<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Prostate%20cancer/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Virtual patients
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Virtual%20cell/" class="md-tabs__link">
        
  
    
  
  Virtual cell

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../RA/" class="md-tabs__link">
        
  
    
  
  RA

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Prostate%20cancer/" class="md-tabs__link">
        
  
    
  
  Prostate cancer

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Virtual patients

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Virtual%20cell/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual cell
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../RA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RA
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Prostate%20cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Prostate cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Virtual patients
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Virtual patients</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-03 06:09:24 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Virtual patients</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Virtual patients</a><br>
      <a href="#recommended_articles">3. Recommended articles on Virtual patients</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Virtual patients</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Virtual patients</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="A quantitative systems toxicology (QST) model for citalopram was established to simulate, in silico, a ‘virtual twin’ of a real patient to predict the occurrence of cardiotoxic events previously reported in patients under various clinical conditions. The QST model considers the effects of citalopram and its most notable electrophysiologically active primary (desmethylcitalopram) and secondary (didesmethylcitalopram) metabolites, on cardiac electrophysiology. The in vitro cardiac ion channel current inhibition data was coupled with the biophysically detailed model of human cardiac electrophysiology to investigate the impact of (i) the inhibition of multiple ion currents (IKr, IKs, ICaL); (ii) the inclusion of metabolites in the QST model; and (iii) unbound or total plasma as the operating drug concentration, in predicting clinically observed QT prolongation. The inclusion of multiple ion channel current inhibition and metabolites in the simulation with unbound plasma citalopram concentration provided the lowest prediction error. The predictive performance of the model was verified with three additional therapeutic and supra-therapeutic drug exposure clinical cases. The results indicate that considering only the hERG ion channel inhibition of only the parent drug is potentially misleading, and the inclusion of active metabolite data and the influence of other ion channel currents should be considered to improve the prediction of potential cardiac toxicity. Mechanistic modelling can help bridge the gaps existing in the quantitative translation from preclinical cardiac safety assessment to clinical toxicology. Moreover, this study shows that the QST models, in combination with appropriate drug and systems parameters, can pave the way towards personalised safety assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/357ea1947a44ce02642c8e40523943dff7116063" target='_blank'>
                Real Patient and its Virtual Twin: Application of Quantitative Systems Toxicology Modelling in the Cardiac Safety Assessment of Citalopram
                </a>
              </td>
          <td>
            N. Patel, B. Wiśniowska, M. Jamei, S. Polak
          </td>
          <td>None</td>
          <td>The AAPS Journal, AAPS Journal</td>
          <td>28</td>
          <td>48</td>

            <td><a href='../recommendations/357ea1947a44ce02642c8e40523943dff7116063' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Virtual clinical trials (VCTs) have gained popularity for their ability to rationalize the drug development process using mathematical and computational modelling, and to provide key insights into the mechanisms regulating patient responses to treatment. In this chapter, we cover approaches for generating virtual cohorts with applications in cancer biology and treatment. VCTs are an effective tool for predicting clinical responses to novel therapeutics and establishing effective treatment strategies. These VCTs allow us to capture inter-individual variability (IIV) which can lead to diversity in patient drug responses. Here we discuss three main methodologies for capturing IIV with a VCT. First, we highlight the use of population pharmacokinetic (PopPK) models, which extrapolate from empirical data population PK parameters that best fits the individual variability seen in drug disposition using non-linear mixed effects models. Next, we show how virtual patients may be sampled from a normal distribution with mean and standard deviation informed from experimental data to estimate parameters in a mechanistic model that regulates drug PKs. Lastly, we show how optimization techniques can be used to calibrate virtual patient parameter values and generate the VCT. Throughout, we compare and contrast these methods to provide a broader view of the generation of virtual patients, and to aid the decision-making process for those looking to leverage virtual clinical trials in their research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93dd39c86854a4c9008387177ccfd2a474105b80" target='_blank'>
                Approaches to generating virtual patient cohorts with applications in oncology
                </a>
              </td>
          <td>
            Anudeep Surendran, Justin Le Sauteur-Robitaille, Dana Kleimeier, J. Gevertz, K. Wilkie, A. Jenner, M. Craig
          </td>
          <td>2022-05-25</td>
          <td>bioRxiv</td>
          <td>8</td>
          <td>18</td>

            <td><a href='../recommendations/93dd39c86854a4c9008387177ccfd2a474105b80' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef39ec91aeb4092e59df228b59ca6261be21ff5" target='_blank'>
                Health digital twins as tools for precision medicine: Considerations for computation, implementation, and regulation
                </a>
              </td>
          <td>
            Kaushik P. Venkatesh, Marium M. Raza, J. Kvedar
          </td>
          <td>2022-09-22</td>
          <td>NPJ Digital Medicine</td>
          <td>97</td>
          <td>46</td>

            <td><a href='../recommendations/9ef39ec91aeb4092e59df228b59ca6261be21ff5' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="A mathematical model of the human response to trauma replicates individual patient outcomes but predicts unexpected results in populations. A virtual large sample size Severe trauma or bleeding evokes an all-hands-on-deck immune response. When properly orchestrated, the myriad cytokines and peptides help to heal the patient. However, this process can easily go awry, and administering the right drug to compensate is a challenge. Brown and colleagues mathematically modeled the complicated immune responses from the data of 33 blunt trauma patients and then generated a larger cohort of 10,000 virtual trauma patients. This large virtual cohort predicted the reactions of smaller validation cohorts, but the surprise was that understanding the response details in a single patient did not predict how the population would act. The author’s virtual clinical trial indicated that inhibition of interleukin-6 (IL-6) produced a small survival benefit, whereas IL-1β inhibition did not help much and tumor necrosis factor–α made things worse. Trauma-induced critical illness is driven by acute inflammation, and elevated systemic interleukin-6 (IL-6) after trauma is a biomarker of adverse outcomes. We constructed a multicompartment, ordinary differential equation model that represents a virtual trauma patient. Individual-specific variants of this model reproduced both systemic inflammation and outcomes of 33 blunt trauma survivors, from which a cohort of 10,000 virtual trauma patients was generated. Model-predicted length of stay in the intensive care unit, degree of multiple organ dysfunction, and IL-6 area under the curve as a function of injury severity were in concordance with the results from a validation cohort of 147 blunt trauma patients. In a subcohort of 98 trauma patients, those with high–IL-6 single-nucleotide polymorphisms (SNPs) exhibited higher plasma IL-6 levels than those with low IL-6 SNPs, matching model predictions. Although IL-6 could drive mortality in individual virtual patients, simulated outcomes in the overall cohort were independent of the propensity to produce IL-6, a prediction verified in the 98-patient subcohort. In silico randomized clinical trials suggested a small survival benefit of IL-6 inhibition, little benefit of IL-1β inhibition, and worse survival after tumor necrosis factor–α inhibition. This study demonstrates the limitations of extrapolating from reductionist mechanisms to outcomes in individuals and populations and demonstrates the use of mechanistic simulation in complex diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfd4fdecd1112b0df0d9bf7248306695434283e2" target='_blank'>
                Trauma in silico: Individual-specific mathematical models and virtual clinical populations
                </a>
              </td>
          <td>
            David Brown, Rami A. Namas, K. Almahmoud, A. Zaaqoq, J. Sarkar, Derek A. Barclay, Jinling Yin, Ali Ghuma, Andrew Abboud, G. Constantine, G. Nieman, R. Zamora, Steven C. Chang, T. Billiar, Y. Vodovotz
          </td>
          <td>2015-04-29</td>
          <td>Science Translational Medicine</td>
          <td>75</td>
          <td>140</td>

            <td><a href='../recommendations/bfd4fdecd1112b0df0d9bf7248306695434283e2' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Purpose We introduce two validated single (SH) and dual hormone (DH) mathematical models that represent an in-silico virtual patient population (VPP) for type 1 diabetes (T1D). The VPP can be used to evaluate automated insulin and glucagon delivery algorithms, so-called artificial pancreas (AP) algorithms that are currently being used to help people with T1D better manage their glucose levels. We present validation results comparing these virtual patients with true clinical patients undergoing AP control and demonstrate that the virtual patients behave similarly to people with T1D. Methods A single hormone virtual patient population (SH-VPP) was created that is comprised of eight differential equations that describe insulin kinetics, insulin dynamics and carbohydrate absorption. The parameters in this model that represent insulin sensitivity were statistically sampled from a normal distribution to create a population of virtual patients with different levels of insulin sensitivity. A dual hormone virtual patient population (DH-VPP) extended this SH-VPP by incorporating additional equations to represent glucagon kinetics and glucagon dynamics. The DH-VPP is comprised of thirteen differential equations and a parameter representing glucagon sensitivity, which was statistically sampled from a normal distribution to create virtual patients with different levels of glucagon sensitivity. We evaluated the SH-VPP and DH-VPP on a clinical data set of 20 people with T1D who participated in a 3.5-day outpatient AP study. Twenty virtual patients were matched with the 20 clinical patients by total daily insulin requirements and body weight. The identical meals given during the AP study were given to the virtual patients and the identical AP control algorithm that was used to control the glucose of the virtual patients was used on the clinical patients. We compared percent time in target range (70–180 mg/dL), time in hypoglycemia (<70 mg/dL) and time in hyperglycemia (>180 mg/dL) for both the virtual patients and the actual patients. Results The subjects in the SH-VPP performed similarly vs. the actual patients (time in range: 78.1 ± 5.1% vs. 74.3 ± 8.1%, p = 0.11; time in hypoglycemia: 3.4 ± 1.3% vs. 2.8 ± 1.7%, p = 0.23). The subjects in the DH-VPP also performed similarly vs. the actual patients (time in range: 75.6 ± 5.5% vs. 71.9 ± 10.9%, p = 0.13; time in hypoglycemia: 0.9 ± 0.8% vs. 1.3 ± 1%, p = 0.19). While the VPPs tended to over-estimate the time in range relative to actual patients, the difference was not statistically significant. Conclusions We have verified that a SH-VPP and a DH-VPP performed comparably with actual patients undergoing AP control using an identical control algorithm. The SH-VPP and DH-VPP may be used as a simulator for pre-evaluation of T1D control algorithms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed50a04c433e06a870d929d0948c3de3cdeaebd" target='_blank'>
                A statistical virtual patient population for the glucoregulatory system in type 1 diabetes with integrated exercise model
                </a>
              </td>
          <td>
            Navid Resalat, J. El Youssef, Nichole S. Tyler, J. Castle, P. Jacobs
          </td>
          <td>2019-07-25</td>
          <td>PLoS ONE</td>
          <td>59</td>
          <td>31</td>

            <td><a href='../recommendations/2ed50a04c433e06a870d929d0948c3de3cdeaebd' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Virtual patients'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Virtual patients</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Background: Salbutamol, a short-acting β2-agonist used in asthma treatment, is available in multiple formulations, including inhalers, nebulizers, oral tablets, and intravenous, intramuscular, and subcutaneous routes. Each formulation exhibits distinct pharmacokinetic (PK) and pharmacodynamic (PD) profiles, influencing therapeutic outcomes and adverse effects. Although asthma management predominantly relies on inhaled salbutamol, understanding how these formulations interact with patient-specific characteristics could improve personalized medicine approaches, potentially uncovering the therapeutic benefits of alternative formulations for an individual patient. Herein, this study aims to analyze how covariates—such as age, weight, gender, body surface area (BSA), cytochrome P450 (CYP) expression, race, and health status—affect the therapeutic regime of orally administered salbutamol using population PK (popPK) modeling. The final model is intended as a tool to support the selection of optimal formulation and dosage regimen based on individual patient profiles. Methods: A dataset of 40 virtual patients derived from a physiologically based PK (PBPK) model of oral salbutamol was included in the popPK model. Results: A two-compartment model with first-order elimination and absorption, with a transit compartment, best described the plasma concentration–time profile following a 4 mg dose. Relationships were identified between gender and mean transit time (Mtt) and clearance (Cl), as well as the effects of weight and BSA on the volume of distribution of the central compartment (V1) and Cl, and a significant impact of health status on Cl. Conclusions: Despite current contraindications for oral salbutamol, our findings suggest that certain individuals, particularly children, may benefit from oral treatment over inhalation. We also identified individual characteristics associated with increased salbutamol toxicity risk, indicating the need for dose adjustment or alternative administration. This study further highlights the potential of popPK modeling in personalized therapy through a fully in silico approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28653842ab9dad79df4265e0faa25cd3ac9fc4ea" target='_blank'>
              Improving Individualized Salbutamol Treatment: A Population Pharmacokinetic Model for Oral Salbutamol in Virtual Patients
              </a>
            </td>
          <td>
            Lara Marques, Nuno Vale
          </td>
          <td>2024-12-30</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Sex differences in cardiac electrophysiology are a crucial factor affecting arrhythmia risk and treatment responses. It is well-documented that females are at a higher risk of drug-induced Torsade de Pointes and sudden cardiac death, largely due to longer QTc intervals compared to males. However, the underrepresentation of females in both basic and clinical research introduces biases that hinder our understanding of sex-specific arrhythmia mechanisms, risk metrics, disease progression, treatment strategies, and outcomes. To address this problem, we developed a quantitative tool that predicts ECG features in females based on data from males (and vice versa) by combining detailed biophysical models of human ventricular excitation-contraction coupling and statistical regression models. We constructed male and female ventricular tissue models incorporating transmural heterogeneity and sex-specific parameterizations and derived pseudo-ECGs from these models. Multivariable lasso regression was employed to generate sets of regression coefficients (a cross-sex translator) that map male ECG features to female ECG features. The predictive ability of the translator was evaluated using an independent dataset that simulates the effects of various drugs and pharmacological agents at different concentrations on male and female models. Furthermore, we demonstrated a proof-of-concept clinical application using ECG data from age-matched subjects of both sexes under various drug regimens. We propose our cross-sex ECG translator as a novel digital health tool that can facilitate sex-specific cardiac safety assessments, ensuring that pharmacotherapy is safe and effective for both sexes, which is a major step forward in addressing disparities in cardiac treatment for females.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25c36cc2d6699af60e0f002f7e1acdd8ebd84695" target='_blank'>
              Development and clinical validation of a cross-sex translator of ECG drug responses
              </a>
            </td>
          <td>
            R. Shetty, S. Morotti, V. Sobota, J. Bayer, H. Ni, E. Grandi
          </td>
          <td>2024-12-29</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background/Objectives: Colorectal cancer (CRC) holds the third and second position among cancers affecting men and women, respectively. Frequently, the first-line treatment for metastatic CRC consists of the intravenous administration of 5-fluorouracil and leucovorin in combination with oxaliplatin or irinotecan. Physiologically-based pharmacokinetic models (PBPK) aim to mechanistically incorporate body physiology and drug physicochemical attributes, enabling the description of both systemic and organ drug exposure based on the treatment specificities. This bottom-up approach represents an opportunity to personalize treatment and minimize the therapeutic risk/benefit ratio through the understanding of drug distribution within colorectal tissue. This project has the goal of characterizing the systemic and tissue exposure of four anti-cancer drugs in humans using a PBPK platform fed with data from the literature. Methods: A literature search was performed to collect clinical data on systemic concentration versus time profiles. Physicochemical features were obtained from the literature, as well as parameters associated with distribution, metabolism, and excretion. The PBPK models were built using PK-Sim®. Results: The data from 51 clinical studies were extracted and combined in one single dataset. The PBPK models successfully described the exposure vs. time profiles with respect to both, with both the typical tendency and dispersion shown by the data. The percentage of observations falling within the two-fold error bounds ranged between 94 and 100%. The colon/plasma AUCinf ratios were similar for 5-FU, oxaliplatin, and leucovorin, but it was significantly higher for irinotecan. Conclusions: The PBPK models support tailored treatment approaches by linking in vitro studies to organ exposure. These models serve as the initial step towards incorporating a dedicated tumor compartment, which will further account for the variability in tumor microenvironment characteristics to improve therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c47bd1801f751033fb61994389dcf83cf82aae7" target='_blank'>
              Development and Application of Physiologically-Based Pharmacokinetic Model to Predict Systemic and Organ Exposure of Colorectal Cancer Drugs
              </a>
            </td>
          <td>
            Sara Peribañez-Dominguez, Zinnia P. Parra-Guillen, I. Trocóniz
          </td>
          <td>2025-01-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0eda19b94dfcd39089ae0957c52f1ac7d92dc5b2" target='_blank'>
              Simulation of clinical trials of oral treprostinil in pulmonary arterial hypertension using a virtual population
              </a>
            </td>
          <td>
            Andrew E. Stine, Jignesh Parmar, Amy K Smith, Zachary Cummins, N. R. Pillalamarri, R. J. Bender
          </td>
          <td>2025-01-15</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Background Remdesivir (RDV) is an intravenous nucleotide prodrug approved for COVID-19 treatment. Its effect on QT interval is unknown. This Phase 1 study evaluated the effects of a supratherapeutic dose of RDV on QT interval.Figure 1. Estimated ΔΔQTcF (90% CI) between RDV and Placebo-to-match RDV. ΔΔQTcF, baseline-adjusted, placebo-matched QTcF; QTcF, QT interval corrected for heart rate using the Fridericia formula; RDV, remdesivir. Methods This study included 2 cohorts of healthy participants aged 18-55 years: an RDV dose-selection cohort (Sentinel Cohort; n=6); and a randomized, placebo- and positive-controlled (moxifloxacin), single-dose crossover cohort (Thorough QT [TQT] Cohort; n=55). The Sentinel Cohort assessed pharmacokinetics (PK) and safety of a supratherapeutic RDV dose (600 mg) and was followed by the TQT Cohort. The primary endpoint for the TQT Cohort was ΔΔQTcF: the difference between baseline-adjusted QTcF (QT corrected for heart rate [HR] using the Fridericia formula) in the RDV treatment period and baseline-adjusted QTcF in the placebo period at each postdose time point. Adverse events (AEs), laboratory abnormalities, and plasma PK were also assessed.Table 1. Plasma PK Parameters for RDV and Its Metabolites (GS-704277 and GS 441524) Following a Single Dose of RDV 600 mg. AUCinf, area under the concentration-time curve extrapolated to infinite time; AUClast, area under the concentration-time curve from dosing to last measurable concentration; Cmax, maximum observed concentration; CV, coefficient of variance; PK, pharmacokinetic; RDV, remdesivir; t½, half-life; Tmax, time at which maximum concentration was observed; TQT, thorough QT. aThe TQT Cohort enrolled 55 participants; 54 participants received RDV. Results Following RDV 600 mg, PK exposures of RDV and its metabolites were similar in both cohorts (Table 1). In the TQT Cohort, the assay sensitivity was established; the estimated lower bound of 96.67% CI for ΔΔQTcF for moxifloxacin was >5 msec for ≥1 of the prespecified 1, 2, and 3h postdose time points. RDV had no clinically relevant effects on QT interval compared to placebo (Figure 1); the estimated upper bound of 90% CI for RDV ΔΔQTcF was < 10 msec at all postdose time points. Most AEs were Grade 1-2 in severity. One participant receiving RDV in the TQT Cohort had a Grade 3 AE (transaminase increased). There were no other Grade ≥3 AEs, tachycardia, or HR AEs (Table 2). 5/54 participants in the TQT Cohort had HR >100 bpm within 12h of receiving RDV (all resolved by 24h postdose). 3 of the 5 participants had Grade 1-2 nausea and vomiting ≤13h postdose; 1 of the 3 participants had treatment-related chills ≤7h postdose.Table 2. TEAEs and Grade ≥3 TELAs TEAE, treatment-emergent adverse event; TELA, treatment-emergent laboratory abnormality; RDV, remdesivir; TQT, thorough QT. Conclusion RDV 600 mg in healthy participants resulted in PK exposures 3-4–fold higher than those observed with RDV 200 mg in prior studies. A single dose of RDV 600 mg was generally safe; there were no AEs associated with QT prolongation, deaths, or SAEs reported. In this positive control–validated study, ΔΔQTcF for RDV was below the regulatory threshold of concern, demonstrating that its clinical regimen has no proarrhythmic potential. Disclosures Olena Anoshchenko, PhD, Gilead Sciences, Inc.: Employee|Gilead Sciences, Inc.: Stocks/Bonds (Public Company) Mazin Abdelghany, MD, Gilead Sciences, Inc.: Employee|Gilead Sciences, Inc.: Stocks/Bonds (Public Company) Robert H. Hyland, DPhil, Gilead Sciences, Inc.: Employee|Gilead Sciences, Inc.: Stocks/Bonds (Public Company) Santosh Davies, MD, Gilead Sciences, Inc.: Employee|Gilead Sciences, Inc.: Stocks/Bonds (Public Company) Anna Kwan, BS, Gilead Sciences, Inc.: Employee|Gilead Sciences, Inc.: Stocks/Bonds (Public Company) Ran Duan, PhD, Gilead Sciences, Inc.: Employee|Gilead Sciences, Inc.: Stocks/Bonds (Public Company) Aryun Kim, PharmD, Gilead Sciences, Inc.: Employee|Gilead Sciences, Inc.: Stocks/Bonds (Public Company)">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a500e3d73dd553cd3273886c7b21331af95d5cb" target='_blank'>
              P-2031. Thorough QT/QTc Clinical Study to Evaluate the Effect of Remdesivir on Cardiac Repolarization in Healthy Participants
              </a>
            </td>
          <td>
            Olena Anoshchenko, Mazin Abdelghany, Robert H Hyland, Santosh Davies, Anna Kwan, Ran Duan, Aryun Kim
          </td>
          <td>2025-01-29</td>
          <td>Open Forum Infectious Diseases</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Objectives To assess the pharmacokinetics and pharmacodynamics of imipenem in a retrospective cohort of hospitalized Chinese older patients. Methods A population pharmacokinetic (PPK) model was constructed utilizing a nonlinear mixed-effects modeling approach. The final model underwent evaluation through bootstrap resampling and visual predictive checks. Additionally, a population pharmacokinetic and pharmacodynamic analysis was conducted employing Monte Carlo simulations to investigate the impact of commonly used dosing regimens (0.25 g every 6 h, 0.5 g every 6 h, 0.5 g every 8 h, 1 g every 6 h, 1 g every 8 h, and 1 g every 12 h) on the likelihood of achieving the target therapeutic outcomes. Results A total of 370 observations available from 142 patients were incorporated in the PPK model. A two-compartment PPK model with linear elimination best predicted the imipenem plasma concentrations, with the creatinine clearance as a significant covariate of clearance. Typical estimates for clearance, inter-compartmental clearance, central and peripheral volume were 13.1 L·h−1, 11.9 L·h−1, 11.7 L, 29.3 L, respectively. Conclusion The pharmacokinetics of imipenem in elderly patients were effectively characterized by the established PPK model, which includes creatinine clearance as a key covariate. This research will enhance our understanding of imipenem elimination and support precision dosing in this patient demographic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47628ead3bd204ff68a69f873bcb56f935e133c1" target='_blank'>
              Population pharmacokinetics and dosing optimization of imipenem in Chinese elderly patients
              </a>
            </td>
          <td>
            Jing Wang, Qiu Fang, Xuemei Luo, Lu Jin, Huaijun Zhu
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16434184a7c396255e1e520823e433a1333c2f54" target='_blank'>
              A Pooled Pharmacokinetic Analysis for Piperacillin/Tazobactam Across Different Patient Populations: From Premature Infants to the Elderly
              </a>
            </td>
          <td>
            Daming Kong, Jason A Roberts, J. Lipman, F. Taccone, M. Cohen-Wolkowiez, F. Sime, Danny Tsai, Pieter De Cock, S. Jaruratanasirikul, S. Dhaese, Andrew A. Udy, Timothy W. Felton, R. Michelet, Céline Thibault, Jeroen V. Koomen, D. Eleveld, M. Struys, Jan J De Waele, Pieter J. Colin
          </td>
          <td>2024-12-25</td>
          <td>Clinical Pharmacokinetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/934ce70965d6e11160dca268c540c46e605b2c8e" target='_blank'>
              Fast and accurate prediction of drug induced proarrhythmic risk with sex specific cardiac emulators
              </a>
            </td>
          <td>
            P. Dominguez-Gomez, Alberto Zingaro, Laura Baldo-Canut, Caterina Balzotti, Borje Darpo, Christopher Morton, Mariano Vázquez, Jazmin Aguado
          </td>
          <td>2024-12-26</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background/Objectives: Vedolizumab (VDZ) is approved in the treatment of patients with moderate to severe ulcerative colitis (UC) or Crohn’s disease (CD). VDZ exhibits considerable variability in its pharmacokinetic (PK) profile, and its exposure-response relationship is not yet fully understood. The aim was to investigate the variability in VDZ trough levels and PK parameters, to assess the relationship between VDZ PK and biochemical response, as well as clinical and endoscopic outcomes. Methods: We included 61 UC and 45 CD patients. Patients’ data and trough VDZ concentrations were retrospectively obtained. Population PK analysis was performed using non-linear mixed-effects modelling with NONMEM (version 7.5). Graphs and statistical analyses were performed using R (version 4.1.3). Results: In total, 116 trough VDZ concentrations from 106 patients were described by a two-compartment model. For a typical patient, clearance (CL) was estimated at 0.159 L/day, while in patients previously treated with anti-TNFα agents, VDZ CL increased by 26.4% on average. In univariate binary logistic regression, VDZ trough concentration was not statistically significant predictor of remission, whereas CL was. Moreover, combined CL and faecal calprotectin (FCP) were a statistically significant predictors of remission. The hazard ratio (HR) for CL above 0.1886 L/day was 0.35 (p = 0.05) and for FCP below 250 µg/g was 2.66 (p = 0.02) in a time-to-event analysis. Conclusions: Our population PK model incorporates the effect of prior anti-TNFα agents on CL, suggesting its association with more severe forms of IBD. VDZ CL emerged as a more robust and clinically relevant predictor of remission in IBD patients than trough concentration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b3f763285f8d609f76a57110a4b866db845438f" target='_blank'>
              Vedolizumab Clearance as a Surrogate Marker for Remission in Inflammatory Bowel Disease Patients: Insights from Real-World Pharmacokinetics
              </a>
            </td>
          <td>
            S. Marković, Đ. Kralj, P. Svorcan, T. Knežević Ivanovski, O. Odanović, Sanja Obradović, A. Homšek, Marija Jovanović, Rada Savic, K. Vučićević
          </td>
          <td>2024-12-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Minimizing harm is a cornerstone of ethical research practices. A drug that has undergone extensive clinical pharmacological testing in healthy participants (HPs) and a diverse selection of patients can be described with a sufficiently predictive population pharmacokinetic (PopPK) model. In impaired clearance trials, recruitment is minimized and underpowered for all but major exposure differences. Virtual HP arms have been reported to support similar conclusions to conventional impaired clearance studies, and further minimize potential harm of drug exposure without medical benefit by eliminating an arm of the study. However, the extent to which the conventional analysis of impairment studies compare to the simulation approach is unknown. Here we assess the operating characteristics of the virtual cohort approach along with the conventional approach through controlled simulations. These simulations included a simple, widely accessible PopPK model and several internal models that have been used in a previous meta-analysis of the virtual cohort approach. In the pairwise comparisons assessed, the virtual cohort simulation approach had greater power per sample size than the conventional approach and the same power under the null hypothesis. Given key methodological differences, it is recommended that the simulation and conventional approaches be treated as having approximately the same power under equivalent conditions. These results provide a strong justification for the use of the virtual cohort approach when an adequate PopPK model is available, minimizing unnecessary exposure to study drugs that will not benefit healthy study participants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a792791d41701cb013d930fd9010e65799e5e43" target='_blank'>
              Operating Characteristics of the Simulated Healthy Participant Approach in Impaired Clearance Studies.
              </a>
            </td>
          <td>
            Sana Gupta, Vivek Purohit, Yuchen Wang, John P Prybylski
          </td>
          <td>2025-01-22</td>
          <td>The AAPS journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="To employ a reduced-order cardiovascular model as a digital twin for personalised medicine, it is essential to understand how uncertainties in the model’s input parameters affect its outputs. The aim is to identify a set of input parameters that can serve as clinical biomarkers, providing insight into a patient’s physiological state. Given the challenge of finding useful clinical data, careful consideration must be given to the experimental design used to acquire patient-specific input parameters. In this paper, we conduct the first quantification of a cardiovascular system’s sloppiness to elucidate the structure of the input parameter space. By utilising Sobol indices and examining various synthetic cardiovascular measures with increasing invasiveness, we uncover how the personalisation process and the cardiovascular system’s sloppiness are contingent upon the chosen experimental design. Our findings reveal that continuous clinical measures induce system sloppiness and increase the number of personalisable biomarkers, whereas discrete clinical measurements produce a non-sloppy system with a reduced number of biomarkers. This study underscores the necessity for careful consideration of available clinical data as differing measurement sets can significantly impact model personalisation. Author Summary In personalised medicine, computational models that replicate physical systems — are becoming vital tools for understanding and predicting individual health. Our study explores cardiovascular models, which simulate heart and circulatory functions from which clinical metrics may be derived. These models aim to provide personalised insights into heart health and treatment planning. A key challenge in building these models is addressing “sloppiness,” a property which provides vital insight into the response surface structure for which one calibrates a model searching for a global minimum point, a position in parameter space which best represents a patients cardiovascular health. In order to personalise a model different types of clinical metrics must be available for a model response to be compared to. We examined how different types of clinical data — ranging from simple discrete blood pressure readings to detailed invasive continuous waveform data — impact model sloppiness and the number of personalisable biomarkers. Our results show that continuous measurements increase the number of personalisable biomarkers but make the personalisation process more complex through increased sloppiness. In contrast, simpler discrete measurements reduce model sloppiness simplifying the personalisation process but yield fewer personalisable biomarkers. By analysing the impact of experimental designs on the personalisation process, our work offers practical insights into improving the reliability of cardiovascular digital twins, supporting their adoption in personalised medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad51bcdf15d70eb358e8f1dceeb4de62915ebed9" target='_blank'>
              The Impact of Experimental Designs & System Sloppiness on the Personalisation Process: A Cardiovascular Perspective
              </a>
            </td>
          <td>
            Harry Saxton, Daniel J. Taylor, Grace Faulkner, Ian Halliday, Thomas Newman, Torsten Schenkel, Paul D. Morris, Richard H. Clayton, Xu Xu
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Objective - To establish an optimized sotalol dosing strategy for fetal tachycardia by using a pregnancy computational model for dose simulations. Methods - A physiologically-based computational model, including pregnancy-related changes and placental transfer values, was established and verified. Simulations of the current dosing advises and prospective dosing scenarios were performed. To avoid maternal dose-related toxicity (QT-prolongation) we aimed for maternal concentrations <2.5 mg/L. Based on neonatal concentration-effect data, we aimed for a fetal Ctrough concentrations of 0.4 - 1.0 mg/L. Results - The pregnancy physiologically-based pharmacokinetic model accurately predicted maternal and fetal exposures. Predictions indicate that almost 16% of maternal plasma concentrations exceed the toxic level of 2.5 mg/L at the maximum oral daily dose of 480 milligram, while 90% of fetuses have a Ctrough concentration within the therapeutic window. When lowering the maximum daily dose to 400 mg, 0.1% of maternal plasma concentrations exceed 2.5 mg/L, while 87% of the fetal plasma concentrations remain in the therapeutic window. Additionally dosing 480 mg in three times daily reduces the risk of high maternal plasma exposure to 0.3%, while maintaining effective fetal Ctrough concentrations in 95% of fetuses. Conclusion - Pregnancy computational modeling can be used to adequately predict maternal and fetal sotalol exposures. Our simulations suggest that daily doses should not exceed 400 milligram and that dividing the oral daily dose over three doses improves the balance between high maternal plasma exposure and effective fetal concentrations. Funding - This publication is based on research funded by the Bill & Melinda Gates Foundation (INV-023795).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/846fe908319230c98c9fc2c13a95eca73be20c27" target='_blank'>
              Sotalol dose optimization for fetal tachycardia: a pregnancy physiologically based pharmacokinetic model study.
              </a>
            </td>
          <td>
            H. van Hove, J. van der Heijden, A. van Uden, V. Gijsen, R. Greupink, S. de Wildt, J. van Drongelen
          </td>
          <td>2024-12-20</td>
          <td>None</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT The pharmaceutical industry constantly strives to improve drug development processes to reduce costs, increase efficiencies, and enhance therapeutic outcomes for patients. Model‐Informed Drug Development (MIDD) uses mathematical models to simulate intricate processes involved in drug absorption, distribution, metabolism, and excretion, as well as pharmacokinetics and pharmacodynamics. Artificial intelligence (AI), encompassing techniques such as machine learning, deep learning, and Generative AI, offers powerful tools and algorithms to efficiently identify meaningful patterns, correlations, and drug–target interactions from big data, enabling more accurate predictions and novel hypothesis generation. The union of MIDD with AI enables pharmaceutical researchers to optimize drug candidate selection, dosage regimens, and treatment strategies through virtual trials to help derisk drug candidates. However, several challenges, including the availability of relevant, labeled, high‐quality datasets, data privacy concerns, model interpretability, and algorithmic bias, must be carefully managed. Standardization of model architectures, data formats, and validation processes is imperative to ensure reliable and reproducible results. Moreover, regulatory agencies have recognized the need to adapt their guidelines to evaluate recommendations from AI‐enhanced MIDD methods. In conclusion, integrating model‐driven drug development with AI offers a transformative paradigm for pharmaceutical innovation. By integrating the predictive power of computational models and the data‐driven insights of AI, the synergy between these approaches has the potential to accelerate drug discovery, optimize treatment strategies, and usher in a new era of personalized medicine, benefiting patients, researchers, and the pharmaceutical industry as a whole.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4c0840bc02532ce5e02fd634295b18c901f94d" target='_blank'>
              Integrating Model‐Informed Drug Development With AI: A Synergistic Approach to Accelerating Pharmaceutical Innovation
              </a>
            </td>
          <td>
            Karthik Raman, Rukmini Kumar, C. Musante, Subha Madhavan
          </td>
          <td>2025-01-01</td>
          <td>Clinical and Translational Science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1c49f7fb420c30b28dbda2831ccc402f2a211e6" target='_blank'>
              Machine learning-based prediction of pharmacokinetic parameters for individualized drug dosage optimization
              </a>
            </td>
          <td>
            Sinenhlanhla Mtshali, Byron A. Jacobs
          </td>
          <td>2024-12-15</td>
          <td>International Journal of Information Technology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Aims Few personalized monitoring models for valproic acid (VPA) in pediatric epilepsy patients (PEPs) incorporate machine learning (ML) algorithms. This study aimed to develop an ensemble ML model for VPA monitoring to enhance clinical precision of VPA usage. Methods A dataset comprising 366 VPA trough concentrations from 252 PEPs, along with 19 covariates and the target variable (VPA trough concentration), was refined by Spearman correlation and multicollinearity testing (366 × 11). The dataset was split into a training set (292) and testing set (74) at a ratio of 8:2. An ensemble model was formulated by Gradient Boosting Regression Trees (GBRT), Random Forest Regression (RFR), and Support Vector Regression (SVR), and assessed by SHapley Additive exPlanations (SHAP) analysis for covariate importance. The model was optimized for R2, relative accuracy, and absolute accuracy, and validated against two independent external datasets (32 in-hospital and 28 out-of-hospital dataset). Results Using the R2 weight ratio of GBRT, RFR and SVR optimized at 5:2:3, the ensemble model demonstrated superior performance in terms of relative accuracy (87.8%), absolute accuracy (78.4%), and R2 (0.50), while also exhibiting a lower Mean Absolute Error (9.87) and Root Mean Squared Error (12.24), as validated by the external datasets. Platelet count (PLT) and VPA daily dose were identified as pivotal covariates. Conclusion The proposed ensemble model effectively monitors VPA trough concentrations in PEPs. By integrating covariates across various ML algorithms, it delivers results closely aligned with clinical practice, offering substantial clinical value for the guided use of VPA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b9cffac23bff7d0cc53ee324115100297e0186" target='_blank'>
              Monitoring of the trough concentration of valproic acid in pediatric epilepsy patients: a machine learning-based ensemble model
              </a>
            </td>
          <td>
            Yue-Wen Chen, Xi-Kai Lin, Si Chen, Ya-Lan Zhang, Wei Wu, Chen Huang, Xin Rao, Zong-Xing Lu, Zhou-Jie Liu
          </td>
          <td>2024-12-18</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="One of the goals of personalized medicine is to tailor diagnostics to individual patients. Diagnostics are performed in practice by measuring quantities, called biomarkers, that indicate the existence and progress of a disease. In common cardiovascular diseases, such as hypertension, biomarkers that are closely related to the clinical representation of a patient can be predicted using computational models. Personalizing computational models translates to considering patient-specific flow conditions, for example, the compliance of blood vessels that cannot be a priori known and quantities such as the patient geometry that can be measured using imaging. Therefore, a patient is identified by a set of measurable and nonmeasurable parameters needed to well-define a computational model; else, the computational model is not personalized, meaning it is prone to large prediction errors. Therefore, to personalize a computational model, sufficient information needs to be extracted from the data. The current methods by which this is done are either inefficient, due to relying on slow-converging optimization methods, or hard to interpret, due to using `black box` deep-learning algorithms. We propose a personalized diagnostic procedure based on a differentiable 0D-1D Navier-Stokes reduced order model solver and fast parameter inference methods that take advantage of gradients through the solver. By providing a faster method for performing parameter inference and sensitivity analysis through differentiability while maintaining the interpretability of well-understood mathematical models and numerical methods, the best of both worlds is combined. The performance of the proposed solver is validated against a well-established process on different geometries, and different parameter inference processes are successfully performed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33e49c22a3d7bc2453830728348a2c5a05bb2115" target='_blank'>
              Accelerated Patient-Specific Calibration via Differentiable Hemodynamics Simulations
              </a>
            </td>
          <td>
            Diego Renner, Georgios Kissas
          </td>
          <td>2024-12-19</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT
Different polymorphisms in genes encoding metabolizing enzymes and drug transporters have been associated with tacrolimus pharmacokinetics. In particular, studies on CYP3A4 and CYP3A5, and their combined cluster have demonstrated their significance in adjusting tacrolimus dosing to minimize under- and overexposure thereby increasing the proportion of patients who achieve tacrolimus therapeutic target. Many factors influence the pharmacokinetics of tacrolimus, contributing to inter-patient variability affecting individual dosing requirements. On the other hand, the growing use of population pharmacokinetic models in solid organ transplantation, including different tacrolimus formulations, has facilitated the integration of pharmacogenetic data and other variables into algorithms to easier implement the personalized dose adjustment in transplant centers. The future of personalized medicine in transplantation lies in implementing these models in clinical practice, with pharmacogenetics as a key factor to account for the high inter-patient variability in tacrolimus exposure. To date, three clinical trials have validated the clinical application of these approaches. The aim of this review is to provide an overview of the current studies regarding the different population pharmacokinetic including pharmacogenetics and those translated to the clinical practice for individualizing tacrolimus dose adjustment in kidney transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/917aa5ac878fa10b13074d397eeb400bec68e0a0" target='_blank'>
              Customizing Tacrolimus Dosing in Kidney Transplantation: Focus on Pharmacogenetics.
              </a>
            </td>
          <td>
            Nuria Lloberas, A. Vidal-Alabró, Helena Colom
          </td>
          <td>2024-12-10</td>
          <td>Therapeutic drug monitoring</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b8ce454573eb115134bdf921ad1f9f130c939a8" target='_blank'>
              A PopPBPK-RL approach for precision dosing of benazepril in renal impaired patients.
              </a>
            </td>
          <td>
            Guillermo Vigueras, Lucía Muñoz-Gil, Valerie Reinisch, Joana T Pinto
          </td>
          <td>2024-12-11</td>
          <td>Journal of pharmacokinetics and pharmacodynamics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In this article, we aim to explore the rapidly developing role of artificial intelligence (AI) in cardiac metabolism research, highlighting its impact on biomarker discovery, precision medicine, and patient stratification. Cardiac metabolism, a key determinant of cardiovascular health, is often disrupted in cardiovascular diseases (CVDs) like heart failure and coronary artery disease. AI’s ability to process and analyze large-scale data offers new chances for understanding and addressing these metabolic dysfunctions. By integrating up-to-date technologies with molecular and clinical insights, AI enables the achievement of personalized treatments, more accurate diagnostics, and the discovery of potential novel therapeutic targets. The main challenges include ethical concerns around data privacy, algorithmic bias, and the need for representative datasets. Future directions focus on developing transparent, accountable, and collaborative AI models that integrate data and enable real-time monitoring, ensuring fairness and accessibility in healthcare. As AI continues to evolve, its role in advancing cardiovascular care is expected to grow, offering new trends in cardiovascular research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/269207580bd167b11f28a5846cb43db813c95f01" target='_blank'>
              Artificial intelligence in cardiac metabolism: the next frontier in cardiovascular health
              </a>
            </td>
          <td>
            , Yuhui Zhang, Jian Zhang
          </td>
          <td>2025-01-07</td>
          <td>Metabolism and Target Organ Damage</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Background Zoliflodacin is a first-in-class oral bacterial gyrase inhibitor being developed as single dose treatment for uncomplicated gonorrhea. Population pharmacokinetic (PPK) analyses using data from 6 Phase 1 studies and the pivotal Phase 3 trial and assessments of probabilities of pharmacokinetic-pharmacodynamic (PK-PD) target attainment (PTA) were conducted to support zoliflodacin dose selection. Methods PPK analyses were performed using plasma PK data from 261 participants, including 24 patients from the PK substudy of the pivotal Phase 3 trial. The final, qualified PPK model was used with non-clinical PK-PD and MIC data to perform PTA analyses among fed and fasted simulated patients with Phase 1 and 3 bioavailability estimates. The PK-PD index associated with efficacy is the ratio of free-drug plasma area under the curve from time zero to infinity (AUC0-∞) to MIC, with a target of 70.6 associated with suppression of Neisseria gonorrhoeae mutant amplification [Jacobsson et al., Front Pharmacol. 2021;12:682135]. PTA by MIC and averaged over the MIC distributions for Phase 3 and in vitro surveillance data was assessed among simulated patients. Results A final PPK model with an oral depot compartment and associated relative bioavailability, a transit chain compartment to describe the absorption profile, and one systemic compartment with linear elimination best described the data. Body weight, pre-dose meal type, study phase, sex, formulation type or manufacturer, and concomitant administration of a strong CYP3A4 inhibitor were statistically significant predictors of inter-individual variability in zoliflodacin PK. Age and race were not identified as statistically significant covariates. PTA was ≥ 96.2% at a MIC ≤ 0.25 µg/mL (at which 100 and 99.6% of in vitro surveillance and Phase 3 isolates were inhibited) in simulated patients under fed and fasted conditions, and by bioavailability estimate/study phase (Figure 1). Averaged over either MIC distribution, PTA was ≥ 99.5% under all conditions assessed. Conclusion A PPK model utilizing Phase 1 and 3 data described the zoliflodacin PK well. The high PTA (≥ 96.2 %) at a MIC ≤ 0.25 µg/mL and over the MIC distributions supported a 3 g single oral dose of zoliflodacin for the treatment of uncomplicated gonorrhea. Disclosures Sujata M. Bhavnani, PharmD; MS; FIDSA, Achaogen Inc.: Grant/Research Support|Adagio Therapeutics, Inc.: Grant/Research Support|AiCuris Anti-infective Cures AG: Grant/Research Support|Albany Medical College: Grant/Research Support|AN2 Therapeutics: Grant/Research Support|Antabio SAS: Grant/Research Support|Apogee Biologics, Inc.: Grant/Research Support|Arcutis Biotherapeutics, Inc.: Grant/Research Support|B. Braun Medical Inc.: Grant/Research Support|Basilea Pharmaceutica: Grant/Research Support|BioFire Diagnostics, LLC.: Grant/Research Support|Cidara Therapeutics Inc.: Grant/Research Support|Cipla USA: Grant/Research Support|Cumberland Pharmaceuticals Inc.: Grant/Research Support|Entasis Therapeutics Inc., an affiliate of Innoviva Specialty Therapeutics, Inc.: Grant/Research Support|Excalibur Pharmaceuticals Inc.: Grant/Research Support|Fedora Pharmaceuticals: Grant/Research Support|Genetech: Grant/Research Support|GlaxoSmithKline: Advisor/Consultant|GlaxoSmithKline: Grant/Research Support|Global Antibiotic Research and Development Partnership: Grant/Research Support|Hoffmann-La Roche: Grant/Research Support|Inoterm: Grant/Research Support|Insmed Inc.: Grant/Research Support|Institute for Clinical Pharmacodynamics, Inc.: Ownership Interest|Iterum Therapeutics Limited: Grant/Research Support|Kaizen Bioscience: Grant/Research Support|Lassen Therapeutics Inc.: Grant/Research Support|Matinas Biopharma: Grant/Research Support|Meiji Seika Pharma Co., Ltd.: Grant/Research Support|Melinta Therapeutics: Grant/Research Support|Mutabilis: Grant/Research Support|Nabriva Therapeutics AG: Grant/Research Support|Novobiotic Pharmaceuticals LLC.: Grant/Research Support|Paratek Pharmaceuticals, Inc.: Grant/Research Support|Pfizer Inc.: Grant/Research Support|Praxis Precision Medicines, Inc.: Grant/Research Support|PTC Therapeutics: Grant/Research Support|PureTech LYT 100 Inc.: Grant/Research Support|Qpex Biopharma: Grant/Research Support|Renibus Therapeutics: Grant/Research Support|Sfunga Therapeutics: Grant/Research Support|Shionogi Inc.: Advisor/Consultant|Shionogi Inc.: Grant/Research Support|Spero Therapeutics: Grant/Research Support|Spruce Biosciences Inc.: Grant/Research Support|Suzhou Sinovent Pharmaceuticals Co.: Grant/Research Support|Theravance: Grant/Research Support|University of Wisconsin: Grant/Research Support|US Food and Drug Administration: Grant/Research Support|UT Southwestern: Grant/Research Support|ValanBio Therapeutics, Inc.: Grant/Research Support|VenatoRx: Grant/Research Support|Zogenix International: Grant/Research Support Anthony P. Cammarata, M.S., Achaogen Inc.: Grant/Research Support|Adagio Therapeutics, Inc.: Grant/Research Support|AiCuris Anti-infective Cures AG: Grant/Research Support|Albany Medical College: Grant/Research Support|AN2 Therapeutics: Grant/Research Support|Antabio SAS: Grant/Research Support|Apogee Biologics, Inc.: Grant/Research Support|Arcutis Biotherapeutics, Inc.: Grant/Research Support|B. Braun Medical Inc.: Grant/Research Support|Basilea Pharmaceutica: Grant/Research Support|BioFire Diagnostics, LLC.: Grant/Research Support|Cidara Therapeutics Inc.: Grant/Research Support|Cipla USA: Grant/Research Support|Cumberland Pharmaceuticals Inc.: Grant/Research Support|Entasis Therapeutics Inc., an affiliate of Innoviva Specialty Therapeutics, Inc.: Grant/Research Support|Excalibur Pharmaceuticals Inc.: Grant/Research Support|Fedora Pharmaceuticals: Grant/Research Support|Genetech: Grant/Research Support|GlaxoSmithKline: Grant/Research Support|Global Antibiotic Research and Development Partnership: Grant/Research Support|Hoffmann-La Roche: Grant/Research Support|Inotrem: Grant/Research Support|Insmed Inc.: Grant/Research Support|Institute for Clinical Pharmacodynamics, Inc.: Employee|Iterum Therapeutics Limited: Grant/Research Support|Kaizen Bioscience: Grant/Research Support|Lassen Therapeutics Inc.: Grant/Research Support|Matinas Biopharma: Grant/Research Support|Meiji Seika Pharma Co., Ltd.: Grant/Research Support|Melinta Therapeutics: Grant/Research Support|Mutabilis: Grant/Research Support|Nabriva Therapeutics AG: Grant/Research Support|Novobiotic Pharmaceuticals LLC.: Grant/Research Support|Paratek Pharmaceuticals, Inc.: Grant/Research Support|Pfizer Inc.: Grant/Research Support|Praxis Precision Medicines, Inc.: Grant/Research Support|PTC Therapeutics: Grant/Research Support|PureTech LYT 100 Inc.: Grant/Research Support|Qpex Biopharma: Grant/Research Support|Renibus Therapeutics: Grant/Research Support|Sfunga Therapeutics: Grant/Research Support|Shionogi Inc.: Grant/Research Support|Spero Therapeutics: Grant/Research Support|Spruce Biosciences Inc.: Grant/Research Support|Suzhou Sinovent Pharmaceuticals Co.: Grant/Research Support|Theravance: Grant/Research Support|University of Wisconsin: Grant/Research Support|US Food and Drug Administration: Grant/Research Support|UT Southwestern: Grant/Research Support|ValanBio Therapeutics, Inc.: Grant/Research Support|VenatoRx: Grant/Research Support|Zogenix International: Grant/Research Support Jeffrey P. Hammel, MS, Achaogen Inc.: Grant/Research Support|Adagio Therapeutics, Inc.: Grant/Research Support|AiCuris Anti-infective Cures AG: Grant/Research Support|Albany Medical College: Grant/Research Support|AN2 Therapeutics: Grant/Research Support|Antabio SAS: Grant/Research Support|Apogee Biologics, Inc.: Grant/Research Support|Arcutis Biotherapeutics, Inc.: Grant/Research Support|B. Braun Medical Inc.: Grant/Research Support|Basilea Pharmaceutica: Grant/Research Support|BioFire Diagnostics, LLC.: Grant/Research Support|Cidara Therapeutics Inc.: Grant/Research Support|Cipla USA: Grant/Research Support|Cumberland Pharmaceuticals Inc.: Grant/Research Support|Entasis Therapeutics Inc., an affiliate of Innoviva Specialty Therapeutics, Inc.: Grant/Research Support|Excalibur Pharmaceuticals Inc.: Grant/Research Support|Fedora Pharmaceuticals: Grant/Research Support|Genetech: Grant/Research Support|GlaxoSmithKline: Grant/Research Support|Global Antibiotic Research and Development Partnership: Grant/Research Support|Hoffmann-La Roche: Grant/Research Support|Inotrem: Grant/Research Support|Insmed Inc.: Grant/Research Support|Institute for Clinical Pharmacodynamics, Inc.: Employee|Iterum Therapeutics Limited: Grant/Research Support|Kaizen Bioscience: Grant/Research Support|Lassen Therapeutics Inc.: Grant/Research Support|Matinas Biopharma: Grant/Research Support|Meiji Seika Pharma Co., Ltd.: Grant/Research Support|Melinta Therapeutics: Grant/Research Support|Mutabilis: Grant/Research Support|Nabriva Therapeutics AG: Grant/Research Support|Novobiotic Pharmaceuticals LLC.: Grant/Research Support|Paratek Pharmaceuticals, Inc.: Grant/Research Support|Pfizer Inc.: Grant/Research Support|Praxis Precision Medicines, Inc.: Grant/Research Support|PTC Therapeutics: Grant/Research Support|PureTech LYT 100 Inc.: Grant/Research Support|Qpex Biopharma: Grant/Research Support|Renibus Therapeutics: Grant/Research Support|Sfunga Therapeutics: Grant/Research Support|Shionogi Inc.: Grant/Research Support|Spero Therapeutics: Grant/Research Support|Spruce Biosciences Inc.: Grant/Research Support|Suzhou Sinovent Pharmaceuticals Co.: Grant/Research Support|Theravance: Grant/Research Support|University of Wisconsin: Grant/Research Support|US Food and Drug Administration: Grant/Research Support|UT Southwestern: Grant/Research Support|ValanBio Therapeutics, Inc.: Grant/Research Support|VenatoRx: Grant/Research Support|Zogenix International: Grant/Research Support Alison Luckey, MD, GARDP: Employee of GARDP, a non-profit Swiss-based foundation, which receives government and private-funded instituti">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46d548210eb73ef90832579c63336d205247b750" target='_blank'>
              P-1251. Pharmacometric Analyses to Support Dose Selection of Zoliflodacin, a First-in-Class Oral Antibiotic Being Developed for the Treatment of Uncomplicated Gonorrhea
              </a>
            </td>
          <td>
            S. Bhavnani, Anthony P Cammarata, J. P. Hammel, Alison Luckey, Pierre Daram, Kajal B Larson, John P. O'Donnell, C. Rubino
          </td>
          <td>2025-01-29</td>
          <td>Open Forum Infectious Diseases</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Whole-body hemodynamics simulators, which model blood flow and pressure waveforms as functions of physiological parameters, are now essential tools for studying cardiovascular systems. However, solving the corresponding inverse problem of mapping observations (e.g., arterial pressure waveforms at specific locations in the arterial network) back to plausible physiological parameters remains challenging. Leveraging recent advances in simulation-based inference, we cast this problem as statistical inference by training an amortized neural posterior estimator on a newly built large dataset of cardiac simulations that we publicly release. To better align simulated data with real-world measurements, we incorporate stochastic elements modeling exogenous effects. The proposed framework can further integrate in-vivo data sources to refine its predictive capabilities on real-world data. In silico, we demonstrate that the proposed framework enables finely quantifying uncertainty associated with individual measurements, allowing trustworthy prediction of four biomarkers of clinical interest--namely Heart Rate, Cardiac Output, Systemic Vascular Resistance, and Left Ventricular Ejection Time--from arterial pressure waveforms and photoplethysmograms. Furthermore, we validate the framework in vivo, where our method accurately captures temporal trends in CO and SVR monitoring on the VitalDB dataset. Finally, the predictive error made by the model monotonically increases with the predicted uncertainty, thereby directly supporting the automatic rejection of unusable measurements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/495af227ea7b6752df29d7721708bde6c1cd1e1e" target='_blank'>
              Leveraging Cardiovascular Simulations for In-Vivo Prediction of Cardiac Biomarkers
              </a>
            </td>
          <td>
            Laura Manduchi, Antoine Wehenkel, Jens Behrmann, Luca Pegolotti, Andy C. Miller, Ozan Sener, Marco Cuturi, Guillermo Sapiro, J. Jacobsen
          </td>
          <td>2024-12-23</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3498e511e9eb5c885a5bd150e27ed48f227d9e8a" target='_blank'>
              Future Directions for Quantitative Systems Pharmacology.
              </a>
            </td>
          <td>
            Birgit Schoeberl, C. Musante, S. Ramanujan
          </td>
          <td>2025-01-16</td>
          <td>Handbook of experimental pharmacology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a07880669d7454058d6934619466fe117b01fd" target='_blank'>
              External validation of AI-based scoring systems in the ICU: a systematic review and meta-analysis
              </a>
            </td>
          <td>
            Patrick Rockenschaub, E. M. Akay, Benjamin Gregory Carlisle, A. Hilbert, Joshua Wendland, Falk Meyer-Eschenbach, Anatol-Fiete Näher, Dietmar Frey, V. Madai
          </td>
          <td>2025-01-06</td>
          <td>BMC Medical Informatics and Decision Making</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Rheumatoid arthritis (RA) is a major public health concern, which can cause serious outcomes. Low-dose methotrexate (MTX) is a cornerstone in RA treatment, but there is significant heterogeneity in clinical response. To evaluate underlying sources of pharmacokinetic variability and clinical response of MTX, a physiologically based pharmacokinetic and pharmacodynamic (PBPK/PD) model was developed using PK-sim and Mobi (version 11.1). The PBPK model included metabolism and transportation by AXO1, FPGS, GGH, RFC, and MRP2, with renal and biliary excretion. We also developed various degrees of renal insufficiency populations with subsequent dosing optimizations. A total of 23 MTX plasma concentration-time profiles were used, with 97% of predicted plasma concentrations within a two-fold range compared to observed data. The PBPK/PD modeling and simulation demonstrated that variability in renal clearance and enzymes related to MTX are likely important drivers of PK variability and there is a quantitative relationship between MTX-PG3 and RA treatment response. The PBPK/PD model could be used to guide improvement in MTX dose regimens for RA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc030539ac38870cb0565e8afef6d1b87e1a0e58" target='_blank'>
              Comprehensive Parent-Metabolite PBPK/PD Modeling Insights Into Methotrexate Personalized Dosing Strategies in Patients With Rheumatoid Arthritis.
              </a>
            </td>
          <td>
            Xin Wang, Jiangfan Wu, Hongjiang Ye, Xiaofang Zhao, Shenyin Zhu
          </td>
          <td>2025-01-08</td>
          <td>CPT: pharmacometrics & systems pharmacology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bc8d16e0d6b207bafba23a7b60598ea67dc9939" target='_blank'>
              Comparison of the power and type 1 error of total score models for drug effect detection in clinical trials
              </a>
            </td>
          <td>
            Elham Haem, Mats O Karlsson, S. Ueckert
          </td>
          <td>2024-12-10</td>
          <td>Journal of Pharmacokinetics and Pharmacodynamics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Background Bayesian software may enhance beta-lactam dosing. Therefore, we analyzed the predictiveness of cefepime pharmacokinetic (PK) models for use as Bayesian priors. Methods Data on adult patients receiving cefepime intravenous therapy were entered into a Bayesian software at 2 sites. Inclusion criteria encompassed patients with ≥ 1 cefepime drug concentration (TDM), including those undergoing continuous renal replacement therapy. Exclusion criteria were dialysis and outlier TDMs defined as > 3 standard deviations above the mean. Adult parametric PK models were identified and implemented using PKPDsim. Measured TDMs were predicted iteratively, using model priors to predict the first set (“a priori”), with subsequent TDMs predicted using Bayesian estimation (“a posteriori”), to mirror the clinical experience (i.e., initial predictions were made using only demographics; updated predictions were made using each additional TDM event). TDMs were analyzed separately as peaks or as random/trough levels. Mean percent error (MPE) assessed bias and root mean square error (RMSE) assessed imprecision. Statistical significance was determined using overlapping 90% confidence intervals from 1000 bootstrap simulations. Results A total of 114 patients with 200 TDMs for a priori and 35 TDMs for a posteriori predictions were included (Table 1). Five PK models were identified (Table 2). The An et al. and Jonckheere et al. models were most predictive, with acceptable RMSE ≤ 25 mg/L and bias < 25% (Fig 1.). The An et al. model was most precise a priori (RMSEpeak: 23 mg/L, RMSErandom/trough: 19 mg/L versus other models ranging from 26-33 mg/L and 23-30 mg/L) but demonstrated bias in predicting peaks a priori (MPE: 24%) that was not improved a posteriori. The Jonckheere et al. model demonstrated similar precision a priori to the An et al. model and showed better precision in predicting peaks a posteriori (RMSEpeak: 6 mg/L versus other models ranging from 13-25 mg/ L; Fig 1.). Conclusion While many models performed similarly (overlapping 90% CI), the An and Jonckheere models both demonstrated acceptable bias and precision for predicting random/trough levels. Centers using a random/trough sampling approach to estimate cefepime exposures may find these models clinically useful. Disclosures Maria-Stephanie Hughes, PharmD, InsightRX: Employee of company|InsightRX: Stocks/Bonds (Private Company) Nathaniel J. Rhodes, PharmD MS, Apothecademy, LLC: Advisor/Consultant Brandon Smith, MD, PharmD, Melinta Therapeutics: Advisor/Consultant|Shionogi, INC: Advisor/Consultant Ryan K. Shields, PharmD, MS, Allergan: Advisor/Consultant|Cidara: Advisor/Consultant|Entasis: Advisor/Consultant|GSK: Advisor/Consultant|Melinta: Advisor/Consultant|Melinta: Grant/Research Support|Menarini: Advisor/Consultant|Merck: Advisor/Consultant|Merck: Grant/Research Support|Pfizer: Advisor/Consultant|Roche: Grant/Research Support|Shionogi: Advisor/Consultant|Shionogi: Grant/Research Support|Utility: Advisor/Consultant|Venatorx: Advisor/Consultant|Venatorx: Grant/Research Support Jasmine Hughes, PhD, InsightRX: Employee|InsightRX: Stocks/Bonds (Private Company)">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f9f27050cfa8c04c8918281ea46a43710d2700a" target='_blank'>
              P-1253. Validating and Comparing the Predictiveness of Cefepime Pharmacokinetic Models Used in Bayesian Dosing
              </a>
            </td>
          <td>
            Maria-Stephanie A Hughes, Nathaniel J Rhodes, Brandon Smith, Ryan K Shields, Samantha Pan, Erin Weslander, Shannon Galvin, Jasmine Hughes
          </td>
          <td>2025-01-29</td>
          <td>Open Forum Infectious Diseases</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Artificial Intelligence (AI) has the disruptive potential to transform patients’ lives via innovations in pharmaceutical sciences, drug development, clinical trials, and manufacturing. However, it presents significant challenges, ethical concerns, and risks across sectors and societies. AI’s rapid advancement has revealed regulatory gaps as existing public policies struggle to keep pace with the challenges posed by these emerging technologies. The term AI itself has become commonplace to argue that greater “human oversight” for “machine intelligence” is needed to harness the power of this revolutionary technology for both potential and risk management, and hence to call for more practical regulatory guidelines, harmonized frameworks, and effective policies to ensure safety, scalability, data privacy, and governance, transparency, and equitable treatment. In this review paper, we employ a holistic multidisciplinary lens to survey the current regulatory landscape with a synopsis of the FDA workshop perspectives on the use of AI in drug and biological product development. We discuss the promises of responsible data-driven AI, challenges and related practices adopted to overcome limitations, and our practical reflections on regulatory oversight. Finally, the paper outlines a path forward and future opportunities for lawful ethical AI. This review highlights the importance of risk-based regulatory oversight, including diverging regulatory views in the field, in reaching a consensus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79cd58da6a4afff75ea6786f8af76f281d5e2ff1" target='_blank'>
              Harnessing the AI/ML in Drug and Biological Products Discovery and Development: The Regulatory Perspective
              </a>
            </td>
          <td>
            Fahimeh Mirakhori, Sarfaraz K. Niazi
          </td>
          <td>2025-01-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Objective. Demographic groups are often represented at different rates in medical datasets. These differences can create bias in machine learning algorithms, with higher levels of performance for better-represented groups. One promising solution to this problem is to generate synthetic data to mitigate potential adverse effects of non-representative data sets. Methods. We build on recent advances in LLM-based synthetic data generation to create a pipeline where the synthetic data is generated separately for each demographic group. We conduct our study using MIMIC-IV and Framingham"Offspring and OMNI-1 Cohorts"datasets. We prompt GPT4-Turbo to create group-specific data, providing training examples and the dataset context. An exploratory analysis is conducted to ascertain the quality of the generated data. We then evaluate the utility of the synthetic data for augmentation of a training dataset in a downstream machine learning task, focusing specifically on model performance metrics across groups. Results. The performance of GPT4-Turbo augmentation is generally superior but not always. In the majority of experiments our method outperforms standard modeling baselines, however, prompting GPT-4-Turbo to produce data specific to a group provides little to no additional benefit over a prompt that does not specify the group. Conclusion. We developed a method for using LLMs out-of-the-box to synthesize group-specific data to address imbalances in demographic representation in medical datasets. As another"tool in the toolbox", this method can improve model fairness and thus health equity. More research is needed to understand the conditions under which LLM generated synthetic data is useful for non-representative medical data sets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afe112ee881d10edfe5471b569a09714504f640b" target='_blank'>
              Improving Equity in Health Modeling with GPT4-Turbo Generated Synthetic Data: A Comparative Study
              </a>
            </td>
          <td>
            Daniel Smolyak, Arshana Welivita, Margr'et V. Bjarnad'ottir, Ritu Agarwal
          </td>
          <td>2024-12-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ec59aea67c1f250bece084437afa85b16714b79" target='_blank'>
              Quantitative Systems Pharmacology Modeling in Immuno-Oncology: Hypothesis Testing, Dose Optimization, and Efficacy Prediction.
              </a>
            </td>
          <td>
            Hanwen Wang, Theinmozhi Arulraj, Alberto Ippolito, Aleksander S. Popel
          </td>
          <td>2024-12-21</td>
          <td>Handbook of experimental pharmacology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Background Vancomycin (VAN) is the most frequently used antibiotic for hospitalized patients in the United States. Wide use occurs as it is a first line treatment for severe gram-positive bacterial infections in both adults and pediatrics. Most VAN dosing in children with multiple organ dysfunction syndrome (MODS) is guided via population pharmacokinetic (PK) approaches based on creatinine clearance equations and frequent samplings. Volumetric absorptive microsampling (VAMS) offers sampling procedures that require as little as 20 mcL of blood. The primary objective of this study was to develop a population PK (popPK) model for VAN in children with MODS using VAMS.Table 1. Population parameter estimation values. Methods We conducted a multi-center prospective observational study of VAN PK in pediatric MODS. Subjects (< 18 years) participating in a larger observational study of children with MODS in the ICU were eligible. Up to 15 PK samples were collected over 3 consecutive days, along with pertinent covariate data. Parametric popPK modeling was performed using Monolix 2024R1. Covariate selection was based on improvements in objective function value and physiologic relevance. Children on CRRT were excluded from the present analyses. Subjects in our interim analysis were also excluded from analysis if they were missing demographic information or were pending data queries on VAN concentrations.Figure 1. Individual weighted residuals according to time and individual predictions. Results 52 subjects from 8 sites were included with a median age of 6.5 y (range: 2 m–17 y) and a median weight of 24.5 kg (range: 3-214). A two-compartment model with clearance adjusted for allometric scaling and estimated glomerular function best described the data. Population predictions were modest (R2=0.28) whereas individualized predictions were improved (R2=0.61). Between subject variation for clearance and volume of distribution was high even after correcting for U25 creatinine clearance (92.7% and 107.9%, CV% respectively).Figure 2. Observed vs. population (A) and individual (B) predictions for the model. Conclusion Children with MODS display high between subject variability of VAN clearance and volume of distribution. Frequent sampling is needed because “stead-state” does not exist in the early period of MODS for children. VAMS can facilitate small volume sampling and future Bayesian models based on data such as these can facilitate fewer sampling events to classify the patient’s dynamic clearance. Disclosures Nathaniel J. Rhodes, PharmD MS, Apothecademy, LLC: Advisor/Consultant Kevin J. Downes, MD, Paratek, Inc.: Grant/Research Support|Veloxis Pharmaceuticals, Inc.: Grant/Research Support Marc H. Scheetz, PharmD, MSc, Abbvie: Advisor/Consultant|Basilea: Advisor/Consultant|Cidara: Advisor/Consultant|DoseMe: Advisor/Consultant|Entasis: Advisor/Consultant|F2G: Advisor/Consultant|GSK: Advisor/Consultant|Lykos: Advisor/Consultant|Roche: Advisor/Consultant|Third Pole Therapeutics: Advisor/Consultant|Xelia: Advisor/Consultant">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d18af9a9c22c47a907b9a1e2a8751ed5410aea1a" target='_blank'>
              P-1236. Vancomycin Population Pharmacokinetics Using Volumetric Absorptive Microsampling in Critically Ill Children with Multiple Organ Dysfunction Syndrome
              </a>
            </td>
          <td>
            Justin Shiau, Victor Amajor, Nathaniel J Rhodes, Athena F Zuppa, Kevin J Downes, John Takyi-Williams, Bo Wen, M. Scheetz
          </td>
          <td>2025-01-29</td>
          <td>Open Forum Infectious Diseases</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Personalized medicine is an evolving paradigm that aims to tailor therapeutic interventions to individual patient characteristics. With a growing understanding of the genetic, epigenetic, and molecular mechanisms underlying diseases, tailored therapies are becoming more feasible and effective. This review highlights the significant advancements in personalized medicine, focusing specifically on pharmacological strategies. The article explores the integration of genomics, transcriptomics, proteomics, and metabolomics in drug development and therapy optimization. Pharmacogenomics, the customization of drug therapy based on an individual's genetic makeup, receives particular emphasis. This leads to the identification of specific biomarkers that can predict therapeutic response, drug toxicity, and susceptibility to various diseases. Together with computational tools and artificial intelligence, these advancements contribute to tailored treatment plans for patients with conditions such as cancer, cardiovascular diseases, and neurological disorders. We also highlight the challenges and ethical considerations in implementing personalized medicine, such as data privacy, cost-effectiveness, and accessibility. We outline future prospects and ongoing research in this field, highlighting the importance of collaborative efforts between researchers, clinicians, pharmacists, and regulatory authorities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/605535a7297e0199c29cb0c1eb6cb59bd895c757" target='_blank'>
              Pharmacological Approaches and Innovative Strategies for Individualized Patient Care.
              </a>
            </td>
          <td>
            Amnesh Kumar Verma, Kuldeep Singh, J. K. Gupta, Shivendra Kumar, Divya Jain
          </td>
          <td>2025-01-20</td>
          <td>Recent patents on biotechnology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Predictive algorithms have myriad potential clinical decision-making implications from prognostic counselling to improving clinical trial efficiency. Large observational (or "real world") cohorts are a common data source for the development and evaluation of such tools. There is significant optimism regarding the benefits and use cases for risk-based care, but there is a notable disparity between the volume of clinical prediction models published and implementation into healthcare systems that drive and realise patient benefit. Considering the perspective of a clinician or clinical researcher that may encounter clinical predictive algorithms in the near future as a user or developer, this editorial: (1) discusses the ways in which prediction models built using observational data could inform better clinical decisions; (2) summarises the main steps in producing a model with special focus on key appraisal factors; and (3) highlights recent work driving evolution in the ways that we should conceptualise, build and evaluate these tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/923fa7aeb1ab03b626dcfc410032a8610ec53466" target='_blank'>
              How Outcome Prediction Could Aid Clinical Practice.
              </a>
            </td>
          <td>
            A. K. Clift
          </td>
          <td>2025-01-17</td>
          <td>British journal of hospital medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5f86b2198182e1b57bdd0f8d27ceac5415002e1" target='_blank'>
              Identify the underlying true model from other models for clinical practice using model performance measures
              </a>
            </td>
          <td>
            Yan Li
          </td>
          <td>2025-01-09</td>
          <td>BMC Medical Research Methodology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Everolimus presents significant dosing challenges due to between- and within-patient pharmacokinetic variabilities. This study aimed to develop and validate a model-informed precision dosing strategy for everolimus in liver transplant recipients. The dosing strategy was initially developed using retrospective data, employing nonlinear mixed-effects modeling. The model included readily measurable covariates, body surface area, albumin, and tacrolimus trough concentration. The dosing strategy was subsequently validated in a prospective trial, recommending 1 to 1.75 mg dosages every 12 h, depending on covariates. Lower dosages were recommended for patients with lower body surface area and albumin with adjustments based on tacrolimus trough concentration. The estimated pharmacokinetic parameters (typical value ± standard error), apparent clearance (CL/F: 15.0 ± 0.5 L/h), and apparent volume of distribution (Vd/F: 862 ± 79.3 L) were refined using prospective clinical data from 20 patients, reducing interindividual variations. This research successfully developed and validated a population pharmacokinetic model for everolimus. The developed "dosE" web-based platform translates our pharmacokinetic model into a practical tool for healthcare providers, exemplifying the application of pharmaceutical research in clinical practice and potentially improving therapeutic outcomes in liver transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc0083acb9d0a84c6987f4f3022cce3566fbdf3d" target='_blank'>
              Development and Clinical Validation of Model-Informed Precision Dosing for Everolimus in Liver Transplant Recipients.
              </a>
            </td>
          <td>
            Jeayoon Lee, In-Wha Kim, S. Hong, N. Han, Kyung-Suk Suh, Jung Mi Oh
          </td>
          <td>2024-12-12</td>
          <td>ACS pharmacology & translational science</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The rising prevalence of chronic diseases poses significant challenges to healthcare systems worldwide, necessitating innovative solutions for effective management. Predictive modelling using machine learning (ML) has emerged as a transformative approach in optimizing healthcare processes and addressing the complexities of chronic disease management. This paper explores the role of ML algorithms in predicting disease progression, improving diagnostic accuracy, and enhancing personalized care. By leveraging large-scale health datasets, ML models enable early identification of at-risk individuals, offering opportunities for timely intervention and preventive care. The application of ML in chronic disease management spans diverse areas, including cardiovascular diseases, diabetes, and cancer. ML algorithms, such as decision trees, neural networks, and ensemble methods, have demonstrated high predictive accuracy, significantly reducing diagnostic errors and hospital readmissions. Furthermore, the integration of ML into electronic health record (EHR) systems facilitates real-time decision-making, ensuring tailored treatment plans for patients. Case studies highlight successful implementations of ML-driven predictive models in reducing healthcare costs and improving patient outcomes. Despite these advancements, challenges remain, including data quality, privacy concerns, and the need for algorithmic transparency. This paper emphasizes the importance of addressing these barriers to foster trust and scalability in ML applications. Collaborative efforts among healthcare providers, data scientists, and policymakers are essential to realizing the full potential of ML in predictive modelling. In conclusion, machine learning offers unprecedented opportunities to revolutionize chronic disease management by optimizing healthcare processes. Its potential to predict disease trajectories and personalize care underscores its transformative role in advancing global health outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1201e7f09310355c7e71bf013cbd3d597be58acd" target='_blank'>
              Machine Learning in Predictive Modelling: Addressing Chronic Disease Management Through Optimized Healthcare Processes
              </a>
            </td>
          <td>
            Olalekan Kehinde A
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Research Publication and Reviews</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="In this systematic review, we conclude that many AI models in health care result from volumetric exploration and almost exclusively focus on black box models whose underlying mechanisms or reasons for making specific predictions are not understood by the user. However, most of the current methods are not transparent and difficult to understand due to their inherent black-box nature. Our analysis also shows that medical decision relevant AI models do not yet consider the factors of explainability, interpretability, reproducibility, robustness, and trustworthiness of deployment in clinical routine. All the above-described properties are essential to ensure the quality and safety of clinical decisions based on AI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e6c7bef6721dfe9241860febeea29ac0f1ea74d" target='_blank'>
              The Future of AI-Driven Technologies in Medical Laboratories
              </a>
            </td>
          <td>
            Moayad Sadek Alshawi, Ali Ahmad Saleh Alsmail, Murtadha Hashim Ahmed Al hashim, Mustafa Abdullah Alkhamis, Hassan Ali Alhadab
          </td>
          <td>2024-12-13</td>
          <td>International Journal For Multidisciplinary Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The pharmacokinetics of renally eliminated antibiotics can be influenced by changes associated with renal function and development in a growing subject. Little is known about the effects of renal insufficiency on the pharmacokinetics of meropenem in pediatric subjects. The aim of this study was to develop a physiologically based pharmacokinetic (PBPK) model of meropenem for pediatric patients that can be used to optimize meropenem dosing in pediatric patients with renal impairment (RI). The PBPK model was developed using GastroPlus™ 9.9 based on clinical data obtained from the literature and then scaled to pediatric patients with RI for dose optimization of meropenem. The goodness of fit of the model was assessed by comparing the predicted values of AUC0-t, AUC0-α, and Cmax with the observed data and the average fold errors (AFE). The AFE values for AUC0-t, AUC0-α, and Cmax in the pediatric population were measured to be 1.60, 1.08, and 1.48, respectively. In addition, dose optimization was performed in virtual pediatric populations with varying degrees of RI and a dose reduction to 10 mg/kg and 7.5 mg/kg was recommended for moderate and severe RI, respectively. In all virtual pediatric populations with RI, the plasma concentration reached the recommended time above the minimum inhibitory concentration (MIC) at all optimized doses. The developed PBPK model for meropenem provides a quantitative tool to assess the impact of RI on the pharmacokinetics of meropenem in pediatric patients, which may be useful for optimizing the dosing regimen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b8c26dc9cb2c89c12babc68c16da9dc0200f3fa" target='_blank'>
              Dosage Optimization Using Physiologically Based Pharmacokinetic Modeling for Pediatric Patients with Renal Impairment: A Case Study of Meropenem.
              </a>
            </td>
          <td>
            N. Rahim, Muhammad Sarfraz, A. Bello, Syed Baqir Shyum Naqvi
          </td>
          <td>2025-01-16</td>
          <td>AAPS PharmSciTech</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Background The impact of extracorporeal membrane oxygenation (EMCO) on cefepime (FEP) pharmacokinetics (PK) in the absence of renal replacement therapy (RRT) is unclear. Figure 1. Observed versus predicted cefepime plasma concentrations in the population (A) and for each individual (B) Methods We evaluated FEP PK in patients with hospital-acquired pneumonia from June 2018 to May 2023, with/without ECMO in the absence of RRT, in a single-center study nested within the Successful Clinical Response In Pneumonia Treatment (SCRIPT) study. Patient data were extracted from electronic health records. FEP dosing was according to institutional protocols based on renal function and indication. Plasma concentrations were quantified by validated LC-MS/MS assay. We used Pmetrics 2.1.1 for R to estimate population model parameter values [e.g., volume (V) and clearance (CL)] and individual target attainment rates, assuming a free (f) fraction of 80%. Targets of 100% fT>1xMIC and 100% fT>4xMIC were evaluated vs. the susceptible breakpoint MICs of Enterobacterales (2 mcg/mL) and P. aeruginosa (8 mcg/mL) and stratified by concurrent ECMO. Figure 2. Pharmacokinetic parameters (CL and V) standardized to body weight (kg) stratified by ECMO status. Results Sixty-two SCRIPT patients (63% male) contributed 95 plasma FEP samples (1-13 per patient). A total of 9 patients (one female) required ECMO. Patients had a mean±SD CRCL of 123±84 mL/min and a mean±SD body weight of 86±24 kg. The median/max first 24-hr FEP dose was 4/8 grams. A two-compartment PK model fitted best (Fig 1). Body weight and creatinine clearance were related to V and CL, respectively (Table 1). ECMO was not significantly associated with FEP CL (P=0.2) but was associated with V (P< 0.005) (Fig 2). ECMO patients had a median 3.5-fold greater Vd vs. non-ECMO patients. For targets of fT >1xMIC and fT >4xMIC, median individual plasma attainment rates were 100% and 100% against Enterobacterales and 100% and 49% against P. aeruginosa, respectively. Attainment rates were lowest in patients requiring ECMO vs. P. aeruginosa at a target of 100% fT >4xMIC, (Fig 3). Figure 3. Individual predicted cefepime PK/PD target attainment stratified by organism breakpoint and concurrent ECMO status for fT>1xMIC and fT>4xMIC. Conclusion We found that ECMO increased FEP V but not CL suggesting that ECMO patients may require loading doses and prolonged infusions to improve target attainment. Protocolized short infusions of FEP in ECMO patients resulted in suboptimal target attainment for a PK/PD goal of 100% fT>4xMIC against P. aeruginosa. More work is needed to identify the impact of FEP target attainment on clinical outcomes in populations such as these. Table 1. Population Median Parameters and 95% Credible Intervals for Cefepime Patients with and without the Requirement of Extracorporeal Membrane Oxygenation Support Disclosures Marc H. Scheetz, PharmD, MSc, Abbvie: Advisor/Consultant|Basilea: Advisor/Consultant|Cidara: Advisor/Consultant|DoseMe: Advisor/Consultant|Entasis: Advisor/Consultant|F2G: Advisor/Consultant|GSK: Advisor/Consultant|Lykos: Advisor/Consultant|Roche: Advisor/Consultant|Third Pole Therapeutics: Advisor/Consultant|Xelia: Advisor/Consultant Nathaniel J. Rhodes, PharmD MS, Apothecademy, LLC: Advisor/Consultant">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42445834e4358d72cd704ac457146f039d0471b8" target='_blank'>
              P-1224. Individual Target Pharmacokinetic/Pharmacodynamic Attainment Rates among Cefepime-treated Patients Admitted to the ICU with Hospital-Acquired Pneumonia with and without ECMO
              </a>
            </td>
          <td>
            Adrian Valadez, M. Scheetz, Michael N. Neely, Paul R. Yarnold, M. Kang, Helen K Donnelly, Kay Dedicatoria, Rachel L. Medernach, S. Nozick, Alan R. Hauser, E. Ozer, Estefani Diaz, Alexander V Misharin, R. Wunderink, Nathaniel J Rhodes
          </td>
          <td>2025-01-29</td>
          <td>Open Forum Infectious Diseases</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Antimicrobial resistance (AMR) poses a critical global health threat, necessitating the optimal use of existing antibiotics. Pharmacokinetic/pharmacodynamic (PK/PD) principles provide a scientific framework for optimizing antimicrobial therapy, particularly to respond to evolving resistance patterns. This review examines PK/PD strategies for antimicrobial dosing optimization, focusing on three key aspects. First, we discuss the importance of drug concentration management for enhancing efficacy while preventing toxicity, considering various patient populations, including pediatric and elderly patients with their unique physiological characteristics. Second, we analyze different PK modeling approaches: the classic top-down approach exemplified by population PK analysis, the bottom-up approach represented by physiologically based PK modeling, and hybrid models combining both approaches for enhanced predictive performance. Third, we explore clinical applications, including nomogram-based dosing strategies, Bayesian estimation, and emerging artificial intelligence applications, for real-time dose optimization. Critical challenges in implementing PK/PD simulation are addressed, particularly the selection of appropriate PK models, the optimization of PK/PD indices, and considerations concerning antimicrobial concentrations at infection sites. Understanding these principles and challenges is crucial for optimizing antimicrobial therapy and combating AMR through improved dosing strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/373ee52c208df4c7894e9c0219e6e75bddfbf868" target='_blank'>
              PK/PD-Guided Strategies for Appropriate Antibiotic Use in the Era of Antimicrobial Resistance
              </a>
            </td>
          <td>
            Tetsushu Onita, Noriyuki Ishihara, Takahisa Yano
          </td>
          <td>2025-01-01</td>
          <td>Antibiotics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Madam,
Precision medicine is a relatively recent approach to managing disease. It considers the genetic, environmental, and lifestyle-related variability unique to each individual, taking into account their predicted response to treatment or their disease risk for prevention strategies [1]. The concept of personalised healthcare to target each individual's medical needs is not exactly new, but the use of emerging technology and innovations, especially in fields like genomics and artificial intelligence (AI), has enabled this model to progress remarkably [2].
The utility of precision medicine for this purpose can be highlighted by pointing out the several domains where individuals can be targeted during the disease management process - for instance, the principles of AI may be used for actions like altering prescriptions based on pharmacogenomic variability, minimising sociodemographic considerations negatively impacting healthcare delivery; making clinical stratifications based on risks of comorbidities; and assessment of genomic cues and clinical patterns to customise therapies, thus improving the disease burden considerably [2]. AI has already been shown to outperform existing methods in its visual diagnostic capacity and ability to integrate and analyse diversified data, thereby becoming better at making distinctive risk predictions and significantly improving individual patient outcomes in the cardinal domain of myocardial infarction [3].
This futuristic mode of healthcare with its exciting prospects inadvertently faces some hurdles the mention of which seems imperative at this stage - for example, AI’s heavy dependence on the quality, applicability, and pertinence of its data to be able to perform the tasks mentioned above; its statistical limitation to make accurate predictions at the individual level; its incapacity as yet to ascertain causation, affecting therapy selection with precision; and the concerns of data safety and privacy [2,4]. In addition, its cost-effectiveness cannot presently be judged as a result of misallocation in the market and systematic biases owing to its novelty in healthcare [5]. Nevertheless, the remarkably accelerating impact of AI in personalising medicine must be acknowledged. In this regard, the upcoming generation of progressive researchers will surely play a profound role in bridging the gap between research and its clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/035b8d6775860a3509c8b840eb6f2d6c605ec773" target='_blank'>
              Optimizing personalized healthcare using artificial intelligence with precision medicine.
              </a>
            </td>
          <td>
            Suhaira Abdullah Arif
          </td>
          <td>2024-12-23</td>
          <td>JPMA. The Journal of the Pakistan Medical Association</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Machine Learning is transforming medical research by improving diagnostic accuracy and personalizing treatments. General ML models trained on large datasets identify broad patterns across populations, but their effectiveness is often limited by the diversity of human biology. This has led to interest in subject-specific models that use individual data for more precise predictions. However, these models are costly and challenging to develop. To address this, we propose a novel validation approach that uses a general ML model to ensure reproducible performance and robust feature importance analysis at both group and subject-specific levels. We tested a single Random Forest (RF) model on nine datasets varying in domain, sample size, and demographics. Different validation techniques were applied to evaluate accuracy and feature importance consistency. To introduce variability, we performed up to 400 trials per subject, randomly seeding the ML algorithm for each trial. This generated 400 feature sets per subject, from which we identified top subject-specific features. A group-specific feature importance set was then derived from all subject-specific results. We compared our approach to conventional validation methods in terms of performance and feature importance consistency. Our repeated trials approach, with random seed variation, consistently identified key features at the subject level and improved group-level feature importance analysis using a single general model. Subject-specific models address biological variability but are resource-intensive. Our novel validation technique provides consistent feature importance and improved accuracy within a general ML model, offering a practical and explainable alternative for clinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c3fc4715dda81205fd1fd7152931f1c0b48f084" target='_blank'>
              Stabilizing Machine Learning for Reproducible and Explainable Results: A Novel Validation Approach to Subject-Specific Insights
              </a>
            </td>
          <td>
            Gideon Vos, L. V. Eijk, Zoltán Sarnyai, M. Azghadi
          </td>
          <td>2024-12-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f28e7786509ccbeb6f2b722ec0470ba03a20d024" target='_blank'>
              Clinical Pharmacokinetics of Antitubercular Drugs in the Overweight and Obese Population: Implications for Dosage Adjustments
              </a>
            </td>
          <td>
            Marlene Prager, V. al Jalali, Markus Zeitlinger
          </td>
          <td>2025-01-10</td>
          <td>Clinical Pharmacokinetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Drug administration at similar doses in patients can often lead to various clinical responses. It has been hypothesized that genetics primarily accounts for variation in drug efficacy and toxicity in individuals. Personalized medicine has been a breakthrough achieved by the Human Genome Project which contributes to enhancing quality-based patient care. It deals with the customization of medication considering the distinct genetic and proteomic data that underpins the originality of every patient and every instance of therapeutic intervention. Many cases have shown that inter-individual differences related to drug response can be traced to genetic polymorphism in the gene alleles that codes for metabolizing enzymes, drug transporters, and genetic variations in a person’s Major Histocompatibility Complex (MHC). So, the paper throws an insight into epidemiological variations due to genetics, variations in response based on metabolic parameters, and transporters. It also covers genetic variation related to immune-related drug toxicities. Pharmacogenetic testing plays an important role in achieving more precise personalized therapeutics for better public health. The paper discusses various tests for human leukocyte antigen variants and metabolic variants. Every technology inclusion comes with advantages and limitations so personalized medicine also faces certain challenges which are discussed in the paper. Once personalized medicine is used in clinical settings, patients will be able to receive the best medications for them based on their unique genetic and protein profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c7390db50d2a6273d88f9ea7a4e94de321d7a18" target='_blank'>
              Role of Personalized Medicine in Clinical Practice: An Overview of Current and Future Perspectives
              </a>
            </td>
          <td>
            R. Ghai, Ashu Mittal, Deepali Pandey, Md Shamshir Alam, Shikha Kaushik, Pasha Ishtiyaq, Yogita Kaushik, S. Mishra
          </td>
          <td>2024-12-30</td>
          <td>Biomedical and Pharmacology Journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background and objective The growing availability of patient data from several clinical settings, fueled by advanced analysis systems and new diagnostics, presents a unique opportunity. These data can be used to understand disease progression and predict future outcomes. However, analysing this vast amount of data requires collaboration between physicians and experts from diverse fields like mathematics and engineering. Methods Mathematical models play a crucial role in interpreting patient data and enable in-silico simulations for diagnosis and treatment. To facilitate the creation and sharing of such models, the CNR-IASI BioMatLab group developed the “Gemini” (MoSpec/Autocoder) system, a framework allowing researchers with basic mathematical knowledge to quickly and correctly translate biological problems into Ordinary Differential Equations models. The system facilitates the development and computation of mathematical models for the interpretation of medical and biological phenomena, also using data from the clinical setting or laboratory experiments for parameter estimation. Results Gemini automatically generates code in multiple languages (C++, Matlab, R, and Julia) and automatically creates documentation, including code, figures, and visualizations. Conclusions This user-friendly approach promotes model sharing and collaboration among researchers, besides vastly increasing group productivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5db4a8b19a2dda0d3b5ba16f4916d27abfc74fa7" target='_blank'>
              “MoSpec”: A customized and integrated system for model development, verification and validation
              </a>
            </td>
          <td>
            M. Pompa, S. Panunzi, Alessandro Borri, Laura D'Orsi, Andrea De Gaetano
          </td>
          <td>2025-01-02</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="OBJECTIVE
This Phase I clinical trial aimed to address the knowledge gap regarding topiroxostat's use outside Japan by characterizing its pharmacokinetic profile, safety, and efficacy in healthy Chinese subjects.


METHODS
The trial followed a randomized, open-label, three-dose group design, enrolling 12 healthy participants and administering topiroxostat at three different dose levels. The study utilized NONMEM software for pharmacokinetic analysis, evaluating the impact of demographic and biochemical covariates on drug disposition.


RESULTS
Pharmacokinetic analysis shows the peak drug concentration (Cmax) under a single oral administration of 20, 40, and 80 mg of Topiroxostat, which was found in healthy subjects to be 215.46 ± 94.04 ng/mL, 473.74 ± 319.83 ng/mL and 1009.63 ± 585.98 ng/mL, respectively. The time to peak drug concentration (Tmax) was longer in females (0.79-0.98 h) than in males (0.53-0.93 h). Activated partial thromboplastin time (APTT) and triglycerides (TG) were included as covariates for the typical value of the absorption rate constant (TVKA) in our pharmacokinetic model. The dose (DOSE) was considered a covariate for the typical value of bioavailability (TVF1), and sex (SEX) was considered a covariate for the typical value of clearance (TVCL). The typical population values for topiroxostat included Q/F at 4.91 L/h, KA at 0.657 h-¹, Vc/F at 32.5 L, Vp/F at 30 L, and CL/F at 124 L/h.


CONCLUSION
The trial successfully established the pharmacokinetic parameters of topiroxostat in a Chinese population, confirming its safety and efficacy. The results support the need for individualized dosing strategies and optimize therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e4d72af0fb91cced87fb3a59ba02e684637e45e" target='_blank'>
              Characterizing Pharmacokinetic Variability of Topiroxostat in Chinese Population: Insights from a Phase I Randomized Clinical Trial.
              </a>
            </td>
          <td>
            Tianqi Zhong, Kaizong Huang, LuYao Han, Wenbo Pang, Yan Xia, Shengjun Qu, Guo Yu, Yangsheng Chen, H. Fan
          </td>
          <td>2024-12-16</td>
          <td>Current drug metabolism</td>
          <td>0</td>
          <td>22</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [2, 1],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>