<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Prostate%20cancer/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Virtual patients
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Virtual%20cell/" class="md-tabs__link">
        
  
    
  
  Virtual cell

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../RA/" class="md-tabs__link">
        
  
    
  
  RA

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Prostate%20cancer/" class="md-tabs__link">
        
  
    
  
  Prostate cancer

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Virtual patients

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Virtual%20cell/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual cell
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../RA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RA
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Prostate%20cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Prostate cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Virtual patients
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Virtual patients</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-29 08:54:58 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Virtual patients</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Virtual patients</a><br>
      <a href="#recommended_articles">3. Recommended articles on Virtual patients</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Virtual patients</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Virtual patients</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="A quantitative systems toxicology (QST) model for citalopram was established to simulate, in silico, a ‘virtual twin’ of a real patient to predict the occurrence of cardiotoxic events previously reported in patients under various clinical conditions. The QST model considers the effects of citalopram and its most notable electrophysiologically active primary (desmethylcitalopram) and secondary (didesmethylcitalopram) metabolites, on cardiac electrophysiology. The in vitro cardiac ion channel current inhibition data was coupled with the biophysically detailed model of human cardiac electrophysiology to investigate the impact of (i) the inhibition of multiple ion currents (IKr, IKs, ICaL); (ii) the inclusion of metabolites in the QST model; and (iii) unbound or total plasma as the operating drug concentration, in predicting clinically observed QT prolongation. The inclusion of multiple ion channel current inhibition and metabolites in the simulation with unbound plasma citalopram concentration provided the lowest prediction error. The predictive performance of the model was verified with three additional therapeutic and supra-therapeutic drug exposure clinical cases. The results indicate that considering only the hERG ion channel inhibition of only the parent drug is potentially misleading, and the inclusion of active metabolite data and the influence of other ion channel currents should be considered to improve the prediction of potential cardiac toxicity. Mechanistic modelling can help bridge the gaps existing in the quantitative translation from preclinical cardiac safety assessment to clinical toxicology. Moreover, this study shows that the QST models, in combination with appropriate drug and systems parameters, can pave the way towards personalised safety assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/357ea1947a44ce02642c8e40523943dff7116063" target='_blank'>
                Real Patient and its Virtual Twin: Application of Quantitative Systems Toxicology Modelling in the Cardiac Safety Assessment of Citalopram
                </a>
              </td>
          <td>
            N. Patel, B. Wiśniowska, M. Jamei, S. Polak
          </td>
          <td>None</td>
          <td>The AAPS Journal, AAPS Journal</td>
          <td>28</td>
          <td>48</td>

            <td><a href='../recommendations/357ea1947a44ce02642c8e40523943dff7116063' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Virtual clinical trials (VCTs) have gained popularity for their ability to rationalize the drug development process using mathematical and computational modelling, and to provide key insights into the mechanisms regulating patient responses to treatment. In this chapter, we cover approaches for generating virtual cohorts with applications in cancer biology and treatment. VCTs are an effective tool for predicting clinical responses to novel therapeutics and establishing effective treatment strategies. These VCTs allow us to capture inter-individual variability (IIV) which can lead to diversity in patient drug responses. Here we discuss three main methodologies for capturing IIV with a VCT. First, we highlight the use of population pharmacokinetic (PopPK) models, which extrapolate from empirical data population PK parameters that best fits the individual variability seen in drug disposition using non-linear mixed effects models. Next, we show how virtual patients may be sampled from a normal distribution with mean and standard deviation informed from experimental data to estimate parameters in a mechanistic model that regulates drug PKs. Lastly, we show how optimization techniques can be used to calibrate virtual patient parameter values and generate the VCT. Throughout, we compare and contrast these methods to provide a broader view of the generation of virtual patients, and to aid the decision-making process for those looking to leverage virtual clinical trials in their research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93dd39c86854a4c9008387177ccfd2a474105b80" target='_blank'>
                Approaches to generating virtual patient cohorts with applications in oncology
                </a>
              </td>
          <td>
            Anudeep Surendran, Justin Le Sauteur-Robitaille, Dana Kleimeier, J. Gevertz, K. Wilkie, A. Jenner, M. Craig
          </td>
          <td>2022-05-25</td>
          <td>bioRxiv</td>
          <td>8</td>
          <td>18</td>

            <td><a href='../recommendations/93dd39c86854a4c9008387177ccfd2a474105b80' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef39ec91aeb4092e59df228b59ca6261be21ff5" target='_blank'>
                Health digital twins as tools for precision medicine: Considerations for computation, implementation, and regulation
                </a>
              </td>
          <td>
            Kaushik P. Venkatesh, Marium M. Raza, J. Kvedar
          </td>
          <td>2022-09-22</td>
          <td>NPJ Digital Medicine</td>
          <td>97</td>
          <td>46</td>

            <td><a href='../recommendations/9ef39ec91aeb4092e59df228b59ca6261be21ff5' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="A mathematical model of the human response to trauma replicates individual patient outcomes but predicts unexpected results in populations. A virtual large sample size Severe trauma or bleeding evokes an all-hands-on-deck immune response. When properly orchestrated, the myriad cytokines and peptides help to heal the patient. However, this process can easily go awry, and administering the right drug to compensate is a challenge. Brown and colleagues mathematically modeled the complicated immune responses from the data of 33 blunt trauma patients and then generated a larger cohort of 10,000 virtual trauma patients. This large virtual cohort predicted the reactions of smaller validation cohorts, but the surprise was that understanding the response details in a single patient did not predict how the population would act. The author’s virtual clinical trial indicated that inhibition of interleukin-6 (IL-6) produced a small survival benefit, whereas IL-1β inhibition did not help much and tumor necrosis factor–α made things worse. Trauma-induced critical illness is driven by acute inflammation, and elevated systemic interleukin-6 (IL-6) after trauma is a biomarker of adverse outcomes. We constructed a multicompartment, ordinary differential equation model that represents a virtual trauma patient. Individual-specific variants of this model reproduced both systemic inflammation and outcomes of 33 blunt trauma survivors, from which a cohort of 10,000 virtual trauma patients was generated. Model-predicted length of stay in the intensive care unit, degree of multiple organ dysfunction, and IL-6 area under the curve as a function of injury severity were in concordance with the results from a validation cohort of 147 blunt trauma patients. In a subcohort of 98 trauma patients, those with high–IL-6 single-nucleotide polymorphisms (SNPs) exhibited higher plasma IL-6 levels than those with low IL-6 SNPs, matching model predictions. Although IL-6 could drive mortality in individual virtual patients, simulated outcomes in the overall cohort were independent of the propensity to produce IL-6, a prediction verified in the 98-patient subcohort. In silico randomized clinical trials suggested a small survival benefit of IL-6 inhibition, little benefit of IL-1β inhibition, and worse survival after tumor necrosis factor–α inhibition. This study demonstrates the limitations of extrapolating from reductionist mechanisms to outcomes in individuals and populations and demonstrates the use of mechanistic simulation in complex diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfd4fdecd1112b0df0d9bf7248306695434283e2" target='_blank'>
                Trauma in silico: Individual-specific mathematical models and virtual clinical populations
                </a>
              </td>
          <td>
            David Brown, Rami A. Namas, K. Almahmoud, A. Zaaqoq, J. Sarkar, Derek A. Barclay, Jinling Yin, Ali Ghuma, Andrew Abboud, G. Constantine, G. Nieman, R. Zamora, Steven C. Chang, T. Billiar, Y. Vodovotz
          </td>
          <td>2015-04-29</td>
          <td>Science Translational Medicine</td>
          <td>75</td>
          <td>140</td>

            <td><a href='../recommendations/bfd4fdecd1112b0df0d9bf7248306695434283e2' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Purpose We introduce two validated single (SH) and dual hormone (DH) mathematical models that represent an in-silico virtual patient population (VPP) for type 1 diabetes (T1D). The VPP can be used to evaluate automated insulin and glucagon delivery algorithms, so-called artificial pancreas (AP) algorithms that are currently being used to help people with T1D better manage their glucose levels. We present validation results comparing these virtual patients with true clinical patients undergoing AP control and demonstrate that the virtual patients behave similarly to people with T1D. Methods A single hormone virtual patient population (SH-VPP) was created that is comprised of eight differential equations that describe insulin kinetics, insulin dynamics and carbohydrate absorption. The parameters in this model that represent insulin sensitivity were statistically sampled from a normal distribution to create a population of virtual patients with different levels of insulin sensitivity. A dual hormone virtual patient population (DH-VPP) extended this SH-VPP by incorporating additional equations to represent glucagon kinetics and glucagon dynamics. The DH-VPP is comprised of thirteen differential equations and a parameter representing glucagon sensitivity, which was statistically sampled from a normal distribution to create virtual patients with different levels of glucagon sensitivity. We evaluated the SH-VPP and DH-VPP on a clinical data set of 20 people with T1D who participated in a 3.5-day outpatient AP study. Twenty virtual patients were matched with the 20 clinical patients by total daily insulin requirements and body weight. The identical meals given during the AP study were given to the virtual patients and the identical AP control algorithm that was used to control the glucose of the virtual patients was used on the clinical patients. We compared percent time in target range (70–180 mg/dL), time in hypoglycemia (<70 mg/dL) and time in hyperglycemia (>180 mg/dL) for both the virtual patients and the actual patients. Results The subjects in the SH-VPP performed similarly vs. the actual patients (time in range: 78.1 ± 5.1% vs. 74.3 ± 8.1%, p = 0.11; time in hypoglycemia: 3.4 ± 1.3% vs. 2.8 ± 1.7%, p = 0.23). The subjects in the DH-VPP also performed similarly vs. the actual patients (time in range: 75.6 ± 5.5% vs. 71.9 ± 10.9%, p = 0.13; time in hypoglycemia: 0.9 ± 0.8% vs. 1.3 ± 1%, p = 0.19). While the VPPs tended to over-estimate the time in range relative to actual patients, the difference was not statistically significant. Conclusions We have verified that a SH-VPP and a DH-VPP performed comparably with actual patients undergoing AP control using an identical control algorithm. The SH-VPP and DH-VPP may be used as a simulator for pre-evaluation of T1D control algorithms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed50a04c433e06a870d929d0948c3de3cdeaebd" target='_blank'>
                A statistical virtual patient population for the glucoregulatory system in type 1 diabetes with integrated exercise model
                </a>
              </td>
          <td>
            Navid Resalat, J. El Youssef, Nichole S. Tyler, J. Castle, P. Jacobs
          </td>
          <td>2019-07-25</td>
          <td>PLoS ONE</td>
          <td>59</td>
          <td>31</td>

            <td><a href='../recommendations/2ed50a04c433e06a870d929d0948c3de3cdeaebd' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Virtual patients'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Virtual patients</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Objective This study aimed to evaluate the predictive performance of published amisulpride population pharmacokinetic (PopPK) models in schizophrenia patients with an external data set and establish remedial dosing regimens for nonadherent amisulpride-treated patients. Methods A systematic search was conducted on PubMed, Embase, and Web of Science to identify PopPK models for evaluation. The evaluation process involved analyzing 390 serum concentration samples obtained from 361 Chinese adult inpatients diagnosed with schizophrenia. Model predictability was evaluated by prediction-based and simulation-based diagnostics. Based on validation results, a modified PopPK model was constructed to characterize amisulpride pharmacokinetic in our patients. Monte Carlo simulation was employed to investigate non-adherence scenarios and the impact of subsequently administered remedial regimens. Results In the five assessed published models, four included trough concentrations from schizophrenia patients, and one combined single-dose data from healthy older adults and trough concentrations from older adults with Alzheimer’s disease. The PE for population and individual predictions ranged from −92.89% to 27.02% and −24.82% to 4.04%, respectively. In the simulation-based diagnostics, the NPDE results indicated noticeable bias in all models. Therefore, a modified one-compartment model, with estimated creatinine clearance(eCLcr) as covariates on the apparent clearance (CL/F) of amisulpride, was developed. For delays in medication dosing, if the delay is within 12 hours, take half the missed dose right away, then resume the normal schedule; if the delay is up to 24 hours, just continue with the regular dosing schedule. Conclusion Existing published models lack the necessary reliability for cross-center application. Future prospective studies are required to assess our model before integrating it into clinical practice. Model-based simulations provided a rational approach to propose remedial strategies for delayed or missed doses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9be1e0b56d2bfcc57c62a4c6fabad9df86d84469" target='_blank'>
              External Validation of the Population Pharmacokinetic Models of Amisulpride and Remedial Strategies for Delayed or Missed Doses
              </a>
            </td>
          <td>
            Desheng Yan, Gehang Ju, Xin Liu, Qing Shao, Yan Zhang, Na Wang, Keyu Yan
          </td>
          <td>2024-12-01</td>
          <td>Drug Design, Development and Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background/Objectives: Colorectal cancer (CRC) holds the third and second position among cancers affecting men and women, respectively. Frequently, the first-line treatment for metastatic CRC consists of the intravenous administration of 5-fluorouracil and leucovorin in combination with oxaliplatin or irinotecan. Physiologically-based pharmacokinetic models (PBPK) aim to mechanistically incorporate body physiology and drug physicochemical attributes, enabling the description of both systemic and organ drug exposure based on the treatment specificities. This bottom-up approach represents an opportunity to personalize treatment and minimize the therapeutic risk/benefit ratio through the understanding of drug distribution within colorectal tissue. This project has the goal of characterizing the systemic and tissue exposure of four anti-cancer drugs in humans using a PBPK platform fed with data from the literature. Methods: A literature search was performed to collect clinical data on systemic concentration versus time profiles. Physicochemical features were obtained from the literature, as well as parameters associated with distribution, metabolism, and excretion. The PBPK models were built using PK-Sim®. Results: The data from 51 clinical studies were extracted and combined in one single dataset. The PBPK models successfully described the exposure vs. time profiles with respect to both, with both the typical tendency and dispersion shown by the data. The percentage of observations falling within the two-fold error bounds ranged between 94 and 100%. The colon/plasma AUCinf ratios were similar for 5-FU, oxaliplatin, and leucovorin, but it was significantly higher for irinotecan. Conclusions: The PBPK models support tailored treatment approaches by linking in vitro studies to organ exposure. These models serve as the initial step towards incorporating a dedicated tumor compartment, which will further account for the variability in tumor microenvironment characteristics to improve therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c47bd1801f751033fb61994389dcf83cf82aae7" target='_blank'>
              Development and Application of Physiologically-Based Pharmacokinetic Model to Predict Systemic and Organ Exposure of Colorectal Cancer Drugs
              </a>
            </td>
          <td>
            Sara Peribañez-Dominguez, Zinnia P. Parra-Guillen, I. Trocóniz
          </td>
          <td>2025-01-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0eda19b94dfcd39089ae0957c52f1ac7d92dc5b2" target='_blank'>
              Simulation of clinical trials of oral treprostinil in pulmonary arterial hypertension using a virtual population
              </a>
            </td>
          <td>
            Andrew E. Stine, Jignesh Parmar, Amy K Smith, Zachary Cummins, N. R. Pillalamarri, R. J. Bender
          </td>
          <td>2025-01-15</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The integration of artificial intelligence [AI] into clinical trials has revolutionized the process of drug development and personalized medicine. Among these advancements, deep learning and predictive modelling have emerged as transformative tools for optimizing clinical trial design, patient recruitment, and real-time monitoring. This study explores the application of deep learning techniques, such as convolutional neural networks [CNNs] and transformer-based models, to stratify patients, forecast adverse events, and personalize treatment plans. Furthermore, predictive modelling approaches, including survival analysis and time-series forecasting, are employed to predict trial outcomes, enhancing efficiency and reducing trial failure rates. To address challenges in analysing unstructured clinical data, such as patient notes and trial protocols, natural language processing [NLP] techniques are utilized for extracting actionable insights. A custom dataset comprising structured patient demographics, genomic data, and unstructured text is curated for training and validating these models. Key metrics, including precision, recall, and F1 scores, are used to evaluate model performance, while trade-offs between accuracy and computational efficiency are examined to identify the optimal model for clinical deployment. This research underscores the potential of AI-driven methods to streamline clinical trial workflows, improve patient-centric outcomes, and reduce costs associated with trial inefficiencies. The findings provide a robust framework for integrating predictive analytics into precision medicine, paving the way for more adaptive and efficient clinical trials. By bridging the gap between technological innovation and real-world applications, this study contributes to advancing the role of AI in healthcare, particularly in fostering personalized care and improving overall trial success rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2be7af1178b1e5b9dcf1c457f1b3a6483e200350" target='_blank'>
              Advancing clinical trial outcomes using deep learning and predictive modelling: bridging precision medicine and patient-centered care
              </a>
            </td>
          <td>
            Sydney Anuyah, Mallika K Singh, Hope Nyavor
          </td>
          <td>2024-12-09</td>
          <td>ArXiv</td>
          <td>16</td>
          <td>3</td>
        </tr>

        <tr id="Objectives To assess the pharmacokinetics and pharmacodynamics of imipenem in a retrospective cohort of hospitalized Chinese older patients. Methods A population pharmacokinetic (PPK) model was constructed utilizing a nonlinear mixed-effects modeling approach. The final model underwent evaluation through bootstrap resampling and visual predictive checks. Additionally, a population pharmacokinetic and pharmacodynamic analysis was conducted employing Monte Carlo simulations to investigate the impact of commonly used dosing regimens (0.25 g every 6 h, 0.5 g every 6 h, 0.5 g every 8 h, 1 g every 6 h, 1 g every 8 h, and 1 g every 12 h) on the likelihood of achieving the target therapeutic outcomes. Results A total of 370 observations available from 142 patients were incorporated in the PPK model. A two-compartment PPK model with linear elimination best predicted the imipenem plasma concentrations, with the creatinine clearance as a significant covariate of clearance. Typical estimates for clearance, inter-compartmental clearance, central and peripheral volume were 13.1 L·h−1, 11.9 L·h−1, 11.7 L, 29.3 L, respectively. Conclusion The pharmacokinetics of imipenem in elderly patients were effectively characterized by the established PPK model, which includes creatinine clearance as a key covariate. This research will enhance our understanding of imipenem elimination and support precision dosing in this patient demographic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47628ead3bd204ff68a69f873bcb56f935e133c1" target='_blank'>
              Population pharmacokinetics and dosing optimization of imipenem in Chinese elderly patients
              </a>
            </td>
          <td>
            Jing Wang, Qiu Fang, Xuemei Luo, Lu Jin, Huaijun Zhu
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16434184a7c396255e1e520823e433a1333c2f54" target='_blank'>
              A Pooled Pharmacokinetic Analysis for Piperacillin/Tazobactam Across Different Patient Populations: From Premature Infants to the Elderly
              </a>
            </td>
          <td>
            Daming Kong, Jason A Roberts, J. Lipman, F. Taccone, M. Cohen-Wolkowiez, F. Sime, Danny Tsai, Pieter A. J. G. De Cock, S. Jaruratanasirikul, S. Dhaese, Andrew A. Udy, Timothy W. Felton, R. Michelet, Céline Thibault, Jeroen V. Koomen, D. Eleveld, M. Struys, Jan J De Waele, Pieter J. Colin
          </td>
          <td>2024-12-25</td>
          <td>Clinical Pharmacokinetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/934ce70965d6e11160dca268c540c46e605b2c8e" target='_blank'>
              Fast and accurate prediction of drug induced proarrhythmic risk with sex specific cardiac emulators
              </a>
            </td>
          <td>
            P. Dominguez-Gomez, Alberto Zingaro, Laura Baldo-Canut, Caterina Balzotti, Borje Darpo, Christopher Morton, Mariano Vázquez, Jazmin Aguado
          </td>
          <td>2024-12-26</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The Comprehensive In Vitro Proarrhythmia Assay (CiPA) evaluates drug-induced torsade de pointes (TdP) risk, with qNet commonly used to classify drugs into low-, intermediate-, and high-risk categories. While most studies focus on single-drug effects, 2-drug fixed-dose combination (FDC) therapy is widely used for cardiovascular disease management. We aimed to develop the CiPA-based methodology to predict adverse effects of FDC therapy. A human ventricular cell model was stimulated under the effects of various drug combinations from twelve well-characterized compounds suggested by CiPA at 1 to 4 maximum plasma concentration, and the qNetavg biomarker as a function of the ratio of two drugs was used to evaluate the TdP risk of combined compounds. Results showed that high-risk and intermediate-risk drug combinations often yielded lower qNetavg than individual drugs, suggesting increased TdP risk. Conversely, combinations involving low-risk drugs tended to reduce TdP risk by raising qNetavg above individual drug levels. Also, we found that the interplay of some major ionic channels caused variations on qNetavg. These findings highlight the importance of evaluating FDC cardiotoxicity to predict risks that may not appear in single-drug analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f42ede5d44162968de78cb0840deff395bdb4e1f" target='_blank'>
              Cardiotoxicity evaluation of two-drug fixed-dose combination therapy under CiPA: a computational study
              </a>
            </td>
          <td>
            Ali Ikhsanul Qauli, Aroli Marcellinus, F. Vanheusden, Ki Moo Lim
          </td>
          <td>2024-12-01</td>
          <td>Translational and Clinical Pharmacology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background/Objectives: Vedolizumab (VDZ) is approved in the treatment of patients with moderate to severe ulcerative colitis (UC) or Crohn’s disease (CD). VDZ exhibits considerable variability in its pharmacokinetic (PK) profile, and its exposure-response relationship is not yet fully understood. The aim was to investigate the variability in VDZ trough levels and PK parameters, to assess the relationship between VDZ PK and biochemical response, as well as clinical and endoscopic outcomes. Methods: We included 61 UC and 45 CD patients. Patients’ data and trough VDZ concentrations were retrospectively obtained. Population PK analysis was performed using non-linear mixed-effects modelling with NONMEM (version 7.5). Graphs and statistical analyses were performed using R (version 4.1.3). Results: In total, 116 trough VDZ concentrations from 106 patients were described by a two-compartment model. For a typical patient, clearance (CL) was estimated at 0.159 L/day, while in patients previously treated with anti-TNFα agents, VDZ CL increased by 26.4% on average. In univariate binary logistic regression, VDZ trough concentration was not statistically significant predictor of remission, whereas CL was. Moreover, combined CL and faecal calprotectin (FCP) were a statistically significant predictors of remission. The hazard ratio (HR) for CL above 0.1886 L/day was 0.35 (p = 0.05) and for FCP below 250 µg/g was 2.66 (p = 0.02) in a time-to-event analysis. Conclusions: Our population PK model incorporates the effect of prior anti-TNFα agents on CL, suggesting its association with more severe forms of IBD. VDZ CL emerged as a more robust and clinically relevant predictor of remission in IBD patients than trough concentration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b3f763285f8d609f76a57110a4b866db845438f" target='_blank'>
              Vedolizumab Clearance as a Surrogate Marker for Remission in Inflammatory Bowel Disease Patients: Insights from Real-World Pharmacokinetics
              </a>
            </td>
          <td>
            S. Marković, Đ. Kralj, P. Svorcan, T. Knežević Ivanovski, O. Odanović, Sanja Obradović, A. Homšek, Marija Jovanović, Rada Savic, K. Vučićević
          </td>
          <td>2024-12-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="It is the era of personalized medicine, ushering us into a new healthcare era of treatment based on the individual characteristics of each. Using advances in genomics, artificial intelligence (AI), and multi-omics technologies, this revolutionary approach promises diagnosis, prevention, and treatment strategies that go far beyond the “one size fits all” model of the past[1].
From Genomics to Multi-Omics: The Precision Healthcare Foundation
The completion of the Human Genome Project was a major step forward in modern medicine, unveiling the sequence of the genetic code that defines each of us. However, the human genome was not the end of the story. With the advent of personalized medicine, we define it through its multi-omics nature, which integrates genomics, transcriptomics, proteomics, metabolomics, and the microbiome. These layers of data give us an understanding of the biological mechanisms driving disease that allow targeted intervention[2].
For instance, genetic biomarkers have made a sea change in oncology. Targeted therapies improve the outcome in breast cancer (e.g. BRCA1/BRCA2) and lung cancer (e.g. mutant EGFR) by detecting such mutations in genomic profiling. Likewise, technologies such as liquid biopsy are advancing cancer care by providing real-time monitoring of circulating tumor DNA without invasive monitoring[3].
Artificial Intelligence and Digital Twins: Accelerating Progress
AI and machine learning have become the new indispensable tools for personalized medicine. However, the vastness of datasets, such as genetic profiles, electronic health records (EHRs), and wearable device data, can be analyzed by AI algorithms to predict disease risks, advise treatments, and optimize clinical decision-making. For example, AI-driven models have shown themselves capable of detecting breast cancer with similar accuracy to radiologists, identifying new biomarkers for the prediction of disease, and personalizing pharmacotherapy[4].
A very exciting advance is digital twins. These are so-called virtual replicas of individual patients who are created using real-time health data, simulations, and predictive models. Digital twins enable healthcare providers to test treatment plans in a virtual environment before applying them in the real world. This innovation reduces risks, shortens clinical trial timelines, and paves the way for truly individualized care[5].
Personalized Medicine in Clinical Practice
Personalized medicine is already being translated into the clinic, albeit at a slower pace. For example, pharmacogenomics helps clinicians optimize drug therapy for an individual’s genetic makeup. Examples include genetic testing-guided dosing of warfarin or the use of targeted therapies in cancers with defined molecular signatures. In addition, smart devices and digital health tools promote continuous health monitoring and allow patients to take an active role in managing their health[6].
Advances in genomics are allowing us to identify people at high risk for cardiovascular disorders, or diabetes, among other diseases, and intervene before the problems happen. For example, BRCA1 mutation carriers have taken proactive steps, like Angelina Jolie has, to mitigate breast and ovarian cancer risks[7].
Challenging Issues and Ethical Issues
The promise of personalized medicine has not gone unchallenged. First, it is still expensive for many healthcare systems to perform multi-omics analysis, AI tools, and genetic testing. If we don’t address equity in access, health disparities will continue to widen[8].
Second, these massive amounts of data are problematic because of the problems those data create around privacy, security, and ethical use. Strong policies and regulations must cover the issue of informed consent and data ownership, as well as protection against the misuse of genetic information[9].
Clinicians and patients alike need to be educated and trained on the many facets of personalized medicine. Streamlined workflows, interoperable health systems, and clinical guidelines are needed for integration into routine care[10].
The Road Ahead: Personal, Predictive, Preventive
Technology, as well as our increased knowledge of biology, is the future of personalized medicine. If you keep investing in genomics, AI, and digital tools we are about to enter a world where disease prevention, early detection, and targeted treatment are the norm[11].
This is really future enabled by truly personalized health, predictive health through advanced models to predict health outcomes, and preventive health to prevent before a disease strikes. The future holds the promise not only of improved individual health outcomes but a more efficient, less costly, more equitable healthcare system on a global scale[12].
Conclusion
In a future of personalized medicine, we have enormous promise from technological advances and a greater understanding of human biology. The move towards more precise, efficient, and patient-centric healthcare is seeing these integrated with genomic variation, AI, and digital tools. However, to get to this future, there are issues of accessibility, ethical issues, and data security. As we stand on the cusp of a new era, we can not achieve personalized medicine without collaboration between researchers, clinicians, policymakers, and technologists. By doing this, not only will we improve individual health outcomes, but we will also change the global healthcare landscape for generations to come.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12cc9c53828c5c03e9ba9feaf83d8ca3b900f2bf" target='_blank'>
              The Future of Personalized Medicine
              </a>
            </td>
          <td>
            Naveed Shuja
          </td>
          <td>2024-12-18</td>
          <td>DEVELOPMENTAL MEDICO-LIFE-SCIENCES</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Minimizing harm is a cornerstone of ethical research practices. A drug that has undergone extensive clinical pharmacological testing in healthy participants (HPs) and a diverse selection of patients can be described with a sufficiently predictive population pharmacokinetic (PopPK) model. In impaired clearance trials, recruitment is minimized and underpowered for all but major exposure differences. Virtual HP arms have been reported to support similar conclusions to conventional impaired clearance studies, and further minimize potential harm of drug exposure without medical benefit by eliminating an arm of the study. However, the extent to which the conventional analysis of impairment studies compare to the simulation approach is unknown. Here we assess the operating characteristics of the virtual cohort approach along with the conventional approach through controlled simulations. These simulations included a simple, widely accessible PopPK model and several internal models that have been used in a previous meta-analysis of the virtual cohort approach. In the pairwise comparisons assessed, the virtual cohort simulation approach had greater power per sample size than the conventional approach and the same power under the null hypothesis. Given key methodological differences, it is recommended that the simulation and conventional approaches be treated as having approximately the same power under equivalent conditions. These results provide a strong justification for the use of the virtual cohort approach when an adequate PopPK model is available, minimizing unnecessary exposure to study drugs that will not benefit healthy study participants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a792791d41701cb013d930fd9010e65799e5e43" target='_blank'>
              Operating Characteristics of the Simulated Healthy Participant Approach in Impaired Clearance Studies.
              </a>
            </td>
          <td>
            Sana Gupta, Vivek Purohit, Yuchen Wang, John P Prybylski
          </td>
          <td>2025-01-22</td>
          <td>The AAPS journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="To employ a reduced-order cardiovascular model as a digital twin for personalised medicine, it is essential to understand how uncertainties in the model’s input parameters affect its outputs. The aim is to identify a set of input parameters that can serve as clinical biomarkers, providing insight into a patient’s physiological state. Given the challenge of finding useful clinical data, careful consideration must be given to the experimental design used to acquire patient-specific input parameters. In this paper, we conduct the first quantification of a cardiovascular system’s sloppiness to elucidate the structure of the input parameter space. By utilising Sobol indices and examining various synthetic cardiovascular measures with increasing invasiveness, we uncover how the personalisation process and the cardiovascular system’s sloppiness are contingent upon the chosen experimental design. Our findings reveal that continuous clinical measures induce system sloppiness and increase the number of personalisable biomarkers, whereas discrete clinical measurements produce a non-sloppy system with a reduced number of biomarkers. This study underscores the necessity for careful consideration of available clinical data as differing measurement sets can significantly impact model personalisation. Author Summary In personalised medicine, computational models that replicate physical systems — are becoming vital tools for understanding and predicting individual health. Our study explores cardiovascular models, which simulate heart and circulatory functions from which clinical metrics may be derived. These models aim to provide personalised insights into heart health and treatment planning. A key challenge in building these models is addressing “sloppiness,” a property which provides vital insight into the response surface structure for which one calibrates a model searching for a global minimum point, a position in parameter space which best represents a patients cardiovascular health. In order to personalise a model different types of clinical metrics must be available for a model response to be compared to. We examined how different types of clinical data — ranging from simple discrete blood pressure readings to detailed invasive continuous waveform data — impact model sloppiness and the number of personalisable biomarkers. Our results show that continuous measurements increase the number of personalisable biomarkers but make the personalisation process more complex through increased sloppiness. In contrast, simpler discrete measurements reduce model sloppiness simplifying the personalisation process but yield fewer personalisable biomarkers. By analysing the impact of experimental designs on the personalisation process, our work offers practical insights into improving the reliability of cardiovascular digital twins, supporting their adoption in personalised medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad51bcdf15d70eb358e8f1dceeb4de62915ebed9" target='_blank'>
              The Impact of Experimental Designs & System Sloppiness on the Personalisation Process: A Cardiovascular Perspective
              </a>
            </td>
          <td>
            Harry Saxton, Daniel J. Taylor, Grace Faulkner, Ian Halliday, Thomas Newman, Torsten Schenkel, Paul D. Morris, Richard H. Clayton, Xu Xu
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT The pharmaceutical industry constantly strives to improve drug development processes to reduce costs, increase efficiencies, and enhance therapeutic outcomes for patients. Model‐Informed Drug Development (MIDD) uses mathematical models to simulate intricate processes involved in drug absorption, distribution, metabolism, and excretion, as well as pharmacokinetics and pharmacodynamics. Artificial intelligence (AI), encompassing techniques such as machine learning, deep learning, and Generative AI, offers powerful tools and algorithms to efficiently identify meaningful patterns, correlations, and drug–target interactions from big data, enabling more accurate predictions and novel hypothesis generation. The union of MIDD with AI enables pharmaceutical researchers to optimize drug candidate selection, dosage regimens, and treatment strategies through virtual trials to help derisk drug candidates. However, several challenges, including the availability of relevant, labeled, high‐quality datasets, data privacy concerns, model interpretability, and algorithmic bias, must be carefully managed. Standardization of model architectures, data formats, and validation processes is imperative to ensure reliable and reproducible results. Moreover, regulatory agencies have recognized the need to adapt their guidelines to evaluate recommendations from AI‐enhanced MIDD methods. In conclusion, integrating model‐driven drug development with AI offers a transformative paradigm for pharmaceutical innovation. By integrating the predictive power of computational models and the data‐driven insights of AI, the synergy between these approaches has the potential to accelerate drug discovery, optimize treatment strategies, and usher in a new era of personalized medicine, benefiting patients, researchers, and the pharmaceutical industry as a whole.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4c0840bc02532ce5e02fd634295b18c901f94d" target='_blank'>
              Integrating Model‐Informed Drug Development With AI: A Synergistic Approach to Accelerating Pharmaceutical Innovation
              </a>
            </td>
          <td>
            Karthik Raman, Rukmini Kumar, C. Musante, Subha Madhavan
          </td>
          <td>2025-01-01</td>
          <td>Clinical and Translational Science</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1c49f7fb420c30b28dbda2831ccc402f2a211e6" target='_blank'>
              Machine learning-based prediction of pharmacokinetic parameters for individualized drug dosage optimization
              </a>
            </td>
          <td>
            Sinenhlanhla Mtshali, Byron A. Jacobs
          </td>
          <td>2024-12-15</td>
          <td>International Journal of Information Technology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Sirolimus is a commonly used immunosuppressant administered after solid organ transplantation. It is characterized by a narrow therapeutic window and highly variable exposure, necessitating the identification of the sources of variability and design of individualized drug therapies. Aim This study aimed to perform a population pharmacokinetic (PK) analysis of sirolimus in adult liver transplant recipients and develop dosing regimen recommendations according to patient characteristics. Methodology A total of 216 measurements of whole blood sirolimus concentrations in 103 adult patients were obtained for analysis. Covariates influencing the PKs of sirolimus were investigated using a stepwise procedure. Monte Carlo simulations were conducted to recommend dosing regimens for patients with different levels of covariates. Results A one-compartment model with first-order elimination provided the best fit of the data. Hematocrit (HCT) significantly influenced the apparent clearance of sirolimus. Monte Carlo simulations showed that for patients with a low HCT level of 28%, dosing regimens of 1.5 mg qd or 1 mg qd alternating with 1.5 mg qd should be recommended. For patients with a normal HCT level, the recommended dosing regimens were 1 mg qd, 2 mg qod, or 0.5 mg qd alternating with 1 mg qd. Conclusion Based on our population PK model of sirolimus in adult liver transplant recipients, which has the largest sample size to date, we recommend to tailor dosing regimens to various HCT levels in such patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c20849fe8ed6c30dc723157d06c4013ad769c7d" target='_blank'>
              Dosing Regimen Recommendations for Sirolimus in Adult Liver Transplant Recipients: Insights from a Population Pharmacokinetic Model
              </a>
            </td>
          <td>
            Juehui Mao, Yunshan Cheng, Dong Liu, Bo Zhang, Xiping Li
          </td>
          <td>2024-12-01</td>
          <td>Drug Design, Development and Therapy</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Introduction The Precision Medicine Program (PMP) at the University of Florida (UF) focuses on advancing pharmacogenomics (PGx) to improve patient care. Methods The UF PMP, in collaboration with the UF Health Pathology Laboratory (UFHPL), utilized Health Level Seven (HL7) standards to integrate PGx data into Epic’s Genomic Module to enhance the management and utilization of PGx data in clinical practice. Results A key feature of the Genomic Module is the introduction of genomic indicators—innovative tools that flag actionable genetic information directly within the electronic health record (EHR). These indicators enable the effective presentation of phenotypic information and, when leveraged with existing clinical decision support (CDS) alerts, help provide timely and informed therapeutic decisions based on genomic data. Discussion This advancement represents a significant shift in the utilization of genetic data, moving beyond traditional PDF reports to provide a comprehensive understanding of PGx data. Ultimately, this integration empowers healthcare providers with genomics-guided recommendations, enhancing precision and personalization in patient care, contributing significantly to the advancement of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5955dfc2e074f4603f4941a47255c69bf616e8b" target='_blank'>
              Integration of pharmacogenetic data in epic genomic module drives clinical decision support alerts
              </a>
            </td>
          <td>
            Kimberly J. Newsom, Bradley Hall, Katherine A. Martinez, Scott Nelson, Petr Starostik, Khoa Nguyen
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Aims Few personalized monitoring models for valproic acid (VPA) in pediatric epilepsy patients (PEPs) incorporate machine learning (ML) algorithms. This study aimed to develop an ensemble ML model for VPA monitoring to enhance clinical precision of VPA usage. Methods A dataset comprising 366 VPA trough concentrations from 252 PEPs, along with 19 covariates and the target variable (VPA trough concentration), was refined by Spearman correlation and multicollinearity testing (366 × 11). The dataset was split into a training set (292) and testing set (74) at a ratio of 8:2. An ensemble model was formulated by Gradient Boosting Regression Trees (GBRT), Random Forest Regression (RFR), and Support Vector Regression (SVR), and assessed by SHapley Additive exPlanations (SHAP) analysis for covariate importance. The model was optimized for R2, relative accuracy, and absolute accuracy, and validated against two independent external datasets (32 in-hospital and 28 out-of-hospital dataset). Results Using the R2 weight ratio of GBRT, RFR and SVR optimized at 5:2:3, the ensemble model demonstrated superior performance in terms of relative accuracy (87.8%), absolute accuracy (78.4%), and R2 (0.50), while also exhibiting a lower Mean Absolute Error (9.87) and Root Mean Squared Error (12.24), as validated by the external datasets. Platelet count (PLT) and VPA daily dose were identified as pivotal covariates. Conclusion The proposed ensemble model effectively monitors VPA trough concentrations in PEPs. By integrating covariates across various ML algorithms, it delivers results closely aligned with clinical practice, offering substantial clinical value for the guided use of VPA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b9cffac23bff7d0cc53ee324115100297e0186" target='_blank'>
              Monitoring of the trough concentration of valproic acid in pediatric epilepsy patients: a machine learning-based ensemble model
              </a>
            </td>
          <td>
            Yue-Wen Chen, Xi-Kai Lin, Si Chen, Ya-Lan Zhang, Wei Wu, Chen Huang, Xin Rao, Zong-Xing Lu, Zhou-Jie Liu
          </td>
          <td>2024-12-18</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Following the advent of biologic therapies for chronic plaque psoriasis, achieving clear/nearly clear skin (disease control) has become a realistic goal. Patients currently continue high-cost biologic therapy indefinitely once they have achieved disease control, which leads to a long-term drug and healthcare burden. Clinical trial data indicate that some individuals can maintain disease control with less frequent doses, especially those who have higher serum drug concentrations, indicating a role for therapeutic drug monitoring (TDM) in guiding dosing decisions. To enable real-world implementation of TDM-guided precision dosing of biologics, we developed, validated, and beta-tested an online interactive TDM dashboard, powered by a pharmacokinetic (PK) model. We searched for population PK models of the exemplar psoriasis biologic risankizumab in PKPDAI and PubMed databases. We externally validated the PK model using an independent multicentre real-world UK psoriasis dataset (n = 50 patients, BSTOP study) prior to embedding it within an online R Shiny interactive dashboard. Two rounds of beta-testing of the TDM dashboard were performed with healthcare professionals to assess usability. A two-compartment PK model with first-order absorption and elimination processes was identified. The PK model was based on Phase I–III trial data and adopted a nonlinear mixed effects approach (13 123 observations from 1899 patients; 71% male; median age, 47; median weight, 97 kg; Suleiman et al., Clin Pharmacokinet 2019). We integrated the model into an online R Shiny interactive dashboard, which generates a personalized dosing interval for risankizumab. The model is used as a Bayesian prior, to predict individual patient parameters using their serum drug concentrations, dose history and other covariates (weight, serum concentrations of anti-drug antibody, albumin and creatinine). Individual predictions were externally validated (mean absolute error 0.95 mg/L; mean percentage error 17.6%; root mean square error 1.7 mg/L; R2 0.8). Beta-testing sessions with 29 clinicians (58% female; mean age 41; 33% doctor; 42% nurse) from 8 UK dermatology centres showed above-average usability of the dashboard (mean System Usability Scale score 72/100, rating ‘good’). All participants evaluated the dashboard as user-friendly and rated it acceptable or completely acceptable. Mean time to generate a dosing interval using the dashboard was 1.9 min (SD 1.2). Mean time to generate a dosing interval using the dashboard was 1.9 min (SD 1.2). Our study provides a novel pipeline for the implementation of TDM-enabled precision dosing of biologics in real-world practice. Our user-friendly online TDM dashboard has the potential to incorporate other biologic therapies and can be extended across disease contexts (including non-response and other immune-mediated inflammatory diseases) for maximal real-world health benefits and cost saving.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aef75303990b8735ebd03c62ef346b126b581a9c" target='_blank'>
              FC30 Precision biologic dosing in psoriasis: development, validation and beta-testing of a therapeutic drug monitoring dashboard
              </a>
            </td>
          <td>

          </td>
          <td>2024-12-01</td>
          <td>British Journal of Dermatology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT
Different polymorphisms in genes encoding metabolizing enzymes and drug transporters have been associated with tacrolimus pharmacokinetics. In particular, studies on CYP3A4 and CYP3A5, and their combined cluster have demonstrated their significance in adjusting tacrolimus dosing to minimize under- and overexposure thereby increasing the proportion of patients who achieve tacrolimus therapeutic target. Many factors influence the pharmacokinetics of tacrolimus, contributing to inter-patient variability affecting individual dosing requirements. On the other hand, the growing use of population pharmacokinetic models in solid organ transplantation, including different tacrolimus formulations, has facilitated the integration of pharmacogenetic data and other variables into algorithms to easier implement the personalized dose adjustment in transplant centers. The future of personalized medicine in transplantation lies in implementing these models in clinical practice, with pharmacogenetics as a key factor to account for the high inter-patient variability in tacrolimus exposure. To date, three clinical trials have validated the clinical application of these approaches. The aim of this review is to provide an overview of the current studies regarding the different population pharmacokinetic including pharmacogenetics and those translated to the clinical practice for individualizing tacrolimus dose adjustment in kidney transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/917aa5ac878fa10b13074d397eeb400bec68e0a0" target='_blank'>
              Customizing Tacrolimus Dosing in Kidney Transplantation: Focus on Pharmacogenetics.
              </a>
            </td>
          <td>
            Nuria Lloberas, A. Vidal-Alabró, Helena Colom
          </td>
          <td>2024-12-10</td>
          <td>Therapeutic drug monitoring</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b8ce454573eb115134bdf921ad1f9f130c939a8" target='_blank'>
              A PopPBPK-RL approach for precision dosing of benazepril in renal impaired patients.
              </a>
            </td>
          <td>
            Guillermo Vigueras, Lucía Muñoz-Gil, Valerie Reinisch, Joana T Pinto
          </td>
          <td>2024-12-11</td>
          <td>Journal of pharmacokinetics and pharmacodynamics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3498e511e9eb5c885a5bd150e27ed48f227d9e8a" target='_blank'>
              Future Directions for Quantitative Systems Pharmacology.
              </a>
            </td>
          <td>
            Birgit Schoeberl, C. Musante, S. Ramanujan
          </td>
          <td>2025-01-16</td>
          <td>Handbook of experimental pharmacology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/616b7e29b399917646cd7667b8a390485f481cd9" target='_blank'>
              Semi-mechanistic population pharmacokinetic modeling of DZIF-10c, a neutralizing antibody against SARS-Cov-2: predicting systemic and lung exposure following inhaled and intravenous administration
              </a>
            </td>
          <td>
            Sree Kurup, Nieves Velez de Mendizabal, Stephan Becker, Erica Bolella, Dorothy De Sousa, Gerd Fätkenheuer, H. Gruell, Florian Klein, Jakob J Malin, Ulrike Schmid, Julia Korell
          </td>
          <td>2024-12-05</td>
          <td>Journal of Pharmacokinetics and Pharmacodynamics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Rheumatoid arthritis (RA) is a major public health concern, which can cause serious outcomes. Low-dose methotrexate (MTX) is a cornerstone in RA treatment, but there is significant heterogeneity in clinical response. To evaluate underlying sources of pharmacokinetic variability and clinical response of MTX, a physiologically based pharmacokinetic and pharmacodynamic (PBPK/PD) model was developed using PK-sim and Mobi (version 11.1). The PBPK model included metabolism and transportation by AXO1, FPGS, GGH, RFC, and MRP2, with renal and biliary excretion. We also developed various degrees of renal insufficiency populations with subsequent dosing optimizations. A total of 23 MTX plasma concentration-time profiles were used, with 97% of predicted plasma concentrations within a two-fold range compared to observed data. The PBPK/PD modeling and simulation demonstrated that variability in renal clearance and enzymes related to MTX are likely important drivers of PK variability and there is a quantitative relationship between MTX-PG3 and RA treatment response. The PBPK/PD model could be used to guide improvement in MTX dose regimens for RA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc030539ac38870cb0565e8afef6d1b87e1a0e58" target='_blank'>
              Comprehensive Parent-Metabolite PBPK/PD Modeling Insights Into Methotrexate Personalized Dosing Strategies in Patients With Rheumatoid Arthritis.
              </a>
            </td>
          <td>
            Xin Wang, Jiangfan Wu, Hongjiang Ye, Xiaofang Zhao, Shenyin Zhu
          </td>
          <td>2025-01-08</td>
          <td>CPT: pharmacometrics & systems pharmacology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bc8d16e0d6b207bafba23a7b60598ea67dc9939" target='_blank'>
              Comparison of the power and type 1 error of total score models for drug effect detection in clinical trials
              </a>
            </td>
          <td>
            Elham Haem, Mats O Karlsson, S. Ueckert
          </td>
          <td>2024-12-10</td>
          <td>Journal of Pharmacokinetics and Pharmacodynamics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="In general, innovation in clinical research often stems not only from aspiration – for example to increase accuracy and efficiency or to optimise costs – but also from shifts in social paradigms or from changes in circumstances that lead to changes in practices, as experienced during the COVID-19 pandemic. Indeed, the challenges clinical research faced during the pandemic led to changes in how clinical trials are conducted. Healthcare personnel had to come up with nontraditional ways to connect with their patients and patients’ families. Researchers had to develop new approaches since it was no longer possible to follow many everyday practices and because they had to answer even more complex questions – often with fewer resources. These challenges accelerated innovation in clinical research, particularly in the area of trial design. This article takes a deep dive into how design innovation has transformed clinical research by providing new operational approaches, such as those applied in decentralised clinical trials (PART 1), and new methodologies, such as those used in complex trials (PART 2).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d8b33bd05d7df4f3cde265e633454e94bef8c88" target='_blank'>
              Innovative clinical trials: Addressing the evolving needs of clinical research and participants with decentralised and complex trials
              </a>
            </td>
          <td>
            Güliz Vanli Jaccard
          </td>
          <td>2024-12-01</td>
          <td>Regulatory Affairs Watch</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Artificial Intelligence (AI) has the disruptive potential to transform patients’ lives via innovations in pharmaceutical sciences, drug development, clinical trials, and manufacturing. However, it presents significant challenges, ethical concerns, and risks across sectors and societies. AI’s rapid advancement has revealed regulatory gaps as existing public policies struggle to keep pace with the challenges posed by these emerging technologies. The term AI itself has become commonplace to argue that greater “human oversight” for “machine intelligence” is needed to harness the power of this revolutionary technology for both potential and risk management, and hence to call for more practical regulatory guidelines, harmonized frameworks, and effective policies to ensure safety, scalability, data privacy, and governance, transparency, and equitable treatment. In this review paper, we employ a holistic multidisciplinary lens to survey the current regulatory landscape with a synopsis of the FDA workshop perspectives on the use of AI in drug and biological product development. We discuss the promises of responsible data-driven AI, challenges and related practices adopted to overcome limitations, and our practical reflections on regulatory oversight. Finally, the paper outlines a path forward and future opportunities for lawful ethical AI. This review highlights the importance of risk-based regulatory oversight, including diverging regulatory views in the field, in reaching a consensus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79cd58da6a4afff75ea6786f8af76f281d5e2ff1" target='_blank'>
              Harnessing the AI/ML in Drug and Biological Products Discovery and Development: The Regulatory Perspective
              </a>
            </td>
          <td>
            Fahimeh Mirakhori, Sarfaraz K. Niazi
          </td>
          <td>2025-01-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Pharmacogenomics (PGx) holds the promise of personalizing medical treatments based on individual genetic profiles, thereby enhancing drug efficacy and safety. However, the current landscape of PGx research is hindered by fragmented data sources, time-consuming manual data extraction processes, and the need for comprehensive and up-to-date information. This study aims to address these challenges by evaluating the ability of Large Language Models (LLMs), specifically Llama3.1-70B, to automate and improve the accuracy of PGx information extraction from the FDA Table of Pharmacogenomic Biomarkers in Drug Labeling (FDA PGx Biomarker table), which is well-structured with drug names, biomarkers, therapeutic area, and related labeling texts. Our primary goal was to test the feasibility of LLMs in streamlining PGx data extraction, as an alternative to traditional, labor-intensive approaches. Llama3.1-70B achieved 91.4% accuracy in identifying drug-biomarker pairs from single labeling texts and 82% from mixed texts, with over 85% consistency in aligning extracted PGx categories from FDA PGx Biomarker table and relevant scientific abstracts, demonstrating its effectiveness for PGx data extraction. By integrating data from diverse sources, including scientific abstracts, this approach can support pharmacologists, regulatory bodies, and healthcare researchers in updating PGx resources more efficiently, making critical information more accessible for applications in personalized medicine. In addition, this approach shows potential of discovering novel PGx information, particularly of underrepresented minority ethnic groups. This study highlights the ability of LLMs to enhance the efficiency and completeness of PGx research, thus laying a foundation for advancements in personalized medicine by ensuring that drug therapies are tailored to the genetic profiles of diverse populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f15ec7badcea056da65587869b10d3ad6a587b5f" target='_blank'>
              Enhancing pharmacogenomic data accessibility and drug safety with large language models: a case study with Llama3.1
              </a>
            </td>
          <td>
            Dan Li, Leihong Wu, Ying-Chi Lin, Ho-Yin Huang, Ebony Cotton, Qi Liu, Ru Chen, Ruihao Huang, Yifan Zhang, Joshua Xu
          </td>
          <td>2024-12-03</td>
          <td>Experimental Biology and Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Personalized medicine is a new concept that transforms health care in the context of tailoring strictly towards the needs of an individual patient, using genetic, molecular, and clinical data. This new wave has been brought about by the advancement of genomics, technology, and artificial intelligence (AI). AI comes into play while analyzing massive datasets, identifying different patterns, and enabling predictive modelling for disease outcomes. Genomic medicine makes potent use of the Human Genome Project by relying on AI technologies to process and interpret enormous genetic data. AI enhances genome sequencing, thereby making it possible to be more sensitive and specific in detecting genetic variations that predispose or cause diseases. In pharmacogenomics, AI optimizes drug prescriptions based on genetic profiles to make therapies more effective and personalized. AI, particularly ML and DL, helps in the advanced genomic analysis in clinical genomics, going beyond what is allowed by the traditional bioinformatics to study genomic patterns that wouldn't have been possible under previous standards. Advanced genomic analysis paves the way for precision medicine, providing insights in terms of cancer diagnosis, optimising treatment, and prognosis by the inspection of complex genomic data including SNVs, structural variations, and CNVs. This paper briefly surveys the integration of AI in personalized medicine and genomic research; it would most likely revolutionize healthcare outcomes. As this paper surveys the following, it is apparent that including AI in personal medicine highly enhances genomic study and treatment personalization. The promise of such an advancement includes improved patient outcomes through precised diagnostics, tailored therapies, and optimized disease management strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e23a9a9e85fcc68c2a89acc1980c35ce61559268" target='_blank'>
              Artificial Intelligence in Personalized Medicine: A Survey on FuncPredict: AI Driven Genetic Variant Analysis
              </a>
            </td>
          <td>
            Dhanapandi V, Mahendran G, Nivetha S A, Sonali M, Susmitha D P
          </td>
          <td>2024-12-04</td>
          <td>2024 3rd International Conference on Automation, Computing and Renewable Systems (ICACRS)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ec59aea67c1f250bece084437afa85b16714b79" target='_blank'>
              Quantitative Systems Pharmacology Modeling in Immuno-Oncology: Hypothesis Testing, Dose Optimization, and Efficacy Prediction.
              </a>
            </td>
          <td>
            Hanwen Wang, Theinmozhi Arulraj, Alberto Ippolito, Aleksander S. Popel
          </td>
          <td>2024-12-21</td>
          <td>Handbook of experimental pharmacology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The pharmaceutical industry has increasingly adopted model-informed drug discovery and development (MID3) to enhance productivity in drug discovery and development. Quantitative systems pharmacology (QSP), which integrates drug action mechanisms and disease complexities to predict clinical endpoints and biomarkers is central to MID3. QSP modeling has proven successful in metabolic and cardiovascular diseases and has expanded into oncology, immunotherapy, and infectious diseases. Despite its benefits, QSP model validation through clinical trial simulations using virtual patients (VPs) is challenging because of parameter variability and high computational costs. To address these challenges, this study proposes a hybrid method that combines Bayesian optimization with machine learning for efficient parameter screening. Our approach achieved an acceptance rate of 27.5% in QSP simulations, which is in sharp contrast with the 2.5% rate of conventional random search methods, indicating more than 10-fold improvement in efficiency. By facilitating faster and more diverse VPs generation, this method promises to advance clinical trial simulations and accelerate drug development in pharmaceutical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b752385c900dbfc4f30d08d8dea12e0ef790e497" target='_blank'>
              Accelerating virtual patient generation with a Bayesian optimization and machine learning surrogate model.
              </a>
            </td>
          <td>
            Hiroaki Iwata, Ryuta Saito
          </td>
          <td>2024-12-04</td>
          <td>CPT: pharmacometrics & systems pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Atrial fibrillation (AF) is a common cardiac arrhythmia with high recurrence rate post-treatment. Personalised computational models can be used to investigate patient-specific mechanisms underlying AF, explore personalised treatment approaches, and for large-scale in silico trials. To ensure accuracy, reproducibility, and realistic uptake in clinical practice, investigations require consistent construction of models from patient-specific data. Clear and user-friendly model pipelines are therefore critical and should be developed with the end-user in mind. The atrial modelling toolkit (atrialmtk: https://gi thub.com/pcmlab/atrialmtk ) allows users to produce atrial meshes for electrophysiological simulation, incorporating atrial regions, fibres, and transmural variations across the atrial wall. Multiple workflows allow the use of several input data types that may be available to the user, while mesh type and fibre distribution options allow users to investigate the effect of various factors on fibril-latory dynamics. The pipeline has been tested by multiple users across operating systems to ensure compatibility and usability, and its success at scale demonstrated in a previous study of 1000 atrial geometries. By expanding testing to a wider audience, we aim to increase clinical engagement and the uptake of modelling and digital">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1f24c7942513b44e4f2779a0b70cc75eb09cd71" target='_blank'>
              Development of a User-Friendly Pipeline for Constructing Atrial Models at Scale: Importance of the End-User for Clinical Uptake
              </a>
            </td>
          <td>
            Laura Bevis, Semhar Biniam Misghina, E. Rauseo, Carlos Edgar Lopez Barrera, Gernot Plank, E. Vigmond, Axel Loewe, Elias Karabelas, Jose Alonso Solis-Lemus, Steven Niederer, Steffen E. Petersen, Gregory G Slabaugh, Anthony Mathur, Caroline H. Roney
          </td>
          <td>2024-12-01</td>
          <td>Computing in Cardiology Conference (CinC)</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Aim: The study's long-term goals, such as determining supratherapeutic ranges according to age distributions specific to the country, adjusting dosages for additional drugs used by patients in different disease groups, and providing the opportunity for etiological studies in the light of diagnosis and drug metabolism perspective, are of great importance in defining the study. Method: Population pharmacokinetics is a method expressed to evaluate processes such as absorption, distribution, metabolism, and elimination of a drug from an individual's blood-plasma concentration. In drug pharmacokinetic experiments, generating data without considering any pharmacokinetic differences among patients prevents the measurement or observation of variability among individuals in the population as a simple approach. The dose-concentration relationship is crucial for individualized dose adjustment. Additionally, the impact of other drugs used by the individual on metabolite levels and the metabolic interactions between drugs play a critical role in the development of personalized treatments. Population approaches provide a foundation that benefits the observation of these effects. The variability in drug metabolism among individuals forms one of the fundamental building blocks of personalized treatment approaches, specifically through Therapeutic Drug Monitoring (TDM), which plays an important role in determining the therapeutic range of drugs. Materials: In this study, drug metabolism findings of patients served at NP Istanbul Brain Hospital between 2010 and 2022 were examined within the repository created along with other patient-specific parameters. Results and Conclusion: The analysis results have been followed up longitudinally, partially demographically, and retrospectively. Thanks to the repository of NP Istanbul Brain Hospital, population pharmacokinetic analyses aimed in this study are being conducted for the first time globally and nationally in terms of scope. The repository has been studied with TDM for individualized treatment methods, and within this project, it is anticipated to perform phenotyping with the population pharmacokinetic approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb28f88dc070a1a3caaebf210751f1c4245347a7" target='_blank'>
              An Extensive Therapeutical Drug Monitoring Repository for Localized Population Pharmacokinetics Research
              </a>
            </td>
          <td>
            Elif Çakır, Pınar Öz, Murat Özdemir, Selma Özilhan, Nevzat Tarhan
          </td>
          <td>2024-12-01</td>
          <td>The Journal of Neurobehavioral Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Precision medicine, also known as personalized medicine, represents a promising  paradigm shift in hypertension management Precision medicine endeavours to  improve treatment effectiveness and reduce adverse effects by customizing  therapies to individual patients, considering their distinct characteristics and genetic  makeup. Over the past few years, the incorporation of artificial intelligence (AI) into  hypertension therapy has significantly advanced the principles of precision  medicine, revolutionizing the assessment, monitoring, and control of high blood  pressure. This review delves into the theoretical underpinnings of precision  medicine in hypertension treatment, elucidating the pivotal role of AI in optimizing  patient care. Moreover, it evaluates the potential benefits and drawbacks  associated with the adoption of these innovative approaches. The aim of this review  is to offer a thorough insight into how precision medicine, empowered by artificial  intelligence, is transforming the landscape of hypertension management.  Ultimately, this exploration aims to underscore the ground-breaking potential of AI  in fundamentally changing the approach to hypertension treatment, offering  insights into future directions for research and clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/927685a5d12772551703a60cececbbf7ebc3f4f8" target='_blank'>
              Revolutionizing Hypertension Management: AI-Powered Precision Medicine Approaches
              </a>
            </td>
          <td>
            Aishwarya Patel, Divyesh Kankva, Parixit Prajapati
          </td>
          <td>2024-12-15</td>
          <td>International Journal of Pharmaceutical Sciences and Nanotechnology(IJPSN)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Patient-reported outcome measures (PROMs) are vital tools in cardiovascular disease (CVD) research and care, providing insights that complement traditional clinical outcomes like mortality and morbidity. PROMs capture patient experiences with CVD, such as quality of life, functional capacity, and emotional well-being, allowing clinicians to assess how interventions impact daily life. PROMs are integral to cardiovascular investigations as well as management, especially in chronic conditions and rehabilitation, where they inform on the impact of personalized care plans by tracking symptom progression and patient adherence. Selecting and applying to cardiovascular research and practice effective PROMs involves evaluating their validity, reliability, and sensitivity to change, with instruments like the Kansas City Cardiomyopathy Questionnaire (KCCQ) and the Seattle Angina Questionnaire (SAQ) widely used for heart failure and coronary artery disease, respectively. Implementing PROMs in real-world practice requires addressing challenges related to workflow integration and patient adherence, emphasizing their value in patient-centered care. As digital health advances, remote PROM data collection through mobile applications and wearable devices will enhance access to and extent of PROMs, and AI-driven analytical tools will provide real-time, automated and plausible more poignant insights for personalized treatment. Future efforts should focus on culturally adapting PROMs for diverse populations to ensure global applicability. PROMs should also be established as essential components of innovative research and responsive, patient-centered cardiovascular care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a3a7f4b4b8dbf9591486f4162a8e4e85c5d7ab" target='_blank'>
              Patient reported outcome measures in cardiovascular research and care: PRO(M)s and CONS.
              </a>
            </td>
          <td>
            Giuseppe Biondi-Zoccai, Giacomo Frati, M. Peruzzi, M. Bernardi, Luigi Spadafora, E. Tremoli
          </td>
          <td>2025-01-23</td>
          <td>Journal of cardiovascular pharmacology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Personalized medicine is an evolving paradigm that aims to tailor therapeutic interventions to individual patient characteristics. With a growing understanding of the genetic, epigenetic, and molecular mechanisms underlying diseases, tailored therapies are becoming more feasible and effective. This review highlights the significant advancements in personalized medicine, focusing specifically on pharmacological strategies. The article explores the integration of genomics, transcriptomics, proteomics, and metabolomics in drug development and therapy optimization. Pharmacogenomics, the customization of drug therapy based on an individual's genetic makeup, receives particular emphasis. This leads to the identification of specific biomarkers that can predict therapeutic response, drug toxicity, and susceptibility to various diseases. Together with computational tools and artificial intelligence, these advancements contribute to tailored treatment plans for patients with conditions such as cancer, cardiovascular diseases, and neurological disorders. We also highlight the challenges and ethical considerations in implementing personalized medicine, such as data privacy, cost-effectiveness, and accessibility. We outline future prospects and ongoing research in this field, highlighting the importance of collaborative efforts between researchers, clinicians, pharmacists, and regulatory authorities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/605535a7297e0199c29cb0c1eb6cb59bd895c757" target='_blank'>
              Pharmacological Approaches and Innovative Strategies for Individualized Patient Care.
              </a>
            </td>
          <td>
            Amnesh Kumar Verma, Kuldeep Singh, J. K. Gupta, Shivendra Kumar, Divya Jain
          </td>
          <td>2025-01-20</td>
          <td>Recent patents on biotechnology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5f86b2198182e1b57bdd0f8d27ceac5415002e1" target='_blank'>
              Identify the underlying true model from other models for clinical practice using model performance measures
              </a>
            </td>
          <td>
            Yan Li
          </td>
          <td>2025-01-09</td>
          <td>BMC Medical Research Methodology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Everolimus presents significant dosing challenges due to between- and within-patient pharmacokinetic variabilities. This study aimed to develop and validate a model-informed precision dosing strategy for everolimus in liver transplant recipients. The dosing strategy was initially developed using retrospective data, employing nonlinear mixed-effects modeling. The model included readily measurable covariates, body surface area, albumin, and tacrolimus trough concentration. The dosing strategy was subsequently validated in a prospective trial, recommending 1 to 1.75 mg dosages every 12 h, depending on covariates. Lower dosages were recommended for patients with lower body surface area and albumin with adjustments based on tacrolimus trough concentration. The estimated pharmacokinetic parameters (typical value ± standard error), apparent clearance (CL/F: 15.0 ± 0.5 L/h), and apparent volume of distribution (Vd/F: 862 ± 79.3 L) were refined using prospective clinical data from 20 patients, reducing interindividual variations. This research successfully developed and validated a population pharmacokinetic model for everolimus. The developed "dosE" web-based platform translates our pharmacokinetic model into a practical tool for healthcare providers, exemplifying the application of pharmaceutical research in clinical practice and potentially improving therapeutic outcomes in liver transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc0083acb9d0a84c6987f4f3022cce3566fbdf3d" target='_blank'>
              Development and Clinical Validation of Model-Informed Precision Dosing for Everolimus in Liver Transplant Recipients.
              </a>
            </td>
          <td>
            Jeayoon Lee, In-Wha Kim, S. Hong, N. Han, Kyung-Suk Suh, Jung Mi Oh
          </td>
          <td>2024-12-12</td>
          <td>ACS pharmacology & translational science</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="AIMS
The accuracy of model-informed precision dosing largely depends on selecting the most appropriate population pharmacokinetic (popPK) model from many available options. This study aims to evaluate the concordance of optimal initial simulated doses among various vancomycin popPK models developed in neonates and to explore the role of predictive performance in explaining the variability in probability of target attainment (PTA).


METHODS
A virtual neonatal patient population was created and 26 previously externally evaluated vancomycin popPK models were used to simulate 5 different dosing regimens. For each simulated scenario, the area under the concentration-time curve and PTA were calculated to assess the agreement on optimal initial doses across the 26 models. A multiple regression was performed to explore the impact of the models' predictive performance on PTA.


RESULTS
For most models (15/26), there was an agreement on the optimal dosing regimen. The highest PTA being achieved by the model with the best a priori predictive performance. The multiple regression model significantly predicted mean ln-transformed PTA, with F(2, 23) = 5.406 and P = .010, yielding an adjusted R2 of .21. PTA was significantly influenced by imprecision (P = .048) but not bias (P = .469).


CONCLUSION
In conclusion, our study demonstrated that, despite the variability in bias and imprecision, there was a consensus on the initial optimal doses for the majority of models; however, models with superior a priori predictive performance yielded higher PTA values. Bias and imprecision alone only seem to predict a small proportion of the variability in PTA, with imprecision having a more pronounced effect.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0d9fa8967c6ac033a37ff1e87a5ebdebd7d04fe" target='_blank'>
              A simulation study to assess the influence of population pharmacokinetic model selection on initial dosing recommendations of vancomycin in neonates.
              </a>
            </td>
          <td>
            Mehdi El Hassani, Mathieu Blouin, A. Marsot
          </td>
          <td>2024-12-03</td>
          <td>British journal of clinical pharmacology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The pharmacokinetics of renally eliminated antibiotics can be influenced by changes associated with renal function and development in a growing subject. Little is known about the effects of renal insufficiency on the pharmacokinetics of meropenem in pediatric subjects. The aim of this study was to develop a physiologically based pharmacokinetic (PBPK) model of meropenem for pediatric patients that can be used to optimize meropenem dosing in pediatric patients with renal impairment (RI). The PBPK model was developed using GastroPlus™ 9.9 based on clinical data obtained from the literature and then scaled to pediatric patients with RI for dose optimization of meropenem. The goodness of fit of the model was assessed by comparing the predicted values of AUC0-t, AUC0-α, and Cmax with the observed data and the average fold errors (AFE). The AFE values for AUC0-t, AUC0-α, and Cmax in the pediatric population were measured to be 1.60, 1.08, and 1.48, respectively. In addition, dose optimization was performed in virtual pediatric populations with varying degrees of RI and a dose reduction to 10 mg/kg and 7.5 mg/kg was recommended for moderate and severe RI, respectively. In all virtual pediatric populations with RI, the plasma concentration reached the recommended time above the minimum inhibitory concentration (MIC) at all optimized doses. The developed PBPK model for meropenem provides a quantitative tool to assess the impact of RI on the pharmacokinetics of meropenem in pediatric patients, which may be useful for optimizing the dosing regimen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b8c26dc9cb2c89c12babc68c16da9dc0200f3fa" target='_blank'>
              Dosage Optimization Using Physiologically Based Pharmacokinetic Modeling for Pediatric Patients with Renal Impairment: A Case Study of Meropenem.
              </a>
            </td>
          <td>
            N. Rahim, Muhammad Sarfraz, A. Bello, Syed Baqir Shyum Naqvi
          </td>
          <td>2025-01-16</td>
          <td>AAPS PharmSciTech</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="OBJECTIVE
Clonidine has been widely used in the pediatric population to treat neonatal abstinence syndrome (NAS), attention deficit hyperactivity disorder (ADHD), sedation, and Tourette's syndrome; however, there is no consensus on dosing. This research aims to recommend optimal dosing of clonidine in the pediatric population using physiologically based pharmacokinetic (PBPK) modeling.


METHODS
The pediatric PBPK model was developed from an adult model by scaling the clearance processes from adults to pediatrics using ontogeny equations. The adult and pediatric models were verified using clinical PK data, and the model performance was evaluated based on visual predictive checks and absolute fold error (AFE). The final pediatric PBPK model was used to simulate clonidine PK in the virtual pediatric population. The optimal dose was recommended based on a target concentration representing clonidine's α-2 central agonist activity (EC50 = 40.5 nM).


RESULTS
The adult and pediatric models predicted well, with more than 90% of observed data captured within the 95% prediction interval of simulated data. The AFE values were within 2-fold for clonidine plasma concentrations from observed and predicted data. The pediatric simulations showed that 30 µg/kg dose orally for neonates and 0.9 mg/day orally for children (6-17 years) are optimal for achieving target concentrations for maximal α-2 adrenergic activity.


CONCLUSIONS
The pediatric PBPK model of clonidine scaled from the adult PBPK model provided optimal dosing recommendations for clonidine in different pediatric age groups. The pediatric PBPK model described in this study can be extended to other pediatric age groups and routes of administration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30281ca1f5e4f1118577666b0d517fcb789a7eb5" target='_blank'>
              Optimal Dosing Recommendations of Clonidine in Pediatrics Using Physiologically Based Pharmacokinetic Modeling.
              </a>
            </td>
          <td>
            Venkata Yellepeddi, Sharlo Bayless, Madison Parrot, Catherine M. Sherwin
          </td>
          <td>2024-12-01</td>
          <td>The journal of pediatric pharmacology and therapeutics : JPPT : the official journal of PPAG</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Antimicrobial resistance (AMR) poses a critical global health threat, necessitating the optimal use of existing antibiotics. Pharmacokinetic/pharmacodynamic (PK/PD) principles provide a scientific framework for optimizing antimicrobial therapy, particularly to respond to evolving resistance patterns. This review examines PK/PD strategies for antimicrobial dosing optimization, focusing on three key aspects. First, we discuss the importance of drug concentration management for enhancing efficacy while preventing toxicity, considering various patient populations, including pediatric and elderly patients with their unique physiological characteristics. Second, we analyze different PK modeling approaches: the classic top-down approach exemplified by population PK analysis, the bottom-up approach represented by physiologically based PK modeling, and hybrid models combining both approaches for enhanced predictive performance. Third, we explore clinical applications, including nomogram-based dosing strategies, Bayesian estimation, and emerging artificial intelligence applications, for real-time dose optimization. Critical challenges in implementing PK/PD simulation are addressed, particularly the selection of appropriate PK models, the optimization of PK/PD indices, and considerations concerning antimicrobial concentrations at infection sites. Understanding these principles and challenges is crucial for optimizing antimicrobial therapy and combating AMR through improved dosing strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/373ee52c208df4c7894e9c0219e6e75bddfbf868" target='_blank'>
              PK/PD-Guided Strategies for Appropriate Antibiotic Use in the Era of Antimicrobial Resistance
              </a>
            </td>
          <td>
            Tetsushu Onita, Noriyuki Ishihara, Takahisa Yano
          </td>
          <td>2025-01-01</td>
          <td>Antibiotics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Drug administration at similar doses in patients can often lead to various clinical responses. It has been hypothesized that genetics primarily accounts for variation in drug efficacy and toxicity in individuals. Personalized medicine has been a breakthrough achieved by the Human Genome Project which contributes to enhancing quality-based patient care. It deals with the customization of medication considering the distinct genetic and proteomic data that underpins the originality of every patient and every instance of therapeutic intervention. Many cases have shown that inter-individual differences related to drug response can be traced to genetic polymorphism in the gene alleles that codes for metabolizing enzymes, drug transporters, and genetic variations in a person’s Major Histocompatibility Complex (MHC). So, the paper throws an insight into epidemiological variations due to genetics, variations in response based on metabolic parameters, and transporters. It also covers genetic variation related to immune-related drug toxicities. Pharmacogenetic testing plays an important role in achieving more precise personalized therapeutics for better public health. The paper discusses various tests for human leukocyte antigen variants and metabolic variants. Every technology inclusion comes with advantages and limitations so personalized medicine also faces certain challenges which are discussed in the paper. Once personalized medicine is used in clinical settings, patients will be able to receive the best medications for them based on their unique genetic and protein profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c7390db50d2a6273d88f9ea7a4e94de321d7a18" target='_blank'>
              Role of Personalized Medicine in Clinical Practice: An Overview of Current and Future Perspectives
              </a>
            </td>
          <td>
            R. Ghai, Ashu Mittal, Deepali Pandey, Md Shamshir Alam, Shikha Kaushik, Pasha Ishtiyaq, Yogita Kaushik, S. Mishra
          </td>
          <td>2024-12-30</td>
          <td>Biomedical and Pharmacology Journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background and objective The growing availability of patient data from several clinical settings, fueled by advanced analysis systems and new diagnostics, presents a unique opportunity. These data can be used to understand disease progression and predict future outcomes. However, analysing this vast amount of data requires collaboration between physicians and experts from diverse fields like mathematics and engineering. Methods Mathematical models play a crucial role in interpreting patient data and enable in-silico simulations for diagnosis and treatment. To facilitate the creation and sharing of such models, the CNR-IASI BioMatLab group developed the “Gemini” (MoSpec/Autocoder) system, a framework allowing researchers with basic mathematical knowledge to quickly and correctly translate biological problems into Ordinary Differential Equations models. The system facilitates the development and computation of mathematical models for the interpretation of medical and biological phenomena, also using data from the clinical setting or laboratory experiments for parameter estimation. Results Gemini automatically generates code in multiple languages (C++, Matlab, R, and Julia) and automatically creates documentation, including code, figures, and visualizations. Conclusions This user-friendly approach promotes model sharing and collaboration among researchers, besides vastly increasing group productivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5db4a8b19a2dda0d3b5ba16f4916d27abfc74fa7" target='_blank'>
              “MoSpec”: A customized and integrated system for model development, verification and validation
              </a>
            </td>
          <td>
            M. Pompa, S. Panunzi, Alessandro Borri, Laura D'Orsi, Andrea De Gaetano
          </td>
          <td>2025-01-02</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="OBJECTIVE
This Phase I clinical trial aimed to address the knowledge gap regarding topiroxostat's use outside Japan by characterizing its pharmacokinetic profile, safety, and efficacy in healthy Chinese subjects.


METHODS
The trial followed a randomized, open-label, three-dose group design, enrolling 12 healthy participants and administering topiroxostat at three different dose levels. The study utilized NONMEM software for pharmacokinetic analysis, evaluating the impact of demographic and biochemical covariates on drug disposition.


RESULTS
Pharmacokinetic analysis shows the peak drug concentration (Cmax) under a single oral administration of 20, 40, and 80 mg of Topiroxostat, which was found in healthy subjects to be 215.46 ± 94.04 ng/mL, 473.74 ± 319.83 ng/mL and 1009.63 ± 585.98 ng/mL, respectively. The time to peak drug concentration (Tmax) was longer in females (0.79-0.98 h) than in males (0.53-0.93 h). Activated partial thromboplastin time (APTT) and triglycerides (TG) were included as covariates for the typical value of the absorption rate constant (TVKA) in our pharmacokinetic model. The dose (DOSE) was considered a covariate for the typical value of bioavailability (TVF1), and sex (SEX) was considered a covariate for the typical value of clearance (TVCL). The typical population values for topiroxostat included Q/F at 4.91 L/h, KA at 0.657 h-¹, Vc/F at 32.5 L, Vp/F at 30 L, and CL/F at 124 L/h.


CONCLUSION
The trial successfully established the pharmacokinetic parameters of topiroxostat in a Chinese population, confirming its safety and efficacy. The results support the need for individualized dosing strategies and optimize therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e4d72af0fb91cced87fb3a59ba02e684637e45e" target='_blank'>
              Characterizing Pharmacokinetic Variability of Topiroxostat in Chinese Population: Insights from a Phase I Randomized Clinical Trial.
              </a>
            </td>
          <td>
            Tianqi Zhong, Kaizong Huang, LuYao Han, Wenbo Pang, Yan Xia, Shengjun Qu, Guo Yu, Yangsheng Chen, H. Fan
          </td>
          <td>2024-12-16</td>
          <td>Current drug metabolism</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Understanding drug dosing to fulfill safety and efficacy requirements in a patient population is an essential part of dose finding in clinical practice and drug development. The majority of current dose finding methods are simulation-based, which can be time consuming and resource intensive. Model-based simulations also do not guarantee that the dose, that will optimally fulfill the safety and efficacy endpoints, will be found. In this work, an advanced dose estimation approach based on the OptiDose concept is utilized to understand therapeutic doses in a patient population and to streamline dose finding. To demonstrate the potential of this concept, two illustrative case studies are presented, each representing specific challenges for dose finding, including different safety and efficacy requirements, complex dose-response relationships, and changes of dynamics in special populations. For both applications, the distributions of therapeutic doses in the population were estimated, such that therapeutic population doses can be determined for any proportion of patients to fulfill the safety and efficacy endpoints. In addition, the therapeutic population dose with which 50% and 95% of the population will fulfill the safety and efficacy requirements was estimated. Dose estimation for both drug applications was implemented in Monolix. The presented OptiDose approach has the potential to identify the "optimal" dose for any pharmacometric and clinical pharmacology scenario, allowing to guide clinical practice and facilitate dose selection in drug development, particularly in special populations such as pediatric or cancer patients. As such, we suggest moving from current "dose simulation" approaches toward a more efficient "dose estimation" paradigm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b815b3695c9a480f54ba533a44e5c9381b271f2" target='_blank'>
              Understanding and Streamlining Dose Finding: From Dose Simulation to Dose Estimation.
              </a>
            </td>
          <td>
            D. Bräm, F. Bachmann, J. Schropp, Verena Gotta, Britta Steffens, J. van den Anker, M. Pfister, Gilbert Koch
          </td>
          <td>2025-01-19</td>
          <td>Journal of clinical pharmacology</td>
          <td>0</td>
          <td>4</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [17, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>