<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              C72b39aad7b09c50a4e21ff0614a4fd0f74061e5
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-20 11:43:52 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>A systems biology approach to model neural stem cell regulation by notch, shh, wnt, and EGF signaling pathways.</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="Recent advancements in machine learning-based data processing techniques have facilitated the inference of gene regulatory interactions and the identification of key genes from multidimensional gene expression data. In this study, we applied a stepwise Bayesian framework to uncover a novel regulatory component involved in differentiation of specific neural and neuronal cells. We treated naive neural precursor cells with Sonic Hedgehog (Shh) at various concentrations and time points, generating comprehensive whole-genome sequencing data that captured dynamic gene expression profiles during differentiation. The genes were categorized into 224 subsets based on their expression profiles, and the relationships between these subsets were extrapolated. To accurately predict gene regulation among subsets, known networks were used as a core model and subsets to be added were tested stepwise. This approach led to the identification of a novel component involved in neural tube patterning within gene regulatory networks (GRNs), which was experimentally validated. Our study highlights the effectiveness of in silico modeling for extrapolating GRNs during neural development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c4c4e2c6b28e65474d7f5776d1bf876b03e2372" target='_blank'>Stepwise Bayesian Machine Learning Uncovers a Novel Gene Regulatory Network Component in Neural Tube Development</a></td>
          <td>
            Chen Xing, Y. Sakumura, Toshiya Kokaji, Katsuyuki Kunida, Noriaki Sasai
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Elucidation of the mechanisms of stem cell differentiation is of great scientific interest. Increasing evidence suggests that stem cell differentiation involves changes at multiple levels of biological regulation, which together orchestrate the complex differentiation process; many related studies have been performed to investigate the various levels of regulation. The resulting valuable data, however, remain scattered. Most of the current stem cell-relevant databases focus on a single level of regulation (mRNA expression) from limited stem cell types; thus, a unifying resource would be of great value to compile the multiple levels of research data available. Here we present a database for this purpose, SyStemCell, deposited with multi-level experimental data from stem cell research. The database currently covers seven levels of stem cell differentiation-associated regulatory mechanisms, including DNA CpG 5-hydroxymethylcytosine/methylation, histone modification, transcript products, microRNA-based regulation, protein products, phosphorylation proteins and transcription factor regulation, all of which have been curated from 285 peer-reviewed publications selected from PubMed. The database contains 43,434 genes, recorded as 942,221 gene entries, for four organisms (Homo sapiens, Mus musculus, Rattus norvegicus, and Macaca mulatta) and various stem cell sources (e.g., embryonic stem cells, neural stem cells and induced pluripotent stem cells). Data in SyStemCell can be queried by Entrez gene ID, symbol, alias, or browsed by specific stem cell type at each level of genetic regulation. An online analysis tool is integrated to assist researchers to mine potential relationships among different regulations, and the potential usage of the database is demonstrated by three case studies. SyStemCell is the first database to bridge multi-level experimental information of stem cell studies, which can become an important reference resource for stem cell researchers. The database is available at http://lifecenter.sgst.cn/SyStemCell/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/231d6fb2a99d6dae3b68122f4268c74aee31249c" target='_blank'>SyStemCell: A Database Populated with Multiple Levels of Experimental Data from Stem Cell Differentiation Research</a></td>
          <td>
            Jian Yu, Xiaobin Xing, Lingyao Zeng, Jiehuan Sun, Wei Li, Han Sun, Ying He, Jing Li, Guoqing Zhang, Chuan Wang, Yixue Li, Lu Xie
          </td>
          <td>2012-07-13</td>
          <td>PLoS ONE</td>
          <td>12</td>
          <td>35</td>
        </tr>

        <tr id="In May this year, Stockholm hosted a Keystone Symposium on Adult Neurogenesis, attracting scientists from around the world despite the lack of customary snow. The symposium offered an extraordinary program, covering diverse topics that ranged from the neural stem cell lineage and regulation of neurogenesis to functional aspects of neurogenesis in homeostasis and disease, and even computational modeling. This Meeting Review describes some of the exciting presentations and emerging themes from the symposium, which reveal how much this young field has matured.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce0dc6536dbbe98ae29901e5f27e3536f2edcb21" target='_blank'>Adult neurogenesis: taking stock in Stockholm</a></td>
          <td>
            Nambirajan Govindarajan, G. Kempermann
          </td>
          <td>2014-10-01</td>
          <td>Development</td>
          <td>1</td>
          <td>94</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b819b59714b1800d4f5b5c0478318b9c27f766de" target='_blank'>A Generalized Gene-Regulatory Network Model of Stem Cell Differentiation for Predicting Lineage Specifiers</a></td>
          <td>
            S. Okawa, Sarah Nicklas, Sascha Zickenrott, J. Schwamborn, A. del Sol
          </td>
          <td>2016-08-18</td>
          <td>Stem Cell Reports</td>
          <td>40</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56469def4a189e7b2e7f86809eb4689acecb0eb1" target='_blank'>Odefy -- From discrete to continuous models</a></td>
          <td>
            J. Krumsiek, Sebastian Pölsterl, Dominik M. Wittmann, Fabian J Theis
          </td>
          <td>2010-05-07</td>
          <td>BMC Bioinformatics</td>
          <td>115</td>
          <td>98</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62eafbe01f4eeaf573d3daac5a39118bde6bade3" target='_blank'>Exploring Notch Pathway to Elucidate Phenotypic Plasticity and Intra-tumor Heterogeneity in Gliomas</a></td>
          <td>
            S. Chowdhury, R. Sarkar
          </td>
          <td>2019-07-01</td>
          <td>Scientific Reports</td>
          <td>5</td>
          <td>21</td>
        </tr>

        <tr id="Experimental data from biological pathways come in many forms: qualitative or quantitative, static or dynamic. By combining a variety of these heterogeneous sources of data, we construct a mathematical model of a critical regulatory network in vertebrate development, the Sonic Hedgehog signaling pathway. The structure of our model is first constrained by several well-established pathway interactions. On top of this, we develop a hierarchical genetic algorithm that is capable of integrating different types of experimental data collected on the pathway's function, including qualitative as well as static and dynamic quantitative data, in order to estimate model parameters. The result is a dynamical model that fits the observed data and is robust to perturbations in its parameters. Since it is based on a canonical power-law representation of biochemical pathways whose parameters can be directly translated into physical interactions between network components, our model provides insight into the nature and strength of pathway interactions and suggests directions for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524cf3385ed32f6c0d64bd0f4140c26a6f4ddbb7" target='_blank'>Using heterogeneous data sources in a systems biology approach to modeling the Sonic Hedgehog signaling pathway.</a></td>
          <td>
            E. Boykin, W. O. Ogle
          </td>
          <td>2010-10-01</td>
          <td>Molecular bioSystems</td>
          <td>6</td>
          <td>21</td>
        </tr>

        <tr id="The segmentation of the vertebrate body is laid down during early embryogenesis. The formation of signaling gradients, the periodic expression of genes of the Notch-, Fgf- and Wnt-pathways and their interplay in the unsegmented presomitic mesoderm (PSM) precedes the rhythmic budding of nascent somites at its anterior end, which later develops into epithelialized structures, the somites. Although many in silico models describing partial aspects of somitogenesis already exist, simulations of a complete causal chain from gene expression in the growth zone via the interaction of multiple cells to segmentation are rare. Here, we present an enhanced gene regulatory network (GRN) for mice in a simulation program that models the growing PSM by many virtual cells and integrates WNT3A and FGF8 gradient formation, periodic gene expression and Delta/Notch signaling. Assuming Hes7 as core of the somitogenesis clock and LFNG as modulator, we postulate a negative feedback of HES7 on Dll1 leading to an oscillating Dll1 expression as seen in vivo. Furthermore, we are able to simulate the experimentally observed wave of activated NOTCH (NICD) as a result of the interactions in the GRN. We esteem our model as robust for a wide range of parameter values with the Hes7 mRNA and protein decays exerting a strong influence on the core oscillator. Moreover, our model predicts interference between Hes1 and HES7 oscillators when their intrinsic frequencies differ. In conclusion, we have built a comprehensive model of somitogenesis with HES7 as core oscillator that is able to reproduce many experimentally observed data in mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0353f62181dd018ca5a1ac92fe20f85da4037ad" target='_blank'>From Dynamic Expression Patterns to Boundary Formation in the Presomitic Mesoderm</a></td>
          <td>
            Hendrik B. Tiedemann, Elida Schneltzer, S. Zeiser, B. Hoesel, J. Beckers, Gerhard K. H. Przemeck, M. H. Angelis
          </td>
          <td>2012-06-01</td>
          <td>PLoS Computational Biology</td>
          <td>24</td>
          <td>59</td>
        </tr>

        <tr id="In the present study we have generated a GRN comprising the process by which neural crest stem cells develop to two types of sensory neurons (Propioceptors and Nocioceptors). We have also been able to find patterns of regulation (motifs) that act cooperatively to control such process. Surprisingly, these motifs take place in similar stages during the development of erythrocytes from hematopoietic stem cells. Regarding the complexity of the GRN found, we then used Random Boolean Networks (RBN) for this purpose, which showed key components as well as the dynamics of the process through changes in initial conditions. Finally, the motifs were reflected in the model, suggesting insights for further studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fae1438aeb103889089e7b1cf2685e551e57ce13" target='_blank'>Modeling of genetic regulatory networks in the differentiation of neural crest stem cells to sensory neurons by means of boolean networks</a></td>
          <td>
            Jorge Marcelo Aráus Patiño, H. G. Restrepo, A. G. Barrios
          </td>
          <td>2011-03-01</td>
          <td>Revista Facultad de Ingeniería Universidad de Antioquia</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Advanced bioinformatics analysis, such as systems biology (SysBio) and artificial intelligence (AI) approaches, including machine learning (ML) and deep learning (DL), is increasingly present in stem cell (SC) research. An approximate timeline on these developments and their global impact is still lacking. We conducted a scoping review on the contribution of SysBio and AI analysis to SC research and therapy development based on literature published in PubMed between 2000 and 2024. We identified an 8-10-fold increase in research output related to all three search terms between 2000 and 2021, with a 10-fold increase in AI-related production since 2010. Use of SysBio and AI still predominates in preclinical basic research with increasing use in clinically oriented translational medicine since 2010. SysBio- and AI-related research was found all over the globe, with SysBio output led by the United States (US, n=1487), United Kingdom (UK, n=1094), Germany (n=355), The Netherlands (n=339), Russia (n=215), and France (n=149), while for AI-related research the US (n=853) and UK (n=258) take a strong lead, followed by Switzerland (n=69), The Netherlands (n=37), and Germany (n=19). The US and UK are most active in SCs publications related to AI/ML and AI/DL. The prominent use of SysBio in ESC research was recently overtaken by prominent use of AI in iPSC and MSC research. This study reveals the global evolution and growing intersection between AI, SysBio, and SC research over the past two decades, with substantial growth in all three fields and exponential increases in AI-related research in the past decade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db16765f7316066330145d1b49c8618dbe1ba09e" target='_blank'>The Global Evolution and Impact of Systems Biology and Artificial Intelligence in Stem Cell Research and Therapeutics Development: A Scoping Review.</a></td>
          <td>
            Thayna Silva-Sousa, J. Usuda, N. Al-Arawe, Francisca Frias, I. Hinterseher, R. Catar, Christian Luecht, Katarina Riesner, Alexander Hackel, L. Schimke, Haroldo Dutra Dias, I. Filgueiras, Helder I Nakaya, N. O. S. Câmara, Stefan Fischer, G. Riemekasten, Olle Ringdén, O. Penack, Tobias Winkler, Georg Duda, D. Fonseca, O. Cabral-Marques, Guido Moll
          </td>
          <td>2024-09-04</td>
          <td>Stem cells</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="SUMMARY
The characterization of the complex phenomenon of cell differentiation is a key goal of both systems and computational biology. GeStoDifferent is a Cytoscape plugin aimed at the generation and the identification of gene regulatory networks (GRNs) describing an arbitrary stochastic cell differentiation process. The (dynamical) model adopted to describe general GRNs is that of noisy random Boolean networks (NRBNs), with a specific focus on their emergent dynamical behavior. GeStoDifferent explores the space of GRNs by filtering the NRBN instances inconsistent with a stochastic lineage differentiation tree representing the cell lineages that can be obtained by following the fate of a stem cell descendant. Matched networks can then be analyzed by Cytoscape network analysis algorithms or, for instance, used to define (multiscale) models of cellular dynamics.


AVAILABILITY
Freely available at http://bimib.disco.unimib.it/index.php/Retronet#GESTODifferent or at the Cytoscape App Store http://apps.cytoscape.org/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c443439acb720c8bdc56c858e863e13f06bb4d45" target='_blank'>GeStoDifferent: a Cytoscape plugin for the generation and the identification of gene regulatory networks describing a stochastic cell differentiation process</a></td>
          <td>
            M. Antoniotti, Gary D Bader, G. Caravagna, Silvia Crippa, Alex Graudenzi, G. Mauri
          </td>
          <td>2013-02-15</td>
          <td>Bioinformatics</td>
          <td>7</td>
          <td>101</td>
        </tr>

        <tr id="Recent efforts in systematically profiling embryonic stem (ES) cells have yielded a wealth of high‐throughput data. Complementarily, emerging databases and computational tools facilitate ES cell studies and further pave the way toward the in silico reconstruction of regulatory networks encompassing multiple molecular layers. Here, we briefly survey databases, algorithms, and software tools used to organize and analyze high‐throughput experimental data collected to study mammalian cellular systems with a focus on ES cells. The vision of using heterogeneous data to reconstruct a complete multi‐layered ES cell regulatory network is discussed. This review also provides an accompanying manually extracted dataset of different types of regulatory interactions from low‐throughput experimental ES cell studies available at http://amp.pharm.mssm.edu/iscmid/literature. Copyright © 2010 John Wiley & Sons, Inc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c2ee046360ac0bd7f578a0667427a0112bc0a36" target='_blank'>Toward a complete in silico, multi‐layered embryonic stem cell regulatory network</a></td>
          <td>
            Huilei Xu, C. Schaniel, I. Lemischka, Avi Ma’ayan
          </td>
          <td>2010-11-01</td>
          <td>Wiley Interdisciplinary Reviews: Systems Biology and Medicine</td>
          <td>30</td>
          <td>66</td>
        </tr>

        <tr id="The Hedgehog Signaling Pathway Database is a curated repository of information pertaining to the Hedgehog developmental pathway. It was designed to provide centralized access to a wide range of relevant information in an organism-agnostic manner. Data are provided for all genes and gene targets known to be involved in the Hh pathway across various organisms. The data provided include DNA and protein sequences as well as domain structure motifs. All known human diseases associated with the Hh pathway are indexed including experimental data on therapeutic agents and their molecular targets. Hh researchers will find useful information on relevant protocols, tissue cell lines and reagents used in current Hh research projects. Curated content is also provided for publications, grants and patents relating to the Hh pathway. The database can be accessed at http://www.hedgehog.sfsu.edu.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd6c413a5ec32f742669243c0bcc5c6c653be7f0" target='_blank'>Hedgehog Signaling Pathway Database: a repository of current annotation efforts and resources for the Hh research community</a></td>
          <td>
            Kieran Hervold, Andrew Martin, Roger A. Kirkpatrick, P. Kenna, F. Ramirez-Weber
          </td>
          <td>2006-12-06</td>
          <td>Nucleic Acids Research</td>
          <td>3</td>
          <td>11</td>
        </tr>

        <tr id="Understanding the molecular basis of stem cell (SC) function is essential to regenerate damaged tissue due to various types of injuries and pathologies. However, there are bottlenecks that prevent the use of Stem cell in the treatment of wide range of diseases. Complete information regarding the control of gene expression in Stem cell is necessary to understand the regulation of Stem cell fates such as self-renewal, differentiation, migration/ homing and apoptosis. However, in the absence of multi-functional, single window tool for the analysis of various omics and drugs data related to Stem cell, the knowledge consolidation is missing to fully harness its potential. A tool which can integrate various databases, analysis software and gives a probable solution for Stem cell mediated regeneration of tissues using available therapeutic options would likely to help in reducing morbidity and mortality associated with various pathologies. This subsumes omics based repurposing of drugs for Stem cell guided tissue regeneration, which will be a boon for clinicians to treat various debilitating diseases/ malfunctions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2c7100a9e1a50c9f9048f3539c7a6204fd61fe4" target='_blank'>A Hypothetical Tool for Integrated Omics, Drugs and Clinical Stem Cell Data Analysis</a></td>
          <td>
            Y. Verma, G. Gangenahalli
          </td>
          <td>2017-08-03</td>
          <td>life science journal</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The molecular components governing neural induction remain largely unknown. Here, we applied a suite of genomic and computational tools to comprehensively identify these components. We performed RNA-seq, ChIP-seq (H3K27ac, H3K27me3) and ATAC-seq on human embryonic stem cells (hESCs) at seven early neural differentiation time points (0-72 hours) and identified thousands of induced genes and regulatory regions. We analyzed the function of ~2,500 selected regions using massively parallel reporter assays at all time points. We found numerous temporal enhancers that correlated with similarly timed epigenetic marks and gene expression. Development of a prioritization method that incorporated all genomic data identified key transcription factors (TFs) involved in neural induction. Individual overexpression of eleven TFs and several combinations in hESCs found novel neural induction regulators. Combined, our results provide a comprehensive map of genes and functional regulatory elements involved in neural induction and identify master regulator TFs that are instrumental for this process. One Sentence Summary Using numerous genomic assays and computational tools we characterized the dynamic changes that take place during neural induction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cc5da55c4a7da9e729c7b17e1eb31edf95c272e" target='_blank'>Massively parallel characterization of regulatory dynamics during neural induction</a></td>
          <td>
            Fumitaka Inoue, Anat Kreimer, Tal Ashuach, N. Ahituv, N. Yosef
          </td>
          <td>2018-07-16</td>
          <td>bioRxiv</td>
          <td>5</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e7126777a5e77b2a5cc673f74da0828568a9217" target='_blank'>A systems biology pipeline identifies regulatory networks for stem cell engineering</a></td>
          <td>
            Melissa A. Kinney, Linda T. Vo, Jenna M. Frame, Jessica Barragan, Ashlee J Conway, Shuai Li, Kwok-kin Wong, J. Collins, Patrick Cahan, T. North, D. Lauffenburger, G. Daley
          </td>
          <td>2019-07-01</td>
          <td>Nature Biotechnology</td>
          <td>17</td>
          <td>124</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7676a0cf09740444102cf705ae30fc2a51e2b089" target='_blank'>Neural tube patterning: from a minimal model for rostrocaudal patterning toward an integrated 3D model</a></td>
          <td>
            Max Brambach, A. Ernst, Sara Nolbrant, J. Drouin-Ouellet, A. Kirkeby, M. Parmar, V. Olariu
          </td>
          <td>2020-10-02</td>
          <td>iScience</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Stem cells offer great promise within the field of regenerative medicine but despite encouraging results, the large scale use of stem cells for therapeutic applications still faces challenges when it comes to controlling signaling pathway responses with respect to environmental perturbations. This step is critical for the elaboration of stable and reproducible differentiation protocols, and computational modeling can be helpful to overcome these difficulties. This article is a comparative review of the mechanism-based methods used for hypothesis-driven approaches and data-driven methods which are two types of computational approaches commonly used for analysing the dynamics of pathways involved in stem cell regulation. We firstly review works based on kinetic modelling. We emphasize the relationships between the dynamics of these pathways and their topological features, and illustrative examples are described to show how the analysis of these relationships can contribute to a more detailed and formal understanding of the signaling dynamics. This discussion is followed by a review of the recent data-driven pathway analysis methods. Based on a simplified description of the pathways, these methods are able to handle high dimensionality data, and topological features of the pathways taken into account in the latest methods improve both accuracy and predictive power. Nevertheless, progress is still needed to clarify the biological meaning of the topological decompositions used by these methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca59937c82be584e209108e87cfb84a136ac2d47" target='_blank'>A comparative review of computational methods for pathway perturbation analysis: dynamical and topological perspectives.</a></td>
          <td>
            Q. Vanhaelen, A. Aliper, Alex Zhavoronkov
          </td>
          <td>2017-08-22</td>
          <td>Molecular bioSystems</td>
          <td>11</td>
          <td>37</td>
        </tr>

        <tr id="Proper tissue development requires that stem/progenitor cells precisely coordinate cell division and differentiation in space and time. Notch-Hes1 intercellular signaling, which affects both differentiation and cell cycle progression and directs cell fate decisions at various developmental stages in many cell types, is central to this process. This study explored whether the pattern of connections among the cell cycle regulatory module, the Notch effector Hes1 and the proneural factor Ngn2 could explain salient aspects of cell fate determination in neural progenitors. A mathematical model that includes mutual interactions between Hes1, Ngn2 and G1-phase regulators was constructed and simulated at the single- and two-cell levels. By differentially regulating G1-phase progression, Hes1 and Ngn2 are shown to induce two contrasting cell cycle arrest states in early and late G1, respectively. Indeed, steady Hes1 overexpression promotes reversible quiescence by downregulating activators of G0/G1 exit and Ngn2. Ngn2 also downregulates activators of G0/G1 exit, but cooperates with Cip/Kip proteins to prevent G1/S transit, whereby it promotes G1-phase lengthening and, ultimately, contributes to reinforcing an irreversible late G1 arrest coincident with terminal differentiation. In this scheme, Hes1 oscillation in single cells is able to maintain a labile proliferation state in dynamic balance with two competing cell fate outputs associated with Hes1-mediated and Ngn2-mediated cell cycle arrest states. In Delta/Notch-connected cells, Hes1 oscillations and a lateral inhibition mechanism combine to establish heterogeneous Hes1, Ngn2 and cell cycle dynamics between proliferating neural progenitors, thereby increasing the chances of asymmetric cell fate decisions and improving the reliability of commitment to differentiation. Summary: Computational modelling deciphers the connections between the cell-cycle machinery and Notch signalling which govern whether neural stem cells proliferate, differentiate or enter quiescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a990d80c915ac6d6a95aec25582fc283f7d85a5b" target='_blank'>A computational model for the coordination of neural progenitor self-renewal and differentiation through Hes1 dynamics</a></td>
          <td>
            Benjamin Pfeuty
          </td>
          <td>2015-02-01</td>
          <td>Development</td>
          <td>34</td>
          <td>15</td>
        </tr>

        <tr id="The Notch signaling pathway controls numerous cell fate decisions during development and adulthood through diverse mechanisms. Thus, whereas it functions as an oscillator during somitogenesis, it can mediate an all-or-none cell fate switch to influence pattern formation in various tissues during development. Furthermore, while in some contexts continuous Notch signaling is required, in others a transient Notch signal is sufficient to influence cell fate decisions. However, the signaling mechanisms that underlie these diverse behaviors in different cellular contexts have not been understood. Notch1 along with two downstream transcription factors hes1 and RBP-Jk forms an intricate network of positive and negative feedback loops, and we have implemented a systems biology approach to computationally study this gene regulation network. Our results indicate that the system exhibits bistability and is capable of switching states at a critical level of Notch signaling initiated by its ligand Delta in a particular range of parameter values. In this mode, transient activation of Delta is also capable of inducing prolonged high expression of Hes1, mimicking the “ON” state depending on the intensity and duration of the signal. Furthermore, this system is highly sensitive to certain model parameters and can transition from functioning as a bistable switch to an oscillator by tuning a single parameter value. This parameter, the transcriptional repression constant of hes1, can thus qualitatively govern the behavior of the signaling network. In addition, we find that the system is able to dampen and reduce the effects of biological noise that arise from stochastic effects in gene expression for systems that respond quickly to Notch signaling. This work thus helps our understanding of an important cell fate control system and begins to elucidate how this context dependent signaling system can be modulated in different cellular settings to exhibit entirely different behaviors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5311ceab2031fba500a30a4ec23343a0d1310db" target='_blank'>Computational Models of the Notch Network Elucidate Mechanisms of Context-dependent Signaling</a></td>
          <td>
            S. Agrawal, C. Archer, D. Schaffer
          </td>
          <td>2009-05-01</td>
          <td>PLoS Computational Biology</td>
          <td>72</td>
          <td>83</td>
        </tr>

        <tr id="The past 20 years have witnessed a considerable interest in the aging phenomenon among researchers in varied fields of biology and medicine. The Buck Institute is currently in a phase of growth, and a major focus will be to expand stem cell research. There are two main reasons, and the first is obvious. It is increasingly apparent that stem cells, either embryonic, adult or induced pluripotent, offer exciting possibilities to develop therapeutic approaches to the diseases of aging. The Institute already has programs to explore the potential of stem cells in the treatment of a range of neurodegenerative disorders, including Parkinson, Huntington and Alzheimer diseases. With respect to Parkinson disease, Dr. Xianmin Zeng is developing clinical-grade human dopaminergic neurons with the intent to replace those that are lost with disease progression. Furthermore, Dr. Deepak Lamba, who has just joined the Buck Institute faculty, will bring an exciting program to develop stem cell therapeutic approaches to treat macular degeneration, which has a major impact on the quality of life of elders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/906f07d2f0ee3459a2799c875d85e24d8b242691" target='_blank'>Computational approaches for the transcriptomics, proteomics and epigenetic analysis in adult stem cells</a></td>
          <td>
            King Jordan
          </td>
          <td>2011-12-15</td>
          <td>Cell Cycle</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cell phenotypic transitions, or cell fate decision making processes, are regulated by complex regulatory networks composed of genes, RNAs, proteins and metabolites. The regulation can take place at the epigenetic, transcriptional, translational, and post-translational levels to name a few. Epigenetic histone modification plays an important role in cell phenotype maintenance and transitions. However, the underlying mechanism relating dynamical histone modifications to stable epigenetic cell memory remains elusive. Incorporating key pieces of molecular level experimental information, we built a statistical mechanics model for the inheritance of epigenetic histone modifications. The model reveals that enzyme selectivity of different histone substrates and cooperativity between neighboring nucleosomes are essential to generate bistability of the epigenetic memory. We then applied the epigenetic modeling framework to the differentiation process of olfactory sensory neurons (OSNs), where the observed ‘one-neuron-one-allele’ phenomenon has remained as a long-standing puzzle. Our model successfully explains this singular behavior in terms of epigenetic competition and enhancer cooperativity during the differentiation process. Epigenetic level events and transcriptional level events cooperate synergistically in the OSN differentiation process. The model also makes a list of testable experimental predictions. In general, the epigenetic modeling framework can be used to study phenotypic transitions when histone modification is a major regulatory element in the system. Post-transcriptional level regulation plays important roles in cell phenotype maintenance. Our integrated experimental and computational studies revealed such a motif regulating the differentiation of definitive endoderm. We identified two RNA binding proteins, hnRNPA1 and KSRP, which repress each other through microRNAs miR-375 and miR-135a. The motif can generate switch behavior and serve as a noise filter in the stem cell differentiation process. Manipulating the motif could enhance the differentiation efficiency toward a specific lineage one desires. Last we performed mathematical modeling on an epithelial-to-mesenchymal transition (EMT) process, which could be used by tumor cells for their migration. Our model predicts that the IL-6 induced EMT is a stepwise process with multiple intermediate states. In summary, our theoretical and computational analyses about cell phenotypic transitions provide novel insights on the underlying mechanism of cell fate decision. The modeling studies revealed general physical principles underlying complex regulatory networks. Theoretical and Computational Studies on the Dynamics and Regulation of Cell Phenotypic Transitions Hang Zhang General Audience Abstract The human body is composed of more than 200 different types of cells, characteristic of muscles, skin, brain, etc. Although different types of cells exhibit many differences in morphology and other properties, they all originate from a single fertilized egg. When a cell divides, the new cell has the potential either to maintain its own cell type or become another type of cell. For example, an unspecialized stem cell can produce another stem cell or develop into other types of cell with specialized functions. The transition between different cell types is called cell phenotypic transition. Cell phenotypic transitions are commonly observed in living organisms. For example, olfactory stem cells can transform to olfactory sensory neuron (OSN) cells. As a result, each olfactory sensory neuron (OSN) is specialized to expresses one and only one type of olfactory receptor (OR) protein on the cell surface, and by this mechanism, the human nose can sense trillions of different odors. As a second example, epithelial cells can convert to mesenchymal cells in human development, cancer development and wound healing. Epithelial cells are tightly connected to each other and to the basement, thus hardly have the ability to move. On the other hand, mesenchymal cells are less connected and more easily to migrate. Epithelial cancers are the most prevalent type of cancer. During cancer metastasis, epithelial cells transform into mesenchymal cells and gain the ability to move to a distant positions. Epithelial to mesenchymal transition (EMT) is not only utilized by cancer cells; normal cell will also utilize this process to move. In human development and wound healing, epithelial cells will transform to mesenchymal cells to migrate to the desired locations. Despite intensive studies, the underlying regulatory mechanisms of cell phenotypic transitions remain largely elusive. In this dissertation, I present a theoretical framework to elucidate some underlying mechanisms of cell-type maintenance and transitions. Based on experimentally observed physical interactions, we built a statistical mechanics model which successfully explained the mechanism of cell type maintenance. As an application, I used this model to study the differentiation process of OSNs. Each OSN cell randomly expresses one and only one OR gene from thousands of candidates with equal probability, however, the mechanism remains one of the biggest puzzles in neuroscience. Our model not only recapitulated singular OR expression, but also elucidated how the olfactory system maximizes and maintains the diversity of OR expression. In addition to the differentiation of OSNs, I also studied the differentiation of embryonic stem cell in human development. Previous studies mainly focused on protein signaling networks regulating this process, our studies revealed a novel regulatory mechanism with microRNA (an important type of regulatory RNA) playing a crucial role. Manipulating the network could enhance the efficiency of differentiation toward a specific lineage that one might desire. Lastly, I examined a mathematical model of the EMT process. The model revealed potential relationships between inflammation and tumor metastasis regarding molecule IL-6 induced EMT. Here, our model predicted that IL-6 induced EMT is a stepwise process with multiple intermediate states and helped us to fully characterize the tumor progression process. Overall, the broad goal of this research is to advance our understanding of the underlying mechanisms of cell phenotypic transitions. It also sheds light on the processes of tissue regeneration and cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea1bfdae2ce3bed82dd02489f402e7ef419d03e" target='_blank'>Theoretical and Computational Studies on the Dynamics and Regulation of Cell Phenotypic Transitions</a></td>
          <td>
            Hang Zhang
          </td>
          <td>2016-04-18</td>
          <td></td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Stem cells are characterized by their potential for self-renewal and their capacity to differentiate into mature cells. These two key features emerge through the interplay of various factors within complex molecular networks. To provide researchers with a dedicated tool to investigate these networks, we have developed StemCellNet, a versatile web server for interactive network analysis and visualization. It rapidly generates focused networks based on a large collection of physical and regulatory interactions identified in human and murine stem cells. The StemCellNet web-interface has various easy-to-use tools for selection and prioritization of network components, as well as for integration of expression data provided by the user. As a unique feature, the networks generated can be screened against a compendium of stemness-associated genes. StemCellNet can also indicate novel candidate genes by evaluating their connectivity patterns. Finally, an optional dataset of generic interactions, which provides large coverage of the human and mouse proteome, extends the versatility of StemCellNet to other biomedical research areas in which stem cells play important roles, such as in degenerative diseases or cancer. The StemCellNet web server is freely accessible at http://stemcellnet.sysbiolab.eu.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65dff5cdc834c36f05d90a6ae7c76b1756f4ca06" target='_blank'>StemCellNet: an interactive platform for network-oriented investigations in stem cell biology</a></td>
          <td>
            J. Pinto, R. Kalathur, Rui S. R. Machado, J. Xavier, J. Bragança, M. Futschik
          </td>
          <td>2014-05-22</td>
          <td>Nucleic Acids Research</td>
          <td>15</td>
          <td>35</td>
        </tr>

        <tr id="We propose a new approach of modeling and simulation for analyzing multicellular biological events by combining colored hybrid Petri nets (CoIHPNs) with newly devised biological experiments that can control the level of a gene quantitatively. We analyzed the patterning of boundary cells in the Drosophila large intestine, where one-cell-wide domain of boundary cells differentiate through Delta-Notch signaling. Biological experiments regulating the level of Delta result in six distinct patterns of boundary cells corresponding with the level of Delta. All these patterns were successfully reproduced by simulation based on ColHPN modeling only by changing the parameter related to the level of Delta. By monitoring the concentration of the active form of Notch in each cell during simulation, it was revealed that these distinct modes of patterning correlate with the fluctuation range of active Notch. Combination of simulation and quantitative manipulation of a gene activity described here is a reliable and powerful approach for analyzing and understanding the patterning process regulated by Notch signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9a945b5febf9353ec1080efe46aad684527bb6c" target='_blank'>Pattern formation analysis by colored Petri nets with quantitative manipulation of a gene</a></td>
          <td>
            Fei Liu, Ena Yamamoto, Katsunobu Shirahama, T. Saitoh, Shuhei Aoyama, Yumiko Harada, R. Murakami, H. Matsuno
          </td>
          <td>2019-11-01</td>
          <td>2019 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="In the developing nervous system, the expression of proneural genes, i.e., Hes1, Neurogenin-2 (Ngn2), and Deltalike-1 (Dll1), oscillates in neural progenitors with a period of 2–3 h, but is persistent in postmitotic neurons. In this paper, we present a computational model for neural fate decisions based on intertwined Notch-Delta signaling involving the Hes1, Notch, and Dll1 proteins. In agreement with experimental observations, the model predicts that Notch-Delta signaling plays critical roles in regulating the choice between remaining as a progenitor and embarking on neural differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de920b268ed7567709f882075cd7d0980535baa9" target='_blank'>Neural fate decisions mediated by notch-delta signaling</a></td>
          <td>
            Ruiqi ang, Kaihui Liu, Luonan Chen
          </td>
          <td>2011-10-03</td>
          <td>2011 IEEE International Conference on Systems Biology (ISB)</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f57191a737642be345b8201347861089d57cae43" target='_blank'>In-silico models of stem cell and developmental systems</a></td>
          <td>
            Yaki Setty
          </td>
          <td>2014-01-08</td>
          <td>Theoretical Biology & Medical Modelling</td>
          <td>18</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f57191a737642be345b8201347861089d57cae43" target='_blank'>In-silico models of stem cell and developmental systems</a></td>
          <td>
            Yaki Setty
          </td>
          <td>2014-01-08</td>
          <td>Theoretical Biology & Medical Modelling</td>
          <td>18</td>
          <td>7</td>
        </tr>

        <tr id="Addressing the high mortality rate and limited treatment options for oral cancer is a global public health challenge. To enhance therapeutic approaches, a comprehensive understanding of oral cancer stem cells (OCSCs) is essential. This review employs advanced bio-inspired methods to identify unique pharmacological targets within OCSCs, considering their heterogeneity and resistance to conventional treatments. The Evolutionary Computation Network for Drug Repositioning (ECN-DR) dissects the intricate signaling pathways and molecular networks within OCSCs using bio-inspired techniques. By integrating machine learning, network analysis, and molecular dynamics simulations, this approach identifies potential targets for both new and existing anticancer drugs. Recognizing the key molecular players in OCSCs enables the design of tailored medicines to disrupt these cells, offering more potent and targeted therapy options with fewer side effects. Molecular dynamics simulations, protein-ligand docking studies, and in silico drug screening predict the binding affinity and therapeutic potential of prospective medications against selected OCSC targets. These simulations contribute to a better understanding of targeting specific proteins in oral cancer therapy. Utilizing bio-inspired methods and computational simulations enhances our knowledge of OCSC biology, advancing the prospects of personalized cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6320190c652e5fb7447f17791436de51a89b5051" target='_blank'>Discovering Novel and Old Drug Targets in Oral Cancer Stem Cells Using bio-inspired methods - A Review</a></td>
          <td>

          </td>
          <td>2023-12-23</td>
          <td>Journal of Angiotherapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Recent studies have associated the transcription factors, Oct4, Sox2 and Nanog as parts of a self-regulating network which is responsible for maintaining embryonic stem cell properties: self renewal and pluripotency. In addition, mutual antagonism between two of these and other master regulators have been shown to regulate lineage determination. In particular, an excess of Cdx2 over Oct4 determines the trophectoderm lineage whereas an excess of Gata-6 over Nanog determines differentiation into the endoderm lineage. Also, under/over-expression studies of the master regulator Oct4 have revealed that some self-renewal/pluripotency as well as differentiation genes are expressed in a biphasic manner with respect to the concentration of Oct4. Methodology/Principal Findings We construct a dynamical model of a minimalistic network, extracted from ChIP-on-chip and microarray data as well as literature studies. The model is based upon differential equations and makes two plausible assumptions; activation of Gata-6 by Oct4 and repression of Nanog by an Oct4–Gata-6 heterodimer. With these assumptions, the results of simulations successfully describe the biphasic behavior as well as lineage commitment. The model also predicts that reprogramming the network from a differentiated state, in particular the endoderm state, into a stem cell state, is best achieved by over-expressing Nanog, rather than by suppression of differentiation genes such as Gata-6. Conclusions The computational model provides a mechanistic understanding of how different lineages arise from the dynamics of the underlying regulatory network. It provides a framework to explore strategies of reprogramming a cell from a differentiated state to a stem cell state through directed perturbations. Such an approach is highly relevant to regenerative medicine since it allows for a rapid search over the host of possibilities for reprogramming to a stem cell state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d51292c18b3d5ff73e18e3b9496ccf5afde33d08" target='_blank'>A Computational Model for Understanding Stem Cell, Trophectoderm and Endoderm Lineage Determination</a></td>
          <td>
            V. Chickarmane, C. Peterson
          </td>
          <td>2008-10-22</td>
          <td>PLoS ONE</td>
          <td>147</td>
          <td>28</td>
        </tr>

        <tr id="Cell signaling networks regulate a variety of developmental and physiological processes, and changes in their response to external stimuli are often implicated in disease initiation and progression. To elucidate how different responses can arise from conserved signaling networks, we have developed a mathematical model of the well-characterized Caenorhabditis vulval development network involving EGF, Wnt and Notch signaling that recapitulates biologically observed behaviors. We experimentally block a specific element of the EGF pathway (MEK), and find different behaviors in vulval development in two Caenorhabditis species, C. elegans and C. briggsae. When we separate our parameters into subsets that correspond to these two responses, they yield model behaviors that are consistent with observed experimental results, despite the initial parameter grouping based on perturbation in a single node of the EGF pathway. Finally, our analysis predicts specific parameters that may be critical for the theoretically and experimentally observed differences, suggesting modifications that might allow intentional switching between the two species' responses. Our results indicate that all manipulations within a signal transduction pathway do not yield the same outcome, and provide a framework to identify the specific genetic perturbations within a conserved network that will confer unique behaviors on the network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b2aeb0aa34ca2ee9a27cbf7f3f7ae8392edf79c" target='_blank'>A computational model predicts genetic nodes that allow switching between species-specific responses in a conserved signaling network.</a></td>
          <td>
            Adriana T. Dawes, David Wu, Karley K. Mahalak, Edward Zitnik, Natalia Kravtsova, Haiwei Su, H. Chamberlin
          </td>
          <td>2017-02-20</td>
          <td>Integrative biology : quantitative biosciences from nano to macro</td>
          <td>5</td>
          <td>20</td>
        </tr>

        <tr id="Recently, the field of computational stem cell biology, a new subdiscipline of developmental biology, has received a remarkable level of attention from the researchers. It leverages computational algorithms and tools to study the mechanisms of regulating the behaviors of embryonic stem cells (ESCs). In particular, the invention of induced pluripotent stem cells (iPSCs) in 2006 raised a question of intense focus: “What are the underlying molecular mechanisms by which iPSCs regain pluripotent potential?” Dynamical systems modeling has revealed the complexity of the regulation of cell fate determination and thus has proven to be a useful method for answering this question. Towards exploring the molecular basis of stem cell fate regulation, the transcriptional regulatory network of core pluripotency transcription factors (TFs) was first modeled to uncover critical network structures that contribute to the cell fate transitions. Subsequently, we integrated the epigenetic network and signaling pathways that support the maintenance of stem cells and the determination of lineage specifications with the TF-based genetic layer of regulation, thereby constructing a hybrid network model of multilayered regulations. However, most of the computational models are mainly limited to exploring the system dynamics by theoretical modeling. More recently, a trend has emerged in computational stem cell biology to incorporate genome-wide sequencing data, especially single-cell profiling, with system-level modeling of stem cells. In this thesis, we concentrate on these two computational methodologies, i.e., the knowledge-based modeling and the data-driven modeling, to facilitate the study of stem cell fate transitions using a well-known metaphor in development biology, namely Waddington’s epigenetic landscape. It describes the outcome of the percolation of genetic lineages by linking the cell phenotypes with the genotypes, through the coordination of gene regulatory networks. Thus, the trajectories of cell state transition on the landscape surface reflect the progressive dynamics of stem cell fate determination. We developed a software tool named “NetLand” to facilitate the knowledgebased modeling and simulation study of cell state transitions driven by kinetics of gene regulatory network in Waddington’s epigenetic landscape. It serves as a useful tool for understanding the dynamics of stem cells. For example, it can be used to study the barriers to the conversion of iPSCs. Although there have been many a plenty of protocols for the generation of iPSCs, the low conversion rate is still a bottleneck. To apply iPSCs in">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de42194ad103d9220e3f6e959626ea64865c6a04" target='_blank'>Mathematical modeling and simulation of stem cell differentiation and reprogramming by Waddington's epigenetic landscape</a></td>
          <td>
            Jing-Jing Guo
          </td>
          <td>2018-08-20</td>
          <td></td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17e5e8fe3ace26508d87b495850933a2fe5ef2c1" target='_blank'>Controlling neural territory patterning from pluripotency using a systems developmental biology approach</a></td>
          <td>
            Katie E. Sears, Keerthi Gullapalli, Divya Trivedi, Alexander Mihas, M. Bukys, J. Jensen
          </td>
          <td>2022-03-21</td>
          <td>iScience</td>
          <td>2</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c91dafd7ab979a95a9c5d09511bdd087cd30aa0f" target='_blank'>Emergent Stem Cell Homeostasis in the C. elegans Germline Is Revealed by Hybrid Modeling</a></td>
          <td>
            B. Hall, Nir Piterman, A. Hajnal, J. Fisher
          </td>
          <td>2015-07-21</td>
          <td>Biophysical Journal</td>
          <td>15</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f7f8366a75c4cd21ff46d3baec6acfbbf2cc2d" target='_blank'>GINsim: a software suite for the qualitative modelling, simulation and analysis of regulatory networks.</a></td>
          <td>
            A. González, A. Naldi, L. Sánchez, D. Thieffry, C. Chaouiya
          </td>
          <td>2006-05-01</td>
          <td>Bio Systems</td>
          <td>182</td>
          <td>55</td>
        </tr>

        <tr id="Neurogenesis (NG), as generation of neurons of the cortical plate (CP), includes processes of proliferation, migration and programmed cell death, which have been thoroughly studied on murine model through the series of experiments by Takahashi, Nowakowski, Caviness et al. Simple 11 cell cycle mathematical model of NG within pseudostratified ventricular epithelia (PVE) has been proposed and magnitude of the programmed cell death in CP during early postnatal life has been estimated. We developed system dynamics model of the NG in mouse neocortex that integrates available experimental data. Simulation experiments were run in order to establish its stability, validity and dependence on initial parameters. Following experimental data were used for PVE and CP at future area 1 of the murine neocortex: 1) cell cycle duration ; 2) q factor (proportion of the ongoing mitosis giving birth to young postmitotic neurons (PN)) during neurogenetic interval (NI) ; 3) growth fraction ; 4) duration of migration ; 5) distribution of young postmitotic neurons within CP as function of their time of birth ; and 6) proportion of cells that undergoes programmed cell death within CP in early postnatal life. Model was developed and tested with system dynamics software Stella 8.0 for Windows XP operating system. Duration of the cell cycle, cortical distribution of young postmitotic neurons and proportion of dying cells were implemented as table-vice-linear function with linear interpolation. Growth fraction and duration of migration were constant. Q factor was estimated by polynomial fit that was obtained by Microsoft Excel 2003 software. Simulation experiments were conducted with starting population of 1*103 cells, equally distributed throughout cell cycle, for the time frame from 11th embryonal (E11) to 22nd postnatal (P22) day. Euler’ s integration method was used ; with integration time (DT) equaled 1/10 hours. NI lasted for 150 hours (E11+4, 9h – E17+10, 9h). Each cell of the starting population of the PVE at the beginning of NI gave rise to ~145 postmitotic neurons. Population of the PVE was largest at E14+13, 5h. Distribution by layers (in %) was as follows: VI-23, 1 ; V-9, 0 ; IV-26, 4 ; III/II-41 ; I-0, 6. During NI, cells of the PVE executed 11 cell cycles if they just entered G1 phase at the onset of neurogenetic interval, but only 10 cycles if they were anywhere in the cell cycle at least 1.6 hours away from the start of G1 phase. Overall reduction PN number within CP in the early postnatal life (P0-P14) was 28, 0%. Reduction by layers (in %) was as follows: VI – 16, 6 ; V-11, 5 ; IV-24, 5 ; III/II-40, 3 ; I-20, 7. Final number of PN linearly depends on the size of the founder population within PVE, but polynomial (2nd order) on the length of the cell cycle. Alteration of q-factor considerably influenced final neuron number. Model was shown to be stable for given parameters providing repeatable results in each performed experiment. Results of simulation experiments were in accordance to experimentally observed facts for onset of NI, its duration, time of maximal size of PVE, and persistent migration of PN during early postnatal life. Cortical distribution of PN and postnatal cell death within CP showed differences when compared with previously published data that might be attributed to the numerical estimation methods used and populations of neurons that were not included in the model (SPP, GE, ML, SP). System dynamics was found suitable for simulation modeling of quantitative developmental cellular processes enabling integration of available experimental data into single model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c7b9837de77f4aa38a7458d9fd347cd2edf5e17" target='_blank'>Neurogenesis : from Experimental Data to System Dynamics Model</a></td>
          <td>
            M. Rimac, Božikov Jadranka
          </td>
          <td>2008-05-06</td>
          <td></td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Mathematical and computational modeling enables biologists to integrate data from observations and experiments into a theoretical framework. In this review, we describe how developmental processes associated with stem‐cell‐driven growth of tissue in both the embryonic and adult nervous system can be modeled using cellular automata (CA). A cellular automaton is defined by its discrete nature in time, space, and state. The discrete space is represented by a uniform grid or lattice containing agents that interact with other agents within their local neighborhood. This possibility of local interactions of agents makes the cellular automata approach particularly well suited for studying through modeling how complex patterns at the tissue level emerge from fundamental developmental processes (such as proliferation, migration, differentiation, and death) at the single‐cell level. As part of this review, we provide a primer for how to define biologically inspired rules governing these processes so that they can be implemented into a CA model. We then demonstrate the power of the CA approach by presenting simulations (in the form of figures and movies) based on building models of three developmental systems: the formation of the enteric nervous system through invasion by neural crest cells; the growth of normal and tumorous neurospheres induced by proliferation of adult neural stem/progenitor cells; and the neural fate specification through lateral inhibition of embryonic stem cells in the neurogenic region of Drosophila.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e925a5488fa6300c9fafd6ec6882619c5a6a11b" target='_blank'>Cellular Automata Modeling of Stem‐Cell‐Driven Development of Tissue in the Nervous System</a></td>
          <td>
            Dávid Lehotzky, G. Zupanc
          </td>
          <td>2019-05-01</td>
          <td>Developmental Neurobiology</td>
          <td>19</td>
          <td>30</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>