<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              E509d9b8f4bae96ed26b214de1804c178e475585
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-27 06:08:39 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Different designs of kinase-phosphatase interactions and phosphatase sequestration shapes the robustness and signal flow in the MAPK cascade</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="Mitogen‐activated protein kinase (MAPK) cascades process a myriad of stimuli received by cell‐surface receptors and generate precise spatiotemporal guidance for multiple target proteins, dictating receptor‐specific cellular outcomes. Computational modeling reveals that the intrinsic topology of MAPK cascades enables them to amplify signal sensitivity and amplitude, reduce noise, and display intricate dynamic properties, which include toggle switches, excitation pulses, and oscillations. Specificity of signaling responses can be brought about by signal‐induced feedback and feedforward wiring imposed on the MAPK cascade backbone. Intracellular gradients of protein activities arise from the spatial separation of opposing reactions in kinase‐phosphatase cycles. The membrane confinement of the initiating kinase in MAPK cascades and cytosolic localization of phosphatases can result in precipitous gradients of phosphorylated signal‐transducers if they spread solely by diffusion. Endocytotic trafficking of active kinases driven by molecular motors and traveling waves of protein phosphorylation can propagate phosphorylation signals from the plasma membrane to the nucleus, especially in large cells, such as Xenopus eggs. Copyright © 2009 John Wiley & Sons, Inc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/549f0ea398377e29460e84284cf31387d6568803" target='_blank'>Four‐dimensional dynamics of MAPK information‐processing systems</a></td>
          <td>
            B. Kholodenko, M. Birtwistle
          </td>
          <td>2009-07-01</td>
          <td>Wiley Interdisciplinary Reviews: Systems Biology and Medicine</td>
          <td>87</td>
          <td>57</td>
        </tr>

        <tr id="The mitogen-activated protein kinase (MAPK) pathway is considered to be a central block in many biological signaling networks. Despite the common core cascade structure, the activation of MAPK in different biological systems can exhibit different types of dynamic behaviors. Computer modeling may help to reveal the mechanisms underlying such variations. However, so far most computational models of the MAPK cascade have been system-specific, or to reflect a particular type among the wide spectrum of possible dynamics. To obtain a general and integrated view of the relationship between the dynamics of MAPK activation and the structures and parameters of the MAPK cascade, we constructed a generic model by comparing previous models covering different specific biological systems. We assumed that reliable qualitative results could be predicted through a qualitative model with pseudo parameters. We used randomly sampled parameters instead of a specific set of “best-fit” parameters to avoid biases towards any particular systems. A range of dynamics behaviors for MAPK activation, including ultrasensitivity, bistability, transient activation and oscillation, were successfully predicted by the generic model. The results indicated that the steady state dynamics (ultrasensitivity and bistability) was jointly determined by the three-tiered structure of the MAPK cascade and the competitive substrate binding in the dual-phosphorylation processes of the central components, while the temporal dynamics (transient activation and oscillation) was mainly affected by the upstream signaling pathway and feedbacks. Moreover, MAPK kinase (MAPKK) played more important roles than the other two components in determining the dynamics of MAPK activation. We hypothesize that this is an important and advantageous property for the regulation and for the functional diversity of MAPK pathways in real cells. Finally, to assist developing generic models for signaling motifs through model comparisons, we proposed a reaction-based database to make the model data more flexible and interoperable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/558a5c17bf4b19875b0696b35de9a583a6c40956" target='_blank'>Random Parameter Sampling of a Generic Three-Tier MAPK Cascade Model Reveals Major Factors Affecting Its Versatile Dynamics</a></td>
          <td>
            Zhong Mai, Haiyan Liu
          </td>
          <td>2013-01-24</td>
          <td>PLoS ONE</td>
          <td>9</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da87720f4767a44da3f92e497492607a5e8a082c" target='_blank'>When kinases meet mathematics: the systems biology of MAPK signalling</a></td>
          <td>
            W. Kolch, Muffy Calder, David R. Gilbert
          </td>
          <td>2005-03-21</td>
          <td>FEBS Letters</td>
          <td>190</td>
          <td>86</td>
        </tr>

        <tr id="All cells receive and respond to signals from their environment, whether they live freely or are part of a tissue. Feedback loops are one among the interactions in signal-transduction cascades, in which the information from the downstream components is sent back to the upstream components of the system. Crosstalks between signal-transduction cascades are able to combine the information's received and processed to trigger the appropriate response. To understand how signaling networks make decisions, the dynamic interactions of proteins in the signal-transduction cascades are important rather than just creating static maps of pathways. Based on reaction kinetics, we have developed mathematical models to predict and analyze the impacts of the feedback loops in JNK cascades and the crosstalks between JNK and P38 kinase cascades. Feedback loops in JNK cascades and crosstalks between JNK and P38 kinase cascades plays the complementary roles in making the impacts on the activation of JNK and P38 kinases. JNK and P38 kinase cascades with feedbacks and crosstalks is a robust system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2880a4c0be242db0837a9696c152f250827dd584" target='_blank'>Dynamic Modeling and Simulation of JNK and P38 Kinase Cascades With Feedbacks and Crosstalks</a></td>
          <td>
            Pandurangan Sundaramurthy, S. Gakkhar
          </td>
          <td>2010-08-19</td>
          <td>IEEE Transactions on NanoBioscience</td>
          <td>7</td>
          <td>24</td>
        </tr>

        <tr id="Networks of Signaling pathways provide a robust mechanism for cells to respond to various biological stimuli. In this paper we demonstrate cell adaptation through the viewpoint of an organizing principle between two interconnected pathways- MAPK and PKC. We use a multi-layered system representation of the pathways to determine the pathway components contributing to the adaptive behavior and coordination. The adaptation can be thought of as being manifested by a change in parameters of the coordinator. In silico experiments are conducted using MAPK/PKC mathematical model in literature to investigate the role of PLA2as a coordinator is reported here. Our results show that varying parameters of the coordinator not only activates the network of pathways where otherwise the pathway activity is very low, but also reveal the ability of the system to activate itself in the absence of the input, indicating relevance of the principle of bounded autonomy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b05bffc3e00de1eef8e2206dcd7c7aca7097059e" target='_blank'>From Networks to Systems to Complex Systems: A Signaling Pathway Coordination Case Study</a></td>
          <td>
            R. Soebiyanto, S. Sreenath, M. Mesarovic, O. Wolkenhauer
          </td>
          <td>2005-12-12</td>
          <td>Proceedings of the 44th IEEE Conference on Decision and Control</td>
          <td>1</td>
          <td>52</td>
        </tr>

        <tr id="For more than a decade, the MAPK (mitogen-activated protein kinase) cascade has been studied using mathematical modelling and quantitative experimentation [1]. The MAPK cascade relays the presence of extracellular stimuli such as growth hormones to the nucleus and controls the expression of hundreds of genes. MAPKs control major cell fate decisions such as proliferation, differentiation and apoptosis, mainly by inducing alterations in gene expression. In this chapter, we discuss how systems biology analysis provides insights into the functioning of this cascade. We show how this pathway assists the cell in responding properly to extracellular cues by filtering out sub-threshold stimuli, while efficiently transmitting physiologically relevant inputs. Several different receptors signal through the MAPK pathway even though they elicit opposite biological responses, thus raising the question of how specificity is achieved in MAPK signalling. Experimental studies revealed that specific biological responses are encoded by quantitative aspects of the MAPK signal such as amplitude or duration. We discuss mechanisms that enable the pathway to generate quantitatively different signals, and also explain how different signals are interpreted by the downstream gene expression machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6240690e740bb11ac1d42f43f91ffcfad059dbaf" target='_blank'>Systems analysis of MAPK signal transduction.</a></td>
          <td>
            N. Blüthgen, S. Legewie
          </td>
          <td>2008-09-30</td>
          <td>Essays in biochemistry</td>
          <td>68</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/380612c9c06b9e10186175edd02ceaf28ac4dc69" target='_blank'>Generalized logical model based on network topology to capture the dynamical trends of cellular signaling pathways</a></td>
          <td>
            Fan Zhang, Haoting Chen, Li na Zhao, Hui Liu, T. Przytycka, Jie Zheng
          </td>
          <td>2016-01-11</td>
          <td>BMC Systems Biology</td>
          <td>2</td>
          <td>51</td>
        </tr>

        <tr id="Among the 13 TLRs in the vertebrate systems, only TLR4 utilizes both Myeloid differentiation factor 88 (MyD88) and Toll/Interleukin-1 receptor (TIR)-domain-containing adapter interferon-β-inducing Factor (TRIF) adaptors to transduce signals triggering host-protective immune responses. Earlier studies on the pathway combined various experimental data in the form of one comprehensive map of TLR signaling. But in the absence of adequate kinetic parameters quantitative mathematical models that reveal emerging systems level properties and dynamic inter-regulation among the kinases/phosphatases of the TLR4 network are not yet available. So, here we used reaction stoichiometry-based and parameter independent logical modeling formalism to build the TLR4 signaling network model that captured the feedback regulations, interdependencies between signaling kinases and phosphatases and the outcome of simulated infections. The analyses of the TLR4 signaling network revealed 360 feedback loops, 157 negative and 203 positive; of which, 334 loops had the phosphatase PP1 as an essential component. The network elements' interdependency (positive or negative dependencies) in perturbation conditions such as the phosphatase knockout conditions revealed interdependencies between the dual-specific phosphatases MKP-1 and MKP-3 and the kinases in MAPK modules and the role of PP2A in the auto-regulation of Calmodulin kinase-II. Our simulations under the specific kinase or phosphatase gene-deficiency or inhibition conditions corroborated with several previously reported experimental data. The simulations to mimic Yersinia pestis and E. coli infections identified the key perturbation in the network and potential drug targets. Thus, our analyses of TLR4 signaling highlights the role of phosphatases as key regulatory factors in determining the global interdependencies among the network elements; uncovers novel signaling connections; identifies potential drug targets for infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21671a5064b220ad54e253c6cfcad83c0acf01de" target='_blank'>Comprehensive Logic Based Analyses of Toll-Like Receptor 4 Signal Transduction Pathway</a></td>
          <td>
            M. Padwal, U. Sarma, B. Saha
          </td>
          <td>2014-04-03</td>
          <td>PLoS ONE</td>
          <td>18</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1060162779f5ba10620f52dde0b038e896d35a8" target='_blank'>Mapping connections in signaling networks with ambiguous modularity</a></td>
          <td>
            Daniel Lill, Oleksii S. Rukhlenko, Anthony James Mc Elwee, E. Kashdan, J. Timmer, B. Kholodenko
          </td>
          <td>2019-05-23</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>9</td>
          <td>73</td>
        </tr>

        <tr id="The advances in proteomics technologies offer an unprecedented opportunity and valuable resources to understand how living organisms execute necessary functions at systems levels. However, little work has been done up to date to utilize the highly accurate spatio-temporal dynamic proteome data generated by phosphoprotemics for mathematical modeling of complex cell signaling pathways. This work proposed a novel computational framework to develop mathematical models based on proteomic datasets. Using the MAP kinase pathway as the test system, we developed a mathematical model including the cytosolic and nuclear subsystems; and applied the genetic algorithm to infer unknown model parameters. Robustness property of the mathematical model was used as a criterion to select the appropriate rate constants from the estimated candidates. Quantitative information regarding the absolute protein concentrations was used to refine the mathematical model. We have demonstrated that the incorporation of more experimental data could significantly enhance both the simulation accuracy and robustness property of the proposed model. In addition, we used the MAP kinase pathway inhibited by phosphatases with different concentrations to predict the signal output influenced by different cellular conditions. Our predictions are in good agreement with the experimental observations when the MAP kinase pathway was inhibited by phosphatase PP2A and MKP3. The successful application of the proposed modeling framework to the MAP kinase pathway suggests that our method is very promising for developing accurate mathematical models and yielding insights into the regulatory mechanisms of complex cell signaling pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9577e3ebca59ca340c92adad68c098433bbcb84b" target='_blank'>Mathematical Modelling of the MAP Kinase Pathway Using Proteomic Datasets</a></td>
          <td>
            Tianhai Tian, Jiangning Song
          </td>
          <td>2012-08-08</td>
          <td>PLoS ONE</td>
          <td>35</td>
          <td>51</td>
        </tr>

        <tr id="The relation between design principles of signaling network motifs and their robustness against intrinsic noise still remains illusive. In this work we investigate the role of cascading for coping with intrinsic noise due to stochasticity in molecular reactions. We use stochastic approaches to quantify fluctuations in the terminal kinase of phosphorylation-dephosphorylation cascade motifs and demonstrate that cascading highly affects these fluctuations. We show that this purely stochastic effect can be explained by time-varying sequestration of upstream kinase molecules. In particular, we discuss conditions on time scales and parameter regimes which lead to a reduction of output fluctuations. Our results are put into biological context by adapting rate parameters of our modeling approach to biologically feasible ranges for general binding-unbinding and phosphorylation-dephosphorylation mechanisms. Overall, this study reveals a novel role of stochastic sequestration for dynamic noise filtering in signaling cascade motifs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dc875615a36630c7bd5940cb11e2c55a7ebeb6d" target='_blank'>The role of stochastic sequestration dynamics for intrinsic noise filtering in signaling network motifs</a></td>
          <td>
            Debdas Paul, N. Radde
          </td>
          <td>2018-03-08</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Intracellular signaling pathways that share common components often elicit distinct physiological responses. In most cases, the biochemical mechanisms responsible for this signal specificity remain poorly understood. Protein scaffolds and cross-inhibition have been proposed as strategies to prevent unwanted cross-talk. Here, we report a mechanism for signal specificity termed “kinetic insulation.” In this approach signals are selectively transmitted through the appropriate pathway based on their temporal profile. In particular, we demonstrate how pathway architectures downstream of a common component can be designed to efficiently separate transient signals from signals that increase slowly over time. Furthermore, we demonstrate that upstream signaling proteins can generate the appropriate input to the common pathway component regardless of the temporal profile of the external stimulus. Our results suggest that multilevel signaling cascades may have evolved to modulate the temporal profile of pathway activity so that stimulus information can be efficiently encoded and transmitted while ensuring signal specificity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34f99baf2a3fce2ef002f0b3da1545674a8824cc" target='_blank'>Kinetic insulation as an effective mechanism for achieving pathway specificity in intracellular signaling networks</a></td>
          <td>
            M. Behar, H. Dohlman, T. Elston
          </td>
          <td>2007-10-09</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>87</td>
          <td>54</td>
        </tr>

        <tr id="Despite much effort, identification of modular structures and study of their organizing and functional roles remain a formidable challenge in molecular systems biology, which, however, is essential in reaching a systematic understanding of large-scale cell regulation networks and hence gaining capacity of exerting effective interference to cell activity. Combining graph theoretic methods with available dynamics information, we successfully retrieved multiple feedback modules of three important signaling networks. These feedbacks are structurally arranged in a hierarchical way and dynamically produce layered temporal profiles of output signals. We found that global and local feedbacks act in very different ways and on distinct features of the information flow conveyed by signal transduction but work highly coordinately to implement specific biological functions. The redundancy embodied with multiple signal-relaying channels and feedback controls bestow great robustness and the reaction hubs seated at junctions of different paths announce their paramount importance through exquisite parameter management. The current investigation reveals intriguing general features of the organization of cell signaling networks and their relevance to biological function, which may find interesting applications in analysis, design and control of bio-networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f1a36176cecd70b5b2e37e209eede6e52ee5091" target='_blank'>Hierarchical Feedback Modules and Reaction Hubs in Cell Signaling Networks</a></td>
          <td>
            Jianfeng Xu, Y. Lan
          </td>
          <td>2015-05-07</td>
          <td>PLoS ONE</td>
          <td>10</td>
          <td>20</td>
        </tr>

        <tr id="Operating regimes characterizing the input-output behaviour of an activated phosphorylation-dephosphorylation reaction cycle (PdPC) such as a single MAPK cascade are dictated by the saturated/unsaturated state of the two underlying enzymatic reactions. Four combinations of the states of these two enzymatic reactions led to identification of distinct operating regimes, viz., Hyperbolic (H), Signal transducing (ST), Threshold-hyperbolic (TH) and Ultrasensitive (U). A single PdPC without an explicit feedback have been classically viewed as a module offering signal flow from upstream to downstream, that is, one-way communication. Recently it has been shown that load due to sequestration of the phosphorylated or unphosphorylated form of the substrate by corresponding downstream targets permits retroactive signalling that offers two-way communication. We systematically characterize the operating regimes of a single PdPC subject to retroactivity in either of the substrate forms. We identify five possible regime transitions that could be achieved by increasing the retroactivity strength on either of the two substrate forms. Remarkably, a retroactivity strength of 0.30 in the unphosphorylated form of the substrate is sufficient to induce a transition from ST to H regime indicating that the input-output behaviour of a PdPC is highly sensitive to the presence of a downstream load. Using sensitivity and rate-balance analysis, we show that modulation of the saturation levels of the two enzymatic reactions by increasing retroactivity is the fundamental mechanism governing operating regime transition. Highlights Characterization of operating regimes in the presence of retroactivity. Retroactivity can induce a transition between different operating regimes. Saturation levels of the two enzymatic reactions govern the regime transition. Sensitivity of the protein levels to retroactivity is dictated by saturation levels.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d027423ced733f8825b5661c384cf2168e2289" target='_blank'>Retroactivity induced operating regime transition in a phosphorylation-dephosphorylation cycle</a></td>
          <td>
            Akshay N. Parundekar, G. Viswanathan
          </td>
          <td>2020-10-02</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="The prediction of causal influences between components of a signaling network requires detailed modeling from large data sets. Single-cell measurements of the phosphorylation state of a panel of signaling proteins with phosphospecific antibodies after various treatments that influenced cellular signaling provided sufficient data so that Sachs et al. (see the Perspective by Brent and Lok) could apply a Bayesian network inference algorithm to map a signaling network and infer causal influences between the components of the network. Known connections were reproduced, and a newly discovered connection was experimentally tested and found indeed to be of biological relevance. K. Sachs, O. Perez, D. Pe'er, D. A. Lauffenburger, G. P. Nolan, Causal protein-signaling networks derived from multiparameter single-cell data. Science 308, 523-529 (2005). [Abstract] [Full Text] R. Brent, L. Lok, A fishing buddy for hypothesis generators. Science 308, 504-506 (2005). [Summary] [Full Text]">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28342c03bc2b1a6ad6c9813d920108ed08dcfb6c" target='_blank'>Modeling Signaling Networks</a></td>
          <td>

          </td>
          <td>2005-04-26</td>
          <td>Science's STKE</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Different environmental stimuli often use the same set of signaling proteins to achieve very different physiological outcomes. The invasive growth and mating pathways in yeast Saccharomyces cerevisiae each employ a mitogen‐activated protein (MAP) kinase cascade that includes several common components (e.g. Ste11, Ste7), after which the pathways diverge leading to activation of different MAP kinases (Fus3 and Kss1, respectively). In order to determine how pathway specificity is achieved, we conducted a quantitative analysis of MAP kinase activity, and devised a series of six computational models that describe their behavior in vivo. As anticipated by one of the models, MAP kinase phosphorylation of shared pathway components results in diminished activation of one MAP kinase but not the other. These findings reveal how one MAP kinase can limit the activity of a competing MAP kinase and thereby ensure signal fidelity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a0034a8b7b900ff225c7b67b37624a166b457e" target='_blank'>Modeling signal specificity by feedback inhibition</a></td>
          <td>
            Nan Hao, Necmettin Yildirim, S. Parnell, B. Errede, T. Elston, H. Dohlman
          </td>
          <td>2007-04-01</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Among the 13 TLRs in the vertebrate systems, only TLR4 utilizes both Myeloid differentiation factor 88 (MyD88) and Toll/ 
Interleukin-1 receptor (TIR)-domain-containing adapter interferon-b-inducing Factor (TRIF) adaptors to transduce signals 
triggering host-protective immune responses. Earlier studies on the pathway combined various experimental data in the 
form of one comprehensive map of TLR signaling. But in the absence of adequate kinetic parameters quantitative 
mathematical models that reveal emerging systems level properties and dynamic inter-regulation among the kinases/ 
phosphatases of the TLR4 network are not yet available. So, here we used reaction stoichiometry-based and parameter 
independent logical modeling formalism to build the TLR4 signaling network model that captured the feedback regulations, 
interdependencies between signaling kinases and phosphatases and the outcome of simulated infections. The analyses of 
the TLR4 signaling network revealed 360 feedback loops, 157 negative and 203 positive; of which, 334 loops had the 
phosphatase PP1 as an essential component. The network elements’ interdependency (positive or negative dependencies) 
in perturbation conditions such as the phosphatase knockout conditions revealed interdependencies between the dualspecific 
phosphatases MKP-1 and MKP-3 and the kinases in MAPK modules and the role of PP2A in the auto-regulation of 
Calmodulin kinase-II. Our simulations under the specific kinase or phosphatase gene-deficiency or inhibition conditions 
corroborated with several previously reported experimental data. The simulations to mimic Yersinia pestis and E. coli 
infections identified the key perturbation in the network and potential drug targets. Thus, our analyses of TLR4 signaling 
highlights the role of phosphatases as key regulatory factors in determining the global interdependencies among the 
network elements; uncovers novel signaling connections; identifies potential drug targets for infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb6593f9456a088a63702b44f973b06b08a49cfb" target='_blank'>Comprehensive Logic Based Analyses of Toll-LikeReceptor 4 Signal Transduction PathwayMahesh Kumar</a></td>
          <td>
            M. Padwal, U. Sarma, B. Saha
          </td>
          <td>2014-04-01</td>
          <td></td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b06d234407efd97d798904ba97ebb9a73b5fb31e" target='_blank'>Methodologies for the modeling and simulation of biochemical networks, illustrated for signal transduction pathways: A primer</a></td>
          <td>
            Nesma ElKalaawy, A. Wassal
          </td>
          <td>2015-03-01</td>
          <td>Bio Systems</td>
          <td>20</td>
          <td>8</td>
        </tr>

        <tr id="Kinases and phosphatases are stars in the signaling universe. Kinases and phosphatases are key regulatory molecules that participate in most cell signaling pathways. Systems-level analyses are providing new insights into phosphorylation sites and kinase specificity, and phosphoproteomic analyses are creating not just a wealth of data, but are also revealing rich revelations about cellular behavior, cellular responses to changing environmental conditions, and mechanisms of disease. Phosphatases are not merely signal terminators, but play active roles in signal transduction and are especially important in redox signaling. In this Focus Issue, Science Signaling highlights protein phosphorylation as the center of the signaling universe.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cced11acef974d2179936b86e6065c7876f93b4" target='_blank'>Focus Issue: Systems Analysis of Protein Phosphorylation</a></td>
          <td>
            N. Gough, J. Foley
          </td>
          <td>2010-08-31</td>
          <td>Science Signaling</td>
          <td>9</td>
          <td>32</td>
        </tr>

        <tr id="Dynamics adds to the complexity of signaling networks. The biochemical processes that control the responses of cells to changes in their internal or external environments form a complex interconnected network. The molecules that participate in these processes are diverse and dynamic, and the networks themselves are affected by the flow of information through them. With this appreciation of the complexity in signaling networks, Science Signaling highlights computational and experimental approaches to understanding both the macroscopic cellular response and the microscopic signaling network and its constituents and dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8045d1bb90c38a9e81507bcbcfd412b519b768f7" target='_blank'>Focus Issue: Unraveling Signaling Complexity</a></td>
          <td>
            N. Gough, J. Foley
          </td>
          <td>2009-07-28</td>
          <td>Science Signaling</td>
          <td>5</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19ea531f407b034c82262a3c743a3f5575d55e9b" target='_blank'>Quantitative analysis of signaling networks.</a></td>
          <td>
            Herbert M. Sauro, B. Kholodenko
          </td>
          <td>2004-09-01</td>
          <td>Progress in biophysics and molecular biology</td>
          <td>213</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/191f8aa325b96b6e58a5ca39c766f19650b93b17" target='_blank'>A systematic framework for the design of reduced-order models for signal transduction pathways from a control theoretic perspective</a></td>
          <td>
            M. Maurya, Santhoji Katare, P. Patkar, A. Rundell, V. Venkatasubramanian
          </td>
          <td>2006-01-15</td>
          <td>Comput. Chem. Eng.</td>
          <td>11</td>
          <td>53</td>
        </tr>

        <tr id="Many distinct signaling pathways allow the cell to receive, process, and respond to information. Often, components of different pathways interact, resulting in signaling networks. Biochemical signaling networks were constructed with experimentally obtained constants and analyzed by computational methods to understand their role in complex biological processes. These networks exhibit emergent properties such as integration of signals across multiple time scales, generation of distinct outputs depending on input strength and duration, and self-sustaining feedback loops. Feedback can result in bistable behavior with discrete steady-state activities, well-defined input thresholds for transition between states and prolonged signal output, and signal modulation in response to transient stimuli. These properties of signaling networks raise the possibility that information for "learned behavior" of biological systems may be stored within intracellular biochemical reactions that comprise signaling pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9de3b7150cc254356bd579ef99e0714c1f3b39a" target='_blank'>Emergent properties of networks of biological signaling pathways.</a></td>
          <td>
            U. Bhalla, R. Iyengar
          </td>
          <td>1999-01-15</td>
          <td>Science</td>
          <td>1645</td>
          <td>64</td>
        </tr>

        <tr id="Cellular signaling pathways are responsible for decision making that sustains life. Most signaling pathways include post-translational modification cycles, that process multiple inputs and are tightly interconnected. Here we consider a model for phosphorylation/dephosphorylation cycles, and we show that under some assumptions they can operate as molecular neurons or perceptrons, that generate sigmoidal-like activation functions by processing sums of inputs with positive and negative weights. We carry out a steady-state and structural stability analysis for single molecular perceptrons as well as for feedforward interconnections, concluding that interconnected phosphorylation/dephosphorylation cycles may work as multi-layer biomolecular neural networks (BNNs) with the capacity to perform a variety of computations. As an application, we design signaling networks that behave as linear and non-linear classifiers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5112a01219843c2f6335c66b9a2bda740eb41fe2" target='_blank'>Signaling-based neural networks for cellular computation</a></td>
          <td>
            Christian Cuba Samaniego, Andrew Moorman, Giulia Giordano, Elisa Franco
          </td>
          <td>2020-11-10</td>
          <td>bioRxiv</td>
          <td>11</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75fd6027d52e75c27e75990da11221684c3f5f16" target='_blank'>Attenuation of noise in ultrasensitive signaling cascades.</a></td>
          <td>
            M. Thattai, A. van Oudenaarden
          </td>
          <td>2002-06-01</td>
          <td>Biophysical journal</td>
          <td>233</td>
          <td>76</td>
        </tr>

        <tr id="Cellular proccess that are crucial for health or disease are orchestrated by the expression of multiple genes in a network of differential complex signaling pathways, consisting of many physical and functional interactions.Unlike the stable genome structure, the highly dynamic interactome represents a major challenge for quantitative mapping of protein–protein interactions (PPIs). In this article, I will discuss the latest systems science advances by using breakthrough technologies to gain an understanding of the whole picture of the spatiotemporal signaling network as a response to dynamic individual genome–lifestyle interactions and translating it to the clinic.Biological signals control cells’ functions and their organization into tissues, organs and a whole organism’s homeostasis. A variety of extracellular and intracellu-lar signaling molecules include hormones, growth factors, cytokines, chemokines, neurotransmitters, extracellular matrix components, receptors, protein kinases, protein phosphatases and DNA-binding proteins in the nucleus. A signal is trans-mitted or amplified from the cell mem-brane to the nucleus through a dynamic complex interactions network">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2702c101690829e49078cf33e2ee81dc4f49e616" target='_blank'>Differential signaling transduction networks for clinical robustness</a></td>
          <td>
            D. Roukos
          </td>
          <td>2012-04-01</td>
          <td>Expert Review of Proteomics</td>
          <td>12</td>
          <td>62</td>
        </tr>

        <tr id="Markevich et al. developed a computational model for a mechanism, not previously described, whereby signaling pathways can exhibit bistable behavior and switch between stable "on" and "off" states. Mitogen-activated protein kinase (MAPK) signaling pathways, which play critical roles in cell proliferation, differentiation, and apoptosis, generally function as a three-tiered cascade in which an activated (phosphorylated) kinase at one level phosphorylates and thereby activates the kinase at the subsequent level. To be fully activated, these MAPKs, MAPK kinases (MAPKKs), and MAPKK kinases must be phosphorylated at two sites. The MAPK cascades are regulated by multiple feedback loops, and the existence of either positive or double-negative feedback regulation is generally thought to be necessary for a cascade to function as a bistable system. Markevich et al., however, present a model in which bistability of the MAPK signaling pathway arises from a distributive kinetic mechanism for the dual phosphorylation and dephosphorylation of the kinase and is independent of feedback regulation. (A distributive mechanism implies that the kinase or phosphatase releases the monophosphorylated intermediate, which must then interact with enzyme again before it is converted into the final doubly phosphorylated or completely dephosphorylated state.) Their model was consistent with the reported kinetic properties of the extracellular signal-regulated kinase (ERK) kinase (MEK) and phosphatase (MKP3). The authors suggest that multisite phosphorylation, which takes place on proteins involved in many signaling systems, may represent a widespread mechanism for conferring switch-like behavior on various signaling systems. N. I. Markevich, J. B. Hoek, B. N. Kholodenko, Signaling switches and bistability arising from multisite phosphorylation in protein kinase cascades. J. Cell Biol. 164, 353-359 (2004). [Abstract] [Full Text]">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7053bd2d18f08169a4698ad2a73b821fce3419" target='_blank'>Flipping a Signaling Switch</a></td>
          <td>

          </td>
          <td>2004-02-10</td>
          <td>Science's STKE</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract A kinase anchoring proteins (AKAPs) bind multiple signaling proteins and have subcellular targeting domains that allow them to greatly impact cellular signaling. AKAPs localize, specify, amplify, and accelerate signal transduction within the cell by bringing signaling proteins together in space and time. AKAPs also organize higher-order network motifs such as feed forward and feedback loops that may create complex network responses, including adaptation, oscillation, and ultrasensitivity. Computational models have begun to provide an insight into how AKAPs regulate signaling dynamics and cardiovascular pathophysiology. Models of mitogen-activated protein kinase and epidermal growth factor receptor scaffolds have revealed additional design principles and new methods for representing signaling scaffolds mathematically. Coupling computational modeling with quantitative experimental approaches will be increasingly necessary for dissecting the diverse information processing functions performed by AKAP signaling complexes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/def404ce92aea068ec191a04eb36d8eafca030dd" target='_blank'>Bigger, Better, Faster: Principles and Models of AKAP Anchoring Protein Signaling</a></td>
          <td>
            Eric C Greenwald, J. Saucerman
          </td>
          <td>2011-11-01</td>
          <td>Journal of Cardiovascular Pharmacology</td>
          <td>35</td>
          <td>35</td>
        </tr>

        <tr id="Cell responses are actuated by tightly controlled signal transduction pathways. Although the concept of an integrated signaling network replete with interpathway cross-talk and feedback regulation is broadly appreciated, kinetic data of the type needed to characterize such interactions in conjunction with mathematical models are lacking. In mammalian cells, the Ras/ERK pathway controls cell proliferation and other responses stimulated by growth factors, and several cross-talk and feedback mechanisms affecting its activation have been identified. In this work, we take a systematic approach to parse the magnitudes of multiple regulatory mechanisms that attenuate ERK activation through canonical (Ras-dependent) and non-canonical (PI3K-dependent) pathways. In addition to regulation of receptor and ligand levels, we consider three layers of ERK-dependent feedback: desensitization of Ras activation, negative regulation of MEK kinase (e.g. Raf) activities, and up-regulation of dual-specificity ERK phosphatases. Our results establish the second of these as the dominant mode of ERK self-regulation in mouse fibroblasts. We further demonstrate that kinetic models of signaling networks, trained on a sufficient diversity of quantitative data, can be reasonably comprehensive, accurate, and predictive in the dynamical sense.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405be6d692d4836d3b3efde9333fdf1789ff05c7" target='_blank'>Systematic Quantification of Negative Feedback Mechanisms in the Extracellular Signal-regulated Kinase (ERK) Signaling Network*</a></td>
          <td>
            M. Cirit, Chun-Chao Wang, J. Haugh
          </td>
          <td>2010-09-16</td>
          <td>The Journal of Biological Chemistry</td>
          <td>91</td>
          <td>33</td>
        </tr>

        <tr id="Simulating signal transduction in cellular signaling networks provides predictions of network dynamics by quantifying the changes in concentration and activity-level of the individual proteins. Since numerical values of kinetic parameters might be difficult to obtain, it is imperative to develop non-parametric approaches that combine the connectivity of a network with the response of individual proteins to signals which travel through the network. The activity levels of signaling proteins computed through existing non-parametric modeling tools do not show significant correlations with the observed values in experimental results. In this work we developed a non-parametric computational framework to describe the profile of the evolving process and the time course of the proportion of active form of molecules in the signal transduction networks. The model is also capable of incorporating perturbations. The model was validated on four signaling networks showing that it can effectively uncover the activity levels and trends of response during signal transduction process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d7b2f82278b1e0396626ed63f2da71203f1c000" target='_blank'>Nonparametric Simulation of Signal Transduction Networks with Semi-Synchronized Update</a></td>
          <td>
            I. Nassiri, A. Masoudi-Nejad, M. Jalili, A. Moeini
          </td>
          <td>2012-06-21</td>
          <td>PLoS ONE</td>
          <td>11</td>
          <td>47</td>
        </tr>

        <tr id="Biochemical networks, including those containing signaling pathways, display a wide range of regulatory properties. These include the ability to propagate information across different time scales and to function as switches and oscillators. The mechanisms underlying these complex behaviors involve many interacting components and cannot be understood by experiments alone. The development of computational models and the integration of these models with experiments provide valuable insight into these complex systems-level behaviors. Here we review current approaches to the development of computational models of biochemical networks and describe the insights gained from models that integrate experimental data, using three examples that deal with ultrasensitivity, flexible bistability and oscillatory behavior. These types of complex behavior from relatively simple networks highlight the necessity of using theoretical approaches in understanding higher order biological functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7892da387cb63b94afbe9f7c3bc0d0b71be6c4c2" target='_blank'>Modeling Signaling Networks</a></td>
          <td>
            Susana R. Neves, R. Iyengar
          </td>
          <td>2002-12-01</td>
          <td>Science</td>
          <td>131</td>
          <td>64</td>
        </tr>

        <tr id="Signal transduction is a complex protein signaling process with a rich network of multifunctional interactions that occur in a non‐linear fashion. Mitogen‐activated protein kinase (MAPK) signal transduction pathways regulate diverse cellular processes ranging from proliferation and differentiation to apoptosis. In mammals, out of five, there are three well characterized subfamilies of MAPKs ‐ ERKs (Extracellular signal‐regulated kinases), JNKs (c‐Jun N‐terminal kinases), and P38 kinases, and their activators, are implicated in human diseases and are targets for drug development. Kinase cascades in MAPK pathways mediate the sensing and processing of stimuli. To understand how cells makes decisions, the dynamic interactions of components of signaling cascades are important rather than just creating static maps. Based on enzyme kinetic reactions, we have developed a mathematical model to analyze the impact of the cross‐talks between JNK and P38 kinase cascades. Cross‐talks between JNK and P38 kinase cascades influence the activities of P38 kinases. Responses of the signals should be studied for network of kinase cascades by considering cross‐talks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef12bcae9ca82339a744c3b732a8bd77d348ac39" target='_blank'>Computational prediction and analysis of impact of the cross‐talks between JNK and P38 kinase cascades</a></td>
          <td>
            Pandurangan Sundaramurthy, S. Gakkhar, R. Sowdhamini
          </td>
          <td>2009-01-12</td>
          <td>Bioinformation</td>
          <td>12</td>
          <td>32</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>