<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              89edceb6395ad6a9a46a561fb37a23bbccd0f06b
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-21 10:41:45 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Computational models in inflammatory bowel disease</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="Inflammatory bowel disease (IBD) is a complex, immune-mediated gastrointestinal disorder with ill-defined etiology, multifaceted diagnostic criteria, and unpredictable treatment response. Innovations in IBD diagnostics, including developments in genomic sequencing and molecular analytics, have generated tremendous interest in leveraging these large data platforms into clinically meaningful tools. Artificial intelligence, through machine learning facilitates the interpretation of large arrays of data, and may provide insight to improving IBD outcomes. While potential applications of machine learning models are vast, further research is needed to generate standardized models that can be adapted to target IBD populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96a50646bcd4ed63d1897a075a4af13ac4e17de3" target='_blank'>Emerging use of artificial intelligence in inflammatory bowel disease</a></td>
          <td>
            A. Kohli, Erik A. Holzwanger, Alexander N. Levy
          </td>
          <td>2020-11-28</td>
          <td>World Journal of Gastroenterology</td>
          <td>21</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Various extrinsic and intrinsic factors such as drug exposures, antibiotic treatments, smoking, lifestyle, genetics, immune responses, and the gut microbiome characterize ulcerative colitis and Crohn's disease, collectively called inflammatory bowel disease (IBD). All these factors contribute to the complexity and heterogeneity of the disease etiology and pathogenesis leading to major challenges for the scientific community in improving management, medical treatments, genetic risk, and exposome impact. Understanding the interaction(s) among these factors and their effects on the immune system in IBD patients has prompted advances in multi‐omics research, the development of new tools as part of system biology, and more recently, artificial intelligence (AI) approaches. These innovative approaches, supported by the availability of big data and large volumes of digital medical datasets, hold promise in better understanding the natural histories, predictors of disease development, severity, complications and treatment outcomes in complex diseases, providing decision support to doctors, and promising to bring us closer to the realization of the “precision medicine” paradigm. This review aims to provide an overview of current IBD omics based on both individual (genomics, transcriptomics, proteomics, metagenomics) and multi‐omics levels, highlighting how AI can facilitate the integration of heterogeneous data to summarize our current understanding of the disease and to identify current gaps in knowledge to inform upcoming research in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ce7ad87d298d804fa5b06fe06196a3d8f7fd6d2" target='_blank'>Inflammatory bowel disease genomics, transcriptomics, proteomics and metagenomics meet artificial intelligence</a></td>
          <td>
            Anna Lucia Cannarozzi, A. Latiano, L. Massimino, F. Bossa, Francesco Giuliani, Matteo Riva, F. Ungaro, Maria Guerra, Anna Laura Di Brina, G. Biscaglia, F. Tavano, S. Carparelli, G. Fiorino, S. Danese, F. Perri, O. Palmieri
          </td>
          <td>2024-08-31</td>
          <td>United European Gastroenterology Journal</td>
          <td>3</td>
          <td>57</td>
        </tr>

        <tr id="Inflammatory bowel disease (IBD) is a complex and multifaceted disorder of the gastrointestinal tract that is increasing in incidence worldwide and associated with significant morbidity. The rapid accumulation of large datasets from electronic health records, high-definition multi-omics (including genomics, proteomics, transcriptomics, and metagenomics), and imaging modalities (endoscopy and endomicroscopy) have provided powerful tools to unravel novel mechanistic insights and help address unmet clinical needs in IBD. Although the application of artificial intelligence (AI) methods has facilitated the analysis, integration, and interpretation of large datasets in IBD, significant heterogeneity in AI methods, datasets, and clinical outcomes and the need for unbiased prospective validations studies are current barriers to incorporation of AI into clinical practice. The purpose of this review is to summarize the most recent advances in the application of AI and machine learning technologies in the diagnosis and risk prediction, assessment of disease severity, and prediction of clinical outcomes in patients with IBD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90ee1b0c19c44b6b49338b7a6f75017f261423c6" target='_blank'>Artificial intelligence applications in inflammatory bowel disease: Emerging technologies and future directions</a></td>
          <td>
            J. Gubatan, Steven Levitte, Akshar Patel, Tatiana Balabanis, M. Wei, S. Sinha
          </td>
          <td>2021-05-07</td>
          <td>World Journal of Gastroenterology</td>
          <td>76</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Introduction Artificial intelligence (AI) is expected to impact all facets of inflammatory bowel disease (IBD) management, including disease assessment, treatment decisions, discovery and development of new biomarkers and therapeutics, as well as clinician–patient communication. Areas covered This perspective paper provides an overview of the application of AI in the clinical management of IBD through a review of the currently available AI models that could be potential tools for prognosis, shared decision-making, and precision medicine. This overview covers models that measure treatment response based on statistical or machine-learning methods, or a combination of the two. We briefly discuss a computational model that allows integration of immune/biological system knowledge with mathematical modeling and also involves a ‘digital twin’, which allows measurement of temporal trends in mucosal inflammatory activity for predicting treatment response. A viewpoint on AI-enabled wearables and nearables and their use to improve IBD management is also included. Expert opinion Although challenges regarding data quality, privacy, and security; ethical concerns; technical limitations; and regulatory barriers remain to be fully addressed, a growing body of evidence suggests a tremendous potential for integration of AI into daily clinical practice to enable precision medicine and shared decision-making. ARTICLE HIGHLIGHTS Advances in artificial intelligence (AI) show promise for improving treatment response prediction, decision-making, and precision medicine in inflammatory bowel disease (IBD). In particular, AI could improve precision medicine for IBD by enabling identification of disease subtypes, prediction of disease progression and treatment response, selection of personalized treatments, and remote monitoring. Predictive models can benefit clinicians and patients alike by optimizing shared decision-making processes; patients can also use AI to cope with daily and long-term challenges of the disease. Beyond patients and practitioners, predictive models may positively impact healthcare structures and payers by enabling effective healthcare-resource utilization. To increase the accuracy and efficiency of AI models, biomarkers, patient-reported outcomes, and disease scores should be combined within predictive models, and the outputs should be compared with clinical trial data and real-world data for validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2835080905d0341827dedcd78db2a0fd8423297a" target='_blank'>Impact of artificial intelligence on prognosis, shared decision-making, and precision medicine for patients with inflammatory bowel disease: a perspective and expert opinion</a></td>
          <td>
            Philippe Pinton
          </td>
          <td>2023-12-12</td>
          <td>Annals of Medicine</td>
          <td>6</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Background Inflammatory bowel disease (IBD) is a gastrointestinal chronic disease with an unpredictable disease course. Computational methods such as machine learning (ML) have the potential to stratify IBD patients for the provision of individualized care. The use of ML methods for IBD was surveyed, with an additional focus on how the field has changed over time. Methods On May 6, 2021, a systematic review was conducted through a search of MEDLINE and Embase databases, with the search structure (“machine learning” OR “artificial intelligence”) AND (“Crohn* Disease” OR “Ulcerative Colitis” OR “Inflammatory Bowel Disease”). Exclusion criteria included studies not written in English, no human patient data, publication before 2001, studies that were not peer reviewed, nonautoimmune disease comorbidity research, and record types that were not primary research. Results Seventy-eight (of 409) records met the inclusion criteria. Random forest methods were most prevalent, and there was an increase in neural networks, mainly applied to imaging data sets. The main applications of ML to clinical tasks were diagnosis (18 of 78), disease course (22 of 78), and disease severity (16 of 78). The median sample size was 263. Clinical and microbiome-related data sets were most popular. Five percent of studies used an external data set after training and testing for additional model validation. Discussion Availability of longitudinal and deep phenotyping data could lead to better modeling. Machine learning pipelines that consider imbalanced data and that feature selection only on training data will generate more generalizable models. Machine learning models are increasingly being applied to more complex clinical tasks for specific phenotypes, indicating progress towards personalized medicine for IBD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e80321ffdbad16fd298859ec042ad938aef9f00" target='_blank'>A Systematic Review of Artificial Intelligence and Machine Learning Applications to Inflammatory Bowel Disease, with Practical Guidelines for Interpretation</a></td>
          <td>
            I. Stafford, M. Gosink, E. Mossotto, S. Ennis, M. Hauben
          </td>
          <td>2022-06-14</td>
          <td>Inflammatory Bowel Diseases</td>
          <td>23</td>
          <td>38</td>
        </tr>

        <tr id="RS The colloquial concept of artificial intelligence (AI) represents a collection of analytic methods that are able to ingest data of any type—numerals, text, sounds, images, or videos—and find patterns that can predict an output (eg, a judgment, clinical outcome, or therapeutic decision). These analytic methods encompass types of machine learning, including support vector machines, ensemble methods such as random forest, and neural networks, with each tool suited for different types of data and situations. AI is neither artificial nor necessarily intelligent, although it can appear so by replicating complex human tasks and interpretations. The availability of data has fueled opportunities that show what AI is capable of achieving. It is very important to remember that AI is just another tool in a doctor’s tool belt; it just happens to be exceptionally powerful when used for the right problem. AI is appealing because treating inflammatory bowel disease (IBD) is difficult and requires many laborious, tedious, and time-consuming tasks. I would love to hand off tasks such as reviewing laboratory results for evidence of medication toxicity or efficacy, watching capsule endoscopy videos to look for ulcers, and grading endoscopic severity as a central reviewer, and these tasks need to be done competently, reliably, and fast, which AI can do. In addition, doctors often do not have the luxury of opening textbooks to figure out what to do or have complete confidence that a course of therapy will be helpful. They Ryan W. Stidham, MD, MS Associate Professor Inflammatory Bowel Disease Center Department of Internal Medicine Department of Computational Medicine and Bioinformatics University of Michigan Ann Arbor, Michigan">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac7ad4ee9b57d3d33aba0ec779c38348a92eb0b7" target='_blank'>Artificial Intelligence in Inflammatory Bowel Disease.</a></td>
          <td>
            R. Stidham
          </td>
          <td>2022-10-01</td>
          <td>Gastroenterology & hepatology</td>
          <td>5</td>
          <td>31</td>
        </tr>

        <tr id="Introduction Inflammatory bowel diseases (IBD) are immune-mediated conditions that are increasing in incidence and prevalence worldwide. Their assessment and monitoring are becoming increasingly important, though complex. The best disease control is achieved through tight monitoring of objective inflammatory parameters (such as serum and stool inflammatory markers), cross-sectional imaging and endoscopic assessment. Considering the complexity of the information obtained throughout a patient’s journey, artificial intelligence (AI) provides an ideal adjunct to existing tools to help diagnose, monitor and predict the course of disease of patients with IBD. Therefore, we propose a scoping review assessing AI’s role in diagnosis, monitoring and prognostication tools in patients with IBD. We aim to detect gaps in the literature and address them in future research endeavours. Methods and analysis We will search electronic databases, including Medline, Embase, Cochrane CENTRAL, CINAHL Complete, Web of Science and IEEE Xplore. Two reviewers will independently screen the abstracts and titles first and then perform the full-text review. A third reviewer will resolve any conflict. We will include both observational studies and clinical trials. Study characteristics will be extracted using a data extraction form. The extracted data will be summarised in a tabular format, following the imaging modality theme and the study outcome assessed. The results will have an accompanying narrative review. Ethics and dissemination Considering the nature of the project, ethical review by an institutional review board is not required. The data will be presented at academic conferences, and the final product will be published in a peer-reviewed journal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ec8fd9e1d27ab210c344af992a7e3639265e9c" target='_blank'>Role of artificial intelligence in imaging and endoscopy for the diagnosis, monitoring and prognostication of inflammatory bowel disease: a scoping review protocol</a></td>
          <td>
            M. Chavannes, Lynn Kysh, Mariangela Allocca, N. Krugliak Cleveland, M. Dolinger, Tom S Robbins, David T. Rubin, S. Sagami, B. Verstockt, Kerri L Novak
          </td>
          <td>2023-12-01</td>
          <td>BMJ Open Gastroenterology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Inflammatory bowel disease encompasses Crohn’s disease and ulcerative colitis and is characterized by uncontrolled, relapsing, and remitting course of inflammation in the gastrointestinal tract. Artificial intelligence represents a new era within the field of gastroenterology, and the amount of research surrounding artificial intelligence in patients with inflammatory bowel disease is on the rise. As clinical trial outcomes and treatment targets evolve in inflammatory bowel disease, artificial intelligence may prove as a valuable tool for providing accurate, consistent, and reproducible evaluations of endoscopic appearance and histologic activity, thereby optimizing the diagnosis process and identifying disease severity. Furthermore, as the applications of artificial intelligence for inflammatory bowel disease continue to expand, they may present an ideal opportunity for improving disease management by predicting treatment response to biologic therapies and for refining the standard of care by setting the basis for future treatment personalization and cost reduction. The purpose of this review is to provide an overview of the unmet needs in the management of inflammatory bowel disease in clinical practice and how artificial intelligence tools can address these gaps to transform patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cbadc4940ca547e9ff6e90a8782b65558d63098" target='_blank'>Artificial intelligence in inflammatory bowel disease: implications for clinical practice and future directions</a></td>
          <td>
            H. Ahmad, J. East, R. Panaccione, S. Travis, J. Canavan, K. Usiskin, Michael F. Byrne
          </td>
          <td>2023-04-20</td>
          <td>Intestinal Research</td>
          <td>10</td>
          <td>95</td>
        </tr>

        <tr id="Our objective was to review and exemplify how selected applications of artificial intelligence (AI) might facilitate and improve inflammatory bowel disease (IBD) care and to identify gaps for future work in this field. IBD is highly complex and associated with significant variation in care and outcomes. The application of AI to IBD has the potential to reduce variation in healthcare delivery and improve quality of care. AI refers to the ability of machines to mimic human intelligence. The range of AI's ability to perform tasks that would normally require human intelligence varies from prediction to complex decision‐making that more closely resembles human thought. Clinical applications of AI have been applied to study pathogenesis, diagnosis, and patient prognosis in IBD. Despite these advancements, AI in IBD is in its early development and has tremendous potential to transform future care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8af3be5ea9089a55c36b64ef58358cf2bac9b2e1" target='_blank'>Clinical applications of artificial intelligence and machine learning‐based methods in inflammatory bowel disease</a></td>
          <td>
            S. Cohen-Mekelburg, Sameer K Berry, R. Stidham, Ji Zhu, A. Waljee
          </td>
          <td>2021-02-01</td>
          <td>Journal of Gastroenterology and Hepatology</td>
          <td>10</td>
          <td>51</td>
        </tr>

        <tr id="Inflammatory bowel disease (IBD), comprising Crohn disease (CD) and ulcerative colitis (UC), is a prototypical complex disease that has been considered a significant healthcare burden in the healthcare system of Western countries. Considered as one major form of AI, machine learning (ML) is an application that performs repeated iterations of models progressively improving the performance of a specific task by adapting several methods including random forest (RF), support vector machine (SVM), longitudinal regression (LR), and elastic net regularized generalized linear model. ML has been used in several clinical studies comprising IBD patients to determine the differential diagnosis of IBD, to assess IBD, and to predict the response and clinical outcomes of medications used to treat IBD. However, ML-assisted diagnostic or predictive measures for the daily medical care of IBD patients has not been applied yet. Here, we briefly reviewed some representative studies and subsequently addressed the challenges and opportunities of AI in the medical care of IBD patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa826aac74e73b7acb7c3fdbf405cfdbff9dadef" target='_blank'>Artificial intelligence in inflammatory bowel disease: current status and opportunities</a></td>
          <td>
            Ji Li, J. Qian
          </td>
          <td>2020-03-02</td>
          <td>Chinese Medical Journal</td>
          <td>9</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11bc72402348f9198a43e78eb74bf93aa40928bf" target='_blank'>Correction to: A Computational Platform Integrating a Mechanistic Model of Crohn’s Disease for Predicting Temporal Progression of Mucosal Damage and Healing</a></td>
          <td>
            Sai Phanindra Venkatapurapu, R. Iwakiri, E. Udagawa, Nikhil Patidar, Zhen Qi, Ryoko Takayama, Kei Kumar, Yuki Sato, Marcelo Behar, P. Offner, G. Dwivedi, Haruna Miyasaka, Ryohsuke Ken Suzuki, A. L. Hamada, P. M. D’Alessandro, Jovelle L Fernandez
          </td>
          <td>2022-12-13</td>
          <td>Advances in Therapy</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Inflammatory bowel disease (IBD) is a complex chronic immune disease with two subtypes: Crohn's disease and ulcerative colitis. Considering the differences in pathogenesis, etiology, clinical presentation, and response to therapy among patients, gastroenterologists mainly rely on endoscopy to diagnose and treat IBD during clinical practice. However, as exemplified by the increasingly comprehensive ulcerative colitis endoscopic scoring system, the endoscopic diagnosis, evaluation, and treatment of IBD still rely on the subjective manipulation and judgment of endoscopists. In recent years, the use of artificial intelligence (AI) has grown substantially in various medical fields, and an increasing number of studies have investigated the use of this emerging technology in the field of gastroenterology. Clinical applications of AI have focused on IBD pathogenesis, etiology, diagnosis, and patient prognosis. Large-scale datasets offer tremendous utility in the development of novel tools to address the unmet clinical and practice needs for treating patients with IBD. However, significant differences among AI methodologies, datasets, and clinical findings limit the incorporation of AI technology into clinical practice. In this review, we discuss practical AI applications in the diagnosis of IBD via gastroenteroscopy and speculate regarding a future in which AI technology provides value for the diagnosis and treatment of IBD patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed361153b7b3ff5baface57c4dbd1e31f3e1a25a" target='_blank'>Artificial Intelligence in Inflammatory Bowel Disease Endoscopy: Advanced Development and New Horizons</a></td>
          <td>
            Yu Chang, Zhi Wang, Hai-Bo Sun, Yu-Qin Li, Tongyu Tang
          </td>
          <td>2023-04-17</td>
          <td>Gastroenterology Research and Practice</td>
          <td>10</td>
          <td>12</td>
        </tr>

        <tr id="


 Inﬂammatory bowel diseases (IBDs) are chronic relapsing inflammatory disorders that result from an inappropriate inflammatory response to the enteric microbiota in a genetically susceptible host. Because the molecular mechanisms of IBD have been the subject of intensive exploration, we assembled the available information into a suite of comprehensive causal biological network models, which offer a better visualisation and understanding of the processes underlying IBD.



 Biological expression language (BEL), which represents biological findings in a computable form was used for scientific text curation. The curated BEL statements were compiled using the OpenBEL framework 3.0.0. Cytoscape was used to visualise and analyse network properties. The network perturbation amplitude methodology was used to score the network models with transcriptomic data from public data repositories.



 The IBD network model suite consists of four independent models that represent signalling pathways contributing to biological processes involved in IBD. In the ‘wound healing’ model, fibroblast growth factor and thymosin β-4-associated signalling pathways emerged as the main routes leading to epithelial cell proliferation and thus to wound repair. Phosphoinositide 3-kinases and carcinoembryonic antigen-related cell adhesion molecule-associated pathways were identified in the ‘bacterial colonisation’ model as the primary pathways modulating bacterial adhesion and internalisation. In the ‘immunity’ model, nuclear factor-κB and myeloid differentiation primary response 88 emerged as the main contributors to immune responses. In the ‘epithelial barrier defence’ model, tumour necrosis factor- and interleukin 1-associated signalling cascades were the main routes leading to tight junction disruption and thereby, to increased permeability in the intestinal epithelium. The scoring of publicly available transcriptomic data sets onto these network models demonstrates that the IBD models capture the perturbation in each dataset accurately.



 This work is a continuation of our approach using computational biological network models and mathematical algorithms in order to interpret high-throughput molecular datasets. The IBD network model suite can provide better mechanistic insights of the transcriptional changes in IBD and constitutes a valuable tool in personalised medicine to further understand molecular heterogeneity and individual drug responses in IBD.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad3789b95779f133e6fb3177487fc1a8274905bc" target='_blank'>P109 Construction of a suite of computable biological network models focused on inflammatory bowel disease</a></td>
          <td>
            P. A. R. Castro, H. Yepiskoposyan, J. Szostak, M. Talikka, J. Hoeng
          </td>
          <td>2020-01-15</td>
          <td>Journal of Crohns & Colitis</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Inflammatory bowel diseases, namely ulcerative colitis and Crohn’s disease, are chronic and relapsing conditions that pose a growing burden on healthcare systems worldwide. Because of their complex and partly unknown etiology and pathogenesis, the management of ulcerative colitis and Crohn’s disease can prove challenging not only from a clinical point of view but also for resource optimization. Artificial intelligence, an umbrella term that encompasses any cognitive function developed by machines for learning or problem solving, and its subsets machine learning and deep learning are becoming ever more essential tools with a plethora of applications in most medical specialties. In this regard gastroenterology is no exception, and due to the importance of endoscopy and imaging numerous clinical studies have been gradually highlighting the relevant role that artificial intelligence has in inflammatory bowel diseases as well. The aim of this review was to summarize the most recent evidence on the use of artificial intelligence in inflammatory bowel diseases in various contexts such as diagnosis, follow-up, treatment, prognosis, cancer surveillance, data collection, and analysis. Moreover, insights into the potential further developments in this field and their effects on future clinical practice were discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c517331f2984e1d64fd3ea9a20594448f05777" target='_blank'>Artificial intelligence and inflammatory bowel disease: Where are we going?</a></td>
          <td>
            L. Da Rio, M. Spadaccini, T. Parigi, R. Gabbiadini, A. Dal Buono, A. Busacca, R. Maselli, A. Fugazza, M. Colombo, S. Carrara, G. Franchellucci, L. Alfarone, A. Facciorusso, C. Hassan, A. Repici, A. Armuzzi
          </td>
          <td>2023-01-21</td>
          <td>World Journal of Gastroenterology</td>
          <td>29</td>
          <td>79</td>
        </tr>

        <tr id="There is a rising interest in use of big data approaches to personalize treatment of inflammatory bowel diseases (IBDs) and to predict and prevent outcomes such as disease flares and therapeutic nonresponse. Machine learning (ML) provides an avenue to identify and quantify features across vast quantities of data to produce novel insights in disease management. In this review, we cover current approaches in ML-driven predictive outcomes modeling for IBD and relate how advances in other fields of medicine may be applied to improve future IBD predictive models. Numerous studies have incorporated clinical, laboratory, or omics data to predict significant outcomes in IBD, including hospitalizations, outpatient corticosteroid use, biologic response, and refractory disease after colectomy, among others, with considerable health care dollars saved as a result. Encouraging results in other fields of medicine support efforts to use ML image analysis-including analysis of histopathology, endoscopy, and radiology-to further advance outcome predictions in IBD. Though obstacles to clinical implementation include technical barriers, bias within data sets, and incongruence between limited data sets preventing model validation in larger cohorts, ML-predictive analytics have the potential to transform the clinical management of IBD. Future directions include the development of models that synthesize all aforementioned approaches to produce more robust predictive metrics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86809a75f12069d0ff2ab407316d108df8710165" target='_blank'>Machine Learning Predictive Outcomes Modeling in Inflammatory Bowel Diseases.</a></td>
          <td>
            A. Javaid, Omer Shahab, W. Adorno, Philip Fernandes, Eve May, S. Syed
          </td>
          <td>2021-08-21</td>
          <td>Inflammatory bowel diseases</td>
          <td>14</td>
          <td>18</td>
        </tr>

        <tr id="Genetic factors play an important role in the etiology of inflammatory bowel disease (IBD). The launch of genome-wide association study (GWAS) represents a landmark in the genetic study of human complex disease. Concurrently, computational methods have undergone rapid development during the past a few years, which led to the identification of numerous disease susceptibility loci. IBD is one of the successful examples of GWAS and related analyses. A total of 163 genetic loci and multiple signaling pathways have been identified to be associated with IBD. Pleiotropic effects were found for many of these loci; and risk prediction models were built based on a broad spectrum of genetic variants. Important gene-gene, gene-environment interactions and key contributions of gut microbiome are being discovered. Here we will review the different types of analyses that have been applied to IBD genetic study, discuss the computational methods for each type of analysis, and summarize the discoveries made in IBD research with the application of these methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0eff76d6f51709a74b60880a1f5fb1ba43b930c2" target='_blank'>Application of computational methods in genetic study of inflammatory bowel disease.</a></td>
          <td>
            Jin Li, Zhi Wei, H. Hakonarson
          </td>
          <td>2016-01-21</td>
          <td>World journal of gastroenterology</td>
          <td>7</td>
          <td>169</td>
        </tr>

        <tr id="Endoscopy, histology, and cross-sectional imaging serve as fundamental pillars in the detection, monitoring, and prognostication of inflammatory bowel disease (IBD). However, interpretation of these studies often relies on subjective human judgment, which can lead to delays, intra- and interobserver variability, and potential diagnostic discrepancies. With the rising incidence of IBD globally coupled with the exponential digitization of these data, there is a growing demand for innovative approaches to streamline diagnosis and elevate clinical decision-making. In this context, artificial intelligence (AI) technologies emerge as a timely solution to address the evolving challenges in IBD. Early studies using deep learning and radiomics approaches for endoscopy, histology, and imaging in IBD have demonstrated promising results for using AI to detect, diagnose, characterize, phenotype, and prognosticate IBD. Nonetheless, the available literature has inherent limitations and knowledge gaps that need to be addressed before AI can transition into a mainstream clinical tool for IBD. To better understand the potential value of integrating AI in IBD, we review the available literature to summarize our current understanding and identify gaps in knowledge to inform future investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75917459323eadc3832c7e76e758611c5cf010d2" target='_blank'>AI-luminating Artificial Intelligence in Inflammatory Bowel Diseases: A Narrative Review on the Role of AI in Endoscopy, Histology, and Imaging for IBD.</a></td>
          <td>
            Phillip Gu, Oreen Mendonca, D. Carter, S. Dube, Paul Wang, Xiuzhen Huang, Debiao Li, Jason H. Moore, Dermot P B McGovern
          </td>
          <td>2024-03-07</td>
          <td>Inflammatory bowel diseases</td>
          <td>4</td>
          <td>7</td>
        </tr>

        <tr id="Inflammatory bowel disease (IBD), which includes ulcerative colitis (UC) and Crohn’s disease (CD), is an idiopathic condition related to a dysregulated immune response to commensal intestinal microflora in a genetically susceptible host. As a global disease, the morbidity of IBD reached a rate of 84.3 per 100,000 persons and reflected a continued gradual upward trajectory. The medical cost of IBD is also notably extremely high. For example, in Europe, it has €3,500 in CD and €2,000 in UC per patient per year, respectively. In addition, taking into account the work productivity loss and the reduced quality of life, the indirect costs are incalculable. In modern times, the diagnosis of IBD is still a subjective judgment based on laboratory tests and medical images. Its early diagnosis and intervention is therefore a challenging goal and also the key to control its progression. Artificial intelligence (AI)-assisted diagnosis and prognosis prediction has proven effective in many fields including gastroenterology. In this study, support vector machines were utilized to distinguish the significant features in IBD. As a result, the reliability of IBD diagnosis due to its impressive performance in classifying and addressing region problems was improved. Convolutional neural networks are advanced image processing algorithms that are currently in existence. Digestive endoscopic images can therefore be better understood by automatically detecting and classifying lesions. This study aims to summarize AI application in the area of IBD, objectively evaluate the performance of these methods, and ultimately understand the algorithm–dataset combination in the studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49a4b92dca55ae65e6b1acb8f7888ada0dc6b357" target='_blank'>Artificial Intelligence Enhances Studies on Inflammatory Bowel Disease</a></td>
          <td>
            Guihua Chen, Jun Shen
          </td>
          <td>2021-07-08</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>14</td>
          <td>5</td>
        </tr>

        <tr id="Advances in next generation sequencing technologies allow the collection of enormous volumes of genomic data on large patient cohorts. Concurrently, machine learning algorithms are rapidly evolving and, together, these technologies represent the new frontier of research and clinical management on a path leading toward personalised medicine. The aims of this thesis are two. Firstly, to develop a mathematical framework for the analysis and integration of next generation sequencing data. Secondly, to model data from patients affected by inflammatory bowel disease (IBD), a common complex autoimmune condition with increasing incidence worldwide, by applying machine learning methodologies to clinical and transformed genomic data. The analyses presented in this thesis are largely based on a cohort of paediatric IBD patients for which clinical data, immunology and whole exome sequencing data were available. This research illustrates a supervised and unsupervised machine learning approach modelling histology and endoscopy data for assigning IBD patients with the correct CD/UC subtypes with superior accuracy. Stratification and classification of IBD patients can be improved by layering ge- nomic data on top of clinical evidence. This thesis also describes the development of GenePy, a mathematical model for transforming patients genomic data into a per-individual per-gene deleteriousness scoring system. GenePy is capable of modelling and implementing important biological information from whole exome sequencing data from patient DNA. GenePy eases the analysis and interpretation of genomic data on an individual basis and concomitantly allows the comparison of genetic profiles across patients. GenePy gene scores can be further combined according to molecular processes or pathways. This work describes eight novel immuno-genomic IBD sybtypes observed on a small cohort for which immune cytokine signalling and response cascades have been specifically profiled and GenePy scores obtained. In addition, the GenePy algorithm is applied using both supervised and unsuper- vised approaches to classify IBD subtypes and to explore alternative disease clas- sifications that discriminate molecular clinical subtypes that are clinically relevant for treatment and prognosis. This thesis reports the current highest performance in discriminating IBD subtypes using exome sequencing data and five novel ge- nomic patient strata defined by different mutational burden of adaptive immune system genes. This work demonstrates the power of integrating 21st century high throughput digital data in machine learning frameworks and the potential to obtain clinically relevant strata for bench to bedside improvements in patient quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d2fda055d35795aef42e566b10844c3cf1521d3" target='_blank'>Advanced modelling of genomic data in Inflammatory Bowel Disease</a></td>
          <td>
            E. Mossotto
          </td>
          <td>2018-05-01</td>
          <td></td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Crohn’s disease and ulcerative colitis remain debilitating disorders, characterized by progressive bowel damage and possible lethal complications. The growing number of applications for artificial intelligence in gastrointestinal endoscopy has already shown great potential, especially in the field of neoplastic and pre-neoplastic lesion detection and characterization, and is currently under evaluation in the field of inflammatory bowel disease management. The application of artificial intelligence in inflammatory bowel diseases can range from genomic dataset analysis and risk prediction model construction to the disease grading severity and assessment of the response to treatment using machine learning. We aimed to assess the current and future role of artificial intelligence in assessing the key outcomes in inflammatory bowel disease patients: endoscopic activity, mucosal healing, response to treatment, and neoplasia surveillance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3c22c2ada22087b9ae3bf034d44d603ca8049ef" target='_blank'>The Role of Artificial Intelligence in Monitoring Inflammatory Bowel Disease—The Future Is Now</a></td>
          <td>
            C. Diaconu, M. State, Mihaela Birligea, Madalina Ifrim, G. Bajdechi, Teodora Georgescu, B. Mateescu, T. Voiosu
          </td>
          <td>2023-02-01</td>
          <td>Diagnostics</td>
          <td>8</td>
          <td>16</td>
        </tr>

        <tr id="Purpose Inflammatory bowel disease (IBD) is difficult to diagnose and classify. The purpose of this study is to establish an artificial intelligence model based on fecal multi-omics data for multi-classification diagnosis of IBD and its subtypes. Materials and Methods A total of 299 clinical cohort studies were included in this study, including 86 healthy people, 140 CD patients and 73 UC patients. Based on the idea of hierarchical modeling for different groups, we model the total population and the groups with self-evaluation of “very well” and “slightly below par”, respectively. The original total features were fecal multi-omics data, including metagenomics, metatranscriptomics, proteomics, metabolomics, viromics, faecal calprotectin. The importance, collinearity and other feature engineering methods were used to evaluate the features. Finally, three individualized diagnosis models with less features and high accuracy were obtained. Results First, we screened 111 features to form the optimal feature set for the total population and established a three-classification individual diagnosis model with AUC of 0.83, which can simultaneously diagnose health, CD and UC. Secondly, according to the hierarchical modeling of the total population, we established two models for population with different self-evaluation. For “very well” population, we screened 59 features and established a three-classification diagnostic model with AUC of 0.85. For the self-evaluation population with “slightly below par”, we finally included 22 features and established a three-classification diagnostic model with AUC of 0.84. Only metabolomics and metatranscriptomics features were included in the optimal feature sets. Conclusion This study provides a valuable method for high accuracy, noninvasive diagnosis and subtype identification of IBD patients. Researchers can choose biomarkers in different models according to different self-evaluation of patients. Simple noninvasive fecal sampling can be used to detect metabolomics and metatranscriptomics data, thus replacing the tedious and painful clinical colonoscopy and biopsy procedures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd3338184b4807f968c1d5e3bf5fe7f1e5b78313" target='_blank'>Application of Artificial Intelligence Modeling Technology Based on Multi-Omics in Noninvasive Diagnosis of Inflammatory Bowel Disease</a></td>
          <td>
            Qiongrong Huang, Xiuli Zhang, Zhiyuan Hu
          </td>
          <td>2021-05-14</td>
          <td>Journal of Inflammation Research</td>
          <td>8</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0c7f1c5a4395dea79aed2fecfb1ff6fac41b5be" target='_blank'>How Artificial Intelligence Will Transform Clinical Care, Research, and Trials for Inflammatory Bowel Disease.</a></td>
          <td>
            Anna L. Silverman, Dennis L. Shung, R. Stidham, G. Kochhar, M. Iacucci
          </td>
          <td>2024-07-01</td>
          <td>Clinical gastroenterology and hepatology : the official clinical practice journal of the American Gastroenterological Association</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="Research of inflammatory bowel disease (IBD) has identified numerous molecular players involved in the disease development. Even so, the understanding of IBD is incomplete, while disease treatment is still far from the precision medicine. Reliable diagnostic and prognostic biomarkers in IBD are limited which may reduce efficient therapeutic outcomes. High-throughput technologies and artificial intelligence emerged as powerful tools in search of unrevealed molecular patterns that could give important insights into IBD pathogenesis and help to address unmet clinical needs. Machine learning, a subtype of artificial intelligence, uses complex mathematical algorithms to learn from existing data in order to predict future outcomes. The scientific community has been increasingly employing machine learning for the prediction of IBD outcomes from comprehensive patient data-clinical records, genomic, transcriptomic, proteomic, metagenomic, and other IBD relevant omics data. This review aims to present fundamental principles behind machine learning modeling and its current application in IBD research with the focus on studies that explored genomic and transcriptomic data. We described different strategies used for dealing with omics data and outlined the best-performing methods. Before being translated into clinical settings, the developed machine learning models should be tested in independent prospective studies as well as randomized controlled trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb61a6c5bab2aff2d029d9d775e291c06ee4180" target='_blank'>Machine Learning Modeling from Omics Data as Prospective Tool for Improvement of Inflammatory Bowel Disease Diagnosis and Clinical Classifications</a></td>
          <td>
            B. Stanković, N. Kotur, G. Nikčević, V. Gasic, B. Zukic, S. Pavlovic
          </td>
          <td>2021-09-01</td>
          <td>Genes</td>
          <td>16</td>
          <td>26</td>
        </tr>

        <tr id="


 Clinical evidence supports the combination therapy of vedolizumab (a gut-selective, anti-lymphocyte trafficking drug inhibiting α4β7-integrin) with JAK 1, 2, 3 inhibitors (JAKi) for Inflammatory Bowel Disease. However, the mechanistic explanation for the improved outcomes in Ulcerative Colitis patients treated with vedolizumab plus JAKi remains unknown and even more in Crohn's Disease (CD). In this study we used an unbiased in silico systems biology and artificial intelligence approach to gain a holistic view of the mechanisms that may underpin the potential benefits of the mentioned combination therapy in the treatment of CD.



 The drugs under study -vedolizumab and JAKi- and CD were characterized at protein level by compiling data from extensive literature search. Through it, human protein interaction networks focused on CD protein effectors were generated and used to construct mathematical models using Therapeutic Performance Mapping System (TPMS) technology (Anaxomics Biotech, Barcelona, Spain). Sampling-based methods were employed to assess the impact on CD of the combined therapy vedolizumab plus JAKi and to describe underlying molecular mechanisms.



 Characterization of CD allowed identifying 4 pathological processes (referred to as motives-M) underlying the manifestation of the disease: intestinal barrier disruption (M1), increased innate immune response (M2), chronic inflammation and Th1/Th17 adaptive immune response (M3) and tissue remodeling (M4) and the molecular effectors or proteins involved in each motive. The combination of vedolizumab and JAKi induced a high degree of reversion of the effectors altered in CD (54.76%), higher than the percentage reverted by the individual drugs. Combined vedolizumab plus JAKi therapy mainly modulated M3, where both drugs mostly converge although by distinct mechanisms, thus providing additional individual benefits. The combination therapy reverted a great percentage of M3 effectors, for example, CCR9 and FASLG which were increased in CD, were reverted (or downregulated). In addition to M3 modulation, the combination therapy also modulated M1 and M4. In M1, vedolizumab is the one that would provide more complementary mechanisms apart from those that already converge in both drugs. Regarding M4, it seems that main contribution comes from complementarity pathways of the different mechanisms of action of each individual drug.



 The constructed in silico models reveal that the combined vedolizumab plus JAKi therapy could modulate a high array of effectors altered in CD, more than any of the therapies alone, by enhancing the effects of the individual drugs, and provide a mechanistic rationale for employing the combination of these drugs as potential therapy for CD.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37afddf4916b5088749544ad27899c601491b710" target='_blank'>P969 Unveiling the molecular mechanisms of the combination of vedolizumab with JAK inhibitors in Crohn’s Disease through a systems biology and artificial intelligence-based approach</a></td>
          <td>
            I. Marín-Jiménez, M. Sierra-Ausín, T. Letosa-Abián, J. Aparicio, C. Montoto-Otero, S. Sánchez-Ramón
          </td>
          <td>2024-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Inflammatory bowel diseases (IBD) are chronic and recurrent inflammatory disorders of the gastrointestinal tract. The elucidation of their etiopathology requires complex and multiple approaches. Systems biology has come to fulfill this need in approaching the pathogenetic mechanisms of IBD and its etiopathology, in a comprehensive way, by combining data from different scientific sources. In combination with bioinformatics and network medicine, it uses principles from computer science, mathematics, physics, chemistry, biology, medicine and computational tools to achieve its purposes. Systems biology utilizes scientific sources that provide data from omics studies (e.g., genomics, transcriptomics, etc.) and clinical observations, whose combined analysis leads to network formation and ultimately to a more integrative image of disease etiopathogenesis. In this review, we analyze the current literature on the methods and the tools utilized by systems biology in order to cover an innovative and exciting field: IBD-omics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07cfd2ecf64a7bb7c4f9450a909d9737a93ec2b1" target='_blank'>Systems biology in inflammatory bowel diseases: on the way to precision medicine</a></td>
          <td>
            N. Dovrolis, E. Filidou, G. Kolios
          </td>
          <td>2019-04-03</td>
          <td>Annals of Gastroenterology</td>
          <td>11</td>
          <td>42</td>
        </tr>

        <tr id="The project aims to develop a system For predicting inflammatory bowel and building the machine learning model based on routinely performed laboratory blood, urine, and fecal tests to supportdifferentiation between IBD patients and non- IBD patients comparison of the effectiveness of our model to standard inflammatory serummarker, that is C-reactive protein (CRP), in the prediction of IBD, creating a website- based application supporting the prediction of the presence of IBD . the age profile of IBD patients is changing and there is an increase in early-onset and late-onset IBD prevalence. Both groups (older adults, which frequently suffer from various comorbidities, as well as children) would particularly benefit from the non-invasive diagnostic test. However, colonoscopy remains a gold standard in IBD diagnosis, monitoring of the disease course, and response to the therapy, as well as colorectal cancer screening [6,7,8,9]. Still, despite its obvious advantages, it is highly invasive, expensive, time- consuming, requires qualified medical personnel and patient‘s preparation, and is often poorly tolerated by patients themselves. Besides, in the pandemic all low-contact medical procedures are preferred. creating a website- based application supporting the prediction ofthe presence of IBD.Therefore, a simple diagnostic methodology based only on markers from blood, urine and stool that can be performed by a GP would be imperative inthe early diagnosis of IBD Keywords : inflammatoryboweldisease;ulcerative colitis; Crohn’s disease; artificial intelligence; machine">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c742c528df3db8aaf6cf16b573d2237b004eeee3" target='_blank'>Classification and prediction of inflammatory bowel diseases</a></td>
          <td>
            Shiva Kumar
          </td>
          <td>2023-12-01</td>
          <td>INTERANTIONAL JOURNAL OF SCIENTIFIC RESEARCH IN ENGINEERING AND MANAGEMENT</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77b5012f30dcc2c3a8a481e15f21fa3171167151" target='_blank'>Artificial intelligence and endo-histo-omics: new dimensions of precision endoscopy and histology in inflammatory bowel disease.</a></td>
          <td>
            M. Iacucci, G. Santacroce, I. Zammarchi, Yasuharu Maeda, R. del Amor, Pablo Meseguer Esbri, B. B. Kolawole, U. Chaudhari, A. Di Sabatino, S. Danese, Y. Mori, E. Grisan, Valery Naranjo, Subrata Ghosh
          </td>
          <td>2024-05-01</td>
          <td>The lancet. Gastroenterology & hepatology</td>
          <td>5</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c7f1d633797ece58d05c35e4f0d109c32da0644" target='_blank'>Real-World Guidance from Artificial Intelligence? Predicting Outcomes of Inflammatory Bowel Disease Using Machine Learning</a></td>
          <td>
            Danny Con, A. Vasudevan
          </td>
          <td>2022-05-03</td>
          <td>Digestive Diseases and Sciences</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Implications of artificial intelligence in inflammatory bowel disease: Diagnosis, prognosis and treatment follow up">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7846dead413ea148b5d3e89a6c526c2d271e0de" target='_blank'>Implications of artificial intelligence in inflammatory bowel disease: Diagnosis, prognosis and treatment follow up</a></td>
          <td>
            A. Almomani, A. Hitawala, Mohammad Abureesh, Thabet Qapaja, Dana Alshaikh, M. Zmaili, M. Saleh, M. Alkhayyat
          </td>
          <td>2021-06-28</td>
          <td>Artificial Intelligence in Gastroenterology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Inflammatory bowel diseases (IBD), subdivided into Crohn’s disease (CD) and ulcerative colitis (UC), are chronic diseases that are characterized by relapsing and remitting periods of inflammation in the gastrointestinal tract. In recent years, the amount of research surrounding digital health (DH) and artificial intelligence (AI) has increased. The purpose of this scoping review is to explore this growing field of research to summarize the role of DH and AI in the diagnosis, treatment, monitoring and prognosis of IBD. A review of 21 articles revealed the impact of both AI algorithms and DH technologies; AI algorithms can improve diagnostic accuracy, assess disease activity, and predict treatment response based on data modalities such as endoscopic imaging and genetic data. In terms of DH, patients utilizing DH platforms experienced improvements in quality of life, disease literacy, treatment adherence, and medication management. In addition, DH methods can reduce the need for in-person appointments, decreasing the use of healthcare resources without compromising the standard of care. These articles demonstrate preliminary evidence of the potential of DH and AI for improving the management of IBD. However, the majority of these studies were performed in a regulated clinical environment. Therefore, further validation of these results in a real-world environment is required to assess the efficacy of these methods in the general IBD population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/865308d73392495514218dc36fb7de7e0a12cdf1" target='_blank'>Role of Digital Health and Artificial Intelligence in Inflammatory Bowel Disease: A Scoping Review</a></td>
          <td>
            Kamila Majidova, J. Handfield, K. Kafi, Ryan D Martin, Ryszard Kubinski
          </td>
          <td>2021-09-22</td>
          <td>Genes</td>
          <td>10</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f177f4c8e465f58a83e51814dfa657200f6e1f82" target='_blank'>Artificial Intelligence for Inflammatory Bowel Diseases (IBD); Accurately Predicting Adverse Outcomes Using Machine Learning</a></td>
          <td>
            A. Zand, Zack Stokes, Arjun Sharma, Welmoed K. van Deen, D. Hommes
          </td>
          <td>2022-04-27</td>
          <td>Digestive Diseases and Sciences</td>
          <td>15</td>
          <td>78</td>
        </tr>

        <tr id="Inflammatory bowel disease (IBD) is a chronic immune-mediated condition arising due to complex interactions between multiple genetic and environmental factors. Despite recent advances, the pathogenesis of the condition is not fully understood and patients still experience suboptimal clinical outcomes. Over the past few years, investigators are increasingly capturing multi-omics data from patient cohorts to better characterise the disease. However, reaching clinically translatable endpoints from these complex multi-omics datasets is an arduous task. Network biology, a branch of systems biology that utilises mathematical graph theory to represent, integrate and analyse biological data through networks, will be key to addressing this challenge. In this narrative review, we provide an overview of various types of network biology approaches that have been utilised in IBD including protein-protein interaction networks, metabolic networks, gene regulatory networks and gene co-expression networks. We also include examples of multi-layered networks that have combined various network types to gain deeper insights into IBD pathogenesis. Finally, we discuss the need to incorporate other data sources including metabolomic, histopathological, and high-quality clinical meta-data. Together with more robust network data integration and analysis frameworks, such efforts have the potential to realise the key goal of precision medicine in IBD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c4537b203113177c1ee4e3661fa0e6604c1e3fd" target='_blank'>Network Biology Approaches to Achieve Precision Medicine in Inflammatory Bowel Disease</a></td>
          <td>
            John P. Thomas, D. Módos, T. Korcsmáros, J. Brooks-Warburton
          </td>
          <td>2021-10-21</td>
          <td>Frontiers in Genetics</td>
          <td>12</td>
          <td>38</td>
        </tr>

        <tr id="Ulcerative colitis (UC) is a serious inflammatory bowel disease (IBD) with high morbidity and mortality worldwide. As the traditional diagnostic techniques have various limitations in the practice and diagnosis of early ulcerative colitis, it is necessary to develop new diagnostic models from molecular biology to supplement the existing methods. In this study, we developed a machine learning-based synthesis to construct an artificial intelligence diagnostic model for ulcerative colitis, and the correctness of the model is verified using an external independent dataset. According to the significantly expressed genes related to the occurrence of UC in the model, an unsupervised quantitative ulcerative colitis related score (UCRScore) based on principal coordinate analysis was established. The UCRScore is not only highly generalizable across UC bulk cohorts at different stages, but also highly generalizable across single-cell datasets, with the same effect in terms of cell numbers, activation pathways and mechanisms. As an important role of screening genes in disease occurrence, based on connectivity map analysis, 5 potential targeting molecular compounds were identified, which can be used as an additional supplement to the therapeutic of UC. Overall, this study provides a potential tool for differential diagnosis and assessment of bio-pathological changes in UC at the macroscopic level, providing an opportunity to optimize the diagnosis and treatment of UC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbc27a7e1fc8552124aa48144e6eaadd57421d36" target='_blank'>An assessment system for clinical and biological interpretability in ulcerative colitis</a></td>
          <td>
            Shiqian Zhang, Ge Zhang, Wenxiu Wang, Song-Bin Guo, Pengpeng Zhang, Fuqi Wang, Quanbo Zhou, Zhaokai Zhou, Yujia Wang, Haifeng Sun, Wenming Cui, Shuaixi Yang, Weitang Yuan
          </td>
          <td>2024-02-16</td>
          <td>Aging (Albany NY)</td>
          <td>3</td>
          <td>12</td>
        </tr>

        <tr id="Artificial intelligence (AI) is an emerging technology predicted to have significant applications in healthcare. This review highlights AI applications that impact the patient journey in inflammatory bowel disease (IBD), from genomics to endoscopic applications in disease classification, stratification and self-monitoring to risk stratification for personalised management. We discuss the practical AI applications currently in use while giving a balanced view of concerns and pitfalls and look to the future with the potential of where AI can provide significant value to the care of the patient with IBD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c81d1ec734a63a46657d7e1ba34bfd6d2556e8a" target='_blank'>Artificial intelligence and inflammatory bowel disease: practicalities and future prospects</a></td>
          <td>
            Johanne Brooks-Warburton, J. Ashton, A. Dhar, T. Tham, P. Allen, S. Hoque, L. Lovat, S. Sebastian
          </td>
          <td>2021-12-10</td>
          <td>Frontline Gastroenterology</td>
          <td>8</td>
          <td>47</td>
        </tr>

        <tr id="A large and growing body of research implicates aberrant immune response and compositional shifts of the intestinal microbiota in the pathogenesis of many intestinal disorders. The molecular and physical interaction between the host and the microbiota, known as the host-microbiota interactome, is one of the key drivers in the pathophysiology of many of these disorders. This host-microbiota interactome is a set of dynamic and complex processes, and needs to be treated as a distinct entity and subject for study. Disentangling this complex web of interactions will require novel approaches, using a combination of data-driven bioinformatics with knowledge-driven computational modeling. This review describes the computational approaches for investigating the host-microbiota interactome, with emphasis on the human intestinal tract and innate immunity, and highlights open challenges and existing gaps in the computation methodology for advancing our knowledge about this important facet of human health.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d77db0d1024922887d540bb6716cf6d6728a411d" target='_blank'>Computational Studies of the Intestinal Host-Microbiota Interactome</a></td>
          <td>
            S. Christley, Chase Cockrell, G. An
          </td>
          <td>2015-01-14</td>
          <td>Computation (Basel, Switzerland)</td>
          <td>7</td>
          <td>39</td>
        </tr>

        <tr id="© Author(s) (or their employer(s)) 2022. No commercial reuse. See rights and permissions. Published by BMJ. The rise of artificial intelligence (AI) in healthcare provides an opportunity to improve clinical care and patient outcomes. Gastroenterology may be seen as a leader in the application of AI, through automatic image recognition and interpretation in endoscopy, and analysis of images gathered through video capsule endoscopy. Despite this, the clinical translation of AI into routine practice has lagged behind its application in research settings. Inflammatory bowel disease (IBD) presents specific clinical challenges for which AI may have solutions, including prediction of therapeutic response, novel subgroup classification, precise molecular diagnosis, complication risk stratification and endoscopic image analysis for scoring of severity of mucosal inflammation in both ulcerative colitis and Crohn’s disease. We are also moving to an era of big data in IBD research, with consortia collecting and collating large cohorts of patients with available genomic, and other multiomic, data. 4 This resource presents an unrivalled opportunity to alter the landscape of disease prediction and classification in IBD, and usher in routine personalisation of diagnosis and treatment (figure 1). A key part of reliable, reproducible and applicable use of AI for patients, is the quality of the clinical phenotyping. Lack of indepth clinical data, systemic bias in data entry, lack of longitudinal outcomes and missing data all pose huge challenges to application of AI, with algorithms reliant on highquality data input to give highquality output. While this constitutes a challenge, it also creates an opportunity to develop robust systems to gather prospective and retrospective data, in structured ways and to use routinely collected data for alternative purposes. Within this opinion article, we focus on the use of AI to predict outcomes, response to therapy, complications and novel disease subgroups, from molecular and ‘big’ clinical data. We aim to highlight the current issues associated with limited clinical phenotyping of patients, reinforce the importance of precise longitudinal clinical data collection, with specific examples of translational opportunities for big data in IBD. We propose potential mechanisms for optimisation of future clinical application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be79199fd71c4989a6b54e7ed458d3406417847c" target='_blank'>The importance of high-quality ‘big data’ in the application of artificial intelligence in inflammatory bowel disease</a></td>
          <td>
            J. Ashton, Johanne Brooks-Warburton, P. Allen, T. Tham, S. Hoque, N. Kennedy, A. Dhar, Shaji Sebastian
          </td>
          <td>2022-11-17</td>
          <td>Frontline Gastroenterology</td>
          <td>2</td>
          <td>44</td>
        </tr>

        <tr id="Inflammatory bowel disease (IBD), with its 2 subtypes, Crohn’s disease and ulcerative colitis, is a complex chronic condition. A precise definition of disease activity and appropriate drug management greatly improve the clinical course while minimizing the risk or cost. Artificial intelligence (AI) has been used in several medical diseases or situations. Herein, we provide an overview of AI for endoscopy in IBD. We discuss how AI can improve clinical practice and how some components have already begun to shape our knowledge. There may be a time when we can use AI in clinical practice. As AI systems contribute to the exact diagnosis and treatment of human disease, we should continue to learn best practices in health care in the field of IBD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dd3f36595b335674338e503a869dc84bbab0b09" target='_blank'>Artificial intelligence for endoscopy in inflammatory bowel disease</a></td>
          <td>
            K. Takenaka, Ami Kawamoto, R. Okamoto, Mamoru Watanabe, K. Ohtsuka
          </td>
          <td>2022-01-07</td>
          <td>Intestinal Research</td>
          <td>10</td>
          <td>68</td>
        </tr>

        <tr id="Crohn's disease (CD) is a complex inflammatory bowel disease whose pathogenesis appears to involve several immunologic defects causing functional impairment of the gut. Its complexity and the reported loss of effectiveness over time of standard of care together with the increase in its worldwide incidence require the application of techniques aiming to find new therapeutic strategies. Currently, systems pharmacology modeling has been gaining importance as it integrates the available knowledge of the system into a single computational model. In this work, the following workflow for robust application of systems pharmacology modeling was followed: 1) scope definition; 2) species selection and circulating plasma levels based on a search in the literature; 3) representation of model topology and parametrization of the interactions, after literature data extraction and curation, and the implementation of ordinary differential equations in SimBiology (MATLAB version R2018b); and 4) model curation and evaluation by visual comparison of simulated interleukin (IL) concentrations with the reported levels in plasma, and sensitivity analysis performed to confirm model robustness and identify the most influential parameters. Finally, 5) exposure to two dose levels of recombinant human IL10 was evaluated by simulation and comparison with reported clinical study results. In summary, we present a quantitative systems pharmacology model for the main ILs involved in CD developed using a standardized methodology and supported by a comprehensive repository summarizing the most relevant literature in the field. However, it has to be taken into account that external validation is still pending as available clinical data were primarily used for model training. SIGNIFICANCE STATEMENT Crohn's disease (CD) is a complex heterogeneous inflammatory bowel disorder. Systems pharmacology modeling offers a great opportunity for integration of the available knowledge on the disease using a computational framework. As a result of this work, a comprehensive repository along with a quantitative systems pharmacology model for the main interleukins involved in CD is provided. This model is useful for the in silico evaluation of biomarkers and potential therapeutic targets and can be adapted to address research gaps regarding CD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1b59cbda9a63547cdc28a859ad8c140d010a08" target='_blank'>A Quantitative Systems Pharmacology Model for the Key Interleukins Involved in Crohn's Disease</a></td>
          <td>
            Violeta Balbas-Martinez, E. Asín-Prieto, Zinnia P. Parra-Guillen, I. Trocóniz
          </td>
          <td>2019-12-10</td>
          <td>The Journal of Pharmacology and Experimental Therapeutics</td>
          <td>8</td>
          <td>32</td>
        </tr>

        <tr id="


 The aim of this project is to model Inflammatory Bowel Diseases (IBD) progression using an innovative approach that considers the manifestation of the disease from a dynamic and multifactorial point of view, with a focus on model explainability.



 With the aim of modelling the course of IBD in the study population of the UR-CARE registry, an innovative approach will be applied consisting of the combined use of two different data-driven Artificial Intelligence techniques, namely dynamic Bayesian networks and Process Mining. Specifically, these two approaches will be jointly used to model IBD progression trajectories, providing a broader overview of the disease through the description of its patterns of progression (including clinical events, treatments and/or outcomes) and the interactions between clinical variables. The potential of the proposed methodology to address the predictive needs of chronic IBD progression, such as the forecasting of the next relapse, the effect of a therapy, or the impact of a risk factor, will also be explored.



 On the one side, IBD patients experience constant uncertainty regarding disease progression, while clinicians, on the other hand, need tools that can support them in understanding the multidimensionality of disease progression. In this scenario, AI can be the key to successfully satisfy these needs, effectively investigating the disease processes, allowing to describe pathological evolution over time, handling and capturing patients’ inter-variability, and providing tools to forecast disease evolution. By adopting an ad-hoc developed analytic approach, this project can help in better understanding IBD mechanisms and best care strategies, defining the relationships among the patient characteristics and the sequence of experienced clinical events, evaluating the impact of key elements on disease’s prognosis. These results would be useful not only to better manage the disease from a clinical and care point of view, but also in economic terms.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9f05d40c9251072474dc926cd10de52d0222352" target='_blank'>IIS Registry Grant Modelling Inflammatory Bowel Diseases trajectories combining dynamic, multifactorial, Artificial Intelligence-based approaches</a></td>
          <td>
            R. I. Comoretto, E. Tavazzi, M. Martinato, D. Azzolina
          </td>
          <td>2024-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Artificial intelligence (AI) has arrived and it will directly impact how we assess, monitor, and manage inflammatory bowel diseases (IBD). Advances in the machine learning methodologies that power AI have produced astounding results for replicating expert judgement and predicting clinical outcomes, particularly in the analysis of imaging. This review will cover general concepts for AI in IBD with descriptions of common machine learning methods including decision trees and neural networks. Applications of AI in IBD will cover recent achievements in endoscopic image interpretation and scoring, new capabilities for cross-sectional image analysis, natural language processing for automated understanding of clinical text, and progress in AI-powered clinical decision support tools. In addition to detailing current evidence supporting the capabilities of AI for replicating expert clinical judgement, speculative commentary on how AI may advance concepts of disease activity assessment, care pathways, and pathophysiologic mechanisms of IBD will be addressed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5254fba3ecccc0c5657cef8ecc206a1a115c807" target='_blank'>Artificial Intelligence for Disease Assessment in IBD: How Will it Change Our Practice?</a></td>
          <td>
            R. Stidham, K. Takenaka
          </td>
          <td>2021-12-14</td>
          <td>Gastroenterology</td>
          <td>37</td>
          <td>31</td>
        </tr>

        <tr id="Crohn’s disease is among the chronic inflammatory bowel diseases that impact the gastrointestinal tract. Understanding and predicting the severity of inflammation in real-time settings is critical to disease management. Extant literature has primarily focused on studies that are conducted in clinical trial settings to investigate the impact of a drug treatment on the remission status of the disease. This research proposes an analytics methodology where three different types of prediction models are developed to predict and to explain the severity of inflammation in patients diagnosed with Crohn’s disease. The results show that machine-learning-based analytic methods such as gradient boosting machines can predict the inflammation severity with a very high accuracy (area under the curve = 92.82%), followed by regularized regression and logistic regression. According to the findings, a combination of baseline laboratory parameters, patient demographic characteristics, and disease location are among the strongest predictors of inflammation severity in Crohn’s disease patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0058bb7c727ab6e55e302a97e71dc1887e1e551c" target='_blank'>Predicting and explaining inflammation in Crohn’s disease patients using predictive analytics methods and electronic medical record data</a></td>
          <td>
            B. Reddy, D. Delen, Rupesh Agrawal
          </td>
          <td>2019-12-01</td>
          <td>Health Informatics Journal</td>
          <td>36</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2726fe6f91ae4f70eb9c7f0f372c4265e6d0b0e" target='_blank'>[Development of the use of artificial intelligence in the management of chronic inflammatory bowel disease].</a></td>
          <td>
            J. Augustin, P. Mclellan, J. Calderaro
          </td>
          <td>2023-03-01</td>
          <td>Annales de pathologie</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed7f8a4abcdabe551cd8ce3581a34ce48ebb694" target='_blank'>Systems biology approach highlights mechanistic differences between Crohn’s disease and ulcerative colitis</a></td>
          <td>
            P. A. Ruiz Castro, H. Yepiskoposyan, Sylvain Gubian, Florian Calvino-Martin, U. Kogel, K. Renggli, M. Peitsch, J. Hoeng, M. Talikka
          </td>
          <td>2021-06-01</td>
          <td>Scientific Reports</td>
          <td>13</td>
          <td>62</td>
        </tr>

        <tr id="Chronic inflammatory diseases (CIDs), including inflammatory bowel disease (IBD), rheumatoid arthritis (RA) and systemic lupus erythematosus (SLE) are thought to emerge from an impaired complex network of inter- and intracellular biochemical interactions among several proteins and small chemical compounds under strong influence of genetic and environmental factors. CIDs are characterised by shared and disease-specific processes, which is reflected by partially overlapping genetic risk maps and pathogenic cells (e.g., T cells). Their pathogenesis involves a plethora of intracellular pathways. The translation of the research findings on CIDs molecular mechanisms into effective treatments is challenging and may explain the low remission rates despite modern targeted therapies. Modelling CID-related causal interactions as networks allows us to tackle the complexity at a systems level and improve our understanding of the interplay of key pathways. Here we report the construction, description, and initial applications of the SYSCID map (https://syscid.elixir-luxembourg.org/), a mechanistic causal interaction network covering the molecular crosstalk between IBD, RA and SLE. We demonstrate that the map serves as an interactive, graphical review of IBD, RA and SLE molecular mechanisms, and helps to understand the complexity of omics data. Examples of such application are illustrated using transcriptome data from time-series gene expression profiles following anti-TNF treatment and data from genome-wide associations studies that enable us to suggest potential effects to altered pathways and propose possible mechanistic biomarkers of treatment response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91e070e7e1b299f3a1a5565b1dc2fe5df95c949d" target='_blank'>The SYSCID map: a graphical and computational resource of molecular mechanisms across rheumatoid arthritis, systemic lupus erythematosus and inflammatory bowel disease</a></td>
          <td>
            M. Acencio, M. Ostaszewski, A. Mazein, Philip Rosenstiel, K. Aden, Neha Mishra, Vibeke Andersen, P. Sidiropoulos, A. Banos, Anastasia Filia, Souad Rahmouni, A. Finckh, Wei Gu, Reinhard Schneider, V. Satagopam
          </td>
          <td>2023-11-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Autoimmune rheumatic diseases (ARD) present a significant global health challenge characterized by a rising prevalence. These highly heterogeneous diseases involve complex pathophysiological mechanisms, leading to variable treatment efficacies across individuals. This variability underscores the need for personalized and precise treatment strategies. Traditionally, clinical practices have depended on empirical treatment selection, which often results in delays in effective disease management and can cause irreversible damage to multiple organs. Such delays significantly affect patient quality of life and prognosis. Artificial intelligence (AI) has recently emerged as a transformative tool in rheumatology, offering new insights and methodologies. Current research explores AI’s capabilities in diagnosing diseases, stratifying risks, assessing prognoses, and predicting treatment responses in ARD. These developments in AI offer the potential for more precise and targeted treatment strategies, fostering optimism for enhanced patient outcomes. This paper critically reviews the latest AI advancements for predicting treatment responses in ARD, highlights the current state of the art, identifies ongoing challenges, and proposes directions for future research. By capitalizing on AI’s capabilities, researchers and clinicians are poised to develop more personalized and effective interventions, improving care and outcomes for patients with ARD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbcb36593c4ceb14c26d697664d3d2325d826571" target='_blank'>Artificial intelligence for predicting treatment responses in autoimmune rheumatic diseases: advancements, challenges, and future perspectives</a></td>
          <td>
            Yanli Yang, Yang Liu, Yu Chen, Di-Cheng Luo, K. Xu, Liyun Zhang
          </td>
          <td>2024-10-22</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b6d19e4a519ae3668404d76c1f20c8c5b12e928" target='_blank'>Multiscale modeling of mucosal immune responses</a></td>
          <td>
            Yongguo Mei, V. Abedi, Adria Carbo, Xiaoying Zhang, Pinyi Lu, C. Philipson, R. Hontecillas, S. Hoops, Nathan Liles, J. Bassaganya-Riera
          </td>
          <td>2015-08-25</td>
          <td>BMC Bioinformatics</td>
          <td>34</td>
          <td>53</td>
        </tr>

        <tr id="   Currently scientists from different countries are exploring the possibilities of using machine learning methods to improve the accuracy of endoscopic and radiation diagnostics in patients with inflammatory bowel diseases (IBD) both to reduce the time spent by doctors on describing the results and to reduce the time needed to verify the diagnosis. Predicting the course of IBD based on artificial intelligence (AI) with the creation of predictive scenarios (models) is another promising area in gastroenterology. This review analyzes the main directions of scientific projects on the introduction of AI and machine learning methods in the diagnosis and prediction of the course of IBD. The article pays special attention to the problems faced by specialists in the application of AI methods, ways to solve them, as well as the prospects for using AI in patients with IBD. The possibilities of using AI for colorectal cancer screening and analysis of medical records are presented.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88360f15e216e489ab8f24bf7a365c11b75aef9b" target='_blank'>Artificial intelligence in the diagnostics and treatment of inflammatory bowel diseases (review)</a></td>
          <td>
            T. S. Fil’, I. G. Bakulin
          </td>
          <td>2024-09-21</td>
          <td>Koloproktologia</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e913f5b38ea1fa181c5c47d04c1ee10840eec9a" target='_blank'>Artificial intelligence guided discovery of a barrier-protective therapy in inflammatory bowel disease</a></td>
          <td>
            D. Sahoo, Lee Swanson, I. Sayed, G. Katkar, Stella-Rita C. Ibeawuchi, Y. Mittal, Rama F. Pranadinata, Courtney Tindle, MacKenzie Fuller, Dominik L. Stec, John T. Chang, W. Sandborn, Soumita Das, P. Ghosh
          </td>
          <td>2021-07-12</td>
          <td>Nature Communications</td>
          <td>42</td>
          <td>180</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>