<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              F96c7a83f219c01d8f6dd860d6912bbf755670b0
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Virtual%20cell/" class="md-tabs__link">
        
  
    
  
  Virtual cell

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../RA/" class="md-tabs__link">
        
  
    
  
  RA

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Prostate%20cancer/" class="md-tabs__link">
        
  
    
  
  Prostate cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Virtual%20patients/" class="md-tabs__link">
        
  
    
  
  Virtual patients

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Virtual%20cell/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual cell
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../RA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RA
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Prostate%20cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Prostate cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Virtual%20patients/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual patients
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-29 08:53:23 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Cell‐level systems biology model to study inflammatory bowel diseases and their treatment options</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="Motivation The literature on complex diseases is abundant but not always quantitative. This is particularly so for Inflammatory Bowel Disease (IBD), where many molecular pathways are qualitatively well described but this information cannot be used in traditional quantitative mathematical models employed in drug development. We propose the elaboration and validation of a logic network for IBD able to capture the information available in the literature that will facilitate the identification/validation of therapeutic targets. Results In this article, we propose a logic model for Inflammatory Bowel Disease (IBD) which consists of 43 nodes and 298 qualitative interactions. The model presented is able to describe the pathogenic mechanisms of the disorder and qualitatively describes the characteristic chronic inflammation. A perturbation analysis performed on the IBD network indicates that the model is robust. Also, as described in clinical trials, a simulation of anti-TNFα, anti-IL2 and Granulocyte and Monocyte Apheresis showed a decrease in the Metalloproteinases node (MMPs), which means a decrease in tissue damage. In contrast, as clinical trials have demonstrated, a simulation of anti-IL17 and anti-IFNγ or IL10 overexpression therapy did not show any major change in MMPs expression, as corresponds to a failed therapy. The model proved to be a promising in silico tool for the evaluation of potential therapeutic targets, the identification of new IBD biomarkers, the integration of IBD polymorphisms to anticipate responders and non-responders and can be reduced and transformed in quantitative model/s.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c655f04a31d1dc93671152d749f2e63985c7bf3" target='_blank'>A systems pharmacology model for inflammatory bowel disease</a></td>
          <td>
            Violeta Balbas-Martinez, Leire Ruiz-Cerdá, I. Irurzun-Arana, I. González-García, A. Vermeulen, J. Gómez-Mantilla, I. Trocóniz
          </td>
          <td>2018-03-07</td>
          <td>PLoS ONE</td>
          <td>31</td>
          <td>32</td>
        </tr>

        <tr id="
 Here we present the newly developed ENteric Immunity SImulator (ENISI) that simulates the antagonistic inflammatory and regulatory immune pathways of the gut as individual immune cells interact with and respond to commensal and foreign bacteria. This tool has been used to reproduce a typical inflammatory response to foreign bacteria as well as the immunopathological effects of autoimmunity to commensal bacteria with 10^6 individual cells. ENISI is built on an agent-based model that incorporates spatial effects and randomness of cell-cell and cell-bacteria contact, which can have a significant impact on whether a triggered inflammatory cascade is successfully regulated or spirals out of control. Preliminary simulations indicate the importance of cell density and epithelial cell-mediated recruitment in determining gut immunopathology. In addition, such representation allows emergent properties such as changes in bacterial demographics and evolution as a result of interaction with host immune cells. This tool will be useful to i) test the plausibility of in vitro observed immune cell responses as explanations for tissue-level damage, ii) propose mechanisms not yet tested in vitro that could explain tissue-level phenomenon, iii) conduct low-cost, preliminary tests of proposed interventions/ treatments for inflammatory diseases, iv) identify specific data to be gathered experimentally in order to characterize mechanisms of immune modulation in the gut.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac1d9b20e9f5580f0a85140fb3b01997e5b68824" target='_blank'>ENteric Immunity SImulator: A tool for in silico study of gut immunopathologies (166.15)</a></td>
          <td>
            K. Wendelsdorf, J. Bassaganya-Riera, Keith Bisset, Stephen Eubank, R. Hontecillas, Madhav V. Marathe
          </td>
          <td>2011-04-01</td>
          <td>The Journal of Immunology</td>
          <td>2</td>
          <td>53</td>
        </tr>

        <tr id="During acute and chronic infectious diseases hosts develop complex immune responses to cope with bacterial persistence. Depending on a variety of host and microbe factors, outcomes range from peaceful co-existence to detrimental disease. Mechanisms underlying immunity to bacterial stimuli span several spatiotemporal magnitudes and the summation of these hierarchical interactions plays a decisive role in pathogenic versus tolerogenic fate for the host. This dissertation integrates diverse data from immunoinformatics analyses, experimental validation and mathematical modeling to investigate a series of hypotheses driven by computational modeling to study mucosal immunity. Two contrasting microbes, enteroaggregative Escherichia coli and Helicobacter pylori, are used to perturb gut immunity in order to discover host-centric targets for modulating the host immune system. These findings have the potential to be broadly applicable to other infectious and immune-mediated diseases and could assist in the development of antibiotic-free and host-targeted treatments that modulate tolerance to prevent disease. This work was supported in part by the National Institute of Allergy and Infectious Diseases Contract No. HHSN272201000056C to Josep Bassaganya-Riera and funds from the Nutritional Immunology and Molecular Medicine Laboratory. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6533733f66c542d395ec6d41c0b1a59550a2e544" target='_blank'>Systems analysis and characterization of mucosal immunity</a></td>
          <td>
            C. Philipson
          </td>
          <td>2015-07-28</td>
          <td></td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Crohn's disease (CD) is a complex inflammatory bowel disease whose pathogenesis appears to involve several immunologic defects causing functional impairment of the gut. Its complexity and the reported loss of effectiveness over time of standard of care together with the increase in its worldwide incidence require the application of techniques aiming to find new therapeutic strategies. Currently, systems pharmacology modeling has been gaining importance as it integrates the available knowledge of the system into a single computational model. In this work, the following workflow for robust application of systems pharmacology modeling was followed: 1) scope definition; 2) species selection and circulating plasma levels based on a search in the literature; 3) representation of model topology and parametrization of the interactions, after literature data extraction and curation, and the implementation of ordinary differential equations in SimBiology (MATLAB version R2018b); and 4) model curation and evaluation by visual comparison of simulated interleukin (IL) concentrations with the reported levels in plasma, and sensitivity analysis performed to confirm model robustness and identify the most influential parameters. Finally, 5) exposure to two dose levels of recombinant human IL10 was evaluated by simulation and comparison with reported clinical study results. In summary, we present a quantitative systems pharmacology model for the main ILs involved in CD developed using a standardized methodology and supported by a comprehensive repository summarizing the most relevant literature in the field. However, it has to be taken into account that external validation is still pending as available clinical data were primarily used for model training. SIGNIFICANCE STATEMENT Crohn's disease (CD) is a complex heterogeneous inflammatory bowel disorder. Systems pharmacology modeling offers a great opportunity for integration of the available knowledge on the disease using a computational framework. As a result of this work, a comprehensive repository along with a quantitative systems pharmacology model for the main interleukins involved in CD is provided. This model is useful for the in silico evaluation of biomarkers and potential therapeutic targets and can be adapted to address research gaps regarding CD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1b59cbda9a63547cdc28a859ad8c140d010a08" target='_blank'>A Quantitative Systems Pharmacology Model for the Key Interleukins Involved in Crohn's Disease</a></td>
          <td>
            Violeta Balbas-Martinez, E. Asín-Prieto, Zinnia P. Parra-Guillen, I. Trocóniz
          </td>
          <td>2019-12-10</td>
          <td>The Journal of Pharmacology and Experimental Therapeutics</td>
          <td>8</td>
          <td>32</td>
        </tr>

        <tr id="Inflammatory bowel diseases (IBD) are chronic and recurrent inflammatory disorders of the gastrointestinal tract. The elucidation of their etiopathology requires complex and multiple approaches. Systems biology has come to fulfill this need in approaching the pathogenetic mechanisms of IBD and its etiopathology, in a comprehensive way, by combining data from different scientific sources. In combination with bioinformatics and network medicine, it uses principles from computer science, mathematics, physics, chemistry, biology, medicine and computational tools to achieve its purposes. Systems biology utilizes scientific sources that provide data from omics studies (e.g., genomics, transcriptomics, etc.) and clinical observations, whose combined analysis leads to network formation and ultimately to a more integrative image of disease etiopathogenesis. In this review, we analyze the current literature on the methods and the tools utilized by systems biology in order to cover an innovative and exciting field: IBD-omics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07cfd2ecf64a7bb7c4f9450a909d9737a93ec2b1" target='_blank'>Systems biology in inflammatory bowel diseases: on the way to precision medicine</a></td>
          <td>
            N. Dovrolis, E. Filidou, G. Kolios
          </td>
          <td>2019-04-03</td>
          <td>Annals of Gastroenterology</td>
          <td>11</td>
          <td>42</td>
        </tr>

        <tr id="Inflammatory bowel disease (IBD) is a chronic and relapsing disease with multiple underlying influences and notable heterogeneity among its clinical and response‐to‐treatment phenotypes. There is no cure for IBD, and none of the currently available therapies have demonstrated clinical efficacies beyond 40%–60%. Data collected about its omics, pathogenesis, and treatment strategies have grown exponentially with time making IBD a prime candidate for artificial intelligence (AI) mediated discovery support. AI can be leveraged to further understand or identify IBD features to improve clinical outcomes. Various treatment candidates are currently under evaluation in clinical trials, offering further approaches and opportunities for increasing the efficacies of treatments. However, currently, therapeutic plans are largely determined using clinical features due to the lack of specific biomarkers, and it has become necessary to step into precision medicine to predict therapeutic responses to guarantee optimal treatment efficacy. This is accompanied by the application of AI and the development of multiscale hybrid models combining mechanistic approaches and machine learning. These models ultimately lead to the creation of digital twins of given patients delivering on the promise of precision dosing and tailored treatment. Interleukin‐6 (IL‐6) is a prominent cytokine in cell‐to‐cell communication in the inflammatory responses’ regulation. Dysregulated IL‐6‐induced signaling leads to severe immunological or proliferative pathologies, such as IBD and colon cancer. This mini‐review explores multiscale models with the aim of predicting the response to therapy in IBD. Modeling IL‐6 biology and generating digital twins enhance the credibility of their prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89edceb6395ad6a9a46a561fb37a23bbccd0f06b" target='_blank'>Computational models in inflammatory bowel disease</a></td>
          <td>
            P. Pinton
          </td>
          <td>2022-02-05</td>
          <td>Clinical and Translational Science</td>
          <td>5</td>
          <td>40</td>
        </tr>

        <tr id="


 Clinical evidence supports the combination therapy of vedolizumab (a gut-selective, anti-lymphocyte trafficking drug inhibiting α4β7-integrin) with JAK 1, 2, 3 inhibitors (JAKi) for Inflammatory Bowel Disease. However, the mechanistic explanation for the improved outcomes in Ulcerative Colitis patients treated with vedolizumab plus JAKi remains unknown and even more in Crohn's Disease (CD). In this study we used an unbiased in silico systems biology and artificial intelligence approach to gain a holistic view of the mechanisms that may underpin the potential benefits of the mentioned combination therapy in the treatment of CD.



 The drugs under study -vedolizumab and JAKi- and CD were characterized at protein level by compiling data from extensive literature search. Through it, human protein interaction networks focused on CD protein effectors were generated and used to construct mathematical models using Therapeutic Performance Mapping System (TPMS) technology (Anaxomics Biotech, Barcelona, Spain). Sampling-based methods were employed to assess the impact on CD of the combined therapy vedolizumab plus JAKi and to describe underlying molecular mechanisms.



 Characterization of CD allowed identifying 4 pathological processes (referred to as motives-M) underlying the manifestation of the disease: intestinal barrier disruption (M1), increased innate immune response (M2), chronic inflammation and Th1/Th17 adaptive immune response (M3) and tissue remodeling (M4) and the molecular effectors or proteins involved in each motive. The combination of vedolizumab and JAKi induced a high degree of reversion of the effectors altered in CD (54.76%), higher than the percentage reverted by the individual drugs. Combined vedolizumab plus JAKi therapy mainly modulated M3, where both drugs mostly converge although by distinct mechanisms, thus providing additional individual benefits. The combination therapy reverted a great percentage of M3 effectors, for example, CCR9 and FASLG which were increased in CD, were reverted (or downregulated). In addition to M3 modulation, the combination therapy also modulated M1 and M4. In M1, vedolizumab is the one that would provide more complementary mechanisms apart from those that already converge in both drugs. Regarding M4, it seems that main contribution comes from complementarity pathways of the different mechanisms of action of each individual drug.



 The constructed in silico models reveal that the combined vedolizumab plus JAKi therapy could modulate a high array of effectors altered in CD, more than any of the therapies alone, by enhancing the effects of the individual drugs, and provide a mechanistic rationale for employing the combination of these drugs as potential therapy for CD.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37afddf4916b5088749544ad27899c601491b710" target='_blank'>P969 Unveiling the molecular mechanisms of the combination of vedolizumab with JAK inhibitors in Crohn’s Disease through a systems biology and artificial intelligence-based approach</a></td>
          <td>
            I. Marín-Jiménez, M. Sierra-Ausín, T. Letosa-Abián, J. Aparicio, C. Montoto-Otero, S. Sánchez-Ramón
          </td>
          <td>2024-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="The immune system is a complex and dynamic network, crucial for combating infections and maintaining health. Developing a comprehensive digital twin of the immune system requires incorporating essential cellular components and their interactions. This study presents the first blueprint for an immune system digital twin, consisting of a comprehensive and simulatable mechanistic model. It integrates 51 innate and adaptive immune cells, 37 secretory factors, and 11 disease conditions, providing the foundation for developing a multi-scale model. The cellular-level model demonstrates its potential in characterizing immune responses to various single and combinatorial disease conditions. By making the model available in easy-to-use formats directly in the Cell Collective platform, the community can easily and further expand it. This blueprint represents a significant step towards developing general-purpose immune digital twins, with far-reaching implications for the future of digital twin technology in life sciences and healthcare, advancing patient care, and accelerating precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6641c34ef9f1144419c9d233fad85729b60ada25" target='_blank'>Immune Digital Twin Blueprint: A Comprehensive Mechanistic Model of the Human Immune System</a></td>
          <td>
            Rada Amin, S. Aghamiri, B. L. Puniya, Lauren Mayo, Dennis Startsev, Kashish Poore, Resa M. K. Helikar, T. Helikar
          </td>
          <td>2020-03-12</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>24</td>
        </tr>

        <tr id="Ulcerative colitis, an inflammatory bowel disease, is a chronic inflammatory disorder that results in ulcers of the colon and rectum without known etiology.Ulcerative colitis causes a huge public health care burden particularly in developed countries.Many studies suggest that ulcerative colitis results from an abnormal immune response against components of commensal microbiota in genetically susceptible individuals.However, understanding of the disease mechanisms at cellular and molecular levels remains largely elusive.In this paper, a network model is developed based on our previous study and computer simulations are performed using an agent-based network modeling to elucidate the dynamics of immune response in ulcerative colitis progression.Our modeling study identifies several important positive feedback loops as a driving force for ulcerative colitis initiation and progression.The results demonstrate that although immune response in ulcerative colitis patients is dominated by anti-inflammatory/regulatory cells such as alternatively activated macrophages and type Ⅱ natural killer T cells, proinflammatory cells including classically activated macrophages, T helper 1 and T helper 17 cells, and their secreted cytokines tumor necrosis factor-α, interleukin-12, interleukin-23, interleukin-17 and interferon-γ remain at certain levels (lower than those in Crohn's disease, another inflammatory bowel disease).Long-term exposure to these proinflammatory components, causes mucosal tissue damage persistently, leading to ulcerative colitis.Our simulation results are qualitatively in agreement with clinical and laboratory measurements, offering novel insight into the disease mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78e59dd74ab5b9b66ead438b5b446f397142aed6" target='_blank'>Agent-Based Network Modeling Study of Immune Responses in Progression of Ulcerative Colitis</a></td>
          <td>
            Dao-rong Wu, Hai-shan Yu, J. Liao
          </td>
          <td>2018-04-27</td>
          <td>Chinese Journal of Chemical Physics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Inﬂammatory bowel diseases (IBDs) are chronic relapsing inflammatory disorders that result from an inappropriate inflammatory response to the enteric microbiota in a genetically susceptible host. Because the molecular mechanisms of IBD have been the subject of intensive exploration, we assembled the available information into a suite of comprehensive causal biological network models, which offer a better visualisation and understanding of the processes underlying IBD.



 Biological expression language (BEL), which represents biological findings in a computable form was used for scientific text curation. The curated BEL statements were compiled using the OpenBEL framework 3.0.0. Cytoscape was used to visualise and analyse network properties. The network perturbation amplitude methodology was used to score the network models with transcriptomic data from public data repositories.



 The IBD network model suite consists of four independent models that represent signalling pathways contributing to biological processes involved in IBD. In the ‘wound healing’ model, fibroblast growth factor and thymosin β-4-associated signalling pathways emerged as the main routes leading to epithelial cell proliferation and thus to wound repair. Phosphoinositide 3-kinases and carcinoembryonic antigen-related cell adhesion molecule-associated pathways were identified in the ‘bacterial colonisation’ model as the primary pathways modulating bacterial adhesion and internalisation. In the ‘immunity’ model, nuclear factor-κB and myeloid differentiation primary response 88 emerged as the main contributors to immune responses. In the ‘epithelial barrier defence’ model, tumour necrosis factor- and interleukin 1-associated signalling cascades were the main routes leading to tight junction disruption and thereby, to increased permeability in the intestinal epithelium. The scoring of publicly available transcriptomic data sets onto these network models demonstrates that the IBD models capture the perturbation in each dataset accurately.



 This work is a continuation of our approach using computational biological network models and mathematical algorithms in order to interpret high-throughput molecular datasets. The IBD network model suite can provide better mechanistic insights of the transcriptional changes in IBD and constitutes a valuable tool in personalised medicine to further understand molecular heterogeneity and individual drug responses in IBD.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad3789b95779f133e6fb3177487fc1a8274905bc" target='_blank'>P109 Construction of a suite of computable biological network models focused on inflammatory bowel disease</a></td>
          <td>
            P. A. R. Castro, H. Yepiskoposyan, J. Szostak, M. Talikka, J. Hoeng
          </td>
          <td>2020-01-15</td>
          <td>Journal of Crohns & Colitis</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b6d19e4a519ae3668404d76c1f20c8c5b12e928" target='_blank'>Multiscale modeling of mucosal immune responses</a></td>
          <td>
            Yongguo Mei, V. Abedi, Adria Carbo, Xiaoying Zhang, Pinyi Lu, C. Philipson, R. Hontecillas, S. Hoops, Nathan Liles, J. Bassaganya-Riera
          </td>
          <td>2015-08-25</td>
          <td>BMC Bioinformatics</td>
          <td>34</td>
          <td>53</td>
        </tr>

        <tr id="Chronic inflammatory diseases (CIDs), including inflammatory bowel disease (IBD), rheumatoid arthritis (RA) and systemic lupus erythematosus (SLE) are thought to emerge from an impaired complex network of inter- and intracellular biochemical interactions among several proteins and small chemical compounds under strong influence of genetic and environmental factors. CIDs are characterised by shared and disease-specific processes, which is reflected by partially overlapping genetic risk maps and pathogenic cells (e.g., T cells). Their pathogenesis involves a plethora of intracellular pathways. The translation of the research findings on CIDs molecular mechanisms into effective treatments is challenging and may explain the low remission rates despite modern targeted therapies. Modelling CID-related causal interactions as networks allows us to tackle the complexity at a systems level and improve our understanding of the interplay of key pathways. Here we report the construction, description, and initial applications of the SYSCID map (https://syscid.elixir-luxembourg.org/), a mechanistic causal interaction network covering the molecular crosstalk between IBD, RA and SLE. We demonstrate that the map serves as an interactive, graphical review of IBD, RA and SLE molecular mechanisms, and helps to understand the complexity of omics data. Examples of such application are illustrated using transcriptome data from time-series gene expression profiles following anti-TNF treatment and data from genome-wide associations studies that enable us to suggest potential effects to altered pathways and propose possible mechanistic biomarkers of treatment response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91e070e7e1b299f3a1a5565b1dc2fe5df95c949d" target='_blank'>The SYSCID map: a graphical and computational resource of molecular mechanisms across rheumatoid arthritis, systemic lupus erythematosus and inflammatory bowel disease</a></td>
          <td>
            M. Acencio, M. Ostaszewski, A. Mazein, Philip Rosenstiel, K. Aden, Neha Mishra, Vibeke Andersen, P. Sidiropoulos, A. Banos, Anastasia Filia, Souad Rahmouni, A. Finckh, Wei Gu, Reinhard Schneider, V. Satagopam
          </td>
          <td>2023-11-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/790e3a8ad8718788e35b3a0fdd34cb22338162ef" target='_blank'>A guide to systems-level immunomics</a></td>
          <td>
            L. Bonaguro, J. Schulte-Schrepping, T. Ulas, A. Aschenbrenner, M. Beyer, J. Schultze
          </td>
          <td>2022-09-22</td>
          <td>Nature Immunology</td>
          <td>46</td>
          <td>93</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/688bdf93e0b5923c287709c96a57a560ed30bee9" target='_blank'>A systems approach to dissecting immunity and inflammation.</a></td>
          <td>
            A. Aderem, Kelly D. Smith
          </td>
          <td>2004-02-01</td>
          <td>Seminars in immunology</td>
          <td>77</td>
          <td>98</td>
        </tr>

        <tr id="Systems biology is an approach that marries complimentary disciplines, encouraging the use of quantitative methods to help define, explain, and predict biological processes. By building computational models of biological systems, we can pose new biologically motivated questions and make falsifiable, quantitative predictions. In this thesis I will discuss the cycle of model building and experimental validation, and how it has provided insight into poorly and understood systems and allowed us to predict the effects of perturbations on these systems, which could have real and significant effects in human health and medicine. First, we model the activation of neutrophils in sepsis. By fitting a single model to two sets of data, coming from animals that survive and succumb to the same bacterial challenge, we create a realistic representation of biological variation, showing how a single network architecture can lead to different outcomes. Additionally, this method allows us to identify markers for sepsis susceptibility and identify and optimize a potential treatment option to lead to improved outcomes. Next, we model signaling downstream of the T cell receptor, and how this leads to differentiation decision making in CD4 T cells. By modeling the dynamics of this signaling network under varying antigen doses, we are able to identify network elements critical to dose discrimination, leading to the production of Treg cells following low dose stimulation and Th cells following high dose stimulation. We can then perturb these elements of the network, to potentially fine tune mature T cell populations to alter the trajectories of autoimmune disorders or cancer. Finally, we model the dynamics of IL-17 signaling. This allows us to understand how ubiquitin scaffolds form following cytokine stimulation, leading to the activation of NF-B, and how the ubiquitin editing enzyme A20 acts as a negative feedback regulator by breaking these chains. This allows us to better understand ubiquitin oligomerization as a fulcrum in the system, and how changes in A20 and ubiquitin binding proteins lead to different profiles of NF-B activation and could play a role in inflammatory disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c5de783dbb55c40e10e29aae4417239bfb08290" target='_blank'>Closing the loop: A combined computational modeling and experimental approach provides novel insights into immune cell signaling systems and their global effects.</a></td>
          <td>
            R. Sheehan
          </td>
          <td>2016-09-08</td>
          <td></td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Various extrinsic and intrinsic factors such as drug exposures, antibiotic treatments, smoking, lifestyle, genetics, immune responses, and the gut microbiome characterize ulcerative colitis and Crohn's disease, collectively called inflammatory bowel disease (IBD). All these factors contribute to the complexity and heterogeneity of the disease etiology and pathogenesis leading to major challenges for the scientific community in improving management, medical treatments, genetic risk, and exposome impact. Understanding the interaction(s) among these factors and their effects on the immune system in IBD patients has prompted advances in multi‐omics research, the development of new tools as part of system biology, and more recently, artificial intelligence (AI) approaches. These innovative approaches, supported by the availability of big data and large volumes of digital medical datasets, hold promise in better understanding the natural histories, predictors of disease development, severity, complications and treatment outcomes in complex diseases, providing decision support to doctors, and promising to bring us closer to the realization of the “precision medicine” paradigm. This review aims to provide an overview of current IBD omics based on both individual (genomics, transcriptomics, proteomics, metagenomics) and multi‐omics levels, highlighting how AI can facilitate the integration of heterogeneous data to summarize our current understanding of the disease and to identify current gaps in knowledge to inform upcoming research in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ce7ad87d298d804fa5b06fe06196a3d8f7fd6d2" target='_blank'>Inflammatory bowel disease genomics, transcriptomics, proteomics and metagenomics meet artificial intelligence</a></td>
          <td>
            Anna Lucia Cannarozzi, A. Latiano, L. Massimino, F. Bossa, Francesco Giuliani, Matteo Riva, F. Ungaro, Maria Guerra, Anna Laura Di Brina, G. Biscaglia, F. Tavano, S. Carparelli, G. Fiorino, S. Danese, F. Perri, O. Palmieri
          </td>
          <td>2024-08-31</td>
          <td>United European Gastroenterology Journal</td>
          <td>3</td>
          <td>57</td>
        </tr>

        <tr id="Clinical symptoms of microbial infection of the gastrointestinal (GI) tract are often exacerbated by inflammation induced pathology. Identifying novel avenues for treating and preventing such pathologies is necessary and complicated by the complexity of interacting immune pathways in the gut, where effector and inflammatory immune cells are regulated by anti-inflammatory or regulatory cells. Here we present new advances in the development of the ENteric Immunity SImulator (ENISI), a simulator of GI immune mechanisms in response to resident commensal bacteria as well as invading pathogens and the effect on the development of intestinal lesions. ENISI is a tool for identifying potential treatment strategies that reduce inflammation-induced damage and, at the same time, ensure pathogen removal by allowing one to test plausibility of in vitro observed behavior as explanations for observations in vivo, propose behaviors not yet tested in vitro that could explain these tissue-level observations, and conduct low-cost, preliminary experiments of proposed interventions/treatments. An example of such application is shown in which we simulate dysentery resulting from Brachyispira hyodysenteriae infection and identify aspects of the host immune pathways that lead to continued inflammation-induced tissue damage even after pathogen elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d95fa8a9091e8ee469d1ee1535064cb80e731604" target='_blank'>ENteric Immunity SImulator: A Tool for In Silico Study of Gastroenteric Infections</a></td>
          <td>
            K. Wendelsdorf, Maksudul Alam, J. Bassaganya-Riera, Keith Bisset, Stephen Eubank, R. Hontecillas, S. Hoops, Madhav V. Marathe
          </td>
          <td>2012-09-10</td>
          <td>IEEE Transactions on NanoBioscience</td>
          <td>40</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11bc72402348f9198a43e78eb74bf93aa40928bf" target='_blank'>Correction to: A Computational Platform Integrating a Mechanistic Model of Crohn’s Disease for Predicting Temporal Progression of Mucosal Damage and Healing</a></td>
          <td>
            Sai Phanindra Venkatapurapu, R. Iwakiri, E. Udagawa, Nikhil Patidar, Zhen Qi, Ryoko Takayama, Kei Kumar, Yuki Sato, Marcelo Behar, P. Offner, G. Dwivedi, Haruna Miyasaka, Ryohsuke Ken Suzuki, A. L. Hamada, P. M. D’Alessandro, Jovelle L Fernandez
          </td>
          <td>2022-12-13</td>
          <td>Advances in Therapy</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The main goal of pharmacogenomics is to study the effects of genetic variation on patient responses to therapies. Its applications range from the evaluation of safety and efficacy of treatment to the optimization of therapies and therapeutic regimens. Pharmacogenomics is becoming increasingly important in immunology, for the development of new generation vaccines, immunotherapies and transplantation. The human immune system is a complex and adaptive learning system which operates at multiple levels: molecules, cells, organs, organisms, and groups of organisms. Immunologic research, both basic and applied, needs to deal with this complexity. We increasingly use mathematical modeling and computational simulation in the study of the immune system and immune responses. Thus, quantitative models that appropriately capture the complexity in architecture and function of the immune system are an integral component of the personalized medicine efforts. In silico models of the immune system can provide answers to a variety of questions, including understanding the general behavior of the immune system, the course of disease, effects of treatment, analysis of cellular and molecular interactions, and simulation of laboratory experiments. We herein present the ImmunoGrid project that integrates molecular and system level models of the immune system and processes for in silico studies of the immune function. The ImmunoGrid simulator uses Grid technologies, enabling computational simulation of the immune system at the natural scale, perform a large number of simulated experiments, capture the diversity of the immune system between individuals, and provide a basis for therapeutic approaches tailored to the individual genetic make-up.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e556d3c380c6ad2ccc73657ca1993140f2f7743" target='_blank'>Computational Simulations of the Immune System for Personalized Medicine: State of the Art and Challenges</a></td>
          <td>
            F. Pappalardo, Ping Zhang, M. Halling-Brown, K. Basford, A. Scalia, A. Shepherd, D. Moss, S. Motta, V. Brusic
          </td>
          <td>2008-12-04</td>
          <td>Current Pharmacogenomics and Personalized Medicine</td>
          <td>18</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b0006e1689d55b986a1f8bbbaaf869960a56e2d" target='_blank'>2DIs: A SBML Compliant Web Platform for the Design and Modeling of Immune System Interactions</a></td>
          <td>
            M. Pennisi, G. Russo, G. Sgroi, Giuseppe Parasiliti, F. Pappalardo
          </td>
          <td>2017-08-07</td>
          <td>MAG, DBLP</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b24d2482771b58ffbe3f62693eb95f609bab357" target='_blank'>An immunology primer for computational modelers</a></td>
          <td>
            W. Hawse, P. Morel
          </td>
          <td>2014-09-20</td>
          <td>Journal of Pharmacokinetics and Pharmacodynamics</td>
          <td>3</td>
          <td>42</td>
        </tr>

        <tr id="Understanding and modelling the complexity of the immune system is a challenge that is shared by the ImmunoComplexiT$^1$ thematic network from the RNSC. The immune system is a complex biological, adaptive, highly diversified, self-organized and degenerative cognitive network of entities, allowing for a robust and resilient system with emergent properties such as anamnestic responses and regulation. The adaptive immune system has evolved into a complex system of billions of highly diversified lymphocytes all interacting as a connective dynamic, multi-scale organised and distributed system, in order to collectively insure body and species preservation. The immune system is characterized by complexity at different levels: network organisation through fluid cell populations with inter-and intra-cell signalling, lymphocyte receptor diversity, cell clonotype selection and competition at cell level, migration and interaction inside the immunological tissues and fluid dissemination through the organism, homeostatic regulation while rapid adaptation to a changing environment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c332120532334ab66de1348ba289da9bc43f684" target='_blank'>Understanding and Modelling the Complexity of the Immune System: Systems Biology for Integration and Dynamical Reconstruction of Lymphocyte Multi-Scale Dynamics</a></td>
          <td>
            V. Thomas-Vaslin
          </td>
          <td>2014-09-09</td>
          <td>arXiv: Quantitative Methods</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16a53e4f5ccd4166c9cfd1b548ed301889dc9292" target='_blank'>Seeking the ultimate challenge</a></td>
          <td>
            P. Brodin
          </td>
          <td>2019-05-24</td>
          <td>Nature Medicine</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc9a47faecb2a97afec94f6155b36f6276dd942d" target='_blank'>Systems immunology: a survey of modeling formalisms, applications and simulation tools</a></td>
          <td>
            V. Narang, James Decraene, S. Wong, B. Aiswarya, Andrew R. Wasem, Shiang Rong Leong, A. Gouaillard
          </td>
          <td>2012-04-13</td>
          <td>Immunologic Research</td>
          <td>37</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed7f8a4abcdabe551cd8ce3581a34ce48ebb694" target='_blank'>Systems biology approach highlights mechanistic differences between Crohn’s disease and ulcerative colitis</a></td>
          <td>
            P. A. Ruiz Castro, H. Yepiskoposyan, Sylvain Gubian, Florian Calvino-Martin, U. Kogel, K. Renggli, M. Peitsch, J. Hoeng, M. Talikka
          </td>
          <td>2021-06-01</td>
          <td>Scientific Reports</td>
          <td>14</td>
          <td>62</td>
        </tr>

        <tr id="The immune system plays a critical role in protecting the human body against various pathogens and diseases. Understanding the complexity and dynamics of the immune system is essential for developing effective therapies and interventions. Agent-based modeling (ABM) has emerged as a powerful tool for simulating and studying the behavior of complex systems, including the immune system. This review examines the advantages, challenges, and applications of ABM in immune system modeling. ABM captures the complexity of immune cell behavior, spatial effects and stochasticity. It has been applied to study immune cell dynamics, immune responses to pathogens, immune cell migration, immunotherapies and immune system disorders. Challenges include parameterization, validation, and computational resource requirements. Future directions involve integrating multi-omics and single-cell data, incorporating machine learning, exploring multi-scale modeling, and developing user-friendly interfaces. ABM holds promise for enhancing our understanding of immune system dynamics and advancing diagnostics and treatments in immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d234c3a3d3e3dfa896bd15157f48d6581b3a663c" target='_blank'>Modeling the Immune System Through Agent-based Modeling: A Mini-review</a></td>
          <td>
            Yousef Jamali
          </td>
          <td>2024-08-01</td>
          <td>Immunoregulation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The immune system is a complex and dynamic network, crucial for combating infections and maintaining health. Developing a comprehensive digital twin of the immune system requires incorporating essential cellular components and their interactions. This study presents the first blueprint for an immune system digital twin, consisting of a comprehensive and simulatable mechanistic model. It integrates 51 innate and adaptive immune cells, 37 secretory factors, and 11 disease conditions, providing the foundation for developing a multi-scale model. The cellular-level model demonstrates its potential in characterizing immune responses to various single and combinatorial disease conditions. By making the model available in easy-to-use formats directly in the Cell Collective platform, the community can easily and further expand it. This blueprint represents a significant step towards developing general-purpose immune digital twins, with far-reaching implications for the future of digital twin technology in life sciences and healthcare, advancing patient care, and accelerating precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6641c34ef9f1144419c9d233fad85729b60ada25" target='_blank'>Immune Digital Twin Blueprint: A Comprehensive Mechanistic Model of the Human Immune System</a></td>
          <td>
            Rada Amin, S. Aghamiri, B. L. Puniya, Lauren Mayo, Dennis Startsev, Kashish Poore, Resa M. K. Helikar, T. Helikar
          </td>
          <td>2020-03-12</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>24</td>
        </tr>

        <tr id="Here we present the ENteric Immunity Simulator (ENISI), a modeling system for the inflammatory and regulatory immune pathways triggered by microbe-immune cell interactions in the gut. With ENISI, immunologists and infectious disease experts can test and generate hypotheses for enteric disease pathology and propose interventions through experimental infection of an in silico gut. ENISI is an agent based simulator, in which individual cells move through the simulated tissues, and engage in context-dependent interactions with the other cells with which they are in contact. The scale of ENISI is unprecedented in this domain, with the ability to simulate 107 cells for 250 simulated days on 576 cores in one and a half hours, with the potential to scale to even larger hardware and problem sizes. In this paper we describe the ENISI simulator for modeling mucosal immune responses to gastrointestinal pathogens. We then demonstrate the utility of ENISI by recreating an experimental infection of a mouse with Helicobacter pylori 26695. The results identify specific processes by which bacterial virulence factors do and do not contribute to pathogenesis associated with H. pylori strain 26695. These modeling results inform general intervention strategies by indicating immunomodulatory mechanisms such as those used in inflammatory bowel disease may be more appropriate therapeutically than directly targeting specific microbial populations through vaccination or by using antimicrobials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69d094f8ca84e0157e1122cb5a03654f650b7321" target='_blank'>High-Performance Interaction-Based Simulation of Gut Immunopathologies with ENteric Immunity Simulator (ENISI)</a></td>
          <td>
            K. Bisset, Md. Maksudul Alam, J. Bassaganya-Riera, Adria Carbo, S. Eubank, R. Hontecillas, S. Hoops, Yongguo Mei, K. Wendelsdorf, D. Xie, Jae-Seung Yeom, M. Marathe
          </td>
          <td>2012-05-21</td>
          <td>2012 IEEE 26th International Parallel and Distributed Processing Symposium</td>
          <td>14</td>
          <td>56</td>
        </tr>

        <tr id="Abstract The immune system is a complex distributed system consisting of cells, which circulate through the body, communicate and turnover in response to antigenic perturbations. We discuss new approaches to modelling the functioning of the immune system of humans and experimental animals with a focus on its ‘complexity’. Emerging mathematical and computer models are reviewed to describe the immune system diversity, the cell/cytokine network communication structures, hierarchical regulation, and evolutionary dynamics of immune repertoires.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84cc5067b6048c367b415023cb486b2bd5b58cfd" target='_blank'>Multiphysics modelling of immune processes using distributed parameter systems</a></td>
          <td>
            Gennady A. Bocharov, D. S. Grebennikov, R. Savinkov
          </td>
          <td>2023-10-01</td>
          <td>Russian Journal of Numerical Analysis and Mathematical Modelling</td>
          <td>2</td>
          <td>7</td>
        </tr>

        <tr id="


 The aim of this project is to model Inflammatory Bowel Diseases (IBD) progression using an innovative approach that considers the manifestation of the disease from a dynamic and multifactorial point of view, with a focus on model explainability.



 With the aim of modelling the course of IBD in the study population of the UR-CARE registry, an innovative approach will be applied consisting of the combined use of two different data-driven Artificial Intelligence techniques, namely dynamic Bayesian networks and Process Mining. Specifically, these two approaches will be jointly used to model IBD progression trajectories, providing a broader overview of the disease through the description of its patterns of progression (including clinical events, treatments and/or outcomes) and the interactions between clinical variables. The potential of the proposed methodology to address the predictive needs of chronic IBD progression, such as the forecasting of the next relapse, the effect of a therapy, or the impact of a risk factor, will also be explored.



 On the one side, IBD patients experience constant uncertainty regarding disease progression, while clinicians, on the other hand, need tools that can support them in understanding the multidimensionality of disease progression. In this scenario, AI can be the key to successfully satisfy these needs, effectively investigating the disease processes, allowing to describe pathological evolution over time, handling and capturing patients’ inter-variability, and providing tools to forecast disease evolution. By adopting an ad-hoc developed analytic approach, this project can help in better understanding IBD mechanisms and best care strategies, defining the relationships among the patient characteristics and the sequence of experienced clinical events, evaluating the impact of key elements on disease’s prognosis. These results would be useful not only to better manage the disease from a clinical and care point of view, but also in economic terms.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9f05d40c9251072474dc926cd10de52d0222352" target='_blank'>IIS Registry Grant Modelling Inflammatory Bowel Diseases trajectories combining dynamic, multifactorial, Artificial Intelligence-based approaches</a></td>
          <td>
            R. I. Comoretto, E. Tavazzi, M. Martinato, D. Azzolina
          </td>
          <td>2024-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The revolutions in biotechnology and information technology have produced clinical data, which complement biological data. These data enable detailed descriptions of various healthy and diseased states and responses to therapies. For the investigation of the physiology and pathology of the immune responses, computer and mathematical models have been used in the last decades, enabling the representation of biological processes. In this modeling effort, a major issue is represented by the communication between models that work at cellular and molecular level, that is, multiscale representation. Here we sketch some attempts to model immune system dynamics at both levels.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d53a65277041949ff080632e0d1f6f7f6dee6a23" target='_blank'>In Silico Modeling of the Immune System: Cellular and Molecular Scale Approaches</a></td>
          <td>
            M. Belfiore, M. Pennisi, G. Aricò, S. Ronsisvalle, F. Pappalardo
          </td>
          <td>2014-04-06</td>
          <td>BioMed Research International</td>
          <td>8</td>
          <td>34</td>
        </tr>

        <tr id="Clinical symptoms of gastrointestinal (GI) infections are often caused by the inflammatory response elicited to eliminate the invading microbe. Here we present ENISI, a simulator of GI immune mechanisms in response to resident commensal bacteria as well as invading pathogens and the effect on host clinical symptoms. ENISI is a tool for identifying treatment strategies that reduce inflammation-induced damage and, at the same time, ensure pathogen removal by allowing one to test plausibility of in vitro observed behavior as explanations for observations in vivo, propose behaviors not yet tested in vitro that could explain these tissue-level observations, and conduct low-cost, preliminary experiments of proposed interventions/ treatments. An example of such application is shown in which we simulate dysentery resulting from B. hyodysenteriae infection and identify aspects of the host immune pathways that lead to continued inflammation-induced tissue damage even after pathogen elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca43cf82d33b6f299ac0772cea320019bb8e2eb8" target='_blank'>ENteric Immunity SImulator: A Tool for in silico Study of Gut Immunopathologies</a></td>
          <td>
            K. Wendelsdorf, J. Bassaganya-Riera, K. Bisset, S. Eubank, R. Hontecillas, M. Marathe
          </td>
          <td>2011-04-01</td>
          <td>2011 IEEE International Conference on Bioinformatics and Biomedicine</td>
          <td>16</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2748e1f482034fb8c6f447ea6439aa0a07c24d05" target='_blank'>System Dynamics Metamodels Supporting the Development of Computational Models of the Human Innate Immune System</a></td>
          <td>
            Igor Knop, A. B. Pigozzo, B. M. Quintela, G. Macedo, C. B. Barbosa, Rodrigo Weber dos Santos, M. Lobosco
          </td>
          <td>2012-06-18</td>
          <td>MAG, DBLP</td>
          <td>2</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67b3e39427e2dcb0bfceddeb6a6271805cac09ab" target='_blank'>A single-cell transcriptomic meta-analysis platform for inflammatory bowel disease</a></td>
          <td>

          </td>
          <td>2023-06-01</td>
          <td>Nature Computational Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Despite advances in the treatment of inflammatory bowel diseases (IBD), fewer than, 40% of patients reach clinical remission, suggesting that improved response rates may require targeting of multiple pathogenic pathways through combination therapy. However, the rational selection of combination therapies based on patient and pre-clinical data remains a significant challenge.



 Here, we used Crohn’s disease (CD) patient-derived molecular networks as a platform for bridging pre-clinical animal models of combination therapy to human disease. Intestinal transcriptional signatures of anti-TNF, anti-IL-23 or combination treatments were generated from the anti-CD40 agonistic antibody murine colitis model and the murine genes mapped to their human orthologues. Humanized gene signatures were intersected with human disease networks to generate treatment subnetworks and enrichment analyses performed.



 The anti-TNF subnetwork was enriched in genes expressed in myeloid cells, chemokine signaling and NFkB signaling, while the anti-IL-23 subnetwork was enriched in genes expressed in the intestinal epithelium, IL-17 signaling, and cell adhesion. The intersection of these two therapeutic gene signature subnetworks was significantly enriched in IBD GWAS loci genes and human CD inflammatory gene signatures, suggesting that the molecular mechanisms reflected in the pre-clinical model faithfully captures aspects of the human disease. Simultaneous inhibition of these two pathways could result in enhanced efficacy through targeting shared inflammatory pathways and complementary biology in myeloid and epithelial cells to reduce colitis. To test this hypothesis, mice were treated with varying doses of anti-IL-23, anti-TNF, or the combination of both. A synergistic response to combination therapy was observed both in reduction of systemic weight loss and inhibition of local colonic tissue inflammation by histopathology. A set of genes uniquely significantly modulated by the combination therapy compared to either monotherapy was mapped to our human IBD network. Upregulated gene networks in the combination therapy enriched in stromal cells, epithelial mesenchymal transition and extracellular matrix pathways, and adhesion pathways while downregulated gene networks were enriched in IBD disease signatures, M1 macrophages, neutrophils and IFN-γ signaling.



 These results provide a novel, data-driven approach to predict effective combination therapies for inflammatory diseases and suggest that anti-TNF and anti-IL-23 combination therapy may drive more patients into deep remission through impacts on both shared and unique molecular pathways involved in IBD pathogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93d6950c50e63c057dc02154fb1a9ca67255bcbd" target='_blank'>P328 In silico evaluation and pre-clinical efficacy of anti-TNF and anti-IL-23 combination therapy in Inflammatory Bowel Disease</a></td>
          <td>
            J. Perrigoue, L. Muniz-Bongers, L. Ong, Y. Chen, L. Chang, K. Ngo, A. Stojmirović, C. O'Brien, M. Germinaro, R. Rao, M. Vetter, J. Towne
          </td>
          <td>2022-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>3</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a044c589e6f5f2b20143495b590c29e0f1ed286a" target='_blank'>The development of a fully-integrated immune response model (FIRM) simulator of the immune response through integration of multiple subset models</a></td>
          <td>
            S. Palsson, T. Hickling, E. Bradshaw-Pierce, Michael Zager, K. Jooss, Peter O'Brien, M. Spilker, B. Palsson, P. Vicini
          </td>
          <td>2013-09-28</td>
          <td>BMC Systems Biology</td>
          <td>69</td>
          <td>142</td>
        </tr>

        <tr id="Immuno Systems Biology aims to study the immune system in the more integrated manner on how cells and molecules participate at different system levels to the immune function. Through this bookKumar Selvarajoointroduces to physicists, chemists, computer scientists, biologists and immunologists the idea of an integrated approach to the understanding of mammalian immune system. Geared towards a researcher with limited immunological and computational analytical experience, the book provides a broad overview to the subject and some instruction in basic computational, theoretical and experimental approaches. The book links complex immunological processes with computational analysis and emphasizes the importance of immunology to themammalian system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/772b7b2154433d1ddc9557f27af279b3fb1a6d6a" target='_blank'>Immuno Systems Biology: A macroscopic approach for immune cell signaling</a></td>
          <td>
            K. Selvarajoo
          </td>
          <td>2013-10-01</td>
          <td></td>
          <td>5</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2794e0a606a07512d2eec9ba3cc756c31bc68462" target='_blank'>The Basic Immune Simulator: An agent-based model to study the interactions between innate and adaptive immunity</a></td>
          <td>
            V. A. Folcik, G. An, C. Orosz
          </td>
          <td>2007-09-27</td>
          <td>Theoretical Biology & Medical Modelling</td>
          <td>163</td>
          <td>40</td>
        </tr>

        <tr id="Immunologists currently face daunting challenges, as a result of the rapid development of new methods for immunological data collection, from high-throughput phenotyping to deep sequencing (1). These and similar methods keep generating humongous amounts of immunological data, which in turn challenge the theoretical immunology community to develop methods for data organization and analysis and mathematical and computational modeling. These challenges and methods were discussed in recent workshops, for example the Lymphocyte Repertoire Workshop (Institute of Advanced Studies of the Hebrew University, Jerusalem, early 2012, organized by myself), and the International Seminar on Multi-Scale Physics of Lymphocyte Development (Max Planck Institute for the Physics of Complex Systems, Dresden, Summer 2012, organized by M. Or-Guil et al.). 

At about the same time, the organizers mentioned above were approached by the Frontiers editorial staff with the idea for a “Frontiers in Immunology” research topic, which was to provide a comprehensive, online, open access snapshot of the current state of the art on immune system modeling and analysis. The research topic was launched, edited, and finalized with the kind help of co-editors Rob de Boer, Miles Davenport, Carmen Molina-Paris, Michal Or-Guil, and Veronika Zarnitsyna. It has been a success, with 35 papers accepted for publication, which attests to the timeliness of the topic. 

The papers included in this Research Topic reflect many of the issues that theoretical immunologists are struggling with. Some of the papers address old questions – such as the targeting of somatic hypermutation (2) and the resulting diversity of B cell repertoires (3, 4), how clonal selection operates in germinal centers (5–8); or how the T cell compartment develops (9–11) and changes with aging (12). However, these papers offer new viewpoints, which emerged thanks to the immunological “data revolution”, in particular next-generation sequencing of lymphocyte repertoires. Others address new methods of extracting (13–15) and analyzing (16–18) comprehensive T and B cell phenotype and repertoire data, and delineate some of the first insights gleaned from sequencing studies regarding how these repertoires emerge, evolve, and function (19–25). Natural killer cells (26), myeloid cells (27), and structural immunology (28–31) are also represented. 

My thanks go to the above-mentioned co-editors, to the responsive and efficient Frontiers editorial staff, to all the authors who contributed papers, and to the reviewers whose work has made publication of all these papers possible.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/465bf75bc3ca87beac5a84f5c0178e1622821207" target='_blank'>Immune System Modeling and Analysis</a></td>
          <td>
            R. Mehr
          </td>
          <td>2014-12-19</td>
          <td>Frontiers in Immunology</td>
          <td>5</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9567528d3e57d85032a1bc54a9caf8aef234cb94" target='_blank'>Towards systems immunology of critical illness at scale: from single cell 'omics to digital twins.</a></td>
          <td>
            Y. Vodovotz
          </td>
          <td>2023-03-01</td>
          <td>Trends in immunology</td>
          <td>11</td>
          <td>73</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>