<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              1eb7cc43e57b554c6e40709726e054ae93f19e6c
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-21 10:42:05 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>A QSP model of prostate cancer immunotherapy to identify effective combination therapies</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="Immunotherapy has shown great potential in the treatment of cancer; however, only a fraction of patients respond to treatment, and many experience autoimmune‐related side effects. The pharmaceutical industry has relied on mathematical models to study the behavior of candidate drugs and more recently, complex, whole‐body, quantitative systems pharmacology (QSP) models have become increasingly popular for discovery and development. QSP modeling has the potential to discover novel predictive biomarkers as well as test the efficacy of treatment plans and combination therapies through virtual clinical trials. In this work, we present a QSP modeling platform for immuno‐oncology (IO) that incorporates detailed mechanisms for important immune interactions. This modular platform allows for the construction of QSP models of IO with varying degrees of complexity based on the research questions. Finally, we demonstrate the use of the platform through two example applications of immune checkpoint therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/750c1aa4fc2f364ea1febcf5761c49bd89bc2f93" target='_blank'>QSP‐IO: A Quantitative Systems Pharmacology Toolbox for Mechanistic Multiscale Modeling for Immuno‐Oncology Applications</a></td>
          <td>
            R. Sové, M. Jafarnejad, Chen Zhao, Hanwen Wang, Huilin Ma, A. Popel
          </td>
          <td>2020-07-02</td>
          <td>CPT: Pharmacometrics & Systems Pharmacology</td>
          <td>39</td>
          <td>69</td>
        </tr>

        <tr id="Modern cancer immunotherapy has revolutionised oncology and carries the potential to radically change the approach to cancer treatment. However, numerous questions remain to be answered to understand immunotherapy response better and further improve the benefit for future cancer patients. Computational models are promising tools that can contribute to accelerated immunotherapy research by providing new clues and hypotheses that could be tested in future trials, based on preceding simulations in addition to the empirical rationale. In this topical review, we briefly summarise the history of cancer immunotherapy, including computational modelling of traditional cancer immunotherapy, and comprehensively review computational models of modern cancer immunotherapy, such as immune checkpoint inhibitors (as monotherapy and combination treatment), co-stimulatory agonistic antibodies, bispecific antibodies, and chimeric antigen receptor T cells. The modelling approaches are classified into one of the following categories: data-driven top-down vs mechanistic bottom-up, simplistic vs detailed, continuous vs discrete, and hybrid. Several common modelling approaches are summarised, such as pharmacokinetic/pharmacodynamic models, Lotka–Volterra models, evolutionary game theory models, quantitative systems pharmacology models, spatio-temporal models, agent-based models, and logic-based models. Pros and cons of each modelling approach are critically discussed, particularly with the focus on the potential for successful translation into immuno-oncology research and routine clinical practice. Specific attention is paid to calibration and validation of each model, which is a necessary prerequisite for any successful model, and at the same time, one of the main obstacles. Lastly, we provide guidelines and suggestions for the future development of the field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62c69387fe4f031dfad70073875b6cf0e2738498" target='_blank'>Computational modelling of modern cancer immunotherapy</a></td>
          <td>
            D. Valentinuzzi, R. Jeraj
          </td>
          <td>2020-10-22</td>
          <td>Physics in Medicine & Biology</td>
          <td>20</td>
          <td>47</td>
        </tr>

        <tr id="Drug development in oncology commonly exploits the tools of molecular biology to gain therapeutic benefit through reprograming of cellular responses. In immuno‐oncology (IO) the aim is to direct the patient’s own immune system to fight cancer. After remarkable successes of antibodies targeting PD1/PD‐L1 and CTLA4 receptors in targeted patient populations, the focus of further development has shifted toward combination therapies. However, the current drug‐development approach of exploiting a vast number of possible combination targets and dosing regimens has proven to be challenging and is arguably inefficient. In particular, the unprecedented number of clinical trials testing different combinations may no longer be sustainable by the population of available patients. Further development in IO requires a step change in selection and validation of candidate therapies to decrease development attrition rate and limit the number of clinical trials. Quantitative systems pharmacology (QSP) proposes to tackle this challenge through mechanistic modeling and simulation. Compounds’ pharmacokinetics, target binding, and mechanisms of action as well as existing knowledge on the underlying tumor and immune system biology are described by quantitative, dynamic models aiming to predict clinical results for novel combinations. Here, we review the current QSP approaches, the legacy of mathematical models available to quantitative clinical pharmacologists describing interaction between tumor and immune system, and the recent development of IO QSP platform models. We argue that QSP and virtual patients can be integrated as a new tool in existing IO drug development approaches to increase the efficiency and effectiveness of the search for novel combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe4c7719ed81361b93340d169c24788387a66ae" target='_blank'>Quantitative Systems Pharmacology Approaches for Immuno‐Oncology: Adding Virtual Patients to the Development Paradigm</a></td>
          <td>
            Vijayalakshmi Chelliah, Georgia Lazarou, Sumit Bhatnagar, J. Gibbs, M. Nijsen, Avijit Ray, B. Stoll, R. Thompson, A. Gulati, S. Soukharev, A. Yamada, Jared Weddell, Hiroyuki Sayama, Masayo Oishi, S. Wittemer-Rump, C. Patel, Christoph Niederalt, R. Burghaus, Christian Scheerans, J. Lippert, S. Kabilan, I. Kareva, N. Belousova, A. Rolfe, A. Zutshi, M. Chenel, Filippo Venezia, Sylvain Fouliard, H. Oberwittler, A. Scholer-Dahirel, H. Lelièvre, D. Bottino, Sabrina C. Collins, Hoa Q. Nguyen, Haiqing Wang, Tomoki Yoneyama, Andy Z. X. Zhu, Piet H. Graaf, A. Kierzek
          </td>
          <td>2020-07-19</td>
          <td>Clinical Pharmacology and Therapeutics</td>
          <td>51</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b65ba1404d8d661ad5e10d9ca98efe2dd8af6c0" target='_blank'>Towards integrative systems pharmacology models in oncology drug development.</a></td>
          <td>
            J. G. V. van Hasselt, Piet H. van der Graaf
          </td>
          <td>2015-08-01</td>
          <td>Drug discovery today. Technologies</td>
          <td>36</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Understanding the intricate interactions of cancer cells with the tumor microenvironment (TME) is a pre-requisite for the optimization of immunotherapy. Mechanistic models such as quantitative systems pharmacology (QSP) provide insights into the TME dynamics and predict the efficacy of immunotherapy in virtual patient populations/digital twins but require vast amounts of multimodal data for parameterization. Large-scale datasets characterizing the TME are available due to recent advances in bioinformatics for multi-omics data. Here, we discuss the perspectives of leveraging omics-derived bioinformatics estimates to inform QSP models and circumvent the challenges of model calibration and validation in immuno-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771e6ee84353828e640c230c07c64013c8803ab5" target='_blank'>Leveraging multi-omics data to empower quantitative systems pharmacology in immuno-oncology</a></td>
          <td>
            Theinmozhi Arulraj, Hanwen Wang, Alberto Ippolito, Shuming Zhang, E. Fertig, Aleksander S. Popel
          </td>
          <td>2024-03-27</td>
          <td>Briefings in Bioinformatics</td>
          <td>3</td>
          <td>45</td>
        </tr>

        <tr id="Cancer immunotherapy focuses on stimulating and promoting the immune system to recognize and eliminate cancer cells, with several FDA approvals in recent years. However, identifying patients best suited for specific immune therapies, and determining optimal treatment regimens continue to be a clinical challenge. Using a molecular-detailed computational systems pharmacology model to identify cellular biomarkers and optimize regimens, we may be able to predict the efficacy of regimens in specific patient populations, and expedite drug development for cancer treatment. We developed a cell/receptor-based multi-compartment systems pharmacology model focusing on the immune response against a growing tumor, with the intent to test the effects of immune checkpoint inhibitors against PD-1, PD-L1 and CTLA-4 administered as mono- and combination therapies. Additionally, the model also allows for testing of other immuno-therapies, such as adoptive cell therapies, which can be combined with the checkpoint inhibitors. The model was designed and developed using the SimBiology plug-in in MATLAB. Simulations were performed with parameters that define the immune response at particular tumor stages of melanoma and NSCLC. All results were qualitatively and quantitatively compared to experimental pre-clinical and clinical data for model validation, or used for the generation of predictions suitable for further experimental testing. In silico, we have identified that administrations of the prescribed doses of 1-10 mg/kg of anti-CLTA-4 (based on binding kinetics) effectively saturates the receptors on the T cells, and promotes both an extended life span of the antigen presenting cells (APCs), and the maximum attainable activation levels of the effector T cells. The model further predicts that the effectiveness of anti-CTLA-4 therapy is limited by the immunogenicity of the system (i.e., the antigen intensity level and number of APCs presenting the antigens) in a monotonic fashion. Furthermore, injecting activated APCs without therapy would show a temporary tumor response and a subsequent recovery by the tumor to its original growth trajectory, while raising the antigen intensity had a sustained effect on tumor response. Other simulations indicate that, despite the lack of apparent tumor response, a sustained immune attack may be ongoing in the body; however, the immune activity is proportionally limited by the tumor and regulatory cells. Lastly, several dose-responses and clinical trials were simulated for both combination and monotherapies, and correlated with published clinical trial data. Future work will focus on uncovering the cellular biomarkers responsible for such results, experimentally validating them, as well as simulating optimal combination treatment regimens for future evaluation. Citation Format: Oleg Milberg, Chang Gong, Bing Wang, Paolo Vicini, Rajesh Narwal, Lorin Roskos, Aleksander Popel. Systems pharmacology to predict cellular biomarkers and optimize mono- and combination-therapy regimens: Focusing on immune checkpoint targets PD-1, PD-L1 and CTLA-4 [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2017; 2017 Apr 1-5; Washington, DC. Philadelphia (PA): AACR; Cancer Res 2017;77(13 Suppl):Abstract nr 4531. doi:10.1158/1538-7445.AM2017-4531">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb75efa5a87e05d34e374d801f59dcff29bafd7" target='_blank'>Abstract 4531: Systems pharmacology to predict cellular biomarkers and optimize mono- and combination-therapy regimens: Focusing on immune checkpoint targets PD-1, PD-L1 and CTLA-4</a></td>
          <td>
            Oleg Milberg, Chang Gong, B. Wang, P. Vicini, R. Narwal, L. Roskos, A. Popel
          </td>
          <td>2017-07-01</td>
          <td>Cancer Research</td>
          <td>3</td>
          <td>69</td>
        </tr>

        <tr id="Although therapeutic vaccination often induces markers of tumor-specific immunity, therapeutic responses remain rare. An improved understanding of patient-specific dynamic interactions of immunity and tumor progression, combined with personalized application of immune therapeutics would increase the efficacy of immunotherapy. Here, we developed a method to predict and enhance the individual response to immunotherapy by using personalized mathematical models, constructed in the early phase of treatment. Our approach includes an iterative real-time in-treatment evaluation of patient-specific parameters from the accruing clinical data, construction of personalized models and their validation, model-based simulation of subsequent response to ongoing therapy, and suggestion of potentially more effective patient-specific modified treatment. Using a mathematical model of prostate cancer immunotherapy, we applied our model to data obtained in a clinical investigation of an allogeneic whole-cell therapeutic prostate cancer vaccine. Personalized models for the patients who responded to treatment were derived and validated by data collected before treatment and during its early phase. Simulations, based on personalized models, suggested that an increase in vaccine dose and administration frequency would stabilize the disease in most patients. Together, our findings suggest that application of our method could facilitate development of a new paradigm for studies of in-treatment personalization of the immune agent administration regimens (P-trials), with treatment modifications restricted to an approved range, resulting in more efficacious immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd27e074fa5c1279e9443c302fd07550052e083" target='_blank'>Reconsidering the paradigm of cancer immunotherapy by computationally aided real-time personalization.</a></td>
          <td>
            Y. Kogan, Karin Halevi-Tobias, M. Elishmereni, Stanimir Vuk-Pavlović, Z. Agur
          </td>
          <td>2012-03-15</td>
          <td>Cancer research</td>
          <td>52</td>
          <td>35</td>
        </tr>

        <tr id="The goal of cancer immunotherapy is to boost a patient’s immune response to a tumor. Yet, the design of an effective immunotherapy is complicated by various factors, including a potentially immunosuppressive tumor microenvironment, immune-modulating effects of conventional treatments, and therapy-related toxicities. These complexities can be incorporated into mathematical and computational models of cancer immunotherapy that can then be used to aid in rational therapy design. In this review, we survey modeling approaches under the umbrella of the major challenges facing immunotherapy development, which encompass tumor classification, optimal treatment scheduling, and combination therapy design. Although overlapping, each challenge has presented unique opportunities for modelers to make contributions using analytical and numerical analysis of model outcomes, as well as optimization algorithms. We discuss several examples of models that have grown in complexity as more biological information has become available, showcasing how model development is a dynamic process interlinked with the rapid advances in tumor-immune biology. We conclude the review with recommendations for modelers both with respect to methodology and biological direction that might help keep modelers at the forefront of cancer immunotherapy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/161a29b3c32e37eaded637a7b1bb3c118aa6e089" target='_blank'>Addressing current challenges in cancer immunotherapy with mathematical and computational modeling</a></td>
          <td>
            A. Konstorum, A. Vella, A. Adler, R. Laubenbacher
          </td>
          <td>2017-06-01</td>
          <td>bioRxiv</td>
          <td>74</td>
          <td>48</td>
        </tr>

        <tr id="Cancer immunotherapy is a rapidly developing field, with numerous drugs and therapy combinations waiting to be tested in pre-clinical and clinical settings. However, the costly and time-consuming trial-and-error approach to development of new treatment paradigms creates a research bottleneck, motivating the development of complementary approaches. Computational modelling is a compelling candidate for this task, however, difficulties associated with the validation of such models limit their use in pre-clinical and clinical settings. Here we propose a bottom-up deterministic computational model to simulate tumour response to treatment with anti-programmed-death-1 antibodies (anti-PD-1). The model was built with validation in mind, and so contains minimum number of parameters, and only four free parameters. Moreover, all model parameters can be measured experimentally. Free parameters were tuned by fitting the model to experimental data from the literature, using B16-F10 murine melanoma implanted into wild type (C57BL/6), as well as into immunodeficient (NSG) mice strains, and treated with anti-PD-1 antibodies. The model’s predictive ability was verified on two independent datasets from literature with different but well-known inputs. To identify possible biomarkers of response to anti-PD-1 immunotherapy, sensitivity study of key model parameters was performed. Good agreement between the simulated tumour growth curves and the experimental data was achieved, with mean relative deviations in the range of 13%–20%. Our sensitivity study demonstrated that major histocompatibility complex (MHC) class I expression was the only parameter able to clearly discriminate responders from non-responders to anti-PD-1 therapy. Additionally, the results of sensitivity studies suggest that MHC class I expression might affect the predictive ability of other biomarkers that are currently used in the clinics, such as PD-1 ligand (PD-L1) expression. Interestingly, our model predicts the best response to anti-PD-1 therapy for subjects with moderate PD-L1 values. Such computational models show promise to support, guide and accelerate future immunotherapy research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88a12b75e53dfbf146a44de21daf57a611378db4" target='_blank'>Predicting tumour response to anti-PD-1 immunotherapy with computational modelling</a></td>
          <td>
            D. Valentinuzzi, U. Simončič, Katja Uršič, M. Vrankar, Maruša Turk, R. Jeraj
          </td>
          <td>2019-01-16</td>
          <td>Physics in Medicine & Biology</td>
          <td>23</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27968845671968d0ba91c79cba3242f30016982d" target='_blank'>Exploring potential therapeutic combinations for castration-sensitive prostate cancer using supercomputers: a proof of concept study</a></td>
          <td>
            Draško Tomić, J. Murgić, A. Fröbe, K. Skala, Antonela Vrljičak, Branka Medved Rogina, Branimir Kolarek, Viktor Bojović
          </td>
          <td>2024-08-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="PHARMACOKINETICS (PK) AND PROTEIN BINDING (PB) OF EPRISTERIDE (E) IN LIVER DISEASE PATIENTS (LDP) AND HEALTHY MALE SUBJECTS (HS). LJ Benincosa*, KA Thompson*, DE Everitt*, H Oldham*, S Patterson*, D Jorkasky, MK Bay* and S Schenker*. SmithKline Beecham Pharmaceuticals, King of Prussia, PA, Univ of Texas Health Science Center, San Antonio, TX E, a potent, selective, uncompetitive inhibitor of steroid 5(xreductase (isoform 2) is being developed for the treatment of benign prostatic hyperplasia. This study compared the safety, PK and PB of E following a single 80 mg oral dose in 8 LDP and 8 age-matched HS. E plasma conc were determined by HPLC and PB was determined using ultrafiltration. Mean (SD) PK parameters are given below (*p<0.05). Parameter (units) LDP HS AUC(0-inf) (ug.h/mL) 21.1 (5.0) 34.0 (13.3) * Cmax (ug/mL) 0.79 (0.20) 1.17 (0.38) * T1/2 (h) 20.3 (4.4) 26.3 (8.5) fraction unbound (%) 2.78 (0.38) 2.12 (0.27) * Based on free E conc, the ratio (LDP:HS) for AUC and Cmax were 0.82 (95% CI: 0.56, 1.19) and 0.89 (95% CI: 0.56, 1.43) respectively. It is likely that decreased serum albumin conc in LDP has contributed to the decreased PB of E as evidenced by a significant negative correlation between serum albumin and fraction unbound (r=-0.912, p<.0001). E was safe and welltolerated in LDP of moderate degree and HS with no marked differences between groups in the PK based on free E conc. PIII-54">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79c6174b22060282af0f785e26731851a3a04d6" target='_blank'>American Society for Clinical Pharmacology and Therapeutics</a></td>
          <td>
            Tian Yu, J. Rower, Xiaoxi Liu, A. Balch, Zachery Halford, E. Korgenski, Catherine M Sherwin
          </td>
          <td>1975-06-01</td>
          <td>Clinical Pharmacology & Therapeutics</td>
          <td>15</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a00267e88f4f567b9de0a596e574080d8c26ce3" target='_blank'>Pharmacokinetics (PK) and Protein Binding (PB) of Epristeride (E) in Liver Disease Patients (LDP) and Healthy Male Subjects (HS)</a></td>
          <td>
            L. Benincosa, K. A. Thompson, D. Everitt, H. Oldham, S. Patterson, D. Jorkasky, M. Bay, Steven Schenker
          </td>
          <td>1996-02-01</td>
          <td>Clinical Pharmacology & Therapeutics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32bc52fdec6be7403d14abf3c4b6d4ea53585fc2" target='_blank'>Abstracts of papers, Quantitative Systems Pharmacology: Multiscale Model‐Based Drug Development Through Integrating Systems Biology and Pharmacometrics Pre‐conference Hyatt Regency, New Orleans, Louisiana, March 3, 2015Abstracts of papers, 2015 Annual Meeting Hyatt Regency, New Orleans, Louisiana, M</a></td>
          <td>

          </td>
          <td>2015-01-22</td>
          <td>Clinical Pharmacology & Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d49da3006712193dab1a2ac75e81e68d16d6d01" target='_blank'>Pharmacodynamic Effects of Interferon ALFA‐2b in Healthy Volunteers</a></td>
          <td>
            Paul Glue, David L. Cutler, Ron Sabo, R. Huhn, P. Witt, Sheila Jacobs, Vijay Batra, M. Affrime
          </td>
          <td>1996-02-01</td>
          <td>Clinical Pharmacology & Therapeutics</td>
          <td>10</td>
          <td>37</td>
        </tr>

        <tr id="Prostate cancer (PRAD) is a major cause of cancer-related deaths. Current monotherapies show limited efficacy due to often rapidly emerging resistance. Combination therapies could provide an alternative solution to address this problem with enhanced therapeutic effect, reduced cytotoxicity, and delayed the appearance of drug resistance. However, it is prohibitively cost and labor-intensive for the experimental approaches to pick out synergistic combinations from the millions of possibilities. Thus, it is highly desired to explore other efficient strategies to assist experimental researches. Inspired by the challenge, we construct the transcriptomics-based and network-based prediction models to quickly screen the potential drug combination for Prostate cancer, and further assess their performance by in vitro assays. The transcriptomics-based method screens nine possible combinations. However, the network-based method gives discrepancies for at least three drug pairs. Further experimental results indicate the dose-dependent effects of the three docetaxel-containing combinations, and confirm the synergistic effects of the other six combinations predicted by the transcriptomics-based model. For the network-based predictions, in vitro tests give opposite results to the two combinations (i.e. mitoxantrone-cyproheptadine and cabazitaxel-cyproheptadine). Namely, the transcriptomics-based method outperforms the network-based one for the specific disease like Prostate cancer, which provide guideline for selection of the computational methods in the drug combination screening. More importantly, six combinations (the three mitoxantrone-containing and the three cabazitaxel-containing combinations) are found to be promising candidates to synergistically conquer Prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce541a238c278e22bbbd7f7b54f5ecb8bfa12efc" target='_blank'>Prediction of Synergistic Drug Combinations for Prostate Cancer by Transcriptomic and Network Characteristics</a></td>
          <td>
            Shiqi Li, Fuhui Zhang, Xiuchan Xiao, Yanzhi Guo, Z. Wen, Meng-long Li, X. Pu
          </td>
          <td>2021-04-12</td>
          <td>Frontiers in Pharmacology</td>
          <td>12</td>
          <td>27</td>
        </tr>

        <tr id="Quantitative systems pharmacology (QSP) models and spatial agent-based models (ABM) are powerful and efficient approaches for the analysis of biological systems and for clinical applications. Although QSP models are becoming essential in discovering predictive biomarkers and developing combination therapies through in silico virtual trials, they are inadequate to capture the spatial heterogeneity and randomness that characterize complex biological systems, and specifically the tumor microenvironment. Here, we extend our recently developed spatial QSP (spQSP) model to analyze tumor growth dynamics and its response to immunotherapy at different spatio-temporal scales. In the model, the tumor spatial dynamics is governed by the ABM, coupled to the QSP model, which includes the following compartments: central (blood system), tumor, tumor-draining lymph node, and peripheral (the rest of the organs and tissues). A dynamic recruitment of T cells and myeloid-derived suppressor cells (MDSC) from the QSP central compartment has been implemented as a function of the spatial distribution of cancer cells. The proposed QSP-ABM coupling methodology enables the spQSP model to perform as a coarse-grained model at the whole-tumor scale and as an agent-based model at the regions of interest (ROIs) scale. Thus, we exploit the spQSP model potential to characterize tumor growth, identify T cell hotspots, and perform qualitative and quantitative descriptions of cell density profiles at the invasive front of the tumor. Additionally, we analyze the effects of immunotherapy at both whole-tumor and ROI scales under different tumor growth and immune response conditions. A digital pathology computational analysis of triple-negative breast cancer specimens is used as a guide for modeling the immuno-architecture of the invasive front.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c62f2ed04de22b4bdf2a40d999eaeef064e9490b" target='_blank'>Simulations of tumor growth and response to immunotherapy by coupling a spatial agent-based model with a whole-patient quantitative systems pharmacology model</a></td>
          <td>
            Alvaro Ruiz-Martinez, Chang Gong, Hanwen Wang, R. Sové, Haoyang Mi, Holly Kimko, A. Popel
          </td>
          <td>2022-07-01</td>
          <td>PLoS Computational Biology</td>
          <td>34</td>
          <td>69</td>
        </tr>

        <tr id="Quantitative systems pharmacology (QSP) modeling has become increasingly important in pharmaceutical research and development, and is a powerful tool to gain mechanistic insights into the complex dynamics of biological systems in response to drug treatment. However, even once a suitable mathematical framework to describe the pathophysiology and mechanisms of interest is established, final model calibration and the exploration of variability can be challenging and time consuming. QSP models are often formulated as multi-scale, multi-compartment nonlinear systems of ordinary differential equations. Commonly accepted modeling strategies, workflows, and tools have promise to greatly improve the efficiency of QSP methods and improve productivity. In this paper, we present the QSP Toolbox, a set of functions, structure array conventions, and class definitions that computationally implement critical elements of QSP workflows including data integration, model calibration, and variability exploration. We present the application of the toolbox to an ordinary differential equations-based model for antibody drug conjugates. As opposed to a single stepwise reference model calibration, the toolbox also facilitates simultaneous parameter optimization and variation across multiple in vitro, in vivo, and clinical assays to more comprehensively generate alternate mechanistic hypotheses that are in quantitative agreement with available data. The toolbox also includes scripts for developing and applying virtual populations to mechanistic exploration of biomarkers and efficacy. We anticipate that the QSP Toolbox will be a useful resource that will facilitate implementation, evaluation, and sharing of new methodologies in a common framework that will greatly benefit the community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a62799c63dcab1a1119a051f38f7f2d59b0ae0d" target='_blank'>QSP Toolbox: Computational Implementation of Integrated Workflow Components for Deploying Multi-Scale Mechanistic Models</a></td>
          <td>
            Yougan Cheng, Craig J. Thalhauser, S. Smithline, Jyotsna Pagidala, M. Miladinov, H. Vezina, Manish Gupta, T. Leil, B. Schmidt
          </td>
          <td>2017-05-24</td>
          <td>The AAPS Journal</td>
          <td>46</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9fcbba9e52ada0386186556bd60b266a2255a9" target='_blank'>Recent applications of quantitative systems pharmacology and machine learning models across diseases</a></td>
          <td>
            S. Aghamiri, Rada Amin, T. Helikar
          </td>
          <td>2021-10-20</td>
          <td>Journal of Pharmacokinetics and Pharmacodynamics</td>
          <td>34</td>
          <td>24</td>
        </tr>

        <tr id="Significance This research develops a computational framework to quickly propose drugs with potential clinical impact for CRPC patients that are resistant to hormonal therapies. Furthermore, we identified a drug—COL-3—which showed high efficacy in clinical CRPC tumors displaying resistance to enzalutamide or abiraterone. We validated COL-3’s preferential efficacy through a pair of isogenic CRPC cell line models and further demonstrated the utility of our pipeline for efficient discovery of drugs with clinical impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b972279c4953defe87df057da9e617d95ed688e" target='_blank'>Computational drug discovery for castration-resistant prostate cancers through in vitro drug response modeling</a></td>
          <td>
            Weijie Zhang, Adam M. Lee, S. Jena, Yingbo Huang, Y. Ho, Kiel T Tietz, C. R. Miller, Mei-Chi Su, Joshua Mentzer, Alexander L. Ling, Yingming Li, S. Dehm, R. Huang
          </td>
          <td>2023-04-17</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>9</td>
          <td>44</td>
        </tr>

        <tr id="Quantitative systems pharmacology (QSP) has emerged as an innovative approach in model‐informed drug discovery and development, supporting program decisions from exploratory research through late‐stage clinical trials. In this commentary, we discuss the unique value of disease‐scale “platform” QSP models that are amenable to reuse and repurposing to support diverse clinical decisions in ways distinct from other pharmacometrics strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75fd303e477a32d2844dd2c8a67fced699191fee" target='_blank'>Quantitative Systems Pharmacology: A Case for Disease Models</a></td>
          <td>
            C. Musante, S. Ramanujan, B. J. Schmidt, OG Ghobrial, James Lu, A. Heatherington
          </td>
          <td>2016-11-18</td>
          <td>Clinical Pharmacology and Therapeutics</td>
          <td>50</td>
          <td>23</td>
        </tr>

        <tr id="The low response rate of immune checkpoint blockade in breast cancer has highlighted the need for predictive biomarkers to identify responders. While a number of clinical trials are ongoing, testing all possible combinations is not feasible. In this study, a quantitative systems pharmacology model is built to integrate immune–cancer cell interactions in patients with breast cancer, including central, peripheral, tumour-draining lymph node (TDLN) and tumour compartments. The model can describe the immune suppression and evasion in both TDLN and the tumour microenvironment due to checkpoint expression, and mimic the tumour response to checkpoint blockade therapy. We investigate the relationship between the tumour response to checkpoint blockade therapy and composite tumour burden, PD-L1 expression and antigen intensity, including their individual and combined effects on the immune system, using model-based simulations. The proposed model demonstrates the potential to make predictions of tumour response of individual patients given sufficient clinical measurements, and provides a platform that can be further adapted to other types of immunotherapy and their combination with molecular-targeted therapies. The patient predictions demonstrate how this systems pharmacology model can be used to individualize immunotherapy treatments. When appropriately validated, these approaches may contribute to optimization of breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/833197fc82a2fcb81242e160b09682a167c4c6f5" target='_blank'>In silico simulation of a clinical trial with anti-CTLA-4 and anti-PD-L1 immunotherapies in metastatic breast cancer using a systems pharmacology model</a></td>
          <td>
            Hanwen Wang, Oleg Milberg, I. Bartelink, P. Vicini, B. Wang, R. Narwal, L. Roskos, Cesar A. Santa-Maria, A. Popel
          </td>
          <td>2019-05-01</td>
          <td>Royal Society Open Science</td>
          <td>56</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d79bbb8f35d419b57478958d0df85426b8647ff7" target='_blank'>Cancer modeling: From mechanistic to data-driven approaches, and from fundamental insights to clinical applications</a></td>
          <td>
            Sophie Bekisz, L. Geris
          </td>
          <td>2020-08-12</td>
          <td>J. Comput. Sci.</td>
          <td>46</td>
          <td>43</td>
        </tr>

        <tr id="e16014 Background: CRPC is a heterogeneous disease, and it is challenging to determine optimal treatment sequences. Mathematical modeling can integrate diverse clinical and biological data to offer...">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bdae280459cf72e844679440757dac7465dce4b" target='_blank'>Optimizing personalized treatment sequences in metastatic castration resistant prostate cancer using tumor-biomarker based computational modeling.</a></td>
          <td>
            Shilpa Gupta, J. Gallaher, C. Lynch, Leah M. Cook, Arturo Araujo, Jacob G. Scott, J. Dhillon, J. Pow-Sang, D. Basanta
          </td>
          <td>2015-05-20</td>
          <td>Journal of Clinical Oncology</td>
          <td>1</td>
          <td>45</td>
        </tr>

        <tr id="Background Late-stage cancer immunotherapy trials strive to demonstrate the clinical efficacy of novel immunotherapies, which is leading to exceptional responses and long-term survival in subsets of patients. To establish the clinical efficacy of an immunotherapy, it is critical to adjust the trial's design to the expected immunotherapy-specific response patterns. Methods In silico cancer immunotherapy trials are virtual clinical trials that simulate the kinetics and outcome of immunotherapy depending on the type and treatment schedule. We used an ordinary differential equation model to simulate (1) cellular interactions within the tumor microenvironment, (2) translates these into disease courses in patients, and (3) assemble populations of virtual patients to simulate in silico late-stage immunotherapy, chemotherapy, or combination trials. We predict trial outcomes and investigate how therapy-specific response patterns affect the probability of their success. Results In silico cancer immunotherapy trials reveal that immunotherapy-derived survival kinetics -- such as delayed curve separation and plateauing curve of the treatment arm -- arise naturally due to biological interactions in the tumor microenvironment. In silico clinical trials are capable of translating these biological interactions into survival kinetics. Considering four aspects of clinical trial design -- sample size calculations, endpoint and randomization rate selection, and interim analysis planning -- we illustrate that failing to consider such distinctive response patterns can significantly reduce the power of novel immunotherapy trials. Conclusion In silico trials have three significant implications for immuno-oncology. First, they provide an economical approach to verify the robustness of biological assumptions underlying an immunotherapy trial and help to scrutinize its design. Second, the biological basis of these trials facilitates and encourages communication between biomedical researchers, doctors, and trialists. Third, its application as an educational tool can illustrate design principles to scientists in training, contributing to improved designs and higher success rates of future immunotherapy trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99a5d7d0179c38221683ce78bf4eccfb96e167d8" target='_blank'>In silico cancer immunotherapy trials uncover the consequences of therapy-specific response patterns for clinical trial design and outcome</a></td>
          <td>
            J. H. Creemers, Kit Roes, Niven Mehra, Carl G. Figdor, I. J. M. de Vries, J. Textor
          </td>
          <td>2021-09-13</td>
          <td>None</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3498e511e9eb5c885a5bd150e27ed48f227d9e8a" target='_blank'>Future Directions for Quantitative Systems Pharmacology.</a></td>
          <td>
            Birgit Schoeberl, C. Musante, S. Ramanujan
          </td>
          <td>2025-01-16</td>
          <td>Handbook of experimental pharmacology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Prostate cancer is the second most occurring cancer in men worldwide. To better understand the mechanisms of tumorigenesis and possible treatment responses, we developed a mathematical model of prostate cancer which considers the major signalling pathways known to be deregulated. We personalised this Boolean model to molecular data to reflect the heterogeneity and specific response to perturbations of cancer patients. A total of 488 prostate samples were used to build patient-specific models and compared to available clinical data. Additionally, eight prostate cell line-specific models were built to validate our approach with dose-response data of several drugs. The effects of single and combined drugs were tested in these models under different growth conditions. We identified 15 actionable points of interventions in one cell line-specific model whose inactivation hinders tumorigenesis. To validate these results, we tested nine small molecule inhibitors of five of those putative targets and found a dose-dependent effect on four of them, notably those targeting HSP90 and PI3K. These results highlight the predictive power of our personalised Boolean models and illustrate how they can be used for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18d52789867538ad9583601890d8157a8f696dd8" target='_blank'>Patient-specific Boolean models of signalling networks guide personalised treatments</a></td>
          <td>
            Arnau Montagud, Jonas Béal, L. Tobalina, P. Traynard, Vigneshwar Subramanian, B. Szalai, R. Alföldi, L. Puskás, A. Valencia, E. Barillot, J. Saez-Rodriguez, Laurence Calzone
          </td>
          <td>2022-02-15</td>
          <td>eLife</td>
          <td>43</td>
          <td>64</td>
        </tr>

        <tr id="Although therapeutic vaccination often induces markers of tumor-specific immunity, therapeutic responses remain rare. An improved understanding of patient-specific dynamic interactions of immunity and tumor progression, combined with personalized application of immune therapeutics would increase the efficacy of immunotherapy. Here, we developed a method to predict and enhance the individual response to immunotherapy by using personalized mathematical models, constructed in the early phase of treatment. Our approach includes an iterative real-time in-treatment evaluation of patient-specific parameters from the accruing clinical data, construction of personalized models and their validation, model-based simulation of subsequent response to ongoing therapy, and suggestion of potentially more effective patient-specific modified treatment. Using a mathematical model of prostate cancer immunotherapy, we applied our model to data obtained in a clinical investigation of an allogeneic whole-cell therapeutic prostate cancer vaccine. Personalized models for the patients who responded to treatment were derived and validated by data collected before treatment and during its early phase. Simulations, based on personalized models, suggested that an increase in vaccine dose and administration frequency would stabilize the disease in most patients. Together, our findings suggest that application of our method could facilitate development of a new paradigm for studies of in-treatment personalization of the immune agent administration regimens (P-trials), with treatment modifications restricted to an approved range, resulting in more efficacious immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd27e074fa5c1279e9443c302fd07550052e083" target='_blank'>Reconsidering the paradigm of cancer immunotherapy by computationally aided real-time personalization.</a></td>
          <td>
            Y. Kogan, Karin Halevi-Tobias, M. Elishmereni, Stanimir Vuk-Pavlović, Z. Agur
          </td>
          <td>2012-03-15</td>
          <td>Cancer research</td>
          <td>52</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6390a7198e07ebd6231e4059b4f39210c8417535" target='_blank'>Quantitative Systems Pharmacology models as a key to translational medicine</a></td>
          <td>
            B. Schoeberl
          </td>
          <td>2019-08-01</td>
          <td>Current Opinion in Systems Biology</td>
          <td>4</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4624f66c438a80c62525b6af71f20d1edb159bd" target='_blank'>Integrating Systems Biology and an Ex Vivo Human Tumor Model Elucidates PD-1 Blockade Response Dynamics</a></td>
          <td>
            Munisha Smalley, Munisha Smalley, M. Przedborski, S. Thiyagarajan, Moriah Pellowe, A. Verma, N. Brijwani, Debika Datta, Misti Jain, Basavaraja U. Shanthappa, Vidushi Kapoor, K. Gopinath, D. C. Doval, K. Sabitha, G. Taroncher-Oldenburg, B. Majumder, P. Majumder, M. Kohandel, Aaron Goldman, Aaron Goldman
          </td>
          <td>2020-06-01</td>
          <td>iScience</td>
          <td>7</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68d54a4ef82873fd3a0e857ad2c136d65fa17db8" target='_blank'>Systems biology informed neural networks (SBINN) predict response and novel combinations for PD-1 checkpoint blockade</a></td>
          <td>
            M. Przedborski, Munisha Smalley, S. Thiyagarajan, A. Goldman, M. Kohandel
          </td>
          <td>2021-07-15</td>
          <td>Communications Biology</td>
          <td>12</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68d54a4ef82873fd3a0e857ad2c136d65fa17db8" target='_blank'>Systems biology informed neural networks (SBINN) predict response and novel combinations for PD-1 checkpoint blockade</a></td>
          <td>
            M. Przedborski, Munisha Smalley, S. Thiyagarajan, A. Goldman, M. Kohandel
          </td>
          <td>2021-07-15</td>
          <td>Communications Biology</td>
          <td>12</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c700941e299210cb7fb770abc9a413bb44672b6" target='_blank'>Improving treatment strategies for patients with metastatic castrate resistant prostate cancer through personalized computational modeling</a></td>
          <td>
            J. Gallaher, Leah M. Cook, Shilpa Gupta, Arturo Araujo, J. Dhillon, Jong Y. Park, Jacob G. Scott, J. Pow-Sang, D. Basanta, C. Lynch
          </td>
          <td>2014-08-31</td>
          <td>Clinical & Experimental Metastasis</td>
          <td>26</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1818d6d2f1497c68120dffa486144c93a75e7c6" target='_blank'>Improving treatment strategies for patients with metastatic castrate resistant prostate cancer through personalized computational modeling</a></td>
          <td>
            J. Gallaher, Leah M. Cook, Shilpa Gupta, Arturo Araujo, J. Dhillon, Jong Y. Park, Jacob G. Scott, J. Pow‐Sang, D. Basanta, C. Lynch
          </td>
          <td>2014-08-31</td>
          <td>Clinical & Experimental Metastasis</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Quantitative systems pharmacology (QSP) modeling has become an increasingly popular approach impacting our understanding of disease mechanisms and helping predict patients’ treatment responses to facilitate study design or development go/no-go decisions. In this paper, we highlight the notable contributions and opportunities that QSP approaches are to offer during the drug development process by sharing three examples that have facilitated internal decisions. The barriers to successful applications and the factors that facilitate the success of the modeling approach is discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15595407c283a7e6ca03ee082cb43d7d405e8328" target='_blank'>Quantitative systems modeling approaches towards model-informed drug development: Perspective through case studies</a></td>
          <td>
            Meghna Verma, Louis Gall, J. Biasetti, G. D. Di Veroli, C. Pichardo-Almarza, M. Gibbs, Holly Kimko
          </td>
          <td>2023-01-12</td>
          <td>None</td>
          <td>5</td>
          <td>21</td>
        </tr>

        <tr id="To allow innovative, progressive treatments to make the transition from concept to clinic, experimentally calibrated and clinically motivated mathematical models are essential. To aid the development of well-informed clinical trials, collaboration between mathematicians, computational modelers, biologists, and clinicians is required to allow qualitative hypotheses to take on a quantitative dimension. Such interdisciplinary science can allow both personalization and optimization of dose and scheduling of immunotherapeutic protocols, both as an independent therapy and in conjunction with other more traditional therapies, to streamline the transition from innovative concept to clinical practice and improve clinical outcomes for individual patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcc145eb2aa83b88341c321a673401f79702dde" target='_blank'>From concept to clinic: mathematically informed immunotherapy</a></td>
          <td>
            R. Walker, H. Enderling
          </td>
          <td>2015-10-01</td>
          <td>bioRxiv</td>
          <td>37</td>
          <td>37</td>
        </tr>

        <tr id="Immunotherapy and immune checkpoint blocking antibodies such as anti-PD-1 are approved and significantly improve the survival of advanced non-small cell lung cancer (NSCLC) patients, but there has been little success in identifying biomarkers capable of separating the responders from non-responders before the onset of the therapy. In this study, we developed a quantitative system pharmacology (QSP) model to represent the anti-tumor immune response in human NSCLC that integrated our knowledge of tumor growth, antigen processing and presentation, T cell activation and distribution, antibody pharmacokinetics, and immune checkpoint dynamics. The model was calibrated with the available data and was used to identify potential biomarkers as well as patient-specific response based on the patient parameters. The model predicted that in addition to tumor mutational burden (TMB), a known biomarker for anti-PD-1 therapy in NSCLC, the number of effector T cells and regulatory T cells in the tumor and blood is a predictor of the responders. Furthermore, the model simulated a set of 12 patients with known TMB and MHC/antigen-binding affinity from a recent clinical trial (ClinicalTrials.gov number, NCT02259621) on neoadjuvant nivolumab therapy in resectable lung cancer and predicted an augmented durable response in patients with adjuvant nivolumab treatment in addition to the clinical trial protocol of neoadjuvant nivolumab treatment followed by resection. Overall, the model provides a valuable framework to model tumor immunity and response to immune checkpoint blockers to enhance biomarker discovery and performing virtual clinical trials to aid in design and interpretation of the current trials with fewer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6107c61042002f08fd5b87757a1769200845b10" target='_blank'>A Computational Model of Neoadjuvant PD-1 Inhibition in Non-Small Cell Lung Cancer</a></td>
          <td>
            M. Jafarnejad, Chang Gong, E. Gabrielson, I. Bartelink, P. Vicini, B. Wang, R. Narwal, L. Roskos, A. Popel
          </td>
          <td>2019-06-24</td>
          <td>The AAPS Journal</td>
          <td>54</td>
          <td>71</td>
        </tr>

        <tr id="ABSTRACT Introduction Prostate cancer (PC) is the most common malignancy and accounts for a significant proportion of cancer deaths among men. Although initial therapy success can often be observed in patients diagnosed with localized PC, many patients eventually develop disease recurrence and metastasis. Without effective treatments, patients with aggressive PC display very poor survival. To curb the current high mortality rate, many investigations have been carried out to identify efficacious therapeutics. Compared to de novo drug designs, computational methods have been widely employed to offer actionable drug predictions in a fast and cost-efficient way. Particularly, powered by an increasing availability of next-generation sequencing molecular profiles from PC patients, computer-aided approaches can be tailored to screen for candidate drugs. Areas covered Herein, the authors review the recent advances in computational methods for drug discovery utilizing molecular profiles from PC patients. Given the uniqueness in PC therapeutic needs, they discuss in detail the drug discovery goals of these studies, highlighting their translational values for clinically impactful drug nomination. Expert opinion Evolving molecular profiling techniques may enable new perspectives for computer-aided approaches to offer drug candidates for different tumor microenvironments. With ongoing efforts to incorporate new compounds into large-scale high-throughput screens, the authors envision continued expansion of drug candidate pools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ab81ced264dcb3b837ac31f1cf1f804ed3d305d" target='_blank'>Computer-aided drug discovery strategies for novel therapeutics for prostate cancer leveraging next-generating sequencing data</a></td>
          <td>
            Weijie Zhang, R. S. Huang
          </td>
          <td>2024-06-11</td>
          <td>Expert Opinion on Drug Discovery</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdfb93d6a47c26dc7c4e0a77652e8f96f15aaed0" target='_blank'>Systems biology applied to vaccine and immunotherapy development</a></td>
          <td>
            L. Buonaguro, E. Wang, M. Tornesello, F. Buonaguro, F. Marincola
          </td>
          <td>2011-09-20</td>
          <td>BMC Systems Biology</td>
          <td>30</td>
          <td>106</td>
        </tr>

        <tr id="Immune checkpoint inhibitors have shown impressive benefits for patients with various types of cancer. However, patients who respond are still a minority. To improve the response rates, combinations of various immunotherapies, as well as combinations with other conventional therapies, have been extensively studied. Due to an unmanageably high number of all possible combinations and dosing regimens, alternatives to the costly and time-consuming trial-and-error approach are of utmost importance. Our main goal was to develop a verifiable computational model that would analyze the tumor response to anti-PD-1 antibodies and provide suggestions about the possible biomarkers of response to anti-PD-1 immunotherapy. Our model was built with validation in mind, and so contains minimum number of parameters. Moreover, all parameters can be measured experimentally. The model was tuned and validated in vivo using 3 murine tumor cell lines (B16-F10, CT26, 4T1) in 3 different settings: (1) growth of tumors in NUDE mice to assess intrinsic tumor growth in the absence of T-cells, (2) growth of tumors in wild-type (WT) mice to assess the effect of the immune system on untreated tumor growth, and (3) growth of tumors in WT mice receiving anti-PD-1 antibodies to assess the therapeutic effect. MHC Class I and PD-L1 expression on tumor cells was measured in vitro using flow cytometry to assess the parameters associated with immunogenicity of selected tumor cell lines. Single nucleotide variations (SNV) data, indicative of the mutational load, were taken from literature. Finally, we performed a sensitivity study of key model parameters to identify possible biomarkers of tumor response to anti-PD-1 therapy. In vitro results showed comparable PD-L1 expression in all 3 cell lines (11%-21%), while MHC class I expression varied significantly between B16-F10 (2.8%), 4T1 (99.5%), and CT26 (99.9%). Additionally, SNV data indicated an order of magnitude higher CT26 SNV (3023) compared to 4T1 (293), and more than 3 times higher compared to B16-F10 (908). Using the above-measured parameters our computational model was able to reproduce all in vivo experiments. The model suggests that the average occupancy of PD-1 receptors on tumor-infiltrating T cells by anti-PD-1 antibodies is much higher in CT26 (74%) compared to 4T1 (30%) or B16-F10 (8%). It indicates that the ability of antibodies to penetrate the tumor might vary depending on the tumor type. The results of the sensitivity study suggested that a combination of MHC class I, PD-L1 and SNV might be superior for predicting tumor response to anti-PD-1 compared to either of the biomarkers alone. Namely, CT26, the cell line with high MHC class I, high SNV and moderate PD-L1 expression, was the only cell line where complete responses to anti-PD-1 antibodies were observed experimentally. Despite simplified description of the reality, our model generates meaningful hypotheses to be tested in future (pre)clinical trials. Such models show promise to support, guide and accelerate immunotherapy research. Citation Format: Damijan Valentinuzzi, Katja Ursic, Urban Simoncic, Matea Maruna, Marusa Turk, Martina Vrankar, Maja Cemazar, Gregor Sersa, Robert Jeraj. Computational modeling analysis of the tumor response to anti-PD-1 immunotherapy [abstract]. In: Proceedings of the AACR Special Conference on Tumor Immunology and Immunotherapy; 2018 Nov 27-30; Miami Beach, FL. Philadelphia (PA): AACR; Cancer Immunol Res 2020;8(4 Suppl):Abstract nr A09.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b2e76a66a7488f297f0eb9c1fcf27ffddc5d2cf" target='_blank'>Abstract A09: Computational modeling analysis of the tumor response to anti-PD-1 immunotherapy</a></td>
          <td>
            D. Valentinuzzi, Katja Uršič, U. Simončič, Matea Maruna, Maruša Turk, M. Vrankar, M. Čemažar, G. Serša, R. Jeraj
          </td>
          <td>2020-04-01</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/283e6b80544ffe4eca14f62894e31da3e1404192" target='_blank'>Drug repositioning in castration-resistant prostate cancer using systems biology and computational drug design techniques.</a></td>
          <td>
            Javad Rafiee, K. Jamialahmadi, M. Bazyari, S. H. Aghaee-Bakhtiari
          </td>
          <td>2024-12-01</td>
          <td>Computational biology and chemistry</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer vaccines are a real application of the extensive knowledge of immunology to the field of oncology. Tumors are dynamic complex systems in which several entities, events, and conditions interact among them resulting in growth, invasion, and metastases. The immune system includes many cells and molecules that cooperatively act to protect the host organism from foreign agents. Interactions between the immune system and the tumor mass include a huge number of biological factors. Testing of some cancer vaccine features, such as the best conditions for vaccine administration or the identification of candidate antigenic stimuli, can be very difficult or even impossible only through experiments with biological models simply because a high number of variables need to be considered at the same time. This is where computational models, and, to this extent, immunoinformatics, can prove handy as they have shown to be able to reproduce enough biological complexity to be of use in suggesting new experiments. Indeed, computational models can be used in addition to biological models. We now experience that biologists and medical doctors are progressively convinced that modeling can be of great help in understanding experimental results and planning new experiments. This will boost this research in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69c47b06e91e2a05513e1c894c0a3634aec792db" target='_blank'>Cancer Vaccines: State of the Art of the Computational Modeling Approaches</a></td>
          <td>
            F. Pappalardo, F. Chiacchio, S. Motta
          </td>
          <td>2012-12-23</td>
          <td>BioMed Research International</td>
          <td>18</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee845585df6b55f57695a1164b68a906719e68a7" target='_blank'>Computational approaches to modelling and optimizing cancer treatment</a></td>
          <td>
            Thomas O. McDonald, Yu-Chen Cheng, Christopher J Graser, Phillip B. Nicol, D. Temko, F. Michor
          </td>
          <td>2023-07-19</td>
          <td>Nature Reviews Bioengineering</td>
          <td>13</td>
          <td>76</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>