<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../MAPK1/">
      
      
        <link rel="next" href="../Virtual%20cell/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              MAPK
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Virtual%20cell/" class="md-tabs__link">
        
  
    
  
  Virtual cell

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../RA/" class="md-tabs__link">
        
  
    
  
  RA

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Prostate%20cancer/" class="md-tabs__link">
        
  
    
  
  Prostate cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Virtual%20patients/" class="md-tabs__link">
        
  
    
  
  Virtual patients

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Virtual%20cell/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual cell
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../RA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RA
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Prostate%20cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Prostate cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Virtual%20patients/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual patients
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>MAPK</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-03 06:08:53 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on MAPK</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on MAPK</a><br>
      <a href="#recommended_articles">3. Recommended articles on MAPK</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on MAPK</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>MAPK</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6794df1533c28e74541f21d11b40c62f080d24e" target='_blank'>
                Computational model of EGFR and IGF1R pathways in lung cancer: a Systems Biology approach for Translational Oncology.
                </a>
              </td>
          <td>
            F. Bianconi, E. Baldelli, V. Ludovini, L. Crinò, A. Flacco, P. Valigi
          </td>
          <td>None</td>
          <td>Biotechnology advances</td>
          <td>47</td>
          <td>85</td>

            <td><a href='../recommendations/a6794df1533c28e74541f21d11b40c62f080d24e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0712e7933144fafdcb01011f5031d0190cc79967" target='_blank'>
                Oscillations in MAPK cascade triggered by two distinct designs of coupled positive and negative feedback loops
                </a>
              </td>
          <td>
            U. Sarma, I. Ghosh
          </td>
          <td>2012-06-13</td>
          <td>BMC Research Notes</td>
          <td>11</td>
          <td>16</td>

            <td><a href='../recommendations/0712e7933144fafdcb01011f5031d0190cc79967' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e509d9b8f4bae96ed26b214de1804c178e475585" target='_blank'>
                Different designs of kinase-phosphatase interactions and phosphatase sequestration shapes the robustness and signal flow in the MAPK cascade
                </a>
              </td>
          <td>
            U. Sarma, I. Ghosh
          </td>
          <td>2012-07-02</td>
          <td>BMC Systems Biology</td>
          <td>14</td>
          <td>16</td>

            <td><a href='../recommendations/e509d9b8f4bae96ed26b214de1804c178e475585' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="The mitogen-activated protein kinase (MAPK) cascade is a highly conserved series of three protein kinases implicated in diverse biological processes. Here we demonstrate that the cascade arrangement has unexpected consequences for the dynamics of MAPK signaling. We solved the rate equations for the cascade numerically and found that MAPK is predicted to behave like a highly cooperative enzyme, even though it was not assumed that any of the enzymes in the cascade were regulated cooperatively. Measurements of MAPK activation in Xenopus oocyte extracts confirmed this prediction. The stimulus/response curve of the MAPK was found to be as steep as that of a cooperative enzyme with a Hill coefficient of 4-5, well in excess of that of the classical allosteric protein hemoglobin. The shape of the MAPK stimulus/ response curve may make the cascade particularly appropriate for mediating processes like mitogenesis, cell fate induction, and oocyte maturation, where a cell switches from one discrete state to another.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfd48bb820b877eaf924310e6c030ff6fa7ac29d" target='_blank'>
                Ultrasensitivity in the mitogen-activated protein kinase cascade.
                </a>
              </td>
          <td>
            Chi-Ying F. Huang, J. Ferrell
          </td>
          <td>1996-09-17</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1231</td>
          <td>62</td>

            <td><a href='../recommendations/bfd48bb820b877eaf924310e6c030ff6fa7ac29d' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Functional organization of signal transduction into protein phosphorylation cascades, such as the mitogen-activated protein kinase (MAPK) cascades, greatly enhances the sensitivity of cellular targets to external stimuli. The sensitivity increases multiplicatively with the number of cascade levels, so that a tiny change in a stimulus results in a large change in the response, the phenomenon referred to as ultrasensitivity. In a variety of cell types, the MAPK cascades are imbedded in long feedback loops, positive or negative, depending on whether the terminal kinase stimulates or inhibits the activation of the initial level. Here we demonstrate that a negative feedback loop combined with intrinsic ultrasensitivity of the MAPK cascade can bring about sustained oscillations in MAPK phosphorylation. Based on recent kinetic data on the MAPK cascades, we predict that the period of oscillations can range from minutes to hours. The phosphorylation level can vary between the base level and almost 100% of the total protein. The oscillations of the phosphorylation cascades and slow protein diffusion in the cytoplasm can lead to intracellular waves of phospho-proteins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1b66e7327a2fb1436df251839828fe387fa75b" target='_blank'>
                Negative feedback and ultrasensitivity can bring about oscillations in the mitogen-activated protein kinase cascades.
                </a>
              </td>
          <td>
            B. Kholodenko
          </td>
          <td>2000-03-01</td>
          <td>European journal of biochemistry</td>
          <td>688</td>
          <td>57</td>

            <td><a href='../recommendations/bb1b66e7327a2fb1436df251839828fe387fa75b' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecad4837f4bc306f173456aafecd9d6b67b62744" target='_blank'>
                Regulation of ERK-MAPK signaling in human epidermis
                </a>
              </td>
          <td>
            Joseph Cursons, Jerry Gao, Daniel G. Hurley, C. Print, P. Rod Dunbar, Marc D. Jacobs, Edmund J Crampin
          </td>
          <td>2015-07-25</td>
          <td>BMC Systems Biology</td>
          <td>40</td>
          <td>20</td>

            <td><a href='../recommendations/ecad4837f4bc306f173456aafecd9d6b67b62744' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Mitogen-Activated Protein Kinases (MAPKs) cascade plays an important role in regulating plant growth and development, generating cellular responses to the extracellular stimuli. MAPKs cascade mainly consist of three sub-families i.e. mitogen-activated protein kinase kinase kinase (MAPKKK), mitogen-activated protein kinase kinase (MAPKK) and mitogen activated protein kinase (MAPK), several cascades of which are activated by various abiotic and biotic stresses. In this work we have modeled the holistic molecular mechanisms essential to MAPKs activation in response to several abiotic and biotic stresses through a system biology approach and performed its simulation studies. As extent of abiotic and biotic stresses goes on increasing, the process of cell division, cell growth and cell differentiation slow down in time dependent manner. The models developed depict the combinatorial and multicomponent signaling triggered in response to several abiotic and biotic factors. These models can be used to predict behavior of cells in event of various stresses depending on their time and exposure through activation of complex signaling cascades.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dfc3ad85074a37ea43ae7199e61a35587ec1d29" target='_blank'>
                Modeling of the MAPK machinery activation in response to various abiotic and biotic stresses in plants by a system biology approach
                </a>
              </td>
          <td>
            R. Pathak, G. Taj, D. Pandey, S. Arora, Anil Kumar
          </td>
          <td>2013-05-25</td>
          <td>Bioinformation</td>
          <td>37</td>
          <td>53</td>

            <td><a href='../recommendations/2dfc3ad85074a37ea43ae7199e61a35587ec1d29' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Mitogen-activated protein kinase (MAPK) cascades can operate as bistable switches residing in either of two different stable states. MAPK cascades are often embedded in positive feedback loops, which are considered to be a prerequisite for bistable behavior. Here we demonstrate that in the absence of any imposed feedback regulation, bistability and hysteresis can arise solely from a distributive kinetic mechanism of the two-site MAPK phosphorylation and dephosphorylation. Importantly, the reported kinetic properties of the kinase (MEK) and phosphatase (MKP3) of extracellular signal–regulated kinase (ERK) fulfill the essential requirements for generating a bistable switch at a single MAPK cascade level. Likewise, a cycle where multisite phosphorylations are performed by different kinases, but dephosphorylation reactions are catalyzed by the same phosphatase, can also exhibit bistability and hysteresis. Hence, bistability induced by multisite covalent modification may be a widespread mechanism of the control of protein activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d1e185348dca2c036fe189abba3ab6cd1fb5a7a" target='_blank'>
                Signaling switches and bistability arising from multisite phosphorylation in protein kinase cascades
                </a>
              </td>
          <td>
            N. Markevich, J. Hoek, B. Kholodenko
          </td>
          <td>2004-02-02</td>
          <td>The Journal of Cell Biology, Journal of Cell Biology</td>
          <td>713</td>
          <td>57</td>

            <td><a href='../recommendations/9d1e185348dca2c036fe189abba3ab6cd1fb5a7a' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Activation of the fibroblast growth factor (FGFR) and melanocyte stimulating hormone (MC1R) receptors stimulates B‐Raf and C‐Raf isoforms that regulate the dynamics of MAPK1,2 signaling. Network topology motifs in mammalian cells include feed‐for ward and feedback loops and bifans where signals from two upstream molecules integrate to modulate the activity of two downstream molecules. We computation ally modeled and experimentally tested signal processing in the FGFR/MC1R/B‐Raf/C‐Raf/MAPK1,2 net work in human melanoma cells; identifying 7 regulatory loops and a bifan motif. Signaling from FGFR leads to sustained activation of MAPK1,2, whereas signaling from MC1R results in transient activation of MAPK1,2. The dynamics of MAPK activation depends critically on the expression level and connectivity to C‐Raf, which is critical for a sustained MAPK1,2 response. A partially incoherent bifan motif with a feedback loop acts as a logic gate to integrate signals and regulate duration of activation of the MAPK signaling cascade. Further reducing a 106‐node ordinary differential equations network encompassing the complete network to a 6‐node network encompassing rate‐limiting processes sustains the feedback loops and the bifan, providing sufficient information to predict biological responses.—Muller, M., Obeyesekere, M., Mills, G. B., Ram, P. T. Network topology determines dynamics of the mammalian MAPK1,2 signaling network: bifan motif regulation of C‐Raf and B‐Raf isoforms by FGFR and MC1R. FASEB J. 22, 1393–1403 (2008)">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec6b95aa83664dddca439b321c3584c8e157db08" target='_blank'>
                Network topology determines dynamics of the mammalian MAPK1,2 signaling network: bifan motif regulation of C‐Raf and B‐Raf isoforms by FGFR and MC1R
                </a>
              </td>
          <td>
            M. Muller, M. Obeyesekere, G. Mills, Prahlad T. Ram
          </td>
          <td>2008-05-01</td>
          <td>The FASEB Journal</td>
          <td>25</td>
          <td>189</td>

            <td><a href='../recommendations/ec6b95aa83664dddca439b321c3584c8e157db08' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="In addition to preventing crosstalk among related signaling pathways, scaffold proteins might facilitate signal transduction by preforming multimolecular complexes that can be rapidly activated by incoming signal. In many cases, such as mitogen-activated protein kinase (MAPK) cascades, scaffold proteins are necessary for full activation of a signaling pathway. To date, however, no detailed biochemical model of scaffold action has been suggested. Here we describe a quantitative computer model of MAPK cascade with a generic scaffold protein. Analysis of this model reveals that formation of scaffold-kinase complexes can be used effectively to regulate the specificity, efficiency, and amplitude of signal propagation. In particular, for any generic scaffold there exists a concentration value optimal for signal amplitude. The location of the optimum is determined by the concentrations of the kinases rather than their binding constants and in this way is scaffold independent. This effect and the alteration of threshold properties of the signal propagation at high scaffold concentrations might alter local signaling properties at different subcellular compartments. Different scaffold levels and types might then confer specialized properties to tune evolutionarily conserved signaling modules to specific cellular contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20ed3469838f814cc78cd08cc427934fcc8b2f7" target='_blank'>
                Scaffold proteins may biphasically affect the levels of mitogen-activated protein kinase signaling and reduce its threshold properties.
                </a>
              </td>
          <td>
            A. Levchenko, Jehoshua Bruck, P. Sternberg
          </td>
          <td>2000-05-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>522</td>
          <td>106</td>

            <td><a href='../recommendations/b20ed3469838f814cc78cd08cc427934fcc8b2f7' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/509f1971d67425c00eb55f435cf67274acb0fe25" target='_blank'>
                Computational modeling of the dynamics of the MAP kinase cascade activated by surface and internalized EGF receptors
                </a>
              </td>
          <td>
            B. Schoeberl, Claudia Eichler-Jonsson, E. Gilles, G. Müller
          </td>
          <td>2002-04-01</td>
          <td>Nature Biotechnology</td>
          <td>1026</td>
          <td>48</td>

            <td><a href='../recommendations/509f1971d67425c00eb55f435cf67274acb0fe25' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="ErbB tyrosine kinase receptors mediate mitogenic signal cascade by binding a variety of ligands and recruiting the different cassettes of adaptor proteins. In the present study, we examined heregulin (HRG)-induced signal transduction of ErbB4 receptor and found that the phosphatidylinositol 3'-kinase (PI3K)-Akt pathway negatively regulated the extracellular signal-regulated kinase (ERK) cascade by phosphorylating Raf-1 on Ser(259). As the time-course kinetics of Akt and ERK activities seemed to be transient and complex, we constructed a mathematical simulation model for HRG-induced ErbB4 receptor signalling to explain the dynamics of the regulation mechanism in this signal transduction cascade. The model reflected well the experimental results observed in HRG-induced ErbB4 cells and in other modes of growth hormone-induced cell signalling that involve Raf-Akt cross-talk. The model suggested that HRG signalling is regulated by protein phosphatase 2A as well as Raf-Akt cross-talk, and protein phosphatase 2A modulates the kinase activity in both the PI3K-Akt and MAPK (mitogen-activated protein kinase) pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5c624f60a57498d015d6990eeb7e2ca9bf3aef4" target='_blank'>
                A computational model on the modulation of mitogen-activated protein kinase (MAPK) and Akt pathways in heregulin-induced ErbB signalling.
                </a>
              </td>
          <td>
            M. Hatakeyama, Shuhei Kimura, T. Naka, T. Kawasaki, N. Yumoto, Mio Ichikawa, Jae-Hoon Kim, K. Saito, Mihoro Saeki, M. Shirouzu, S. Yokoyama, A. Konagaya
          </td>
          <td>2003-07-15</td>
          <td>Biochemical Journal, The Biochemical journal</td>
          <td>259</td>
          <td>112</td>

            <td><a href='../recommendations/c5c624f60a57498d015d6990eeb7e2ca9bf3aef4' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/011d43546dfc34d1075f28232de2a553fc797695" target='_blank'>
                Thermodynamically consistent model calibration in chemical kinetics
                </a>
              </td>
          <td>
            Garrett Jenkinson, J. Goutsias
          </td>
          <td>2011-05-06</td>
          <td>BMC Systems Biology</td>
          <td>9</td>
          <td>29</td>

            <td><a href='../recommendations/011d43546dfc34d1075f28232de2a553fc797695' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/150e1f4a4ec7838cc3188f00cdd15be74d14e417" target='_blank'>
                Ligand-Specific c-Fos Expression Emerges from the Spatiotemporal Control of ErbB Network Dynamics
                </a>
              </td>
          <td>
            T. Nakakuki, M. Birtwistle, Y. Saeki, N. Yumoto, K. Ide, T. Nagashima, L. Brusch, B. Ogunnaike, M. Okada‐Hatakeyama, B. Kholodenko
          </td>
          <td>2010-05-28</td>
          <td>Cell</td>
          <td>231</td>
          <td>57</td>

            <td><a href='../recommendations/150e1f4a4ec7838cc3188f00cdd15be74d14e417' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0c22a2a310dc721699b021da353da90a5c2c2f2" target='_blank'>
                Optimal enumeration of state space of finitely buffered stochastic molecular networks and exact computation of steady state landscape probability
                </a>
              </td>
          <td>
            Youfang Cao, Jie Liang
          </td>
          <td>2008-03-29</td>
          <td>BMC Systems Biology</td>
          <td>58</td>
          <td>12</td>

            <td><a href='../recommendations/b0c22a2a310dc721699b021da353da90a5c2c2f2' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for MAPK'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>MAPK</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Cells compartmentalize biochemical processes using physical barriers in the form of membranes. Eukaryotes have a wide diversity of membrane-based compartments that can be used in this context, with the main ones being the extracellular membrane, which separates the inside from the outside of the cell, and the nuclear membrane, which separates the nucleus from the cytoplasm. The nuclear membrane not only isolates and protects the DNA and the transcription and replication processes from the other processes that are occurring in the cytoplasm but also has an active role in the regulation of cellular signaling. The TGF-β pathway is one of the most important and conserved signaling cascades, and it achieves compartmentalization using a well-tuned balance between the import and export rates of the active and inactive forms of key proteins. Thus, compartmentalization serves as an additional regulatory mechanism, physically isolating transcription factors from their targets, influencing the dynamics and strength of signal transduction. This contribution focuses on this biophysical layer of regulation, using the TGF-β pathway to illustrate the molecular mechanisms underlying this process, as well as the biological consequences of this compartmentalization. We also introduce a simplified mathematical formulation for studying the dynamics of this process using a generalized approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e8fd0965c2c60ebf4f5853c92d53e53c16cef7f" target='_blank'>
              Cellular Compartmentalization as a Physical Regulatory Mechanism of Signaling Pathways
              </a>
            </td>
          <td>
            Ahmed N. Fayad, Diego Mazó-Durán, David G. Míguez
          </td>
          <td>2024-12-10</td>
          <td>Biophysica</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract The KRAS gene is nearly ubiquitously subjected to activating mutation in pancreatic adenocarcinomas (PDAC), occurring at a frequency of over 90% in tumors. Mutant KRAS drives sustained signaling through the MAPK pathway to affect frequently disrupted cancer phenotypes including transcription, proliferation and cell survival. Recent research has shown that PDAC tumor growth and survival required a guanine nucleotide exchange factor for RAS homolog family member A (RhoA) called GEF-H1. The GEF-H1 protein, encoded by the ARHGEF2 gene, is a microtubule-associated GEF for RhoA that promotes invasion-migration of PDAC cells via activation of RhoA. Unexpectedly, independent of its RhoGEF activity, GEF-H1 was found to potentiate MAPK signaling by scaffolding protein phosphatase 2A (PP2A) to the kinase suppressor of Ras 1 (KSR-1). In a feedback-dependent manner, enhanced MAPK activity drives expression of ARHGEF2 via regulation of transcription factors ETS and SP, and the RAS responsive element-binding protein 1 (RREB1). RREB1 a negative regulator of ARHGEF2 expression, is downregulated in PDAC cells, which permits sustained expression of GEF-H1 for PDAC tumor survival and subsequent MAPK pathway activation. Given that MAPK targeted therapies show limited clinical efficacy, highlights the need for novel targets. This review describes the unexpected complexity of GEF-H1 function leading to positive feedback that potentiates RAS-MAPK signaling and suggests inhibition of GEF-H1 as a therapeutic strategy for RAS-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f610b92c6965aec299b0a5a0325df9a496fbf4a" target='_blank'>
              Dynamic Coupling of MAPK Signaling to the Guanine Nucleotide Exchange Factor GEF-H1
              </a>
            </td>
          <td>
            Kévin Leguay, Oliver Kent
          </td>
          <td>2025-01-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Biological systems in general operates out of equilibrium which demands the requirement for constant supply of energy due to non-equilibrium entropy production. The thermodynamic uncertainty relation (TUR) essentially imposes a bound on minimum current fluctuation the system can have given a entropy production rate. The fluctuation eventually impacts the signal to noise ratio imposing an upper bound on information transmission accuracy. In this study, we explored the role of TUR on the information transmission capacity of a set of cellular signaling systems using a coupled mathematical and machine learning approaches to experimental data in yeast under several stress conditions. Cell signaling systems are involved in sensing the changes in environment by activating a set of transcription factors which typically diffuse inside the nucleus to trigger transcription of the required genes. However, the inherent stochasticity of the biochemical pathways associated with signaling processes severely limits the accuracy of estimating the environmental input by the transcription factors. Application of TUR reveals a general picture about the working principle of the transcription factors. We found that the the activation followed by biased diffusion of transcription factors (TF) towards the nucleus triggers entropy production which amplifies magnitude of the overall TF currents towards the nucleus as well as reduces the fluctuations. These outcomes significantly improve the accuracy of information transmission carried out by the transcription factors following the bound imposed by TUR. Thus, experimental observations coupled with TUR based theoretical models demonstrate the role of thermodynamic fluctuation and entropy production on cellular information processing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ba0927e8af8fd7516c0db7b7efdf39231317eb0" target='_blank'>
              Thermodynamic uncertainty relation constraints information transmission through cell signaling systems
              </a>
            </td>
          <td>
            Shreyansh Verma, Vishva Saravanan, Bhaswar Ghosh
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cell states evolve through the combined activity of signaling pathways and gene regulatory networks. While transcription factors can direct cell fate, these factors rely on a cell state that is receptive to transitions in cell identity. How signaling levels contribute to the emergence of receptive cell states remains poorly defined in primary cells. Using a well-defined model of direct conversion, we examined how levels of the MAPK-activating oncogene HRASG12V influence direct conversion of primary fibroblasts to induced motor neurons. We demonstrate that an optimal ‘Goldilocks’ level of MAPK signaling efficiently drives cell-fate programming. Rates of direct conversion respond biphasically to increasing HRASG12V levels. While intermediate HRASG12V levels increase the rate of conversion, high levels of HRASG12V induce senescence. Through chemogenetic tuning, we set optimal MAPK activity for high rates of conversion in the absence of HRAS mutants. As MAPK pathways influence cell-fate transitions in development and disease, our results highlight the need to tune therapeutic interventions within a non-monotonic landscape that is shaped by genetics and levels of gene expression. Highlights MAPK signaling drives proliferation and conversion of fibroblasts to motor neurons Cell-fate programming responds biphasically to HRASG12V expression High HRASG12V expression induces senescence, which reduces conversion Chemogenetic tuning of MAPK activity increases conversion rates A small-molecule MAPK inducer drives high rates of conversion in the absence of HRASG12V">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fa4a4e600cf88c5b704f193ed57b3b5cfc188f0" target='_blank'>
              Chemogenetic tuning reveals optimal MAPK signaling for cell-fate programming
              </a>
            </td>
          <td>
            Brittany A. Lende-Dorn, Jane C. Atkinson, Yunbeen Bae, Kate E. Galloway
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Reversible protein phosphorylation directs essential cellular processes including cell division, cell growth, cell death, inflammation, and differentiation. Because protein phosphorylation drives diverse diseases, kinases and phosphatases have been targets for drug discovery, with some achieving remarkable clinical success. Most protein kinases are activated by phosphorylation of their activation loops, which shifts the conformational equilibrium of the kinase toward the active state. To turn off the kinase, protein phosphatases dephosphorylate these sites, but how the conformation of the dynamic activation loop contributes to dephosphorylation was not known. To answer this, we modulated the activation loop conformational equilibrium of human p38α ΜΑP kinase with existing kinase inhibitors that bind and stabilize specific inactive activation loop conformations. From this, we identified three inhibitors that increase the rate of dephosphorylation of the activation loop phospho-threonine by the PPM serine/threonine phosphatase WIP1. Hence, these compounds are "dual-action" inhibitors that simultaneously block the active site and promote p38α dephosphorylation. Our X-ray crystal structures of phosphorylated p38α bound to the dual-action inhibitors reveal a shared flipped conformation of the activation loop with a fully accessible phospho-threonine. In contrast, our X-ray crystal structure of phosphorylated apo human p38α reveals a different activation loop conformation with an inaccessible phospho-threonine, thereby explaining the increased rate of dephosphorylation upon inhibitor binding. These findings reveal a conformational preference of phosphatases for their targets and suggest a unique approach to achieving improved potency and specificity for therapeutic kinase inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60818e7ae2fbfb6a37b9e778746a117f7ce8c1e2" target='_blank'>
              Dual-action kinase inhibitors influence p38α MAP kinase dephosphorylation.
              </a>
            </td>
          <td>
            Emily J Stadnicki, Hannes Ludewig, Ramasamy P Kumar, Xicong Wang, Youwei Qiao, Dorothee Kern, Niels Bradshaw
          </td>
          <td>2024-12-31</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Mitogen-activated protein kinase (MAPK) and c-Jun N-terminal kinase (JNK) pathways exhibit complex signal transduction pathways that modulate multiple processes and the deregulated form of MAPK is involved in many types of cancers including skin cancer. These signaling pathways regulate cellular processes such as differentiation, inflammation, cell proliferation, and survival. Moreover, MAPKs are crucial signaling pathways in regulating various cellular processes, including cell proliferation, differentiation, survival, and apoptosis. Their dysregulation is commonly observed in cancer, making them attractive targets for therapeutic intervention. The critical mechanisms and pathways of MAPKs and JNKs in skin cancer including basal cell carcinoma (BCC), squamous cell carcinoma (SCC), and melanoma are summarized in this review. In addition to providing new insights into MAPK and JNK pathways in cancer progression and proliferation, we also offered the significance of the mechanism of JNK in drug resistance and the association of KLF4 in influencing JNKs for future endeavors to be targeted as therapeutic approaches as understanding the molecular aspects of these signaling pathways are crucial for novel therapeutic breakthroughs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44c1ad35e8bd78ba9ef3762f50fa41c883998750" target='_blank'>
              Role of MAPK and JNK Signaling Pathways in Skin Cancer Progression and Therapeutic Approach
              </a>
            </td>
          <td>
            Hamza Khaliq
          </td>
          <td>2025-01-07</td>
          <td>Dermatological Health</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract In response to distinct cellular stresses, the p53 exhibits distinct dynamics. These p53 dynamics subsequently control cell fate. However, different stresses can generate the same p53 dynamics with different cell fate outcomes, suggesting that the integration of dynamic information from other pathways is important for cell fate regulation. The interactions between miRNA-125b, p53, and reactive oxygen species (ROS) are significant in the context of cellular stress responses and apoptosis. However, the regulating mechanism of miR-125b with p53 is not fully studied. The dynamics of p53 and its response to the miR-125b regulation are still open questions. In the present study, we try to answer some of these fundamental questions based on basic model built from available experimental reports. The miR-125b-p53 regulatory network is modeled using a set of 11 molecular species variables. The biochemical network of miR-125b-p53, described by 22 reaction channels, is represented by coupled ordinary differential equations (ODEs) using the mass action law of chemical kinetics. These ODEs are solved numerically using the standard fourth-order Runge–Kutta method to analyze the dynamical behavior of the system. The biochemical network model we designed is based on both experimental and theoretical reported data. The p53 dynamics driven by miR-125b exhibit five distinct dynamical states: first and second stable states, first and second dynamical states, and a sustained oscillation state. These different p53 dynamical states may correspond to various cellular conditions. If the stress induced by miR-125b is weak, the system will be weakly activated, favoring a return to normal functioning. However, if the stress is significantly strong, the system will move to an active state. To sustain this active state, which is far from equilibrium with little scope for returning to normal conditions, the system may transition to an apoptotic state by crossing through other intermediate states, as it is unlikely to regain normal functioning. The p53 dynamical states show a multifractal nature, contributed by both short- and long-range correlations. The networks illustrated from these dynamical states follow hierarchical scale-free features, exhibiting an assortative nature with an absence of the centrality-lethality rule. Furthermore, the active dynamical state is generally closer to hierarchical characteristics and is self-organized. Our research study reveals that significant activity of miR-125b on the p53 regulatory network and its dynamics can only be observed when the system is slightly activated by ROS. However, this process does not necessarily require the direct study of ROS activity. These findings elucidate the mechanisms by which cells integrate signaling pathways with distinct temporal activity patterns to encode stress specificity and direct diverse cell fate decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49d808bb15892aef64993cb3f1670cfa188db07c" target='_blank'>
              Complex p53 dynamics regulated by miR-125b in cellular responses to reactive oxidative stress and DNA damage
              </a>
            </td>
          <td>
            Md. Zubbair Malik, Mohammed Dashti, Amit Jangid, A. Channanath, Sumi Elsa John, R. K. B. Singh, F. Al-Mulla, Thangavel Alphonse Thanaraj
          </td>
          <td>2024-11-22</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Signaling pathways enable cells to coordinate collective behaviors by exchanging specific information. Many pathways utilize multiple ligand variants to activate the same intracellular signaling cascade, raising the question of how cells discriminate between these seemingly redundant signals. It has been shown that individual cells can discriminate between signals based on their induced level of activity, temporal dynamics or combinatorial effect. Here, we demonstrate that ligand discrimination could also emerge at the population level. Using mathematical models of ligand-receptor interactions, we examine how response heterogeneity at the population level can encode ligand identity. We introduce a local scaling metric to quantify how variation in pathway components affects the cellular response. Our results reveal that for pathways with dimeric receptors, and more significantly for heterodimeric receptors, biochemical parameters of the ligands control the resulting heterogeneity in the response of a population of cells. Furthermore, we show that the population-level heterogeneity encodes the enzymatic activity of the resulting receptor complex. This suggests a functional advantage for utilizing heterodimeric receptor complexes in pathways acting across a population of cells, such as the type I interferon pathway, which shows several of the characteristics of our model. This contrasts to juxtacrine pathways, such as Notch, that do not act at the population level and use a single component receptor. Overall, our findings highlight a novel mechanism by which receptor architecture enables cells to encode ligand-specific information through population-level heterogeneity, offering insights into immune regulation, tissue development, and synthetic biology. Significance Statement Cells often communicate using distinct molecular signals (ligands) that activate the same pathway. Yet, how cells distinguish between these signals remains unclear. This study reveals that ligand discrimination can emerge at the level of entire cell populations, not just individual cells. By analyzing mathematical models, we show that certain receptor architectures, especially those involving heterodimeric receptors, allow signals to control the heterogeneity of the response level across a population. This mechanism enables cell populations to decode specific information about their environment, impacting processes like immune responses and tissue development. Our findings provide new insights into how signals coordinate collective behaviors and suggest strategies for designing synthetic systems to precisely control biological responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2c9e614dd76f4eb67621730f0b39dda4cadda4e" target='_blank'>
              Receptor dimerization enables ligand discrimination through tunable response heterogeneity
              </a>
            </td>
          <td>
            Assaf Biran, Yaron E. Antebi
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Inflammasomes are crucial mediators of both antimicrobial host defense and inflammatory pathology, requiring stringent regulation at multiple levels. This review explores the pivotal role of mitogen-activated protein kinase (MAPK) signaling in modulating inflammasome activation through various regulatory mechanisms. We detail recent advances in understanding MAPK-mediated regulation of NLRP3 inflammasome priming, licensing and activation, with emphasis on MAPK-induced activator protein-1 (AP-1) signaling in NLRP3 priming, ERK1 and JNK in NLRP3 licensing, and TAK1 in connecting death receptor signaling to NLRP3 inflammasome activation. Furthermore, we discuss novel insights into MAPK signaling in human NLRP1 inflammasome activation, focusing on the MAP3K member ZAKα as a key kinase linking ribosomal stress to inflammasome activation. Lastly, we review recent work elucidating how Bacillus anthracis lethal toxin (LeTx) manipulates host MAPK signaling to induce macrophage apoptosis as an immune evasion strategy, and the counteraction of this effect through genotype-specific Nlrp1b inflammasome activation in certain rodent strains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac0fd5e7681ac38fdf1388dde27f9b65c1416877" target='_blank'>
              MAP Kinase Signaling at the Crossroads of Inflammasome Activation.
              </a>
            </td>
          <td>
            Alex Vervaeke, Mohamed Lamkanfi
          </td>
          <td>2025-01-01</td>
          <td>Immunological reviews</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Intracellular signaling pathways form networks through which information is transmitted, often in the form of kinase-mediated phosphorylation events, to interpret extracellular signals and elicit appropriate cellular responses. Yet, even isogenic cells in a homogenous environment show heterogeneity in their intracellular “cell-state”, as well as their response to extracellular signals. Here, we aimed to better understand this relation between these phenomena by investigating how information flows through the EGF-receptor centered network upon targeted drug treatment, and how this is affected by cell-to-cell-state differences. Using single-cell ID-seq, we profiled the cell-state and signaling activity of primary human epidermal stem cells by measuring 69 (phospho-)proteins upon inhibition of the Erk/MAPK (p90RSK) and Akt/mTOR (p70S6K) routes downstream of the EGF pathway. We found that the effects of drug treatment propagated from the EGF-signaling pathway to other connected parts of the cellular signaling network, indicating altered signaling flow. We identified nine distinct cell-states that show pervasive state-dependent drug-responses for many (phospho-)proteins. Computational modeling of the signaling network using single-cell Comparative Network Reconstruction showed that many interactions between phospho-proteins (i.e. network wiring) were quantitatively different between cell-states. Furthermore, (phospho-)proteins with a cell-state dependent drug response, were more likely to be involved in interactions that showed a cell-state dependent strength. Overall, our results indicate that drug treatment response and signaling interactions between proteins are closely related and modulated by cell-state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5aafd958806435b041f6f614bd7ef297f3ec839a" target='_blank'>
              Cell-state specific drug-responses are associated with differences in signaling network wiring
              </a>
            </td>
          <td>
            Niels Krämer, Roderick van Eijl, Tim Stohn, Sabine E. J. Tanis, Lodewyk F. A. Wessels, Evert Bosdriesz, Klaas W. Mulder
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="A lack of tools for detecting receptor activity in vivo has limited our ability to fully explore receptor-level control of developmental patterning. Here, we extend a new class of biosensors for receptor tyrosine kinase (RTK) activity, the pYtag system, to visualize endogenous RTK activity in Drosophila. We build biosensors for three Drosophila RTKs that function across developmental stages and tissues. By characterizing Torso::pYtag during terminal patterning in the early embryo, we find that Torso activity differs from downstream ERK activity in two surprising ways: Torso activity is narrowly restricted to the poles but produces a broader gradient of ERK, and Torso activity decreases over developmental time while ERK activity is sustained. This decrease in Torso activity is driven by ERK pathway-dependent negative feedback. Our results suggest an updated model of terminal patterning where a narrow domain of Torso activity, tuned in amplitude by negative feedback, locally activates signaling effectors which diffuse through the syncytial embryo to form the ERK gradient. Altogether, this work highlights the usefulness of pYtags for investigating receptor-level regulation of developmental patterning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdd4f36abf0e565dedd3a0af403a70c10f3cbae2" target='_blank'>
              In vivo measurements of receptor tyrosine kinase activity reveal feedback regulation of a developmental gradient
              </a>
            </td>
          <td>
            Emily K. Ho, Rebecca P. Kim-Yip, Alison G Simpkins, Payam E. Farahani, Harrison R. Oatman, Eszter Posfai, Stanislav Y. Shvartsman, Jared E. Toettcher
          </td>
          <td>2025-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Understanding how cells migrate following exogenous cues is one of the most fundamental questions for biology, medicine, and bioengineering. Growing evidence shows that electrotaxis, the directed cell migration toward electric potential gradients, represents a precise and programmable method to control cell migration. Most data suggest that the polarization of membrane components and the following downstream signaling are central to electrotaxis. Unfortunately, how these multiple mechanisms coordinate with the motile machinery of the cell to respond to an electric field is still poorly understood. Here, we develop a mechanistic model that explains and recapitulate electrotaxis across different cell types. Using the zebrafish proteome, we identify membrane proteins directly related to migration signaling pathways that polarize anodally and cathodally. Further, we show that simultaneous and asymmetric distribution of charged membrane receptors towards the anode and the cathode establishes multiple cooperative and competing stimuli for downstream signaling pathways. The resulting polarization of signals controls the actomyosin network dynamics, directing the anodal and cathodal migration of the cell. Our theoretical framework rationalizes the physical processes that determine electrotaxis across cell types and provides a physical framework to test multiple electrotactic pathways. These results together show us how to control cell migration to, e.g., enhance, cancel, or switch directed cell migration, which opens up new avenues not only to promote tissue regeneration or arrest tumor progression but also to design better biomimetic-engineered tissue constructs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94fec75f867824b781d8dea0a23a5fcc6f603b02" target='_blank'>
              Competing signaling pathways controls electrotaxis
              </a>
            </td>
          <td>
            S. Kulkarni, F. Tebar, C. Rentero, M. Zhao, P. Sáez
          </td>
          <td>2025-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="he increasing number of diseases associated with aberrant MAPK signaling underscores the need for a deeper understanding of its regulatory mechanisms: the MAPK pathway is closely associated with cellular responses to various stimuli, and its dysregulation can cause pathologies. This study focuses on the regulatory mechanisms of the MAPK signaling pathway and the role of MAPK phosphatase (MKP) in regulating its activity. The goal of this study is to elucidate the specific function of MKP in the regulation of MAPK signaling and to identify potential therapeutic targets for diseases associated with MAPK dysregulation. In this study, the interaction between MKP and MAPK signaling was investigated using cellular models and biochemical experiments. The primary targets were cells expressing MAPK and MKP, and protein-protein interactions and signaling dynamics were analyzed using methods such as co-immunoprecipitation, confocal microscopy, and flow cytometry. According to this study, MKP is crucial for the exact temporal and spatial modulation of MAPK signaling through its negative regulation. An imbalance in this balance can lead to pathological conditions. This study highlights the importance of MKP in cellular homeostasis and highlights its potential as a therapeutic agent.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8819fe5eb2f02fb5c10960c9542ee800ae69220" target='_blank'>
              A novel regulatory mechanism of the MAPK signaling pathway: the negative regulatory role of MKP-1 and its pathophysiological significance
              </a>
            </td>
          <td>
            Ziyue Wei
          </td>
          <td>2024-12-06</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extracellular signal-regulated kinase (ERK) regulates multiple cellular functions through distinct activation patterns. Genetically encoded fluorescent probes are instrumental in dissecting the ERK activity dynamics in living cells. Here we modified a previously reported Förster resonance energy transfer (FRET) probe for ERK, EKAREN5 by replacing its mTurquoise2 and YPet sequences with mTurquoise-GL and a synonymous codon variant of YPet, respectively. The modified biosensor, EKAREN5-gl showed an increased sensitivity to EGF-induced ERK activation responding to a very low dose (20 pg/ml) of EGF stimulation. We quantitatively characterized two FRET-based ERK probes, EKAREN5 and EKAREN5-gl, and a subcellular kinase translocation-based probe, ERK-KTR. We found the three biosensors differently respond to EGF stimulations with different intensity, duration, and latency. Furthermore, we investigated how the minimal EGF-induced ERK activation affects the downstream transcription in HeLa cells by comprehensive transcriptional analysis. We found the minimal ERK activation leads to a distinct transcriptional pattern from those induced by higher ERK activations. Our study highlights the significance of sensitive fluorescent probes to understand cellular signal dynamics and the role of minimal ERK activation in regulating transcription.Key words: fluorescent probe, ERK, FRET, KTR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab60d10f742a6c10270988f0a442871b1aa59200" target='_blank'>
              A sensitive ERK fluorescent probe reveals the significance of minimal EGF-induced transcription.
              </a>
            </td>
          <td>
            Weisheng Zhang, Jun Nakayama, Yukino Inomata, Shigeki Higashiyama, Toru Hiratsuka
          </td>
          <td>2024-12-18</td>
          <td>Cell structure and function</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Photo-proximity labeling proteomics (PLP) methods have recently shown that the cell surface receptors can dynamically form lateral interactome networks. Here, we present a paired set of PLP workflows that simultaneously track neighborhood changes for oncogenic epidermal growth factor receptor (EGFR) with temporal resolution, both outside and inside of cells. We achieved this by augmenting the multiscale PLP workflow we call MultiMap, where three photo-probes with different labeling ranges were photo-activated by one photocatalyst, Eosin Y. By anchoring Eosin Y extracellularly and intracellularly on EGFR, we captured hundreds of proteins on both sides of the cell membrane that change in proximity to EGFR upon EGF activation. Neighbors engaged with EGFR within minutes to over an hour, reflecting dynamic interactomes during early, mid- and late-signaling including phosphorylation, internalization, degradation and transcriptional regulation. This rapid “photographic” labeling approach provides snapshots of signaling neighborhoods, revealing their dynamic nature, and potential for drug targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85961c05301971a038d9185d7b8c97f67296dbe6" target='_blank'>
              Spatiotemporal photocatalytic proximity labeling proteomics reveals ligand-activated extracellular and intracellular EGFR neighborhoods
              </a>
            </td>
          <td>
            Zhi Lin, Wayne Ngo, Yu-Ting Chou, Harry Wu, K. Susa, Young-wook Jun, T. Bivona, Jennifer A. Doudna, James A. Wells
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Signaling by G protein-coupled receptors (GPCRs) is regulated by temporally distinct processes including receptor desensitization, internalization, and lysosomal sorting, and are tightly controlled by post-translational modifications. While the role of phosphorylation in regulating GPCR signaling is well studied and established, the mechanisms by which other post-translational modifications, such as ubiquitination, regulate GPCR signaling are not clearly defined. We hypothesize that GPCR ubiquitination and deubiquitination is critical for proper signaling and cellular responses. In the present study, we show that the deubiquitinase ubiquitin-specific protease-34 (USP34) regulates thrombin-stimulated protease-activated receptor-1 (PAR1)-induced p38 autophosphorylation and activation. The PAR1-stimulated p38 signaling pathway is driven by ubiquitination. Interestingly, small interfering RNA-induced knockdown of USP34 expression markedly increased PAR1 cell surface abundance and protein expression without modulating PAR1 ubiquitination or the ubiquitination status of p38 signaling pathway components. In addition, increased PAR1 expression observed in USP34-depleted cells was not caused by altered PAR1 constitutive internalization, agonist-induced internalization, or receptor degradation. Rather, we report that loss of USP34 expression increased mRNA transcript expression of the PAR1-encoding gene, F2R. This study unexpectedly identified a critical role for USP34 in regulation of F2R mRNA transcript expression, which modulates PAR1 cell surface levels and thrombin-stimulated p38 mitogen-activated protein kinase signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1490e9b25de663f6480902fb7c3d9489e6eba90f" target='_blank'>
              USP34 regulates endothelial PAR1 mRNA transcript expression and cellular signaling.
              </a>
            </td>
          <td>
            Norton Cheng, Monica L. Gonzalez Ramirez, Chloe Edwards, J. Trejo
          </td>
          <td>2024-12-20</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a90212d3ced7e505412366e3e7ffc4b08c5ef8f2" target='_blank'>
              Oncogenic PIK3CA corrupts growth factor signaling specificity.
              </a>
            </td>
          <td>
            R. Madsen, A. Le Marois, O. Mruk, M. Voliotis, S. Yin, J. Sufi, X. Qin, Salome J Zhao, J. Gorczynska, D. Morelli, L. Davidson, E. Sahai, V. Korolchuk, C. Tape, B. Vanhaesebroeck
          </td>
          <td>2024-12-20</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="ConspectusA key challenge in modern chemistry research is to mimic life-like functions using simple molecular networks and the integration of such networks into the first functional artificial cell. Central to this endeavor is the development of signaling elements that can regulate the cell function in time and space by producing entities of code with specific information to induce downstream activity. Such artificial signaling motifs can emerge in nonequilibrium systems, exhibiting complex dynamic behavior like bistability, multistability, oscillations, and chaos. However, the de novo, bottom-up design of such systems remains challenging, primarily because the kinetic characteristics and energy aspects yielding bifurcation have not yet been globally defined. We herein review our recent work that focuses on the design and functional analysis of peptide-based networks, propelled by replication reactions and exhibiting bistable behavior. Furthermore, we rationalize and discuss their exploitation and implementation as variable signaling motifs in homogeneous and heterogeneous environments.The bistable reactions constitute reversible second-order autocatalysis as positive feedback to generate two distinct product distributions at steady state (SS), the low-SS and high-SS. Quantitative analyses reveal that a phase transition from simple reversible equilibration dynamics into bistability takes place when the system is continuously fueled, using a reducing agent, to keep it far from equilibrium. In addition, an extensive set of experimental, theoretical, and simulation studies highlight a defined parameter space where bistability operates.Analogous to the arrangement of protein-based bistable motifs in intracellular signaling pathways, sequential concatenation of the synthetic bistable networks is used for signal processing in homogeneous media. The cascaded network output signals are switched and erased or transduced by manipulating the order of addition of the components, allowing it to reach "on demand" either the low-SS or high-SS. The pre-encoded bistable networks are also useful as a programming tool for the downstream regulation of nanoscale materials properties, bridging together the Systems Chemistry and Nanotechnology fields. In such heterogeneous cascade pathways, the outputs of the bistable network serve as input signals for consecutive nanoparticle formation reaction and growth processes, which-depending on the applied conditions-regulate various features of (Au) nanoparticle shape and assembly.Our work enables the design and production of various signaling apparatus that feature higher complexity than previously observed in chemical networks. Future studies, briefly discussed at the end of the Account, will be directed at the design and analysis of more elaborate functionality, such as bistability under flow conditions, multistability, and oscillations. We propose that a profound understanding of the design principles facilitating the replication-based bistability and related functions bear implications for exploring the origin of protein functionality prior to the highly evolved replication-translation-transcription machinery. The integration of our peptide-based signaling motifs within future synthetic cells seems to be a straightforward development of the two alternating states as memory and switch elements for controlling cell growth and division and even communication among different cells. We furthermore suggest that such systems can be introduced into living cells for various biotechnology applications, such as switches for cell temporal and spatial manipulations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/021cab477bd5cee1ac8ab0c1f9f6423947694de2" target='_blank'>
              Bistable Functions and Signaling Motifs in Systems Chemistry: Taking the Next Step Toward Synthetic Cells.
              </a>
            </td>
          <td>
            Indrajit Maity, N. Wagner, Dharm Dev, G. Ashkenasy
          </td>
          <td>2025-01-22</td>
          <td>Accounts of chemical research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The p38 MAP kinase (MAPK) signaling pathway plays pivotal roles in various cellular processes. Phosphorylation serves as a canonical way to regulate p38α activation through a phosphorylation cascade. Thus, understanding the mechanism governing p38α phosphorylation is important. The present study demonstrated that p38α undergoes methylation at K165, which promote its phosphorylation in tumor cells. Inhibition of p38α methylation impairs p38α phosphorylation, repressing tumor progression in vitro and in vivo. Mechanistically, KDM5D is a demethylase that interacts with p38α, mediating demethylation at K165 and inhibiting p38α phosphorylation. Moreover, KDM5D is expressed at low levels in non-small cell lung cancer (NSCLC), and high KDM5D expression is positively correlated with cancer survival. KDM5D markedly inhibits cell proliferation and migration via inactivating p38α, thereby slowing cancer progression in xenograft models. In summary, these findings highlight KDM5D as a demethylase of p38α at K165, elucidating a unique role for lysine demethylation in integrating cytoplasmic kinase-signaling cascades. The present results revealed the critical role of KDM5D in suppressing tumor progression, suggesting that KDM5D can serve as a potential drug target for combating hyperactive p38α-driven lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cceeb36986eb6be9d7aadc46f1da63b8e738fd94" target='_blank'>
              KDM5D histone demethylase mediates p38α inactivation via its enzymatic activity to inhibit cancer progression.
              </a>
            </td>
          <td>
            Jingying Chen, Ting Wang, Dongzhe Zhang, Huiling Wang, Zhiang Huang, Zhongxin Yang, Jizhuo Li, Tianyi Hu, Xin Wang, Xia Li
          </td>
          <td>2024-12-05</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The allosteric mechanism of G-protein-coupled receptors (GPCRs) involves a population shift from inactive to active receptor conformations in response to the binding of ligand agonists. Two possible kinetic mechanisms for this population shift are induced fit and conformational selection. The two mechanisms differ in the temporal sequence of binding events and conformational changes: Ligand bindings occurs prior to the change from the inactive to the active receptor conformation in the induced-fit mechanism, and after the conformational change in the conformational-selection mechanism. In this article, we discuss the current evidence from experiments that probe the binding kinetics of GPCRs to identify the allosteric mechanism. For the peptide-activated neurotensin receptor 1, the modeling of kinetic data from stopped-flow mixing experiments indicates an induced-fit mechanism 1. The conformational exchange rates of the induced-fit mechanism obtained from this modeling agree with rates measured by saturation transfer difference NMR experiments of the peptide-receptor complex, which corroborates the mechanism. For the small-molecule-activated β2-andrenergic receptor, an induced-fit mechanism has been inferred from a comparison of ligand-association rates for the inactive and the active receptor conformation 2. A stabilization of the active receptor conformation by G proteins or nanobodies leads to a decrease of ligand association rates, which indicates that ligand binding occurs in the inactive conformation and, thus, prior to the change from the inactive to the active conformation as in the induced-fit mechanism. A structural explanation for the induced-fit mechanism of the β2-andrenergic receptor is a closed lid over the binding site that blocks ligand entry in the active conformation. Since constriction and closing of the ligand-binding site in the active conformation is rather common for small-molecule-activated and peptide-activated GPCRs, induced fit likely is shared as allosteric mechanism by these GPCRs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5174eb8b355e2621db203af2c729461950ad5d36" target='_blank'>
              The allosteric mechanism of G-protein-coupled receptors is induced fit, not conformational selection
              </a>
            </td>
          <td>
            Kazem Asadollahi, P. Gooley, T. Weikl
          </td>
          <td>2025-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Animal bodies are equipped to sense injuries and respond, and rapid activation of ERK signaling at wound sites has emerged as a recurring mechanism across metazoans. During whole-body regeneration, distant species including cnidarians, planarians and echinoderms activate ERK at wounds to drive early transcriptional changes. However, it is unclear to what extent these are distinct phenomena, utilizing a common effector pathway but with different upstream inputs and downstream functions, or are more deeply alike, sharing upstream inputs, activating cell type(s), or spatiotemporal dynamics. To facilitate thorough comparisons across animals, we examined wound-induced ERK during whole-body regeneration in the acoel Hofstenia miamia. ERK signaling is induced at wound sites within 10 minutes post-amputation and peaks in activity within one hour. ERK is required for head and tail regeneration and inhibiting ERK suppresses the expression of wound-induced genes, which are themselves required for regeneration. Quantification of spatial dynamics revealed that active ERK spreads rapidly away from the wound site during the first 30 minutes post-amputation and continues to expand in space through 3-6 hours, albeit at lower levels. Multiple cell types activate ERK including neoblasts (stem cells) and muscle, and ERK is spatially dynamic within both cell types, however, ERK activation in muscle does not require neoblasts. Finally, neuregulin-1, a putative ligand produced exclusively by muscle cells, is required for ERK activation at wounds. Altogether, our data identify a key signaling role of muscle cells in driving ERK activation in multiple cell types after injury, and describe a spatial spreading phenomenon with parallels to dynamic ERK patterns in other systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d1a2403c0474215191c4d15b98eef5af7d83fb4" target='_blank'>
              A spreading, multi-tissue wound signal initiates whole-body regeneration
              </a>
            </td>
          <td>
            Catriona Breen, M. Srivastava
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Mitogen-activated protein kinase 1 (MAPK1) is a serine/threonine kinase that plays a crucial role in the MAP kinase signaling transduction pathway. This pathway plays a crucial role in various cellular processes, including cell proliferation, differentiation, adhesion, migration, and survival. Besides, many chemotherapeutic drugs targeting the MAPK pathway are used in clinical practice, and novel inhibitors of MAPK1 with improved specificity and efficacy are required. Hence, targeting MAPK1 can be crucial to control metastasis in cancer therapeutics. In this study, we utilized a structure-guided virtual screening approach to screen a library of thousands of natural compounds from the ZINC database. The Lipinski rule of five (RO5) was used as a criterion for the primary selection of natural compounds. The screened compounds were prioritized based on their binding affinity, docking scores, and specificity towards the kinase domain of MAPK1 during the molecular docking process. Subsequently, the selected hits underwent rigorous screening that included the identification of potential pan-assay interference compounds (PAINS), ADMET evaluation, and prediction of pharmacological activities using PASS analysis. Afterwards, we performed a comprehensive interaction analysis to explore the binding prototypes of the screened molecules with the key residues within the MAPK1 kinase domain. Finally, selected molecules underwent extensive all-atom molecular dynamics (MD) simulations for a time duration of 200 nanoseconds. The study pinpointed three natural compounds with ZINC database IDs ZINC0209285, ZINC02130647, and ZINC02133691 as potential inhibitors of MAPK1. The study highlights that these compounds could be explored further in preclinical and clinical investigations to develop anticancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f224494acb4d86bdceddd0a86d84714f762e4b9" target='_blank'>
              Structure-guided identification of mitogen-activated protein kinase-1 inhibitors towards anticancer therapeutics
              </a>
            </td>
          <td>
            Md Nayab Sulaimani, Shazia Ahmed, F. Anjum, Taj Mohammad, Anas Shamsi, Ravins Dohare, Md Imtiyaz Hassan
          </td>
          <td>2025-01-24</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb205e617609411844c74b772c07c6d871da38db" target='_blank'>
              GRK5 is required for adipocyte differentiation through ERK activation.
              </a>
            </td>
          <td>
            Mary E Seramur, Bailey McDonald, M. Davis, Tony E Reeves, Leah C. Solberg Woods, Chia-Chi Chuang Key
          </td>
          <td>2025-01-21</td>
          <td>International journal of obesity</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="OFD1 is a protein involved in many cellular processes, including cilia biogenesis, mitotic spindle assembly, translation, autophagy and the repair of double-strand DNA breaks. Despite many potential interactors identified in high-throughput studies, only a few have been directly confirmed with their binding sites identified. We performed an analysis of the evolutionary conservation of the OFD1 sequence in three clades: 80 Tetrapoda, 144 Vertebrata or 26 Animalia species, and identified 59 protein-binding motifs localized in the OFD1 regions conserved in various clades. Our results indicate that OFD1 contains 14 potential post-translational modification (PTM) sites targeted by at least eight protein kinases, seven motifs bound by proteins recognizing phosphorylated aa residues and a binding site for phosphatase 2A. Moreover, OFD1 harbors both a motif that enables its phosphorylation by mitogen-activated protein kinases (MAPKs) and a specific docking site for these proteins. Generally, our results suggest that OFD1 forms a scaffold for interaction with many proteins and is tightly regulated by PTMs and ligands. Future research on OFD1 should focus on the regulation of OFD1 function and localization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fce2a1dcf805b72e6f83f9890506825bd166fb0" target='_blank'>
              Conservation of OFD1 Protein Motifs: Implications for Discovery of Novel Interactors and the OFD1 Function
              </a>
            </td>
          <td>
            P. Jagodzik, E. Ziętkiewicz, Zuzanna Bukowy-Bieryłło
          </td>
          <td>2025-01-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Systemic signaling is an essential hallmark of multicellular life. Pathogen encounter occurs locally but triggers organ-scale and organismic immune responses. In plants, elicitor perception provokes systemically expanding Ca2+ and H2O2 signals conferring immunity. Here, we identify a Ca2+ sensing bi-kinase module as becoming super-activated through mutual phosphorylation and as imposing synergistically enhanced NADPH oxidase activation. A combined two-layer bi-kinase/substrate phospho-code allows for sensitized signaling initiation already by near-resting elevations of Ca2+ concentration. Subsequently, it facilitates further signal wave proliferation with minimal Ca2+ amplitude requirement, triggering protective defense responses throughout the plant. Our study reveals how plants build and perpetuate trans-cellular immune signal proliferation while avoiding disturbance of ongoing cellular signaling along the path of response dissemination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66de1c2c0075cafbaa1efdaa9d75ca99565ee8f4" target='_blank'>
              A bi-kinase module sensitizes and potentiates plant immune signaling
              </a>
            </td>
          <td>
            P. Köster, Gefeng He, Changyun Liu, Qiuyan Dong, Katarina Hake, Ina Schmitz-Thom, Paulina Heinkow, Jürgen Eirich, L. Wallrad, Kenji Hashimoto, Stefanie Schültke, I. Finkemeier, Tina Romeis, Jörg Kudla
          </td>
          <td>2025-01-24</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Signalling molecules often contribute to several signalling pathways that produce distinct transcriptional outputs and cellular phenotypes. One of the major unanswered questions in cell biology is how multiple activities of signalling molecules are coordinated in space and time in vivo. Here, we focus on the Signal Transducer and Activator of Transcription (STAT) protein as a paradigm of signalling molecules involved in several independent signalling pathways. In addition to the canonical pathways, whereby STAT is phosphorylated and activated by Janus Kinases, STAT is involved in at least two non-canonical pathways. In one pathway, STAT is activated by the Epidermal Growth Factor Receptor (EGFR) promoting apoptosis. In another, it binds the Heterochromatin Protein 1 (HP1) to enhance heterochromatin formation. Using Drosophila wing discs as an epithelial system, we demonstrate that all three STAT activities coexist in this tissue. We provide evidence that while the canonical STAT signalling is dominant over non-canonical pathways, EGFR:STAT and HP1:STAT pathways compete for the availability of unphosphorylated STAT. We also describe the role of the cell-cell adhesion protein E-cadherin in EGFR:STAT signalling. Both EGFR and STAT colocalise with E-cadherin at cell-cell junctions and on intracellular vesicles. Elevated intracellular E-cadherin promotes EGFR:STAT pathway leading to apoptosis, whereas blocking E-cadherin endocytosis prevents apoptosis induction. Altogether, we propose that E-cadherin endocytosis controls the balance between two non-canonical STAT activities in a potential tumour-suppressive mechanism – junctional disassembly in dysregulated epithelial-to-mesenchymal transitions would shift this balance towards the EGFR:STAT signalling to promote apoptosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a1a715500cec1bfcf17c9fc4e5bea5ac5403855" target='_blank'>
              E-cadherin endocytosis promotes non-canonical EGFR:STAT signalling to induce cell death and inhibit heterochromatinisation
              </a>
            </td>
          <td>
            Miguel Ramírez-Moreno, Amy Quinton, Eleanor Jacobsen, Przemyslaw A. Stempor, Martin P. Zeidler, N. Bulgakova
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The proliferative effects of angiotensin (Ang) II in vascular smooth muscle cells (VSMCs) through its ability to stimulate extracellular signal-regulated kinases 1 and 2 (ERK1/2) pathway have been established. The main goal of this study was to explore whether Ang III induces ERK1/2 MAPK and VSMC proliferation in cultured Wistar VSMCs. Further, the Ang III actions were compared to those observed in VSMCs derived from the spontaneously hypertensive rat (SHR). We hypothesized that in VSMCs Ang III will have similar actions as Ang II to induce ERK1/2 MAPK and cellular proliferation and this ability may be different in VSMCs isolated from Wistar versus SHR rats. Time and/or concentration-dependent effects of Ang III and Ang II were determined in VSMCs using western blot analysis and DNA incorporation assay. The results showed that ERK1/2 MAPK phosphorylation mediated by Ang II or Ang III were concentration- and time-dependent in Wistar VSMCs. Moreover, Ang III was less effective in mediating ERK1/2 phosphorylation in SHR VSMCs as compared to effects seen in Wistar rat VSMCs. Ang III induced ERK1/2 phosphorylation through the AT1 receptors activation. Ang II and Ang III induced VSMC DNA synthesis via the AT1 receptor in a concentration-dependent manner in Wistar VSMCs. Moreover, Ang III induced VSMC proliferation and significant differences existed in the peptide's proliferation effects in Wistar versus SHR VSMCs. These results indicate that Ang III stimulates ERK1/2 MAPK and DNA synthesis in VSMCs via AT1 receptors. However, its ability to stimulate these pathways is reduced in SHR VSMCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e7eb373b5bfb0644ce69d41441c68da8c17c8d2" target='_blank'>
              Angiotensin III activates ERK1/2 mitogen activated protein kinases and proliferation of rat vascular smooth muscle cells.
              </a>
            </td>
          <td>
            A. Alanazi, Michelle A Clark
          </td>
          <td>2025-01-13</td>
          <td>Journal of receptor and signal transduction research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Vasopressin (VP) activates protein kinase A (PKA), resulting in phosphorylation events and membrane accumulation of aquaporin-2 (AQP2). Epidermal growth factor receptor (EGFR) inhibition with erlotinib also induces AQP2 membrane trafficking with a phosphorylation pattern similar to VP, but without increasing PKA activity. Here, we identify the ribosomal s6 kinase (RSK) as a major mediator phosphorylating AQP2 in this novel, erlotinib-induced pathway. We found that RSK was expressed in collecting duct principal cells in rat kidneys. RSK inhibition with BI-D1870 blocked erlotinib-induced AQP2 serine 256 (S256) phosphorylation and membrane accumulation. CRISPR-generated RSK knockout cells failed to show increased S256 phosphorylation in response to erlotinib. Like PKA, RSK was able to phosphorylate AQP2 S256 in vitro. Inhibition of PDK1, a known activator of RSK, blocked erlotinib-induced AQP2 S256 phosphorylation and membrane accumulation. We conclude that RSK is a crucial terminal kinase phosphorylating AQP2 at S256 upon EGFR inhibition by erlotinib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745c964db62c85eff50aa395776077f82b81e561" target='_blank'>
              Ribosomal s6 kinase (RSK) is a mediator of aquaporin-2 S256 phosphorylation and membrane accumulation after EGFR inhibition with erlotinib.
              </a>
            </td>
          <td>
            Richard S E Babicz, N. Baylor, Abby Terlouw, Daphne A. Faber, Kazuhiko Fukushima, Ricardo M Biondi, R. Bouley, Dennis Brown
          </td>
          <td>2025-01-17</td>
          <td>American journal of physiology. Renal physiology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="KRAS is a pivotal oncoprotein that regulates cell proliferation and survival through interactions with downstream effectors such as RAF1. Oncogenic mutations in KRAS, including G12V, G13D, and Q61R, drive constitutive activation and hyperactivation of signaling pathways, contributing to cancer progression. Despite significant advances in understanding KRAS biology, the structural and dynamic mechanisms of KRAS binding and allostery by which oncogenic mutations enhance KRAS-RAF1 binding and signaling remain incompletely understood. In this study, we employ microsecond molecular dynamics simulations, Markov State Modeling, mutational scanning and binding free energy calculations together with dynamic network modeling to elucidate the effect of KRAS mutations and characterize the thermodynamic and allosteric drivers and hotspots of KRAS binding and oncogenic activation. Our simulations revealed that oncogenic mutations stabilize the open active conformation of KRAS by differentially modulating the flexibility of the switch I and switch II regions, thereby enhancing RAF1 binding affinity. The G12V mutation rigidifies both switch I and switch II, locking KRAS in a stable, active state. In contrast, the G13D mutation moderately reduces switch I flexibility while increasing switch II dynamics, restoring a balance between stability and flexibility. The Q61R mutation induces a more complex conformational landscape, characterized by the increased switch II flexibility and expansion of functional macrostates, which promotes prolonged RAF1 binding and signaling. Mutational scanning of KRAS-RAF1 complexes identified key binding affinity hotspots, including Y40, E37, D38, and D33, and together with the MM-GBSA analysis revealed the hotspots leverage synergistic electrostatic and hydrophobic binding interactions in stabilizing the KRAS-RAF1 complexes. Network-based analysis of allosteric communication identifies critical KRAS residues (e.g., L6, E37, D57, R97) that mediate long-range interactions between the KRAS core and the RAF1 binding interface. The central β-sheet of KRAS emerges as a hub for transmitting conformational changes, linking distant functional sites and facilitating allosteric regulation. Strikingly, the predicted allosteric hotspots align with experimentally identified allosteric binding hotspots that define the energy landscape of KRAS allostery. This study highlights the power of integrating computational modeling with experimental data to unravel the complex dynamics of KRAS and its mutants. The identification of binding hotspots and allosteric communication routes offers new opportunities for developing targeted therapies to disrupt KRAS-RAF1 interactions and inhibit oncogenic signaling. Our results underscore the potential of computational approaches to guide the design of allosteric inhibitors and mutant-specific therapies for KRAS-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6b6e2fe14e85ad43d29650e24307f1c5b387ed0" target='_blank'>
              Accurate Characterization of the Allosteric Energy Landscapes, Binding Hotspots and Long-Range Communications for KRAS Complexes with Effector Proteins : Integrative Approach Using Microsecond Molecular Dynamics, Deep Mutational Scanning of Binding Energetics and Allosteric Network Modeling
              </a>
            </td>
          <td>
            Sian Xiao, Mohammed Alshahrani, Guang Hu, Peng-Chu Tao, Gennady M. Verkhivker
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Negative feedback is a well-known mechanism for attenuating noise and enhancing robustness in biological systems. When coupled with cell-cell signaling, it provides a strategy for achieving population-level control in multicellular systems. While cell-cell signaling alone tends to reduce cell-to-cell variability by averaging fluctuations across cells, its interplay with negative feedback can produce contrasting effects on noise regulation at single-cell and population levels. Therefore, to design population-level controllers that achieve robust behaviors with attenuated noise, a systematically understanding of how different noise sources impact cell gene expressions at these levels become a critical challenge. Here, we investigate noise regulation in a quorum sensing-based negative feedback system, focusing on two extrinsic noise sources: process noise from target gene dynamics and measurement noise from quorum sensing dynamics. Our results reveal that signal-based negative feedback significantly reduces process noise at the population level, especially for dynamic noise. However, at the single-cell level, it enhances variability, with increased noise levels under faster signal diffusion and higher population density. In contrast, measurement noise is consistently attenuated at both single-cell and population levels through the coupled cell-cell signaling and negative feedback, under conditions of faster diffusion and higher density.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6400c5c874769f8ada1910afa8f5f008c6dc532f" target='_blank'>
              The Interplay Between Cell-cell Signaling and Negative Feedback Reduces Population Noise While Enhancing Single-cell Variability
              </a>
            </td>
          <td>
            Xinying Ren
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The regulation of PKC epsilon (PKCε) and its downstream effects is still not fully understood, making it challenging to develop targeted therapies or interventions. A more precise tool that enables spatiotemporal control of PKCε activity is thus required. Here, we describe a photo-activatable optogenetic PKCε probe (Opto-PKCε) consisting of an engineered PKCε catalytic domain and a blue-light inducible dimerization domain. Molecular dynamics and AlphaFold simulations enable rationalization of the dark-light activity of the optogenetic probe. We first characterize the binding partners of Opto-PKCε, which are similar to those of PKCε. Subsequent validation of the Opto-PKCε tool is performed with phosphoproteome analysis, which reveals that only PKCε substrates are phosphorylated upon light activation. Opto-PKCε could be engineered for recruitment to specific subcellular locations. Activation of Opto-PKCε in isolated hepatocytes reveals its sustained activation at the plasma membrane is required for its phosphorylation of the insulin receptor at Thr1160. In addition, Opto-PKCε recruitment to the mitochondria results in its lowering of the spare respiratory capacity through phosphorylation of complex I NDUFS4. These results demonstrate that Opto-PKCε may have broad applications for the studies of PKCε signaling with high specificity and spatiotemporal resolution. Summary We have developed a photo-activatable optogenetic PKCε probe, which demonstrates differential activity in light versus dark. The tool is subsequently validated with protein association studies and phosphoproteome analysis. It enables dissection of signaling events arising from its activation at defined subcellular locations. For instance, sustained activation of PKCε at the plasma membrane is required for its phosphorylation of the insulin receptor at Thr1160.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dca0f2b39c8a9615b473a16cc7d15c3dce10b1ca" target='_blank'>
              Optogenetic control of Protein Kinase C-epsilon activity reveals its intrinsic signaling properties with spatiotemporal resolution
              </a>
            </td>
          <td>
            Qunxiang Ong, C. Lim, Yilie Liao, Justin Tze-Yang Ng, Ler Ting Rachel Lim, Shernys Xuan Yi Koh, Sher En Chan, Pheobe Lee Yu Ying, Huijun Lim, Chen Rui Ye, Loo Chien Wang, Siok Ghee Ler, Radoslaw M Sobota, Y. S. Tan, Gerald I. Shulman, Xiaoyong Yang, Weiping Han
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="β-arrestins (βarrs) play a crucial role in regulating G protein-coupled receptor (GPCR) signaling and trafficking. Canonically, interactions of βarr with the phosphorylated intracellular GPCR-tail induce a multi-step conformational transition that results in the activation of βarr. Depending on the specific interaction pattern with the receptor, βarrs adopt multiple conformational states, each tightly linked to a specific functional outcome of βarr recruitment. Despite its physiological relevance, the structural determinants of βarr activation remain poorly understood. Using a combination of molecular dynamics simulations, biochemical and cell-based experiments, we reveal how specific interactions with a prototypical GPCR promote the unbinding of the βarr2 C-tail—a crucial step in arrestin activation. Importantly, we observe that the expulsion of the C-tail is promoted by the displacement of a conserved arginine residue (Arg394) within the βarr polar core, which we dub “the arginine switch.” Our study uncovers a previously unknown molecular switch that, upon engagement, destabilizes the polar core as a crucial step in the GPCR-induced βarr activation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fef98b16777c5704c651da391b756cc37914e5e5" target='_blank'>
              An arginine switch drives the stepwise activation of β-arrestin
              </a>
            </td>
          <td>
            Jeongsek Ji, Yaejin Yun, Tomek Stepniewski, Hye-Jin Yoon, Kyungjin Min, Ji Young Park, Chiwoon Chung, Ka Young Chung, J. Selent, Hyung Ho Lee
          </td>
          <td>2025-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="A cell’s global physical state is characterized by its volume and dry mass. The ratio of cell mass to volume is the cell mass density (CMD), which is also a measure of macromolecular crowding and concentrations of all proteins. Using the Fluorescence eXclusion method (FXm) and Quantitative Phase Microscopy (QPM), we investigate CMD dynamics after exposure to sudden media osmolarity change. We find that while the cell volume and mass exhibit complex behavior after osmotic shock, CMD follows a straightforward monotonic recovery in 48 hours. The recovery is cell-cycle independent and relies on a coordinated adjustment of protein synthesis and volume growth rates. Surprisingly, we find that the protein synthesis rate decreases when CMD increases. This result is explained by CMD-dependent nucleoplasm-cytoplasm transport, which serves as negative regulatory feedback on CMD. The Na+/H+ exchanger NHE plays a role in regulating CMD by affecting both protein synthesis and volume change. Taken together, we reveal that cells possess a robust control system that actively regulates CMD during environmental change.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7af39ff469ca36dbd9f6970bfe7f2bf7fb886565" target='_blank'>
              Cells Prioritize the Regulation of Cell Mass Density
              </a>
            </td>
          <td>
            Jinyu Fu, Qin Ni, Yufei Wu, Anoushka Gupta, Zhuoxu Ge, Hongru Yang, Yasin Afrida, Ishan Barman, Sean X. Sun
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Dual-specificity tyrosine phosphorylation-regulated kinase (DYRK) belongs to the CMGC group of kinases that are named after cyclin-dependent kinases (CDKs), mitogen-activated protein kinases (MAPKs), glycogen synthase kinases (GSKs), and CDC-like kinases (CLKs).  DYRK-related kinases comprise a novel subfamily of protein kinases with unique structural, biochemical, and enzymatic features. In humans, DYRKs phosphorylate a set of proteins and play a critical role in apoptosis, DNA damage repair, cell proliferation, survival, and development. Dysregulation of DYRK protein kinases has been associated with cancer biology. In recent years, several studies have reported some kinase inhibitors affecting cancer development and progression, making them potential therapeutic drugs. However, challenges remain in understanding the molecular mechanisms and roles of each member of DYRK family in cancer initiation and progression. In this review, we will highlight the importance of DYRK kinases in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3c66e607bdd7085fb3d920e9a3c3d8681f8cfbd" target='_blank'>
              Dual-specificity tyrosine-regulated kinases (DYRKs) and cancer
              </a>
            </td>
          <td>
            S. A. Moududee, Zongbing You
          </td>
          <td>2024-12-06</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Signaling pathways play key roles in many important biological processes, such as cell division, differentiation, and migration. Phosphorylation site-specific antibodies specifically target proteins phosphorylated on a given tyrosine, threonine, or serine residue. The use of phospho-specific antibodies facilitates the analysis of signaling pathway regulation and activity. Given the usefulness of phospho-specific antibodies, a number of collections of these antibodies have been generated, typically for the detection of phosphorylated mammalian proteins. Anecdotal evidence shows that some of these are also useful for the detection of phosphorylated forms of orthologous proteins in model organisms. We propose that anti-phospho-mammalian protein antibody collections comprise an untapped resource for research in other species. To systematically analyze the potential utility of anti-phospho-mammalian protein antibodies in other species, we developed the Cross-species Epitope Sequence Analysis software tool (CESA). CESA identifies and aligns orthologous proteins in model species and then analyzes the conservation of antibody target sites. We used CESA to predict what phospho-specific antibodies in a collection from Cell Signaling Technology (CST) might be useful for studies in Drosophila melanogaster and other species. CESA predicts that more than 232 sites on 116 Drosophila proteins can potentially be targeted by the antibodies initially developed at CST to detect human, mouse, or rat phosphoproteins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5cf5a5efc46b9d28ff8c1b851d6a96d2e920a4e" target='_blank'>
              Cross-Species Epitope Sequence Analysis for Discovery of Existing Antibodies Useful for Phospho-Specific Protein Detection in Model Species
              </a>
            </td>
          <td>
            Yanhui Hu, Chenxi Gao, William McKenna, Baolong Xia, Majd M. Ariss, S. Mohr, Norbert Perrimon
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="This study investigated possible mechanisms underlying differences between heterophilic and homophilic cadherin adhesions that influence intercellular mechanics and multicellular organization. Results suggest that homophilic cadherin ligation selectively activates force-transduction, such that resulting signaling and mechano-transduction amplitudes are independent of cadherin binding affinities. Epithelial (E-) and neural (N-) cadherin cooperate with distinct growth factors to mechanically activate force-transduction cascades. Prior results demonstrated that E-cadherin and epidermal growth factor receptor form force-sensitive complexes at intercellular junctions. Here, results show that the reconstitution of N-cadherin force transduction required the co-expression of N-cadherin and fibroblast growth factor receptor. Mechanical measurements further demonstrated that homophilic ligation initiates receptor tyrosine kinase-dependent force transduction cascades, but heterophilic cadherin ligands fail to activate signaling or generate stereotypical mechano-transduction signatures. The all-or-nothing contrast between mechano-transduction by heterophilic versus homophilic cadherin adhesions supersedes differences in cadherin adhesion strength. This mechano-selectivity impacts cell spreading and traction generation on cadherin substrates. Homophilic ligation appears to be a key that selectively unlocks cadherin mechano-transduction. These findings may reconcile the roles of cadherin recognition and cell mechanics in the organization of multicellular assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71c3ab047999e3a775e46eac56430142c7d2c1f5" target='_blank'>
              Cadherins and Growth Factor Receptors: ligand selective mechano-switches at cadherin junctions.
              </a>
            </td>
          <td>
            Vinh H. Vu, Brendan Sullivan, Evan Hebner, Zainab Rahil, Yubo Zou, Deborah Leckband
          </td>
          <td>2025-01-16</td>
          <td>Journal of cell science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Intracellular networks process cellular-level information and control cell fate. They can be computationally modeled using Boolean networks, which are implicit-time causal models of discrete binary events. These networks can be embedded in computational agents to drive cellular behavior. To explore this integration, we construct a set of candidate interventions that induce apoptosis in a cell-survival network of a rare leukemia using exhaustive search simulation, stable motif control, and an individual-based mean field approach (IBMFA). Due to inherent algorithmic limitations, these interventions are most suitable for cell-level determinations, not the more realistic multicellular setting. To address these limitations, we treat the target control solutions as putative targets for therapeutic interventions and develop a pipeline to translate them to continuous-time multicellular, agent-based models. We set the discrete-to-continuous transitions between the Boolean network and multicellular model via thresholding and produce simple computational simulations designed to emulate situations in experimental and translational biology. These include a series of simulations: constant substrate gradients, global substrate pulses, and time-varying boundary conditions. We find that interventions that perform equally well in the implicit-time single-cell setting are separable in the multiscale setting in ability to impact population growth and spatial distribution. Further analysis shows that the population and spatial distribution differences arise from differences in internal dynamics (stable motif controls versus target controls) and network distance between the intervention and output nodes. This proof of concept work demonstrates the importance of accounting for internal dynamics in multicellular simulations as well as impacts on understanding of Boolean network control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/537e9b2f3d968280dde541ad2a97539dc9831dd5" target='_blank'>
              Translating and evaluating single-cell Boolean network interventions in the multiscale setting
              </a>
            </td>
          <td>
            John Metzcar, Katie Pletz, Heber L. Rocha, Jordan C Rozum
          </td>
          <td>2025-01-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="G protein-coupled receptors (GPCRs), critical for cellular communication and signaling, represent the largest cell surface protein family and play important roles in numerous pathophysiological processes. Consequently, GPCRs have become a primary focus in drug discovery efforts. Beyond their traditional G protein-dependent signaling pathways, GPCRs are also capable of activating alternative signaling mechanisms, including G protein-independent signaling, biased signaling, and signaling crosstalk. A particularly novel signaling mode employed by these receptors is GPCR transactivation, which enables cross-communication between GPCRs and other receptor types. Intriguingly, GPCR transactivation by distinct GPCRs has also been identified. In this review, I provide an overview of the known GPCR transactivation mechanisms and explore recently uncovered GPCR transactivation mediated by adhesion-class GPCRs (aGPCRs). These aGPCR-GPCR transactivation processes regulate unique cell type-specific functions, offering an exciting opportunity to develop therapies that precisely modulate specific GPCR-mediated biological effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd7640b6c71340efccc23a107fa020552a4d9e72" target='_blank'>
              An Alternative Mode of GPCR Transactivation: Activation of GPCRs by Adhesion GPCRs
              </a>
            </td>
          <td>
            Hsi-Hsien Lin
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Summary Plasma membrane (PM) nanodomains have emerged as pivotal elements in the regulation of plant cellular functions and signal transduction. These nanoscale membrane regions, enriched in specific lipids and proteins, behave as regulatory/signaling hubs spatially and temporally coordinating critical cellular functions. In this review, we first examine the mechanisms underlying the formation and maintenance of PM nanodomains in plant cells, highlighting the roles of PM lipid composition, protein oligomerization and interactions with cytoskeletal and cell wall components. Then, we discuss how nanodomains act as organizing centers by mediating protein–protein interactions that orchestrate essential processes such as symbiosis, defense against pathogens, ion transport or hormonal and reactive oxygen species (ROS) signaling. Finally, we introduce the concept of nanoenvironments, where localized physicochemical variations are generated in the very close proximity of PM nanodomains, in response to stimuli. After decoding by a dedicated machinery likely localized in the vicinity of nanodomains, this enrichment of secondary messengers, such as ROS or Ca2+, would allow specific downstream cellular responses. This review provides insights into the dynamic nature of nanodomains and proposes future research to better understand their contribution to the intricate signaling networks that govern plant development and stress responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e41e337300f09187de33033a21b3dbdff5727fb" target='_blank'>
              Membrane nanodomains to shape plant cellular functions and signaling
              </a>
            </td>
          <td>
            Omar Hdedeh, Caroline Mercier, Arthur Poitout, A. Martinière, Enric Zelazny
          </td>
          <td>2024-12-25</td>
          <td>The New Phytologist</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The Janus kinase-signal transducer and activator of transcription (JAK-STAT) signalling pathway is a key player in animal development and physiology. Although it functions in a variety of processes, the net output of JAK-STAT signalling depends on its spatiotemporal activation, as well as extensive crosstalk with other signalling pathways. Drosophila, with its relatively simple signal transduction pathways and plethora of genetic analysis tools, is an ideal system for dissecting JAK-STAT signalling interactions. In this review, we explore studies in Drosophila revealing that JAK-STAT signalling lies at the nexus of a complex network of interlinked pathways, including epidermal growth factor receptor (EGFR), c-Jun N-terminal kinase (JNK), Notch, Insulin, Hippo, bone morphogenetic protein (BMP), Hedgehog (Hh) and Wingless (Wg). These pathways can synergise with or antagonise one another to produce a variety of outcomes. Given the conserved nature of signal transduction pathways, we conclude with our perspective on the implication of JAK-STAT signalling dysregulation in human diseases, and how studies in Drosophila have the potential to inform and influence clinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9346cda2020e36d154472fdd666cca4f77fbd875" target='_blank'>
              What a tangled web we weave: crosstalk between JAK-STAT and other signalling pathways during development in Drosophila.
              </a>
            </td>
          <td>
            Amartya Mukherjee, Chaithra Anoop, U. Nongthomba
          </td>
          <td>2025-01-16</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="YAP/TAZ (Yes-associated protein/paralog transcriptional co-activator with PDZ-binding domain) are transcriptional cofactors that are the key and major downstream effectors of the Hippo signaling pathway. Both are known to play a crucial role in defining cellular outcomes, including cell differentiation, cell proliferation, and apoptosis. Aside from the canonical Hippo signaling cascade with the key components MST1/2 (mammalian STE20-like kinase 1/2), SAV1 (Salvador homologue 1), MOB1A/B (Mps one binder kinase activator 1A/B) and LATS1/2 (large tumor suppressor kinase 1/2) upstream of YAP/TAZ, YAP/TAZ activation is also influenced by numerous other signaling pathways. Such non-canonical regulation of YAP/TAZ includes well-known growth factor signaling pathways such as the epidermal growth factor receptor (EGFR)/ErbB family, Notch, and Wnt signaling as well as cell-cell adhesion, cell-matrix interactions and mechanical cues from a cell’s microenvironment. This puts YAP/TAZ at the center of a complex signaling network capable of regulating developmental processes and tissue regeneration. On the other hand, dysregulation of YAP/TAZ signaling has been implicated in numerous diseases including various cancers and neurodevelopmental disorders. Indeed, in recent years, parallels between cancer development and neurodevelopmental disorders have become apparent with YAP/TAZ signaling being one of these pathways. This review discusses the role of YAP/TAZ in brain development, cancer and neurodevelopmental disorders with a special focus on the interconnection in the role of YAP/TAZ in these different conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7cb7c32d9c2c91673e28a047a795c76d30bad9b" target='_blank'>
              YAP/TAZ-associated cell signaling – at the crossroads of cancer and neurodevelopmental disorders
              </a>
            </td>
          <td>
            Aderonke O. Ajongbolo, Sigrid A. Langhans
          </td>
          <td>2025-01-28</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The GPCR-like protein Smoothened (Smo) plays a pivotal role in the Hedgehog (Hh) pathway. To initiate Hh signaling, active Smo binds to and inhibits the catalytic subunit of PKA in the primary cilium, a process facilitated by G protein-coupled receptor kinase 2 (Grk2). However, the precise regulatory mechanisms underlying this process, as well as the events preceding and following Smo activation, remain poorly understood. To address this question, we leveraged the proximity labeling tool TurboID and conducted a time-resolved proteomic study of Smo-associated proteins over the course of Hh signaling activation. Our results not only confirmed previously reported Smo interactors but also uncovered new Smo-associated proteins. We characterized one of these new Smo interactors, Grk-interacting protein 1 (Git1), previously known to modulate GPCR signaling. We found that Git1 localizes to the base of the primary cilium, where it controls the cilium transport of Grk2, an early event in Hh signaling. Loss of Git1 impairs Smo phosphorylation by Grk2, a critical step for Smo-PKA interaction, leading to attenuated Hh signaling and reduced cell proliferation in granule neuron precursors. These results revealed a critical regulatory mechanism of Grk2 phosphorylation on Smo in the primary cilium. Our Smo-TurboID proteomic dataset provides a unique resource for investigating Smo regulations across different stages of Hh pathway activation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29c8dcbc43d7a8febed745a5c8130ac386f37304" target='_blank'>
              Proximity based proteomics reveals Git1 as a regulator of Smoothened signaling
              </a>
            </td>
          <td>
            Jingyi Zhang, Gurleen Kaur, Eva Cai, Oscar Torres Gutierrez, Xiaoliang Liu, Sabyasachi Baboo, Jolene K. Diedrich, Ju‐Fen Zhu, Benjamin R. Myers, John R. Yates, Xuecai Ge
          </td>
          <td>2025-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79ee2f17fff89faf152b4a5da94498408a6df77f" target='_blank'>
              Upregulation of the MAP2K4 gene triggers endothelial-mesenchymal transition in COVID-19.
              </a>
            </td>
          <td>
            Esra Yılmaz, Dilek Yilmaz, Can Gokay Yildiz, E. Cacan
          </td>
          <td>2025-01-31</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="T cell receptor signaling must operate reliably under tight time constraints. While assuming quite different mechanisms, two prominent models of T cell receptor activation, kinetic segregation and kinetic proofreading, both introduce a distinct time scale. However, a clear understanding of whether and how those characteristic times give rise to a consistent timing of T cell receptor activation in the presence of stochastic fluctuations has been lacking so far. Here, using a simulation approach capable of modeling molecular interactions between adjacent cell membranes, we explore a stochastic model that combines elements of kinetic segregation and proofreading. Our simulations suggest that the two mechanisms interoperate, thereby rendering the corresponding stochastic times biologically functional. Receptor activation relies on rare molecular events that are not well characterized by the mean of the underlying probability density function. Yet, a consistent timing of receptor activation can be ensured by a modest number of proofreading steps.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e17737963e27d31d61fce5b0a274943bb2601965" target='_blank'>
              Timing consistency of T cell receptor activation in a stochastic model combining kinetic segregation and proofreading
              </a>
            </td>
          <td>
            T. Prüstel, Martin Meier-Schellersheim
          </td>
          <td>2024-12-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Epidermal growth factor (EGF) binds with its surface receptor to stimulate gene expression and cancer cell proliferation. EGF stimulates cancer cell growth via phosphoinositide 3‑kinase (PI3K) and programmed cell death ligand 1 (PD‑L1) pathways. As an integrin αvβ3 antagonist, heteronemin exhibits potent cytotoxic effects against cancer cells. It inhibits critical signal transduction pathways promoted by the EGF. In the current study, EGF‑induced signal activation and proliferative effects were investigated in cholangiocarcinoma cells and its molecular targets using qPCR and western blotting analyses. In addition, cell viability assays were performed to assess the growth effects of EGF and heteronemin. Heteronemin reversed the effects of EGF and was further enhanced by blockage of PI3K's activity. In summary, EGF stimulates cholangiocarcinoma cell growth. On the other hand, heteronemin inhibited PI3K activation and PD‑L1 expression to reverse the stimulative effects of EGF‑induced gene expression and proliferation in cholangiocarcinoma cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e828f2b4a1b2fa7921839ab4f28d7fb594b973e" target='_blank'>
              Heteronemin suppresses EGF‑induced proliferation through the PI3K/PD‑L1 signaling pathways in cholangiocarcinoma.
              </a>
            </td>
          <td>
            Y. S. Yang, Chung-Che Tsai, Yung-Ning Yang, Feng-Cheng Liu, Sheng Y. Lee, Jen-Chang Yang, Dana R Crawford, Hsien-Chung Chiu, Mei-Chin Lu, Zian Li, Yi-Chen Chen, Tin-Yi Chu, Jacqueline Whang-Peng, Hung-Yun Lin, Kuan Wang
          </td>
          <td>2025-01-09</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>19</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [1, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>