<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Ac952ba3508fc13b6ca5e8d7e11d7a7086f9b72d
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-21 10:41:53 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>A quantitative systems pharmacology model of the pathophysiology and treatment of COVID-19 predicts optimal timing of pharmacological interventions</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="Significance A distinctive feature of COVID-19 is its extreme heterogeneity—illness ranges from minimally symptomatic to life threatening. Heterogeneity results from a poorly understood combination of patient factors, viral dynamics, antiviral and immune modulating therapies, and dynamics of the innate and adaptive immune responses. In order to better understand clinical heterogeneity and optimal treatment, we developed a comprehensive mathematical model incorporating elements of the innate and adaptive immune responses, the renin−angiotensin system (which the virus exploits for cellular entry), rates of viral replication, inflammatory cytokines, and the coagulation cascade. Our model reveals divergent treatment responses and clinical outcomes as a function of comorbidities, age, and details of the innate and adaptive immune responses which can provide a framework for understanding individual patients’ trajectories. Understanding the underlying mechanisms of COVID-19 progression and the impact of various pharmaceutical interventions is crucial for the clinical management of the disease. We developed a comprehensive mathematical framework based on the known mechanisms of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection, incorporating the renin−angiotensin system and ACE2, which the virus exploits for cellular entry, key elements of the innate and adaptive immune responses, the role of inflammatory cytokines, and the coagulation cascade for thrombus formation. The model predicts the evolution of viral load, immune cells, cytokines, thrombosis, and oxygen saturation based on patient baseline condition and the presence of comorbidities. Model predictions were validated with clinical data from healthy people and COVID-19 patients, and the results were used to gain insight into identified risk factors of disease progression including older age; comorbidities such as obesity, diabetes, and hypertension; and dysregulated immune response. We then simulated treatment with various drug classes to identify optimal therapeutic protocols. We found that the outcome of any treatment depends on the sustained response rate of activated CD8+ T cells and sufficient control of the innate immune response. Furthermore, the best treatment—or combination of treatments—depends on the preinfection health status of the patient. Our mathematical framework provides important insight into SARS-CoV-2 pathogenesis and could be used as the basis for personalized, optimal management of COVID-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e26745913090baa9c6bf3b311f751b5c6602d112" target='_blank'>In silico dynamics of COVID-19 phenotypes for optimizing clinical management</a></td>
          <td>
            C. Voutouri, M. Nikmaneshi, C. Hardin, A. Patel, A. Verma, M. Khandekar, S. Dutta, T. Stylianopoulos, L. Munn, R. Jain
          </td>
          <td>2021-01-05</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>23</td>
          <td>76</td>
        </tr>

        <tr id="The current, rapidly diversifying pandemic has accelerated the need for efficient and effective identification of potential drug candidates for COVID-19. Knowledge on host-immune response to SARS-CoV-2 infection, however, remains limited with very few drugs approved to date. Viable strategies and tools are rapidly arising to address this, especially with repurposing of existing drugs offering significant promise. Here we introduce a systems biology tool, the PHENotype SIMulator, which – by leveraging available transcriptomic and proteomic databases – allows modeling of SARS-CoV-2 infection in host cells in silico to i) determine with high sensitivity and specificity (both>96%) the viral effects on cellular host-immune response, resulting in a specific cellular SARS-CoV-2 signature and ii) utilize this specific signature to narrow down promising repurposable therapeutic strategies. Powered by this tool, coupled with domain expertise, we have identified several potential COVID-19 drugs including methylprednisolone and metformin, and further discern key cellular SARS-CoV-2-affected pathways as potential new drugable targets in COVID-19 pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25cdce26ef41c74f981cbcf9a8ba01790bfd4f28" target='_blank'>Rapid Identification of Druggable Targets and the Power of the PHENotype SIMulator for Effective Drug Repurposing in COVID-19</a></td>
          <td>
            Naomi I Maria, Rosaria Valentina Rapicavoli, S. Alaimo, E. Bischof, Alessia Stasuzzo, J. Broek, A. Pulvirenti, B. Mishra, A. Duits, A. Ferro
          </td>
          <td>2021-04-14</td>
          <td>Research Square</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="Undefirstanding the underlying mechanisms of COVID-19 progression and the impact of various pharmaceutical interventions is crucial for the clinical management of the disease. We developed a comprehensive mathematical framework based on the known mechanisms of the SARS-CoV-2 virus infection, incorporating the renin-angiotensin system and ACE2, which the virus exploits for cellular entry, key elements of the innate and adaptive immune responses, the role of inflammatory cytokines and the coagulation cascade for thrombus formation. The model predicts the evolution of viral load, immune cells, cytokines, thrombosis, and oxygen saturation based on patient baseline condition and the presence of co-morbidities. Model predictions were validated with clinical data from healthy people and COVID-19 patients, and the results were used to gain insight into identified risk factors of disease progression including older age, co-morbidities such as obesity, diabetes, and hypertension, and dysregulated immune response1,2. We then simulated treatment with various drug classes to identify optimal therapeutic protocols. We found that the outcome of any treatment depends on the sustained response rate of activated CD8+ T cells and sufficient control of the innate immune response. Furthermore, the best treatment –or combination of treatments – depends on the pre-infection health status of the patient. Our mathematical framework provides important insight into SARS-CoV-2 pathogenesis and could be used as the basis for personalized, optimal management of COVID-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ededcf0e8206d71722e7c02d229b6ec876920251" target='_blank'>In silico dynamics of COVID-19 phenotypes for optimizing clinical management</a></td>
          <td>
            C. Voutouri, M. Nikmaneshi, C. Hardin, A. Patel, A. Verma, M. Khandekar, S. Dutta, T. Stylianopoulos, L. Munn, R. Jain
          </td>
          <td>2020-09-03</td>
          <td>Research Square</td>
          <td>21</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de6603feaed05335c7a88ee1a1994af49f78226a" target='_blank'>Application of the PHENotype SIMulator for rapid identification of potential candidates in effective COVID-19 drug repurposing</a></td>
          <td>
            N. Maria, Rosaria Valentina Rapicavoli, S. Alaimo, E. Bischof, Alessia Stasuzzo, J. Broek, A. Pulvirenti, Bud Mishra, A. Duits, A. Ferro
          </td>
          <td>2023-03-01</td>
          <td>Heliyon</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdba71c8198592733c2f3c0e94873428e96682cb" target='_blank'>Advancing therapies for viral infections using mechanistic computational models of the dynamic interplay between the virus and host immune response</a></td>
          <td>
            V. Zarnitsyna, J. F. Gianlupi, A. Hagar, T. Sego, J. Glazier
          </td>
          <td>2021-08-24</td>
          <td>Current Opinion in Virology</td>
          <td>7</td>
          <td>51</td>
        </tr>

        <tr id="Emerging diseases—and none as recently or devastatingly impactful toward humans as COVID-19—pose an immense challenge to researchers concerned with infectious disease. This study is tasked with expanding the computational probe of treatment regimes in a differential equations-based model of the SARS-CoV-2 host–virus interaction. Parameters within the model are tweaked to simulate dose specifications. Further, parametric variations are introduced in a timed manner to infer the importance of dose timing. Arming in silico testing, and eventually, clinical testing, with abundant information on simulated therapeutic regimes is the overall contribution of this pharmacodynamic model; thus, a wide range of dose and timing combinations are examined. Therapeutic interventions that block viral replication inhibit viral entry into host cells, and vaccination-induced antibodies are all studied alone and in combination. Especially during early detection, exhaustive parameter sweeps of well-suited within-host models are often the first step in the clinical response to a novel disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d60b4a93424cfd625c87dd3301212a8d4e25d54" target='_blank'>A Computational Framework for Exploring SARS-CoV-2 Pharmacodynamic Dose and Timing Regimes</a></td>
          <td>
            J. Pateras, Preetam Ghosh
          </td>
          <td>2022-10-11</td>
          <td>Mathematics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/169d884ca56ee95bcaa837d0ac317dfb34294c23" target='_blank'>Executable network of SARS-CoV-2-host interaction predicts drug combination treatments</a></td>
          <td>
            Rowan S M Howell, Matthew A. Clarke, Ann-Kathrin Reuschl, Tianyi Chen, Sean Abbott-Imboden, M. Singer, D. Lowe, Clare L. Bennett, B. Chain, C. Jolly, J. Fisher
          </td>
          <td>2021-07-27</td>
          <td>NPJ Digital Medicine</td>
          <td>9</td>
          <td>93</td>
        </tr>

        <tr id="The COVID-19 pandemic significantly transformed the field of mathematical modeling in immunology. International collaboration among numerous research groups yielded a substantial amount of experimental data, which greatly facilitated model validation and led to the development of new mathematical models. The aim of the study is an improvement of system understanding of the immune response to SARS-CoV-2 infection based on the development of a modular mathematical model which provides a foundation for further research on host-pathogen interactions. We utilized the open-source BioUML platform to develop a model using ordinary, delay and stochastic differential equations. The model was validated using experimental data from middle-aged individuals with moderate COVID-19 progression, including measurements of viral load, antibodies, CD4+ and CD8+ T cells, and interleukin-6 levels. Parameter optimization and sensitivity analysis were conducted to refine the model’s accuracy. The model effectively reproduces moderate, severe, and critical COVID-19 progressions, consistent with experimental observations. We investigated the efficiency and contributions of innate and adaptive immunity in response to SARS-CoV-2 infection and assessed immune system behavior during co-infection with HIV and organ transplantation. Additionally, we studied therapy methods, such as interferon administration. The developed model can be employed as a framework for simulating other infectious diseases taking into account follow-up immune response. Author summary Despite the significant progress reached in understanding of COVID-19, traditional methods still struggle to analyze and interpret the extensive and sometimes controversial experimental data on SARS-CoV-2 infection. Mathematical and systems biology approaches attempt to address this challenge by developing mathematical models of the immune response. We aimed not only to investigate the disease at a systemic level but also to provide a framework for further research on host-pathogen interactions, both existing and forthcoming. To achieve this, we constructed a model incorporating both innate and adaptive immunity, as well as cellular and humoral components. This together allowed us to conduct a series of in silico experiments, exploring the immune response across various levels and compartments. The results of these investigations offer valuable insights into the complex dynamics of the immune system and can guide future research and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fba63039c2028ccefcb6ba1155b931e933bffd6f" target='_blank'>A modular model of immune response as a computational platform to investigate a pathogenesis of infection disease</a></td>
          <td>
            M.I. Miroshnichenko, F. Kolpakov, I. Akberdin
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Computational modeling is an effective tool for studying complex disease. However, solutions to many models are purely mathematical and cannot immediately provide clinical insights. To overcome this barrier, we propose a series of quantitative scoring metrics that can be used in combination with drug-target interaction data to identify solutions that are readily clinically actionable. Furthermore, we introduce methods for the prediction and ranking of pharmaceutical interventions that closely align with these high-scoring solutions, with an emphasis on robustness across multiple solutions. We demonstrate these methods on a previously-described model of COVID-19 induced cytokine storm. These scoring methods ultimately identify multiple pharmaceutical candidates that have been shown to be effective in reducing mortality rates in COVID-19 patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c667e5eae3273dda154ffc7cf714140d62ee891f" target='_blank'>Leveraging Drug-Target Interaction Data for the Translation of Computational Models into Clinically Actionable Interventions</a></td>
          <td>
            Spencer C. Richmanz, Cole A. Lyman, M. Morris, Hongbao Cao, Anastasia Nesterovay, Anton Yuryevy, C. Cheadle, G. Broderick
          </td>
          <td>2021-12-09</td>
          <td>2021 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Rare diseases, affecting millions globally, present significant drug development challenges. This is due to the limited patient populations and the unique pathophysiology of these diseases, which can make traditional clinical trial designs unfeasible. Quantitative Systems Pharmacology (QSP) models offer a promising approach to expedite drug development, particularly in rare diseases. QSP models provide a mechanistic representation of the disease and drug response in virtual patients that can complement routinely applied empirical modeling and simulation approaches. QSP models can generate digital twins of actual patients and mechanistically simulate the disease progression of rare diseases, accounting for phenotypic heterogeneity. QSP models can also support drug development in various drug modalities, such as gene therapy. Impactful QSP models case studies are presented here to illustrate their value in supporting various aspects of drug development in rare indications. As these QSP model applications continue to mature, there is a growing possibility that they could be more widely integrated into routine drug development steps. This integration could provide a robust framework for addressing some of the inherent challenges in rare disease drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/501aa60f69949c250c5052e0be685b26e600d873" target='_blank'>Quantitative Systems Pharmacology Models: Potential Tools for Advancing Drug Development for Rare Diseases.</a></td>
          <td>
            Susana Neves-Zaph, C. Kaddi
          </td>
          <td>2024-09-28</td>
          <td>Clinical pharmacology and therapeutics</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaf4d9b3ef75711609791b736ba048ab52657727" target='_blank'>Mechanistic modeling of the SARS-CoV-2 disease map</a></td>
          <td>
            Kinza Rian, M. Esteban-Medina, Marta R. Hidalgo, C. Çubuk, M. M. Falco, C. Loucera, Devrim Gunyel, M. Ostaszewski, M. Peña-Chilet, J. Dopazo
          </td>
          <td>2020-04-12</td>
          <td>BioData Mining</td>
          <td>18</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfaa41dbd4992987d3c637f8ddc9d354ff98ad9e" target='_blank'>Computational drug repurposing in the age of COVID-19: mixing antiviral cocktails in silico</a></td>
          <td>
            James A. Diao, Marium M. Raza, Kaushik P. Venkatesh, J. Kvedar
          </td>
          <td>2022-04-22</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35ef433ad73bb3c9a619744dabe0baad0e3e6ab" target='_blank'>Therapeutics for COVID-19: from computation to practices—where we are, where we are heading to</a></td>
          <td>
            P. Ojha, S. Kar, J. Krishna, K. Roy, J. Leszczynski
          </td>
          <td>2020-09-02</td>
          <td>Molecular Diversity</td>
          <td>49</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35ef433ad73bb3c9a619744dabe0baad0e3e6ab" target='_blank'>Therapeutics for COVID-19: from computation to practices—where we are, where we are heading to</a></td>
          <td>
            P. Ojha, S. Kar, J. Krishna, K. Roy, J. Leszczynski
          </td>
          <td>2020-09-02</td>
          <td>Molecular Diversity</td>
          <td>49</td>
          <td>77</td>
        </tr>

        <tr id="Seriously ill patients infected with severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) and hospitalized in intensive care units (ICUs) are commonly given a combination of drugs, a process known as multi‐drug treatment. After extracting data on drug–drug interactions with clinical relevance from available online platforms, we hypothesize that an overall interaction map can be generated for all drugs administered. Furthermore, by combining this approach with simulations of cellular biochemical pathways, we may be able to explain the general clinical outcome. Finally, we postulate that by applying this strategy retrospectively to a cohort of patients hospitalized in ICU, a prediction of the timing of developing acute kidney injury (AKI) could be made. Whether or not this approach can be extended to other diseases is uncertain. Still, we believe it represents a valuable pharmacological insight to help improve clinical outcomes for severely ill patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a72166c17973a2720876862242c821a8d36a54bb" target='_blank'>How far are we from predicting multi‐drug interactions during treatment for COVID‐19 infection?</a></td>
          <td>
            Benjamin Lozano, J. Santibáñez, Nicolás Severino, C. Saldias, M. Vera, J. Retamal, Soledad Torres, Nelson P. Barrera
          </td>
          <td>2022-07-01</td>
          <td>British Journal of Pharmacology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9831894a2a2f7afb3d16b198fdf79df52f396f09" target='_blank'>Optimizing antiviral therapy for COVID-19 with learned pathogenic model</a></td>
          <td>
            Abhishek Dutta
          </td>
          <td>2021-09-21</td>
          <td>Scientific Reports</td>
          <td>8</td>
          <td>3</td>
        </tr>

        <tr id="mrgsolve is an open‐source R package available on the Comprehensive R Archive Network. It combines R and C++ coding for simulation from hierarchical, ordinary differential equation–based models. Its efficient simulation engine and integration into a parallelizable, R‐based workflow makes mrgsolve a convenient tool both for simple and complex models and thus is ideal for physiologically‐based pharmacokinetic (PBPK) and quantitative systems pharmacology (QSP) model. This tutorial will first introduce the basics of the mrgsolve simulation workflow, including model specification, the introduction of interventions (dosing events) into the simulation, and simulated results postprocessing. An applied simulation example is then presented using a PBPK model for voriconazole, including a model validation step against adult and pediatric data sets. A final simulation example is then presented using a previously published QSP model for mitogen‐activated protein kinase signaling in colorectal cancer, illustrating population simulation of different combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f58af03b1918b56e424198aaef83d3aa5da5080" target='_blank'>Quantitative Systems Pharmacology and Physiologically‐Based Pharmacokinetic Modeling With mrgsolve: A Hands‐On Tutorial</a></td>
          <td>
            Ahmed Elmokadem, M. Riggs, K. Baron
          </td>
          <td>2019-11-14</td>
          <td>CPT: Pharmacometrics & Systems Pharmacology</td>
          <td>52</td>
          <td>13</td>
        </tr>

        <tr id="COVID-19 vaccines have been instrumental tools in reducing the impact of SARS-CoV-2 infections around the world by preventing 80% to 90% of hospitalizations and deaths from reinfection, in addition to preventing 40% to 65% of symptomatic illnesses. However, the simultaneous large-scale vaccination of the global population will indubitably unveil heterogeneity in immune responses as well as in the propensity to developing post-vaccine adverse events, especially in vulnerable individuals. Herein, we applied a systems biology workflow, integrating vaccine transcriptional signatures with chemogenomics, to study the pharmacological effects of mRNA vaccines. First, we derived transcriptional signatures and predicted their biological effects using pathway enrichment and network approaches. Second, we queried the Connectivity Map (CMap) to prioritize adverse events hypotheses. Finally, we accepted higher-confidence hypotheses that have been predicted by independent approaches. Our results reveal that the mRNA-based BNT162b2 vaccine affects immune response pathways related to interferon and cytokine signaling, which should lead to vaccine success, but may also result in some adverse events. Our results emphasize the effects of BNT162b2 on calcium homeostasis, which could be contributing to some frequently encountered adverse events related to mRNA vaccines. Notably, cardiac side effects were signaled in the CMap query results. In summary, our approach has identified mechanisms underlying both the expected protective effects of vaccination as well as possible post-vaccine adverse effects. Our study illustrates the power of systems biology approaches in improving our understanding of the comprehensive biological response to vaccination against COVID-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c652900a53bbc0d23326e5456aad640e30b1afd2" target='_blank'>Analyzing the Systems Biology Effects of COVID-19 mRNA Vaccines to Assess Their Safety and Putative Side Effects</a></td>
          <td>
            R. Hajjo, D. Sabbah, A. Tropsha
          </td>
          <td>2022-06-29</td>
          <td>Pathogens</td>
          <td>13</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6390a7198e07ebd6231e4059b4f39210c8417535" target='_blank'>Quantitative Systems Pharmacology models as a key to translational medicine</a></td>
          <td>
            B. Schoeberl
          </td>
          <td>2019-08-01</td>
          <td>Current Opinion in Systems Biology</td>
          <td>4</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f961e824126c57712d91a25515e980c44c44a69" target='_blank'>Drug repurposing for COVID-19 using machine learning and mechanistic models of signal transduction circuits related to SARS-CoV-2 infection</a></td>
          <td>
            C. Loucera, M. Esteban-Medina, Kinza Rian, M. M. Falco, J. Dopazo, M. Peña-Chilet
          </td>
          <td>2020-12-01</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>42</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9a266c73dc6cd9686d43728a2480c3881d75c3c" target='_blank'>Computational repurposing of drugs for viral diseases and current and future pandemics</a></td>
          <td>
            David A. Winkler
          </td>
          <td>2024-02-05</td>
          <td>Journal of Mathematical Chemistry</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Quantitative systems pharmacology (QSP) has emerged as an innovative approach in model‐informed drug discovery and development, supporting program decisions from exploratory research through late‐stage clinical trials. In this commentary, we discuss the unique value of disease‐scale “platform” QSP models that are amenable to reuse and repurposing to support diverse clinical decisions in ways distinct from other pharmacometrics strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75fd303e477a32d2844dd2c8a67fced699191fee" target='_blank'>Quantitative Systems Pharmacology: A Case for Disease Models</a></td>
          <td>
            C. Musante, S. Ramanujan, B. J. Schmidt, OG Ghobrial, James Lu, A. Heatherington
          </td>
          <td>2016-11-18</td>
          <td>Clinical Pharmacology and Therapeutics</td>
          <td>50</td>
          <td>23</td>
        </tr>

        <tr id="The new Coronavirus, (A.K.A. COVID-19) created a global crisis that impacted the world not only in health perspectives but in most aspects of life. Many teams across the globe from difference, science, and health domains continue as of the time of writing this paper to find proper treatments for COVID-19. In this scope, our effort in this paper is to follow a data-analytic approach and summarize candidate COVID-19 treatments, their mechanism of action and side effects, and research progress in this regard. In addition, we provide a mathematical model to measure the transmission of the disease. We also conducted a comparison analysis between those candidate treatments. Our analysis indicated that among the most popular terms in research articles and social networks, there is a mixture between the different drug treatments, versus their classifications and commercial names. While research publications may reflect states, public, and private sector efforts on the different alternatives of treatments, popularity in social networks are less structured and may combine a mixture of treatments, their categories, and also manufacturing and marketing names. Many of COVID-19 treatments evolve from earlier Coronavirus treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e78d500bb5d5819d0512fa3996c44269a46c533f" target='_blank'>Data Analytics and Assessment of the Drugs for COVID-19</a></td>
          <td>
            Tanima Bose, I. Alsmadi, M. Al-Ramahi, Ummugul Bulut
          </td>
          <td>2020-09-09</td>
          <td></td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Before the availability of vaccines, many countries have resorted multiple times to drastic social restrictions to prevent saturation of their health care system, and to regain control over an otherwise exponentially increasing COVID-19 pandemic. With the advent of data-sharing, computational approaches are key to efficiently control a pandemic with non-pharmaceutical interventions (NPIs). Here we develop a data-driven computational framework based on a time discrete and age-stratified compartmental model to control a pandemic evolution inside and outside hospitals in a constantly changing environment with NPIs. Besides the calendrical time, we introduce a second time-scale for the infection history, which allows for non-exponential transition probabilities. We develop inference methods and feedback procedures to successively recalibrate model parameters as new data becomes available. As a showcase, we calibrate the framework to study the pandemic evolution inside and outside hospitals in France until February 2021. We combine national hospitalization statistics from governmental websites with clinical data from a single hospital to calibrate hospitalization parameters. We infer changes in social contact matrices as a function of NPIs from positive testing and new hospitalization data. We use simulations to infer hidden pandemic properties such as the fraction of infected population, the hospitalisation probability, or the infection fatality ratio. We show how reproduction numbers and herd immunity levels depend on the underlying social dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1c9b79b129944891585ee1cec63263d3e9682fa" target='_blank'>Data-driven multiscale dynamical framework to control a pandemic evolution with non-pharmaceutical interventions</a></td>
          <td>
            J. Reingruber, A. Papale, S. Ruckly, J. Timsit, D. Holcman
          </td>
          <td>2023-01-17</td>
          <td>PLOS ONE</td>
          <td>3</td>
          <td>105</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c35ba611bd27ca4269fdfb7a8e2e63fa6faba431" target='_blank'>Chapter 18 In silico disease modeling for COVID-19</a></td>
          <td>
            E. Yiannakopoulou
          </td>
          <td>2023-12-31</td>
          <td>Omics Approaches and Technologies in COVID-19</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="BACKGROUND
Microbial diseases, specifically originating from viruses are the major cause of human mortality all over the world. The current COVID-19 pandemic is a case in point, where the dynamics of the viral-human interactions are still not completely understood, making its treatment a case of trial and error. Scientists are struggling to devise a strategy to contain the pandemic for over a year and this brings to light the lack of understanding of how the virus grows and multiplies in the human body.


METHODS
This paper presents the perspective of the authors on the applicability of computational tools for deep learning and understanding of host-microbe interaction, disease progression and management, drug resistance and immune modulation through in-silico methodologies which can aid in effective and selective drug development. The paper has summarized advances in the last five years. The studies published and indexed in leading databases were included in the review.


RESULTS
Computational systems biology works on an interface of biology and mathematics and intends to unravel the complex mechanisms plying between the biological systems and the inter and intra species dynamics using computational tools, and high-throughput technologies developed on algorithms, networks and complex connections to simulate cellular biological processes.


CONCLUSION
Computational strategies and modelling integrate and prioritize microbial-host interactions and may predict the conditions in which the fine-tuning attenuates. These microbial-host interactions and working mechanisms are important from the aspect of effective drug designing and fine-tuning the therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d2b4b9fd28b57e5a8206f26883d2873eca8e8f9" target='_blank'>Exploration of Computational Aids for Effective Drug Designing and Management of Viral Diseases: A Comprehensive Review.</a></td>
          <td>
            Tahmeena Khan, S. Raza
          </td>
          <td>2023-02-01</td>
          <td>Current topics in medicinal chemistry</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Quantitative systems pharmacology (QSP) places an emphasis on dynamic systems modeling, incorporating considerations from systems biology modeling and pharmacodynamics. The goal of QSP is often to quantitatively predict the effects of clinical therapeutics, their combinations, and their doses on clinical biomarkers and endpoints. In order to achieve this goal, strategies for incorporating clinical data into model calibration are critical. Virtual population (VPop) approaches facilitate model calibration while faced with challenges encountered in QSP model application, including modeling a breadth of clinical therapies, biomarkers, endpoints, utilizing data of varying structure and source, capturing observed clinical variability, and simulating with models that may require more substantial computational time and resources than often found in pharmacometrics applications. VPops are frequently developed in a process that may involve parameterization of isolated pathway models, integration into a larger QSP model, incorporation of clinical data, calibration, and quantitative validation that the model with the accompanying, calibrated VPop is suitable to address the intended question or help with the intended decision. Here, we introduce previous strategies for developing VPops in the context of a variety of therapeutic and safety areas: metabolic disorders, drug-induced liver injury, autoimmune diseases, and cancer. We introduce methodological considerations, prior work for sensitivity analysis and VPop algorithm design, and potential areas for future advancement. Finally, we give a more detailed application example of a VPop calibration algorithm that illustrates recent progress and many of the methodological considerations. In conclusion, although methodologies have varied, VPop strategies have been successfully applied to give valid clinical insights and predictions with the assistance of carefully defined and designed calibration and validation strategies. While a uniform VPop approach for all potential QSP applications may be challenging given the heterogeneity in use considerations, we anticipate continued innovation will help to drive VPop application for more challenging cases of greater scale while developing new rigorous methodologies and metrics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68337aa80c1390201d9ea07d9ae681ebb37bfaa5" target='_blank'>Virtual Populations for Quantitative Systems Pharmacology Models.</a></td>
          <td>
            Yougan Cheng, R. Straube, Abed Alnaif, Lu Huang, T. Leil, B. Schmidt
          </td>
          <td>2021-08-24</td>
          <td>Methods in molecular biology</td>
          <td>22</td>
          <td>23</td>
        </tr>

        <tr id="Quantitative systems pharmacology (QSP) modeling has become an increasingly popular approach impacting our understanding of disease mechanisms and helping predict patients’ treatment responses to facilitate study design or development go/no-go decisions. In this paper, we highlight the notable contributions and opportunities that QSP approaches are to offer during the drug development process by sharing three examples that have facilitated internal decisions. The barriers to successful applications and the factors that facilitate the success of the modeling approach is discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15595407c283a7e6ca03ee082cb43d7d405e8328" target='_blank'>Quantitative systems modeling approaches towards model-informed drug development: Perspective through case studies</a></td>
          <td>
            Meghna Verma, Louis Gall, J. Biasetti, G. D. Di Veroli, C. Pichardo-Almarza, M. Gibbs, Holly Kimko
          </td>
          <td>2023-01-12</td>
          <td>None</td>
          <td>5</td>
          <td>21</td>
        </tr>

        <tr id="We propose the use of in silico mathematical models to provide insights that optimize therapeutic interventions designed to effectively treat respiratory infection during a pandemic. A modelling and simulation framework is provided using SARS‐CoV‐2 as an example, considering applications for both treatment and prophylaxis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7d17c85eeb47bb2c1cbb8821a767dc1f217518d" target='_blank'>Using in silico viral kinetic models to guide therapeutic strategies during a pandemic: An example in SARS‐CoV‐2</a></td>
          <td>
            Kashyap Patel, Michael G. Dodds, António Gonçalves, M. Kamal, C. Rayner, C. Kirkpatrick, Patrick F. Smith
          </td>
          <td>2020-09-10</td>
          <td>British Journal of Clinical Pharmacology</td>
          <td>6</td>
          <td>43</td>
        </tr>

        <tr id="Background The development and approval of novel drugs are typically time-intensive and expensive. Leveraging a computational drug repurposing framework that integrates disease-relevant genetically regulated gene expression (GReX) and large longitudinal electronic medical record (EMR) databases can expedite the repositioning of existing medications. However, validating computational predictions of the drug repurposing framework remains a challenge. Methods To benchmark the drug repurposing framework, we first performed a 5-method-rank-based computational drug prioritization pipeline by integrating multi-tissue GReX associated with COVID-19-related hospitalization, with drug transcriptional signature libraries from the Library of Integrated Network-Based Cellular Signatures. We prioritized FDA-approved medications from the 10 top-ranked compounds, and assessed their association with COVID-19 incidence within the Veterans Health Administration (VHA) cohort (~9 million individuals). In parallel, we evaluated in vitro SARS-CoV-2 replication inhibition in human lung epithelial cells for the selected candidates. Results Our in silico pipeline identified seven FDA-approved drugs among the top ten candidates. Six (imiquimod, nelfinavir and saquinavir, everolimus, azathioprine, and retinol) had sufficient prescribing rates or feasibility for further testing. In the VHA cohort, azathioprine (odds ratio [OR]=0.69, 95% CI 0.62-0.77) and retinol (OR=0.81, 95% CI 0.72-0.92) were significantly associated with reduced COVID-19 incidence. Conversely, nelfinavir and saquinavir demonstrated potent SARS-CoV-2 inhibition in vitro (~95% and ~65% viral load reduction, respectively). No single compound showed robust protection in both in vivo and in vitro settings. Conclusions These findings underscore the power of GReX-based drug repurposing in rapidly identifying existing therapies with potential clinical relevance; four out of six compounds showed a protective effect in one of the two validation approaches. Crucially, our results highlight how a complementary evaluation-combining epidemiological data and in vitro assays-helps refine the most promising candidates for subsequent mechanistic studies and clinical trials. This integrated validation approach may prove vital for accelerating therapeutic development against current and future health challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd1b6ef90b2615a2aaf5ec0fd73e960bcd6dd48c" target='_blank'>A genetically based computational drug repurposing framework for rapid identification of candidate compounds: application to COVID-19</a></td>
          <td>
            G. Voloudakis, K. M. Lee, J. Vicari, W. Zhang, D. Hoagland, S. Venkatesh, J. Bian, M. Anyfantakis, Z. Wu, S. Rahman, L. Gao, , J. S. Lee, S. K. Iyengar, S.-W. Luoh, T. L. Assimes, G. Hoffman, B. R. tenOever, J. Fullard, J. A. Lynch, P. Roussos
          </td>
          <td>2025-01-14</td>
          <td>None</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e169fcd3d0b5df283ef2703057590403120e244b" target='_blank'>Natural experiment concept to accelerate the Re-purposing of existing therapeutics for Covid-19</a></td>
          <td>
            C. Daughton
          </td>
          <td>2020-05-15</td>
          <td>Global Epidemiology</td>
          <td>4</td>
          <td>36</td>
        </tr>

        <tr id="Introduction Epidemiological modeling is widely used to offer insights into the COVID-19 pandemic situation in Asia. We reviewed published computational (mathematical/simulation) models conducted in Asia that assessed impacts of pharmacological and non-pharmacological interventions against COVID-19 and their implications for vaccination strategy. Methods A search of the PubMed database for peer-reviewed, published, and accessible articles in English was performed up to November 2022 to capture studies in Asian populations based on computational modeling of outcomes in the COVID-19 pandemic. Extracted data included model type (mechanistic compartmental/agent-based, statistical, both), intervention type (pharmacological, non-pharmacological), and procedures for parameterizing age. Findings are summarized with descriptive statistics and discussed in terms of the evolving COVID-19 situation. Results The literature search identified 378 results, of which 59 met criteria for data extraction. China, Japan, and South Korea accounted for approximately half of studies, with fewer from South and South-East Asia. Mechanistic models were most common, either compartmental (61.0%), agent-based (1.7%), or combination (18.6%) models. Statistical modeling was applied less frequently (11.9%). Pharmacological interventions were examined in 59.3% of studies, and most considered vaccination, except one study of an antiviral treatment. Non-pharmacological interventions were also considered in 84.7% of studies. Infection, hospitalization, and mortality were outcomes in 91.5%, 30.5%, and 30.5% of studies, respectively. Approximately a third of studies accounted for age, including 10 that also examined mortality. Four of these studies emphasized benefits in terms of mortality from prioritizing older adults for vaccination under conditions of a limited supply; however, one study noted potential benefits to infection rates from early vaccination of younger adults. Few studies (5.1%) considered the impact of vaccination among children. Conclusion Early in the COVID-19 pandemic, non-pharmacological interventions helped to mitigate the health burden of COVID-19; however, modeling indicates that high population coverage of effective vaccines will complement and reduce reliance on such interventions. Thus, increasing and maintaining immunity levels in populations through regular booster shots, particularly among at-risk and vulnerable groups, including older adults, might help to protect public health. Future modeling efforts should consider new vaccines and alternative therapies alongside an evolving virus in populations with varied vaccination histories.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279bb918c32726b40663c1600d5a7f05ac8483b9" target='_blank'>Impact of vaccination and non-pharmacological interventions on COVID-19: a review of simulation modeling studies in Asia</a></td>
          <td>
            Karan Thakkar, J. Spinardi, Jingyan Yang, M. Kyaw, Egemen Ozbilgili, C. F. Mendoza, H. Oh
          </td>
          <td>2023-09-25</td>
          <td>Frontiers in Public Health</td>
          <td>5</td>
          <td>32</td>
        </tr>

        <tr id="leveraged Covid-19 portal to develop machine learning models that estimate anti-SARS-COV-2 activity of molecular compounds. The models evaluate the degree to which different molecular compounds target viral infectivity, viral entry, viral replication, and human cell toxicity. The platform provides a robust and rapid approach to screening compounds for anti-SARS-COV-2 activity, and may streamline identification and development of novel pharmacologic interventions. I applaud the authors on developing a novel platform that has potential to accelerate identification of new Covid-19 therapies. My concerns relate primarily to the structure of the manuscript and potential undescribed limitations of the models. Below are some minor comments:">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb90da33e681d22a368cebf7a4f94e549793efa0" target='_blank'>Review of: "A machine learning platform to estimate anti-SARS-CoV-2 activities"</a></td>
          <td>
            C. Eckhardt
          </td>
          <td>2021-07-18</td>
          <td></td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fbb8f15e393f8d276ece5379ed35fe9b5ec6d29" target='_blank'>From predictions to prescriptions: A data-driven response to COVID-19</a></td>
          <td>
            D. Bertsimas, L. Boussioux, Ryan Cory-Wright, A. Delarue, V. Digalakis, A. Jacquillat, D. LAHLOU KITANE, G. Lukin, M. L. Li, L. Mingardi, O. Nohadani, A. Orfanoudaki, T. Papalexopoulos, I. Paskov, J. Pauphilet, O. Skali Lami, B. Stellato, H. Tazi Bouardi, K. Villalobos Carballo, H. Wiberg, C. Zeng
          </td>
          <td>2020-06-29</td>
          <td>Health Care Management Science</td>
          <td>67</td>
          <td>93</td>
        </tr>

        <tr id="
 The PHENotype SIMulator, is a systems biology tool, which leverages available transcriptomic and proteomic databases to model of SARS-CoV-2 host-infection in silico. PHENSIM can determine the viral effects on cellular host-immune responses, which we recently applied to in silico drug repurposing for COVID-19. There is a clear importance of previously imprinted viral infections on the host-immune response against SARS-CoV2. In this study we explore the potential clinical applicability of PHENSIM in addressing co-infection of SARS-CoV2 (SCoV2) and human rhinovirus (HRV), or SCoV2 and Influenza A Virus (IAV).
 We leveraged PHENSIM to simulate HRV-infection of A549 lung alveolar cells at 24hours in silico, yielding comparable cellular transcriptomic signatures as previously published for in vitro HRV-infection, including antiviral and inflammatory gene transcriptomics. Next, we simulated viral co-infection of HRV/SCoV2 or IAV/SCoV2 using PHENSIM, and assessed co-infection effects in silico.
 Similar to recent in vitro studies, our in silico results indicated HRV-infection prior to SCoV2 exposure induced ISG responses and downregulated SCoV2 transcriptomic activity. Further analysis revealed several key SCoV2 affected pathways, including IFN Type-I & -III and MHC-II pathways, to be negatively affected by HRV-infection. In contrast, co-infection with IAV and SCoV2 resulted in overall increased SCoV2 transcriptomic activity.
 Our PHENSIM results confirm HRV-imprinted downmodulation of SARS-CoV2 activity and the importance of timely IFN pathway activation for effective immune response and viral clearance. Identified anti-correlating pathways are of interest for therapeutic targeting and effective drug development.
 N.I.M. was funded in part by a fellowship award from the Netherlands-Caribbean Foundation for Clinical Higher Education (NASKHO). S.A., A.F. and A.P. have been partially supported by the MIUR PON research project BILIGeCT “Liquid Biopsies for Cancer Clinical Management”. National Cancer Institute Physical Sciences-Oncology Center Grant U54 CA193313-01 (to B.M.), and US Army grant W911NF1810427 (to B.M.)">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a4f76301fa1217ef4f600a5dc75f41952a86121" target='_blank'>Potential Clinical Applicability of The PHENotype SIMulator for In Silico Viral Co-Infection Studies in COVID-19</a></td>
          <td>
            Naomi I Maria, Rosaria Valentina Rapicavoli, S. Alaimo, E. Bischof, A. Pulvirenti, Bud Mishra, A. Duits, A. Ferro
          </td>
          <td>2022-05-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background Drug repurposing is a fast and effective way to develop drugs for an emerging disease such as COVID-19. The main challenges of effective drug repurposing are the discoveries of the right therapeutic targets and the right drugs for combating the disease. Methods Here, we present a systematic repurposing approach, combining Homopharma and hierarchal systems biology networks (HiSBiN), to predict 327 therapeutic targets and 21,233 drug-target interactions of 1,592 FDA drugs for COVID-19. Among these multi-target drugs, eight candidates (along with pimozide and valsartan) were tested and methotrexate was identified to affect 14 therapeutic targets suppressing SARS-CoV-2 entry, viral replication, and COVID-19 pathologies. Through the use of in vitro (EC50 = 0.4 μM) and in vivo models, we show that methotrexate is able to inhibit COVID-19 via multiple mechanisms. Results Our in vitro studies illustrate that methotrexate can suppress SARS-CoV-2 entry and replication by targeting furin and DHFR of the host, respectively. Additionally, methotrexate inhibits all four SARS-CoV-2 variants of concern. In a Syrian hamster model for COVID-19, methotrexate reduced virus replication, inflammation in the infected lungs. By analysis of transcriptomic analysis of collected samples from hamster lung, we uncovered that neutrophil infiltration and the pathways of innate immune response, adaptive immune response and thrombosis are modulated in the treated animals. Conclusions We demonstrate that this systematic repurposing approach is potentially useful to identify pharmaceutical targets, multi-target drugs and regulated pathways for a complex disease. Our findings indicate that methotrexate is established as a promising drug against SARS-CoV-2 variants and can be used to treat lung damage and inflammation in COVID-19, warranting future evaluation in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d4aa9be0693f525725211246982c69d6c8e1a6" target='_blank'>Methotrexate inhibition of SARS-CoV-2 entry, infection and inflammation revealed by bioinformatics approach and a hamster model</a></td>
          <td>
            Yun-Ti Chen, Yu-Hsiu Chang, Nikhil Pathak, Shey-Cherng Tzou, Yong-Chun Luo, Yen-Chao Hsu, Tian-Neng Li, Jung-Yu Lee, Yi-Cyun Chen, Yu-Wei Huang, Hsin J. Yang, Nung-Yu Hsu, H. Tsai, Tein-Yao Chang, Shu-Chen Hsu, Ping-Cheng Liu, Yuan-Fan Chin, Wen-Chin Lin, Chuen-Mi Yang, Hsuehling Wu, Chia-Ying Lee, H. Hsu, Yi-Chun Liu, J. Chu, L. Wang, Jann-Yuan Wang, Chih-Heng Huang, Chi-Hung Lin, P. Hsieh, Y. Wu Lee, Y. Hung, Jinn-Moon Yang
          </td>
          <td>2022-12-21</td>
          <td>Frontiers in Immunology</td>
          <td>6</td>
          <td>43</td>
        </tr>

        <tr id="Quantitative systems pharmacology (QSP) modeling has become increasingly important in pharmaceutical research and development, and is a powerful tool to gain mechanistic insights into the complex dynamics of biological systems in response to drug treatment. However, even once a suitable mathematical framework to describe the pathophysiology and mechanisms of interest is established, final model calibration and the exploration of variability can be challenging and time consuming. QSP models are often formulated as multi-scale, multi-compartment nonlinear systems of ordinary differential equations. Commonly accepted modeling strategies, workflows, and tools have promise to greatly improve the efficiency of QSP methods and improve productivity. In this paper, we present the QSP Toolbox, a set of functions, structure array conventions, and class definitions that computationally implement critical elements of QSP workflows including data integration, model calibration, and variability exploration. We present the application of the toolbox to an ordinary differential equations-based model for antibody drug conjugates. As opposed to a single stepwise reference model calibration, the toolbox also facilitates simultaneous parameter optimization and variation across multiple in vitro, in vivo, and clinical assays to more comprehensively generate alternate mechanistic hypotheses that are in quantitative agreement with available data. The toolbox also includes scripts for developing and applying virtual populations to mechanistic exploration of biomarkers and efficacy. We anticipate that the QSP Toolbox will be a useful resource that will facilitate implementation, evaluation, and sharing of new methodologies in a common framework that will greatly benefit the community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a62799c63dcab1a1119a051f38f7f2d59b0ae0d" target='_blank'>QSP Toolbox: Computational Implementation of Integrated Workflow Components for Deploying Multi-Scale Mechanistic Models</a></td>
          <td>
            Yougan Cheng, Craig J. Thalhauser, S. Smithline, Jyotsna Pagidala, M. Miladinov, H. Vezina, Manish Gupta, T. Leil, B. Schmidt
          </td>
          <td>2017-05-24</td>
          <td>The AAPS Journal</td>
          <td>46</td>
          <td>24</td>
        </tr>

        <tr id="Systems biology is an approach that marries complimentary disciplines, encouraging the use of quantitative methods to help define, explain, and predict biological processes. By building computational models of biological systems, we can pose new biologically motivated questions and make falsifiable, quantitative predictions. In this thesis I will discuss the cycle of model building and experimental validation, and how it has provided insight into poorly and understood systems and allowed us to predict the effects of perturbations on these systems, which could have real and significant effects in human health and medicine. First, we model the activation of neutrophils in sepsis. By fitting a single model to two sets of data, coming from animals that survive and succumb to the same bacterial challenge, we create a realistic representation of biological variation, showing how a single network architecture can lead to different outcomes. Additionally, this method allows us to identify markers for sepsis susceptibility and identify and optimize a potential treatment option to lead to improved outcomes. Next, we model signaling downstream of the T cell receptor, and how this leads to differentiation decision making in CD4 T cells. By modeling the dynamics of this signaling network under varying antigen doses, we are able to identify network elements critical to dose discrimination, leading to the production of Treg cells following low dose stimulation and Th cells following high dose stimulation. We can then perturb these elements of the network, to potentially fine tune mature T cell populations to alter the trajectories of autoimmune disorders or cancer. Finally, we model the dynamics of IL-17 signaling. This allows us to understand how ubiquitin scaffolds form following cytokine stimulation, leading to the activation of NF-B, and how the ubiquitin editing enzyme A20 acts as a negative feedback regulator by breaking these chains. This allows us to better understand ubiquitin oligomerization as a fulcrum in the system, and how changes in A20 and ubiquitin binding proteins lead to different profiles of NF-B activation and could play a role in inflammatory disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c5de783dbb55c40e10e29aae4417239bfb08290" target='_blank'>Closing the loop: A combined computational modeling and experimental approach provides novel insights into immune cell signaling systems and their global effects.</a></td>
          <td>
            R. Sheehan
          </td>
          <td>2016-09-08</td>
          <td></td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Significance The COVID-19 pandemic has highlighted the importance of prioritizing approved drugs to treat severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infections. Here, we deployed algorithms relying on artificial intelligence, network diffusion, and network proximity to rank 6,340 drugs for their expected efficacy against SARS-CoV-2. We experimentally screened 918 drugs, allowing us to evaluate the performance of the existing drug-repurposing methodologies, and used a consensus algorithm to increase the accuracy of the predictions. Finally, we screened in human cells the top-ranked drugs, identifying six drugs that reduced viral infection, four of which could be repurposed to treat COVID-19. The developed strategy has significance beyond COVID-19, allowing us to identify drug-repurposing candidates for neglected diseases. The COVID-19 pandemic has highlighted the need to quickly and reliably prioritize clinically approved compounds for their potential effectiveness for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infections. Here, we deployed algorithms relying on artificial intelligence, network diffusion, and network proximity, tasking each of them to rank 6,340 drugs for their expected efficacy against SARS-CoV-2. To test the predictions, we used as ground truth 918 drugs experimentally screened in VeroE6 cells, as well as the list of drugs in clinical trials that capture the medical community’s assessment of drugs with potential COVID-19 efficacy. We find that no single predictive algorithm offers consistently reliable outcomes across all datasets and metrics. This outcome prompted us to develop a multimodal technology that fuses the predictions of all algorithms, finding that a consensus among the different predictive methods consistently exceeds the performance of the best individual pipelines. We screened in human cells the top-ranked drugs, obtaining a 62% success rate, in contrast to the 0.8% hit rate of nonguided screenings. Of the six drugs that reduced viral infection, four could be directly repurposed to treat COVID-19, proposing novel treatments for COVID-19. We also found that 76 of the 77 drugs that successfully reduced viral infection do not bind the proteins targeted by SARS-CoV-2, indicating that these network drugs rely on network-based mechanisms that cannot be identified using docking-based strategies. These advances offer a methodological pathway to identify repurposable drugs for future pathogens and neglected diseases underserved by the costs and extended timeline of de novo drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/836e965cd93369089027d31d92a55b5819cae267" target='_blank'>Network medicine framework for identifying drug-repurposing opportunities for COVID-19</a></td>
          <td>
            D. Gysi, Ì. Valle, M. Zitnik, Asher Ameli, Xiao Gan, Onur Varol, Helia Sanchez, R. Baron, D. Ghiassian, J. Loscalzo, A. Barabási
          </td>
          <td>2020-04-15</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>414</td>
          <td>150</td>
        </tr>

        <tr id="Significance The COVID-19 pandemic has highlighted the importance of prioritizing approved drugs to treat severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infections. Here, we deployed algorithms relying on artificial intelligence, network diffusion, and network proximity to rank 6,340 drugs for their expected efficacy against SARS-CoV-2. We experimentally screened 918 drugs, allowing us to evaluate the performance of the existing drug-repurposing methodologies, and used a consensus algorithm to increase the accuracy of the predictions. Finally, we screened in human cells the top-ranked drugs, identifying six drugs that reduced viral infection, four of which could be repurposed to treat COVID-19. The developed strategy has significance beyond COVID-19, allowing us to identify drug-repurposing candidates for neglected diseases. The COVID-19 pandemic has highlighted the need to quickly and reliably prioritize clinically approved compounds for their potential effectiveness for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infections. Here, we deployed algorithms relying on artificial intelligence, network diffusion, and network proximity, tasking each of them to rank 6,340 drugs for their expected efficacy against SARS-CoV-2. To test the predictions, we used as ground truth 918 drugs experimentally screened in VeroE6 cells, as well as the list of drugs in clinical trials that capture the medical community’s assessment of drugs with potential COVID-19 efficacy. We find that no single predictive algorithm offers consistently reliable outcomes across all datasets and metrics. This outcome prompted us to develop a multimodal technology that fuses the predictions of all algorithms, finding that a consensus among the different predictive methods consistently exceeds the performance of the best individual pipelines. We screened in human cells the top-ranked drugs, obtaining a 62% success rate, in contrast to the 0.8% hit rate of nonguided screenings. Of the six drugs that reduced viral infection, four could be directly repurposed to treat COVID-19, proposing novel treatments for COVID-19. We also found that 76 of the 77 drugs that successfully reduced viral infection do not bind the proteins targeted by SARS-CoV-2, indicating that these network drugs rely on network-based mechanisms that cannot be identified using docking-based strategies. These advances offer a methodological pathway to identify repurposable drugs for future pathogens and neglected diseases underserved by the costs and extended timeline of de novo drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/836e965cd93369089027d31d92a55b5819cae267" target='_blank'>Network medicine framework for identifying drug-repurposing opportunities for COVID-19</a></td>
          <td>
            D. Gysi, Ì. Valle, M. Zitnik, Asher Ameli, Xiao Gan, Onur Varol, Helia Sanchez, R. Baron, D. Ghiassian, J. Loscalzo, A. Barabási
          </td>
          <td>2020-04-15</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>414</td>
          <td>150</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11722d02b48c06efb6af22187a458abc5098f51e" target='_blank'>Quantitative systems pharmacology in model-informed drug development and therapeutic use</a></td>
          <td>
            S. Wicha, C. Kloft
          </td>
          <td>2018-08-01</td>
          <td>Current Opinion in Systems Biology</td>
          <td>3</td>
          <td>35</td>
        </tr>

        <tr id="The recent global COVID-19 outbreak, attributed by the World Health Organization to the rapid spread of the severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2), underscores the need for an extensive exploration of virological intricacies, fundamental pathophysiology, and immune responses. This investigation is vital to unearth potential therapeutic avenues and preventive strategies. Our study delves into the intricate interaction between SARS-CoV-2 and the immune system, coupled with exploring therapeutic interventions to counteract dysfunctional immune responses like the ‘cytokine storm’ (CS), a driver of disease progression. Understanding these immunological dimensions informs the design of precise multiepitopetargeted peptide vaccines using advanced immunoinformatics and equips us with tools to confront the cytokine storm. Employing a control theory-based approach, we scrutinize the perturbed behavior of key proteins associated with cytokine storm during COVID-19 infection. Our findings support ACE2 activation as a potential drug target for CS control and confirm AT1R inhibition as an alternative strategy. Leveraging deep learning, we identify potential drugs to individually target ACE2 and AT1R, with Lomefloxacin and Fostamatinib emerging as standout options due to their close interaction with ACE2. Their stability within the protein-drug complex suggests superior efficacy among many drugs from our deep-learning analysis. Moreover, there is a significant scope for optimization in fine-tuning protein-drug interactions. Strong binding alone may not be the sole determining factor for potential drugs; precise adjustments are essential. The application of advanced computational power offers novel solutions, circumventing time-consuming lab work. In scenarios necessitating both ACE2 and AT1R targeting, optimal drug combinations can be derived from our analysis of drug-drug interactions, as detailed in the manuscript.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e06463d6d53de9244d9b984a9b0c2f6e796bed" target='_blank'>In Silico Therapeutic Intervention on Cytokine Storm in COVID-19</a></td>
          <td>
            Abhisek Bakshi, Kaustav Gangopadhyay, Sujit Basak, Amlan Chakrabarti, Abhijit Dasgupta, Rajat K. De
          </td>
          <td>2023-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Immunotherapy has shown great potential in the treatment of cancer; however, only a fraction of patients respond to treatment, and many experience autoimmune‐related side effects. The pharmaceutical industry has relied on mathematical models to study the behavior of candidate drugs and more recently, complex, whole‐body, quantitative systems pharmacology (QSP) models have become increasingly popular for discovery and development. QSP modeling has the potential to discover novel predictive biomarkers as well as test the efficacy of treatment plans and combination therapies through virtual clinical trials. In this work, we present a QSP modeling platform for immuno‐oncology (IO) that incorporates detailed mechanisms for important immune interactions. This modular platform allows for the construction of QSP models of IO with varying degrees of complexity based on the research questions. Finally, we demonstrate the use of the platform through two example applications of immune checkpoint therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/750c1aa4fc2f364ea1febcf5761c49bd89bc2f93" target='_blank'>QSP‐IO: A Quantitative Systems Pharmacology Toolbox for Mechanistic Multiscale Modeling for Immuno‐Oncology Applications</a></td>
          <td>
            R. Sové, M. Jafarnejad, Chen Zhao, Hanwen Wang, Huilin Ma, A. Popel
          </td>
          <td>2020-07-02</td>
          <td>CPT: Pharmacometrics & Systems Pharmacology</td>
          <td>39</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3498e511e9eb5c885a5bd150e27ed48f227d9e8a" target='_blank'>Future Directions for Quantitative Systems Pharmacology.</a></td>
          <td>
            Birgit Schoeberl, C. Musante, S. Ramanujan
          </td>
          <td>2025-01-16</td>
          <td>Handbook of experimental pharmacology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Data-driven computational analysis and simulation identified drug repurposing opportunities for COVID-19. The global outbreak of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) necessitates the rapid development of new therapies against coronavirus disease 2019 (COVID-19) infection. Here, we present the identification of 200 approved drugs, appropriate for repurposing against COVID-19. We constructed a SARS-CoV-2–induced protein network, based on disease signatures defined by COVID-19 multiomics datasets, and cross-examined these pathways against approved drugs. This analysis identified 200 drugs predicted to target SARS-CoV-2–induced pathways, 40 of which are already in COVID-19 clinical trials, testifying to the validity of the approach. Using artificial neural network analysis, we classified these 200 drugs into nine distinct pathways, within two overarching mechanisms of action (MoAs): viral replication (126) and immune response (74). Two drugs (proguanil and sulfasalazine) implicated in viral replication were shown to inhibit replication in cell assays. This unbiased and validated analysis opens new avenues for the rapid repurposing of approved drugs into clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ddd817afac82978ecbf7fd104865e65bee7c75" target='_blank'>Identification of SARS-CoV-2–induced pathways reveals drug repurposing strategies</a></td>
          <td>
            N. Han, W. Hwang, K. Tzelepis, Patrick Schmerer, E. Yankova, Méabh MacMahon, Winnie Lei, Nicholas M Katritsis, Anika Liu, Ulrike Felgenhauer, A. Schuldt, Rebecca Harris, Kathryn L Chapman, F. McCaughan, F. Weber, T. Kouzarides
          </td>
          <td>2021-06-01</td>
          <td>Science Advances</td>
          <td>33</td>
          <td>108</td>
        </tr>

        <tr id="The ongoing COVID-19 pandemic is arguably one of the most challenging health crises in modern times. The development of effective strategies to control the spread of SARS-CoV-2 were major goals for governments and policy makers. Mathematical modeling and machine learning emerged as potent tools to guide and optimize the different control measures. This review briefly summarizes the SARS-CoV-2 pandemic evolution during the first 3 years. It details the main public health challenges focusing on the contribution of mathematical modeling to design and guide government action plans and spread mitigation interventions of SARS-CoV-2. Next describes the application of machine learning methods in a series of study cases, including COVID-19 clinical diagnosis, the analysis of epidemiological variables, and drug discovery by protein engineering techniques. Lastly, it explores the use of machine learning tools for investigating long COVID, by identifying patterns and relationships of symptoms, predicting risk indicators, and enabling early evaluation of COVID-19 sequelae.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07cb656b375fad48b95a91a26d61881478421db6" target='_blank'>The role of machine learning in health policies during the COVID-19 pandemic and in long COVID management</a></td>
          <td>
            Lindybeth Sarmiento Varón, Jorge González-Puelma, David Medina-Ortiz, Jacqueline Aldridge, Diego Alvarez-Saravia, R. Uribe-Paredes, Marcelo A Navarrete
          </td>
          <td>2023-04-11</td>
          <td>Frontiers in Public Health</td>
          <td>8</td>
          <td>10</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>