<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../RA/">
      
      
        <link rel="next" href="../Virtual%20patients/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Prostate cancer
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Virtual%20cell/" class="md-tabs__link">
        
  
    
  
  Virtual cell

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../RA/" class="md-tabs__link">
        
  
    
  
  RA

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Prostate cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Virtual%20patients/" class="md-tabs__link">
        
  
    
  
  Virtual patients

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Virtual%20cell/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual cell
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../RA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RA
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Prostate cancer
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Virtual%20patients/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual patients
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Prostate cancer</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-03 06:09:13 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Prostate cancer</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Prostate cancer</a><br>
      <a href="#recommended_articles">3. Recommended articles on Prostate cancer</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Prostate cancer</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Prostate cancer</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Background Therapeutic vaccination against disseminated prostate cancer (PCa) is partially effective in some PCa patients. We hypothesized that the efficacy of treatment will be enhanced by individualized vaccination regimens tailored by simple mathematical models. Methodology/Principal Findings We developed a general mathematical model encompassing the basic interactions of a vaccine, immune system and PCa cells, and validated it by the results of a clinical trial testing an allogeneic PCa whole-cell vaccine. For model validation in the absence of any other pertinent marker, we used the clinically measured changes in prostate-specific antigen (PSA) levels as a correlate of tumor burden. Up to 26 PSA levels measured per patient were divided into each patient's training set and his validation set. The training set, used for model personalization, contained the patient's initial sequence of PSA levels; the validation set contained his subsequent PSA data points. Personalized models were simulated to predict changes in tumor burden and PSA levels and predictions were compared to the validation set. The model accurately predicted PSA levels over the entire measured period in 12 of the 15 vaccination-responsive patients (the coefficient of determination between the predicted and observed PSA values was R 2 = 0.972). The model could not account for the inconsistent changes in PSA levels in 3 of the 15 responsive patients at the end of treatment. Each validated personalized model was simulated under many hypothetical immunotherapy protocols to suggest alternative vaccination regimens. Personalized regimens predicted to enhance the effects of therapy differed among the patients. Conclusions/Significance Using a few initial measurements, we constructed robust patient-specific models of PCa immunotherapy, which were retrospectively validated by clinical trial results. Our results emphasize the potential value and feasibility of individualized model-suggested immunotherapy protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c181942f9c3307163599684cc55f9a9c096219a9" target='_blank'>
                Predicting Outcomes of Prostate Cancer Immunotherapy by Personalized Mathematical Models
                </a>
              </td>
          <td>
            N. Kronik, Y. Kogan, M. Elishmereni, Karin Halevi-Tobias, S. Vuk-Pavlović, Z. Agur
          </td>
          <td>2010-12-08</td>
          <td>PLoS ONE</td>
          <td>117</td>
          <td>35</td>

            <td><a href='../recommendations/c181942f9c3307163599684cc55f9a9c096219a9' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c453e732e4940c2b85b4ef9a46c4d3bf3b77936" target='_blank'>
                Parameter identification of a model for prostate cancer treated by intermittent therapy.
                </a>
              </td>
          <td>
            Clément Draghi, F. Denis, A. Tolédano, C. Letellier
          </td>
          <td>2019-01-01</td>
          <td>Journal of theoretical biology</td>
          <td>3</td>
          <td>36</td>

            <td><a href='../recommendations/6c453e732e4940c2b85b4ef9a46c4d3bf3b77936' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Prostate cancer'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Prostate cancer</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="BACKGROUND
Prostate-specific antigen (PSA) kinetics has been investigated as a prognostic marker in post hoc analyses of clinical trials. This study validated the prognostic value of rapid and deep PSA decline in metastatic hormone-sensitive prostate cancer (mHSPC) using real-world data.


METHODS
In total, 1296 patients with mHSPC were retrospectively reviewed. We assessed the prognostic value of a PSA decline to ≤ 0.2 ng/mL after 12 weeks of treatment and investigated several potential risk factors for a poor PSA response.


RESULTS
Of 1296 patients, 714 (cohort 1: 55.1%) were treated with conventional hormonal therapy, while 582 (cohort 2: 44.9%) received androgen signaling inhibitors. There were significant differences in progression-free survival and overall survival between patients with PSA decline to ≤ 0.2 ng/mL by 12 weeks of treatment and others (p < 0.001 for each). In addition, patients with an initial PSA ≥ 200 ng/mL, Clinical T4 and Grade Group 5 were less likely to achieve PSA decline to ≤ 0.2 ng/mL by 12 weeks of treatment, with odds ratios of 0.31 (p < 0.001), 0.67 (p = 0.039) and 0.70 (p = 0.043), respectively.


CONCLUSION
Our findings suggested that PSA decline to ≤ 0.2 ng/mL by 12 weeks of treatment may be a useful prognostic biomarker for mHSPC in the real-world setting. The prognostic value of this should be further investigated in a prospective cohort, and identification of an optimal cutoff value is necessary for its application in clinical trial design or clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed73e5db844c11c13f937c2018307bd7b8c8ee4a" target='_blank'>
              Rapid and Deep Prostate-Specific Antigen Decline is a Prognostic Marker in Metastatic Hormone-Sensitive Prostate Cancer: A Real-World Multi-Intuitional Analysis.
              </a>
            </td>
          <td>
            Kotaro Suzuki, T. Hara, H. Watanabe, K. Nakane, Kiyoshi Takahara, T. Naiki, T. Yasui, R. Shiroki, Takuya Koie, Hideaki Miyake
          </td>
          <td>2024-12-27</td>
          <td>The Prostate</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa72a42061c33f14558fe96c7d73cbe119e62a50" target='_blank'>
              Predicting intermediate-risk prostate cancer using machine learning.
              </a>
            </td>
          <td>
            M. Stojadinovic, Milorad Stojadinovic, Slobodan Jankovic
          </td>
          <td>2025-01-03</td>
          <td>International urology and nephrology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="BACKGROUND
Radiotherapy is often given with androgen deprivation therapy for prostate cancer which causes a reduction in testosterone levels, which when below castrate levels, can cause the prostate specific antigen (PSA) levels to be artificially low.


AIM
To determine if high-level radiotherapy clinical trials are underestimating biochemical recurrence (BCR) rates due to inadequate measurement of testosterone levels.


METHODS
The study plans for clinical trials performed by the Radiation Therapy Oncology Group (RTOG [now NRG]) on clinicaltrials.gov were reviewed for details on testosterone measurement in trials from 1994 to 2023, namely if the testosterone levels were indexed to PSA levels.


OUTCOMES
PSA being indexed to testosterone levels and other metrics of testosterone measurement, including time of day of measurement, assay used, and mean testosterone measurement.


RESULTS
Five of 21 (24%) trials stipulated that testosterone levels should be indexed to PSA levels. Eleven of 21 (52%) trials made no mention of testosterone. No trial reported testosterone assay or time of day of measurement. Thirteen of 21 (62%) trials did not require regular follow-up testosterone measurements.


CLINICAL IMPLICATIONS
The number of clinical trials indexing or regularly measuring testosterone was surprisingly low, which could cause an underestimation of BCR rates.


STRENGTHS AND LIMITATIONS
Strengths include being the first study, to our knowledge, to analyze the details of testosterone measurement in high-level radiotherapy trials. Limitations include only analyzing RTOG/NRG trials, analyzing unpublished data, and using clinicaltrials.gov rather than official trial protocols to determine details of testosterone measurement.


CONCLUSION
Indexing of testosterone levels to PSA levels in high-level radiotherapy trials using androgen deprivation therapy was uncommon, possibly rendering data on BCR unreliable, potentially underestimating BCR rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af53258b3b46b94006440e2dfbfe6622810c19e9" target='_blank'>
              Contemporary prostate cancer radiation therapy trials may underestimate the risk of biochemical recurrence.
              </a>
            </td>
          <td>
            M. West, J. Flores, J. Mulhall
          </td>
          <td>2024-12-30</td>
          <td>The journal of sexual medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76012f1c5791d6df11bd6e8fe93bfd3261005d7e" target='_blank'>
              Prostate-specific antigen screening at low thresholds of men with pathogenic BRCA1/2 variants.
              </a>
            </td>
          <td>
            H. V. Stroomberg, K. Brasso, A. A. Blak, A. Byrjalsen, T. V. O. Hansen, Andreas Røder
          </td>
          <td>2025-01-21</td>
          <td>Prostate cancer and prostatic diseases</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a8ec3ff03860e7ef5588a739968da229a8489fd" target='_blank'>
              Prognostic Testing for Prostate Cancer-A Cost-Effectiveness Analysis Comparing a Prostatype P-Score Biomarker Approach to Standard Clinical Practice.
              </a>
            </td>
          <td>
            A. Fridhammar, Oskar Frisell, Karin Wahlberg, Emelie Berglund, Pontus Röbeck, Sofie Persson
          </td>
          <td>2025-01-11</td>
          <td>PharmacoEconomics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Despite the expanding therapeutic options available to cancer patients, therapeutic resistance, disease recurrence, and metastasis persist as hallmark challenges in the treatment of cancer. The rise to prominence of generative artificial intelligence (GenAI) in many realms of human activities is compelling the consideration of its capabilities as a potential lever to advance the development of effective cancer treatments. This article presents a hypothetical case study on the application of generative pre-trained transformers (GPTs) to the treatment of metastatic prostate cancer (mPC). The case explores the design of GPT-supported adaptive intermittent therapy for mPC. Testosterone and prostate-specific antigen (PSA) are assumed to be repeatedly monitored while treatment may involve a combination of androgen deprivation therapy (ADT), androgen receptor-signalling inhibitors (ARSI), chemotherapy, and radiotherapy. The analysis covers various questions relevant to the configuration, training, and inferencing of GPTs for the case of mPC treatment with a particular attention to risk mitigation regarding the hallucination problem and its implications to clinical integration of GenAI technologies. The case study provides elements of an actionable pathway to the realization of GenAI-assisted adaptive treatment of metastatic prostate cancer. As such, the study is expected to help facilitate the design of clinical trials of GenAI-supported cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7244f1484e4d500495398cd3c86ccd9b214a6176" target='_blank'>
              Adaptive Treatment of Metastatic Prostate Cancer Using Generative Artificial Intelligence
              </a>
            </td>
          <td>
            Y. Derbal
          </td>
          <td>2025-01-01</td>
          <td>Clinical Medicine Insights. Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="BACKGROUND
Prior studies have concentrated exclusively on how different prostate-specific antigen (PSA) levels affect the prognosis of high-grade prostate cancer (PCa), often overlooking the prognosis of low-grade PCa.


METHODS
The present cohort study included individuals diagnosed with PCa from the Surveillance, Epidemiology, and End Results (SEER) database between 2010 and 2021. The all-cause mortality (ACM) and prostate cancer-specific mortality (PCSM) for each treatment group was calculated stratified by the four PSA levels (≤ 4.0, 4.1-10.0, 10.1-20.0, and > 20.0 ng/mL). Fine and Gray competing-risks analyses were conducted to calculate hazard ratios (HRs) with 95% confidence intervals (CIs). Cox proportional hazards regression analyses using PSA as a continuous variable with restricted cubic splines (RCS) were conducted to allow for potential nonlinear relationships.


RESULTS
This study encompassed 416,825 male patients diagnosed with PCa. Compared to individuals with PSA value between 4.1 and 10.0 ng/mL, a significant association between low levels of PSA (≤ 4.0 ng/mL) and an increased risk of ACM (AHR = 1.15, 95% CI: 1.12-1.19; p < 0.001) and PCSM (AHR = 1.49, 95% CI: 1.38-1.61; p < 0.001) was observed. Additionally, the increased risk of ACM (AHR = 1.35, 95% CI: 1.29-1.40; p < 0.001) and PCSM (AHR = 1.84, 95% CI: 1.67-2.02; p < 0.001) are more pronounced within the first 5 years post-diagnosis. In most subgroups, similar results were observed. The RCS curves further corroborated the correlation between PSA value and the risk of mortality.


CONCLUSION
Low PSA levels are notably linked to a heightened risk for both ACM and PCSM, irrespective of the grade of PCa being high or low. There is a need to initiate new studies that tackle novel diagnostics and therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a355fb3f5aa3b30edd938e9353a83e60e77a81bb" target='_blank'>
              Reassessing the Role of Low PSA in Prognosis Across Grades of Prostate Cancer: A Cohort Study.
              </a>
            </td>
          <td>
            Cheng Shen, Zhan Chen, Fei-Hong Hu, Wei Wang, Yong-Shen Pan, Yong Zhang, Wei Zhang, Xin-feng Chen, Hong-Lin Chen, Hua Zhu, Bing Zheng
          </td>
          <td>2025-01-29</td>
          <td>The Prostate</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background: To establish a machine learning (ML) model for predicting prostate biopsy outcomes using prostate-specific antigen (PSA) values, multiparametric magnetic resonance imaging (mpMRI) findings, and hematologic parameters. Methods: The medical records of the patients who had undergone a prostate biopsy were evaluated. Laboratory findings, mpMRI findings, and prostate biopsy results were collected. Patients with benign prostate pathology were classified as Group 1, and those with prostate cancer (PCa) were classified as Group 2. The following ML algorithms were used to create the ML model: ExtraTrees classifier, Light Gradient-Boosting Machine (LGBM) classifier, eXtreme Gradient Boosting (XGB) classifier, Logistic Regression, and Random Forest classifier. Results: A total of 244 male patients who met the inclusion criteria were included in this study. Among them, 171 (71.1%) were categorized in Group 1, and 73 (29.9%) in Group 2. The LGBM classifier model demonstrated the highest performance, achieving an accuracy rate of 81.6% and an AUC–ROC (area under the curve–receiver operating characteristic) of 78.4%, with sensitivity and specificity values of 66.7% and 88.2%, respectively, in predicting prostate biopsy outcomes. Conclusions: Pathological results can be predicted by ML models using PSA values, mpMRI findings, and hematologic parameters prior to a prostate biopsy, potentially reducing unnecessary biopsy procedures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc5cd8a9e47869c7ef74561e32e26a158e97f12c" target='_blank'>
              Machine Learning-Based Prediction of Prostate Biopsy Necessity Using PSA, MRI, and Hematologic Parameters
              </a>
            </td>
          <td>
            Mustafa Sungur, Aykut Aykaç, Mehmet Erhan Aydın, Ozer Celik, Coşkun Kaya
          </td>
          <td>2024-12-31</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Objectives to construct a prediction model for clinically significant prostate cancer (csPCa) based on prostate-specific antigen (PSA) levels, digital rectal examination (DRE), and transrectal ultrasonography (TRUS). Methods We retrospectively analysed 1196 Asian patients who underwent transrectal ultrasound-guided biopsy (TRUSB) between June 2000 and February 2023. Patients were randomly divided into a training set of 837 cases (70%) and a validation set of 359 patients (30%). A csPCa risk prediction model was established using the logistic regression. The performance of the model was examined based on calibration curves, receiver operating characteristic (ROC) curves, decision curve analysis (DCA), and clinical impact curves (CIC). Results Serum PSA levels, age, DRE results, prostatic shape, prostatic border and hypoechoic area were associated with pathological outcomes. The area under the ROC curve of the training set was 0.890 (95%CI: 0.865-0.816). The optimal cut-off value was 0.279. The calibration curves indicated good calibration, and the DCA and CIC results demonstrated good clinical utility. Significantly, the prediction model has higher negative predictive value (89.8%) and positive predictive value (68.0%) compared with MRI. Subsequently, we developed an online calculator (https://jiwentong0.shinyapps.io/dynnomapp/) with six variables for biopsy optimization. Conclusion This study incorporated the results of three traditional diagnostic methods to establish a cost-effective and highly accurate model for predicting csPCa before biopsy. With this model, we aim to provide a non-invasive and cost-effective tool for csPCa detection in Asia and other underdeveloped areas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b86d00009bbf7d99e3325c5ad947b1d59f371a89" target='_blank'>
              Construction of a clinically significant prostate cancer risk prediction model based on traditional diagnostic methods
              </a>
            </td>
          <td>
            Wen-Tong Ji, Yong-Kun Wang, Zhan-Yang Han, Si-Qi Wang, Yao Wang
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="OBJECTIVE
To assess the association between prostate-specific antigen (PSA) density (PSAD) and prostate cancer mortality after a benign result on systematic transrectal ultrasonography (TRUS)-guided prostate biopsy.


PATIENTS AND METHODS
This retrospective study used data from the Finnish Randomised Study of Screening for Prostate Cancer (FinRSPC) collected between 1996 and 2020. We identified men aged 55-71 years randomised to the screening arm with PSA ≥4.0 ng/mL and a benign systematic TRUS-guided biopsy result. The cumulative prostate cancer mortality of men stratified by a PSAD cutoff of 0.15 ng/mL/cm3 was modelled with competing risk functions. The ability of PSAD, PSA, and base variables (age at biopsy, DRE result, socioeconomic status, 5α-reductase inhibitor usage, family history, and Charlson Comorbidity Index (CCI)) to predict prostate cancer death was compared using c-statistics and a likelihood ratio test.


RESULTS
After excluding 10 men without PSA data within 2 years of the biopsy and 65 without prostate volume data, 2276 men were eligible for inclusion in the study. A total of 50 men died from prostate cancer and 1028 from other causes during a median (interquartile range) follow-up of 17.4 (13.2-20.9) years. The cumulative prostate cancer mortality of men with PSAD <0.15 ng/mL/cm3 was significantly lower than that of men with PSAD ≥0.15 ng/mL/cm3: 0.5% (95% confidence interval [CI] 0.2%-1.1%) vs 2.0% (95% CI 1.2%-3.1%) at 15 years (Grey's test, P = 0.001). The model consisting of PSAD, PSA and the base variables predicted prostate cancer mortality (c-statistic 0.781) significantly better than either the base variables alone (c-statistic 0.737; likelihood-ratio test, P = 0.003) or the base variables and PSA (c-statistic 0.765; likelihood-ratio test, P = 0.039).


CONCLUSION
Prostate cancer mortality after a benign systematic TRUS-guided biopsy is low. In these patients, PSAD predicts prostate cancer mortality and provides additional value to other clinical variables. PSAD-based stratification can be used to guide follow-up strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0eb4aa4e0452d583b006e63dc668f43b41541032" target='_blank'>
              Association of prostate-specific antigen density with prostate cancer mortality after a benign systematic prostate biopsy result.
              </a>
            </td>
          <td>
            J. Pylväläinen, K. Talala, Jani Raitanen, Antti S. Rannikko, Anssi Auvinen
          </td>
          <td>2025-01-22</td>
          <td>BJU international</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Objective To evaluate the efficacy of a machine learning model for predicting prostate-specific antigen (PSA) persistence after radical prostatectomy (RP). Methods Data from 470 patients who underwent RP at the Affiliated Hospital of Qingdao University from January 2018 to June 2021 were retrospectively analyzed. Ten risk factors, including age, body mass index (BMI), preoperative PSA, biopsy Gleason score, total prostate specific antigen density (PSAD), clinical tumor stage, clinical lymph node status, seminal vesicle invasion, capsular invasion and positive surgical margin, were included in the analysis. The data were randomly divided into a training set and a test set at a ratio of 7:3, and seven different machine learning algorithms were compared. The confusion matrix, receiver operating characteristic (ROC) curve and area under the ROC curve (AUC) were used to evaluate the diagnostic performance of the model, and the random forest algorithm found to be the optimal prediction model. Results In the entire cohort, 142 (30.21%) patients developed PSA persistence. Based on all included risk factors, the random forest model had the best effect among the seven models, with an AUC of 0.8607 in the training set and 0.8011 in the test set. The feature importance results showed that capsular invasion, positive surgical margin, preoperative PSA and biopsy Gleason score were the four most important risk factors for PSA persistence after RP. Conclusion The Random Forest algorithm performed excellently in this study and can be used to construct a predictive model for PSA persistence. By incorporating clinical data from the Asian region and exploring the risk factors for PSA persistence, this study contributes to the existing research and aids clinicians in assessing the risk of PSA persistence occurrence, enabling timely treatment planning and improving patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a0e87565b869dbc8c66d479a6f0957290edb0c" target='_blank'>
              Predictive model for PSA persistence after radical prostatectomy using machine learning algorithms
              </a>
            </td>
          <td>
            Haotian Du, Guipeng Wang, Yongchao Yan, Shengxian Li, Xuecheng Yang
          </td>
          <td>2024-12-06</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b3a954b5d643cfac67348476d195f89c2861f7" target='_blank'>
              Androgen receptor and osteoglycin gene expression predicting prognosis of metastatic prostate cancer
              </a>
            </td>
          <td>
            Tomohiro Kameda, Toru Sugihara, D. Obinata, Masashi Oshima, Y. Yamada, Naoki Kimura, Ken‐ichi Takayama, Satoshi Inoue, Satoru Takahashi, Tetsuya Fujimura
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="BACKGROUND
Previous studies found that Midlife Baseline PSA (MB PSA) predicts the risk of developing lethal prostate cancer (PCa), although the cohorts were homogenous in terms of racial compositions. We aimed to investigate racial disparities in the predictive value of MB PSA for lethal PCa in a diverse, contemporary, North American population.


METHODS
Our cohort included White and Black men aged 40-59 years, who underwent MB PSA through our health system. Cumulative incidence curves depicted lethal PCa stratified by race and MB PSA above/below the median. We utilized time-dependent Receiver Operating Characteristic (ROC) curves and Area Under the ROC Curve (AUC) to compare the performance of MB PSA in predicting lethal PCa based on race. Multivariable regression (MVA) was used to examine the impact of the MB PSA in predicting lethal PCa by race.


RESULTS
We included 112,967 men, of whom 27% were Black. The cumulative incidence estimate with MB PSA values equal to the median at 15 years of follow-up was 0.13 (0.04, 0.32) for White men and 0.55 (0.24, 1.11) for Black men. AUCs comparison showed no statistically significant differences in the predictive role of MB PSA for lethal PCa between White and Black men. At MVA, using White patients with PSA ≤ median as the reference group, the HR of lethal PCa for White men with PSA > median aged 40-44, 45-49, 50-54, and 55-59 was respectively 2.98 (1.59-5.57), 3.01 (1.89-4.81), 5.10 (3.38-7.70), and 3.38 (2.32-4.92). While for Black men was respectively 5.50 (2.94-10.27), 4.19 (2.59-6.78), 9.79 (6.37-15.04), and 7.53 (5.03-11.26) (all p < 0.001).


CONCLUSION
Our findings indicate that for the same MB PSA and within the same age category, Black men have a greater risk of developing lethal PCa than White men. A separate cut-off should be created for MB PSA, if this is to be used to guide PSA screening in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91b4ddcfa9e3d75fa7360f9973f7e503d19788ff" target='_blank'>
              Racial Disparities in Future Development of Lethal Prostate Cancer Based on Midlife Baseline Prostate-Specific Antigen.
              </a>
            </td>
          <td>
            Giuseppe Chiarelli, Matthew Davis, A. Stephens, M. Finati, G. Cirulli, Chase Morrison, Akshay Sood, G. Carrieri, A. Briganti, Francesco Montorsi, Giovanni Lughezzani, Nicolòs Buffi, Craig G. Rogers, F. Abdollah
          </td>
          <td>2024-12-09</td>
          <td>The Prostate</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Simple Summary The ideal timing of androgen deprivation therapy (ADT) for biochemical recurrent (BCR) prostate cancer (PCa) is debated due to its side effects and uncertain survival benefits. Our review of 26 studies found that early ADT may delay PCa progression, but its impact on PCa-specific mortality is still unclear. The PSA thresholds for starting ADT vary, complicating decisions. Key predictors of progression include a short PSA doubling time (PSADT), a high Gleason score (GS), and a brief interval to BCR post-radiotherapy. The combination of ADT with androgen receptor pathway inhibitors (ARPIs) has shown improved metastasis-free survival in high-risk patients. Stratifying patients by their risk factors can guide treatment and, in high-risk patients, delaying ADT should be avoided.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f989560068257393c83972e73ec90567ca4ecac4" target='_blank'>
              Early Versus Delayed Androgen Deprivation Therapy for Biochemical Recurrence After Local Curative Treatment in Non-Metastatic Hormone-Sensitive Prostate Cancer: A Systematic Review of the Literature
              </a>
            </td>
          <td>
            Muneeb Uddin Karim, S. Tisseverasinghe, R. Cartes, Constanza Martinez, B. Bahoric, T. Niazi
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="

 The neutrophil-to-lymphocyte ratio (NLR), one of the inflammatory markers, can be calculated simply from peripheral blood samples. In this study, the role of NLR in predicting significant prostate cancer (sPC) before prostate biopsy was investigated.



 Patients who underwent prostate biopsy between 2017 and 2022 at the National Taiwan University Hospital were enrolled prospectively. NLR was calculated before the biopsy. sPC was defined as an unfavorable-intermediate or higher risk of the National Comprehensive Cancer Network risk classification.



 Among 484 enrolled patients, 164 sPC (33.9%) and 320 benign/insignificant prostate cancer (isPC; 66.1%) patients were diagnosed. The sPC group was older and had a higher prostate-specific antigen (PSA), smaller prostate volume, higher PSA density, more advanced clinical T stage, and more family history of prostate cancer than the benign/isPC group. There was no difference in neutrophil count (P = 0.929), lymphocyte count (P = 0.927), or NLR (P = 0.931) between groups. NLR failed to predict sPC in univariable (odds ratio, 1.02; 95% confidence interval [CI], 0.93–1.12; P = 0.7) and multivariable (odds ratio, 0.94; 95% CI, 0.79–1.12; P = 0.506) logistic regression models, respectively. The area under the curve of the receiver operating characteristic curve in the model consisted of age, PSA, prostate volume, T stage, and family history of prostate cancer was 0.882 (95% CI, 0.849–0.914). There was no statistical difference when the model included NLR (P = 0.625).



 NLR did not differ between sPC and benign/isPC groups. NLR cannot predict sPC successfully before prostate biopsy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ff45438802797b5f84c1b477033069f520b3e7a" target='_blank'>
              Predictive value of neutrophil-to-lymphocyte ratio for clinically significant prostate cancer
              </a>
            </td>
          <td>
            Siang Ren Jhuang, Jian-Hua Hong, Chao-Yuan Huang, Shih Liu, Po-Ming Chow, Shih-Chieh Jeff Chueh, Chung-Hsin Chen, Yeong-Shiau Pu
          </td>
          <td>2024-12-12</td>
          <td>Urological Science</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="OBJECTIVE
Early and accurate diagnosis of prostate cancer (PC) is crucial for effective treatment. Diagnosing  clinically insignificant cancers can lead to overdiagnosis and overtreatment, highlighting the importance of accurately selecting patients for further evaluation based on improved risk prediction tools. Novel biomarkers offer promise for enhancing this diagnostic process. In this study, we aimed to externally validate a previously developed urine and plasma biomarker test in a biopsy-naïve population.


MATERIALS AND METHODS
Urine and blood samples were prospectively collected from 362 biopsy-naïve men with suspected PC before they underwent transrectal prostate biopsies. The expression levels of a 10-gene mRNA panel were quantified using reverse transcription/quantitative polymerase chain reaction of both urine and plasma. These gene expression levels, combined with clinical features and plasma prostate-specific antigen (PSA) levels, were used to predict the presence of International Society of Urological Pathology grade group ≥ 2 PC.


RESULTS
Complete data were available for 314 patients. The sensitivity and specificity of the biomarker test were 87% (95% CI: 79-93%) and 42% (95% CI: 36-49%), respectively. The area under the curve was 0.76 (95% CI: 0.7-0.82) for the biomarker test probability and 0.65 (95% CI: 0.59-0.72) for PSA (p = 0.02). The test's negative predictive value was 89% (CI: 81-94%).


CONCLUSION
This study did not replicate the previously reported high accuracy of the biomarker test, highlighting the need for further refinement and robust external validation to ensure reliable performance across diverse patient populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cfaba648f44663b161eff9436c6c5f096045194" target='_blank'>
              Prospective validation study of a combined urine and plasma test for predicting high-grade prostate cancer in biopsy naïve men.
              </a>
            </td>
          <td>
            Torben Brøchner Pedersen, Mads Hvid Poulsen, M. Lund, Søren Feddersen, M. Albitar, Charlotte Aaberg Poulsen, Lars Lund
          </td>
          <td>2025-01-14</td>
          <td>Scandinavian journal of urology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a significant global health concern, ranking as the second most prevalent cancer among men worldwide. Genetic factors, particularly germline pathogenic variants (PVs) in DNA repair genes (DRGs), play a crucial role in PCa predisposition. Our study aimed to assess patients' adherence to a targeted PCa screening program targeting high‐risk individuals with DRG PVs and evaluate the potential reduction in biopsy and MRI rates by employing our screening protocol.We conducted a prospective ongoing trial evaluating targeted PCa screening in men with documented PVs in DRGs. Screening involved annual assessment of medical history, physical examination, prostate‐specific antigen (PSA) testing, Prostate Health Index (PHI), and multiparametric magnetic resonance imaging (mpMRI) when indicated. Descriptive statistics were used to analyse patient characteristics, and adherence to screening was evaluated at three time points: baseline (T0), one year (T1), and two years (T2) from enrolment.A total of 101 high‐risk individuals were enrolled, with a median age of 52 years. Adherence to screening was high, with 72.3% of patients attending the first annual follow‐up (T1) and 100% attending the second follow‐up (T2). Despite elevated PSA levels in some patients, no PCa was detected during the study period. However, our screening protocol demonstrated the potential in reducing unnecessary biopsies and MRIs, particularly in patients with elevated PSA but low PHI values. Limitations include the ongoing nature of the study, small sample size, and lack of non‐carrier controls.Our findings described a new PCa screening strategy integrated with genetic risk factors. The incorporation of PHI shows promise in improving the efficiency of diagnostic procedures while minimizing unnecessary interventions. High adherence among high‐risk individuals underscores the potential effectiveness of targeted screening programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2850b9b640f39bbb237a45684226f358be600580" target='_blank'>
              Impact of prostate cancer screening in European ancestry un‐affected men with germline DNA repair pathogenic variants
              </a>
            </td>
          <td>
            V. Fasulo, Giuseppe Chiarelli, G. Garofano, Carla Barbara Ripamonti, M. Barile, Paolo Bianchi, Emanuela Morenghi, A. Benetti, M. Aljoulani, A. Finocchiaro, M. Paciotti, P. Avolio, E. Beatrici, P. Arena, A. Saita, R. Hurle, F. Maura, Giorgio Da Rin, R. Asselta, Anita Capalbo, G. Soldà, P. Casale, Nicolòs Buffi, Giovanni Lughezzani, Massimo Lazzeri
          </td>
          <td>2025-01-31</td>
          <td>BJUI Compass</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Aggressive prostate cancer (PC) represents a significant health concern worldwide. Conventional initial screening methods, primarily based on prostate-specific antigen (PSA) levels, lack specificity, leading to a high rate of unnecessary biopsies and an urgent need for more accurate diagnostic tools. This study addresses the gap by exploring the potential of integrating clinical and routine blood laboratory parameters including a comprehensive hormone assessment to enhance the non-invasive prediction of aggressive PC. Methods In a pilot study of 578 patients who were scheduled for a prostate biopsy due to suspicion of PC, we analyzed an extensive panel of 28 laboratory values alongside data on family history, diet, and lifestyle. A logistic regression classifier was developed, and model performance was evaluated using repeated k-fold cross-validation on the complete dataset (n=282). Participants were histologically categorized into three risk groups: healthy, moderate PC (ISUP 1-2 PC), and aggressive PC (ISUP 3-5 PC). Results Significant associations were found between PC aggressiveness and lower levels of androstenedione, Dehydroepiandrosterone-Sulfate (DHEA-S) and free PSA percentage, as well as higher levels of Sex Hormone Binding Globulin (SHBG). The integration of these serum markers with clinical parameters into a new multi-stage risk classifier for PC prediction significantly improved the predictive accuracy. The risk model outperformed PSA-only methods, demonstrating higher sensitivity and specificity in predicting aggressive PC. Conclusions Incorporating serum markers DHEA-S, androstenedione, and SHBG into a novel risk classifier can improve early detection of aggressive PC. These widely available and cost-effective blood biomarkers could reduce reliance on invasive prostate biopsies and expensive magnetic resonance imaging by providing a more targeted approach to non-invasive prediction of aggressive PC following PSA testing. Our pilot study lays the groundwork for larger-scale research to further explore the integration of androgens and SHBG in future risk stratification models for improved clinical decision making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3254586a7ca082e9d1b49e74cbf797d18bfdeb3f" target='_blank'>
              Integration of serum androgens and Sex Hormone-Binding Globulin for optimized early detection of aggressive prostate cancer
              </a>
            </td>
          <td>
            O. Lazareva, A. Riediger, O. Stegle, H. Sueltmann, M. Hohenfellner, M. Goertz
          </td>
          <td>2024-12-06</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1861b45b76a529d7121c388efe03a05c095aeb5f" target='_blank'>
              Analysis of the predictive value of the prostate-specific antigen-to-neutrophil ratio for the diagnosis of prostate cancer
              </a>
            </td>
          <td>
            Yuxuan Chen, Haisheng Yan, Yaoqin Xu, Kexin Chen, Runqin Yang, Jiali Yang, Ruian Zhu, Rui Lin, Jiang Wang, Jie Liu, Pingsheng Gao, Lei Pang, Lexin Wang
          </td>
          <td>2025-01-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d92332ae486c3b94c41f8d10e6867cf69c3624d" target='_blank'>
              Unveiling the molecular profile of a prostate carcinoma: implications for personalized medicine
              </a>
            </td>
          <td>
            Massimiliano Agostini, Erica Giacobbi, F. Servadei, Julia Bishof, Likas Funke, Giuseppe Sica, Valentina Rovella, Marco Carilli, V. Iacovelli, Yufang Shi, Jianquan Hou, E. Candi, G. Melino, Giulio Cervelli, M. Scimeca, A. Mauriello, Pierluigi Bove
          </td>
          <td>2024-12-31</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39d9ad3dc0962f7c31b170d54804528397e6b774" target='_blank'>
              Clinical assessment of urinary prostate cancer antigen 3 in Chinese population: a large-scale, prospective and multicenter study
              </a>
            </td>
          <td>
            Xuan Shu, Jiaming Wang, Wen Cai, Shen Lin, Jiangfeng Li, Xueyou Ma, Yufan Ying, Yat Sai Terry Wang, Xiao Wang, Hong Chen, Chunyu Jin, Ben Liu, Liping Xie, Jindan Luo
          </td>
          <td>2024-12-31</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f94c0dce4bda7d7bc76945e81393c99f021f70c1" target='_blank'>
              Genomic biomarkers of survival in patients with metastatic hormone-sensitive prostate cancer undergoing intensified androgen deprivation therapy.
              </a>
            </td>
          <td>
            G. Gebrael, N. Sayegh, C. Hage Chehade, Y. Jo, Arshit Narang, B. Chigarira, N. Tripathi, Ayana Srivastava, C. Tandar, Jessica F Williams, Diya Garg, Richard Ji, Benjamin L. Maughan, Umang Swami, N. Agarwal
          </td>
          <td>2025-01-31</td>
          <td>Prostate cancer and prostatic diseases</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Despite treatment, prostate cancer commonly progresses into castration-resistant prostate cancer (CRPC), which remains largely incurable, requiring the development of new interventions. Darolutamide is an orally administered second-generation androgen receptor inhibitor indicated for patients with non-metastatic CRPC or metastatic hormone-sensitive prostate cancer. Here, we evaluated the effect of androgen receptor (AR) inhibition by darolutamide in combination with DNA double-strand-break-inducing targeted radium-223 alpha therapy in vitro and in an intratibial LNCaP xenograft model mimicking prostate cancer metastasized to bone. The results highlight the synergistic antitumor efficacy of darolutamide in combination with radium-223 both in vitro and in vivo. This effect was most likely driven by the downregulation of genes involved in DDR signaling, which was demonstrated in vitro by a gene set enrichment analysis. The combination treatment also reduced pathological tumor-induced effects in bone by decreasing the number of osteoblasts and osteoclasts and reducing abnormal bone formation in tumor-bearing bone. Additionally, it was shown that darolutamide does not affect the uptake of radium-223 into bone tissue. These results support the investigation of darolutamide in combination with radium-223 for the treatment of patients with CRPC metastasized to bone.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ce30ab729d0d77e80798b42707093a8ce4cbf2e" target='_blank'>
              Darolutamide in Combination with Radium-223 Exhibits Synergistic Antitumor Efficacy in LNCaP Prostate Cancer Models
              </a>
            </td>
          <td>
            U. Hagemann, Christoph A. Schatz, M. Suominen, A. Schlicker, M. Knuuttila, Timothy G Wilson, E. Alhoniemi, Sanna-Maria Käkönen, B. Haendler, Arne Scholz
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f81ce93d78378eddde4edc3e25f6e66eadf8ee3" target='_blank'>
              Stathmin 1 expression in neuroendocrine and proliferating prostate cancer
              </a>
            </td>
          <td>
            Yingli Shi, Yunshin A. Yeh, Siyuan Cheng, Xin Gu, Shu Yang, Lin Li, Nazih P. Khater, Susan Kasper, Xiuping Yu
          </td>
          <td>2025-01-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Breast cancer currently occupies the first place in the structure of cancer morbidity in the female population worldwide. Approximately 15–20% of breast cancer patients have overexpression and/or amplification of human epidermal growth factor receptor type 2 (HER2), which causes aggressive biological behavior of the tumor and is associated with poor prognosis. Anti-HER2 therapies have significantly improved the prognosis of patients with HER2-positive breast cancer. Starting from the earliest stages (T2 and/or N+), patients should be treated with neoadjuvant drug therapy, which allows direct assessment of treatment efficacy, increases the chances of organ-preserving surgery, and helps to tailor postneoadjuvant therapy. Anthracycline-based chemotherapy regimens have historically been the first treatment option for HER2-positive breast cancer, as HER2 status is a predictor of anthracycline efficacy. However, the development of new agents and the resulting emergence of novel combinations with multiple anti-HER2 agents have provided an opportunity to explore anthracycline-free regimens aimed at reducing adverse events and improving patient quality of life. The results of the BCIRG-006 trial investigating the adjuvant use of such regimens and, later, neoadjuvant assignment in the TRAIN-2 protocol marked a turning point in the use of non-anthracycline regimens. One important open question is the appropriateness of adding carboplatin, as direct comparisons of non-anthracycline combinations with or without carboplatin are lacking. There is now strong evidence of the impact of achieving a complete pathomorphological response (pCR) after the neoadjuvant phase of treatment on long-term oncologic outcomes. The development and validation of possible biomarkers that predict achievement of pCR is extremely promising.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0396a2b43ed6cf482496094a85af86cb41fc6bc3" target='_blank'>
              Current approaches to neoadjuvant drug therapy for primary operable HER2-positive breast cancer
              </a>
            </td>
          <td>
            M. S. Ruban, L. V. Bolotina, Yu. B. Karagodina
          </td>
          <td>2025-01-21</td>
          <td>Medical alphabet</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Patient stratification remains a challenge for optimal treatment of prostate cancer (PCa). This clinical heterogeneity implies intra-tumoural heterogeneity, with different prostate epithelial cell subtypes not all targeted by current treatments. We reported that such cell subtypes are traceable in liquid biopsies through representative transcripts. Expanding on this concept, we included 57 genes representing cell subtypes, drug targets and relevant to resistance as non-invasive biomarkers for stratification. This panel was tested by RT-qPCR (quantitative reverse transcription polymerase chain reaction) in blood of controls and different categories of PCa patients. Overall, circulating transcripts showed predictive value throughout the disease. Those with aggressive pathological features such as intra-ductal carcinoma at diagnosis showed more genes over-expressed. In metastatic patients, signatures of subtypes or resistance were associated with treatments, progression-free survival and overall survival. Altogether, testing markers of cell diversity, an intrinsic feature of tumours, and drug targets via liquid biopsies represents a valuable means to stratify patients and predict responses to current or new therapeutic modalities. Over-expressed drug target genes suggest potential benefit from targeted treatments, justifying new clinical trials to offer patient-tailored strategies to eventually impact on PCa mortality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db693f90ecf986ea4405cf60ddb76edec574f4c1" target='_blank'>
              Clinical significance of stratifying prostate cancer patients through specific circulating genes.
              </a>
            </td>
          <td>
            Seta Derderian, Edouard Jarry, Arynne Santos, Quentin Vesval, Lucie Hamel, R. Sanchez-Salas, A. Rompré-Brodeur, W. Kassouf, Raghu Rajan, F. Brimo, Marie Duclos, A. Aprikian, Simone Chevalier
          </td>
          <td>2025-01-22</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb34e831896ce5d03efd1eaef81cedce5a6fa384" target='_blank'>
              Accelerated growth and local progression of radiorecurrent prostate cancer in an orthotopic bioluminescent mouse model
              </a>
            </td>
          <td>
            Gavin Frame, Xiaoyong Huang, Roni Haas, Kabir A Khan, Hon S Leong, T. Kislinger, P. Boutros, Michelle R. Downes, Stanley K Liu
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/823fac8baba17d873c8cfc6928961f769be662ad" target='_blank'>
              Exploring the Effects of Opioid-Related Drugs on the Clinical Outcome of Prostate Cancer Patients Via Integrated Bioinformatics Analysis.
              </a>
            </td>
          <td>
            Yunxuan Zhang, Yuenan Liu, Kailei Chen, Qi-zeng Miao, Qi Cao, Xiaoping Zhang
          </td>
          <td>2025-01-20</td>
          <td>Molecular biotechnology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Background Prostate cancer is a common cancer with a variable prognosis. Its management is currently guided by histological and biological markers such as the Gleason score and PSA. Developments in molecular biology are now making it possible to identify new targets for better classification of prostate cancer. Among emerging biomarker, DNA methylation, an epigenetic process, is increasingly being studied in carcinogenesis. Techniques for analyzing DNA methylation are constantly improving, and digital PCR now allows absolute methylation quantification with high sensitivity. These techniques can be performed on circulating tumor DNA. Materials & Methods We conducted a literature review of scientific articles addressing the topic of DNA methylation in prostate cancer. Results & discussion This review summarizes the different genes whose methylation is involved in carcinogenesis and their clinical implications, both diagnostic and prognostic. Methylation monitoring could also be useful for the prediction of treatment response. However, most studies are retrospective, and prospective studies are needed to validate these data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59c901f5c6197904e2aeab3c865a402741cdc3a" target='_blank'>
              DNA Methylation in Prostate Cancer: Clinical Implications and Potential Applications
              </a>
            </td>
          <td>
            Romane Muletier, C. Bourgne, Laurent Guy, Aurore Dougé
          </td>
          <td>2025-01-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Purpose: The standard criteria available for the diagnosis of prostate cancer often do not sufficiently predict the course or outcome of this common disease. Evaluating molecular tissue markers, especially those that mark apoptosis and proliferation, in an integrated mathematical model, could contribute to a better understanding of the development and pathogenesis of this disease, aid in the selection of adequate therapies, and open new avenues of therapeutic research. 
Materials and methods: Paraffin-embedded prostatectomy specimens were stained according to an immunohistochemical protocol for p53, bcl-2, and CD105. A mathematical model was developed, which incorporated values of tissue markers according to their apoptotic/proliferative characteristics. 
Results: The results of tissue marker expression and mathematical modeling were correlated with patient clinical data. We showed that tissue marker values correlated better with clinical data when tissue markers values were incorporated in a mathematical model than when applied alone.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5759f34397366d68e94b56b915b98f78f69c6eda" target='_blank'>
              Molecular tissue markers integrated: A mathematical model approach to evaluating and predicting prostate cancer using molecular tissue markers
              </a>
            </td>
          <td>
            Mihael Munda
          </td>
          <td>2024-12-24</td>
          <td>Acta Medico-Biotechnica</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The global incidence of prostate cancer (PCa) is rising. Localized PCa can be managed through surgical intervention or radiotherapy, but certain patients may experience recurrence or develop metastatic disease following localized treatment. Despite aggressive therapeutic approaches, the majority of metastatic patients with PCa will eventually progress to metastatic castration‑resistant PCa, with limited treatment alternatives and a dismal prognosis. The treatment options for advanced PCa are continuously evolving, yet there remains a demand for further innovative therapeutic approaches. Antibody‑drug conjugates (ADCs) represent a novel class of targeted medications comprising a humanized monoclonal antibody, a linker and a cytotoxic payload. ADCs primarily bind to antigens that are upregulated on the surface of PCa cells but are minimally expressed on normal cells. At present, a variety of targets for ADCs have been identified in the treatment of PCa, encompassing prostate‑specific membrane antigen, STEAP family member 1, trophoblast cell‑surface antigen 2, CD46, B7‑H3, tissue factor and delta‑like protein 3, each with one or more specific ADC that has shown encouraging results in the PCa field. In the present review, the developmental course, composition and mechanism of action of ADCs are explored, with a specific focus on recently published studies and ongoing trials aimed at investigating the efficacy and safety of ADCs in treating PCa. Lastly, ongoing challenges in ADC development and corresponding strategies to combat them are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb6647ef3243af9b93ecaada082dfb6baa67d85" target='_blank'>
              Antibody‑drug conjugates in prostate cancer: Emerging strategies to enhance therapeutic index and current clinical landscape (Review).
              </a>
            </td>
          <td>
            Chadanfeng Yang, Limei Wang, Chen Gong, Dihao Lv, Haihao Li, Yinglong Huang, Jiting Li, Wujie Chen, Shi Fu, Zhiyong Tan, Mingxia Ding
          </td>
          <td>2024-12-13</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a heterogeneous disease that exhibits androgen sensitivity and responsiveness to androgen deprivation therapy (ADT). However, ADT induces only temporary remission, and the majority of PCa cases eventually progress to castration-resistant PCa (CRPC). During the development and progression of CRPC, androgen sensitivity and androgen receptor (AR) dependency in PCa cells are often deceased or lost due to ADT or spontaneously arising AR variants even before starting ADT. To prevent CRPC, a clinical PCa model derived from an AR-positive cancer cell line with weak or no androgen sensitivity is required. The human prostate LNCaP cell line is a good model for PCa because of its androgen sensitivity and AR dependency in terms of cell growth and gene expression. Notably, LNCaP cells are heterogeneous cells comprising different clones with natural variations in androgen sensitivity and AR dependency resulting from spontaneously occurring changes. In our group, to obtain androgen-insensitive or weakly sensitive clones spontaneously derived from parental LNCaP cells, we performed a limiting dilution of parental LNCaP cells and obtained several sublines with varying levels of androgen sensitivity and AR dependency. In addition, we established an androgen-insensitive subline from parental LNCaP cells by continuous passage under hormone-depleted conditions. This article provides a unique perspective on our original PCa progression model interacting with fibroblasts and its application in preclinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3bad0e1df464a656ec2794ad0a821d8db32a285" target='_blank'>
              Application of Original Prostate Cancer Progression Model Interacting with Fibroblasts in Preclinical Research
              </a>
            </td>
          <td>
            K. Ishii, K. Iguchi, Chise Matsuda, Y. Hirokawa, Yoshiki Sugimura, Masatoshi Watanabe
          </td>
          <td>2024-12-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background: Prostate cancer is a major global health burden, with biochemical recurrence (BCR) following radical prostatectomy affecting 20–40% of patients and posing significant challenges to prognosis and treatment. Emerging evidence suggests a critical role for differentially expressed in normal and neoplastic cell (DENN) domain-containing genes in oncogenesis; however, their implications in prostate cancer and BCR risk remain underexplored. Methods: This study systematically evaluated 151 single-nucleotide polymorphisms in DENN domain-containing genes in 458 patients with prostate cancer and BCR, followed by validation in an independent cohort of 185 patients. Results: Multivariate Cox regression analyses identified DENND2D rs610261 G>A as significantly associated with improved BCR-free survival in both cohorts (adjusted hazard ratio = 0.39, 95% confidence interval = 0.23–0.66, p = 0.001). Functional analysis revealed rs610261’s regulatory potential, with the protective A allele correlating with increased DENND2D expression in various human tissues. Compared to normal prostate tissues, DENND2D expression was reduced in prostate cancer, with higher expression being linked to favorable patient prognosis (p = 0.03). Gene set enrichment analysis revealed an association between DENND2D expression and the negative regulation of MYC target genes, including MAD2L1, ERH, and CLNS1A, which are overexpressed in prostate cancer and associated with poor survival. Furthermore, the elevated DENND2D expression promotes immune infiltration in prostate cancer, supporting its role in immune modulation. Conclusions: DENND2D is a prognostic biomarker for BCR in prostate cancer and offers new avenues for personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac27074b32a882cf963d9c0a17256435bfb014b4" target='_blank'>
              Unveiling DENND2D as a Novel Prognostic Biomarker for Prostate Cancer Recurrence: From Gene to Prognosis
              </a>
            </td>
          <td>
            Chi-Fen Chang, Lih‐Chyang Chen, Yei-Tsung Chen, Chaoyuan Huang, Chia-Cheng Yu, V. C. Lin, Te-Ling Lu, Shu-Pin Huang, B. Bao
          </td>
          <td>2024-12-26</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61ad3571a382fc7c0257f7357d8200c5c78b4492" target='_blank'>
              Targeting mitochondria and programmed cell death as potential interventions for metastatic castration-resistant prostate cancer.
              </a>
            </td>
          <td>
            Amonlaya Amantakul, Akara Amantakul, S. Pojchamarnwiputh, N. Chattipakorn, S. Chattipakorn, J. Sripetchwandee
          </td>
          <td>2024-12-16</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Prostate cancer is the most commonly diagnosed cancer and a leading cause of cancer-related mortality among men in the United States. Androgen deprivation therapy (ADT) has long been the cornerstone of treatment for advanced prostate cancer. However, most patients eventually develop resistance, leading to progression into the lethal castration-resistant prostate cancer (CRPC) stage, which is associated with poor survival rates. While next-generation androgen receptor signaling inhibitors (ARSIs), such as enzalutamide and abiraterone, have revolutionized CRPC treatment, resistance to these therapies remains a significant challenge, rendering the disease incurable. Recent studies have identified Polo-like kinase 1 (PLK1), a master regulator of mitosis, as a key driver of prostate cancer progression, particularly in CRPC. PLK1 activation has been linked to critical pathways, including oxidative stress responses, lipid metabolism, and androgen receptor (AR) signaling, which collectively promote tumor growth and ARSIs resistance. Moreover, PLK1 can regulate chromosomal stability and DNA damage repair pathways, contributing to sensitivity to irradiation therapy. Additionally, PLK1 plays a role in driving resistance to targeted therapies, such as PARP inhibitors, in BRCA-mutant CRPC. Compelling evidence further suggests that combining PLK1 inhibition with paclitaxel could serve as an effective therapeutic strategy to overcome resistance in CRPC by targeting microtubule dynamics. This review examines the mechanistic role and broader implications of PLK1 in CRPC treatment, offering valuable insights into potential combination therapies to improve efficacy and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be1739c67f1f729180f6441e8a334eb27ea83080" target='_blank'>
              Plk1, a promising THERAPEUTIC target for prostate cancer treatment
              </a>
            </td>
          <td>
            Jia Peng, Mansoureh Nouri, Hamed Maasoumyhaghighi, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-17</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND
The efficacy of abiraterone acetate varies among patients with high-risk metastatic castration-sensitive prostate cancer (mCSPC). Both androgen receptor (AR) and cytokeratin 18 (CK18) are markers of the luminal lineage of prostate cancer, and their expression levels have been suggested to affect the response to androgen deprivation therapy (ADT). This study aimed to predict the efficacy of abiraterone acetate in high-risk mCSPC via immunohistochemical staining of biopsy specimens obtained at the time of prostate cancer diagnosis.


METHODS
We retrospectively analyzed 44 patients treated with abiraterone acetate in combination with ADT. AR and CK18 expression in prostate biopsy specimens were assessed using immunohistochemical staining.


RESULTS
AR and CK18 staining was not significantly associated with overall survival (OS). However, low AR staining was significantly associated with a shorter time to castration-resistant prostate cancer (TTCRPC) compared with high AR staining (log-rank test, p = 0.018). Similarly, low CK18 staining was significantly associated with a shorter TTCRPC compared with high CK18 staining (log-rank test, p = 0.037).


CONCLUSIONS
Immunohistochemical analysis of AR or CK18 expression in biopsy specimens may serve as a predictive biomarker of high-risk mCSPC treated with abiraterone acetate.


TRIAL REGISTRATION
None.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc6a0a33f858f3e308765431c1f90fef4309b55" target='_blank'>
              Immunohistochemical Analysis of Androgen Receptor Expression Predicts the Prognosis of Metastatic Castration-Sensitive Prostate Cancer Patients Receiving Abiraterone Acetate.
              </a>
            </td>
          <td>
            Mitsuhisa Nishimoto, M. D. De Velasco, Yutaka Yamamoto, S. Fujimoto, Yasunori Akashi, Shingo Toyoda, Mamoru Hashimoto, Shogo Adomi, E. Banno, Y. Saito, Takafumi Minami, Akihide Hirayama, K. Yoshimura, H. Uemura, Kazutoshi Fujita
          </td>
          <td>2025-01-31</td>
          <td>The Prostate</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Glioblastoma is the most common primary intracranial malignant tumor, classified by the World Health Organization as a grade IV glioma of the central nervous system due to its high mortality, recurrence rate, and morbidity. This disease has difficulty to treat. Currently commonly used therapies, such as surgery, chemotherapy and radiotherapy, have poor prognosis. Therefore, new treatments are needed to control tumor progression and improve patient outcomes. After years of development, cancer vaccines have gradually become an important part of tumor immunotherapy, and they have shown a more rapid development trend in the current era. Cancer vaccines introduce tumor antigens, genes encoding tumor antigens, immune cells, and molecules into the body. Cancer vaccines activate specific immune responses that control tumor growth and ultimately clean it. In recent years, a number of research groups at home and abroad have explored the safety and feasibility of different types of therapeutic vaccines for the treatment of GBM according to the characteristics of GBM. This review introduces results, mechanism, safety and efficacy of virus-like particles (VLPs) vaccine, mRNA vaccine and dendritic cells (DC) vaccine. The feasibility of using these vaccines to treat GBM was discussed to provide reference for further research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/997912c61b4c890ec8bb13a132b53de0daa3f614" target='_blank'>
              Emerging Role of Vaccines in Glioblastoma Treatment
              </a>
            </td>
          <td>
            Ruidong Guo
          </td>
          <td>2024-12-24</td>
          <td>Transactions on Materials, Biotechnology and Life Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dde8ccf4e0d0a3dff90375c5f0667b0c93c98779" target='_blank'>
              Exploring the role of pomalidomide in androgen-dependent prostate cancer: a computational analysis.
              </a>
            </td>
          <td>
            Shivani Pathak, Vipendra Kumar Singh, P. Gupta, Arun Kumar Mahapatra, Rajanish Giri, Rashmi Sahu, Rohit Sharma, Neha Garg
          </td>
          <td>2024-12-21</td>
          <td>Molecular diversity</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Prostate cancer is one of the most prevalent malignant tumors in men, particularly in regions with a high Human Development Index. While the long-term survival rate for localized prostate cancer is relatively high, the mortality rate remains significantly elevated once the disease progresses to advanced stages, even with various intensive treatment modalities. The primary obstacle to curing advanced prostate cancer is the absence of comprehensive treatment strategies that effectively target the highly heterogeneous tumors at both genetic and molecular levels. Prostate cancer development is a complex, multigenic, and multistep process that involves numerous gene mutations, alteration in gene expression, and changes in signaling pathways. Key genetic and pathway alterations include the amplification and/or mutation of the androgen receptor, the loss of Rb, PTEN, and p53, the activation of the WNT signaling pathway, and the amplification of the MYC oncogene. This review summarizes the mechanisms by which these genes influence the progression of prostate cancer and highlights the interactions between multiple genes and their relationship with prostate cancer. Additionally, we reviewed the current state of treatments targeting these genes and signaling pathways, providing a comprehensive overview of therapeutic approaches in the context of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38fab9adcd074d173268fdd423c799e2e23c962b" target='_blank'>
              Interactions between key genes and pathways in prostate cancer progression and therapy resistance
              </a>
            </td>
          <td>
            Fan Wu, Heng Zhang, Miaomiao Hao
          </td>
          <td>2025-01-23</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Androgen-indifferent prostate cancer (AIPC) is increasingly common and particularly lethal. Data describing these tumors are sparse, and AIPC remains a poorly understood malignancy. Utilizing the Oncology Research Information Exchange Network (ORIEN) database, we enriched for tumors with features of AIPC using previously described characteristics. Our AIPC cohort included three subgroups: aggressive variant prostate cancer (AVPC), neuroendocrine PC (NEPC), and double-negative PC (DNPC). Of 1496 total PC patients available for analysis, we identified 323 (22%) as MCRPC. Of those, 39 (12%) met AIPC criteria (17 AVPC, 13 NEPC, 9 DNPC) and 284 (88%) were non-AIPC. Forty-three percent of AIPC patients had de novo metastatic disease vs. 15% for non-AIPC (p = 0.003). Homologous recombination deficiency (HRD) and tumor mutational burden (TMB) did not differ between cohorts, but microsatellite instability scores (MSI) were significantly higher in AIPC (p = 0.019). Using Gene Set Enrichment Analysis (GSEA), we found that genes defining response to androgens and genes involved in oxidative phosphorylation were the most downregulated, whereas genes involved in epithelial–mesenchymal transition (EMT) and immune signaling were significantly upregulated in AIPC vs. non-AIPC. Our study demonstrates the potential for predefined criteria that aim to enrich for AIPC and suggests opportunities for therapeutic investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7c8e30f8cc18e46120ea1d42d1f7f591cfde2eb" target='_blank'>
              Clinical and Genomic Features of Androgen Indifferent Prostate Cancer
              </a>
            </td>
          <td>
            Jack Masur, A. Ratan, Krzysztof Wierbiłowicz, Adanma Ayanambakkam, Michelle L Churchman, Laura S. Graham, G. D. Grass, Sumati Gupta, Sean Kern, Jennifer King, Zin W. Myint, Robert J Rounbehler, B. Salhia, Eric A Singer, Yousef Zakharia, B. Paschal, Paul V. Viscuse
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Pancreatic cancer remains a highly malignant and challenging tumor with a dismal 5-year survival rate of only 13%. The majority of patients are diagnosed at advanced stages, where surgical options are limited, and prognosis is poor. Immunotherapy, particularly PD-1 inhibitors, has shown limited success in pancreatic cancer due to its unique tumor immune microenvironment. However, certain genetic profiles, such as BRCA1/2 mutations, high tumor mutational burden (TMB), or microsatellite instability-high (MSI-H), may enhance sensitivity to these therapies. This report presents two cases of advanced pancreatic cancer with BRCA1/2 mutations treated with a combination of chemotherapy and immune checkpoint inhibitors. The first patient, with TMB-H and stable microsatellites, achieved complete remission after conversion therapy and remains disease-free for over two years post-surgery. The second patient, with MSI-H and low TMB, experienced significant tumor regression and improved quality of life with a prolonged progression-free survival, although the patient ultimately declined surgery. These cases suggest that combined chemotherapy and immunotherapy may offer a promising treatment option for select pancreatic cancer patients, particularly those with specific genetic profiles, warranting further investigation into personalized approaches to immunotherapy in this malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a93c4c22949785a73a6a2556482978253791d306" target='_blank'>
              Case report: Achieving significant tumor reduction in advanced pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Hongying Liu, Yun-Long Wang, Qian Zhang, Nengwen Ke
          </td>
          <td>2024-12-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b351a91b1b339944d457e7db859962fb6728ec9" target='_blank'>
              Patient-derived xenografts from circulating cancer stem cells as a preclinical model for personalized pancreatic cancer research
              </a>
            </td>
          <td>
            Benedikt J Wagner, Andreas Ettner-Sitter, Nicolas A Ihlo, Merle Behr, Sebastian Koelbl, Stefan M Brunner, Florian Weber, Bettina M Rau, Hans J Schlitt, Christoph Brochhausen, Rebecca Schoenmehl, Annalena Artinger, D. Schott, M. Pizon, Katharina Pachmann, Thiha Aung, Silke Haerteis, Christina Hackl
          </td>
          <td>2025-01-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Immunotherapy represents a groundbreaking therapeutic approach, based on the immune system’s intrinsic capacity to interfere with tumor progression, that opens the horizons in the treatment of endometrial cancer. However, the clinical efficacy of immunotherapy is hampered by the development of resistance in patients. The resistance to immunotherapy is multifactorial mechanism, encompassed genetic and epigenetic alterations in tumor cells modulating immune checkpoint molecules, resulted in escaping immune surveillance. The tumor microenvironment can orchestrate an immunosuppressive milieu, attenuating the immune response and facilitating tumor progression. To overcome immunotherapeutic resistance in endometrial cancer we must bring to light the mechanisms of intricate interplay between neoplastic cells, the host immune system, and the tumor microenvironment. The identification of predictive biomarkers for immunotherapeutic response and the innovative agents capable of reversing resistance pathways must be developed. Our review summarizes accumulated data on the role of cells of the tumor microenvironment and their regulatory molecules in the mechanisms underlying therapeutic effects of immune checkpoint inhibitors, including resistance to therapy. Major question we raise here – which group of patients is the most favorable to achieve durable immunotherapy response in endometrial cancer?">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cd4ff458f46525f9be97feafd21ef2784338c06" target='_blank'>
              Predicting immunotherapy efficacy in endometrial cancer: focus on the tumor microenvironment
              </a>
            </td>
          <td>
            L. Tashireva, I. V. Larionova, N. A. Ermak, A. Maltseva, Ekaterina I. Livanos, A. Kalinchuk, M. Stakheyeva, L. Kolomiets
          </td>
          <td>2025-01-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Adoption of immunotherapy has completely transformed the treatment landscape of cancer. Patients with advanced cancer treated with immunotherapy may benefit from durable tumor response and long-term survival. The most widely used immunotherapy in solid tumors is anti-programmed-death (ligand) protein (PD-(L)1), which is now an integral part of non-small cell lung cancer (NSCLC) treatment irrespective of histological cell types and tumor stage. However, the vast majority of patients with advanced NSCLC treated with anti-PD-(L)1 still develop therapeutic resistance, and the prognosis after anti-PD-(L)1 resistance is poor. Resistance mechanisms to PD-1 blockade are often complex and encompass a combination of defects within the cancer-immunity cycle. These defects include failure in antigen presentation and T-cell priming, presence of co-inhibitory immune checkpoints, inability of immune cells to infiltrate the tumor, and presence of immunosuppressive tumor microenvironment. Recently, advances in drug design, genomic sequencing, and gene editing technologies have led to development of next-generation immunotherapies that may potentially overcome these resistance mechanisms. In this review, we will discuss the anti-PD-(L)1 resistance mechanism landscape in NSCLC and four novel modalities of immunotherapy in detail, namely novel immune checkpoint inhibitor and targeted therapy combinations, bispecific antibodies, cancer vaccine, and cell therapy. These novel therapeutics have all demonstrated early clinical data in NSCLC treatment and may work synergistically with each other to restore anticancer immunity. In addition, we share our perspectives on the future promises and challenges in the transformation of these novel immunotherapies to standard clinical care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdef5bc62054de2d611913bc2fb5655a0f727fb9" target='_blank'>
              Next-generation immunotherapy: igniting new hope for lung cancer
              </a>
            </td>
          <td>
            M. S. Li, Andrew L S Chan, Kevin K S Mok, L. Chan, Tony S K Mok
          </td>
          <td>2024-01-01</td>
          <td>Therapeutic Advances in Medical Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Immunotherapy has revolutionized cancer treatment, but predicting patient response remains challenging. Herein, we present iPRISM (Intelligent Predicting Response to cancer Immunotherapy through Systematic Modeling), which is a novel network‐based model that integrates multiomics data to predict immunotherapy outcomes. In this approach, iPRISM incorporates gene expression, biological functional network, tumor microenvironment characteristics, immune‐related pathways, and clinical data to provide a comprehensive view of factors influencing immunotherapy efficacy. Using stepwise logistic regression, we identified key predictive features and validated iPRISM across multiple cohorts including melanoma, bladder cancer, non‐small cell lung cancer, and stomach adenocarcinoma. We also find that iPRISM outperforms the existing methods, achieving high predictive accuracy and demonstrating significant prognostic value for overall and progression‐free survival. By identifying key genetic and immunological factors, this model provides a new insight for more personalized treatment strategies and combination therapies to overcome resistance mechanisms. iPRISM can be accessed at CRAN: https://CRAN.R‐project.org/package=iPRISM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed045e8d91285b3e11584131d35ba5346ad63121" target='_blank'>
              iPRISM: Intelligent Predicting Response to Cancer Immunotherapy through Systematic Modeling
              </a>
            </td>
          <td>
            Yinchun Su, Siyuan Li, Qian Wang, Bingyue Pan, Jiyin Lai, Guangyou Wang, Junwei Han, Qingfei Kong
          </td>
          <td>2025-01-14</td>
          <td>Advanced Intelligent Systems</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cancer is a burning problem in the modern health field. In this research mainly we focused on how we can treat for a cancer by using chemotherapy and immune therapy as individual monotherapies and as a combined therapy. In previously, researchers have constructed mathematical models to analyse the combined therapy treatment with saturation effects for immune therapy treatment but here we introduce mass action kinetics for immune therapy and this model reflects as a continues attacking process to tumour cells by immune cells with the help of chemotherapy drug. We introduce some threshold levels which we can remove cancer completely and control the cancer in a constant level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/161cbe6543ac6ec0ae6bbad4c65a8698e774643d" target='_blank'>
              A Mathematical Model to Treat for a Cancer Using Chemotherapy and Immunotherapy under Mass Action Kinetics for Immunotherapy
              </a>
            </td>
          <td>
            A. M. D. Clotilda, G. V. R. K. Vithanage, D. D. Lakshika
          </td>
          <td>2024-12-19</td>
          <td>MOLECULAR SCIENCES AND APPLICATIONS</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aggressive variant prostate cancer (AVPC) is characterized by a molecular signature involving combined defects in TP53, RB1, and/or PTEN (AVPC-TSGs), identifiable through immunohistochemistry or genomic analysis. The reported prevalence of AVPC-TSG alterations varies widely, reflecting differences in assay sensitivity, treatment pressure, and disease stage evolution. Although robust clinical evidence is still emerging, the study of AVPC-TSG alterations in prostate cancer (PCa) is promising. Alterations in TP53, RB1, and PTEN, as well as the combined loss of AVPC-TSGs, may have significant implications for prognosis and treatment. These biomarkers might help predict responses to various therapies, including hormonal treatments, cytotoxic agents, radiotherapy, and targeted therapies. Understanding the impact of these molecular alterations in patients with PCa is crucial for personalized management. In this review, we provide a comprehensive overview of the emerging prognostic and predictive roles of AVPC-TSG alterations across PCa stages. Moreover, we discuss the implications of different methods used for detecting AVPC-TSG alterations and summarize factors influencing their prevalence. As our comprehension of the genomic landscape of PCa disease deepens, incorporating genomic profiling into clinical decision making will become increasingly important for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc76168bc6cc3f716d4ad9c939472b94406c6f4f" target='_blank'>
              The Emerging Predictive and Prognostic Role of Aggressive-Variant-Associated Tumor Suppressor Genes Across Prostate Cancer Stages
              </a>
            </td>
          <td>
            Martino Pedrani, Jessica Barizzi, Giuseppe Salfi, Alessandro Nepote, Irene Testi, Sara Merler, Luis Castelo-Branco, R. P. Mestre, Fabio Turco, Luigi Tortola, Jean-Philippe P. Theurillat, S. Gillessen, U. Vogl
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Simple Summary With the discovery of immunotherapy (treatments that increase the immune system’s ability to kill cancer cells) and targeted therapy (treatments that target the specific pathway the cancer uses to grow and survive), we have made great progress in the treatment of advanced kidney cancer. However, their roles in a localized setting (a tumor limited to the kidney and/or lymph nodes around the kidney) are not well defined. This review will look at the available data on the use of immunotherapy and targeted therapy for patients with localized kidney cancer, where the goals may be to decrease tumor size, to make surgery easier and less invasive, to save as much kidney function as possible, and/or to kill the cancer cells that may have escaped the kidney.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2476f892f32a000cbcb6f0c49b52d56b374d134" target='_blank'>
              A Review of Neoadjuvant Therapy for Localized and Locally Advanced Renal Cell Carcinoma
              </a>
            </td>
          <td>
            Qian Qin, Isamu Tachibana, V. Margulis, Jeffrey A. Cadeddu, Tian Zhang
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>66</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [1, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>