<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../RA/">
      
      
        <link rel="next" href="../Virtual%20patients/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Prostate cancer
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Virtual%20cell/" class="md-tabs__link">
        
  
    
  
  Virtual cell

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../RA/" class="md-tabs__link">
        
  
    
  
  RA

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Prostate cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Virtual%20patients/" class="md-tabs__link">
        
  
    
  
  Virtual patients

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Virtual%20cell/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual cell
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../RA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RA
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Prostate cancer
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Virtual%20patients/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual patients
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Prostate cancer</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-29 08:54:46 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Prostate cancer</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Prostate cancer</a><br>
      <a href="#recommended_articles">3. Recommended articles on Prostate cancer</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Prostate cancer</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Prostate cancer</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Background Therapeutic vaccination against disseminated prostate cancer (PCa) is partially effective in some PCa patients. We hypothesized that the efficacy of treatment will be enhanced by individualized vaccination regimens tailored by simple mathematical models. Methodology/Principal Findings We developed a general mathematical model encompassing the basic interactions of a vaccine, immune system and PCa cells, and validated it by the results of a clinical trial testing an allogeneic PCa whole-cell vaccine. For model validation in the absence of any other pertinent marker, we used the clinically measured changes in prostate-specific antigen (PSA) levels as a correlate of tumor burden. Up to 26 PSA levels measured per patient were divided into each patient's training set and his validation set. The training set, used for model personalization, contained the patient's initial sequence of PSA levels; the validation set contained his subsequent PSA data points. Personalized models were simulated to predict changes in tumor burden and PSA levels and predictions were compared to the validation set. The model accurately predicted PSA levels over the entire measured period in 12 of the 15 vaccination-responsive patients (the coefficient of determination between the predicted and observed PSA values was R 2 = 0.972). The model could not account for the inconsistent changes in PSA levels in 3 of the 15 responsive patients at the end of treatment. Each validated personalized model was simulated under many hypothetical immunotherapy protocols to suggest alternative vaccination regimens. Personalized regimens predicted to enhance the effects of therapy differed among the patients. Conclusions/Significance Using a few initial measurements, we constructed robust patient-specific models of PCa immunotherapy, which were retrospectively validated by clinical trial results. Our results emphasize the potential value and feasibility of individualized model-suggested immunotherapy protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c181942f9c3307163599684cc55f9a9c096219a9" target='_blank'>
                Predicting Outcomes of Prostate Cancer Immunotherapy by Personalized Mathematical Models
                </a>
              </td>
          <td>
            N. Kronik, Y. Kogan, M. Elishmereni, Karin Halevi-Tobias, S. Vuk-Pavlović, Z. Agur
          </td>
          <td>2010-12-08</td>
          <td>PLoS ONE</td>
          <td>117</td>
          <td>35</td>

            <td><a href='../recommendations/c181942f9c3307163599684cc55f9a9c096219a9' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c453e732e4940c2b85b4ef9a46c4d3bf3b77936" target='_blank'>
                Parameter identification of a model for prostate cancer treated by intermittent therapy.
                </a>
              </td>
          <td>
            Clément Draghi, F. Denis, A. Tolédano, C. Letellier
          </td>
          <td>2019-01-01</td>
          <td>Journal of theoretical biology</td>
          <td>3</td>
          <td>36</td>

            <td><a href='../recommendations/6c453e732e4940c2b85b4ef9a46c4d3bf3b77936' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Prostate cancer'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Prostate cancer</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="BACKGROUND
Prostate-specific antigen (PSA) kinetics has been investigated as a prognostic marker in post hoc analyses of clinical trials. This study validated the prognostic value of rapid and deep PSA decline in metastatic hormone-sensitive prostate cancer (mHSPC) using real-world data.


METHODS
In total, 1296 patients with mHSPC were retrospectively reviewed. We assessed the prognostic value of a PSA decline to ≤ 0.2 ng/mL after 12 weeks of treatment and investigated several potential risk factors for a poor PSA response.


RESULTS
Of 1296 patients, 714 (cohort 1: 55.1%) were treated with conventional hormonal therapy, while 582 (cohort 2: 44.9%) received androgen signaling inhibitors. There were significant differences in progression-free survival and overall survival between patients with PSA decline to ≤ 0.2 ng/mL by 12 weeks of treatment and others (p < 0.001 for each). In addition, patients with an initial PSA ≥ 200 ng/mL, Clinical T4 and Grade Group 5 were less likely to achieve PSA decline to ≤ 0.2 ng/mL by 12 weeks of treatment, with odds ratios of 0.31 (p < 0.001), 0.67 (p = 0.039) and 0.70 (p = 0.043), respectively.


CONCLUSION
Our findings suggested that PSA decline to ≤ 0.2 ng/mL by 12 weeks of treatment may be a useful prognostic biomarker for mHSPC in the real-world setting. The prognostic value of this should be further investigated in a prospective cohort, and identification of an optimal cutoff value is necessary for its application in clinical trial design or clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed73e5db844c11c13f937c2018307bd7b8c8ee4a" target='_blank'>
              Rapid and Deep Prostate-Specific Antigen Decline is a Prognostic Marker in Metastatic Hormone-Sensitive Prostate Cancer: A Real-World Multi-Intuitional Analysis.
              </a>
            </td>
          <td>
            Kotaro Suzuki, T. Hara, H. Watanabe, K. Nakane, Kiyoshi Takahara, T. Naiki, T. Yasui, R. Shiroki, Takuya Koie, Hideaki Miyake
          </td>
          <td>2024-12-27</td>
          <td>The Prostate</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa72a42061c33f14558fe96c7d73cbe119e62a50" target='_blank'>
              Predicting intermediate-risk prostate cancer using machine learning.
              </a>
            </td>
          <td>
            M. Stojadinovic, Milorad Stojadinovic, Slobodan Jankovic
          </td>
          <td>2025-01-03</td>
          <td>International urology and nephrology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76012f1c5791d6df11bd6e8fe93bfd3261005d7e" target='_blank'>
              Prostate-specific antigen screening at low thresholds of men with pathogenic BRCA1/2 variants.
              </a>
            </td>
          <td>
            H. V. Stroomberg, K. Brasso, A. A. Blak, A. Byrjalsen, T. V. O. Hansen, Andreas Røder
          </td>
          <td>2025-01-21</td>
          <td>Prostate cancer and prostatic diseases</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a8ec3ff03860e7ef5588a739968da229a8489fd" target='_blank'>
              Prognostic Testing for Prostate Cancer-A Cost-Effectiveness Analysis Comparing a Prostatype P-Score Biomarker Approach to Standard Clinical Practice.
              </a>
            </td>
          <td>
            A. Fridhammar, Oskar Frisell, Karin Wahlberg, Emelie Berglund, Pontus Röbeck, Sofie Persson
          </td>
          <td>2025-01-11</td>
          <td>PharmacoEconomics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background: To establish a machine learning (ML) model for predicting prostate biopsy outcomes using prostate-specific antigen (PSA) values, multiparametric magnetic resonance imaging (mpMRI) findings, and hematologic parameters. Methods: The medical records of the patients who had undergone a prostate biopsy were evaluated. Laboratory findings, mpMRI findings, and prostate biopsy results were collected. Patients with benign prostate pathology were classified as Group 1, and those with prostate cancer (PCa) were classified as Group 2. The following ML algorithms were used to create the ML model: ExtraTrees classifier, Light Gradient-Boosting Machine (LGBM) classifier, eXtreme Gradient Boosting (XGB) classifier, Logistic Regression, and Random Forest classifier. Results: A total of 244 male patients who met the inclusion criteria were included in this study. Among them, 171 (71.1%) were categorized in Group 1, and 73 (29.9%) in Group 2. The LGBM classifier model demonstrated the highest performance, achieving an accuracy rate of 81.6% and an AUC–ROC (area under the curve–receiver operating characteristic) of 78.4%, with sensitivity and specificity values of 66.7% and 88.2%, respectively, in predicting prostate biopsy outcomes. Conclusions: Pathological results can be predicted by ML models using PSA values, mpMRI findings, and hematologic parameters prior to a prostate biopsy, potentially reducing unnecessary biopsy procedures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc5cd8a9e47869c7ef74561e32e26a158e97f12c" target='_blank'>
              Machine Learning-Based Prediction of Prostate Biopsy Necessity Using PSA, MRI, and Hematologic Parameters
              </a>
            </td>
          <td>
            Mustafa Sungur, Aykut Aykaç, Mehmet Erhan Aydın, Ozer Celik, Coşkun Kaya
          </td>
          <td>2024-12-31</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Objectives to construct a prediction model for clinically significant prostate cancer (csPCa) based on prostate-specific antigen (PSA) levels, digital rectal examination (DRE), and transrectal ultrasonography (TRUS). Methods We retrospectively analysed 1196 Asian patients who underwent transrectal ultrasound-guided biopsy (TRUSB) between June 2000 and February 2023. Patients were randomly divided into a training set of 837 cases (70%) and a validation set of 359 patients (30%). A csPCa risk prediction model was established using the logistic regression. The performance of the model was examined based on calibration curves, receiver operating characteristic (ROC) curves, decision curve analysis (DCA), and clinical impact curves (CIC). Results Serum PSA levels, age, DRE results, prostatic shape, prostatic border and hypoechoic area were associated with pathological outcomes. The area under the ROC curve of the training set was 0.890 (95%CI: 0.865-0.816). The optimal cut-off value was 0.279. The calibration curves indicated good calibration, and the DCA and CIC results demonstrated good clinical utility. Significantly, the prediction model has higher negative predictive value (89.8%) and positive predictive value (68.0%) compared with MRI. Subsequently, we developed an online calculator (https://jiwentong0.shinyapps.io/dynnomapp/) with six variables for biopsy optimization. Conclusion This study incorporated the results of three traditional diagnostic methods to establish a cost-effective and highly accurate model for predicting csPCa before biopsy. With this model, we aim to provide a non-invasive and cost-effective tool for csPCa detection in Asia and other underdeveloped areas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b86d00009bbf7d99e3325c5ad947b1d59f371a89" target='_blank'>
              Construction of a clinically significant prostate cancer risk prediction model based on traditional diagnostic methods
              </a>
            </td>
          <td>
            Wen-Tong Ji, Yong-Kun Wang, Zhan-Yang Han, Si-Qi Wang, Yao Wang
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="OBJECTIVE
To assess the association between prostate-specific antigen (PSA) density (PSAD) and prostate cancer mortality after a benign result on systematic transrectal ultrasonography (TRUS)-guided prostate biopsy.


PATIENTS AND METHODS
This retrospective study used data from the Finnish Randomised Study of Screening for Prostate Cancer (FinRSPC) collected between 1996 and 2020. We identified men aged 55-71 years randomised to the screening arm with PSA ≥4.0 ng/mL and a benign systematic TRUS-guided biopsy result. The cumulative prostate cancer mortality of men stratified by a PSAD cutoff of 0.15 ng/mL/cm3 was modelled with competing risk functions. The ability of PSAD, PSA, and base variables (age at biopsy, DRE result, socioeconomic status, 5α-reductase inhibitor usage, family history, and Charlson Comorbidity Index (CCI)) to predict prostate cancer death was compared using c-statistics and a likelihood ratio test.


RESULTS
After excluding 10 men without PSA data within 2 years of the biopsy and 65 without prostate volume data, 2276 men were eligible for inclusion in the study. A total of 50 men died from prostate cancer and 1028 from other causes during a median (interquartile range) follow-up of 17.4 (13.2-20.9) years. The cumulative prostate cancer mortality of men with PSAD <0.15 ng/mL/cm3 was significantly lower than that of men with PSAD ≥0.15 ng/mL/cm3: 0.5% (95% confidence interval [CI] 0.2%-1.1%) vs 2.0% (95% CI 1.2%-3.1%) at 15 years (Grey's test, P = 0.001). The model consisting of PSAD, PSA and the base variables predicted prostate cancer mortality (c-statistic 0.781) significantly better than either the base variables alone (c-statistic 0.737; likelihood-ratio test, P = 0.003) or the base variables and PSA (c-statistic 0.765; likelihood-ratio test, P = 0.039).


CONCLUSION
Prostate cancer mortality after a benign systematic TRUS-guided biopsy is low. In these patients, PSAD predicts prostate cancer mortality and provides additional value to other clinical variables. PSAD-based stratification can be used to guide follow-up strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0eb4aa4e0452d583b006e63dc668f43b41541032" target='_blank'>
              Association of prostate-specific antigen density with prostate cancer mortality after a benign systematic prostate biopsy result.
              </a>
            </td>
          <td>
            J. Pylväläinen, K. Talala, Jani Raitanen, Antti S. Rannikko, Anssi Auvinen
          </td>
          <td>2025-01-22</td>
          <td>BJU international</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Objective To evaluate the efficacy of a machine learning model for predicting prostate-specific antigen (PSA) persistence after radical prostatectomy (RP). Methods Data from 470 patients who underwent RP at the Affiliated Hospital of Qingdao University from January 2018 to June 2021 were retrospectively analyzed. Ten risk factors, including age, body mass index (BMI), preoperative PSA, biopsy Gleason score, total prostate specific antigen density (PSAD), clinical tumor stage, clinical lymph node status, seminal vesicle invasion, capsular invasion and positive surgical margin, were included in the analysis. The data were randomly divided into a training set and a test set at a ratio of 7:3, and seven different machine learning algorithms were compared. The confusion matrix, receiver operating characteristic (ROC) curve and area under the ROC curve (AUC) were used to evaluate the diagnostic performance of the model, and the random forest algorithm found to be the optimal prediction model. Results In the entire cohort, 142 (30.21%) patients developed PSA persistence. Based on all included risk factors, the random forest model had the best effect among the seven models, with an AUC of 0.8607 in the training set and 0.8011 in the test set. The feature importance results showed that capsular invasion, positive surgical margin, preoperative PSA and biopsy Gleason score were the four most important risk factors for PSA persistence after RP. Conclusion The Random Forest algorithm performed excellently in this study and can be used to construct a predictive model for PSA persistence. By incorporating clinical data from the Asian region and exploring the risk factors for PSA persistence, this study contributes to the existing research and aids clinicians in assessing the risk of PSA persistence occurrence, enabling timely treatment planning and improving patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a0e87565b869dbc8c66d479a6f0957290edb0c" target='_blank'>
              Predictive model for PSA persistence after radical prostatectomy using machine learning algorithms
              </a>
            </td>
          <td>
            Haotian Du, Guipeng Wang, Yongchao Yan, Shengxian Li, Xuecheng Yang
          </td>
          <td>2024-12-06</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b3a954b5d643cfac67348476d195f89c2861f7" target='_blank'>
              Androgen receptor and osteoglycin gene expression predicting prognosis of metastatic prostate cancer
              </a>
            </td>
          <td>
            Tomohiro Kameda, Toru Sugihara, D. Obinata, Masashi Oshima, Y. Yamada, Naoki Kimura, Ken‐ichi Takayama, Satoshi Inoue, Satoru Takahashi, Tetsuya Fujimura
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="

 The neutrophil-to-lymphocyte ratio (NLR), one of the inflammatory markers, can be calculated simply from peripheral blood samples. In this study, the role of NLR in predicting significant prostate cancer (sPC) before prostate biopsy was investigated.



 Patients who underwent prostate biopsy between 2017 and 2022 at the National Taiwan University Hospital were enrolled prospectively. NLR was calculated before the biopsy. sPC was defined as an unfavorable-intermediate or higher risk of the National Comprehensive Cancer Network risk classification.



 Among 484 enrolled patients, 164 sPC (33.9%) and 320 benign/insignificant prostate cancer (isPC; 66.1%) patients were diagnosed. The sPC group was older and had a higher prostate-specific antigen (PSA), smaller prostate volume, higher PSA density, more advanced clinical T stage, and more family history of prostate cancer than the benign/isPC group. There was no difference in neutrophil count (P = 0.929), lymphocyte count (P = 0.927), or NLR (P = 0.931) between groups. NLR failed to predict sPC in univariable (odds ratio, 1.02; 95% confidence interval [CI], 0.93–1.12; P = 0.7) and multivariable (odds ratio, 0.94; 95% CI, 0.79–1.12; P = 0.506) logistic regression models, respectively. The area under the curve of the receiver operating characteristic curve in the model consisted of age, PSA, prostate volume, T stage, and family history of prostate cancer was 0.882 (95% CI, 0.849–0.914). There was no statistical difference when the model included NLR (P = 0.625).



 NLR did not differ between sPC and benign/isPC groups. NLR cannot predict sPC successfully before prostate biopsy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ff45438802797b5f84c1b477033069f520b3e7a" target='_blank'>
              Predictive value of neutrophil-to-lymphocyte ratio for clinically significant prostate cancer
              </a>
            </td>
          <td>
            Siang Ren Jhuang, Jian-Hua Hong, Chao-Yuan Huang, Shih Liu, Po-Ming Chow, Shih-Chieh Jeff Chueh, Chung-Hsin Chen, Yeong-Shiau Pu
          </td>
          <td>2024-12-12</td>
          <td>Urological Science</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="OBJECTIVE
Early and accurate diagnosis of prostate cancer (PC) is crucial for effective treatment. Diagnosing  clinically insignificant cancers can lead to overdiagnosis and overtreatment, highlighting the importance of accurately selecting patients for further evaluation based on improved risk prediction tools. Novel biomarkers offer promise for enhancing this diagnostic process. In this study, we aimed to externally validate a previously developed urine and plasma biomarker test in a biopsy-naïve population.


MATERIALS AND METHODS
Urine and blood samples were prospectively collected from 362 biopsy-naïve men with suspected PC before they underwent transrectal prostate biopsies. The expression levels of a 10-gene mRNA panel were quantified using reverse transcription/quantitative polymerase chain reaction of both urine and plasma. These gene expression levels, combined with clinical features and plasma prostate-specific antigen (PSA) levels, were used to predict the presence of International Society of Urological Pathology grade group ≥ 2 PC.


RESULTS
Complete data were available for 314 patients. The sensitivity and specificity of the biomarker test were 87% (95% CI: 79-93%) and 42% (95% CI: 36-49%), respectively. The area under the curve was 0.76 (95% CI: 0.7-0.82) for the biomarker test probability and 0.65 (95% CI: 0.59-0.72) for PSA (p = 0.02). The test's negative predictive value was 89% (CI: 81-94%).


CONCLUSION
This study did not replicate the previously reported high accuracy of the biomarker test, highlighting the need for further refinement and robust external validation to ensure reliable performance across diverse patient populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cfaba648f44663b161eff9436c6c5f096045194" target='_blank'>
              Prospective validation study of a combined urine and plasma test for predicting high-grade prostate cancer in biopsy naïve men.
              </a>
            </td>
          <td>
            Torben Brøchner Pedersen, Mads Hvid Poulsen, M. Lund, Søren Feddersen, M. Albitar, Charlotte Aaberg Poulsen, Lars Lund
          </td>
          <td>2025-01-14</td>
          <td>Scandinavian journal of urology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Aggressive prostate cancer (PC) represents a significant health concern worldwide. Conventional initial screening methods, primarily based on prostate-specific antigen (PSA) levels, lack specificity, leading to a high rate of unnecessary biopsies and an urgent need for more accurate diagnostic tools. This study addresses the gap by exploring the potential of integrating clinical and routine blood laboratory parameters including a comprehensive hormone assessment to enhance the non-invasive prediction of aggressive PC. Methods In a pilot study of 578 patients who were scheduled for a prostate biopsy due to suspicion of PC, we analyzed an extensive panel of 28 laboratory values alongside data on family history, diet, and lifestyle. A logistic regression classifier was developed, and model performance was evaluated using repeated k-fold cross-validation on the complete dataset (n=282). Participants were histologically categorized into three risk groups: healthy, moderate PC (ISUP 1-2 PC), and aggressive PC (ISUP 3-5 PC). Results Significant associations were found between PC aggressiveness and lower levels of androstenedione, Dehydroepiandrosterone-Sulfate (DHEA-S) and free PSA percentage, as well as higher levels of Sex Hormone Binding Globulin (SHBG). The integration of these serum markers with clinical parameters into a new multi-stage risk classifier for PC prediction significantly improved the predictive accuracy. The risk model outperformed PSA-only methods, demonstrating higher sensitivity and specificity in predicting aggressive PC. Conclusions Incorporating serum markers DHEA-S, androstenedione, and SHBG into a novel risk classifier can improve early detection of aggressive PC. These widely available and cost-effective blood biomarkers could reduce reliance on invasive prostate biopsies and expensive magnetic resonance imaging by providing a more targeted approach to non-invasive prediction of aggressive PC following PSA testing. Our pilot study lays the groundwork for larger-scale research to further explore the integration of androgens and SHBG in future risk stratification models for improved clinical decision making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3254586a7ca082e9d1b49e74cbf797d18bfdeb3f" target='_blank'>
              Integration of serum androgens and Sex Hormone-Binding Globulin for optimized early detection of aggressive prostate cancer
              </a>
            </td>
          <td>
            O. Lazareva, A. Riediger, O. Stegle, H. Sueltmann, M. Hohenfellner, M. Goertz
          </td>
          <td>2024-12-06</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65472bcda3911cb955eb72558b66bb1c50911197" target='_blank'>
              Intratumoral heterogeneity drives acquired therapy resistance in a patient with metastatic prostate cancer
              </a>
            </td>
          <td>
            Dena P. Rhinehart, Jiaying Lai, David E Sanin, Varsha Vakkala, A. Mendes, Christopher Bailey, E. Antonarakis, C. Paller, Xiaojun Wu, T. Lotan, Rachel Karchin, Laura A Sena
          </td>
          <td>2024-12-02</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1861b45b76a529d7121c388efe03a05c095aeb5f" target='_blank'>
              Analysis of the predictive value of the prostate-specific antigen-to-neutrophil ratio for the diagnosis of prostate cancer
              </a>
            </td>
          <td>
            Yuxuan Chen, Haisheng Yan, Yaoqin Xu, Kexin Chen, Runqin Yang, Jiali Yang, Ruian Zhu, Rui Lin, Jiang Wang, Jie Liu, Pingsheng Gao, Lei Pang, Lexin Wang
          </td>
          <td>2025-01-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d92332ae486c3b94c41f8d10e6867cf69c3624d" target='_blank'>
              Unveiling the molecular profile of a prostate carcinoma: implications for personalized medicine
              </a>
            </td>
          <td>
            Massimiliano Agostini, Erica Giacobbi, F. Servadei, Julia Bishof, Likas Funke, Giuseppe Sica, Valentina Rovella, Marco Carilli, V. Iacovelli, Yufang Shi, Jianquan Hou, E. Candi, G. Melino, Giulio Cervelli, M. Scimeca, A. Mauriello, Pierluigi Bove
          </td>
          <td>2024-12-31</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39d9ad3dc0962f7c31b170d54804528397e6b774" target='_blank'>
              Clinical assessment of urinary prostate cancer antigen 3 in Chinese population: a large-scale, prospective and multicenter study
              </a>
            </td>
          <td>
            Xuan Shu, Jiaming Wang, Wen Cai, Shen Lin, Jiangfeng Li, Xueyou Ma, Yufan Ying, Yat Sai Terry Wang, Xiao Wang, Hong Chen, Chunyu Jin, Ben Liu, Liping Xie, Jindan Luo
          </td>
          <td>2024-12-31</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The prediction of survival outcomes is a key factor in making decisions for prostate cancer (PCa) treatment. Advances in computer-based technologies have increased the role of machine learning (ML) methods in predicting cancer prognosis. Due to the various effective treatments available for each non-linear landscape of PCa, the integration of ML can help offer tailored treatment strategies and precision medicine approaches, thus improving survival in patients with PCa. There has been an upsurge of studies utilizing ML to predict the survival of these patients using complex datasets, including patient and tumor features, radiographic data, and population-based databases. This review aims to explore the evolving role of ML in predicting survival outcomes associated with PCa. Specifically, we will focus on the applications of ML in forecasting biochemical recurrence-free, progression to castration-resistance-free, metastasis-free, and overall survivals. Additionally, we will suggest areas in need of further research in the future to enhance the utility of ML for a more clinically-utilizable PCa prognosis prediction and treatment optimization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85de4d129f3584c3ef8b66fa578599797953c397" target='_blank'>
              Harnessing machine learning to predict prostate cancer survival: a review
              </a>
            </td>
          <td>
            Sungun Bang, Young Jin Ahn, K. Koo
          </td>
          <td>2025-01-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Simple Summary In this study, machine learning models were developed to predict lymph node invasion in Japanese prostate cancer patients, using a structured approach to improve predictive accuracy. Traditional methods, such as the Briganti nomogram, have been widely used but may lack flexibility and adaptability to diverse clinical datasets. Our machine learning models, especially Light Gradient-Boosting Machine and Random Forest, showed higher flexibility and accuracy compared to conventional models. Through a thorough data analysis and feature engineering, including the use of predictive features such as PSA levels and tumor stage, we enhanced model performance and interpretability. A decision curve analysis indicated significant clinical benefits, suggesting that these models can better identify candidates for lymph node dissection, thereby minimizing unnecessary procedures. However, validation with larger and multi-institutional datasets is needed to confirm the utility of these models in broader clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad87c2880dac6f6698c93f1bc02afd160fce358e" target='_blank'>
              Utility of Machine Learning Models to Predict Lymph Node Metastasis of Japanese Localized Prostate Cancer
              </a>
            </td>
          <td>
            H. Ueki, T. Terakawa, T. Hara, Munenori Uemura, Y. Okamura, Kotaro Suzuki, Yukari Bando, Jun Teishima, Yuzo Nakano, Raizo Yamaguchi, Hideaki Miyake
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Breast cancer currently occupies the first place in the structure of cancer morbidity in the female population worldwide. Approximately 15–20% of breast cancer patients have overexpression and/or amplification of human epidermal growth factor receptor type 2 (HER2), which causes aggressive biological behavior of the tumor and is associated with poor prognosis. Anti-HER2 therapies have significantly improved the prognosis of patients with HER2-positive breast cancer. Starting from the earliest stages (T2 and/or N+), patients should be treated with neoadjuvant drug therapy, which allows direct assessment of treatment efficacy, increases the chances of organ-preserving surgery, and helps to tailor postneoadjuvant therapy. Anthracycline-based chemotherapy regimens have historically been the first treatment option for HER2-positive breast cancer, as HER2 status is a predictor of anthracycline efficacy. However, the development of new agents and the resulting emergence of novel combinations with multiple anti-HER2 agents have provided an opportunity to explore anthracycline-free regimens aimed at reducing adverse events and improving patient quality of life. The results of the BCIRG-006 trial investigating the adjuvant use of such regimens and, later, neoadjuvant assignment in the TRAIN-2 protocol marked a turning point in the use of non-anthracycline regimens. One important open question is the appropriateness of adding carboplatin, as direct comparisons of non-anthracycline combinations with or without carboplatin are lacking. There is now strong evidence of the impact of achieving a complete pathomorphological response (pCR) after the neoadjuvant phase of treatment on long-term oncologic outcomes. The development and validation of possible biomarkers that predict achievement of pCR is extremely promising.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0396a2b43ed6cf482496094a85af86cb41fc6bc3" target='_blank'>
              Current approaches to neoadjuvant drug therapy for primary operable HER2-positive breast cancer
              </a>
            </td>
          <td>
            M. S. Ruban, L. V. Bolotina, Yu. B. Karagodina
          </td>
          <td>2025-01-21</td>
          <td>Medical alphabet</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a common malignancy in men and is among the leading causes of cancer-related death worldwide. Genomic tests assess disease aggressiveness and guide treatment, particularly in low- and intermediate-risk PCa. We reviewed the literature on the use of four genomic tests (Prolaris®, Promark®, Oncotype DX®, and Decipher®) in assessing the prognosis of PCa and their use in treatment decision-making. Most of the studies showed that Prolaris® has a strong correlation with biochemical recurrence, metastasis risk, PCa-specific mortality (PCSM), and pathological features. Similarly, three studies on Promark® indicated a connection between results and pathological features in the subsequent prostatectomy, time to metastasis, and biochemical recurrence. Fourteen studies on Oncotype DX® showed a clear correlation between high scores, death, and PCSM. One study found that routine biopsy pathology reports, combined with serum PSA levels, provide a risk assessment comparable to Oncotype DX® testing. Results from 22 studies on Decipher® were controversial. The test was associated with conservative management, suggesting that patients with a high GC score are more likely to need radiation after surgery. Comparative studies indicated that Oncotype DX® is preferable for assessing PCSM, Decipher® for predicting metastasis, and Prolaris® for predicting recurrence. With the incidence rate of PCa dramatically increasing, genomic tests appear to be useful adjunctive precision medicine tools with significant potential in improving prognostic discrimination, facilitating better risk stratification, and guiding personalized treatment, especially in the intermediate-risk patient group. Large-scale, prospective, multi-sectional studies are required to validate the utility of these tests prior to their integration into clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32da13d38e5bb8ddbed1c81ce924e80aa8cc6e07" target='_blank'>
              Integration of Genomic Tests in Prostate Cancer Care: Implications for Clinical Practice and Patient Outcomes
              </a>
            </td>
          <td>
            Christos Roidos, Anastasios Anastasiadis, Stavros Tsiakaras, C. Loutradis, Panagiotis Baniotis, Dimitrios Memmos, G. Dimitriadis, Maria Papaioannou
          </td>
          <td>2024-12-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Patient stratification remains a challenge for optimal treatment of prostate cancer (PCa). This clinical heterogeneity implies intra-tumoural heterogeneity, with different prostate epithelial cell subtypes not all targeted by current treatments. We reported that such cell subtypes are traceable in liquid biopsies through representative transcripts. Expanding on this concept, we included 57 genes representing cell subtypes, drug targets and relevant to resistance as non-invasive biomarkers for stratification. This panel was tested by RT-qPCR (quantitative reverse transcription polymerase chain reaction) in blood of controls and different categories of PCa patients. Overall, circulating transcripts showed predictive value throughout the disease. Those with aggressive pathological features such as intra-ductal carcinoma at diagnosis showed more genes over-expressed. In metastatic patients, signatures of subtypes or resistance were associated with treatments, progression-free survival and overall survival. Altogether, testing markers of cell diversity, an intrinsic feature of tumours, and drug targets via liquid biopsies represents a valuable means to stratify patients and predict responses to current or new therapeutic modalities. Over-expressed drug target genes suggest potential benefit from targeted treatments, justifying new clinical trials to offer patient-tailored strategies to eventually impact on PCa mortality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db693f90ecf986ea4405cf60ddb76edec574f4c1" target='_blank'>
              Clinical significance of stratifying prostate cancer patients through specific circulating genes.
              </a>
            </td>
          <td>
            Seta Derderian, Edouard Jarry, Arynne Santos, Quentin Vesval, Lucie Hamel, R. Sanchez-Salas, A. Rompré-Brodeur, W. Kassouf, Raghu Rajan, F. Brimo, Marie Duclos, A. Aprikian, Simone Chevalier
          </td>
          <td>2025-01-22</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/823fac8baba17d873c8cfc6928961f769be662ad" target='_blank'>
              Exploring the Effects of Opioid-Related Drugs on the Clinical Outcome of Prostate Cancer Patients Via Integrated Bioinformatics Analysis.
              </a>
            </td>
          <td>
            Yunxuan Zhang, Yuenan Liu, Kailei Chen, Qi-zeng Miao, Qi Cao, Xiaoping Zhang
          </td>
          <td>2025-01-20</td>
          <td>Molecular biotechnology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Background Prostate cancer is a common cancer with a variable prognosis. Its management is currently guided by histological and biological markers such as the Gleason score and PSA. Developments in molecular biology are now making it possible to identify new targets for better classification of prostate cancer. Among emerging biomarker, DNA methylation, an epigenetic process, is increasingly being studied in carcinogenesis. Techniques for analyzing DNA methylation are constantly improving, and digital PCR now allows absolute methylation quantification with high sensitivity. These techniques can be performed on circulating tumor DNA. Materials & Methods We conducted a literature review of scientific articles addressing the topic of DNA methylation in prostate cancer. Results & discussion This review summarizes the different genes whose methylation is involved in carcinogenesis and their clinical implications, both diagnostic and prognostic. Methylation monitoring could also be useful for the prediction of treatment response. However, most studies are retrospective, and prospective studies are needed to validate these data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59c901f5c6197904e2aeab3c865a402741cdc3a" target='_blank'>
              DNA Methylation in Prostate Cancer: Clinical Implications and Potential Applications
              </a>
            </td>
          <td>
            Romane Muletier, C. Bourgne, Laurent Guy, Aurore Dougé
          </td>
          <td>2025-01-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The rising incidence of breast cancer and its heterogeneity necessitate precise tools for predicting patient prognosis and tailoring personalized treatments. Epigenetic changes play a critical role in breast cancer progression and therapy responses, providing a foundation for prognostic model development.We developed the Machine Learning-derived Epigenetic Model (MLEM) to identify prognostic epigenetic gene patterns in breast cancer. Using multi-cohort transcriptomic datasets, MLEM was constructed with rigorous machine learning techniques and validated across independent datasets. The model’s performance was further corroborated through immunohistochemical validation on clinical samples.MLEM effectively stratified breast cancer patients into high- and low-risk groups. Low-MLEM patients exhibited improved prognosis, characterized by enhanced immune cell infiltration and higher responsiveness to immunotherapy. High-MLEM patients showed poorer prognosis but were more responsive to chemotherapy, with vincristine identified as a promising therapeutic option. The model demonstrated robust performance across independent validation datasets.MLEM is a powerful prognostic tool for predicting breast cancer outcomes and tailoring personalized treatments. By integrating epigenetic insights with machine learning, this model has the potential to improve clinical decision-making and optimize therapeutic strategies for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30a59be16fce55c3394d46cf1517a7214190e840" target='_blank'>
              Epigenetic profiling for prognostic stratification and personalized therapy in breast cancer
              </a>
            </td>
          <td>
            Xiao Guo, Chuanbo Feng, Jiaying Xing, Yuyan Cao, Tengda Liu, Wenchuang Yang, Runhong Mu, Tao Wang
          </td>
          <td>2025-01-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The global incidence of prostate cancer (PCa) is rising. Localized PCa can be managed through surgical intervention or radiotherapy, but certain patients may experience recurrence or develop metastatic disease following localized treatment. Despite aggressive therapeutic approaches, the majority of metastatic patients with PCa will eventually progress to metastatic castration‑resistant PCa, with limited treatment alternatives and a dismal prognosis. The treatment options for advanced PCa are continuously evolving, yet there remains a demand for further innovative therapeutic approaches. Antibody‑drug conjugates (ADCs) represent a novel class of targeted medications comprising a humanized monoclonal antibody, a linker and a cytotoxic payload. ADCs primarily bind to antigens that are upregulated on the surface of PCa cells but are minimally expressed on normal cells. At present, a variety of targets for ADCs have been identified in the treatment of PCa, encompassing prostate‑specific membrane antigen, STEAP family member 1, trophoblast cell‑surface antigen 2, CD46, B7‑H3, tissue factor and delta‑like protein 3, each with one or more specific ADC that has shown encouraging results in the PCa field. In the present review, the developmental course, composition and mechanism of action of ADCs are explored, with a specific focus on recently published studies and ongoing trials aimed at investigating the efficacy and safety of ADCs in treating PCa. Lastly, ongoing challenges in ADC development and corresponding strategies to combat them are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb6647ef3243af9b93ecaada082dfb6baa67d85" target='_blank'>
              Antibody‑drug conjugates in prostate cancer: Emerging strategies to enhance therapeutic index and current clinical landscape (Review).
              </a>
            </td>
          <td>
            Chadanfeng Yang, Limei Wang, Chen Gong, Dihao Lv, Haihao Li, Yinglong Huang, Jiting Li, Wujie Chen, Shi Fu, Zhiyong Tan, Mingxia Ding
          </td>
          <td>2024-12-13</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df35383baab7cdd35c05579c5bac4deb31b8c4ef" target='_blank'>
              Predicting chemotherapy responsiveness in gastric cancer through machine learning analysis of genome, immune, and neutrophil signatures.
              </a>
            </td>
          <td>
            Shota Sasagawa, Yoshitaka Honma, Xinxin Peng, Kazuhiro Maejima, K. Nagaoka, Yukari Kobayashi, Ayako Oosawa, Todd A Johnson, Yuki Okawa, Han Liang, Kazuhiro Kakimi, Yasuhide Yamada, H. Nakagawa
          </td>
          <td>2024-12-02</td>
          <td>Gastric cancer : official journal of the International Gastric Cancer Association and the Japanese Gastric Cancer Association</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7bc4401981d242587cb65181c99817f023e6204" target='_blank'>
              Blood-based tumor mutational burden impacts clinical outcomes of immune checkpoint inhibitor treated breast and prostate cancers
              </a>
            </td>
          <td>
            Reagan M Barnett, Albert Jang, Sree M Lanka, Pingfu Fu, L. Bucheit, Hani Babiker, Alan H Bryce, Haley M. Meyer, Yujin Choi, Casey Moore, R. Garje, Xin Gao, Dae Won Kim, Richard Y. Chang, Pat Gulhati, Ryne Ramaker, Rani Bansal, Tian Zhang, A. Oliver Sartor, Andrew J Armstrong, M. Bilen, Pedro C Barata
          </td>
          <td>2024-12-02</td>
          <td>Communications Medicine</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Prostate cancer is one of the most prevalent malignant tumors in men, particularly in regions with a high Human Development Index. While the long-term survival rate for localized prostate cancer is relatively high, the mortality rate remains significantly elevated once the disease progresses to advanced stages, even with various intensive treatment modalities. The primary obstacle to curing advanced prostate cancer is the absence of comprehensive treatment strategies that effectively target the highly heterogeneous tumors at both genetic and molecular levels. Prostate cancer development is a complex, multigenic, and multistep process that involves numerous gene mutations, alteration in gene expression, and changes in signaling pathways. Key genetic and pathway alterations include the amplification and/or mutation of the androgen receptor, the loss of Rb, PTEN, and p53, the activation of the WNT signaling pathway, and the amplification of the MYC oncogene. This review summarizes the mechanisms by which these genes influence the progression of prostate cancer and highlights the interactions between multiple genes and their relationship with prostate cancer. Additionally, we reviewed the current state of treatments targeting these genes and signaling pathways, providing a comprehensive overview of therapeutic approaches in the context of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38fab9adcd074d173268fdd423c799e2e23c962b" target='_blank'>
              Interactions between key genes and pathways in prostate cancer progression and therapy resistance
              </a>
            </td>
          <td>
            Fan Wu, Heng Zhang, Miaomiao Hao
          </td>
          <td>2025-01-23</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pancreatic cancer remains a highly malignant and challenging tumor with a dismal 5-year survival rate of only 13%. The majority of patients are diagnosed at advanced stages, where surgical options are limited, and prognosis is poor. Immunotherapy, particularly PD-1 inhibitors, has shown limited success in pancreatic cancer due to its unique tumor immune microenvironment. However, certain genetic profiles, such as BRCA1/2 mutations, high tumor mutational burden (TMB), or microsatellite instability-high (MSI-H), may enhance sensitivity to these therapies. This report presents two cases of advanced pancreatic cancer with BRCA1/2 mutations treated with a combination of chemotherapy and immune checkpoint inhibitors. The first patient, with TMB-H and stable microsatellites, achieved complete remission after conversion therapy and remains disease-free for over two years post-surgery. The second patient, with MSI-H and low TMB, experienced significant tumor regression and improved quality of life with a prolonged progression-free survival, although the patient ultimately declined surgery. These cases suggest that combined chemotherapy and immunotherapy may offer a promising treatment option for select pancreatic cancer patients, particularly those with specific genetic profiles, warranting further investigation into personalized approaches to immunotherapy in this malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a93c4c22949785a73a6a2556482978253791d306" target='_blank'>
              Case report: Achieving significant tumor reduction in advanced pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Hongying Liu, Yun-Long Wang, Qian Zhang, Nengwen Ke
          </td>
          <td>2024-12-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b351a91b1b339944d457e7db859962fb6728ec9" target='_blank'>
              Patient-derived xenografts from circulating cancer stem cells as a preclinical model for personalized pancreatic cancer research
              </a>
            </td>
          <td>
            Benedikt J Wagner, Andreas Ettner-Sitter, Nicolas A Ihlo, Merle Behr, Sebastian Koelbl, Stefan M Brunner, Florian Weber, Bettina M Rau, Hans J Schlitt, Christoph Brochhausen, Rebecca Schoenmehl, Annalena Artinger, D. Schott, M. Pizon, Katharina Pachmann, Thiha Aung, Silke Haerteis, Christina Hackl
          </td>
          <td>2025-01-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Adoption of immunotherapy has completely transformed the treatment landscape of cancer. Patients with advanced cancer treated with immunotherapy may benefit from durable tumor response and long-term survival. The most widely used immunotherapy in solid tumors is anti-programmed-death (ligand) protein (PD-(L)1), which is now an integral part of non-small cell lung cancer (NSCLC) treatment irrespective of histological cell types and tumor stage. However, the vast majority of patients with advanced NSCLC treated with anti-PD-(L)1 still develop therapeutic resistance, and the prognosis after anti-PD-(L)1 resistance is poor. Resistance mechanisms to PD-1 blockade are often complex and encompass a combination of defects within the cancer-immunity cycle. These defects include failure in antigen presentation and T-cell priming, presence of co-inhibitory immune checkpoints, inability of immune cells to infiltrate the tumor, and presence of immunosuppressive tumor microenvironment. Recently, advances in drug design, genomic sequencing, and gene editing technologies have led to development of next-generation immunotherapies that may potentially overcome these resistance mechanisms. In this review, we will discuss the anti-PD-(L)1 resistance mechanism landscape in NSCLC and four novel modalities of immunotherapy in detail, namely novel immune checkpoint inhibitor and targeted therapy combinations, bispecific antibodies, cancer vaccine, and cell therapy. These novel therapeutics have all demonstrated early clinical data in NSCLC treatment and may work synergistically with each other to restore anticancer immunity. In addition, we share our perspectives on the future promises and challenges in the transformation of these novel immunotherapies to standard clinical care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdef5bc62054de2d611913bc2fb5655a0f727fb9" target='_blank'>
              Next-generation immunotherapy: igniting new hope for lung cancer
              </a>
            </td>
          <td>
            M. S. Li, Andrew L S Chan, Kevin K S Mok, L. Chan, Tony S K Mok
          </td>
          <td>2024-01-01</td>
          <td>Therapeutic Advances in Medical Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Immunotherapy has revolutionized cancer treatment, but predicting patient response remains challenging. Herein, we present iPRISM (Intelligent Predicting Response to cancer Immunotherapy through Systematic Modeling), which is a novel network‐based model that integrates multiomics data to predict immunotherapy outcomes. In this approach, iPRISM incorporates gene expression, biological functional network, tumor microenvironment characteristics, immune‐related pathways, and clinical data to provide a comprehensive view of factors influencing immunotherapy efficacy. Using stepwise logistic regression, we identified key predictive features and validated iPRISM across multiple cohorts including melanoma, bladder cancer, non‐small cell lung cancer, and stomach adenocarcinoma. We also find that iPRISM outperforms the existing methods, achieving high predictive accuracy and demonstrating significant prognostic value for overall and progression‐free survival. By identifying key genetic and immunological factors, this model provides a new insight for more personalized treatment strategies and combination therapies to overcome resistance mechanisms. iPRISM can be accessed at CRAN: https://CRAN.R‐project.org/package=iPRISM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed045e8d91285b3e11584131d35ba5346ad63121" target='_blank'>
              iPRISM: Intelligent Predicting Response to Cancer Immunotherapy through Systematic Modeling
              </a>
            </td>
          <td>
            Yinchun Su, Siyuan Li, Qian Wang, Bingyue Pan, Jiyin Lai, Guangyou Wang, Junwei Han, Qingfei Kong
          </td>
          <td>2025-01-14</td>
          <td>Advanced Intelligent Systems</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/743d452f13e817151bd0f6e466497fc0d19ac479" target='_blank'>
              Spatial heterogeneity of the hepatocellular carcinoma microenvironment determines the efficacy of immunotherapy
              </a>
            </td>
          <td>
            M. Zhang, Kailin Huang, Qiushi Yin, Xueqin Wu, Mingyue Zhu, Mengsen Li
          </td>
          <td>2025-01-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Aggressive variant prostate cancer (AVPC) is characterized by a molecular signature involving combined defects in TP53, RB1, and/or PTEN (AVPC-TSGs), identifiable through immunohistochemistry or genomic analysis. The reported prevalence of AVPC-TSG alterations varies widely, reflecting differences in assay sensitivity, treatment pressure, and disease stage evolution. Although robust clinical evidence is still emerging, the study of AVPC-TSG alterations in prostate cancer (PCa) is promising. Alterations in TP53, RB1, and PTEN, as well as the combined loss of AVPC-TSGs, may have significant implications for prognosis and treatment. These biomarkers might help predict responses to various therapies, including hormonal treatments, cytotoxic agents, radiotherapy, and targeted therapies. Understanding the impact of these molecular alterations in patients with PCa is crucial for personalized management. In this review, we provide a comprehensive overview of the emerging prognostic and predictive roles of AVPC-TSG alterations across PCa stages. Moreover, we discuss the implications of different methods used for detecting AVPC-TSG alterations and summarize factors influencing their prevalence. As our comprehension of the genomic landscape of PCa disease deepens, incorporating genomic profiling into clinical decision making will become increasingly important for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc76168bc6cc3f716d4ad9c939472b94406c6f4f" target='_blank'>
              The Emerging Predictive and Prognostic Role of Aggressive-Variant-Associated Tumor Suppressor Genes Across Prostate Cancer Stages
              </a>
            </td>
          <td>
            Martino Pedrani, Jessica Barizzi, Giuseppe Salfi, Alessandro Nepote, Irene Testi, Sara Merler, Luis Castelo-Branco, R. P. Mestre, Fabio Turco, Luigi Tortola, Jean-Philippe P. Theurillat, S. Gillessen, U. Vogl
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Simple Summary With the discovery of immunotherapy (treatments that increase the immune system’s ability to kill cancer cells) and targeted therapy (treatments that target the specific pathway the cancer uses to grow and survive), we have made great progress in the treatment of advanced kidney cancer. However, their roles in a localized setting (a tumor limited to the kidney and/or lymph nodes around the kidney) are not well defined. This review will look at the available data on the use of immunotherapy and targeted therapy for patients with localized kidney cancer, where the goals may be to decrease tumor size, to make surgery easier and less invasive, to save as much kidney function as possible, and/or to kill the cancer cells that may have escaped the kidney.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2476f892f32a000cbcb6f0c49b52d56b374d134" target='_blank'>
              A Review of Neoadjuvant Therapy for Localized and Locally Advanced Renal Cell Carcinoma
              </a>
            </td>
          <td>
            Qian Qin, Isamu Tachibana, V. Margulis, Jeffrey A. Cadeddu, Tian Zhang
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="ABSTRACT



 Prostate cancer is one of the most prevalent cancers among men worldwide. The identification of genetic markers that contribute to the risk of prostate cancer can significantly enhance early diagnosis and personalized treatment strategies. Next-Generation Sequencing (NGS) has emerged as a powerful tool for uncovering genetic variations associated with various cancers, including prostate cancer.



 A cohort of 200 men, consisting 100 diagnosed with prostate cancer and 100 healthy controls, was recruited for this study. DNA samples were extracted from blood and tumor tissues, followed by library preparation and sequencing using an Illumina platform. Bioinformatic analysis was conducted to identify single nucleotide polymorphisms (SNPs), copy number variations (CNVs), and other genetic alterations linked to prostate cancer. Statistical analysis was performed to assess the association between identified genetic markers and prostate cancer risk.



 NGS identified a total of 450 genetic variants, with 50 SNPs showing a strong association with prostate cancer risk (P < 0.001). Notably, the BRCA1, BRCA2, and HOXB13 genes exhibited significant variants in 20% of the prostate cancer patients. Furthermore, CNVs in the MYC and PTEN genes were observed in 15% and 10% of the patients, respectively. The control group displayed a significantly lower frequency of these genetic alterations, reinforcing their potential role as risk markers for prostate cancer.



 This study demonstrates the efficacy of NGS in identifying genetic markers associated with prostate cancer risk.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef36374ed470edf97757cba9059ce326a4977025" target='_blank'>
              Efficacy of Next-Generation Sequencing in Identifying Genetic Markers for Prostate Cancer Risk
              </a>
            </td>
          <td>
            Fahad H. Alaithan, Raghad H. Alaithan, Tauheed Ahmed
          </td>
          <td>2024-12-05</td>
          <td>Journal of Pharmacy and Bioallied Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Importance
Only a small fraction of patients with advanced non-small cell lung cancer (NSCLC) respond to immune checkpoint inhibitor (ICI) treatment. For optimal personalized NSCLC care, it is imperative to identify patients who are most likely to benefit from immunotherapy.


Objective
To develop a supervised deep learning-based ICI response prediction method; evaluate its performance alongside other known predictive biomarkers; and assess its association with clinical outcomes in patients with advanced NSCLC.


Design, Setting, and Participants
This multicenter cohort study developed and independently validated a deep learning-based response stratification model for predicting ICI treatment outcome in patients with advanced NSCLC from whole slide hematoxylin and eosin-stained images. Images for model development and validation were obtained from 1 participating center in the US and 3 in the European Union (EU) from August 2014 to December 2022. Data analyses were performed from September 2022 to May 2024.


Exposure
Monotherapy with ICIs.


Main Outcomes and Measures
Model performance measured by clinical end points and objective response rate (ORR) differentiation power vs other predictive biomarkers, ie, programmed death-ligand 1 (PD-L1), tumor mutational burden (TMB), and tumor-infiltrating lymphocytes (TILs).


Results
A total of 295 581 image tiles from 958 patients (mean [SD] age, 66.0 [10.6] years; 456 [48%] females and 502 [52%] males) treated with ICI for NSCLC were included in the analysis. The US-based development cohort consisted of 614 patients with median (IQR) follow-up time of 54.5 (38.2-68.1) months, and the EU-based validation cohort, 344 patients with 43.3 (27.4-53.9) months of follow-up. The ORR to ICI was 26% in the developmental cohort and 28% in the validation cohort. The deep learning model's area under the receiver operating characteristic curve (AUC) for ORR was 0.75 (95% CI, 0.64-0.85) in the internal test set and 0.66 (95% CI, 0.60-0.72) in the validation cohort. In a multivariable analysis, the deep learning model's score was an independent predictor of ICI response in the validation cohort for both progression-free (hazard ratio, 0.56; 95% CI, 0.42-0.76; P < .001) and overall survival (hazard ratio, 0.53; 95% CI, 0.39-0.73; P < .001). The tuned deep learning model achieved a higher AUC than TMB, TILs, and PD-L1 in the internal set; in the validation cohort, it was superior to TILs and comparable with PD-L1 (AUC, 0.67; 95% CI, 0.60-0.74), with a 10-percentage point improvement in specificity. In the validation cohort, combining the deep learning model with PD-L1 scores achieved an AUC of 0.70 (95% CI, 0.63-0.76), outperforming either marker alone, with a response rate of 51% compared to 41% for PD-L1 (≥50%) alone.


Conclusions and Relevance
The findings of this cohort study demonstrate a strong and independent deep learning-based feature associated with ICI response in patients with NSCLC across various cohorts. Clinical use of this deep learning model could refine treatment precision and better identify patients who are likely to benefit from ICI for treatment of advanced NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cb3cabb4e9d3f987624a772fec9e7ae5fd89698" target='_blank'>
              Deep Learning Model for Predicting Immunotherapy Response in Advanced Non-Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            M. Rakaee, M. Tafavvoghi, B. Ricciuti, J. Alessi, Alessio Cortellini, F. Citarella, L. Nibid, Giuseppe Perrone, E. Adib, C. Fulgenzi, Cassio Murilo Hidalgo Filho, A. Di Federico, Falah Jabar, Sayed Hashemi, I. Houda, E. Richardsen, Lill-Tove Rasmussen Busund, T. Dønnem, I. Bahce, D. Pinato, Å. Helland, L. Sholl, M. M. Awad, David J. Kwiatkowski
          </td>
          <td>2024-12-26</td>
          <td>JAMA oncology</td>
          <td>1</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82afc183b4d0410b01d96c21827443e7a561932e" target='_blank'>
              Prediction of checkpoint inhibitor immunotherapy efficacy for cancer using routine blood tests and clinical data.
              </a>
            </td>
          <td>
            Seong-Keun Yoo, C. Fitzgerald, B. A. Cho, Bailey G Fitzgerald, Catherine Han, Elizabeth S. Koh, Abhinav Pandey, Hannah Sfreddo, Fionnuala Crowley, Michelle Rudshteyn Korostin, Neha Debnath, Yan Leyfman, Cristina Valero, Mark Lee, J. Vos, A. Lee, K. Zhao, S. Lam, Ezekiel Olumuyide, F. Kuo, E. Wilson, Pauline Hamon, C. Hennequin, Miriam Saffern, Lynda Vuong, A. A. Hakimi, Brian D Brown, Miram Merad, S. Gnjatic, N. Bhardwaj, M. Galsky, Eric E. Schadt, R. Samstein, T. Marron, M. Gönen, L. Morris, D. Chowell
          </td>
          <td>2025-01-06</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Treatment for castration-resistant prostate cancer (CRPC) primarily involves the suppression of androgen receptor (AR) activity using androgen receptor signaling inhibitors (ARSIs). While ARSIs have extended patient survival, resistance inevitably develops. Mechanisms of resistance include genomic aberrations at the AR locus that reactivate AR signaling, or lineage plasticity that drives emergence of AR-independent phenotypes. Given the diverse mechanisms of ARSI resistance in CRPC, there is a need for more effective monitoring strategies that detect signs of resistance to inform prognosis and guide use of alternative therapies. Liquid biopsy is a blood test that has emerged as a powerful, minimally invasive tool for investigating advanced cancer. In CRPC, liquid biopsy has been shown to reflect genomic and transcriptomic features in tumor tissue and has been utilized to detect an array of resistance signatures. Liquid biopsy is uninhibited by spatial restrictions and allows for longitudinal monitoring of disease progression. However, current clinical liquid biopsy tests provide limited actionable information. This review highlights recent advancements to the understanding of mechanisms driving treatment resistance in CRPC through use of research-grade liquid biopsy assays. We explore novel methods of disease characterization developed using liquid biopsy and emphasize the clinical potential of a multi-omic molecular profiling approach to comprehensively detect emerging therapeutic resistance. Routine assessment of therapy resistance via a liquid biopsy assay has the potential to enhance prognostication and improve outcomes of men with CRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad53e1b9e5a73acf01cdbea070656fe0feed870c" target='_blank'>
              Systems-level liquid biopsy in advanced prostate cancer.
              </a>
            </td>
          <td>
            Jacqueline Lyman, S. Dehm
          </td>
          <td>2025-01-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Despite significant research progress, tumor heterogeneity remains elusive, and its complexity poses a barrier to anticancer drug discovery and cancer treatment. Response to the same drug varies across patients, and the timing of treatment is an important factor in determining prognosis. Therefore, development of patient-specific preclinical models that can predict a patient's drug response within a short period is imperative. In this study, a printed gastric cancer (pGC) model is developed for preclinical chemotherapy using extrusion-based 3D bioprinting technology and tissue-specific bioinks containing patient-derived tumor chunks. The pGC model retained the original tumor characteristics and enabled rapid drug evaluation within 2 weeks of its isolation from the patient. In fact, it is confirmed that the drug response-related gene profile of pGC tissues co-cultured with human gastric fibroblasts (hGaFibro) is similar to that of patient tissues. This suggested that the application of the pGC model can potentially overcome the challenges associated with accurate drug evaluation in preclinical models (e.g., patient-derived xenografts) owing to the deficiency of stromal cells derived from the patient. Consequently, the pGC model manifested a remarkable similarity with patients in terms of response to chemotherapy and prognostic predictability. Hence, it is considered a promising preclinical tool for personalized and precise treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48628cf6b57bb61c17a9a6597bd1fac3fe372a0f" target='_blank'>
              Prediction of Patient Drug Response via 3D Bioprinted Gastric Cancer Model Utilized Patient-Derived Tissue Laden Tissue-Specific Bioink.
              </a>
            </td>
          <td>
            Yoo-Mi Choi, Deukchae Na, Goeun Yoon, Jisoo Kim, Seoyeon Min, Hee-Gyeong Yi, Soo-Jeong Cho, Jae Hee Cho, Charles Lee, Jinah Jang
          </td>
          <td>2025-01-02</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Our research aims to understand the adaptive, ergo potentially metastatic, responses of prostate cancer to changing microenvironments. Emerging evidence implicates a role of the Polyaneuploid Cancer Cell (PACC) state in metastasis, positing the PACC state as capable of conferring metastatic competency. Mounting in vitro evidence supports increased metastatic potential of cells in the PACC state. Additionally, our recent retrospective study revealed that PACC presence in patient prostate tumors at the time of radical prostatectomy was predictive of future metastasis. To test for a causative relationship between PACC state biology and metastasis in prostate cancer, we leveraged a novel method designed for flow-cytometric detection of circulating tumor cells (CTCs) and disseminated tumor cells (DTCs) from animal models. This approach provides both quantitative and qualitative information about the number and PACC-status of recovered CTCs and DTCs. Specifically, we applied this approach to analysis of subcutaneous, caudal artery, and intracardiac murine models. Collating data from all models, we found that 74% of recovered CTCs and DTCs were in the PACC state. Furthermore, in vivo colonization assays proved PACC populations can regain proliferative capacity at metastatic sites. Additional in vitro analyses revealed a PACC-specific partial Epithelial-to-Mesenchymal-Transition phenotype and a pro-metastatic secretory profile, together providing preliminary evidence of pro-metastatic mechanisms specific to the PACC state. Implications: Considering that many anti-cancer agents induce the PACC state, our data positions the increased metastatic competency of PACC state cells as an important unforeseen ramification of neoadjuvant regimens, which may help explain clinical correlations between chemotherapy and metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c130a571d99ddcd76ddd1df86bc68f827f48ad3" target='_blank'>
              Cells in the Polyaneuploid Cancer Cell State are Pro-Metastatic.
              </a>
            </td>
          <td>
            Mikaela M. Mallin, Louis T A Rolle, Michael J Schmidt, Shilpa Priyadarsini Nair, A. Zurita, Peter Kuhn, James B. Hicks, Kenneth J Pienta, S. Amend
          </td>
          <td>2024-12-10</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Purpose Circulating-tumor DNA (ctDNA) is an emerging, minimally invasive diagnostic and prognostic biomarker for patients receiving a variety of cancer therapies. Comprehensive and robust longitudinal monitoring of ctDNA can provide an understanding of tumor burden, heterogeneity, and response or resistance to treatment. Experimental Design ctDNA of 28 metastatic cancer patients receiving an individualized neoantigen-directed immunotherapy was monitored longitudinally, up to two years, using a unique hybrid next generation sequencing assay targeting tumor-informed and tumor-naïve variants. Patient-specific panels were designed targeting an average of 144 variants per patient. A tumor-naïve universal panel was also designed for inclusion with patient-specific panels to monitor recurrently mutated tumor hotspots (e.g., KRAS and TP53) and genes implicated in immunotherapy resistance (B2M, TAP1/2). Results Analytical characterization of the assay established linearity with a mean variant allele frequency (VAF) ≥0.049%, and a variant-level limit of detection (LOD95) of 0.12%. Tumor-informed variants were detected in 26/28 patients, and de novo variants were observed in 25/28 patients. HLA LOH was also observed. Longitudinal ctDNA data provided key insights into patients’ responses to vaccine treatment. Conclusions The hybrid design of the ctDNA monitoring assay provides the sensitivity and specificity required for evaluating patient samples undergoing individualized therapy. It provides an improved capability to understand patient response to experimental therapies and further supports the utility of ctDNA as a cancer biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89f4ac7a21d9b5bade1ff45fda945bd8d7ea8aff" target='_blank'>
              Comprehensive Longitudinal ctDNA Monitoring in Metastatic Cancer Patients Treated with an Individualized Neoantigen-directed Vaccine
              </a>
            </td>
          <td>
            Desiree Schenk, Matthew J. Davis, R. Zhou, Alexis Mantilla, Madeline Galbraith, O. Spiro, Olivia Petrillo, Italo Faria do Valle, Andrew R. Ferguson, Karin Jooss, Ankur Dhanik
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Most patients with stage IV breast cancer cannot be cured; hence, the goal of therapy is to prolong survival while maintaining a good quality of life. Guidance for treating patients in the nearly limitless scenarios after disease progression following the first few lines of therapy comes from clinical trials conducted in patients who have had limited treatment and do not address these many complex scenarios. We used a form of artificial intelligence called deep learning to investigate patients with stage IV breast cancer from the SEER-Medicare dataset to develop predictive modeling in individual patients. We achieved approximately 90% accuracy in survival predictions in individual patients by combining features from the patient, the cancer, their treatments, and adverse events from treatment. These models can guide decision-making regarding patients’ treatment, where guidance is unavailable, to extend their lives while maintaining a good quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65e7eb5c429566ac53105aeccdbaf7576bea539" target='_blank'>
              Predictive Modeling of Long-Term Survivors with Stage IV Breast Cancer Using the SEER-Medicare Dataset
              </a>
            </td>
          <td>
            Nabil Adam, R. Wieder
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Despite advancements in the treatment of advanced prostate cancer, including metastatic castration-resistant prostate cancer (mCRPC), the disease remains lethal, often developing resistance to conventional therapies such as androgen deprivation therapy (ADT) and chemotherapy. Therapy resistance poses a significant challenge, particularly in aggressive forms of prostate cancer. Data on TP53 mutations, especially gain-of-function (GOF) mutations, and their specific impact on prostate cancer metabolism remain limited, underscoring the need for a deeper understanding to uncover new therapeutic avenues. This study aimed to explore the metabolic consequences of TP53 GOF mutations (R175H and R273H) in prostate cancer progression using metabolomics analysis. Approximately 300 metabolites were significantly altered across PC3-GFP (KO), PC3-TP53WT (WT), and PC3-TP53 mutant (R175H & R273H) models. Key findings revealed dysregulation of essential metabolites, such as α-ketoglutarate and spermidine, which support cancer cell survival and progression. These insights not only enhance our understanding of the interplay between TP53 mutations and metabolic alterations but also identify potential biomarkers for prognosis and therapeutic targeting in therapy-resistant prostate cancer. In the TP53-R175H mutant, numerous essential metabolites were dysregulated (downregulated: glutamine, GAP/DHAP, and α-ketoglutarate; Upregulated: deoxyuridine, spermidine, and pseudouridine) when compared with the control (KO) thereby causing metabolic shifts that favor cancer cell survival. Conversely, metabolites such as GlaNAC/GlcNAC, adenosine, acetoacetic acid, acetylcholine, octopamine, itaconic acid, lactic acid, and NADP+ are all downregulated in the R273H mutant. While spermine, spermidine, N-acetyl-DL-serine, pseudouridine, deoxyguanosine, sucrose, and AMP/dGMP are upregulated by the R273H mutant but downregulated in GFP (KO). These findings showed that TP53 GOF mutations (R175H & R273H) cause metabolic alterations in numerous metabolites essential for tumor survival and progression. Furthermore, it may provide therapeutic benefits by identifying appropriate biomarkers that could be used as diagnostic and/or prognostic tools for prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6235a81e6d316a547376d1bb83acc9177dd3e97b" target='_blank'>
              Impact of TP53 Gain-of-Function Mutations on Metabolic Reprogramming in Prostate Cancer Progression
              </a>
            </td>
          <td>
            Farida Jazuli Usman, D. G. Gupta, S. Abdulkadir, Ibrahim Muhammad Usman, Adamu Jibrin Alhassan, Young A Yoo, Haris Jaafar Bello, Binta G. Kurfi, K. Babagana
          </td>
          <td>2025-01-10</td>
          <td>Asian Journal of Biochemistry, Genetics and Molecular Biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The prognosis for pancreatic cancer is still poor, because it is typically identified at an advanced, frequently metastatic stage. Recently, a sequential treatment for metastatic pancreatic ductal adenocarcinoma (mPDAC) has been developed. When the performance status is good, combination chemotherapy regimens such FOLFIRINOX and gemcitabine plus nab-paclitaxel are options for first-line treatment. Gemcitabine monotherapy is a viable choice for patients with lower performance status. Recently, olaparib, a PARP inhibitor, was showed to enhance the amount of time patients with metastatic pancreatic ductal adenocarcinoma (mPDAC) and a BRCA1/-2 germ line mutation can live without their disease worsening. If the initial treatment is unsuccessful, patients should be provided with an alternative treatment option if their ECOG score allows for continued treatment. The combination of 5-FU/FA with nanoliposomal irinotecan has demonstrated better overall survival compared to 5-FU/FA alone. Immune checkpoint drugs like PD1/PD-L1 mAbs are particularly efficacious in malignancies with high microsatellite instability (MSI-h). Limited data in mPDACs suggests that only a part of the already tiny subgroup of MSI-H mPDACs (frequency approximately 1%) appears to benefit substantially from a checkpoint inhibitor treatment. The identification of other subgroups, may further increase therapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d0ee6a06e92b5e768c019d1f67f584b44f71b5b" target='_blank'>
              Systemic Therapy in Metastatic Pancreatic Cancer: A Review
              </a>
            </td>
          <td>
            Alfredo Colombo, Concetta Maria Porretto
          </td>
          <td>2024-12-28</td>
          <td>International Journal of Advanced Multidisciplinary Research and Studies</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38518d8cdbf882d74bd9cdd46e825049be172ea5" target='_blank'>
              Personalized decision making of neoadjuvant chemotherapy vs. upfront surgery in pancreatic cancer by a simple blood collection?
              </a>
            </td>
          <td>
            P. Kirchweger, B. Doleschal, Holger Rumpold, H. Wundsam, Matthias Biebl
          </td>
          <td>2024-12-18</td>
          <td>memo - Magazine of European Medical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a765de539c809febe05924111aeac006808c8bb" target='_blank'>
              Characterization of breast cancer tumors in older patients who show de novo resistance to endocrine therapy
              </a>
            </td>
          <td>
            Y. Ishizuka, Y. Horimoto, Men Yuan, Yuko Ueki, H. Onagi, Harumi Saeki, Takuo Hayashi, Tsuyoshi Saito, T. Kawate, Takashi Ishikawa, H. Eguchi, Junichiro Watanabe, G. Kutomi
          </td>
          <td>2024-12-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>35</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [2, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>