<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              2dfc3ad85074a37ea43ae7199e61a35587ec1d29
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-21 10:42:26 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Modeling of the MAPK machinery activation in response to various abiotic and biotic stresses in plants by a system biology approach</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="The 2020 world population data sheet estimates that 30 years from now, the population is intended to outstretch approximately up to 9.9 billion. The exponential rise in population has led to the agitation of land and the environment. Abrupt changes in climatic conditions and increasing global population emphasize the propagation of novel crops and acclimatization of the available plants and crops to obtain sufficient food in the extended future. In addition to ‘Omics’, bioinformatics plays an essential role in understanding the underlying mechanism of molecular functional systems in various plants. Bioinformatics contributes towards multi-disciplinary interactions and helps to reshape agricultural tradition and production, providing knowledge for enhanced plant quality, and it also provides a plan of action for protection against adverse environmental conditions. This review examines recent approaches in systems biology for predicting the functionality of genes and their networks (GRNs) along with its influential effect in current research involving various disciplines like genomics, proteomics, transcriptomics, phenomics, interactomics, ionomics, epigenomics and metabolomics, which will be very useful to researchers in plant sciences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea77f1665e540ce96f242b605195fa88c0ba5b4d" target='_blank'>Recent approaches of Systems Biology and Omics in Plant Research</a></td>
          <td>
            Sweety Guria Rani, Hewanti Kumari, Purbasha Pati, Sadaf Naaz, Tannu Prasad, Ratna Kumari, Humdah Qayyum, Naiyar Naaz, Mukesh Nitin
          </td>
          <td>2021-06-24</td>
          <td>Journal of Current Opinion in Crop Science</td>
          <td>2</td>
          <td>7</td>
        </tr>

        <tr id="For more than a decade, the MAPK (mitogen-activated protein kinase) cascade has been studied using mathematical modelling and quantitative experimentation [1]. The MAPK cascade relays the presence of extracellular stimuli such as growth hormones to the nucleus and controls the expression of hundreds of genes. MAPKs control major cell fate decisions such as proliferation, differentiation and apoptosis, mainly by inducing alterations in gene expression. In this chapter, we discuss how systems biology analysis provides insights into the functioning of this cascade. We show how this pathway assists the cell in responding properly to extracellular cues by filtering out sub-threshold stimuli, while efficiently transmitting physiologically relevant inputs. Several different receptors signal through the MAPK pathway even though they elicit opposite biological responses, thus raising the question of how specificity is achieved in MAPK signalling. Experimental studies revealed that specific biological responses are encoded by quantitative aspects of the MAPK signal such as amplitude or duration. We discuss mechanisms that enable the pathway to generate quantitatively different signals, and also explain how different signals are interpreted by the downstream gene expression machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6240690e740bb11ac1d42f43f91ffcfad059dbaf" target='_blank'>Systems analysis of MAPK signal transduction.</a></td>
          <td>
            N. Blüthgen, S. Legewie
          </td>
          <td>2008-09-30</td>
          <td>Essays in biochemistry</td>
          <td>68</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54562e54df78d295197115d989eb4c681b24700" target='_blank'>Modeling and simulation in signal transduction pathways: a systems biology approach.</a></td>
          <td>
            C. V. Suresh Babu, Eun Joo Song, Y. Yoo
          </td>
          <td>2006-03-01</td>
          <td>Biochimie</td>
          <td>39</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da87720f4767a44da3f92e497492607a5e8a082c" target='_blank'>When kinases meet mathematics: the systems biology of MAPK signalling</a></td>
          <td>
            W. Kolch, Muffy Calder, David R. Gilbert
          </td>
          <td>2005-03-21</td>
          <td>FEBS Letters</td>
          <td>190</td>
          <td>86</td>
        </tr>

        <tr id="The advances in proteomics technologies offer an unprecedented opportunity and valuable resources to understand how living organisms execute necessary functions at systems levels. However, little work has been done up to date to utilize the highly accurate spatio-temporal dynamic proteome data generated by phosphoprotemics for mathematical modeling of complex cell signaling pathways. This work proposed a novel computational framework to develop mathematical models based on proteomic datasets. Using the MAP kinase pathway as the test system, we developed a mathematical model including the cytosolic and nuclear subsystems; and applied the genetic algorithm to infer unknown model parameters. Robustness property of the mathematical model was used as a criterion to select the appropriate rate constants from the estimated candidates. Quantitative information regarding the absolute protein concentrations was used to refine the mathematical model. We have demonstrated that the incorporation of more experimental data could significantly enhance both the simulation accuracy and robustness property of the proposed model. In addition, we used the MAP kinase pathway inhibited by phosphatases with different concentrations to predict the signal output influenced by different cellular conditions. Our predictions are in good agreement with the experimental observations when the MAP kinase pathway was inhibited by phosphatase PP2A and MKP3. The successful application of the proposed modeling framework to the MAP kinase pathway suggests that our method is very promising for developing accurate mathematical models and yielding insights into the regulatory mechanisms of complex cell signaling pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9577e3ebca59ca340c92adad68c098433bbcb84b" target='_blank'>Mathematical Modelling of the MAP Kinase Pathway Using Proteomic Datasets</a></td>
          <td>
            Tianhai Tian, Jiangning Song
          </td>
          <td>2012-08-08</td>
          <td>PLoS ONE</td>
          <td>35</td>
          <td>51</td>
        </tr>

        <tr id="All cells receive and respond to signals from their environment, whether they live freely or are part of a tissue. Feedback loops are one among the interactions in signal-transduction cascades, in which the information from the downstream components is sent back to the upstream components of the system. Crosstalks between signal-transduction cascades are able to combine the information's received and processed to trigger the appropriate response. To understand how signaling networks make decisions, the dynamic interactions of proteins in the signal-transduction cascades are important rather than just creating static maps of pathways. Based on reaction kinetics, we have developed mathematical models to predict and analyze the impacts of the feedback loops in JNK cascades and the crosstalks between JNK and P38 kinase cascades. Feedback loops in JNK cascades and crosstalks between JNK and P38 kinase cascades plays the complementary roles in making the impacts on the activation of JNK and P38 kinases. JNK and P38 kinase cascades with feedbacks and crosstalks is a robust system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2880a4c0be242db0837a9696c152f250827dd584" target='_blank'>Dynamic Modeling and Simulation of JNK and P38 Kinase Cascades With Feedbacks and Crosstalks</a></td>
          <td>
            Pandurangan Sundaramurthy, S. Gakkhar
          </td>
          <td>2010-08-19</td>
          <td>IEEE Transactions on NanoBioscience</td>
          <td>7</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b06d234407efd97d798904ba97ebb9a73b5fb31e" target='_blank'>Methodologies for the modeling and simulation of biochemical networks, illustrated for signal transduction pathways: A primer</a></td>
          <td>
            Nesma ElKalaawy, A. Wassal
          </td>
          <td>2015-03-01</td>
          <td>Bio Systems</td>
          <td>20</td>
          <td>8</td>
        </tr>

        <tr id="In recent years, rapid developments in several omics platforms and next generation sequencing technology have generated a huge amount of biological data about plants. Systems biology aims to develop and use well-organized and efficient algorithms, data structure, visualization, and communication tools for the integration of these biological data with the goal of computational modeling and simulation. It studies crop plant systems by systematically perturbing them, checking the gene, protein, and informational pathway responses; integrating these data; and finally, formulating mathematical models that describe the structure of system and its response to individual perturbations. Consequently, systems biology approaches, such as integrative and predictive ones, hold immense potential in understanding of molecular mechanism of agriculturally important complex traits linked to agricultural productivity. This has led to identification of some key genes and proteins involved in networks of pathways involved in input use efficiency, biotic and abiotic stress resistance, photosynthesis efficiency, root, stem and leaf architecture, and nutrient mobilization. The developments in the above fields have made it possible to design smart crops with superior agronomic traits through genetic manipulation of key candidate genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db40e5699072be10227d6af18ce2ab039767b78c" target='_blank'>Systems Biology for Smart Crops and Agricultural Innovation: Filling the Gaps between Genotype and Phenotype for Complex Traits Linked with Robust Agricultural Productivity and Sustainability.</a></td>
          <td>
            Anil Kumar, R. Pathak, S. Gupta, V. S. Gaur, D. Pandey
          </td>
          <td>2015-10-20</td>
          <td>Omics : a journal of integrative biology</td>
          <td>77</td>
          <td>53</td>
        </tr>

        <tr id="System dynamics is a popular approach in many fields of science and technology, but it has not been investigated for cell signaling pathways yet. It is a well formulated methodology used to analyze the components of a system considering the cause-effect relationships. The two main components of system dynamics modeling, Causal Loop and Stocks-Flow diagrams, make possible to model the dynamics of the system with the ability to analyze it both qualitatively and quantitatively. In this paper, after introducing the system dynamics modeling approach, and giving a simple example from its usage for Michaelis Menton reactions, a three-step process is proposed for signal transduction modeling. Then in a complete example, it is applied to a case study in cell signaling pathways, namely the RKIP Influence on the ERK signaling pathway and the results is compared with an integrative modeling approach based on Petri net and ODEs. Computational simulations show the success of the system dynamic in easier and effective modeling of the cell signaling pathways, in addition to its diverse options for understanding and testing the pathway quantitatively and quantitatively at the same time in relation to other methods. More intestinally, the suggested approach here helps one to identify all biochemical reaction paths and loops for complex cell signaling pathways. It will be a good step forward to define dominant systems for such complex cell signaling pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4149b50e4189896acbe292b78085dbf48d60c22" target='_blank'>Causal loop and Stock-Flow Modeling of Signal Transduction Pathways</a></td>
          <td>
            S. Sulaimany, G. Bidkhori, Sarbaz H. A. Khoshnaw
          </td>
          <td>2024-09-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The International Journal of Systems Biology ISSN: 0975–2900 & E-ISSN: 0975–9204 aims to publish all the latest and outstanding research articles, reviews and letters in all areas of systems biology, bioinformatics, functional genomics and proteomics studies on organisms. Current emphasis is on the bioinformatics, statistics, biology, modeling, simulation, data mining, computational techniques and medicine basis for common and complex diseases and on the functional mechanism, architecture and evolution of gene networks, studied by experimental perturbation. Each issue contains a series of timely, in-depth written articles by leaders in the field, covering a wide range of the integration of multidimensional challenges of genomics, proteomics, systems biology science including integration issues of the gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6afc1aaedeeab4e4791fb0c99f89f57175291f9c" target='_blank'>International Journal of Systems Biology</a></td>
          <td>
            Publisher Bioinfo Publications
          </td>
          <td>2013-09-25</td>
          <td></td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="[This corrects the article DOI: 10.1371/journal.pcbi.1005488.].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/431b4fb0315b7164b03ba7a2c8baa3e155daeb5a" target='_blank'>Correction: A dynamic genetic-hormonal regulatory network model explains multiple cellular behaviors of the root apical meristem of Arabidopsis thaliana</a></td>
          <td>

          </td>
          <td>2019-06-01</td>
          <td>PLoS Computational Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mohamed A El-Esawi1,2* and Enas M El-Ballat2,3 1Division of Crop Biotechnics, KU Leuven, B-3001 Leuven, Belgium 2Botany Department, Faculty of Science, Tanta University, Tanta, Egypt 3Department of Plant Sciences, School of Biosciences, University of Nottingham, UK *Corresponding author: El-Esawi MA, Division of Crop Biotechnics, KU Leuven, B-3001 Leuven, Belgium, E-mail: mohamed.elesawi@science.tanta.edu.eg">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a84186f3829deda114f0fd0ac94871cee52de0" target='_blank'>Integrative Systems Biology: Addressing Biological Processes and Analysis of High-Throughput Bioinformatics Data</a></td>
          <td>
            M. El-Esawi, E. El-Ballat
          </td>
          <td>2016-05-22</td>
          <td></td>
          <td>3</td>
          <td>48</td>
        </tr>

        <tr id="1116-1135">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75b69f21b7b1f504fbad4573f8a9154ac3a688dc" target='_blank'>A Review on Modeling of Plant Cell Growth</a></td>
          <td>
            Jiawei Pei, Changyu Liu, Aimin Wu, Ziwei Cheng, Hansheng Wang, Jingjing Zheng, Hao Huang, Chunyin Wu
          </td>
          <td>2024-03-18</td>
          <td>Computer-Aided Design and Applications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="MAPK (Mitogen Activated Protein Kinase) is a Ser/Thr kinase, which plays a crucial role in plant growth and development, transferring the extra cellular stimuli into intracellular response etc. Manual identification of these MAPK in the plant genome is tedious and time taking process. There are number of online servers which predict the P-site (phosphorylation site), find the motifs and domain but there is no specific tool which can identify all them together. In order to identify the P-Site, phosphorylation site consensus sequences and domain of the MAPK in plant genome, we developed a tool, MAP Kinase analyzer. MAP kinase analyzer take protein sequence as input in the fasta format and the output of tool includes: 1) The prediction of the phosphorylation site viz., Serine (S), Threonine (T), and Tyrosine (Y), Contex, Position, Score and phosphorylating kinase as well as the graphical output; 2) Phosphorylation site consensus sequence pattern for different kinases and 3) Domain information about the MAPK's. The MAP kinase analyser tool and supplementary files can be downloaded from http://www.bioinfogbpuat/mapk_OWN_1/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2b19fba8b3c1bb3be647de5831306fc7a209d05" target='_blank'>MAP Kinase analyser: A tool for plant kinase and substrate analysis</a></td>
          <td>
            Saranga Dhar Samantaray, Parvendra Singh, B. Suresh, Sugandha B. Sharma, G. Taj, Anil Kumar
          </td>
          <td>2011-06-23</td>
          <td>Bioinformation</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e507b8bc19c546b1fd45e787ebfea3b3c1ce5943" target='_blank'>Modelling dynamic plant cells.</a></td>
          <td>
            Junli Liu, Claire S Grieson, A. Webb, P. Hussey
          </td>
          <td>2010-12-01</td>
          <td>Current opinion in plant biology</td>
          <td>17</td>
          <td>56</td>
        </tr>

        <tr id="The study of complex biological systems necessitates computational modeling approaches that are currently underutilized in plant biology. Many plant biologists have trouble identifying or adopting modeling methods to their research, particularly mechanistic mathematical modeling. Here we address challenges that limit the use of computational modeling methods, particularly mechanistic mathematical modeling. We divide computational modeling techniques into either pattern models (e.g., bioinformatics, machine learning, or morphology) or mechanistic mathematical models (e.g., biochemical reactions, biophysics, or population models), which both contribute to plant biology research at different scales to answer different research questions. We present arguments and recommendations for the increased adoption of modeling by plant biologists interested in incorporating more modeling into their research programs. As some researchers find math and quantitative methods to be an obstacle to modeling, we provide suggestions for easy-to-use tools for non-specialists and for collaboration with specialists. This may especially be the case for mechanistic mathematical modeling, and we spend some extra time discussing this. Through a more thorough appreciation and awareness of the power of different kinds of modeling in plant biology, we hope to facilitate interdisciplinary, transformative research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/210c7df7d345eb30c7e316d4670725b68ace3ad1" target='_blank'>Overcoming the Challenges to Enhancing Experimental Plant Biology With Computational Modeling</a></td>
          <td>
            Renee Dale, S. Oswald, A. Jalihal, Marion Laporte, D. Fletcher, Allen H Hubbard, Shin-Han Shiu, Andrew D. Nelson, Alexander Bucksch
          </td>
          <td>2021-07-20</td>
          <td>Frontiers in Plant Science</td>
          <td>7</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2b828d8388c5c4bd707b9d5dc694d13bf8d7094" target='_blank'>Bioinformatics analyses for signal transduction networks</a></td>
          <td>
            Wei Liu, Dong Li, Yun-ping Zhu, F. He
          </td>
          <td>2008-11-07</td>
          <td>Science in China Series C: Life Sciences</td>
          <td>3</td>
          <td>69</td>
        </tr>

        <tr id="The development of multicellular organisms requires the coordinated accomplishment of many molecular and cellular processes, like cell division and differentiation, as well as metabolism. Regulation of those processes must be very reliable and capable of resisting fluctuations of the internal and external environments. Without such homeostatic capacity, the viability of the organism would be compromised. Cellular processes are finely controlled by a number of regulatory molecules. In particular, transcription factors are amongst the proteins that determine the transcription rate of genes, including those involved in development and morphogenesis. For this reason, the molecular mechanisms responsible for the homeostatic capacity and coordinated behavior of the transcriptional machinery have become the focus of several laboratories. Modern techniques of molecular genetics have greatly increased the rate at which genes are recognized and their primary sequences determined. High throughput methods for the identification of gene-gene and protein-protein interactions exist and continue to be refined, raising the prospect of high-throughput determination of networks of interactions in discrete cell types. Still, classic biochemical and physiological studies are necessary to identify the targets, and to understand the functions of the encoded proteins. For such reasons, the rate at which pathways are described is much slower than the rate of genome sequencing. The large quantity of available sequences creates the challenge for molecular geneticists of linking genes and proteins into functional pathways or networks. Biologists are often interested in particular subsets of these very extensive networks obtained with">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48b19165642a0d59940f0f5e4481dc2acb228434" target='_blank'>Strengths and Weaknesses of Selected Modeling Methods Used in Systems Biology</a></td>
          <td>
            P. Kahlem, Alessandro DiCara, M. Durot, John M. Hancock, E. Klipp, V. Schächter, E. Segal, I. Xenarios, E. Birney, L. Mendoza
          </td>
          <td>2011-09-12</td>
          <td></td>
          <td>4</td>
          <td>129</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/206917f5b0eabb49bba3d18dd657b52ffb26a89c" target='_blank'>Computational systems biology of cellular processes in Arabidopsis thaliana: an overview</a></td>
          <td>
            Pascal Holzheu, U. Kummer
          </td>
          <td>2019-11-25</td>
          <td>Cellular and Molecular Life Sciences</td>
          <td>11</td>
          <td>27</td>
        </tr>

        <tr id="The mitogen-activated protein kinase (MAPK) pathway is considered to be a central block in many biological signaling networks. Despite the common core cascade structure, the activation of MAPK in different biological systems can exhibit different types of dynamic behaviors. Computer modeling may help to reveal the mechanisms underlying such variations. However, so far most computational models of the MAPK cascade have been system-specific, or to reflect a particular type among the wide spectrum of possible dynamics. To obtain a general and integrated view of the relationship between the dynamics of MAPK activation and the structures and parameters of the MAPK cascade, we constructed a generic model by comparing previous models covering different specific biological systems. We assumed that reliable qualitative results could be predicted through a qualitative model with pseudo parameters. We used randomly sampled parameters instead of a specific set of “best-fit” parameters to avoid biases towards any particular systems. A range of dynamics behaviors for MAPK activation, including ultrasensitivity, bistability, transient activation and oscillation, were successfully predicted by the generic model. The results indicated that the steady state dynamics (ultrasensitivity and bistability) was jointly determined by the three-tiered structure of the MAPK cascade and the competitive substrate binding in the dual-phosphorylation processes of the central components, while the temporal dynamics (transient activation and oscillation) was mainly affected by the upstream signaling pathway and feedbacks. Moreover, MAPK kinase (MAPKK) played more important roles than the other two components in determining the dynamics of MAPK activation. We hypothesize that this is an important and advantageous property for the regulation and for the functional diversity of MAPK pathways in real cells. Finally, to assist developing generic models for signaling motifs through model comparisons, we proposed a reaction-based database to make the model data more flexible and interoperable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/558a5c17bf4b19875b0696b35de9a583a6c40956" target='_blank'>Random Parameter Sampling of a Generic Three-Tier MAPK Cascade Model Reveals Major Factors Affecting Its Versatile Dynamics</a></td>
          <td>
            Zhong Mai, Haiyan Liu
          </td>
          <td>2013-01-24</td>
          <td>PLoS ONE</td>
          <td>9</td>
          <td>26</td>
        </tr>

        <tr id="There is general agreement that a systems approach is needed for a better understanding of causal and functional relationships that generate the dynamics of biological networks and pathways. These observations have been the basis for efforts to get the engineering and physical sciences involved in life sciences. The emergence of systems biology as a new area of research is evidence for these developments. Dynamic modelling and simulation of signal transduction pathways is an important theme in systems biology and is getting growing attention from researchers with an interest in the analysis of dynamic systems. This paper introduces systems biology in terms of the analysis and modelling of signal transduction pathways. Focusing on mathematical representations of cellular dynamics, a number of emerging challenges and perspectives are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9596a211e449e9b94f067c6038d88522e603242f" target='_blank'>Analysis and modelling of signal transduction pathways in systems biology.</a></td>
          <td>
            Kwang-Hyun Cho, Olaf Wolkenhauer
          </td>
          <td>2003-12-01</td>
          <td>Biochemical Society transactions</td>
          <td>65</td>
          <td>7</td>
        </tr>

        <tr id="In the past few years, systems biology methods have been widely used to explore the mechanism of cancer. Various signaling pathways play the important role in cancer. And a large number of computational methods are used to model the signaling pathway. Here, we reviewed many existing mechanisms and modeling techniques, which are used to model different signaling pathways. Then we introduced the systematic workflow of a system biology approach and some fundamental analysis methods. Finally we discuss some known challenges in the process of modeling signaling pathways and what we should handle in our future work.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05a9ab7b8011464e527bd7731032008ad51b69be" target='_blank'>Review of the computational models for the signaling pathway</a></td>
          <td>
            Meijing Kong, Chao Wang, Le Zhang
          </td>
          <td>2016-05-21</td>
          <td></td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The Signal transduction pathway uses docking interaction with kinases which may also be regulated by phosphorylation. Regulation of these docking interactions by phosphorylation allows an additional level of control over the diverse biological processes including Crosstalk induction of defense. Amino acid sequences surrounding the phosphorylation motif have been extensively studied. However, the present study, attempts have been made to identify the presence of important determinants/motifs for substrate specificity of Mitogen activated protein kinase (MAPK), CaM kinase II, Protein kinase C, Receptor tyrosine kinase in the 54 identified MAPK 3 and MAPK 6 substrates of Arabidopsis thaliana. All identified substrates do not possess the known (p) S/ (p) T-P phosphorylation sites. Out of 54 substrates, 47 have S/T-P site and 7 are lacking such sites for interaction. Likewise 8 shows XXRRX(p)S pattern, 28 shows HYDXRXX(p)SXX pattern , 35 shows HYDXXRXX(p)S pattern, 12 shows R/K(1-3)X(p) S/T(HYD)R/K(1-3) pattern and 17 shows Na(1-3)-X-(p)Y-XX-HYD pattern. In order to ascertain the role of surrounding hydrophobic amino acids in the interaction, the other conserved pattern/Motif were also identified in the MAPK substrates which include XXRRX(p)S, HYDXRXX(p)SXX, HYDXXRXX(p)S, R/K(1-3)X(p)S/T(HYD)R/K(1-3), Na(1-3)-X- (p)Y-XX-HYD (Na-Hydrophilic; HYD- Hydrophobic; X- Any amino acid) pattern. These conserved patterns show activation sites for other kinases viz. MAPK Activated Protein kinase-1, MAPK Activated Protein kinase -2, CaM Kinase II, Protein kinase C, Tyrosine protein kinase respectively. Identification of interacting partners based on the surrounding amino acids of phosphorylation sites could be useful in the understanding of such complex hierarchical networks involved in controlling the defense signaling pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c92a6f93c20fc90022ab73e10bb229590cc0498" target='_blank'>In-silico Approaches for Studying Cross-talk of Different Kinases Associated in Diverse Biological Processes with their Interacting Substrates Partners</a></td>
          <td>
            G. Taj, Payal Agarwal, Anil Kumar
          </td>
          <td>2011-04-15</td>
          <td>Journal of Proteomics & Bioinformatics</td>
          <td>3</td>
          <td>53</td>
        </tr>

        <tr id="Signal transduction pathways are very complex biochemical networks that include a large number of vital biomolecules and chemical compounds whose activities are dependent on member(s) involved in the network under consideration as well as other networks. Modularized study of such networks will be more appropriate for deciphering the underlying mechanisms that control physiological behavior of cell(s). Here we have proposed an algorithm for modularizing signal transduction network which is applied to MAPK pathway of D. melanogaster and H. sapiens. The idea "a member whose activity is controlled by maximum number of other members in the network under consideration, tends to be the center of a module" is used to divide a network into different modules. These modules are created by taking a certain level of complexity c into account, i.e., in an expanding module, a member having more than c relations that lie outside the module, are ignored. Recursive modularization can create modules at different stages of various complexities that finally lead to convergence of the entire network into a single module">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f952e5620dae3eae38b5b53b30ed0d524112a48f" target='_blank'>MAPK Signaling Pathways and Their Recursive Modularization</a></td>
          <td>
            R. K. De, Losiana Nayak
          </td>
          <td>2006-11-21</td>
          <td>2006 15th International Conference on Computing</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="In their Perspective “Systems biology (un)certainties” (23 October 2015, p. [386][1]), P. D. W. Kirk et al. may give readers the impression that mathematical modeling and computer simulations of biological systems have fallen on hard times, owing to difficulties in measuring parameter values.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/233ea41db6c5c236a0a95ad117b55ac4aa2d14b4" target='_blank'>Mechanistic biological modeling thrives.</a></td>
          <td>
            B. O’Shaughnessy, T. Pollard
          </td>
          <td>2016-01-15</td>
          <td>Science</td>
          <td>3</td>
          <td>112</td>
        </tr>

        <tr id="The main aim of this research project is to develop computer-based modelling of the cell cycle as a cell signaling network. This is done as to obtain analytical results based on computer modelling as opposed to the traditional method by experimental lab works. It is also done as to prove that computer-based modelling could give similar results as proposed by the literature review data. The computer-based modelling is based upon the mathematical models of two types of cell cycles; which are the Goldbeter’s minimal cascade model of the mitotic (mitosis) oscillations in fission yeast and Igor Deineko’s mitogen dependent G1/S transition in mammalian cell cycle. These two modelling were successfully developed via Simulink® simulation software, a toolbox within MATLAB®. Sensitivities analyses were also done on two main parameters of the fission yeast mitotic cell cycle, which were the constant rate of cyclin synthesis, vi and the maximum rate of cyclin degradation by protease, vd. It was determined that by maintaining one of these parameters and varying the values of the other parameter, it affected the mitotic oscillations for the fission yeast cell cycle. For the sensitivities analyses done, the vi parameter was determined to have a larger effect on the fission yeast mitotic oscillations; as it was the regulator of the mitotic cell cycle (made the progression in the mitotic cell cycle possible).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1160ea8b456878456cc8620d1a2a98b4b5d9ee3c" target='_blank'>Computer based modelling of the cell cycle as a cell signaling network</a></td>
          <td>
            M. Salimi, C. A. Kent
          </td>
          <td>2021-05-03</td>
          <td>PROCEEDINGS OF GREEN DESIGN AND MANUFACTURE 2020</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="In the last few decades, extensive computational studies have been conducted on signaling pathways with both logic modeling (knowledge-based) and datadriven modeling. The increasing availability of high-throughput molecular data and multiplex techniques for uncovering cellular systems have made predictive in silico modeling more accurate which is important for understanding and engineering cellular functions. Generally, signal transduction inside a cell involves various modifications, such as protein phosphorylation. Through a cascade of biochemical reactions, the signals are transmitted downward to the nucleus or other cellular organelles to regulate physiological functions and finally control the cellular phenotypes (e.g., apoptosis, proliferation and cell cycle). It is widely believed that the dysregulation of signal transduction is one of the most important pathogeneses of many human diseases including cancer, which makes the study of signaling pathways crucial for discovery of new anticancer therapies. However, there remain several challenges for the systems modeling of signaling pathways, such as how to integrate prior knowledge with real data into a context-specific model. This thesis presents the studies I have carried out with my collaborators to address some of the challenges. We first propose a generalized logical model that is able to simulate the graded responses to degradations and the effects of scheduled perturbations to the cells. A network is constructed for computational simulation based on the knowledge extracted from databases and literatures. In the network, the nodes and the edges represent the signaling proteins and the phosphorylation">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5522fe1dd2edf371b0e49a01e45f12ca95cbbd" target='_blank'>Computational prediction of cell fates by systems modelling of cellular signalling pathways</a></td>
          <td>
            Fan Zhang
          </td>
          <td>2016-11-28</td>
          <td></td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Genomics, structural biology, and bioinformatics deal with identification, cataloguing and characterization of the components that make up a cell. The focus on individual genes and proteins, which has proven so powerful for the molecular genetics in the past century, is in itself inadequate to describe the dynamic processes involving interactions among tens, hundreds and even thousands of components. Cell function, including growth, differentiation, division, and apoptosis, are temporal processes and we will only be able to  understand them if we treat them as dynamic systems. There is a general agreement that systems approach is necessary to understand the causal and functional relationships that generate the dynamics of biological networks and pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50615aeb8f2bf6231792a03ece29714ac13e875" target='_blank'>Systems Biology: To Meet Challenges for Biology in the Twenty-first Century</a></td>
          <td>
            Sundaramurthy Pandurangan, S. Gakkhar
          </td>
          <td>2010-12-21</td>
          <td>Research in Biotechnology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19ea531f407b034c82262a3c743a3f5575d55e9b" target='_blank'>Quantitative analysis of signaling networks.</a></td>
          <td>
            Herbert M. Sauro, B. Kholodenko
          </td>
          <td>2004-09-01</td>
          <td>Progress in biophysics and molecular biology</td>
          <td>213</td>
          <td>57</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>