<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              C5c624f60a57498d015d6990eeb7e2ca9bf3aef4
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-21 10:42:32 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>A computational model on the modulation of mitogen-activated protein kinase (MAPK) and Akt pathways in heregulin-induced ErbB signalling.</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="Extra-cellular signal regulated kinase (ERK) signal transduction is an important signaling pathway, involved in a variety of processes in cell and closely related to the occurrences of diverse tumors. This raises the question how these diverse functions are specified and coordinated. In this work, we present an integrated computational model of ERK signal transduction pathway, investigating the regulatory mechanism of ERK signal pathway by variation of the concentration of epidermal growth factor receptor (EGFR) as well as the binding affinity of adaptor proteins. The results show that EGFR and distinct adaptor proteins engender different activation patterns of ERK, which will bring different cellular responses. It could be helpful to understand the mechanism of ERK signal pathway and the relationship between ERK signal network and tumor development or therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8834b7200d12495f21190553b8a4f1751ce0a835" target='_blank'>Dynamic Model and Regulatory Mechanism of Integrated ERK Signal Pathway Activated by Epidermal Growth Factor</a></td>
          <td>
            H. Fang, Yan Li, Yonghua Wang, Shuwei Zhang
          </td>
          <td>2008-10-18</td>
          <td>2008 Fourth International Conference on Natural Computation</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/582489cbec6dab9bd9024775f6d61c58a66145d3" target='_blank'>Kinetic analysis of the MAPK and PI3K/Akt signaling pathways.</a></td>
          <td>
            Suresh Babu C V, Sheikh Md. Enayetul Babar, E. J. Song, Eulsik Oh, Y. Yoo
          </td>
          <td>2008-05-01</td>
          <td>Molecules and cells</td>
          <td>16</td>
          <td>28</td>
        </tr>

        <tr id="Extra-cellular signal regulated kinase (ERK) signal network plays an important role in regulating the fundamental cellular functions such as cell proliferation, survival, differentiation and motility. Development and analysis of mathematical model can help us acquire a deep understanding of the complex behavior of ERK signal transduction network. This paper presents a computational model that offers an integrated quantitative and dynamic simulation of ERK signal transduction network, activated by epidermal growth factor (EGF). The mathematic model contains the activation kinetics of the pathway, a multitude of feedback loops and connection of scaffold proteins. The model provides insight into signal-response relationships between the binding of EGF to its receptor at the cell surface and activation of downstream proteins in the signaling cascade. The different influence of positive and negative feedback loops of the ERK signal transduction pathway were mainly analyzed, indicating that feedback loops were the main impacting factor to the oscillation of ERK signal transduction pathway. The predictions of this oscillation of ERK activation agree well with the literature. It can lead to signal waves of the downstream substrates and induce corresponding biological behaviors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7bf832c4d17a797f80bffbcb87cce1c1078a91f" target='_blank'>Dynamic Analysis of Feedback Loops in Extra-Cellular Signal Regulated Kinase Signal Network</a></td>
          <td>
            Yong Han, Yan Li, Yonghua Wang, H. Fang, Shuwei Zhang
          </td>
          <td>2009-03-31</td>
          <td>2009 WRI World Congress on Computer Science and Information Engineering</td>
          <td>3</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfbf73b4d67dd34c855cf8e556071b7e092bb704" target='_blank'>Input–output behavior of ErbB signaling pathways as revealed by a mass action model trained against dynamic data</a></td>
          <td>
            William W. Chen, B. Schoeberl, P. Jasper, M. Niepel, U. Nielsen, D. Lauffenburger, P. Sorger
          </td>
          <td>2009-01-20</td>
          <td>Molecular Systems Biology</td>
          <td>429</td>
          <td>119</td>
        </tr>

        <tr id="Cell signaling pathways control cells' responses to their environment through an intricate network of proteins and small molecules partitioned by intracellular structures, such as the cytoskeleton and nucleus. Our understanding of these pathways has been revised recently with the advent of more advanced experimental techniques; no longer are signaling pathways viewed as linear cascades of information flowing from membrane-bound receptors to the nucleus. Instead, such pathways must be understood in the context of networks, and studying such networks requires an integration of computational and experimental approaches. This understanding is becoming more important in designing novel therapies for diseases such as cancer. Using the MAPK (mitogen-activated protein kinase) and PI3K (class I phosphoinositide-3' kinase) pathways as case studies of cellular signaling, we give an overview of these pathways and their functions. We then describe, using a number of case studies, how computational modeling has aided in understanding these pathways' deregulation in cancer, and how such understanding can be used to optimally tailor current therapies or help design new therapies against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/653dd939407ee3714dc5a23c5389f2d840b8639e" target='_blank'>Modeling Signaling Networks to Advance New Cancer Therapies.</a></td>
          <td>
            J. Saez-Rodriguez, Aidan MacNamara, S. Cook
          </td>
          <td>2015-12-07</td>
          <td>Annual review of biomedical engineering</td>
          <td>31</td>
          <td>59</td>
        </tr>

        <tr id="Abstract Computer simulations and mathematical modeling of biological processes are becoming increasingly popular, and yet the complexity of the biochemical systems or the differences between experimental setups make it very difficult to establish a standard formula for these modeling projects. Before we can start using computer‐based models for predictions or targeted experiment designs, it is very important to establish a reliable model on which those predictions can be based and experimentally tested. Here we attempt to present a computer model for the mitogen‐activated protein kinase (MAPK) signaling cascade which is consistent with previously published experimental results. In this study we have focused our attention to a generic MAPK ERK (extracellular signal‐regulated kinase) pathway activated by epidermal growth factor (EGF) in an attempt to understand how receptors may achieve different activation kinetics of the MAPK signaling. We successfully show that the level of receptor expression is one key determinant in this regulation, and that the binding affinity of the active receptor to adaptor proteins can have a small but albeit direct effect on the downstream activation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dde366641aca5125603232bb38f2d61b4eda2a5" target='_blank'>A Computer‐Based Model for the Regulation of Mitogen Activated Protein Kinase (MAPK) Activation</a></td>
          <td>
            I. Aksan, M. Levent Kurnaz
          </td>
          <td>2003-01-01</td>
          <td>Journal of Receptors and Signal Transduction</td>
          <td>26</td>
          <td>4</td>
        </tr>

        <tr id="3509 Signal transduction is the primary mechanism used by cells to respond to external stimuli. Defects of signaling pathways cause perturbation in signal flow leading to pathological states including cancer. New targeted therapies are being developed to correct such defects by either selective blocking of pro-survival or activation of pro-apoptotic pathways. However, individual signaling pathways do not act in isolation. They interact with each other, forming complex signaling networks that respond to diverse, often contradictory, stimuli. Therefore, a systems level understanding of signaling network is needed to improve the outcome of targeted therapies.  Our efforts focus on the signaling network formed by the interaction of three signaling pathways: Epidermal Growth Factor (EGF), Insulin-like Growth Factor (IGF), and Insulin (Ins). All three pathways are pro-survival and have been experimentally linked to cancer. Here we report, to our best knowledge, the first computational and experimental analysis of the crosstalk occurring between these three pathways.  First, using STKE data base, we created network encompassing EGF, IGF and Ins receptor pathways. The network was represented as a weighted directed graph with two types of edges: activation and inhibition edges. To determine the role of each signaling molecule in signal transduction, we introduced two measurements: pathway crosstalk and network crosstalk. Based on the static analysis of the network, we selected 13 molecules with grater than zero network and pathway crosstalk value: MAPK, Grb2, c-Src, Raf-1, Akt1, Shc, PI3/4//5/P3, PKC, JNK, CREB, p70S6K, PI3K Ia, Rho. All these molecules were dynamically tested using a simple computational model. The effects of changes in activation of three receptors: EGF receptor, IGF1 receptor, and Ins receptor on signal transduction through the selected molecules was simulated. The results showed that each tested molecule has its unique activation pattern.  Partial in vitro validation of the obtained results was performed using ovarian cancer cell line SKOV3 expressing all three receptors. Serum starved cells were differentially stimulated with EGF, IGF and Insulin, reflecting simulation conditions. Resulting changes in the activation of pro-survival and pro-proliferation pathways were monitored by measurement of phosphorylation of Akt1 and Erk1/2, respectively, using MSD biomarker detection assay. Obtained results indicated that even in its simplest version, the model was able to predict some (although not all) experimentally observed trends and thus it is a good starting point for our iterative model building procedure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58cc142622e0ac0e831ba4f75cc5b95c2459cbb6" target='_blank'>Systemic approach to study the crosstalk between EGF, IGF, and Insulin Pathways - computational and experimental analysis</a></td>
          <td>
            Rafal Zieliński, P. Przytycki, T. Przytycka, J. Capala
          </td>
          <td>2008-05-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="The epidermal growth factor receptor (EGFR) signalling pathway is a complex signalling process with a wide network of interactions. The activation of the mitogen-activated protein kinases (MAPKs) cascade by this activated EGFR has been well studied. MAPKs form a highly integrated network, which is essential for certain specialised cell functions. This paper presents a kinetic model for the MAPK pathway downstream of the EGFR using a biochemical simulator. The model includes 30 signalling events and 29 signalling molecules. The time course data were examined for the activation of each signalling component. The simulation provides a large volume of data, by monitoring the kinetics of the signalling components, which were compared experimentally using the PC12 cell line. The kinetic model corresponded well with the experimental results observed in the EGFR induced activation of proteins. An examination of the kinetic analysis of the multiple signalling events provides a quantitative framework for representing the EGFR signalling network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ab4070943b1039a9e896666b7a81fc1863c7dc4" target='_blank'>Simulation and sensitivity analysis of phosphorylation of EGFR signal transduction pathway in PC12 cell model.</a></td>
          <td>
            C. V. S. Babu, S. Yoon, H. Nam, Young Sook Yoo
          </td>
          <td>2004-12-01</td>
          <td>Systems biology</td>
          <td>20</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0b74a9d445f7e808ca6701314073d3087d2ea19" target='_blank'>Model analysis of difference between EGF pathway and FGF pathway.</a></td>
          <td>
            Satoshi Yamada, Takaharu Taketomi, A. Yoshimura
          </td>
          <td>2004-02-20</td>
          <td>Biochemical and biophysical research communications</td>
          <td>79</td>
          <td>121</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ce2d9991d11077a69e456a1b3c9243e4f3a19ad" target='_blank'>Applications of sensitivity analysis for drug discovery and development in the ErbB receptor network</a></td>
          <td>
            B. Harms, Allen Lee, Ricardo Paxson, U. Nielsen, B. Schoeberl
          </td>
          <td>2007-05-08</td>
          <td>BMC Systems Biology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="In the past few years, systems biology methods have been widely used to explore the mechanism of cancer. Various signaling pathways play the important role in cancer. And a large number of computational methods are used to model the signaling pathway. Here, we reviewed many existing mechanisms and modeling techniques, which are used to model different signaling pathways. Then we introduced the systematic workflow of a system biology approach and some fundamental analysis methods. Finally we discuss some known challenges in the process of modeling signaling pathways and what we should handle in our future work.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05a9ab7b8011464e527bd7731032008ad51b69be" target='_blank'>Review of the computational models for the signaling pathway</a></td>
          <td>
            Meijing Kong, Chao Wang, Le Zhang
          </td>
          <td>2016-05-21</td>
          <td></td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Everyday cells encounter various stimuli ranging from growth signals to bacterial infections to UV insult to death signals. They must somehow receive these signals, interpret them, integrate different stimuli, and generate the required output. They can do so by an intricate mechanism named intracellular signal transduction. There are various signal transduction pathways within a cell, each of which are designed for a particular stimulus, and all of which can crosstalk among themselves for the careful integration of all stimuli. In this report, only one of these pathways, the mitogen-activated protein kinase (MAPK) pathway has been simulated. This pathway is activated upon binding of growth factors to their respective cell surface-bound receptors. Activated receptors relay the incoming signal to the cell interior via a cascade of proteins, which are thought to be involved in both the amplification of the signal, and the specificity of the pathway. A generic MAPK pathway activated by the EGF (epidermal growth factor) and the effect of receptor overexpression has been studied, and consistent with experimental evidence, it is shown that the number of EGF receptors on the cell surface is a key factor in the response generated by the pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60459cf132bf5912a5192bbf5cbc9a589b40edec" target='_blank'>Computer simulation of differential kinetics of MAPK activation upon EGF receptor overexpression</a></td>
          <td>
            I. Aksan, M. Sen, M. Araz, M. Levent Kurnaz
          </td>
          <td>2001-10-25</td>
          <td>2001 Conference Proceedings of the 23rd Annual International Conference of the IEEE Engineering in Medicine and Biology Society</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Vascular endothelial growth factor (VEGF) signal transduction is central to angiogenesis in development and in pathological conditions such as cancer, retinopathy and ischemic diseases. However, no detailed mass-action models of VEGF receptor signaling have been developed. We constructed and validated the first computational model of VEGFR2 trafficking and signaling, to study the opposing roles of Gab1 and Gab2 in regulation of Akt phosphorylation in VEGF-stimulated endothelial cells. Trafficking parameters were optimized against 5 previously published in vitro experiments, and the model was validated against six independent published datasets. The model showed agreement at several key nodes, involving scaffolding proteins Gab1, Gab2 and their complexes with Shp2. VEGFR2 recruitment of Gab1 is greater in magnitude, slower, and more sustained than that of Gab2. As Gab2 binds VEGFR2 complexes more transiently than Gab1, VEGFR2 complexes can recycle and continue to participate in other signaling pathways. Correspondingly, the simulation results show a log-linear relationship between a decrease in Akt phosphorylation and Gab1 knockdown while a linear relationship was observed between an increase in Akt phosphorylation and Gab2 knockdown. Global sensitivity analysis demonstrated the importance of initial-concentration ratios of antagonistic molecular species (Gab1/Gab2 and PI3K/Shp2) in determining Akt phosphorylation profiles. It also showed that kinetic parameters responsible for transient Gab2 binding affect the system at specific nodes. This model can be expanded to study multiple signaling contexts and receptor crosstalk and can form a basis for investigation of therapeutic approaches, such as tyrosine kinase inhibitors (TKIs), overexpression of key signaling proteins or knockdown experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ecea14e5a0ecfd87840ca47c28228c8a46d59ca" target='_blank'>Computational Model of Gab1/2-Dependent VEGFR2 Pathway to Akt Activation</a></td>
          <td>
            W. Tan, A. Popel, F. Mac Gabhann
          </td>
          <td>2013-06-21</td>
          <td>PLoS ONE</td>
          <td>41</td>
          <td>69</td>
        </tr>

        <tr id="表皮生长因子受体(epidermal growth factor receptor: EGFR)在细胞信号转导中发挥了重要作用，调节细胞的增殖、分化与迁移。EGFR与多种癌症密切相关，这使其成为一个理想的抗肿瘤药物靶点。本文对EGFR的结构生物学及计算生物学的研究进展做一综述。通过结构生物学家的不懈努力，已经对EGFR的三维空间结构有了较为清晰的认识。计算机模拟是研究生物大分子动态过程的重要手段之一，可以在EGFR静态结构的基础上，模拟其在配体诱导的激活过程中发生的大尺度、长时程的动态构象变化。对EGFR的结构及其分子内部运动的研究，有助于揭示其发挥生物学功能的分子机理，为抗肿瘤药物的设计开发提供指导。 Epidermal growth factor receptor (EGFR) plays a critical role in the network of signal transduction, and regulates cell proliferation, differentiation, and migration. EGFR is implicated in different kinds of cancers, which makes it an excellent target for anti-tumor therapy. This review considers structural and computational studies of EGFR, and their impact on our understanding of EGFR function. With continuing efforts of biologists, many high-resolution structures of different regions in EGFR, which include the extracellular region and the intracellular tyrosine kinase do-main, have been solved. Starting from these structures, computer simulation techniques enable us to investigate large length-scale and long time-scale conformational changes during the EGFR activation. All the studies will be helpful to reveal the molecular mechanism in EGFR regulation, and provide valuable knowledge for developing anti-cancer drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/304b3eca9d24041459b6626ebf25b125ee7fd861" target='_blank'>表皮生长因子受体的结构与计算生物学研究进展 Progress in Structural and Computational Studies of Epidermal Growth Factor Receptor</a></td>
          <td>
            张泳辉, 张志勇
          </td>
          <td>2013-05-24</td>
          <td>Biophysics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f1a9d531192299eb19e35e6abb7a3ac2875cf6d" target='_blank'>Inspecting the Role of PI3K/AKT Signaling Pathway in Cancer Development Using an In Silico Modeling and Simulation Approach</a></td>
          <td>
            Pedro Pablo González Pérez, M. Cárdenas-García
          </td>
          <td>2018-04-25</td>
          <td>MAG, DBLP</td>
          <td>3</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dd8c0e59cc6f86078f2c60ed2d73016f4951d76" target='_blank'>Computational modelling of cancerous mutations in the EGFR/ERK signalling pathway</a></td>
          <td>

          </td>
          <td>2009-10-05</td>
          <td>BMC Systems Biology</td>
          <td>64</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6319b79882fa8bd07267c8cafc082740cd9779b6" target='_blank'>Computational modeling of the EGFR network elucidates control mechanisms regulating signal dynamics</a></td>
          <td>
            Dennis Wang, L. Cardelli, Andrew Phillips, Nir Piterman, J. Fisher
          </td>
          <td>2009-12-22</td>
          <td>BMC Systems Biology</td>
          <td>40</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b0d4a5d209975c2d5166fd635acb02a22157e56" target='_blank'>The crosstalk between EGF, IGF, and Insulin cell signaling pathways - computational and experimental analysis</a></td>
          <td>
            Rafal Zieliński, P. Przytycki, Jie Zheng, David Zhang, T. Przytycka, J. Capala
          </td>
          <td>2009-09-04</td>
          <td>BMC Systems Biology</td>
          <td>62</td>
          <td>51</td>
        </tr>

        <tr id="The epidermal growth factor receptor (EGFR) activated extracellular-signal regulated kinase (ERK) pathway is a central cell signalling pathway that mediates many biological responses including cell proliferation, transformation, survival and motility. Deregulation of the pathway either through mutation of components or overexpression of EGFRs is associated with several forms of cancer. Under normal conditions, EGF stimulates a rapid but transient activation of ERK as the signal is rapidly shutdown, whereas under cancerous conditions, the ERK signal cannot be shutdown and is sustained. Computational modelling techniques have been used to investigate the signalling dynamics of the EGFR/ERK pathway, focusing on identifying the key processes involved in signal termination and what role the ERK to son of sevenless (SOS) negative feedback loop plays in generating a transient response. This model predicts that this negative feedback loop is not needed to achieve a transient activation of ERK as the process of receptor degradation alone is enough to terminate the signal. Importantly, the behaviour and predictions of this model are verified with laboratory data, as is essential for modern systems biology approaches. Further analysis showed that the feedback loop and receptor degradation were both redundant processes, as each could compensate for the absence of the other. This led to the prediction that in the case of a receptor which is not degraded, such as the insulin receptor, the negative feedback loop to SOS will actually be essential for a transient response to be achieved. Overall, the results shed new light on the role of negative feedback in EGF receptor signalling and suggest that different receptors are dependent on different features within the ERK pathway when relaying their signals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35e7094d70c15e7e0196f4a688c075edd2e156e0" target='_blank'>Computational modelling reveals feedback redundancy within the epidermal growth factor receptor/extracellular-signal regulated kinase signalling pathway.</a></td>
          <td>
            Richard J. Orton, Oliver E. Sturm, Amelie Gormand, Walter Kolch, David R. Gilbert
          </td>
          <td>2008-08-01</td>
          <td>IET systems biology</td>
          <td>21</td>
          <td>13</td>
        </tr>

        <tr id="Abstract : Therapeutic targeting of the PI3K/Akt/mTOR pathway is expected to be highly effective against breast cancer, and numerous clinical trials are underway for compounds targeting pathway components. However, current research is revealing that the connectivity and dynamical behavior of this network is highly complex, due to numerous feedback loops and crosstalk connections. The goal of this research project is to develop quantitative models of the PI3K network that will enable the effects of inhibitors to be predicted across different types of cell lines and tumors. We have now constructed and are validating two different quantitative models of signaling in the PI3K network. The first, a mechanistic rule-based model, recapitulates the molecular steps involved in the membrane recruitment and phosphorylation of Akt. In developing this model, we have constructed a number of new experimental tools, and these have revealed a novel positive feedback loop controlling phosphorylation at Ser473, a key step in Akt activation. The second model is a data-driven model that is based on high throughput imaging and high-content quantitation of individual cells responding to growth factors and signaling pathway inhibitors. This model seeks to link the signaling states of individual cells to the behavior of the population, and has revealed quantitative differences between normal and oncogene-driven cell proliferation in the efficiency of PI3K/Akt/mTOR signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b14c2cd9d82834bc4882f54621d663826d673cf" target='_blank'>Experimental Analysis and Computational Modeling of Network States and Drug Responses in the PI3K/Akt/mTOR Network</a></td>
          <td>
            J. Albeck
          </td>
          <td>2010-09-01</td>
          <td></td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54562e54df78d295197115d989eb4c681b24700" target='_blank'>Modeling and simulation in signal transduction pathways: a systems biology approach.</a></td>
          <td>
            C. V. Suresh Babu, Eun Joo Song, Y. Yoo
          </td>
          <td>2006-03-01</td>
          <td>Biochimie</td>
          <td>39</td>
          <td>20</td>
        </tr>

        <tr id="The phosphorylation of cellular proteins plays a crucial role in the transduction of various signals from outside the cell into the nucleus. The signals are transduced by phosphorylation chain reactions within multiple pathways; however, determining which pathways are responsible for each defined signal has proven challenging. To estimate the activity of each pathway, we developed a phosphorylation array platform comprising a protein array with 1200 proteins belonging to 376 signalling pathways and an analytical method to estimate pathway activity based on the phosphorylation levels of proteins. The performance of our system was assessed by reconstructing kinase–substrate relationships, as well as by estimating pathway activity upon epidermal growth factor (EGF) stimulation and the pharmacological inhibition of epidermal growth factor receptor (EGFR). As a result, kinase–substrate relationships were reliably reconstructed based on the precise measurement of phosphorylation levels of constituent proteins on the array. Furthermore, the pathway activities associated with EGF stimulation and EGFR inhibition were successfully traced through the related pathways from the outer membrane to the nucleus along a time course. Thus, our phosphorylation array system can effectively assess the activity of specific signalling pathways that are perturbed by extracellular stimuli, such as various drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f9c27ddb3e4d2f7897fe0cb3dc1a2581cdd3b73" target='_blank'>Assessing the activation/inhibition of tyrosine kinase‐related pathways with a newly developed platform</a></td>
          <td>
            Harumi Kagiwada, Takayuki Kiboku, Hitomi Matsuo, M. Kitazawa, K. Fukui, K. Horimoto
          </td>
          <td>2021-06-21</td>
          <td>PROTEOMICS</td>
          <td>5</td>
          <td>47</td>
        </tr>

        <tr id="MOTIVATION
G protein-coupled receptors (GPCRs) are central to how cells respond to their environment and a major class of pharmacological targets. However, comprehensive knowledge of which pathways are activated and deactivated by these essential sensors is largely unknown. To better understand the mechanism of GPCR signaling system, we integrated five independent genome-wide expression datasets, representing 275 human tissues and cell lines, with protein-protein interactions and functional pathway data.


RESULTS
We found that tissue-specificity plays a crucial part in the function of GPCR signaling system. Only a few GPCRs are expressed in each tissue, which are coupled by different combinations of G-proteins or β-arrestins to trigger specific downstream pathways. Based on this finding, we predicted the downstream pathways of GPCR in human tissues and validated our results with L1000 knockdown data. In total, we identified 154,988 connections between 294 GPCRs and 690 pathways in 240 tissues and cell types.


AVAILABILITY AND IMPLEMENTATION
The source code and results supporting the conclusions of this article are available at http://tatonettilab.org/resources/GOTE/source_code/ CONTACT: nick.tatonetti@columbia.eduSupplementary information: Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ec3eaebfb381a5149f07397973b3f2a70bdc1f" target='_blank'>Predicting G protein-coupled receptor downstream signaling by tissue expression</a></td>
          <td>
            Yun Hao, N. Tatonetti
          </td>
          <td>2016-08-02</td>
          <td>Bioinformatics</td>
          <td>13</td>
          <td>41</td>
        </tr>

        <tr id="An error led to part of Figure 2 being removed, breaking the red line that should have linked the grey circles “PKC” and “EGFR”. The correct version of Figure 2 is now displayed on both the PDF and HTML versions of the article.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/892ad05c7eb0020aec314c7a7afee5e1e1dd1b95" target='_blank'>Correction: Integrative Modelling of the Influence of MAPK Network on Cancer Cell Fate Decision</a></td>
          <td>
            L. Grieco, Laurence Calzone, I. Bernard-Pierrot, F. Radvanyi, B. Kahn-Perlès, D. Thieffry
          </td>
          <td>2013-11-01</td>
          <td>PLoS Computational Biology</td>
          <td>41</td>
          <td>61</td>
        </tr>

        <tr id="Embryonic development and adult tissue homeostasis are controlled through activation of intracellular signal transduction pathways by extracellular growth factors. In the past, signal transduction has largely been regarded as a linear process. However, more recent data from large‐scale and high‐throughput experiments indicate that there is extensive cross‐talk between individual signaling cascades leading to the notion of a signaling network. The behavior of such complex networks cannot be predicted by simple intuitive approaches but requires sophisticated models and computational simulations. The purpose of such models is to generate experimentally testable hypotheses and to find explanations for unexpected experimental results. Here, we discuss the need for, and the future impact of, mathematical models for exploring signal transduction in different biological contexts such as for example development. BioEssays 30:1110–1125, 2008. © 2008 Wiley Periodicals, Inc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8545f3dc8505b17afcd909dbddc0e64abbc53ce2" target='_blank'>Network modeling of signal transduction: establishing the global view</a></td>
          <td>
            H. Kestler, Christian Wawra, Barbara Kracher, M. Kühl
          </td>
          <td>2008-11-01</td>
          <td>BioEssays</td>
          <td>94</td>
          <td>56</td>
        </tr>

        <tr id="MicroRNAs have gained significant interest due to their widespread occurrence and diverse functions as regulatory molecules, which are essential for cell division, growth, development and apoptosis in eukaryotes. The epidermal growth factor receptor (EGFR) signaling pathway is one of the best investigated cellular signaling pathways regulating important cellular processes and its deregulation is associated with severe diseases, such as cancer. In this study, we introduce a systems biological model of the EGFR signaling pathway integrating validated miRNA-target information according to diverse studies, in order to demonstrate essential roles of miRNA within this pathway. The model consists of 1241 reactions and contains 241 miRNAs. We analyze the impact of 100 specific miRNA inhibitors (anit-miRNAs) on this pathway and propose that the embedded miRNA-network can help to identify new drug targets of the EGFR signaling pathway and thereby support the development of new therapeutic strategies against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6498d712ff2f4991ed44e3c86cf3f3b3f164ebfa" target='_blank'>Modeling of miRNA and Drug Action in the EGFR Signaling Pathway</a></td>
          <td>
            Jian Li, V. Pandey, T. Kessler, H. Lehrach, C. Wierling
          </td>
          <td>2012-01-11</td>
          <td>PLoS ONE</td>
          <td>13</td>
          <td>124</td>
        </tr>

        <tr id="Over the past decade we have seen a shift in cancer therapy from broadly cytotoxic drugs to molecular therapies targeting “driver” mutations. Although targeted therapy has seen great success for some cancers (e.g. imatinib for leukemia), it has struggled with poor efficacy in treating other cancers that can sometimes possess multiple “driver” mutations. This highlights the complex, and at times non-intuitive, interplay between multiple players in a signaling cascade, which can be highly dependent on the biological context – that is, gene expression levels and mutational architecture – of a tumor or cell line. A quantitative, mechanistic, biologically-tailored understanding of how these signaling dynamics drive proliferation and death could improve precision pharmacology approaches to treat cancer. Here, we constructed the first highly detailed, large-scale ordinary differential equation (ODE) mechanistic mathematical model depicting the most commonly mutated cancer signaling pathways across human cancers, as indicated by a pan-cancer analysis by The Cancer Genome Atlas (TCGA). The model includes the RTK/Ras/MAPK, PI3K/AKT/mTOR, CDK/RB cell cycle, p53/MDM2 DNA damage response, and BCL/Caspases apoptosis pathways. The adjustable parameters of the model can be informed by measurements from patients or cell lines, including copy number alterations, mutations, and gene expression levels. This single-cell model links stochastic gene expression processes to quantitative signaling dynamics, and once tailored to a biological context can be used to simulate the effect of various anti-cancer therapies on cell fate behavior such as proliferation and death for a population of cells. The first instance of the model integrated genomic, transcriptomic, and proteomic data from the MCF10A cell line, a non-transformed cell line with predictable phenotypic behaviors. We trained the model using western blot and flow cytometry experiments to refine various biochemical parameters and phenotypic outcomes. Many fundamental questions in signal transduction arose during this process, such as how EGF and insulin synergize to drive S-phase entry or how a specific biological context confers sensitivity or resistance to inhibitors of the ERK and AKT pathways. Simultaneously, we are tailoring the model to patient-derived genetic information from primary glioblastoma tumors and screening brain-penetrable compounds in a patient-specific manner. In conclusion, a quantitative, mechanistic, biologically-tailored mathematical model depicting the major cancer pathways allows us to probe the mechanisms that underlie how signaling dynamics drive proliferation and death in response to various perturbations, and gain insight into their dependence on the biological context of cell lines and patient tumors. Note: This abstract was not presented at the meeting. Citation Format: Mehdi Bouhaddou, Anne Marie Barrette, Rick J. Koch, Marc R. Birtwistle. Predicting stochastic proliferation and death in response to drugs with mechanistic models tailored to genomic, transcriptomic, and proteomic data [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2017; 2017 Apr 1-5; Washington, DC. Philadelphia (PA): AACR; Cancer Res 2017;77(13 Suppl):Abstract nr 1568. doi:10.1158/1538-7445.AM2017-1568">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b9460f53b1393fa834eeb46b9c94177968a96a" target='_blank'>Abstract 1568: Predicting stochastic proliferation and death in response to drugs with mechanistic models tailored to genomic, transcriptomic, and proteomic data</a></td>
          <td>
            M. Bouhaddou, A. Barrette, R. J. Koch, M. Birtwistle
          </td>
          <td>2017-07-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Targeted cancer therapies are powerful alternatives to chemotherapies or can be used complementary to these. Yet, the response to targeted treatments depends on a variety of factors, including mutations and expression levels, and therefore their outcome is difficult to predict. Here, we develop a mechanistic model of gastric cancer to study response and resistance factors for cetuximab treatment. The model captures the EGFR, ERK and AKT signaling pathways in two gastric cancer cell lines with different mutation patterns. We train the model using a comprehensive selection of time and dose response measurements, and provide an assessment of parameter and prediction uncertainties. We demonstrate that the proposed model facilitates the identification of causal differences between the cell lines. Furthermore, our study shows that the model provides accurate predictions for the responses to different perturbations, such as knockdown and knockout experiments. Among other results, the model predicted the effect of MET mutations on cetuximab sensitivity. These predictive capabilities render the model a powerful basis for the assessment of gastric cancer signaling and for the development and discovery of predictive biomarkers. Author Summary Unraveling the causal differences between drug responders and non-responders is an important challenge. The information can help to understand molecular mechanisms and to guide the selection and design of targeted therapies. Here, we approach this problem for cetuximab treatment for gastric cancer using mechanistic mathematical modeling. The proposed model describes multiple gastric cancer cell lines and can accurately predict the response in several validation experiments. Our analysis provides a differentiated view on mutations and explains, for instance, the relevance of MET mutations and the insignificance of PIK3CA mutation in the considered cell lines. The model might provide the basis for understanding the recent failure of several clinical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9c95ce3ab03d0c251ffaf0c848e40145e703fa6" target='_blank'>Model-based analysis of response and resistance factors of cetuximab treatment in gastric cancer cell lines</a></td>
          <td>
            E. Raimúndez, S. Keller, G. Zwingenberger, K. Ebert, S. Hug, Fabian J Theis, D. Maier, B. Luber, J. Hasenauer
          </td>
          <td>2019-05-31</td>
          <td>PLoS Computational Biology</td>
          <td>9</td>
          <td>98</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d85c2d54218ee5c7b97e6ccbac485cc8ab585bd" target='_blank'>Reconstruction and analysis of a signal transduction network using HeLa cell protein–protein interaction data</a></td>
          <td>
            Pei-Jung Wu, Wu-Hsiung Wu, Tzu-Chi Chen, Kuan-Ting Lin, Jin-Mei Lai, Chi-Ying F. Huang, Feng-Sheng Wang
          </td>
          <td>2014-11-01</td>
          <td>Journal of The Taiwan Institute of Chemical Engineers</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Cell responses are actuated by tightly controlled signal transduction pathways. Although the concept of an integrated signaling network replete with interpathway cross-talk and feedback regulation is broadly appreciated, kinetic data of the type needed to characterize such interactions in conjunction with mathematical models are lacking. In mammalian cells, the Ras/ERK pathway controls cell proliferation and other responses stimulated by growth factors, and several cross-talk and feedback mechanisms affecting its activation have been identified. In this work, we take a systematic approach to parse the magnitudes of multiple regulatory mechanisms that attenuate ERK activation through canonical (Ras-dependent) and non-canonical (PI3K-dependent) pathways. In addition to regulation of receptor and ligand levels, we consider three layers of ERK-dependent feedback: desensitization of Ras activation, negative regulation of MEK kinase (e.g. Raf) activities, and up-regulation of dual-specificity ERK phosphatases. Our results establish the second of these as the dominant mode of ERK self-regulation in mouse fibroblasts. We further demonstrate that kinetic models of signaling networks, trained on a sufficient diversity of quantitative data, can be reasonably comprehensive, accurate, and predictive in the dynamical sense.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405be6d692d4836d3b3efde9333fdf1789ff05c7" target='_blank'>Systematic Quantification of Negative Feedback Mechanisms in the Extracellular Signal-regulated Kinase (ERK) Signaling Network*</a></td>
          <td>
            M. Cirit, Chun-Chao Wang, J. Haugh
          </td>
          <td>2010-09-16</td>
          <td>The Journal of Biological Chemistry</td>
          <td>91</td>
          <td>33</td>
        </tr>

        <tr id="The main objective of this review is to emphasize the role and importance of the careful mathematical/computational modeling of signaling networks for the understanding of aberrant signaling in cancer and for the development of targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c07288770fb4f39f25e31957e3e4132e5412974" target='_blank'>On the role of cell signaling models in cancer research.</a></td>
          <td>
            A. Ventura, Trachette L. Jackson, S. Merajver
          </td>
          <td>2009-01-15</td>
          <td>Cancer research</td>
          <td>39</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c47f6df71a9c01156d35a1778b1a714fa9e175c" target='_blank'>A Multiscale Computational Approach to Dissect Early Events in the Erb Family Receptor Mediated Activation, Differential Signaling, and Relevance to Oncogenic Transformations</a></td>
          <td>
            Yingting Liu, J. Purvis, Andrew J. Shih, Joshua A. Weinstein, Neeraj J. Agrawal, R. Radhakrishnan
          </td>
          <td>2007-02-02</td>
          <td>Annals of Biomedical Engineering</td>
          <td>39</td>
          <td>40</td>
        </tr>

        <tr id="Simple Summary Temporal signaling dynamics are important for controlling the fate decisions of mammalian cells. In this study, we developed BioMASS, a computational platform for prediction and analysis of signaling dynamics using RNA-sequencing gene expression data. We first constructed a detailed mechanistic model of early transcriptional regulation mediated by ErbB receptor signaling pathway. After training the model parameters against phosphoprotein time-course datasets obtained from breast cancer cell lines, the model successfully predicted signaling activities of another untrained cell line. The result indicates that the parameters of molecular interactions in these different cell types are not particularly unique to the cell type, and the expression levels of the components of the signaling network are sufficient to explain the complex dynamics of the networks. Our method can be further expanded to predict signaling activity from clinical gene expression data for in silico drug screening for personalized medicine. Abstract A current challenge in systems biology is to predict dynamic properties of cell behaviors from public information such as gene expression data. The temporal dynamics of signaling molecules is critical for mammalian cell commitment. We hypothesized that gene expression levels are tightly linked with and quantitatively control the dynamics of signaling networks regardless of the cell type. Based on this idea, we developed a computational method to predict the signaling dynamics from RNA sequencing (RNA-seq) gene expression data. We first constructed an ordinary differential equation model of ErbB receptor → c-Fos induction using a newly developed modeling platform BioMASS. The model was trained with kinetic parameters against multiple breast cancer cell lines using autologous RNA-seq data obtained from the Cancer Cell Line Encyclopedia (CCLE) as the initial values of the model components. After parameter optimization, the model proceeded to prediction in another untrained breast cancer cell line. As a result, the model learned the parameters from other cells and was able to accurately predict the dynamics of the untrained cells using only the gene expression data. Our study suggests that gene expression levels of components within the ErbB network, rather than rate constants, can explain the cell-specific signaling dynamics, therefore playing an important role in regulating cell fate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39b85b8aebc0242791c8944fd3435e2529943c50" target='_blank'>A Computational Framework for Prediction and Analysis of Cancer Signaling Dynamics from RNA Sequencing Data—Application to the ErbB Receptor Signaling Pathway</a></td>
          <td>
            Hiroaki Imoto, Suxiang Zhang, Mariko Okada
          </td>
          <td>2020-10-01</td>
          <td>Cancers</td>
          <td>21</td>
          <td>12</td>
        </tr>

        <tr id="The site-specific function of EGFR’s C-terminal tyrosines is interrogated by phosphotyrosine phosphoproteomics to build a data-driven model that links cell response to signaling nodes and elucidates novel associations in the EGFR signaling network. The epidermal growth factor receptor (EGFR) has been studied extensively because of its critical role in cellular signaling and association with disease. Previous models have elucidated interactions between EGFR and downstream adaptor proteins or showed phenotypes affected by EGFR. However, the link between specific EGFR phosphorylation sites and phenotypic outcomes is still poorly understood. Here, we employed a suite of isogenic cell lines expressing site-specific mutations at each of the EGFR C-terminal phosphorylation sites to interrogate their role in the signaling network and cell biological response to stimulation. Our results demonstrate the resilience of the EGFR network, which was largely similar even in the context of multiple Y-to-F mutations in the EGFR C-terminal tail, while also revealing nodes in the network that have not previously been linked to EGFR signaling. Our data-driven model highlights the signaling network nodes associated with distinct EGF-driven cell responses, including migration, proliferation, and receptor trafficking. Application of this same approach to less-studied RTKs should provide a plethora of novel associations that should lead to an improved understanding of these signaling networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d1bedc5d16450a77fda0fc2c816d3cd1cf3570c" target='_blank'>Predictive data-driven modeling of C-terminal tyrosine function in the EGFR signaling network</a></td>
          <td>
            Jacqueline S. Gerritsen, Joseph S Faraguna, R. Bonavia, F. Furnari, F. White
          </td>
          <td>2023-05-11</td>
          <td>Life Science Alliance</td>
          <td>2</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/679d063f3ab44475bb957757a0b179292e634ea3" target='_blank'>Systems Biology in Translational Oncology: Computational and Experimental Study of EGFR and IGF1R Pathways in NSCLC Cell Lines</a></td>
          <td>
            F. Bianconi, K. Perruccio, V. Ludovini, E. Baldelli, G. Bellezza, A. Flacco, P. Valigi, L. Crinò
          </td>
          <td>2012-09-01</td>
          <td>Annals of Oncology</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="The biophysical properties of ligand binding heavily influence the ability of receptors to specify cell fates. Understanding the rules by which ligand binding kinetics impact cell phenotype is challenging, however, because of the coupled information transfers that occur from receptors to downstream signaling effectors and from effectors to phenotypes. Here, we address that issue by developing an integrated mechanistic and data-driven computational modeling platform to predict cell responses to different ligands for the epidermal growth factor receptor (EGFR). Experimental data for model training and validation were generated using MCF7 human breast cancer cells treated with the high- and low-affinity ligands epidermal growth factor (EGF) and epiregulin (EREG), respectively. The integrated model captures the unintuitive, concentration-dependent abilities of EGF and EREG to drive signals and phenotypes differently, even at similar levels of receptor occupancy. For example, the model correctly predicts the dominance of EREG over EGF in driving a cell differentiation phenotype through AKT signaling at intermediate and saturating ligand concentrations and the ability of EGF and EREG to drive a broadly concentration-sensitive migration phenotype through cooperative ERK and AKT signaling. Parameter sensitivity analysis identifies EGFR endocytosis, which is differentially regulated by EGF and EREG, as one of the most important determinants of the alternative phenotypes driven by different ligands. The integrated model provides a new platform to predict how phenotypes are controlled by the earliest biophysical rate processes in signal transduction and may eventually be leveraged to understand receptor signaling system performance depends on cell context. One-sentence summary Integrated kinetic and data-driven EGFR signaling model identifies the specific signaling mechanisms that dictate cell responses to EGFR activation by different ligands.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccddd2d0cde0d83a0a8cfb56bef3dd0624d5226" target='_blank'>An integrated mechanistic and data-driven computational model predicts cell responses to high- and low-affinity EGFR ligands</a></td>
          <td>
            Paul J. Myers, Sung Hyun Lee, M. Lazzara
          </td>
          <td>2023-06-26</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>22</td>
        </tr>

        <tr id="The receptor tyrosine kinases (RTKs) are key drivers of cancer progression and targets for drug therapy. A major challenge in anti-RTK treatment is the dependence of drug effectiveness on co-expression of multiple RTKs which defines resistance to single drug therapy. Reprogramming of the RTK network leading to alteration in RTK co-expression in response to drug intervention is a dynamic mechanism of acquired resistance to single drug therapy in many cancers. One route to overcome this resistance is combination therapy. We describe the results of a joint in silico, in vitro, and in vivo investigations on the efficacy of trastuzumab, pertuzumab and their combination to target the HER2 receptors. Computational modelling revealed that these two drugs alone and in combination differentially suppressed RTK network activation depending on RTK co-expression. Analyses of mRNA expression in SKOV3 ovarian tumour xenograft showed up-regulation of HER3 following treatment. Considering this in a computational model revealed that HER3 up-regulation reprograms RTK kinetics from HER2 homodimerisation to HER3/HER2 heterodimerisation. The results showed synergy of the trastuzumab and pertuzumab combination treatment of the HER2 overexpressing tumour can be due to an independence of the combination effect on HER3/HER2 composition when it changes due to drug-induced RTK reprogramming.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/328d5cf4ea98961ec78538f4e034e1bd9b7f84fe" target='_blank'>Systems Analysis of Drug-Induced Receptor Tyrosine Kinase Reprogramming Following Targeted Mono- and Combination Anti-Cancer Therapy</a></td>
          <td>
            A. Goltsov, Y. Deeni, Hilal S. Khalil, Tero Soininen, Stylianos Kyriakidis, Huizhong Hu, S. Langdon, D. Harrison, J. Bown
          </td>
          <td>2014-06-01</td>
          <td>Cells</td>
          <td>28</td>
          <td>60</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>