<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              150e1f4a4ec7838cc3188f00cdd15be74d14e417
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Virtual%20cell/" class="md-tabs__link">
        
  
    
  
  Virtual cell

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../RA/" class="md-tabs__link">
        
  
    
  
  RA

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Prostate%20cancer/" class="md-tabs__link">
        
  
    
  
  Prostate cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Virtual%20patients/" class="md-tabs__link">
        
  
    
  
  Virtual patients

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Virtual%20cell/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual cell
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../RA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RA
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Prostate%20cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Prostate cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Virtual%20patients/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual patients
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-03 06:08:47 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Ligand-Specific c-Fos Expression Emerges from the Spatiotemporal Control of ErbB Network Dynamics</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/509f1971d67425c00eb55f435cf67274acb0fe25" target='_blank'>Computational modeling of the dynamics of the MAP kinase cascade activated by surface and internalized EGF receptors</a></td>
          <td>
            B. Schoeberl, Claudia Eichler-Jonsson, E. Gilles, G. Müller
          </td>
          <td>2002-04-01</td>
          <td>Nature Biotechnology</td>
          <td>1026</td>
          <td>48</td>
        </tr>

        <tr id="Simple Summary Temporal signaling dynamics are important for controlling the fate decisions of mammalian cells. In this study, we developed BioMASS, a computational platform for prediction and analysis of signaling dynamics using RNA-sequencing gene expression data. We first constructed a detailed mechanistic model of early transcriptional regulation mediated by ErbB receptor signaling pathway. After training the model parameters against phosphoprotein time-course datasets obtained from breast cancer cell lines, the model successfully predicted signaling activities of another untrained cell line. The result indicates that the parameters of molecular interactions in these different cell types are not particularly unique to the cell type, and the expression levels of the components of the signaling network are sufficient to explain the complex dynamics of the networks. Our method can be further expanded to predict signaling activity from clinical gene expression data for in silico drug screening for personalized medicine. Abstract A current challenge in systems biology is to predict dynamic properties of cell behaviors from public information such as gene expression data. The temporal dynamics of signaling molecules is critical for mammalian cell commitment. We hypothesized that gene expression levels are tightly linked with and quantitatively control the dynamics of signaling networks regardless of the cell type. Based on this idea, we developed a computational method to predict the signaling dynamics from RNA sequencing (RNA-seq) gene expression data. We first constructed an ordinary differential equation model of ErbB receptor → c-Fos induction using a newly developed modeling platform BioMASS. The model was trained with kinetic parameters against multiple breast cancer cell lines using autologous RNA-seq data obtained from the Cancer Cell Line Encyclopedia (CCLE) as the initial values of the model components. After parameter optimization, the model proceeded to prediction in another untrained breast cancer cell line. As a result, the model learned the parameters from other cells and was able to accurately predict the dynamics of the untrained cells using only the gene expression data. Our study suggests that gene expression levels of components within the ErbB network, rather than rate constants, can explain the cell-specific signaling dynamics, therefore playing an important role in regulating cell fate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39b85b8aebc0242791c8944fd3435e2529943c50" target='_blank'>A Computational Framework for Prediction and Analysis of Cancer Signaling Dynamics from RNA Sequencing Data—Application to the ErbB Receptor Signaling Pathway</a></td>
          <td>
            Hiroaki Imoto, Suxiang Zhang, Mariko Okada
          </td>
          <td>2020-10-01</td>
          <td>Cancers</td>
          <td>21</td>
          <td>12</td>
        </tr>

        <tr id="Cell responses are actuated by tightly controlled signal transduction pathways. Although the concept of an integrated signaling network replete with interpathway cross-talk and feedback regulation is broadly appreciated, kinetic data of the type needed to characterize such interactions in conjunction with mathematical models are lacking. In mammalian cells, the Ras/ERK pathway controls cell proliferation and other responses stimulated by growth factors, and several cross-talk and feedback mechanisms affecting its activation have been identified. In this work, we take a systematic approach to parse the magnitudes of multiple regulatory mechanisms that attenuate ERK activation through canonical (Ras-dependent) and non-canonical (PI3K-dependent) pathways. In addition to regulation of receptor and ligand levels, we consider three layers of ERK-dependent feedback: desensitization of Ras activation, negative regulation of MEK kinase (e.g. Raf) activities, and up-regulation of dual-specificity ERK phosphatases. Our results establish the second of these as the dominant mode of ERK self-regulation in mouse fibroblasts. We further demonstrate that kinetic models of signaling networks, trained on a sufficient diversity of quantitative data, can be reasonably comprehensive, accurate, and predictive in the dynamical sense.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405be6d692d4836d3b3efde9333fdf1789ff05c7" target='_blank'>Systematic Quantification of Negative Feedback Mechanisms in the Extracellular Signal-regulated Kinase (ERK) Signaling Network*</a></td>
          <td>
            M. Cirit, Chun-Chao Wang, J. Haugh
          </td>
          <td>2010-09-16</td>
          <td>The Journal of Biological Chemistry</td>
          <td>91</td>
          <td>33</td>
        </tr>

        <tr id="The Ras/ERK pathway drives cell proliferation and other oncogenic behaviors, and quantifying its activity in situ is of high interest in cancer diagnosis and therapy. Pathway activation is often assayed by measuring phosphorylated ERK. However, this form of measurement overlooks dynamic aspects of signaling that can only be observed over time. In this study, we combine a live, single-cell ERK biosensor approach with multiplexed immunofluorescence staining of downstream target proteins to ask how well immunostaining captures the dynamic history of ERK activity. Combining linear regression, machine learning, and differential equation models, we develop an interpretive framework for immunostains, in which Fra-1 and pRb levels imply long term activation of ERK signaling, while Egr-1 and c-Myc indicate recent activation. We show that this framework can distinguish different classes of ERK dynamics within a heterogeneous population, providing a tool for annotating ERK dynamics within fixed tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b822e494292e7a44173abfef978a7e61987d4a1c" target='_blank'>Deciphering the History of ERK Activity from Fixed-Cell Immunofluorescence Measurements</a></td>
          <td>
            A. Ram, M. Pargett, Yongin Choi, D. Murphy, Markhus Cabel, N. Kosaisawe, G. Quon, John G. Albeck
          </td>
          <td>2024-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Embryonic development and adult tissue homeostasis are controlled through activation of intracellular signal transduction pathways by extracellular growth factors. In the past, signal transduction has largely been regarded as a linear process. However, more recent data from large‐scale and high‐throughput experiments indicate that there is extensive cross‐talk between individual signaling cascades leading to the notion of a signaling network. The behavior of such complex networks cannot be predicted by simple intuitive approaches but requires sophisticated models and computational simulations. The purpose of such models is to generate experimentally testable hypotheses and to find explanations for unexpected experimental results. Here, we discuss the need for, and the future impact of, mathematical models for exploring signal transduction in different biological contexts such as for example development. BioEssays 30:1110–1125, 2008. © 2008 Wiley Periodicals, Inc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8545f3dc8505b17afcd909dbddc0e64abbc53ce2" target='_blank'>Network modeling of signal transduction: establishing the global view</a></td>
          <td>
            H. Kestler, Christian Wawra, Barbara Kracher, M. Kühl
          </td>
          <td>2008-11-01</td>
          <td>BioEssays</td>
          <td>94</td>
          <td>56</td>
        </tr>

        <tr id="Extra-cellular signal regulated kinase (ERK) signal transduction is an important signaling pathway, involved in a variety of processes in cell and closely related to the occurrences of diverse tumors. This raises the question how these diverse functions are specified and coordinated. In this work, we present an integrated computational model of ERK signal transduction pathway, investigating the regulatory mechanism of ERK signal pathway by variation of the concentration of epidermal growth factor receptor (EGFR) as well as the binding affinity of adaptor proteins. The results show that EGFR and distinct adaptor proteins engender different activation patterns of ERK, which will bring different cellular responses. It could be helpful to understand the mechanism of ERK signal pathway and the relationship between ERK signal network and tumor development or therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8834b7200d12495f21190553b8a4f1751ce0a835" target='_blank'>Dynamic Model and Regulatory Mechanism of Integrated ERK Signal Pathway Activated by Epidermal Growth Factor</a></td>
          <td>
            H. Fang, Yan Li, Yonghua Wang, Shuwei Zhang
          </td>
          <td>2008-10-18</td>
          <td>2008 Fourth International Conference on Natural Computation</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c47eca070ef4fc936f0fdb8bd32925e209619e" target='_blank'>Self-consistent signal transduction analysis for modeling context-specific signaling cascades and perturbations</a></td>
          <td>
            John Cole
          </td>
          <td>2024-07-19</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cellular proccess that are crucial for health or disease are orchestrated by the expression of multiple genes in a network of differential complex signaling pathways, consisting of many physical and functional interactions.Unlike the stable genome structure, the highly dynamic interactome represents a major challenge for quantitative mapping of protein–protein interactions (PPIs). In this article, I will discuss the latest systems science advances by using breakthrough technologies to gain an understanding of the whole picture of the spatiotemporal signaling network as a response to dynamic individual genome–lifestyle interactions and translating it to the clinic.Biological signals control cells’ functions and their organization into tissues, organs and a whole organism’s homeostasis. A variety of extracellular and intracellu-lar signaling molecules include hormones, growth factors, cytokines, chemokines, neurotransmitters, extracellular matrix components, receptors, protein kinases, protein phosphatases and DNA-binding proteins in the nucleus. A signal is trans-mitted or amplified from the cell mem-brane to the nucleus through a dynamic complex interactions network">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2702c101690829e49078cf33e2ee81dc4f49e616" target='_blank'>Differential signaling transduction networks for clinical robustness</a></td>
          <td>
            D. Roukos
          </td>
          <td>2012-04-01</td>
          <td>Expert Review of Proteomics</td>
          <td>12</td>
          <td>62</td>
        </tr>

        <tr id="Cancer is a multifactorial disease. Aberrant functioning of the underlying complex signaling network that orchestrates cellular response to external or internal cues governs incidence, progression, and recurrence of cancer. Detailed understanding of cancer’s etiology can offer useful insights into arriving at novel therapeutic and disease management strategies. Such an understanding for most cancers is currently limited due to unavailability of a predictive large-scale, integrated signaling model accounting for all tumor orchestrating factors. We suggest that the potential of Boolean dynamic (BD) modeling approaches, though qualitative, can be harnessed for developing holistic models capturing multi-scale, multi-cellular signaling processes involved in cancer incidence and progression. We believe that constraining such an integrated BD model with variety of omics data at different scales from laboratory and clinical settings could offer deeper insights into causal mechanisms governing the disease leading to better prognosis. We review the recent literature employing different BD modeling strategies to model variety of cancer signaling programs leading to identification of cancer-specific prognostic markers such as SMAD proteins, which may also serve as early predictors of tumor cells hijacking the epithelial-mesenchymal plasticity program. In silico simulations of BD models of different cancer signaling networks combined with attractor landscape analysis and validated with experimental data predicted the nature of short-and long-term response of standard targeted therapeutic agents such as Nutlin-3, a small molecule inhibitor for p53-MDM2 interaction. BD simulations also offered a mechanistic view of emerging resistance to drugs such as Trastuzumab for">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be0d7c2f350f03724a68a5b4624bf8d8da62bbaf" target='_blank'>Boolean dynamic modeling of cancer signaling networks: Prognosis, progression, and therapeutics</a></td>
          <td>
            S. Sherekar, G. Viswanathan
          </td>
          <td>2021-05-01</td>
          <td>Computational and Systems Oncology</td>
          <td>13</td>
          <td>12</td>
        </tr>

        <tr id="The biophysical properties of ligand binding heavily influence the ability of receptors to specify cell fates. Understanding the rules by which ligand binding kinetics impact cell phenotype is challenging, however, because of the coupled information transfers that occur from receptors to downstream signaling effectors and from effectors to phenotypes. Here, we address that issue by developing an integrated mechanistic and data-driven computational modeling platform to predict cell responses to different ligands for the epidermal growth factor receptor (EGFR). Experimental data for model training and validation were generated using MCF7 human breast cancer cells treated with the high- and low-affinity ligands epidermal growth factor (EGF) and epiregulin (EREG), respectively. The integrated model captures the unintuitive, concentration-dependent abilities of EGF and EREG to drive signals and phenotypes differently, even at similar levels of receptor occupancy. For example, the model correctly predicts the dominance of EREG over EGF in driving a cell differentiation phenotype through AKT signaling at intermediate and saturating ligand concentrations and the ability of EGF and EREG to drive a broadly concentration-sensitive migration phenotype through cooperative ERK and AKT signaling. Parameter sensitivity analysis identifies EGFR endocytosis, which is differentially regulated by EGF and EREG, as one of the most important determinants of the alternative phenotypes driven by different ligands. The integrated model provides a new platform to predict how phenotypes are controlled by the earliest biophysical rate processes in signal transduction and may eventually be leveraged to understand receptor signaling system performance depends on cell context. One-sentence summary Integrated kinetic and data-driven EGFR signaling model identifies the specific signaling mechanisms that dictate cell responses to EGFR activation by different ligands.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccddd2d0cde0d83a0a8cfb56bef3dd0624d5226" target='_blank'>An integrated mechanistic and data-driven computational model predicts cell responses to high- and low-affinity EGFR ligands</a></td>
          <td>
            Paul J. Myers, Sung Hyun Lee, M. Lazzara
          </td>
          <td>2023-06-26</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>22</td>
        </tr>

        <tr id="Cell responses are actuated by tightly controlled signal transduction pathways. Although the concept of an integrated signaling network replete with interpathway cross-talk and feedback regulation is broadly appreciated, kinetic data of the type needed to characterize such interactions in conjunction with mathematical models are lacking. In mammalian cells, the Ras/ERK pathway controls cell proliferation and other responses stimulated by growth factors, and several cross-talk and feedback mechanisms affecting its activation have been identified. In this work, we take a systematic approach to parse the magnitudes of multiple regulatory mechanisms that attenuate ERK activation through canonical (Ras-dependent) and non-canonical (PI3K-dependent) pathways. In addition to regulation of receptor and ligand levels, we consider three layers of ERK-dependent feedback: desensitization of Ras activation, negative regulation of MEK kinase (e.g. Raf) activities, and up-regulation of dual-specificity ERK phosphatases. Our results establish the second of these as the dominant mode of ERK self-regulation in mouse fibroblasts. We further demonstrate that kinetic models of signaling networks, trained on a sufficient diversity of quantitative data, can be reasonably comprehensive, accurate, and predictive in the dynamical sense.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405be6d692d4836d3b3efde9333fdf1789ff05c7" target='_blank'>Systematic Quantification of Negative Feedback Mechanisms in the Extracellular Signal-regulated Kinase (ERK) Signaling Network*</a></td>
          <td>
            M. Cirit, Chun-Chao Wang, J. Haugh
          </td>
          <td>2010-09-16</td>
          <td>The Journal of Biological Chemistry</td>
          <td>91</td>
          <td>33</td>
        </tr>

        <tr id="Cells integrate many inputs through complex networks of interacting signaling pathways. Systems approaches as well as computer-aided reductionist approaches attempt to "untangle the wires" and gain an intimate understanding of cells. But "understanding" any system is just the way that the human mind gains the ability to predict behavior. Computer simulations are an alternative way to achieve this goal--quite possibly the only way for complex systems. We have new tools to probe large sets of unknown interactions, and we have amassed enough detailed information to quantitatively describe many functional modules. Cell physiology has passed the threshold: the time to begin modeling is now.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c21a264fd9a79b61a366d39499498f73ac5931" target='_blank'>Intracellular signaling: spatial and temporal control.</a></td>
          <td>
            I. Moraru, L. Loew
          </td>
          <td>2005-06-01</td>
          <td>Physiology</td>
          <td>34</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6a045551eda8ac4bf3c6dd072aafd66f5f9d95" target='_blank'>Simplistic pathways or complex networks?</a></td>
          <td>
            C. Jørgensen, R. Linding
          </td>
          <td>2010-02-01</td>
          <td>Current opinion in genetics & development</td>
          <td>40</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ac9c640391085d36e30bce8cd2eff1affe6082c" target='_blank'>CDK2 activity crosstalk on the ERK kinase translocation reporter can be resolved computationally.</a></td>
          <td>
            Timothy E. Hoffman, Chengzhe Tian, Varuna Nangia, Chen Yang, S. Regot, Luca Gerosa, Sabrina L Spencer
          </td>
          <td>2025-01-01</td>
          <td>Cell systems</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Department of Biophysics and Cell Biology, Research Center for Molecular Medicine, University of Debrecen, Debrecen, Hungarya; German Cancer Research Center (DKFZ), Biophysics of Macromolecules, Heidelberg, Germanyb; HAS-UD Vascular Biology, Thrombosis and Hemostasis Research Group, Hungarian Academy of Sciences, Debrecen, Hungaryc; Division of Clinical Laboratory Science, Department of Laboratory Medicine, Faculty of Medicine, University of Debrecen, Debrecen, Hungaryd; Sanford Burnham Prebys Medical Discovery Institute at Lake Nona, Orlando, Florida, USAe">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c7e8c85d599e1833d8069eabbbf45566bb41dd6" target='_blank'>Erratum for Szalóki et al., “Evidence for Homodimerization of the c-Fos Transcription Factor in Live Cells Revealed by Fluorescence Microscopy and Computer Modeling”</a></td>
          <td>
            N. Szalóki, J. Krieger, I. Komáromi, K. Toth, G. Vámosi
          </td>
          <td>2018-07-16</td>
          <td>Molecular and Cellular Biology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Detecting relative rather than absolute changes in extracellular signals enables cells to make decisions in constantly fluctuating environments. It is currently not well understood how mammalian signaling networks store the memories of past stimuli and subsequently use them to compute relative signals, that is perform fold change detection. Using the growth factor-activated PI3K-Akt signaling pathway, we develop here computational and analytical models, and experimentally validate a novel non-transcriptional mechanism of relative sensing in mammalian cells. This mechanism relies on a new form of cellular memory, where cells effectively encode past stimulation levels in the abundance of cognate receptors on the cell surface. The surface receptor abundance is regulated by background signal-dependent receptor endocytosis and down-regulation. We show the robustness and specificity of relative sensing for two physiologically important ligands, epidermal growth factor (EGF) and hepatocyte growth factor (HGF), and across wide ranges of background stimuli. Our results suggest that similar mechanisms of cell memory and fold change detection may be important in diverse signaling cascades and multiple biological contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f33ec71f173b318ab19494c7750c3eece01916c" target='_blank'>Receptor-based mechanism of relative sensing and cell memory in mammalian signaling networks</a></td>
          <td>
            E. Lyashenko, M. Niepel, P. Dixit, S. Lim, P. Sorger, Dennis Vitkup
          </td>
          <td>2020-01-21</td>
          <td>eLife</td>
          <td>17</td>
          <td>108</td>
        </tr>

        <tr id="The epidermal growth factor receptor (EGFR) activated extracellular-signal regulated kinase (ERK) pathway is a central cell signalling pathway that mediates many biological responses including cell proliferation, transformation, survival and motility. Deregulation of the pathway either through mutation of components or overexpression of EGFRs is associated with several forms of cancer. Under normal conditions, EGF stimulates a rapid but transient activation of ERK as the signal is rapidly shutdown, whereas under cancerous conditions, the ERK signal cannot be shutdown and is sustained. Computational modelling techniques have been used to investigate the signalling dynamics of the EGFR/ERK pathway, focusing on identifying the key processes involved in signal termination and what role the ERK to son of sevenless (SOS) negative feedback loop plays in generating a transient response. This model predicts that this negative feedback loop is not needed to achieve a transient activation of ERK as the process of receptor degradation alone is enough to terminate the signal. Importantly, the behaviour and predictions of this model are verified with laboratory data, as is essential for modern systems biology approaches. Further analysis showed that the feedback loop and receptor degradation were both redundant processes, as each could compensate for the absence of the other. This led to the prediction that in the case of a receptor which is not degraded, such as the insulin receptor, the negative feedback loop to SOS will actually be essential for a transient response to be achieved. Overall, the results shed new light on the role of negative feedback in EGF receptor signalling and suggest that different receptors are dependent on different features within the ERK pathway when relaying their signals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35e7094d70c15e7e0196f4a688c075edd2e156e0" target='_blank'>Computational modelling reveals feedback redundancy within the epidermal growth factor receptor/extracellular-signal regulated kinase signalling pathway.</a></td>
          <td>
            Richard J. Orton, Oliver E. Sturm, Amelie Gormand, Walter Kolch, David R. Gilbert
          </td>
          <td>2008-08-01</td>
          <td>IET systems biology</td>
          <td>21</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfbf73b4d67dd34c855cf8e556071b7e092bb704" target='_blank'>Input–output behavior of ErbB signaling pathways as revealed by a mass action model trained against dynamic data</a></td>
          <td>
            William W. Chen, B. Schoeberl, P. Jasper, M. Niepel, U. Nielsen, D. Lauffenburger, P. Sorger
          </td>
          <td>2009-01-20</td>
          <td>Molecular Systems Biology</td>
          <td>430</td>
          <td>119</td>
        </tr>

        <tr id="Over the past decade we have seen a shift in cancer therapy from broadly cytotoxic drugs to molecular therapies targeting “driver” mutations. Although targeted therapy has seen great success for some cancers (e.g. imatinib for leukemia), it has struggled with poor efficacy in treating other cancers that can sometimes possess multiple “driver” mutations. This highlights the complex, and at times non-intuitive, interplay between multiple players in a signaling cascade, which can be highly dependent on the biological context – that is, gene expression levels and mutational architecture – of a tumor or cell line. A quantitative, mechanistic, biologically-tailored understanding of how these signaling dynamics drive proliferation and death could improve precision pharmacology approaches to treat cancer. Here, we constructed the first highly detailed, large-scale ordinary differential equation (ODE) mechanistic mathematical model depicting the most commonly mutated cancer signaling pathways across human cancers, as indicated by a pan-cancer analysis by The Cancer Genome Atlas (TCGA). The model includes the RTK/Ras/MAPK, PI3K/AKT/mTOR, CDK/RB cell cycle, p53/MDM2 DNA damage response, and BCL/Caspases apoptosis pathways. The adjustable parameters of the model can be informed by measurements from patients or cell lines, including copy number alterations, mutations, and gene expression levels. This single-cell model links stochastic gene expression processes to quantitative signaling dynamics, and once tailored to a biological context can be used to simulate the effect of various anti-cancer therapies on cell fate behavior such as proliferation and death for a population of cells. The first instance of the model integrated genomic, transcriptomic, and proteomic data from the MCF10A cell line, a non-transformed cell line with predictable phenotypic behaviors. We trained the model using western blot and flow cytometry experiments to refine various biochemical parameters and phenotypic outcomes. Many fundamental questions in signal transduction arose during this process, such as how EGF and insulin synergize to drive S-phase entry or how a specific biological context confers sensitivity or resistance to inhibitors of the ERK and AKT pathways. Simultaneously, we are tailoring the model to patient-derived genetic information from primary glioblastoma tumors and screening brain-penetrable compounds in a patient-specific manner. In conclusion, a quantitative, mechanistic, biologically-tailored mathematical model depicting the major cancer pathways allows us to probe the mechanisms that underlie how signaling dynamics drive proliferation and death in response to various perturbations, and gain insight into their dependence on the biological context of cell lines and patient tumors. Note: This abstract was not presented at the meeting. Citation Format: Mehdi Bouhaddou, Anne Marie Barrette, Rick J. Koch, Marc R. Birtwistle. Predicting stochastic proliferation and death in response to drugs with mechanistic models tailored to genomic, transcriptomic, and proteomic data [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2017; 2017 Apr 1-5; Washington, DC. Philadelphia (PA): AACR; Cancer Res 2017;77(13 Suppl):Abstract nr 1568. doi:10.1158/1538-7445.AM2017-1568">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b9460f53b1393fa834eeb46b9c94177968a96a" target='_blank'>Abstract 1568: Predicting stochastic proliferation and death in response to drugs with mechanistic models tailored to genomic, transcriptomic, and proteomic data</a></td>
          <td>
            M. Bouhaddou, A. Barrette, R. J. Koch, M. Birtwistle
          </td>
          <td>2017-07-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The site-specific function of EGFR’s C-terminal tyrosines is interrogated by phosphotyrosine phosphoproteomics to build a data-driven model that links cell response to signaling nodes and elucidates novel associations in the EGFR signaling network. The epidermal growth factor receptor (EGFR) has been studied extensively because of its critical role in cellular signaling and association with disease. Previous models have elucidated interactions between EGFR and downstream adaptor proteins or showed phenotypes affected by EGFR. However, the link between specific EGFR phosphorylation sites and phenotypic outcomes is still poorly understood. Here, we employed a suite of isogenic cell lines expressing site-specific mutations at each of the EGFR C-terminal phosphorylation sites to interrogate their role in the signaling network and cell biological response to stimulation. Our results demonstrate the resilience of the EGFR network, which was largely similar even in the context of multiple Y-to-F mutations in the EGFR C-terminal tail, while also revealing nodes in the network that have not previously been linked to EGFR signaling. Our data-driven model highlights the signaling network nodes associated with distinct EGF-driven cell responses, including migration, proliferation, and receptor trafficking. Application of this same approach to less-studied RTKs should provide a plethora of novel associations that should lead to an improved understanding of these signaling networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d1bedc5d16450a77fda0fc2c816d3cd1cf3570c" target='_blank'>Predictive data-driven modeling of C-terminal tyrosine function in the EGFR signaling network</a></td>
          <td>
            Jacqueline S. Gerritsen, Joseph S Faraguna, R. Bonavia, F. Furnari, F. White
          </td>
          <td>2023-05-11</td>
          <td>Life Science Alliance</td>
          <td>2</td>
          <td>66</td>
        </tr>

        <tr id="The EGF-induced MAP kinase cascade is one of the most important and best characterized networks in intracellular signalling. It has a vital role in the development and maturation of living organisms. However, when deregulated, it is involved in the onset of a number of diseases. Based on a computational model describing a “surface” and an “internalized” parallel route, we use systems biology techniques to characterize aspects of the network’s functional organization. We examine the re-organization of protein groups from low to high external stimulation, define functional groups of proteins within the network, determine the parameter best encoding for input intensity and predict the effect of protein removal to the system’s output response. Extensive functional re-organization of proteins is observed in the lower end of stimulus concentrations. As we move to higher concentrations the variability is less pronounced. 6 functional groups have emerged from a consensus clustering approach, reflecting different dynamical aspects of the network. Mutual information investigation revealed that the maximum activation rate of the two output proteins best encodes for stimulus intensity. Removal of each protein of the network resulted in a range of graded effects, from complete silencing to intense activation. Our results provide a new “vista” of the EGF-induced MAP kinase cascade, from the perspective of complex self-organizing systems. Functional grouping of the proteins reveals an organizational scheme contrasting the current understanding of modular topology. The six identified groups may provide the means to experimentally follow the dynamics of this complex network. Also, the vulnerability analysis approach may be used for the development of novel therapeutic targets in the context of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccd7041af39a1cc5e66884c8aacd4b45fc5c205d" target='_blank'>Functional Aspects of the EGF-Induced MAP Kinase Cascade: A Complex Self-Organizing System Approach</a></td>
          <td>
            E. Kosmidis, V. Moschou, Georgios Ziogas, I. Boukovinas, M. Albani, N. Laskaris
          </td>
          <td>2014-11-05</td>
          <td>PLoS ONE</td>
          <td>1</td>
          <td>38</td>
        </tr>

        <tr id="Detecting relative rather than absolute changes in external signals enables cells to make decisions in fluctuating environments and diverse biological contexts. However, how mammalian signaling networks store the memories of past stimuli and use them to compute relative signals is not well understood. Using the growth factor-activated PI3K-Akt signaling pathway, we develop computational and analytical models, and experimentally validate a novel mechanism of relative sensing in mammalian cells. This non-transcriptional mechanism relies on a new form of cellular memory, where cells effectively encode past stimulation levels in the abundance of cognate receptors on the cell surface. We show the robustness and specificity of the relative sensing for two physiologically important ligands, epidermal growth factor (EGF) and hepatocyte growth factor (HGF), and across wide ranges of background stimuli. The described memory and sensing mechanism could play a role in multiple other sensory cascades where stimulation leads to a proportional reduction in the abundance of cell surface receptors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/512be8d672f1e5beaa81e95ea265f3755a48516c" target='_blank'>Receptor-Based Mechanism of Cell Memory and Relative Sensing in Mammalian Signaling Networks</a></td>
          <td>
            E. Lyashenko, M. Niepel, P. Dixit, S. Lim, P. Sorger, Dennis Vitkup
          </td>
          <td>2017-07-02</td>
          <td>bioRxiv</td>
          <td>7</td>
          <td>108</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec321c10f940fc23c2ce5e932e9b885bd8d56920" target='_blank'>11 From benchside to byteside: what we can learn from computational modelling</a></td>
          <td>
            W. Kolch, M. Birtwistle, B. Kholodenko, J. Grindlay, Oliver E. Sturm, R. Orton, David R. Gilbert, A. Kriegsheim
          </td>
          <td>2010-06-01</td>
          <td>MAG</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Computational modeling of protein localization dynamics yields new information about Smad signaling. The molecular mechanisms that transmit information from cell surface receptors to the nucleus are exceedingly complex; thus, much effort has been expended in developing computational models to understand these processes. A recent study on modeling the nuclear-cytoplasmic shuttling of Smad2-Smad4 complexes in response to transforming growth factor–β (TGF-β) receptor activation has provided substantial insight into how this signaling network translates the degree of TGF-β receptor activation (input) into the amount of nuclear Smad2-Smad4 complexes (output). The study addressed this question by combining a simple, mechanistic model with targeted experiments, an approach that proved particularly powerful for exploring the fundamental properties of a complex signaling network. The mathematical model revealed that Smad nuclear-cytoplasmic dynamics enables a proportional but time-delayed coupling between the input and the output. As a result, the output can faithfully track gradual changes in the input while the rapid input fluctuations that constitute signaling noise are dampened out.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261e159904d02ed3d3fffa418a2ef59b1e043adc" target='_blank'>Smad Signaling Dynamics: Insights from a Parsimonious Model</a></td>
          <td>
            H. Shankaran, H. Wiley
          </td>
          <td>2008-09-09</td>
          <td>Science Signaling</td>
          <td>9</td>
          <td>44</td>
        </tr>

        <tr id="Building mechanistic models of kinase-driven signaling pathways requires quantitative measurements of protein phosphorylation across physiologically relevant conditions, but this is rarely done because of the insensitivity of traditional technologies. By using a multiplexed deep phosphoproteome profiling workflow, we were able to generate a deep phosphoproteomics dataset of the EGFR-MAPK pathway in non-transformed MCF10A cells across physiological ligand concentrations with a time resolution of <12 min and in the presence and absence of multiple kinase inhibitors. An improved phosphosite mapping technique allowed us to reliably identify >46,000 phosphorylation sites on >6600 proteins, of which >4500 sites from 2110 proteins displayed a >2-fold increase in phosphorylation in response to EGF. This data was then placed into a cellular context by linking it to 15 previously published protein databases. We found that our results were consistent with much, but not all previously reported data regarding the activation and negative feedback phosphorylation of core EGFR-ERK pathway proteins. We also found that EGFR signaling is biphasic with substrates downstream of RAS/MAPK activation showing a maximum response at <3ng/ml EGF while direct substrates, such as HGS and STAT5B, showing no saturation. We found that RAS activation is mediated by at least 3 parallel pathways, two of which depend on PTPN11. There appears to be an approximately 4-minute delay in pathway activation at the step between RAS and RAF, but subsequent pathway phosphorylation was extremely rapid. Approximately 80 proteins showed a >2-fold increase in phosphorylation across all experiments and these proteins had a significantly higher median number of phosphorylation sites (~18) relative to total cellular phosphoproteins (~4). Over 60% of EGF-stimulated phosphoproteins were downstream of MAPK and included mediators of cellular processes such as gene transcription, transport, signal transduction and cytoskeletal arrangement. Their phosphorylation was either linear with respect to MAPK activation or biphasic, corresponding to the biphasic signaling seen at the level of the EGFR. This deep, integrated phosphoproteomics data resource should be useful in building mechanistic models of EGFR and MAPK signaling and for understanding how downstream responses are regulated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c1f72033b1ae81f91de5f869e07e891d11153d8" target='_blank'>A Phosphoproteomics Data Resource for Systems-level Modeling of Kinase Signaling Networks</a></td>
          <td>
            Song Feng, J. Sanford, Thomas Weber, Chelsea Hutchinson-Bunch, Panshak P. Dakup, V. Paurus, Kwame Attah, H. Sauro, W. Qian, H. Wiley
          </td>
          <td>2023-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Members of the ErbB receptor family are associated with several cancers and appear to be providing useful targets for pharmacological therapeutics for tumours of the lung and breast. Further improvements of these therapies may be guided by a quantitative, dynamic integrative systems understanding of the complexities of ErbB dimerisation, trafficking and activation, for it is these complexities that render difficult intuiting how perturbations such as drug intervention will affect ErbB signalling activities. Towards this goal, we have developed a computational model implementing commonly accepted principles governing ErbB receptor interaction, trafficking, phosphorylation and dephosphorylation. Using this model, we are able to investigate several hypotheses regarding the compartmental localisation of dephosphorylation. Model results applied to experimental data on ErbB 1, ErbB2 and ErbB3 phosphorylation in H292 human lung carcinoma cells support a hypothesis that key dephosphorylation activity for these receptors occurs largely in an intracellular, endosomal compartment rather than at the cell surface plasma membrane. Thus, the endocytic trafficking-related compartmentalisation of dephosphorylation may define a critical aspect of the ErbB signalling response to ligand.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eec0e3aca46d055f273c84125cd34b0aa75d8cd6" target='_blank'>Computational modelling of ErbB family phosphorylation dynamics in response to transforming growth factor alpha and heregulin indicates spatial compartmentation of phosphatase activity.</a></td>
          <td>
            B. Hendriks, J. Cook, J. Burke, J. Beusmans, D. Lauffenburger, D. D. Graaf
          </td>
          <td>2006-01-30</td>
          <td>Systems biology</td>
          <td>48</td>
          <td>119</td>
        </tr>

        <tr id="The Hippo pathway is an evolutionarily conserved signaling network that integrates diverse cues to regulate cell fate and organ homeostasis. The central downstream pathway protein is the transcriptional co-activator Yes-associated protein (YAP). Although capable of inducing gene transcription, YAP cannot bind DNA directly. Instead, it mediates transcriptional activity through interaction with distinct DNA-binding transcriptional factors (TFs), including TEAD, SMAD, and p73, to form active and functionally opposing transcriptional complexes. Growing evidence in mammals demonstrates that YAP has a dual role and can either promote cell proliferation or apoptosis, which underpin its ability to function as both an oncogene or a tumour suppressor depending on the specific context. However, the mechanisms by which YAP coordinates its distinct transcriptional complexes and mediates context-dependent function remain poorly defined. This is in part due to the lack of systems-level studies that can decrypt the complexities of upstream signalling pathways and their crosstalk, which together dictate the transcriptional regulation at the YAP level. Here, we undertake an integrative systems-based approach combining computational network modelling and experimental studies to interrogate the dynamic formation of and transition between the YAP-SMAD and YAP-p73 transcriptional complexes, which control proliferative and apoptotic gene expression, respectively. We developed a new experimentally-validated mathematical model of the TGF-β/Hippo signalling crosstalk and used this model to elucidate dynamic network behaviour. Our integrative studies uncovered previously unknown molecular switches that control the YAP-SMAD/p73 complexes in an on/off, switch-like manner. RASSF1A and ITCH were identified as major regulators of the switches, whereby a graded increase in ITCH expression can trigger YAP to abruptly switch from binding p73 to SMAD, swiftly promoting proliferative gene expression. Further, adjusting the model to reflect cell type-specific protein expression profiles using both in-house and publicly available experimental data enabled us to study the YAP switches under diverse and varied cellular contexts. Overall, our studies provide a new quantitative and systems-level understanding of the dynamic regulation of functionally opposing YAP transcriptional complexes in mammalian cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19c5dda2d4e195cda51b4f133731444ae6b27e53" target='_blank'>Systems modelling of TGF-β/Hippo signalling crosstalk uncovers molecular switches that coordinate YAP transcriptional complexes (submitted to iScience)</a></td>
          <td>
            Milad Ghomlaghi, Mandy Theocharous, Sung-young Shin, Eric O' Neill, Leann Nguyen
          </td>
          <td>2022-04-08</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Extra-cellular signal regulated kinase (ERK) signal network plays an important role in regulating the fundamental cellular functions such as cell proliferation, survival, differentiation and motility. Development and analysis of mathematical model can help us acquire a deep understanding of the complex behavior of ERK signal transduction network. This paper presents a computational model that offers an integrated quantitative and dynamic simulation of ERK signal transduction network, activated by epidermal growth factor (EGF). The mathematic model contains the activation kinetics of the pathway, a multitude of feedback loops and connection of scaffold proteins. The model provides insight into signal-response relationships between the binding of EGF to its receptor at the cell surface and activation of downstream proteins in the signaling cascade. The different influence of positive and negative feedback loops of the ERK signal transduction pathway were mainly analyzed, indicating that feedback loops were the main impacting factor to the oscillation of ERK signal transduction pathway. The predictions of this oscillation of ERK activation agree well with the literature. It can lead to signal waves of the downstream substrates and induce corresponding biological behaviors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7bf832c4d17a797f80bffbcb87cce1c1078a91f" target='_blank'>Dynamic Analysis of Feedback Loops in Extra-Cellular Signal Regulated Kinase Signal Network</a></td>
          <td>
            Yong Han, Yan Li, Yonghua Wang, H. Fang, Shuwei Zhang
          </td>
          <td>2009-03-31</td>
          <td>2009 WRI World Congress on Computer Science and Information Engineering</td>
          <td>3</td>
          <td>34</td>
        </tr>

        <tr id="Motivation Signal transduction networks regulate a multitude of essential biological processes and are frequently aberrated in diseases such as cancer. Developing a mechanistic understanding of such networks is essential to understand disease or cell population specific signaling and to design effective treatment strategies. Typically, such networks are computationally reconstructed based on systematic perturbation experiments, followed by quantification of signaling protein activity. Recent technological advances now allow for the quantification of the activity of many (signaling) proteins simultaneously in single cells. This makes it feasible to reconstruct signaling networks from single cell data. Results Here we introduce single cell Comparative Network Reconstruction (scCNR) to derive signal transduction networks by exploiting the heterogeneity of single cell (phospho)protein measurements. scCNR treats stochastic variation in total protein abundances as natural perturbation experiments, whose effects propagate through the network. scCNR reconstructs cell population specific networks of the same underlying topology for cells from diverse populations. We extensively validated scCNR on simulated single cell data, and we applied it to a dataset of EGFR-inhibitor treated keratinocytes to recover signaling differences downstream of EGFR and in protein interactions associated with proliferation. scCNR will help to unravel the mechanistic signaling differences between cell populations by making use of single-cell data, and will subsequently guide the development of well-informed treatment strategies. Availability and implementation scCNR is available as a python module at https://github.com/ibivu/scmra. Additionally, code to reproduce all figures is available at https://github.com/tstohn/scmra_analysis. Supplementary information Supplementary information and data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/516c13660aa9eadff6cefbce4ac0292281fdc217" target='_blank'>Reconstructing and comparing signal transduction networks from single cell protein quantification data</a></td>
          <td>
            Tim Stohn, Roderick van Eijl, Klaas W. Mulder, L. Wessels, Evert Bosdriesz
          </td>
          <td>2024-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6319b79882fa8bd07267c8cafc082740cd9779b6" target='_blank'>Computational modeling of the EGFR network elucidates control mechanisms regulating signal dynamics</a></td>
          <td>
            Dennis Wang, L. Cardelli, Andrew Phillips, Nir Piterman, J. Fisher
          </td>
          <td>2009-12-22</td>
          <td>BMC Systems Biology</td>
          <td>40</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcdad5f1bff9567dba95a080790737079a3c8dd2" target='_blank'>Prediction and validation of the distinct dynamics of transient and sustained ERK activation</a></td>
          <td>
            S. Sasagawa, Yu-ichi Ozaki, Kazuhiro A. Fujita, Shinya Kuroda
          </td>
          <td>2005-03-27</td>
          <td>Nature Cell Biology</td>
          <td>455</td>
          <td>27</td>
        </tr>

        <tr id="To understand how integration of multiple data types can help decipher cellular responses at the systems level, we analyzed the mitogenic response of human mammary epithelial cells to epidermal growth factor (EGF) using whole genome microarrays, mass spectrometry-based proteomics and large-scale western blots with over 1000 antibodies. A time course analysis revealed significant differences in the expression of 3172 genes and 596 proteins, including protein phosphorylation changes measured by western blot. Integration of these disparate data types showed that each contributed qualitatively different components to the observed cell response to EGF and that varying degrees of concordance in gene expression and protein abundance measurements could be linked to specific biological processes. Networks inferred from individual data types were relatively limited, whereas networks derived from the integrated data recapitulated the known major cellular responses to EGF and exhibited more highly connected signaling nodes than networks derived from any individual dataset. While cell cycle regulatory pathways were altered as anticipated, we found the most robust response to mitogenic concentrations of EGF was induction of matrix metalloprotease cascades, highlighting the importance of the EGFR system as a regulator of the extracellular environment. These results demonstrate the value of integrating multiple levels of biological information to more accurately reconstruct networks of cellular response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f4d23a95c29e3c93f9d7bdac57936862d5438a5" target='_blank'>Network Analysis of Epidermal Growth Factor Signaling Using Integrated Genomic, Proteomic and Phosphorylation Data</a></td>
          <td>
            K. Waters, Tao Liu, Ryan D. Quesenberry, A. Willse, Somnath Bandyopadhyay, L. Kathmann, T. Weber, Richard D. Smith, H. Wiley, B. Thrall
          </td>
          <td>2012-03-29</td>
          <td>PLoS ONE</td>
          <td>46</td>
          <td>133</td>
        </tr>

        <tr id="Dynamics adds to the complexity of signaling networks. The biochemical processes that control the responses of cells to changes in their internal or external environments form a complex interconnected network. The molecules that participate in these processes are diverse and dynamic, and the networks themselves are affected by the flow of information through them. With this appreciation of the complexity in signaling networks, Science Signaling highlights computational and experimental approaches to understanding both the macroscopic cellular response and the microscopic signaling network and its constituents and dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8045d1bb90c38a9e81507bcbcfd412b519b768f7" target='_blank'>Focus Issue: Unraveling Signaling Complexity</a></td>
          <td>
            N. Gough, J. Foley
          </td>
          <td>2009-07-28</td>
          <td>Science Signaling</td>
          <td>5</td>
          <td>32</td>
        </tr>

        <tr id="Epithelial-mesenchymal transition (EMT) is well established as playing a crucial role in cancer progression and being a potential therapeutic target. To elucidate the gene regulation that drives the decision making of EMT, many previous studies have been conducted to model EMT gene regulatory circuits (GRCs) using interactions from the literature. While this approach can depict the generic regulatory interactions, it falls short of capturing context-specific features. Here, we explore the effectiveness of a combined bioinformatics and mathematical modeling approach to construct context-specific EMT GRCs directly from transcriptomics data. Using time-series single cell RNA-sequencing data from four different cancer cell lines treated with three EMT-inducing signals, we identify context-specific activity dynamics of common EMT transcription factors. In particular, we observe distinct paths during the forward and backward transitions, as is evident from the dynamics of major regulators such as NF-KB (e.g., NFKB2 and RELB) and AP-1 (e.g., FOSL1 and JUNB). For each experimental condition, we systematically sample a large set of network models and identify the optimal GRC capturing context-specific EMT states using a mathematical modeling method named Random Circuit Perturbation (RACIPE). The results demonstrate that the approach can build high quality GRCs in certain cases, but not others and, meanwhile, elucidate the role of common bioinformatics parameters and properties of network structures in determining the quality of GRCs. We expect the integration of top-down bioinformatics and bottom-up systems biology modeling to be a powerful and generally applicable approach to elucidate gene regulatory mechanisms of cellular state transitions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3c4278005e08109ced6029bce3e30144603bc5" target='_blank'>Toward Modeling Context-Specific EMT Regulatory Networks Using Temporal Single Cell RNA-Seq Data</a></td>
          <td>
            Daniel Ramirez, V. Kohar, Mingyang Lu
          </td>
          <td>2020-04-23</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>24</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b0d4a5d209975c2d5166fd635acb02a22157e56" target='_blank'>The crosstalk between EGF, IGF, and Insulin cell signaling pathways - computational and experimental analysis</a></td>
          <td>
            Rafal Zieliński, P. Przytycki, Jie Zheng, David Zhang, T. Przytycka, J. Capala
          </td>
          <td>2009-09-04</td>
          <td>BMC Systems Biology</td>
          <td>62</td>
          <td>51</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>