---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-17 18:05:08 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Quantitative systems pharmacology model‐based investigation of adverse gastrointestinal events associated with prolonged treatment with PI3‐kinase inhibitors</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Inflammatory bowel disorders are non-infectious chronic inflammation of the esophageal tract (IBD). IBD is divided into two types: ulcerative colitis (UC) and Crohn's disease. The highest incidence and prevalence rates of ulcerative colitis are seen in North America and northern Europe, with incidence rates ranging from 9 to 20 cases per 100 000 person-years and prevalence rates ranging from 156 to 291 cases per 100 000 persons. OZM is contraindicated in patients with previous history of cardiovascular complications. The dose and dosage of OZM is 0.23 mg by oral route (qDay) for 1 to 4 days followed by 5 to 7:0.46 mg and on eighth day titrated to 0.92 mg PO (qDay). OZM is not highly recommended in pregnant and lactating women since existing animal studies not reported with enough evidence. The in silico analysis of Ozanimod (OZM) is not yet been fully characterized. The current study was designed to investigate the pharmacokinetic profile of the OZM and its interaction potential with UC target protein.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8146d60559fce654360081441e7243fe352e72cd" target='_blank'>Network pharmacology and in silico pharmacokinetic prediction of Ozanimod in the management of ulcerative colitis: A computational study</a></td>
          <td>
            Priya Sivasakthi, Sarvesh Sabarathinam, T. Vijayakumar
          </td>
          <td>2022-01-01</td>
          <td>Health Science Reports</td>
          <td>5</td>
          <td>9</td>
        </tr>
    
        <tr id="
 
 
 Clinical evidence supports the combination therapy of vedolizumab (a gut-selective, anti-lymphocyte trafficking drug inhibiting α4β7-integrin) with JAK 1, 2, 3 inhibitors (JAKi) for Inflammatory Bowel Disease. However, the mechanistic explanation for the improved outcomes in Ulcerative Colitis patients treated with vedolizumab plus JAKi remains unknown and even more in Crohn's Disease (CD). In this study we used an unbiased in silico systems biology and artificial intelligence approach to gain a holistic view of the mechanisms that may underpin the potential benefits of the mentioned combination therapy in the treatment of CD.
 
 
 
 The drugs under study -vedolizumab and JAKi- and CD were characterized at protein level by compiling data from extensive literature search. Through it, human protein interaction networks focused on CD protein effectors were generated and used to construct mathematical models using Therapeutic Performance Mapping System (TPMS) technology (Anaxomics Biotech, Barcelona, Spain). Sampling-based methods were employed to assess the impact on CD of the combined therapy vedolizumab plus JAKi and to describe underlying molecular mechanisms.
 
 
 
 Characterization of CD allowed identifying 4 pathological processes (referred to as motives-M) underlying the manifestation of the disease: intestinal barrier disruption (M1), increased innate immune response (M2), chronic inflammation and Th1/Th17 adaptive immune response (M3) and tissue remodeling (M4) and the molecular effectors or proteins involved in each motive. The combination of vedolizumab and JAKi induced a high degree of reversion of the effectors altered in CD (54.76%), higher than the percentage reverted by the individual drugs. Combined vedolizumab plus JAKi therapy mainly modulated M3, where both drugs mostly converge although by distinct mechanisms, thus providing additional individual benefits. The combination therapy reverted a great percentage of M3 effectors, for example, CCR9 and FASLG which were increased in CD, were reverted (or downregulated). In addition to M3 modulation, the combination therapy also modulated M1 and M4. In M1, vedolizumab is the one that would provide more complementary mechanisms apart from those that already converge in both drugs. Regarding M4, it seems that main contribution comes from complementarity pathways of the different mechanisms of action of each individual drug.
 
 
 
 The constructed in silico models reveal that the combined vedolizumab plus JAKi therapy could modulate a high array of effectors altered in CD, more than any of the therapies alone, by enhancing the effects of the individual drugs, and provide a mechanistic rationale for employing the combination of these drugs as potential therapy for CD.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37afddf4916b5088749544ad27899c601491b710" target='_blank'>P969 Unveiling the molecular mechanisms of the combination of vedolizumab with JAK inhibitors in Crohn’s Disease through a systems biology and artificial intelligence-based approach</a></td>
          <td>
            I. Marín-Jiménez, M. Sierra-Ausín, T. Letosa-Abián, J. Aparicio, C. Montoto-Otero, S. Sánchez-Ramón
          </td>
          <td>2024-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>1</td>
          <td>7</td>
        </tr>
    
        <tr id="Crohn's disease (CD) is a complex inflammatory bowel disease whose pathogenesis appears to involve several immunologic defects causing functional impairment of the gut. Its complexity and the reported loss of effectiveness over time of standard of care together with the increase in its worldwide incidence require the application of techniques aiming to find new therapeutic strategies. Currently, systems pharmacology modeling has been gaining importance as it integrates the available knowledge of the system into a single computational model. In this work, the following workflow for robust application of systems pharmacology modeling was followed: 1) scope definition; 2) species selection and circulating plasma levels based on a search in the literature; 3) representation of model topology and parametrization of the interactions, after literature data extraction and curation, and the implementation of ordinary differential equations in SimBiology (MATLAB version R2018b); and 4) model curation and evaluation by visual comparison of simulated interleukin (IL) concentrations with the reported levels in plasma, and sensitivity analysis performed to confirm model robustness and identify the most influential parameters. Finally, 5) exposure to two dose levels of recombinant human IL10 was evaluated by simulation and comparison with reported clinical study results. In summary, we present a quantitative systems pharmacology model for the main ILs involved in CD developed using a standardized methodology and supported by a comprehensive repository summarizing the most relevant literature in the field. However, it has to be taken into account that external validation is still pending as available clinical data were primarily used for model training. SIGNIFICANCE STATEMENT Crohn's disease (CD) is a complex heterogeneous inflammatory bowel disorder. Systems pharmacology modeling offers a great opportunity for integration of the available knowledge on the disease using a computational framework. As a result of this work, a comprehensive repository along with a quantitative systems pharmacology model for the main interleukins involved in CD is provided. This model is useful for the in silico evaluation of biomarkers and potential therapeutic targets and can be adapted to address research gaps regarding CD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1b59cbda9a63547cdc28a859ad8c140d010a08" target='_blank'>A Quantitative Systems Pharmacology Model for the Key Interleukins Involved in Crohn's Disease</a></td>
          <td>
            Violeta Balbas-Martinez, E. Asín-Prieto, Zinnia P. Parra-Guillen, I. Trocóniz
          </td>
          <td>2019-12-10</td>
          <td>The Journal of Pharmacology and Experimental Therapeutics</td>
          <td>8</td>
          <td>32</td>
        </tr>
    
        <tr id="ABSTRACT Severe mortality due to the COVID-19 pandemic resulted from the lack of effective treatment. Although COVID-19 vaccines are available, their side effects have become a challenge for clinical use in patients with chronic diseases, especially cancer patients. In the current report, we applied network pharmacology and systematic bioinformatics to explore the use of biochanin A in patients with colorectal cancer (CRC) and COVID-19 infection. Using the network pharmacology approach, we identified two clusters of genes involved in immune response (IL1A, IL2, and IL6R) and cell proliferation (CCND1, PPARG, and EGFR) mediated by biochanin A in CRC/COVID-19 condition. The functional analysis of these two gene clusters further illustrated the effects of biochanin A on interleukin-6 production and cytokine-cytokine receptor interaction in CRC/COVID-19 pathology. In addition, pathway analysis demonstrated the control of PI3K-Akt and JAK-STAT signaling pathways by biochanin A in the treatment of CRC/COVID-19. The findings of this study provide a therapeutic option for combination therapy against COVID-19 infection in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916657a05db2cf3c7c148f16e01b739551950794" target='_blank'>Bioinformatics and in-silico findings reveal medical features and pharmacological targets of biochanin A against colorectal cancer and COVID-19</a></td>
          <td>
            Jingru Qin, C. Guo, Lu Yang, Xiaoliu Liang, Ai-Jun Jiao, K. Lai, Bin Yang
          </td>
          <td>2021-12-20</td>
          <td>Bioengineered</td>
          <td>5</td>
          <td>34</td>
        </tr>
    
        <tr id="[This retracts the article DOI: 10.1155/2022/7102500.].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee8964d4d366bc0df5964c9fdf0ea5152179ff69" target='_blank'>Retracted: Network Pharmacology-Based Strategy to Investigate the Mechanisms of Lenvatinib in the Treatment of Hepatocellular Carcinoma</a></td>
          <td>
            Computational Intelligence and Neuroscience
          </td>
          <td>2023-11-29</td>
          <td>Computational Intelligence and Neuroscience</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr id="Department of Gastroenterology, Guangdong Provincial People’s Hospital, Guangdong Academy of Medical Sciences, Southern Medical University, Guangzhou, China, The Second School of Clinical Medicine, Southern Medical University, Guangzhou, China, Shantou University Medical College, Shantou University, Shantou, China, Department of Computer Science, The University of Hong Kong, Hong Kong, Hong Kong SAR, China, Sepulveda Ambulatory Care Center, VA Greater Los Angeles Healthcare System, Los Angeles, CA, United States, University of California Los Angeles David Geffen School of Medicine, Los Angeles, CA, United States">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27aebc14245b473fcf6eae2729d7890fa59877d" target='_blank'>Editorial: Exploration of genetic variation, drug response, and interactions between gastrointestinal disorders and other diseases</a></td>
          <td>
            Shuangshuang Tong, Yanlin Lyu, Ruibang Luo, Felix W. Leung, W. Sha, Hao Chen
          </td>
          <td>2023-09-13</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>14</td>
        </tr>
    
        <tr id="With the advent of effective immunotherapies to battle cancers and diseases, an obstacle in recovery has become the potential side effects, specifically cytokine release syndrome (CRS). As there is little quantitative understanding of risks for developing CRS and the degree of its severity, this work explored a model-based approach to produce a library of in silico patients through sensitivity analysis of cytokine interaction parameters and a Monte Carlo sampling. The objective of producing the in silico patients was to correlate a known grading system of cytokine release syndrome severity and thus design a new formula for grading CRS. Using our CRS grading system as the foundation, this work produced not only a formula which related the in silico patient data to the different grades, but we effectively demonstrated a selective approach to reduce the grade of CRS with sequential cytokine inhibition targets. We achieved the reduction of grades by applying the insight from the sensitivity analysis, beginning with the most sensitive targets. Cytokines IL-1, IL-8, TNF-α, INF-γ, IL-6, IL-2, IL-4, IL-10, and IL-12 were in turn the best targets for inhibition to alleviate CRS. Using this approach, patient cytokine time profiles in real-time can be related to the CRS grading system and if the grade is severe enough, action can be taken earlier during the treatment to prevent potentially life-threatening symptoms. What’s more, the identified inhibition sequence of the 9 cytokines provides guidance for clinical intervention of CRS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdbed3634ba13aed3fc2c3d954812638cfe069fa" target='_blank'>A Model-Based Investigation of Cytokine Dynamics in Immunotherapies</a></td>
          <td>
            B. Hopkins, Yiming Pan, M. Tucker, Z. Huang
          </td>
          <td>2018-12-30</td>
          <td>Processes</td>
          <td>5</td>
          <td>18</td>
        </tr>
    
        <tr id="Weighted gene co-expression network analysis, single-cell sequencing analysis, immune inﬁltration analysis, machine learning, DEG analysis and network pharmacology were the six analysis included in the study. Results showed that the co-morbidity between type 2 diabetes mellitus and ulcerative colitis is primarily associated with immune-inﬂammatory pathways, including IL-17, TNF, chemokine and toll-like receptor signaling pathways. Machine learning studies identiﬁed IGFBP3 as a biomarker for Gegen Qinlian Decoction in treating type 2 diabetes mellitus, while BACE2, EPHB4 and EPHA2 emerged as biomarkers for Gegen Qinlian decoction in ulcerative colitis treatment. The study provides insights into the shared pathogenesis of type 2 diabetes mellitus and ulcerative colitis and proposes novel targets and therapeutic strategies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bf5ade925b543b7cde2a6150f5b418f376b210c" target='_blank'>Editorial: Molecular informatics in precision medicine</a></td>
          <td>
            Alice Chen, S. Abdulazeez, Hari S. Sharma, J. Borgio
          </td>
          <td>2024-12-03</td>
          <td>Frontiers in Medicine</td>
          <td>1</td>
          <td>20</td>
        </tr>
    
        <tr id="The immune system is a complex and dynamic network, crucial for combating infections and maintaining health. Developing a comprehensive digital twin of the immune system requires incorporating essential cellular components and their interactions. This study presents the first blueprint for an immune system digital twin, consisting of a comprehensive and simulatable mechanistic model. It integrates 51 innate and adaptive immune cells, 37 secretory factors, and 11 disease conditions, providing the foundation for developing a multi-scale model. The cellular-level model demonstrates its potential in characterizing immune responses to various single and combinatorial disease conditions. By making the model available in easy-to-use formats directly in the Cell Collective platform, the community can easily and further expand it. This blueprint represents a significant step towards developing general-purpose immune digital twins, with far-reaching implications for the future of digital twin technology in life sciences and healthcare, advancing patient care, and accelerating precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6641c34ef9f1144419c9d233fad85729b60ada25" target='_blank'>Immune Digital Twin Blueprint: A Comprehensive Mechanistic Model of the Human Immune System</a></td>
          <td>
            Rada Amin, S. Aghamiri, B. L. Puniya, Lauren Mayo, Dennis Startsev, Kashish Poore, Resa M. K. Helikar, T. Helikar
          </td>
          <td>2020-03-12</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>24</td>
        </tr>
    
        <tr id="We address the need for modelling and predicting adverse outcomes in immunotoxicology to improve non-clinical assessments of immunomodulatory therapy safety and efficacy. The integrated approach includes, first, the adverse outcome pathway concept established in the toxicology field, and, second, the systems medicine disease map approach for describing molecular mechanisms involved in a particular pathology. The proposed systems immunotoxicology workflow is demonstrated with CAR T cell treatment as a use case. To this end, the linear adverse outcome pathway (AOP) is expanded into a molecular interaction model in standard systems biology formats. Then it is shown how knowledge related to immunotoxic events can be integrated, encoded, managed and explored to benefit the research community. The map is accessible online via the MINERVA Platform for browsing, commenting and data visualisation (https://minerva.pages.uni.lu). Our work transforms a graphical illustration of an AOP into a digitally structured and standardised form, featuring precise and controlled vocabulary and supporting reproducible computational analyses. Because of annotations to source literature and databases, the map can be further expanded to match the evolving knowledge and research questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d13f7d84fd51fe9ce0ef217f6267f61784e78d9" target='_blank'>Using interactive platforms to encode, manage and explore immune-related adverse outcome pathways</a></td>
          <td>
            A. Mazein, Muhammad Shoaib, M. Alb, C. Sakellariou, Charline Sommer, K. Sewald, K. Reiche, Patricia Gogesch, Luise A Roser, Samira Ortega Iannazzo, S. Sheth, S. Schiffmann, Z. Waibler, V. Neuhaus, Susann Dehmel, V. Satagopam, Reinhard Schneider, M. Ostaszewski, Wei Gu
          </td>
          <td>2023-03-23</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>30</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>