---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-21 10:41:42 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Quantitative systems pharmacology model‐based investigation of adverse gastrointestinal events associated with prolonged treatment with PI3‐kinase inhibitors</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Inflammatory bowel disorders are non-infectious chronic inflammation of the esophageal tract (IBD). IBD is divided into two types: ulcerative colitis (UC) and Crohn's disease. The highest incidence and prevalence rates of ulcerative colitis are seen in North America and northern Europe, with incidence rates ranging from 9 to 20 cases per 100 000 person-years and prevalence rates ranging from 156 to 291 cases per 100 000 persons. OZM is contraindicated in patients with previous history of cardiovascular complications. The dose and dosage of OZM is 0.23 mg by oral route (qDay) for 1 to 4 days followed by 5 to 7:0.46 mg and on eighth day titrated to 0.92 mg PO (qDay). OZM is not highly recommended in pregnant and lactating women since existing animal studies not reported with enough evidence. The in silico analysis of Ozanimod (OZM) is not yet been fully characterized. The current study was designed to investigate the pharmacokinetic profile of the OZM and its interaction potential with UC target protein.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8146d60559fce654360081441e7243fe352e72cd" target='_blank'>Network pharmacology and in silico pharmacokinetic prediction of Ozanimod in the management of ulcerative colitis: A computational study</a></td>
          <td>
            Priya Sivasakthi, Sarvesh Sabarathinam, T. Vijayakumar
          </td>
          <td>2022-01-01</td>
          <td>Health Science Reports</td>
          <td>5</td>
          <td>9</td>
        </tr>
    
        <tr id="
 
 
 Clinical evidence supports the combination therapy of vedolizumab (a gut-selective, anti-lymphocyte trafficking drug inhibiting α4β7-integrin) with JAK 1, 2, 3 inhibitors (JAKi) for Inflammatory Bowel Disease. However, the mechanistic explanation for the improved outcomes in Ulcerative Colitis patients treated with vedolizumab plus JAKi remains unknown and even more in Crohn's Disease (CD). In this study we used an unbiased in silico systems biology and artificial intelligence approach to gain a holistic view of the mechanisms that may underpin the potential benefits of the mentioned combination therapy in the treatment of CD.
 
 
 
 The drugs under study -vedolizumab and JAKi- and CD were characterized at protein level by compiling data from extensive literature search. Through it, human protein interaction networks focused on CD protein effectors were generated and used to construct mathematical models using Therapeutic Performance Mapping System (TPMS) technology (Anaxomics Biotech, Barcelona, Spain). Sampling-based methods were employed to assess the impact on CD of the combined therapy vedolizumab plus JAKi and to describe underlying molecular mechanisms.
 
 
 
 Characterization of CD allowed identifying 4 pathological processes (referred to as motives-M) underlying the manifestation of the disease: intestinal barrier disruption (M1), increased innate immune response (M2), chronic inflammation and Th1/Th17 adaptive immune response (M3) and tissue remodeling (M4) and the molecular effectors or proteins involved in each motive. The combination of vedolizumab and JAKi induced a high degree of reversion of the effectors altered in CD (54.76%), higher than the percentage reverted by the individual drugs. Combined vedolizumab plus JAKi therapy mainly modulated M3, where both drugs mostly converge although by distinct mechanisms, thus providing additional individual benefits. The combination therapy reverted a great percentage of M3 effectors, for example, CCR9 and FASLG which were increased in CD, were reverted (or downregulated). In addition to M3 modulation, the combination therapy also modulated M1 and M4. In M1, vedolizumab is the one that would provide more complementary mechanisms apart from those that already converge in both drugs. Regarding M4, it seems that main contribution comes from complementarity pathways of the different mechanisms of action of each individual drug.
 
 
 
 The constructed in silico models reveal that the combined vedolizumab plus JAKi therapy could modulate a high array of effectors altered in CD, more than any of the therapies alone, by enhancing the effects of the individual drugs, and provide a mechanistic rationale for employing the combination of these drugs as potential therapy for CD.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37afddf4916b5088749544ad27899c601491b710" target='_blank'>P969 Unveiling the molecular mechanisms of the combination of vedolizumab with JAK inhibitors in Crohn’s Disease through a systems biology and artificial intelligence-based approach</a></td>
          <td>
            I. Marín-Jiménez, M. Sierra-Ausín, T. Letosa-Abián, J. Aparicio, C. Montoto-Otero, S. Sánchez-Ramón
          </td>
          <td>2024-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>1</td>
          <td>7</td>
        </tr>
    
        <tr id="Crohn's disease (CD) is a complex inflammatory bowel disease whose pathogenesis appears to involve several immunologic defects causing functional impairment of the gut. Its complexity and the reported loss of effectiveness over time of standard of care together with the increase in its worldwide incidence require the application of techniques aiming to find new therapeutic strategies. Currently, systems pharmacology modeling has been gaining importance as it integrates the available knowledge of the system into a single computational model. In this work, the following workflow for robust application of systems pharmacology modeling was followed: 1) scope definition; 2) species selection and circulating plasma levels based on a search in the literature; 3) representation of model topology and parametrization of the interactions, after literature data extraction and curation, and the implementation of ordinary differential equations in SimBiology (MATLAB version R2018b); and 4) model curation and evaluation by visual comparison of simulated interleukin (IL) concentrations with the reported levels in plasma, and sensitivity analysis performed to confirm model robustness and identify the most influential parameters. Finally, 5) exposure to two dose levels of recombinant human IL10 was evaluated by simulation and comparison with reported clinical study results. In summary, we present a quantitative systems pharmacology model for the main ILs involved in CD developed using a standardized methodology and supported by a comprehensive repository summarizing the most relevant literature in the field. However, it has to be taken into account that external validation is still pending as available clinical data were primarily used for model training. SIGNIFICANCE STATEMENT Crohn's disease (CD) is a complex heterogeneous inflammatory bowel disorder. Systems pharmacology modeling offers a great opportunity for integration of the available knowledge on the disease using a computational framework. As a result of this work, a comprehensive repository along with a quantitative systems pharmacology model for the main interleukins involved in CD is provided. This model is useful for the in silico evaluation of biomarkers and potential therapeutic targets and can be adapted to address research gaps regarding CD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1b59cbda9a63547cdc28a859ad8c140d010a08" target='_blank'>A Quantitative Systems Pharmacology Model for the Key Interleukins Involved in Crohn's Disease</a></td>
          <td>
            Violeta Balbas-Martinez, E. Asín-Prieto, Zinnia P. Parra-Guillen, I. Trocóniz
          </td>
          <td>2019-12-10</td>
          <td>The Journal of Pharmacology and Experimental Therapeutics</td>
          <td>8</td>
          <td>32</td>
        </tr>
    
        <tr id="ABSTRACT Severe mortality due to the COVID-19 pandemic resulted from the lack of effective treatment. Although COVID-19 vaccines are available, their side effects have become a challenge for clinical use in patients with chronic diseases, especially cancer patients. In the current report, we applied network pharmacology and systematic bioinformatics to explore the use of biochanin A in patients with colorectal cancer (CRC) and COVID-19 infection. Using the network pharmacology approach, we identified two clusters of genes involved in immune response (IL1A, IL2, and IL6R) and cell proliferation (CCND1, PPARG, and EGFR) mediated by biochanin A in CRC/COVID-19 condition. The functional analysis of these two gene clusters further illustrated the effects of biochanin A on interleukin-6 production and cytokine-cytokine receptor interaction in CRC/COVID-19 pathology. In addition, pathway analysis demonstrated the control of PI3K-Akt and JAK-STAT signaling pathways by biochanin A in the treatment of CRC/COVID-19. The findings of this study provide a therapeutic option for combination therapy against COVID-19 infection in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916657a05db2cf3c7c148f16e01b739551950794" target='_blank'>Bioinformatics and in-silico findings reveal medical features and pharmacological targets of biochanin A against colorectal cancer and COVID-19</a></td>
          <td>
            Jingru Qin, C. Guo, Lu Yang, Xiaoliu Liang, Ai-Jun Jiao, K. Lai, Bin Yang
          </td>
          <td>2021-12-20</td>
          <td>Bioengineered</td>
          <td>5</td>
          <td>34</td>
        </tr>
    
        <tr id="[This retracts the article DOI: 10.1155/2022/7102500.].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee8964d4d366bc0df5964c9fdf0ea5152179ff69" target='_blank'>Retracted: Network Pharmacology-Based Strategy to Investigate the Mechanisms of Lenvatinib in the Treatment of Hepatocellular Carcinoma</a></td>
          <td>
            Computational Intelligence and Neuroscience
          </td>
          <td>2023-11-29</td>
          <td>Computational Intelligence and Neuroscience</td>
          <td>0</td>
          <td>3</td>
        </tr>
    
        <tr id="Rare diseases, affecting millions globally, present significant drug development challenges. This is due to the limited patient populations and the unique pathophysiology of these diseases, which can make traditional clinical trial designs unfeasible. Quantitative Systems Pharmacology (QSP) models offer a promising approach to expedite drug development, particularly in rare diseases. QSP models provide a mechanistic representation of the disease and drug response in virtual patients that can complement routinely applied empirical modeling and simulation approaches. QSP models can generate digital twins of actual patients and mechanistically simulate the disease progression of rare diseases, accounting for phenotypic heterogeneity. QSP models can also support drug development in various drug modalities, such as gene therapy. Impactful QSP models case studies are presented here to illustrate their value in supporting various aspects of drug development in rare indications. As these QSP model applications continue to mature, there is a growing possibility that they could be more widely integrated into routine drug development steps. This integration could provide a robust framework for addressing some of the inherent challenges in rare disease drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/501aa60f69949c250c5052e0be685b26e600d873" target='_blank'>Quantitative Systems Pharmacology Models: Potential Tools for Advancing Drug Development for Rare Diseases.</a></td>
          <td>
            Susana Neves-Zaph, C. Kaddi
          </td>
          <td>2024-09-28</td>
          <td>Clinical pharmacology and therapeutics</td>
          <td>1</td>
          <td>10</td>
        </tr>
    
        <tr id="With the advent of effective immunotherapies to battle cancers and diseases, an obstacle in recovery has become the potential side effects, specifically cytokine release syndrome (CRS). As there is little quantitative understanding of risks for developing CRS and the degree of its severity, this work explored a model-based approach to produce a library of in silico patients through sensitivity analysis of cytokine interaction parameters and a Monte Carlo sampling. The objective of producing the in silico patients was to correlate a known grading system of cytokine release syndrome severity and thus design a new formula for grading CRS. Using our CRS grading system as the foundation, this work produced not only a formula which related the in silico patient data to the different grades, but we effectively demonstrated a selective approach to reduce the grade of CRS with sequential cytokine inhibition targets. We achieved the reduction of grades by applying the insight from the sensitivity analysis, beginning with the most sensitive targets. Cytokines IL-1, IL-8, TNF-α, INF-γ, IL-6, IL-2, IL-4, IL-10, and IL-12 were in turn the best targets for inhibition to alleviate CRS. Using this approach, patient cytokine time profiles in real-time can be related to the CRS grading system and if the grade is severe enough, action can be taken earlier during the treatment to prevent potentially life-threatening symptoms. What’s more, the identified inhibition sequence of the 9 cytokines provides guidance for clinical intervention of CRS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdbed3634ba13aed3fc2c3d954812638cfe069fa" target='_blank'>A Model-Based Investigation of Cytokine Dynamics in Immunotherapies</a></td>
          <td>
            B. Hopkins, Yiming Pan, M. Tucker, Z. Huang
          </td>
          <td>2018-12-30</td>
          <td>Processes</td>
          <td>5</td>
          <td>18</td>
        </tr>
    
        <tr id="Quantitative systems pharmacology (QSP) places an emphasis on dynamic systems modeling, incorporating considerations from systems biology modeling and pharmacodynamics. The goal of QSP is often to quantitatively predict the effects of clinical therapeutics, their combinations, and their doses on clinical biomarkers and endpoints. In order to achieve this goal, strategies for incorporating clinical data into model calibration are critical. Virtual population (VPop) approaches facilitate model calibration while faced with challenges encountered in QSP model application, including modeling a breadth of clinical therapies, biomarkers, endpoints, utilizing data of varying structure and source, capturing observed clinical variability, and simulating with models that may require more substantial computational time and resources than often found in pharmacometrics applications. VPops are frequently developed in a process that may involve parameterization of isolated pathway models, integration into a larger QSP model, incorporation of clinical data, calibration, and quantitative validation that the model with the accompanying, calibrated VPop is suitable to address the intended question or help with the intended decision. Here, we introduce previous strategies for developing VPops in the context of a variety of therapeutic and safety areas: metabolic disorders, drug-induced liver injury, autoimmune diseases, and cancer. We introduce methodological considerations, prior work for sensitivity analysis and VPop algorithm design, and potential areas for future advancement. Finally, we give a more detailed application example of a VPop calibration algorithm that illustrates recent progress and many of the methodological considerations. In conclusion, although methodologies have varied, VPop strategies have been successfully applied to give valid clinical insights and predictions with the assistance of carefully defined and designed calibration and validation strategies. While a uniform VPop approach for all potential QSP applications may be challenging given the heterogeneity in use considerations, we anticipate continued innovation will help to drive VPop application for more challenging cases of greater scale while developing new rigorous methodologies and metrics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68337aa80c1390201d9ea07d9ae681ebb37bfaa5" target='_blank'>Virtual Populations for Quantitative Systems Pharmacology Models.</a></td>
          <td>
            Yougan Cheng, R. Straube, Abed Alnaif, Lu Huang, T. Leil, B. Schmidt
          </td>
          <td>2021-08-24</td>
          <td>Methods in molecular biology</td>
          <td>22</td>
          <td>23</td>
        </tr>
    
        <tr id="Weighted gene co-expression network analysis, single-cell sequencing analysis, immune inﬁltration analysis, machine learning, DEG analysis and network pharmacology were the six analysis included in the study. Results showed that the co-morbidity between type 2 diabetes mellitus and ulcerative colitis is primarily associated with immune-inﬂammatory pathways, including IL-17, TNF, chemokine and toll-like receptor signaling pathways. Machine learning studies identiﬁed IGFBP3 as a biomarker for Gegen Qinlian Decoction in treating type 2 diabetes mellitus, while BACE2, EPHB4 and EPHA2 emerged as biomarkers for Gegen Qinlian decoction in ulcerative colitis treatment. The study provides insights into the shared pathogenesis of type 2 diabetes mellitus and ulcerative colitis and proposes novel targets and therapeutic strategies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bf5ade925b543b7cde2a6150f5b418f376b210c" target='_blank'>Editorial: Molecular informatics in precision medicine</a></td>
          <td>
            Alice Chen, S. Abdulazeez, Hari S. Sharma, J. Borgio
          </td>
          <td>2024-12-03</td>
          <td>Frontiers in Medicine</td>
          <td>1</td>
          <td>20</td>
        </tr>
    
        <tr id="A quantitative systems pharmacology (QSP) model of the pathogenesis and treatment of SARS-CoV-2 infection can streamline and accelerate the development of novel medicines to treat COVID-19. Simulation of clinical trials allows in silico exploration of the uncertainties of clinical trial design and can rapidly inform their protocols. We previously published a preliminary model of the immune response to SARS-CoV-2 infection. To further our understanding of COVID-19 and treatment, we significantly updated the model by matching a curated dataset spanning viral load and immune responses in plasma and lung. We identified a population of parameter sets to generate heterogeneity in pathophysiology and treatment and tested this model against published reports from interventional SARS-CoV-2 targeting mAb and antiviral trials. Upon generation and selection of a virtual population, we match both the placebo and treated responses in viral load in these trials. We extended the model to predict the rate of hospitalization or death within a population. Via comparison of the in silico predictions with clinical data, we hypothesize that the immune response to virus is log-linear over a wide range of viral load. To validate this approach, we show the model matches a published subgroup analysis, sorted by baseline viral load, of patients treated with neutralizing Abs. By simulating intervention at different time points post infection, the model predicts efficacy is not sensitive to interventions within five days of symptom onset, but efficacy is dramatically reduced if more than five days pass post symptom onset prior to treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac952ba3508fc13b6ca5e8d7e11d7a7086f9b72d" target='_blank'>A quantitative systems pharmacology model of the pathophysiology and treatment of COVID-19 predicts optimal timing of pharmacological interventions</a></td>
          <td>
            Rohit Rao, C. Musante, R. Allen
          </td>
          <td>2021-12-08</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>8</td>
          <td>18</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b6d19e4a519ae3668404d76c1f20c8c5b12e928" target='_blank'>Multiscale modeling of mucosal immune responses</a></td>
          <td>
            Yongguo Mei, V. Abedi, Adria Carbo, Xiaoying Zhang, Pinyi Lu, C. Philipson, R. Hontecillas, S. Hoops, Nathan Liles, J. Bassaganya-Riera
          </td>
          <td>2015-08-25</td>
          <td>BMC Bioinformatics</td>
          <td>34</td>
          <td>53</td>
        </tr>
    
        <tr id="The immune system is a complex and dynamic network, crucial for combating infections and maintaining health. Developing a comprehensive digital twin of the immune system requires incorporating essential cellular components and their interactions. This study presents the first blueprint for an immune system digital twin, consisting of a comprehensive and simulatable mechanistic model. It integrates 51 innate and adaptive immune cells, 37 secretory factors, and 11 disease conditions, providing the foundation for developing a multi-scale model. The cellular-level model demonstrates its potential in characterizing immune responses to various single and combinatorial disease conditions. By making the model available in easy-to-use formats directly in the Cell Collective platform, the community can easily and further expand it. This blueprint represents a significant step towards developing general-purpose immune digital twins, with far-reaching implications for the future of digital twin technology in life sciences and healthcare, advancing patient care, and accelerating precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6641c34ef9f1144419c9d233fad85729b60ada25" target='_blank'>Immune Digital Twin Blueprint: A Comprehensive Mechanistic Model of the Human Immune System</a></td>
          <td>
            Rada Amin, S. Aghamiri, B. L. Puniya, Lauren Mayo, Dennis Startsev, Kashish Poore, Resa M. K. Helikar, T. Helikar
          </td>
          <td>2020-03-12</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>24</td>
        </tr>
    
        <tr id="We address the need for modelling and predicting adverse outcomes in immunotoxicology to improve non-clinical assessments of immunomodulatory therapy safety and efficacy. The integrated approach includes, first, the adverse outcome pathway concept established in the toxicology field, and, second, the systems medicine disease map approach for describing molecular mechanisms involved in a particular pathology. The proposed systems immunotoxicology workflow is demonstrated with CAR T cell treatment as a use case. To this end, the linear adverse outcome pathway (AOP) is expanded into a molecular interaction model in standard systems biology formats. Then it is shown how knowledge related to immunotoxic events can be integrated, encoded, managed and explored to benefit the research community. The map is accessible online via the MINERVA Platform for browsing, commenting and data visualisation (https://minerva.pages.uni.lu). Our work transforms a graphical illustration of an AOP into a digitally structured and standardised form, featuring precise and controlled vocabulary and supporting reproducible computational analyses. Because of annotations to source literature and databases, the map can be further expanded to match the evolving knowledge and research questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d13f7d84fd51fe9ce0ef217f6267f61784e78d9" target='_blank'>Using interactive platforms to encode, manage and explore immune-related adverse outcome pathways</a></td>
          <td>
            A. Mazein, Muhammad Shoaib, M. Alb, C. Sakellariou, Charline Sommer, K. Sewald, K. Reiche, Patricia Gogesch, Luise A Roser, Samira Ortega Iannazzo, S. Sheth, S. Schiffmann, Z. Waibler, V. Neuhaus, Susann Dehmel, V. Satagopam, Reinhard Schneider, M. Ostaszewski, Wei Gu
          </td>
          <td>2023-03-23</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>30</td>
        </tr>
    
        <tr id="mrgsolve is an open‐source R package available on the Comprehensive R Archive Network. It combines R and C++ coding for simulation from hierarchical, ordinary differential equation–based models. Its efficient simulation engine and integration into a parallelizable, R‐based workflow makes mrgsolve a convenient tool both for simple and complex models and thus is ideal for physiologically‐based pharmacokinetic (PBPK) and quantitative systems pharmacology (QSP) model. This tutorial will first introduce the basics of the mrgsolve simulation workflow, including model specification, the introduction of interventions (dosing events) into the simulation, and simulated results postprocessing. An applied simulation example is then presented using a PBPK model for voriconazole, including a model validation step against adult and pediatric data sets. A final simulation example is then presented using a previously published QSP model for mitogen‐activated protein kinase signaling in colorectal cancer, illustrating population simulation of different combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f58af03b1918b56e424198aaef83d3aa5da5080" target='_blank'>Quantitative Systems Pharmacology and Physiologically‐Based Pharmacokinetic Modeling With mrgsolve: A Hands‐On Tutorial</a></td>
          <td>
            Ahmed Elmokadem, M. Riggs, K. Baron
          </td>
          <td>2019-11-14</td>
          <td>CPT: Pharmacometrics & Systems Pharmacology</td>
          <td>52</td>
          <td>13</td>
        </tr>
    
        <tr id="To investigate the mechanism of action of Chenpi in the treatment of acute ulcerative colitis, this experiment employed a combination of network pharmacology and molecular docking technology to make a preliminary prediction of its mechanism of action. The TCMSP and PubChem databases were employed to identify the active components of Chenpi and their respective targets. In contrast, the SwissTargetPrediction database was utilized to predict the potential targets of action of Chenpi. The relevant targets of acute ulcerative colitis were obtained from the GeneCards disease database and subsequently intersected with the targets of Chen Pi to identify the key targets of Chen Pi for the treatment of acute ulcerative colitis. The results of the network pharmacological analysis indicated that Chenpi may exert its effects through ESR1, potentially via the action of sitosterol, naringenin, 5,7-dihydroxy-2-(3-hydroxy-4-methoxyphenyl) chroman-4-one, Citromitin, and nobiletin, five active components, as well as PTGS2, PPARG, BCL2, SRC, and MMP9, and an additional 104 core targets. Following the enrichment analysis of the aforementioned 104 core targets, it was determined that pathways such as "Pathways in cancer," "Arachidonic acid metabolism," "Prostate cancer," "EGFR tyrosine kinase inhibitor resistance," and others were involved in the treatment of Chen Pi. The results of molecular docking also demonstrated that the active ingredients of Chen Pi exhibited favorable binding affinities with the key targets, thereby substantiating the precision of the network pharmacology-predicted outcomes. The results of this experiment suggest that Chenpi may exert its therapeutic effects on acute ulcerative colitis through a multi-component, multi-target, and multi-pathway mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d20b2fc0c45174d5913c1461e9e75f579403316" target='_blank'>Exploring the Mechanism of Chen Pi in Treating Acute Ulcerative Colitis Based on Network Pharmacology and Molecular Docking</a></td>
          <td>
            Nai-dan Zhang, Tun Zhang
          </td>
          <td>2024-08-06</td>
          <td>Scholars Journal of Agriculture and Veterinary Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>
    
        <tr id="Motivation The literature on complex diseases is abundant but not always quantitative. This is particularly so for Inflammatory Bowel Disease (IBD), where many molecular pathways are qualitatively well described but this information cannot be used in traditional quantitative mathematical models employed in drug development. We propose the elaboration and validation of a logic network for IBD able to capture the information available in the literature that will facilitate the identification/validation of therapeutic targets. Results In this article, we propose a logic model for Inflammatory Bowel Disease (IBD) which consists of 43 nodes and 298 qualitative interactions. The model presented is able to describe the pathogenic mechanisms of the disorder and qualitatively describes the characteristic chronic inflammation. A perturbation analysis performed on the IBD network indicates that the model is robust. Also, as described in clinical trials, a simulation of anti-TNFα, anti-IL2 and Granulocyte and Monocyte Apheresis showed a decrease in the Metalloproteinases node (MMPs), which means a decrease in tissue damage. In contrast, as clinical trials have demonstrated, a simulation of anti-IL17 and anti-IFNγ or IL10 overexpression therapy did not show any major change in MMPs expression, as corresponds to a failed therapy. The model proved to be a promising in silico tool for the evaluation of potential therapeutic targets, the identification of new IBD biomarkers, the integration of IBD polymorphisms to anticipate responders and non-responders and can be reduced and transformed in quantitative model/s.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c655f04a31d1dc93671152d749f2e63985c7bf3" target='_blank'>A systems pharmacology model for inflammatory bowel disease</a></td>
          <td>
            Violeta Balbas-Martinez, Leire Ruiz-Cerdá, I. Irurzun-Arana, I. González-García, A. Vermeulen, J. Gómez-Mantilla, I. Trocóniz
          </td>
          <td>2018-03-07</td>
          <td>PLoS ONE</td>
          <td>31</td>
          <td>32</td>
        </tr>
    
        <tr id="[This retracts the article DOI: 10.1155/2022/4640849.].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b703523f60961ba983f02a3568650635b680874d" target='_blank'>Retracted: Identification of Molecular Targets and Underlying Mechanisms of Xiaoji Recipe against Pancreatic Cancer Based on Network Pharmacology</a></td>
          <td>
            Computational and Mathematical Methods in Medicine
          </td>
          <td>2023-12-06</td>
          <td>Computational and Mathematical Methods in Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/234074db21cc720b2506084d5f8bd01f9f92fa08" target='_blank'>Physiological Indirect Response Model to Omics-Powered Quantitative Systems Pharmacology Model.</a></td>
          <td>
            Aydar Uatay, Louis Gall, Linda Irons, Shivendra G. Tewari, Xu (Sue) Zhu, Megan Gibbs, Holly Kimko
          </td>
          <td>2023-10-01</td>
          <td>Journal of pharmaceutical sciences</td>
          <td>1</td>
          <td>14</td>
        </tr>
    
        <tr id="In this tutorial, we introduce a differential equation simulation model for use in pharmacometrics involving NONMEM, Berkeley Madonna, and R. We report components of the simulation code and similarities/differences between software, rather than how to use each software. Depending on the purpose of the simulation, an appropriate tool can be selected for effective communication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4007f41d1a36622679f4e8d3656811335c5f035a" target='_blank'>Pharmacometric models simulation using NONMEM, Berkeley Madonna and R</a></td>
          <td>
            W. Park
          </td>
          <td>2017-09-01</td>
          <td>Translational and Clinical Pharmacology</td>
          <td>6</td>
          <td>7</td>
        </tr>
    
        <tr id="Quantitative systems pharmacology (QSP) has emerged as an innovative approach in model‐informed drug discovery and development, supporting program decisions from exploratory research through late‐stage clinical trials. In this commentary, we discuss the unique value of disease‐scale “platform” QSP models that are amenable to reuse and repurposing to support diverse clinical decisions in ways distinct from other pharmacometrics strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75fd303e477a32d2844dd2c8a67fced699191fee" target='_blank'>Quantitative Systems Pharmacology: A Case for Disease Models</a></td>
          <td>
            C. Musante, S. Ramanujan, B. J. Schmidt, OG Ghobrial, James Lu, A. Heatherington
          </td>
          <td>2016-11-18</td>
          <td>Clinical Pharmacology and Therapeutics</td>
          <td>50</td>
          <td>23</td>
        </tr>
    
        <tr id="CPT: Pharmacometrics & Systems Pharmacology (PSP) is a crossdisciplinary journal which publishes research articles, reviews and tutorials that bridge various disciplines including pharmacometrics, systems pharmacology modeling, physiologically based pharmacokinetics (PBPK), translational research, modelbased metaanalyses (MBMA) of clinical trials, computational pharmacology and bioinformatics. PSP was launched in September 2012, as the second journal of American Society for Clinical Pharmacology and Therapeutics (ASCPT), with the first being Clinical Pharmacology and Therapeutics (CPT). Since the launch of Clinical & Translational Science (CTS), PSP is now part of a family of three journals. In June 2021, approaching its 9th anniversary, PSP was granted for the first time, an impact factor. We are proud to announce that the impact factor for 2020 is 4.054. Of note, the cite score of PSP was already increasing from 4.1 in 2015 to 6.3 in 2020, which is great news for both the journal and our scientific community. Hence, for the July 2021 issue, we introduce our first videoabstract, and in this editorial present our vision for the journal for the next ten years, and last, but certainly not least, present our new cover (Figure 1). It was selected by the Editor in Chief, associate editors and the ASCPT board members to illustrate the various aspects of PSP (molecules, equations, models, patients) and their interplay but also the cycling of the different components of modelinformed drug development (MIDD) and development and application of models as they continue to evolve.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff206807101a81dfa131bdd9e99907654d28bea6" target='_blank'>CPT: Pharmacometrics & Systems Pharmacology – Inception, Maturation, and Future Vision</a></td>
          <td>
            Karen Rowland Yeo, S. Hennig, S. Krishnaswami, Natasha Strydom, Vivaswath S. Ayyar, J. French, V. Sinha, E. Sobie, Ping Zhao, L. Friberg, F. Mentré
          </td>
          <td>2021-07-01</td>
          <td>CPT: Pharmacometrics & Systems Pharmacology</td>
          <td>6</td>
          <td>60</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47d3797090cb650abd41db66efa55bed366c4d77" target='_blank'>gPKPDSim: a SimBiology®-based GUI application for PKPD modeling in drug development</a></td>
          <td>
            I. Hosseini, A. Gajjala, Daniela Bumbaca Yadav, S. Sukumaran, S. Ramanujan, Ricardo Paxson, K. Gadkar
          </td>
          <td>2018-01-04</td>
          <td>Journal of Pharmacokinetics and Pharmacodynamics</td>
          <td>25</td>
          <td>24</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47d3797090cb650abd41db66efa55bed366c4d77" target='_blank'>gPKPDSim: a SimBiology®-based GUI application for PKPD modeling in drug development</a></td>
          <td>
            I. Hosseini, A. Gajjala, Daniela Bumbaca Yadav, S. Sukumaran, S. Ramanujan, Ricardo Paxson, K. Gadkar
          </td>
          <td>2018-01-04</td>
          <td>Journal of Pharmacokinetics and Pharmacodynamics</td>
          <td>25</td>
          <td>24</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6891c99e59946101257e77b47551ca66c733bd55" target='_blank'>Quantitative and systems pharmacology 2. In silico polypharmacology of G protein‐coupled receptor ligands via network‐based approaches</a></td>
          <td>
            Zengrui Wu, Weiqiang Lu, Weiwei Yu, Tianduanyi Wang, Weihua Li, Guixia Liu, Hankun Zhang, Xiufeng Pang, Jin Huang, Mingyao Liu, F. Cheng, Yun Tang
          </td>
          <td>2017-11-10</td>
          <td>Pharmacological Research</td>
          <td>24</td>
          <td>67</td>
        </tr>
    
        <tr id="Immunotherapy has shown great potential in the treatment of cancer; however, only a fraction of patients respond to treatment, and many experience autoimmune‐related side effects. The pharmaceutical industry has relied on mathematical models to study the behavior of candidate drugs and more recently, complex, whole‐body, quantitative systems pharmacology (QSP) models have become increasingly popular for discovery and development. QSP modeling has the potential to discover novel predictive biomarkers as well as test the efficacy of treatment plans and combination therapies through virtual clinical trials. In this work, we present a QSP modeling platform for immuno‐oncology (IO) that incorporates detailed mechanisms for important immune interactions. This modular platform allows for the construction of QSP models of IO with varying degrees of complexity based on the research questions. Finally, we demonstrate the use of the platform through two example applications of immune checkpoint therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/750c1aa4fc2f364ea1febcf5761c49bd89bc2f93" target='_blank'>QSP‐IO: A Quantitative Systems Pharmacology Toolbox for Mechanistic Multiscale Modeling for Immuno‐Oncology Applications</a></td>
          <td>
            R. Sové, M. Jafarnejad, Chen Zhao, Hanwen Wang, Huilin Ma, A. Popel
          </td>
          <td>2020-07-02</td>
          <td>CPT: Pharmacometrics & Systems Pharmacology</td>
          <td>39</td>
          <td>69</td>
        </tr>
    
        <tr id="Intravenous immunoglobulin (IVIg) is used as treatment for several autoimmune and inflammatory conditions, but its specific mechanisms are not fully understood. Herein, we aimed to evaluate, using systems biology and artificial intelligence techniques, the differences in the pathophysiological pathways of autoimmune and inflammatory conditions that show diverse responses to IVIg treatment. We also intended to determine the targets of IVIg involved in the best treatment response of the evaluated diseases. Our selection and classification of diseases was based on a previously published systematic review, and we performed the disease characterization through manual curation of the literature. Furthermore, we undertook the mechanistic evaluation with artificial neural networks and pathway enrichment analyses. A set of 26 diseases was selected, classified, and compared. Our results indicated that diseases clearly benefiting from IVIg treatment were mainly characterized by deregulated processes in B cells and the complement system. Indeed, our results show that proteins related to B-cell and complement system pathways, which are targeted by IVIg, are involved in the clinical response. In addition, targets related to other immune processes may also play an important role in the IVIg response, supporting its wide range of actions through several mechanisms. Although B-cell responses and complement system have a key role in diseases benefiting from IVIg, protein targets involved in such processes are not necessarily the same in those diseases. Therefore, IVIg appeared to have a pleiotropic effect that may involve the collaborative participation of several proteins. This broad spectrum of targets and ‘non-specificity’ of IVIg could be key to its efficacy in very different diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad00dddd98df7f6f73eaf2590f07560a1de3655" target='_blank'>Systems biology and artificial intelligence analysis highlights the pleiotropic effect of IVIg therapy in autoimmune diseases with a predominant role on B cells and complement system</a></td>
          <td>
            C. Segú-Vergés, S. Caño, Elisabeth Calderón‐Gómez, H. Bartra, Teresa Sardon, S. Kaveri, J. Terencio
          </td>
          <td>2022-09-30</td>
          <td>Frontiers in Immunology</td>
          <td>7</td>
          <td>73</td>
        </tr>
    
        <tr id="In the last ten years, many advances have been made in the treatment and diagnosis of immune-mediated diseases [...].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b54a56078412c3f8ce10026ceaef38f6f42908b" target='_blank'>Personalized Medicine and Machine Learning: A Roadmap for the Future</a></td>
          <td>
            M. Sebastiani, C. Vacchi, A. Manfredi, G. Cassone
          </td>
          <td>2022-07-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>16</td>
          <td>44</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdba71c8198592733c2f3c0e94873428e96682cb" target='_blank'>Advancing therapies for viral infections using mechanistic computational models of the dynamic interplay between the virus and host immune response</a></td>
          <td>
            V. Zarnitsyna, J. F. Gianlupi, A. Hagar, T. Sego, J. Glazier
          </td>
          <td>2021-08-24</td>
          <td>Current Opinion in Virology</td>
          <td>7</td>
          <td>51</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b67c725d14beca01cd9dd910301260fc0dd1397" target='_blank'>Computational modeling of complex bioenergetic mechanisms that modulate CD4+ T cell effector and regulatory functions</a></td>
          <td>
            Ryan Baker, R. Hontecillas, Nuria Tubau-Juni, Andrew Leber, Shiv D. Kale, J. Bassaganya-Riera
          </td>
          <td>2022-11-22</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>6</td>
          <td>53</td>
        </tr>
    
        <tr id="Quantitative systems pharmacology (QSP) modeling has become increasingly important in pharmaceutical research and development, and is a powerful tool to gain mechanistic insights into the complex dynamics of biological systems in response to drug treatment. However, even once a suitable mathematical framework to describe the pathophysiology and mechanisms of interest is established, final model calibration and the exploration of variability can be challenging and time consuming. QSP models are often formulated as multi-scale, multi-compartment nonlinear systems of ordinary differential equations. Commonly accepted modeling strategies, workflows, and tools have promise to greatly improve the efficiency of QSP methods and improve productivity. In this paper, we present the QSP Toolbox, a set of functions, structure array conventions, and class definitions that computationally implement critical elements of QSP workflows including data integration, model calibration, and variability exploration. We present the application of the toolbox to an ordinary differential equations-based model for antibody drug conjugates. As opposed to a single stepwise reference model calibration, the toolbox also facilitates simultaneous parameter optimization and variation across multiple in vitro, in vivo, and clinical assays to more comprehensively generate alternate mechanistic hypotheses that are in quantitative agreement with available data. The toolbox also includes scripts for developing and applying virtual populations to mechanistic exploration of biomarkers and efficacy. We anticipate that the QSP Toolbox will be a useful resource that will facilitate implementation, evaluation, and sharing of new methodologies in a common framework that will greatly benefit the community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a62799c63dcab1a1119a051f38f7f2d59b0ae0d" target='_blank'>QSP Toolbox: Computational Implementation of Integrated Workflow Components for Deploying Multi-Scale Mechanistic Models</a></td>
          <td>
            Yougan Cheng, Craig J. Thalhauser, S. Smithline, Jyotsna Pagidala, M. Miladinov, H. Vezina, Manish Gupta, T. Leil, B. Schmidt
          </td>
          <td>2017-05-24</td>
          <td>The AAPS Journal</td>
          <td>46</td>
          <td>24</td>
        </tr>
    
        <tr id="Computational modeling is an effective tool for studying complex disease. However, solutions to many models are purely mathematical and cannot immediately provide clinical insights. To overcome this barrier, we propose a series of quantitative scoring metrics that can be used in combination with drug-target interaction data to identify solutions that are readily clinically actionable. Furthermore, we introduce methods for the prediction and ranking of pharmaceutical interventions that closely align with these high-scoring solutions, with an emphasis on robustness across multiple solutions. We demonstrate these methods on a previously-described model of COVID-19 induced cytokine storm. These scoring methods ultimately identify multiple pharmaceutical candidates that have been shown to be effective in reducing mortality rates in COVID-19 patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c667e5eae3273dda154ffc7cf714140d62ee891f" target='_blank'>Leveraging Drug-Target Interaction Data for the Translation of Computational Models into Clinically Actionable Interventions</a></td>
          <td>
            Spencer C. Richmanz, Cole A. Lyman, M. Morris, Hongbao Cao, Anastasia Nesterovay, Anton Yuryevy, C. Cheadle, G. Broderick
          </td>
          <td>2021-12-09</td>
          <td>2021 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</td>
          <td>0</td>
          <td>43</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e913f5b38ea1fa181c5c47d04c1ee10840eec9a" target='_blank'>Artificial intelligence guided discovery of a barrier-protective therapy in inflammatory bowel disease</a></td>
          <td>
            D. Sahoo, Lee Swanson, I. Sayed, G. Katkar, Stella-Rita C. Ibeawuchi, Y. Mittal, Rama F. Pranadinata, Courtney Tindle, MacKenzie Fuller, Dominik L. Stec, John T. Chang, W. Sandborn, Soumita Das, P. Ghosh
          </td>
          <td>2021-07-12</td>
          <td>Nature Communications</td>
          <td>42</td>
          <td>180</td>
        </tr>
    
        <tr id="Significance A distinctive feature of COVID-19 is its extreme heterogeneity—illness ranges from minimally symptomatic to life threatening. Heterogeneity results from a poorly understood combination of patient factors, viral dynamics, antiviral and immune modulating therapies, and dynamics of the innate and adaptive immune responses. In order to better understand clinical heterogeneity and optimal treatment, we developed a comprehensive mathematical model incorporating elements of the innate and adaptive immune responses, the renin−angiotensin system (which the virus exploits for cellular entry), rates of viral replication, inflammatory cytokines, and the coagulation cascade. Our model reveals divergent treatment responses and clinical outcomes as a function of comorbidities, age, and details of the innate and adaptive immune responses which can provide a framework for understanding individual patients’ trajectories. Understanding the underlying mechanisms of COVID-19 progression and the impact of various pharmaceutical interventions is crucial for the clinical management of the disease. We developed a comprehensive mathematical framework based on the known mechanisms of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection, incorporating the renin−angiotensin system and ACE2, which the virus exploits for cellular entry, key elements of the innate and adaptive immune responses, the role of inflammatory cytokines, and the coagulation cascade for thrombus formation. The model predicts the evolution of viral load, immune cells, cytokines, thrombosis, and oxygen saturation based on patient baseline condition and the presence of comorbidities. Model predictions were validated with clinical data from healthy people and COVID-19 patients, and the results were used to gain insight into identified risk factors of disease progression including older age; comorbidities such as obesity, diabetes, and hypertension; and dysregulated immune response. We then simulated treatment with various drug classes to identify optimal therapeutic protocols. We found that the outcome of any treatment depends on the sustained response rate of activated CD8+ T cells and sufficient control of the innate immune response. Furthermore, the best treatment—or combination of treatments—depends on the preinfection health status of the patient. Our mathematical framework provides important insight into SARS-CoV-2 pathogenesis and could be used as the basis for personalized, optimal management of COVID-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e26745913090baa9c6bf3b311f751b5c6602d112" target='_blank'>In silico dynamics of COVID-19 phenotypes for optimizing clinical management</a></td>
          <td>
            C. Voutouri, M. Nikmaneshi, C. Hardin, A. Patel, A. Verma, M. Khandekar, S. Dutta, T. Stylianopoulos, L. Munn, R. Jain
          </td>
          <td>2021-01-05</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>23</td>
          <td>76</td>
        </tr>
    
        <tr id="Tongxie Yaofang (TXYF), a classical traditional Chinese medicine, is commonly used in China to treat ulcerative colitis (UC). The aim of this study was to integrate network pharmacology with molecular docking and molecular dynamics simulations to explore the mechanism of Tongxie Yaofang in the treatment of UC. The traditional Chinese medicine systems pharmacology database was used to retrieve the relevant chemical compositions of the herbs contained in TXYF. The DisGeNET, GeneCards, Online Mendelian Inheritance in Man, and Therapeutic Target Database databases were used to retrieve UC-related targets. To construct protein–protein interaction networks and screen for key targets, gene ontology and Kyoto Encyclopedia of Genes and Genomes analyses of the key targets of TXYF in the treatment of UC were performed using R 4.3.2 software. AutoDock Tools 1.5.7 was used for molecular docking. Molecular dynamics simulations of protein complexes and complexes of proteins with small-molecule ligands and eutectic ligands were carried out with Gromacs 2022 software. Network pharmacology analysis revealed that TXYF could act on UC through multiple targets and pathways. It may exert therapeutic effects mainly through the AGE/RAGE, TOLL, JAK/STAT, and Th17 signaling pathways. The possible targets of TXYF in the treatment of UC could be AKT1, BCL2, EGFR, HMOX1, HSP90AA1, and TGFβ1. Molecular docking analysis revealed that AKT1 had the highest binding energy (‐10.55 kcal/mol). Molecular dynamics simulations revealed that the complexes formed by the AKT1 protein and the chemical compounds MOL001910 and MOL00035 had good stability and high binding strength. AKT1 may be the most critical target of TXYF in treating UC, and the key chemical components of TXYF in treating UC may include β-sitosterol (MOL000358) and 11alpha,12alpha-epoxy-3beta-23-dihydroxy-30-norolean-20-en-28,12beta-olide (MOL00 1910). This study revealed that TXYF may exert therapeutic effects on UC through multiple targets, multiple biological functions, and multiple signaling pathways. This study provides a new insight into the pharmacological mechanism of TXYF in treating UC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e640a4c55a99a557b81e524fe5d0f52985f64641" target='_blank'>Network pharmacology integrated with molecular docking and molecular dynamics simulations to explore the mechanism of Tongxie Yaofang in the treatment of ulcerative colitis</a></td>
          <td>
            Lili Tang, Yuedong Liu, Hongwu Tao, Wenzhe Feng, Cong Ren
          </td>
          <td>2024-09-06</td>
          <td>Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr id="Quantitative systems pharmacology (QSP) modeling has become an increasingly popular approach impacting our understanding of disease mechanisms and helping predict patients’ treatment responses to facilitate study design or development go/no-go decisions. In this paper, we highlight the notable contributions and opportunities that QSP approaches are to offer during the drug development process by sharing three examples that have facilitated internal decisions. The barriers to successful applications and the factors that facilitate the success of the modeling approach is discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15595407c283a7e6ca03ee082cb43d7d405e8328" target='_blank'>Quantitative systems modeling approaches towards model-informed drug development: Perspective through case studies</a></td>
          <td>
            Meghna Verma, Louis Gall, J. Biasetti, G. D. Di Veroli, C. Pichardo-Almarza, M. Gibbs, Holly Kimko
          </td>
          <td>2023-01-12</td>
          <td>None</td>
          <td>5</td>
          <td>21</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e7d956fccc7015105d96d6e72ec20565cd792ad" target='_blank'>Accelerating robust plausible virtual patient cohort generation by substituting ODE simulations with parameter space mapping</a></td>
          <td>
            Thibaud Derippe, Sylvain Fouliard, X. Declèves, D. Mager
          </td>
          <td>2022-10-31</td>
          <td>Journal of Pharmacokinetics and Pharmacodynamics</td>
          <td>4</td>
          <td>42</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43da20acd05724870e9de70bf8ac10a786f3b4f" target='_blank'>Finding multiple target optimal intervention in disease-related molecular network</a></td>
          <td>
            Kun Yang, Hongjun Bai, Ouyang Qi, L. Lai, Chao Tang
          </td>
          <td>2008-11-04</td>
          <td>Molecular Systems Biology</td>
          <td>176</td>
          <td>59</td>
        </tr>
    
        <tr id="Spleen qi deficiency (SQD) syndrome is one of the basic traditional Chinese medicine (TCM) syndromes related to various diseases including chronic inflammation and hypertension and guides the use of many herbal formulae. However, the biological basis of SQD syndrome has not been clearly elucidated due to the lack of appropriate methodologies. Here, we propose a network pharmacology strategy integrating computational, clinical, and experimental investigation to study the biological basis of SQD syndrome. From computational aspects, we used a powerful disease gene prediction algorithm to predict the SQD syndrome biomolecular network which is significantly enriched in biological functions including immune regulation, oxidative stress, and lipid metabolism. From clinical aspects, SQD syndrome is involved in both the local and holistic disorders, that is, the digestive diseases and the whole body's dysfunctions. We, respectively, investigate SQD syndrome-related digestive diseases including chronic gastritis and irritable bowel syndrome and the whole body's dysfunctions such as chronic fatigue syndrome and hypertension. We found innate immune and oxidative stress modules of SQD syndrome biomolecular network dysfunction in chronic gastritis patients and irritable bowel syndrome patients. Lymphocyte modules were downregulated in chronic fatigue syndrome patients and hypertension patients. From experimental aspects, network pharmacology analysis suggested that targets of Radix Astragali and other four herbs commonly used for SQD syndrome are significantly enriched in the SQD syndrome biomolecular network. Experiments further validated that Radix Astragali ingredients promoted immune modules such as macrophage proliferation and lymphocyte proliferation. These findings indicate that the biological basis of SQD syndrome is closely related to insufficient immune response including decreased macrophage activity and reduced lymphocyte proliferation. This study not only demonstrates the potential biological basis of SQD syndrome but also provides a novel strategy for exploring relevant molecular mechanisms of disease-syndrome-herb from the network pharmacology perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b26358d4dcef0beeee485ada0cb5da8c0811ca52" target='_blank'>Network Pharmacology to Uncover the Biological Basis of Spleen Qi Deficiency Syndrome and Herbal Treatment</a></td>
          <td>
            Xin Wang, Min Wu, Xinxing Lai, Jiahui Zheng, Minghua Hu, Yan Li, Shao Li
          </td>
          <td>2020-08-27</td>
          <td>Oxidative Medicine and Cellular Longevity</td>
          <td>26</td>
          <td>39</td>
        </tr>
    
        <tr id="Abstract Quantitative systems pharmacology (QSP) is a relatively new modelling discipline, formed within the ever-growing domain of model-informed drug development and actively evolving throughout the last decade. This modelling technique is based on the systems analysis and is used to get a quantitative rather than qualitative understanding of systems dynamics and explore the mechanisms of action of a drug. However, there is no well-defined methodology for the QSP model development, which significantly complicates the practical application of these models. In the current work, we overview the existing mathematical models of antidiabetic therapies and propose a modelling method, which overcomes common limitations and is able to produce a physiologically based mechanistic model describing gliflozin action in type 2 diabetes mellitus. From the practical standpoint, sensitivity analysis preformed in this work helped to reveal subpopulation of patients with better response to gliflozin therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5072babf295017ae3c97f3b5cddd82324956b058" target='_blank'>Algorithms and methodological challenges in the development and application of quantitative systems pharmacology models: a case study in type 2 diabetes</a></td>
          <td>
            V. Sokolov
          </td>
          <td>2022-11-01</td>
          <td>Russian Journal of Numerical Analysis and Mathematical Modelling</td>
          <td>1</td>
          <td>7</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6390a7198e07ebd6231e4059b4f39210c8417535" target='_blank'>Quantitative Systems Pharmacology models as a key to translational medicine</a></td>
          <td>
            B. Schoeberl
          </td>
          <td>2019-08-01</td>
          <td>Current Opinion in Systems Biology</td>
          <td>4</td>
          <td>25</td>
        </tr>
    
        <tr id="Basic and Molecular Epidemiology of Gastrointestinal Disorders Research Center, Research Institute for Gastroenterology and Liver Diseases, Shahid Beheshti University of Medical Sciences, Tehran, Iran, Department of Immunology, School of Medicine, Tehran University of Medical Sciences, Tehran, Iran, Gastroenterology and Liver Diseases Research Center, Research Institute for Gastroenterology and Liver Diseases, Shahid Beheshti University of Medical Sciences, Tehran, Iran">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd866dab0990e5f0b322d263bc26e2b2007aba60" target='_blank'>Editorial: Computational methods for multi-omics data analysis in cancer precision medicine</a></td>
          <td>
            Moein Piroozkhah, Radman Mazloomnejad, Zahra Salehi, E. Nazemalhosseini-Mojarad
          </td>
          <td>2023-07-05</td>
          <td>Frontiers in Genetics</td>
          <td>2</td>
          <td>17</td>
        </tr>
    
        <tr id="During acute and chronic infectious diseases hosts develop complex immune responses to cope with bacterial persistence. Depending on a variety of host and microbe factors, outcomes range from peaceful co-existence to detrimental disease. Mechanisms underlying immunity to bacterial stimuli span several spatiotemporal magnitudes and the summation of these hierarchical interactions plays a decisive role in pathogenic versus tolerogenic fate for the host. This dissertation integrates diverse data from immunoinformatics analyses, experimental validation and mathematical modeling to investigate a series of hypotheses driven by computational modeling to study mucosal immunity. Two contrasting microbes, enteroaggregative Escherichia coli and Helicobacter pylori, are used to perturb gut immunity in order to discover host-centric targets for modulating the host immune system. These findings have the potential to be broadly applicable to other infectious and immune-mediated diseases and could assist in the development of antibiotic-free and host-targeted treatments that modulate tolerance to prevent disease. This work was supported in part by the National Institute of Allergy and Infectious Diseases Contract No. HHSN272201000056C to Josep Bassaganya-Riera and funds from the Nutritional Immunology and Molecular Medicine Laboratory. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6533733f66c542d395ec6d41c0b1a59550a2e544" target='_blank'>Systems analysis and characterization of mucosal immunity</a></td>
          <td>
            C. Philipson
          </td>
          <td>2015-07-28</td>
          <td></td>
          <td>0</td>
          <td>19</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>