---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-21 10:42:20 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Computational model of EGFR and IGF1R pathways in lung cancer: a Systems Biology approach for Translational Oncology.</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="e21096 Background: Systems biology together with translational oncology is a new approach to discover sensitive pathways in specific cancers. In this study, we propose a computational modeling technique to investigate the possible effects various alterations, such as protein overexpression, gene amplification or mutations, may have on signaling, through of the EGFR and IGF1R pathways, in non-small cell lung cancer (NSCLC). Methods: EGFR and IGF1R pathways and the downstream MAPK and PIK3 networks have been reproduced through a mathematical model. One hundred-twenty five tumors from surgical NSCLC patients were evaluated for EGFR and IGF1R protein expression, by immunohistochemistry (IHC) and gene amplification, by fluorescence in situ hybridization (FISH). KRAS mutations (exons 2 and 3) were evaluated by direct sequencing Results: To correlate EGFR and IGF1R expression levels, and KRAS mutations to tumor cell proliferation, we focused on the ERK signaling pathway, which plays a central role in several ste...">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8679c2128e1b66f12aa92e65e81a57e1ab5ab00" target='_blank'>Systems biology in personalized medicine in oncology: A case study in non-small cell lung cancer.</a></td>
          <td>
            F. Bianconi, K. Perruccio, V. Ludovini, E. Baldelli, G. Bellezza, A. Flacco, M. Ragusa, P. Valigi, L. Crinò
          </td>
          <td>2012-05-20</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>85</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/679d063f3ab44475bb957757a0b179292e634ea3" target='_blank'>Systems Biology in Translational Oncology: Computational and Experimental Study of EGFR and IGF1R Pathways in NSCLC Cell Lines</a></td>
          <td>
            F. Bianconi, K. Perruccio, V. Ludovini, E. Baldelli, G. Bellezza, A. Flacco, P. Valigi, L. Crinò
          </td>
          <td>2012-09-01</td>
          <td>Annals of Oncology</td>
          <td>0</td>
          <td>85</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a3bacc99c6a54871af3f882420412a90c593d1" target='_blank'>Corrigendum to "Computational model of EGFR and IGF1R pathways in lung cancer: A Systems Biology approach for Translational Oncology" (Biotechnol Adv 30 (2012) 142-153)</a></td>
          <td>
            F. Bianconi, E. Baldelli, V. Ludovini, L. Crinò, A. Flacco, P. Valigi
          </td>
          <td>2013-03-01</td>
          <td>Biotechnology Advances</td>
          <td>2</td>
          <td>85</td>
        </tr>
    
        <tr id="Insulin and insulin-like growth factor I (IGF1) have been shown to influence cancer risk and progression through poorly understood mechanisms. Here, new insights on the mechanisms of differential MAPK and Akt activation are revealed by an iterative quantitative systems pharmacology approach. In the first iteration, I combined proteomic screening with computational network inference to uncover differences in IGF1 and insulin induced signaling. Using reverse phase protein array of 21 breast cancer cell lines treated with a time course of IGF1 and insulin, I constructed directed protein expression networks using three separate methods: (i) lasso regression, (ii) conventional matrix inversion, and (iii) entropy maximization. These networks, named here as the time translation models, were analyzed and the inferred interactions were ranked by differential magnitude to identify pathway differences. The two top candidates, chosen for experimental validation, were shown to regulate IGF1/insulin induced phosphorylation events. Both of the knock-down perturbations caused phosphorylation responses stronger in IGF1 stimulated cells compared with insulin. Overall, the time-translation modeling coupled to wet-lab experiments has proven to be powerful in inferring differential interactions downstream of IGF1 and insulin signaling, in vitro. In the second iteration, mechanistic representation of IGF1 and insulin dual signaling cascades by a set of ODEs is generated by rule-based modeling. The mechanistic network modeling provided a framework to elucidate experimental targets downstream of two receptors, which were treated as indistinguishable in previous models. The model included cascades of both mitogen-activated protein kinase (MAPK) and Akt signaling, as well as the crosstalk and feedback loops in between. The parameter perturbation scanning employed for seven different models of seven cell lines yielded new experimental hypotheses on how differential responses of MAPK and Akt originate. Complementary to the first iteration, the results in this part suggested that regulation of insulin receptor substrate 1 (IRS1) is critical in inducing differential MAPK or Akt activation. Compensation and activating feedback mechanisms collectively depressed the efficacy of anti-IGF1R/InsR therapies. With the quantitative systems pharmacologic approach, the networks of signal transduction constructed in this thesis are aimed to discern novel downstream components of the IGF1R/InsR system, and to direct patients with suitable tumor subclasses to efficient personalized clinical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef89964fed22761b45a25e84a934a11b5aa0b6a7" target='_blank'>EMPLOYING QUANTITATIVE SYSTEMS PHARMACOLOGY TO CHARACTERIZE DIFFERENCES IN IGF1 AND INSULIN SIGNALING PATHWAYS IN BREAST CANCER</a></td>
          <td>
            Cemal Erdem
          </td>
          <td>2018-03-29</td>
          <td></td>
          <td>0</td>
          <td>4</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d79bbb8f35d419b57478958d0df85426b8647ff7" target='_blank'>Cancer modeling: From mechanistic to data-driven approaches, and from fundamental insights to clinical applications</a></td>
          <td>
            Sophie Bekisz, L. Geris
          </td>
          <td>2020-08-12</td>
          <td>J. Comput. Sci.</td>
          <td>46</td>
          <td>43</td>
        </tr>
    
        <tr id="Targeted cancer therapies are powerful alternatives to chemotherapies or can be used complementary to these. Yet, the response to targeted treatments depends on a variety of factors, including mutations and expression levels, and therefore their outcome is difficult to predict. Here, we develop a mechanistic model of gastric cancer to study response and resistance factors for cetuximab treatment. The model captures the EGFR, ERK and AKT signaling pathways in two gastric cancer cell lines with different mutation patterns. We train the model using a comprehensive selection of time and dose response measurements, and provide an assessment of parameter and prediction uncertainties. We demonstrate that the proposed model facilitates the identification of causal differences between the cell lines. Furthermore, our study shows that the model provides accurate predictions for the responses to different perturbations, such as knockdown and knockout experiments. Among other results, the model predicted the effect of MET mutations on cetuximab sensitivity. These predictive capabilities render the model a powerful basis for the assessment of gastric cancer signaling and for the development and discovery of predictive biomarkers. Author Summary Unraveling the causal differences between drug responders and non-responders is an important challenge. The information can help to understand molecular mechanisms and to guide the selection and design of targeted therapies. Here, we approach this problem for cetuximab treatment for gastric cancer using mechanistic mathematical modeling. The proposed model describes multiple gastric cancer cell lines and can accurately predict the response in several validation experiments. Our analysis provides a differentiated view on mutations and explains, for instance, the relevance of MET mutations and the insignificance of PIK3CA mutation in the considered cell lines. The model might provide the basis for understanding the recent failure of several clinical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9c95ce3ab03d0c251ffaf0c848e40145e703fa6" target='_blank'>Model-based analysis of response and resistance factors of cetuximab treatment in gastric cancer cell lines</a></td>
          <td>
            E. Raimúndez, S. Keller, G. Zwingenberger, K. Ebert, S. Hug, Fabian J Theis, D. Maier, B. Luber, J. Hasenauer
          </td>
          <td>2019-05-31</td>
          <td>PLoS Computational Biology</td>
          <td>9</td>
          <td>98</td>
        </tr>
    
        <tr id="Cancer progresses due to changes in the dynamic interactions of multidimensional factors associated with gene mutations. Cancer research has actively adopted computational methods, including data‐driven and mathematical model‐driven approaches, to identify causative factors and regulatory rules that can explain the complexity and diversity of cancers. A data‐driven, statistics‐based approach revealed correlations between gene alterations and clinical outcomes in many types of cancers. A model‐driven mathematical approach has elucidated the dynamic features of cancer networks and identified the mechanisms of drug efficacy and resistance. More recently, machine learning methods have emerged that can be used for mining omics data and classifying patient. However, as the strengths and weaknesses of each method becoming apparent, new analytical tools are emerging to combine and improve the methodologies and maximize their predictive power for classifying cancer subtypes and prognosis. Here, we introduce recent advances in cancer systems biology aimed at personalized medicine, with focus on the receptor tyrosine kinase signaling network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81a4ea8f563e2bfdcc86df72bb7526e1c81b8ae5" target='_blank'>Building patient‐specific models for receptor tyrosine kinase signaling networks</a></td>
          <td>
            Kyoichi Ebata, Sawa Yamashiro, Keita Iida, Mariko Okada
          </td>
          <td>2021-03-23</td>
          <td>The FEBS Journal</td>
          <td>7</td>
          <td>12</td>
        </tr>
    
        <tr id="MicroRNAs have gained significant interest due to their widespread occurrence and diverse functions as regulatory molecules, which are essential for cell division, growth, development and apoptosis in eukaryotes. The epidermal growth factor receptor (EGFR) signaling pathway is one of the best investigated cellular signaling pathways regulating important cellular processes and its deregulation is associated with severe diseases, such as cancer. In this study, we introduce a systems biological model of the EGFR signaling pathway integrating validated miRNA-target information according to diverse studies, in order to demonstrate essential roles of miRNA within this pathway. The model consists of 1241 reactions and contains 241 miRNAs. We analyze the impact of 100 specific miRNA inhibitors (anit-miRNAs) on this pathway and propose that the embedded miRNA-network can help to identify new drug targets of the EGFR signaling pathway and thereby support the development of new therapeutic strategies against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6498d712ff2f4991ed44e3c86cf3f3b3f164ebfa" target='_blank'>Modeling of miRNA and Drug Action in the EGFR Signaling Pathway</a></td>
          <td>
            Jian Li, V. Pandey, T. Kessler, H. Lehrach, C. Wierling
          </td>
          <td>2012-01-11</td>
          <td>PLoS ONE</td>
          <td>13</td>
          <td>124</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22f6a8911d41d6247bf9a4a0523f534fc2a00a1a" target='_blank'>A systems biology view of cancer.</a></td>
          <td>
            R. Laubenbacher, Valerie Hower, A. Jarrah, S. Torti, V. Shulaev, P. Mendes, F. Torti, S. Akman
          </td>
          <td>2009-12-01</td>
          <td>Biochimica et biophysica acta</td>
          <td>103</td>
          <td>75</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec321c10f940fc23c2ce5e932e9b885bd8d56920" target='_blank'>11 From benchside to byteside: what we can learn from computational modelling</a></td>
          <td>
            W. Kolch, M. Birtwistle, B. Kholodenko, J. Grindlay, Oliver E. Sturm, R. Orton, David R. Gilbert, A. Kriegsheim
          </td>
          <td>2010-06-01</td>
          <td>MAG</td>
          <td>0</td>
          <td>86</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f379feef34e99e989c72c542c8d132f15116aa" target='_blank'>Cancer: a Systems Biology disease.</a></td>
          <td>
            J. Hornberg, F. Bruggeman, H. Westerhoff, J. Lankelma
          </td>
          <td>2006-02-01</td>
          <td>Bio Systems</td>
          <td>384</td>
          <td>87</td>
        </tr>
    
        <tr id="Insulin and insulin-like growth factor-1 (IGF1), acting respectively via the insulin (INSR) and IGF1 (IGF1R) receptors, play key developmental and metabolic roles throughout life. In addition, both signaling pathways fulfill important roles in cancer initiation and progression. The present study was aimed at identifying mechanistic differences between INSR and IGF1R using a recently developed bioinformatics tool, the Biological Network Simulator (BioNSi). This application allows to import and merge multiple pathways and interaction information from the KEGG database into a single network representation. The BioNsi network simulation tool allowed us to exploit the availability of gene expression data derived from breast cancer cell lines with specific disruptions of the INSR or IGF1R genes in order to investigate potential differences in protein expression that might be linked to biological attributes of the specific receptor networks. Modeling-generated information was corroborated by experimental and biological assays. BioNSi analyses revealed that the expression of 75 and 71 genes changed during simulation of IGF1R-KD and INSR-KD, compared to control cells, respectively. Out of 16 proteins that BioNSi analysis was based on, validated by Western blotting, nine were shown to be involved in DNA repair, eight in cell cycle checkpoints, six in proliferation, eight in apoptosis, seven in oxidative stress, six in cell migration, two in energy homeostasis, and three in senescence. Taken together, analyses identified a number of commonalities and, most importantly, dissimilarities between the IGF1R and INSR pathways that might help explain the basis for the biological differences between these networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfbdf5ae42849023343e8752bac70680c1cef9a9" target='_blank'>Systems Analysis of Insulin and IGF1 Receptors Networks in Breast Cancer Cells Identifies Commonalities and Divergences in Expression Patterns</a></td>
          <td>
            R. Sarfstein, A. Yeheskel, Tali Sinai-Livne, M. Pasmanik-Chor, H. Werner
          </td>
          <td>2020-07-07</td>
          <td>Frontiers in Endocrinology</td>
          <td>2</td>
          <td>57</td>
        </tr>
    
        <tr id="An error led to part of Figure 2 being removed, breaking the red line that should have linked the grey circles “PKC” and “EGFR”. The correct version of Figure 2 is now displayed on both the PDF and HTML versions of the article.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/892ad05c7eb0020aec314c7a7afee5e1e1dd1b95" target='_blank'>Correction: Integrative Modelling of the Influence of MAPK Network on Cancer Cell Fate Decision</a></td>
          <td>
            L. Grieco, Laurence Calzone, I. Bernard-Pierrot, F. Radvanyi, B. Kahn-Perlès, D. Thieffry
          </td>
          <td>2013-11-01</td>
          <td>PLoS Computational Biology</td>
          <td>41</td>
          <td>61</td>
        </tr>
    
        <tr id="Over the past decade we have seen a shift in cancer therapy from broadly cytotoxic drugs to molecular therapies targeting “driver” mutations. Although targeted therapy has seen great success for some cancers (e.g. imatinib for leukemia), it has struggled with poor efficacy in treating other cancers that can sometimes possess multiple “driver” mutations. This highlights the complex, and at times non-intuitive, interplay between multiple players in a signaling cascade, which can be highly dependent on the biological context – that is, gene expression levels and mutational architecture – of a tumor or cell line. A quantitative, mechanistic, biologically-tailored understanding of how these signaling dynamics drive proliferation and death could improve precision pharmacology approaches to treat cancer. Here, we constructed the first highly detailed, large-scale ordinary differential equation (ODE) mechanistic mathematical model depicting the most commonly mutated cancer signaling pathways across human cancers, as indicated by a pan-cancer analysis by The Cancer Genome Atlas (TCGA). The model includes the RTK/Ras/MAPK, PI3K/AKT/mTOR, CDK/RB cell cycle, p53/MDM2 DNA damage response, and BCL/Caspases apoptosis pathways. The adjustable parameters of the model can be informed by measurements from patients or cell lines, including copy number alterations, mutations, and gene expression levels. This single-cell model links stochastic gene expression processes to quantitative signaling dynamics, and once tailored to a biological context can be used to simulate the effect of various anti-cancer therapies on cell fate behavior such as proliferation and death for a population of cells. The first instance of the model integrated genomic, transcriptomic, and proteomic data from the MCF10A cell line, a non-transformed cell line with predictable phenotypic behaviors. We trained the model using western blot and flow cytometry experiments to refine various biochemical parameters and phenotypic outcomes. Many fundamental questions in signal transduction arose during this process, such as how EGF and insulin synergize to drive S-phase entry or how a specific biological context confers sensitivity or resistance to inhibitors of the ERK and AKT pathways. Simultaneously, we are tailoring the model to patient-derived genetic information from primary glioblastoma tumors and screening brain-penetrable compounds in a patient-specific manner. In conclusion, a quantitative, mechanistic, biologically-tailored mathematical model depicting the major cancer pathways allows us to probe the mechanisms that underlie how signaling dynamics drive proliferation and death in response to various perturbations, and gain insight into their dependence on the biological context of cell lines and patient tumors. Note: This abstract was not presented at the meeting. Citation Format: Mehdi Bouhaddou, Anne Marie Barrette, Rick J. Koch, Marc R. Birtwistle. Predicting stochastic proliferation and death in response to drugs with mechanistic models tailored to genomic, transcriptomic, and proteomic data [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2017; 2017 Apr 1-5; Washington, DC. Philadelphia (PA): AACR; Cancer Res 2017;77(13 Suppl):Abstract nr 1568. doi:10.1158/1538-7445.AM2017-1568">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b9460f53b1393fa834eeb46b9c94177968a96a" target='_blank'>Abstract 1568: Predicting stochastic proliferation and death in response to drugs with mechanistic models tailored to genomic, transcriptomic, and proteomic data</a></td>
          <td>
            M. Bouhaddou, A. Barrette, R. J. Koch, M. Birtwistle
          </td>
          <td>2017-07-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>
    
        <tr id="Purpose of review Significant insight can be gained into complex biologic mechanisms of cancer via a combined computational and experimental systems biology approach. This review highlights some of the major systems biology efforts that were applied to cancer in the past year. Recent findings Two main approaches to computational systems biology are discussed: mechanistic dynamical simulations and inferential data mining. Significant developments have occurred in both areas. For example, mechanistic simulations of the EGFR pathway are promoting understanding of cancer, and Bayesian inference approaches allow for the reconstruction of regulatory networks. In addition, the article reports on advancements in experimental systems biology for determining protein-protein interactions and quantifying protein expression to generate the necessary data for computational modeling and inferential data mining. Emerging approaches will further improve the ability to bridge the gap between in vitro systems and in vivo human biology. Technologies paving the way include in vitro models that better reflect in vivo tumors, microfabricated devices of human physiology, and improved animal models. Summary An important challenge facing the field is how better to translate in vitro discoveries to the clinic. Computational systems biology approaches that use omic data to predict biology along with novel experimental systems that better represent human in vivo biology will prove useful in bridging this gap. Although still early, the potential application of systems biology and the future evolution of the field will significantly affect understanding of cancer disease mechanisms and the ability to devise effective therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12264090cd2e2f6934ed20e099ae6b87f38d6484" target='_blank'>Systems biology for cancer</a></td>
          <td>
            I. Khalil, C. Hill
          </td>
          <td>2005-01-01</td>
          <td>Current Opinion in Oncology</td>
          <td>51</td>
          <td>6</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dd8c0e59cc6f86078f2c60ed2d73016f4951d76" target='_blank'>Computational modelling of cancerous mutations in the EGFR/ERK signalling pathway</a></td>
          <td>
            
          </td>
          <td>2009-10-05</td>
          <td>BMC Systems Biology</td>
          <td>64</td>
          <td>0</td>
        </tr>
    
        <tr id="Receptor tyrosine kinases (RTKs) are high-affinity cell surface receptors for growth factors that are frequently deregulated in cancer. Signaling through these receptors has been associated with increased cancer cell proliferation and resistance to cytotoxic therapies. To block this detrimental signaling, many companies are developing inhibitory antibodies against various RTKs. A key challenge in clinical studies is the optimal stratification of patients who may benefit from these therapies. For an RTK targeted antibody, the detection of the respective growth factor in the tumor microenvironment may be an important bio-marker. Beyond the physical presence of the growth factor, the decision whether a cancer cell will respond to growth factor-induced signals is governed by complex intra-cellular signaling networks. We compared different approaches to predict cellular responses and will highlight a hybrid approach that combines mechanistic modeling based on ordinary differential equations with a machine learning algorithm. The models are trained on in vitro drug response screens and then applied to predict response in patient samples. The mechanistic models are trained on quantitative data from signal transduction studies as well as RNAseq data for cellular characterization. Using the hybrid approach, a correlation between growth factor expression in the tumor microenvironment and its predicted response was identified. This supports the hypothesis of addiction of tumors to growth factors abundant in the tumor microenvironment, and might enable more robust patient stratification in the future. Citation Format: Helge Hass, Kristina Masson, Sibylle Wohlgemuth, Violette Paragas, John E. Allen, Mark Sevecka, Emily Pace, Jens Timmer, Joerg Stelling, Gavin MacBeath, Birgit Schoeberl, Andreas Raue. Predicting ligand-dependent tumors from multi-dimensional signaling features [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2018; 2018 Apr 14-18; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2018;78(13 Suppl):Abstract nr 1312.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bca6a59dae02c0c3a5fa53c09131e688a0c1325" target='_blank'>Abstract 1312: Predicting ligand-dependent tumors from multi-dimensional signaling features</a></td>
          <td>
            Helge Hass, Kristina Masson, Sibylle Wohlgemuth, V. Paragas, John E. Allen, M. Sevecka, E. Pace, Jens Timmer, Joerg Stelling, G. MacBeath, B. Schoeberl, A. Raue
          </td>
          <td>2018-07-01</td>
          <td>Bioinformatics and Systems Biology</td>
          <td>0</td>
          <td>40</td>
        </tr>
    
        <tr id="Drawing on concepts from experimental biology, computer science, informatics, mathematics, and statistics, systems biologists integrate data across diverse platforms and scales of time and space to create computational and mathematical models of the integrative, holistic functions of living systems. Endocrine-related cancers are well suited to study from a systems perspective because of the signaling complexities arising from the roles of growth factors, hormones, and their receptors as critical regulators of cancer cell biology and from the interactions among cancer cells, normal cells, and signaling molecules in the tumor microenvironment. Moreover, growth factors, hormones, and their receptors are often effective targets for therapeutic intervention, such as estrogen biosynthesis, estrogen receptors, or HER2 in breast cancer, and androgen receptors in prostate cancer. Given the complexity underlying the molecular control networks in these cancers, a simple, intuitive understanding of how endocrine-related cancers respond to therapeutic protocols has proved incomplete and unsatisfactory. Systems biology offers an alternative paradigm for understanding these cancers and their treatment. To correctly interpret the results of systems-based studies requires some knowledge of how in silico models are built, and how they are used to describe a system and to predict the effects of perturbations on system function. In this review, we provide a general perspective on the field of cancer systems biology, and we explore some of the advantages, limitations, and pitfalls associated with using predictive multiscale modeling to study endocrine-related cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf192696377134863f44821ea66938672e87510b" target='_blank'>Systems biology: perspectives on multiscale modeling in research on endocrine-related cancers.</a></td>
          <td>
            R. Clarke, J. Tyson, M. Tan, W. Baumann, Lu Jin, J. Xuan, Yue Wang
          </td>
          <td>2019-06-01</td>
          <td>Endocrine-related cancer</td>
          <td>13</td>
          <td>67</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6319b79882fa8bd07267c8cafc082740cd9779b6" target='_blank'>Computational modeling of the EGFR network elucidates control mechanisms regulating signal dynamics</a></td>
          <td>
            Dennis Wang, L. Cardelli, Andrew Phillips, Nir Piterman, J. Fisher
          </td>
          <td>2009-12-22</td>
          <td>BMC Systems Biology</td>
          <td>40</td>
          <td>74</td>
        </tr>
    
        <tr id="Mathematical modeling is a key process in Systems Biology and the use of computational tools such as Cytoscape for omics data processing, need to be integrated in the modeling activity. In this paper we propose a new methodology for modeling signaling networks by combining ordinary differential equation models and a gene recommender system, GeneMANIA. We started from existing models, that are stored in the BioModels database, and we generated a query to use as input for the GeneMANIA algorithm. The output of the recommender system was then led back to the kinetic reactions that were finally added to the starting model. We applied the proposed methodology to EGFR-IGF1R signal transduction network, which plays an important role in translational oncology and cancer therapy of non small cell lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58a20a83d17961a6e4a510e9726e000bcd9b2619" target='_blank'>An approach for optimally extending mathematical models of signaling networks using omics data</a></td>
          <td>
            F. Bianconi, Federico Patiti, E. Baldelli, L. Crinò, P. Valigi
          </td>
          <td>2015-11-05</td>
          <td>2015 37th Annual International Conference of the IEEE Engineering in Medicine and Biology Society (EMBC)</td>
          <td>1</td>
          <td>85</td>
        </tr>
    
        <tr id="Despite many evidences supporting the concept of “oncogene addiction” and many hypotheses rationalizing it, there is still a lack of detailed understanding to the precise molecular mechanism underlying oncogene addiction. In this account, we developed a mathematic model of epidermal growth factor receptor (EGFR) associated signaling network, which involves EGFR-driving proliferation/pro-survival signaling pathways Ras/extracellular-signal-regulated kinase (ERK) and phosphoinositol-3 kinase (PI3K)/AKT, and pro-apoptotic signaling pathway apoptosis signal-regulating kinase 1 (ASK1)/p38. In the setting of sustained EGFR activation, the simulation results show a persistent high level of proliferation/pro-survival effectors phospho-ERK and phospho-AKT, and a basal level of pro-apoptotic effector phospho-p38. The potential of p38 activation (apoptotic potential) due to the elevated level of reactive oxygen species (ROS) is largely suppressed by the negative crosstalk between PI3K/AKT and ASK1/p38 pathways. Upon acute EGFR inactivation, the survival signals decay rapidly, followed by a fast increase of the apoptotic signal due to the release of apoptotic potential. Overall, our systems biology modeling together with experimental validations reveals that inhibition of survival signals and concomitant release of apoptotic potential jointly contribute to the tumor cell death following the inhibition of addicted oncogene in EGFR addicted cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec2cb47b81031125cdc9974d23a6e32694ec2d97" target='_blank'>Systems Biology Modeling Reveals a Possible Mechanism of the Tumor Cell Death upon Oncogene Inactivation in EGFR Addicted Cancers</a></td>
          <td>
            Jian-Ping Zhou, Xin Chen, Shan Feng, Shi-Dong Luo, You-Li Pan, L. Zhong, P. Ji, Zerong Wang, Shuang-Jiang Ma, Linli Li, Yuquan Wei, Sheng-yong Yang
          </td>
          <td>2011-12-14</td>
          <td>PLoS ONE</td>
          <td>11</td>
          <td>42</td>
        </tr>
    
        <tr id="Cancer has been a paradigmatic example of a complex disease. It is the second leading cause of death worldwide. Knowledge acquired on the mechanisms and potential causes underlying the origins and progression of cancer from single-scale studies have led to the development of personalized treatments, which in turn have decreasedmortality rates in several types of cancer. However, because it is not a single and isolated disease, but rather a heterogeneous set of multiscale alterations, current research requires computational and systems biology approaches that deal with complex systems on gene regulatory networks. Multi-omics is one of these approaches. Aimed at developing multi-scale mathematical modeling, it allows us to integrate the dynamics of biological perturbations for different types of cancer. Moreover, if tissue/organ-specific path is studied we are able to create mechanistic models for a particular type of the disease. That, combined with current low-cost high throughput technologies have made possible massive cohorts of -omics and clinical data to be publicly available. This immeasurable amount of information promotes harnessing data science approaches for the analysis, integration, and mining of multi-omics data. Large and multidisciplinary groups can now focus on creating models and provide theoretical frameworks that describe cancer in a variety of ways. Besides cancer cell transformation or cancer evolution, currently single cell genomics and spatial single cell transcriptomics is moving forward with deeper understanding of the disease. Also, cancer driver mutations and mutational processes, development of targetspecific treatments have been leading to a prediction of drug responses. Notwithstanding, several challenges that remain to be undertaken such asmechanisms ofmetastasis, resistance to treatment, intra-tumoral heterogeneity, molecular, cellular, and metabolic changes during progression stages, epigenetic modifications, to mention a few. This cross-disciplinary interaction at these levels of description have contributed to develop more efficient models that improve the predictive capacity and ultimately help clinicians and medical scientists in the treatment and therapies. OPEN ACCESS">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18a5c9568422971ebe3df8c55afef2906c57168e" target='_blank'>Editorial: Cancer systems biology</a></td>
          <td>
            J. Espinal-Enríquez, C. Rangel-Escareño, F. Emmert-Streib
          </td>
          <td>2022-11-18</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>41</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41b8753d78a9f7180d251c2057dfc04eded8d0a8" target='_blank'>L7.2 Potential Contributions of Systems Biology Modeling and Simulation for Cancer Phenomenology Understanding</a></td>
          <td>
            P. Johnson, K. Schuerer, M. Lopes
          </td>
          <td>2012-06-01</td>
          <td>Annals of Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ce2d9991d11077a69e456a1b3c9243e4f3a19ad" target='_blank'>Applications of sensitivity analysis for drug discovery and development in the ErbB receptor network</a></td>
          <td>
            B. Harms, Allen Lee, Ricardo Paxson, U. Nielsen, B. Schoeberl
          </td>
          <td>2007-05-08</td>
          <td>BMC Systems Biology</td>
          <td>0</td>
          <td>29</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b17eff14eed0d23295b1d0eab1dcd343cb4b6d7" target='_blank'>Multi-Network Modeling of Cancer Cell States</a></td>
          <td>
            M. Blinov, A. Udyavar, W. Yarbrough, Jialiang Wang, L. Estrada, V. Quaranta
          </td>
          <td>2012-01-31</td>
          <td>Biophysical Journal</td>
          <td>2</td>
          <td>70</td>
        </tr>
    
        <tr id="In recent years it has become clear that carcinogenesis is a complex process, both at the molecular and cellular levels. Understanding the origins, growth and spread of cancer, therefore requires an integrated or system-wide approach. Computational systems biology is an emerging sub-discipline in systems biology that utilizes the wealth of data from genomic, proteomic and metabolomic studies to build computer simulations of intra and intercellular processes. Several useful descriptive and predictive models of the origin, growth and spread of cancers have been developed in an effort to better understand the disease and potential therapeutic approaches. In this review we describe and assess the practical and theoretical underpinnings of commonly-used modeling approaches, including ordinary and partial differential equations, petri nets, cellular automata, agent based models and hybrid systems. A number of computer-based formalisms have been implemented to improve the accessibility of the various approaches to researchers whose primary interest lies outside of model development. We discuss several of these and describe how they have led to novel insights into tumor genesis, growth, apoptosis, vascularization and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0011f20e735fc6221c2006aab163d2383185fd2" target='_blank'>Computational systems biology in cancer: modeling methods and applications.</a></td>
          <td>
            W. Materi, David S Wishart, David S Wishart
          </td>
          <td>2007-05-01</td>
          <td>Gene regulation and systems biology</td>
          <td>184</td>
          <td>7</td>
        </tr>
    
        <tr id="In this work high throughput technology and computational analysis were used to study two stage IV lung adenocarcinoma patients treated with standard chemotherapy with markedly different survival (128 months vs 6 months, respectively) and whose tumor samples exhibit a dissimilar protein activation pattern of the signal transduction. Tumor samples of the two patients were subjected to Reverse Phase Protein Microarray (RPPA) analysis to explore the expression/activation levels of 51 signaling proteins. We selected the most divergent proteins based on the ratio of their RPPA values in the two patients with short (s-OS) and long (l-OS) overall survival (OS) and tested them against a EGFR-IGF1R mathematical model. The model with RPPA data showed that the activation levels of 19 proteins were different in the two patients. The four proteins that most distinguished the two patients were BADS155/136 and c-KITY703/719 having a higher activation level in the patient with short survival and p70S6S371/T389 and b-RAFS445 that had a lower activation level in the s-OS patient. The final model describes the interactions between the MAPK and PI3K-mTOR pathways, including 21 nodes. According to our model mTOR and ERK activation levels were predicted to be lower in the s-OS patient than the l-OS patient, while the AMPK activation level was higher in the s-OS patient. Moreover, KRAS activation was predicted to be higher in the l-OS KRAS-mutated patient. In accordance with their different biological properties, the Moment Independent Robustness Indicator in s-OS and l-OS predicted the interaction of MAPK and mTOR and the crosstalk AKT with p90RSK as candidates to be prognostic factors and drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70e98ae04fb971eb0960ddde99b8d179b7f1b1a4" target='_blank'>Reverse phase protein array (RPPA) combined with computational analysis to unravel relevant prognostic factors in non- small cell lung cancer (NSCLC): a pilot study</a></td>
          <td>
            V. Ludovini, R. Chiari, Lorenzo Tomassoni, C. Antonini, E. Baldelli, S. Baglivo, A. Siggillino, F. Tofanetti, G. Bellezza, K. A. Hodge, E. Petricoin, M. Pierobon, L. Crinò, F. Bianconi
          </td>
          <td>2017-06-14</td>
          <td>Oncotarget</td>
          <td>1</td>
          <td>92</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c838b14687f4277a581731712f06b718448e6d4" target='_blank'>Cancer systems biology and modeling: microscopic scale and multiscale approaches.</a></td>
          <td>
            A. Masoudi-Nejad, G. Bidkhori, Saman Hosseini Ashtiani, Ali Najafi, J. H. Bozorgmehr, Edwin Wang
          </td>
          <td>2015-02-01</td>
          <td>Seminars in cancer biology</td>
          <td>6</td>
          <td>36</td>
        </tr>
    
        <tr id="Two studies show the power of systems biology methods to improve cancer drug development and treatment strategy. The identification of “driver mutations” in cancers initiated rapid development of targeted drugs for the clinic. Despite promising outcomes initially in patients, the ultimate success of oncogene-targeted drugs has been hampered by the redundancy and remarkable complexity of cellular signaling pathways. Two studies in Science Signaling show that understanding these intricate networks and considering them during tumor classification and drug design can better predict drug response. These studies exemplify the potential of using systems analysis and computational modeling approaches to improve therapeutic strategies and outcomes in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/222e5a136d1e41e7af9834e0988f5c8ac6e9f630" target='_blank'>What Lies Beneath: Looking Beyond Tumor Genetics Shows the Complexity of Signaling Networks Underlying Drug Sensitivity</a></td>
          <td>
            V. Quaranta, D. Tyson
          </td>
          <td>2013-09-24</td>
          <td>Science Signaling</td>
          <td>12</td>
          <td>70</td>
        </tr>
    
        <tr id="In this review we make the statement that hybrid models in oncology are required as a mean for enhanced data integration. In the context of systems oncology, experimental and clinical data need to be at the heart of the models developments from conception to validation to ensure a relevant use of the models in the clinical context. The main applications pursued are to improve diagnosis and to optimize therapies.We first present the Successes achieved thanks to hybrid modelling approaches to advance knowledge, treatments or drug discovery. Then we present the Challenges than need to be addressed to allow for a better integration of the model parts and of the data into the models. And Finally, the Hopes with a focus towards making personalised medicine a reality. Mathematics Subject Classification. 35Q92, 68U20, 68T05, 92-08, 92B05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c63c58cf385c3a4d184205dfce839d6f40ea2ad5" target='_blank'>Hybrid Modelling in Oncology: Sucesses, Challenges and Hopes</a></td>
          <td>
            Ang'elique St'ephanou, P. Ballet, Gibin G. Powathil
          </td>
          <td>2019-01-17</td>
          <td>arXiv: Quantitative Methods</td>
          <td>1</td>
          <td>16</td>
        </tr>
    
        <tr id="Systems biology is a multidisciplinary methodology in which quantitative biological experimental data are dissected using mathematical modelling and other computational and network biology tools, aiming at understanding the structure, function and dynamical regulation of biochemical networks. Systems biology will play a major role in the future molecular and clinical oncology because (a) it can be used for the analysis of cancer-relevant high-throughput data, (b) it provides tools for the reconstruction of the large multilevel regulatory networks that govern critical cancer phenotypes and (c) it is necessary when investigating cancer-relevant networks holding multiple overlapping nonlinear regulatory motifs like feedback and feedforward loops. Here, the value of the systems biology approach in handling cancer genomics and transcriptomics data, the reconstruction of cancer networks and the use of mathematical modelling in the elucidation of cancer networks as well as in the design of anticancer therapies are discusses. Some recent case studies as proof of principle are highlighted. 
 
 
 
Key Concepts: 
 
 
 
Systems biology is a multidisciplinary approach that involves the analysis of quantitative experimental data via mathematical modelling and computational and network biology. 
 
 
 
 
Modelling of cancer-related networks is necessary because these networks display multiple nonlinear regulatory motifs like feedback loops. 
 
 
 
 
Mathematical models have been used to elucidate the structure, dynamics, dysregulation and crosstalk of cancer pathways like the JAK/STAT, p53, MAPK, NFkB and intrinsinc/extrinsic apoptosis signalling pathways, as well as in microRNA cancer regulation. 
 
 
 
 
Multi-scale mathematical models can accurately describe the spatial configuration of tumours, their crosstalk with the surrounding microenvironment and their role in physiological mechanisms like angiogenesis and immune system response. 
 
 
 
 
Mathematical modelling has been used to assess and personalise conventional anticancer therapies, but it is also used in the detection of new anticancer drug targets and the development of combined or immune therapies. 
 
 
 
 
Cancer bioinformatics facilitates efficient analyses and integration of diverse types of biomedical high-throughput data for the elucidation of causes of tumour initiation, progression and metastasis but also for the management, storage and exchange of experimental and clinical data. 
 
 
 
 
Cancer systems biology integrates algorithms, methodologies, software tools and web resources into customised analysis workflows to explore and understand the molecular mechanisms underlying this complex disease. 
 
 
 
 
Data integration and analysis is used to identify novel robust and precise biomarkers, to identify network biomarkers and to design individual patient-tailored therapeutic interventions. 
 
 
 
 
 
 
Keywords: 
 
anticancer therapies; 
biochemical regulatory networks; 
feedback and feedforward loops; 
drug target detection; 
multi-scale cancer modelling; 
miRNA regulation; 
cancer genomics; 
cancer transcriptomics; 
patient stratification">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c803d03517c3c4e7fd228d897400800d00bca0f" target='_blank'>Cancer Systems Biology: Current Achievements in – Omics Data Analysis, Network Reconstruction and Mathematical Modelling</a></td>
          <td>
            J. Vera, O. Wolkenhauer, U. Schmitz
          </td>
          <td>2014-08-15</td>
          <td></td>
          <td>1</td>
          <td>52</td>
        </tr>
    
        <tr id="Cancer is a complex disease that involves multiple types of biological interactions across diverse physical, temporal, and biological scales. This complexity presents substantial challenges for the characterization of cancer biology, and motivates the study of cancer in the context of molecular, cellular, and physiological systems. Computational models of cancer are being developed to aid both biological discovery and clinical medicine. The development of these in silico models is facilitated by rapidly advancing experimental and analytical tools that generate information‐rich, high‐throughput biological data. Statistical models of cancer at the genomic, transcriptomic, and pathway levels have proven effective in developing diagnostic and prognostic molecular signatures, as well as in identifying perturbed pathways. Statistically inferred network models can prove useful in settings where data overfitting can be avoided, and provide an important means for biological discovery. Mechanistically based signaling and metabolic models that apply a priori knowledge of biochemical processes derived from experiments can also be reconstructed where data are available, and can provide insight and predictive ability regarding the behavior of these systems. At longer length scales, continuum and agent‐based models of the tumor microenvironment and other tissue‐level interactions enable modeling of cancer cell populations and tumor progression. Even though cancer has been among the most‐studied human diseases using systems approaches, significant challenges remain before the enormous potential of in silico cancer biology can be fully realized. Copyright © 2009 John Wiley & Sons, Inc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b61b75dbe460d375a6102db8ce287d8cd4dadb6d" target='_blank'>In silico models of cancer</a></td>
          <td>
            Lucas B. Edelman, James A. Eddy, N. Price
          </td>
          <td>2010-07-01</td>
          <td>Wiley Interdisciplinary Reviews: Systems Biology and Medicine</td>
          <td>122</td>
          <td>68</td>
        </tr>
    
        <tr id="Background Breast cancer (BC) is one of the leading cause of death among females worldwide. The increasing incidence of BC is due to various genetic and environmental changes which lead to the disruption of cellular signaling network(s). It is a complex disease in which several interlinking signaling cascades play a crucial role in establishing a complex regulatory network. The logical modeling approach of René Thomas has been applied to analyze the behavior of estrogen receptor-alpha (ER-α) associated Biological Regulatory Network (BRN) for a small part of complex events that leads to BC metastasis. Methods A discrete model was constructed using the kinetic logic formalism and its set of logical parameters were obtained using the model checking technique implemented in the SMBioNet software which is consistent with biological observations. The discrete model was further enriched with continuous dynamics by converting it into an equivalent Petri Net (PN) to analyze the logical parameters of the involved entities. Results In-silico based discrete and continuous modeling of ER-α associated signaling network involved in BC provides information about behaviors and gene-gene interaction in detail. The dynamics of discrete model revealed, imperative behaviors represented as cyclic paths and trajectories leading to pathogenic states such as metastasis. Results suggest that the increased expressions of receptors ER-α, IGF-1R and EGFR slow down the activity of tumor suppressor genes (TSGs) such as BRCA1, p53 and Mdm2 which can lead to metastasis. Therefore, IGF-1R and EGFR are considered as important inhibitory targets to control the metastasis in BC. Conclusion The in-silico approaches allow us to increase our understanding of the functional properties of living organisms. It opens new avenues of investigations of multiple inhibitory targets (ER-α, IGF-1R and EGFR) for wet lab experiments as well as provided valuable insights in the treatment of cancers such as BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d34a33a696e40053b08331930e2dcfb646fa0cd" target='_blank'>Formal modeling and analysis of ER-α associated Biological Regulatory Network in breast cancer</a></td>
          <td>
            Samra Khalid, Rumeza Hanif, Samar H. K. Tareen, Amnah Siddiqa, Zurah Bibi, Jamil Ahmad
          </td>
          <td>2016-10-20</td>
          <td>PeerJ</td>
          <td>7</td>
          <td>17</td>
        </tr>
    
        <tr id="Clinical Cancer models need to incorporate a wide variety of patient data and tumor heterogeneity which requires integration of multiple models. Due to differences in time and length scales of individual processes, such a model integration is a challenging task. Here we have developed an integrated framework combining ErbB receptor mediated Ras-MAPK and PI3K/AKT pathway with p53 mediated DNA damage response pathway. We have applied this in a clinical setting to predict patient specific response of different treatments in cancers of prostate, lung and kidney.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b10cc136d8a75dec9727530bae483c35df9f28c" target='_blank'>Heterogeneous multi-scale framework for cancer systems models and clinical applications</a></td>
          <td>
            Alokendra Ghosh, R. Radhakrishnan
          </td>
          <td>2019-05-16</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>40</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfbf73b4d67dd34c855cf8e556071b7e092bb704" target='_blank'>Input–output behavior of ErbB signaling pathways as revealed by a mass action model trained against dynamic data</a></td>
          <td>
            William W. Chen, B. Schoeberl, P. Jasper, M. Niepel, U. Nielsen, D. Lauffenburger, P. Sorger
          </td>
          <td>2009-01-20</td>
          <td>Molecular Systems Biology</td>
          <td>429</td>
          <td>119</td>
        </tr>
    
        <tr id="In the past few years, systems biology methods have been widely used to explore the mechanism of cancer. Various signaling pathways play the important role in cancer. And a large number of computational methods are used to model the signaling pathway. Here, we reviewed many existing mechanisms and modeling techniques, which are used to model different signaling pathways. Then we introduced the systematic workflow of a system biology approach and some fundamental analysis methods. Finally we discuss some known challenges in the process of modeling signaling pathways and what we should handle in our future work.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05a9ab7b8011464e527bd7731032008ad51b69be" target='_blank'>Review of the computational models for the signaling pathway</a></td>
          <td>
            Meijing Kong, Chao Wang, Le Zhang
          </td>
          <td>2016-05-21</td>
          <td></td>
          <td>0</td>
          <td>4</td>
        </tr>
    
        <tr id="Cell signaling pathways control cells' responses to their environment through an intricate network of proteins and small molecules partitioned by intracellular structures, such as the cytoskeleton and nucleus. Our understanding of these pathways has been revised recently with the advent of more advanced experimental techniques; no longer are signaling pathways viewed as linear cascades of information flowing from membrane-bound receptors to the nucleus. Instead, such pathways must be understood in the context of networks, and studying such networks requires an integration of computational and experimental approaches. This understanding is becoming more important in designing novel therapies for diseases such as cancer. Using the MAPK (mitogen-activated protein kinase) and PI3K (class I phosphoinositide-3' kinase) pathways as case studies of cellular signaling, we give an overview of these pathways and their functions. We then describe, using a number of case studies, how computational modeling has aided in understanding these pathways' deregulation in cancer, and how such understanding can be used to optimally tailor current therapies or help design new therapies against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/653dd939407ee3714dc5a23c5389f2d840b8639e" target='_blank'>Modeling Signaling Networks to Advance New Cancer Therapies.</a></td>
          <td>
            J. Saez-Rodriguez, Aidan MacNamara, S. Cook
          </td>
          <td>2015-12-07</td>
          <td>Annual review of biomedical engineering</td>
          <td>31</td>
          <td>59</td>
        </tr>
    
        <tr id="Motivation: Gene expression measurements (microarray of RNA-seq) are affordable ways to survey cell activity. However, they constitute low-informative, decontextualized values that often lack a mechanistic link with real cell functional outcomes. Mathematical modeling of biological pathways is emerging as a useful tool to understand the molecular mechanisms that govern the cell behavior or fate, revealing disease mechanisms and drug mechanisms of action, and providing guidance on therapeutic decisions (Gustafsson et al., 2014). Methods: Signaling KEGG pathways are used as basic maps of cell functionality over which gene expression values are modeled to obtain probabilities of signal transduction and, consequently, cell function activations (Hidalgo et al, 2017). Results: Here we propose a new method that models cell signaling using biological knowledge on signal transduction (Hidalgo et al., 2017). The method recodes individual gene expression values (and/or gene mutations) into accurate measurements of changes in the activity of signaling circuits, which ultimately constitute high-throughput estimations of cell functionalities caused by gene activity within the pathway. Signaling circuit activities can predict cancer outcome and have also demonstrated to be excellent predictors of drug response (Amadoz et al., 2015) Conclusions: A comprehensive, systems-based understanding of the way in which genes interact to shape the phenotype is required to realistically manage complex diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c190c1cf23ea278e01ad05d0b33bcd2bba4058ce" target='_blank'>Systems medicine approach to model cell signaling activity uncovers disease mechanisms and predicts cancer outcome</a></td>
          <td>
            J. Dopazo
          </td>
          <td>2018-02-28</td>
          <td></td>
          <td>0</td>
          <td>78</td>
        </tr>
    
        <tr id="In this paper, we outline the status quo and approaches to further development of the systems biology concepts with focus on applications in cancer prevention science. We discuss the biological aspects of cancer research that are of primary importance in cancer prevention, motivations for their mathematical modeling and some recent advances in computational oncology. We also make an attempt to outline in big conceptual terms the contours of future work aimed at creation of large-scale computational and informational infrastructure for using as a routine tool in cancer prevention science and decision making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7fb4f02876405ebc06c15068cdaec8eb6e936b1" target='_blank'>Systems Biology and Cancer Prevention: All Options on the Table</a></td>
          <td>
            S. Rosenfeld, I. Kapetanovic
          </td>
          <td>2008-10-10</td>
          <td>Gene Regulation and Systems Biology</td>
          <td>13</td>
          <td>30</td>
        </tr>
    
        <tr id="With over 200 types of cancer diagnosed to date, researchers the world over have been forced to rapidly update their understanding of the biology of cancer. In fact, only the study of the basic cellular processes, and how these are altered in cancer cells, can ultimately provide a background for rational therapies. Bringing together the state-of-the-art contributions of international experts, Systems Biology of Cancer proposes an ultimate research goal for the whole scientific community: exploiting systems biology to generate in-depth knowledge based on blueprints that are unique to each type of cancer. Readers are provided with a realistic view of what is known and what is yet to be uncovered on the aberrations in the fundamental biological processes, deregulation of major signaling networks, alterations in major cancers and the strategies for using the scientific knowledge for effective diagnosis, prognosis and drug discovery to improve public health.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3642b7d09aea6a4000f02689a883d6e3c9f01b93" target='_blank'>Systems biology of cancer</a></td>
          <td>
            S. Thiagalingam
          </td>
          <td>2015-04-01</td>
          <td></td>
          <td>20</td>
          <td>34</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>