---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-29 08:54:23 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Optimal enumeration of state space of finitely buffered stochastic molecular networks and exact computation of steady state landscape probability</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Stochasticity plays central role in molecular networks of small copy numbers, including those important in protein synthesis and gene regulation. The combination of copy numbers of molecular species defines the microscopic state of molecular interactions. With this formulation, nonlinear reactions can be effectively modeled through chemical master equations. However, currently little is known about the state space associated with stochastic networks, other than the defeatist admission that it is exponentially large. There is neither closed-form solution nor computational algorithm that can effectively characterize the state space of molecular networks. Such a characterization is a prerequisite for directly solving the chemical master equation. In this study, we describe an algorithm that can exhaustively characterize all possible states of a molecular networks with small copy numbers of species for a given initial condition. Our algorithm works for networks of arbitrary stoichiometry, and is optimal in both storage and time complexity. It allows the approach of solving chemical master equation to be applicable to a larger class of stochastic molecular networks. We show an example of application of our method to the MAPK cascade network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44e21eb6bc9ea710777238baa4da307e589cfa27" target='_blank'>An optimal algorithm for enumerating state space of stochastic molecular networks with small copy numbers of molecules</a></td>
          <td>
            Youfang Cao, Jie Liang
          </td>
          <td>2007-10-22</td>
          <td>2007 29th Annual International Conference of the IEEE Engineering in Medicine and Biology Society</td>
          <td>1</td>
          <td>40</td>
        </tr>
    
        <tr id="The discrete chemical master equation (dCME) provides a fundamental framework for studying stochasticity in mesoscopic networks. Because of the multi-scale nature of many networks where reaction rates have large disparity, directly solving dCMEs is intractable due to the exploding size of the state space. It is important to truncate the state space effectively with quantified errors, so accurate solutions can be computed. It is also important to know if all major probabilistic peaks have been computed. Here we introduce the Accurate CME (ACME) algorithm for obtaining direct solutions to dCMEs. With multi-finite buffers for reducing the state space by O(n!), exact steady-state and time-evolving network probability landscapes can be computed. We further describe a theoretical framework of aggregating microstates into a smaller number of macrostates by decomposing a network into independent aggregated birth and death processes, and give an a priori method for rapidly determining steady-state truncation errors. The maximal sizes of the finite buffers for a given error tolerance can also be pre-computed without costly trial solutions of dCMEs. We show exactly computed probability landscapes of three multi-scale networks, namely, a 6-node toggle switch, 11-node phage-lambda epigenetic circuit, and 16-node MAPK cascade network, the latter two with no known solutions. We also show how probabilities of rare events can be computed from first-passage times, another class of unsolved problems challenging for simulation-based techniques due to large separations in time scales. Overall, the ACME method enables accurate and efficient solutions of the dCME for a large class of networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffbbbb7bc5dd79aef76dc4805a3cfb803faf3572" target='_blank'>Accurate Chemical Master Equation Solution Using Multi-Finite Buffers</a></td>
          <td>
            Youfang Cao, A. Terebus, Jie Liang
          </td>
          <td>2016-06-01</td>
          <td>Multiscale modeling & simulation : a SIAM interdisciplinary journal</td>
          <td>38</td>
          <td>40</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f522d23ad294932ea7fc3c12ae06c5601de33aec" target='_blank'>Stochastic Multistability from Simple Network Motifs</a></td>
          <td>
            A. Terebus, Youfang Cao, Jie Liang
          </td>
          <td>2015-01-27</td>
          <td>Biophysical Journal</td>
          <td>0</td>
          <td>40</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06eeb112de42683bdd7fd643f03e0f3f8b02490e" target='_blank'>Multi-Finite Buffer Method for Direct Solution of Discrete Chemical Master Equation</a></td>
          <td>
            Youfang Cao, A. Terebus, Jie Liang
          </td>
          <td>2014-01-28</td>
          <td>Biophysical Journal</td>
          <td>1</td>
          <td>40</td>
        </tr>
    
        <tr id="Stochastic models of reaction networks are becoming increasingly important in Systems Biology. In these models, the dynamics is generally represented by a continuous-time Markov chain whose states denote the copy-numbers of the constituent species. The state-space on which this process resides is a subset of non-negative integer lattice and for many examples of interest, this state-space is countably infinite. This causes numerous problems in analyzing the Markov chain and understanding its long-term behavior. These problems are further confounded by the presence of conservation relations among species which constrain the dynamics in complicated ways. In this paper we provide a linear-algebraic procedure to disentangle these conservation relations and represent the state-space in a special decomposed form, based on the copy-number ranges of various species and dependencies among them. This decomposed form is advantageous for analyzing the stochastic model and for a large class of networks we demonstrate how this form can be used for finding all the closed communication classes for the Markov chain within the infinite state-space. Such communication classes support the extremal stationary distributions and hence our results provide important insights into the long-term behavior and stability properties of stochastic models of reaction networks. We discuss how the knowledge of these communication classes can be used in many ways such as speeding-up stochastic simulations of multiscale networks or in identifying the stationary distributions of complex-balanced networks. We illustrate our results with several examples of gene-expression networks from Systems Biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a9afb46e2dd2e7698aa73f083fb4c46716ed970" target='_blank'>A generic state-space decomposition method for analyzing stochastic biomolecular reaction networks</a></td>
          <td>
            Ankit Gupta, M. Khammash
          </td>
          <td>2015-05-25</td>
          <td>arXiv: Probability</td>
          <td>2</td>
          <td>53</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4345aab3dedc2b8aa969aa501c1d1d81dbd0972" target='_blank'>Exact Computation of Velocity Field and Probability Flux of Time-Evolving Probability Landscape of Stochastic Networks</a></td>
          <td>
            A. Terebus, Chun Liu, Jie Liang
          </td>
          <td>2016-02-16</td>
          <td>Biophysical Journal</td>
          <td>0</td>
          <td>40</td>
        </tr>
    
        <tr id="Stochastic processes play an important role in gene regulatory networks. For many years, methods and algorithms have been developed to solve the problems regarding stochastic mechanisms in the cellular reaction system. Discrete Chemical Master Equation (dCME) is a method developed to analyze biological networks by computing the exact probability distribution of the microstates. With this method, because all computations and analyses of probability distribution can be processed based on the enumerated microstates, network microstates enumeration has been considered as a significant and prerequisite step. However, there is no efficient enumeration method. Applications will perform poorly when enumeration must address a complex or large network. To improve these microstate computation and analysis methods, we propose an efficient algorithm to enumerate microstates using Matrix Network, a new data structure we designed. Unlike traditional methods that perform the enumeration using simulation to apply reactions, the proposed approach utilizes the correlation of the microstate values and the geometric structure of the microstate map to accelerate the enumeration computation. In this paper, the theoretical basis, features and algorithms of Matrix Network are discussed. Moreover, sample applications demonstrating computation and analysis using Matrix Network are provided.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdb7fcf07c3d79ef2745c54f6282b9c753709a98" target='_blank'>Matrix Network: A New Data Structure for Efficient Enumeration of Microstates of a Genetic Regulatory Network</a></td>
          <td>
            Xiao Cong, T. Akutsu
          </td>
          <td>2015-11-01</td>
          <td>J. Inf. Process.</td>
          <td>0</td>
          <td>52</td>
        </tr>
    
        <tr id="Stochastic models of reaction networks are becoming increasingly important in Systems Biology. In these models, the dynamics is generally represented by a continuous-time Markov chain whose states denote the copy-numbers of the constituent species. The state-space on which this process resides is a subset of non-negative integer lattice and for many examples of interest, this state-space is countably infinite. This causes numerous problems in analyzing the Markov chain and understanding its long-term behavior. These problems are further confounded by the presence of conservation relations among species which constrain the dynamics in complicated ways. In this paper we provide a linear-algebraic procedure to disentangle these conservation relations and represent the state-space in a special decomposed form, based on the copy-number ranges of various species and dependencies among them. This decomposed form is advantageous for analyzing the stochastic model and for a large class of networks we demonstrate how this form can be used for finding all the closed communication classes for the Markov chain within the infinite state-space. Such communication classes are irreducible state-spaces for the dynamics and they support all the extremal stationary distributions for the Markov chain. Hence our results provide important insights into the long-term behavior and stability properties of stochastic models of reaction networks. We discuss how the knowledge of these irreducible state-spaces can be used in many ways such as speeding-up stochastic simulations of multiscale networks or in identifying the stationary distributions of complex-balanced networks. We illustrate our results with several examples of gene-expression networks from Systems Biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d5233449950ec516ca1d5619b8cb16a689cfb1" target='_blank'>Computational Identification of Irreducible State-Spaces for Stochastic Reaction Networks</a></td>
          <td>
            Ankit Gupta, M. Khammash
          </td>
          <td>2015-05-25</td>
          <td>SIAM J. Appl. Dyn. Syst.</td>
          <td>17</td>
          <td>53</td>
        </tr>
    
        <tr id="The last decade has witnessed a surge of theoretical and computational models to describe the dynamics of complex gene regulatory networks, and how these interactions can give rise to multistable and heterogeneous cell populations. As the use of theoretical modeling to describe genetic and biochemical circuits becomes more widespread, theoreticians with mathematical and physical backgrounds routinely apply concepts from statistical physics, non-linear dynamics, and network theory to biological systems. This review aims at providing a clear overview of the most important methodologies applied in the field while highlighting current and future challenges. It also includes hands-on tutorials to solve and simulate some of the archetypical biological system models used in the field. Furthermore, we provide concrete examples from the existing literature for theoreticians that wish to explore this fast-developing field. Whenever possible, we highlight the similarities and differences between biochemical and regulatory networks and ‘classical’ systems typically studied in non-equilibrium statistical and quantum mechanics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16424e863cb8b16b8b7a07cc5adeab669c7c3ee9" target='_blank'>Theoretical and computational tools to model multistable gene regulatory networks</a></td>
          <td>
            F. Bocci, D. Jia, Qing Nie, M. Jolly, J. Onuchic
          </td>
          <td>2023-08-02</td>
          <td>Reports on Progress in Physics</td>
          <td>5</td>
          <td>96</td>
        </tr>
    
        <tr id="The last decade has witnessed a surge of theoretical and computational models to describe the dynamics of complex gene regulatory networks, and how these interactions can give rise to multistable and heterogeneous cell populations. As the use of theoretical modeling to describe genetic and biochemical circuits becomes more widespread, theoreticians with mathematical and physical backgrounds routinely apply concepts from statistical physics, non-linear dynamics, and network theory to biological systems. This review aims at providing a clear overview of the most important methodologies applied in the field while highlighting current and future challenges. It also includes hands-on tutorials to solve and simulate some of the archetypical biological system models used in the field. Furthermore, we provide concrete examples from the existing literature for theoreticians that wish to explore this fast-developing field. Whenever possible, we highlight the similarities and differences between biochemical and regulatory networks and 'classical' systems typically studied in non-equilibrium statistical and quantum mechanics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1aa30e5baf1fcd11c81e0bb2acf328de042faff" target='_blank'>Theoretical and computational tools to model multistable gene regulatory networks</a></td>
          <td>
            F. Bocci, D. Jia, Q. Nie, M. Jolly, J. Onuchic
          </td>
          <td>2023-02-14</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>96</td>
        </tr>
    
        <tr id="MOTIVATION
Regulatory gene networks contain generic modules such as feedback loops that are essential for the regulation of many biological functions. The study of the stochastic mechanisms of gene regulation is instrumental for the understanding of how cells maintain their expression at levels commensurate with their biological role, as well as to engineer gene expression switches of appropriate behavior. The lack of precise knowledge on the steady-state distribution of gene expression requires the use of Gillespie algorithms and Monte-Carlo approximations.


METHODOLOGY
In this study, we provide new exact formulas and efficient numerical algorithms for computing/modeling the steady-state of a class of self-regulated genes, and we use it to model/compute the stochastic expression of a gene of interest in an engineered network introduced in mammalian cells. The behavior of the genetic network is then analyzed experimentally in living cells.


RESULTS
Stochastic models often reveal counter-intuitive experimental behaviors, and we find that this genetic architecture displays a unimodal behavior in mammalian cells, which was unexpected given its known bimodal response in unicellular organisms. We provide a molecular rationale for this behavior, and we implement it in the mathematical picture to explain the experimental results obtained from this network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5795aed48f835cbf512f4f7c8f14c5b583aed3e1" target='_blank'>Steady-state expression of self-regulated genes</a></td>
          <td>
            Thomas Fournier, J. Gabriel, C. Mazza, Jerome Pasquier, J. Galbete, N. Mermod
          </td>
          <td>2007-12-01</td>
          <td>Bioinformatics</td>
          <td>25</td>
          <td>39</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc08fd96dc69e849f23546b01210fce1a2f9cd7" target='_blank'>Numerical simulation of stochastic gene circuits</a></td>
          <td>
            H. Salis, Y. Kaznessis
          </td>
          <td>2005-02-15</td>
          <td>Comput. Chem. Eng.</td>
          <td>35</td>
          <td>39</td>
        </tr>
    
        <tr id="Noise is often indispensable to key cellular activities, such as gene expression, necessitating the use of stochastic models to capture its dynamics. The chemical master equation (CME) is a commonly used stochastic model of Kolmogorov forward equations that describe how the probability distribution of a chemically reacting system varies with time. Finding analytic solutions to the CME can have benefits, such as expediting simulations of multiscale biochemical reaction networks and aiding the design of distributional responses. However, analytic solutions are rarely known. A recent method of computing analytic stationary solutions relies on gluing simple state spaces together recursively at one or two states. We explore the capabilities of this method and introduce algorithms to derive analytic stationary solutions to the CME. We first formally characterize state spaces that can be constructed by performing single-state gluing of paths, cycles or both sequentially. We then study stochastic biochemical reaction networks that consist of reversible, elementary reactions with two-dimensional state spaces. We also discuss extending the method to infinite state spaces and designing the stationary behaviour of stochastic biochemical reaction networks. Finally, we illustrate the aforementioned ideas using examples that include two interconnected transcriptional components and biochemical reactions with two-dimensional state spaces.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77c32d987974cccd85316669516603f9fa74bd6b" target='_blank'>Recursively constructing analytic expressions for equilibrium distributions of stochastic biochemical reaction networks</a></td>
          <td>
            X. Flora Meng, Ania-Ariadna Baetica, Vipul Singhal, Richard M. Murray
          </td>
          <td>2017-05-01</td>
          <td>Journal of The Royal Society Interface</td>
          <td>3</td>
          <td>8</td>
        </tr>
    
        <tr id="We quantify the influence of the topology of a transcriptional regulatory network on its ability to process environmental signals. By posing the problem in terms of information theory, we do this without specifying the function performed by the network. Specifically, we study the maximum mutual information between the input (chemical) signal and the output (genetic) response attainable by the network in the context of an analytic model of particle number fluctuations. We perform this analysis for all biochemical circuits, including various feedback loops, that can be built out of 3 chemical species, each under the control of one regulator. We find that a generic network, constrained to low molecule numbers and reasonable response times, can transduce more information than a simple binary switch and, in fact, manages to achieve close to the optimal information transmission fidelity. These high-information solutions are robust to tenfold changes in most of the networks' biochemical parameters; moreover they are easier to achieve in networks containing cycles with an odd number of negative regulators (overall negative feedback) due to their decreased molecular noise (a result which we derive analytically). Finally, we demonstrate that a single circuit can support multiple high-information solutions. These findings suggest a potential resolution of the “cross-talk” phenomenon as well as the previously unexplained observation that transcription factors that undergo proteolysis are more likely to be auto-repressive.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/624dcbdaed19b95c6d57658867c2ca7fded3889d" target='_blank'>Optimal Signal Processing in Small Stochastic Biochemical Networks</a></td>
          <td>
            E. Ziv, I. Nemenman, C. Wiggins
          </td>
          <td>2006-12-21</td>
          <td>PLoS ONE</td>
          <td>129</td>
          <td>32</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57b5b7707f6c5b8d63e348857b2b44037b6214f7" target='_blank'>Exact Probability Distributions of Selected Species in Stochastic Chemical Reaction Networks</a></td>
          <td>
            F. López-Caamal, T. Marquez-Lago
          </td>
          <td>2014-08-26</td>
          <td>Bulletin of Mathematical Biology</td>
          <td>7</td>
          <td>33</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57b5b7707f6c5b8d63e348857b2b44037b6214f7" target='_blank'>Exact Probability Distributions of Selected Species in Stochastic Chemical Reaction Networks</a></td>
          <td>
            F. López-Caamal, T. Marquez-Lago
          </td>
          <td>2014-08-26</td>
          <td>Bulletin of Mathematical Biology</td>
          <td>7</td>
          <td>33</td>
        </tr>
    
        <tr id="At the molecular level, biochemical processes are governed by random interactions between reactant molecules, and the dynamics of such systems are inherently stochastic. When the copy numbers of reactants are large, a deterministic description is adequate, but when they are small, such systems are often modeled as continuous-time Markov jump processes that can be described by the chemical master equation. Gillespie's Stochastic Simulation Algorithm (SSA) generates exact trajectories of these systems, but the amount of computational work required for each step of the original SSA is proportional to the number of reaction channels, leading to computational complexity that scales linearly with the problem size. The original SSA is therefore inefficient for large problems, which has prompted the development of several alternative formulations with improved scaling properties. We describe an exact SSA that uses a table data structure with event time binning to achieve constant computational complexity with respect to the number of reaction channels for weakly coupled reaction networks. We present a novel adaptive binning strategy and discuss optimal algorithm parameters. We compare the computational efficiency of the algorithm to existing methods and demonstrate excellent scaling for large problems. This method is well suited for generating exact trajectories of large weakly coupled models, including those that can be described by the reaction-diffusion master equation that arises from spatially discretized reaction-diffusion processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6135db96c31e88d34130fad5abbeb4d51f9697a0" target='_blank'>Constant-complexity Stochastic Simulation Algorithm with Optimal Binning</a></td>
          <td>
            Kevin R. Sanft, H. Othmer
          </td>
          <td>2015-03-19</td>
          <td>The Journal of chemical physics</td>
          <td>13</td>
          <td>56</td>
        </tr>
    
        <tr id="Lecture notes, slides, and a student assignment illustrate a strategy for teaching stochastic modeling of biological systems. This Teaching Resource provides lecture notes, slides, and a student assignment for a two-part lecture that introduces stochastic modeling of biological systems. The first lecture uses biological examples to present the concept of cell-to-cell variability and makes the connection between the variability of single-cell measurements and concepts from statistical mechanics and probability theory. This section makes the point that for low copy number of a species, the usual differential equation formalism is no longer applicable and needs to be replaced by a probabilistic approach based on the so-called Master Equation. As an example, a simple model of gene transcription is discussed in detail, the different contributions to the relevant Master Equation are highlighted, and the equation itself is derived. The second lecture describes how, for more complex and biologically interesting applications, direct solution of the Master Equation becomes difficult. Gillespie’s algorithm, which is used in most cases of biological interest, is then introduced as a practical alternative. The lecture delves into the crux of Gillespie’s algorithm, which entails the drawing of two random numbers at each time step. It establishes the corresponding formalism, details the connection between chemical rate constants and Gillespie rates, and culminates in a description and explanation of a core MATLAB program for the transcriptional model considered in the first lecture. This core program, written for a single cell, is expanded by the students in the homework assignment to consider both transcription and translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1eb9ae2445a77984468c49fdeec59bc2be5a481" target='_blank'>Simulations of Stochastic Biological Phenomena</a></td>
          <td>
            F. Hayot
          </td>
          <td>2011-09-27</td>
          <td>Science Signaling</td>
          <td>2</td>
          <td>20</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/658d0398cd88cdb9dcbecc43642621dedbb01e1f" target='_blank'>Abstraction-based segmental simulation of reaction networks using adaptive memoization</a></td>
          <td>
            Martin Helfrich, Roman Andriushchenko, Milan Ceska, Jan Kretínský, Stefan Marticek, David Šafránek
          </td>
          <td>2024-11-08</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>
    
        <tr id="Description: By providing expositions to modeling principles, theories, computational solutions, and open problems, this reference presents a full scope on relevant biological phenomena, modeling frameworks, technical challenges, and algorithms. Up–to–date developments of structures of biomolecules, systems biology, advanced models, and algorithms Sampling techniques for estimating evolutionary rates and generating molecular structures Accurate computation of probability landscape of stochastic networks, solving discrete chemical master equations End–of–chapter exercises">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2b039afe6cfc30d675761d47b291ad474cb5bc1" target='_blank'>Models and Algorithms for Biomolecules and Molecular Networks</a></td>
          <td>
            Bhaskar DasGupta, Jie Liang
          </td>
          <td>2016-01-26</td>
          <td></td>
          <td>18</td>
          <td>40</td>
        </tr>
    
        <tr id="Biochemical reaction networks (BRNs) in a cell frequently consist of reactions with disparate timescales. The stochastic simulations of such multiscale BRNs are prohibitively slow due to high computational cost for the simulations of fast reactions. One way to resolve this problem uses the fact that fast species regulated by fast reactions quickly equilibrate to their stationary distribution while slow species are unlikely to be changed. Thus, on a slow timescale, fast species can be replaced by their quasi-steady state (QSS): their stationary conditional expectation values for given slow species. As the QSS are determined solely by the state of slow species, such replacement leads to a reduced model, where fast species are eliminated. However, it is challenging to derive the QSS in the presence of nonlinear reactions. While various approximation schemes for the QSS have been developed, they often lead to considerable errors. Here, we propose two classes of multiscale BRNs which can be reduced by deriving an exact QSS rather than approximations. Specifically, if fast species constitute either a feedforward network or a complex balanced network, the reduced model based on the exact QSS can be derived. Such BRNs are frequently observed in a cell as the feedforward network is one of fundamental motifs of gene or protein regulatory networks. Furthermore, complex balanced networks also include various types of fast reversible bindings such as bindings between transcriptional factors and gene regulatory sites. The reduced models based on exact QSS, which can be calculated by the computational packages provided in this work, accurately approximate the slow scale dynamics of the original full model with much lower computational cost.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a72c97ea7e880a56bc76eac8e674fc3f3285c4af" target='_blank'>Reduction of multiscale stochastic biochemical reaction networks using exact moment derivation</a></td>
          <td>
            Jae Kyoung Kim, Eduardo Sontag
          </td>
          <td>2017-06-01</td>
          <td>PLoS Computational Biology</td>
          <td>38</td>
          <td>96</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c83df151df861fdee633438490e5ddbcd743c5c" target='_blank'>Periodic synchronization of isolated network elements facilitates simulating and inferring gene regulatory networks including stochastic molecular kinetics</a></td>
          <td>
            Johannes Hettich, J. Gebhardt
          </td>
          <td>2022-01-05</td>
          <td>BMC Bioinformatics</td>
          <td>2</td>
          <td>19</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c83df151df861fdee633438490e5ddbcd743c5c" target='_blank'>Periodic synchronization of isolated network elements facilitates simulating and inferring gene regulatory networks including stochastic molecular kinetics</a></td>
          <td>
            Johannes Hettich, J. Gebhardt
          </td>
          <td>2022-01-05</td>
          <td>BMC Bioinformatics</td>
          <td>2</td>
          <td>19</td>
        </tr>
    
        <tr id="In stochastic models of biochemical reaction networks, the dynamics is usually represented by a Markov process which describes the evolution of the copy-numbers or molecular counts of the constituent species. It is often of biological interest to determine if this Markov process has a unique stationary distribution and for this to hold, it is necessary that the state-space is irreducible in the sense that all the states are reachable from each other in a finite time, with a positive probability. Finding such irreducible state-spaces is quite challenging, because the Markovian dynamics can usually access infinitely many states and the presence of conservation relations among species can constrain the dynamics in complicated ways. The aim of this paper is to develop a computational framework for finding irreducible state-spaces for reaction networks that typically arise in Systems and Synthetic Biology. Our results can help in assessing the long-term behavior of a network and also in explicitly obtaining the stationary distributions in certain cases. The framework we present relies only on elementary linear algebra and linear programming, which makes it highly scalable and efficient, even for very large networks. We illustrate the wide applicability of our framework through several examples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88b0d12247df2f2427de11dc635c8d4a99e53233" target='_blank'>A scalable method for finding irreducible state-spaces for stochastic models of biochemical reaction networks</a></td>
          <td>
            Ankit Gupta, M. Khammash
          </td>
          <td>2015-05-25</td>
          <td>arXiv: Probability</td>
          <td>1</td>
          <td>53</td>
        </tr>
    
        <tr id="In the biological systems, Monte Carlo approaches are used to provide the stochastic simulation of the chemical reactions. The major stochastic simulation algorithms (SSAs) are the direct method, also known as the Gillespie algorithm, the first reaction method and the next reaction method. While these methods give accurate generation of the results, they are computationally demanding for large complex systems. To increase the computational efficiency of SSAs, approximate SSAs can be option. The approximate methods rely on the leap condition. This condition means that the propensity function during the time interval $ t $ to $[ t+\tau ]$ should not be altered for the chosen time step $\tau$. Here, to proceed with the system's history axis from one time step to the next, we compute how many times each reaction can be realized in each small time interval $\tau$ so that we can observe plausible simultaneous reactions. Hence, this study aims to generate a realistic and close confidence interval for the parameter which denotes the underlying numbers of simultaneous reactions in the system by satifying the leap condition. For this purpose, the poisson $\tau$-leap algorithm and the approximate Gillespie algorithm, as the extension of the Gillespie algorithm, are handled. In the estimation for the associated parameters in both algorithms, we derive their maximum likelihood estimators, moment estimatora and bayesian estimators. From the derivations, we theoretically show that our novel confidence intervals are narrower than the current confidence intervals under the leap condition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3246d08a1348415d202a26a33b454e3af86e7ade" target='_blank'>Extension of Leap Condition in Approximate Stochastic Simulation Algorithms of Biological Networks</a></td>
          <td>
            Saliha Demirbüken, V. Purutçuoğlu
          </td>
          <td>2022-04-05</td>
          <td>Turkish Journal of Mathematics and Computer Science</td>
          <td>0</td>
          <td>7</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/751f4a7dec4ce0d39ac1e88818817955965d895f" target='_blank'>Biochemical Network Stochastic Simulator (BioNetS): software for stochastic modeling of biochemical networks</a></td>
          <td>
            D. Adalsteinsson, D. McMillen, T. Elston
          </td>
          <td>2004-03-08</td>
          <td>BMC Bioinformatics</td>
          <td>157</td>
          <td>41</td>
        </tr>
    
        <tr id="Simulation of biomolecular networks is now indispensable for studying biological systems, from small reaction networks to large ensembles of cells. Here we present a novel approach for stochastic simulation of networks embedded in the dynamic environment of the cell and its surroundings. We thus sample trajectories of the stochastic process described by the chemical master equation with time-varying propensities. A comparative analysis shows that existing approaches can either fail dramatically, or else can impose impractical computational burdens due to numerical integration of reaction propensities, especially when cell ensembles are studied. Here we introduce the Extrande method which, given a simulated time course of dynamic network inputs, provides a conditionally exact and several orders-of-magnitude faster simulation solution. The new approach makes it feasible to demonstrate—using decision-making by a large population of quorum sensing bacteria—that robustness to fluctuations from upstream signaling places strong constraints on the design of networks determining cell fate. Our approach has the potential to significantly advance both understanding of molecular systems biology and design of synthetic circuits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c156d226781ae1f81a37325c954857f6c791597e" target='_blank'>Stochastic Simulation of Biomolecular Networks in Dynamic Environments</a></td>
          <td>
            M. Voliotis, Philipp Thomas, R. Grima, Clive G. Bowsher
          </td>
          <td>2015-11-04</td>
          <td>PLoS Computational Biology</td>
          <td>72</td>
          <td>39</td>
        </tr>
    
        <tr id="Gene regulatory networks depict the interactions among genes, proteins, and other components of the cell. These interactions are stochastic when large differences in reaction rates and small copy number of molecules are involved. Discrete Chemical Master Equation (dCME) provides a general framework for understanding the stochastic nature of these networks. Here we used the Accurate Chemical Master Equation method to directly compute the exact steady state probability landscape of the feed-forward loop motif (FFL). FFL is one of the most abundant gene regulatory networks motifs where the regulation is carried out from the top nodes to the bottom ones. We examine the behavior of stochastic FFLs under different conditions of various regulation intensities. Under the conditions with slow promoter binding, we show how FFL can exhibit different multistabilities in their landscapes. We also study the sensitivities of regulations of FFLs and introduce a new definition of stochastic sensitivity to characterize how FFLs respond in their probability distributions at the steady state to perturbations of system parameters. We show how change in gene expression under FFL regulations are sensitive to system parameters, including the state of multistability in FFLs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/683e9f072e000d1cf131fef686fffe7dbc539074" target='_blank'>Sensitivities of Regulation Intensities in Feed-Forward Loops with Multistability</a></td>
          <td>
            A. Terebus, Youfang Cao, Jie Liang
          </td>
          <td>2019-07-01</td>
          <td>2019 41st Annual International Conference of the IEEE Engineering in Medicine and Biology Society (EMBC)</td>
          <td>1</td>
          <td>40</td>
        </tr>
    
        <tr id="MOTIVATION
Stochastic reaction networks are a widespread model to describe biological systems where the presence of noise is relevant, such as in cell regulatory processes. Unfortunately, in all but simplest models the resulting discrete state-space representation hinders analytical tractability and makes numerical simulations expensive. Reduction methods can lower complexity by computing model projections that preserve dynamics of interest to the user.


RESULTS
We present an exact lumping method for stochastic reaction networks with mass-action kinetics. It hinges on an equivalence relation between the species, resulting in a reduced network where the dynamics of each macro-species is stochastically equivalent to the sum of the original species in each equivalence class, for any choice of the initial state of the system. Furthermore, by an appropriate encoding of kinetic parameters as additional species, the method can establish equivalences that do not depend on specific values of the parameters. The method is supported by an efficient algorithm to compute the largest species equivalence, thus the maximal lumping. The effectiveness and scalability of our lumping technique, as well as the physical interpretability of resulting reductions, is demonstrated in several models of signaling pathways and epidemic processes on complex networks.


AVAILABILITY
The algorithms for species equivalence have been implemented in the software tool ERODE, freely available for download from https://www.erode.eu.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/424b71ed746317352adca4960f80128f05a07e3d" target='_blank'>Exact maximal reduction of stochastic reaction networks by species lumping</a></td>
          <td>
            L. Cardelli, Isabel Cristina Pérez-Verona, M. Tribastone, Max Tschaikowski, Andrea Vandin, Tabea Waizmann
          </td>
          <td>2021-01-09</td>
          <td>Bioinformatics</td>
          <td>11</td>
          <td>73</td>
        </tr>
    
        <tr id="ABSTRACT We consider stochastic models of biochemical networks. In the stochastic modeling of the biochemical networks, species in the networks are modeled as discrete random variables; we let ni(t) be the number of molecules of ith species at time t and let n(t) = (n1(t), . . . , ns(t)) be a random vector with probability density p(n, t). The governing equation of a network with r reactions, the so-called chemical master equation is dp(n, t) dt = r">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/113766b3d54357d3e3059f7267b6511d64d6de6a" target='_blank'>Stochastic Analysis and Computation of Biochemical Networks</a></td>
          <td>
            Chang Hyeong Lee
          </td>
          <td>2009-11-01</td>
          <td></td>
          <td>0</td>
          <td>11</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93cc69fa9f8ca57001a92bfea3eb04a82d7e8da2" target='_blank'>Dynamical properties of Discrete Reaction Networks</a></td>
          <td>
            Loïc Paulevé, G. Craciun, H. Koeppl
          </td>
          <td>2013-02-14</td>
          <td>Journal of Mathematical Biology</td>
          <td>34</td>
          <td>29</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93cc69fa9f8ca57001a92bfea3eb04a82d7e8da2" target='_blank'>Dynamical properties of Discrete Reaction Networks</a></td>
          <td>
            Loïc Paulevé, G. Craciun, H. Koeppl
          </td>
          <td>2013-02-14</td>
          <td>Journal of Mathematical Biology</td>
          <td>34</td>
          <td>29</td>
        </tr>
    
        <tr id="While noise is generally associated with uncertainties and often has a negative connotation in engineering, living organisms have evolved to adapt to (and even exploit) such uncertainty to ensure the survival of a species or implement certain functions that would have been difficult or even impossible otherwise. In this article, we review the role and impact of noise in systems and synthetic biology, with a particular emphasis on its role in the genetic control of biological systems, an area we refer to as cybergenetics. The main modeling paradigm is that of stochastic reaction networks, whose applicability goes beyond biology, as these networks can represent any population dynamics system, including ecological, epidemiological, and opinion dynamics networks. We review different ways to mathematically represent these systems, and we notably argue that the concept of ergodicity presents a particularly suitable way to characterize their stability. We then discuss noise-induced properties and show that noise can be both an asset and a nuisance in this setting. Finally, we discuss recent results on (stochastic) cybergenetics and explore their relationships to noise. Along the way, we detail the different technical and biological constraints that need to be respected when designing synthetic biological circuits. Finally, we discuss the concepts, problems, and solutions exposed in the article; raise criticisms and concerns about current ideas and approaches; suggest current (open) problems with potential solutions; and provide some ideas for future research directions. Expected final online publication date for the Annual Review of Control, Robotics, and Autonomous Systems, Volume 14 is May 2023. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/283de7de591b4d86bd9ab67c43262183bae3b127" target='_blank'>Noise in Biomolecular Systems: Modeling, Analysis, and Control Implications</a></td>
          <td>
            Corentin Briat, M. Khammash
          </td>
          <td>2022-09-28</td>
          <td>ArXiv</td>
          <td>12</td>
          <td>53</td>
        </tr>
    
        <tr id="The property of multicellular organisms that allows cells with the same genetic code to maintain distinct identities for the entire life of the organism is known as epigenetic cell memory (ECM). Recently, chromatin modifications have appeared to have a key role in ECM. Here, we conduct a stochastic analysis of a chromatin modification circuit to determine the effect of time scale separation among principal system processes on the extent to which the system can keep a stable steady state in the face of noise. To this end, we first obtain a reduced circuit model and determine an analytical expression for both the stationary probability distribution and the switching time between repressed and active chromatin states. Then, we validate these analytical results with stochastic simulations of the original full set of reactions. Our results show that when the basal decay of all chromatin marks is sufficiently slower with respect to the auto and cross-catalysis and the recruited erasure of all the marks, the stationary distribution shows bimodality, with two concentrated peaks in correspondence of the active and repressed states, but biased towards the repressed state. In accordance with these results, slower basal decay extends the memory of the active and repressed states, suggesting, more broadly, a critical design principle for long-lasting memory of gene expression states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a1001c658127ce80d2c435596a7856956f6ca39" target='_blank'>Analytical and computational study of the stochastic behavior of a chromatin modification circuit *</a></td>
          <td>
            Simone Bruno, Ruth J. Williams, D. Vecchio
          </td>
          <td>2022-12-06</td>
          <td>2022 IEEE 61st Conference on Decision and Control (CDC)</td>
          <td>0</td>
          <td>31</td>
        </tr>
    
        <tr id="Spatial stochastic models of single cell kinetics are capable of capturing both fluctuations in molecular numbers and the spatial dependencies of the key steps of intracellular regulatory networks. The spatial stochastic model can be simulated both on a detailed microscopic level using particle tracking and on a mesoscopic level using the reaction–diffusion master equation. However, despite substantial progress on simulation efficiency for spatial models in the last years, the computational cost quickly becomes prohibitively expensive for tasks that require repeated simulation of thousands or millions of realizations of the model. This limits the use of spatial models in applications such as multicellular simulations, likelihood-free parameter inference, and robustness analysis. Further approximation of the spatial dynamics is needed to accelerate such computational engineering tasks. We here propose a multiscale model where a compartment-based model approximates a detailed spatial stochastic model. The compartment model is constructed via a first-exit time analysis on the spatial model, thus capturing critical spatial aspects of the fine-grained simulations, at a cost close to the simple well-mixed model. We apply the multiscale model to a canonical model of negative-feedback gene regulation, assess its accuracy over a range of parameters, and demonstrate that the approximation can yield substantial speedups for likelihood-free parameter inference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87f6de0565c823bf32e3deda9d5794b3131346ab" target='_blank'>A multiscale compartment-based model of stochastic gene regulatory networks using hitting-time analysis</a></td>
          <td>
            A. Coulier, Stefan Hellander, A. Hellander
          </td>
          <td>2021-05-11</td>
          <td>The Journal of Chemical Physics</td>
          <td>9</td>
          <td>23</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3e90da0a020c7cf1e31a0b5bec4492affd182ca" target='_blank'>Are the complex distributed genetic networks inherently oscillatory and chaotic?</a></td>
          <td>
            V. Zhdanov, V. Zhdanov
          </td>
          <td>2011-03-24</td>
          <td>JETP Letters</td>
          <td>2</td>
          <td>47</td>
        </tr>
    
        <tr id="Noise is often indispensable to key cellular activities, such as gene expression, necessitating the use of stochastic models to capture its dynamics. The chemical master equation (CME) is a commonly used stochastic model that describes how the probability distribution of a chemically reacting system varies with time. Knowing analytic solutions to the CME can have benefits, such as expediting simulations of multiscale biochemical reaction networks and aiding the design of distributional responses. However, analytic solutions are rarely known. A recent method of computing analytic stationary solutions relies on gluing simple state spaces together recursively at one or two states. We explore the capabilities of this method and introduce algorithms to derive analytic stationary solutions to the CME. We first formally characterise state spaces that can be constructed by performing single-state gluing of paths, cycles or both sequentially. We then study stochastic biochemical reaction networks that consist of reversible, elementary reactions with two-dimensional state spaces. We also discuss extending the method to infinite state spaces and designing stationary distributions that satisfy user-specified constraints. Finally, we illustrate the aforementioned ideas using examples that include two interconnected transcriptional components and chemical reactions with two-dimensional state spaces. Subject Areas Systems biology, synthetic biology, biomathematics, bioengineering">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27079f79b50163596e322e669cd5da36bd4086ed" target='_blank'>Finding analytic stationary solutions to the chemical master equation by gluing state spaces at one or two states recursively</a></td>
          <td>
            Flora Meng, Ania-Ariadna Baetica, Vipul Singhal, R. Murray
          </td>
          <td>2017-03-03</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>95</td>
        </tr>
    
        <tr id="An important invariant of a chemical reaction network is its maximum number of positive steady states. This number, however, is in general difficult to compute. Nonetheless, there is an upper bound on this number -- namely, a network's mixed volume -- that is easy to compute. Moreover, recent work has shown that, for certain biological signaling networks, the mixed volume does not greatly exceed the maximum number of positive steady states. Continuing this line of research, we further investigate this overcount and also compute the mixed volumes of small networks, those with only a few species or reactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc2f8e0d84a30d7b23bedf810b9448d660dae47f" target='_blank'>Mixed volume of small reaction networks</a></td>
          <td>
            N. Obatake, Anne Shiu, Dilruba Sofia
          </td>
          <td>2020-04-29</td>
          <td>Involve, a Journal of Mathematics</td>
          <td>8</td>
          <td>22</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>