---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-29 08:54:18 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Scaffold proteins may biphasically affect the levels of mitogen-activated protein kinase signaling and reduce its threshold properties.</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="The assembly of multiple signaling proteins into a complex by a scaffold protein guides many cellular decisions. Despite recent advances, the overarching principles that govern scaffold function are not well understood. We carried out a computational study using kinetic Monte Carlo simulations to understand how spatial localization of kinases on a scaffold may regulate signaling under different physiological conditions. Our studies identify regulatory properties of scaffold proteins that allow them to both amplify and attenuate incoming signals in different biological contexts. These properties are not caused by the well established prozone or combinatorial inhibition effect. These results bring coherence to seemingly paradoxical observations and suggest that cells have evolved design rules that enable scaffold proteins to regulate widely disparate cellular functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d58218d364f6073cc6e67d12e57293a72afbfda" target='_blank'>Scaffold proteins confer diverse regulatory properties to protein kinase cascades</a></td>
          <td>
            J. Locasale, A. Shaw, A. Chakraborty
          </td>
          <td>2007-04-01</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>78</td>
          <td>86</td>
        </tr>
    
        <tr id="Scaffold proteins play an important role in coordinating signal transduction cascades. However, their exact mechanism of action and the ultimate effect they have on the signal output remain unclear. Ferrell discusses how computer simulations have provided insight into the multiple possible functions that scaffold proteins may have. What remains is to test the predictions in real cells to determine what difference the presence of a scaffold really makes in the output of a signaling pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46d8d662e7add2368c27003f63021cc354dc74e6" target='_blank'>What Do Scaffold Proteins Really Do?</a></td>
          <td>
            J. Ferrell
          </td>
          <td>2000-10-03</td>
          <td>Science's STKE</td>
          <td>92</td>
          <td>62</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da87720f4767a44da3f92e497492607a5e8a082c" target='_blank'>When kinases meet mathematics: the systems biology of MAPK signalling</a></td>
          <td>
            W. Kolch, Muffy Calder, David R. Gilbert
          </td>
          <td>2005-03-21</td>
          <td>FEBS Letters</td>
          <td>190</td>
          <td>86</td>
        </tr>
    
        <tr id="Scaffold proteins play a crucial role in facilitating signal transduction in eukaryotes by bringing together multiple signaling components. In this study, we performed a systematic analysis of scaffold proteins in signal transduction by integrating protein-protein interaction and kinase-substrate relationship networks. We predicted 212 scaffold proteins that are involved in 605 distinct signaling pathways. The computational prediction was validated using a protein microarray-based approach. The predicted scaffold proteins showed several interesting characteristics, as we expected from the functionality of scaffold proteins. We found that the scaffold proteins are likely to interact with each other, which is consistent with previous finding that scaffold proteins tend to form homodimers and heterodimers. Interestingly, a single scaffold protein can be involved in multiple signaling pathways by interacting with other scaffold protein partners. Furthermore, we propose two possible regulatory mechanisms by which the activity of scaffold proteins is coordinated with their associated pathways through phosphorylation process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de68e00914c4f26923915b114bc6e4235ea4624c" target='_blank'>Systematic Prediction of Scaffold Proteins Reveals New Design Principles in Scaffold-Mediated Signal Transduction</a></td>
          <td>
            Jianfei Hu, J. Neiswinger, Jin Zhang, Heng Zhu, Jiang Qian
          </td>
          <td>2015-09-01</td>
          <td>PLoS Computational Biology</td>
          <td>15</td>
          <td>51</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d443b3671784c408e03c96fb4e03c3b35351c68" target='_blank'>Computational Modeling Implicates Protein Scaffolding in p38 Regulation of Akt.</a></td>
          <td>
            Patrick C. Kinnunen, G. Luker, K. Luker, J. Linderman
          </td>
          <td>2022-10-01</td>
          <td>Journal of theoretical biology</td>
          <td>5</td>
          <td>53</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f757fb478734b45800bc91e4b9a506fded6441b" target='_blank'>Phenotypic Properties of Scaffold-Based Signaling Paradigms</a></td>
          <td>
            Ryan Suderman, A. Schauer, Eric J. Deeds
          </td>
          <td>2015-01-27</td>
          <td>Biophysical Journal</td>
          <td>0</td>
          <td>21</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a12f4100a691e0f63f07dd6467fec2df4d5adef" target='_blank'>ScaPD: a database for human scaffold proteins</a></td>
          <td>
            Xiaomei Han, Jenny Wang, Jie Wang, Sheng Liu, Jianfei Hu, Hengying Zhu, J. Qian
          </td>
          <td>2017-10-01</td>
          <td>BMC Bioinformatics</td>
          <td>11</td>
          <td>31</td>
        </tr>
    
        <tr id="Intracellular signaling pathways that share common components often elicit distinct physiological responses. In most cases, the biochemical mechanisms responsible for this signal specificity remain poorly understood. Protein scaffolds and cross-inhibition have been proposed as strategies to prevent unwanted cross-talk. Here, we report a mechanism for signal specificity termed “kinetic insulation.” In this approach signals are selectively transmitted through the appropriate pathway based on their temporal profile. In particular, we demonstrate how pathway architectures downstream of a common component can be designed to efficiently separate transient signals from signals that increase slowly over time. Furthermore, we demonstrate that upstream signaling proteins can generate the appropriate input to the common pathway component regardless of the temporal profile of the external stimulus. Our results suggest that multilevel signaling cascades may have evolved to modulate the temporal profile of pathway activity so that stimulus information can be efficiently encoded and transmitted while ensuring signal specificity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34f99baf2a3fce2ef002f0b3da1545674a8824cc" target='_blank'>Kinetic insulation as an effective mechanism for achieving pathway specificity in intracellular signaling networks</a></td>
          <td>
            M. Behar, H. Dohlman, T. Elston
          </td>
          <td>2007-10-09</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>87</td>
          <td>54</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81da740350ca8c380e1a66047472929307659a28" target='_blank'>Building with a scaffold: emerging strategies for high- to low-level cellular modeling.</a></td>
          <td>
            T. Ideker, D. Lauffenburger
          </td>
          <td>2003-06-01</td>
          <td>Trends in biotechnology</td>
          <td>202</td>
          <td>119</td>
        </tr>
    
        <tr id="Abstract A kinase anchoring proteins (AKAPs) bind multiple signaling proteins and have subcellular targeting domains that allow them to greatly impact cellular signaling. AKAPs localize, specify, amplify, and accelerate signal transduction within the cell by bringing signaling proteins together in space and time. AKAPs also organize higher-order network motifs such as feed forward and feedback loops that may create complex network responses, including adaptation, oscillation, and ultrasensitivity. Computational models have begun to provide an insight into how AKAPs regulate signaling dynamics and cardiovascular pathophysiology. Models of mitogen-activated protein kinase and epidermal growth factor receptor scaffolds have revealed additional design principles and new methods for representing signaling scaffolds mathematically. Coupling computational modeling with quantitative experimental approaches will be increasingly necessary for dissecting the diverse information processing functions performed by AKAP signaling complexes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/def404ce92aea068ec191a04eb36d8eafca030dd" target='_blank'>Bigger, Better, Faster: Principles and Models of AKAP Anchoring Protein Signaling</a></td>
          <td>
            Eric C Greenwald, J. Saucerman
          </td>
          <td>2011-11-01</td>
          <td>Journal of Cardiovascular Pharmacology</td>
          <td>35</td>
          <td>35</td>
        </tr>
    
        <tr id="Mitogen‐activated protein kinase (MAPK) cascades process a myriad of stimuli received by cell‐surface receptors and generate precise spatiotemporal guidance for multiple target proteins, dictating receptor‐specific cellular outcomes. Computational modeling reveals that the intrinsic topology of MAPK cascades enables them to amplify signal sensitivity and amplitude, reduce noise, and display intricate dynamic properties, which include toggle switches, excitation pulses, and oscillations. Specificity of signaling responses can be brought about by signal‐induced feedback and feedforward wiring imposed on the MAPK cascade backbone. Intracellular gradients of protein activities arise from the spatial separation of opposing reactions in kinase‐phosphatase cycles. The membrane confinement of the initiating kinase in MAPK cascades and cytosolic localization of phosphatases can result in precipitous gradients of phosphorylated signal‐transducers if they spread solely by diffusion. Endocytotic trafficking of active kinases driven by molecular motors and traveling waves of protein phosphorylation can propagate phosphorylation signals from the plasma membrane to the nucleus, especially in large cells, such as Xenopus eggs. Copyright © 2009 John Wiley & Sons, Inc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/549f0ea398377e29460e84284cf31387d6568803" target='_blank'>Four‐dimensional dynamics of MAPK information‐processing systems</a></td>
          <td>
            B. Kholodenko, M. Birtwistle
          </td>
          <td>2009-07-01</td>
          <td>Wiley Interdisciplinary Reviews: Systems Biology and Medicine</td>
          <td>87</td>
          <td>57</td>
        </tr>
    
        <tr id="The mitogen-activated protein kinase (MAPK) pathway is considered to be a central block in many biological signaling networks. Despite the common core cascade structure, the activation of MAPK in different biological systems can exhibit different types of dynamic behaviors. Computer modeling may help to reveal the mechanisms underlying such variations. However, so far most computational models of the MAPK cascade have been system-specific, or to reflect a particular type among the wide spectrum of possible dynamics. To obtain a general and integrated view of the relationship between the dynamics of MAPK activation and the structures and parameters of the MAPK cascade, we constructed a generic model by comparing previous models covering different specific biological systems. We assumed that reliable qualitative results could be predicted through a qualitative model with pseudo parameters. We used randomly sampled parameters instead of a specific set of “best-fit” parameters to avoid biases towards any particular systems. A range of dynamics behaviors for MAPK activation, including ultrasensitivity, bistability, transient activation and oscillation, were successfully predicted by the generic model. The results indicated that the steady state dynamics (ultrasensitivity and bistability) was jointly determined by the three-tiered structure of the MAPK cascade and the competitive substrate binding in the dual-phosphorylation processes of the central components, while the temporal dynamics (transient activation and oscillation) was mainly affected by the upstream signaling pathway and feedbacks. Moreover, MAPK kinase (MAPKK) played more important roles than the other two components in determining the dynamics of MAPK activation. We hypothesize that this is an important and advantageous property for the regulation and for the functional diversity of MAPK pathways in real cells. Finally, to assist developing generic models for signaling motifs through model comparisons, we proposed a reaction-based database to make the model data more flexible and interoperable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/558a5c17bf4b19875b0696b35de9a583a6c40956" target='_blank'>Random Parameter Sampling of a Generic Three-Tier MAPK Cascade Model Reveals Major Factors Affecting Its Versatile Dynamics</a></td>
          <td>
            Zhong Mai, Haiyan Liu
          </td>
          <td>2013-01-24</td>
          <td>PLoS ONE</td>
          <td>9</td>
          <td>26</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc002018eee636962f38bf960ab4d4ffcec84fad" target='_blank'>Prediction of scaffold proteins based on protein interaction and domain architectures</a></td>
          <td>
            Kimin Oh, G. Yi
          </td>
          <td>2015-10-22</td>
          <td>BMC Bioinformatics</td>
          <td>5</td>
          <td>23</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc002018eee636962f38bf960ab4d4ffcec84fad" target='_blank'>Prediction of scaffold proteins based on protein interaction and domain architectures</a></td>
          <td>
            Kimin Oh, G. Yi
          </td>
          <td>2015-10-22</td>
          <td>BMC Bioinformatics</td>
          <td>5</td>
          <td>23</td>
        </tr>
    
        <tr id="Intracellular transport of proteins by motors along cytoskeletal filaments is crucial to the proper functioning of many eukaryotic cells. Since most proteins are synthesized at the cell body, mechanisms are required to deliver them to the growing periphery. In this article, we use computational modeling to study the strategies of protein transport in the context of JNK (c-JUN NH2-terminal kinase) transport along microtubules to the terminals of neuronal cells. One such strategy for protein transport is for the proteins of the JNK signaling cascade to bind to scaffolds, and to have the whole protein-scaffold cargo transported by kinesin motors along microtubules. We show how this strategy outperforms protein transport by diffusion alone, using metrics such as signaling rate and signal amplification. We find that there exists a range of scaffold concentrations for which JNK transport is optimal. Increase in scaffold concentration increases signaling rate and signal amplification but an excess of scaffolds results in the dilution of reactants. Similarly, there exists a range of kinesin motor speeds for which JNK transport is optimal. Signaling rate and signal amplification increases with kinesin motor speed until the speed of motor translocation becomes faster than kinase/scaffold-motor binding. Finally, we suggest experiments that can be performed to validate whether, in physiological conditions, neuronal cells do indeed adopt such an optimal strategy. Understanding cytoskeletal-assisted protein transport is crucial since axonal and cell body accumulation of organelles and proteins is a histological feature in many human neurodegenerative diseases. In this paper, we have shown that axonal transport performance changes with altered transport component concentrations and transport speeds wherein these aspects can be modulated to improve axonal efficiency and prevent or slowdown axonal deterioration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79647686dab07a1d661a9d84a256007318adbc75" target='_blank'>Computational Modeling Reveals Optimal Strategy for Kinase Transport by Microtubules to Nerve Terminals</a></td>
          <td>
            Yen Ling Koon, C. Koh, K. Chiam
          </td>
          <td>2014-04-01</td>
          <td>PLoS ONE</td>
          <td>5</td>
          <td>27</td>
        </tr>
    
        <tr id="For more than a decade, the MAPK (mitogen-activated protein kinase) cascade has been studied using mathematical modelling and quantitative experimentation [1]. The MAPK cascade relays the presence of extracellular stimuli such as growth hormones to the nucleus and controls the expression of hundreds of genes. MAPKs control major cell fate decisions such as proliferation, differentiation and apoptosis, mainly by inducing alterations in gene expression. In this chapter, we discuss how systems biology analysis provides insights into the functioning of this cascade. We show how this pathway assists the cell in responding properly to extracellular cues by filtering out sub-threshold stimuli, while efficiently transmitting physiologically relevant inputs. Several different receptors signal through the MAPK pathway even though they elicit opposite biological responses, thus raising the question of how specificity is achieved in MAPK signalling. Experimental studies revealed that specific biological responses are encoded by quantitative aspects of the MAPK signal such as amplitude or duration. We discuss mechanisms that enable the pathway to generate quantitatively different signals, and also explain how different signals are interpreted by the downstream gene expression machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6240690e740bb11ac1d42f43f91ffcfad059dbaf" target='_blank'>Systems analysis of MAPK signal transduction.</a></td>
          <td>
            N. Blüthgen, S. Legewie
          </td>
          <td>2008-09-30</td>
          <td>Essays in biochemistry</td>
          <td>68</td>
          <td>42</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a4cf812f83a2debae57b3aeae72356c1244f2a3" target='_blank'>A systems biology perspective on protein structural dynamics and signal transduction.</a></td>
          <td>
            F. Rousseau, J. Schymkowitz
          </td>
          <td>2005-02-01</td>
          <td>Current opinion in structural biology</td>
          <td>62</td>
          <td>59</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199632536955065b7efc9d1b12aaace0eb90f3e9" target='_blank'>Systems biology approaches to macromolecules: the role of dynamic protein assemblies in information processing.</a></td>
          <td>
            Oleksii S. Rukhlenko, B. Kholodenko, W. Kolch
          </td>
          <td>2020-10-23</td>
          <td>Current opinion in structural biology</td>
          <td>2</td>
          <td>86</td>
        </tr>
    
        <tr id="Embryonic development and adult tissue homeostasis are controlled through activation of intracellular signal transduction pathways by extracellular growth factors. In the past, signal transduction has largely been regarded as a linear process. However, more recent data from large‐scale and high‐throughput experiments indicate that there is extensive cross‐talk between individual signaling cascades leading to the notion of a signaling network. The behavior of such complex networks cannot be predicted by simple intuitive approaches but requires sophisticated models and computational simulations. The purpose of such models is to generate experimentally testable hypotheses and to find explanations for unexpected experimental results. Here, we discuss the need for, and the future impact of, mathematical models for exploring signal transduction in different biological contexts such as for example development. BioEssays 30:1110–1125, 2008. © 2008 Wiley Periodicals, Inc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8545f3dc8505b17afcd909dbddc0e64abbc53ce2" target='_blank'>Network modeling of signal transduction: establishing the global view</a></td>
          <td>
            H. Kestler, Christian Wawra, Barbara Kracher, M. Kühl
          </td>
          <td>2008-11-01</td>
          <td>BioEssays</td>
          <td>94</td>
          <td>56</td>
        </tr>
    
        <tr id="Graphical Abstract A comprehensive mechanistic understanding of cell signaling networks is crucial to elucidate biological function in human health and disease. The observed diversity of cell and tissue phenotypes result from the integration of subcellular components and molecular constituents, including subnetworks that each are predictable in isolation but cooperate in complex ways. Unraveling the dynamic role of each protein and pathway is experimentally challenging. Quantifying the individual protein-protein interaction events (e.g. binding kinetics, thermodynamics and post-translational modifications) and structure (receptors adopt multiple conformations to enhance or silence signaling activity) is invaluable in understanding recognition (specificity or promiscuity). But to understand whole network complexity, typical analytical assays that assess binary interactions and individual molecular states are insufficient, or resource-prohibitive at scale. High-throughput systems-level in vitro or in vivo experiments provide global and often quantitative insights into key signaling proteins, with results usually depicted as static signaling pathway connection maps. Spatio-temporal dynamics of protein signaling and gene regulatory networks need more than traditional biochemical measurements or systems-scale high-dimensional data alone.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f686a0c084f941a72e614e40389bd938f0ca42bf" target='_blank'>Computational Systems Biochemistry: Beyond the Static Interactome.</a></td>
          <td>
            S. Sarabipour, F. Mac Gabhann
          </td>
          <td>2018-01-09</td>
          <td>Biochemistry</td>
          <td>1</td>
          <td>31</td>
        </tr>
    
        <tr id="Cells integrate many inputs through complex networks of interacting signaling pathways. Systems approaches as well as computer-aided reductionist approaches attempt to "untangle the wires" and gain an intimate understanding of cells. But "understanding" any system is just the way that the human mind gains the ability to predict behavior. Computer simulations are an alternative way to achieve this goal--quite possibly the only way for complex systems. We have new tools to probe large sets of unknown interactions, and we have amassed enough detailed information to quantitatively describe many functional modules. Cell physiology has passed the threshold: the time to begin modeling is now.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c21a264fd9a79b61a366d39499498f73ac5931" target='_blank'>Intracellular signaling: spatial and temporal control.</a></td>
          <td>
            I. Moraru, L. Loew
          </td>
          <td>2005-06-01</td>
          <td>Physiology</td>
          <td>34</td>
          <td>62</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b06d234407efd97d798904ba97ebb9a73b5fb31e" target='_blank'>Methodologies for the modeling and simulation of biochemical networks, illustrated for signal transduction pathways: A primer</a></td>
          <td>
            Nesma ElKalaawy, A. Wassal
          </td>
          <td>2015-03-01</td>
          <td>Bio Systems</td>
          <td>20</td>
          <td>8</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/064578cc79a6606ce7ee464cc02cd148c3fe62eb" target='_blank'>Tony Pawson 1952–2013</a></td>
          <td>
            John D. Scott
          </td>
          <td>2013-10-01</td>
          <td>Nature Structural &Molecular Biology</td>
          <td>0</td>
          <td>31</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9beb2ca6ead42809864ef51fc5c3af9f30e28116" target='_blank'>Evolution of complex probability distributions in enzyme cascades.</a></td>
          <td>
            Y. Lan, G. Papoian
          </td>
          <td>2007-10-07</td>
          <td>Journal of theoretical biology</td>
          <td>10</td>
          <td>34</td>
        </tr>
    
        <tr id="Many biological signaling pathways employ proteins that competitively dimerize in diverse combinations. These dimerization networks can perform biochemical computations, in which the concentrations of monomers (inputs) determine the concentrations of dimers (outputs). Despite their prevalence, little is known about the range of input-output computations that dimerization networks can perform (their “expressivity”) and how it depends on network size and connectivity. Using a systematic computational approach, we demonstrate that even small dimerization networks (3-6 monomers) are expressive, performing diverse multi-input computations. Further, dimerization networks are versatile, performing different computations when their protein components are expressed at different levels, such as in different cell types. Remarkably, individual networks with random interaction affinities, when large enough (≥8 proteins), can perform nearly all (∼90%) potential one-input network computations merely by tuning their monomer expression levels. Thus, even the simple process of competitive dimerization provides a powerful architecture for multi-input, cell-type-specific signal processing. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86c5a8e378f7caf57c6480e9543301fef865b763" target='_blank'>Principles of Computation by Competitive Protein Dimerization Networks</a></td>
          <td>
            Jacob Parres-Gold, Matthew E. Levine, Benjamin L. Emert, Andrew Stuart, M. Elowitz
          </td>
          <td>2023-11-02</td>
          <td>bioRxiv</td>
          <td>7</td>
          <td>60</td>
        </tr>
    
        <tr id="Extra-cellular signal regulated kinase (ERK) signal network plays an important role in regulating the fundamental cellular functions such as cell proliferation, survival, differentiation and motility. Development and analysis of mathematical model can help us acquire a deep understanding of the complex behavior of ERK signal transduction network. This paper presents a computational model that offers an integrated quantitative and dynamic simulation of ERK signal transduction network, activated by epidermal growth factor (EGF). The mathematic model contains the activation kinetics of the pathway, a multitude of feedback loops and connection of scaffold proteins. The model provides insight into signal-response relationships between the binding of EGF to its receptor at the cell surface and activation of downstream proteins in the signaling cascade. The different influence of positive and negative feedback loops of the ERK signal transduction pathway were mainly analyzed, indicating that feedback loops were the main impacting factor to the oscillation of ERK signal transduction pathway. The predictions of this oscillation of ERK activation agree well with the literature. It can lead to signal waves of the downstream substrates and induce corresponding biological behaviors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7bf832c4d17a797f80bffbcb87cce1c1078a91f" target='_blank'>Dynamic Analysis of Feedback Loops in Extra-Cellular Signal Regulated Kinase Signal Network</a></td>
          <td>
            Yong Han, Yan Li, Yonghua Wang, H. Fang, Shuwei Zhang
          </td>
          <td>2009-03-31</td>
          <td>2009 WRI World Congress on Computer Science and Information Engineering</td>
          <td>3</td>
          <td>34</td>
        </tr>
    
        <tr id="Modeling of signal driven transcriptional reprogramming is critical for understanding of organism development, human disease, and cell biology. Many current modeling techniques discount key features of the biological sub-systems when modeling multiscale, organism-level processes. We present a mechanistic hybrid model, GESSA, which integrates a novel pooled probabilistic Boolean network model of cell signaling and a stochastic simulation of transcription and translation responding to a diffusion model of extracellular signals. We apply the model to simulate the well studied cell fate decision process of the vulval precursor cells (VPCs) in C. elegans, using experimentally derived rate constants wherever possible and shared parameters to avoid overfitting. We demonstrate that GESSA recovers (1) the effects of varying scaffold protein concentration on signal strength, (2) amplification of signals in expression, (3) the relative external ligand concentration in a known geometry, and (4) feedback in biochemical networks. We demonstrate that setting model parameters based on wild-type and LIN-12 loss-of-function mutants in C. elegans leads to correct prediction of a wide variety of mutants including partial penetrance of phenotypes. Moreover, the model is relatively insensitive to parameters, retaining the wild-type phenotype for a wide range of cell signaling rate parameters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91e24642ab074172f0fc8e6534215d48996eff7c" target='_blank'>Hybrid Modeling of Cell Signaling and Transcriptional Reprogramming and Its Application in C. elegans Development</a></td>
          <td>
            E. Fertig, Ludmila V. Danilova, Alexander V. Favorov, M. Ochs
          </td>
          <td>2011-08-24</td>
          <td>Frontiers in Genetics</td>
          <td>9</td>
          <td>49</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/387ed2dce130621d6a85d7cc5a97e4c9eb4c37a8" target='_blank'>Cellular Advantages to Signaling in a Digital World.</a></td>
          <td>
            Christopher P. Mancuso, S. Kiriakov, Ahmad S. Khalil
          </td>
          <td>2016-08-24</td>
          <td>Cell systems</td>
          <td>1</td>
          <td>34</td>
        </tr>
    
        <tr id="All cells receive and respond to signals from their environment, whether they live freely or are part of a tissue. Feedback loops are one among the interactions in signal-transduction cascades, in which the information from the downstream components is sent back to the upstream components of the system. Crosstalks between signal-transduction cascades are able to combine the information's received and processed to trigger the appropriate response. To understand how signaling networks make decisions, the dynamic interactions of proteins in the signal-transduction cascades are important rather than just creating static maps of pathways. Based on reaction kinetics, we have developed mathematical models to predict and analyze the impacts of the feedback loops in JNK cascades and the crosstalks between JNK and P38 kinase cascades. Feedback loops in JNK cascades and crosstalks between JNK and P38 kinase cascades plays the complementary roles in making the impacts on the activation of JNK and P38 kinases. JNK and P38 kinase cascades with feedbacks and crosstalks is a robust system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2880a4c0be242db0837a9696c152f250827dd584" target='_blank'>Dynamic Modeling and Simulation of JNK and P38 Kinase Cascades With Feedbacks and Crosstalks</a></td>
          <td>
            Pandurangan Sundaramurthy, S. Gakkhar
          </td>
          <td>2010-08-19</td>
          <td>IEEE Transactions on NanoBioscience</td>
          <td>7</td>
          <td>24</td>
        </tr>
    
        <tr id="Dynamics adds to the complexity of signaling networks. The biochemical processes that control the responses of cells to changes in their internal or external environments form a complex interconnected network. The molecules that participate in these processes are diverse and dynamic, and the networks themselves are affected by the flow of information through them. With this appreciation of the complexity in signaling networks, Science Signaling highlights computational and experimental approaches to understanding both the macroscopic cellular response and the microscopic signaling network and its constituents and dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8045d1bb90c38a9e81507bcbcfd412b519b768f7" target='_blank'>Focus Issue: Unraveling Signaling Complexity</a></td>
          <td>
            N. Gough, J. Foley
          </td>
          <td>2009-07-28</td>
          <td>Science Signaling</td>
          <td>5</td>
          <td>32</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dee9a3add69cd76d8a98df8d7bbf08919cc69ef7" target='_blank'>Review of "System Modeling in Cellular Biology: From Concepts to Nuts and Bolts" by Szallasi, Stelling and Periwal</a></td>
          <td>
            
          </td>
          <td>2007-01-30</td>
          <td>BioMedical Engineering OnLine</td>
          <td>1</td>
          <td>0</td>
        </tr>
    
        <tr id="Computational approaches can reveal how cells interpret, process, and respond to signals. This issue of Science Signaling, which complements the 13 April issue of Science, showcases the application of computational approaches to signal transduction research. Research and commentary highlight how mathematical modeling combined with experimental data can yield new insights into how cells interpret, process, and respond to external and internal cues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3ba07b97173020030f4ef6126bbe5e31f7fb242" target='_blank'>Focus Issue: Adding Math to the Signaling Toolkit</a></td>
          <td>
            W. Wong, J. Foley
          </td>
          <td>2012-04-17</td>
          <td>Science Signaling</td>
          <td>2</td>
          <td>32</td>
        </tr>
    
        <tr id="Modelling biological systems would be straightforward if we knew the structure of the model and the parameters governing their dynamics. For the overwhelming majority of biological processes, however, such parameter values are unknown and often impossible to measure directly. This means that we have to estimate or infer these parameters from observed data. Here we argue that it is also important to appreciate the uncertainty inherent in these estimates. We discuss a statistical approach--approximate Bayesian computation (ABC)--which allows us to approximate the posterior distribution over parameters and show how this can add insights into our understanding of the system dynamics. We illustrate the application of this approach and how the resulting posterior distribution can be analyzed in the context of the mitogen-activated protein kinase phosphorylation cascade. Our analysis also highlights the added benefit of using the distribution of parameters rather than point estimates of parameter values when considering the notion of sloppy models in systems biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1284603181ae4fa6130b2bd26a4e595d7e4e1af" target='_blank'>The ABC of reverse engineering biological signalling systems.</a></td>
          <td>
            Maria Secrier, Tina Toni, M. Stumpf
          </td>
          <td>2009-11-12</td>
          <td>Molecular bioSystems</td>
          <td>54</td>
          <td>56</td>
        </tr>
    
        <tr id="Different environmental stimuli often use the same set of signaling proteins to achieve very different physiological outcomes. The invasive growth and mating pathways in yeast Saccharomyces cerevisiae each employ a mitogen‐activated protein (MAP) kinase cascade that includes several common components (e.g. Ste11, Ste7), after which the pathways diverge leading to activation of different MAP kinases (Fus3 and Kss1, respectively). In order to determine how pathway specificity is achieved, we conducted a quantitative analysis of MAP kinase activity, and devised a series of six computational models that describe their behavior in vivo. As anticipated by one of the models, MAP kinase phosphorylation of shared pathway components results in diminished activation of one MAP kinase but not the other. These findings reveal how one MAP kinase can limit the activity of a competing MAP kinase and thereby ensure signal fidelity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a0034a8b7b900ff225c7b67b37624a166b457e" target='_blank'>Modeling signal specificity by feedback inhibition</a></td>
          <td>
            Nan Hao, Necmettin Yildirim, S. Parnell, B. Errede, T. Elston, H. Dohlman
          </td>
          <td>2007-04-01</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>54</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>