---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-17 18:05:08 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>A Dynamic Quantitative Systems Pharmacology Model of Inflammatory Bowel Disease: Part 1 – Model Framework</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Crohn's disease (CD) is a complex inflammatory bowel disease whose pathogenesis appears to involve several immunologic defects causing functional impairment of the gut. Its complexity and the reported loss of effectiveness over time of standard of care together with the increase in its worldwide incidence require the application of techniques aiming to find new therapeutic strategies. Currently, systems pharmacology modeling has been gaining importance as it integrates the available knowledge of the system into a single computational model. In this work, the following workflow for robust application of systems pharmacology modeling was followed: 1) scope definition; 2) species selection and circulating plasma levels based on a search in the literature; 3) representation of model topology and parametrization of the interactions, after literature data extraction and curation, and the implementation of ordinary differential equations in SimBiology (MATLAB version R2018b); and 4) model curation and evaluation by visual comparison of simulated interleukin (IL) concentrations with the reported levels in plasma, and sensitivity analysis performed to confirm model robustness and identify the most influential parameters. Finally, 5) exposure to two dose levels of recombinant human IL10 was evaluated by simulation and comparison with reported clinical study results. In summary, we present a quantitative systems pharmacology model for the main ILs involved in CD developed using a standardized methodology and supported by a comprehensive repository summarizing the most relevant literature in the field. However, it has to be taken into account that external validation is still pending as available clinical data were primarily used for model training. SIGNIFICANCE STATEMENT Crohn's disease (CD) is a complex heterogeneous inflammatory bowel disorder. Systems pharmacology modeling offers a great opportunity for integration of the available knowledge on the disease using a computational framework. As a result of this work, a comprehensive repository along with a quantitative systems pharmacology model for the main interleukins involved in CD is provided. This model is useful for the in silico evaluation of biomarkers and potential therapeutic targets and can be adapted to address research gaps regarding CD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1b59cbda9a63547cdc28a859ad8c140d010a08" target='_blank'>A Quantitative Systems Pharmacology Model for the Key Interleukins Involved in Crohn's Disease</a></td>
          <td>
            Violeta Balbas-Martinez, E. Asín-Prieto, Zinnia P. Parra-Guillen, I. Trocóniz
          </td>
          <td>2019-12-10</td>
          <td>The Journal of Pharmacology and Experimental Therapeutics</td>
          <td>8</td>
          <td>32</td>
        </tr>
    
        <tr id="Motivation The literature on complex diseases is abundant but not always quantitative. This is particularly so for Inflammatory Bowel Disease (IBD), where many molecular pathways are qualitatively well described but this information cannot be used in traditional quantitative mathematical models employed in drug development. We propose the elaboration and validation of a logic network for IBD able to capture the information available in the literature that will facilitate the identification/validation of therapeutic targets. Results In this article, we propose a logic model for Inflammatory Bowel Disease (IBD) which consists of 43 nodes and 298 qualitative interactions. The model presented is able to describe the pathogenic mechanisms of the disorder and qualitatively describes the characteristic chronic inflammation. A perturbation analysis performed on the IBD network indicates that the model is robust. Also, as described in clinical trials, a simulation of anti-TNFα, anti-IL2 and Granulocyte and Monocyte Apheresis showed a decrease in the Metalloproteinases node (MMPs), which means a decrease in tissue damage. In contrast, as clinical trials have demonstrated, a simulation of anti-IL17 and anti-IFNγ or IL10 overexpression therapy did not show any major change in MMPs expression, as corresponds to a failed therapy. The model proved to be a promising in silico tool for the evaluation of potential therapeutic targets, the identification of new IBD biomarkers, the integration of IBD polymorphisms to anticipate responders and non-responders and can be reduced and transformed in quantitative model/s.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c655f04a31d1dc93671152d749f2e63985c7bf3" target='_blank'>A systems pharmacology model for inflammatory bowel disease</a></td>
          <td>
            Violeta Balbas-Martinez, Leire Ruiz-Cerdá, I. Irurzun-Arana, I. González-García, A. Vermeulen, J. Gómez-Mantilla, I. Trocóniz
          </td>
          <td>2018-03-07</td>
          <td>PLoS ONE</td>
          <td>31</td>
          <td>32</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b6d19e4a519ae3668404d76c1f20c8c5b12e928" target='_blank'>Multiscale modeling of mucosal immune responses</a></td>
          <td>
            Yongguo Mei, V. Abedi, Adria Carbo, Xiaoying Zhang, Pinyi Lu, C. Philipson, R. Hontecillas, S. Hoops, Nathan Liles, J. Bassaganya-Riera
          </td>
          <td>2015-08-25</td>
          <td>BMC Bioinformatics</td>
          <td>34</td>
          <td>53</td>
        </tr>
    
        <tr id="
 
 
 The aim of this project is to model Inflammatory Bowel Diseases (IBD) progression using an innovative approach that considers the manifestation of the disease from a dynamic and multifactorial point of view, with a focus on model explainability.
 
 
 
 With the aim of modelling the course of IBD in the study population of the UR-CARE registry, an innovative approach will be applied consisting of the combined use of two different data-driven Artificial Intelligence techniques, namely dynamic Bayesian networks and Process Mining. Specifically, these two approaches will be jointly used to model IBD progression trajectories, providing a broader overview of the disease through the description of its patterns of progression (including clinical events, treatments and/or outcomes) and the interactions between clinical variables. The potential of the proposed methodology to address the predictive needs of chronic IBD progression, such as the forecasting of the next relapse, the effect of a therapy, or the impact of a risk factor, will also be explored.
 
 
 
 On the one side, IBD patients experience constant uncertainty regarding disease progression, while clinicians, on the other hand, need tools that can support them in understanding the multidimensionality of disease progression. In this scenario, AI can be the key to successfully satisfy these needs, effectively investigating the disease processes, allowing to describe pathological evolution over time, handling and capturing patients’ inter-variability, and providing tools to forecast disease evolution. By adopting an ad-hoc developed analytic approach, this project can help in better understanding IBD mechanisms and best care strategies, defining the relationships among the patient characteristics and the sequence of experienced clinical events, evaluating the impact of key elements on disease’s prognosis. These results would be useful not only to better manage the disease from a clinical and care point of view, but also in economic terms.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9f05d40c9251072474dc926cd10de52d0222352" target='_blank'>IIS Registry Grant Modelling Inflammatory Bowel Diseases trajectories combining dynamic, multifactorial, Artificial Intelligence-based approaches</a></td>
          <td>
            R. I. Comoretto, E. Tavazzi, M. Martinato, D. Azzolina
          </td>
          <td>2024-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>0</td>
          <td>1</td>
        </tr>
    
        <tr id="Ulcerative colitis, an inflammatory bowel disease, is a chronic inflammatory disorder that results in ulcers of the colon and rectum without known etiology.Ulcerative colitis causes a huge public health care burden particularly in developed countries.Many studies suggest that ulcerative colitis results from an abnormal immune response against components of commensal microbiota in genetically susceptible individuals.However, understanding of the disease mechanisms at cellular and molecular levels remains largely elusive.In this paper, a network model is developed based on our previous study and computer simulations are performed using an agent-based network modeling to elucidate the dynamics of immune response in ulcerative colitis progression.Our modeling study identifies several important positive feedback loops as a driving force for ulcerative colitis initiation and progression.The results demonstrate that although immune response in ulcerative colitis patients is dominated by anti-inflammatory/regulatory cells such as alternatively activated macrophages and type Ⅱ natural killer T cells, proinflammatory cells including classically activated macrophages, T helper 1 and T helper 17 cells, and their secreted cytokines tumor necrosis factor-α, interleukin-12, interleukin-23, interleukin-17 and interferon-γ remain at certain levels (lower than those in Crohn's disease, another inflammatory bowel disease).Long-term exposure to these proinflammatory components, causes mucosal tissue damage persistently, leading to ulcerative colitis.Our simulation results are qualitatively in agreement with clinical and laboratory measurements, offering novel insight into the disease mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78e59dd74ab5b9b66ead438b5b446f397142aed6" target='_blank'>Agent-Based Network Modeling Study of Immune Responses in Progression of Ulcerative Colitis</a></td>
          <td>
            Dao-rong Wu, Hai-shan Yu, J. Liao
          </td>
          <td>2018-04-27</td>
          <td>Chinese Journal of Chemical Physics</td>
          <td>0</td>
          <td>7</td>
        </tr>
    
        <tr id="During acute and chronic infectious diseases hosts develop complex immune responses to cope with bacterial persistence. Depending on a variety of host and microbe factors, outcomes range from peaceful co-existence to detrimental disease. Mechanisms underlying immunity to bacterial stimuli span several spatiotemporal magnitudes and the summation of these hierarchical interactions plays a decisive role in pathogenic versus tolerogenic fate for the host. This dissertation integrates diverse data from immunoinformatics analyses, experimental validation and mathematical modeling to investigate a series of hypotheses driven by computational modeling to study mucosal immunity. Two contrasting microbes, enteroaggregative Escherichia coli and Helicobacter pylori, are used to perturb gut immunity in order to discover host-centric targets for modulating the host immune system. These findings have the potential to be broadly applicable to other infectious and immune-mediated diseases and could assist in the development of antibiotic-free and host-targeted treatments that modulate tolerance to prevent disease. This work was supported in part by the National Institute of Allergy and Infectious Diseases Contract No. HHSN272201000056C to Josep Bassaganya-Riera and funds from the Nutritional Immunology and Molecular Medicine Laboratory. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6533733f66c542d395ec6d41c0b1a59550a2e544" target='_blank'>Systems analysis and characterization of mucosal immunity</a></td>
          <td>
            C. Philipson
          </td>
          <td>2015-07-28</td>
          <td></td>
          <td>0</td>
          <td>19</td>
        </tr>
    
        <tr id="Managing inflammatory bowel disease (IBD) is a major challenge for physicians and patients during the COVID-19 pandemic. To understand the impact of the pandemic on patient behaviors and disruptions in medical care, we used a combination of population-based modeling, system dynamics simulation, and linear optimization. Synthetic IBD populations in Tokyo and Hokkaido were created by localizing an existing US-based synthetic IBD population using data from the Ministry of Health, Labor, and Welfare in Japan. A clinical pathway of IBD-specific disease progression was constructed and calibrated using longitudinal claims data from JMDC Inc for patients with IBD before and during the COVID-19 pandemic. Key points considered for disruptions in patient behavior (demand) and medical care (supply) were diagnosis of new patients, clinic visits for new patients seeking care and diagnosed patients receiving continuous care, number of procedures, and the interval between procedures or biologic prescriptions. COVID-19 had a large initial impact and subsequent smaller impacts on demand and supply despite higher infection rates. Our population model (Behavior Predictor) and patient treatment simulation model (Demand Simulator) represent the dynamics of clinical care demand among patients with IBD in Japan, both in recapitulating historical demand curves and simulating future demand during disruption scenarios, such as pandemic, earthquake, and economic crisis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2eba10c40af9c8c70960855ac4b2db4f66c0bdc" target='_blank'>Computational Simulation Model to Predict Behavior Changes in Inflammatory Bowel Disease Patients during the COVID-19 Pandemic: Analysis of Two Regional Japanese Populations</a></td>
          <td>
            Gen Suzuki, R. Iwakiri, E. Udagawa, Sindy Ma, Ryoko Takayama, H. Nishiura, K. Nakamura, S. P. Burns, P. M. D’Alessandro, Jovelle L Fernandez
          </td>
          <td>2023-01-18</td>
          <td>Journal of Clinical Medicine</td>
          <td>1</td>
          <td>61</td>
        </tr>
    
        <tr id="Inflammatory bowel diseases (IBD) are chronic and recurrent inflammatory disorders of the gastrointestinal tract. The elucidation of their etiopathology requires complex and multiple approaches. Systems biology has come to fulfill this need in approaching the pathogenetic mechanisms of IBD and its etiopathology, in a comprehensive way, by combining data from different scientific sources. In combination with bioinformatics and network medicine, it uses principles from computer science, mathematics, physics, chemistry, biology, medicine and computational tools to achieve its purposes. Systems biology utilizes scientific sources that provide data from omics studies (e.g., genomics, transcriptomics, etc.) and clinical observations, whose combined analysis leads to network formation and ultimately to a more integrative image of disease etiopathogenesis. In this review, we analyze the current literature on the methods and the tools utilized by systems biology in order to cover an innovative and exciting field: IBD-omics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07cfd2ecf64a7bb7c4f9450a909d9737a93ec2b1" target='_blank'>Systems biology in inflammatory bowel diseases: on the way to precision medicine</a></td>
          <td>
            N. Dovrolis, E. Filidou, G. Kolios
          </td>
          <td>2019-04-03</td>
          <td>Annals of Gastroenterology</td>
          <td>11</td>
          <td>42</td>
        </tr>
    
        <tr id="Inflammatory bowel disease (IBD) is a chronic and relapsing disease with multiple underlying influences and notable heterogeneity among its clinical and response‐to‐treatment phenotypes. There is no cure for IBD, and none of the currently available therapies have demonstrated clinical efficacies beyond 40%–60%. Data collected about its omics, pathogenesis, and treatment strategies have grown exponentially with time making IBD a prime candidate for artificial intelligence (AI) mediated discovery support. AI can be leveraged to further understand or identify IBD features to improve clinical outcomes. Various treatment candidates are currently under evaluation in clinical trials, offering further approaches and opportunities for increasing the efficacies of treatments. However, currently, therapeutic plans are largely determined using clinical features due to the lack of specific biomarkers, and it has become necessary to step into precision medicine to predict therapeutic responses to guarantee optimal treatment efficacy. This is accompanied by the application of AI and the development of multiscale hybrid models combining mechanistic approaches and machine learning. These models ultimately lead to the creation of digital twins of given patients delivering on the promise of precision dosing and tailored treatment. Interleukin‐6 (IL‐6) is a prominent cytokine in cell‐to‐cell communication in the inflammatory responses’ regulation. Dysregulated IL‐6‐induced signaling leads to severe immunological or proliferative pathologies, such as IBD and colon cancer. This mini‐review explores multiscale models with the aim of predicting the response to therapy in IBD. Modeling IL‐6 biology and generating digital twins enhance the credibility of their prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89edceb6395ad6a9a46a561fb37a23bbccd0f06b" target='_blank'>Computational models in inflammatory bowel disease</a></td>
          <td>
            P. Pinton
          </td>
          <td>2022-02-05</td>
          <td>Clinical and Translational Science</td>
          <td>5</td>
          <td>40</td>
        </tr>
    
        <tr id="
 Here we present the newly developed ENteric Immunity SImulator (ENISI) that simulates the antagonistic inflammatory and regulatory immune pathways of the gut as individual immune cells interact with and respond to commensal and foreign bacteria. This tool has been used to reproduce a typical inflammatory response to foreign bacteria as well as the immunopathological effects of autoimmunity to commensal bacteria with 10^6 individual cells. ENISI is built on an agent-based model that incorporates spatial effects and randomness of cell-cell and cell-bacteria contact, which can have a significant impact on whether a triggered inflammatory cascade is successfully regulated or spirals out of control. Preliminary simulations indicate the importance of cell density and epithelial cell-mediated recruitment in determining gut immunopathology. In addition, such representation allows emergent properties such as changes in bacterial demographics and evolution as a result of interaction with host immune cells. This tool will be useful to i) test the plausibility of in vitro observed immune cell responses as explanations for tissue-level damage, ii) propose mechanisms not yet tested in vitro that could explain tissue-level phenomenon, iii) conduct low-cost, preliminary tests of proposed interventions/ treatments for inflammatory diseases, iv) identify specific data to be gathered experimentally in order to characterize mechanisms of immune modulation in the gut.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac1d9b20e9f5580f0a85140fb3b01997e5b68824" target='_blank'>ENteric Immunity SImulator: A tool for in silico study of gut immunopathologies (166.15)</a></td>
          <td>
            K. Wendelsdorf, J. Bassaganya-Riera, Keith Bisset, Stephen Eubank, R. Hontecillas, Madhav V. Marathe
          </td>
          <td>2011-04-01</td>
          <td>The Journal of Immunology</td>
          <td>2</td>
          <td>53</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>