---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-17 18:05:45 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>A systems biology approach to model neural stem cell regulation by notch, shh, wnt, and EGF signaling pathways.</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Recent advancements in machine learning-based data processing techniques have facilitated the inference of gene regulatory interactions and the identification of key genes from multidimensional gene expression data. In this study, we applied a stepwise Bayesian framework to uncover a novel regulatory component involved in differentiation of specific neural and neuronal cells. We treated naive neural precursor cells with Sonic Hedgehog (Shh) at various concentrations and time points, generating comprehensive whole-genome sequencing data that captured dynamic gene expression profiles during differentiation. The genes were categorized into 224 subsets based on their expression profiles, and the relationships between these subsets were extrapolated. To accurately predict gene regulation among subsets, known networks were used as a core model and subsets to be added were tested stepwise. This approach led to the identification of a novel component involved in neural tube patterning within gene regulatory networks (GRNs), which was experimentally validated. Our study highlights the effectiveness of in silico modeling for extrapolating GRNs during neural development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c4c4e2c6b28e65474d7f5776d1bf876b03e2372" target='_blank'>Stepwise Bayesian Machine Learning Uncovers a Novel Gene Regulatory Network Component in Neural Tube Development</a></td>
          <td>
            Chen Xing, Y. Sakumura, Toshiya Kokaji, Katsuyuki Kunida, Noriaki Sasai
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>
    
        <tr id="Elucidation of the mechanisms of stem cell differentiation is of great scientific interest. Increasing evidence suggests that stem cell differentiation involves changes at multiple levels of biological regulation, which together orchestrate the complex differentiation process; many related studies have been performed to investigate the various levels of regulation. The resulting valuable data, however, remain scattered. Most of the current stem cell-relevant databases focus on a single level of regulation (mRNA expression) from limited stem cell types; thus, a unifying resource would be of great value to compile the multiple levels of research data available. Here we present a database for this purpose, SyStemCell, deposited with multi-level experimental data from stem cell research. The database currently covers seven levels of stem cell differentiation-associated regulatory mechanisms, including DNA CpG 5-hydroxymethylcytosine/methylation, histone modification, transcript products, microRNA-based regulation, protein products, phosphorylation proteins and transcription factor regulation, all of which have been curated from 285 peer-reviewed publications selected from PubMed. The database contains 43,434 genes, recorded as 942,221 gene entries, for four organisms (Homo sapiens, Mus musculus, Rattus norvegicus, and Macaca mulatta) and various stem cell sources (e.g., embryonic stem cells, neural stem cells and induced pluripotent stem cells). Data in SyStemCell can be queried by Entrez gene ID, symbol, alias, or browsed by specific stem cell type at each level of genetic regulation. An online analysis tool is integrated to assist researchers to mine potential relationships among different regulations, and the potential usage of the database is demonstrated by three case studies. SyStemCell is the first database to bridge multi-level experimental information of stem cell studies, which can become an important reference resource for stem cell researchers. The database is available at http://lifecenter.sgst.cn/SyStemCell/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/231d6fb2a99d6dae3b68122f4268c74aee31249c" target='_blank'>SyStemCell: A Database Populated with Multiple Levels of Experimental Data from Stem Cell Differentiation Research</a></td>
          <td>
            Jian Yu, Xiaobin Xing, Lingyao Zeng, Jiehuan Sun, Wei Li, Han Sun, Ying He, Jing Li, Guoqing Zhang, Chuan Wang, Yixue Li, Lu Xie
          </td>
          <td>2012-07-13</td>
          <td>PLoS ONE</td>
          <td>12</td>
          <td>35</td>
        </tr>
    
        <tr id="In May this year, Stockholm hosted a Keystone Symposium on Adult Neurogenesis, attracting scientists from around the world despite the lack of customary snow. The symposium offered an extraordinary program, covering diverse topics that ranged from the neural stem cell lineage and regulation of neurogenesis to functional aspects of neurogenesis in homeostasis and disease, and even computational modeling. This Meeting Review describes some of the exciting presentations and emerging themes from the symposium, which reveal how much this young field has matured.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce0dc6536dbbe98ae29901e5f27e3536f2edcb21" target='_blank'>Adult neurogenesis: taking stock in Stockholm</a></td>
          <td>
            Nambirajan Govindarajan, G. Kempermann
          </td>
          <td>2014-10-01</td>
          <td>Development</td>
          <td>1</td>
          <td>94</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b819b59714b1800d4f5b5c0478318b9c27f766de" target='_blank'>A Generalized Gene-Regulatory Network Model of Stem Cell Differentiation for Predicting Lineage Specifiers</a></td>
          <td>
            S. Okawa, Sarah Nicklas, Sascha Zickenrott, J. Schwamborn, A. del Sol
          </td>
          <td>2016-08-18</td>
          <td>Stem Cell Reports</td>
          <td>40</td>
          <td>43</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56469def4a189e7b2e7f86809eb4689acecb0eb1" target='_blank'>Odefy -- From discrete to continuous models</a></td>
          <td>
            J. Krumsiek, Sebastian PÃ¶lsterl, Dominik M. Wittmann, Fabian J Theis
          </td>
          <td>2010-05-07</td>
          <td>BMC Bioinformatics</td>
          <td>115</td>
          <td>98</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62eafbe01f4eeaf573d3daac5a39118bde6bade3" target='_blank'>Exploring Notch Pathway to Elucidate Phenotypic Plasticity and Intra-tumor Heterogeneity in Gliomas</a></td>
          <td>
            S. Chowdhury, R. Sarkar
          </td>
          <td>2019-07-01</td>
          <td>Scientific Reports</td>
          <td>5</td>
          <td>21</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>