---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-29 08:54:01 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Principles behind the multifarious control of signal transduction</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="For more than a decade, the MAPK (mitogen-activated protein kinase) cascade has been studied using mathematical modelling and quantitative experimentation [1]. The MAPK cascade relays the presence of extracellular stimuli such as growth hormones to the nucleus and controls the expression of hundreds of genes. MAPKs control major cell fate decisions such as proliferation, differentiation and apoptosis, mainly by inducing alterations in gene expression. In this chapter, we discuss how systems biology analysis provides insights into the functioning of this cascade. We show how this pathway assists the cell in responding properly to extracellular cues by filtering out sub-threshold stimuli, while efficiently transmitting physiologically relevant inputs. Several different receptors signal through the MAPK pathway even though they elicit opposite biological responses, thus raising the question of how specificity is achieved in MAPK signalling. Experimental studies revealed that specific biological responses are encoded by quantitative aspects of the MAPK signal such as amplitude or duration. We discuss mechanisms that enable the pathway to generate quantitatively different signals, and also explain how different signals are interpreted by the downstream gene expression machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6240690e740bb11ac1d42f43f91ffcfad059dbaf" target='_blank'>Systems analysis of MAPK signal transduction.</a></td>
          <td>
            N. Blüthgen, S. Legewie
          </td>
          <td>2008-09-30</td>
          <td>Essays in biochemistry</td>
          <td>68</td>
          <td>42</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da87720f4767a44da3f92e497492607a5e8a082c" target='_blank'>When kinases meet mathematics: the systems biology of MAPK signalling</a></td>
          <td>
            W. Kolch, Muffy Calder, David R. Gilbert
          </td>
          <td>2005-03-21</td>
          <td>FEBS Letters</td>
          <td>190</td>
          <td>86</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b06d234407efd97d798904ba97ebb9a73b5fb31e" target='_blank'>Methodologies for the modeling and simulation of biochemical networks, illustrated for signal transduction pathways: A primer</a></td>
          <td>
            Nesma ElKalaawy, A. Wassal
          </td>
          <td>2015-03-01</td>
          <td>Bio Systems</td>
          <td>20</td>
          <td>8</td>
        </tr>
    
        <tr id="Mitogen‐activated protein kinase (MAPK) cascades process a myriad of stimuli received by cell‐surface receptors and generate precise spatiotemporal guidance for multiple target proteins, dictating receptor‐specific cellular outcomes. Computational modeling reveals that the intrinsic topology of MAPK cascades enables them to amplify signal sensitivity and amplitude, reduce noise, and display intricate dynamic properties, which include toggle switches, excitation pulses, and oscillations. Specificity of signaling responses can be brought about by signal‐induced feedback and feedforward wiring imposed on the MAPK cascade backbone. Intracellular gradients of protein activities arise from the spatial separation of opposing reactions in kinase‐phosphatase cycles. The membrane confinement of the initiating kinase in MAPK cascades and cytosolic localization of phosphatases can result in precipitous gradients of phosphorylated signal‐transducers if they spread solely by diffusion. Endocytotic trafficking of active kinases driven by molecular motors and traveling waves of protein phosphorylation can propagate phosphorylation signals from the plasma membrane to the nucleus, especially in large cells, such as Xenopus eggs. Copyright © 2009 John Wiley & Sons, Inc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/549f0ea398377e29460e84284cf31387d6568803" target='_blank'>Four‐dimensional dynamics of MAPK information‐processing systems</a></td>
          <td>
            B. Kholodenko, M. Birtwistle
          </td>
          <td>2009-07-01</td>
          <td>Wiley Interdisciplinary Reviews: Systems Biology and Medicine</td>
          <td>87</td>
          <td>57</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8794a6dee41d037835f81c214643f20f1ec20aee" target='_blank'>How quantitative measures unravel design principles in multi-stage phosphorylation cascades.</a></td>
          <td>
            Simone Frey, Thomas Millat, S. Hohmann, O. Wolkenhauer
          </td>
          <td>2008-09-07</td>
          <td>Journal of theoretical biology</td>
          <td>7</td>
          <td>59</td>
        </tr>
    
        <tr id="Cells integrate many inputs through complex networks of interacting signaling pathways. Systems approaches as well as computer-aided reductionist approaches attempt to "untangle the wires" and gain an intimate understanding of cells. But "understanding" any system is just the way that the human mind gains the ability to predict behavior. Computer simulations are an alternative way to achieve this goal--quite possibly the only way for complex systems. We have new tools to probe large sets of unknown interactions, and we have amassed enough detailed information to quantitatively describe many functional modules. Cell physiology has passed the threshold: the time to begin modeling is now.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c21a264fd9a79b61a366d39499498f73ac5931" target='_blank'>Intracellular signaling: spatial and temporal control.</a></td>
          <td>
            I. Moraru, L. Loew
          </td>
          <td>2005-06-01</td>
          <td>Physiology</td>
          <td>34</td>
          <td>62</td>
        </tr>
    
        <tr id="Cell responses are actuated by tightly controlled signal transduction pathways. Although the concept of an integrated signaling network replete with interpathway cross-talk and feedback regulation is broadly appreciated, kinetic data of the type needed to characterize such interactions in conjunction with mathematical models are lacking. In mammalian cells, the Ras/ERK pathway controls cell proliferation and other responses stimulated by growth factors, and several cross-talk and feedback mechanisms affecting its activation have been identified. In this work, we take a systematic approach to parse the magnitudes of multiple regulatory mechanisms that attenuate ERK activation through canonical (Ras-dependent) and non-canonical (PI3K-dependent) pathways. In addition to regulation of receptor and ligand levels, we consider three layers of ERK-dependent feedback: desensitization of Ras activation, negative regulation of MEK kinase (e.g. Raf) activities, and up-regulation of dual-specificity ERK phosphatases. Our results establish the second of these as the dominant mode of ERK self-regulation in mouse fibroblasts. We further demonstrate that kinetic models of signaling networks, trained on a sufficient diversity of quantitative data, can be reasonably comprehensive, accurate, and predictive in the dynamical sense.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405be6d692d4836d3b3efde9333fdf1789ff05c7" target='_blank'>Systematic Quantification of Negative Feedback Mechanisms in the Extracellular Signal-regulated Kinase (ERK) Signaling Network*</a></td>
          <td>
            M. Cirit, Chun-Chao Wang, J. Haugh
          </td>
          <td>2010-09-16</td>
          <td>The Journal of Biological Chemistry</td>
          <td>91</td>
          <td>33</td>
        </tr>
    
        <tr id="Cell growth critically depends on signalling pathways whose regulation is the focus of intense research. Without utilizing a priori knowledge of the relative importance of pathway components, we have applied in silico computational methods to the EGF-induced MAPK cascade. Specifically, we systematically perturbed the entire parameter space, including initial conditions, using a Monte Carlo approach, and investigate which protein components or kinetic reaction steps contribute to the differentiation of ERK responses. The model, based on previous work by Brightman and Fell (2000), is composed of 28 reactions, 27 protein molecules, and 48 parameters from both mass action and Michaelis-Menten kinetics. Our multi-parametric systems analysis confirms that Raf inactivation is one of the key steps regulating ERK responses to be either transient or sustained. Furthermore, the results of amplitude-differential ERK phosphorylations within the transient case are mainly attributed to the balance between activation and inactivation of Ras while duration-differential ERK responses for the sustained case are, in addition to Ras, markedly affected by dephospho-/phosphorylation of both MEK and ERK. Our sub-module perturbations showed that MEK and ERK's contribution to this differential ERK activation originates from fluctuations in intermediate pathway module components such as Ras and Raf, implicating a cooperative regulatory mode among the key components. The initial protein concentrations of corresponding reactions such as Ras, GAP, and Raf also influence the distinct signalling outputs of ERK activation. We then compare these results with those obtained from a single-parametric perturbation approach using an overall state sensitivity (OSS) analysis. The OSS findings indicate a more pronounced role of ERK's inhibitory feedback effect on catalysing the dissociation of the SOS complex. Both approaches reveal the presence of multiple specific reactions involved in the distinct dynamics of ERK responses and the cell fate decisions they trigger. This work adds a mechanistic insight of the contribution of key pathway components, thus may support the identification of biomarkers for pharmaceutical drug discovery processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0951d8155ada6c35ec896fb6208894bfd088af2" target='_blank'>Investigating Differential Dynamics of the MAPK Signaling Cascade Using a Multi-Parametric Global Sensitivity Analysis</a></td>
          <td>
            Jeongah Yoon, T. Deisboeck
          </td>
          <td>2009-02-23</td>
          <td>PLoS ONE</td>
          <td>40</td>
          <td>40</td>
        </tr>
    
        <tr id="Cell responses are actuated by tightly controlled signal transduction pathways. Although the concept of an integrated signaling network replete with interpathway cross-talk and feedback regulation is broadly appreciated, kinetic data of the type needed to characterize such interactions in conjunction with mathematical models are lacking. In mammalian cells, the Ras/ERK pathway controls cell proliferation and other responses stimulated by growth factors, and several cross-talk and feedback mechanisms affecting its activation have been identified. In this work, we take a systematic approach to parse the magnitudes of multiple regulatory mechanisms that attenuate ERK activation through canonical (Ras-dependent) and non-canonical (PI3K-dependent) pathways. In addition to regulation of receptor and ligand levels, we consider three layers of ERK-dependent feedback: desensitization of Ras activation, negative regulation of MEK kinase (e.g. Raf) activities, and up-regulation of dual-specificity ERK phosphatases. Our results establish the second of these as the dominant mode of ERK self-regulation in mouse fibroblasts. We further demonstrate that kinetic models of signaling networks, trained on a sufficient diversity of quantitative data, can be reasonably comprehensive, accurate, and predictive in the dynamical sense.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405be6d692d4836d3b3efde9333fdf1789ff05c7" target='_blank'>Systematic Quantification of Negative Feedback Mechanisms in the Extracellular Signal-regulated Kinase (ERK) Signaling Network*</a></td>
          <td>
            M. Cirit, Chun-Chao Wang, J. Haugh
          </td>
          <td>2010-09-16</td>
          <td>The Journal of Biological Chemistry</td>
          <td>91</td>
          <td>33</td>
        </tr>
    
        <tr id="Recent work has generated whole-cell and whole-process models capable of predicting phenotype in simple organisms. The approaches used are hindered in higher organisms and more-complex cells by a lack of kinetic parameters for reactions and events, and the difficulty of measuring and estimating these. Here, we outline a large, two-process model capable of predicting the effects of gene expression on a signal transduction network. Our method models signal transduction and the processes involved in gene expression as two separate systems, solved iteratively. We show that this approach is sufficient to capture functionally significant behaviour resulting from common network motifs. We further demonstrate that our method is scalable and efficient to the size of the largest signal transduction databases currently available. This approach enables analysis and prediction in the absence of kinetic data, but is itself held back by the lack of detailed large-scale gene expression models. However, research consortia such as ENCODE and FANTOM are rapidly adding to the knowledge of transcriptional regulation, and we anticipate that incorporating this data into our regulatory model could allow the modelling of complex cellular phenomena such as the structured progression seen in cellular differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4b902f193dc7e94a88fdff58390d34b9008d89" target='_blank'>Towards a Large Integrated Model of Signal Transduction and Gene Regulation Events in Mammalian Cells</a></td>
          <td>
            Liam G. Fearnley, M. Ragan, L. Nielsen
          </td>
          <td>2014-03-03</td>
          <td>MAG, DBLP</td>
          <td>0</td>
          <td>66</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6319b79882fa8bd07267c8cafc082740cd9779b6" target='_blank'>Computational modeling of the EGFR network elucidates control mechanisms regulating signal dynamics</a></td>
          <td>
            Dennis Wang, L. Cardelli, Andrew Phillips, Nir Piterman, J. Fisher
          </td>
          <td>2009-12-22</td>
          <td>BMC Systems Biology</td>
          <td>40</td>
          <td>73</td>
        </tr>
    
        <tr id="Biphasic responses are encountered at all levels in biological systems. At the cellular level, biphasic dose-responses are widely encountered in cell signaling and post-translational modification systems and represent safeguards against overactivation or overexpression of species. In this paper, we provide a unified theoretical synthesis of biphasic responses in cell signaling systems, by assessing signaling systems ranging from basic biochemical building blocks to canonical network structures to well-characterized exemplars on one hand, and examining different types of doses on the other. By using analytical and computational approaches applied to a range of systems across levels (described by broadly employed models), we reveal (i) design principles enabling the presence of biphasic responses, including in almost all instances, an explicit characterization of the parameter space (ii) structural factors which preclude the possibility of biphasic responses (iii) different combinations of the presence or absence of enzyme-biphasic and substrate-biphasic responses, representing safeguards against overactivation and overexpression, respectively (iv) the possibility of broadly robust biphasic responses (v) the complete alteration of signaling behavior in a network due to biphasic interactions between species (biphasic regulation) (vi) the propensity of different co-existing biphasic responses in the Erk signaling network. These results both individually and in totality have a number of important consequences for systems and synthetic biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04cbc7a5b2b069e8271048cbc768ee98bc5cd44e" target='_blank'>A unified approach to dissecting biphasic responses in cell signaling</a></td>
          <td>
            Vaidhiswaran Ramesh, J. Krishnan
          </td>
          <td>2023-12-06</td>
          <td>eLife</td>
          <td>0</td>
          <td>2</td>
        </tr>
    
        <tr id="Intracellular signaling pathways that share common components often elicit distinct physiological responses. In most cases, the biochemical mechanisms responsible for this signal specificity remain poorly understood. Protein scaffolds and cross-inhibition have been proposed as strategies to prevent unwanted cross-talk. Here, we report a mechanism for signal specificity termed “kinetic insulation.” In this approach signals are selectively transmitted through the appropriate pathway based on their temporal profile. In particular, we demonstrate how pathway architectures downstream of a common component can be designed to efficiently separate transient signals from signals that increase slowly over time. Furthermore, we demonstrate that upstream signaling proteins can generate the appropriate input to the common pathway component regardless of the temporal profile of the external stimulus. Our results suggest that multilevel signaling cascades may have evolved to modulate the temporal profile of pathway activity so that stimulus information can be efficiently encoded and transmitted while ensuring signal specificity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34f99baf2a3fce2ef002f0b3da1545674a8824cc" target='_blank'>Kinetic insulation as an effective mechanism for achieving pathway specificity in intracellular signaling networks</a></td>
          <td>
            M. Behar, H. Dohlman, T. Elston
          </td>
          <td>2007-10-09</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>87</td>
          <td>54</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96df4b032feacafb2fb594f9048a1ee018956867" target='_blank'>Implication of dynamics in signal transduction and targeted disruption analyses of signaling networks</a></td>
          <td>
            F. Vital-Lopez, Amit Varshney, C. Maranas, A. Armaou
          </td>
          <td>2008-09-26</td>
          <td>Comput. Chem. Eng.</td>
          <td>2</td>
          <td>77</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6a045551eda8ac4bf3c6dd072aafd66f5f9d95" target='_blank'>Simplistic pathways or complex networks?</a></td>
          <td>
            C. Jørgensen, R. Linding
          </td>
          <td>2010-02-01</td>
          <td>Current opinion in genetics & development</td>
          <td>40</td>
          <td>40</td>
        </tr>
    
        <tr id="One challenge in biology is to make sense of the complexity of biological networks. A good system to approach this is signaling pathways, whose well-characterized molecular details allow us to relate the internal processes of each pathway to their input-output behavior. In this study, we analyzed mathematical models of three metazoan signaling pathways: the canonical Wnt, MAPK/ERK, and Tgfβ pathways. We find an unexpected convergence: the three pathways behave in some physiological contexts as linear signal transmitters. Testing the results experimentally, we present direct measurements of linear input-output behavior in the Wnt and ERK pathways. Analytics from each model further reveal that linearity arises through different means in each pathway, which we tested experimentally in the Wnt and ERK pathways. Linearity is a desired property in engineering where it facilitates fidelity and superposition in signal transmission. Our findings illustrate how cells tune different complex networks to converge on the same behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/781a8e83c82ec64fcdccde644f303e16d21146e9" target='_blank'>Signaling pathways as linear transmitters</a></td>
          <td>
            Harry Nunns, Lea Goentoro
          </td>
          <td>2018-09-17</td>
          <td>eLife</td>
          <td>19</td>
          <td>13</td>
        </tr>
    
        <tr id="Dysregulation in signal transduction pathways can lead to a variety of complex disorders, including cancer. Computational approaches such as network analysis are important tools to understand system dynamics as well as to identify critical components that could be further explored as therapeutic targets. Here, we performed perturbation analysis of a large-scale signal transduction model in extracellular environments that stimulate cell death, growth, motility, and quiescence. Each of the model’s components was perturbed under both loss-of-function and gain-of-function mutations. Using 1,300 simulations under both types of perturbations across various extracellular conditions, we identified the most and least influential components based on the magnitude of their influence on the rest of the system. Based on the premise that the most influential components might serve as better drug targets, we characterized them for biological functions, housekeeping genes, essential genes, and druggable proteins. The most influential components under all environmental conditions were enriched with several biological processes. The inositol pathway was found as most influential under inactivating perturbations, whereas the kinase and small lung cancer pathways were identified as the most influential under activating perturbations. The most influential components were enriched with essential genes and druggable proteins. Moreover, known cancer drug targets were also classified in influential components based on the affected components in the network. Additionally, the systemic perturbation analysis of the model revealed a network motif of most influential components which affect each other. Furthermore, our analysis predicted novel combinations of cancer drug targets with various effects on other most influential components. We found that the combinatorial perturbation consisting of PI3K inactivation and overactivation of IP3R1 can lead to increased activity levels of apoptosis-related components and tumor-suppressor genes, suggesting that this combinatorial perturbation may lead to a better target for decreasing cell proliferation and inducing apoptosis. Finally, our approach shows a potential to identify and prioritize therapeutic targets through systemic perturbation analysis of large-scale computational models of signal transduction. Although some components of the presented computational results have been validated against independent gene expression data sets, more laboratory experiments are warranted to more comprehensively validate the presented results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f61827b05e3b39d1a6c6640e56addecb0b990c6a" target='_blank'>Systems Perturbation Analysis of a Large-Scale Signal Transduction Model Reveals Potentially Influential Candidates for Cancer Therapeutics</a></td>
          <td>
            B. L. Puniya, Laura Allen, C. Hochfelder, M. Majumder, T. Helikar
          </td>
          <td>2015-11-10</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>18</td>
          <td>24</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12f692030dee53f9a37ef82c2e360d4e048c22df" target='_blank'>Information is critical for cellular life</a></td>
          <td>
            R. Iyengar
          </td>
          <td>2009-02-01</td>
          <td>Biophysical Journal</td>
          <td>0</td>
          <td>64</td>
        </tr>
    
        <tr id="Computational approaches can reveal how cells interpret, process, and respond to signals. This issue of Science Signaling, which complements the 13 April issue of Science, showcases the application of computational approaches to signal transduction research. Research and commentary highlight how mathematical modeling combined with experimental data can yield new insights into how cells interpret, process, and respond to external and internal cues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3ba07b97173020030f4ef6126bbe5e31f7fb242" target='_blank'>Focus Issue: Adding Math to the Signaling Toolkit</a></td>
          <td>
            W. Wong, J. Foley
          </td>
          <td>2012-04-17</td>
          <td>Science Signaling</td>
          <td>2</td>
          <td>32</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/191f8aa325b96b6e58a5ca39c766f19650b93b17" target='_blank'>A systematic framework for the design of reduced-order models for signal transduction pathways from a control theoretic perspective</a></td>
          <td>
            M. Maurya, Santhoji Katare, P. Patkar, A. Rundell, V. Venkatasubramanian
          </td>
          <td>2006-01-15</td>
          <td>Comput. Chem. Eng.</td>
          <td>11</td>
          <td>53</td>
        </tr>
    
        <tr id="The EGF-induced MAP kinase cascade is one of the most important and best characterized networks in intracellular signalling. It has a vital role in the development and maturation of living organisms. However, when deregulated, it is involved in the onset of a number of diseases. Based on a computational model describing a “surface” and an “internalized” parallel route, we use systems biology techniques to characterize aspects of the network’s functional organization. We examine the re-organization of protein groups from low to high external stimulation, define functional groups of proteins within the network, determine the parameter best encoding for input intensity and predict the effect of protein removal to the system’s output response. Extensive functional re-organization of proteins is observed in the lower end of stimulus concentrations. As we move to higher concentrations the variability is less pronounced. 6 functional groups have emerged from a consensus clustering approach, reflecting different dynamical aspects of the network. Mutual information investigation revealed that the maximum activation rate of the two output proteins best encodes for stimulus intensity. Removal of each protein of the network resulted in a range of graded effects, from complete silencing to intense activation. Our results provide a new “vista” of the EGF-induced MAP kinase cascade, from the perspective of complex self-organizing systems. Functional grouping of the proteins reveals an organizational scheme contrasting the current understanding of modular topology. The six identified groups may provide the means to experimentally follow the dynamics of this complex network. Also, the vulnerability analysis approach may be used for the development of novel therapeutic targets in the context of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccd7041af39a1cc5e66884c8aacd4b45fc5c205d" target='_blank'>Functional Aspects of the EGF-Induced MAP Kinase Cascade: A Complex Self-Organizing System Approach</a></td>
          <td>
            E. Kosmidis, V. Moschou, Georgios Ziogas, I. Boukovinas, M. Albani, N. Laskaris
          </td>
          <td>2014-11-05</td>
          <td>PLoS ONE</td>
          <td>1</td>
          <td>38</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/514f78dc7cf8c1400823726efd75d16c61a574ca" target='_blank'>The systems biology of signaling pathways</a></td>
          <td>
            I. N. Lavrik, I. N. Lavrik, M. Samsonova
          </td>
          <td>2016-01-01</td>
          <td>Biophysics</td>
          <td>3</td>
          <td>23</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/514f78dc7cf8c1400823726efd75d16c61a574ca" target='_blank'>The systems biology of signaling pathways</a></td>
          <td>
            I. N. Lavrik, I. N. Lavrik, M. Samsonova
          </td>
          <td>2016-01-01</td>
          <td>Biophysics</td>
          <td>3</td>
          <td>23</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d038165c25fb61341a0913a0ecc39aa1efeb74f" target='_blank'>Signal execution modes emerge in biochemical reaction networks calibrated to experimental data</a></td>
          <td>
            Oscar O. Ortega, Mustafa Ozen, Blake A. Wilson, James C. Pino, Michael W. Irvin, Geena V. Ildefonso, Shawn P. Garbett, Carlos F. Lopez
          </td>
          <td>2024-05-01</td>
          <td>iScience</td>
          <td>1</td>
          <td>7</td>
        </tr>
    
        <tr id="Signal transduction in cells has been an area of intense study for many years. However, until recently, the focus was on interactions between individual components rather than on the global behavior of the cell signaling networks. New experimental technologies, such as protein and DNA microarrays, high-throughput screening instrumentation, and diverse compound libraries, allow for many interactions to be examined simultaneously. Along with new experimental methods, quantitative models with mathematical methods, such as deterministic, stochastic, and statistical analyses and graph theory, have been useful in the mapping and analysis of intracellular signaling networks. Here we describe current experimental and computational approaches that are used to develop a global understanding of the complex behavior of intracellular signaling networks. We also discuss important applications of signaling network analysis, including the discovery of new drug targets, the identification of the signaling components responsible for the side (off-target) effects of drugs, and the development of combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c4bf7b7b63ac6d23884d07c5f5850b3a00a1d6" target='_blank'>Signaling Networks in Biology</a></td>
          <td>
            Anamika Sarkar, A. D. Wist, R. Iyengar
          </td>
          <td>2008-05-15</td>
          <td></td>
          <td>0</td>
          <td>64</td>
        </tr>
    
        <tr id="Signaling pathways come together to form networks that connect receptors to many different cellular machines. Such networks not only receive and transmit signals but also process information. The complexity of these networks requires the use of computational models to understand how information is processed and how input-output relationships are determined. Two major computational approaches used to study signaling networks are graph theory and dynamical modeling. Both approaches are useful; network analysis (application of graph theory) helps us understand how the signaling network is organized and what its information-processing capabilities are, whereas dynamical modeling helps us determine how the system changes in time and space upon receiving stimuli. Computational models have helped us identify a number of emergent properties that signaling networks possess. Such properties include ultrasensitivity, bistability, robustness, and noise-filtering capabilities. These properties endow cell-signaling networks with the ability to ignore small or transient signals and/or amplify signals to drive cellular machines that spawn numerous physiological functions associated with different cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/382a84e1cb7587f5957c26d66bbc05aaa65d01e9" target='_blank'>Signaling networks: information flow, computation, and decision making.</a></td>
          <td>
            E. Azeloglu, R. Iyengar
          </td>
          <td>2015-04-01</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>94</td>
          <td>64</td>
        </tr>
    
        <tr id="Motivation: Cellular information processing can be described mathematically using differential equations. Often, external stimulation of cells by compounds such as drugs or hormones leading to activation has to be considered. Mathematically, the stimulus is represented by a time-dependent input function. Parameters such as rate constants of the molecular interactions are often unknown and need to be estimated from experimental data, e.g. by maximum likelihood estimation. For this purpose, the input function has to be defined for all times of the integration interval. This is usually achieved by approximating the input by interpolation or smoothing of the measured data. This procedure is suboptimal since the input uncertainties are not considered in the estimation process which often leads to overoptimistic confidence intervals of the inferred parameters and the model dynamics. Results: This article presents a new approach which includes the input estimation into the estimation process of the dynamical model parameters by minimizing an objective function containing all parameters simultaneously. We applied this comprehensive approach to an illustrative model with simulated data and compared it to alternative methods. Statistical analyses revealed that our method improves the prediction of the model dynamics and the confidence intervals leading to a proper coverage of the confidence intervals of the dynamic parameters. The method was applied to the JAK-STAT signaling pathway. Availability: MATLAB code is available on the authors' website http://www.fdmold.uni-freiburg.de/~schelker/. Contact: max.schelker@fdm.uni-freiburg.de Supplementary Information: Additional information is available at Bioinformatics Online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cef06906f914ec31cc5eb3e853e26d5a68003b0" target='_blank'>Comprehensive estimation of input signals and dynamics in biochemical reaction networks</a></td>
          <td>
            Max Schelker, A. Raue, J. Timmer, C. Kreutz
          </td>
          <td>2012-09-01</td>
          <td>Bioinformatics</td>
          <td>29</td>
          <td>73</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e94e12aae1769fa195dc5bbe54b3d6a5afe4916" target='_blank'>Cell-signalling dynamics in time and space</a></td>
          <td>
            B. Kholodenko
          </td>
          <td>2006-03-01</td>
          <td>Nature Reviews Molecular Cell Biology</td>
          <td>1326</td>
          <td>57</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>