---
hide:
 - navigation
---
<!DOCTYPE html>
#
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-29 08:54:18 UTC</i>
  </p>
  
  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Network topology determines dynamics of the mammalian MAPK1,2 signaling network: bifan motif regulation of C‐Raf and B‐Raf isoforms by FGFR and MC1R</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>
    
        <tr id="Representation of intracellular signaling networks as directed graphs allows for the identification of regulatory motifs. Regulatory motifs are groups of nodes with the same connectivity structure, capable of processing information. The bifan motif, made of two source nodes directly crossregulating two target nodes, is an overrepresented motif in a mammalian cell signaling network and in transcriptional networks. One example of a bifan is the two MAP-kinases, p38, and JNK that phosphorylate and activate the two transcription factors ATF2 and Elk-1. We have used a system of coupled ordinary differential equations to analyze the regulatory capability of this bifan motif by itself, and when it interacts with other motifs such as positive and negative feedback loops. Our results indicate that bifans provide temporal regulation of signal propagation and act as signal sorters, filters, and synchronizers. Bifans that have OR gate configurations show rapid responses whereas AND gate bifans can introduce delays and allow prolongation of signal outputs. Bifans that have AND gates can filter noisy signal inputs. The p38/JNK-ATF2/Elk-1bifan synchronizes the output of activated transcription factors. Synchronization is a robust property of bifans and is exhibited even when the bifan is adjacent to a positive feedback loop. The presence of the bifan promotes the transcription and translation of the dual specificity protein phosphatase MKP-1 that inhibits p38 and JNK thus enabling a negative feedback loop. These results indicate that bifan motifs in cell signaling networks can contribute to signal processing capability both intrinsically and by enabling the functions of other regulatory motifs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae55dfe0229d24fa1a892630fb538c06db0cc554" target='_blank'>Functions of bifans in context of multiple regulatory motifs in signaling networks.</a></td>
          <td>
            Azi Lipshtat, Sudarshan P. Purushothaman, R. Iyengar, Avi Ma’ayan
          </td>
          <td>2007-11-30</td>
          <td>Biophysical journal</td>
          <td>42</td>
          <td>66</td>
        </tr>
    
        <tr id="With the advent of high-throughput biology, we now routinely scan cells and organisms at practically all levels, from genome to protein, metabolism, signaling and other cellular functions. This methodology allowed biological studies to move from a reductionist approach, such as isolation of specific pathways and mechanisms, to a more integrative approach, where biological systems are seen as a network of interconnected components that provide specific outputs and functions in response to stimuli. Recent literature on biological networks demonstrates two important concepts that we will consider in this review: (i) cellular pathways are highly interconnected and should not be studied separately, but as a network; (ii) simple, recurrent feedback motifs within the network can produce very specific functions that favor their modular use. The first theme differs from the traditional approach in biology because it provides a framework (i.e., the network view) in which large datasets are analyzed with an unbiased view. The second theme (feedback motifs) shows the importance of locally analyzing the dynamic properties of biological networks in order to better understand their functionality. We will review these themes with examples from cell signaling networks, gene regulatory networks and metabolic pathways. The deregulation of cellular networks (metabolism, signaling etc.) is involved in cancer, but the size of the networks and resulting non-linear behavior do not allow for intuitive reasoning. In that context, we argue that the qualitative classification of the 'building blocs' of biological networks (i.e. the motifs) in terms of dynamics and functionality will be critical to improve our understanding of cancer biology and rationalize the wealth of information from high-throughput experiments. From the examples highlighted in this review, it is clear that dynamic feedback motifs can be used to provide a unified view of various cellular processes involved in cancer and this will be critical for future research on personalized and predictive cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3f282afaa65efc5fc707b0c0c159353a19a8d31" target='_blank'>Dynamic modeling and analysis of cancer cellular network motifs.</a></td>
          <td>
            M. Cloutier, Edwin Wang
          </td>
          <td>2011-07-01</td>
          <td>Integrative biology : quantitative biosciences from nano to macro</td>
          <td>48</td>
          <td>15</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6a045551eda8ac4bf3c6dd072aafd66f5f9d95" target='_blank'>Simplistic pathways or complex networks?</a></td>
          <td>
            C. Jørgensen, R. Linding
          </td>
          <td>2010-02-01</td>
          <td>Current opinion in genetics & development</td>
          <td>40</td>
          <td>40</td>
        </tr>
    
        <tr id="Emerging technologies have enabled the acquisition of large genomics and proteomics data sets. However, current methodologies for analysis do not permit interpretation of the data in ways that unravel cellular networking. We propose a quantitative method for determining functional interactions in cellular signaling and gene networks. It can be used to explore cell systems at a mechanistic level or applied within a “modular” framework, which dramatically decreases the number of variables to be assayed. This method is based on a mathematical derivation that demonstrates how the topology and strength of network connections can be retrieved from experimentally measured network responses to successive perturbations of all modules. Importantly, our analysis can reveal functional interactions even when the components of the system are not all known. Under these circumstances, some connections retrieved by the analysis will not be direct but correspond to the interaction routes through unidentified elements. The method is tested and illustrated by using computer-generated responses of a modeled mitogen-activated protein kinase cascade and gene network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c5bdebfafe34ab22c712f7f0a323869c0857185" target='_blank'>Untangling the wires: A strategy to trace functional interactions in signaling and gene networks</a></td>
          <td>
            D. Angeli, J. Ferrell, Eduardo Sontag
          </td>
          <td>2002-09-19</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1292</td>
          <td>96</td>
        </tr>
    
        <tr id="Emerging technologies have enabled the acquisition of large genomics and proteomics data sets. However, current methodologies for analysis do not permit interpretation of the data in ways that unravel cellular networking. We propose a quantitative method for determining functional interactions in cellular signaling and gene networks. It can be used to explore cell systems at a mechanistic level or applied within a “modular” framework, which dramatically decreases the number of variables to be assayed. This method is based on a mathematical derivation that demonstrates how the topology and strength of network connections can be retrieved from experimentally measured network responses to successive perturbations of all modules. Importantly, our analysis can reveal functional interactions even when the components of the system are not all known. Under these circumstances, some connections retrieved by the analysis will not be direct but correspond to the interaction routes through unidentified elements. The method is tested and illustrated by using computer-generated responses of a modeled mitogen-activated protein kinase cascade and gene network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c5bdebfafe34ab22c712f7f0a323869c0857185" target='_blank'>Untangling the wires: A strategy to trace functional interactions in signaling and gene networks</a></td>
          <td>
            D. Angeli, J. Ferrell, Eduardo Sontag
          </td>
          <td>2002-09-19</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1292</td>
          <td>96</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75f581981a222f5b0749f3a969807d0aa573b574" target='_blank'>Creating and analyzing pathway and protein interaction compendia for modelling signal transduction networks</a></td>
          <td>
            D. Kirouac, J. Sáez-Rodríguez, J. Swantek, J. Burke, D. Lauffenburger, P. Sorger
          </td>
          <td>2012-05-01</td>
          <td>BMC Systems Biology</td>
          <td>77</td>
          <td>119</td>
        </tr>
    
        <tr id="Signaling pathways come together to form networks that connect receptors to many different cellular machines. Such networks not only receive and transmit signals but also process information. The complexity of these networks requires the use of computational models to understand how information is processed and how input-output relationships are determined. Two major computational approaches used to study signaling networks are graph theory and dynamical modeling. Both approaches are useful; network analysis (application of graph theory) helps us understand how the signaling network is organized and what its information-processing capabilities are, whereas dynamical modeling helps us determine how the system changes in time and space upon receiving stimuli. Computational models have helped us identify a number of emergent properties that signaling networks possess. Such properties include ultrasensitivity, bistability, robustness, and noise-filtering capabilities. These properties endow cell-signaling networks with the ability to ignore small or transient signals and/or amplify signals to drive cellular machines that spawn numerous physiological functions associated with different cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/382a84e1cb7587f5957c26d66bbc05aaa65d01e9" target='_blank'>Signaling networks: information flow, computation, and decision making.</a></td>
          <td>
            E. Azeloglu, R. Iyengar
          </td>
          <td>2015-04-01</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>94</td>
          <td>64</td>
        </tr>
    
        <tr id="Mitogen‐activated protein kinase (MAPK) cascades process a myriad of stimuli received by cell‐surface receptors and generate precise spatiotemporal guidance for multiple target proteins, dictating receptor‐specific cellular outcomes. Computational modeling reveals that the intrinsic topology of MAPK cascades enables them to amplify signal sensitivity and amplitude, reduce noise, and display intricate dynamic properties, which include toggle switches, excitation pulses, and oscillations. Specificity of signaling responses can be brought about by signal‐induced feedback and feedforward wiring imposed on the MAPK cascade backbone. Intracellular gradients of protein activities arise from the spatial separation of opposing reactions in kinase‐phosphatase cycles. The membrane confinement of the initiating kinase in MAPK cascades and cytosolic localization of phosphatases can result in precipitous gradients of phosphorylated signal‐transducers if they spread solely by diffusion. Endocytotic trafficking of active kinases driven by molecular motors and traveling waves of protein phosphorylation can propagate phosphorylation signals from the plasma membrane to the nucleus, especially in large cells, such as Xenopus eggs. Copyright © 2009 John Wiley & Sons, Inc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/549f0ea398377e29460e84284cf31387d6568803" target='_blank'>Four‐dimensional dynamics of MAPK information‐processing systems</a></td>
          <td>
            B. Kholodenko, M. Birtwistle
          </td>
          <td>2009-07-01</td>
          <td>Wiley Interdisciplinary Reviews: Systems Biology and Medicine</td>
          <td>87</td>
          <td>57</td>
        </tr>
    
        <tr id="Dynamics adds to the complexity of signaling networks. The biochemical processes that control the responses of cells to changes in their internal or external environments form a complex interconnected network. The molecules that participate in these processes are diverse and dynamic, and the networks themselves are affected by the flow of information through them. With this appreciation of the complexity in signaling networks, Science Signaling highlights computational and experimental approaches to understanding both the macroscopic cellular response and the microscopic signaling network and its constituents and dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8045d1bb90c38a9e81507bcbcfd412b519b768f7" target='_blank'>Focus Issue: Unraveling Signaling Complexity</a></td>
          <td>
            N. Gough, J. Foley
          </td>
          <td>2009-07-28</td>
          <td>Science Signaling</td>
          <td>5</td>
          <td>32</td>
        </tr>
    
        <tr id="Bayesian inference–based modeling identifies the most likely paths through a signaling network. Picking the Right Path Signaling networks have become increasingly complex as large-scale analysis and experiments in multiple systems add new potential connections and players. Xu et al. present a mathematical approach to rank the possible paths through a signaling pathway and develop hypotheses that can be rationally tested. They call their approach BIBm for Bayesian inference–based modeling and apply BIBm to explore the signaling pathways by which epidermal growth factor (EGF) stimulates extracellular signal–regulated kinase (ERK). Using a limited set of biochemical experiments, the authors tested four models and found that the one that relied on two Raf family members ranked the highest. This model was then experimentally validated in two cell lines to show that both Raf-1 and B-Raf contribute to ERK activation in response to EGF. The specification of biological decisions by signaling pathways is encoded by the interplay between activation dynamics and network topologies. Although we can describe complex networks, we cannot easily determine which topology the cell actually uses to transduce a specific signal. Experimental testing of all plausible topologies is infeasible because of the combinatorially large number of experiments required to explore the complete hypothesis space. Here, we demonstrate that Bayesian inference–based modeling provides an approach to explore and constrain this hypothesis space, permitting the rational ranking of pathway models. Our approach can use measurements of a limited number of biochemical species when combined with multiple perturbations. As proof of concept, we examined the activation of the extracellular signal–regulated kinase (ERK) pathway by epidermal growth factor. The predicted and experimentally validated model shows that both Raf-1 and, unexpectedly, B-Raf are needed to fully activate ERK in two different cell lines. Thus, our formal methodology rationally infers evidentially supported pathway topologies even when a limited number of biochemical and kinetic measurements are available.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00d54fdde2e7f199813e86ab03a7ab6805f23394" target='_blank'>Inferring Signaling Pathway Topologies from Multiple Perturbation Measurements of Specific Biochemical Species</a></td>
          <td>
            
          </td>
          <td>2010-03-16</td>
          <td>Science Signaling</td>
          <td>136</td>
          <td>0</td>
        </tr>
    
        <tr id="CIRIT, MURAT. Predictive Computational Models of Mitogen-activated Protein Kinase Signaling Networks Driven by Quantitative Measurements of System Dynamics. (Under the direction of Jason M. Haugh.) Signal transduction networks in mammalian cells, comprised of a limited set of interacting biochemical pathways, are accessed by various growth factor and cytokine receptors to elicit distinct cell responses. This raises the question as to how specificity of the stimulus-response relationship is encoded at the molecular level. In mammalian cells, the Ras/ERK pathway controls cell proliferation and other responses stimulated by growth factors, and several crosstalk and feedback mechanisms affecting its activation have been identified. Here we present a systematic, comparative analysis of the signaling networks accessed by fibroblast growth factor (FGF) and platelet-derived growth factor (PDGF) receptors in mouse fibroblasts, in which the extracellular signal-regulated kinase (ERK) cascade is activated by Rasand phosphoinositide 3-kinase (PI3K)-dependent pathways and is subjected to three layers of ERK-dependent feedback: desensitization of Ras activation, negative regulation of MEK kinase (e.g. Raf) activities, and upregulation of dual-specificity ERK phosphatases. Through the formulation and analysis of data-driven mathematical models, we show that while the FGF stimulation of PI3K signaling is relatively weak, this deficiency is compensated for by a more potent, Ras-dependent activation of ERK. We take a systematic approach to parse the magnitudes of multiple regulatory mechanisms that attenuate ERK activation through canonical (Ras-dependent) and non-canonical (PI3Kdependent) pathways. Our results establish the second of these as the dominant mode of ERK self-regulation in mouse fibroblasts. We further demonstrate that models of signaling networks, trained on a sufficient diversity of quantitative data, can be reasonably comprehensive, accurate, and predictive in the dynamical sense. Predictive Computational Models of Mitogen-activated Protein Kinase Signaling Networks Driven by Quantitative Measurements of System Dynamics">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/989d2770c00ced44898542d66799c5523c3124e4" target='_blank'>Predictive Computational Models of Mitogen-activated Protein Kinase Signaling Networks Driven by Quantitative Measurements of System Dynamics.</a></td>
          <td>
            M. Cirit
          </td>
          <td>2010-07-23</td>
          <td></td>
          <td>0</td>
          <td>22</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12f692030dee53f9a37ef82c2e360d4e048c22df" target='_blank'>Information is critical for cellular life</a></td>
          <td>
            R. Iyengar
          </td>
          <td>2009-02-01</td>
          <td>Biophysical Journal</td>
          <td>0</td>
          <td>64</td>
        </tr>
    
        <tr id="Cell responses are actuated by tightly controlled signal transduction pathways. Although the concept of an integrated signaling network replete with interpathway cross-talk and feedback regulation is broadly appreciated, kinetic data of the type needed to characterize such interactions in conjunction with mathematical models are lacking. In mammalian cells, the Ras/ERK pathway controls cell proliferation and other responses stimulated by growth factors, and several cross-talk and feedback mechanisms affecting its activation have been identified. In this work, we take a systematic approach to parse the magnitudes of multiple regulatory mechanisms that attenuate ERK activation through canonical (Ras-dependent) and non-canonical (PI3K-dependent) pathways. In addition to regulation of receptor and ligand levels, we consider three layers of ERK-dependent feedback: desensitization of Ras activation, negative regulation of MEK kinase (e.g. Raf) activities, and up-regulation of dual-specificity ERK phosphatases. Our results establish the second of these as the dominant mode of ERK self-regulation in mouse fibroblasts. We further demonstrate that kinetic models of signaling networks, trained on a sufficient diversity of quantitative data, can be reasonably comprehensive, accurate, and predictive in the dynamical sense.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405be6d692d4836d3b3efde9333fdf1789ff05c7" target='_blank'>Systematic Quantification of Negative Feedback Mechanisms in the Extracellular Signal-regulated Kinase (ERK) Signaling Network*</a></td>
          <td>
            M. Cirit, Chun-Chao Wang, J. Haugh
          </td>
          <td>2010-09-16</td>
          <td>The Journal of Biological Chemistry</td>
          <td>91</td>
          <td>33</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b06d234407efd97d798904ba97ebb9a73b5fb31e" target='_blank'>Methodologies for the modeling and simulation of biochemical networks, illustrated for signal transduction pathways: A primer</a></td>
          <td>
            Nesma ElKalaawy, A. Wassal
          </td>
          <td>2015-03-01</td>
          <td>Bio Systems</td>
          <td>20</td>
          <td>8</td>
        </tr>
    
        <tr id="Cellular signaling networks display complex architecture. Defining the design principle of this architecture is crucial for our understanding of various biological processes. Using a mathematical model for three-node feed-forward loops, we identify that the organization of motifs in specific manner within the network serves as an important regulator of signal processing. Further, incorporating a systemic stochastic perturbation to the model we could propose a possible design principle, for higher-order organization of motifs into larger networks in order to achieve specific biological output. The design principle was then verified in a large, complex human cancer signaling network. Further analysis permitted us to classify signaling nodes of the network into robust and vulnerable nodes as a result of higher order motif organization. We show that distribution of these nodes within the network at strategic locations then provides for the range of features displayed by the signaling network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24367176c9c2619ae9bcefeb7c1d76c1de5a5e79" target='_blank'>Unraveling the Design Principle for Motif Organization in Signaling Networks</a></td>
          <td>
            S. Chatterjee, Dhiraj Kumar
          </td>
          <td>2011-12-02</td>
          <td>PLoS ONE</td>
          <td>4</td>
          <td>21</td>
        </tr>
    
        <tr id="Single-cell RNA-sequencing data can revolutionize our understanding of the patterns of cell-cell and ligand-receptor connectivity that influence the function of tissues and organs. However, the quantification and visualization of these patterns are major computational and epistemological challenges. Here, we present Connectome, a software package for R which facilitates rapid calculation, and interactive exploration, of cell-cell signaling network topologies contained in single-cell RNA-sequencing data. Connectome can be used with any reference set of known ligand-receptor mechanisms. It has built-in functionality to facilitate differential and comparative connectomics, in which complete mechanistic networks are quantitatively compared between systems. Connectome includes computational and graphical tools designed to analyze and explore cell-cell connectivity patterns across disparate single-cell datasets. We present approaches to quantify these topologies and discuss some of the biologic theory leading to their design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/410f8c84736d79491044843b70dcd764f7837be9" target='_blank'>Connectome: computation and visualization of cell-cell signaling topologies in single-cell systems data</a></td>
          <td>
            M. Raredon, Junchen Yang, J. Garritano, Meng Wang, D. Kushnir, J. Schupp, T. Adams, A. Greaney, K. Leiby, N. Kaminski, Y. Kluger, A. Levchenko, L. Niklason
          </td>
          <td>2021-01-21</td>
          <td>bioRxiv</td>
          <td>31</td>
          <td>98</td>
        </tr>
    
        <tr id="Cellular proccess that are crucial for health or disease are orchestrated by the expression of multiple genes in a network of differential complex signaling pathways, consisting of many physical and functional interactions.Unlike the stable genome structure, the highly dynamic interactome represents a major challenge for quantitative mapping of protein–protein interactions (PPIs). In this article, I will discuss the latest systems science advances by using breakthrough technologies to gain an understanding of the whole picture of the spatiotemporal signaling network as a response to dynamic individual genome–lifestyle interactions and translating it to the clinic.Biological signals control cells’ functions and their organization into tissues, organs and a whole organism’s homeostasis. A variety of extracellular and intracellu-lar signaling molecules include hormones, growth factors, cytokines, chemokines, neurotransmitters, extracellular matrix components, receptors, protein kinases, protein phosphatases and DNA-binding proteins in the nucleus. A signal is trans-mitted or amplified from the cell mem-brane to the nucleus through a dynamic complex interactions network">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2702c101690829e49078cf33e2ee81dc4f49e616" target='_blank'>Differential signaling transduction networks for clinical robustness</a></td>
          <td>
            D. Roukos
          </td>
          <td>2012-04-01</td>
          <td>Expert Review of Proteomics</td>
          <td>12</td>
          <td>62</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63eb764d9a6760bb8317b892dc9911062cdc8b21" target='_blank'>Computation and visualization of cell–cell signaling topologies in single-cell systems data using Connectome</a></td>
          <td>
            M. Raredon, Junchen Yang, James Garritano, Meng Wang, D. Kushnir, J. Schupp, T. Adams, A. Greaney, K. Leiby, N. Kaminski, Y. Kluger, A. Levchenko, L. Niklason
          </td>
          <td>2022-03-09</td>
          <td>Scientific Reports</td>
          <td>52</td>
          <td>68</td>
        </tr>
    
        <tr id="Successful drug development is a risky and lengthy process that can take over ten years and consume billions of dollars. Target discovery is a critical stage of drug development for the identification of key molecules and pathways that can be targeted by novel therapeutics to find more effective treatments. Due to the rapid development in artificial intelligence and machine learning techniques over the past decade, computational approaches have now emerged as powerful tools to unravel complex interactions within biological systems to identify novel therapeutic targets. In particular, modeling and analysis of intracellular molecular networks play a pivotal role in target discovery by enabling researchers to efficiently and simultaneously navigate massive amounts of biological data to identify potential therapeutic targets. Such technologies can significantly accelerate the prolonged process of development of innovative therapies for complex diseases. Besides highlighting the findings of the recently introduced extreme signaling failures in intracellular molecular networks, here we briefly review various methods for modeling and analysis of intracellular molecular networks and discuss how they can be utilized to predict potential drug targets within such complex signaling systems. Overall, this review emphasizes the significance of modeling and analysis of molecular networks for fast-tracking and rapid discovery of novel therapeutic targets; to pave the way for the development of more effective treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524c46fe6dd0743cfbc96de3c126a77081afdd10" target='_blank'>From Data to Knowledge: A Mini-Review on Molecular Network Modeling and Analysis for Therapeutic Target Discovery</a></td>
          <td>
            Mustafa Ozen, E. Emamian, A. Abdi
          </td>
          <td>2023-07-05</td>
          <td>Archives of Pharmacology and Therapeutics</td>
          <td>0</td>
          <td>16</td>
        </tr>
    
        <tr id="Despite much effort, identification of modular structures and study of their organizing and functional roles remain a formidable challenge in molecular systems biology, which, however, is essential in reaching a systematic understanding of large-scale cell regulation networks and hence gaining capacity of exerting effective interference to cell activity. Combining graph theoretic methods with available dynamics information, we successfully retrieved multiple feedback modules of three important signaling networks. These feedbacks are structurally arranged in a hierarchical way and dynamically produce layered temporal profiles of output signals. We found that global and local feedbacks act in very different ways and on distinct features of the information flow conveyed by signal transduction but work highly coordinately to implement specific biological functions. The redundancy embodied with multiple signal-relaying channels and feedback controls bestow great robustness and the reaction hubs seated at junctions of different paths announce their paramount importance through exquisite parameter management. The current investigation reveals intriguing general features of the organization of cell signaling networks and their relevance to biological function, which may find interesting applications in analysis, design and control of bio-networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f1a36176cecd70b5b2e37e209eede6e52ee5091" target='_blank'>Hierarchical Feedback Modules and Reaction Hubs in Cell Signaling Networks</a></td>
          <td>
            Jianfeng Xu, Y. Lan
          </td>
          <td>2015-05-07</td>
          <td>PLoS ONE</td>
          <td>10</td>
          <td>20</td>
        </tr>
    
        <tr id="An error led to part of Figure 2 being removed, breaking the red line that should have linked the grey circles “PKC” and “EGFR”. The correct version of Figure 2 is now displayed on both the PDF and HTML versions of the article.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/892ad05c7eb0020aec314c7a7afee5e1e1dd1b95" target='_blank'>Correction: Integrative Modelling of the Influence of MAPK Network on Cancer Cell Fate Decision</a></td>
          <td>
            L. Grieco, Laurence Calzone, I. Bernard-Pierrot, F. Radvanyi, B. Kahn-Perlès, D. Thieffry
          </td>
          <td>2013-11-01</td>
          <td>PLoS Computational Biology</td>
          <td>41</td>
          <td>61</td>
        </tr>
    
        <tr id="Signaling network construction and analysis provide insights into biology and medicine. The advancements in “omics” (proteomics, genomics, lipidomics, and metabolomics) technologies have yielded large inventories of genes, transcripts, proteins, and metabolites. The challenge is to find out how these entities work together to regulate the processes by which cells respond to external and internal signals. Mathematical and computational modeling of signaling networks has a key role in this task, and network analysis provides insights into biological systems and has applications for medicine. Here, we review experimental and theoretical progress and future challenges toward this goal. We focus on how networks are reconstructed from data, how these networks are structured to control the flow of biological information, and how the design features of the networks specify biological decisions. How do cells perceive their environment and produce appropriate responses? The sensing and coordination of proper responses is coordinated by intricate biochemical networks that integrate signals and compute biological responses with high fidelity and reproducibility. Modern “omics” (proteomics, genomics, lipidomics, and metabolomics) techniques have identified many of the components of these networks, but researchers are now faced with the challenge of unraveling how the components are connected and how they compute. To elucidate these network properties, various mathematical and computational modeling techniques are used because their complexity defeats human intuition. The results indicate that signaling networks show a wide diversity of behaviors encoded in their design, but that they are also very flexible, with cells adapting their networks in response to different contexts by changing connectivities. The remaining challenges include integration of data generated with methods for the analysis of different types of networks (gene regulation, protein modification, and metabolic networks), and generation of coherent network interpretations that are widely accessible and can generate new hypotheses that are experimentally testable and that will push the knowledge frontier in biology and biomedicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd91467a3bfbd49f004addfc80c5358488ea942d" target='_blank'>Computational Approaches for Analyzing Information Flow in Biological Networks</a></td>
          <td>
            B. Kholodenko, M. Yaffe, W. Kolch
          </td>
          <td>2012-04-17</td>
          <td>Science Signaling</td>
          <td>171</td>
          <td>105</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/486e692a589332225293a4accd0672423a72245e" target='_blank'>Computational approaches for modeling regulatory cellular networks.</a></td>
          <td>
            N. J. Eungdamrong, R. Iyengar
          </td>
          <td>2004-12-01</td>
          <td>Trends in cell biology</td>
          <td>66</td>
          <td>64</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b0d4a5d209975c2d5166fd635acb02a22157e56" target='_blank'>The crosstalk between EGF, IGF, and Insulin cell signaling pathways - computational and experimental analysis</a></td>
          <td>
            Rafal Zieliński, P. Przytycki, Jie Zheng, David Zhang, T. Przytycka, J. Capala
          </td>
          <td>2009-09-04</td>
          <td>BMC Systems Biology</td>
          <td>62</td>
          <td>51</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/514f78dc7cf8c1400823726efd75d16c61a574ca" target='_blank'>The systems biology of signaling pathways</a></td>
          <td>
            I. N. Lavrik, I. N. Lavrik, M. Samsonova
          </td>
          <td>2016-01-01</td>
          <td>Biophysics</td>
          <td>3</td>
          <td>23</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/514f78dc7cf8c1400823726efd75d16c61a574ca" target='_blank'>The systems biology of signaling pathways</a></td>
          <td>
            I. N. Lavrik, I. N. Lavrik, M. Samsonova
          </td>
          <td>2016-01-01</td>
          <td>Biophysics</td>
          <td>3</td>
          <td>23</td>
        </tr>
    
        <tr id="Large-scale models of signaling networks are beginning to be reconstructed and corresponding analysis frameworks are being developed. Herein, a reconstruction of the JAK-STAT signaling system in the human B-cell is described and a scalable framework for its network analysis is presented. This approach is called extreme signaling pathway analysis and involves the description of network properties with systemically independent basis vectors called extreme pathways. From the extreme signaling pathways, emergent systems properties of the JAK-STAT signaling network have been characterized, including 1), a mathematical definition of network crosstalk; 2), an analysis of redundancy in signaling inputs and outputs; 3), a study of reaction participation in the network; and 4), a delineation of 85 correlated reaction sets, or systemic signaling modules. This study is the first such analysis of an actual biological signaling system. Extreme signaling pathway analysis is a topologically based approach and assumes a balanced use of the signaling network. As large-scale reconstructions of signaling networks emerge, such scalable analyses will lead to a description of the fundamental systems properties of signal transduction networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c04f8f4c6952cf87ecc92003d384f64ce02c12ba" target='_blank'>The JAK-STAT signaling network in the human B-cell: an extreme signaling pathway analysis.</a></td>
          <td>
            J. Papin, B. Palsson
          </td>
          <td>2004-07-01</td>
          <td>Biophysical journal</td>
          <td>137</td>
          <td>142</td>
        </tr>
    
        <tr id="Relating structure and dynamics of molecular networks remains very challenging. The era of genome sequencing has produced long lists of the molecular parts from which cellular machines are constructed. A fundamental goal in systems biology is to understand how cellular behavior emerges from the interaction in time and space of genetically encoded molecular parts, as well as nongenetically encoded small molecules. Networks provide a natural framework for the organization and quantitative representation of all the available data about molecular interactions. The structural and dynamic properties of molecular networks have been the subject of intense research. Despite major advances, bridging network structure to dynamics—and therefore to behavior—remains challenging. A key concept of modern engineering that recurs in the functional analysis of biological networks is modularity. Most approaches to molecular network analysis rely to some extent on the assumption that molecular networks are modular—that is, they are separable and can be studied to some degree in isolation. We describe recent advances in the analysis of modularity in biological networks, focusing on the increasing realization that a dynamic perspective is essential to grouping molecules into modules and determining their collective function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/742ade5f48d35538b4121002f621d1e13fba1f4f" target='_blank'>Understanding Modularity in Molecular Networks Requires Dynamics</a></td>
          <td>
            Roger P. Alexander, Philip M. Kim, T. Emonet, M. Gerstein
          </td>
          <td>2009-07-28</td>
          <td>Science Signaling</td>
          <td>93</td>
          <td>168</td>
        </tr>
    
        <tr id="Embryonic development and adult tissue homeostasis are controlled through activation of intracellular signal transduction pathways by extracellular growth factors. In the past, signal transduction has largely been regarded as a linear process. However, more recent data from large‐scale and high‐throughput experiments indicate that there is extensive cross‐talk between individual signaling cascades leading to the notion of a signaling network. The behavior of such complex networks cannot be predicted by simple intuitive approaches but requires sophisticated models and computational simulations. The purpose of such models is to generate experimentally testable hypotheses and to find explanations for unexpected experimental results. Here, we discuss the need for, and the future impact of, mathematical models for exploring signal transduction in different biological contexts such as for example development. BioEssays 30:1110–1125, 2008. © 2008 Wiley Periodicals, Inc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8545f3dc8505b17afcd909dbddc0e64abbc53ce2" target='_blank'>Network modeling of signal transduction: establishing the global view</a></td>
          <td>
            H. Kestler, Christian Wawra, Barbara Kracher, M. Kühl
          </td>
          <td>2008-11-01</td>
          <td>BioEssays</td>
          <td>94</td>
          <td>56</td>
        </tr>
    
        <tr id="A hybrid modeling framework integrating a highly specific mechanistic model with highly abstract empirical model is presented. With the growing interest in the scientific and medical community for identification of therapeutic targets in treatment of disease, it is necessary to develop predictive models that can describe cellular behavior in response to environmental cues. Intracellular signaling pathways form complex networks that regulate cellular response in both health and disease. Mechanistic (or white-box) models of biochemical networks are often unable to explain comprehensive cellular response due to lack of knowledge and/or intractable complexity (especially in events distal from the cell membrane). Empirical (or black-box) models may provide a less than accurate representation of cellular response due to data deficiency and/or loss of mechanistic detail. In the proposed framework, we use a mechanistic model to capture early signaling events and apply the resulting generated internal signals (along with external inputs) to a downstream empirical sub-model. The key construct in the approach is the treatment of a cell’s biochemical network as an encoder that creates a functional internal representation of external environmental cues. The signals derived from this representation are then used to inform downstream behaviors. Using this idea, we are able to create a comprehensive framework that describes important mechanisms with sufficient detail, while representing complex or unknown mechanisms in a more abstract form. The model is verified using published biological data describing T-Cells in immune response.Copyright © 2013 by ASME">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f470c01ee028f44eb08c099f9bfa4ffe012ae147" target='_blank'>Integrated Mechanistic-Empirical Modeling of Cellular Response Based on Intracellular Signaling Dynamics</a></td>
          <td>
            Michaelle N. Mayalu, H. Asada
          </td>
          <td>2013-10-01</td>
          <td></td>
          <td>1</td>
          <td>40</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eddc32d300363080a2ed33b1d0ee44efa9570413" target='_blank'>Manatee invariants reveal functional pathways in signaling networks</a></td>
          <td>
            Leonie Amstein, Jörg Ackermann, Jennifer Scheidel, S. Fulda, I. Dikič, I. Koch
          </td>
          <td>2017-07-28</td>
          <td>BMC Systems Biology</td>
          <td>10</td>
          <td>117</td>
        </tr>
    
        <tr id="The EGF-induced MAP kinase cascade is one of the most important and best characterized networks in intracellular signalling. It has a vital role in the development and maturation of living organisms. However, when deregulated, it is involved in the onset of a number of diseases. Based on a computational model describing a “surface” and an “internalized” parallel route, we use systems biology techniques to characterize aspects of the network’s functional organization. We examine the re-organization of protein groups from low to high external stimulation, define functional groups of proteins within the network, determine the parameter best encoding for input intensity and predict the effect of protein removal to the system’s output response. Extensive functional re-organization of proteins is observed in the lower end of stimulus concentrations. As we move to higher concentrations the variability is less pronounced. 6 functional groups have emerged from a consensus clustering approach, reflecting different dynamical aspects of the network. Mutual information investigation revealed that the maximum activation rate of the two output proteins best encodes for stimulus intensity. Removal of each protein of the network resulted in a range of graded effects, from complete silencing to intense activation. Our results provide a new “vista” of the EGF-induced MAP kinase cascade, from the perspective of complex self-organizing systems. Functional grouping of the proteins reveals an organizational scheme contrasting the current understanding of modular topology. The six identified groups may provide the means to experimentally follow the dynamics of this complex network. Also, the vulnerability analysis approach may be used for the development of novel therapeutic targets in the context of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccd7041af39a1cc5e66884c8aacd4b45fc5c205d" target='_blank'>Functional Aspects of the EGF-Induced MAP Kinase Cascade: A Complex Self-Organizing System Approach</a></td>
          <td>
            E. Kosmidis, V. Moschou, Georgios Ziogas, I. Boukovinas, M. Albani, N. Laskaris
          </td>
          <td>2014-11-05</td>
          <td>PLoS ONE</td>
          <td>1</td>
          <td>38</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da87720f4767a44da3f92e497492607a5e8a082c" target='_blank'>When kinases meet mathematics: the systems biology of MAPK signalling</a></td>
          <td>
            W. Kolch, Muffy Calder, David R. Gilbert
          </td>
          <td>2005-03-21</td>
          <td>FEBS Letters</td>
          <td>190</td>
          <td>86</td>
        </tr>
    
        <tr id="BioNetGen allows a user to create a computational model that characterizes the dynamics of a signal transduction system, and that accounts comprehensively and precisely for specified enzymatic activities, potential post-translational modifications and interactions of the domains of signaling molecules. The output defines and parameterizes the network of molecular species that can arise during signaling and provides functions that relate model variables to experimental readouts of interest. Models that can be generated are relevant for rational drug discovery, analysis of proteomic data and mechanistic studies of signal transduction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab2f8a8beda2d4681b64f77e86e6fdf0c6f65e26" target='_blank'>BioNetGen: software for rule-based modeling of signal transduction based on the interactions of molecular domains</a></td>
          <td>
            M. Blinov, J. Faeder, B. Goldstein, W. Hlavacek
          </td>
          <td>2004-11-22</td>
          <td>Bioinformatics</td>
          <td>435</td>
          <td>44</td>
        </tr>
    
        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6c5f3716c8b99a84acb38f62b6e1f710b2e866" target='_blank'>The use of machine learning to discover regulatory networks controlling biological systems.</a></td>
          <td>
            Rossin Erbe, Jessica Gore, K. Gemmill, Daria A. Gaykalova, E. Fertig
          </td>
          <td>2022-01-01</td>
          <td>Molecular cell</td>
          <td>19</td>
          <td>45</td>
        </tr>
    
  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  
  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>