<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Ecad4837f4bc306f173456aafecd9d6b67b62744
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Virtual%20cell/" class="md-tabs__link">
        
  
    
  
  Virtual cell

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../RA/" class="md-tabs__link">
        
  
    
  
  RA

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Prostate%20cancer/" class="md-tabs__link">
        
  
    
  
  Prostate cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Virtual%20patients/" class="md-tabs__link">
        
  
    
  
  Virtual patients

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Virtual%20cell/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual cell
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../RA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RA
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Prostate%20cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Prostate cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Virtual%20patients/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual patients
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-03 06:08:34 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Regulation of ERK-MAPK signaling in human epidermis</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="Cell responses are actuated by tightly controlled signal transduction pathways. Although the concept of an integrated signaling network replete with interpathway cross-talk and feedback regulation is broadly appreciated, kinetic data of the type needed to characterize such interactions in conjunction with mathematical models are lacking. In mammalian cells, the Ras/ERK pathway controls cell proliferation and other responses stimulated by growth factors, and several cross-talk and feedback mechanisms affecting its activation have been identified. In this work, we take a systematic approach to parse the magnitudes of multiple regulatory mechanisms that attenuate ERK activation through canonical (Ras-dependent) and non-canonical (PI3K-dependent) pathways. In addition to regulation of receptor and ligand levels, we consider three layers of ERK-dependent feedback: desensitization of Ras activation, negative regulation of MEK kinase (e.g. Raf) activities, and up-regulation of dual-specificity ERK phosphatases. Our results establish the second of these as the dominant mode of ERK self-regulation in mouse fibroblasts. We further demonstrate that kinetic models of signaling networks, trained on a sufficient diversity of quantitative data, can be reasonably comprehensive, accurate, and predictive in the dynamical sense.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405be6d692d4836d3b3efde9333fdf1789ff05c7" target='_blank'>Systematic Quantification of Negative Feedback Mechanisms in the Extracellular Signal-regulated Kinase (ERK) Signaling Network*</a></td>
          <td>
            M. Cirit, Chun-Chao Wang, J. Haugh
          </td>
          <td>2010-09-16</td>
          <td>The Journal of Biological Chemistry</td>
          <td>91</td>
          <td>33</td>
        </tr>

        <tr id="The Ras/ERK pathway drives cell proliferation and other oncogenic behaviors, and quantifying its activity in situ is of high interest in cancer diagnosis and therapy. Pathway activation is often assayed by measuring phosphorylated ERK. However, this form of measurement overlooks dynamic aspects of signaling that can only be observed over time. In this study, we combine a live, single-cell ERK biosensor approach with multiplexed immunofluorescence staining of downstream target proteins to ask how well immunostaining captures the dynamic history of ERK activity. Combining linear regression, machine learning, and differential equation models, we develop an interpretive framework for immunostains, in which Fra-1 and pRb levels imply long term activation of ERK signaling, while Egr-1 and c-Myc indicate recent activation. We show that this framework can distinguish different classes of ERK dynamics within a heterogeneous population, providing a tool for annotating ERK dynamics within fixed tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b822e494292e7a44173abfef978a7e61987d4a1c" target='_blank'>Deciphering the History of ERK Activity from Fixed-Cell Immunofluorescence Measurements</a></td>
          <td>
            A. Ram, M. Pargett, Yongin Choi, D. Murphy, Markhus Cabel, N. Kosaisawe, G. Quon, John G. Albeck
          </td>
          <td>2024-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Extra-cellular signal regulated kinase (ERK) signal transduction is an important signaling pathway, involved in a variety of processes in cell and closely related to the occurrences of diverse tumors. This raises the question how these diverse functions are specified and coordinated. In this work, we present an integrated computational model of ERK signal transduction pathway, investigating the regulatory mechanism of ERK signal pathway by variation of the concentration of epidermal growth factor receptor (EGFR) as well as the binding affinity of adaptor proteins. The results show that EGFR and distinct adaptor proteins engender different activation patterns of ERK, which will bring different cellular responses. It could be helpful to understand the mechanism of ERK signal pathway and the relationship between ERK signal network and tumor development or therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8834b7200d12495f21190553b8a4f1751ce0a835" target='_blank'>Dynamic Model and Regulatory Mechanism of Integrated ERK Signal Pathway Activated by Epidermal Growth Factor</a></td>
          <td>
            H. Fang, Yan Li, Yonghua Wang, Shuwei Zhang
          </td>
          <td>2008-10-18</td>
          <td>2008 Fourth International Conference on Natural Computation</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Building mechanistic models of kinase-driven signaling pathways requires quantitative measurements of protein phosphorylation across physiologically relevant conditions, but this is rarely done because of the insensitivity of traditional technologies. By using a multiplexed deep phosphoproteome profiling workflow, we were able to generate a deep phosphoproteomics dataset of the EGFR-MAPK pathway in non-transformed MCF10A cells across physiological ligand concentrations with a time resolution of <12 min and in the presence and absence of multiple kinase inhibitors. An improved phosphosite mapping technique allowed us to reliably identify >46,000 phosphorylation sites on >6600 proteins, of which >4500 sites from 2110 proteins displayed a >2-fold increase in phosphorylation in response to EGF. This data was then placed into a cellular context by linking it to 15 previously published protein databases. We found that our results were consistent with much, but not all previously reported data regarding the activation and negative feedback phosphorylation of core EGFR-ERK pathway proteins. We also found that EGFR signaling is biphasic with substrates downstream of RAS/MAPK activation showing a maximum response at <3ng/ml EGF while direct substrates, such as HGS and STAT5B, showing no saturation. We found that RAS activation is mediated by at least 3 parallel pathways, two of which depend on PTPN11. There appears to be an approximately 4-minute delay in pathway activation at the step between RAS and RAF, but subsequent pathway phosphorylation was extremely rapid. Approximately 80 proteins showed a >2-fold increase in phosphorylation across all experiments and these proteins had a significantly higher median number of phosphorylation sites (~18) relative to total cellular phosphoproteins (~4). Over 60% of EGF-stimulated phosphoproteins were downstream of MAPK and included mediators of cellular processes such as gene transcription, transport, signal transduction and cytoskeletal arrangement. Their phosphorylation was either linear with respect to MAPK activation or biphasic, corresponding to the biphasic signaling seen at the level of the EGFR. This deep, integrated phosphoproteomics data resource should be useful in building mechanistic models of EGFR and MAPK signaling and for understanding how downstream responses are regulated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c1f72033b1ae81f91de5f869e07e891d11153d8" target='_blank'>A Phosphoproteomics Data Resource for Systems-level Modeling of Kinase Signaling Networks</a></td>
          <td>
            Song Feng, J. Sanford, Thomas Weber, Chelsea Hutchinson-Bunch, Panshak P. Dakup, V. Paurus, Kwame Attah, H. Sauro, W. Qian, H. Wiley
          </td>
          <td>2023-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ac9c640391085d36e30bce8cd2eff1affe6082c" target='_blank'>CDK2 activity crosstalk on the ERK kinase translocation reporter can be resolved computationally.</a></td>
          <td>
            Timothy E. Hoffman, Chengzhe Tian, Varuna Nangia, Chen Yang, S. Regot, Luca Gerosa, Sabrina L Spencer
          </td>
          <td>2025-01-01</td>
          <td>Cell systems</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dd8c0e59cc6f86078f2c60ed2d73016f4951d76" target='_blank'>Computational modelling of cancerous mutations in the EGFR/ERK signalling pathway</a></td>
          <td>

          </td>
          <td>2009-10-05</td>
          <td>BMC Systems Biology</td>
          <td>64</td>
          <td>0</td>
        </tr>

        <tr id="An error led to part of Figure 2 being removed, breaking the red line that should have linked the grey circles “PKC” and “EGFR”. The correct version of Figure 2 is now displayed on both the PDF and HTML versions of the article.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/892ad05c7eb0020aec314c7a7afee5e1e1dd1b95" target='_blank'>Correction: Integrative Modelling of the Influence of MAPK Network on Cancer Cell Fate Decision</a></td>
          <td>
            L. Grieco, Laurence Calzone, I. Bernard-Pierrot, F. Radvanyi, B. Kahn-Perlès, D. Thieffry
          </td>
          <td>2013-11-01</td>
          <td>PLoS Computational Biology</td>
          <td>41</td>
          <td>61</td>
        </tr>

        <tr id="Cell responses are actuated by tightly controlled signal transduction pathways. Although the concept of an integrated signaling network replete with interpathway cross-talk and feedback regulation is broadly appreciated, kinetic data of the type needed to characterize such interactions in conjunction with mathematical models are lacking. In mammalian cells, the Ras/ERK pathway controls cell proliferation and other responses stimulated by growth factors, and several cross-talk and feedback mechanisms affecting its activation have been identified. In this work, we take a systematic approach to parse the magnitudes of multiple regulatory mechanisms that attenuate ERK activation through canonical (Ras-dependent) and non-canonical (PI3K-dependent) pathways. In addition to regulation of receptor and ligand levels, we consider three layers of ERK-dependent feedback: desensitization of Ras activation, negative regulation of MEK kinase (e.g. Raf) activities, and up-regulation of dual-specificity ERK phosphatases. Our results establish the second of these as the dominant mode of ERK self-regulation in mouse fibroblasts. We further demonstrate that kinetic models of signaling networks, trained on a sufficient diversity of quantitative data, can be reasonably comprehensive, accurate, and predictive in the dynamical sense.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405be6d692d4836d3b3efde9333fdf1789ff05c7" target='_blank'>Systematic Quantification of Negative Feedback Mechanisms in the Extracellular Signal-regulated Kinase (ERK) Signaling Network*</a></td>
          <td>
            M. Cirit, Chun-Chao Wang, J. Haugh
          </td>
          <td>2010-09-16</td>
          <td>The Journal of Biological Chemistry</td>
          <td>91</td>
          <td>33</td>
        </tr>

        <tr id="To understand how integration of multiple data types can help decipher cellular responses at the systems level, we analyzed the mitogenic response of human mammary epithelial cells to epidermal growth factor (EGF) using whole genome microarrays, mass spectrometry-based proteomics and large-scale western blots with over 1000 antibodies. A time course analysis revealed significant differences in the expression of 3172 genes and 596 proteins, including protein phosphorylation changes measured by western blot. Integration of these disparate data types showed that each contributed qualitatively different components to the observed cell response to EGF and that varying degrees of concordance in gene expression and protein abundance measurements could be linked to specific biological processes. Networks inferred from individual data types were relatively limited, whereas networks derived from the integrated data recapitulated the known major cellular responses to EGF and exhibited more highly connected signaling nodes than networks derived from any individual dataset. While cell cycle regulatory pathways were altered as anticipated, we found the most robust response to mitogenic concentrations of EGF was induction of matrix metalloprotease cascades, highlighting the importance of the EGFR system as a regulator of the extracellular environment. These results demonstrate the value of integrating multiple levels of biological information to more accurately reconstruct networks of cellular response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f4d23a95c29e3c93f9d7bdac57936862d5438a5" target='_blank'>Network Analysis of Epidermal Growth Factor Signaling Using Integrated Genomic, Proteomic and Phosphorylation Data</a></td>
          <td>
            K. Waters, Tao Liu, Ryan D. Quesenberry, A. Willse, Somnath Bandyopadhyay, L. Kathmann, T. Weber, Richard D. Smith, H. Wiley, B. Thrall
          </td>
          <td>2012-03-29</td>
          <td>PLoS ONE</td>
          <td>46</td>
          <td>133</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17b108d1790cf60844d2f4cd74ce00efffd7bc4b" target='_blank'>The genome-scale metabolic model iIN800 of Saccharomyces cerevisiae and its validation: a scaffold to query lipid metabolism</a></td>
          <td>
            I. Nookaew, Michael C. Jewett, Asawin Meechai, Chinae Thammarongtham, Kobkul Laoteng, Supapon Cheevadhanarak, Jens Nielsen, Sakarindr Bhumiratana
          </td>
          <td>2008-08-07</td>
          <td>BMC Systems Biology</td>
          <td>383</td>
          <td>4</td>
        </tr>

        <tr id="The phosphorylation of cellular proteins plays a crucial role in the transduction of various signals from outside the cell into the nucleus. The signals are transduced by phosphorylation chain reactions within multiple pathways; however, determining which pathways are responsible for each defined signal has proven challenging. To estimate the activity of each pathway, we developed a phosphorylation array platform comprising a protein array with 1200 proteins belonging to 376 signalling pathways and an analytical method to estimate pathway activity based on the phosphorylation levels of proteins. The performance of our system was assessed by reconstructing kinase–substrate relationships, as well as by estimating pathway activity upon epidermal growth factor (EGF) stimulation and the pharmacological inhibition of epidermal growth factor receptor (EGFR). As a result, kinase–substrate relationships were reliably reconstructed based on the precise measurement of phosphorylation levels of constituent proteins on the array. Furthermore, the pathway activities associated with EGF stimulation and EGFR inhibition were successfully traced through the related pathways from the outer membrane to the nucleus along a time course. Thus, our phosphorylation array system can effectively assess the activity of specific signalling pathways that are perturbed by extracellular stimuli, such as various drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f9c27ddb3e4d2f7897fe0cb3dc1a2581cdd3b73" target='_blank'>Assessing the activation/inhibition of tyrosine kinase‐related pathways with a newly developed platform</a></td>
          <td>
            Harumi Kagiwada, Takayuki Kiboku, Hitomi Matsuo, M. Kitazawa, K. Fukui, K. Horimoto
          </td>
          <td>2021-06-21</td>
          <td>PROTEOMICS</td>
          <td>5</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6a42ae353dba2ac0298dd73047678c91811541f" target='_blank'>From pathway to population – a multiscale model of juxtacrine EGFR-MAPK signalling</a></td>
          <td>
            Dawn C. Walker, N. T. Georgopoulos, Jennifer Southgate
          </td>
          <td>2008-11-26</td>
          <td>BMC Systems Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17b108d1790cf60844d2f4cd74ce00efffd7bc4b" target='_blank'>The genome-scale metabolic model iIN800 of Saccharomyces cerevisiae and its validation: a scaffold to query lipid metabolism</a></td>
          <td>
            I. Nookaew, Michael C. Jewett, Asawin Meechai, Chinae Thammarongtham, Kobkul Laoteng, Supapon Cheevadhanarak, Jens Nielsen, Sakarindr Bhumiratana
          </td>
          <td>2008-08-07</td>
          <td>BMC Systems Biology</td>
          <td>383</td>
          <td>4</td>
        </tr>

        <tr id="Extra-cellular signal regulated kinase (ERK) signal network plays an important role in regulating the fundamental cellular functions such as cell proliferation, survival, differentiation and motility. Development and analysis of mathematical model can help us acquire a deep understanding of the complex behavior of ERK signal transduction network. This paper presents a computational model that offers an integrated quantitative and dynamic simulation of ERK signal transduction network, activated by epidermal growth factor (EGF). The mathematic model contains the activation kinetics of the pathway, a multitude of feedback loops and connection of scaffold proteins. The model provides insight into signal-response relationships between the binding of EGF to its receptor at the cell surface and activation of downstream proteins in the signaling cascade. The different influence of positive and negative feedback loops of the ERK signal transduction pathway were mainly analyzed, indicating that feedback loops were the main impacting factor to the oscillation of ERK signal transduction pathway. The predictions of this oscillation of ERK activation agree well with the literature. It can lead to signal waves of the downstream substrates and induce corresponding biological behaviors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7bf832c4d17a797f80bffbcb87cce1c1078a91f" target='_blank'>Dynamic Analysis of Feedback Loops in Extra-Cellular Signal Regulated Kinase Signal Network</a></td>
          <td>
            Yong Han, Yan Li, Yonghua Wang, H. Fang, Shuwei Zhang
          </td>
          <td>2009-03-31</td>
          <td>2009 WRI World Congress on Computer Science and Information Engineering</td>
          <td>3</td>
          <td>34</td>
        </tr>

        <tr id="Identification of dynamic signaling mechanisms on different cellular layers is now facilitated as the increased usage of various high-throughput techniques goes along with decreasing costs for individual experiments. A lot of these signaling mechanisms are known to be coordinated by their dynamics, turning time-course data sets into valuable information sources for inference of regulatory mechanisms. However, the combined analysis of parallel time-course measurements from different high-throughput platforms still constitutes a major challenge requiring sophisticated bioinformatic tools in order to ease biological interpretation. We developed a new pathway-based integration approach for the analysis of coupled omics time-series data, which we implemented in the R package pwOmics. Unlike many other approaches, our approach acknowledges the role of the different cellular layers of measurement and infers consensus profiles and time profile clusters for further biological interpretation. We investigated a time-course data set on epidermal growth factor stimulation of human mammary epithelial cells generated on the two layers of RNA and proteins. The data was analyzed using our new approach with a focus on feedback signaling and pathway crosstalk. We could confirm known regulatory patterns relevant in the physiological cellular response to epidermal growth factor stimulation as well as identify interesting new interactions in this signaling context, such as the regulatory influence of the connective tissue growth factor on transferrin receptor or the influence of growth arrest and DNA-damage-inducible alpha on the connective tissue growth factor. Thus, we show that integrated cross-platform analysis provides a deeper understanding of regulatory signaling mechanisms. Combined with time-course information it enables the characterization of dynamic signaling processes and leads to the identification of important regulatory interactions which might be dysregulated in disease with adverse effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f1eaf336391af0aad332a04b8889875c71007df" target='_blank'>Decoding Cellular Dynamics in Epidermal Growth Factor Signaling Using a New Pathway-Based Integration Approach for Proteomics and Transcriptomics Data</a></td>
          <td>
            A. Wachter, T. Beissbarth
          </td>
          <td>2016-01-07</td>
          <td>Frontiers in Genetics</td>
          <td>3</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/509f1971d67425c00eb55f435cf67274acb0fe25" target='_blank'>Computational modeling of the dynamics of the MAP kinase cascade activated by surface and internalized EGF receptors</a></td>
          <td>
            B. Schoeberl, Claudia Eichler-Jonsson, E. Gilles, G. Müller
          </td>
          <td>2002-04-01</td>
          <td>Nature Biotechnology</td>
          <td>1026</td>
          <td>48</td>
        </tr>

        <tr id="Cellular signaling lies at the core of cellular behavior, and is central for the understanding of many pathologic conditions. To comprehend how signal transduction is orchestrated at the molecular level remains the ultimate challenge for cell biology. In the last years there has been a revolution in the development of high-throughput methodologies in proteomics and genomics, which have provided extensive knowledge about expression profiles and molecular interaction-networks. However, these methods have typically provided qualitative and static information. This is about to turn, and several high-throughput methods are now available that provide quantitative and temporal information. These data are well suited for analysis by computational methods and bioinformatics, which are becoming increasingly valuable tools to grasp the complexity of cellular networks. At present, several cellular pathways have been modeled in silico and the analysis provides new understanding of the underlying properties that contribute to their dynamic features. Here, we review methodologies that are used for in silico modeling as well as methods to obtain large-scale quantitative data, and discuss how they can be integrated to generate powerful and predictive models of cellular processes. We argue that the generation of such models provide powerful tools to understand how systems properties emerges in healthy and pathologic states, and to generate efficient strategies for pharmacological intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95fc342e11b6855073464184482f87d7d1258c83" target='_blank'>Proteomic and computational methods in systems modeling of cellular signaling.</a></td>
          <td>
            R. Kleppe, E. Kjarland, F. Selheim
          </td>
          <td>2006-05-31</td>
          <td>Current pharmaceutical biotechnology</td>
          <td>9</td>
          <td>21</td>
        </tr>

        <tr id="The EGF-induced MAP kinase cascade is one of the most important and best characterized networks in intracellular signalling. It has a vital role in the development and maturation of living organisms. However, when deregulated, it is involved in the onset of a number of diseases. Based on a computational model describing a “surface” and an “internalized” parallel route, we use systems biology techniques to characterize aspects of the network’s functional organization. We examine the re-organization of protein groups from low to high external stimulation, define functional groups of proteins within the network, determine the parameter best encoding for input intensity and predict the effect of protein removal to the system’s output response. Extensive functional re-organization of proteins is observed in the lower end of stimulus concentrations. As we move to higher concentrations the variability is less pronounced. 6 functional groups have emerged from a consensus clustering approach, reflecting different dynamical aspects of the network. Mutual information investigation revealed that the maximum activation rate of the two output proteins best encodes for stimulus intensity. Removal of each protein of the network resulted in a range of graded effects, from complete silencing to intense activation. Our results provide a new “vista” of the EGF-induced MAP kinase cascade, from the perspective of complex self-organizing systems. Functional grouping of the proteins reveals an organizational scheme contrasting the current understanding of modular topology. The six identified groups may provide the means to experimentally follow the dynamics of this complex network. Also, the vulnerability analysis approach may be used for the development of novel therapeutic targets in the context of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccd7041af39a1cc5e66884c8aacd4b45fc5c205d" target='_blank'>Functional Aspects of the EGF-Induced MAP Kinase Cascade: A Complex Self-Organizing System Approach</a></td>
          <td>
            E. Kosmidis, V. Moschou, Georgios Ziogas, I. Boukovinas, M. Albani, N. Laskaris
          </td>
          <td>2014-11-05</td>
          <td>PLoS ONE</td>
          <td>1</td>
          <td>38</td>
        </tr>

        <tr id="For more than a decade, the MAPK (mitogen-activated protein kinase) cascade has been studied using mathematical modelling and quantitative experimentation [1]. The MAPK cascade relays the presence of extracellular stimuli such as growth hormones to the nucleus and controls the expression of hundreds of genes. MAPKs control major cell fate decisions such as proliferation, differentiation and apoptosis, mainly by inducing alterations in gene expression. In this chapter, we discuss how systems biology analysis provides insights into the functioning of this cascade. We show how this pathway assists the cell in responding properly to extracellular cues by filtering out sub-threshold stimuli, while efficiently transmitting physiologically relevant inputs. Several different receptors signal through the MAPK pathway even though they elicit opposite biological responses, thus raising the question of how specificity is achieved in MAPK signalling. Experimental studies revealed that specific biological responses are encoded by quantitative aspects of the MAPK signal such as amplitude or duration. We discuss mechanisms that enable the pathway to generate quantitatively different signals, and also explain how different signals are interpreted by the downstream gene expression machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6240690e740bb11ac1d42f43f91ffcfad059dbaf" target='_blank'>Systems analysis of MAPK signal transduction.</a></td>
          <td>
            N. Blüthgen, S. Legewie
          </td>
          <td>2008-09-30</td>
          <td>Essays in biochemistry</td>
          <td>68</td>
          <td>42</td>
        </tr>

        <tr id="Motivation Signal transduction networks regulate a multitude of essential biological processes and are frequently aberrated in diseases such as cancer. Developing a mechanistic understanding of such networks is essential to understand disease or cell population specific signaling and to design effective treatment strategies. Typically, such networks are computationally reconstructed based on systematic perturbation experiments, followed by quantification of signaling protein activity. Recent technological advances now allow for the quantification of the activity of many (signaling) proteins simultaneously in single cells. This makes it feasible to reconstruct signaling networks from single cell data. Results Here we introduce single cell Comparative Network Reconstruction (scCNR) to derive signal transduction networks by exploiting the heterogeneity of single cell (phospho)protein measurements. scCNR treats stochastic variation in total protein abundances as natural perturbation experiments, whose effects propagate through the network. scCNR reconstructs cell population specific networks of the same underlying topology for cells from diverse populations. We extensively validated scCNR on simulated single cell data, and we applied it to a dataset of EGFR-inhibitor treated keratinocytes to recover signaling differences downstream of EGFR and in protein interactions associated with proliferation. scCNR will help to unravel the mechanistic signaling differences between cell populations by making use of single-cell data, and will subsequently guide the development of well-informed treatment strategies. Availability and implementation scCNR is available as a python module at https://github.com/ibivu/scmra. Additionally, code to reproduce all figures is available at https://github.com/tstohn/scmra_analysis. Supplementary information Supplementary information and data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/516c13660aa9eadff6cefbce4ac0292281fdc217" target='_blank'>Reconstructing and comparing signal transduction networks from single cell protein quantification data</a></td>
          <td>
            Tim Stohn, Roderick van Eijl, Klaas W. Mulder, L. Wessels, Evert Bosdriesz
          </td>
          <td>2024-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Cascades of phosphorylation between protein kinases comprise a core mechanism in the integration and propagation of intracellular signals. Although we have accumulated a wealth of knowledge around some such pathways, this is subject to study biases and much remains to be uncovered. Phosphoproteomics, the identification and quantification of phosphorylated proteins on a proteomic scale, provides a high-throughput means of interrogating the state of intracellular phosphorylation, both at the pathway level and at the whole-cell level. In this review, we discuss methods for using human quantitative phosphoproteomic data to reconstruct the underlying signalling networks that generated it. We address several challenges imposed by the data on such analyses and we consider promising advances towards reconstructing unbiased, kinome-scale signalling networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be50267d98477e448ac4e49d08a64c94ebfea1b1" target='_blank'>Reconstructing phosphorylation signalling networks from quantitative phosphoproteomic data</a></td>
          <td>
            B. Invergo, P. Beltrão
          </td>
          <td>2018-08-02</td>
          <td>Essays in Biochemistry</td>
          <td>31</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75f581981a222f5b0749f3a969807d0aa573b574" target='_blank'>Creating and analyzing pathway and protein interaction compendia for modelling signal transduction networks</a></td>
          <td>
            D. Kirouac, J. Sáez-Rodríguez, J. Swantek, J. Burke, D. Lauffenburger, P. Sorger
          </td>
          <td>2012-05-01</td>
          <td>BMC Systems Biology</td>
          <td>77</td>
          <td>119</td>
        </tr>

        <tr id="Signal transduction is a complex protein signaling process with a rich network of multifunctional interactions that occur in a non‐linear fashion. Mitogen‐activated protein kinase (MAPK) signal transduction pathways regulate diverse cellular processes ranging from proliferation and differentiation to apoptosis. In mammals, out of five, there are three well characterized subfamilies of MAPKs ‐ ERKs (Extracellular signal‐regulated kinases), JNKs (c‐Jun N‐terminal kinases), and P38 kinases, and their activators, are implicated in human diseases and are targets for drug development. Kinase cascades in MAPK pathways mediate the sensing and processing of stimuli. To understand how cells makes decisions, the dynamic interactions of components of signaling cascades are important rather than just creating static maps. Based on enzyme kinetic reactions, we have developed a mathematical model to analyze the impact of the cross‐talks between JNK and P38 kinase cascades. Cross‐talks between JNK and P38 kinase cascades influence the activities of P38 kinases. Responses of the signals should be studied for network of kinase cascades by considering cross‐talks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef12bcae9ca82339a744c3b732a8bd77d348ac39" target='_blank'>Computational prediction and analysis of impact of the cross‐talks between JNK and P38 kinase cascades</a></td>
          <td>
            Pandurangan Sundaramurthy, S. Gakkhar, R. Sowdhamini
          </td>
          <td>2009-01-12</td>
          <td>Bioinformation</td>
          <td>12</td>
          <td>32</td>
        </tr>

        <tr id="The site-specific function of EGFR’s C-terminal tyrosines is interrogated by phosphotyrosine phosphoproteomics to build a data-driven model that links cell response to signaling nodes and elucidates novel associations in the EGFR signaling network. The epidermal growth factor receptor (EGFR) has been studied extensively because of its critical role in cellular signaling and association with disease. Previous models have elucidated interactions between EGFR and downstream adaptor proteins or showed phenotypes affected by EGFR. However, the link between specific EGFR phosphorylation sites and phenotypic outcomes is still poorly understood. Here, we employed a suite of isogenic cell lines expressing site-specific mutations at each of the EGFR C-terminal phosphorylation sites to interrogate their role in the signaling network and cell biological response to stimulation. Our results demonstrate the resilience of the EGFR network, which was largely similar even in the context of multiple Y-to-F mutations in the EGFR C-terminal tail, while also revealing nodes in the network that have not previously been linked to EGFR signaling. Our data-driven model highlights the signaling network nodes associated with distinct EGF-driven cell responses, including migration, proliferation, and receptor trafficking. Application of this same approach to less-studied RTKs should provide a plethora of novel associations that should lead to an improved understanding of these signaling networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d1bedc5d16450a77fda0fc2c816d3cd1cf3570c" target='_blank'>Predictive data-driven modeling of C-terminal tyrosine function in the EGFR signaling network</a></td>
          <td>
            Jacqueline S. Gerritsen, Joseph S Faraguna, R. Bonavia, F. Furnari, F. White
          </td>
          <td>2023-05-11</td>
          <td>Life Science Alliance</td>
          <td>2</td>
          <td>66</td>
        </tr>

        <tr id="The biophysical properties of ligand binding heavily influence the ability of receptors to specify cell fates. Understanding the rules by which ligand binding kinetics impact cell phenotype is challenging, however, because of the coupled information transfers that occur from receptors to downstream signaling effectors and from effectors to phenotypes. Here, we address that issue by developing an integrated mechanistic and data-driven computational modeling platform to predict cell responses to different ligands for the epidermal growth factor receptor (EGFR). Experimental data for model training and validation were generated using MCF7 human breast cancer cells treated with the high- and low-affinity ligands epidermal growth factor (EGF) and epiregulin (EREG), respectively. The integrated model captures the unintuitive, concentration-dependent abilities of EGF and EREG to drive signals and phenotypes differently, even at similar levels of receptor occupancy. For example, the model correctly predicts the dominance of EREG over EGF in driving a cell differentiation phenotype through AKT signaling at intermediate and saturating ligand concentrations and the ability of EGF and EREG to drive a broadly concentration-sensitive migration phenotype through cooperative ERK and AKT signaling. Parameter sensitivity analysis identifies EGFR endocytosis, which is differentially regulated by EGF and EREG, as one of the most important determinants of the alternative phenotypes driven by different ligands. The integrated model provides a new platform to predict how phenotypes are controlled by the earliest biophysical rate processes in signal transduction and may eventually be leveraged to understand receptor signaling system performance depends on cell context. One-sentence summary Integrated kinetic and data-driven EGFR signaling model identifies the specific signaling mechanisms that dictate cell responses to EGFR activation by different ligands.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccddd2d0cde0d83a0a8cfb56bef3dd0624d5226" target='_blank'>An integrated mechanistic and data-driven computational model predicts cell responses to high- and low-affinity EGFR ligands</a></td>
          <td>
            Paul J. Myers, Sung Hyun Lee, M. Lazzara
          </td>
          <td>2023-06-26</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>22</td>
        </tr>

        <tr id="The transforming growth factor-β (TGF-β) signal transduction pathway controls many cellular processes, including differentiation, proliferation and apoptosis. It plays a fundamental role during development and it is dysregulated in many diseases. The factors that control the dynamics of the pathway, however, are not fully elucidated yet and so far computational approaches have been very limited in capturing the distinct types of behaviour observed under different cellular backgrounds and conditions into a single-model description. Here, we develop a detailed computational model for TGF-β signalling that incorporates elements of previous models together with crosstalking between Smad1/5/8 and Smad2/3 channels through a negative feedback loop dependent on Smad7. The resulting model accurately reproduces the diverse behaviour of experimental datasets for human keratinocytes, bovine aortic endothelial cells and mouse mesenchymal cells, capturing the dynamics of activation and nucleocytoplasmic shuttling of both R-Smad channels. The analysis of the model dynamics and its system properties revealed Smad7-mediated crosstalking between Smad1/5/8 and Smad2/3 channels as a major determinant in shaping the distinct responses to single and multiple ligand stimulation for different cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9e9f9f231660dad9ab1748d623f51d45358b5c5" target='_blank'>Computational modelling of Smad-mediated negative feedback and crosstalk in the TGF-β superfamily network</a></td>
          <td>
            D. Nicklas, L. Saiz
          </td>
          <td>2013-09-06</td>
          <td>Journal of The Royal Society Interface</td>
          <td>29</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Computer simulations and mathematical modeling of biological processes are becoming increasingly popular, and yet the complexity of the biochemical systems or the differences between experimental setups make it very difficult to establish a standard formula for these modeling projects. Before we can start using computer‐based models for predictions or targeted experiment designs, it is very important to establish a reliable model on which those predictions can be based and experimentally tested. Here we attempt to present a computer model for the mitogen‐activated protein kinase (MAPK) signaling cascade which is consistent with previously published experimental results. In this study we have focused our attention to a generic MAPK ERK (extracellular signal‐regulated kinase) pathway activated by epidermal growth factor (EGF) in an attempt to understand how receptors may achieve different activation kinetics of the MAPK signaling. We successfully show that the level of receptor expression is one key determinant in this regulation, and that the binding affinity of the active receptor to adaptor proteins can have a small but albeit direct effect on the downstream activation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dde366641aca5125603232bb38f2d61b4eda2a5" target='_blank'>A Computer‐Based Model for the Regulation of Mitogen Activated Protein Kinase (MAPK) Activation</a></td>
          <td>
            I. Aksan, M. Levent Kurnaz
          </td>
          <td>2003-01-01</td>
          <td>Journal of Receptors and Signal Transduction</td>
          <td>26</td>
          <td>4</td>
        </tr>

        <tr id="The epidermal growth factor receptor (EGFR) signalling pathway is a complex signalling process with a wide network of interactions. The activation of the mitogen-activated protein kinases (MAPKs) cascade by this activated EGFR has been well studied. MAPKs form a highly integrated network, which is essential for certain specialised cell functions. This paper presents a kinetic model for the MAPK pathway downstream of the EGFR using a biochemical simulator. The model includes 30 signalling events and 29 signalling molecules. The time course data were examined for the activation of each signalling component. The simulation provides a large volume of data, by monitoring the kinetics of the signalling components, which were compared experimentally using the PC12 cell line. The kinetic model corresponded well with the experimental results observed in the EGFR induced activation of proteins. An examination of the kinetic analysis of the multiple signalling events provides a quantitative framework for representing the EGFR signalling network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ab4070943b1039a9e896666b7a81fc1863c7dc4" target='_blank'>Simulation and sensitivity analysis of phosphorylation of EGFR signal transduction pathway in PC12 cell model.</a></td>
          <td>
            C. V. S. Babu, S. Yoon, H. Nam, Young Sook Yoo
          </td>
          <td>2004-12-01</td>
          <td>Systems biology</td>
          <td>20</td>
          <td>5</td>
        </tr>

        <tr id="Signal network assumes a vital part in directing the principal cell capacities, for example, cell expansion, survival, separation and motility. Improvement and investigation of scientific model can help us gain a profound comprehension of the unpredictable conduct of ERK flag transduction organizes. This paper exhibits a computational model that offers an incorporated quantitative and dynamic reproduction of ERK flag transduction arranges, actuated by epidermal development figure. The mathematic demonstrate contains the enactment energy of the pathway, a huge number of input circles and association of platform proteins. The model gives knowledge into flag reaction connections between the authoritative of EGF to its receptor at the phone surface and actuation of downstream proteins in the flagging course. The diverse impact of positive and negative input circles of the ERK flag transduction pathway were for the most part examined, showing that criticism circles were the primary affecting variable to the swaying of ERK flag transduction pathway. The forecasts of this wavering of ERK enactment concur well with the writing. It can prompt flag floods of the downstream substrates and instigate relating natural practices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b0c50699f15bb521375edbc6061e541e8ccaea4" target='_blank'>Analysis of Regulated Kinase Signal Network through Feedback Loops in Extra-Cellular Signal</a></td>
          <td>
            C. Periyasamy Periyasamy
          </td>
          <td>2017-11-01</td>
          <td>Indonesian Journal of Electrical Engineering and Computer Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cellular information processing in signaling networks forms the basis of responses to environmental stimuli. At any given time, cells receive multiple simultaneous input cues, which are processed and integrated to determine cellular responses such as migration, proliferation, apoptosis, or differentiation. Protein phosphorylation events play a major role in this process and are often involved in fundamental biological and cellular processes such as protein‐protein interactions, enzyme activity, and immune responses. Determining which kinases phosphorylate specific phospho sites poses a challenge; this information is critical when trying to elucidate key proteins involved in specific cellular responses. Here, methods to generate high‐quality quantitative phosphorylation data from cell lysates originating from primary cells, and how to analyze the generated data to construct quantitative signaling network models, are presented. These models can subsequently be used to guide follow‐up in vitro/in vivo validation studies. Curr. Protoc. Immunol. 104:11.11.1‐11.11.23. © 2014 by John Wiley & Sons, Inc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a0c788b0085bd2d7a1596f03e8f17950e8a0d6e" target='_blank'>Experimental and Computational Tools for Analysis of Signaling Networks in Primary Cells</a></td>
          <td>
            E. Schoof, R. Linding
          </td>
          <td>2014-02-01</td>
          <td>Current Protocols in Immunology</td>
          <td>2</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55d7c99e47cbd5fed329f3182a9921e9443d5c69" target='_blank'>Integrated Platform for Monitoring Single-cell MAPK Kinetics in Computer-controlled Temporal Stimulations</a></td>
          <td>
            H. Ryu, M. Chung, Jiyoung Song, Sung Sik Lee, O. Pertz, N. Jeon
          </td>
          <td>2018-07-24</td>
          <td>Scientific Reports</td>
          <td>10</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832b8a763bca673f9ef9a922d08084ef49416b11" target='_blank'>Systems Analysis of EGF Receptor Signaling Dynamics with Micro-Western Arrays</a></td>
          <td>
            Mark F. Ciaccio, Joel Wagner, Chih-Pin Chuu, D. Lauffenburger, R. Jones
          </td>
          <td>2010-01-24</td>
          <td>Nature methods</td>
          <td>209</td>
          <td>119</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34fe2e9cd75c191dcaf20c391be13e5dd1d08d0e" target='_blank'>Integrating Multiplex Single-Molecule Pull-Down (SiMPull) Data and Computational Modeling to Understand EGFR Signaling</a></td>
          <td>
            Emanuel Salazar-Cavazos, B. Wilson, K. Lidke, W. Hlavacek, D. Lidke
          </td>
          <td>2018-02-01</td>
          <td>Biophysical Journal</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background The ErbB family of receptors activates intracellular signaling pathways that control cellular proliferation, growth, differentiation and apoptosis. Given these central roles, it is not surprising that overexpression of the ErbB receptors is often associated with carcinogenesis. Therefore, extensive laboratory studies have been devoted to understanding the signaling events associated with ErbB activation. Methodology/Principal Findings Systems biology has contributed significantly to our current understanding of ErbB signaling networks. However, although computational models have grown in complexity over the years, little work has been done to consider the spatial-temporal dynamics of receptor interactions and to evaluate how spatial organization of membrane receptors influences signaling transduction. Herein, we explore the impact of spatial organization of the epidermal growth factor receptor (ErbB1/EGFR) on the initiation of downstream signaling. We describe the development of an algorithm that couples a spatial stochastic model of membrane receptors with a nonspatial stochastic model of the reactions and interactions in the cytosol. This novel algorithm provides a computationally efficient method to evaluate the effects of spatial heterogeneity on the coupling of receptors to cytosolic signaling partners. Conclusions/Significance Mathematical models of signal transduction rarely consider the contributions of spatial organization due to high computational costs. A hybrid stochastic approach simplifies analyses of the spatio-temporal aspects of cell signaling and, as an example, demonstrates that receptor clustering contributes significantly to the efficiency of signal propagation from ligand-engaged growth factor receptors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f507b023b72e131b6fdde1ebc1c7bd9aef05c4a7" target='_blank'>Coupled Stochastic Spatial and Non-Spatial Simulations of ErbB1 Signaling Pathways Demonstrate the Importance of Spatial Organization in Signal Transduction</a></td>
          <td>
            Michelle N. Costa, K. Radhakrishnan, B. Wilson, D. Vlachos, J. Edwards
          </td>
          <td>2009-07-23</td>
          <td>PLoS ONE</td>
          <td>32</td>
          <td>95</td>
        </tr>

        <tr id="The epidermal growth factor receptor (EGFR) activated extracellular-signal regulated kinase (ERK) pathway is a central cell signalling pathway that mediates many biological responses including cell proliferation, transformation, survival and motility. Deregulation of the pathway either through mutation of components or overexpression of EGFRs is associated with several forms of cancer. Under normal conditions, EGF stimulates a rapid but transient activation of ERK as the signal is rapidly shutdown, whereas under cancerous conditions, the ERK signal cannot be shutdown and is sustained. Computational modelling techniques have been used to investigate the signalling dynamics of the EGFR/ERK pathway, focusing on identifying the key processes involved in signal termination and what role the ERK to son of sevenless (SOS) negative feedback loop plays in generating a transient response. This model predicts that this negative feedback loop is not needed to achieve a transient activation of ERK as the process of receptor degradation alone is enough to terminate the signal. Importantly, the behaviour and predictions of this model are verified with laboratory data, as is essential for modern systems biology approaches. Further analysis showed that the feedback loop and receptor degradation were both redundant processes, as each could compensate for the absence of the other. This led to the prediction that in the case of a receptor which is not degraded, such as the insulin receptor, the negative feedback loop to SOS will actually be essential for a transient response to be achieved. Overall, the results shed new light on the role of negative feedback in EGF receptor signalling and suggest that different receptors are dependent on different features within the ERK pathway when relaying their signals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35e7094d70c15e7e0196f4a688c075edd2e156e0" target='_blank'>Computational modelling reveals feedback redundancy within the epidermal growth factor receptor/extracellular-signal regulated kinase signalling pathway.</a></td>
          <td>
            Richard J. Orton, Oliver E. Sturm, Amelie Gormand, Walter Kolch, David R. Gilbert
          </td>
          <td>2008-08-01</td>
          <td>IET systems biology</td>
          <td>21</td>
          <td>13</td>
        </tr>

        <tr id="5371 Background: The overexpression/aberrant activation of the epidermal growth factor receptor and its downstream signaling pathways are common factors associated with the progression of many human tumors. While western blot (WB), RIA, or ELISA are typically used to evaluate relative levels of expression/activation, immunohistochemistry (IHC) provides the ability to quantify and localize intracellular molecules in individual cells and/or within regions of the malignancy and the use of archival pathological specimens. We have validated an approach of IHC staining combined with automated digital microscopy (ADM) analysis against WB using a series of cell culture model systems. This has provided a system by which routine IHC can be optimized to maintain reproducible, precise, and objective quantification. Methods: PhosphoEGFR (pEGFR) was evaluated in A431 cells following stimulation with EGF. Similarly, phosphoMAPK (pMAPK) and phosphoAkt (pAKT) were assessed in Jurkat cells stimulated with PMA or LY294002, respectively. EGFR and HER2 were evaluated in A431, MDA-231, MDA-175, HeLa-S3, and SKBR-3 cell lines. Cell suspensions were split immediately following treatment for WB analysis and formalin fixation/paraffin embedding and IHC with ADM analysis. Immunoblots were digitally scanned and quantified by image analysis. Further refinement of the IHC staining procedure was performed, included varying antigen retrieval, antibody, and substrate conditions, to develop optimal quantitative conditions. Results: Average measurement of pEGFR, pAkt and pMAPK in cell pellets by ADM revealed 5- to 20-fold changes in signal with varying treatments, correlating well with WB analysis (r = 0.960, p">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdf21403eb0730dd7642a0c5b356cdf7b9426a18" target='_blank'>Quantitative immunohistochemistry of ErbB cellular signaling pathways using automated digital microscopy</a></td>
          <td>
            K. Gottlieb, S. Webster, K. Bauer
          </td>
          <td>2004-04-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Signal transduction systems coordinate complex cellular information to regulate biological events such as cell proliferation and differentiation. Although the accumulating evidence on widespread association of signaling molecules has revealed essential contribution of phosphorylation-dependent interaction networks to cellular regulation, their dynamic behavior is mostly yet to be analyzed. Recent technological advances regarding mass spectrometry-based quantitative proteomics have enabled us to describe the comprehensive status of phosphorylated molecules in a time-resolved manner. Computational analyses based on the phosphoproteome dynamics accelerate generation of novel methodologies for mathematical analysis of cellular signaling. Phosphoproteomics-based numerical modeling can be used to evaluate regulatory network elements from a statistical point of view. Integration with transcriptome dynamics also uncovers regulatory hubs at the transcriptional level. These omics-based computational methodologies, which have firstly been applied to representative signaling systems such as the epidermal growth factor receptor pathway, have now opened up a gate for systems analysis of signaling networks involved in immune response and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c162fd711ea80a291593e11af7d278a54f2d18" target='_blank'>Phosphoproteomics-Based Systems Analysis of Signal Transduction Networks</a></td>
          <td>
            H. Kozuka-Hata, S. Tasaki, M. Oyama
          </td>
          <td>2012-01-03</td>
          <td>Frontiers in Physiology</td>
          <td>8</td>
          <td>38</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>