<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Ecad4837f4bc306f173456aafecd9d6b67b62744
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-27 06:08:41 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Regulation of ERK-MAPK signaling in human epidermis</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="Cell responses are actuated by tightly controlled signal transduction pathways. Although the concept of an integrated signaling network replete with interpathway cross-talk and feedback regulation is broadly appreciated, kinetic data of the type needed to characterize such interactions in conjunction with mathematical models are lacking. In mammalian cells, the Ras/ERK pathway controls cell proliferation and other responses stimulated by growth factors, and several cross-talk and feedback mechanisms affecting its activation have been identified. In this work, we take a systematic approach to parse the magnitudes of multiple regulatory mechanisms that attenuate ERK activation through canonical (Ras-dependent) and non-canonical (PI3K-dependent) pathways. In addition to regulation of receptor and ligand levels, we consider three layers of ERK-dependent feedback: desensitization of Ras activation, negative regulation of MEK kinase (e.g. Raf) activities, and up-regulation of dual-specificity ERK phosphatases. Our results establish the second of these as the dominant mode of ERK self-regulation in mouse fibroblasts. We further demonstrate that kinetic models of signaling networks, trained on a sufficient diversity of quantitative data, can be reasonably comprehensive, accurate, and predictive in the dynamical sense.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405be6d692d4836d3b3efde9333fdf1789ff05c7" target='_blank'>Systematic Quantification of Negative Feedback Mechanisms in the Extracellular Signal-regulated Kinase (ERK) Signaling Network*</a></td>
          <td>
            M. Cirit, Chun-Chao Wang, J. Haugh
          </td>
          <td>2010-09-16</td>
          <td>The Journal of Biological Chemistry</td>
          <td>91</td>
          <td>33</td>
        </tr>

        <tr id="The Ras/ERK pathway drives cell proliferation and other oncogenic behaviors, and quantifying its activity in situ is of high interest in cancer diagnosis and therapy. Pathway activation is often assayed by measuring phosphorylated ERK. However, this form of measurement overlooks dynamic aspects of signaling that can only be observed over time. In this study, we combine a live, single-cell ERK biosensor approach with multiplexed immunofluorescence staining of downstream target proteins to ask how well immunostaining captures the dynamic history of ERK activity. Combining linear regression, machine learning, and differential equation models, we develop an interpretive framework for immunostains, in which Fra-1 and pRb levels imply long term activation of ERK signaling, while Egr-1 and c-Myc indicate recent activation. We show that this framework can distinguish different classes of ERK dynamics within a heterogeneous population, providing a tool for annotating ERK dynamics within fixed tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b822e494292e7a44173abfef978a7e61987d4a1c" target='_blank'>Deciphering the History of ERK Activity from Fixed-Cell Immunofluorescence Measurements</a></td>
          <td>
            A. Ram, M. Pargett, Yongin Choi, D. Murphy, Markhus Cabel, N. Kosaisawe, G. Quon, John G. Albeck
          </td>
          <td>2024-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Extra-cellular signal regulated kinase (ERK) signal transduction is an important signaling pathway, involved in a variety of processes in cell and closely related to the occurrences of diverse tumors. This raises the question how these diverse functions are specified and coordinated. In this work, we present an integrated computational model of ERK signal transduction pathway, investigating the regulatory mechanism of ERK signal pathway by variation of the concentration of epidermal growth factor receptor (EGFR) as well as the binding affinity of adaptor proteins. The results show that EGFR and distinct adaptor proteins engender different activation patterns of ERK, which will bring different cellular responses. It could be helpful to understand the mechanism of ERK signal pathway and the relationship between ERK signal network and tumor development or therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8834b7200d12495f21190553b8a4f1751ce0a835" target='_blank'>Dynamic Model and Regulatory Mechanism of Integrated ERK Signal Pathway Activated by Epidermal Growth Factor</a></td>
          <td>
            H. Fang, Yan Li, Yonghua Wang, Shuwei Zhang
          </td>
          <td>2008-10-18</td>
          <td>2008 Fourth International Conference on Natural Computation</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Building mechanistic models of kinase-driven signaling pathways requires quantitative measurements of protein phosphorylation across physiologically relevant conditions, but this is rarely done because of the insensitivity of traditional technologies. By using a multiplexed deep phosphoproteome profiling workflow, we were able to generate a deep phosphoproteomics dataset of the EGFR-MAPK pathway in non-transformed MCF10A cells across physiological ligand concentrations with a time resolution of <12 min and in the presence and absence of multiple kinase inhibitors. An improved phosphosite mapping technique allowed us to reliably identify >46,000 phosphorylation sites on >6600 proteins, of which >4500 sites from 2110 proteins displayed a >2-fold increase in phosphorylation in response to EGF. This data was then placed into a cellular context by linking it to 15 previously published protein databases. We found that our results were consistent with much, but not all previously reported data regarding the activation and negative feedback phosphorylation of core EGFR-ERK pathway proteins. We also found that EGFR signaling is biphasic with substrates downstream of RAS/MAPK activation showing a maximum response at <3ng/ml EGF while direct substrates, such as HGS and STAT5B, showing no saturation. We found that RAS activation is mediated by at least 3 parallel pathways, two of which depend on PTPN11. There appears to be an approximately 4-minute delay in pathway activation at the step between RAS and RAF, but subsequent pathway phosphorylation was extremely rapid. Approximately 80 proteins showed a >2-fold increase in phosphorylation across all experiments and these proteins had a significantly higher median number of phosphorylation sites (~18) relative to total cellular phosphoproteins (~4). Over 60% of EGF-stimulated phosphoproteins were downstream of MAPK and included mediators of cellular processes such as gene transcription, transport, signal transduction and cytoskeletal arrangement. Their phosphorylation was either linear with respect to MAPK activation or biphasic, corresponding to the biphasic signaling seen at the level of the EGFR. This deep, integrated phosphoproteomics data resource should be useful in building mechanistic models of EGFR and MAPK signaling and for understanding how downstream responses are regulated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c1f72033b1ae81f91de5f869e07e891d11153d8" target='_blank'>A Phosphoproteomics Data Resource for Systems-level Modeling of Kinase Signaling Networks</a></td>
          <td>
            Song Feng, J. Sanford, Thomas Weber, Chelsea Hutchinson-Bunch, Panshak P. Dakup, V. Paurus, Kwame Attah, H. Sauro, W. Qian, H. Wiley
          </td>
          <td>2023-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="MOTIVATION
Systems biology is currently focused on integrating intracellular networks, although clinically, diseases are largely defined by their histological features. For example, no computational model can simulate today the formation of a horizontally layered epidermis. Since the epidermis is the most complex structured epithelial tissue, systems biology models could yield important insights in epithelial tissue, in which most of all human cancers arise.


RESULTS
We describe the algorithms of a system, capable of simulating the tissue homeostasis in human epidermis leading to a horizontally layered tissue with cells of different differentiation stages. The system predicts epidermal morphology, tissue kinetics and 2D flow of Ca2+ ions. Predicted properties of an epidermis with a healthy and a disturbed barrier are compared with the literature. The system closely mimics the respecting physiological situations.


AVAILABILITY
Additional information and films of the simulation are available at the website. Source code is available on request. http://www.zbh.uni-hamburg.de/research/ESB/index.php


CONTACT
grabe@zbh.uni-hamburg.de">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bae5b7abd4dad0014a2a2f9c8dffb359519c3e95" target='_blank'>A multicellular systems biology model predicts epidermal morphology, kinetics and Ca2+ flow</a></td>
          <td>
            Niels Grabe, Karsten Neuber
          </td>
          <td>2005-09-01</td>
          <td>Bioinformatics</td>
          <td>46</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ac9c640391085d36e30bce8cd2eff1affe6082c" target='_blank'>CDK2 activity crosstalk on the ERK kinase translocation reporter can be resolved computationally.</a></td>
          <td>
            Timothy E. Hoffman, Chengzhe Tian, Varuna Nangia, Chen Yang, S. Regot, Luca Gerosa, Sabrina L Spencer
          </td>
          <td>2025-01-01</td>
          <td>Cell systems</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Psoriasis is an incurable chronic inflammatory skin disease characterised by immune cytokine-stimulated epidermal hyperproliferation. This results in the skin becoming red with scaly plaques that can appear anywhere on the body, decreasing the quality of life for patients. Previous modelling studies of psoriasis have been limited to 2D models and lacked cell-cell interactions. We have developed a 3D agent-based model of epidermal cell dynamics to gain insights into how immune cytokine stimuli induces hyperproliferation in psoriasis to better understand disease formation and structural changes. Three main keratinocytes, stem, transit-amplifying (TA), differentiated and T cells, are modelled with proliferation and division governed by various nutrients and immune cytokines. Each cell has a set of attributes (growth rate, division probability, position, etc) whose values are governed by processes such as monod-based cellular growth model, probability-based division based on calcium and cytokine concentration and various forces to form the epidermal layers. The model has 2 steady states, healthy (non-lesional) and psoriatic (lesional) skin. Transition from healthy to psoriatic state is triggered by a temporary cytokine stimulus which causes hyperproliferation to occur, a hallmark of psoriasis. This results in the deepening of rete ridges and thickening of the epidermal structure. Model outputs has been validated against population ratios of stem, TA, differentiated, and T cells, cell cycle and turnover times in vivo. The model simulates the structural properties of epidermis, including layer stratification, formation of wave-like rete ridges, change in epidermal height and length of rete ridges from healthy to psoriasis. This has provided some insights on the complex spatio-temporal changes when transitioning between the 2 steady states and how a shot of temporary cytokine stimulus can induce different severity of psoriasis and alters proliferation between healthy and psoriatic skin in line with known literature. This provides the basis to study different cytokine simulation variations of psoriasis development and tracking of cell proliferation in the lab. It also provides a baseline to model the effects of psoriasis treatments such as narrowband-ultraviolet B (NB-UVB) or biologics and predict potential treatment outcomes for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e4255f1b3392770de0cb10786397739acde9a05" target='_blank'>Computational Modelling of Immune Interaction and Epidermal Homeostasis in Psoriasis</a></td>
          <td>
            D. Paramalingam, Bowen Li, N. Reynolds, P. Zuliani
          </td>
          <td>2023-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dd8c0e59cc6f86078f2c60ed2d73016f4951d76" target='_blank'>Computational modelling of cancerous mutations in the EGFR/ERK signalling pathway</a></td>
          <td>

          </td>
          <td>2009-10-05</td>
          <td>BMC Systems Biology</td>
          <td>64</td>
          <td>0</td>
        </tr>

        <tr id="An error led to part of Figure 2 being removed, breaking the red line that should have linked the grey circles “PKC” and “EGFR”. The correct version of Figure 2 is now displayed on both the PDF and HTML versions of the article.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/892ad05c7eb0020aec314c7a7afee5e1e1dd1b95" target='_blank'>Correction: Integrative Modelling of the Influence of MAPK Network on Cancer Cell Fate Decision</a></td>
          <td>
            L. Grieco, Laurence Calzone, I. Bernard-Pierrot, F. Radvanyi, B. Kahn-Perlès, D. Thieffry
          </td>
          <td>2013-11-01</td>
          <td>PLoS Computational Biology</td>
          <td>41</td>
          <td>61</td>
        </tr>

        <tr id="Cell responses are actuated by tightly controlled signal transduction pathways. Although the concept of an integrated signaling network replete with interpathway cross-talk and feedback regulation is broadly appreciated, kinetic data of the type needed to characterize such interactions in conjunction with mathematical models are lacking. In mammalian cells, the Ras/ERK pathway controls cell proliferation and other responses stimulated by growth factors, and several cross-talk and feedback mechanisms affecting its activation have been identified. In this work, we take a systematic approach to parse the magnitudes of multiple regulatory mechanisms that attenuate ERK activation through canonical (Ras-dependent) and non-canonical (PI3K-dependent) pathways. In addition to regulation of receptor and ligand levels, we consider three layers of ERK-dependent feedback: desensitization of Ras activation, negative regulation of MEK kinase (e.g. Raf) activities, and up-regulation of dual-specificity ERK phosphatases. Our results establish the second of these as the dominant mode of ERK self-regulation in mouse fibroblasts. We further demonstrate that kinetic models of signaling networks, trained on a sufficient diversity of quantitative data, can be reasonably comprehensive, accurate, and predictive in the dynamical sense.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405be6d692d4836d3b3efde9333fdf1789ff05c7" target='_blank'>Systematic Quantification of Negative Feedback Mechanisms in the Extracellular Signal-regulated Kinase (ERK) Signaling Network*</a></td>
          <td>
            M. Cirit, Chun-Chao Wang, J. Haugh
          </td>
          <td>2010-09-16</td>
          <td>The Journal of Biological Chemistry</td>
          <td>91</td>
          <td>33</td>
        </tr>

        <tr id="To understand how integration of multiple data types can help decipher cellular responses at the systems level, we analyzed the mitogenic response of human mammary epithelial cells to epidermal growth factor (EGF) using whole genome microarrays, mass spectrometry-based proteomics and large-scale western blots with over 1000 antibodies. A time course analysis revealed significant differences in the expression of 3172 genes and 596 proteins, including protein phosphorylation changes measured by western blot. Integration of these disparate data types showed that each contributed qualitatively different components to the observed cell response to EGF and that varying degrees of concordance in gene expression and protein abundance measurements could be linked to specific biological processes. Networks inferred from individual data types were relatively limited, whereas networks derived from the integrated data recapitulated the known major cellular responses to EGF and exhibited more highly connected signaling nodes than networks derived from any individual dataset. While cell cycle regulatory pathways were altered as anticipated, we found the most robust response to mitogenic concentrations of EGF was induction of matrix metalloprotease cascades, highlighting the importance of the EGFR system as a regulator of the extracellular environment. These results demonstrate the value of integrating multiple levels of biological information to more accurately reconstruct networks of cellular response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f4d23a95c29e3c93f9d7bdac57936862d5438a5" target='_blank'>Network Analysis of Epidermal Growth Factor Signaling Using Integrated Genomic, Proteomic and Phosphorylation Data</a></td>
          <td>
            K. Waters, Tao Liu, Ryan D. Quesenberry, A. Willse, Somnath Bandyopadhyay, L. Kathmann, T. Weber, Richard D. Smith, H. Wiley, B. Thrall
          </td>
          <td>2012-03-29</td>
          <td>PLoS ONE</td>
          <td>46</td>
          <td>133</td>
        </tr>

        <tr id="Preface Howard I. Maibach Foreword Bernard Querleux PART 1: Skin color Multilayer Modeling of Skin Color and Translucency Gladimir V. G. Baranoski, Tenn F. Chen, and Aravind Krishnaswamy Dermal component based optical modeling of the skin translucency: impact on the skin color Igor Meglinski, Alexander Doronin, Alexey N. Bashkatov, Elina A. Genina, and Valery V. Tuchin Mathematics and biological process of skin pigmentation Josef Thingnes, Leiv Oyehaug, and Eivind Hovig PART 2: Skin biomechanics State-of-the-art constitutive models of skin biomechanics Georges Limbert Fiber-matrix models of the dermis Cormac Flynn Cellular scale model of the stratum corneum Roberto Santoprete, and Bernard Querleux PART 3: Skin barrier Mathematical models of skin permeability: microscopic transport models and their predictions Gerald B. Kasting, and Johannes M. Nitsche Cellular scale modelling of the skin barrier Dirk Feuchter, Michael Heisig, Arne Naegel, Martin Scherer, and Gabriel Wittum Molecular scale modeling of skin permeation Sophie Martel, and Pierre-Alain Carrupt Accessing the molecular organization of the stratum corneum using high resolution electron microscopy and computer simulation Lars Norlen, Jamshed Anwar, and Ozan Oktem PART 4: Skin fluids and components Water diffusion through stratum corneum Robert E. Imhof, and Peng Xiao Accurate multi-scale skin model suitable for determining the sensitivity and specificity of changes of skin components Jurg Frohlich, Sonja Hulova, Christian Beyer, and Daniel Erni Model based Quantification of Skin Microcirculatory Perfusion Ingemar Fredriksson, Marcus Larsson, and Tomas Stromberg PART 5: Skin homeostasis Graphical Multi-Scale Modeling of Epidermal Homeostasis with EPISIM Thomas Sutterlin, and Niels Grabe Heuristic Modeling Applied to Epidermal Homeostasis Francois Iris, Manuel Gea, Paul-Henri Lampe, and Bernard Querleux">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9209c423174e09c6f3b87fd9998382242a8365d1" target='_blank'>Computational Biophysics of the Skin</a></td>
          <td>
            B. Querleux
          </td>
          <td>2014-07-25</td>
          <td></td>
          <td>27</td>
          <td>21</td>
        </tr>

        <tr id="The phosphorylation of cellular proteins plays a crucial role in the transduction of various signals from outside the cell into the nucleus. The signals are transduced by phosphorylation chain reactions within multiple pathways; however, determining which pathways are responsible for each defined signal has proven challenging. To estimate the activity of each pathway, we developed a phosphorylation array platform comprising a protein array with 1200 proteins belonging to 376 signalling pathways and an analytical method to estimate pathway activity based on the phosphorylation levels of proteins. The performance of our system was assessed by reconstructing kinase–substrate relationships, as well as by estimating pathway activity upon epidermal growth factor (EGF) stimulation and the pharmacological inhibition of epidermal growth factor receptor (EGFR). As a result, kinase–substrate relationships were reliably reconstructed based on the precise measurement of phosphorylation levels of constituent proteins on the array. Furthermore, the pathway activities associated with EGF stimulation and EGFR inhibition were successfully traced through the related pathways from the outer membrane to the nucleus along a time course. Thus, our phosphorylation array system can effectively assess the activity of specific signalling pathways that are perturbed by extracellular stimuli, such as various drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f9c27ddb3e4d2f7897fe0cb3dc1a2581cdd3b73" target='_blank'>Assessing the activation/inhibition of tyrosine kinase‐related pathways with a newly developed platform</a></td>
          <td>
            Harumi Kagiwada, Takayuki Kiboku, Hitomi Matsuo, M. Kitazawa, K. Fukui, K. Horimoto
          </td>
          <td>2021-06-21</td>
          <td>PROTEOMICS</td>
          <td>5</td>
          <td>47</td>
        </tr>

        <tr id="Extra-cellular signal regulated kinase (ERK) signal network plays an important role in regulating the fundamental cellular functions such as cell proliferation, survival, differentiation and motility. Development and analysis of mathematical model can help us acquire a deep understanding of the complex behavior of ERK signal transduction network. This paper presents a computational model that offers an integrated quantitative and dynamic simulation of ERK signal transduction network, activated by epidermal growth factor (EGF). The mathematic model contains the activation kinetics of the pathway, a multitude of feedback loops and connection of scaffold proteins. The model provides insight into signal-response relationships between the binding of EGF to its receptor at the cell surface and activation of downstream proteins in the signaling cascade. The different influence of positive and negative feedback loops of the ERK signal transduction pathway were mainly analyzed, indicating that feedback loops were the main impacting factor to the oscillation of ERK signal transduction pathway. The predictions of this oscillation of ERK activation agree well with the literature. It can lead to signal waves of the downstream substrates and induce corresponding biological behaviors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7bf832c4d17a797f80bffbcb87cce1c1078a91f" target='_blank'>Dynamic Analysis of Feedback Loops in Extra-Cellular Signal Regulated Kinase Signal Network</a></td>
          <td>
            Yong Han, Yan Li, Yonghua Wang, H. Fang, Shuwei Zhang
          </td>
          <td>2009-03-31</td>
          <td>2009 WRI World Congress on Computer Science and Information Engineering</td>
          <td>3</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/509f1971d67425c00eb55f435cf67274acb0fe25" target='_blank'>Computational modeling of the dynamics of the MAP kinase cascade activated by surface and internalized EGF receptors</a></td>
          <td>
            B. Schoeberl, Claudia Eichler-Jonsson, E. Gilles, G. Müller
          </td>
          <td>2002-04-01</td>
          <td>Nature Biotechnology</td>
          <td>1027</td>
          <td>48</td>
        </tr>

        <tr id="Signaling network responses can be highly heterogeneous across cells in a tissue because of many sources of genetic and non-genetic variance. The emergence of multiplexed single-cell technologies has made it possible to evaluate this heterogeneity. In this review, we categorize currently established single-cell signaling network profiling approaches by their methodology, coverage, and application, and we discuss the advantages and limitations of these technologies. We describe the computational tools for network characterization using single-cell data and discuss potential confounding factors that may affect single-cell analyses. Graphical Abstract Highlights Signaling networks can be highly heterogeneous across cells in a tissue. Various technologies allow analyzing signaling networks at single-cell resolution. The advantages and limitations of each single-cell approach are summarized. Confounding factors in single-cell signaling network analysis are discussed. Signaling networks process intra- and extracellular information to modulate the functions of a cell. Deregulation of signaling networks results in abnormal cellular physiological states and often drives diseases. Network responses to a stimulus or a drug treatment can be highly heterogeneous across cells in a tissue because of many sources of cellular genetic and non-genetic variance. Signaling network heterogeneity is the key to many biological processes, such as cell differentiation and drug resistance. Only recently, the emergence of multiplexed single-cell measurement technologies has made it possible to evaluate this heterogeneity. In this review, we categorize currently established single-cell signaling network profiling approaches by their methodology, coverage, and application, and we discuss the advantages and limitations of each type of technology. We also describe the available computational tools for network characterization using single-cell data and discuss potential confounding factors that need to be considered in single-cell signaling network analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60a57678d8c2ca100491abb88d9a332d1b9e22a7" target='_blank'>Profiling Cell Signaling Networks at Single-cell Resolution</a></td>
          <td>
            Xiao-Kang Lun, B. Bodenmiller
          </td>
          <td>2020-03-04</td>
          <td>Molecular & Cellular Proteomics : MCP</td>
          <td>27</td>
          <td>53</td>
        </tr>

        <tr id="Psoriasis is the most common and chronic skin disease that affects individuals from every age group. The rate of psoriasis is increasing over the time in both developed and developing countries. Studies have revealed the possibility of association of psoriasis with skin cancers, particularly non-melanoma skin cancers (NMSC), which, include basal cell carcinoma and cutaneous squamous cell carcinoma (cSCC). There is a need to analyze the disease at molecular level to propose potential biomarkers and therapeutic targets in comparison to cSCC. Therefore, the second analyzed disease of this study is cSCC. It is the second most common prevalent skin cancer all over the world with the potential to metastasize and recur. There is an urge to validate the proposed biomarkers and discover new potential biomarkers as well. In order to achieve the goals and objectives of the study, microarray and RNA-sequencing data analyses were performed followed by network analysis. Afterwards, quantitative systems biology was implemented to analyze the results at a holistic level. The aim was to predict the molecular patterns that can lead psoriasis to cancer. The current study proposed potential biomarkers and therapeutic targets for psoriasis and cSCC. IL-17 signaling pathway is also identified as significant pathway in both diseases. Moreover, the current study proposed that autoimmune pathology, neutrophil recruitment, and immunity to extracellular pathogens are sensitive towards MAPKs (MAPK13 and MAPK14) and genes for AP-1 (FOSL1 and FOS). Therefore, these genes should be further studied in gene knock down based studies as they may play significant role in leading psoriasis towards cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51951f9a64263bc92df7d5fd48350b3a60d17ae7" target='_blank'>A Computational Systems Analyses to Identify Biomarkers and Mechanistic Link in Psoriasis and Cutaneous Squamous Cell Carcinoma</a></td>
          <td>
            Sidra Adil, R. Z. Paracha, Salma Tariq, Maryum Nisar, S. Ijaz, Amnah Siddiqa, Z. Hussain, A. Amir
          </td>
          <td>2021-06-18</td>
          <td>Frontiers in Immunology</td>
          <td>7</td>
          <td>13</td>
        </tr>

        <tr id="The integrity of the reconstructed human epidermis generated in vitro can be assessed using histological analyses combined with immunohistochemical staining of keratinocyte differentiation markers. Technical differences during the preparation and capture of stained images may influence the outcome of computational methods. Due to the specific nature of the analyzed material, no annotated datasets or dedicated methods are publicly available. Using a dataset with 598 unannotated images showing cross-sections of in vitro reconstructed human epidermis stained with DAB-based immunohistochemistry reaction to visualize four different keratinocyte differentiation marker proteins (filaggrin, keratin 10, Ki67, HSPA2) and counterstained with hematoxylin, we developed an unsupervised method for the detection and quantification of immunohistochemical staining. The pipeline consists of the following steps: (i) color normalization; (ii) color deconvolution; (iii) morphological operations; (iv) automatic image rotation; and (v) clustering. The most effective combination of methods includes (i) Reinhard’s normalization; (ii) Ruifrok and Johnston color-deconvolution method; (iii) proposed image-rotation method based on boundary distribution of image intensity; and (iv) k-means clustering. The results of the work should enhance the performance of quantitative analyses of protein markers in reconstructed human epidermis samples and enable the comparison of their spatial distribution between different experimental conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/874dfbf781c5c9e21f63defe344910e7cf3ec374" target='_blank'>EpidermaQuant: Unsupervised Detection and Quantification of Epidermal Differentiation Markers on H-DAB-Stained Images of Reconstructed Human Epidermis</a></td>
          <td>
            Dawid Zamojski, Agnieszka Gogler, Dorota Scieglinska, Michał Marczyk
          </td>
          <td>2024-06-05</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="For more than a decade, the MAPK (mitogen-activated protein kinase) cascade has been studied using mathematical modelling and quantitative experimentation [1]. The MAPK cascade relays the presence of extracellular stimuli such as growth hormones to the nucleus and controls the expression of hundreds of genes. MAPKs control major cell fate decisions such as proliferation, differentiation and apoptosis, mainly by inducing alterations in gene expression. In this chapter, we discuss how systems biology analysis provides insights into the functioning of this cascade. We show how this pathway assists the cell in responding properly to extracellular cues by filtering out sub-threshold stimuli, while efficiently transmitting physiologically relevant inputs. Several different receptors signal through the MAPK pathway even though they elicit opposite biological responses, thus raising the question of how specificity is achieved in MAPK signalling. Experimental studies revealed that specific biological responses are encoded by quantitative aspects of the MAPK signal such as amplitude or duration. We discuss mechanisms that enable the pathway to generate quantitatively different signals, and also explain how different signals are interpreted by the downstream gene expression machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6240690e740bb11ac1d42f43f91ffcfad059dbaf" target='_blank'>Systems analysis of MAPK signal transduction.</a></td>
          <td>
            N. Blüthgen, S. Legewie
          </td>
          <td>2008-09-30</td>
          <td>Essays in biochemistry</td>
          <td>68</td>
          <td>42</td>
        </tr>

        <tr id="In vivo and in vitro studies give a paradoxical picture of the actions of the key regulatory factor TGF-β1 in epidermal wound healing with it stimulating migration of keratinocytes but also inhibiting their proliferation. To try to reconcile these into an easily visualized 3D model of wound healing amenable for experimentation by cell biologists, a multiscale model of the formation of a 3D skin epithelium was established with TGF-β1 literature–derived rule sets and equations embedded within it. At the cellular level, an agent-based bottom-up model that focuses on individual interacting units (keratinocytes) was used. This was based on literature-derived rules governing keratinocyte behavior and keratinocyte/ECM interactions. The selection of these rule sets is described in detail in this paper. The agent-based model was then linked with a subcellular model of TGF-β1 production and its action on keratinocytes simulated with a complex pathway simulator. This multiscale model can be run at a cellular level only or at a combined cellular/subcellular level. It was then initially challenged (by wounding) to investigate the behavior of keratinocytes in wound healing at the cellular level. To investigate the possible actions of TGF-β1, several hypotheses were then explored by deliberately manipulating some of these rule sets at subcellular levels. This exercise readily eliminated some hypotheses and identified a sequence of spatial-temporal actions of TGF-β1 for normal successful wound healing in an easy-to-follow 3D model. We suggest this multiscale model offers a valuable, easy-to-visualize aid to our understanding of the actions of this key regulator in wound healing, and provides a model that can now be used to explore pathologies of wound healing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fb98ebdf840f7b0da6f61004f31cec8ba6cc1a3" target='_blank'>Exploring Hypotheses of the Actions of TGF-β1 in Epidermal Wound Healing Using a 3D Computational Multiscale Model of the Human Epidermis</a></td>
          <td>
            T. Sun, S. Adra, R. Smallwood, M. Holcombe, S. MacNeil
          </td>
          <td>2009-12-31</td>
          <td>PLoS ONE</td>
          <td>70</td>
          <td>60</td>
        </tr>

        <tr id="MOTIVATION
For the efficient application of multi-agent systems to spatial and functional modeling of tissues flexible and intuitive modeling tools are needed, which allow the graphical specification of cellular behavior in a tissue context without presuming specialized programming skills.


RESULTS
We developed a graphical modeling system for multi-agent based simulation of tissue homeostasis. An editor allows the intuitive and hierarchically structured specification of cellular behavior. The models are then automatically compiled into highly efficient source code and dynamically linked to an interactive graphical simulation environment. The system allows the quantitative analysis of the morphological and functional tissue properties emerging from the cell behavioral model. We demonstrate the relevance of the approach using a recently published model of epidermal homeostasis as well as a series of cell-cycle models.


AVAILABILITY
The complete software is available in binary executables for MS-Windows and Linux at tiga.uni-hd.de.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06beec78b7bad308d9f21aa35465aea3444cf6be" target='_blank'>Modeling multi-cellular behavior in epidermal tissue homeostasis via finite state machines in multi-agent systems</a></td>
          <td>
            Thomas Sütterlin, S. Huber, H. Dickhaus, N. Grabe
          </td>
          <td>2009-08-01</td>
          <td>Bioinformatics</td>
          <td>35</td>
          <td>36</td>
        </tr>

        <tr id="Motivation Signal transduction networks regulate a multitude of essential biological processes and are frequently aberrated in diseases such as cancer. Developing a mechanistic understanding of such networks is essential to understand disease or cell population specific signaling and to design effective treatment strategies. Typically, such networks are computationally reconstructed based on systematic perturbation experiments, followed by quantification of signaling protein activity. Recent technological advances now allow for the quantification of the activity of many (signaling) proteins simultaneously in single cells. This makes it feasible to reconstruct signaling networks from single cell data. Results Here we introduce single cell Comparative Network Reconstruction (scCNR) to derive signal transduction networks by exploiting the heterogeneity of single cell (phospho)protein measurements. scCNR treats stochastic variation in total protein abundances as natural perturbation experiments, whose effects propagate through the network. scCNR reconstructs cell population specific networks of the same underlying topology for cells from diverse populations. We extensively validated scCNR on simulated single cell data, and we applied it to a dataset of EGFR-inhibitor treated keratinocytes to recover signaling differences downstream of EGFR and in protein interactions associated with proliferation. scCNR will help to unravel the mechanistic signaling differences between cell populations by making use of single-cell data, and will subsequently guide the development of well-informed treatment strategies. Availability and implementation scCNR is available as a python module at https://github.com/ibivu/scmra. Additionally, code to reproduce all figures is available at https://github.com/tstohn/scmra_analysis. Supplementary information Supplementary information and data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/516c13660aa9eadff6cefbce4ac0292281fdc217" target='_blank'>Reconstructing and comparing signal transduction networks from single cell protein quantification data</a></td>
          <td>
            Tim Stohn, Roderick van Eijl, Klaas W. Mulder, L. Wessels, Evert Bosdriesz
          </td>
          <td>2024-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Signal transduction is a complex protein signaling process with a rich network of multifunctional interactions that occur in a non‐linear fashion. Mitogen‐activated protein kinase (MAPK) signal transduction pathways regulate diverse cellular processes ranging from proliferation and differentiation to apoptosis. In mammals, out of five, there are three well characterized subfamilies of MAPKs ‐ ERKs (Extracellular signal‐regulated kinases), JNKs (c‐Jun N‐terminal kinases), and P38 kinases, and their activators, are implicated in human diseases and are targets for drug development. Kinase cascades in MAPK pathways mediate the sensing and processing of stimuli. To understand how cells makes decisions, the dynamic interactions of components of signaling cascades are important rather than just creating static maps. Based on enzyme kinetic reactions, we have developed a mathematical model to analyze the impact of the cross‐talks between JNK and P38 kinase cascades. Cross‐talks between JNK and P38 kinase cascades influence the activities of P38 kinases. Responses of the signals should be studied for network of kinase cascades by considering cross‐talks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef12bcae9ca82339a744c3b732a8bd77d348ac39" target='_blank'>Computational prediction and analysis of impact of the cross‐talks between JNK and P38 kinase cascades</a></td>
          <td>
            Pandurangan Sundaramurthy, S. Gakkhar, R. Sowdhamini
          </td>
          <td>2009-01-12</td>
          <td>Bioinformation</td>
          <td>12</td>
          <td>32</td>
        </tr>

        <tr id="Immunofluorescence (IF) and in situ proximity ligation assay (isPLA) are techniques that are used for in situ protein expression and colocalisation analysis, respectively. However, an efficient quantitative method to analyse both IF and isPLA staining on skin sections is lacking. Therefore, we developed a new method for semi‐automatic quantitative layer‐by‐layer measurement of protein expression and colocalisation in skin sections using the free open‐source software CellProfiler. As a proof of principle, IF and isPLA of ichthyosis‐related proteins TGm‐1 and SDR9C7 were examined. The results indicate that this new method can be used for protein expression and colocalisation analysis in skin sections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e4c50ca3f38be221c43505d198e1b08f9149cdc" target='_blank'>Quantitative image analysis of protein expression and colocalisation in skin sections</a></td>
          <td>
            Hanqian Zhang, Maja Ericsson, M. Virtanen, S. Weström, Carolina Wählby, A. Vahlquist, H. Törmä
          </td>
          <td>2018-01-09</td>
          <td>Experimental Dermatology</td>
          <td>8</td>
          <td>45</td>
        </tr>

        <tr id="The site-specific function of EGFR’s C-terminal tyrosines is interrogated by phosphotyrosine phosphoproteomics to build a data-driven model that links cell response to signaling nodes and elucidates novel associations in the EGFR signaling network. The epidermal growth factor receptor (EGFR) has been studied extensively because of its critical role in cellular signaling and association with disease. Previous models have elucidated interactions between EGFR and downstream adaptor proteins or showed phenotypes affected by EGFR. However, the link between specific EGFR phosphorylation sites and phenotypic outcomes is still poorly understood. Here, we employed a suite of isogenic cell lines expressing site-specific mutations at each of the EGFR C-terminal phosphorylation sites to interrogate their role in the signaling network and cell biological response to stimulation. Our results demonstrate the resilience of the EGFR network, which was largely similar even in the context of multiple Y-to-F mutations in the EGFR C-terminal tail, while also revealing nodes in the network that have not previously been linked to EGFR signaling. Our data-driven model highlights the signaling network nodes associated with distinct EGF-driven cell responses, including migration, proliferation, and receptor trafficking. Application of this same approach to less-studied RTKs should provide a plethora of novel associations that should lead to an improved understanding of these signaling networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d1bedc5d16450a77fda0fc2c816d3cd1cf3570c" target='_blank'>Predictive data-driven modeling of C-terminal tyrosine function in the EGFR signaling network</a></td>
          <td>
            Jacqueline S. Gerritsen, Joseph S Faraguna, R. Bonavia, F. Furnari, F. White
          </td>
          <td>2023-05-11</td>
          <td>Life Science Alliance</td>
          <td>2</td>
          <td>66</td>
        </tr>

        <tr id="The biophysical properties of ligand binding heavily influence the ability of receptors to specify cell fates. Understanding the rules by which ligand binding kinetics impact cell phenotype is challenging, however, because of the coupled information transfers that occur from receptors to downstream signaling effectors and from effectors to phenotypes. Here, we address that issue by developing an integrated mechanistic and data-driven computational modeling platform to predict cell responses to different ligands for the epidermal growth factor receptor (EGFR). Experimental data for model training and validation were generated using MCF7 human breast cancer cells treated with the high- and low-affinity ligands epidermal growth factor (EGF) and epiregulin (EREG), respectively. The integrated model captures the unintuitive, concentration-dependent abilities of EGF and EREG to drive signals and phenotypes differently, even at similar levels of receptor occupancy. For example, the model correctly predicts the dominance of EREG over EGF in driving a cell differentiation phenotype through AKT signaling at intermediate and saturating ligand concentrations and the ability of EGF and EREG to drive a broadly concentration-sensitive migration phenotype through cooperative ERK and AKT signaling. Parameter sensitivity analysis identifies EGFR endocytosis, which is differentially regulated by EGF and EREG, as one of the most important determinants of the alternative phenotypes driven by different ligands. The integrated model provides a new platform to predict how phenotypes are controlled by the earliest biophysical rate processes in signal transduction and may eventually be leveraged to understand receptor signaling system performance depends on cell context. One-sentence summary Integrated kinetic and data-driven EGFR signaling model identifies the specific signaling mechanisms that dictate cell responses to EGFR activation by different ligands.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccddd2d0cde0d83a0a8cfb56bef3dd0624d5226" target='_blank'>An integrated mechanistic and data-driven computational model predicts cell responses to high- and low-affinity EGFR ligands</a></td>
          <td>
            Paul J. Myers, Sung Hyun Lee, M. Lazzara
          </td>
          <td>2023-06-26</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>22</td>
        </tr>

        <tr id="Abstract. Significance: Reflectance confocal microscopy (RCM) is a noninvasive, in vivo technology that offers near histopathological resolution at the cellular level. It is useful in the study of phenomena for which obtaining a biopsy is impractical or would cause unnecessary tissue damage and trauma to the patient. Aim: This review covers the use of RCM in the study of skin and the use of machine learning to automate information extraction. It has two goals: (1) an overview of information provided by RCM on skin structure and how it changes over time in response to stimuli and in disease and (2) an overview of machine learning approaches developed to automate the extraction of key morphological features from RCM images. Approach: A PubMed search was conducted with additional literature obtained from references lists. Results: The application of RCM as an in vivo tool in dermatological research and the biologically relevant information derived from it are presented. Algorithms for image classification to epidermal layers, delineation of the dermal–epidermal junction, classification of skin lesions, and demarcation of individual cells within an image, all important factors in the makeup of the skin barrier, were reviewed. Application of image analysis methods in RCM is hindered by low image quality due to noise and/or poor contrast. Use of supervised machine learning is limited by time-consuming manual labeling of RCM images. Conclusions: RCM has great potential in the study of skin structures. The use of artificial intelligence could enable an easier, more reproducible, precise, and rigorous study of RCM images for the understanding of skin structures, skin barrier, and skin inflammation and lesions. Although several attempts have been made, further work is still needed to provide a definite gold standard and overcome issues related to image quality, limited labeled datasets, and lack of phenotype variability in available databases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/396c8fc021018b1827e39b12b7c621312411048a" target='_blank'>Automating reflectance confocal microscopy image analysis for dermatological research: a review</a></td>
          <td>
            I. Lboukili, G. Stamatas, X. Descombes
          </td>
          <td>2022-07-01</td>
          <td>Journal of Biomedical Optics</td>
          <td>4</td>
          <td>35</td>
        </tr>

        <tr id="The transforming growth factor-β (TGF-β) signal transduction pathway controls many cellular processes, including differentiation, proliferation and apoptosis. It plays a fundamental role during development and it is dysregulated in many diseases. The factors that control the dynamics of the pathway, however, are not fully elucidated yet and so far computational approaches have been very limited in capturing the distinct types of behaviour observed under different cellular backgrounds and conditions into a single-model description. Here, we develop a detailed computational model for TGF-β signalling that incorporates elements of previous models together with crosstalking between Smad1/5/8 and Smad2/3 channels through a negative feedback loop dependent on Smad7. The resulting model accurately reproduces the diverse behaviour of experimental datasets for human keratinocytes, bovine aortic endothelial cells and mouse mesenchymal cells, capturing the dynamics of activation and nucleocytoplasmic shuttling of both R-Smad channels. The analysis of the model dynamics and its system properties revealed Smad7-mediated crosstalking between Smad1/5/8 and Smad2/3 channels as a major determinant in shaping the distinct responses to single and multiple ligand stimulation for different cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9e9f9f231660dad9ab1748d623f51d45358b5c5" target='_blank'>Computational modelling of Smad-mediated negative feedback and crosstalk in the TGF-β superfamily network</a></td>
          <td>
            D. Nicklas, L. Saiz
          </td>
          <td>2013-09-06</td>
          <td>Journal of The Royal Society Interface</td>
          <td>29</td>
          <td>24</td>
        </tr>

        <tr id="In vivo and in vitro studies give a paradoxical picture of the actions of the key regulatory factor TGF-β1 in epidermal wound healing with it stimulating migration of keratinocytes but also inhibiting their proliferation. To try to reconcile these into an easily visualized 3D model of wound healing amenable for experimentation by cell biologists, a multiscale model of the formation of a 3D skin epithelium was established with TGF-β1 literature–derived rule sets and equations embedded within it. At the cellular level, an agent-based bottom-up model that focuses on individual interacting units (keratinocytes) was used. This was based on literature-derived rules governing keratinocyte behavior and keratinocyte/ECM interactions. The selection of these rule sets is described in detail in this paper. The agent-based model was then linked with a subcellular model of TGF-β1 production and its action on keratinocytes simulated with a complex pathway simulator. This multiscale model can be run at a cellular level only or at a combined cellular/subcellular level. It was then initially challenged (by wounding) to investigate the behavior of keratinocytes in wound healing at the cellular level. To investigate the possible actions of TGF-β1, several hypotheses were then explored by deliberately manipulating some of these rule sets at subcellular levels. This exercise readily eliminated some hypotheses and identified a sequence of spatial-temporal actions of TGF-β1 for normal successful wound healing in an easy-to-follow 3D model. We suggest this multiscale model offers a valuable, easy-to-visualize aid to our understanding of the actions of this key regulator in wound healing, and provides a model that can now be used to explore pathologies of wound healing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fb98ebdf840f7b0da6f61004f31cec8ba6cc1a3" target='_blank'>Exploring Hypotheses of the Actions of TGF-β1 in Epidermal Wound Healing Using a 3D Computational Multiscale Model of the Human Epidermis</a></td>
          <td>
            T. Sun, S. Adra, R. Smallwood, M. Holcombe, S. MacNeil
          </td>
          <td>2009-12-31</td>
          <td>PLoS ONE</td>
          <td>70</td>
          <td>60</td>
        </tr>

        <tr id="Abstract Computer simulations and mathematical modeling of biological processes are becoming increasingly popular, and yet the complexity of the biochemical systems or the differences between experimental setups make it very difficult to establish a standard formula for these modeling projects. Before we can start using computer‐based models for predictions or targeted experiment designs, it is very important to establish a reliable model on which those predictions can be based and experimentally tested. Here we attempt to present a computer model for the mitogen‐activated protein kinase (MAPK) signaling cascade which is consistent with previously published experimental results. In this study we have focused our attention to a generic MAPK ERK (extracellular signal‐regulated kinase) pathway activated by epidermal growth factor (EGF) in an attempt to understand how receptors may achieve different activation kinetics of the MAPK signaling. We successfully show that the level of receptor expression is one key determinant in this regulation, and that the binding affinity of the active receptor to adaptor proteins can have a small but albeit direct effect on the downstream activation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dde366641aca5125603232bb38f2d61b4eda2a5" target='_blank'>A Computer‐Based Model for the Regulation of Mitogen Activated Protein Kinase (MAPK) Activation</a></td>
          <td>
            I. Aksan, M. Levent Kurnaz
          </td>
          <td>2003-01-01</td>
          <td>Journal of Receptors and Signal Transduction</td>
          <td>26</td>
          <td>4</td>
        </tr>

        <tr id="The epidermal growth factor receptor (EGFR) signalling pathway is a complex signalling process with a wide network of interactions. The activation of the mitogen-activated protein kinases (MAPKs) cascade by this activated EGFR has been well studied. MAPKs form a highly integrated network, which is essential for certain specialised cell functions. This paper presents a kinetic model for the MAPK pathway downstream of the EGFR using a biochemical simulator. The model includes 30 signalling events and 29 signalling molecules. The time course data were examined for the activation of each signalling component. The simulation provides a large volume of data, by monitoring the kinetics of the signalling components, which were compared experimentally using the PC12 cell line. The kinetic model corresponded well with the experimental results observed in the EGFR induced activation of proteins. An examination of the kinetic analysis of the multiple signalling events provides a quantitative framework for representing the EGFR signalling network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ab4070943b1039a9e896666b7a81fc1863c7dc4" target='_blank'>Simulation and sensitivity analysis of phosphorylation of EGFR signal transduction pathway in PC12 cell model.</a></td>
          <td>
            C. V. S. Babu, S. Yoon, H. Nam, Young Sook Yoo
          </td>
          <td>2004-12-01</td>
          <td>Systems biology</td>
          <td>20</td>
          <td>5</td>
        </tr>

        <tr id="Signal network assumes a vital part in directing the principal cell capacities, for example, cell expansion, survival, separation and motility. Improvement and investigation of scientific model can help us gain a profound comprehension of the unpredictable conduct of ERK flag transduction organizes. This paper exhibits a computational model that offers an incorporated quantitative and dynamic reproduction of ERK flag transduction arranges, actuated by epidermal development figure. The mathematic demonstrate contains the enactment energy of the pathway, a huge number of input circles and association of platform proteins. The model gives knowledge into flag reaction connections between the authoritative of EGF to its receptor at the phone surface and actuation of downstream proteins in the flagging course. The diverse impact of positive and negative input circles of the ERK flag transduction pathway were for the most part examined, showing that criticism circles were the primary affecting variable to the swaying of ERK flag transduction pathway. The forecasts of this wavering of ERK enactment concur well with the writing. It can prompt flag floods of the downstream substrates and instigate relating natural practices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b0c50699f15bb521375edbc6061e541e8ccaea4" target='_blank'>Analysis of Regulated Kinase Signal Network through Feedback Loops in Extra-Cellular Signal</a></td>
          <td>
            C. Periyasamy Periyasamy
          </td>
          <td>2017-11-01</td>
          <td>Indonesian Journal of Electrical Engineering and Computer Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The objective of this paper is to provide a review on some aspects of the mathematical and computational modelling of skin biophysics, with special focus on constitutive theories based on nonlinear continuum mechanics from elasticity, through anelasticity, including growth, to thermoelasticity. Microstructural and phenomenological approaches combining imaging techniques are also discussed. Finally, recent research applications on skin wrinkles will be presented to highlight the potential of physics-based modelling of skin in tackling global challenges such as ageing of the population and the associated skin degradation, diseases and traumas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bace6d856b266f606c29e3f7e79efeca473bb26" target='_blank'>Mathematical and computational modelling of skin biophysics: a review</a></td>
          <td>
            G. Limbert
          </td>
          <td>2017-07-01</td>
          <td>Proceedings of the Royal Society A: Mathematical, Physical and Engineering Sciences</td>
          <td>109</td>
          <td>22</td>
        </tr>

        <tr id="Cellular information processing in signaling networks forms the basis of responses to environmental stimuli. At any given time, cells receive multiple simultaneous input cues, which are processed and integrated to determine cellular responses such as migration, proliferation, apoptosis, or differentiation. Protein phosphorylation events play a major role in this process and are often involved in fundamental biological and cellular processes such as protein‐protein interactions, enzyme activity, and immune responses. Determining which kinases phosphorylate specific phospho sites poses a challenge; this information is critical when trying to elucidate key proteins involved in specific cellular responses. Here, methods to generate high‐quality quantitative phosphorylation data from cell lysates originating from primary cells, and how to analyze the generated data to construct quantitative signaling network models, are presented. These models can subsequently be used to guide follow‐up in vitro/in vivo validation studies. Curr. Protoc. Immunol. 104:11.11.1‐11.11.23. © 2014 by John Wiley & Sons, Inc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a0c788b0085bd2d7a1596f03e8f17950e8a0d6e" target='_blank'>Experimental and Computational Tools for Analysis of Signaling Networks in Primary Cells</a></td>
          <td>
            E. Schoof, R. Linding
          </td>
          <td>2014-02-01</td>
          <td>Current Protocols in Immunology</td>
          <td>2</td>
          <td>40</td>
        </tr>

        <tr id="In this study, we aimed to develop a novel method for non-invasively determining intracellular protein levels, which is essential for understanding cellular phenomena. This understanding hinges on insights into gene expression, cell morphology, dynamics, and intercellular interactions. Traditional cell analysis techniques, such as immunostaining, live imaging, next-generation sequencing, and single-cell analysis, despite rapid advancements, face challenges in comprehensively integrating gene and protein expression data with spatiotemporal information. Leveraging advances in machine learning for image analysis, we designed a new model to estimate cellular biomarker protein levels using a blend of phase-contrast and fluorescent immunostaining images of epidermal keratinocytes. By iterating this process across various proteins, our model can estimate multiple protein levels from a single phase-contrast image. Additionally, we developed a system for analyzing multiple protein expression levels alongside spatiotemporal data through live imaging and phase-contrast methods. Our study offers valuable tools for cell-based research and presents a new avenue for addressing molecular biological challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aa9254d0ba8475aa1788186f38552bbe8ff94b1" target='_blank'>Machine Learning-Enhanced Estimation of Cellular Protein Levels from Bright-Field Images</a></td>
          <td>
            T. Tohgasaki, Arisa Touyama, Shohei Kousai, Kaita Imai
          </td>
          <td>2024-07-31</td>
          <td>Bioengineering</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55d7c99e47cbd5fed329f3182a9921e9443d5c69" target='_blank'>Integrated Platform for Monitoring Single-cell MAPK Kinetics in Computer-controlled Temporal Stimulations</a></td>
          <td>
            H. Ryu, M. Chung, Jiyoung Song, Sung Sik Lee, O. Pertz, N. Jeon
          </td>
          <td>2018-07-24</td>
          <td>Scientific Reports</td>
          <td>10</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832b8a763bca673f9ef9a922d08084ef49416b11" target='_blank'>Systems Analysis of EGF Receptor Signaling Dynamics with Micro-Western Arrays</a></td>
          <td>
            Mark F. Ciaccio, Joel Wagner, Chih-Pin Chuu, D. Lauffenburger, R. Jones
          </td>
          <td>2010-01-24</td>
          <td>Nature methods</td>
          <td>209</td>
          <td>119</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>