<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              42b73dd553ae42926deeaa8377af81aa8ce9864c
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-20 11:50:03 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Quantitative systems pharmacology model‐based investigation of adverse gastrointestinal events associated with prolonged treatment with PI3‐kinase inhibitors</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="Inflammatory bowel disorders are non-infectious chronic inflammation of the esophageal tract (IBD). IBD is divided into two types: ulcerative colitis (UC) and Crohn's disease. The highest incidence and prevalence rates of ulcerative colitis are seen in North America and northern Europe, with incidence rates ranging from 9 to 20 cases per 100 000 person-years and prevalence rates ranging from 156 to 291 cases per 100 000 persons. OZM is contraindicated in patients with previous history of cardiovascular complications. The dose and dosage of OZM is 0.23 mg by oral route (qDay) for 1 to 4 days followed by 5 to 7:0.46 mg and on eighth day titrated to 0.92 mg PO (qDay). OZM is not highly recommended in pregnant and lactating women since existing animal studies not reported with enough evidence. The in silico analysis of Ozanimod (OZM) is not yet been fully characterized. The current study was designed to investigate the pharmacokinetic profile of the OZM and its interaction potential with UC target protein.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8146d60559fce654360081441e7243fe352e72cd" target='_blank'>Network pharmacology and in silico pharmacokinetic prediction of Ozanimod in the management of ulcerative colitis: A computational study</a></td>
          <td>
            Priya Sivasakthi, Sarvesh Sabarathinam, T. Vijayakumar
          </td>
          <td>2022-01-01</td>
          <td>Health Science Reports</td>
          <td>5</td>
          <td>9</td>
        </tr>

        <tr id="


 Clinical evidence supports the combination therapy of vedolizumab (a gut-selective, anti-lymphocyte trafficking drug inhibiting α4β7-integrin) with JAK 1, 2, 3 inhibitors (JAKi) for Inflammatory Bowel Disease. However, the mechanistic explanation for the improved outcomes in Ulcerative Colitis patients treated with vedolizumab plus JAKi remains unknown and even more in Crohn's Disease (CD). In this study we used an unbiased in silico systems biology and artificial intelligence approach to gain a holistic view of the mechanisms that may underpin the potential benefits of the mentioned combination therapy in the treatment of CD.



 The drugs under study -vedolizumab and JAKi- and CD were characterized at protein level by compiling data from extensive literature search. Through it, human protein interaction networks focused on CD protein effectors were generated and used to construct mathematical models using Therapeutic Performance Mapping System (TPMS) technology (Anaxomics Biotech, Barcelona, Spain). Sampling-based methods were employed to assess the impact on CD of the combined therapy vedolizumab plus JAKi and to describe underlying molecular mechanisms.



 Characterization of CD allowed identifying 4 pathological processes (referred to as motives-M) underlying the manifestation of the disease: intestinal barrier disruption (M1), increased innate immune response (M2), chronic inflammation and Th1/Th17 adaptive immune response (M3) and tissue remodeling (M4) and the molecular effectors or proteins involved in each motive. The combination of vedolizumab and JAKi induced a high degree of reversion of the effectors altered in CD (54.76%), higher than the percentage reverted by the individual drugs. Combined vedolizumab plus JAKi therapy mainly modulated M3, where both drugs mostly converge although by distinct mechanisms, thus providing additional individual benefits. The combination therapy reverted a great percentage of M3 effectors, for example, CCR9 and FASLG which were increased in CD, were reverted (or downregulated). In addition to M3 modulation, the combination therapy also modulated M1 and M4. In M1, vedolizumab is the one that would provide more complementary mechanisms apart from those that already converge in both drugs. Regarding M4, it seems that main contribution comes from complementarity pathways of the different mechanisms of action of each individual drug.



 The constructed in silico models reveal that the combined vedolizumab plus JAKi therapy could modulate a high array of effectors altered in CD, more than any of the therapies alone, by enhancing the effects of the individual drugs, and provide a mechanistic rationale for employing the combination of these drugs as potential therapy for CD.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37afddf4916b5088749544ad27899c601491b710" target='_blank'>P969 Unveiling the molecular mechanisms of the combination of vedolizumab with JAK inhibitors in Crohn’s Disease through a systems biology and artificial intelligence-based approach</a></td>
          <td>
            I. Marín-Jiménez, M. Sierra-Ausín, T. Letosa-Abián, J. Aparicio, C. Montoto-Otero, S. Sánchez-Ramón
          </td>
          <td>2024-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Crohn's disease (CD) is a complex inflammatory bowel disease whose pathogenesis appears to involve several immunologic defects causing functional impairment of the gut. Its complexity and the reported loss of effectiveness over time of standard of care together with the increase in its worldwide incidence require the application of techniques aiming to find new therapeutic strategies. Currently, systems pharmacology modeling has been gaining importance as it integrates the available knowledge of the system into a single computational model. In this work, the following workflow for robust application of systems pharmacology modeling was followed: 1) scope definition; 2) species selection and circulating plasma levels based on a search in the literature; 3) representation of model topology and parametrization of the interactions, after literature data extraction and curation, and the implementation of ordinary differential equations in SimBiology (MATLAB version R2018b); and 4) model curation and evaluation by visual comparison of simulated interleukin (IL) concentrations with the reported levels in plasma, and sensitivity analysis performed to confirm model robustness and identify the most influential parameters. Finally, 5) exposure to two dose levels of recombinant human IL10 was evaluated by simulation and comparison with reported clinical study results. In summary, we present a quantitative systems pharmacology model for the main ILs involved in CD developed using a standardized methodology and supported by a comprehensive repository summarizing the most relevant literature in the field. However, it has to be taken into account that external validation is still pending as available clinical data were primarily used for model training. SIGNIFICANCE STATEMENT Crohn's disease (CD) is a complex heterogeneous inflammatory bowel disorder. Systems pharmacology modeling offers a great opportunity for integration of the available knowledge on the disease using a computational framework. As a result of this work, a comprehensive repository along with a quantitative systems pharmacology model for the main interleukins involved in CD is provided. This model is useful for the in silico evaluation of biomarkers and potential therapeutic targets and can be adapted to address research gaps regarding CD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1b59cbda9a63547cdc28a859ad8c140d010a08" target='_blank'>A Quantitative Systems Pharmacology Model for the Key Interleukins Involved in Crohn's Disease</a></td>
          <td>
            Violeta Balbas-Martinez, E. Asín-Prieto, Zinnia P. Parra-Guillen, I. Trocóniz
          </td>
          <td>2019-12-10</td>
          <td>The Journal of Pharmacology and Experimental Therapeutics</td>
          <td>8</td>
          <td>32</td>
        </tr>

        <tr id="ABSTRACT Severe mortality due to the COVID-19 pandemic resulted from the lack of effective treatment. Although COVID-19 vaccines are available, their side effects have become a challenge for clinical use in patients with chronic diseases, especially cancer patients. In the current report, we applied network pharmacology and systematic bioinformatics to explore the use of biochanin A in patients with colorectal cancer (CRC) and COVID-19 infection. Using the network pharmacology approach, we identified two clusters of genes involved in immune response (IL1A, IL2, and IL6R) and cell proliferation (CCND1, PPARG, and EGFR) mediated by biochanin A in CRC/COVID-19 condition. The functional analysis of these two gene clusters further illustrated the effects of biochanin A on interleukin-6 production and cytokine-cytokine receptor interaction in CRC/COVID-19 pathology. In addition, pathway analysis demonstrated the control of PI3K-Akt and JAK-STAT signaling pathways by biochanin A in the treatment of CRC/COVID-19. The findings of this study provide a therapeutic option for combination therapy against COVID-19 infection in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916657a05db2cf3c7c148f16e01b739551950794" target='_blank'>Bioinformatics and in-silico findings reveal medical features and pharmacological targets of biochanin A against colorectal cancer and COVID-19</a></td>
          <td>
            Jingru Qin, C. Guo, Lu Yang, Xiaoliu Liang, Ai-Jun Jiao, K. Lai, Bin Yang
          </td>
          <td>2021-12-20</td>
          <td>Bioengineered</td>
          <td>5</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bf91901f6646b9e981fd3c56b8f32b335dae8f8" target='_blank'>Training in Systems Pharmacology: Predoctoral Program in Pharmacology and Systems Biology at Mount Sinai School of Medicine</a></td>
          <td>
            E. Sobie, Sherry L. Jenkins, Ravi Iyengar, T. A. Krulwich
          </td>
          <td>2010-07-01</td>
          <td>Clinical Pharmacology & Therapeutics</td>
          <td>10</td>
          <td>55</td>
        </tr>

        <tr id="[This retracts the article DOI: 10.1155/2022/7102500.].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee8964d4d366bc0df5964c9fdf0ea5152179ff69" target='_blank'>Retracted: Network Pharmacology-Based Strategy to Investigate the Mechanisms of Lenvatinib in the Treatment of Hepatocellular Carcinoma</a></td>
          <td>
            Computational Intelligence and Neuroscience
          </td>
          <td>2023-11-29</td>
          <td>Computational Intelligence and Neuroscience</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Rare diseases, affecting millions globally, present significant drug development challenges. This is due to the limited patient populations and the unique pathophysiology of these diseases, which can make traditional clinical trial designs unfeasible. Quantitative Systems Pharmacology (QSP) models offer a promising approach to expedite drug development, particularly in rare diseases. QSP models provide a mechanistic representation of the disease and drug response in virtual patients that can complement routinely applied empirical modeling and simulation approaches. QSP models can generate digital twins of actual patients and mechanistically simulate the disease progression of rare diseases, accounting for phenotypic heterogeneity. QSP models can also support drug development in various drug modalities, such as gene therapy. Impactful QSP models case studies are presented here to illustrate their value in supporting various aspects of drug development in rare indications. As these QSP model applications continue to mature, there is a growing possibility that they could be more widely integrated into routine drug development steps. This integration could provide a robust framework for addressing some of the inherent challenges in rare disease drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/501aa60f69949c250c5052e0be685b26e600d873" target='_blank'>Quantitative Systems Pharmacology Models: Potential Tools for Advancing Drug Development for Rare Diseases.</a></td>
          <td>
            Susana Neves-Zaph, C. Kaddi
          </td>
          <td>2024-09-28</td>
          <td>Clinical pharmacology and therapeutics</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Department of Gastroenterology, Guangdong Provincial People’s Hospital, Guangdong Academy of Medical Sciences, Southern Medical University, Guangzhou, China, The Second School of Clinical Medicine, Southern Medical University, Guangzhou, China, Shantou University Medical College, Shantou University, Shantou, China, Department of Computer Science, The University of Hong Kong, Hong Kong, Hong Kong SAR, China, Sepulveda Ambulatory Care Center, VA Greater Los Angeles Healthcare System, Los Angeles, CA, United States, University of California Los Angeles David Geffen School of Medicine, Los Angeles, CA, United States">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27aebc14245b473fcf6eae2729d7890fa59877d" target='_blank'>Editorial: Exploration of genetic variation, drug response, and interactions between gastrointestinal disorders and other diseases</a></td>
          <td>
            Shuangshuang Tong, Yanlin Lyu, Ruibang Luo, Felix W. Leung, W. Sha, Hao Chen
          </td>
          <td>2023-09-13</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="With the advent of effective immunotherapies to battle cancers and diseases, an obstacle in recovery has become the potential side effects, specifically cytokine release syndrome (CRS). As there is little quantitative understanding of risks for developing CRS and the degree of its severity, this work explored a model-based approach to produce a library of in silico patients through sensitivity analysis of cytokine interaction parameters and a Monte Carlo sampling. The objective of producing the in silico patients was to correlate a known grading system of cytokine release syndrome severity and thus design a new formula for grading CRS. Using our CRS grading system as the foundation, this work produced not only a formula which related the in silico patient data to the different grades, but we effectively demonstrated a selective approach to reduce the grade of CRS with sequential cytokine inhibition targets. We achieved the reduction of grades by applying the insight from the sensitivity analysis, beginning with the most sensitive targets. Cytokines IL-1, IL-8, TNF-α, INF-γ, IL-6, IL-2, IL-4, IL-10, and IL-12 were in turn the best targets for inhibition to alleviate CRS. Using this approach, patient cytokine time profiles in real-time can be related to the CRS grading system and if the grade is severe enough, action can be taken earlier during the treatment to prevent potentially life-threatening symptoms. What’s more, the identified inhibition sequence of the 9 cytokines provides guidance for clinical intervention of CRS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdbed3634ba13aed3fc2c3d954812638cfe069fa" target='_blank'>A Model-Based Investigation of Cytokine Dynamics in Immunotherapies</a></td>
          <td>
            B. Hopkins, Yiming Pan, M. Tucker, Z. Huang
          </td>
          <td>2018-12-30</td>
          <td>Processes</td>
          <td>5</td>
          <td>18</td>
        </tr>

        <tr id="Quantitative systems pharmacology (QSP) places an emphasis on dynamic systems modeling, incorporating considerations from systems biology modeling and pharmacodynamics. The goal of QSP is often to quantitatively predict the effects of clinical therapeutics, their combinations, and their doses on clinical biomarkers and endpoints. In order to achieve this goal, strategies for incorporating clinical data into model calibration are critical. Virtual population (VPop) approaches facilitate model calibration while faced with challenges encountered in QSP model application, including modeling a breadth of clinical therapies, biomarkers, endpoints, utilizing data of varying structure and source, capturing observed clinical variability, and simulating with models that may require more substantial computational time and resources than often found in pharmacometrics applications. VPops are frequently developed in a process that may involve parameterization of isolated pathway models, integration into a larger QSP model, incorporation of clinical data, calibration, and quantitative validation that the model with the accompanying, calibrated VPop is suitable to address the intended question or help with the intended decision. Here, we introduce previous strategies for developing VPops in the context of a variety of therapeutic and safety areas: metabolic disorders, drug-induced liver injury, autoimmune diseases, and cancer. We introduce methodological considerations, prior work for sensitivity analysis and VPop algorithm design, and potential areas for future advancement. Finally, we give a more detailed application example of a VPop calibration algorithm that illustrates recent progress and many of the methodological considerations. In conclusion, although methodologies have varied, VPop strategies have been successfully applied to give valid clinical insights and predictions with the assistance of carefully defined and designed calibration and validation strategies. While a uniform VPop approach for all potential QSP applications may be challenging given the heterogeneity in use considerations, we anticipate continued innovation will help to drive VPop application for more challenging cases of greater scale while developing new rigorous methodologies and metrics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68337aa80c1390201d9ea07d9ae681ebb37bfaa5" target='_blank'>Virtual Populations for Quantitative Systems Pharmacology Models.</a></td>
          <td>
            Yougan Cheng, R. Straube, Abed Alnaif, Lu Huang, T. Leil, B. Schmidt
          </td>
          <td>2021-08-24</td>
          <td>Methods in molecular biology</td>
          <td>22</td>
          <td>23</td>
        </tr>

        <tr id="Weighted gene co-expression network analysis, single-cell sequencing analysis, immune inﬁltration analysis, machine learning, DEG analysis and network pharmacology were the six analysis included in the study. Results showed that the co-morbidity between type 2 diabetes mellitus and ulcerative colitis is primarily associated with immune-inﬂammatory pathways, including IL-17, TNF, chemokine and toll-like receptor signaling pathways. Machine learning studies identiﬁed IGFBP3 as a biomarker for Gegen Qinlian Decoction in treating type 2 diabetes mellitus, while BACE2, EPHB4 and EPHA2 emerged as biomarkers for Gegen Qinlian decoction in ulcerative colitis treatment. The study provides insights into the shared pathogenesis of type 2 diabetes mellitus and ulcerative colitis and proposes novel targets and therapeutic strategies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bf5ade925b543b7cde2a6150f5b418f376b210c" target='_blank'>Editorial: Molecular informatics in precision medicine</a></td>
          <td>
            Alice Chen, S. Abdulazeez, Hari S. Sharma, J. Borgio
          </td>
          <td>2024-12-03</td>
          <td>Frontiers in Medicine</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="A quantitative systems pharmacology (QSP) model of the pathogenesis and treatment of SARS-CoV-2 infection can streamline and accelerate the development of novel medicines to treat COVID-19. Simulation of clinical trials allows in silico exploration of the uncertainties of clinical trial design and can rapidly inform their protocols. We previously published a preliminary model of the immune response to SARS-CoV-2 infection. To further our understanding of COVID-19 and treatment, we significantly updated the model by matching a curated dataset spanning viral load and immune responses in plasma and lung. We identified a population of parameter sets to generate heterogeneity in pathophysiology and treatment and tested this model against published reports from interventional SARS-CoV-2 targeting mAb and antiviral trials. Upon generation and selection of a virtual population, we match both the placebo and treated responses in viral load in these trials. We extended the model to predict the rate of hospitalization or death within a population. Via comparison of the in silico predictions with clinical data, we hypothesize that the immune response to virus is log-linear over a wide range of viral load. To validate this approach, we show the model matches a published subgroup analysis, sorted by baseline viral load, of patients treated with neutralizing Abs. By simulating intervention at different time points post infection, the model predicts efficacy is not sensitive to interventions within five days of symptom onset, but efficacy is dramatically reduced if more than five days pass post symptom onset prior to treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac952ba3508fc13b6ca5e8d7e11d7a7086f9b72d" target='_blank'>A quantitative systems pharmacology model of the pathophysiology and treatment of COVID-19 predicts optimal timing of pharmacological interventions</a></td>
          <td>
            Rohit Rao, C. Musante, R. Allen
          </td>
          <td>2021-12-08</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>8</td>
          <td>18</td>
        </tr>

        <tr id="Digital twins are computational models of complex systems, which aim to understand and optimize those systems more effectively than would be possible in real life. Ideally, digital twins can be translated to individual patients, to characterize and computationally treat their diseases with thousands of drugs, to select the drug or drugs that cure the patients. The background problem is that many patients do not respond adequately to drug treatment. This problem reflects a wide gap between the complexity of diseases and clinical practice. Each disease may involve altered interactions between thousands of genes that vary between different cell types in different organs. To our knowledge, these altered interactions have not been characterized on a genome-, cellulome-, and organ-wide scale in any disease. Thus, clinical translation of the digital twin ideal for predictive, preventive, personalized and participatory treatment involves formidable challenges, which are close to the limits of, or beyond today's technologies. Here, I discuss recent developments and challenges in relation to that ideal focusing on immune-mediated inflammatory diseases, as well as examples from other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/046005959d52420484ac7499efe6a496943efe38" target='_blank'>Digital Twins for Predictive, Preventive Personalized, and Participatory Treatment of Immune-Mediated Diseases.</a></td>
          <td>
            M. Benson
          </td>
          <td>2023-01-26</td>
          <td>Arteriosclerosis, thrombosis, and vascular biology</td>
          <td>5</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c536333e7bdcd1289228f777b10c660165468f60" target='_blank'>Computational systems biology in disease modeling and control, review and perspectives</a></td>
          <td>
            Rongting Yue, Abhishek Dutta
          </td>
          <td>2022-10-03</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>34</td>
          <td>2</td>
        </tr>

        <tr id="
 Omics and drug molecules become increasingly influential in identifying disease mechanisms and drug response. Because diseases and drug responses are co-expressed and regulated in the relevant omics interactions, the traditional way that grabbing molecular data from single isolated layers cannot always obtain valuable inference. Also, adverse effects exist in drugs that impair patients, and launching new medicines for diseases is costly. To resolve the above difficulties, systems biology is then applied to predict potential molecular interaction elements by integrating omics data from genomic, proteomic, transcriptional, and metabolic layers. Combined with known drug reactions, the resulting models improve medicines’ therapeutical performance by re-purposing the existing drugs and combining drug molecules without off-target effects. Based on the identified computational models, drug administration control laws are designed to balance toxicity and efficacy. This review introduces biomedical applications and analyses of interactions among omics and drug molecules for modeling disease mechanism and drug response. The therapeutical performance can be improved by combining the predictive and computational models with drug administration designed by control laws. The challenges are discussed for its clinic uses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1209aa00eb2b7665c9076ebc887ae3b0626ec5e" target='_blank'>Systems biology in disease modeling and control, analysis and perspectives</a></td>
          <td>
            Rongting Yue, Abhishek Dutta
          </td>
          <td>2021-06-02</td>
          <td></td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Systems biology uses experimental and computational approaches to characterize large sample populations systematically, process large datasets, examine and analyze regulatory networks, and model reactions to determine how components are joined to form functional systems. Systems biology technologies, data and knowledge are particularly useful in understanding disease processes and drug actions. An important area of integration between systems biology and drug discovery is the concept of polypharmacology: the treatment of diseases by modulating more than one target. Polypharmacology for complex diseases is likely to involve multiple drugs acting on distinct targets that are part of a network regulating physiological responses. This review discusses the current state of the systems-level understanding of diseases and both the therapeutic and adverse mechanisms of drug actions. Drug-target networks can be used to identify multiple targets and to determine suitable combinations of drug targets or drugs. Thus, the discovery of new drug therapies for complex diseases may be greatly aided by systems biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c8d6d727872a3a2e8a4f216b1fb0d4986fa5c04" target='_blank'>Systems approaches to polypharmacology and drug discovery.</a></td>
          <td>
            A. Boran, R. Iyengar
          </td>
          <td>2010-05-01</td>
          <td>Current opinion in drug discovery & development</td>
          <td>265</td>
          <td>64</td>
        </tr>

        <tr id="Systems approaches have long been used in pharmacology to understand drug action at the organ and organismal levels. The application of computational and experimental systems biology approaches to pharmacology allows us to expand the definition of systems pharmacology to include network analyses at multiple scales of biological organization and to explain both therapeutic and adverse effects of drugs. Systems pharmacology analyses rely on experimental "omics" technologies that are capable of measuring changes in large numbers of variables, often at a genome-wide level, to build networks for analyzing drug action. A major use of omics technologies is to relate the genomic status of an individual to the therapeutic efficacy of a drug of interest. Combining pathway and network analyses, pharmacokinetic and pharmacodynamic models, and a knowledge of polymorphisms in the genome will enable the development of predictive models of therapeutic efficacy. Network analyses based on publicly available databases such as the U.S. Food and Drug Administration's Adverse Event Reporting System allow us to develop an initial understanding of the context within which molecular-level drug-target interactions can lead to distal effectors in a process that results in adverse phenotypes at the organ and organismal levels. The current state of systems pharmacology allows us to formulate a set of questions that could drive future research in the field. The long-term goal of such research is to develop polypharmacology for complex diseases and predict therapeutic efficacy and adverse event risk for individuals prior to commencement of therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eff87d5850df2c45dab80bb553ce9e3f2d562ade" target='_blank'>Systems pharmacology: network analysis to identify multiscale mechanisms of drug action.</a></td>
          <td>
            Shan Zhao, R. Iyengar
          </td>
          <td>2012-01-11</td>
          <td>Annual review of pharmacology and toxicology</td>
          <td>306</td>
          <td>64</td>
        </tr>

        <tr id="The immune system is a complex and dynamic network, crucial for combating infections and maintaining health. Developing a comprehensive digital twin of the immune system requires incorporating essential cellular components and their interactions. This study presents the first blueprint for an immune system digital twin, consisting of a comprehensive and simulatable mechanistic model. It integrates 51 innate and adaptive immune cells, 37 secretory factors, and 11 disease conditions, providing the foundation for developing a multi-scale model. The cellular-level model demonstrates its potential in characterizing immune responses to various single and combinatorial disease conditions. By making the model available in easy-to-use formats directly in the Cell Collective platform, the community can easily and further expand it. This blueprint represents a significant step towards developing general-purpose immune digital twins, with far-reaching implications for the future of digital twin technology in life sciences and healthcare, advancing patient care, and accelerating precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6641c34ef9f1144419c9d233fad85729b60ada25" target='_blank'>Immune Digital Twin Blueprint: A Comprehensive Mechanistic Model of the Human Immune System</a></td>
          <td>
            Rada Amin, S. Aghamiri, B. L. Puniya, Lauren Mayo, Dennis Startsev, Kashish Poore, Resa M. K. Helikar, T. Helikar
          </td>
          <td>2020-03-12</td>
          <td>bioRxiv</td>
          <td>4</td>
          <td>24</td>
        </tr>

        <tr id="Drug development in oncology commonly exploits the tools of molecular biology to gain therapeutic benefit through reprograming of cellular responses. In immuno‐oncology (IO) the aim is to direct the patient’s own immune system to fight cancer. After remarkable successes of antibodies targeting PD1/PD‐L1 and CTLA4 receptors in targeted patient populations, the focus of further development has shifted toward combination therapies. However, the current drug‐development approach of exploiting a vast number of possible combination targets and dosing regimens has proven to be challenging and is arguably inefficient. In particular, the unprecedented number of clinical trials testing different combinations may no longer be sustainable by the population of available patients. Further development in IO requires a step change in selection and validation of candidate therapies to decrease development attrition rate and limit the number of clinical trials. Quantitative systems pharmacology (QSP) proposes to tackle this challenge through mechanistic modeling and simulation. Compounds’ pharmacokinetics, target binding, and mechanisms of action as well as existing knowledge on the underlying tumor and immune system biology are described by quantitative, dynamic models aiming to predict clinical results for novel combinations. Here, we review the current QSP approaches, the legacy of mathematical models available to quantitative clinical pharmacologists describing interaction between tumor and immune system, and the recent development of IO QSP platform models. We argue that QSP and virtual patients can be integrated as a new tool in existing IO drug development approaches to increase the efficiency and effectiveness of the search for novel combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe4c7719ed81361b93340d169c24788387a66ae" target='_blank'>Quantitative Systems Pharmacology Approaches for Immuno‐Oncology: Adding Virtual Patients to the Development Paradigm</a></td>
          <td>
            Vijayalakshmi Chelliah, Georgia Lazarou, Sumit Bhatnagar, J. Gibbs, M. Nijsen, Avijit Ray, B. Stoll, R. Thompson, A. Gulati, S. Soukharev, A. Yamada, Jared Weddell, Hiroyuki Sayama, Masayo Oishi, S. Wittemer-Rump, C. Patel, Christoph Niederalt, R. Burghaus, Christian Scheerans, J. Lippert, S. Kabilan, I. Kareva, N. Belousova, A. Rolfe, A. Zutshi, M. Chenel, Filippo Venezia, Sylvain Fouliard, H. Oberwittler, A. Scholer-Dahirel, H. Lelièvre, D. Bottino, Sabrina C. Collins, Hoa Q. Nguyen, Haiqing Wang, Tomoki Yoneyama, Andy Z. X. Zhu, Piet H. Graaf, A. Kierzek
          </td>
          <td>2020-07-19</td>
          <td>Clinical Pharmacology and Therapeutics</td>
          <td>51</td>
          <td>33</td>
        </tr>

        <tr id="We address the need for modelling and predicting adverse outcomes in immunotoxicology to improve non-clinical assessments of immunomodulatory therapy safety and efficacy. The integrated approach includes, first, the adverse outcome pathway concept established in the toxicology field, and, second, the systems medicine disease map approach for describing molecular mechanisms involved in a particular pathology. The proposed systems immunotoxicology workflow is demonstrated with CAR T cell treatment as a use case. To this end, the linear adverse outcome pathway (AOP) is expanded into a molecular interaction model in standard systems biology formats. Then it is shown how knowledge related to immunotoxic events can be integrated, encoded, managed and explored to benefit the research community. The map is accessible online via the MINERVA Platform for browsing, commenting and data visualisation (https://minerva.pages.uni.lu). Our work transforms a graphical illustration of an AOP into a digitally structured and standardised form, featuring precise and controlled vocabulary and supporting reproducible computational analyses. Because of annotations to source literature and databases, the map can be further expanded to match the evolving knowledge and research questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d13f7d84fd51fe9ce0ef217f6267f61784e78d9" target='_blank'>Using interactive platforms to encode, manage and explore immune-related adverse outcome pathways</a></td>
          <td>
            A. Mazein, Muhammad Shoaib, M. Alb, C. Sakellariou, Charline Sommer, K. Sewald, K. Reiche, Patricia Gogesch, Luise A Roser, Samira Ortega Iannazzo, S. Sheth, S. Schiffmann, Z. Waibler, V. Neuhaus, Susann Dehmel, V. Satagopam, Reinhard Schneider, M. Ostaszewski, Wei Gu
          </td>
          <td>2023-03-23</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>30</td>
        </tr>

        <tr id="mrgsolve is an open‐source R package available on the Comprehensive R Archive Network. It combines R and C++ coding for simulation from hierarchical, ordinary differential equation–based models. Its efficient simulation engine and integration into a parallelizable, R‐based workflow makes mrgsolve a convenient tool both for simple and complex models and thus is ideal for physiologically‐based pharmacokinetic (PBPK) and quantitative systems pharmacology (QSP) model. This tutorial will first introduce the basics of the mrgsolve simulation workflow, including model specification, the introduction of interventions (dosing events) into the simulation, and simulated results postprocessing. An applied simulation example is then presented using a PBPK model for voriconazole, including a model validation step against adult and pediatric data sets. A final simulation example is then presented using a previously published QSP model for mitogen‐activated protein kinase signaling in colorectal cancer, illustrating population simulation of different combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f58af03b1918b56e424198aaef83d3aa5da5080" target='_blank'>Quantitative Systems Pharmacology and Physiologically‐Based Pharmacokinetic Modeling With mrgsolve: A Hands‐On Tutorial</a></td>
          <td>
            Ahmed Elmokadem, M. Riggs, K. Baron
          </td>
          <td>2019-11-14</td>
          <td>CPT: Pharmacometrics & Systems Pharmacology</td>
          <td>52</td>
          <td>13</td>
        </tr>

        <tr id="To investigate the mechanism of action of Chenpi in the treatment of acute ulcerative colitis, this experiment employed a combination of network pharmacology and molecular docking technology to make a preliminary prediction of its mechanism of action. The TCMSP and PubChem databases were employed to identify the active components of Chenpi and their respective targets. In contrast, the SwissTargetPrediction database was utilized to predict the potential targets of action of Chenpi. The relevant targets of acute ulcerative colitis were obtained from the GeneCards disease database and subsequently intersected with the targets of Chen Pi to identify the key targets of Chen Pi for the treatment of acute ulcerative colitis. The results of the network pharmacological analysis indicated that Chenpi may exert its effects through ESR1, potentially via the action of sitosterol, naringenin, 5,7-dihydroxy-2-(3-hydroxy-4-methoxyphenyl) chroman-4-one, Citromitin, and nobiletin, five active components, as well as PTGS2, PPARG, BCL2, SRC, and MMP9, and an additional 104 core targets. Following the enrichment analysis of the aforementioned 104 core targets, it was determined that pathways such as "Pathways in cancer," "Arachidonic acid metabolism," "Prostate cancer," "EGFR tyrosine kinase inhibitor resistance," and others were involved in the treatment of Chen Pi. The results of molecular docking also demonstrated that the active ingredients of Chen Pi exhibited favorable binding affinities with the key targets, thereby substantiating the precision of the network pharmacology-predicted outcomes. The results of this experiment suggest that Chenpi may exert its therapeutic effects on acute ulcerative colitis through a multi-component, multi-target, and multi-pathway mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d20b2fc0c45174d5913c1461e9e75f579403316" target='_blank'>Exploring the Mechanism of Chen Pi in Treating Acute Ulcerative Colitis Based on Network Pharmacology and Molecular Docking</a></td>
          <td>
            Nai-dan Zhang, Tun Zhang
          </td>
          <td>2024-08-06</td>
          <td>Scholars Journal of Agriculture and Veterinary Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Irritable Bowel Syndrome (IBS) is characterized by symptoms that are dominated by abdominal pain and abnormal bowel habits, as defined by the Rome criteria. The complexity of the disorder is exemplified by the heterogeneity of symptom profiles and the number of putative pathophysiological mechanisms. Currently it is unclear whether IBS is a multifactorial disorder or rather a summary diagnosis for several distinct disease entities displaying similar symptoms. This thesis aims to identify subgroups of clinical relevance by developing and demonstrating symptomand mechanism-based stratification approaches, as well as an integrative analysis pipeline aiming to link different pathophysiological mechanisms. In a clinical sample of IBS patients, as well as in subjects fulfilling IBS in a population-based sample, symptom-based stratification yielded reproducible subgroups, characterized by combinations of gastrointestinal, extra-intestinal somatic and psychological symptoms. In the population-based sample this subgrouping was associated with differences in healthcare utilization. Mechanism-based stratification, focusing on the function of the autonomic nervous system (ANS), demonstrated altered ANS function in IBS patients compared to healthy controls, and identified a subgroup of IBS patients with aberrant overall ANS function, which was associated with more severe diarrhea. This thesis also introduces a stepwise multilevel integrative analysis pipeline using network theory, which presents associations of host-gene expression with mucosa-adherent gut microbiota as well as key IBS symptoms, revealing distinct IBS-specific associations. In conclusion, IBS patients show reproducible subgroups with specific profiles of a comprehensive set of IBS related symptoms and differences in healthcare needs based on these subgroups. Further, multivariate comparisons between IBS patients and healthy controls aid in identifying individuals for which specific complex pathophysiological mechanisms may be of relevance, as demonstrated by identifying a subset of IBS patients with aberrant overall ANS function. This stratification approach could be applied to other pathophysiological mechanisms. Our stepwise multilevel integrative analysis pipeline showed differences in variable associations at the gut mucosal level between IBS patients and healthy controls, and is therefore a model for further, comprehensive analysis of the complex pathophysiology of IBS">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6504f56649d0fdf449768eba336c5b475f44531" target='_blank'>Stratification and model-based analysis of patients with Irritable Bowel Syndrome using advanced biostatistics and medical data mining techniques</a></td>
          <td>
            A. Polster
          </td>
          <td>2018-05-24</td>
          <td></td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Motivation The literature on complex diseases is abundant but not always quantitative. This is particularly so for Inflammatory Bowel Disease (IBD), where many molecular pathways are qualitatively well described but this information cannot be used in traditional quantitative mathematical models employed in drug development. We propose the elaboration and validation of a logic network for IBD able to capture the information available in the literature that will facilitate the identification/validation of therapeutic targets. Results In this article, we propose a logic model for Inflammatory Bowel Disease (IBD) which consists of 43 nodes and 298 qualitative interactions. The model presented is able to describe the pathogenic mechanisms of the disorder and qualitatively describes the characteristic chronic inflammation. A perturbation analysis performed on the IBD network indicates that the model is robust. Also, as described in clinical trials, a simulation of anti-TNFα, anti-IL2 and Granulocyte and Monocyte Apheresis showed a decrease in the Metalloproteinases node (MMPs), which means a decrease in tissue damage. In contrast, as clinical trials have demonstrated, a simulation of anti-IL17 and anti-IFNγ or IL10 overexpression therapy did not show any major change in MMPs expression, as corresponds to a failed therapy. The model proved to be a promising in silico tool for the evaluation of potential therapeutic targets, the identification of new IBD biomarkers, the integration of IBD polymorphisms to anticipate responders and non-responders and can be reduced and transformed in quantitative model/s.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c655f04a31d1dc93671152d749f2e63985c7bf3" target='_blank'>A systems pharmacology model for inflammatory bowel disease</a></td>
          <td>
            Violeta Balbas-Martinez, Leire Ruiz-Cerdá, I. Irurzun-Arana, I. González-García, A. Vermeulen, J. Gómez-Mantilla, I. Trocóniz
          </td>
          <td>2018-03-07</td>
          <td>PLoS ONE</td>
          <td>31</td>
          <td>32</td>
        </tr>

        <tr id="Background Our previous studies demonstrate that ARL4C is the most critical clinical biomarker for gastric cancer (GC) patients among ARL family members (ARLs) and functions as an oncogene in GC. However, its underlying mechanisms in GC need to be further illustrated. In this study, we aim to explore the upstream and downstream molecular mechanisms of ARL4C in GC cells. Methods The genetic alteration of ARL4C in GC is analyzed by cBioPortal database. Potential ARL4C-targeted microRNAs (miRs) are predicted by three databases. The high-throughput RNA sequencing is performed to explore the underlying mechanisms of ARL4C in GC cells. The effects of predicted microRNAs on ARL4C, the RNA-sequencing results validation and the biological functions of ARL4C in GC cells are illustrated by in vitro experiments. Results Genetic analyses indicate that ARL4C is significantly upregulated in GC, which is not caused by gene amplification. MicroRNAs prediction shows the high relevance between ARL4C and miR-302 members. Moreover, miR-302c or miR-302d transfection reduces ARL4C protein expression in GC cells. Based on the high-throughput RNA sequencing of ARL4C-knockdown cells, enrichment analyses demonstrate that ARL4C is closely related to cell growth and involved in p53 signaling. Moreover, there are strong gene–gene interactions between ARL4C and genes in p53 signaling, and ARL4C downregulation could inhibit the protein expression of MDM2, a critical gene in p53 pathway. Further functional experiments demonstrate that ARL4C silencing leads to cell cycle arrest and increased cell apoptosis in AGS and MKN45 cells. Conclusion Our data suggest that miR-302c and miR-302d may function as the upstream regulators of ARL4C. And, ARL4C might promote GC cell cycle progression via regulating p53 signaling. Our findings provide novel insights into the key role of ARL4C and the underlying mechanisms in GC progression, thus facilitating the development of ARL4C-targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fad5c4b719c8fbe160566ddcf1c7a8c8d118ad9" target='_blank'>MicroRNA-302s Might Regulate ARL4C-Mediated Gastric Cancer Progression via p53 Signaling: Bioinformatics Analysis and Experiments Validation</a></td>
          <td>
            N. Xie, Yifei Pan, Jian Wu, Yunfan Bai, Cailan Xiao, Xiaoliang Gao, Jinhai Wang, Na Liu
          </td>
          <td>2021-04-13</td>
          <td>OncoTargets and therapy</td>
          <td>5</td>
          <td>33</td>
        </tr>

        <tr id=".............................................................................................iii DEDICATION.............................................................................................v ACKNOWLEDGEMENTS.............................................................................vi LIST OF TABLES.......................................................................................xv LIST OF FIGURES...................................................................................xviii ABBREVIATIONS......................................................................................xx Chapter">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/580c604d921e2e1348c562d2af6df501aca94108" target='_blank'>In silico strategies to study polypharmacology of G-protein-coupled receptors</a></td>
          <td>
            R. Hajjo
          </td>
          <td>2010-12-01</td>
          <td></td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Understanding the intricate interactions of cancer cells with the tumor microenvironment (TME) is a pre-requisite for the optimization of immunotherapy. Mechanistic models such as quantitative systems pharmacology (QSP) provide insights into the TME dynamics and predict the efficacy of immunotherapy in virtual patient populations/digital twins but require vast amounts of multimodal data for parameterization. Large-scale datasets characterizing the TME are available due to recent advances in bioinformatics for multi-omics data. Here, we discuss the perspectives of leveraging omics-derived bioinformatics estimates to inform QSP models and circumvent the challenges of model calibration and validation in immuno-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771e6ee84353828e640c230c07c64013c8803ab5" target='_blank'>Leveraging multi-omics data to empower quantitative systems pharmacology in immuno-oncology</a></td>
          <td>
            Theinmozhi Arulraj, Hanwen Wang, Alberto Ippolito, Shuming Zhang, E. Fertig, Aleksander S. Popel
          </td>
          <td>2024-03-27</td>
          <td>Briefings in Bioinformatics</td>
          <td>3</td>
          <td>45</td>
        </tr>

        <tr id="[This retracts the article DOI: 10.1155/2022/4237040.].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48cbd887b4ef7e0a81c0771e22576967cf38e556" target='_blank'>Retracted: Mechanisms of Banxia Xiexin Decoction Underlying Chronic Atrophic Gastritis via Network Pharmacology, Molecular Docking, and Molecular Dynamics Simulations</a></td>
          <td>
            Computational and Mathematical Methods in Medicine
          </td>
          <td>2023-12-06</td>
          <td>Computational and Mathematical Methods in Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="[This retracts the article DOI: 10.1155/2022/4640849.].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b703523f60961ba983f02a3568650635b680874d" target='_blank'>Retracted: Identification of Molecular Targets and Underlying Mechanisms of Xiaoji Recipe against Pancreatic Cancer Based on Network Pharmacology</a></td>
          <td>
            Computational and Mathematical Methods in Medicine
          </td>
          <td>2023-12-06</td>
          <td>Computational and Mathematical Methods in Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Systems pharmacology involves the application of systems biology approaches, combining large‐scale experimental studies with computational analyses, to the study of drugs, drug targets, and drug effects. Many of these initial studies have focused on identifying new drug targets, new uses of known drugs, and systems‐level properties of existing drugs. This review focuses on systems pharmacology studies that aim to better understand drug side effects and adverse events. By studying the drugs in the context of cellular networks, these studies provide insights into adverse events caused by off‐targets of drugs as well as adverse events‐mediated complex network responses. This allows rapid identification of biomarkers for side effect susceptibility. In this way, systems pharmacology will lead to not only newer and more effective therapies, but safer medications with fewer side effects. WIREs Syst Biol Med 2011 3 129–135 DOI: 10.1002/wsbm.114">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d54d677661075fc06b44974971c44a6cd366199a" target='_blank'>Role of systems pharmacology in understanding drug adverse events</a></td>
          <td>
            Seth I. Berger, R. Iyengar
          </td>
          <td>2011-03-01</td>
          <td>Wiley Interdisciplinary Reviews: Systems Biology and Medicine</td>
          <td>112</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/234074db21cc720b2506084d5f8bd01f9f92fa08" target='_blank'>Physiological Indirect Response Model to Omics-Powered Quantitative Systems Pharmacology Model.</a></td>
          <td>
            Aydar Uatay, Louis Gall, Linda Irons, Shivendra G. Tewari, Xu (Sue) Zhu, Megan Gibbs, Holly Kimko
          </td>
          <td>2023-10-01</td>
          <td>Journal of pharmaceutical sciences</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="Quantitative systems pharmacology (QSP) has emerged as an innovative approach in model‐informed drug discovery and development, supporting program decisions from exploratory research through late‐stage clinical trials. In this commentary, we discuss the unique value of disease‐scale “platform” QSP models that are amenable to reuse and repurposing to support diverse clinical decisions in ways distinct from other pharmacometrics strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75fd303e477a32d2844dd2c8a67fced699191fee" target='_blank'>Quantitative Systems Pharmacology: A Case for Disease Models</a></td>
          <td>
            C. Musante, S. Ramanujan, B. J. Schmidt, OG Ghobrial, James Lu, A. Heatherington
          </td>
          <td>2016-11-18</td>
          <td>Clinical Pharmacology and Therapeutics</td>
          <td>50</td>
          <td>23</td>
        </tr>

        <tr id="CPT: Pharmacometrics & Systems Pharmacology (PSP) is a crossdisciplinary journal which publishes research articles, reviews and tutorials that bridge various disciplines including pharmacometrics, systems pharmacology modeling, physiologically based pharmacokinetics (PBPK), translational research, modelbased metaanalyses (MBMA) of clinical trials, computational pharmacology and bioinformatics. PSP was launched in September 2012, as the second journal of American Society for Clinical Pharmacology and Therapeutics (ASCPT), with the first being Clinical Pharmacology and Therapeutics (CPT). Since the launch of Clinical & Translational Science (CTS), PSP is now part of a family of three journals. In June 2021, approaching its 9th anniversary, PSP was granted for the first time, an impact factor. We are proud to announce that the impact factor for 2020 is 4.054. Of note, the cite score of PSP was already increasing from 4.1 in 2015 to 6.3 in 2020, which is great news for both the journal and our scientific community. Hence, for the July 2021 issue, we introduce our first videoabstract, and in this editorial present our vision for the journal for the next ten years, and last, but certainly not least, present our new cover (Figure 1). It was selected by the Editor in Chief, associate editors and the ASCPT board members to illustrate the various aspects of PSP (molecules, equations, models, patients) and their interplay but also the cycling of the different components of modelinformed drug development (MIDD) and development and application of models as they continue to evolve.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff206807101a81dfa131bdd9e99907654d28bea6" target='_blank'>CPT: Pharmacometrics & Systems Pharmacology – Inception, Maturation, and Future Vision</a></td>
          <td>
            Karen Rowland Yeo, S. Hennig, S. Krishnaswami, Natasha Strydom, Vivaswath S. Ayyar, J. French, V. Sinha, E. Sobie, Ping Zhao, L. Friberg, F. Mentré
          </td>
          <td>2021-07-01</td>
          <td>CPT: Pharmacometrics & Systems Pharmacology</td>
          <td>6</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6891c99e59946101257e77b47551ca66c733bd55" target='_blank'>Quantitative and systems pharmacology 2. In silico polypharmacology of G protein‐coupled receptor ligands via network‐based approaches</a></td>
          <td>
            Zengrui Wu, Weiqiang Lu, Weiwei Yu, Tianduanyi Wang, Weihua Li, Guixia Liu, Hankun Zhang, Xiufeng Pang, Jin Huang, Mingyao Liu, F. Cheng, Yun Tang
          </td>
          <td>2017-11-10</td>
          <td>Pharmacological Research</td>
          <td>24</td>
          <td>67</td>
        </tr>

        <tr id="Immunotherapy has shown great potential in the treatment of cancer; however, only a fraction of patients respond to treatment, and many experience autoimmune‐related side effects. The pharmaceutical industry has relied on mathematical models to study the behavior of candidate drugs and more recently, complex, whole‐body, quantitative systems pharmacology (QSP) models have become increasingly popular for discovery and development. QSP modeling has the potential to discover novel predictive biomarkers as well as test the efficacy of treatment plans and combination therapies through virtual clinical trials. In this work, we present a QSP modeling platform for immuno‐oncology (IO) that incorporates detailed mechanisms for important immune interactions. This modular platform allows for the construction of QSP models of IO with varying degrees of complexity based on the research questions. Finally, we demonstrate the use of the platform through two example applications of immune checkpoint therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/750c1aa4fc2f364ea1febcf5761c49bd89bc2f93" target='_blank'>QSP‐IO: A Quantitative Systems Pharmacology Toolbox for Mechanistic Multiscale Modeling for Immuno‐Oncology Applications</a></td>
          <td>
            R. Sové, M. Jafarnejad, Chen Zhao, Hanwen Wang, Huilin Ma, A. Popel
          </td>
          <td>2020-07-02</td>
          <td>CPT: Pharmacometrics & Systems Pharmacology</td>
          <td>39</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ebc77a0835b6cda84893be93d324bfe05e1d5f" target='_blank'>An agent-based modeling framework linking inflammation and cancer using evolutionary principles: description of a generative hierarchy for the hallmarks of cancer and developing a bridge between mechanism and epidemiological data.</a></td>
          <td>
            G. An, S. Kulkarni
          </td>
          <td>2015-02-01</td>
          <td>Mathematical biosciences</td>
          <td>27</td>
          <td>39</td>
        </tr>

        <tr id="Intravenous immunoglobulin (IVIg) is used as treatment for several autoimmune and inflammatory conditions, but its specific mechanisms are not fully understood. Herein, we aimed to evaluate, using systems biology and artificial intelligence techniques, the differences in the pathophysiological pathways of autoimmune and inflammatory conditions that show diverse responses to IVIg treatment. We also intended to determine the targets of IVIg involved in the best treatment response of the evaluated diseases. Our selection and classification of diseases was based on a previously published systematic review, and we performed the disease characterization through manual curation of the literature. Furthermore, we undertook the mechanistic evaluation with artificial neural networks and pathway enrichment analyses. A set of 26 diseases was selected, classified, and compared. Our results indicated that diseases clearly benefiting from IVIg treatment were mainly characterized by deregulated processes in B cells and the complement system. Indeed, our results show that proteins related to B-cell and complement system pathways, which are targeted by IVIg, are involved in the clinical response. In addition, targets related to other immune processes may also play an important role in the IVIg response, supporting its wide range of actions through several mechanisms. Although B-cell responses and complement system have a key role in diseases benefiting from IVIg, protein targets involved in such processes are not necessarily the same in those diseases. Therefore, IVIg appeared to have a pleiotropic effect that may involve the collaborative participation of several proteins. This broad spectrum of targets and ‘non-specificity’ of IVIg could be key to its efficacy in very different diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad00dddd98df7f6f73eaf2590f07560a1de3655" target='_blank'>Systems biology and artificial intelligence analysis highlights the pleiotropic effect of IVIg therapy in autoimmune diseases with a predominant role on B cells and complement system</a></td>
          <td>
            C. Segú-Vergés, S. Caño, Elisabeth Calderón‐Gómez, H. Bartra, Teresa Sardon, S. Kaveri, J. Terencio
          </td>
          <td>2022-09-30</td>
          <td>Frontiers in Immunology</td>
          <td>7</td>
          <td>73</td>
        </tr>

        <tr id="In the last ten years, many advances have been made in the treatment and diagnosis of immune-mediated diseases [...].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b54a56078412c3f8ce10026ceaef38f6f42908b" target='_blank'>Personalized Medicine and Machine Learning: A Roadmap for the Future</a></td>
          <td>
            M. Sebastiani, C. Vacchi, A. Manfredi, G. Cassone
          </td>
          <td>2022-07-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>16</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdba71c8198592733c2f3c0e94873428e96682cb" target='_blank'>Advancing therapies for viral infections using mechanistic computational models of the dynamic interplay between the virus and host immune response</a></td>
          <td>
            V. Zarnitsyna, J. F. Gianlupi, A. Hagar, T. Sego, J. Glazier
          </td>
          <td>2021-08-24</td>
          <td>Current Opinion in Virology</td>
          <td>7</td>
          <td>51</td>
        </tr>

        <tr id="Low drug productivity has been a significant problem of the pharmaceutical industry for several decades even though numerous novel technologies were introduced during this period. Currently pharmacologic dogma, "single drug, single target, single disease", is at the root of the lack of drug productivity. From a systems biology viewpoint, network pharmacology has been proposed to complement the established guiding pharmacologic approaches. The rationale for network pharmacology as a major component of drug discovery and development is that a disease can be caused by perturbation of the disease-causing network and a drug may be designed to interact with multiple targets for modulation of such a network from the disease status toward normal status. Therefore, network pharmacology has been applied to guide and assist in drug repositioning. Drugs exerting their therapeutic effects may directly target disease-associated proteins, but they may also modulate the pathways involved in the pathological process. In this review, we discuss the progresses and prospects in network pharmacology, focusing on drug off-targets discovery, disease-associated protein identification, and pathway analysis for elucidating relationships between drug targets and disease-associated proteins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c01918b4e2ebbd2ab78c1e5c4d399d0e217ea6b4" target='_blank'>Drug Repositioning Through Network Pharmacology.</a></td>
          <td>
            Hao Ye, Jia Wei, Kailin Tang, R. Feuers, H. Hong
          </td>
          <td>2016-11-30</td>
          <td>Current topics in medicinal chemistry</td>
          <td>71</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b67c725d14beca01cd9dd910301260fc0dd1397" target='_blank'>Computational modeling of complex bioenergetic mechanisms that modulate CD4+ T cell effector and regulatory functions</a></td>
          <td>
            Ryan Baker, R. Hontecillas, Nuria Tubau-Juni, Andrew Leber, Shiv D. Kale, J. Bassaganya-Riera
          </td>
          <td>2022-11-22</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>6</td>
          <td>53</td>
        </tr>

        <tr id="Since its launch in 1960, Clinical Pharmacology & Therapeutics (CPT) has evolved as a crossdisciplinary premier journal in clinical pharmacology. The editorial leadership of CPT has set original research articles (ORAs) as a priority to advance innovation in the discipline. As a result, the majority of CPT ’s content now consists of ORAs, which cover a broad range of topics including clinical trials as well as post hoc analyses spanning diverse therapeutic areas, different (e.g., academic, industry, and regulatory) settings, and novel concepts in clinical pharmacology. In recent years, novel computational methods such as pharmacometrics and machine learning technologies, precision dosing approaches including pharmacogenomics, and pharmacological modalities beyond small molecules have revolutionized clinical pharmacology. Additionally, new developments in clinical trial design and increasingly inclusive patient recruitment have further advanced the field. Recently, realworld evidence, i.e., analyses of observational studies or routinely collected (“realworld”) data (RWD; e.g., from electronic health records), has gained increasing interest for hypothesis generation for adverse drug reactions and drug– drug interactions, and has been used to support regulatory decision making. To identify how these new developments in clinical pharmacology have been reflected in CPT publications, and how the focus of the journal has shifted, we evaluated key areas and therapeutic modalities studied in ORAs published between 2015 and 2021 (Figure 1). This effort was undertaken by CPT’s editorsintraining (I.K.M., D.M.S.), which is a role for early career scientists to learn about the editorial process and assist with the manuscript consideration process and other journal activities. ORAs published in CPT in 2015, 2018, and 2021 were manually assigned to associated therapeutic fields, drug modalities, and key areas of interest by two independent authors (I.K.M., D.M.S.). Discordant entries were discussed until consensus was met. Key areas were defined by the editorinchief, deputyeditorinchief, associate editors, and the editorsintraining, and included pharmacometrics / machine learning (PMx/ML), pharmacogenomics (PGx), transporters/enzymes, RWD, regulatory science, drug– drug interactions (DDIs), and “other.” Categorization was performed based on article content. For example, publications were assigned as PGx if they included genomewide association studies, candidate gene studies, PGx implementation, or related research. PMx/ ML encompassed publications that described, among others, the development and use of population pharmacokinetic/pharmacodynamic models, physiologybased models, machine learning, or related methodology. The category transporters/enzymes was assigned if an article described enzymes (e.g., cytochrome P450) or">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95c5d9926e461285ef3a7a16ded31bb0753d36a" target='_blank'>Trends in Clinical Pharmacology and Therapeutics</a></td>
          <td>
            Iris K. Minichmayr, D. Max Smith, K. Giacomini, Piet H. Graaf
          </td>
          <td>2023-06-19</td>
          <td>Clinical Pharmacology & Therapeutics</td>
          <td>5</td>
          <td>81</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>