<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              011d43546dfc34d1075f28232de2a553fc797695
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Virtual%20cell/" class="md-tabs__link">
        
  
    
  
  Virtual cell

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../RA/" class="md-tabs__link">
        
  
    
  
  RA

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Prostate%20cancer/" class="md-tabs__link">
        
  
    
  
  Prostate cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Virtual%20patients/" class="md-tabs__link">
        
  
    
  
  Virtual patients

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Virtual%20cell/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual cell
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../RA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RA
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Prostate%20cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Prostate cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Virtual%20patients/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual patients
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-03 06:08:45 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Thermodynamically consistent model calibration in chemical kinetics</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4961a5e558b6ec1d53c868d28e006fcf98882ebe" target='_blank'>Thermodynamically consistent Bayesian analysis of closed biochemical reaction systems</a></td>
          <td>
            Garrett Jenkinson, Xiaogang Zhong, J. Goutsias
          </td>
          <td>2010-11-05</td>
          <td>BMC Bioinformatics</td>
          <td>7</td>
          <td>29</td>
        </tr>

        <tr id="Mathematical and computational models of biological systems are increasingly complex, typically comprised of hybrid multi-scale methods such as ordinary differential equations, partial differential equations, agent-based and rule-based models, etc. These mechanistic models concurrently simulate detail at resolutions of whole host, multi-organ, organ, tissue, cellular, molecular, and genomic dynamics. Lacking analytical and numerical methods, solving complex biological models requires iterative parameter sampling-based approaches to establish appropriate ranges of model parameters that capture corresponding experimental datasets. However, these models typically comprise large numbers of parameters and therefore large degrees of freedom. Thus, fitting these models to multiple experimental datasets over time and space presents significant challenges. In this work we undertake the task of reviewing, testing, and advancing calibration practices across models and dataset types to compare methodologies for model calibration. Evaluating the process of calibrating models includes weighing strengths and applicability of each approach as well as standardizing calibration methods. Our work compares the performance of our model agnostic Calibration Protocol (CaliPro) with approximate Bayesian computing (ABC) to highlight strengths, weaknesses, synergies, and differences among these methods. We also present next-generation updates to CaliPro. We explore several model implementations and suggest a decision tree for selecting calibration approaches to match dataset types and modeling constraints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/886c9abed50f9657cff7804c05cd0031e17c3074" target='_blank'>Calibration methods to fit parameters within complex biological models</a></td>
          <td>
            Pariksheet Nanda, Denise E. Kirschner
          </td>
          <td>2023-10-18</td>
          <td>Frontiers in applied mathematics and statistics</td>
          <td>2</td>
          <td>2</td>
        </tr>

        <tr id="Ordinary differential equation (ODE) models are widely used to describe biochemical processes, since they effectively represent mass action kinetics. Optimization-based calibration of ODE models on experimental data can be challenging, even for low-dimensional problems. However, reliable model calibration is a prerequisite for uncertainty analysis, model comparison, and biological interpretation. Multiple hypotheses have been advanced to explain why optimization based calibration of biochemical models is challenging, but there are few comprehensive studies that test these hypotheses, likely because tools for performing such studies are also lacking. We implemented an established trust-region method as a modular Python framework (fides) to enable systematic comparison of different approaches to ODE model calibration involving various Hessian approximation schemes. We evaluated fides on a set of benchmark problems for which real experimental data are available. Unexpectedly, we observed high variability in optimizer performance among different implementations of the same algorithm. Overall, fides performed most reliably and efficiently. Our investigation of possible sources of poor optimizer performance identified drawbacks in the widely used Gauss-Newton, BFGS and SR1 Hessian approximations. We address these drawbacks by proposing a novel hybrid Hessian approximation scheme that enhances optimizer performance and outperforms existing hybrid approaches. We expect fides to be broadly useful for ODE constrained optimization problems and to enable future methods development. Availability fides is published under the permissive BSD-3-Clause license with source code publicly available at https://github.com/fides-dev/fides. Citeable releases are archived on Zenodo. Code to reproduce results presented in this manuscript is available at https://github.com/fides-dev/fides-benchmark.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30eaa3ce5bdf1c94216f0e5004673e2ec3378521" target='_blank'>Fides: Reliable trust-region optimization for parameter estimation of ordinary differential equation models</a></td>
          <td>
            Fabian Fröhlich, P. Sorger
          </td>
          <td>2021-05-22</td>
          <td>PLoS Computational Biology</td>
          <td>23</td>
          <td>108</td>
        </tr>

        <tr id="Estimating the rate constants of a biochemical reaction model of cellular function is an important, albeit computationally intensive, problem in systems biology. In this paper, a variance-based sensitivity analysis approach is proposed, which can be used, as a pre-screening step, to identify parameters in a biochemical reaction system that do not appreciably influence the cost of estimation and, therefore, whose values cannot be precisely determined by parameter estimation. By only estimating the remaining parameters, appreciable qualitative and quantitative improvements can be achieved. A subset of a well-known biochemical reaction model of the EGF/ERK signaling pathway is used to illustrate the benefits achieved by the proposed method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35dbd291e679b82a7da0772b437e9abc9638f529" target='_blank'>A screening method for dimensionality reduction in biochemical reaction system calibration</a></td>
          <td>
            Garrett Jenkinson, J. Goutsias
          </td>
          <td>2010-11-01</td>
          <td>2010 IEEE International Workshop on Genomic Signal Processing and Statistics (GENSIPS)</td>
          <td>4</td>
          <td>29</td>
        </tr>

        <tr id="The identification of suitable model parameters for biochemical reactions has been recognized as a quite difficult endeavor. Parameter values from literature or experiments can often not directly be combined in complex reaction systems. Nature-inspired optimization techniques can find appropriate sets of parameters that calibrate a model to experimentally obtained time series data. We present SBMLsimulator, a tool that combines the Systems Biology Simulation Core Library for dynamic simulation of biochemical models with the heuristic optimization framework EvA2. SBMLsimulator provides an intuitive graphical user interface with various options as well as a fully-featured command-line interface for large-scale and script-based model simulation and calibration. In a parameter estimation study based on a published model and artificial data we demonstrate the capability of SBMLsimulator to identify parameters. SBMLsimulator is useful for both, the interactive simulation and exploration of the parameter space and for the large-scale model calibration and estimation of uncertain parameter values.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/802c221d0efd3005aa01fde76864c58c89e37925" target='_blank'>SBMLSimulator: A Java Tool for Model Simulation and Parameter Estimation in Systems Biology</a></td>
          <td>
            Alexander Dörr, Roland Keller, A. Zell, Andreas Dräger
          </td>
          <td>2014-12-01</td>
          <td>Computation (Basel, Switzerland)</td>
          <td>15</td>
          <td>56</td>
        </tr>

        <tr id="Abstract Ordinary differential equation models are nowadays widely used for the mechanistic description of biological processes and their temporal evolution. These models typically have many unknown and nonmeasurable parameters, which have to be determined by fitting the model to experimental data. In order to perform this task, known as parameter estimation or model calibration, the modeller faces challenges such as poor parameter identifiability, lack of sufficiently informative experimental data and the existence of local minima in the objective function landscape. These issues tend to worsen with larger model sizes, increasing the computational complexity and the number of unknown parameters. An incorrectly calibrated model is problematic because it may result in inaccurate predictions and misleading conclusions. For nonexpert users, there are a large number of potential pitfalls. Here, we provide a protocol that guides the user through all the steps involved in the calibration of dynamic models. We illustrate the methodology with two models and provide all the code required to reproduce the results and perform the same analysis on new models. Our protocol provides practitioners and researchers in biological modelling with a one-stop guide that is at the same time compact and sufficiently comprehensive to cover all aspects of the problem.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c44fc63d08720ccf6be63b150ec181fbba6a29b" target='_blank'>A protocol for dynamic model calibration</a></td>
          <td>
            A. F. Villaverde, Dilan Pathirana, Fabian Fröhlich, J. Hasenauer, J. Banga
          </td>
          <td>2021-05-25</td>
          <td>Briefings in Bioinformatics</td>
          <td>34</td>
          <td>55</td>
        </tr>

        <tr id="Motivation Mechanistic models of biochemical reaction networks facilitate the quantitative understanding of biological processes and the integration of heterogeneous datasets. However, some biological processes require the consideration of comprehensive reaction networks and therefore large-scale models. Parameter estimation for such models poses great challenges, in particular when the data are on a relative scale. Results Here, we propose a novel hierarchical approach combining (i) the efficient analytic evaluation of optimal scaling, offset, and error model parameters with (ii) the scalable evaluation of objective function gradients using adjoint sensitivity analysis. We evaluate the properties of the methods by parameterizing a pan-cancer ordinary differential equation model (> 1000 state variables, > 4000 parameters) using relative protein, phospho-protein and viability measurements. The hierarchical formulation improves optimizer performance considerably. Furthermore, we show that this approach allows estimating error model parameters with negligible computational overhead when no experimental estimates are available, pro-viding an unbiased way to weight heterogeneous data. Overall, our hierarchical formulation is applicable to a wide range of models, and allows for the efficient parameterization of large-scale models based on heterogeneous relative measurements. Contact jan.hasenauer@helmholtz-muenchen.de Supplementary information Supplementary information are available at bioRxiv online. Supplementary code and data are available online at http://doi.org/10.5281/zenodo.2593839 and http://doi.org/10.5281/zenodo.2592186.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6410a37106ae5d68d99c17f729e603b5c1642a" target='_blank'>Efficient parameterization of large-scale dynamic models based on relative measurements</a></td>
          <td>
            Leonard Schmiester, Yannik Schälte, Fabian Froehlich, J. Hasenauer, Daniel Weindl
          </td>
          <td>2019-03-16</td>
          <td>Bioinformatics</td>
          <td>30</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Summary Molecular mechanisms of biological functions and disease processes are exceptionally complex, and our ability to interrogate and understand relationships is becoming increasingly dependent on the use of computational modeling. We have developed “BioModME,” a standalone R-based web application package, providing an intuitive and comprehensive graphical user interface to help investigators build, solve, visualize, and analyze computational models of complex biological systems. Some important features of the application package include multi-region system modeling, custom reaction rate laws and equations, unit conversion, model parameter estimation utilizing experimental data, and import and export of model information in the Systems Biology Matkup Language format. The users can also export models to MATLAB, R, and Python languages and the equations to LaTeX and Mathematical Markup Language formats. Other important features include an online model development platform, multi-modality visualization tool, and efficient numerical solvers for differential-algebraic equations and optimization. Availability and implementation All relevant software information including documentation and tutorials can be found at https://mcw.marquette.edu/biomedical-engineering/computational-systems-biology-lab/biomodme.php. Deployed software can be accessed at https://biomodme.ctsi.mcw.edu/. Source code is freely available for download at https://github.com/MCWComputationalBiologyLab/BioModME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef27698508bdbe5f1c9def5398c346fda4ae7ae6" target='_blank'>BioModME for building and simulating dynamic computational models of complex biological systems</a></td>
          <td>
            Justin Womack, Viren Shah, S. Audi, Scott S. Terhune, R. K. Dash
          </td>
          <td>2024-02-20</td>
          <td>Bioinformatics Advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12d5cf0c48097c6ed1866e1e3f1d971ef1cbf1f5" target='_blank'>PCA-based synthetic sensitivity coefficients for chemical reaction network in cancer</a></td>
          <td>
            Giorgia Biddau, Giacomo Caviglia, Michele Piana, Sara Sommariva
          </td>
          <td>2024-07-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="UNLABELLED
Modeling of dynamical systems using ordinary differential equations is a popular approach in the field of systems biology. Two of the most critical steps in this approach are to construct dynamical models of biochemical reaction networks for large datasets and complex experimental conditions and to perform efficient and reliable parameter estimation for model fitting. We present a modeling environment for MATLAB that pioneers these challenges. The numerically expensive parts of the calculations such as the solving of the differential equations and of the associated sensitivity system are parallelized and automatically compiled into efficient C code. A variety of parameter estimation algorithms as well as frequentist and Bayesian methods for uncertainty analysis have been implemented and used on a range of applications that lead to publications.


AVAILABILITY AND IMPLEMENTATION
The Data2Dynamics modeling environment is MATLAB based, open source and freely available at http://www.data2dynamics.org.


CONTACT
andreas.raue@fdm.uni-freiburg.de


SUPPLEMENTARY INFORMATION
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f047a957ff17bc9c01c20c5e37723f5206e58e7a" target='_blank'>Data2Dynamics: a modeling environment tailored to parameter estimation in dynamical systems</a></td>
          <td>
            A. Raue, Bernhard Steiert, Max Schelker, C. Kreutz, T. Maiwald, Helge Hass, J. Vanlier, C. Tönsing, L. Adlung, R. Engesser, W. Mader, T. Heinemann, J. Hasenauer, M. Schilling, T. Höfer, E. Klipp, Fabian J Theis, U. Klingmüller, B. Schoeberl, J. Timmer
          </td>
          <td>2015-11-01</td>
          <td>Bioinformatics</td>
          <td>215</td>
          <td>99</td>
        </tr>

        <tr id="Mechanistic mathematical modeling of biochemical reaction networks using ordinary differential equation (ODE) models has improved our understanding of small-and medium-scale biological processes. While the same should in principle hold for large-and genome-scale processes, the computational methods for the analysis of ODE models which describe hundreds or thousands of biochemical species and reactions are missing so far. While individual simulations are feasible, the inference of the model parameters from experimental data is computationally too intensive. In this manuscript, we evaluate adjoint sensitivity analysis for parameter estimation in large scale biochemical reaction networks. We present the approach for time-discrete measurement and compare it to state-of-the-art methods used in systems and computational biology. Our comparison reveals a significantly improved computational efficiency and a superior scalability of adjoint sensitivity analysis. The computational complexity is effectively independent of the number of parameters, enabling the analysis of large-and genome-scale models. Our study of a comprehensive kinetic model of ErbB signaling shows that parameter estimation using adjoint sensitivity analysis requires a fraction of the computation time of established methods. The proposed method will facilitate mechanistic modeling of genome-scale cellular processes, as required in the age of omics. Author Summary In this manuscript, we introduce a scalable method for parameter estimation for genome-scale biochemical reaction networks. Mechanistic models for genome-scale biochemical reaction networks describe the behavior of thousands of chemical species using thousands of parameters. Standard methods for parameter estimation are usually computationally intractable at these scales. Adjoint sensitivity based approaches have been suggested to have superior scalability but any rigorous evaluation is lacking. We implement a toolbox for adjoint sensitivity analysis for biochemical reaction network which also supports the import of SBML models. We show by means of a set of benchmark models that adjoint sensitivity based approaches unequivocally outperform standard approaches for large-scale models and that the achieved speedup increases with respect to both the number of parameters and the number of chemical species in the model. This demonstrates the applicability of adjoint sensitivity based approaches to parameter estimation for genome-scale mechanistic model. The MATLAB toolbox implementing the developed methods is available from http://ICB-DCM.github.io/AMICI/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b85cc5a3ee1709c8fad8040b683e7d720cf5c746" target='_blank'>Scalable Parameter Estimation for Genome-Scale Biochemical Reaction Networks</a></td>
          <td>
            Fabian Fröhlich, B. Kaltenbacher, Fabian J Theis, J. Hasenauer
          </td>
          <td>2016-11-22</td>
          <td>PLoS Computational Biology</td>
          <td>127</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d038165c25fb61341a0913a0ecc39aa1efeb74f" target='_blank'>Signal execution modes emerge in biochemical reaction networks calibrated to experimental data</a></td>
          <td>
            Oscar O. Ortega, Mustafa Ozen, Blake A. Wilson, James C. Pino, Michael W. Irvin, Geena V. Ildefonso, Shawn P. Garbett, Carlos F. Lopez
          </td>
          <td>2024-05-01</td>
          <td>iScience</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Mathematical models of biomolecular networks are commonly used to study cellular processes; however, their usefulness to explain and predict dynamic behaviors is often questioned due to the unclear relationship between parameter uncertainty and network dynamics. In this work, we introduce PyDyNo (Python Dynamic analysis of biochemical NetwOrks), a non-equilibrium reaction-flux based analysis to identify dominant reaction paths within a biochemical reaction network calibrated to experimental data. We first show, in a simplified apoptosis execution model, that Bayesian parameter optimization can yield thousands of parameter vectors with equally good fits to experimental data. Our analysis however enables us to identify the dynamic differences between these parameter sets and identify three dominant execution modes. We further demonstrate that parameter vectors from each execution mode exhibit varying sensitivity to perturbations. We then apply our methodology to JAK2/STAT5 network in colony-forming unit-erythroid (CFU-E) cells to identify its signal execution modes. Our analysis identifies a previously unrecognized mechanistic explanation for the survival responses of the CFU-E cell population that would have been impossible to deduce with traditional protein-concentration based analyses. Impact Statement Given the mechanistic models of network-driven cellular processes and the associated parameter uncertainty, we present a framework that can identify dominant reaction paths that could in turn lead to unique signal execution modes (i.e., dominant paths of flux propagation), providing a novel statistical and mechanistic insights to explain and predict signal processing and execution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/768c3894ff88395cd904f714ce789bccb719931e" target='_blank'>Emergent signal execution modes in biochemical reaction networks calibrated to experimental data</a></td>
          <td>
            Oscar O. Ortega, Mustafa Ozen, Blake A. Wilson, James C. Pino, Michael W. Irvin, Geena V. Ildefonso, Shawn P. Garbett, Carlos F. Lopez
          </td>
          <td>2021-01-27</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e213a890b84f948163c925b9db5c0b052e4ac9a" target='_blank'>Identifying effective evolutionary strategies-based protocol for uncovering reaction kinetic parameters under the effect of measurement noises</a></td>
          <td>
            Hock Chuan Yeo, Varsheni Vijay, Kumar Selvarajoo
          </td>
          <td>2024-10-14</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The integration of computational models with experimental data is a cornerstone for gaining insight into biomedical applications. However, parameter fitting procedures often require an immense availability and frequency of data that are challenging to obtain from a single source. Here, we present a novel methodology designed to integrate qualitative data from multiple laboratories, overcoming the constraints of single-lab data collection. Our method harmonizes disparate qualitative assessments—ranging from expert annotations to categorical observations—into a unified framework for parameter estimation. These qualitative constraints are represented as dynamic “qualitative windows” that capture significant trends that models must adhere to. For numerical implementation, we developed a GPU-accelerated version of differential evolution to navigate in the residuals that integrated quantitative and qualitative data. We validate our approach across a series of case studies, demonstrating significant improvements in model accuracy and parameter identifiability. This work opens new avenues for collaborative science, enabling a methodology to combine and compare findings between studies to improve our understanding of biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e23aa0adba9184d0e5e8a63c3015c30765e6cd19" target='_blank'>Integrating qualitative and quantitative data across multiple labs for model calibration</a></td>
          <td>
            Rodolfo Blanco-Rodríguez, Tanya A. Miura, Esteban A. Hernández-Vargas
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Motivation Dynamic models are used in systems biology to study and understand cellular processes like gene regulation or signal transduction. Frequently, ordinary differential equation (ODE) models are used to model the time and dose dependency of the abundances of molecular compounds as well as interactions and translocations. A multitude of computational approaches have been developed within recent years. However, many of these approaches lack proper testing in application settings because a comprehensive set of benchmark problems is yet missing. Results We present a collection of 20 ODE models developed given experimental data as benchmark problems in order to evaluate new and existing methodologies, e.g. for parameter estimation or uncertainty analysis. In addition to the equations of the dynamical system, the benchmark collection provides experimental measurements as well as observation functions and assumptions about measurement noise distributions and parameters. The presented benchmark models comprise problems of different size, complexity and numerical demands. Important characteristics of the models and methodological requirements are summarized, estimated parameters are provided, and some example studies were performed for illustrating the capabilities of the presented benchmark collection. Availability The models are provided in several standardized formats, including an easy-to-use human readable form and machine-readable SBML files. The data is provided as Excel sheets. All files are available at https://github.com/Benchmarking-Initiative/Benchmark-Models, with MATLAB code to process and simulate the models. Contact jan.hasenauer@helmholtz-muenchen.de, ckreutz@fdm.uni-freiburg.de">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/514b1f7c7156dd84fb84d9eb4654e4511eeaf584" target='_blank'>Benchmark problems for dynamic modeling of intracellular processes</a></td>
          <td>
            Helge Hass, Carolin Loos, Elba Raimundez Alvarez, J. Timmer, J. Hasenauer, C. Kreutz
          </td>
          <td>2018-08-30</td>
          <td>Bioinformatics</td>
          <td>62</td>
          <td>73</td>
        </tr>

        <tr id="The size and complexity of cellular systems make building predictive models an extremely difficult task. In principle dynamical time-course data can be used to elucidate the structure of the underlying molecular mechanisms, but a central and recurring problem is that many and very different models can be fitted to experimental data, especially when the latter are limited and subject to noise. Even given a model, estimating its parameters remains challenging in real-world systems. Here we present a comprehensive analysis of 180 systems biology models, which allows us to classify the parameters with respect to their contribution to the overall dynamical behaviour of the different systems. Our results reveal candidate elements of control in biochemical pathways that differentially contribute to dynamics. We introduce sensitivity profiles that concisely characterize parameter sensitivity and demonstrate how this can be connected to variability in data. Systematically linking data and model sloppiness allows us to extract features of dynamical systems that determine how well parameters can be estimated from time-course measurements, and associates the extent of data required for parameter inference with the model structure, and also with the global dynamical state of the system. The comprehensive analysis of so many systems biology models reaffirms the inability to estimate precisely most model or kinetic parameters as a generic feature of dynamical systems, and provides safe guidelines for performing better inferences and model predictions in the context of reverse engineering of mathematical models for biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/330e8c6df2bf2adb506714cd065e9d1b21b1dc82" target='_blank'>Practical limits for reverse engineering of dynamical systems: a statistical analysis of sensitivity and parameter inferability in systems biology models.</a></td>
          <td>
            K. Erguler, M. Stumpf
          </td>
          <td>2011-05-01</td>
          <td>Molecular bioSystems</td>
          <td>94</td>
          <td>56</td>
        </tr>

        <tr id="In silico models of biological systems are usually very complex and rely on a large number of parameters describing physical and biological properties that require validation. As such, parameter space exploration is an essential component of computational model development to fully characterize and validate simulation results. Experimental data may also be used to constrain parameter space (or enable model calibration) to enhance the biological relevance of model parameters. One widely used computational platform in the mathematical biology community is PhysiCell, which provides a standardized approach to agent-based models of biological phenomena at different time and spatial scales. Nonetheless, one limitation of PhysiCell is the lack of a generalized approach for parameter space exploration and calibration that can be run without high-performance computing access. Here, we present PhysiCOOL, an open-source Python library tailored to create standardized calibration and optimization routines for PhysiCell models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93b3fdecfb536ee9bd949807c6fd7054ba8d434b" target='_blank'>PhysiCOOL: A generalized framework for model Calibration and Optimization Of modeLing projects</a></td>
          <td>
            Inês G. Gonçalves, D. Hormuth, Sandhya Prabhakaran, C. Phillips, J. García-Aznar
          </td>
          <td>2023-02-28</td>
          <td>GigaByte</td>
          <td>3</td>
          <td>43</td>
        </tr>

        <tr id="Computational dynamic ODE models of cell function describing biochemical reactions have been created for decades, but on a small scale. Still, they have been highly effective in describing and predicting behaviors. For example, oscillatory phospho-ERK levels were predicted and confirmed in MAPK signaling encompassing both positive and negative feedback loops. These models typically were limited and not adapted to large datasets so commonly found today. But importantly, ODE models describe reaction networks in well-mixed systems representing the cell and can be simulated with ordinary differential equations that are solved deterministically. Stochastic solutions, which can account for noisy reaction networks, in some cases, also improve predictions. Today, dynamic ODE models rarely encompass an entire cell even though it might be expected that an upload of the large genomic, transcriptomic, and proteomic datasets may allow whole cell models. It is proposed here to combine output from simulated dynamic ODE models, completed with omics data, to discover both biomarkers in cancer a priori and molecular targets in the Machine Learning setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/500b567ec04a718411eb4a59b5b37f83439cb68d" target='_blank'>Capturing Biomarkers and Molecular Targets in Cellular Landscapes From Dynamic Reaction Network Models and Machine Learning</a></td>
          <td>
            S. Mertins
          </td>
          <td>2022-01-21</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88c271f2fa41f5319499d498ee6ad8987ce5aca8" target='_blank'>CaliBayes: Integration of GRID based simulation and data resources for Bayesian calibration of biological models</a></td>
          <td>
            R. Boys, D. Henderson, T. Kirkwood, D. Wilkinson, W. Wolski
          </td>
          <td>2005-09-21</td>
          <td>BMC Bioinformatics</td>
          <td>8</td>
          <td>65</td>
        </tr>

        <tr id="Hierarchical modelling is essential to achieving complex, large-scale models. However, not all modelling schemes support hierarchical composition, and correctly mapping points of connection between models requires comprehensive knowledge of each model’s components and assumptions. To address these challenges in integrating biosimulation models, we propose an approach to automatically and confidently compose biosimulation models. The approach uses bond graphs to combine aspects of physical and thermodynamics-based modelling with biological semantics. We improved on existing approaches by using semantic annotations to automate the recognition of common components. The approach is illustrated by coupling a model of the Ras-MAPK cascade to a model of the upstream activation of EGFR. Through this methodology, we aim to assist researchers and modellers in readily having access to more comprehensive biological systems models. Author summary Detailed, multi-scale computational models bridging from biomolecular processes to entire organs and bodies have the potential to revolutionise medicine by enabling personalised treatments. One of the key challenges to achieving these models is connecting together the vast number of isolated biosimulation models into a coherent whole. Using recent advances in both modelling techniques and biological standards in the scientific community, we developed an approach to integrate and compose models in a physics-based environment. This provides significant advantages, including the automation of model composition and post-model-composition adjustments. We anticipate that our approach will enable the faster development of realistic and accurate models to understand complex biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/065a0d90e6587ed4030e8602450e7300c608a995" target='_blank'>A semantics, energy-based approach to automate biomodel composition</a></td>
          <td>
            Niloofar Shahidi, Michael Pan, K. Tran, E. Crampin, D. Nickerson
          </td>
          <td>2021-11-13</td>
          <td>PLoS ONE</td>
          <td>6</td>
          <td>36</td>
        </tr>

        <tr id="These lecture materials describe how to create kinetic models based on information in the literature and include a student assignment. This Teaching Resource provides lecture notes, slides, and a student assignment for a lecture on strategies for the development of mathematical models. Many biological processes can be represented mathematically as systems of ordinary differential equations (ODEs). Simulations with these mathematical models can provide mechanistic insight into the underlying biology of the system. A prerequisite for running simulations, however, is the identification of kinetic parameters that correspond closely with the biological reality. This lecture presents an overview of the steps required for the development of kinetic ODE models and describes experimental methods that can yield kinetic parameters and concentrations of reactants, which are essential for the development of kinetic models. Strategies are provided to extract necessary parameters from published data. The homework assignment requires students to find parameters appropriate for a well-studied biological regulatory system, convert these parameters into appropriate units, and interpret how different values of these parameters may lead to different biological behaviors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb8be00d6778f6c9b7792433130275ebd3490a09" target='_blank'>Obtaining and Estimating Kinetic Parameters from the Literature</a></td>
          <td>
            Susana R. Neves
          </td>
          <td>2011-09-20</td>
          <td>Science Signaling</td>
          <td>11</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d049b4f815695b19bfcf2d1561200d20ae72310" target='_blank'>Guidelines for benchmarking of optimization-based approaches for fitting mathematical models</a></td>
          <td>
            C. Kreutz
          </td>
          <td>2019-07-08</td>
          <td>Genome Biology</td>
          <td>9</td>
          <td>36</td>
        </tr>

        <tr id="Lecture notes, slides, and problem set illustrate methods for curve fitting and error analysis relevant to development of computational models for biomedical applications. This Teaching Resource introduces students to curve fitting and error analysis; it is the second of two lectures on developing mathematical models of biomedical systems. The first focused on identifying, extracting, and converting required constants—such as kinetic rate constants—from experimental literature. To understand how such constants are determined from experimental data, this lecture introduces the principles and practice of fitting a mathematical model to a series of measurements. We emphasize using nonlinear models for fitting nonlinear data, avoiding problems associated with linearization schemes that can distort and misrepresent the data. To help ensure proper interpretation of model parameters estimated by inverse modeling, we describe a rigorous six-step process: (i) selecting an appropriate mathematical model; (ii) defining a “figure-of-merit” function that quantifies the error between the model and data; (iii) adjusting model parameters to get a “best fit” to the data; (iv) examining the “goodness of fit” to the data; (v) determining whether a much better fit is possible; and (vi) evaluating the accuracy of the best-fit parameter values. Implementation of the computational methods is based on MATLAB, with example programs provided that can be modified for particular applications. The problem set allows students to use these programs to develop practical experience with the inverse-modeling process in the context of determining the rates of cell proliferation and death for B lymphocytes using data from BrdU-labeling experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24e4f19b79133b31dd0b75fcccb496c66f3c27e4" target='_blank'>Biomedical Model Fitting and Error Analysis</a></td>
          <td>
            K. Costa, S. Kleinstein, Uri Hershberg
          </td>
          <td>2011-09-27</td>
          <td>Science Signaling</td>
          <td>13</td>
          <td>61</td>
        </tr>

        <tr id="The transition from explanative modelling of fitted data to the predictive modelling of unseen data for systems biology endeavors necessitates the effective recovery of reaction parameters. Yet, the relative efficacy of optimization algorithms in doing so remains under-studied, as to the specific reaction kinetics and the effect of measurement noises. To this end, we simulate the reactions of an artificial pathway using 4 kinetic formulations: generalized mass action (GMA), Michaelis-Menten, linear-logarithmic, and convenience kinetics. We then compare the effectiveness of 5 evolutionary algorithms (CMAES, DE, SRES, ISRES, G3PCX) for objective function optimization in kinetic parameter hyperspace to determine the corresponding estimated parameters. We quickly dropped the DE algorithm due to its poor performance. Baring measurement noise, we find CMAES algorithm to only require a fraction of the computational cost incurred by other EAs for both GMA and linear-logarithmic kinetics yet performing as well by other criteria. However, with increasing noise, SRES and ISRES perform more reliably for GMA kinetics, but at considerably higher computational cost. Conversely, G3PCX is among the most efficacious for estimating Michaelis-Menten parameters regardless of noise, while achieving numerous folds saving in computational cost. Cost aside, we find SRES to be versatilely applicable across GMA, Michaelis-Menten, and linear-logarithmic kinetics, with good resilience to noise. On the other hand, we could not identify the parameters of convenience kinetics using any algorithm. Together, we find algorithms that are effective under marked measurement noise for specific reaction kinetics, as a step towards predictive modelling for systems biology endeavors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d9b5f01cba7ef80849690a58034e44d2bcf713e" target='_blank'>Identifying effective evolutionary strategies for uncovering reaction kinetic parameters under the effect of measurement noises</a></td>
          <td>
            Hock Chuan Yeo, Vijay Varsheni, Kumar Selvarajoo
          </td>
          <td>2024-03-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ab366e6d4332164b4c018a4b813cbcbfb9293d3" target='_blank'>Improved Parameter Estimation in Kinetic Models: Selection and Tuning of Regularization Methods</a></td>
          <td>
            A. Gábor, J. Banga
          </td>
          <td>2014-11-17</td>
          <td>DBLP, MAG</td>
          <td>6</td>
          <td>55</td>
        </tr>

        <tr id="We address the problem of evaluating the transfer entropy (TE) produced by biochemical reactions from experimentally measured data. Although these reactions are generally nonlinear and nonstationary processes making it challenging to achieve accurate modeling, Gaussian approximation can facilitate the TE assessment only by estimating covariance matrices using multiple data obtained from simultaneously measured time series representing the activation levels of biomolecules such as proteins. Nevertheless, the nonstationary nature of biochemical signals makes it difficult to theoretically assess the sampling distributions of TE, which are necessary for evaluating the statistical confidence and significance of the data-driven estimates. We resolve this difficulty by computationally assessing the sampling distributions using techniques from computational statistics. The computational methods are tested by using them in analyzing data generated from a theoretically tractable time-varying signal model, which leads to the development of a method to screen only statistically significant estimates. The usefulness of the developed method is examined by applying it to real biological data experimentally measured from the ERBB-RAS-MAPK system that superintends diverse cell fate decisions. A comparison between cells containing wild-type and mutant proteins exhibits a distinct difference in the time evolution of TE while any apparent difference is hardly found in average profiles of the raw signals. Such a comparison may help in unveiling important pathways of biochemical reactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa8d4d3ba8a60846ca34bd27cd31ea0120b36094" target='_blank'>Assessing transfer entropy from biochemical data.</a></td>
          <td>
            T. Imaizumi, N. Umeki, Ryo Yoshizawa, T. Obuchi, Y. Sako, Y. Kabashima
          </td>
          <td>2021-11-09</td>
          <td>Physical review. E</td>
          <td>4</td>
          <td>40</td>
        </tr>

        <tr id="Kinetic models are increasingly relevant in medical research. In systems biology, more than 10 years of experience with the development of standards and tools to construct and analyse kinetic models exists. This has supported the sharing of kinetic models, increased their reuse, and thereby has helped to reproduce and validate scientific results. Given this expertise, it seems natural to consider the application and development of standards and tools to meet the requirements of medical scientists. In this paper, we discuss challenges and opportunities for standards and tools from systems biology in medical research, and we suggest criteria for the safe use of simulations. We conclude that standards, tools and infrastructure need to be extended to ensure the quality, reliability and safety required when working with medical and patient data. This will foster the adaptation of modelling in the clinic, providing tools for improved diagnosis, prognosis and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f673e5c5ecd7d602c5af9468051c8b3a16fcfce" target='_blank'>Challenges and opportunities for system biology standards and tools in medical research</a></td>
          <td>
            M. König, A. Oellrich, Dagmar Waltemath, R. Dobson, Tim J. P. Hubbard, O. Wolkenhauer
          </td>
          <td>2016-09-29</td>
          <td>DBLP, MAG</td>
          <td>3</td>
          <td>52</td>
        </tr>

        <tr id="Due to the high complexity of biological data it is difficult to disentangle cellular processes relying only on intuitive interpretation of measurements. A Systems Biology approach that combines quantitative experimental data with dynamic mathematical modeling promises to yield deeper insights into these processes. Nevertheless, with growing complexity and increasing amount of quantitative experimental data, building realistic and reliable mathematical models can become a challenging task: the quality of experimental data has to be assessed objectively, unknown model parameters need to be estimated from the experimental data, and numerical calculations need to be precise and efficient. Here, we discuss, compare and characterize the performance of computational methods throughout the process of quantitative dynamic modeling using two previously established examples, for which quantitative, dose- and time-resolved experimental data are available. In particular, we present an approach that allows to determine the quality of experimental data in an efficient, objective and automated manner. Using this approach data generated by different measurement techniques and even in single replicates can be reliably used for mathematical modeling. For the estimation of unknown model parameters, the performance of different optimization algorithms was compared systematically. Our results show that deterministic derivative-based optimization employing the sensitivity equations in combination with a multi-start strategy based on latin hypercube sampling outperforms the other methods by orders of magnitude in accuracy and speed. Finally, we investigated transformations that yield a more efficient parameterization of the model and therefore lead to a further enhancement in optimization performance. We provide a freely available open source software package that implements the algorithms and examples compared here.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68e00b675f75325a43dd4db74228f0d7177c00ee" target='_blank'>Lessons Learned from Quantitative Dynamical Modeling in Systems Biology</a></td>
          <td>
            A. Raue, M. Schilling, J. Bachmann, Andrew Matteson, Max Schelke, Daniel Kaschek, S. Hug, C. Kreutz, B. Harms, Fabian J Theis, U. Klingmüller, J. Timmer
          </td>
          <td>2013-09-30</td>
          <td>PLoS ONE</td>
          <td>298</td>
          <td>99</td>
        </tr>

        <tr id="Quantitative dynamical models facilitate the understanding of biological processes and the prediction of their dynamics. The parameters of these models are commonly estimated from experimental data. Yet, experimental data generated from different techniques do not provide direct information about the state of the system but a non-linear (monotonic) transformation of it. For such semi-quantitative data, when this transformation is unknown, it is not apparent how the model simulations and the experimental data can be compared. Here, we propose a versatile spline-based approach for the integration of a broad spectrum of semi-quantitative data into parameter estimation. We derive analytical formulas for the gradients of the hierarchical objective function and show that this substantially increases the estimation efficiency. Subsequently, we demonstrate that the method allows for the reliable discovery of unknown measurement transformations. Furthermore, we show that this approach can significantly improve the parameter inference based on semi-quantitative data in comparison to available methods. Modelers can easily apply our method by using our implementation in the open-source Python Parameter EStimation TOolbox (pyPESTO).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89f706fe0ee36f858690aba9db910a13d68c883d" target='_blank'>Efficient parameter estimation for ODE models of cellular processes using semi-quantitative data</a></td>
          <td>
            Domagoj Dorešić, Stephan Grein, Jan Hasenauer
          </td>
          <td>2024-01-30</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="There are many mathematical models of biochemical cell signaling pathways that contain a large number of elements (species and reactions). This is sometimes a big issue for identifying critical model elements and describing the model dynamics. Thus, techniques of model reduction can be used as a mathematical tool in order to minimize the number of variables and parameters. In this thesis, we review some well-known methods of model reduction for cell signaling pathways. We have also developed some approaches that provide us a great step forward in model reduction. The techniques are quasi steady state approximation (QSSA), quasi equilibrium approximation (QEA), lumping of species and entropy production analysis. They are applied on protein translation pathways with microRNA mechanisms, chemical reaction networks, extracellular signal regulated kinase (ERK) pathways, NFkB signal transduction pathways, elongation factors EFTu and EFTs signaling pathways and Dihydrofolate reductase (DHFR) pathways. The main aim of this thesis is to reduce the complex cell signaling pathway models. This provides one a better understanding of the dynamics of such models and gives an accurate approximate solution. Results show that there is a good agreement between the original models and the simplified models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badbb8adee6c29733dbf304c97c828b0645152b7" target='_blank'>Techniques of Model Reductions in Biochemical Cell Signaling Pathways</a></td>
          <td>
            H. Rasool, Sarbaz H. A. Khoshnaw
          </td>
          <td>2021-09-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/921e340d00314b9b2f248cd4d7c8a147d92d91bb" target='_blank'>Parameterization of mechanistic models from qualitative data using an efficient optimal scaling approach</a></td>
          <td>
            Leonard Schmiester, Daniel Weindl, J. Hasenauer
          </td>
          <td>2020-07-21</td>
          <td>Journal of Mathematical Biology</td>
          <td>9</td>
          <td>39</td>
        </tr>

        <tr id="Mathematical models of biological processes have various applications: to assist in understanding the functioning of a system, to simulate experiments before actually performing them, to study situations that cannot be dealt with experimentally, etc. Some parameters in the model can be directly obtained from experiments or from the literature. Others have to be inferred by comparing model results to experiments. In this minireview, we discuss the identifiability of models, both intrinsic to the model and taking into account the available data. Furthermore, we give an overview of the most frequently used approaches to search the parameter space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/905b9a95408a5a30b41f304dadcc255cb6af57d3" target='_blank'>Systems biology: parameter estimation for biochemical models</a></td>
          <td>
            M. Ashyraliyev, Y. Fomekong-Nanfack, J. Kaandorp, J. Blom
          </td>
          <td>2009-02-01</td>
          <td>The FEBS Journal</td>
          <td>343</td>
          <td>32</td>
        </tr>

        <tr id="Motivation: Modeling biological signaling networks using ordinary differential equations (ODEs) has proven to be a powerful technique for generating insight into cellular dynamics, but it typically requires estimating rate parameters based on experimentally observed concentrations. New measurement methods can measure concentrations for all molecular species in a pathway, which creates a new opportunity to decompose the optimization of rate parameters. Results: In contrast with conventional methods that minimize the disagreement between simulated and observed concentrations, the BPPE method fits a spline curve through the observed concentration points, and then matches the derivatives of the spline-curve to the production and consumption of each species. Whereas traditional methods follow the ODEs exactly and then attempt to match the data, BPPE follows the data exactly and then attempts to match the ODEs. The new objective function is an extreme decomposition of the problem because each factor in the function is enforcing the equality of one ODE at one timeslice. A "loopy belief propagation" algorithm solves this factorized approximation of the parameter estimation problem providing systematic coverage of the search space and unique asymptotic behavior; the run time is polynomial in the number of molecules and timepoints, but exponential in the degree of the biochemical network. The implementation is a global-local hybrid optimization, and we compare with the performance of local, global, and hybrid methods. BPPE is demonstrated for a novel model of Akt activation dynamics including redox-mediated inactivation of PTEN. Availability: Software and supplementary information are available at this http URL . Contact: LisaTK@nus.edu.sg . Keywords: probabilistic graphical models, physico-chemical modeling, systems biology, signal transduction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef71b981e6491bccb1e61863a1a09606d9645a3" target='_blank'>Estimating Reaction Rate Parameters in Cell Signaling Pathways Using Extreme Decomposition and Belief Propagation Tailored for Data-Rich Cases</a></td>
          <td>
            Tri Hieu Nim, Le Luo, Marie-V'eronique Cl'ement, Jacob K. White, L. Tucker-Kellogg
          </td>
          <td>2011-03-04</td>
          <td>arXiv: Molecular Networks</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Ordinary differential equation (ODE) models are frequently used to mathematically represent the dynamic behavior of cellular components, e.g.\ for describing biochemical reaction networks. Solutions of these ODE models depend non-linearly on parameters, which can be estimated using experimental data by minimizing the discrepancy between data and the model trajectories. In realistic applications, only relative, sparse and noisy data is available which makes model fitting a challenging optimization problem. In order to take account for the non-convexity of the objective function and to reveal the existence of local optima within the parameter search space, optimization is performed with multiple initial guesses. For statistically valid conclusions it is of general interest, whether distinct optimization outcomes are correctly identified as local optima originating from the non-convexity of the objective function as typically presumed, or if they are only a result of incompletely converged optimization runs and could be merged by a connecting path to a single optimum by e.g.\ fine-tuning of the numeric algorithms. To clarify this question in application settings, we present an approach for finding optimal paths between parameter estimates in complex objective function landscapes. By analyzing the profiles of these paths, conclusions about non-trivial connections of parameter estimates can be made and by this, it can be reliably determined if two different optimization results belong to the same or to distinct local optima. For optimal path finding, we adapt the nudged elastic band (NEB) method and apply our approach to a benchmark model with a suboptimal optimization result, which yields a preferable grouping of fits afterwards.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d22082d136f6e1770d51baf8bac7f7aaeb3bb502" target='_blank'>Optimal Paths Between Parameter Estimates in Non-linear ODE Systems Using the Nudged Elastic Band Method</a></td>
          <td>
            C. Tönsing, J. Timmer, C. Kreutz
          </td>
          <td>2019-10-09</td>
          <td>Frontiers in Physics</td>
          <td>3</td>
          <td>73</td>
        </tr>

        <tr id="ABSTRACT Understanding the dynamics of biological systems in evolving environments is a challenge due to their scale and complexity. Here, we present a computational framework for the timescale decomposition of biochemical reaction networks to distill essential patterns from their intricate dynamics. This approach identifies timescale hierarchies, concentration pools, and coherent structures from time-series data, providing a system-level description of reaction networks at physiologically important timescales. We apply this technique to kinetic models of hypothetical and biological pathways, validating it by reproducing analytically characterized or previously known concentration pools of these pathways. Moreover, by analyzing the timescale hierarchy of the glycolytic pathway, we elucidate the connections between the stoichiometric and dissipative structures of reaction networks and the temporal organization of coherent structures. Specifically, we show that glycolysis is a cofactor-driven pathway, the slowest dynamics of which are described by a balance between high-energy phosphate bond and redox trafficking. Overall, this approach provides more biologically interpretable characterizations of network dynamics than large-scale kinetic models, thus facilitating model reduction and personalized medicine applications. IMPORTANCE Complex interactions within interconnected biochemical reaction networks enable cellular responses to a wide range of unpredictable environmental perturbations. Understanding how biological functions arise from these intricate interactions has been a long-standing problem in biology. Here, we introduce a computational approach to dissect complex biological systems' dynamics in evolving environments. This approach characterizes the timescale hierarchies of complex reaction networks, offering a system-level understanding at physiologically relevant timescales. Analyzing various hypothetical and biological pathways, we show how stoichiometric properties shape the way energy is dissipated throughout reaction networks. Notably, we establish that glycolysis operates as a cofactor-driven pathway, where the slowest dynamics are governed by a balance between high-energy phosphate bonds and redox trafficking. This approach enhances our understanding of network dynamics and facilitates the development of reduced-order kinetic models with biologically interpretable components. Complex interactions within interconnected biochemical reaction networks enable cellular responses to a wide range of unpredictable environmental perturbations. Understanding how biological functions arise from these intricate interactions has been a long-standing problem in biology. Here, we introduce a computational approach to dissect complex biological systems' dynamics in evolving environments. This approach characterizes the timescale hierarchies of complex reaction networks, offering a system-level understanding at physiologically relevant timescales. Analyzing various hypothetical and biological pathways, we show how stoichiometric properties shape the way energy is dissipated throughout reaction networks. Notably, we establish that glycolysis operates as a cofactor-driven pathway, where the slowest dynamics are governed by a balance between high-energy phosphate bonds and redox trafficking. This approach enhances our understanding of network dynamics and facilitates the development of reduced-order kinetic models with biologically interpretable components.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7c61f7e43334e2ee6ee31f41b307bf37cd86d3d" target='_blank'>A data-driven approach for timescale decomposition of biochemical reaction networks</a></td>
          <td>
            Amir Akbari, Zachary B. Haiman, Bernhard O. Palsson
          </td>
          <td>2024-01-23</td>
          <td>mSystems</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="We present a nonlinear programming (NLP) framework for the scalable solution of parameter estimation problems that arise in dynamic modeling of biological systems. Such problems are computationally challenging because they often involve highly nonlinear and stif differential equations as well as many experimental data sets and parameters. The proposed framework uses cutting-edge modeling and solution tools which are computationally efficient, robust, and easy-to-use. Specifically, our framework uses a time discretization approach that: i) avoids repetitive simulations of the dynamic model, ii) enables fully algebraic model implementations and computation of derivatives, and iii) enables the use of computationally efficient nonlinear interior point solvers that exploit sparse and structured linear algebra techniques. We demonstrate these capabilities by solving estimation problems for synthetic human gut microbiome community models. We show that an instance with 156 parameters, 144 differential equations, and 1,704 experimental data points can be solved in less than 3 minutes using our proposed framework (while an off-the-shelf simulation-based solution framework requires over 7 hours). We also create large instances to show that the proposed framework is scalable and can solve problems with up to 2,352 parameters, 2,304 differential equations, and 20,352 data points in less than 15 minutes. Competing methods reported in the computational biology literature cannot address problems of this level of complexity. The proposed framework is flexible, can be broadly applied to dynamic models of biological systems, and enables the implementation of sophisticated estimation techniques to quantify parameter uncertainty, to diagnose observability/uniqueness issues, to perform model selection, and to handle outliers. Author summary Constructing and validating dynamic models of biological systems spanning biomolecular networks to ecological systems is a challenging problem. Here we present a scalable computational framework to rapidly infer parameters in complex dynamic models of biological systems from large-scale experimental data. The framework was applied to infer parameters of a synthetic microbial community model from large-scale time series data. We also demonstrate that this framework can be used to analyze parameter uncertainty, to diagnose whether the experimental data are sufficient to uniquely determine the parameters, to determine the model that best describes the data, and to infer parameters in the face of data outliers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e7c847333832320468481208d42aad629f89aa" target='_blank'>Scalable nonlinear programming framework for parameter estimation in dynamic biological system models</a></td>
          <td>
            Sungho Shin, Ophelia S. Venturelli, V. Zavala
          </td>
          <td>2018-09-07</td>
          <td>PLoS Computational Biology</td>
          <td>17</td>
          <td>39</td>
        </tr>

        <tr id="The key processes in biological and chemical systems are described by networks of chemical reactions. From molecular biology to biotechnology applications, computational models of reaction networks are used extensively to elucidate their non-linear dynamics. The model dynamics are crucially dependent on the parameter values which are often estimated from observations. Over the past decade, the interest in parameter and state estimation in models of (bio-) chemical reaction networks (BRNs) grew considerably. The related inference problems are also encountered in many other tasks including model calibration, discrimination, identifiability, and checking, and optimum experiment design, sensitivity analysis, and bifurcation analysis. The aim of this review paper is to examine the developments in literature to understand what BRN models are commonly used, and for what inference tasks and inference methods. The initial collection of about 700 documents concerning estimation problems in BRNs excluding books and textbooks in computational biology and chemistry were screened to select over 270 research papers and 20 graduate research theses. The paper selection was facilitated by text mining scripts to automate the search for relevant keywords and terms. The outcomes are presented in tables revealing the levels of interest in different inference tasks and methods for given models in the literature as well as the research trends are uncovered. Our findings indicate that many combinations of models, tasks and methods are still relatively unexplored, and there are many new research opportunities to explore combinations that have not been considered—perhaps for good reasons. The most common models of BRNs in literature involve differential equations, Markov processes, mass action kinetics, and state space representations whereas the most common tasks are the parameter inference and model identification. The most common methods in literature are Bayesian analysis, Monte Carlo sampling strategies, and model fitting to data using evolutionary algorithms. The new research problems which cannot be directly deduced from the text mining data are also discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08ceffbbee9477c793d5defeea1b024ae83595a3" target='_blank'>Comprehensive Review of Models and Methods for Inferences in Bio-Chemical Reaction Networks</a></td>
          <td>
            P. Loskot, Komlan Atitey, L. Mihaylova
          </td>
          <td>2019-02-15</td>
          <td>Frontiers in Genetics</td>
          <td>42</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40b05f5afb7831d8956217893bac8bb1bd63c79e" target='_blank'>A comparison of approximation techniques for variance-based sensitivity analysis of biochemical reaction systems</a></td>
          <td>
            Hong-Xuan Zhang, J. Goutsias
          </td>
          <td>2010-05-12</td>
          <td>BMC Bioinformatics</td>
          <td>15</td>
          <td>29</td>
        </tr>

        <tr id="The extraction of models from data (in a sense, the “understanding” of the physical laws giving rise to the data) is a fundamental cognitive as well as scientific challenge. We show a geometric/analytic learning algorithm capable of creating minimal descriptions of parametrically dependent unknown nonlinear dynamical systems. This is accomplished by the data-driven discovery of useful intrinsic-state variables and parameters in terms of which one can empirically model the underlying dynamics. We discuss an informed observation geometry that enables us to formulate models without first principles as well as without closed form equations. (See pp. E7865–E7874.)">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2df6b9b2ee1bdef1834c343bb952182a7807bda" target='_blank'>PNAS Plus Significance Statements</a></td>
          <td>
            Ronald R Coifman, David A. Kessler, A. Goodkind
          </td>
          <td>2017-09-19</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>40</td>
          <td>13</td>
        </tr>

        <tr id="Motivation: Computational models of biological signalling networks, based on ordinary differential equations (ODEs), have generated many insights into cellular dynamics, but the model-building process typically requires estimating rate parameters based on experimentally observed concentrations. New proteomic methods can measure concentrations for all molecular species in a pathway; this creates a new opportunity to decompose the optimization of rate parameters. Results: In contrast with conventional parameter estimation methods that minimize the disagreement between simulated and observed concentrations, the SPEDRE method fits spline curves through observed concentration points, estimates derivatives and then matches the derivatives to the production and consumption of each species. This reformulation of the problem permits an extreme decomposition of the high-dimensional optimization into a product of low-dimensional factors, each factor enforcing the equality of one ODE at one time slice. Coarsely discretized solutions to the factors can be computed systematically. Then the discrete solutions are combined using loopy belief propagation, and refined using local optimization. SPEDRE has unique asymptotic behaviour with runtime polynomial in the number of molecules and timepoints, but exponential in the degree of the biochemical network. SPEDRE performance is comparatively evaluated on a novel model of Akt activation dynamics including redox-mediated inactivation of PTEN (phosphatase and tensin homologue). Availability and implementation: Web service, software and supplementary information are available at www.LtkLab.org/SPEDRE Supplementary information: Supplementary data are available at Bioinformatics online. Contact: LisaTK@nus.edu.sg">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89bb5bb8611e6a373a5968d7e5fdabc96d56a9c6" target='_blank'>Systematic parameter estimation in data-rich environments for cell signalling dynamics</a></td>
          <td>
            Tri Hieu Nim, Le Luo, M. Clément, Jacob K. White, L. Tucker-Kellogg
          </td>
          <td>2013-02-19</td>
          <td>Bioinformatics</td>
          <td>12</td>
          <td>53</td>
        </tr>

        <tr id="Mechanistic models have been used for centuries to describe complex interconnected processes, including biological ones. As the scope of these models has widened, so have their computational demands. This complexity can limit their suitability when running many simulations or when real-time results are required. Surrogate machine learning (ML) models can be used to approximate the behaviour of complex mechanistic models, and once built, their computational demands are several orders of magnitude lower. This paper provides an overview of the relevant literature, both from an applicability and a theoretical perspective. For the latter, the paper focuses on the design and training of the underlying ML models. Application-wise, we show how ML surrogates have been used to approximate different mechanistic models. We present a perspective on how these approaches can be applied to models representing biological processes with potential industrial applications (e.g., metabolism and whole-cell modelling) and show why surrogate ML models may hold the key to making the simulation of complex biological systems possible using a typical desktop computer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32a8d184439de9c319adbc70e1c96a7f22e6fd9a" target='_blank'>Bridging the gap between mechanistic biological models and machine learning surrogates</a></td>
          <td>
            Ioana M. Gherman, Z. Abdallah, Wei Pang, T. Gorochowski, C. Grierson, L. Marucci
          </td>
          <td>2023-04-01</td>
          <td>PLOS Computational Biology</td>
          <td>18</td>
          <td>33</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>