<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../IBD/">
      
      
        <link rel="next" href="../Cancer/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Covid
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Covid</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-21 10:42:03 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Covid</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Covid</a><br>
      <a href="#recommended_articles">3. Recommended articles on Covid</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Covid</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Covid</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="A quantitative systems pharmacology (QSP) model of the pathogenesis and treatment of SARS-CoV-2 infection can streamline and accelerate the development of novel medicines to treat COVID-19. Simulation of clinical trials allows in silico exploration of the uncertainties of clinical trial design and can rapidly inform their protocols. We previously published a preliminary model of the immune response to SARS-CoV-2 infection. To further our understanding of COVID-19 and treatment, we significantly updated the model by matching a curated dataset spanning viral load and immune responses in plasma and lung. We identified a population of parameter sets to generate heterogeneity in pathophysiology and treatment and tested this model against published reports from interventional SARS-CoV-2 targeting mAb and antiviral trials. Upon generation and selection of a virtual population, we match both the placebo and treated responses in viral load in these trials. We extended the model to predict the rate of hospitalization or death within a population. Via comparison of the in silico predictions with clinical data, we hypothesize that the immune response to virus is log-linear over a wide range of viral load. To validate this approach, we show the model matches a published subgroup analysis, sorted by baseline viral load, of patients treated with neutralizing Abs. By simulating intervention at different time points post infection, the model predicts efficacy is not sensitive to interventions within five days of symptom onset, but efficacy is dramatically reduced if more than five days pass post symptom onset prior to treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac952ba3508fc13b6ca5e8d7e11d7a7086f9b72d" target='_blank'>
                A quantitative systems pharmacology model of the pathophysiology and treatment of COVID-19 predicts optimal timing of pharmacological interventions
                </a>
              </td>
          <td>
            Rohit Rao, C. Musante, R. Allen
          </td>
          <td>2021-12-08</td>
          <td>medRxiv, NPJ Systems Biology and Applications</td>
          <td>8</td>
          <td>18</td>

            <td><a href='../recommendations/ac952ba3508fc13b6ca5e8d7e11d7a7086f9b72d' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e2719064be04fd488be75a358c40fc717730faf" target='_blank'>
                Modelling the COVID-19 epidemic and implementation of population-wide interventions in Italy
                </a>
              </td>
          <td>
            G. Giordano, F. Blanchini, R. Bruno, P. Colaneri, A. Di Filippo, A. Di Matteo, M. Colaneri
          </td>
          <td>2020-03-22</td>
          <td>Nature Network Boston, Nature Medicine</td>
          <td>1581</td>
          <td>55</td>

            <td><a href='../recommendations/7e2719064be04fd488be75a358c40fc717730faf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Since the emergence of the first cases in Wuhan, China, the novel coronavirus (2019-nCoV) infection has been quickly spreading out to other provinces and neighboring countries. Estimation of the basic reproduction number by means of mathematical modeling can be helpful for determining the potential and severity of an outbreak and providing critical information for identifying the type of disease interventions and intensity. A deterministic compartmental model was devised based on the clinical progression of the disease, epidemiological status of the individuals, and intervention measures. The estimations based on likelihood and model analysis show that the control reproduction number may be as high as 6.47 (95% CI 5.71–7.23). Sensitivity analyses show that interventions, such as intensive contact tracing followed by quarantine and isolation, can effectively reduce the control reproduction number and transmission risk, with the effect of travel restriction adopted by Wuhan on 2019-nCoV infection in Beijing being almost equivalent to increasing quarantine by a 100 thousand baseline value. It is essential to assess how the expensive, resource-intensive measures implemented by the Chinese authorities can contribute to the prevention and control of the 2019-nCoV infection, and how long they should be maintained. Under the most restrictive measures, the outbreak is expected to peak within two weeks (since 23 January 2020) with a significant low peak value. With travel restriction (no imported exposed individuals to Beijing), the number of infected individuals in seven days will decrease by 91.14% in Beijing, compared with the scenario of no travel restriction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a74cec58b08636c930fc8d9d12dad6471ae1cc29" target='_blank'>
                Estimation of the Transmission Risk of the 2019-nCoV and Its Implication for Public Health Interventions
                </a>
              </td>
          <td>
            B. Tang, Xia Wang, Qian Li, N. Bragazzi, Sanyi Tang, Yanni Xiao, Jianhong Wu
          </td>
          <td>2020-02-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>1312</td>
          <td>71</td>

            <td><a href='../recommendations/a74cec58b08636c930fc8d9d12dad6471ae1cc29' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d71eda036617bbbfc9d6b8b916d94aa4de9a705" target='_blank'>
                Why is it difficult to accurately predict the COVID-19 epidemic?
                </a>
              </td>
          <td>
            W. Roda, M. Varughese, Donglin Han, Michael Y. Li
          </td>
          <td>2020-03-25</td>
          <td>Infectious Disease Modelling</td>
          <td>557</td>
          <td>35</td>

            <td><a href='../recommendations/4d71eda036617bbbfc9d6b8b916d94aa4de9a705' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2cd248811b0d9bdcfeb250e86518d19e3b08239" target='_blank'>
                An updated estimation of the risk of transmission of the novel coronavirus (2019-nCov)
                </a>
              </td>
          <td>
            B. Tang, N. Bragazzi, Qian Li, Sanyi Tang, Yanni Xiao, Jianhong Wu
          </td>
          <td>2020-02-11</td>
          <td>Infectious Disease Modelling</td>
          <td>701</td>
          <td>71</td>

            <td><a href='../recommendations/c2cd248811b0d9bdcfeb250e86518d19e3b08239' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Covid'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Covid</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/521d4a775314528e820191f84f21e703df5d6f81" target='_blank'>
              Modeling COVID-19 dynamics in the Basque Country: characterizing population immunity profile from 2020 to 2022
              </a>
            </td>
          <td>
            Bechir Naffeti, Zeineb Ounissi, A. K. Srivastav, N. Stollenwerk, Joseba Bidaurrazaga Van-Dierdonck, M. Aguiar
          </td>
          <td>2025-01-02</td>
          <td>BMC Infectious Diseases</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The COVID-19 pandemic has highlighted the crucial role of health sector decision-makers in establishing and evaluating effective treatment and prevention policies. To inform sound decisions, it is essential to simultaneously monitor multiple pandemic characteristics, including transmission rates, infection rates, recovery rates (which indicate treatment efficacy), and fatality rates. This study introduces an innovative application of existing methodologies: the Multivariate Exponentially Weighted Moving Average (MEWMA) and Multivariate Cumulative Sum (MCUSUM) control charts (CCs), used for monitoring the parameters of the Susceptible, Exposed, Infected, Recovered, Death, and Vaccination (SEIRDV) model. The methodology is applied to COVID-19 data from the State of Qatar, offering new insights into the pandemic’s dynamics. By monitoring changes in the model parameters, this study aims to assess the effectiveness of interventions and track the impact of emerging variants. The results underscore the practical utility of these methodologies for decision-making during similar pandemics. Additionally, this study employs an augmented particle Markov chain Monte Carlo scheme that enables real-time monitoring of SEIRDV model parameters, offering improved estimation accuracy and robustness compared to traditional approaches. The results demonstrate that MEWMA and MCUSUM charts are effective tools for monitoring SEIRDV model parameters and can support decision-making in any similar pandemic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0111b671ddde2006b8670201c09fbfb282162f99" target='_blank'>
              Multivariate Techniques for Monitoring Susceptible, Exposed, Infected, Recovered, Death, and Vaccination Model Parameters for the COVID-19 Pandemic for Qatar
              </a>
            </td>
          <td>
            Abdel-Salam G. Abdel-Salam, Edward L. Boone, R. Ghanam
          </td>
          <td>2024-11-27</td>
          <td>International Journal of Environmental Research and Public Health</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="This study used a non-linear SEIR model to analyze the spread of COVID-19 in Malaysia. The model considered vaccines, booster doses, hospitalized cases, and various infectious cases. The model was fitted to real data using MATLAB's ode solver and the particle swarm optimization method. The results revealed that booster vaccines significantly reduced the total number of infected cases by 62% when 70% of the population received them. This model can help policymakers to make crucial decisions to prevent future outbreaks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4ffb074ba169a8f98dcae5f488174260070e455" target='_blank'>
              The Impact of Booster Vaccine on the Spread Of COVID-19 Using an Extended SEIR Model
              </a>
            </td>
          <td>
            Sadia Sristy, Nor Hamzah, Muhamad Aziz
          </td>
          <td>2024-11-30</td>
          <td>Proceedings of the Bulgarian Academy of Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In the early stage of an infectious disease outbreak, public health strategies tend to gravitate towards non-pharmaceutical interventions (NPIs) given the time required to develop targeted treatments and vaccines. One of the most common NPIs is Test-Trace-Isolate (TTI). One of the factors determining the effectiveness of TTI is the ability to identify contacts of infected individuals. In this study, we propose a multi-layer temporal contact network to model transmission dynamics and assess the impact of different TTI implementations, using SARS-CoV-2 as a case study. The model was used to evaluate TTI effectiveness both in containing an outbreak and mitigating the impact of an epidemic. We estimated that a TTI strategy based on home isolation and testing of both primary and secondary contacts can contain outbreaks only when the reproduction number is up to 1.3, at which the epidemic prevention potential is 88.2% (95% CI: 87.9%-88.5%). On the other hand, for higher value of the reproduction number, TTI is estimated to noticeably mitigate disease burden but at high social costs (e.g., over a month in isolation/quarantine per person for reproduction numbers of 1.7 or higher). We estimated that strategies considering quarantine of contacts have a larger epidemic prevention potential than strategies that either avoid tracing contacts or require contacts to be tested before isolation. Combining TTI with other social distancing measures can improve the likelihood of successfully containing an outbreak but the estimated epidemic prevention potential remains lower than 50% for reproduction numbers higher than 2.1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3921f5ce38f94d6168eda925012c7a87f0c5f10" target='_blank'>
              Assessing the effectiveness of test-trace-isolate interventions using a multi-layered temporal network
              </a>
            </td>
          <td>
            Yunyi Cai, Weiyi Wang, Lanlan Yu, Ruixiang Wang, Gui-Quan Sun, A. G. Kummer, Paulo C. Ventura, Jiancheng Lv, M. Ajelli, Quan-Hui Liu
          </td>
          <td>2024-12-18</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="This study examines the effect of a partially protective vaccine on COVID-19 infection with the original and mutant virus with the help of a deterministic mathematical model developed. The model we developed consists of five compartments and thirteen parameters. The model consists of S (susceptible), V (vaccinated), I_1 (infected with original virus), I_2 (infected with mutant virus) and R (recover) subcompartments. With the established model, imperfect vaccines and mutant virus for the COVID-19 pandemic in Turkey were examined. We examined the effect of both artificial active immunity (vaccinated) and natural active immunity (passing disease) in the model. Since it is known that the recovery and death rates of the original virus and the mutant virus are different in COVID-19, we considered it in the study. We performed local stability and global stability analysis by calculating the disease-free equilibrium point and endemic equilibrium point of the model. We also obtained the basic reproduction number with the help of the next generation matrix method. We estimated three model parameters with parameter estimation and identified model-sensitive parameters with local sensitivity analysis. Next, we showed the existence of a backward bifurcation using the Castillo-Chavez and Song Bifurcation Theorem. Additionally, to illustrate the underlying fundamental mechanisms of the model dynamics and to support the analytical findings, we presented three distinct simulations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a051dab63819892023151c62b3019bcb72c2f35c" target='_blank'>
              Mathematical model of COVID-19 with imperfect vaccine and virus mutation
              </a>
            </td>
          <td>
            Ceren Gurbuz Can, Sebaheddin cSevgin
          </td>
          <td>2024-12-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Purpose: This research is about a new COVID-19 SIR model containing three classes; susceptible S(t), infected I(t), and recovered R(t) with the convex incident rate. 
Methodology: The NCOVID-19 model was formulated in the following system, the whole population N(t) was divided into three classes S(t), I(t), and R(t), which represented Susceptible, Infected, and Recovered compartments in the form of differential equations. Lyapunov functions were used to validate the stability of the equilibrium of the ordinary differential equations, linearization of the system was also done using Jacobian matrices by finding the derivatives of f(x) for x.   
Findings: Covid-19 is an infectious disease caused by the novel coronavirus identified as Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). The people infected by COVID-19 experience mild respiratory problems such as; Fever, dry cough, throat infection, and fatigue. People may also have symptoms such as nasal infection, aches, and sore throat. The pandemic has led to a dramatic loss of human life in Kenya, Africa, and the whole world as it presents an unprecedented challenge to public health, food systems, and the world of work.  This case study seeks to model covid-19 virus after lifting preventive measures with a major focus on Kisii County, the subject model was presented in the form of differential equations and the disease-free and endemic equilibrium was calculated for the model. Also, the basic reproduction number R0 = 0.7831 was calculated and the disease-free equilibrium was found to be asymptotically stable meaning that the virus could be eliminated from the population, this showed that the county government of Kisii was in good control of the COVID-19 situation., in addition, The global stability of the model was calculated using the Lyapunov function construction while the Local stability was calculated using the Jacobian matrices. The numerical solutions were calculated using the non-standard finite difference scheme (NFDS) and MATLAB software. 
Unique Contribution to Theory, Practice and Policy: This study has laid a foundation for future research in the area. In the future, a study that can include the rate of COVID-19 virus mutation and its impacts is recommended.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce9835f62721bfe92da0a6d7fda096fb1c6ebff" target='_blank'>
              Modeling Covid-19 Virus after Lifting Preventive Measures: A Case Study of Kisii County
              </a>
            </td>
          <td>
            Michael Kamweti, Winifred Mutuku, Paul Wanjau
          </td>
          <td>2024-12-03</td>
          <td>International Journal of Natural Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="During the SARS-CoV-2 pandemic, the effective reproduction number (R-eff) has frequently been used to describe the course of the pandemic. Analytical properties of R-eff are rarely studied. We analytically examine how and under which conditions the conventional susceptible–infected–removed (SIR) model (without infection age) serves as an approximation to the infection-age-structured SIR model. Special emphasis is given to the role of R-eff, which is an implicit parameter in the infection-age-structured SIR model and an explicit parameter in the approximation. The analytical findings are illustrated by a simulation study about an hypothetical intervention during a SARS-CoV-2 outbreak and by historical data from an influenza outbreak in Prussian army camps in the region of Arnsberg (Germany), 1918–1919.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1047f62343d7a3b5b742c03357fece792b6d7792" target='_blank'>
              Approximation of the infection-age-structured SIR model by the conventional SIR model of infectious disease epidemiology
              </a>
            </td>
          <td>
            Ralph Brinks, A. Hoyer
          </td>
          <td>2024-12-17</td>
          <td>Frontiers in Epidemiology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68d0fe4ee28c1422d8f92db8edba91c81566aec6" target='_blank'>
              Transmissibility and control of tuberculosis in school outbreaks: a modeling study based on four outbreaks in China
              </a>
            </td>
          <td>
            Qiao Liu, Meng Yang, Qiuping Chen, Chan Liu, Yue He, L. Gavotte, Ze-yu Zhao, Yan-hua Su, Roger Frutos, Kaiwei Luo, Tianmu Chen
          </td>
          <td>2024-11-27</td>
          <td>BMC Infectious Diseases</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="From the Black Death to SARS to influenza A (H1N1), communicable diseases have always been among the uppermost problem endangering human health and have caused crises all over the world countless times. Practical relevance of a thorough knowledge of the transmission pathways and related mechanisms of infectious diseases are complex. Investigating the laws of infectious illness transmission mostly depends on the building of dynamic models. Designing corresponding models based on specific transmission mechanisms can more realistically depict the transmission dynamics of infectious diseases and provide higher accuracy for parameter estimation. This study suggested an enhanced SEIR model and performed a comprehensive investigation of its equilibrium point and stability, acknowledging that individuals infected with the novel coronavirus are contagious in time of their incubation period. Consequently, an empirical investigation was undertaken utilizing this model to analyze the overall trend of the epidemic in China and the outbreak aboard the Diamond Princess cruise ship. Key model parameters, such as the infection rate coefficient, control coefficient, basic reproduction number, and the particular time at which the effective reproduction number declines to 1, were ascertained by the maximum likelihood estimation approach. The cumulative confirmed case fitting curves for the national pandemic in China and the Diamond Princess outbreak were presented concurrently to further validate the model's efficacy">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26d24990cca8d35f34dadd9d197a49fd3d8b5a60" target='_blank'>
              The Study of COVID-19 Virus Infection Based on the SEIR Model
              </a>
            </td>
          <td>
            Fan Gao
          </td>
          <td>2025-01-13</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The current situation of COVID-19 measures makes it difficult to accurately assess the prevalence of SARS-CoV-2 due to a decrease in reporting rates, leading to missed initial transmission events and subsequent outbreaks. There is growing recognition that wastewater virus data assist in estimating potential infections, including asymptomatic and unreported infections. Understanding the COVID-19 situation hidden behind the reported cases is critical for decision-making when choosing appropriate social intervention measures. However, current models implicitly assume homogeneity in human behaviour, such as virus shedding patterns within the population, making it challenging to predict the emergence of new variants due to variant-specific transmission or shedding parameters. This can result in predictions with considerable uncertainty. In this study, we established a state-space model based on wastewater viral load to predict both reported cases and potential infection numbers. Our model using wastewater virus data showed high goodness-of-fit to COVID-19 case numbers despite the dataset including waves of two distinct variants. Furthermore, the model successfully provided estimates of potential infection, reflecting the superspreading nature of SARS-CoV-2 transmission. This study supports the notion that wastewater surveillance and state-space modelling have the potential to effectively predict both reported cases and potential infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5232d775c78201a50b0460e68ccc225fae5b9300" target='_blank'>
              State-space modelling using wastewater virus and epidemiological data to estimate reported COVID-19 cases and the potential infection numbers.
              </a>
            </td>
          <td>
            Syun-suke Kadoya, Yubing Li, Yilei Wang, Hiroyuki Katayama, Daisuke Sano
          </td>
          <td>2025-01-01</td>
          <td>Journal of the Royal Society, Interface</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The COVID-19 pandemic highlighted the need to improve the modeling, estimation, and prediction of how infectious diseases spread. SEIR-like models have been particularly successful in providing accurate short-term predictions. This study fills a notable literature gap by exploring the following question: Is it possible to incorporate a nonparametric susceptible-exposed-infected-removed (SEIR) COVID-19 model into the inverse-problem regularization framework when the transmission coefficient varies over time? Our positive response considers varying degrees of disease severity, vaccination, and other time-dependent parameters. In addition, we demonstrate the continuity, differentiability, and injectivity of the operator that link the transmission parameter to the observed infection numbers. By employing Tikhonov-type regularization to the corresponding inverse problem, we establish the existence and stability of regularized solutions. Numerical examples using both synthetic and real data illustrate the model's estimation accuracy and its ability to fit the data effectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23cd97f0c273d8a742128750b1758b89b841edfe" target='_blank'>
              On the Estimation of the Time-Dependent Transmission Rate in Epidemiological Models
              </a>
            </td>
          <td>
            J. Zubelli, J. Loria, V. Albani
          </td>
          <td>2024-12-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="As SARS-CoV-2 has transitioned from a novel pandemic-causing pathogen into an established seasonal respiratory virus, focus has shifted to post-acute sequelae of COVID-19 (PASC, colloquially ‘long COVID’). We use compartmental mathematical models simulating emergence of new variants to help identify key sources of uncertainty in PASC trajectories. Some parameters (such as the duration and equilibrium prevalence of infection, as well as the fraction of infections that develop PASC) matter more than others (such as the duration of immunity and secondary vaccine efficacy against PASC). Even if newer variants carry the same risk of PASC as older types, the dynamics of selection can give rise to greater PASC prevalence. However, identifying plausible PASC prevalence trajectories requires accurate knowledge of the transmission potential of COVID-19 variants in the endemic phase. Precise estimates for secondary vaccine efficacy and duration of immunity will not greatly improve forecasts for PASC prevalence. Researchers involved with Living Evidence Synthesis, or other similar initiatives focused on PASC, are well advised to ascertain primary efficacy against infection, duration of infection and prevalence of active infection in order to facilitate predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c8d312f12194ae9ef319c5ce75edd9a2c02ed54" target='_blank'>
              Uncertainty in COVID-19 transmission could undermine our ability to predict long COVID
              </a>
            </td>
          <td>
            Alexander B. Beams, D. Earn, Caroline Colijn
          </td>
          <td>2024-12-01</td>
          <td>Journal of the Royal Society Interface</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Given the significant impact of the more than three-year-long COVID-19 pandemic on people’s health, social order, and economic performance, as well as the potential re-emergence of a new variant and the epidemic “Disease X,” it is crucial to examine its developmental trends and suggest countermeasures to address community epidemics of severe respiratory infectious diseases. Methods The epidemiological characterization of various strains of COVID-19 was modeled using an improved Susceptible-Exposed-Infectious-Recovered (SEIR) model to simulate the infections of different strains of COVID-19 under different scenarios, taking as an example an urban area of a prefecture-level city in Shandong Province, China, with a resident population of 2 million. Scenarios 1–5 are scenario-based simulations the Omicron strain, and 6–8 simulate the original COVID-19 strain, with different parameters for each scenario. Scenarios 1 and 6 do not consider community NPIs and represent natural epidemic scenarios. Scenarios 2–4 assess the impact of different NPIs on the original COVID-19 strain. Scenarios 1–4 and 6–8 compare the effects of the same measures on different strains. Scenario 5 simulates the effects of implementing NPIs after an outbreak has spread widely. Compare scenarios 4 and 9 to analyze the effect of high grades versus dynamic clearing of NPIs. By analyzing the time at which the peak number of cases was reached and the maximum number of cases, we were able to calculate the effectiveness of urban community control measures (NPIs) and the impact of vaccination on disease trends. Based on our research into the degree of restriction of social activities in different levels of control areas during real-world epidemics, we categorized the NPIs into three levels, with controls becoming increasingly stringent from levels 1 to 3 as low-, medium-, and high-risk areas are, respectively, controlled. Results In simulation scenarios 1–5 and 9, where the epidemic strain is Omicron and the susceptible population receives three doses of vaccine, it was found that the real-time peak number of cases in scenario 2, which implemented level 1 controls, was reduced by 18.19%, and in scenario 3, which implemented level 2 controls, it was reduced by 38.94%, compared with scenario 1, where no control measures were taken. Level 1 and level 2 controls do not block transmission but significantly reduce peak incidence and delay the peak time. In scenario 5, even with a high number of initial cases, the implementation of level 3 controls can still control the outbreak quickly, but it requires a longer period of time. However, Omicron has a low rate of severe illness, and the existing beds in City A could largely cope even if the control measures had not been implemented. Analyzing scenarios 4 and 9, level 3 community control and dynamic zeroing of the three zones were similarly successful in interrupting the spread of the epidemic. In simulation scenarios 6–8, where the prevalent strain was the original COVID-19 strain, only level 3 community control was able to rapidly extinguish the outbreak. Unchecked, the outbreak is severe, characterized by high peaks and substantial medical stress. Although level 2 controls reduced real-time incidence and peak new infections by 39.81 and 61.33%, and delayed the peaks by 55 and 52 days, respectively, the high rate of severe illnesses may still overwhelm the medical system. Conclusion Control effects are related to the level, timing and virus characteristics. Level 3 and dynamic zeroing measures can interrupt community transmission in the early stages of an outbreak. During a pandemic, different NPIs must be implemented, considering the virus’s status and cost of control, and ensuring that medical resources are sufficient to maintain medical order.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65fdb948aeeaf8b557503abf03b9bf9bf5a03efc" target='_blank'>
              Effectiveness of a scenario-based, community-based intervention in containing COVID-19 in China
              </a>
            </td>
          <td>
            Binghang Li, Yalin Zhou, Ting Zhang, Anning Ma, Wenhao Hao
          </td>
          <td>2024-11-27</td>
          <td>Frontiers in Public Health</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The outbreak of "pneumonia of unknown etiology" marked the beginning of a new era in global health care and public life. The disease quickly spread throughout the world, turning into a pandemic that posed a serious threat to the international community. Ukraine, like many other countries, has been seriously affected by the consequences of this pandemic. Effective prediction of the spread of COVID-19 is a key task for informing health management and making strategic decisions to minimize the consequences of the pandemic. Many studies have been conducted using different methods and approaches to predict the spread of COVID-19. However, most of these methods do not adequately take into account all aspects that may affect the spread of the disease. This paper analyzes the known methods of forecasting COVID-19, evaluates the impact of various arguments on the spread of the disease, and develops a mathematical model for forecasting. For this, data on the spread of COVID-19 in Ukraine was used.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aac6b961533c30c424b00202647d7f8b5a862271" target='_blank'>
              CREATION AND RESEARCH OF FORECASTING MODELS OF THE SPREAD AND DEMOGRAPHIC CONSEQUENCES OF COVID-19
              </a>
            </td>
          <td>
            Olena Skakalina
          </td>
          <td>2024-11-28</td>
          <td>Системи управління, навігації та зв’язку. Збірник наукових праць</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="COVID-19 is an infectious disease primarily transmitted to individuals through direct contact with respiratory droplets. The infection, caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), continues to spread globally infecting around 776 million confirmed cases, including over 7 million deaths. Meanwhile, dengue is a vector-borne disease caused by the Flaviviridae virus and is transmitted through bites from female mosquitoes, primarily Aedes aegypti and Aedes albopictus. It is estimated that 390 million dengue virus infections occur per year caused by four distinct virus serotypes–DENV-1, DENV-2, DENV-3, and DENV-4. The COVID-19 pandemic has further strained public health systems, particularly in tropical and subtropical regions where dengue is endemic. The overlapping presence of these infectious diseases heightens the risk of co-infection, posing additional diagnostic and treatment challenges. Co-infection of COVID-19 and dengue cases were already reported and confirmed in several countries. In this study, an 11-compartmentalized deterministic mathematical model was developed to understand the transmission dynamics of COVID-19 and dengue co-infection. This modeling approach was described by a system of ordinary differential equations (ODEs), examining disease progression over time, offering insights into potential co-infection scenarios andcontrol strategies to help guide public health interventions. The well-posedness of the model was verified, ensuring the existence and uniqueness of its solutions based on continuity, local Lipschitz conditions, and invariance over a compact feasible region. The basic reproduction number (R0), a significant indicator of disease transmission, was calculated using the Next Generation Method (NGM). Four equilibrium points were identified: the disease-free, COVID-19-only, dengue-only, and COVID-19-dengue co-infection equilibrium points. Threshold values of the basic reproduction number were calculated to establish the conditions for the existence and stability of the equilibrium points. These equilibrium points and threshold values provide critical insight into the conditions necessary for eradicating or controlling each disease, serving as a guide for developing interventions during different stages of an epidemic or pandemic. Furthermore, a phase diagram of two parameters sensitive to R0 (COVID-19 transmission βc and dengue vector-to-human transmission Cvh) was established which presented six distinct regions of existence and stability states of the equilibrium points. These regions described different stable epidemiological scenarios whenever the parameter values were varied. Numerical simulations were conducted to verify the stability results and to analyze the effects of varied parameter values on the model solution. The simulations illustrated the positive impacts of reducing the recovery period on the spread of infections even with increasing transmission rates. This demonstrates the effectiveness of timely interventions, such as accelerated recovery through early diagnosis and treatment, in mitigating the severity of outbreaks. All the algebraic calculations, analysis, and numerical simulations were conducted with the aid of MATLAB R2023b and Maple software.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0534425c77422f99071018357bcfac705974c8f1" target='_blank'>
              Modeling the Co-Infection Dynamics of COVID-19 and Dengue: Well-posedness, Analysis of Equilibrium Properties and Numerical Simulations
              </a>
            </td>
          <td>
            Wyeth Ian C. Blase, A. J. Balsomo, S. G. Sabinay
          </td>
          <td>2024-12-18</td>
          <td>Communication in Biomathematical Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The recurrence of epidemic waves has been a hallmark of infectious disease outbreaks. Repeated surges in infections pose significant challenges to public health systems, yet the mechanisms that drive these waves remain insufficiently understood. Most prior models attribute epidemic waves to exogenous factors, such as transmission seasonality, viral mutations, or implementation of public health interventions. We show that epidemic waves can emerge autonomously from the feedback loop between infection dynamics and human behavior. Our results are based on a behavioral framework in which individuals continuously adjust their level of risk mitigation subject to their perceived risk of infection, which depends on information availability and disease severity. We show that delayed behavioral responses alone can lead to the emergence of multiple epidemic waves. The magnitude and frequency of these waves depend on the interplay between behavioral factors (delay, severity, and sensitivity of responses) and disease factors (transmission and recovery rates). Notably, if the response is either too prompt or excessively delayed, multiple waves cannot emerge. Our results further align with previous observations that adaptive human behavior can produce non-monotonic final epidemic sizes, shaped by the trade-offs between various biological and behavioral factors--namely, risk sensitivity, response stringency, and disease generation time. Interestingly, we found that the minimal final epidemic size occurs on regimes that exhibit a few damped oscillations. Altogether, our results emphasize the importance of integrating social and operational factors into infectious disease models, in order to capture the joint evolution of adaptive behavioral responses and epidemic dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea83bc0a54f8cd80c7c02559cc1162d09fb1cf77" target='_blank'>
              Adaptive human behavior and delays in information availability autonomously modulate epidemic waves
              </a>
            </td>
          <td>
            M. S. Mahmud, S. Eshun, B. Espinoza, C. Kadelka
          </td>
          <td>2024-11-26</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In this work, we introduce a new SEVIR epidemic model to analyze the dynamics of infectious diseases by incorporating the key compartments such as susceptible, exposed, vaccinated, infected, and recovered populations. We add the vaccination compartment in the current model and calculate the consequences concerning the disease spreading; it is significantly relevant at the present day considering the general policies of worldwide vaccinations. For a stringent assessment of stability, it should involve derivation of a basic reproduction number, as this parameter characterizes whether an infection will be spread at all. Our results demonstrate that when we have R0 < 1, the disease-free equilibrium is locally asymptotically stable, which implies that an infection will eventually die out. We also identify two distinguished disease-dependent equilibrium points for the first time, giving much deeper insight into long term behavior of the disease. Numerical simulations show the efficacy of vaccination toward reducing the newly infected individuals and suggest the model can predict stabilization for all compartments over time. The exposed and recovered population increases and stabilizes as the susceptible, infected, and vaccinated populations decline. In this regard, these are valuable insights into the progression of epidemics and put emphasis on vaccination programs. Our model provides a perspective on disease control by taking into account the interplay between vaccination and infection dynamics. It can be very useful for predicting future outbreaks and can guide public health policy. Future work will extend the model by adding quarantine measures to further hone our understanding of disease transmission. ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/115601009af6a1c5b7a6d9863670bb5f300e0d82" target='_blank'>
              Essential Analysis of New SEVIR Model: A Five Case Epidemic Model
              </a>
            </td>
          <td>
            Kalpana Umapathy, Balaganesan Palanivelu, Prasantha Bharathi Dhandapani, Udhayakumar Ramalingam
          </td>
          <td>2024-12-03</td>
          <td>Contemporary Mathematics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c290c18283aed288fa071ab877e79e316e669e8c" target='_blank'>
              Unveiling pandemic patterns: a detailed analysis of transmission and severity parameters across four COVID-19 waves in Bogotá, Colombia
              </a>
            </td>
          <td>
            David Santiago Quevedo, Nicolás T. Domínguez, Diego Fernando Perez, Maria Alejandra Cabrera Polanía, Juan David Serrano Medina, Felipe Segundo Abril-Bermúdez, Diane Moyano Romero, Diana Sofia Rios Oliveros, Manuel Alfredo González Mayorga, Charles Whittaker, Zulma M. Cucunubá
          </td>
          <td>2024-12-10</td>
          <td>BMC Global and Public Health</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The limited availability of COVID-19 vaccines has prompted extensive research on optimal vaccination strategies. Previous studies have considered various non-pharmaceutical interventions, vaccine efficacy and distribution strategies. In this work, we address the combined effects of inter-group contacts and vaccination rates under contact reduction, analysing the Spanish population’s demographic and age group contact patterns and incorporating reinfection dynamics. We conduct an exhaustive analysis, evaluating 362 880 permutations of nine age groups across six vaccination rates and two distinct, empirically quantified scenarios for social contacts. Our results show that at intermediate-to-high vaccination rates with unrestricted social contacts, optimal age-based vaccination strategies only slightly deviate from older-to-younger prioritization, yielding marginal reductions in deaths and infections. However, when significant reductions in social contacts are enforced—similar to the lockdowns in 2020—there are substantial improvements, particularly at moderate vaccination rates. These restrictions lead to a transition where infection propagation is halted, a scenario that became achievable during the pandemic with the observed vaccination rates. Our findings emphasize the importance of combining appropriate social contact reductions with vaccination to optimize age-based vaccination strategies, underscoring the complex, nonlinear dynamics involved in pandemic dynamics and the necessity for tailored context-specific interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c32e25389219f04d668057e2bdaaec1961e20b9" target='_blank'>
              Optimal COVID-19 vaccine prioritization by age depends critically on inter-group contacts and vaccination rates
              </a>
            </td>
          <td>
            I. Atienza-Diez, Gabriel Rodriguez-Maroto, Saúl Ares, Susanna Manrubia
          </td>
          <td>2024-12-01</td>
          <td>Royal Society Open Science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In December 2019, the world faced a new challenge — an outbreak of the disease caused by the new SARS-CoV-2 coronavirus. On March 11, 2020, WHO declared the COVID-19 pandemic. In order to overcome it, previously tested strategies and measures of prevention and control were implemented, as well as the latest digital approaches, new political, diplomatic and socio-economic solutions were also applied, which, together with classical approaches, made it possible to finally overcome the spread of SARS-CoV-2 and the consequences of the COVID-19 pandemic.
Objective — to investigate the development of the pandemic of the acute respiratory disease COVID-19 caused by the new coronavirus SARS-CoV-2 in Ukraine and the organization of epidemiologic surveillance for COVID-19 in the country.
Materials and methods. An observational population study was conducted. Epidemiological, medical-statistical and bibliosemantic methods were used.
Results and discussion. The weekly morbidity rate varied from 8.28 to 371.39, mortality — from 0.26 to 11.19, lethality — from 1.29 to 5.93 %. The rate of increase/decrease of confirmed cases ranged from —60.92 to 101.21 %, deaths — from —53.66 to 131.25 %. In the age structure of those infected with COVID-19, the largest share is people aged 50—59 years old (18.3 %), among the dead — people > 70 years old (55.64 %). The highest mortality rate in the age group of ≥ 70 years old is 9.78 %. The morbidity rate by region ranged from 1.93 to 753.08, the mortality rate from 0.06 to 25.3. The Electronic Integrated Surveillance System for Infectious Diseases was launched in May 2020 to monitor COVID-19, and data exchange with the Electronic Health Care System was subsequently implemented.
Conclusions. The development and implementation of effective strategies to combat new infectious diseases, including preparedness and response plans, allow you to prepare for future epidemics and pandemics, an important role in this is played by data analysis, in particular the systematic collection, processing and interpretation of data, in order to assess the current situation and forecast and determine the effectiveness measures taken.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9331edd2643cec161aa1a5ef76d7fda173fdd55c" target='_blank'>
              Characteristics of The Development of the COVID-19 Pandemic in Ukraine and Peculiarities of Epidemiologic Surveillance
              </a>
            </td>
          <td>
            V. V. Korolenko, H. A. Mokhort, T.V. Petrusevych, V. V. Korolenko
          </td>
          <td>2024-12-05</td>
          <td>Tuberculosis, Lung Diseases, HIV Infection</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Objectives To understand the epidemic characteristics of various SARS-CoV-2 variants, we mainly focus on analyzing general epidemic profiles, viral mutation, and evolution of COVID-19 outbreaks caused by different SARS-CoV-2 variants of concern (VOCs) in China as of August 2022. Methods We systematically sorted out the general epidemic profiles of outbreaks caused by various SARS-CoV-2 VOCs in China, compared the differences of outbreaks caused by Delta and Omicron VOCs, and analyzed the mutational changes of subvariants between the same outbreak and different outbreaks. Findings By 15 August 2022, a total of 2, 33, and 124 COVID-19 outbreaks caused by Alpha, Delta, and Omicron VOCs, respectively, were reported in different regions of China. In terms of the number of outbreaks, the extent of affected areas, and the total number of confirmed cases, Omicron VOCs were more widespread than the other variants. The most frequently circulating PANGO lineages in China were B.1.617.2 and AY.122 in Delta VOCs, and BA.2.2.1, BA.2, BA.2.2, and BA.5 for Omicron VOCs. Additional mutations in the genome of the SARS-CoV-2 strain were frequently observed in outbreaks with longer duration and higher numbers of infections. Conclusion Through the comprehensive analysis of the COVID-19 outbreaks, the influences, and the evolution of the SARS-CoV-2 variants in China, we found differences between outbreaks caused by Delta and Omicron VOCs. The genome of SARS-CoV-2 continued to evolve within the same outbreak and across outbreaks occurring in different locations or at different times. These findings suggest that rapidly containing an Omicron virus outbreak can not only reduce the spread of the virus but also delay the virus’s mutation frequency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0037b93ba99a2840b17e750fba95ef37bf7ca1e" target='_blank'>
              COVID-19 outbreaks caused by different SARS-CoV-2 variants: a descriptive, comparative study from China
              </a>
            </td>
          <td>
            Cao Chen, Yenan Feng, Zeyuan Yin, Mingfan Pang, Qiang Shi, Xuejun Ma, X. Dong
          </td>
          <td>2024-12-12</td>
          <td>Frontiers in Public Health</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="COVID-19 a novel Corona Virus Disease which was caused by SARS-CoV-2 (Severe Acute Respiratory Syndrome Corona Virus 2) continues to pose a critical and urgent threat to global health. When an infected person comes in contact with a normal individual or when the infected person sneezes or coughs, the virus that triggers COVID-19 spreads.The spread of novel SARS-CoV-2 was increasing, and the threats caused by it were becoming more severe in 2021. To counter the disease and save countless lives in danger, it is necessary to predict the trend of the number of cases and deaths and then implement the policies accordingly. In this paper, the trends of the growth rate of worldwide cases and deaths were studied, and the future growth for 100 days was predicted using the Neural network model and Polynomial Regression model. For efficient planning, the countries were grouped using Principal Component Analysis and the predictions were made. The cases and deaths in different countries and states were related through the Pearson coefficient, and the heat maps were studied. Additionally, in this paper, a case study to predict the trend of cases, number of deaths and recoveries in India was also performed. The Indian states were grouped into four groups based on the Principal Component Analysis (PCA) results, and relevant remarks and trends were suggested. The growth of cases and deaths was studied, and the peaks were predicted for the next 200 days. In recent months, COVID-19 has generated a significant deal of anxiety as a global pandemic, and an increasing number of studies have been published in this area. Consequently, a bibliometric examination of these papers may offer insight into current research hot subjects and trends. We are the first to join stochastic analysis of COVID-19 effects with bibliometric analysis of COVID-19. This prediction, if taken into consideration strategically during the planning of preventive measures of COVID-19 can help to reduce the cases to a great extent.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc4e7b743ec7642cfbaf75ec461c7bd9b30a2ac2" target='_blank'>
              A stochastic analysis and bibliometric analysis of COVID-19
              </a>
            </td>
          <td>
            Nisha Gaur, Rakshita Chaudhary, Mohit Yadav, Siddharth Srivastava, Vanshika Chaudhary, Mohd Asif Shah
          </td>
          <td>2024-12-15</td>
          <td>Indian Journal of Microbiology Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The Nipah virus outbreak in Kerala, India, in 2024 presented significant public health challenges, necessitating an in-depth analysis of its transmission dynamics and the effectiveness of response measures. This study employs the SIR (Susceptible, Infected, Recovered) epidemiological model to simulate the outbreak, focusing on key parameters such as transmission and recovery rates. The model reveals that the outbreak peaked around Day 10, with a maximum of approximately 631 infected individuals, highlighting the rapid spread of the virus. However, timely public health interventions, including contact tracing, isolation, and community engagement, effectively curtailed transmission, resulting in a swift decline in infected cases and a significant recovery rate. By Day 100, nearly the entire population had either recovered or remained uninfected, underscoring the success of containment efforts. This study emphasizes the critical role of proactive health strategies in managing infectious diseases and provides valuable insights for future preparedness against zoonotic outbreaks. The findings highlight the necessity of continuous monitoring and community involvement in strengthening public health infrastructure to mitigate the impact of similar outbreaks in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba5f2f7bb33996aee2e9839fd507babda6f679f2" target='_blank'>
              Epidemiological insights into Nipah virus: A sir model perspective
              </a>
            </td>
          <td>
            M. Muthuvel
          </td>
          <td>2024-11-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="During the Covid-19 pandemic, India was one of the worst hit
countries in the world. A wide variation was observed in the reporting
of cases across the States/Union Territories and across the districts
within each State in the country. However, a majority of the SARS1 CoV-2 infected patients remained asymptomatic, and undiscovered .
Despite the global number of infections reaching tens of millions
during its peak, the overall infection rate was relatively low within the
2
local infection hotspots . The seroprevalence of antibodies was thus
important to understand the transmission intricacies of the virus such
as, estimating total infections, including mild and asymptomatic
3
individuals who might not have tested ; and the possibility of
transmission interruption through the depletion of susceptible
individuals if seroconversion was associated with robust immunity.
Facility-based surveillance efforts, although were found to be useful
were not representative of the population especially in hilly areas, as
that of Uttarakhand. Population-based sero-epidemiological studies
were therefore, recommended to measure the extent of spread of
4
infection in a given area . We thus aimed to measure serum levels of
SARS-CoV-2 antibodies in the population of hilly areas and to assess
the association with SARS-CoV-2 antibody titters with ABO blood
groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0add455249605e1c8fd924875d9ff4375a4b6246" target='_blank'>
              ASSESSING COVID-19 IN UTTARAKHAND: A SERO-EPIDEMIOLOGICAL COMMUNITY SURVEY
              </a>
            </td>
          <td>
            J. Semwal, A. Srivastava, Rajender Singh Panwar, Akanksha Uniyal, Deepti Nagrath, Garima Mittal, M. Raturi
          </td>
          <td>2024-12-01</td>
          <td>INDIAN JOURNAL OF APPLIED RESEARCH</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="In the face of the COVID-19 pandemic, understanding the dynamics of disease transmission is crucial for effective public health interventions. This study explores the concept of symmetry within compartmental models, employing compartmental analysis and numerical simulations to investigate the intricate interactions between compartments and their implications for disease spread. Our findings reveal the conditions under which the disease-free equilibrium is globally asymptotically stable while the endemic equilibrium exhibits local stability. Additionally, we investigate the phenomenon of backward bifurcation, shedding light on the critical role of quarantine measures in controlling outbreaks. By integrating the concept of symmetry into our model, we enhance our understanding of transmission dynamics and provide a robust framework for evaluating intervention strategies. The insights gained from this research are vital for policymakers and health authorities aiming to mitigate the impact of infectious diseases in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e66d4cc634f6e96f8a980008f8ce554c17dbb18a" target='_blank'>
              Exploring Symmetry in an Epidemiological Model: Numerical Analysis of Backward Bifurcation and Sensitivity Indices
              </a>
            </td>
          <td>
            Fathia Moh, Al Samma, Avinash, P. Chellamani, Nafisa A. Albasheir, Ameni Gargouri, G. Britto, Antony Xavier, Mohammed M. A. Almazah
          </td>
          <td>2024-11-26</td>
          <td>Symmetry</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="ABSTRACT Background: Post-pandemic surveillance data on COVID-19 infections may help inform future public health policies regarding SARS-CoV-2 testing, vaccinations or other COVID-19 measures. We estimate the total SARS-CoV-2 infections in Austria after the end of the pandemic (May 5, 2023, per WHO) up to May 2024 from wastewater data. Those estimates are used in an agent-based model (ABM) to estimate average national levels of SARS-CoV-2 infection protection (IP) and COVID-19 death protection (DP). Methods: We use a previously published model estimating total infections in Austria from wastewater data and extrapolate the approach up to May 2024. Utilizing those estimates in an ABM, we estimate daily national average IP and DP. These estimates are based on waning immunity estimates of previous literature and incorporate documented vaccinations. Findings: We estimate approximately 3.2 million infections between May 6, 2023, and May 23, 2024, with a total of 17.8 million infections following May 12, 2020. The ABM estimates that 95% of people in Austria were infected with SARS-CoV-2 at least once. It also shows very high levels of national average DP a year after the end of the pandemic. National IP remained relatively low after the onset of Omicron. Interpretation: The estimated high number of SARS-CoV-2 infections since the end of the COVID-19 pandemic in Austria has kept the national average DP very high. These findings should be considered for public health decisions on SARS-CoV-2 testing practices and vaccine booster administrations. Funding: Austrian Science Fund (FWF) KLI 1188.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041c58fbf50c446af6f1e8837368177128b7302a" target='_blank'>
              Estimates of SARS-CoV-2 infections and population immunity after the COVID-19 pandemic in Austria: Analysis of national wastewater data
              </a>
            </td>
          <td>
            U. Riedmann, A. Chalupka, L. Richter, M. Sprenger, W. Rauch, H. Schenk, R. Krause, P. Willeit, H. Oberacher, T. B. Hoeg, J. Ioannidis, S. Pilz
          </td>
          <td>2024-11-23</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The COVID-19 pandemic, driven by the rapid evolution of the SARS-CoV-2 virus, presents ongoing challenges to global public health. SARS-CoV-2 is characterized by rapidly evolving mutations, especially in (but not limited to) the spike protein, complicating predictions about its evolutionary trajectory. These mutations have significantly affected transmissibility, immune evasion, and vaccine efficacy, leading to multiple pandemic waves with over half a billion cases and seven million deaths globally. Despite several strategies, from rapid vaccine development and administration to the design and availability of antivirals, including monoclonal antibodies, already having been employed, the persistent circulation of the virus and the emergence of new variants continue to result in high case numbers and fatalities. In the past four years, immense research efforts have contributed much to our understanding of the viral pathogenesis mechanism, the COVID-19 syndrome, and the host–microbe interactions, leading to the development of effective vaccines, diagnostic tools, and treatments. The focus of this review is to provide a comprehensive analysis of the functional impact of mutations on diagnosis, treatments, and vaccine effectiveness. We further discuss vaccine safety in pregnancy and the implications of hybrid immunity on long-term protection against infection, as well as the latest developments on a pan-coronavirus vaccine and nasal formulations, emphasizing the need for continued surveillance, research, and adaptive public health strategies in response to the ongoing SARS-CoV-2 evolution race.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9366d548cb0510b7d43a3b64ec5fee743469505" target='_blank'>
              SARS-CoV-2 Evolution: Implications for Diagnosis, Treatment, Vaccine Effectiveness and Development
              </a>
            </td>
          <td>
            Fabrizio Angius, Silvia Puxeddu, Silvio Zaimi, Serena Canton, Sepehr Nematollahzadeh, Andrea Pibiri, Ilenia Delogu, G. Alvisi, Meng Ling Moi, Aldo Manzin
          </td>
          <td>2024-12-28</td>
          <td>Vaccines</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="

 During the three years of the COVID-19 epidemic, the disease spread to 205 locations across Saudi Arabia’s 13 administrative areas. There were serious repercussions of mortality and morbidity. However, since to the hard work of all the sectors and authorities involved, the pandemic was successfully managed..



 This analytical investigation of COVID-19 aims to pinpoint the locations affected over three years. It also investigates the morbidity and mortality burden of the epidemic and identifies locations (governorates) and administrative areas for the volume and severity of COVID-19 cases, year-wise.



 Daily reports of COVID-19 cases published by the Saudi Arabian Ministry of Health from March 15, 2020, to April 30, 2023, were analyzed in this research. Annual counts, estimates of crude infection rates, and case fatality rates with monthly averages were calculated for the entire country, its administrative areas, and 205 affected locations.



 Despite a progressive trend, the impact of the second and third waves of COVID-19 on morbidity and mortality varied across administrative areas, governorates, and locations. Both major administrative areas of Riyadh, Makkah,Madina, and the Eastern Region, as well as smaller ones, such as Al-Baha, Jazan and Aseer, faced serious repercussions of morbidity. Additionally, mortality was notable in the smaller administrative areas, namely, Al-Jouf, Hail, Jazan, Aseer, Northern Borders, and Al-Baha. Beyond, certain locations within these administrative areas outside populous cities and administrative headquarters emerged as COVID-19 hotspots, particularly in terms of mortality.



 The COVID-19-related morbidity and mortality rates in Saudi Arabia exhibited a discernible pattern across the administrative areas, governorates, and locations, with numbers and infection rates fluctuating every month, showing both increases and decreases. Although mortality rates were higher in the first year, significant control measures were implemented that led to a reduction in this rate later. The fatality rate displayed notable variations across administrative areas. This analysis holds significance for the development of epidemic control systems, the implementation of resistance measures at the grassroots level, and the establishment of surveillance systems. Furthermore, it contributes to a clearer understanding of the situation that will facilitate international comparisons and the formation of collaborative networks.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ca7149880158e189264fae10d9081b3e68d427" target='_blank'>
              COVID-19 Morbidity and Mortality in Saudi Arabia: A Year-wise Geographic Analysis
              </a>
            </td>
          <td>
            H. Aldossari
          </td>
          <td>2024-12-31</td>
          <td>The Open Public Health Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The COVID-19 pandemic has caused widespread disruption and prompted the implementation of nonpharmaceutical measures in several countries to contain the spread of the disease until a vaccine became available. Urban mobility reduction has historically been employed to limit the transmission of epidemics. However, only some studies have quantified the effectiveness of such restrictions across different sectors of society. To address this gap, we have adapted an agent-based model that utilizes data from the Google Community Mobility Report to simulate the dynamics of COVID-19 across 14 Brazilian capitals. Each agent in the model has a network of contacts established from mobility data across various categories. We simulated six scenarios for each capital, each with different probabilities of contagion in each category. Our findings show that different scenarios are more effective in describing the curve of infected people and deaths in each city. In particular, our results indicate that the same scenario was optimal for describing the number of cases and deaths in Belo Horizonte. However, the first peak in the number of deaths could not be reproduced in the model, possibly due to issues with the data recording. Our proposed model can be further developed to incorporate additional elements related to the dynamics of an epidemic and can serve as an additional tool in understanding and planning actions to contain the spread of COVID-19 beyond mobility, particularly in urban centers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5b4f95bcbe7c33f113a8caf8c452500c23b92e6" target='_blank'>
              Understanding the Role of Daily Activities in the Transmission of COVID-19 in Urban Settings Using an Agent-Based Model
              </a>
            </td>
          <td>
            L. L. Lima, Elizabeth F. Wanner, A. Atman
          </td>
          <td>2024-12-15</td>
          <td>Complex Syst.</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The ongoing COVID-19 pandemic, fueled by the highly transmissible and mutable SARS-CoV-2 virus, has disrupted global health, economies, and daily life since late 2019. The virus's capacity to mutate into novel variants poses significant challenges to prevention and control efforts, altering its transmissibility, virulence, and potential for immune evasion. Monitoring the circulation of SARS-CoV-2 variants is crucial as they may differ in transmissibility, disease severity, and impact on healthcare systems. Furthermore, mutant strains can compromise vaccine effectiveness, necessitating continuous surveillance to inform public health interventions and ensure the efficacy of preventive measures. This study underscores the importance of tracking variant dissemination to anticipate and mitigate the evolving threat posed by SARS-CoV-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11d0e31f9422c95e9744ac48447b22d71f3974ba" target='_blank'>
              Benchmarking Mutation Type of SARS-CoV-2's Different Variants and Their Relationship with Infectiousness
              </a>
            </td>
          <td>
            Jingzhe Zhang
          </td>
          <td>2024-12-06</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Mathematical models of viral dynamics are crucial in understanding infection trajectories. However, severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) viral load data often includes limited sparse observations with significant heterogeneity. This study aims to: (1) understand the impact of patient characteristics in shaping the temporal viral load trajectory and (2) establish a data collection protocol (DCP) to reliably reconstruct individual viral load trajectories. We collected longitudinal viral load data for SARS‐CoV‐2 Delta and Omicron variants from 243 patients in Singapore (2021–2022). A viral dynamics model was calibrated using patients' age, symptom presence, and vaccination status. We accessed associations between these patient characteristics and aspects of viral dynamics using linear regression models. We evaluated the accuracy of viral load trajectory estimation under different simulated DCPs by varying patient numbers, test frequencies, and test intervals. Older unvaccinated individuals had a longer viral shedding duration due to lower infection and cell death rates. Higher peak viral loads were found in older, symptomatic, and vaccinated individuals, with earlier peaks in younger vaccinated individuals. Symptom presence and vaccination resulted in a shorter time from infection to diagnosis. To accurately estimate viral dynamics, more frequent tests, longer test intervals, and larger patient samples are required. For 500 patients, a 21‐day follow‐up with measurements every 3 days and an 8‐day follow‐up with daily measurements was optimal for the Delta and Omicron variants, respectively. Patient characteristics significantly impacted viral dynamics. Our analytic approach and recommended DCPs can enhance preparedness and response to emerging pathogens beyond SARS‐CoV‐2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/820685edf6e7dddb8e879ebcae761b20ded41791" target='_blank'>
              Defining the Critical Requisites for Accurate Simulation of SARS‐CoV‐2 Viral Dynamics: Patient Characteristics and Data Collection Protocol
              </a>
            </td>
          <td>
            Hoong Kai Chua, Ananya Singh, Yuqian Wang, Y. Goh, Conrad En Zuo Chan, J. Chavatte, Raymond Valentine Tzer Pin Lin, Yvonne C F Su, M. Ajelli, P. Chia, S. W. Ong, David Chien Lye, B. Young, Keisuke Ejima
          </td>
          <td>2025-01-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4734fc3059a064e8f4d1d120ff045e91d4a80cf" target='_blank'>
              The role of a vaccine booster for a fractional order model of the dynamic of COVID-19: a case study in Thailand
              </a>
            </td>
          <td>
            P. Pongsumpun, P. Pongsumpun, I. Tang, J. Lamwong
          </td>
          <td>2025-01-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="By considering the recently introduced SIRU model, in this paper we study the dynamic of COVID-19 pandemic under the temporally varying public intervention in the Chilean context. More precisely, we propose a method to forecast cumulative daily reported cases CR(t), and a systematic way to identify the unreported daily cases given CR(t) data. We firstly base on the recently introduced epidemic model SIRU (Susceptible, Asymptomatic Infected, Reported infected, Unreported infected), and focus on the transmission rate parameter τ. To understand the dynamic of the data, we extend the scalar τ to an unknown function τ(t) in the SIRU system, which is then inferred directly from the historical CR(t) data, based on nonparametric estimation. The estimation of τ(t) leads to the estimation of other unobserved functions in the system, including the daily unreported cases. Furthermore, the estimation of τ(t) allows us to build links between the pandemic evolution and the public intervention, which is modeled by logistic regression. We then employ polynomial approximation to construct a predicted curve which evolves with the latest trend of CR(t). In addition, we regularize the evolution of the forecast in such a way that it corresponds to the future intervention plan based on the previously obtained link knowledge. We test the proposed predictor on different time windows. The promising results show the effectiveness of the proposed methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bab420c1f475b889c90af43509cbdcecb1074ba7" target='_blank'>
              Extended SIRU model with dynamic transmission rate and its application in the forecasting of COVID-19 under temporally varying public intervention
              </a>
            </td>
          <td>
            Yiye Jiang, Gaston Vergara-Hermosilla
          </td>
          <td>2024-12-31</td>
          <td>BIOMATH</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Coronavirus disease (COVID-19) is an extremely transmissible disease that causes tremendous human suffering worldwide, with millions of people dying within two years. In Malaysia, the COVID-19 pandemic continues with an increasing number of positive cases. In order to monitor and prevent the disease, disease mapping is proposed for quantifying and describing the disease risk. The primary objective of this research is to estimate the COVID-19 relative risk using one of the most widely employed Bayesian approaches, namely the Poisson-gamma model, and eventually display a disease map to show the COVID-19 risk level in each state. The relative risk results are divided into five different levels to classify the COVID-19 risk level in each Malaysian state. The illustration of five different shades in the disease mapping has distinguished the risk levels within each state, where the most light-coloured area indicates very low risk whilst the most dark-coloured area indicates very high risk. According to the results, the area with the lowest risk of contracting COVID-19 in 2020 is Perlis, and the area with the highest risk is Labuan, whereas in 2021, Perlis remains the lowest risk area while Selangor is the highest risk area.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b232c053619e7f30e2176eebf4ae784f40d01d3" target='_blank'>
              COVID-19 Disease Mapping in Malaysia Using Poisson-gamma Model to Safeguard Community
              </a>
            </td>
          <td>
            Nazrina Aziz
          </td>
          <td>2024-12-19</td>
          <td>ASM Science Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97f6d580fd974a0a884517a540a8e0e223047f66" target='_blank'>
              Leveraging dynamics informed neural networks for predictive modeling of COVID-19 spread: a hybrid SEIRV-DNNs approach
              </a>
            </td>
          <td>
            Cheng Cheng, E. Aruchunan, Muhamad Hifzhudin Noor Aziz
          </td>
          <td>2025-01-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="A novel time-dependent deterministic SEIRS model, extended with vaccination, hospitalization, and vital dynamics, is introduced. Time-varying basic and effective reproduction numbers associated with this model are defined, which are crucial metrics in understanding epidemic dynamics. Furthermore, a parameter identification approach has been used to develop a numerical method to compute these numbers for long-term epidemics. We analyze the actual COVID-19 data from the USA, Italy, and Bulgaria to solve appropriate inverse problems and gain an understanding of the time evolution behavior of the basic and effective reproduction numbers. Moreover, an insightful comparison of key coronavirus data and epidemiological parameters across these countries has been conducted. For this purpose, while the basic and effective reproduction numbers provide insights into the virus transmission potential, we propose data-driven criteria for assessing the actual realization of the transmission potential of the SARS-CoV-2 virus and the effectiveness of the applied restrictive measures. To obtain these results, we conduct a mathematical analysis to demonstrate various biological properties of the new differential model, including non-negativity, boundedness, existence, and uniqueness of the solution. The new model and the associated numerical simulation tools proposed herein could be applied to COVID-19 data in any country worldwide and hold a promising potential for the transmission capacity and impact of the virus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83a73e826a0200a6008e95af641e2671d8dfbd89" target='_blank'>
              Computing the COVID-19 Basic and Effective Reproduction Numbers Using Actual Data: SEIRS Model with Vaccination and Hospitalization
              </a>
            </td>
          <td>
            S. Margenov, N. Popivanov, T. Hristov, Veneta Koleva
          </td>
          <td>2024-12-19</td>
          <td>Mathematics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="PurposeOwing to the limitations of traditional infectious disease dynamic systems in accurately encapsulating the nuances of China’s dynamic epidemic prevention policies and considering the varying sensitivity of local governments to the unfolding of public health emergencies (PHEs), this paper introduces a novel infectious disease dynamic system.Design/methodology/approachThis system, rooted in the distinct characteristics of infectious diseases and nuanced prevention and control measures, leverages a learning model for enhanced precision. It intricately incorporates factors such as the infectivity in sealed and controlled areas and the role of asymptomatic patients, thereby refining the dynamics of isolation, sealing, control and the transition from asymptomatic to confirmed cases. Employing the Markov Chain Monte Carlo (MCMC) parameter estimation approach significantly augments the accuracy in pinpointing the valid parameters of disease spread. Empirical analysis was meticulously carried out, using data from the Shanghai epidemic from 1 Mar 2022 to 1 Jul 2022.FindingsThis analysis not only illuminates the profound impact of control efforts on the trajectory of the epidemic but also underscores the pivotal role of social distancing in curbing the rapid transmission of infectious diseases. Furthermore, it reveals that an accelerated detection rate during the swift spread and peak of the epidemic paradoxically leads to a surge in confirmed cases and a consequent strain on medical resources, thereby impeding the pace of medical intervention.Originality/valueA stage-wise dissection of the Shanghai epidemic and comparative analyses against the evolution profiles in ASEAN countries elucidates the five stages of PHE risk evolution in alignment with the crisis lifecycle theory. These stages encompass hidden transmission, multi-point dissemination, multi-chain parallelism, rapid spread, fluctuation rebound and multi-community spread, each presenting unique challenges and dynamics in the control and management of the epidemic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20bdaf0a3e993eda4a73cbc9d58ac9ae13549307" target='_blank'>
              Evolution of transmission risk in public health emergencies: an MCMC-based approach in the Chinese context
              </a>
            </td>
          <td>
            Chaoyu Zheng, Zhaoqiang Zhong, Baiyu Wu, Xuan Zhao, Mu Yue, Benhong Peng
          </td>
          <td>2024-11-29</td>
          <td>Kybernetes</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Households are a major driver of transmission of acute respiratory viruses, such as SARS-CoV-2 or Influenza. Until now antiviral treatments have mostly been used as a curative treatment in symptomatic individuals. During an outbreak, more aggressive strategies involving pre- or post-exposure prophylaxis (PrEP or PEP) could be employed to further reduce the risk of severe disease but also prevent transmission to household contacts. In order to understand the effectiveness of such strategies and the factors that may modulate them, we developed a multi-scale model that follows the infection at both the individual-level (viral dynamics) and the population-level (transmission dynamics) in households. Using a simulation study we explored different antiviral treatment strategies, evaluating their effectiveness on reducing the transmission risk and the virological burden in households for a range of virus characteristics (e.g., secondary attack rate—SAR, or time to peak viral load). We found that when the index case can be identified and treated before symptom onset, both transmission and virological burden are reduced by > 75% for most SAR values and time to peak viral load, with minimal benefit to treat additionally household contacts. While treatment initiated after index symptom onset does not reduce the risk of transmission, it can still reduce the virological burden in the household, a proxy for severe disease and subsequent transmission risk outside the household. In that case optimal strategies involve treatment of both index case and household contacts as PEP, with efficacy > 50% when peak viral load occurs after symptom onset, and 30-50% otherwise. In all the considered cases, antiviral treatment strategies were optimal for SAR ranging 20-60%, and for larger household sizes. This study highlights the opportunity of antiviral drug-based interventions in households during an outbreak to minimize viral transmission and disease burden.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e658863dba1557462cd20ae53a9f724d0642f36" target='_blank'>
              Modelling the effectiveness of antiviral treatment strategies to prevent household transmission of acute respiratory viruses
              </a>
            </td>
          <td>
            Hind Zaaraoui, Clarisse Schumer, Xavier Duval, Bruno Hoen, L. Opatowski, Jérémie Guedj
          </td>
          <td>2024-12-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Disentangling contributions of different hosts to disease emergence and spread is highly complex, but critical for improving predictions, surveillance, and response. This is particularly challenging in wildlife, with pathogens often infecting multiple species and data collection being difficult. Using the emergence of Usutu virus (USUV) in the Netherlands as a case study, we demonstrate the use of an Approximate Bayesian Computation framework on diverse data sources to uncover hidden drivers of spatio-temporal wildlife disease emergence. We calibrated single- and multi-host mechanistic transmission models to five types of wildlife surveillance and research data, describing molecular and serological evidence of USUV in wild birds. Although Eurasian blackbirds, the primary sentinel species, were the most severely affected species, our models indicated that USUV could not persist in blackbirds alone. Our data-model framework provided statistical support for additional bird species to have contributed to transmission. This population of bird species is characterised by limited infection mortality, a longer lifespan, and likely further dispersal than blackbirds. Immunity in this reservoir population appears to have protected blackbirds from further USUV-related population decline. Continued surveillance could help identify these reservoir species. Our results underscore the importance of considering multiple host populations to understand outbreak dynamics. Neglecting the multi-host context of disease transmission can impact the reliability of future predictions and the projected impact of intervention strategies. Significance statement Understanding which species contribute to disease transmission is critical for improving predictions, surveillance, and response. However, disentangling these contributions is difficult, especially for wildlife diseases where pathogens often infect multiple species and data collection is notoriously difficult. Using Usutu virus (USUV) in the Netherlands as an example, we show how a computational framework can integrate diverse wildlife data sources to uncover hidden contributors to transmission. Our findings show that blackbirds, the most severely affected species, cannot sustain USUV transmission alone. Additional bird species, characterised by lower infection mortality and longer lifespans, are important for transmission and may protect more vulnerable species. This work emphasizes that studying transmission in a multi-host context is crucial for accurate outbreak predictions and effective interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e24fd07cbc29680e2d92999ea9c06f9ca2ce6236" target='_blank'>
              Silent reservoirs are shaping disease emergence: the case of Usutu virus in the Netherlands
              </a>
            </td>
          <td>
            M. de Wit, G. Beaunée, M. Dellar, E. Münger, L. Krol, N. Atama, J. van Irsel, H. P. van der Jeugd, M. Koopmans, M. de Jong, R. S. Sikkema, Q. T. ten Bosch
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The corona virus pandemic has affected the population worldwide leading to SARS-COV-2 virus, traumatic impacts on health systems, public health, societies and individuals globally. To control the spread of covid-19 government imposed national lockdowns, travel restrictions and instructed social distancing along with the use of facemasks and sanitizers in many countries. As a result of this, many researchers are opting for digital data collection methods rather than manual way of collection. Data collection via phone, emails, and various other digital platforms were being preferred. This study will help to understand that how covid infection affects the various human body systems including the age, occupation, gender, blood groups and co-morbidities. It will also help to analyze between the patient affectivity rate (more positivity rate in 1st wave or 2nd wave), role of vaccination and its effectivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33f7e77b18b71e92175344c2e2759bba2f193557" target='_blank'>
              Study the Risk Factors Influencing Covid-19 Infection in Humans
              </a>
            </td>
          <td>
            Subhash D. Pawar, Deepak B. Shirgaonkar
          </td>
          <td>2024-12-22</td>
          <td>International Journal For Multidisciplinary Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>