<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../IBD/">
      
      
        <link rel="next" href="../Cancer/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Covid
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Covid</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-17 18:05:27 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Covid</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Covid</a><br>
      <a href="#recommended_articles">3. Recommended articles on Covid</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Covid</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Covid</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="A quantitative systems pharmacology (QSP) model of the pathogenesis and treatment of SARS-CoV-2 infection can streamline and accelerate the development of novel medicines to treat COVID-19. Simulation of clinical trials allows in silico exploration of the uncertainties of clinical trial design and can rapidly inform their protocols. We previously published a preliminary model of the immune response to SARS-CoV-2 infection. To further our understanding of COVID-19 and treatment, we significantly updated the model by matching a curated dataset spanning viral load and immune responses in plasma and lung. We identified a population of parameter sets to generate heterogeneity in pathophysiology and treatment and tested this model against published reports from interventional SARS-CoV-2 targeting mAb and antiviral trials. Upon generation and selection of a virtual population, we match both the placebo and treated responses in viral load in these trials. We extended the model to predict the rate of hospitalization or death within a population. Via comparison of the in silico predictions with clinical data, we hypothesize that the immune response to virus is log-linear over a wide range of viral load. To validate this approach, we show the model matches a published subgroup analysis, sorted by baseline viral load, of patients treated with neutralizing Abs. By simulating intervention at different time points post infection, the model predicts efficacy is not sensitive to interventions within five days of symptom onset, but efficacy is dramatically reduced if more than five days pass post symptom onset prior to treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac952ba3508fc13b6ca5e8d7e11d7a7086f9b72d" target='_blank'>
                A quantitative systems pharmacology model of the pathophysiology and treatment of COVID-19 predicts optimal timing of pharmacological interventions
                </a>
              </td>
          <td>
            Rohit Rao, C. Musante, R. Allen
          </td>
          <td>2021-12-08</td>
          <td>medRxiv, NPJ Systems Biology and Applications</td>
          <td>8</td>
          <td>18</td>

            <td><a href='../recommendations/ac952ba3508fc13b6ca5e8d7e11d7a7086f9b72d' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Covid'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Covid</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="The emergence of COVID-19 in December 2019 spurred a global effort to develop effective medical interventions. Therapeutic monoclonal antibodies (mAbs) have emerged as a promising strategy to combat the SARS-CoV-2 virus. Several mAbs targeting the receptor-binding domain (RBD) of the SARS-CoV-2 spike protein have received Emergency Use Authorization (EUA) for treating mild to moderate COVID-19. Additionally, human mAbs and hyperimmune plasma derived from recovered COVID-19 patients have been explored as potential therapeutic options. This review delves into the potential of mAbs for the diagnosis and treatment of COVID-19 infection. We discuss the mechanisms of action of mAbs, as well as their advantages and limitations. Furthermore, we explore the ongoing research and development efforts to optimize mAb-based therapies for COVID-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9f9d506498cea799c0f5d3d46a343d0fe32bca" target='_blank'>
              Monoclonal Antibodies for Treatment of COVID-19: An Updated Review of Current Evidence
              </a>
            </td>
          <td>
            Saurabh Nimesh, Pratibha Kumari, Rohit Kumar, Gosiya Gosiya, Md. Quamuddin, Md. Iftekhar Ahmad
          </td>
          <td>2024-11-30</td>
          <td>Borneo Journal of Pharmacy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Remdesivir inhibits the SARS-CoV-2 RNA-dependent RNA polymerase (RdRp; Nsp12). Here, we conducted viral resistance analyses from the Phase 3 PINETREE trial of remdesivir in nonhospitalized participants at risk of severe COVID-19. Nasopharyngeal swabs (collected at baseline [Day 1], Days 2, 3, 7, and 14) were eligible for analysis if their viral load was above the lower limit of quantification for the RT-qPCR assay (2228 copies/mL). The SARS-CoV-2 genome was sequenced for all remdesivir participants and 50% of placebo participants (baseline, Days 3, 7, and 14) and for participants who progressed to COVID-19-related hospitalization or all-cause death (all time points). Emergent substitutions in Nsp12 and other replication complex proteins were phenotyped using site-directed mutagenesis in a SARS-CoV-2 subgenomic replicon system. Overall, emergent Nsp12 substitutions were detected in 8/115 (7.0%) remdesivir participants and 7/129 (5.4%) placebo participants (1 substitution overlap between groups). Based on a structural analysis, none of the emergent Nsp12 substitutions were in direct contact with the incoming nucleoside triphosphate substrate, the RNA, or the RNA template 5' overhang. One substitution (A376V) showed reduced susceptibility to remdesivir (12.6-fold change in remdesivir half-maximal concentration [EC50]); it also showed reduced fitness when introduced in the SARS-CoV-2 replicon and virus in vitro. Other substitutions had <1.1-fold change in remdesivir EC50. None of the emergent substitutions in Nsp8, Nsp10, Nsp13, or Nsp14 (remdesivir, 10/115 [8.7%]; placebo, 10/129 [7.8%]) showed reduced remdesivir susceptibility. In conclusion, emergent substitutions in the SARS-CoV-2 RdRp complex with reduced remdesivir susceptibility were uncommon, indicating a high barrier to remdesivir resistance.CLINICAL TRIALSThis study is registered with ClinicalTrials.gov as NCT04501952.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee67dedddaf4dc05c98bf1aca4440f825403ea3d" target='_blank'>
              SARS-CoV-2 resistance analyses from the Phase 3 PINETREE study of remdesivir treatment in nonhospitalized participants.
              </a>
            </td>
          <td>
            Lauren Rodriguez, Hery W. Lee, Jiani Li, Ross Martin, Dong Han, Simin Xu, Jasmine Moshiri, Nadine Peinovich, Gregory Camus, Jason K. Perry, Robert H Hyland, D. Porter, Mazin Abdelghany, Matthias Götte, C. Hedskog
          </td>
          <td>2024-12-19</td>
          <td>Antimicrobial agents and chemotherapy</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The COVID-19 pandemic, driven by the rapid evolution of the SARS-CoV-2 virus, presents ongoing challenges to global public health. SARS-CoV-2 is characterized by rapidly evolving mutations, especially in (but not limited to) the spike protein, complicating predictions about its evolutionary trajectory. These mutations have significantly affected transmissibility, immune evasion, and vaccine efficacy, leading to multiple pandemic waves with over half a billion cases and seven million deaths globally. Despite several strategies, from rapid vaccine development and administration to the design and availability of antivirals, including monoclonal antibodies, already having been employed, the persistent circulation of the virus and the emergence of new variants continue to result in high case numbers and fatalities. In the past four years, immense research efforts have contributed much to our understanding of the viral pathogenesis mechanism, the COVID-19 syndrome, and the host–microbe interactions, leading to the development of effective vaccines, diagnostic tools, and treatments. The focus of this review is to provide a comprehensive analysis of the functional impact of mutations on diagnosis, treatments, and vaccine effectiveness. We further discuss vaccine safety in pregnancy and the implications of hybrid immunity on long-term protection against infection, as well as the latest developments on a pan-coronavirus vaccine and nasal formulations, emphasizing the need for continued surveillance, research, and adaptive public health strategies in response to the ongoing SARS-CoV-2 evolution race.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9366d548cb0510b7d43a3b64ec5fee743469505" target='_blank'>
              SARS-CoV-2 Evolution: Implications for Diagnosis, Treatment, Vaccine Effectiveness and Development
              </a>
            </td>
          <td>
            Fabrizio Angius, Silvia Puxeddu, Silvio Zaimi, Serena Canton, Sepehr Nematollahzadeh, Andrea Pibiri, Ilenia Delogu, G. Alvisi, Meng Ling Moi, Aldo Manzin
          </td>
          <td>2024-12-28</td>
          <td>Vaccines</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Few pathogens have historically been subjected to as intense scientific and clinical scrutiny as SARS-CoV-2. The genetic, immunological, and environmental factors influencing disease severity and post-infection clinical outcomes, known as correlates of immunity, remain largely undefined. Clinical outcomes of SARS-CoV-2 infection vary widely, ranging from asymptomatic cases to those with life-threatening COVID-19 symptoms. While most infected individuals return to their former health and fitness within a few weeks, some develop debilitating chronic symptoms, referred to as long-COVID. Autoimmune responses have been proposed as one of the factors influencing long-COVID and the severity of SARS-CoV-2 infection. The association between viral infections and autoimmune pathologies is not new. Viruses such as Epstein-Barr virus and cytomegalovirus, among others, have been shown to induce the production of autoantibodies and the onset of autoimmune conditions. Given the extensive literature on SARS-CoV-2, here we review current evidence on SARS-CoV-2-induced autoimmune pathologies, with a focus on autoantibodies. We closely examine mechanisms driving autoantibody production, particularly their connection with disease severity and long-COVID.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3a7e3e65e1ca564ed6ea146ebd780ffc685597a" target='_blank'>
              Autoantibodies in COVID-19: implications for disease severity and clinical outcomes
              </a>
            </td>
          <td>
            Y. Galipeau, C. Cooper, M.-A. Langlois
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d8b890eebc91c909e5f9d774a6b588b8b0af22b" target='_blank'>
              The consequences of SARS-CoV-2 within-host persistence.
              </a>
            </td>
          <td>
            A. Sigal, R. Neher, R. Lessells
          </td>
          <td>2024-11-25</td>
          <td>Nature reviews. Microbiology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Introduction Dysregulated host cytokine responses to SARS-CoV-2 infection are a primary cause of progression to severe disease, whereas early neutralizing antibody responses are considered protective. However, there are gaps in understanding the early temporal dynamics of these immune responses, and the profile of productive immune responses generated by non-hospitalized people with mild infections in the community. Methods Here we conducted a prospective cohort study of people with suspected infections/exposures in the US state of North Carolina, before vaccine availability. We recruited participants not only in hospitals/clinics, but also in their homes. With serial sampling, we compared virologic and immunologic factors in 258 community cases versus 114 hospital cases of COVID-19 to define factors associated with severity. Results We found that high early neutralizing antibodies were associated with lower nasal viral load, but not protection from hospitalization. Cytokine responses were evaluated in 125 cases, with subsets at first versus second week of illness to assess for time-dependent trajectories. The hospital group demonstrated a higher magnitude of serum IL-6, IL-1R antagonist, IP-10, and MIG; prolonged upregulation of IL-17; and lesser downregulation of GROα, IL-1R antagonist, and MCP1, in comparison to the community group suggesting that these factors may contribute to immunopathology. In the second week of illness, 2-fold increases in IL-6, IL-1R antagonist, and IP-10 were associated with 2.2, 1.8, and 10-fold higher odds of hospitalization respectively, whereas a 2-fold increase in IL-10 was associated with 63% reduction in odds of hospitalization (p<0.05). Moreover, antibody responses at 3-6 months post mild SARS-CoV-2 infections in the community revealed long-lasting antiviral IgM and IgA antibodies as well as a stable set point of neutralizing antibodies that were not waning. Discussion Our data provide valuable temporal cytokine benchmarks to track the progression of immunopathology in COVID-19 patients and guide improvements in immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c160351a23ee38ffff5d4077c00d47cb8a7c363" target='_blank'>
              Dynamics of cytokine and antibody responses in community versus hospital SARS-CoV-2 infections
              </a>
            </td>
          <td>
            Tulika Singh, Andrew N. Macintyre, Thomas W Burke, Jack Anderson, Elizabeth Petzold, Erica L. Stover, M. J. French, Thomas H. Oguin, Todd Demarco, M. T. McClain, E. Ko, Lawrence P. Park, Thomas Denny, Gregory D. Sempowski, C. Woods
          </td>
          <td>2024-11-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Long coronavirus disease (COVID) is characterized by persistent symptoms following severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) infection and has emerged as a significant health concern. As SARS‐CoV‐2 evolved from the wild‐type strain to the Alpha, Beta, Delta, and Omicron variants, there may be a variant‐specific influence on long COVID akin to the acute disease. This review aims to summarize our current knowledge of variant‐specific influences in long COVID incidence, symptom profile as well as mechanisms of pathogenesis. We highlight that long COVID incidence may be lower with the Omicron variants. The symptom profile of long COVID may also show some dependence on the different variants, with a reduction in cardiopulmonary symptoms with more recent SARS‐CoV‐2 variants. This heterogeneity of long COVID may also be related to the variant‐specific differences in affecting the immune system, viral persistence, and autoimmunity. However, emerging data also suggest that vaccinations may play a big role in shaping the presentation of long COVID. We also highlight ongoing work on long COVID incidence and symptom profiles in populations infected only by the Omicron variants. This will be beneficial toward more useful disease definitions and the development of effective diagnostic and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71d8acaecf6da03f3adb7ca5627bc1ea6f032923" target='_blank'>
              Long COVID across SARS‐CoV‐2 variants: Clinical features, pathogenesis, and future directions
              </a>
            </td>
          <td>
            L. S. C. Lok, Shuvam Sarkar, Calista Chi In Lam, Chak-Fun Law, Sin Teng Chau, Chun Yip Thomas Leung, Wai Hin Cheang, Ting Li, O. Monteiro, D. Baptista‐Hon
          </td>
          <td>2024-12-01</td>
          <td>MedComm – Future Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Molnupiravir is an antiviral medicine that induces lethal copying errors into SARS-CoV-2 RNA. Molnupiravir reduced hospitalization in one pivotal trial by 50% and had variable effects on reducing viral RNA levels in three separate trials. We used mathematical models to simulate these trials and closely recapitulated their virologic outcomes. Model simulations suggest lower antiviral potency against pre-omicron SARS-CoV-2 variants than against omicron. We estimate that in vitro assays underestimate in vivo potency 7-8 fold against omicron variants. Our model suggests that because polymerase chain reaction detects molnupiravir mutated variants, the true reduction in genetically intact viral RNA is underestimated by ~0.5 log in the two trials, which included participants with omicron. Our results reinforce past work suggesting that in vitro assays are unreliable for estimating in vivo antiviral drug potency and suggest that virologic endpoints for respiratory virus clinical trials should be catered to the drug mechanism of action.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/555cb89c949b6622e4105d45224e93cc98ccdeb2" target='_blank'>
              Molnupiravir clinical trial simulation suggests that polymerase chain reaction underestimates antiviral potency against SARS-CoV-2
              </a>
            </td>
          <td>
            S. Esmaeili-Wellman, K. Owens, J. Wagoner, S. J. Polyak, S. Zhang, J. Standing, D. M. Lowe, J. Schiffer
          </td>
          <td>2024-11-22</td>
          <td>None</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The ongoing COVID-19 pandemic, fueled by the highly transmissible and mutable SARS-CoV-2 virus, has disrupted global health, economies, and daily life since late 2019. The virus's capacity to mutate into novel variants poses significant challenges to prevention and control efforts, altering its transmissibility, virulence, and potential for immune evasion. Monitoring the circulation of SARS-CoV-2 variants is crucial as they may differ in transmissibility, disease severity, and impact on healthcare systems. Furthermore, mutant strains can compromise vaccine effectiveness, necessitating continuous surveillance to inform public health interventions and ensure the efficacy of preventive measures. This study underscores the importance of tracking variant dissemination to anticipate and mitigate the evolving threat posed by SARS-CoV-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11d0e31f9422c95e9744ac48447b22d71f3974ba" target='_blank'>
              Benchmarking Mutation Type of SARS-CoV-2's Different Variants and Their Relationship with Infectiousness
              </a>
            </td>
          <td>
            Jingzhe Zhang
          </td>
          <td>2024-12-06</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Amidst the ongoing global challenge of the SARS-CoV-2 pandemic, the quest for effective antiviral medications remains paramount. This comprehensive review delves into the dynamic landscape of FDA-approved medications repurposed for COVID-19, categorized as antiviral and non-antiviral agents. Our focus extends beyond conventional narratives, encompassing vaccination targets, repurposing efficacy, clinical studies, innovative treatment modalities, and future outlooks. Unveiling the genomic intricacies of SARS-CoV-2 variants, including the WHO-designated Omicron variant, we explore diverse antiviral categories such as fusion inhibitors, protease inhibitors, transcription inhibitors, neuraminidase inhibitors, nucleoside reverse transcriptase, and non-antiviral interventions like importin α/β1-mediated nuclear import inhibitors, neutralizing antibodies, and convalescent plasma. Notably, Molnupiravir emerges as a pivotal player, now licensed in the UK. This review offers a fresh perspective on the historical evolution of COVID-19 therapeutics, from repurposing endeavors to the latest developments in oral anti-SARS-CoV-2 treatments, ushering in a new era of hope in the battle against the pandemic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/851dfc76baf4d4a6141703277e48148e0c04b17e" target='_blank'>
              Navigating the COVID-19 Therapeutic Landscape: Unveiling Novel Perspectives on FDA-Approved Medications, Vaccination Targets, and Emerging Novel Strategies
              </a>
            </td>
          <td>
            R. Barghash, Donato Gemmati, Ahmed M. Awad, Mustafa M. M. Elbakry, V. Tisato, Kareem Awad, A. Singh
          </td>
          <td>2024-11-25</td>
          <td>Molecules</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The focus of this study was to identify risk factors for severe and critical COVID-19, evaluate local respiratory immune responses to SARS-CoV-2 infection, and develop a prognostic tool for COVID-19 severity using accessible early indicators. Using nasopharyngeal swab samples from hospitalized patients with COVID-19 of varying severity during the first wave of the pandemic from March to May 2020 in Louisiana, we evaluated the association between COVID-19 severity and viral load, respiratory immune mediators, and demographic/clinical factors. We found that the SpO2/FiO2 ratio at triage, total comorbidity burden (represented by Charlson Comorbidity Index), and gender were significantly associated with COVID-19 severity. Using these early significant indicators, we developed a prognostic tool for COVID-19 severity that is simple and convenient. Additionally, our study demonstrated that elevated levels of respiratory immune mediators, including IL-10, IL-6, MCP-1, and MCP-3, were significantly associated with COVID-19 severity. We also found that viral load at the time of admission was associated with disease severity. Our findings highlight the feasibility and importance of evaluating the humoral component of local mucosal immune responses and viral load at the infected site using convenient nasopharyngeal swab samples, which could be an effective method to understand the relationship between viral infection and immune responses at the early stages of infection. Our proposed prognostic tool has the potential to be useful for COVID-19 management in clinical settings, as it utilizes accessible and easy-to-collect variables at the time of admission.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e148339238a01b2d4bec183d27c2e7c1bd282904" target='_blank'>
              Predicting severe COVID-19 using readily available admission indicators: SpO2/FiO2 ratio, comorbidity index, and gender
              </a>
            </td>
          <td>
            Luan D. Vu, R. Christofferson, H. O'Neal, Diana Hamer, Anh Phan, Katie M. Vance, E. Turner, Avinash Kumar, I. M. Yola, Natalie Lim, Beverly Ogden, S. Cormier
          </td>
          <td>2024-11-20</td>
          <td>Experimental Biology and Medicine</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Background: Investigation of the effect of SARS-CoV-2 variants and COVID-19 vaccination on inflammatory and immune response to SARS-CoV-2 infection is limited in South Asia. Objectives: We aimed to examine the impact of COVID-19 vaccination and waves of COVID-19 on inflammatory and immunological biomarkers among COVID-19 patients in India. Methods: This cross-sectional analysis used baseline data from a randomized controlled trial of vitamin D and zinc during COVID-19 infection in India (N=181). Blood samples and data regarding vaccination doses were collected. The second (Delta) or third (Omicron) wave was determined by date of enrolment. Mixed effects linear regression with robust standard errors was used to examine associations between COVID-19 vaccination dose or wave at enrolment and C-Reactive Protein (CRP), ferritin, lactate dehydrogenase (LDH), D-dimer, interleukin-6 (IL-6), angiopoietin-2 (Ang-2), soluble triggering receptor expressed on myeloid cells-1 (sTREM-1), immunoglobulin G (IgG) and immunoglobulin M (IgM). Results: Compared to no vaccination, full vaccination was associated with lower LDH (P<0.001), D-dimer (P=0.521) and Ang-2 (P=0.046), and higher IgG levels (P<0.001). Partial vaccination was associated with lower IL-6 (P=0.040) and higher IgG (P<0.001). Enrolment during the third wave was associated with lower IL-6 (P<0.001), CRP (P=0.056), IgM (P=0.013), and IgG (P<0.001), but higher D-dimer levels (P<0.001). Conclusions: COVID-19 vaccination status and SARS-CoV-2 variant influence the inflammatory and immunologic response during SARS-CoV-2 infection, contributing to the severity of clinical presentation. Keywords: COVID-19, vaccine, variants, inflammation, immunology, biomarkers">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86e5f8be0e7533459d4243ee2965b0799209a5a6" target='_blank'>
              Impact of COVID-19 vaccine doses and viral waves on inflammatory and immunological responses to COVID-19 infections in India
              </a>
            </td>
          <td>
            K. K. Sharma, U. Pratap, Y. Marathe, S. Shaikh, P. D Costa, G. Gupta, M. Wang, W. Fawzi, K. C. Kain, N. Mistry, Y. Dholakia
          </td>
          <td>2024-12-21</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which causes coronavirus disease 2019 (COVID-19), has emerged as a global pandemic pathogen with high mortality. While treatments have been developed to reduce morbidity and mortality of COVID-19, more antivirals with broad-spectrum activities are still needed. Here, we identified lonafarnib (LNF), a Food and Drug Administration–approved inhibitor of cellular farnesyltransferase (FTase), as an effective anti–SARS-CoV-2 agent. LNF inhibited SARS-CoV-2 infection and acted synergistically with known anti-SARS antivirals. LNF was equally active against diverse SARS-CoV-2 variants. Mechanistic studies suggested that LNF targeted multiple steps of the viral life cycle. Using other structurally diverse FTase inhibitors and a LNF-resistant FTase mutant, we demonstrated a key role of FTase in the SARS-CoV-2 life cycle. To demonstrate in vivo efficacy, we infected SARS-CoV-2–susceptible humanized mice expressing human angiotensin-converting enzyme 2 (ACE2) and treated them with LNF. LNF at a clinically relevant dose suppressed the viral titer in the respiratory tract and improved pulmonary pathology and clinical parameters. Our study demonstrated that LNF, an approved oral drug with excellent human safety data, is a promising antiviral against SARS-CoV-2 that warrants further clinical assessment for treatment of COVID-19 and potentially other viral infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2c745b1ace96ab08adab58b9be7ba2d23c176f1" target='_blank'>
              Repurposing of lonafarnib as a treatment for SARS-CoV-2 infection
              </a>
            </td>
          <td>
            Mohsin Khan, Parker Irvin, Seung Bum Park, Hannah M Ivester, Inna Ricardo-Lax, Madeleine Leek, Ailis Grieshaber, Eun Sun Jang, Sheryl L Coutermarsh-Ott, Qi Zhang, N. Maio, Jian-Kang Jiang, Bing Li, Wenwei Huang, Amy Q. Wang, Xin Xu, Zongyi Hu, Wei Zheng, Yihong Ye, Tracey A Rouault, Charles M. Rice, Irving C Allen, T. J. Liang
          </td>
          <td>2024-12-03</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57740ae6cc36d8eceefd763d1df8e05a15264247" target='_blank'>
              A deep learning approach predicting the activity of COVID-19 therapeutics and vaccines against emerging variants
              </a>
            </td>
          <td>
            Robert P Matson, I. Comba, E. Silvert, Michiel J. M. Niesen, K. Murugadoss, Dhruti Patwardhan, R. Suratekar, Elizabeth-Grace Goel, Brittany J. Poelaert, Kanny K Wan, K. Brimacombe, A. Venkatakrishnan, V. Soundararajan
          </td>
          <td>2024-11-27</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05fa7450019e671626a9887682bd03540bccf7c4" target='_blank'>
              Mitigating the risks of post-acute sequelae of SARS-CoV-2 infection (PASC) with intranasal chlorpheniramine: perspectives from the ACCROS studies.
              </a>
            </td>
          <td>
            Fernando Valerio-Pascua, Fernando Baires, A. Sekhon, Mari L. Tesch, E. Pineda, Syed A A Rizvi, Jarmanjeet Singh, David Abraham Cortes-Bandy, Amy C. Madril, Jana Radwanski, Anita S Lewis, Miguel Sierra-Hoffman, Mark L. Stevens, Franck F. Rahaghi
          </td>
          <td>2024-11-26</td>
          <td>BMC infectious diseases</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="COVID-19 (Coronavirus disease 19) is caused by infection with SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) in the respiratory system and other organ systems. Tissue injuries resulting from viral infection and host hyperinflammatory responses may lead to moderate to severe pneumonia, systemic complications, and even death. While anti-inflammatory agents have been used to treat patients with severe COVID-19, their therapeutic effects are limited. GPR4 (G protein-coupled receptor 4) is a pro-inflammatory receptor expressed on vascular endothelial cells, regulating leukocyte infiltration and inflammatory responses. In this study, we evaluated the effects of a GPR4 antagonist, NE-52-QQ57, in the SARS-CoV-2-infected K18-hACE2 transgenic mouse model. Our results demonstrated that GPR4 antagonist treatment increased the survival rate in this severe COVID-19 mouse model. The inflammatory response, characterized by proinflammatory cytokines and chemokines, was reduced in the GPR4 antagonist group compared with the vehicle group. Additionally, both SARS-CoV-2 RNA copy numbers and infectious viral titers in the mouse lung were decreased in the GPR4 antagonist group. The percentage of SARS-CoV-2 antigen-positive mouse brains was also decreased in the GPR4 antagonist group compared to the vehicle group. Furthermore, the GPR4 antagonist inhibited SARS-CoV-2 propagation in Vero E6 cells. Together, these results suggest that GPR4 antagonism may be explored as a novel approach for the treatment of COVID-19 and other similar viral diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1717e32fe30fa040e0983d6008503ffb984362a" target='_blank'>
              The GPR4 antagonist NE-52-QQ57 increases survival, mitigates the hyperinflammatory response and reduces viral load in SARS-CoV-2-infected K18-hACE2 transgenic mice
              </a>
            </td>
          <td>
            Xin-Jun Wu, Karen A. Oppelt, Ming Fan, M. Marie, Madison M. Swyers, Ashley J. Williams, Isabelle M. Lemasson, Rachel L. Roper, Paul Bolin, Li V. Yang
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Monoclonal antibodies (mAbs) targeting the SARS-CoV-2 Spike protein were deployed during the COVID-19 pandemic. While all of the clinically authorized mAbs were eventually defeated by SARS-CoV-2 variants, they were highly effective in preventing disease progression when given early in the course of the disease. The experience with mAbs to SARS-CoV-2 offers important lessons for the use of mAbs in future infectious disease emergencies, such as choosing mAbs that target conserved epitopes and designing cocktails to reduce the emergence of escape variants. Planning for future use must include the creation of infusion centers and the development of strategies to minimize the emergence of escape variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/455ffcfc0d92f1201bea776d53cb6c667bedf4ea" target='_blank'>
              Lessons from the Use of Monoclonal Antibodies to SARS-CoV-2 Spike Protein During the COVID-19 Pandemic.
              </a>
            </td>
          <td>
            A. Casadevall, Daniele Focosi
          </td>
          <td>2024-12-04</td>
          <td>Annual review of medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Coronaviruses (CoVs) share key genomic elements critical for viral replication, suggesting the feasibility of developing therapeutics with efficacy across different viruses. In a previous work, we demonstrated the antiviral activity of the antipsychotic drug lurasidone against both SARS-CoV-2 and HCoV-OC43. In this study, our investigations on the mechanism of action of lurasidone suggested that the drug exhibits antiviral activity by targeting the papain-like protease (PL-Pro) of both viruses, and the Spike protein of SARS-CoV-2, thereby hampering both the entry and the viral replication. In vitro assays demonstrate that lurasidone significantly reduces viral load in infected cells, showing that the drug is a promising candidate for further development as a dual-action antiviral, offering a potential new strategy in the fight against COVID-19 and other coronavirus-related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4afadf908c1219f8566a34856714c1e4fd598a6" target='_blank'>
              The antipsychotic drug lurasidone inhibits coronaviruses by affecting multiple targets
              </a>
            </td>
          <td>
            Sara Baroni, Tea Carletti, M. Donalisio, Irene Arduino, Irene Cazzaniga, Toni Giorgino, F. Esposito, Alessia Porta, Luisa Diomede, Ada De Luigi, Marco Gobbi, D. Lembo, A. Marcello, E. Tramontano, Mario Milani, E. Mastrangelo
          </td>
          <td>2024-11-25</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="SARS-CoV-2 can cause clinical and inapparent disease and mortality in several animals cohabitating with humans, and sheep are susceptible to SARS-CoV-2 due to virus–receptor interactions similar to those in humans. Hence, sheep have the potential to be infected, spread, and develop neutralising antibodies (NAbs) against SARS-CoV-2. The aim of this study was to investigate the prevalence of SARS-CoV-2 NAbs in farm animals after natural exposure to the virus. Serum samples were collected from sheep in the Serra da Estrela region in Portugal, both prior to and during the COVID-19 pandemic. The sera were tested by established SARS-CoV-2 pseudovirus systems for multiple SARS-CoV-2 variants (early—Wuhan, mid—Delta, Omicron—BA.1, and late—Omicron XBB, BQ.1.1). Partial neutralisation activity in Pre-pandemic and Mid-pandemic samples was observed, while no NAb activity was observed in Late-pandemic samples tested. Different levels of NAbs were observed between Pre-pandemic samples and those collected during the Mid-pandemic and Late-pandemic periods (p ≤ 0.01). Our results indicate that SARS-CoV-2 cross-species transmission may have occurred through human–sheep contacts on sheep farms during the pandemic, and that farm animals could contribute to the One Health Approach in zoonotic virus surveillance and pandemic preparedness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/743282a174f0b236b4564cae6c3dea8be5662d29" target='_blank'>
              Investigating SARS-CoV-2 Neutralising Antibody Response in Sheep
              </a>
            </td>
          <td>
            Milena Samojlović, JR Mesquita, Sérgio Santos-Silva, Malin Neptin, J. Esbjörnsson
          </td>
          <td>2024-12-30</td>
          <td>Microorganisms</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The COVID-19 pandemic, caused by SARS-CoV-2, has underscored the urgent need for innovative therapeutic approaches. Recent studies have revealed a complex interplay between the circadian clock and SARS-CoV-2 infection in lung cells, opening new avenues for targeted interventions. This systems pharmacology study investigates this intricate relationship, focusing on the circadian protein BMAL1. BMAL1 plays a dual role in viral dynamics, driving the expression of the viral entry receptor ACE2 while suppressing interferon-stimulated antiviral genes. Its critical position at the host-pathogen interface suggests potential as a therapeutic target, albeit requiring a nuanced approach to avoid disrupting essential circadian regulation. To enable precise tuning of potential interventions, we constructed a computational model integrating the lung cellular clock with viral infection components. We validated this model against literature data to establish a platform for drug administration simulation studies using the REV-ERB agonist SR9009. Our simulations of optimized SR9009 dosing reveal circadian-based strategies that potentially suppress viral infection while minimizing clock disruption. This quantitative framework offers insights into the viral-circadian interface, aiming to guide the development of chronotherapy-based antivirals. More broadly, it underscores the importance of understanding the connections between circadian timing, respiratory viral infections, and therapeutic responses for advancing precision medicine. Such approaches are vital for responding effectively to the rapid spread of coronaviruses like SARS-CoV-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3890528cb3ee8471012b52716c256b605f84c4cb" target='_blank'>
              A theoretical systems chronopharmacology approach for COVID-19: Modeling circadian regulation of lung infection and potential precision therapies.
              </a>
            </td>
          <td>
            Yu-Yao Tseng
          </td>
          <td>2024-11-19</td>
          <td>CPT: pharmacometrics & systems pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="It is important to be able to retrospectively determine severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infections with high accuracy, both for post-coronavirus disease 2019 (COVID-19) epidemiological studies, and to distinguish between Long COVID and other multi-syndromic diseases that have overlapping symptoms. Although serum antibody levels can be measured to retrospectively diagnose SARS-CoV-2 infections, peptide stimulation of memory T cell responses is a more sensitive approach. This is because robust memory T cells are generated after SARS-CoV-2 infection and persist even after antibodies wane below detectability thresholds. In this study, we compare T cell responses using FluoroSpot-based methods and overnight stimulation of whole blood with SARS-CoV-2 peptides followed by an ELISA. Both approaches have comparable sensitivity and specificity but require different equipment and samples to be used. Furthermore, the elimination of peptides that cross-react with other coronaviruses increases the assay specificity but trades off some sensitivity. Finally, this approach can be used on archival, cryopreserved PBMCs. This work shows comparative advantages for several methods to measure SARS-CoV-2 T cell responses that could be utilized by any laboratory studying the effects of the coronavirus disease 2019 pandemic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76bc9067833dedebd3ed73f0817abbc2d4f678bf" target='_blank'>
              Comparison of different T cell assays for the retrospective determination of SARS-CoV-2 infection.
              </a>
            </td>
          <td>
            Ben Krishna, M. Metaxaki, Marianne Perera, Mark Wills, N. Sithole
          </td>
          <td>2024-12-01</td>
          <td>The Journal of general virology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Novel and repurposed antiviral drugs are available for the treatment of coronavirus disease 2019 (COVID-19). However, antiviral combinations may be more potent and lead to faster viral clearance, but the methods for screening antiviral combinations against respiratory viruses are not well established and labor-intensive. Here, we describe a time-efficient (72-96 h) and simple in vitro drug-sensitivity assay for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) using standard 96-well plates. We employ different synergy models (zero interaction potency, highest single agent, Loewe, Bliss) to determine the efficacy of antiviral therapies and synergistic combinations against ancestral and emerging clinical SARS-CoV-2 strains. We found that monotherapy of remdesivir, nirmatrelvir, and active metabolite of molnupiravir (EIDD-1931) demonstrated baseline EC50s within clinically achievable levels of 4.34 mg/L (CI: 3.74-4.94 mg/L), 1.25 mg/L (CI: 1.10-1.45 mg/L), and 0.25 mg/L (CI: 0.20-0.30 mg/L), respectively, against the ancestral SARS-CoV-2 strain. However, their efficacy varied against newer Omicron variants BA.1.1.15 and BA.2, particularly with the protease inhibitor nirmatrelvir. We also found that remdesivir and nirmatrelvir have a consistent, strong synergistic effect (Bliss synergy score >10) at clinically relevant drug concentrations (nirmatrelvir 0.25-1 mg/L with remdesivir 1-4 mg/L) across all SARS-CoV-2 strains tested. This method offers a practical tool that streamlines the identification of effective combination therapies and the detection of antiviral resistance. Our findings support the use of antiviral drug combinations targeting multiple viral components to enhance COVID-19 treatment efficacy, particularly in the context of emerging viral strains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea4cf31b0ee0f770b3925763785de3f93d6f8191" target='_blank'>
              Efficient in vitro assay for evaluating drug efficacy and synergy against emerging SARS-CoV-2 strains.
              </a>
            </td>
          <td>
            M. Woodall, Samuel Ellis, Shengyuan Zhang, Japhette E Kembou-Ringert, Kerry-Anne Kite, Laura Buggiotti, Amy I. Jacobs, A. Agyeman, T. Masonou, Machaela Palor, T. D. McHugh, J. Breuer, J. Standing, Claire M. Smith
          </td>
          <td>2024-12-17</td>
          <td>Antimicrobial agents and chemotherapy</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the causative agent of coronavirus disease 2019 (COVID-19), remains a significant threat to global public health. Immunopathological damage plays a role in driving pneumonia, acute respiratory distress syndrome (ARDS), and multiorgan failure in severe COVID-19. Therefore, dissecting the pulmonary immune response to SARS-CoV-2 infection is critical to understand disease pathogenesis and identify immune pathways targetable by therapeutic intervention. Considering that the type 2 cytokine IL-13 enhances COVID-19 disease severity, therapeutic targeting of upstream signals that drive type 2 immunity may confer further protection. In this study, we investigate the role of the IL-33/ST2 signaling axis, a potent inducer of type 2 immunity in the lung, in a mouse model of COVID-19. Upon infection with mouse-adapted SARS-CoV-2 MA10, ST2-/- mice had significantly improved weight loss and survival (69.2% vs 13.3% survival; P = 0.0005), as compared to wild-type mice. In a complementary pharmacologic approach, IL-33/ST2 signaling was inhibited using HpBARI_Hom2, a helminth derived protein that binds to mouse ST2 and blocks IL-33 signaling. In SARS-CoV-2 MA10 infection, HpBARI_Hom2-treated mice had significantly improved weight loss and survival (60% vs 10% survival; P = 0.0035), as compared to inert control-treated mice. These data demonstrate that loss of IL-33/ST2 signaling confers protection during acute SARS-CoV-2 MA10 infection, implicating the IL-33/ST2 signaling axis as an enhancer of COVID-19 disease severity. The protection conferred by pharmacologic blockade of IL-33/ST2 signaling was independent of viral control, as HpBARI_Hom2-treated mice had no reduction in viral titers. This finding suggests an immunopathogenic role for IL-33/ST2 signaling. One potential mechanism through which IL-33/ST2 signaling may drive severe disease is through enhancement of type 2 immune pathways including IL-5 production, as pulmonary IL-5 concentrations were found to depend on IL-33/ST2 signaling in acute SARS-CoV-2 MA10 infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5baf1b81c8384802417d37ffb56312ddf02da88" target='_blank'>
              The IL-33/ST2 signaling axis drives pathogenesis in acute SARS-CoV-2 infection
              </a>
            </td>
          <td>
            Claire Fleming, H. McSorley, J. Allen, William A. Petri
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="This study explores the relationship between specific SARS-CoV-2 mutations and obesity, focusing on how these mutations may influence COVID-19 severity and outcomes in high-BMI individuals. We analyzed 205 viral mutations from a cohort of 675 patients, examining the association of mutations with BMI, hospitalization, and mortality rates. Logistic regression models and statistical analyses were applied to assess the impact of significant mutations on clinical outcomes, including inflammatory markers and antibody levels. Our findings revealed three key mutations—C14599T, A20268G, and C313T—that were associated with elevated BMI. Notably, C14599T appeared to be protective against hospitalization, suggesting context-dependent effects, while A20268G was linked to a 50% increase in hospitalization risk and elevated antibody levels, potentially indicating an adaptive immune response. C313T showed a 428% increase in mortality risk, marking it as a possible poor-prognosis marker. Interestingly, all three mutations were synonymous, suggesting adaptive roles in obesity-driven environments despite not altering viral protein structures. These results emphasize the importance of studying mutations within the broader context of comorbidities, other mutations, and regional factors to enhance our understanding of SARS-CoV-2 adaptation in high-risk groups. Further validation in larger cohorts is necessary to confirm these associations and to assess their clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/859ccd96719b9e66c00e911f33260b65f4b1e5c3" target='_blank'>
              Impact of Obesity-Associated SARS-CoV-2 Mutations on COVID-19 Severity and Clinical Outcomes
              </a>
            </td>
          <td>
            Ana B. Martínez-Martinez, Alexander Tristancho-Baró, Beatriz Garcia-Rodriguez, Marina Clavel-Millan, M. P. Palacián, A. Milagro, A. Rezusta, J. Arbonés-Mainar
          </td>
          <td>2024-12-30</td>
          <td>Viruses</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c97e8db683625f06e7cea8cde2b0c156ba692b9" target='_blank'>
              Microbial Dynamics in COVID-19: Unraveling the Impact of Human Microbiome on Disease Susceptibility and Therapeutic Strategies.
              </a>
            </td>
          <td>
            Soumya Kanti Guha, Sougata Niyogi
          </td>
          <td>2024-12-25</td>
          <td>Current microbiology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="During the COVID-19 pandemic, while most infected individuals experienced mild to moderate symptoms, a significant subset developed severe illness. A clinical test distinguishing between mild and severe cases could inform effective treatment strategies. Toward the latter stages of the pandemic, it became evident that vaccination or prior infection cannot entirely prevent reinfection. However, they are crucial in reducing the risk of severe disease by inducing T-cell memory. T cell receptors (TCRs), which can be obtained from human blood, serve as valuable biomarkers for monitoring T cell responses to SARS-CoV-2 infection. In this study, we investigated the associations between TCR metrics and COVID-19 severity and found significant associations. Furthermore, such associations could depend on the subset of TCRs used (e.g., TCRs from CD8+ T or CD4+ T cells) and when the TCRs were collected.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc6f35cf359129707f021ba286236c89ea61af30" target='_blank'>
              Association between COVID-19 Disease Severity and T Cell Receptor Repertoire
              </a>
            </td>
          <td>
            Xinyi Chen, Yapeng Su, Chloe Dai, Jason D. Goldman, James R. Heath, Siming Liu, Wei Sun
          </td>
          <td>2024-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Since the outbreak of coronavirus disease 2019 (COVID-19) in late 2019 and early 2020, the identification of drugs to control severe acute respiratory syndrome coro-navirus 2 (SARS-CoV-2) infection and its symptoms has been a pressing focus of research. Cytokine storm and acute respiratory distress syndrome (ARDS) are the leading causes of mortality following infection. In this review, we discuss immune pathogenesis and four medications, including Remdesivir, Tocilizumab, Dexamethasone, and Annual SZ for COVID-19. A comparison of the effectiveness and therapeutic usage of drugs as reported in clinical trials and reports was made at different disease levels as well. Clinical studies indicate that Annual SZ with mild side effects was more affordable and might be more effective than other medications. Additionally, Annual SZ was capable of reducing the lev-els of pro-inflammatory cytokines as well as viral attachment and RNA replication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07fb451119b0209bb2685b8a0ade52189955e7df" target='_blank'>
              Annual SZ: An Alternative Immunotherapy for COVID-19 and Long COVID.
              </a>
            </td>
          <td>
            Fatemeh Heidari, Faranak Farahighasreaboonasr, Zuhair Mohammad Hassan, Pooria Fazeli, Maryam Hosseini, M. Ebtekar
          </td>
          <td>2024-12-20</td>
          <td>Infectious disorders drug targets</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2dfb69f02bfa21d02bbb353330dffd43640e4f1" target='_blank'>
              Neutralizing antibody test supports booster strategy for young individuals after SARS-CoV-2 Omicron breakthrough
              </a>
            </td>
          <td>
            Yichuan Yao, Yunru Yang, Qiqin Wu, Mengyao Liu, Wei Bao, Qiutong Wang, Meijun Cheng, Yunuo Chen, Yiting Yu, Yuan Cai, Mei Zhang, Jingxue Yao, Hongliang He, Changjiang Jin, Changcheng Zheng, Tengchuan Jin, Dali Tong
          </td>
          <td>2025-01-06</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea4a894c76f5b0d5264e7ac8126e7826261e49b6" target='_blank'>
              Is COVID-19 Still a Threat? An Expert Opinion Review on the Continued Healthcare Burden in Immunocompromised Individuals.
              </a>
            </td>
          <td>
            C. Spinner, Samira Bell, H. Einsele, Cécile Tremblay, Michel Goldman, Z. Chagla, Axel Finckh, Christopher J Edwards, I. Aurer, O. Launay, Carolina Casañas I Comabella, Samantha James, S. Dube, Katarzyna Borkowska, Fungwe Jah, Walid Kandeil, R. Yokota, Cécile Artaud, J. Gottenberg, L. Gesualdo, Dominique Bertrand, S. Arnetorp, G. Magiorkinis
          </td>
          <td>2024-12-16</td>
          <td>Advances in therapy</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="After the end of the COVID-19 public health emergency, we analysed the relationship between Systemic Lupus Erythematosous (SLE) and COVID-19 from the virologist's perspective based on recent findings. SLE and COVID-19 co-morbidity present unique challenges, as individuals with SLE may be at increased risk for severe COVID-19 illness due to immune system abnormalities and ongoing therapies. Effective management of both diseases requires careful monitoring, adherence to vaccination programs, preventive measures and approved and patient-tailored therapies. This review covers various aspects, including the clinical outcome of SLE patients infected by SARS-CoV-2, the impact of this infection on SLE onset or flare-ups and the benefits of vaccination for this population. Furthermore, this review presents the most recent recommendations on clinical management of COVID-19 in rheumatic patients, including those with SLE, discussing the currently available therapeutic options. Finally, we explore the most effective tools for SARS-CoV-2 diagnosis in autoimmune conditions and examine prognostic biomarkers in COVID-19 rheumatic patients with potential implications on their clinical oversight. By adopting a comprehensive approach, we address these complexities from the virologist's perspective, aiming to improve health care for this vulnerable population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/476d05353ecd35595cd2722e981e0249a4c1987c" target='_blank'>
              Lupus and SARS-CoV-2: What have we learned after the pandemic?
              </a>
            </td>
          <td>
            R. Francese, Massimo Rittà, David Lembo, M. Donalisio
          </td>
          <td>2024-12-17</td>
          <td>Lupus</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f4dc415894774ce156f27b45fb5fee1dbf04fa6" target='_blank'>
              Similar Kinetics of Pulmonary SARS-CoV-2 Load in Intensive Care Unit Patients with COVID-19 Pneumonia with or Without Autoantibodies Neutralizing Type I Interferons.
              </a>
            </td>
          <td>
            Valentine Le Stang, P. Bastard, Elise Langouet, M. Pineton de Chambrun, J. Chommeloux, A. Gervais, L. Bizien, Anne Puel, A. Cobat, J. Mayaux, Alexandre Demoule, J. Casanova, D. Boutolleau, Alain Combes, S. Burrel, C. Luyt
          </td>
          <td>2024-11-20</td>
          <td>Journal of clinical immunology</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="The severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2), causes acute coronavirus disease-19 (COVID-19) that has emerged on a pandemic level. Coronaviruses are well-known to have a negative impact on the lungs and cardiovascular system. SARS-CoV-2 induces a cytokine storm that primarily targets the lungs, causing widespread clinical disorders, including COVID-19. Although, SARS-CoV-2 positive individuals often show no or mild upper respiratory tract symptoms, severe cases can progress to acute respiratory distress syndrome (ARDS). Novel CoV-2 infection in 2019 resulted in viral pneumonia as well as other complications and extrapulmonary manifestation. ARDS is also linked to a higher risk of death. Now, it is essential to develop our perception of the long term sequelae coronavirus infection for the identification of COVID-19 survivors who are at higher risk of developing the chronic lung fibrosis. This review study was planned to provide an overview of the effects of SARS-CoV-2 infection on various parts of the respiratory system such as airways, pulmonary vascular, lung parenchymal and respiratory neuromuscular system as well as the potential mechanism of the ARDS related respiratory complications including the lung fibrosis in patients with severe COVID-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d283d5ea548a06245ea64ffa4c281208389a48" target='_blank'>
              COVID-19: A threat to the respiratory system
              </a>
            </td>
          <td>
            Ghulam Rasool, Waqas Ahmed Khan, Arif Muhammad Khan, Muhammad Riaz, Mazhar Abbas, Aziz Ur Rehman, Saba Irshad, Saeed Ahmad
          </td>
          <td>2024-01-01</td>
          <td>International Journal of Immunopathology and Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Although recommended isolation periods for Coronavirus disease 2019 (COVID-19) have been shortened as the pandemic has subsided, prolonged Severe Acute Respiratory Syndrome-Coronavirus-2 (SARS-CoV-2) shedding remains common in immunocompromised patients. This study estimated the probability of viral clearance in these patients based on elapsed days and specific risk factors.


METHODS
We prospectively enrolled immunocompromised patients with a confirmed COVID-19 diagnosis from January 2022 to May 2023 during the Omicron variant era. We collected weekly respiratory specimens for viral load measurement and culture. We identified significant predictors of viral culture negative conversion through univariate and multivariate analyses and estimated viral clearance probabilities using a Cox time-varying proportional hazard model.


RESULTS
Among 70 patients with serial 319 respiratory specimens with positive SARS-CoV-2 genomic polymerase chain reaction results that underwent cell culture, ∼69% (48) had haematologic malignancies and 31% (22) underwent solid organ transplants. B-cell depleting agents and viral copy number significantly influenced viral culture negative conversion. The probability of culture-negative conversion for immunocompromised patients not treated with B-cell-depleting agents increased over time, with over 90% achieving negative conversion by Day 84. Patients treated with B-cell depleting agents showed lower conversion rates. By Day 84, <90% of patients with cycle threshold values 23-28 [4.85-6.35 log copies/mL] achieved culture-negative conversion. The results indicate more prolonged shedding than in patients without B-cell depletion.


CONCLUSION
Estimating SARS-CoV-2 clearance probabilities based on specific risk factors can guide individualised isolation decisions for immunocompromised patients, tailoring policies to each patient's delayed viral clearance risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/176a50de4d46a97a00b283a86f459198e88b41cb" target='_blank'>
              Predicting persistent SARS-CoV-2 shedding in immunocompromised patients: a probability-based approach.
              </a>
            </td>
          <td>
            Euijin Chang, Jun-Won Kim, Choi-Young Jang, Ji Yeun Kim, S. Kang, S. Bae, Jiwon Jung, Min Jae Kim, YONG-PIL Chong, Sang-Oh Lee, Sang-Ho Choi, S. Yun, Yang Soo Kim, Jeong-Sun Yang, Kyung-Chang Kim, Joo-Yeon Lee, Sung-Han Kim
          </td>
          <td>2025-01-03</td>
          <td>Infectious diseases</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the etiological agent of the coronavirus disease 2019 (COVID-19) pandemic, remains endemic worldwide. Circulating levels of the chemokine CXCL10 are strongly positively associated with poor outcome; however, its precise role in SARS-CoV-2 pathogenesis and its suitability as a therapeutic target have remained undefined. Here, we challenged mice genetically deficient in Cxcl10 with a mouse-adapted strain of SARS-CoV-2. Infected male, but not female, Cxcl10-/- mice displayed increased mortality compared to wild type controls. Histopathological damage, inflammatory gene induction and virus load in the lungs of male mice were not broadly influenced by Cxcl10 deficiency. However, accumulation of B and T lymphocytes in the lung parenchyma of infected mice was reduced in the absence of Cxcl10. Thus, during acute SARS-CoV-2 infection, Cxcl10 regulates lymphocyte infiltration in lung and confers protection against mortality. Our preclinical model results do not support targeting CXCL10 therapeutically in severe COVID-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9771b78d58fa9333b82d633972313c2bf67faad" target='_blank'>
              Cxcl10 is protective during mouse-adapted SARS-CoV-2 infection.
              </a>
            </td>
          <td>
            S. Majumdar, Joseph D. Weaver, Sergio M. Pontejo, Mahnaz Minai, Xinping Lu, Ji-Liang Gao, Gibran Holmes, Reed F Johnson, Hongwei Zhang, Brian L. Kelsall, Joshua M. Farber, Derron A. Alves, Philip M Murphy
          </td>
          <td>2024-11-28</td>
          <td>Journal of leukocyte biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Abstract Background We examined the clinical effectiveness of molnupiravir in reducing deaths in a real-world cohort of adult patients with COVID-19 during the Omicron outbreak. Methods This was a population-wide retrospective cohort study in the Czech Republic. We analyzed all 74 541 patients with an officially registered diagnosis of SARS-CoV-2 infection between 1 January and 31 December 2022, aged 18 years or older, treated with molnupiravir. The primary outcome was 30-day all-cause mortality; the secondary outcome was 30-day COVID-19-related mortality. Hazard ratios (HRs) were estimated using stratified Cox regression and the Fine-Gray model. Results The use of molnupiravir in adult SARS-CoV-2 positive patients was associated with a lower risk of both 30-day all-cause mortality: adjusted HR 0.58 (95% confidence interval, 0.53–0.64; P < .001) and 30-day COVID-19–related mortality: adjusted HR 0.50 (95% confidence interval, 0.42–0.58; P < .001). The effect of molnupiravir was highly significant regardless of sex, Deyo-Charlson Comorbidity Index score, hospitalization status, COVID-19 vaccination status, and patients older than age 65 years. Conclusions In this cohort study, early initiation of molnupiravir was associated with a significant reduction in 30-day all-cause and COVID-19–related mortality in adult SARS-CoV-2 positive patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba187ab3a9aa0f68d939ef6b02bdb85a4d2581ae" target='_blank'>
              Real Clinical Effectiveness of Molnupiravir Against 30-day Mortality Among 74 541 SARS-CoV-2–Positive Patients: A Nationwide Cohort Study From the Czech Republic
              </a>
            </td>
          <td>
            Tomáš Pavlík, Jiří Jarkovský, Ondřej Šanca, Martina Koziar Vašáková, Pavel Dlouhý, Vladimír Černý, Petr Štourač, Vlastimil Válek, Ladislav Dušek
          </td>
          <td>2024-11-20</td>
          <td>Open Forum Infectious Diseases</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The global COVID-19 pandemic, caused by SARS-CoV-2, has led to significant morbidity and mortality, with a profound impact on cardiovascular health. This review investigates the mechanisms of SARS-CoV-2's interaction with cardiac tissue, particularly emphasizing the role of the Spike protein and ACE2 receptor in facilitating viral entry and subsequent cardiac complications. We dissect the structural features of the virus, its interactions with host cell receptors, and the resulting pathophysiological changes in the heart. Highlighting SARS-CoV-2's broad organ tropism, especially its effects on cardiomyocytes via ACE2 and TMPRSS2, the review addresses how these interactions exacerbate cardiovascular issues in patients with pre-existing conditions such as diabetes and hypertension. Additionally, we assess both direct and indirect mechanisms of virus-induced cardiac damage, including myocarditis, arrhythmias, and long-term complications such as 'long COVID'. This review underscores the complexity of SARS-CoV-2's impact on the heart, emphasizing the need for ongoing research to fully understand its long-term effects on cardiovascular health. Key words: COVID-19, Heart, ACE2, Spike protein, Cardiomyocytes, Myocarditis, Long COVID.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/266e41b959d3e3a6d83e96bbcd62e70babc37489" target='_blank'>
              Risk impact of SARS-CoV-2 coronavirus and spike protein on cardiac tissue: a comprehensive review.
              </a>
            </td>
          <td>
            O. Šerý, R. Dziedzinska
          </td>
          <td>2024-12-31</td>
          <td>Physiological research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The outbreak of COVID-19, led to an ongoing pandemic with devastating consequences for the global economy and human health. With the global spread of SARS-CoV-2, multidisciplinary initiatives were launched to explore new diagnostic, therapeutic, and vaccination strategies. From this perspective, proteomics could help to understand the mechanisms associated with SARS-CoV-2 infection and to identify new therapeutic options. A TMT-based quantitative proteomics and phosphoproteomics analysis was performed to study the proteome remodeling of human lung alveolar cells expressing human ACE2 (A549-ACE2) after infection with SARS-CoV-2. Detectability and the prognostic value of selected proteins was analyzed by targeted PRM. A total of 6802 proteins and 6428 phospho-sites were identified in A549-ACE2 cells after infection with SARS-CoV-2. The differential proteins here identified revealed that A549-ACE2 cells undergo a time-dependent regulation of essential processes, delineating the precise intervention of the cellular machinery by the viral proteins. From this mechanistic background and by applying machine learning modeling, 29 differential proteins were selected and detected in the serum of COVID-19 patients, 14 of which showed promising prognostic capacity. Targeting these proteins and the protein kinases responsible for the reported phosphorylation changes may provide efficient alternative strategies for the clinical management of COVID-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd9098cef45a8ea591819e5cb9c752c7b15027f" target='_blank'>
              Dynamic Cellular Proteome Remodeling during SARS-CoV-2 Infection. Identification of Plasma Protein Readouts
              </a>
            </td>
          <td>
            Fátima Milhano dos Santos, Jorge Vindel-Alfageme, Sergio Ciordia, Victoria Castro, Irene Orera, U. Garaigorta, Pablo Gastaminza, Fernando Corrales
          </td>
          <td>2024-11-26</td>
          <td>Journal of Proteome Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Coronavirus disease 2019 (COVID-19) still causes death in elderly and immunocompromised individuals, for whom the sustainability of the vaccine response may be limited. Antiviral treatments, such as remdesivir or molnupiravir, have demonstrated limited clinical efficacy. Nirmatrelvir, an acute respiratory syndrome coronavirus 2 (SARS-CoV-2) major protease inhibitor, is clinically effective but has been associated with viral rebound and antiviral resistance. It is thus necessary to study novel and repurposed antivirals for the treatment of COVID-19. We previously demonstrated that daclatasvir (DCV), an inhibitor of the hepatitis C virus (HCV) NS5A protein, impairs SARS-CoV-2 replication by targeting viral RNA polymerase and exonuclease, but the doses of DCV used to inhibit the new coronavirus are greater than the standard human plasma exposure for hepatitis C. Because any potential use of DCV against SARS-CoV-2 would be shorter than that reported here and short-term toxicological studies on DCV show that higher doses are tolerable, we searched for doses of DCV that could protect transgenic mice expressing the human ACE2 receptor (K18-hACE-2) from lethal challenge with SARS-CoV-2. We found that a dose of 60 mg/kg/day provides this protection by reducing virus replication and virus-induced lung insult. This dose is tolerable in different animal models. Taken together, our data provide preclinical evidence that can support phase I clinical trials to confirm the safety, tolerability, and pharmacokinetics of new doses of daclatasvir for a short duration in humans to further advance this compound’s utility against COVID-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cac01b54daac5a56e0406a0821642bfb7d787ef" target='_blank'>
              Newly Proposed Dose of Daclatasvir to Prevent Lethal SARS-CoV-2 Infection in Human Transgenic ACE-2 Mice
              </a>
            </td>
          <td>
            Mayara Mattos, Carolina Q. Sacramento, André C. Ferreira, N. Fintelman-Rodrigues, Filipe S. Pereira-Dutra, Caroline S. de Freitas, J. Gesto, J. Temerozo, Aline de Paula Dias Da Silva, Mariana T. G. Moreira, Rafael S. C. Silva, Gabriel P. E. Silveira, Douglas P. Pinto, H. M. Pereira, Laís B. Fonseca, Marcelo Alves Ferreira, Camilla Blanco, J. P. Viola, D. Bou-Habib, Patricia T. Bozza, T. M. L. Souza
          </td>
          <td>2024-11-29</td>
          <td>Viruses</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Genome-wide association studies performed in patients with coronavirus disease 2019 (COVID-19) have uncovered various loci significantly associated with susceptibility to SARS-CoV-2 infection and COVID-19 disease severity. However, the underlying cis-regulatory genetic factors that contribute to heterogeneity in the response to SARS-CoV-2 infection and their impact on clinical phenotypes remain enigmatic. Here, we used single-cell RNA-sequencing to quantify genetic contributions to cis-regulatory variation in 361,119 peripheral blood mononuclear cells across 63 COVID-19 patients during acute infection, 39 samples collected in the convalescent phase, and 106 healthy controls. Expression quantitative trait loci (eQTL) mapping across cell types within each disease state group revealed thousands of cis-associated variants, of which hundreds were detected exclusively in immune cells derived from acute COVID-19 patients. Patient-specific genetic effects dissipated as infection resolved, suggesting that distinct gene regulatory networks are at play in the active infection state. Further, 17.2% of tested loci demonstrated significant cell state interactions with genotype, with pathways related to interferon responses and oxidative phosphorylation showing pronounced cell state-dependent variation, predominantly in CD14+ monocytes. Overall, we estimate that 25.6% of tested genes exhibit gene-environment interaction effects, highlighting the importance of environmental modifiers in the transcriptional regulation of the immune response to SARS-CoV-2. Our findings underscore the importance of expanding the study of regulatory variation to relevant cell types and disease contexts and argue for the existence of extensive gene-environment effects among patients responding to an infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3dc4b0b10cafa790b68c9b9c45d0340142af04f" target='_blank'>
              Widespread gene-environment interactions shape the immune response to SARS-CoV-2 infection in hospitalized COVID-19 patients
              </a>
            </td>
          <td>
            Haley E. Randolph, Raúl Aguirre-Gamboa, Elsa Brunet-Ratnasingham, T. Nakanishi, Veronica Locher, Ellen Ketter, C. Brandolino, Catherine Larochelle, Alexandre Prat, Nathalie Arbour, A. Dumaine, Andrés Finzi, Madeleine Durand, J. B. Richards, Daniel E Kaufmann, Luis B Barreiro
          </td>
          <td>2024-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc9d1b0cbbfd015f55118b992113459d381d924a" target='_blank'>
              Across-the-board review on Omicron SARS-CoV-2 variant.
              </a>
            </td>
          <td>
            Rufaida Wasim, Sumaiya, Asad Ahmad
          </td>
          <td>2024-12-23</td>
          <td>Inflammopharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Introduction The coronavirus disease 2019 (COVID-19) global pandemic has been the most severe public health emergency since 2019. Currently, the Omicron variant of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has been the most dominant. The most prominent symptom of SARS-CoV-2 infection is respiratory. Meanwhile, the fatality of COVID-19 was mainly from pneumonia. However ,in patients with SARS-CoV-2 infection who have pneumonia and those who do not, the differences in the immune repertoire still require further investigation. Methods We conducted seven-chain adaptome immune repertoire analyses on patients with SARS-CoV-2 Omicron infection, both with and without pulmonary infiltration. Results Patients with pulmonary infiltration exhibit lymphopenia, a decreased proportion of the overall TCR repertoire alongside an increased BCR repertoire, reduced IGHD and IGHM isotype expression, a shorter mean CDR3 length for TRG, and a longer mean length for TRD, as well as diminished clonality and diversity in the TCR/BCR repertoire. Meanwhile, patients with pulmonary infiltration have distinct V-J gene usage and unique CDR3 signature, as well as BCR class switch recombination pattern. Finally, prior vaccination triggered less BCR IGHM/IGHD somatic hypermutation response, preserved the diversity of the entire adaptive immune repertoire, and provided clinical protection against severe or critical conditions following Omicron infection. Discussion We report a unique, comprehensive adaptive immune system signature in patients with pulmonary infiltration, which may serve as potential immunological biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d4ed936a031f9b310c6148e0d4110b111bb361f" target='_blank'>
              A comprehensive immune repertoire signature distinguishes pulmonary infiltration in SARS-CoV-2 Omicron variant infection
              </a>
            </td>
          <td>
            Xuechuan Li, Hongyi Zhu, Peipei Xu, Jie Zhang, Zhe Wang, Hui He, Fang Shen, Yi Jiang, Lijuan Shen, Jing Xiang, Linhua Yang, Chao Yang, Hao Jiang, Ganglong Gao, Junshuo Jin, Huojian Shen, Yinping Wang, Linshi Wu, Changlin Qian, Dejun Liu, Weiqing Qiu, Qiwei Li, Yuanwen Chen, Fujun Lin, Yun Liu
          </td>
          <td>2024-12-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="In 2019—2023, the world experienced a Public Health Emergency of International Concern related to the pandemic of the novel coronavirus disease caused by SARS-CoV-2, declared by the World Health Organization (WHO), during which the virus mutated and spread, changing its transmissibility, infectivity, and ability to cause severe forms of the disease. Since the beginning of the pandemic, the number of reports of skin manifestations accompanying infection caused by the ­SARS-CoV-2 virus has increased. 
Objective — to summarize data on the specific features of skin manifestations of SARS-CoV-2 infection during the COVID-19 pandemic and to improve further research in this direction.
Materials and methods. The bibliosemantic research method was used. The research materials are publications in foreign and Ukrainian scientific journals related to the meta-analysis of studies of skin manifestations in COVID-19, as well as individual clinical observations.
Results and discussion. A summary of a number of studies in the meta-analysis format conducted on materials from patient contingents in European, Asian and American countries showed the early appearance of skin manifestations in about 10 % of patients with COVID-19 before the onset of the respiratory process. Skin manifestations can occur in both asymptomatic/paucisymptomatic patients and patients with severe disease. No clear correlation was found between the severity of typical clinical manifestations of COVID-19, viral load and the nature of skin manifestations. The most common cutaneous manifestations of SARS-CoV-2 infection are urticaria, acral pernio-like lesions, maculopapular lesions, vesicular «pox-like» and «morbilliform» erythematous lesions, and skin lesions resembling erythema multiforme, livedo reticularis, and livedo racemosa.
Conclusions. Dermatologists can play an important role in the response to the COVID-19 pandemic by early recognition of skin lesions, especially in paucisymptomatic infections, where this recognition can guide early diagnosis of the infection, which certainly leads to a better prognosis. Skin manifestations can be a prominent feature of SARS-CoV-2 infection, and these lesions may be poorly recognized due to the lack of routine dermatological consultations during the pandemic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fb3d9366a36d3b9bcd58e99112d7d8d69919ea4" target='_blank'>
              Skin manifestations of infection caused by the SARS-CoV-2 virus during its pandemic (review)
              </a>
            </td>
          <td>
            V. V. Korolenko, V. V. Korolenko
          </td>
          <td>2024-12-30</td>
          <td>Ukrainian Journal of Dermatology, Venerology, Cosmetology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The variants of concern (VOCs) of SARS-CoV-2 have exhibited different phenotypic characteristics in clinical settings which are yet to be fully explored. This study aimed to characterize the viral replication features of major VOCs of SARS-CoV-2 and their association with pathogenicity. The Alpha, Delta, and Omicron variants of SARS-CoV-2 isolated from the COVID-19 patients in Japan were propagated in VeroE6/TMPRSS2 cells. The viral replication and pathological features were evaluated by laser and electron microscopy at different time points. The results revealed that the Delta variant dominantly infected the VeroE6/TMPRSS2 cells and formed increased syncytia compared to the Alpha and Omicron variants. Relatively large numbers of virions and increased immunoreactivities of the SARS-CoV-2 N-protein were detected in the endoplasmic reticulum and intracellular vesicles of Delta-infected cells. Interestingly, the N-protein and virions were detected in the nucleus of Delta-infected cells, while such properties were not observed in the case of Alpha and Omicron variants. In addition, early nuclear membrane damage followed by severe cellular damage was prominent in Delta-infected cells. A unique mutation (G215C) in the N-protein of the Delta variant is thought to be associated with severe cell damage. In conclusion, this study highlights the distinct replicative and pathogenic characteristics of the Delta variant of SARS-CoV-2 compared to the Alpha and Omicron variants, shedding light on the potential mechanisms underlying its increased pathogenicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b651cc14ed8cd1a951bdfb4e54787ddb9cddf8e1" target='_blank'>
              Exploring the Replication and Pathogenic Characteristics of Alpha, Delta, and Omicron Variants of SARS-CoV-2
              </a>
            </td>
          <td>
            Sakirul Khan, Takaaki Yahiro, K. Kimitsuki, Takehiro Hashimoto, Keiko Matsuura, Shinji Yano, Kazuko Noguchi, Akane Sonezaki, Kaori Yoshizawa, Yoko Kumasako, S. M. Akbar, Akira Nishizono
          </td>
          <td>2024-11-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The Coronavirus disease 2019 (COVID-19) pandemic caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has resulted in an unprecedented public health crisis. Owing to the novelty of the virus and its high mutability, there are currently limited SARS-CoV-2-specific treatments available despite extensive research that has been performed on the virus. Therefore, rapid development of effective therapies, especially small molecules against SARS-CoV-2, is urgently needed. In this study, we targeted three core structural proteins of the virus, spike glycoprotein, replicase polyprotein 1a, and replicase polyprotein 1ab, which have been implicated in advancing viral infection. Using our GenAI system, we generated forty-seven novel small molecules targeting these three proteins. We further tested, in silico, how these molecules compared to the existing drugs, Paxlovid and Remdesivir. Our data showed that several of the predicted molecules outperform existing drugs with higher binding affinities. Our system presents a much quicker and inexpensive method to develop small-molecule drugs against specific protein targets, representing an important advancement in the fight against novel viruses and their resulting diseases. Keywords: SARS-CoV-2, Small molecules, Spike protein, GenAI, Drug Discovery Aims Research Journal Reference Format: Obi, E. D., Yentumi, J.A, Mbatuegwu, D., Ayobami, F. & Obi, T. (2024). Generating Novel Small Molecule Drugs for Selected SARS-CoV-2 Proteins: The Medgnosis GenAI Approach. Advances in Multidisciplinary and Scientific Research Journal Vol. 10. No. 4. Pp 7-18. www.isteams.net/aimsjournal. dx.doi.org/10.22624/AIMS/V10N4P1">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00e0e3b3e6e0c6eef86cf377eac3fc4b8b017add" target='_blank'>
              Generating Novel Small Molecule Drugs for Selected SARS-CoV-2 Proteins: The Medgnosis GenAI Approach
              </a>
            </td>
          <td>
            E.O Obi, J.A. Yentumi, D. Mbatuegwu, F. Ayobami, T. Obi
          </td>
          <td>2024-11-21</td>
          <td>Advances in Multidisciplinary &amp; Scientific Research Journal Publication</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Rationale: Although the mortality of severe coronavirus disease 2019 (COVID-19) has decreased after the emergence of the Omicron variant, it remains high in patients on hemodialysis (HD). Remdesivir (RDV) is considered as the first line drug for hospitalized COVID-19 patients, however the evidence regarding the usage in HD patients is lacking because clinical trials of RDV have excluded HD patients for safety reasons. Thus, accumulation of knowledge on the regimen, efficacy, and tolerability of RDV in HD patients is important. Patient concerns: A nosocomial COVID-19 cluster was occurred from August 31 to October 12 in 2021 when the Delta variant was predominant. During the cluster, 11 health-care workers and 20 inpatients including 7 HD patients were infected with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Diagnoses: The diagnosis of COVID-19 was confirmed by the real-time polymerase chain reaction (PCR) for SARS-CoV-2. Interventions: RDV was initiated within 16 hours after the onset of fever (≥ 37.4°C) or positive PCR result in all HD patients, and continued at 100 mg/day intravenously once daily for either consecutive 5 or 10 days. Outcomes: All patients fully recovered within 2 weeks and did not develop severe COVID-19. Two patients experienced mild liver dysfunction, but it was temporary and remitted spontaneously even continuing RDV treatment. Discontinuation of RDV therapy due to adverse events was not required in any patients. Lessons: Present cases indicated early intervention with RDV may contribute the favorable outcome and daily administration of RDV for up to 10 days was well tolerated even in HD patients. Literature review showed no previous article reported the efficacy and safety of such earlier and longer administration of remdesivir as in the present cases, therefore this report is informative for clinicians to consider the usage of RDV in HD patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59d270f8744239ba5ea0f913b37ea4ccbc2a0cca" target='_blank'>
              Efficacy and safety of early administration of remdesivir in hemodialysis patients with COVID-19: A case report and literature review
              </a>
            </td>
          <td>
            Nanako Oshiro, T. Kinjo, Daigo Aharen, Yuki Kudo, Eisuke Katsuren, Kumiko Omine, Takuto Nakamura, Ryo Zamami, A. Ishida, K. Miyagi, M. Nakamatsu, Kazuko Yamamoto, K. Kusunose, Jiro Fujita, Yusuke Ohya, K. Kohagura
          </td>
          <td>2024-11-29</td>
          <td>Medicine</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f30dc589774d6151de03a03e218b4b78da3dac7" target='_blank'>
              Phenotypic evolution of SARS-CoV-2 spike during the COVID-19 pandemic.
              </a>
            </td>
          <td>
            Wilhelm Furnon, V. Cowton, G. De Lorenzo, R. Orton, V. Herder, Diego Cantoni, G. Ilia, Diogo Correa Mendonca, Karen Kerr, Jay Allan, Nicole Upfold, Gavin R Meehan, S. Bakshi, Udeet Ranjan Das, Sergi Molina Arias, Marion McElwee, Sarah Little, Nicola Logan, Kirsty Kwok, Katherine Smollett, Brian J. Willett, A. da Silva Filipe, David L Robertson, Joe Grove, Arvind H Patel, Massimo Palmarini
          </td>
          <td>2025-01-03</td>
          <td>Nature microbiology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="During the COVID-19 pandemic the effectiveness of vaccines against SARS-CoV-2 in immunodeficient patients did not only affect the individual risk of these vulnerable patients but endangered the selection of new variants of concern due to prolonged virus shedding by these patients. In a tertiary center for pulmonary diseases, we investigated the immune response of 11 patients with primary humoral immunodeficiency and 13 healthy controls on the humoral and cellular level after full vaccination with a mRNA or vector vaccine against SARS-CoV-2. In the majority of patients (73%), we found antibodies against the Spike protein above the thresh-old of positivity. Likewise, patients showed a promising cellular response: the upregulated production of INFγ, TNFα and CXCL10 by T cells did not differ from the response of healthy controls. These results stress the importance to further discern an adequate immunological correlate of protection and the need to follow the effect of booster immunizations in this population at risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a650db8fa44fe06a2f15d588827ce5a25f540f9" target='_blank'>
              SARS-CoV-2-vaccination in primary humoral immunodeficiency - Experience from a German lung clinic.
              </a>
            </td>
          <td>
            Sophie Rosendahl, F. Trudzinski, M. Polke, F. Herth, Michael Kreuter, Thomas Giese
          </td>
          <td>2024-12-19</td>
          <td>Respiration; international review of thoracic diseases</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction: COVID-19 manifests with symptoms ranging from mild to severe, including respiratory and cardiovascular complications. Severe cases are often marked by a cytokine storm, causing pulmonary damage and multi-organ dysfunction. Current treatments lack a specific antiviral, though several pharmacological options, including immunomodulatory and antiviral drugs, are under investigation. 
Purpose: The purpose of this article is to provide an evaluation of the efficacy and safety of baricitinib, a JAK inhibitor, combined with standard care for hospitalized adults with COVID-19. By reviewing recent literature, it aims to assess baricitinib's effectiveness in improving patient outcomes. 
Material and methods: A systematic review of clinical trials and case studies was conducted to analyze baricitinib’s impact on COVID-19 mortality. Data includes treatment-emergent adverse events and patient outcomes in moderate to severe cases. 
Discussion: Baricitinib has demonstrated potential in managing hyperinflammation in severe COVID-19 by inhibiting multiple proinflammatory cytokines and enhancing immune response. Clinical trials show that baricitinib, particularly in combination with corticosteroids, reduces mortality and serious infections compared to standard care alone. Due to potential adverse events, continuous monitoring of hematological and lipid levels is essential. These findings highlight baricitinib's dual role as an immunomodulatory and antiviral agent, suggesting its importance in treating severe COVID-19. Future research should focus on optimizing treatment regimens and further exploring the long-term implications of baricitinib therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3b09cfadd6cc9d3ad08e9bf5b6d31268d4aae44" target='_blank'>
              Evaluating the Efficacy and Safety of Baricitinib in Reducing COVID-19 Mortality: A Comprehensive Review of Clinical Evidence and Pharmacological Strategies
              </a>
            </td>
          <td>
            P. Kwaśniewska, Patryk Graczyk, Anna Wilewska, Kinga Borowiec, Agnieszka Borowiec, Julia Biernikiewicz, Konstanty Alabrudziński, Milena Biernikiewicz, Bartosz Pomirski, Agata Pomirska
          </td>
          <td>2024-12-08</td>
          <td>Journal of Education, Health and Sport</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="During the COVID-19 pandemic, masks were widely promoted and mandated as a key measure to help reduce the transmission of SARS-CoV-2. These policies were primarily informed by laboratory evidence demonstrating the effectiveness of particle filtration, alongside observational studies. While several meta-analyses have indicated that masks may contribute to reducing viral transmission, many of these analyses rely heavily on observational data. There also appears to be a trend where the inclusion of more randomized controlled trials in a meta-analysis is associated with a lower estimate of mask effectiveness. It is important to recognize that success in laboratory settings does not always directly translate to the same outcomes in clinical trials or real-world conditions. This phenomenon is often seen in drug development, where therapies with promising mechanistic evidence may not always perform as expected in trials. In this regard, masks share similarities with other interventions that, while theoretically sound, require further testing in varied contexts to fully assess their real-world impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99dd7b4c013f57d26988ab86fc49ba49f223e74b" target='_blank'>
              Reevaluating Mask Effectiveness: Insights From Evidence-Based Medicine and Clinical Trials
              </a>
            </td>
          <td>
            Beny Spira
          </td>
          <td>2024-12-01</td>
          <td>Cureus</td>
          <td>0</td>
          <td>0</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>