<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              40659e189c761411bfdb0dd31a07f7925e41adf0
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-21 10:42:11 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>A hidden oncogenic positive feedback loop caused by crosstalk between Wnt and ERK Pathways</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="Cell responses are actuated by tightly controlled signal transduction pathways. Although the concept of an integrated signaling network replete with interpathway cross-talk and feedback regulation is broadly appreciated, kinetic data of the type needed to characterize such interactions in conjunction with mathematical models are lacking. In mammalian cells, the Ras/ERK pathway controls cell proliferation and other responses stimulated by growth factors, and several cross-talk and feedback mechanisms affecting its activation have been identified. In this work, we take a systematic approach to parse the magnitudes of multiple regulatory mechanisms that attenuate ERK activation through canonical (Ras-dependent) and non-canonical (PI3K-dependent) pathways. In addition to regulation of receptor and ligand levels, we consider three layers of ERK-dependent feedback: desensitization of Ras activation, negative regulation of MEK kinase (e.g. Raf) activities, and up-regulation of dual-specificity ERK phosphatases. Our results establish the second of these as the dominant mode of ERK self-regulation in mouse fibroblasts. We further demonstrate that kinetic models of signaling networks, trained on a sufficient diversity of quantitative data, can be reasonably comprehensive, accurate, and predictive in the dynamical sense.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405be6d692d4836d3b3efde9333fdf1789ff05c7" target='_blank'>Systematic Quantification of Negative Feedback Mechanisms in the Extracellular Signal-regulated Kinase (ERK) Signaling Network*</a></td>
          <td>
            M. Cirit, Chun-Chao Wang, J. Haugh
          </td>
          <td>2010-09-16</td>
          <td>The Journal of Biological Chemistry</td>
          <td>91</td>
          <td>33</td>
        </tr>

        <tr id="Cellular proccess that are crucial for health or disease are orchestrated by the expression of multiple genes in a network of differential complex signaling pathways, consisting of many physical and functional interactions.Unlike the stable genome structure, the highly dynamic interactome represents a major challenge for quantitative mapping of protein–protein interactions (PPIs). In this article, I will discuss the latest systems science advances by using breakthrough technologies to gain an understanding of the whole picture of the spatiotemporal signaling network as a response to dynamic individual genome–lifestyle interactions and translating it to the clinic.Biological signals control cells’ functions and their organization into tissues, organs and a whole organism’s homeostasis. A variety of extracellular and intracellu-lar signaling molecules include hormones, growth factors, cytokines, chemokines, neurotransmitters, extracellular matrix components, receptors, protein kinases, protein phosphatases and DNA-binding proteins in the nucleus. A signal is trans-mitted or amplified from the cell mem-brane to the nucleus through a dynamic complex interactions network">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2702c101690829e49078cf33e2ee81dc4f49e616" target='_blank'>Differential signaling transduction networks for clinical robustness</a></td>
          <td>
            D. Roukos
          </td>
          <td>2012-04-01</td>
          <td>Expert Review of Proteomics</td>
          <td>12</td>
          <td>62</td>
        </tr>

        <tr id="Epistasis among driver mutations is pervasive and explains relevant features of cancer, such as differential therapy response and convergence towards well-characterized molecular subtypes. Furthermore, a growing body of evidence suggests that tumor development could be hampered by the accumulation of slightly deleterious passenger mutations. In this work, we combined empirical epistasis networks, computer simulations, and mathematical models to explore how synergistic interactions among driver mutations affect cancer progression under the burden of slightly deleterious passengers. We found that epistasis plays a crucial role in tumor development by promoting the transformation of precancerous clones into rapidly growing tumors through a process that is analogous to evolutionary rescue. The triggering of epistasis-driven rescue is strongly dependent on the intensity of epistasis and could be a key rate-limiting step in many tumors, contributing to their unpredictability. As a result, central genes in cancer epistasis networks appear as key intervention targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c91226be0555313595065598908b1bdc235666f1" target='_blank'>Synergistic epistasis among cancer drivers can rescue early tumors from the accumulation of deleterious passengers</a></td>
          <td>
            Carla Alejandre, Jorge Calle-Espinosa, Jaime Iranzo
          </td>
          <td>2024-04-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cell responses are actuated by tightly controlled signal transduction pathways. Although the concept of an integrated signaling network replete with interpathway cross-talk and feedback regulation is broadly appreciated, kinetic data of the type needed to characterize such interactions in conjunction with mathematical models are lacking. In mammalian cells, the Ras/ERK pathway controls cell proliferation and other responses stimulated by growth factors, and several cross-talk and feedback mechanisms affecting its activation have been identified. In this work, we take a systematic approach to parse the magnitudes of multiple regulatory mechanisms that attenuate ERK activation through canonical (Ras-dependent) and non-canonical (PI3K-dependent) pathways. In addition to regulation of receptor and ligand levels, we consider three layers of ERK-dependent feedback: desensitization of Ras activation, negative regulation of MEK kinase (e.g. Raf) activities, and up-regulation of dual-specificity ERK phosphatases. Our results establish the second of these as the dominant mode of ERK self-regulation in mouse fibroblasts. We further demonstrate that kinetic models of signaling networks, trained on a sufficient diversity of quantitative data, can be reasonably comprehensive, accurate, and predictive in the dynamical sense.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405be6d692d4836d3b3efde9333fdf1789ff05c7" target='_blank'>Systematic Quantification of Negative Feedback Mechanisms in the Extracellular Signal-regulated Kinase (ERK) Signaling Network*</a></td>
          <td>
            M. Cirit, Chun-Chao Wang, J. Haugh
          </td>
          <td>2010-09-16</td>
          <td>The Journal of Biological Chemistry</td>
          <td>91</td>
          <td>33</td>
        </tr>

        <tr id="TGF-β, an important cytokine that plays a key role in many diseases regulates a wide array of cellular and physiologic processes via several TGF-β-driven signaling cascades, including the SMAD and non-SMAD-driven pathways. However, the detailed mechanisms by which TGF-β induces such diverse responses remain poorly understood. In particular, compared to the SMAD-dependent pathway, SMAD-independent pathways such as the ERK/MAPK pathway, which is critical in cancer progression, are less characterized. Here, we develop an integrated mechanistic model of the TGF-β-triggered ERK activation pathway and its crosstalk with the SMAD pathway, an analysis of which demonstrates how SMAD dynamics can be significantly modulated and regulated by the ERK pathway. In particular, SMAD-mediated transcription can be altered and delayed due to expedited phosphorylation of the linker of SMAD by TGF-β-activated ERK; and enhanced ERK activity, but attenuated SMAD activity, can be achieved simultaneously by fast turnover of TGF-β receptors via lipid-rafts. Also, in silico mutations of the TGF-β pathways reveal that the dynamic characteristics of both SMAD and ERK signaling may change significantly during cancer development. Specifically, normal cells may exhibit enhanced and sustained SMAD signaling with transient ERK activation, whereas cancerous cells may produce elevated and prolonged ERK signaling with enervated SMAD activation. These distinctive differences between normal and cancerous signaling behavior provide clues concerning, and potential explanations for, the seemingly contradictory roles played by TGF-β during cancer progression. We demonstrate how crosstalk among various branch pathways of TGF-β can influence overall cellular behavior. Based on model analysis, we hypothesize that aberrant molecular alterations drive changes in the intensity and duration of SMAD and ERK signaling during cancer progression and ultimately lead to an imbalance between the SMAD and ERK pathways in favor of tumor promotion. Thus, to treat cancer patients with a genetic signature of oncogenic Ras effectively may require at least a combination therapy to restore both the expression of TGF-β receptors and the GTPase activity of Ras.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e15eee20cb91183ca7ba0fc4da9fd5299f68c7f" target='_blank'>Computational Modeling and Analysis of the TGF-β-induced ERK and SMAD Pathways</a></td>
          <td>
            Seung-Wook Chung, Carlton R. Cooper, M. Farach‐Carson, B. Ogunnaike
          </td>
          <td>2024-11-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Oncogenic mutations in Ras GTPases result in overactivation of the Erk-MAPK pathway, which regulate cellular proliferation, and have been shown to have strong clinical correlations with a variety of pancreatic, colonic, and other cancers. To date, a viable inhibitor of Ras has remained elusive and most Ras mutation cancers are treated with downstream inhibitors to reduce the activity of ERK. However, these therapies have been shown to provide minor anticancer effects in some patients. To examine other viable therapeutic alternatives, potential drug actions that activate "off" mechanisms of this pathway are examined in a computational model. The described model recapitulates empirical data of the Erk-MAPK pathway under EGF-stimulation and is modified to include the effects of a constitutively active Ras. Serving as an experimental predictor of the potential drug actions against the effects of constant Ras activation, activators of protein phosphatase 2 (PP2A) and Raf1 phosphatase catalytic activity are shown to be able to restore Erk responses in such cancers to dynamic responses that are seen in wild-type EGF-stimulated cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af69827096a6688cdc58a1021e1e1f93a8589ba3" target='_blank'>Ras mutations in the Erk-MAPK pathway and Cancer: Alternate therapies examined in an ODE-based computational model</a></td>
          <td>
            Jiarui Li
          </td>
          <td>2020-09-11</td>
          <td>Proceedings of the 1st International Symposium on Artificial Intelligence in Medical Sciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="To investigate the unregulated Ras activation associated with cancer, we developed and validated a mathematical model of Ras signaling. The model-based predictions and associated experiments help explain why only one of two classes of activating Ras point mutations with in vitro transformation potential is commonly found in cancers. Model-based analysis of these mutants uncovered a systems-level process that contributes to total Ras activation in cells. This predicted behavior was supported by experimental observations. We also used the model to identify a strategy in which a drug could cause stronger inhibition on the cancerous Ras network than on the wild-type network. This system-level analysis of the oncogenic Ras network provides new insights and potential therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7537afb7c22d6da2070fdb14b037b930aa4022f3" target='_blank'>Network Analysis of Oncogenic Ras Activation in Cancer</a></td>
          <td>
            E. Stites, P. Trampont, Zhong Ma, K. Ravichandran
          </td>
          <td>2007-10-19</td>
          <td>Science</td>
          <td>118</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6319b79882fa8bd07267c8cafc082740cd9779b6" target='_blank'>Computational modeling of the EGFR network elucidates control mechanisms regulating signal dynamics</a></td>
          <td>
            Dennis Wang, L. Cardelli, Andrew Phillips, Nir Piterman, J. Fisher
          </td>
          <td>2009-12-22</td>
          <td>BMC Systems Biology</td>
          <td>40</td>
          <td>74</td>
        </tr>

        <tr id="Extra-cellular signal regulated kinase (ERK) signal network plays an important role in regulating the fundamental cellular functions such as cell proliferation, survival, differentiation and motility. Development and analysis of mathematical model can help us acquire a deep understanding of the complex behavior of ERK signal transduction network. This paper presents a computational model that offers an integrated quantitative and dynamic simulation of ERK signal transduction network, activated by epidermal growth factor (EGF). The mathematic model contains the activation kinetics of the pathway, a multitude of feedback loops and connection of scaffold proteins. The model provides insight into signal-response relationships between the binding of EGF to its receptor at the cell surface and activation of downstream proteins in the signaling cascade. The different influence of positive and negative feedback loops of the ERK signal transduction pathway were mainly analyzed, indicating that feedback loops were the main impacting factor to the oscillation of ERK signal transduction pathway. The predictions of this oscillation of ERK activation agree well with the literature. It can lead to signal waves of the downstream substrates and induce corresponding biological behaviors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7bf832c4d17a797f80bffbcb87cce1c1078a91f" target='_blank'>Dynamic Analysis of Feedback Loops in Extra-Cellular Signal Regulated Kinase Signal Network</a></td>
          <td>
            Yong Han, Yan Li, Yonghua Wang, H. Fang, Shuwei Zhang
          </td>
          <td>2009-03-31</td>
          <td>2009 WRI World Congress on Computer Science and Information Engineering</td>
          <td>3</td>
          <td>34</td>
        </tr>

        <tr id="Precision medicine has changed thinking in cancer therapy, highlighting a better understanding of the individual clinical interventions. But what role do the drivers and pathways identified from pan-cancer genome analysis play in the tumor? In this letter, we will highlight the importance of in silico modeling in precision medicine. In the current era of big data, tumor engines and pathways derived from pan-cancer analysis should be integrated into in silico models to understand the mutational tumor status and individual molecular pathway mechanism at a deeper level. This allows to pre-evaluate the potential therapy response and develop optimal patient-tailored treatment strategies which pave the way to support precision medicine in the clinic of the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3d85cd48654368783018b1d69f1208b8adc1339" target='_blank'>In silico signaling modeling to understand cancer pathways and treatment responses</a></td>
          <td>
            Meik Kunz, J. Jeromin, M. Fuchs, J. Christoph, G. Veronesi, M. Flentje, Sarah Nietzer, Gudrun Dandekar, T. Dandekar
          </td>
          <td>2020-05-21</td>
          <td>Briefings in bioinformatics</td>
          <td>7</td>
          <td>72</td>
        </tr>

        <tr id="A Boolean dynamical system integrating the main signaling pathways involved in cancer is constructed based on the currently known protein-protein interaction network. This system exhibits stationary protein activation patterns – attractors – dependent on the cell's microenvironment. These dynamical attractors were determined through simulations and their stabilities against mutations were tested. In a higher hierarchical level, it was possible to group the network attractors into distinct cell phenotypes and determine driver mutations that promote phenotypic transitions. We find that driver nodes are not necessarily central in the network topology, but at least they are direct regulators of central components towards which converge or through which crosstalk distinct cancer signaling pathways. The predicted drivers are in agreement with those pointed out by diverse census of cancer genes recently performed for several human cancers. Furthermore, our results demonstrate that cell phenotypes can evolve towards full malignancy through distinct sequences of accumulated mutations. In particular, the network model supports routes of carcinogenesis known for some tumor types. Finally, the Boolean network model is employed to evaluate the outcome of molecularly targeted cancer therapies. The major find is that monotherapies were additive in their effects and that the association of targeted drugs is necessary for cancer eradication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f953c36291fe5927f1caed5ea3236ace33bf67de" target='_blank'>Boolean Network Model for Cancer Pathways: Predicting Carcinogenesis and Targeted Therapy Outcomes</a></td>
          <td>
            Herman F. Fumiã, M. L. Martins
          </td>
          <td>2013-07-26</td>
          <td>PLoS ONE</td>
          <td>141</td>
          <td>16</td>
        </tr>

        <tr id="Most cancer cells harbor multiple drivers whose epistasis and interactions with expression context clouds drug sensitivity prediction. We constructed a mechanistic computational model that is context-tailored by omics data to capture regulation of stochastic proliferation and death by pan-cancer driver pathways. Simulations and experiments explore how the coordinated dynamics of RAF/MEK/ERK and PI-3K/AKT kinase activities in response to synergistic mitogen or drug combinations control cell fate in a specific cellular context. In this context, synergistic ERK and AKT inhibitor-induced death is likely mediated by BIM rather than BAD. AKT dynamics explain S-phase entry synergy between EGF and insulin, but stochastic ERK dynamics seem to drive cell-to-cell proliferation variability, which in simulations are predictable from pre-stimulus fluctuations in C-Raf/B-Raf levels. Simulations predict MEK alteration negligibly influences transformation, consistent with clinical data. Our model mechanistically interprets context-specific landscapes between driver pathways and cell fates, moving towards more rational cancer combination therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75c6b1397a9feec802ccb9b0533160ce3d97c34d" target='_blank'>An Integrated Mechanistic Model of Pan-Cancer Driver Pathways Predicts Stochastic Proliferation and Death</a></td>
          <td>
            M. Bouhaddou, A. Barrette, R. J. Koch, Matthew S. DiStefano, Eric A. Riesel, Alan D. Stern, Luis C. Santos, A. Tan, Alex E. Mertz, M. Birtwistle
          </td>
          <td>2017-04-19</td>
          <td>bioRxiv</td>
          <td>5</td>
          <td>26</td>
        </tr>

        <tr id="Wnt signaling regulates cell survival, proliferation, and differentiation throughout development and is aberrantly regulated in cancer. The pathway is activated when Wnt ligands bind to specific receptors on the cell surface, resulting in the stabilization and nuclear accumulation of the transcriptional co‐activator β‐catenin. Mathematical and computational models have been used to study the spatial and temporal regulation of the Wnt/β‐catenin pathway and to investigate the functional impact of mutations in key components. Such models range in complexity, from time‐dependent, ordinary differential equations that describe the biochemical interactions between key pathway components within a single cell, to complex, multiscale models that incorporate the role of the Wnt/β‐catenin pathway target genes in tissue homeostasis and carcinogenesis. This review aims to summarize recent progress in mathematical modeling of the Wnt pathway and to highlight new biological results that could form the basis for future theoretical investigations designed to increase the utility of theoretical models of Wnt signaling in the biomedical arena. WIREs Syst Biol Med 2013, 5:391–407. doi: 10.1002/wsbm.1221">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99d13a9a38a69da9b028d49d8eabfa9f4c7fdc54" target='_blank'>Toward a quantitative understanding of the Wnt/β‐catenin pathway through simulation and experiment</a></td>
          <td>
            B. Lloyd-Lewis, A. Fletcher, T. Dale, H. Byrne
          </td>
          <td>2013-07-01</td>
          <td>Wiley Interdisciplinary Reviews: Systems Biology and Medicine</td>
          <td>36</td>
          <td>58</td>
        </tr>

        <tr id="Relationships between genetic alterations, such as co-occurrence or mutual exclusivity, are often observed in cancer, where their understanding may provide new insights into etiology and clinical management. In this study, we combined statistical analyses and computational modeling to explain patterns of genetic alterations seen in 178 patients with bladder tumors (either muscle-invasive or non-muscle-invasive). A statistical analysis on frequently altered genes identified pair associations, including co-occurrence or mutual exclusivity. Focusing on genetic alterations of protein-coding genes involved in growth factor receptor signaling, cell cycle, and apoptosis entry, we complemented this analysis with a literature search to focus on nine pairs of genetic alterations of our dataset, with subsequent verification in three other datasets available publicly. To understand the reasons and contexts of these patterns of associations while accounting for the dynamics of associated signaling pathways, we built a logical model. This model was validated first on published mutant mice data, then used to study patterns and to draw conclusions on counter-intuitive observations, allowing one to formulate predictions about conditions where combining genetic alterations benefits tumorigenesis. For example, while CDKN2A homozygous deletions occur in a context of FGFR3-activating mutations, our model suggests that additional PIK3CA mutation or p21CIP deletion would greatly favor invasiveness. Furthermore, the model sheds light on the temporal orders of gene alterations, for example, showing how mutual exclusivity of FGFR3 and TP53 mutations is interpretable if FGFR3 is mutated first. Overall, our work shows how to predict combinations of the major gene alterations leading to invasiveness through two main progression pathways in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7fc3545d04bd868b429976b0c7f9747ab1b9298" target='_blank'>A Modeling Approach to Explain Mutually Exclusive and Co-Occurring Genetic Alterations in Bladder Tumorigenesis.</a></td>
          <td>
            É. Remy, S. Rebouissou, C. Chaouiya, A. Zinovyev, F. Radvanyi, Laurence Calzone
          </td>
          <td>2015-10-01</td>
          <td>Cancer research</td>
          <td>84</td>
          <td>61</td>
        </tr>

        <tr id="The modeling of the dynamics of interaction between ERK and STAT signaling pathways in the cell needs to establish the biochemical diagram of the corresponding proteins interactions as well as the corresponding reaction-diffusion scheme. Starting from the verbal description available in the literature of the cross talk between the two pathways, a simple diagram of interaction between ERK and STAT5a proteins is chosen to write corresponding kinetic equations. The dynamics of interaction is modeled in a form of two-dimensional nonlinear dynamical system for ERK—and STAT5a —protein concentrations. Then the spatial modeling of the interaction is accomplished by introducing an appropriate diffusion-reaction scheme. The obtained system of partial differential equations is analyzed and it is argued that the possibility of Turing bifurcation is presented by loss of stability of the homogeneous steady state and forms dissipative structures in the ERK and STAT interaction process. In these terms, a possible scaffolding effect in the protein interaction is related to the process of stabilization and destabilization of the dissipative structures (pattern formation) inherent to the model of ERK and STAT cross talk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9d9343d3e0504a8941aeb614c09b6514df14167" target='_blank'>Reaction-Diffusion Modeling ERK- and STAT-Interaction Dynamics</a></td>
          <td>
            N. Georgiev, V. Petrov, G. Georgiev
          </td>
          <td>2006-11-29</td>
          <td>EURASIP Journal on Bioinformatics and Systems Biology</td>
          <td>2</td>
          <td>13</td>
        </tr>

        <tr id="Fibroblasts as key components of tumor microenvironment show different features in the interaction with cancer cells. Although, Normal fibroblasts demonstrate anti-tumor effects, cancer associated fibroblasts are principal participant in tumor growth and invasion. The ambiguity of fibroblasts function can be regarded as two heads of its behavioral spectrum and can be subjected for mathematical modeling to identify their switching behavior. In this research, an agent-based model of mutual interactions between fibroblast and cancer cell was created. The proposed model is based on nonlinear differential equations which describes biochemical reactions of the main factors involved in fibroblasts and cancer cells communication. Also, most of the model parameters are estimated using hybrid unscented Kalman filter. The interactions between two cell types are illustrated by the dynamic modeling of TGFβ and LIF pathways as well as their crosstalk. Using analytical and computational approaches, reciprocal effects of cancer cells and fibroblasts are constructed and the role of signaling molecules in tumor progression or prevention are determined. Finally, the model is validated using a set of experimental data. The proposed dynamic modeling might be useful for designing more efficient therapies in cancer metastasis treatment and prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e96d37b2042318f1f7c2bbd7f9e4cd59c1ca8240" target='_blank'>The double-edged sword role of fibroblasts in the interaction with cancer cells; an agent-based modeling approach</a></td>
          <td>
            Zarifeh Heidary, J. Ghaisari, Shiva Moein, S. Haghjooy Javanmard
          </td>
          <td>2020-05-08</td>
          <td>PLoS ONE</td>
          <td>17</td>
          <td>20</td>
        </tr>

        <tr id="The epidermal growth factor receptor (EGFR) activated extracellular-signal regulated kinase (ERK) pathway is a central cell signalling pathway that mediates many biological responses including cell proliferation, transformation, survival and motility. Deregulation of the pathway either through mutation of components or overexpression of EGFRs is associated with several forms of cancer. Under normal conditions, EGF stimulates a rapid but transient activation of ERK as the signal is rapidly shutdown, whereas under cancerous conditions, the ERK signal cannot be shutdown and is sustained. Computational modelling techniques have been used to investigate the signalling dynamics of the EGFR/ERK pathway, focusing on identifying the key processes involved in signal termination and what role the ERK to son of sevenless (SOS) negative feedback loop plays in generating a transient response. This model predicts that this negative feedback loop is not needed to achieve a transient activation of ERK as the process of receptor degradation alone is enough to terminate the signal. Importantly, the behaviour and predictions of this model are verified with laboratory data, as is essential for modern systems biology approaches. Further analysis showed that the feedback loop and receptor degradation were both redundant processes, as each could compensate for the absence of the other. This led to the prediction that in the case of a receptor which is not degraded, such as the insulin receptor, the negative feedback loop to SOS will actually be essential for a transient response to be achieved. Overall, the results shed new light on the role of negative feedback in EGF receptor signalling and suggest that different receptors are dependent on different features within the ERK pathway when relaying their signals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35e7094d70c15e7e0196f4a688c075edd2e156e0" target='_blank'>Computational modelling reveals feedback redundancy within the epidermal growth factor receptor/extracellular-signal regulated kinase signalling pathway.</a></td>
          <td>
            Richard J. Orton, Oliver E. Sturm, Amelie Gormand, Walter Kolch, David R. Gilbert
          </td>
          <td>2008-08-01</td>
          <td>IET systems biology</td>
          <td>21</td>
          <td>13</td>
        </tr>

        <tr id="Signal transduction is a complex protein signaling process with a rich network of multifunctional interactions that occur in a non‐linear fashion. Mitogen‐activated protein kinase (MAPK) signal transduction pathways regulate diverse cellular processes ranging from proliferation and differentiation to apoptosis. In mammals, out of five, there are three well characterized subfamilies of MAPKs ‐ ERKs (Extracellular signal‐regulated kinases), JNKs (c‐Jun N‐terminal kinases), and P38 kinases, and their activators, are implicated in human diseases and are targets for drug development. Kinase cascades in MAPK pathways mediate the sensing and processing of stimuli. To understand how cells makes decisions, the dynamic interactions of components of signaling cascades are important rather than just creating static maps. Based on enzyme kinetic reactions, we have developed a mathematical model to analyze the impact of the cross‐talks between JNK and P38 kinase cascades. Cross‐talks between JNK and P38 kinase cascades influence the activities of P38 kinases. Responses of the signals should be studied for network of kinase cascades by considering cross‐talks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef12bcae9ca82339a744c3b732a8bd77d348ac39" target='_blank'>Computational prediction and analysis of impact of the cross‐talks between JNK and P38 kinase cascades</a></td>
          <td>
            Pandurangan Sundaramurthy, S. Gakkhar, R. Sowdhamini
          </td>
          <td>2009-01-12</td>
          <td>Bioinformation</td>
          <td>12</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b17eff14eed0d23295b1d0eab1dcd343cb4b6d7" target='_blank'>Multi-Network Modeling of Cancer Cell States</a></td>
          <td>
            M. Blinov, A. Udyavar, W. Yarbrough, Jialiang Wang, L. Estrada, V. Quaranta
          </td>
          <td>2012-01-31</td>
          <td>Biophysical Journal</td>
          <td>2</td>
          <td>70</td>
        </tr>

        <tr id="An error led to part of Figure 2 being removed, breaking the red line that should have linked the grey circles “PKC” and “EGFR”. The correct version of Figure 2 is now displayed on both the PDF and HTML versions of the article.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/892ad05c7eb0020aec314c7a7afee5e1e1dd1b95" target='_blank'>Correction: Integrative Modelling of the Influence of MAPK Network on Cancer Cell Fate Decision</a></td>
          <td>
            L. Grieco, Laurence Calzone, I. Bernard-Pierrot, F. Radvanyi, B. Kahn-Perlès, D. Thieffry
          </td>
          <td>2013-11-01</td>
          <td>PLoS Computational Biology</td>
          <td>41</td>
          <td>61</td>
        </tr>

        <tr id="Proceedings: AACR 104th Annual Meeting 2013; Apr 6-10, 2013; Washington, DC

Introduction

Targeted drug treatment requires reliable companion diagnostics for therapy selection. Genomic and transcriptomic data can provide input for this, provided tools exist to convert this complex data into meaningful clinical information. We develop computational models of oncogenic pathways, to assess which one drives tumor growth in an individual patient and what is the causing (epi)genetic defect.

Computational pathway models

Based on a selection of experimentally validated direct target genes, we built initial models of the Wnt, ER, AR and Hedgehog pathways, covering their transcriptional program. We have modeled each pathway by a Bayesian network, which interprets the target genes’ mRNA levels (Affymetrix U133Plus2.0), and infers a probability that the respective pathway is active in a certain sample. Model parameters are based on literature insights and experimental data.

Results

A first Wnt model, calibrated on cell line data, validated perfectly on 32 normal colon samples and 32 colon adenomas from patients ([GSE8671][1]). A second Wnt model, calibrated on these 64 patient samples, correctly predicted no Wnt activity in all 44 normal colon samples, and Wnt activity in 97 of 101 colon cancer samples from [GSE20916][2].

Next, we tested the second Wnt model on other cancer types. On 25 breast cancer cell lines from [GSE12777][3] with known Wnt status, the model correctly identified the two samples with an active pathway. On two patient data sets ([GSE12276][4], n=204; [GSE21653][5], n=266) Wnt activity was predicted in a higher number of basal samples compared to other subtypes (p=0.021 and p=2.7e-5, respectively), in line with increasing evidence for Wnt activity in this subtype. Finally, tests on liver ([GSE9843][6], [GSE6764][7]) and medulloblastoma sets ([GSE10327][8]) confirm the power of these models to predict Wnt pathway activity.

A first ER model was calibrated on estrogen-deprived and -stimulated MCF7 cell lines ([GSE8697][9]). Applied on breast cancer cell line data from [GSE21618][10], increased incidence of ER pathway activity was found in tamoxifen-sensitive cell lines compared to resistant ones. On breast cancer patient data ([GSE12276][4], [GSE9195][11], [GSE6532][12]) the model showed no pathway activity in ER- samples, and an active ER pathway in 26-38% of the ER+ samples. Within the latter group, model-predicted ER activity correlated with improved survival. Clinical utility studies to correlate ER activity to hormone therapy response are in progress.

Finally, the AR model showed promising results on prostate cancer cell lines ([GSE34211][13], [GSE36133][14]), as did the Hedgehog model on medulloblastoma samples ([GSE10327][8]).

Conclusion

Our computational pathway models predict functional activity of oncogenic pathways for an individual patient based on mRNA data, complementary to existing molecular and histopathology staining tests. Clinical utility for therapy response prediction is currently being validated with clinical partners.

Citation Format: Wim Verhaegh, Henk van Ooijen, Marcia Alves de Inda, Kalyan Dulla, Ralf Hoffmann, Dianne van Strijp, Pantelis Hatzis, Hans Clevers, Anja van de Stolpe. Identifying tumor driving signaling pathways for companion diagnostics using computational pathway models. [abstract]. In: Proceedings of the 104th Annual Meeting of the American Association for Cancer Research; 2013 Apr 6-10; Washington, DC. Philadelphia (PA): AACR; Cancer Res 2013;73(8 Suppl):Abstract nr 59. doi:10.1158/1538-7445.AM2013-59

 [1]: /lookup/external-ref?link_type=NCBIGEO&access_num=GSE8671&atom=%2Fcanres%2F73%2F8_Supplement%2F59.atom
 [2]: /lookup/external-ref?link_type=NCBIGEO&access_num=GSE20916&atom=%2Fcanres%2F73%2F8_Supplement%2F59.atom
 [3]: /lookup/external-ref?link_type=NCBIGEO&access_num=GSE12777&atom=%2Fcanres%2F73%2F8_Supplement%2F59.atom
 [4]: /lookup/external-ref?link_type=NCBIGEO&access_num=GSE12276&atom=%2Fcanres%2F73%2F8_Supplement%2F59.atom
 [5]: /lookup/external-ref?link_type=NCBIGEO&access_num=GSE21653&atom=%2Fcanres%2F73%2F8_Supplement%2F59.atom
 [6]: /lookup/external-ref?link_type=NCBIGEO&access_num=GSE9843&atom=%2Fcanres%2F73%2F8_Supplement%2F59.atom
 [7]: /lookup/external-ref?link_type=NCBIGEO&access_num=GSE6764&atom=%2Fcanres%2F73%2F8_Supplement%2F59.atom
 [8]: /lookup/external-ref?link_type=NCBIGEO&access_num=GSE10327&atom=%2Fcanres%2F73%2F8_Supplement%2F59.atom
 [9]: /lookup/external-ref?link_type=NCBIGEO&access_num=GSE8697&atom=%2Fcanres%2F73%2F8_Supplement%2F59.atom
 [10]: /lookup/external-ref?link_type=NCBIGEO&access_num=GSE21618&atom=%2Fcanres%2F73%2F8_Supplement%2F59.atom
 [11]: /lookup/external-ref?link_type=NCBIGEO&access_num=GSE9195&atom=%2Fcanres%2F73%2F8_Supplement%2F59.atom
 [12]: /lookup/external-ref?link_type=NCBIGEO&access_num=GSE6532&atom=%2Fcanres%2F73%2F8_Supplement%2F59.atom
 [13]: /lookup/external-ref?link_type=NCBIGEO&access_num=GSE34211&atom=%2Fcanres%2F73%2F8_Supplement%2F59.atom
 [14]: /lookup/external-ref?link_type=NCBIGEO&access_num=GSE36133&atom=%2Fcanres%2F73%2F8_Supplement%2F59.atom">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55a79dd61700b1d45dc6533eb1c74c9b2eadb62e" target='_blank'>Abstract 59: Identifying tumor driving signaling pathways for companion diagnostics using computational pathway models.</a></td>
          <td>
            W. Verhaegh, H. V. Ooijen, M. Inda, K. Dulla, R. Hoffmann, D. V. Strijp, P. Hatzis, H. Clevers, A. Stolpe
          </td>
          <td>2013-04-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>218</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f379feef34e99e989c72c542c8d132f15116aa" target='_blank'>Cancer: a Systems Biology disease.</a></td>
          <td>
            J. Hornberg, F. Bruggeman, H. Westerhoff, J. Lankelma
          </td>
          <td>2006-02-01</td>
          <td>Bio Systems</td>
          <td>384</td>
          <td>87</td>
        </tr>

        <tr id="The multistep development of cancer involves the cooperation between multiple molecular lesions, as well as complex interactions between cancer cells and the surrounding tumour microenvironment. The search for these synergistic interactions using experimental models made tremendous contributions to our understanding of oncogenesis. Yet, these approaches remain labour intensive and challenging. To tackle such a hurdle, an integrative, multidisciplinary effort is required. In this article, we highlight the use of logical computational models combined to experimental validations as an effective approach to identify cooperative mechanisms and therapeutic strategies in the context of cancer biology. In silico models overcome limitations of reductionist approaches by capturing tumour complexity, and by generating powerful testable hypotheses. We review representative examples of logical models reported in the literature and their validation. We then provide further analyses of our logical model of Epithelium to Mesenchymal Transition (EMT), searching for additional cooperative interactions involving inputs from the tumour microenvironment and gain of function mutations in NOTCH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/185e466d1764a2dd1e860acdd2297327e5220130" target='_blank'>In Silico Logical Modelling to Uncover Cooperative Interactions in Cancer</a></td>
          <td>
            G. Selvaggio, C. Chaouiya, Florence Janody
          </td>
          <td>2021-03-12</td>
          <td>International Journal of Molecular Sciences</td>
          <td>2</td>
          <td>35</td>
        </tr>

        <tr id="Most cancer cells harbor multiple drivers whose epistasis and interactions with expression context clouds drug and drug combination sensitivity prediction. We constructed a mechanistic computational model that is context-tailored by omics data to capture regulation of stochastic proliferation and death by pan-cancer driver pathways. Simulations and experiments explore how the coordinated dynamics of RAF/MEK/ERK and PI-3K/AKT kinase activities in response to synergistic mitogen or drug combinations control cell fate in a specific cellular context. In this MCF10A cell context, simulations suggest that synergistic ERK and AKT inhibitor-induced death is likely mediated by BIM rather than BAD, which is supported by prior experimental studies. AKT dynamics explain S-phase entry synergy between EGF and insulin, but simulations suggest that stochastic ERK, and not AKT, dynamics seem to drive cell-to-cell proliferation variability, which in simulations is predictable from pre-stimulus fluctuations in C-Raf/B-Raf levels. Simulations suggest MEK alteration negligibly influences transformation, consistent with clinical data. Tailoring the model to an alternate cell expression and mutation context, a glioma cell line, allows prediction of increased sensitivity of cell death to AKT inhibition. Our model mechanistically interprets context-specific landscapes between driver pathways and cell fates, providing a framework for designing more rational cancer combination therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1908d4d5c01d330ef1966c89fae24a6a2acd1e66" target='_blank'>A mechanistic pan-cancer pathway model informed by multi-omics data interprets stochastic cell fate responses to drugs and mitogens</a></td>
          <td>
            M. Bouhaddou, A. Barrette, Alan D. Stern, R. J. Koch, Matthew S. DiStefano, Eric A. Riesel, Luis C. Santos, A. Tan, Alex E. Mertz, M. Birtwistle
          </td>
          <td>2018-03-01</td>
          <td>PLoS Computational Biology</td>
          <td>42</td>
          <td>26</td>
        </tr>

        <tr id="Cross-talk among abnormal pathways widely occurs in human cancer and generally leads to insensitivity to cancer treatment. Moreover, alterations in the abnormal pathways are not limited to single molecular level. Therefore, we proposed a strategy that integrates a large number of biological sources at multiple levels for systematic identification of cross-talk among risk pathways in cancer by random walk on protein interaction network. We applied the method to multi-Omics breast cancer data from The Cancer Genome Atlas (TCGA), including somatic mutation, DNA copy number, DNA methylation and gene expression profiles. We identified close cross-talk among many known cancer-related pathways with complex change patterns. Furthermore, we identified key genes (linkers) bridging these cross-talks and showed that these genes carried out consistent biological functions with the linked cross-talking pathways. Through identification of leader genes in each pathway, the architecture of cross-talking pathways was built. Notably, we observed that linkers cooperated with leaders to form the fundamentation of cross-talk of pathways which play core roles in deterioration of breast cancer. As an example, we observed that KRAS showed a direct connection to numerous cancer-related pathways, such as MAPK signaling pathway, suggesting that it may be a central communication hub. In summary, we offer an effective way to characterize complex cross-talk among disease pathways, which can be applied to other diseases and provide useful information for the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3efa819dc6ffcfc2c1c69be2d2d0b6abe54484a7" target='_blank'>Integrating Multi-Omics for Uncovering the Architecture of Cross-Talking Pathways in Breast Cancer</a></td>
          <td>
            Li Wang, Yun Xiao, Yanyan Ping, Jing Li, Hongying Zhao, Feng Li, Jing Hu, Hongyi Zhang, Yulan Deng, J. Tian, Xia Li
          </td>
          <td>2014-08-19</td>
          <td>PLoS ONE</td>
          <td>24</td>
          <td>45</td>
        </tr>

        <tr id="System dynamics is a popular approach in many fields of science and technology, but it has not been investigated for cell signaling pathways yet. It is a well formulated methodology used to analyze the components of a system considering the cause-effect relationships. The two main components of system dynamics modeling, Causal Loop and Stocks-Flow diagrams, make possible to model the dynamics of the system with the ability to analyze it both qualitatively and quantitatively. In this paper, after introducing the system dynamics modeling approach, and giving a simple example from its usage for Michaelis Menton reactions, a three-step process is proposed for signal transduction modeling. Then in a complete example, it is applied to a case study in cell signaling pathways, namely the RKIP Influence on the ERK signaling pathway and the results is compared with an integrative modeling approach based on Petri net and ODEs. Computational simulations show the success of the system dynamic in easier and effective modeling of the cell signaling pathways, in addition to its diverse options for understanding and testing the pathway quantitatively and quantitatively at the same time in relation to other methods. More intestinally, the suggested approach here helps one to identify all biochemical reaction paths and loops for complex cell signaling pathways. It will be a good step forward to define dominant systems for such complex cell signaling pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4149b50e4189896acbe292b78085dbf48d60c22" target='_blank'>Causal loop and Stock-Flow Modeling of Signal Transduction Pathways</a></td>
          <td>
            S. Sulaimany, G. Bidkhori, Sarbaz H. A. Khoshnaw
          </td>
          <td>2024-09-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fbbe0f2e20e5d329dc11bd2b7769f94c2f6e4f5" target='_blank'>Automated oncogene detection in complex protein networks with applications to the MAPK signal transduction pathway.</a></td>
          <td>
            D. Pant, A. Ghosh
          </td>
          <td>2005-03-01</td>
          <td>Biophysical chemistry</td>
          <td>11</td>
          <td>29</td>
        </tr>

        <tr id="Cancer is a multifactorial disease. Aberrant functioning of the underlying complex signaling network that orchestrates cellular response to external or internal cues governs incidence, progression, and recurrence of cancer. Detailed understanding of cancer’s etiology can offer useful insights into arriving at novel therapeutic and disease management strategies. Such an understanding for most cancers is currently limited due to unavailability of a predictive large-scale, integrated signaling model accounting for all tumor orchestrating factors. We suggest that the potential of Boolean dynamic (BD) modeling approaches, though qualitative, can be harnessed for developing holistic models capturing multi-scale, multi-cellular signaling processes involved in cancer incidence and progression. We believe that constraining such an integrated BD model with variety of omics data at different scales from laboratory and clinical settings could offer deeper insights into causal mechanisms governing the disease leading to better prognosis. We review the recent literature employing different BD modeling strategies to model variety of cancer signaling programs leading to identification of cancer-specific prognostic markers such as SMAD proteins, which may also serve as early predictors of tumor cells hijacking the epithelial-mesenchymal plasticity program. In silico simulations of BD models of different cancer signaling networks combined with attractor landscape analysis and validated with experimental data predicted the nature of short-and long-term response of standard targeted therapeutic agents such as Nutlin-3, a small molecule inhibitor for p53-MDM2 interaction. BD simulations also offered a mechanistic view of emerging resistance to drugs such as Trastuzumab for">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be0d7c2f350f03724a68a5b4624bf8d8da62bbaf" target='_blank'>Boolean dynamic modeling of cancer signaling networks: Prognosis, progression, and therapeutics</a></td>
          <td>
            S. Sherekar, G. Viswanathan
          </td>
          <td>2021-05-01</td>
          <td>Computational and Systems Oncology</td>
          <td>13</td>
          <td>12</td>
        </tr>

        <tr id="Biochemical networks related to similar functional pathways are often correlated due to cross-talk among the homologous proteins in the different networks. Using a stochastic framework, we address the functional significance of the cross-talk between two pathways. Our theoretical analysis on generic MAPK pathways reveals cross-talk is responsible for developing coordinated fluctuations between the pathways. The extent of correlation evaluated in terms of the information theoretic measure provides directionality to net information propagation. Stochastic time series and scattered plot suggest that the cross-talk generates synchronization within a cell as well as in a cellular population. Depending on the number of input and output, we identify signal integration and signal bifurcation motif that arise due to inter-pathway connectivity in the composite network. Analysis using partial information decomposition quantifies the net synergy in the information propagation through these branched pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c0ebb8b35dda6d869dfe84cf69e0ed74fdc8b95" target='_blank'>Information Theoretical Study of Cross-Talk Mediated Signal Transduction in MAPK Pathways</a></td>
          <td>
            A. Maity, P. Chaudhury, S. Banik
          </td>
          <td>2015-10-16</td>
          <td>Entropy</td>
          <td>7</td>
          <td>20</td>
        </tr>

        <tr id="Extra-cellular signal regulated kinase (ERK) signal transduction is an important signaling pathway, involved in a variety of processes in cell and closely related to the occurrences of diverse tumors. This raises the question how these diverse functions are specified and coordinated. In this work, we present an integrated computational model of ERK signal transduction pathway, investigating the regulatory mechanism of ERK signal pathway by variation of the concentration of epidermal growth factor receptor (EGFR) as well as the binding affinity of adaptor proteins. The results show that EGFR and distinct adaptor proteins engender different activation patterns of ERK, which will bring different cellular responses. It could be helpful to understand the mechanism of ERK signal pathway and the relationship between ERK signal network and tumor development or therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8834b7200d12495f21190553b8a4f1751ce0a835" target='_blank'>Dynamic Model and Regulatory Mechanism of Integrated ERK Signal Pathway Activated by Epidermal Growth Factor</a></td>
          <td>
            H. Fang, Yan Li, Yonghua Wang, Shuwei Zhang
          </td>
          <td>2008-10-18</td>
          <td>2008 Fourth International Conference on Natural Computation</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cdfd30fd7f8176bda665766985e833fecc32815" target='_blank'>Network-based approach to understand dynamic behaviour of Wnt signaling pathway regulatory elements in colorectal cancer</a></td>
          <td>
            A. Shukla, T. Singh
          </td>
          <td>2018-09-07</td>
          <td>Network Modeling Analysis in Health Informatics and Bioinformatics</td>
          <td>3</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6826d9b55d8610a319cd1e55a62f54d400dde7" target='_blank'>Network-based approach to understand dynamic behaviour of Wnt signaling pathway regulatory elements in colorectal cancer</a></td>
          <td>
            A. Shukla, T. Singh
          </td>
          <td>2018-09-07</td>
          <td>Network Modeling Analysis in Health Informatics and Bioinformatics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="This perspective article gathers the latest developments in mathematical and computational oncology tools that exploit network approaches for the mathematical modelling, analysis, and simulation of cancer development and therapy design. It instigates the community to explore new paths and synergies under the umbrella of the Special Issue “Networks in Cancer: From Symmetry Breaking to Targeted Therapy”. The focus of the perspective is to demonstrate how networks can model the physics, analyse the interactions, and predict the evolution of the multiple processes behind tumour-host encounters across multiple scales. From agent-based modelling and mechano-biology to machine learning and predictive modelling, the perspective motivates a methodology well suited to mathematical and computational oncology and suggests approaches that mark a viable path towards adoption in the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0eb726c69e19f02f76a826053b1a3fd630ab477d" target='_blank'>The Multiple Dimensions of Networks in Cancer: A Perspective</a></td>
          <td>
            Cristian Axenie, R. Bauer, María Rodríguez Martínez
          </td>
          <td>2021-08-25</td>
          <td>Symmetry</td>
          <td>4</td>
          <td>16</td>
        </tr>

        <tr id="Embryonic development and adult tissue homeostasis are controlled through activation of intracellular signal transduction pathways by extracellular growth factors. In the past, signal transduction has largely been regarded as a linear process. However, more recent data from large‐scale and high‐throughput experiments indicate that there is extensive cross‐talk between individual signaling cascades leading to the notion of a signaling network. The behavior of such complex networks cannot be predicted by simple intuitive approaches but requires sophisticated models and computational simulations. The purpose of such models is to generate experimentally testable hypotheses and to find explanations for unexpected experimental results. Here, we discuss the need for, and the future impact of, mathematical models for exploring signal transduction in different biological contexts such as for example development. BioEssays 30:1110–1125, 2008. © 2008 Wiley Periodicals, Inc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8545f3dc8505b17afcd909dbddc0e64abbc53ce2" target='_blank'>Network modeling of signal transduction: establishing the global view</a></td>
          <td>
            H. Kestler, Christian Wawra, Barbara Kracher, M. Kühl
          </td>
          <td>2008-11-01</td>
          <td>BioEssays</td>
          <td>94</td>
          <td>56</td>
        </tr>

        <tr id="Two studies show the power of systems biology methods to improve cancer drug development and treatment strategy. The identification of “driver mutations” in cancers initiated rapid development of targeted drugs for the clinic. Despite promising outcomes initially in patients, the ultimate success of oncogene-targeted drugs has been hampered by the redundancy and remarkable complexity of cellular signaling pathways. Two studies in Science Signaling show that understanding these intricate networks and considering them during tumor classification and drug design can better predict drug response. These studies exemplify the potential of using systems analysis and computational modeling approaches to improve therapeutic strategies and outcomes in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/222e5a136d1e41e7af9834e0988f5c8ac6e9f630" target='_blank'>What Lies Beneath: Looking Beyond Tumor Genetics Shows the Complexity of Signaling Networks Underlying Drug Sensitivity</a></td>
          <td>
            V. Quaranta, D. Tyson
          </td>
          <td>2013-09-24</td>
          <td>Science Signaling</td>
          <td>12</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b0d4a5d209975c2d5166fd635acb02a22157e56" target='_blank'>The crosstalk between EGF, IGF, and Insulin cell signaling pathways - computational and experimental analysis</a></td>
          <td>
            Rafal Zieliński, P. Przytycki, Jie Zheng, David Zhang, T. Przytycka, J. Capala
          </td>
          <td>2009-09-04</td>
          <td>BMC Systems Biology</td>
          <td>62</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da87720f4767a44da3f92e497492607a5e8a082c" target='_blank'>When kinases meet mathematics: the systems biology of MAPK signalling</a></td>
          <td>
            W. Kolch, Muffy Calder, David R. Gilbert
          </td>
          <td>2005-03-21</td>
          <td>FEBS Letters</td>
          <td>190</td>
          <td>86</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>