<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              509f1971d67425c00eb55f435cf67274acb0fe25
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-27 06:08:46 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Computational modeling of the dynamics of the MAP kinase cascade activated by surface and internalized EGF receptors</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="Extra-cellular signal regulated kinase (ERK) signal network plays an important role in regulating the fundamental cellular functions such as cell proliferation, survival, differentiation and motility. Development and analysis of mathematical model can help us acquire a deep understanding of the complex behavior of ERK signal transduction network. This paper presents a computational model that offers an integrated quantitative and dynamic simulation of ERK signal transduction network, activated by epidermal growth factor (EGF). The mathematic model contains the activation kinetics of the pathway, a multitude of feedback loops and connection of scaffold proteins. The model provides insight into signal-response relationships between the binding of EGF to its receptor at the cell surface and activation of downstream proteins in the signaling cascade. The different influence of positive and negative feedback loops of the ERK signal transduction pathway were mainly analyzed, indicating that feedback loops were the main impacting factor to the oscillation of ERK signal transduction pathway. The predictions of this oscillation of ERK activation agree well with the literature. It can lead to signal waves of the downstream substrates and induce corresponding biological behaviors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7bf832c4d17a797f80bffbcb87cce1c1078a91f" target='_blank'>Dynamic Analysis of Feedback Loops in Extra-Cellular Signal Regulated Kinase Signal Network</a></td>
          <td>
            Yong Han, Yan Li, Yonghua Wang, H. Fang, Shuwei Zhang
          </td>
          <td>2009-03-31</td>
          <td>2009 WRI World Congress on Computer Science and Information Engineering</td>
          <td>3</td>
          <td>34</td>
        </tr>

        <tr id="Abstract Computer simulations and mathematical modeling of biological processes are becoming increasingly popular, and yet the complexity of the biochemical systems or the differences between experimental setups make it very difficult to establish a standard formula for these modeling projects. Before we can start using computer‐based models for predictions or targeted experiment designs, it is very important to establish a reliable model on which those predictions can be based and experimentally tested. Here we attempt to present a computer model for the mitogen‐activated protein kinase (MAPK) signaling cascade which is consistent with previously published experimental results. In this study we have focused our attention to a generic MAPK ERK (extracellular signal‐regulated kinase) pathway activated by epidermal growth factor (EGF) in an attempt to understand how receptors may achieve different activation kinetics of the MAPK signaling. We successfully show that the level of receptor expression is one key determinant in this regulation, and that the binding affinity of the active receptor to adaptor proteins can have a small but albeit direct effect on the downstream activation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dde366641aca5125603232bb38f2d61b4eda2a5" target='_blank'>A Computer‐Based Model for the Regulation of Mitogen Activated Protein Kinase (MAPK) Activation</a></td>
          <td>
            I. Aksan, M. Levent Kurnaz
          </td>
          <td>2003-01-01</td>
          <td>Journal of Receptors and Signal Transduction</td>
          <td>26</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/494fb14b7bdb5aeed98f0bf7250842a98ec95388" target='_blank'>Computational modeling reveals molecular details of epidermal growth factor binding</a></td>
          <td>
            K. Mayawala, D. Vlachos, J. Edwards
          </td>
          <td>2005-11-30</td>
          <td>BMC Cell Biology</td>
          <td>40</td>
          <td>95</td>
        </tr>

        <tr id="Background The ErbB family of receptors activates intracellular signaling pathways that control cellular proliferation, growth, differentiation and apoptosis. Given these central roles, it is not surprising that overexpression of the ErbB receptors is often associated with carcinogenesis. Therefore, extensive laboratory studies have been devoted to understanding the signaling events associated with ErbB activation. Methodology/Principal Findings Systems biology has contributed significantly to our current understanding of ErbB signaling networks. However, although computational models have grown in complexity over the years, little work has been done to consider the spatial-temporal dynamics of receptor interactions and to evaluate how spatial organization of membrane receptors influences signaling transduction. Herein, we explore the impact of spatial organization of the epidermal growth factor receptor (ErbB1/EGFR) on the initiation of downstream signaling. We describe the development of an algorithm that couples a spatial stochastic model of membrane receptors with a nonspatial stochastic model of the reactions and interactions in the cytosol. This novel algorithm provides a computationally efficient method to evaluate the effects of spatial heterogeneity on the coupling of receptors to cytosolic signaling partners. Conclusions/Significance Mathematical models of signal transduction rarely consider the contributions of spatial organization due to high computational costs. A hybrid stochastic approach simplifies analyses of the spatio-temporal aspects of cell signaling and, as an example, demonstrates that receptor clustering contributes significantly to the efficiency of signal propagation from ligand-engaged growth factor receptors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f507b023b72e131b6fdde1ebc1c7bd9aef05c4a7" target='_blank'>Coupled Stochastic Spatial and Non-Spatial Simulations of ErbB1 Signaling Pathways Demonstrate the Importance of Spatial Organization in Signal Transduction</a></td>
          <td>
            Michelle N. Costa, K. Radhakrishnan, B. Wilson, D. Vlachos, J. Edwards
          </td>
          <td>2009-07-23</td>
          <td>PLoS ONE</td>
          <td>32</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2968ecd67263a54d9569b6d4f2eff3b4b1daf7a1" target='_blank'>Current challenges in quantitative modeling of epidermal growth factor signaling</a></td>
          <td>
            R. Breitling, Daniela Hoeller
          </td>
          <td>2005-11-21</td>
          <td>FEBS Letters</td>
          <td>30</td>
          <td>64</td>
        </tr>

        <tr id="The epidermal growth factor receptor (EGFR) activated extracellular-signal regulated kinase (ERK) pathway is a central cell signalling pathway that mediates many biological responses including cell proliferation, transformation, survival and motility. Deregulation of the pathway either through mutation of components or overexpression of EGFRs is associated with several forms of cancer. Under normal conditions, EGF stimulates a rapid but transient activation of ERK as the signal is rapidly shutdown, whereas under cancerous conditions, the ERK signal cannot be shutdown and is sustained. Computational modelling techniques have been used to investigate the signalling dynamics of the EGFR/ERK pathway, focusing on identifying the key processes involved in signal termination and what role the ERK to son of sevenless (SOS) negative feedback loop plays in generating a transient response. This model predicts that this negative feedback loop is not needed to achieve a transient activation of ERK as the process of receptor degradation alone is enough to terminate the signal. Importantly, the behaviour and predictions of this model are verified with laboratory data, as is essential for modern systems biology approaches. Further analysis showed that the feedback loop and receptor degradation were both redundant processes, as each could compensate for the absence of the other. This led to the prediction that in the case of a receptor which is not degraded, such as the insulin receptor, the negative feedback loop to SOS will actually be essential for a transient response to be achieved. Overall, the results shed new light on the role of negative feedback in EGF receptor signalling and suggest that different receptors are dependent on different features within the ERK pathway when relaying their signals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35e7094d70c15e7e0196f4a688c075edd2e156e0" target='_blank'>Computational modelling reveals feedback redundancy within the epidermal growth factor receptor/extracellular-signal regulated kinase signalling pathway.</a></td>
          <td>
            Richard J. Orton, Oliver E. Sturm, Amelie Gormand, Walter Kolch, David R. Gilbert
          </td>
          <td>2008-08-01</td>
          <td>IET systems biology</td>
          <td>21</td>
          <td>13</td>
        </tr>

        <tr id="Everyday cells encounter various stimuli ranging from growth signals to bacterial infections to UV insult to death signals. They must somehow receive these signals, interpret them, integrate different stimuli, and generate the required output. They can do so by an intricate mechanism named intracellular signal transduction. There are various signal transduction pathways within a cell, each of which are designed for a particular stimulus, and all of which can crosstalk among themselves for the careful integration of all stimuli. In this report, only one of these pathways, the mitogen-activated protein kinase (MAPK) pathway has been simulated. This pathway is activated upon binding of growth factors to their respective cell surface-bound receptors. Activated receptors relay the incoming signal to the cell interior via a cascade of proteins, which are thought to be involved in both the amplification of the signal, and the specificity of the pathway. A generic MAPK pathway activated by the EGF (epidermal growth factor) and the effect of receptor overexpression has been studied, and consistent with experimental evidence, it is shown that the number of EGF receptors on the cell surface is a key factor in the response generated by the pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60459cf132bf5912a5192bbf5cbc9a589b40edec" target='_blank'>Computer simulation of differential kinetics of MAPK activation upon EGF receptor overexpression</a></td>
          <td>
            I. Aksan, M. Sen, M. Araz, M. Levent Kurnaz
          </td>
          <td>2001-10-25</td>
          <td>2001 Conference Proceedings of the 23rd Annual International Conference of the IEEE Engineering in Medicine and Biology Society</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The biophysical properties of ligand binding heavily influence the ability of receptors to specify cell fates. Understanding the rules by which ligand binding kinetics impact cell phenotype is challenging, however, because of the coupled information transfers that occur from receptors to downstream signaling effectors and from effectors to phenotypes. Here, we address that issue by developing an integrated mechanistic and data-driven computational modeling platform to predict cell responses to different ligands for the epidermal growth factor receptor (EGFR). Experimental data for model training and validation were generated using MCF7 human breast cancer cells treated with the high- and low-affinity ligands epidermal growth factor (EGF) and epiregulin (EREG), respectively. The integrated model captures the unintuitive, concentration-dependent abilities of EGF and EREG to drive signals and phenotypes differently, even at similar levels of receptor occupancy. For example, the model correctly predicts the dominance of EREG over EGF in driving a cell differentiation phenotype through AKT signaling at intermediate and saturating ligand concentrations and the ability of EGF and EREG to drive a broadly concentration-sensitive migration phenotype through cooperative ERK and AKT signaling. Parameter sensitivity analysis identifies EGFR endocytosis, which is differentially regulated by EGF and EREG, as one of the most important determinants of the alternative phenotypes driven by different ligands. The integrated model provides a new platform to predict how phenotypes are controlled by the earliest biophysical rate processes in signal transduction and may eventually be leveraged to understand receptor signaling system performance depends on cell context. One-sentence summary Integrated kinetic and data-driven EGFR signaling model identifies the specific signaling mechanisms that dictate cell responses to EGFR activation by different ligands.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccddd2d0cde0d83a0a8cfb56bef3dd0624d5226" target='_blank'>An integrated mechanistic and data-driven computational model predicts cell responses to high- and low-affinity EGFR ligands</a></td>
          <td>
            Paul J. Myers, Sung Hyun Lee, M. Lazzara
          </td>
          <td>2023-06-26</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>22</td>
        </tr>

        <tr id="Extra-cellular signal regulated kinase (ERK) signal transduction is an important signaling pathway, involved in a variety of processes in cell and closely related to the occurrences of diverse tumors. This raises the question how these diverse functions are specified and coordinated. In this work, we present an integrated computational model of ERK signal transduction pathway, investigating the regulatory mechanism of ERK signal pathway by variation of the concentration of epidermal growth factor receptor (EGFR) as well as the binding affinity of adaptor proteins. The results show that EGFR and distinct adaptor proteins engender different activation patterns of ERK, which will bring different cellular responses. It could be helpful to understand the mechanism of ERK signal pathway and the relationship between ERK signal network and tumor development or therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8834b7200d12495f21190553b8a4f1751ce0a835" target='_blank'>Dynamic Model and Regulatory Mechanism of Integrated ERK Signal Pathway Activated by Epidermal Growth Factor</a></td>
          <td>
            H. Fang, Yan Li, Yonghua Wang, Shuwei Zhang
          </td>
          <td>2008-10-18</td>
          <td>2008 Fourth International Conference on Natural Computation</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286a6247bc3277b67992359ef1ee23e48dac6bc1" target='_blank'>Computational and Mathematical Modelling of the EGF Receptor System</a></td>
          <td>
            Colin G. Johnson, Emmet McIntyre, W. Gullick
          </td>
          <td>2008-10-01</td>
          <td></td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="The EGF-induced MAP kinase cascade is one of the most important and best characterized networks in intracellular signalling. It has a vital role in the development and maturation of living organisms. However, when deregulated, it is involved in the onset of a number of diseases. Based on a computational model describing a “surface” and an “internalized” parallel route, we use systems biology techniques to characterize aspects of the network’s functional organization. We examine the re-organization of protein groups from low to high external stimulation, define functional groups of proteins within the network, determine the parameter best encoding for input intensity and predict the effect of protein removal to the system’s output response. Extensive functional re-organization of proteins is observed in the lower end of stimulus concentrations. As we move to higher concentrations the variability is less pronounced. 6 functional groups have emerged from a consensus clustering approach, reflecting different dynamical aspects of the network. Mutual information investigation revealed that the maximum activation rate of the two output proteins best encodes for stimulus intensity. Removal of each protein of the network resulted in a range of graded effects, from complete silencing to intense activation. Our results provide a new “vista” of the EGF-induced MAP kinase cascade, from the perspective of complex self-organizing systems. Functional grouping of the proteins reveals an organizational scheme contrasting the current understanding of modular topology. The six identified groups may provide the means to experimentally follow the dynamics of this complex network. Also, the vulnerability analysis approach may be used for the development of novel therapeutic targets in the context of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccd7041af39a1cc5e66884c8aacd4b45fc5c205d" target='_blank'>Functional Aspects of the EGF-Induced MAP Kinase Cascade: A Complex Self-Organizing System Approach</a></td>
          <td>
            E. Kosmidis, V. Moschou, Georgios Ziogas, I. Boukovinas, M. Albani, N. Laskaris
          </td>
          <td>2014-11-05</td>
          <td>PLoS ONE</td>
          <td>1</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641cf547279786303f7caf10ec313d2a72cc1061" target='_blank'>Exploring the spatial effects influencing the EGFR/ERK pathway dynamics with machine learning surrogate models</a></td>
          <td>
            Juan A. Garcia, A. Bouchnita
          </td>
          <td>2024-11-01</td>
          <td>Bio Systems</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dd8c0e59cc6f86078f2c60ed2d73016f4951d76" target='_blank'>Computational modelling of cancerous mutations in the EGFR/ERK signalling pathway</a></td>
          <td>

          </td>
          <td>2009-10-05</td>
          <td>BMC Systems Biology</td>
          <td>64</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c47f6df71a9c01156d35a1778b1a714fa9e175c" target='_blank'>A Multiscale Computational Approach to Dissect Early Events in the Erb Family Receptor Mediated Activation, Differential Signaling, and Relevance to Oncogenic Transformations</a></td>
          <td>
            Yingting Liu, J. Purvis, Andrew J. Shih, Joshua A. Weinstein, Neeraj J. Agrawal, R. Radhakrishnan
          </td>
          <td>2007-02-02</td>
          <td>Annals of Biomedical Engineering</td>
          <td>40</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcdad5f1bff9567dba95a080790737079a3c8dd2" target='_blank'>Prediction and validation of the distinct dynamics of transient and sustained ERK activation</a></td>
          <td>
            S. Sasagawa, Yu-ichi Ozaki, Kazuhiro A. Fujita, Shinya Kuroda
          </td>
          <td>2005-03-27</td>
          <td>Nature Cell Biology</td>
          <td>455</td>
          <td>27</td>
        </tr>

        <tr id="Cell responses are actuated by tightly controlled signal transduction pathways. Although the concept of an integrated signaling network replete with interpathway cross-talk and feedback regulation is broadly appreciated, kinetic data of the type needed to characterize such interactions in conjunction with mathematical models are lacking. In mammalian cells, the Ras/ERK pathway controls cell proliferation and other responses stimulated by growth factors, and several cross-talk and feedback mechanisms affecting its activation have been identified. In this work, we take a systematic approach to parse the magnitudes of multiple regulatory mechanisms that attenuate ERK activation through canonical (Ras-dependent) and non-canonical (PI3K-dependent) pathways. In addition to regulation of receptor and ligand levels, we consider three layers of ERK-dependent feedback: desensitization of Ras activation, negative regulation of MEK kinase (e.g. Raf) activities, and up-regulation of dual-specificity ERK phosphatases. Our results establish the second of these as the dominant mode of ERK self-regulation in mouse fibroblasts. We further demonstrate that kinetic models of signaling networks, trained on a sufficient diversity of quantitative data, can be reasonably comprehensive, accurate, and predictive in the dynamical sense.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405be6d692d4836d3b3efde9333fdf1789ff05c7" target='_blank'>Systematic Quantification of Negative Feedback Mechanisms in the Extracellular Signal-regulated Kinase (ERK) Signaling Network*</a></td>
          <td>
            M. Cirit, Chun-Chao Wang, J. Haugh
          </td>
          <td>2010-09-16</td>
          <td>The Journal of Biological Chemistry</td>
          <td>91</td>
          <td>33</td>
        </tr>

        <tr id="The development of novel high-throughput technologies has opened up the opportunity to deeply characterize patient tissues at various molecular levels and has given rise to a paradigm shift in medicine towards personalized therapies. Computational analysis plays a pivotal role in integrating the various genome data and understanding the cellular response to a drug. Based on that data, molecular models can be constructed that incorporate the known downstream effects of drug-targeted receptor molecules and that predict optimal therapy decisions. In this article, we describe the different steps in the conceptual framework of computational modeling. We review resources that hold information on molecular pathways that build the basis for constructing the model interaction maps, highlight network analysis concepts that have been helpful in identifying predictive disease patterns, and introduce the basic concepts of kinetic modeling. Finally, we illustrate this framework with selected studies related to the modeling of important target pathways affected by drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e133d29b87cf2a314abfa9308ee05434a58d6cf" target='_blank'>Computational modeling of drug response with applications to neuroscience</a></td>
          <td>
            R. Herwig
          </td>
          <td>2014-12-01</td>
          <td>Dialogues in Clinical Neuroscience</td>
          <td>3</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b0d4a5d209975c2d5166fd635acb02a22157e56" target='_blank'>The crosstalk between EGF, IGF, and Insulin cell signaling pathways - computational and experimental analysis</a></td>
          <td>
            Rafal Zieliński, P. Przytycki, Jie Zheng, David Zhang, T. Przytycka, J. Capala
          </td>
          <td>2009-09-04</td>
          <td>BMC Systems Biology</td>
          <td>62</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6319b79882fa8bd07267c8cafc082740cd9779b6" target='_blank'>Computational modeling of the EGFR network elucidates control mechanisms regulating signal dynamics</a></td>
          <td>
            Dennis Wang, L. Cardelli, Andrew Phillips, Nir Piterman, J. Fisher
          </td>
          <td>2009-12-22</td>
          <td>BMC Systems Biology</td>
          <td>40</td>
          <td>74</td>
        </tr>

        <tr id="表皮生长因子受体(epidermal growth factor receptor: EGFR)在细胞信号转导中发挥了重要作用，调节细胞的增殖、分化与迁移。EGFR与多种癌症密切相关，这使其成为一个理想的抗肿瘤药物靶点。本文对EGFR的结构生物学及计算生物学的研究进展做一综述。通过结构生物学家的不懈努力，已经对EGFR的三维空间结构有了较为清晰的认识。计算机模拟是研究生物大分子动态过程的重要手段之一，可以在EGFR静态结构的基础上，模拟其在配体诱导的激活过程中发生的大尺度、长时程的动态构象变化。对EGFR的结构及其分子内部运动的研究，有助于揭示其发挥生物学功能的分子机理，为抗肿瘤药物的设计开发提供指导。 Epidermal growth factor receptor (EGFR) plays a critical role in the network of signal transduction, and regulates cell proliferation, differentiation, and migration. EGFR is implicated in different kinds of cancers, which makes it an excellent target for anti-tumor therapy. This review considers structural and computational studies of EGFR, and their impact on our understanding of EGFR function. With continuing efforts of biologists, many high-resolution structures of different regions in EGFR, which include the extracellular region and the intracellular tyrosine kinase do-main, have been solved. Starting from these structures, computer simulation techniques enable us to investigate large length-scale and long time-scale conformational changes during the EGFR activation. All the studies will be helpful to reveal the molecular mechanism in EGFR regulation, and provide valuable knowledge for developing anti-cancer drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/304b3eca9d24041459b6626ebf25b125ee7fd861" target='_blank'>表皮生长因子受体的结构与计算生物学研究进展 Progress in Structural and Computational Studies of Epidermal Growth Factor Receptor</a></td>
          <td>
            张泳辉, 张志勇
          </td>
          <td>2013-05-24</td>
          <td>Biophysics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/482aa18a6e8b3e568e08d3cc16aa642e322fd257" target='_blank'>Computational model of VEGFR2 pathway to ERK activation and modulation through receptor trafficking.</a></td>
          <td>
            W. Tan, A. Popel, F. Mac Gabhann
          </td>
          <td>2013-12-01</td>
          <td>Cellular signalling</td>
          <td>40</td>
          <td>68</td>
        </tr>

        <tr id="Successful drug development is a risky and lengthy process that can take over ten years and consume billions of dollars. Target discovery is a critical stage of drug development for the identification of key molecules and pathways that can be targeted by novel therapeutics to find more effective treatments. Due to the rapid development in artificial intelligence and machine learning techniques over the past decade, computational approaches have now emerged as powerful tools to unravel complex interactions within biological systems to identify novel therapeutic targets. In particular, modeling and analysis of intracellular molecular networks play a pivotal role in target discovery by enabling researchers to efficiently and simultaneously navigate massive amounts of biological data to identify potential therapeutic targets. Such technologies can significantly accelerate the prolonged process of development of innovative therapies for complex diseases. Besides highlighting the findings of the recently introduced extreme signaling failures in intracellular molecular networks, here we briefly review various methods for modeling and analysis of intracellular molecular networks and discuss how they can be utilized to predict potential drug targets within such complex signaling systems. Overall, this review emphasizes the significance of modeling and analysis of molecular networks for fast-tracking and rapid discovery of novel therapeutic targets; to pave the way for the development of more effective treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524c46fe6dd0743cfbc96de3c126a77081afdd10" target='_blank'>From Data to Knowledge: A Mini-Review on Molecular Network Modeling and Analysis for Therapeutic Target Discovery</a></td>
          <td>
            Mustafa Ozen, E. Emamian, A. Abdi
          </td>
          <td>2023-07-05</td>
          <td>Archives of Pharmacology and Therapeutics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The site-specific function of EGFR’s C-terminal tyrosines is interrogated by phosphotyrosine phosphoproteomics to build a data-driven model that links cell response to signaling nodes and elucidates novel associations in the EGFR signaling network. The epidermal growth factor receptor (EGFR) has been studied extensively because of its critical role in cellular signaling and association with disease. Previous models have elucidated interactions between EGFR and downstream adaptor proteins or showed phenotypes affected by EGFR. However, the link between specific EGFR phosphorylation sites and phenotypic outcomes is still poorly understood. Here, we employed a suite of isogenic cell lines expressing site-specific mutations at each of the EGFR C-terminal phosphorylation sites to interrogate their role in the signaling network and cell biological response to stimulation. Our results demonstrate the resilience of the EGFR network, which was largely similar even in the context of multiple Y-to-F mutations in the EGFR C-terminal tail, while also revealing nodes in the network that have not previously been linked to EGFR signaling. Our data-driven model highlights the signaling network nodes associated with distinct EGF-driven cell responses, including migration, proliferation, and receptor trafficking. Application of this same approach to less-studied RTKs should provide a plethora of novel associations that should lead to an improved understanding of these signaling networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d1bedc5d16450a77fda0fc2c816d3cd1cf3570c" target='_blank'>Predictive data-driven modeling of C-terminal tyrosine function in the EGFR signaling network</a></td>
          <td>
            Jacqueline S. Gerritsen, Joseph S Faraguna, R. Bonavia, F. Furnari, F. White
          </td>
          <td>2023-05-11</td>
          <td>Life Science Alliance</td>
          <td>2</td>
          <td>66</td>
        </tr>

        <tr id="A hybrid modeling framework integrating a highly specific mechanistic model with highly abstract empirical model is presented. With the growing interest in the scientific and medical community for identification of therapeutic targets in treatment of disease, it is necessary to develop predictive models that can describe cellular behavior in response to environmental cues. Intracellular signaling pathways form complex networks that regulate cellular response in both health and disease. Mechanistic (or white-box) models of biochemical networks are often unable to explain comprehensive cellular response due to lack of knowledge and/or intractable complexity (especially in events distal from the cell membrane). Empirical (or black-box) models may provide a less than accurate representation of cellular response due to data deficiency and/or loss of mechanistic detail. In the proposed framework, we use a mechanistic model to capture early signaling events and apply the resulting generated internal signals (along with external inputs) to a downstream empirical sub-model. The key construct in the approach is the treatment of a cell’s biochemical network as an encoder that creates a functional internal representation of external environmental cues. The signals derived from this representation are then used to inform downstream behaviors. Using this idea, we are able to create a comprehensive framework that describes important mechanisms with sufficient detail, while representing complex or unknown mechanisms in a more abstract form. The model is verified using published biological data describing T-Cells in immune response.Copyright © 2013 by ASME">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f470c01ee028f44eb08c099f9bfa4ffe012ae147" target='_blank'>Integrated Mechanistic-Empirical Modeling of Cellular Response Based on Intracellular Signaling Dynamics</a></td>
          <td>
            Michaelle N. Mayalu, H. Asada
          </td>
          <td>2013-10-01</td>
          <td></td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832b8a763bca673f9ef9a922d08084ef49416b11" target='_blank'>Systems Analysis of EGF Receptor Signaling Dynamics with Micro-Western Arrays</a></td>
          <td>
            Mark F. Ciaccio, Joel Wagner, Chih-Pin Chuu, D. Lauffenburger, R. Jones
          </td>
          <td>2010-01-24</td>
          <td>Nature methods</td>
          <td>209</td>
          <td>119</td>
        </tr>

        <tr id="One of the most exciting developments in signal transduction research has been the proliferation of studies in which a biological discovery was initiated by computational modeling. In this study, we review the major efforts that enable such studies. First, we describe the experimental technologies that are generally used to identify the molecular components and interactions in, and dynamic behavior exhibited by, a network of interest. Next, we review the mathematical approaches that are used to model signaling network behavior. Finally, we focus on three specific instances of ‘model‐driven discovery’: cases in which computational modeling of a signaling network has led to new insights that have been verified experimentally. Copyright © 2009 John Wiley & Sons, Inc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77dcdce9256a395c2cd486a7e8652b15cea1b2b4" target='_blank'>Computational modeling of mammalian signaling networks</a></td>
          <td>
            J. Hughey, Timothy K Lee, M. Covert
          </td>
          <td>2010-03-01</td>
          <td>Wiley Interdisciplinary Reviews: Systems Biology and Medicine</td>
          <td>60</td>
          <td>41</td>
        </tr>

        <tr id="Mitogen‐activated protein kinase (MAPK) cascades process a myriad of stimuli received by cell‐surface receptors and generate precise spatiotemporal guidance for multiple target proteins, dictating receptor‐specific cellular outcomes. Computational modeling reveals that the intrinsic topology of MAPK cascades enables them to amplify signal sensitivity and amplitude, reduce noise, and display intricate dynamic properties, which include toggle switches, excitation pulses, and oscillations. Specificity of signaling responses can be brought about by signal‐induced feedback and feedforward wiring imposed on the MAPK cascade backbone. Intracellular gradients of protein activities arise from the spatial separation of opposing reactions in kinase‐phosphatase cycles. The membrane confinement of the initiating kinase in MAPK cascades and cytosolic localization of phosphatases can result in precipitous gradients of phosphorylated signal‐transducers if they spread solely by diffusion. Endocytotic trafficking of active kinases driven by molecular motors and traveling waves of protein phosphorylation can propagate phosphorylation signals from the plasma membrane to the nucleus, especially in large cells, such as Xenopus eggs. Copyright © 2009 John Wiley & Sons, Inc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/549f0ea398377e29460e84284cf31387d6568803" target='_blank'>Four‐dimensional dynamics of MAPK information‐processing systems</a></td>
          <td>
            B. Kholodenko, M. Birtwistle
          </td>
          <td>2009-07-01</td>
          <td>Wiley Interdisciplinary Reviews: Systems Biology and Medicine</td>
          <td>87</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96df4b032feacafb2fb594f9048a1ee018956867" target='_blank'>Implication of dynamics in signal transduction and targeted disruption analyses of signaling networks</a></td>
          <td>
            F. Vital-Lopez, Amit Varshney, C. Maranas, A. Armaou
          </td>
          <td>2008-09-26</td>
          <td>Comput. Chem. Eng.</td>
          <td>2</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c171d43aff092d38e347f3e910c28c47a3be8cb2" target='_blank'>In silico prediction of ErbB signal activation from receptor expression profiles through a data analytics pipeline</a></td>
          <td>
            Arya A. Das, Elizabeth Jacob
          </td>
          <td>2018-04-07</td>
          <td>Journal of Biosciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88fea0d189be00b791e24ad740042c366d24234f" target='_blank'>In silico prediction of ErbB signal activation from receptor expression profiles through a data analytics pipeline</a></td>
          <td>
            Arya A. Das, Elizabeth Jacob
          </td>
          <td>2018-04-07</td>
          <td>Journal of Biosciences</td>
          <td>2</td>
          <td>5</td>
        </tr>

        <tr id="Signal network assumes a vital part in directing the principal cell capacities, for example, cell expansion, survival, separation and motility. Improvement and investigation of scientific model can help us gain a profound comprehension of the unpredictable conduct of ERK flag transduction organizes. This paper exhibits a computational model that offers an incorporated quantitative and dynamic reproduction of ERK flag transduction arranges, actuated by epidermal development figure. The mathematic demonstrate contains the enactment energy of the pathway, a huge number of input circles and association of platform proteins. The model gives knowledge into flag reaction connections between the authoritative of EGF to its receptor at the phone surface and actuation of downstream proteins in the flagging course. The diverse impact of positive and negative input circles of the ERK flag transduction pathway were for the most part examined, showing that criticism circles were the primary affecting variable to the swaying of ERK flag transduction pathway. The forecasts of this wavering of ERK enactment concur well with the writing. It can prompt flag floods of the downstream substrates and instigate relating natural practices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b0c50699f15bb521375edbc6061e541e8ccaea4" target='_blank'>Analysis of Regulated Kinase Signal Network through Feedback Loops in Extra-Cellular Signal</a></td>
          <td>
            C. Periyasamy Periyasamy
          </td>
          <td>2017-11-01</td>
          <td>Indonesian Journal of Electrical Engineering and Computer Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54562e54df78d295197115d989eb4c681b24700" target='_blank'>Modeling and simulation in signal transduction pathways: a systems biology approach.</a></td>
          <td>
            C. V. Suresh Babu, Eun Joo Song, Y. Yoo
          </td>
          <td>2006-03-01</td>
          <td>Biochimie</td>
          <td>39</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/957146d2142e84b2405b52eb0ff5cc23562a2d7b" target='_blank'>A multiscale study on the mechanisms of spatial organization in ligand-receptor interactions on cell surfaces</a></td>
          <td>
            Zhaoqian Su, Kalyani Dhusia, Yinghao Wu
          </td>
          <td>2021-03-23</td>
          <td>Computational and Structural Biotechnology Journal</td>
          <td>7</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/380612c9c06b9e10186175edd02ceaf28ac4dc69" target='_blank'>Generalized logical model based on network topology to capture the dynamical trends of cellular signaling pathways</a></td>
          <td>
            Fan Zhang, Haoting Chen, Li na Zhao, Hui Liu, T. Przytycka, Jie Zheng
          </td>
          <td>2016-01-11</td>
          <td>BMC Systems Biology</td>
          <td>2</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b06d234407efd97d798904ba97ebb9a73b5fb31e" target='_blank'>Methodologies for the modeling and simulation of biochemical networks, illustrated for signal transduction pathways: A primer</a></td>
          <td>
            Nesma ElKalaawy, A. Wassal
          </td>
          <td>2015-03-01</td>
          <td>Bio Systems</td>
          <td>20</td>
          <td>8</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>