<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              C181942f9c3307163599684cc55f9a9c096219a9
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Virtual%20cell/" class="md-tabs__link">
        
  
    
  
  Virtual cell

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../RA/" class="md-tabs__link">
        
  
    
  
  RA

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Prostate%20cancer/" class="md-tabs__link">
        
  
    
  
  Prostate cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Virtual%20patients/" class="md-tabs__link">
        
  
    
  
  Virtual patients

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Virtual%20cell/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual cell
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../RA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RA
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Prostate%20cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Prostate cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Virtual%20patients/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual patients
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-29 08:54:39 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Predicting Outcomes of Prostate Cancer Immunotherapy by Personalized Mathematical Models</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="Although therapeutic vaccination often induces markers of tumor-specific immunity, therapeutic responses remain rare. An improved understanding of patient-specific dynamic interactions of immunity and tumor progression, combined with personalized application of immune therapeutics would increase the efficacy of immunotherapy. Here, we developed a method to predict and enhance the individual response to immunotherapy by using personalized mathematical models, constructed in the early phase of treatment. Our approach includes an iterative real-time in-treatment evaluation of patient-specific parameters from the accruing clinical data, construction of personalized models and their validation, model-based simulation of subsequent response to ongoing therapy, and suggestion of potentially more effective patient-specific modified treatment. Using a mathematical model of prostate cancer immunotherapy, we applied our model to data obtained in a clinical investigation of an allogeneic whole-cell therapeutic prostate cancer vaccine. Personalized models for the patients who responded to treatment were derived and validated by data collected before treatment and during its early phase. Simulations, based on personalized models, suggested that an increase in vaccine dose and administration frequency would stabilize the disease in most patients. Together, our findings suggest that application of our method could facilitate development of a new paradigm for studies of in-treatment personalization of the immune agent administration regimens (P-trials), with treatment modifications restricted to an approved range, resulting in more efficacious immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd27e074fa5c1279e9443c302fd07550052e083" target='_blank'>Reconsidering the paradigm of cancer immunotherapy by computationally aided real-time personalization.</a></td>
          <td>
            Y. Kogan, Karin Halevi-Tobias, M. Elishmereni, Stanimir Vuk-Pavlović, Z. Agur
          </td>
          <td>2012-03-15</td>
          <td>Cancer research</td>
          <td>52</td>
          <td>35</td>
        </tr>

        <tr id="Although therapeutic vaccination often induces markers of tumor-specific immunity, therapeutic responses remain rare. An improved understanding of patient-specific dynamic interactions of immunity and tumor progression, combined with personalized application of immune therapeutics would increase the efficacy of immunotherapy. Here, we developed a method to predict and enhance the individual response to immunotherapy by using personalized mathematical models, constructed in the early phase of treatment. Our approach includes an iterative real-time in-treatment evaluation of patient-specific parameters from the accruing clinical data, construction of personalized models and their validation, model-based simulation of subsequent response to ongoing therapy, and suggestion of potentially more effective patient-specific modified treatment. Using a mathematical model of prostate cancer immunotherapy, we applied our model to data obtained in a clinical investigation of an allogeneic whole-cell therapeutic prostate cancer vaccine. Personalized models for the patients who responded to treatment were derived and validated by data collected before treatment and during its early phase. Simulations, based on personalized models, suggested that an increase in vaccine dose and administration frequency would stabilize the disease in most patients. Together, our findings suggest that application of our method could facilitate development of a new paradigm for studies of in-treatment personalization of the immune agent administration regimens (P-trials), with treatment modifications restricted to an approved range, resulting in more efficacious immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd27e074fa5c1279e9443c302fd07550052e083" target='_blank'>Reconsidering the paradigm of cancer immunotherapy by computationally aided real-time personalization.</a></td>
          <td>
            Y. Kogan, Karin Halevi-Tobias, M. Elishmereni, Stanimir Vuk-Pavlović, Z. Agur
          </td>
          <td>2012-03-15</td>
          <td>Cancer research</td>
          <td>52</td>
          <td>35</td>
        </tr>

        <tr id="We review the evolution, achievements, and limitations of the current paradigm shift in medicine, from the "one-size-fits-all" model to "Precision Medicine." Precision, or personalized, medicine - tailoring the medical treatment to the personal characteristics of each patient - engages advanced statistical methods to evaluate the relationships between static patient profiling, e.g., genomic and proteomic, and a simple clinically-motivated output, e.g., yes/no responder. Today, precision medicine technologies that have facilitated groundbreaking advances in oncology, notably in cancer immunotherapy, are approaching the limits of their potential. A different approach to treatment personalization involves methodologies focusing on the dynamic interactions in the patient-disease-drug system, as portrayed in mathematical modeling. Achievements of this scientific approach, in the form of algorithms for predicting personal disease dynamics and in individual patients under immunotherapeutic drugs, are reviewed as well. The contribution of the dynamic approaches to precision medicine is limited, at present, due to insufficient applicability and validation. Yet, the time is ripe for amalgamating together these two approaches, for maximizing their joint potential to personalize and improve cancer immunotherapy. We suggest the roadmaps towards achieving this goal, technologically, and urge clinicians, pharmacologists and computational biologists to join forces along the pharmaco-clinical track of this development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6066680b4266566576bc91fc28a95f18b898f502" target='_blank'>Accelerating the Development of Personalized Cancer Immunotherapy by Integrating Molecular Patients' Profiles with Dynamic Mathematical Models.</a></td>
          <td>
            Z. Agur, M. Elishmereni, U. Foryś, Y. Kogan
          </td>
          <td>2020-06-14</td>
          <td>Clinical pharmacology and therapeutics</td>
          <td>12</td>
          <td>35</td>
        </tr>

        <tr id="ABSTRACT Introduction: Recently, cancer immunotherapy has shown considerable success, but due to the complexity of the immune-cancer interactions, clinical outcomes vary largely between patients. A possible approach to overcome this difficulty may be to develop new methodologies for personal predictions of therapy outcomes, by the integration of patient data with dynamical mathematical models of the drug-affected pathophysiological processes. Areas covered: This review unfolds the story of mathematical modeling in cancer immunotherapy, and examines the feasibility of using these models for immunotherapy personalization. The reviewed studies suggest that response to immunotherapy can be improved by patient-specific regimens, which can be worked out by personalized mathematical models. The studies further indicate that personalized models can be constructed and validated relatively early in treatment. Expert opinion: The suggested methodology has the potential to raise the overall efficacy of the developed immunotherapy. If implemented already during drug development it may increase the prospects of the technology being approved for clinical use. However, schedule personalization, per se, does not comply with the current, ‘one size fits all,’ paradigm of clinical trials. It is worthwhile considering adjustment of the current paradigm to involve personally tailored immunotherapy regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa1f27465c2447e092b81bfaf901def44db5893e" target='_blank'>Employing dynamical computational models for personalizing cancer immunotherapy</a></td>
          <td>
            Z. Agur, Karin Halevi-Tobias, Y. Kogan, O. Shlagman
          </td>
          <td>2016-08-26</td>
          <td>Expert Opinion on Biological Therapy</td>
          <td>29</td>
          <td>35</td>
        </tr>

        <tr id="e16014 Background: CRPC is a heterogeneous disease, and it is challenging to determine optimal treatment sequences. Mathematical modeling can integrate diverse clinical and biological data to offer...">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bdae280459cf72e844679440757dac7465dce4b" target='_blank'>Optimizing personalized treatment sequences in metastatic castration resistant prostate cancer using tumor-biomarker based computational modeling.</a></td>
          <td>
            Shilpa Gupta, J. Gallaher, C. Lynch, Leah M. Cook, Arturo Araujo, Jacob G. Scott, J. Dhillon, J. Pow-Sang, D. Basanta
          </td>
          <td>2015-05-20</td>
          <td>Journal of Clinical Oncology</td>
          <td>1</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c700941e299210cb7fb770abc9a413bb44672b6" target='_blank'>Improving treatment strategies for patients with metastatic castrate resistant prostate cancer through personalized computational modeling</a></td>
          <td>
            J. Gallaher, Leah M. Cook, Shilpa Gupta, Arturo Araujo, J. Dhillon, Jong Y. Park, Jacob G. Scott, J. Pow-Sang, D. Basanta, C. Lynch
          </td>
          <td>2014-08-31</td>
          <td>Clinical & Experimental Metastasis</td>
          <td>26</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e0a1fad7eb147dd27cb52b88e6e64016bf8eb59" target='_blank'>Data-driven translational prostate cancer research: from biomarker discovery to clinical decision</a></td>
          <td>
            Yuxin Lin, Xiaojun Zhao, Zhijun Miao, Zhixin Ling, Xue-dong Wei, J. Pu, Jianquan Hou, Bairong Shen
          </td>
          <td>2020-03-07</td>
          <td>Journal of Translational Medicine</td>
          <td>23</td>
          <td>39</td>
        </tr>

        <tr id="Prostate cancer is the second most occurring cancer in men worldwide. To better understand the mechanisms of tumorigenesis and possible treatment responses, we developed a mathematical model of prostate cancer which considers the major signalling pathways known to be deregulated. We personalised this Boolean model to molecular data to reflect the heterogeneity and specific response to perturbations of cancer patients. A total of 488 prostate samples were used to build patient-specific models and compared to available clinical data. Additionally, eight prostate cell line-specific models were built to validate our approach with dose-response data of several drugs. The effects of single and combined drugs were tested in these models under different growth conditions. We identified 15 actionable points of interventions in one cell line-specific model whose inactivation hinders tumorigenesis. To validate these results, we tested nine small molecule inhibitors of five of those putative targets and found a dose-dependent effect on four of them, notably those targeting HSP90 and PI3K. These results highlight the predictive power of our personalised Boolean models and illustrate how they can be used for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18d52789867538ad9583601890d8157a8f696dd8" target='_blank'>Patient-specific Boolean models of signalling networks guide personalised treatments</a></td>
          <td>
            Arnau Montagud, Jonas Béal, L. Tobalina, P. Traynard, Vigneshwar Subramanian, B. Szalai, R. Alföldi, L. Puskás, A. Valencia, E. Barillot, J. Saez-Rodriguez, Laurence Calzone
          </td>
          <td>2022-02-15</td>
          <td>eLife</td>
          <td>44</td>
          <td>65</td>
        </tr>

        <tr id="The prediction of survival outcomes is a key factor in making decisions for prostate cancer (PCa) treatment. Advances in computer-based technologies have increased the role of machine learning (ML) methods in predicting cancer prognosis. Due to the various effective treatments available for each non-linear landscape of PCa, the integration of ML can help offer tailored treatment strategies and precision medicine approaches, thus improving survival in patients with PCa. There has been an upsurge of studies utilizing ML to predict the survival of these patients using complex datasets, including patient and tumor features, radiographic data, and population-based databases. This review aims to explore the evolving role of ML in predicting survival outcomes associated with PCa. Specifically, we will focus on the applications of ML in forecasting biochemical recurrence-free, progression to castration-resistance-free, metastasis-free, and overall survivals. Additionally, we will suggest areas in need of further research in the future to enhance the utility of ML for a more clinically-utilizable PCa prognosis prediction and treatment optimization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85de4d129f3584c3ef8b66fa578599797953c397" target='_blank'>Harnessing machine learning to predict prostate cancer survival: a review</a></td>
          <td>
            Sungun Bang, Young Jin Ahn, K. Koo
          </td>
          <td>2025-01-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1818d6d2f1497c68120dffa486144c93a75e7c6" target='_blank'>Improving treatment strategies for patients with metastatic castrate resistant prostate cancer through personalized computational modeling</a></td>
          <td>
            J. Gallaher, Leah M. Cook, Shilpa Gupta, Arturo Araujo, J. Dhillon, Jong Y. Park, Jacob G. Scott, J. Pow‐Sang, D. Basanta, C. Lynch
          </td>
          <td>2014-08-31</td>
          <td>Clinical & Experimental Metastasis</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="The goal of cancer immunotherapy is to boost a patient’s immune response to a tumor. Yet, the design of an effective immunotherapy is complicated by various factors, including a potentially immunosuppressive tumor microenvironment, immune-modulating effects of conventional treatments, and therapy-related toxicities. These complexities can be incorporated into mathematical and computational models of cancer immunotherapy that can then be used to aid in rational therapy design. In this review, we survey modeling approaches under the umbrella of the major challenges facing immunotherapy development, which encompass tumor classification, optimal treatment scheduling, and combination therapy design. Although overlapping, each challenge has presented unique opportunities for modelers to make contributions using analytical and numerical analysis of model outcomes, as well as optimization algorithms. We discuss several examples of models that have grown in complexity as more biological information has become available, showcasing how model development is a dynamic process interlinked with the rapid advances in tumor-immune biology. We conclude the review with recommendations for modelers both with respect to methodology and biological direction that might help keep modelers at the forefront of cancer immunotherapy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/161a29b3c32e37eaded637a7b1bb3c118aa6e089" target='_blank'>Addressing current challenges in cancer immunotherapy with mathematical and computational modeling</a></td>
          <td>
            A. Konstorum, A. Vella, A. Adler, R. Laubenbacher
          </td>
          <td>2017-06-01</td>
          <td>bioRxiv</td>
          <td>74</td>
          <td>48</td>
        </tr>

        <tr id="Cancer immunotherapy is a rapidly developing field, with numerous drugs and therapy combinations waiting to be tested in pre-clinical and clinical settings. However, the costly and time-consuming trial-and-error approach to development of new treatment paradigms creates a research bottleneck, motivating the development of complementary approaches. Computational modelling is a compelling candidate for this task, however, difficulties associated with the validation of such models limit their use in pre-clinical and clinical settings. Here we propose a bottom-up deterministic computational model to simulate tumour response to treatment with anti-programmed-death-1 antibodies (anti-PD-1). The model was built with validation in mind, and so contains minimum number of parameters, and only four free parameters. Moreover, all model parameters can be measured experimentally. Free parameters were tuned by fitting the model to experimental data from the literature, using B16-F10 murine melanoma implanted into wild type (C57BL/6), as well as into immunodeficient (NSG) mice strains, and treated with anti-PD-1 antibodies. The model’s predictive ability was verified on two independent datasets from literature with different but well-known inputs. To identify possible biomarkers of response to anti-PD-1 immunotherapy, sensitivity study of key model parameters was performed. Good agreement between the simulated tumour growth curves and the experimental data was achieved, with mean relative deviations in the range of 13%–20%. Our sensitivity study demonstrated that major histocompatibility complex (MHC) class I expression was the only parameter able to clearly discriminate responders from non-responders to anti-PD-1 therapy. Additionally, the results of sensitivity studies suggest that MHC class I expression might affect the predictive ability of other biomarkers that are currently used in the clinics, such as PD-1 ligand (PD-L1) expression. Interestingly, our model predicts the best response to anti-PD-1 therapy for subjects with moderate PD-L1 values. Such computational models show promise to support, guide and accelerate future immunotherapy research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88a12b75e53dfbf146a44de21daf57a611378db4" target='_blank'>Predicting tumour response to anti-PD-1 immunotherapy with computational modelling</a></td>
          <td>
            D. Valentinuzzi, U. Simončič, Katja Uršič, M. Vrankar, Maruša Turk, R. Jeraj
          </td>
          <td>2019-01-16</td>
          <td>Physics in Medicine & Biology</td>
          <td>23</td>
          <td>48</td>
        </tr>

        <tr id="Modern cancer immunotherapy has revolutionised oncology and carries the potential to radically change the approach to cancer treatment. However, numerous questions remain to be answered to understand immunotherapy response better and further improve the benefit for future cancer patients. Computational models are promising tools that can contribute to accelerated immunotherapy research by providing new clues and hypotheses that could be tested in future trials, based on preceding simulations in addition to the empirical rationale. In this topical review, we briefly summarise the history of cancer immunotherapy, including computational modelling of traditional cancer immunotherapy, and comprehensively review computational models of modern cancer immunotherapy, such as immune checkpoint inhibitors (as monotherapy and combination treatment), co-stimulatory agonistic antibodies, bispecific antibodies, and chimeric antigen receptor T cells. The modelling approaches are classified into one of the following categories: data-driven top-down vs mechanistic bottom-up, simplistic vs detailed, continuous vs discrete, and hybrid. Several common modelling approaches are summarised, such as pharmacokinetic/pharmacodynamic models, Lotka–Volterra models, evolutionary game theory models, quantitative systems pharmacology models, spatio-temporal models, agent-based models, and logic-based models. Pros and cons of each modelling approach are critically discussed, particularly with the focus on the potential for successful translation into immuno-oncology research and routine clinical practice. Specific attention is paid to calibration and validation of each model, which is a necessary prerequisite for any successful model, and at the same time, one of the main obstacles. Lastly, we provide guidelines and suggestions for the future development of the field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62c69387fe4f031dfad70073875b6cf0e2738498" target='_blank'>Computational modelling of modern cancer immunotherapy</a></td>
          <td>
            D. Valentinuzzi, R. Jeraj
          </td>
          <td>2020-10-22</td>
          <td>Physics in Medicine & Biology</td>
          <td>20</td>
          <td>48</td>
        </tr>

        <tr id="e17554Background: The ability to understand and predict at the time of diagnosis the trajectories of prostate cancer patients is critical for deciding the appropriate treatment plan. Evidence-based...">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/950af550cd9c98af1f1904139ec8b43aaaccee20" target='_blank'>Using longitudinal PSA values and machine learning for predicting progression of early stage prostate cancer in veterans.</a></td>
          <td>
            I. Danciu, Samantha Erwin, G. Agasthya, Tate Janet, Benjamin H. McMahon, G. Tourassi, A. Justice
          </td>
          <td>2020-05-20</td>
          <td>Journal of Clinical Oncology</td>
          <td>3</td>
          <td>103</td>
        </tr>

        <tr id="ABSTRACT Introduction Prostate cancer (PC) is the most common malignancy and accounts for a significant proportion of cancer deaths among men. Although initial therapy success can often be observed in patients diagnosed with localized PC, many patients eventually develop disease recurrence and metastasis. Without effective treatments, patients with aggressive PC display very poor survival. To curb the current high mortality rate, many investigations have been carried out to identify efficacious therapeutics. Compared to de novo drug designs, computational methods have been widely employed to offer actionable drug predictions in a fast and cost-efficient way. Particularly, powered by an increasing availability of next-generation sequencing molecular profiles from PC patients, computer-aided approaches can be tailored to screen for candidate drugs. Areas covered Herein, the authors review the recent advances in computational methods for drug discovery utilizing molecular profiles from PC patients. Given the uniqueness in PC therapeutic needs, they discuss in detail the drug discovery goals of these studies, highlighting their translational values for clinically impactful drug nomination. Expert opinion Evolving molecular profiling techniques may enable new perspectives for computer-aided approaches to offer drug candidates for different tumor microenvironments. With ongoing efforts to incorporate new compounds into large-scale high-throughput screens, the authors envision continued expansion of drug candidate pools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ab81ced264dcb3b837ac31f1cf1f804ed3d305d" target='_blank'>Computer-aided drug discovery strategies for novel therapeutics for prostate cancer leveraging next-generating sequencing data</a></td>
          <td>
            Weijie Zhang, R. S. Huang
          </td>
          <td>2024-06-11</td>
          <td>Expert Opinion on Drug Discovery</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Existing companion diagnostics have helped to match drug treatments to patients. However, they are largely restricted to single-molecule, single-time-point measurements, which cannot capture the full dynamic complexity of cancer biology. The development of multivariate and even dynamic biomarkers for diagnostic assays could allow more patients to benefit from improved drug regimens. Here we describe our work which provides a case study of multivariate biomarkers where we integrated experimental data generated using multivariate profiling technologies with a variety of computational modeling and simulation methods to identify such biomarkers and make clinical predictions on their therapeutic utility. We believe this approach of integrating multivariate profiling technologies and computational models, and iterating between experimental discovery and model predictions, will be required to develop the next generation of multivariate diagnostics and realize the promise of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00b3fb240007ac9db87c89bbfb3dec6bbfd7f53" target='_blank'>Many are better than one - next generation multivariate biomarkers for precision oncology</a></td>
          <td>
            Jinyan Du, D. Kirouac, B. Schoeberl
          </td>
          <td>2017-01-09</td>
          <td>Cancer cell & microenvironment</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract : TGF is a key regulator of bone development and is a key regulator in the vicious cycle of Prostate to Bone Metastasis. However, from a therapeutic standpoint, targeting TGF presents a dilemma given its noted pleiotropic roles. We hypothesize that an integrated computational modeling approach can be used to predict the temporal behavior of bone metastatic prostate cancer heterogeneous for TGF and further can be used to test the efficacy of putative therapies based on TGF inhibition. Multiple simulations and robust statistical post analyses will determine the role of TGF in the process. Our goal is to obtain a computational model of bone metastatic prostate cancer that can rapidly assess a putative TGF therapy in regards to efficacy and how to synergize with existing therapies to tackle heterogeneous metastases. This specialized research will be ultimately be used develop the PI professionally transition into an independent researcher.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d77f11161939c885dc4276795008c0c24714a14" target='_blank'>Deconvoluting the Complexity of Bone Metastatic Prostate Cancer via Computational Modeling</a></td>
          <td>
            Arturo Araujo
          </td>
          <td>2016-09-01</td>
          <td></td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Background Surgery and radiotherapy are primary nonconservative treatments for prostate cancer (PCa). However, personalizing treatment options between these treatment modalities is challenging due to unclear criteria. We developed an artificial intelligence (AI)-based model that can identify patients with localized PCa who would benefit more from either radiotherapy or surgery, thereby providing personalized clinical decision-making. Material and Methods Data from consecutive patients with localized PCa who received radiotherapy or surgery with complete records of clinicopathological variables and follow-up results in 12 registries of the Surveillance, Epidemiology, and End Results database were analyzed. Patients from 7 registries were randomly assigned to training (TD) and internal validation datasets (IVD) at a 9:1 ratio. The remaining 5 registries constituted the external validation dataset (EVD). TD was divided into training-radiotherapy (TRD) and training-surgery (TSD) datasets, and IVD was divided into internal-radiotherapy (IRD) and internal-surgery (ISD) datasets. Six models for radiotherapy and surgery were trained using TRD and TSD to predict radiotherapy survival probability (RSP) and surgery survival probability (SSP), respectively. The models with the highest concordance index (C-index) on IRD and ISD were chosen to form the final treatment recommendation model (FTR). FTR recommendations were based on the higher value between RSP and SSP. Kaplan-Meier curves were generated for patients receiving recommended (consistent group) and nonrecommended treatments (inconsistent group), which were compared using the log-rank test. Results The study included 118 236 patients, categorized into TD (TRD: 44 621; TSD: 41 500), IVD (IRD: 4949; ISD: 4621), and EVD (22 545). Both radiotherapy and surgery models accurately predicted RSP and SSP (C-index: 0.735-0.787 and 0.769-0.797, respectively). The consistent group exhibited higher survival rates than the inconsistent group, particularly among patients not suitable for active surveillance (P < .001). Conclusion FTR accurately identifies patients with localized PCa who would benefit more from either radiotherapy or surgery, offering clinicians an effective AI tool to make informed choices between these 2 treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8a01b4d5858eecb655570a9b27491181ab33fc9" target='_blank'>Artificial intelligence-based personalized clinical decision-making for patients with localized prostate cancer: surgery versus radiotherapy</a></td>
          <td>
            Yuwei Liu, Litao Zhao, Jiangang Liu, Liang Wang
          </td>
          <td>2024-07-31</td>
          <td>The Oncologist</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1634282e7ae75c7d334f1e1bfa1c9349ffa63ceb" target='_blank'>A Quantitative Paradigm for Decision-Making in Precision Oncology.</a></td>
          <td>
            D. Engelhardt, F. Michor
          </td>
          <td>2021-02-23</td>
          <td>Trends in cancer</td>
          <td>9</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35e3452557c3ff3edc2b9159f25f2193cdef0538" target='_blank'>Advancing Traditional Prostate-specific Antigen Kinetics in the Detection of Prostate Cancer: A Machine Learning Model.</a></td>
          <td>
            M. Perera, Lewis G Smith, I. Thompson, Geoff Breemer, N. Papa, Manish I. Patel, P. Swindle, Elliot Smith
          </td>
          <td>2021-12-01</td>
          <td>European urology focus</td>
          <td>5</td>
          <td>29</td>
        </tr>

        <tr id="Immune-cell therapy and targeting therapy are in rapid development to treat tumor diseases. However, current immune-cell therapy and targeting immunotherapy often face three challenges (three Ss): safety challenges such as cytokine releasing syndrome (C.R.S.); specificity targeting problems such as low efficacy caused by off-targeting tumor cells; unsatisfying payment are confounded to clinical patients and physicians. We have been studying immunotherapy for more than thirty years, and recently, personalized immunotherapy to treat tumor disease has been proposed. After we discovered quiescent genes from immune cells within the tumor microenvironment, we set up single-cell genomics analysis, studying heterogeneous immune responses from multiple tumor antigens (neo-antigen); here, we further introduce a new generation of immunotherapy module by using a machine-learning model to assess optimal immunotherapy. The machine-learning model combined with single-cell genomic analysis can predict optimal immune-cell (such as T-cells) and other optimal targeting drugs such as PD1 and CTLA4 inhibitors for the patient to use.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4a4206daa7daa879dd0d5fced51164f5c0d5807" target='_blank'>Machine-learning Modeling for Personalized Immunotherapy- An Evaluation Module</a></td>
          <td>
            Xiaonan Ying, Biaoru Li
          </td>
          <td>2022-11-18</td>
          <td>Biomedical journal of scientific & technical research</td>
          <td>2</td>
          <td>19</td>
        </tr>

        <tr id="Background/Objectives: Radioligandtherapy (RLT) with [177Lu]Lu-PSMA has been newly introduced as a routine treatment for metastatic castration-resistant prostate cancer (mCRPC). However, not all patients can tolerate the entire therapeutic sequence, and in some cases, the treatment may prove ineffective. In real-world conditions, the aim is to distinguish between patients who fully benefit from treatment (those who respond effectively and tolerate the entire therapeutic sequence) and those who do not respond or cannot tolerate the entire sequence. This study explores predictive factors to distinguish between fully beneficial RLT treatment patients (FBTP) and not fully beneficial RLT treatment patients (NFBTP). The objective was to enhance the understanding of predictive factors influencing RLT effectiveness and to highlight the significance of machine learning in optimizing patient selection for treatment planning. Methods: Data from 25 mCRPC patients, categorized as FBTP (11) or NFBTP (14) to RLT, were analyzed. The dataset included clinical, imaging, and biological parameters. Data analysis techniques, including exploratory data analysis and feature engineering, were used to develop machine learning models for predicting patient outcomes. Results: Imaging data analysis revealed statistically significant differences in the renal uptake intensity of Choline between the two groups. A discordance of FDG+ and PSMA− was identified as a potential indicator of NFBTP. The integration of biological data enhanced the model’s predictive capability, achieving an accuracy of 0.92, a sensitivity of 0.96, and a precision of 0.96. Adding blood parameters like neutrophils, leukocytes, and alkaline phosphatase greatly increased prediction accuracy. Conclusions: This study emphasizes the significance of an integrated approach that merges imaging and biological data, thereby augmenting the predictive accuracy of patient outcomes in RLT with [177Lu]Lu-PSMA. In particular, including Choline PET among the imaging parameters provides unique insights into the predictive factors affecting RLT efficacy. This approach not only deepens the understanding of predictive factors but also underscores the utility of machine learning in refining the patient selection process for optimized treatment planning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f4e67b540e59e7eed8dce320a7738ed116cb0c3" target='_blank'>Predicting Response to [177Lu]Lu-PSMA Therapy in mCRPC Using Machine Learning</a></td>
          <td>
            Kaiyuan Gong, Baptiste Magnier, Salomé L’hostis, Fanny Borrely, Sébastien Le Bon, Nadine Houede, Adel Mamou, L. Maimoun, P. Kotzki, Vincent Boudousq
          </td>
          <td>2024-10-23</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4624f66c438a80c62525b6af71f20d1edb159bd" target='_blank'>Integrating Systems Biology and an Ex Vivo Human Tumor Model Elucidates PD-1 Blockade Response Dynamics</a></td>
          <td>
            Munisha Smalley, Munisha Smalley, M. Przedborski, S. Thiyagarajan, Moriah Pellowe, A. Verma, N. Brijwani, Debika Datta, Misti Jain, Basavaraja U. Shanthappa, Vidushi Kapoor, K. Gopinath, D. C. Doval, K. Sabitha, G. Taroncher-Oldenburg, B. Majumder, P. Majumder, M. Kohandel, Aaron Goldman, Aaron Goldman
          </td>
          <td>2020-06-01</td>
          <td>iScience</td>
          <td>7</td>
          <td>73</td>
        </tr>

        <tr id="Cancer vaccines are a real application of the extensive knowledge of immunology to the field of oncology. Tumors are dynamic complex systems in which several entities, events, and conditions interact among them resulting in growth, invasion, and metastases. The immune system includes many cells and molecules that cooperatively act to protect the host organism from foreign agents. Interactions between the immune system and the tumor mass include a huge number of biological factors. Testing of some cancer vaccine features, such as the best conditions for vaccine administration or the identification of candidate antigenic stimuli, can be very difficult or even impossible only through experiments with biological models simply because a high number of variables need to be considered at the same time. This is where computational models, and, to this extent, immunoinformatics, can prove handy as they have shown to be able to reproduce enough biological complexity to be of use in suggesting new experiments. Indeed, computational models can be used in addition to biological models. We now experience that biologists and medical doctors are progressively convinced that modeling can be of great help in understanding experimental results and planning new experiments. This will boost this research in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69c47b06e91e2a05513e1c894c0a3634aec792db" target='_blank'>Cancer Vaccines: State of the Art of the Computational Modeling Approaches</a></td>
          <td>
            F. Pappalardo, F. Chiacchio, S. Motta
          </td>
          <td>2012-12-23</td>
          <td>BioMed Research International</td>
          <td>18</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10ba119ea0ec71f9d779c39e894ffc9ebbc7c71e" target='_blank'>Evaluation of machine learning strategies for imaging confirmed prostate cancer recurrence prediction on electronic health records</a></td>
          <td>
            J. Beinecke, Patrick Anders, T. Schurrat, D. Heider, M. Luster, D. Librizzi, Anne-Christin Hauschild
          </td>
          <td>2022-02-01</td>
          <td>Computers in biology and medicine</td>
          <td>8</td>
          <td>53</td>
        </tr>

        <tr id="5065 Background: Overall survival (OS) remains the gold standard endpoint for clinical trials in metastatic prostate cancer (metPC) yet requires extensive follow-up. We sought to develop a model utilizing short term (≤ 4 months) prostate specific antigen (PSA) kinetic data to predict the OS readout from phase 3 clinical trials, with the goal of accelerating readout time of trials for patients (pts) with metPC. Methods: Clinical and PSA data were obtained from 7 metPC trials: 6 randomized, double-blind, phase 3 trials (TITAN, COU-AA-301, COU-AA-302, LATITUDE, ACIS, and MAGNITUDE), and one multicenter, phase II trial (GALAHAD). We developed 18 PSA kinetic variables from the first 4 months of enrollment including: 50% decline in PSA (PSA50), 90% decline in PSA, PSA=0.1, and PSA=0.2 within 1, 2, 3, and 4 months; and slope and base of the exponential fit of PSA over the first 4 months on study). To balance pt and disease characteristics, TITAN, COU-AA-301, and ACIS were used to train the intermediate endpoint, with remaining studies held out as an external testing cohort. A random 60/40% split of the pooled pt-level data was performed as a sensitivity analysis. Adaptive least absolute shrinkage and selection operator (aLASSO)-based Cox proportional hazards models were applied to select previously identified prognostic baseline clinical variables with and without PSA kinetic variables most predictive of OS on five-fold cross validation. Performance with and without PSA kinetics was assessed using C-index, integrated Brier score (IBS), and time-dependent area under the receiver operating characteristic curve (tAUC) at 12, 24, 36, and 48 months. Results: The study included 6,451 pts with median follow-up of 22 months and 85,785 PSA values. aLASSO selected Eastern Cooperative Oncology Group performance status, disease extent, lactate dehydrogenase, albumin, hemoglobin, PSA, and alkaline phosphatase as baseline variables for the prognostic comparator model. In the model including PSA kinetics, aLASSO identified PSA50 at 4 months, PSA0.1 at 1 month, and PSA slope as kinetics providing additional intra-treatment information to improve prediction of OS. The model with PSA kinetics showed improved performance on the test trials: C-index (0.72 vs. 0.66), IBS (0.158 vs. 0.173), and tAUC (0.84, 0.78, 0.78, and 0.76 vs. 0.78, 0.71, 0.69, and 0.65 at times 12, 24, 36, and 48 months, respectively; p < 0.05.) Sensitivity analysis using a random split of the pooled pt-level data supported the importance of PSA50 at 4 months and PSA slope. Conclusions: Data from 6,451 mPC pts enrolled on prospective clinical trials were used to develop a machine learning model which included kinetic PSA data and predicted the long-term OS readout in Phase 3 trials with just the first four months of data. This model will be validated in independent data sets comprised of other completed phase 3 trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dac296b786d21d2a4c231f4e0d52d3f32e79335" target='_blank'>Development of a machine learning model to predict overall survival results from randomized clinical trials of patients with metastatic prostate cancer.</a></td>
          <td>
            A. Sabbagh, David A Quigley, Nicholas Lillis, Li Zhang, I. Friesner, A. Bailey, Rahul R Aggarwal, Meera R. Chappidi, Hari Singhal, Ke Zhang, J. Greshock, Jinhui Li, Margaret K. Yu, Eric J. Small, Julian C. Hong
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26940d74f22258b4d1b09adaa5657b39de18c8d6" target='_blank'>Survival analysis of localized prostate cancer with deep learning</a></td>
          <td>
            Xin Dai, Ji Hwan Park, Shinjae Yoo, N. D'Imperio, B. McMahon, C. Rentsch, J. Tate, A. Justice
          </td>
          <td>2022-10-24</td>
          <td>Scientific Reports</td>
          <td>6</td>
          <td>103</td>
        </tr>

        <tr id="Despite a growing body of knowledge on the mechanisms underlying the onset and progression of cancer, treatment success rates in oncology are at best modest. Current approaches use statistical methods that fail to embrace the inherent and expansive complexity of the tumor/patient/drug interaction. Computational modeling, in particular mechanistic modeling, has the power to resolve this complexity. Using fundamental knowledge on the interactions occurring between the components of a complex biological system, large-scale in silico models with predictive capabilities can be generated. Here, we describe how mechanistic virtual patient models, based on systematic molecular characterization of patients and their diseases, have the potential to shift the theranostic paradigm for oncology, both in the fields of personalized medicine and targeted drug development. In particular, we highlight the mechanistic modeling platform ModCell™ for individualized prediction of patient responses to treatment, emphasizing modeling techniques and avenues of application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed989fd87c7141bf188cf54581345f04a17c5b1" target='_blank'>Predictive Modeling of Drug Treatment in the Area of Personalized Medicine</a></td>
          <td>
            L. Ogilvie, C. Wierling, T. Kessler, H. Lehrach, B. Lange
          </td>
          <td>2015-12-06</td>
          <td>Cancer Informatics</td>
          <td>21</td>
          <td>124</td>
        </tr>

        <tr id="To allow innovative, progressive treatments to make the transition from concept to clinic, experimentally calibrated and clinically motivated mathematical models are essential. To aid the development of well-informed clinical trials, collaboration between mathematicians, computational modelers, biologists, and clinicians is required to allow qualitative hypotheses to take on a quantitative dimension. Such interdisciplinary science can allow both personalization and optimization of dose and scheduling of immunotherapeutic protocols, both as an independent therapy and in conjunction with other more traditional therapies, to streamline the transition from innovative concept to clinical practice and improve clinical outcomes for individual patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcc145eb2aa83b88341c321a673401f79702dde" target='_blank'>From concept to clinic: mathematically informed immunotherapy</a></td>
          <td>
            R. Walker, H. Enderling
          </td>
          <td>2015-10-01</td>
          <td>bioRxiv</td>
          <td>37</td>
          <td>37</td>
        </tr>

        <tr id="Supplemental Digital Content is available in the text">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dccb4073cae609b7eae12bb31450de91e35bdf1d" target='_blank'>Identification of novel biomarkers correlated with prostate cancer progression by an integrated bioinformatic analysis</a></td>
          <td>
            Zhifang Ma, Jianming Wang, Ling Ding, Yujun Chen
          </td>
          <td>2020-07-10</td>
          <td>Medicine</td>
          <td>5</td>
          <td>10</td>
        </tr>

        <tr id="Despite its great promise, personalized oncology still faces many hurdles, and it is increasingly clear that targeted drugs and molecular biomarkers alone yield only modest clinical benefit. One reason is the complex relationships between biomarkers and the patient's response to drugs, obscuring the true weight of the biomarkers in the overall patient's response. This complexity can be disentangled by computational models that integrate the effects of personal biomarkers into a simulator of drug–patient dynamic interactions, for predicting the clinical outcomes. Several computational tools have been developed for personalized oncology, notably evidence‐based tools for simulating pharmacokinetics, Bayesian‐estimated tools for predicting survival, etc. We describe representative statistical and mathematical tools, and discuss their merits, shortcomings and preliminary clinical validation attesting to their potential. Yet, the individualization power of mathematical models alone, or statistical models alone, is limited. More accurate and versatile personalization tools can be constructed by a new application of the statistical/mathematical nonlinear mixed effects modeling (NLMEM) approach, which until recently has been used only in drug development. Using these advanced tools, clinical data from patient populations can be integrated with mechanistic models of disease and physiology, for generating personal mathematical models. Upon a more substantial validation in the clinic, this approach will hopefully be applied in personalized clinical trials, P‐trials, hence aiding the establishment of personalized medicine within the main stream of clinical oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcee04c7c821c9ddaa231b2e95facdaa1201995e" target='_blank'>Personalizing oncology treatments by predicting drug efficacy, side‐effects, and improved therapy: mathematics, statistics, and their integration</a></td>
          <td>
            Z. Agur, M. Elishmereni, Y. Kheifetz
          </td>
          <td>2014-05-01</td>
          <td>Wiley Interdisciplinary Reviews: Systems Biology and Medicine</td>
          <td>38</td>
          <td>35</td>
        </tr>

        <tr id="Purpose PSA is currently the most commonly used screening indicator for prostate cancer. However, it has limited specificity for the diagnosis of prostate cancer. We aim to construct machine learning-based models and enhance the prediction of prostate cancer. Methods The data of 551 patients who underwent prostate biopsy were retrospectively retrieved and divided into training and test datasets in a 3:1 ratio. We constructed five PCa prediction models with four supervised machine learning algorithms, including tPSA univariate logistic regression (LR), multivariate LR, decision tree (DT), random forest (RF), and support vector machine (SVM). The five prediction models were compared based on model performance metrics, such as the area under the receiver operating characteristic curve (AUC), accuracy, sensitivity, specificity, calibration curve, and clinical decision curve analysis (DCA). Results All five models had good calibration in the training dataset. In the training dataset, the RF, DT, and multivariate LR models showed better discrimination, with AUCs of 1.0, 0.922 and 0.91, respectively, than the tPSA univariate LR and SVM models. In the test dataset, the multivariate LR model exhibited the best discrimination (AUC=0.918). The multivariate LR model and SVM model had better extrapolation and generalizability, with little change in performance between the training and test datasets. Compared with the DCA curves of the tPSA LR model, the other four models exhibited better net clinical benefits. Conclusion The results of the current retrospective study suggest that machine learning techniques can predict prostate cancer with significantly better AUC, accuracy, and net clinical benefits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7f6aeb94be850109d60e084d301edfd00b61cc9" target='_blank'>Machine Learning-Based Models Enhance the Prediction of Prostate Cancer</a></td>
          <td>
            Su-Jie Chen, Tengteng Jian, Changliang Chi, Yitian Liang, Xiao Liang, Ying Yu, F. Jiang, Ji Lu
          </td>
          <td>2022-07-06</td>
          <td>Frontiers in Oncology</td>
          <td>12</td>
          <td>7</td>
        </tr>

        <tr id="A major barrier for broadening the efficacy of immunotherapies for cancer is identifying key mechanisms that limit the efficacy of tumor infiltrating lymphocytes. Yet, identifying these mechanisms using human samples and mouse models for cancer remains a challenge. While interactions between cancer and the immune system are dynamic and non-linear, identifying the relative roles that biological components play in regulating anti-tumor immunity commonly relies on human intuition alone, which can be limited by cognitive biases. To assist natural intuition, modeling and simulation play an emerging role in identifying therapeutic mechanisms. To illustrate the approach, we developed a multi-scale mechanistic model to describe the control of tumor growth by a primary response of CD8+ T cells against defined tumor antigens using the B16 C57Bl/6 mouse model for malignant melanoma. The mechanistic model was calibrated to data obtained following adenovirus-based immunization and validated to data obtained following adoptive transfer of transgenic CD8+ T cells. More importantly, we use simulation to test whether the postulated network topology, that is the modeled biological components and their associated interactions, is sufficient to capture the observed anti-tumor immune response. Given the available data, the simulation results also provided a statistical basis for quantifying the relative importance of different mechanisms that underpin CD8+ T cell control of B16F10 growth. By identifying conditions where the postulated network topology is incomplete, we illustrate how this approach can be used as part of an iterative design-build-test cycle to expand the predictive power of the model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76e33d0ae43f2975586e25f3a91b591e8f0f75c5" target='_blank'>Inferring the Impact of Regulatory Mechanisms that Underpin CD8+ T Cell Control of B16 Tumor Growth In vivo Using Mechanistic Models and Simulation</a></td>
          <td>
            D. Klinke, Qing Wang
          </td>
          <td>2017-01-04</td>
          <td>Frontiers in Pharmacology</td>
          <td>4</td>
          <td>22</td>
        </tr>

        <tr id="Purpose Patients with advanced prostate cancer (PCa) often develop castration-resistant PCa (CRPC) with poor prognosis. Prognostic information obtained from multiparametric magnetic resonance imaging (mpMRI) and histopathology specimens can be effectively utilized through artificial intelligence (AI) techniques. The objective of this study is to construct an AI-based CRPC progress prediction model by integrating multimodal data. Methods and materials Data from 399 patients diagnosed with PCa at three medical centers between January 2018 and January 2021 were collected retrospectively. We delineated regions of interest (ROIs) from 3 MRI sequences viz, T2WI, DWI, and ADC and utilized a cropping tool to extract the largest section of each ROI. We selected representative pathological hematoxylin and eosin (H&E) slides for deep-learning model training. A joint combined model nomogram was constructed. ROC curves and calibration curves were plotted to assess the predictive performance and goodness of fit of the model. We generated decision curve analysis (DCA) curves and Kaplan–Meier (KM) survival curves to evaluate the clinical net benefit of the model and its association with progression-free survival (PFS). Results The AUC of the machine learning (ML) model was 0.755. The best deep learning (DL) model for radiomics and pathomics was the ResNet-50 model, with an AUC of 0.768 and 0.752, respectively. The nomogram graph showed that DL model contributed the most, and the AUC for the combined model was 0.86. The calibration curves and DCA indicate that the combined model had a good calibration ability and net clinical benefit. The KM curve indicated that the model integrating multimodal data can guide patient prognosis and management strategies. Conclusion The integration of multimodal data effectively improves the prediction of risk for the progression of PCa to CRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77ca645b49d19e3fed51f2de578bd01fc218d482" target='_blank'>Multimodal data integration for predicting progression risk in castration-resistant prostate cancer using deep learning: a multicenter retrospective study</a></td>
          <td>
            Chuan Zhou, Yunfeng Zhang, Sheng Guo, Yu-Qian Huang, Xiao-Ni Qiao, Rong Wang, Lian-Ping Zhao, De-Hui Chang, Li-Ming Zhao, Ming-Xu Da, Feng-Hai Zhou
          </td>
          <td>2024-03-14</td>
          <td>Frontiers in Oncology</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99d1d7540b7731d81c56b4f5c14763e8415bdeda" target='_blank'>Development of machine learning prognostic models for overall survival of prostate cancer patients with lymph node-positive</a></td>
          <td>
            Zi-he Peng, Juan-Hua Tian, Bo-Hong Chen, Hai-Bin Zhou, Hang Bi, Min-Xin He, Ming-Rui Li, Xin-Yu Zheng, Ya-Wen Wang, Tie Chong, Zhao-Lun Li
          </td>
          <td>2023-10-27</td>
          <td>Scientific Reports</td>
          <td>4</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86351c379548fe2384cb132fa07d7715d9811f7" target='_blank'>Bioinformatics for cancer immunology and immunotherapy</a></td>
          <td>
            P. Charoentong, Mihaela Angelova, M. Efremova, R. Gallasch, H. Hackl, J. Galon, Z. Trajanoski
          </td>
          <td>2012-09-18</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>43</td>
          <td>88</td>
        </tr>

        <tr id="This systematic review and meta-analysis evaluated the performance of machine learning models in predicting prostate cancer progression using lifestyle factors as predictive biomarkers to improve prognostic accuracy. Various models were analyzed, including Support Vector Machine (SVM), Logistic Regression (LR), Random Forest (RF), Multi-Layer Perceptron (MLP), and Convolutional Neural Networks (CNN). These models were applied to identify diagnostic and prognostic biomarkers and to enhance the forecasting of prostate cancer progression. The meta-analysis demonstrated high predictive effectiveness across models, with mean performance metrics of 0.901 AUC (Area Under the Curve), 0.914 F1 Score, 0.889 accuracy, and 0.914 sensitivity. Among the models, the Multi-Layer Perceptron (MLP) emerged as the most effective, achieving 97% accuracy and an AUC of 95.8%. These findings underscore the potential of machine learning to integrate lifestyle factors as predictive biomarkers, advancing precision oncology in prostate cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/332077d70a4f9e69fd71d16ac96395158080d88c" target='_blank'>Evaluating Machine Learning Models for Predicting Prostate Cancer Progression Using Lifestyle Factors: A Systematic Review and Meta-Analysis</a></td>
          <td>
            Ernest E. Onuiri, Oluwabamise J. Adeniyi
          </td>
          <td>2024-04-28</td>
          <td>Asian Journal of Engineering and Applied Technology</td>
          <td>3</td>
          <td>3</td>
        </tr>

        <tr id="Bone metastasis will impact most men with advanced prostate cancer. The vicious cycle of bone degradation and formation driven by metastatic prostate cells in bone yields factors that drive cancer growth. Mechanistic insights into this vicious cycle have suggested new therapeutic opportunities, but complex temporal and cellular interactions in the bone microenvironment make drug development challenging. We have integrated biologic and computational approaches to generate a hybrid cellular automata model of normal bone matrix homeostasis and the prostate cancer-bone microenvironment. The model accurately reproduces the basic multicellular unit bone coupling process, such that introduction of a single prostate cancer cell yields a vicious cycle similar in cellular composition and pathophysiology to models of prostate-to-bone metastasis. Notably, the model revealed distinct phases of osteolytic and osteogenic activity, a critical role for mesenchymal stromal cells in osteogenesis, and temporal changes in cellular composition. To evaluate the robustness of the model, we assessed the effect of established bisphosphonate and anti-RANKL therapies on bone metastases. At approximately 100% efficacy, bisphosphonates inhibited cancer progression while, in contrast with clinical observations in humans, anti-RANKL therapy fully eradicated metastases. Reducing anti-RANKL yielded clinically similar results, suggesting that better targeting or dosing could improve patient survival. Our work establishes a computational model that can be tailored for rapid assessment of experimental therapies and delivery of precision medicine to patients with prostate cancer with bone metastases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b154f3a85253de3ad746f7ae195a974020ddf1d8" target='_blank'>An integrated computational model of the bone microenvironment in bone-metastatic prostate cancer.</a></td>
          <td>
            Arturo Araujo, Leah M. Cook, C. Lynch, D. Basanta
          </td>
          <td>2014-05-01</td>
          <td>Cancer research</td>
          <td>75</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e254ac6a544fcd53988b822de2d8bf5741841de" target='_blank'>Predicting disease progression from short biomarker series using expert advice algorithm</a></td>
          <td>
            K. Morino, Yoshito Hirata, Ryota Tomioka, H. Kashima, K. Yamanishi, N. Hayashi, S. Egawa, K. Aihara
          </td>
          <td>2015-05-20</td>
          <td>Scientific Reports</td>
          <td>17</td>
          <td>73</td>
        </tr>

        <tr id="Biomarkers are cornerstones of clinical medicine, and personalized medicine, in particular, is highly dependent on reliable and highly accurate biomarkers for individualized diagnosis and treatment choice. Modern omics technologies, such as genome sequencing, allow molecular profiling of individual patients with unprecedented resolution, but biomarkers based on these technologies often lack the dynamic element to follow the progression of a disease or response to therapy. Here, we discuss computational models as a new conceptual approach to biomarker discovery and design. Being able to integrate a large amount of information, including dynamic information, computational models can simulate disease evolution and response to therapy with high sensitivity and specificity. By populating these models with personal data, they can be highly individualized and will provide a powerful new tool in the armory of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98e004d691f04078b1adb30f6413d975c5cf60e9" target='_blank'>Personalized Computational Models as Biomarkers</a></td>
          <td>
            W. Kolch, D. Fey
          </td>
          <td>2017-09-01</td>
          <td>Journal of Personalized Medicine</td>
          <td>14</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71f601bc802902739d1caf2eef5c5ece8999da52" target='_blank'>External Validation of a Digital Pathology-based Multimodal Artificial Intelligence Architecture in the NRG/RTOG 9902 Phase 3 Trial.</a></td>
          <td>
            Ashley E. Ross, Jingbin Zhang, Huei-Chung Huang, Rikiya Yamashita, Jessica Keim-Malpass, J. Simko, S. DeVries, Todd M. Morgan, L. Souhami, M. C. Dobelbower, L. S. McGinnis, Christopher U. Jones, R. Dess, K. Zeitzer, Kwang Choi, A. Hartford, J. Michalski, A. Raben, Leonard G. Gomella, A. O. Sartor, S. Rosenthal, H. Sandler, D. E. Spratt, S. Pugh, O. Mohamad, A. Esteva, E. Chen, E. Schaeffer, Phuoc T. Tran, F. Feng
          </td>
          <td>2024-01-31</td>
          <td>European urology oncology</td>
          <td>5</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c2eea065afcfb78730c57c62aa84588c986af9e" target='_blank'>Multi-scale Modeling in Clinical Oncology: Opportunities and Barriers to Success</a></td>
          <td>
            T. Yankeelov, G. An, O. Saut, E. Luebeck, A. Popel, B. Ribba, P. Vicini, Xiaobo Zhou, J. Weis, Kaiming Ye, G. Genin
          </td>
          <td>2016-07-06</td>
          <td>Annals of Biomedical Engineering</td>
          <td>61</td>
          <td>68</td>
        </tr>

        <tr id="Abstract : The vascular endothelial growth factor (VEGF) family of cytokines promotes vascularization, tumorigenesis and metastasis in many cancers. Our goal is to develop computational models that combine mechanistic topological data on the VEGF protein interaction network with gene expression datasets for a large population of prostate cancers. We have assembled databases of prostate cancer gene expression data, and analyzed the data using bioinformatic techniques, identifying key VEGF-based subgroups of prostate cancer plus biomarkers that identify these groups. This year, we have submitted and revised a manuscript based on this analysis, we expect it to be accepted soon. We have also created new computational models to simulate prostate cancer, based on the individualized gene expression data. We have used these models to simulate therapies that target the VEGF pathway, including anti-ligands such as bevacizumab and anti-receptors. Among are predictions are estimates of the high degree of variability in response to drugs that is predicted by the model, which is reflected in the results of clinical trials targeting VEGF in prostate cancer. This progress will continue into Year 3, where we will expand the VEGF family members included in our models, and extract key biomarkers indicative of likely success of VEGF- or VEGFR-targeting therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943f0ae223d83867545f3ddfaa3e3665b421fe82" target='_blank'>Computational Models of Anti-VEGF Therapies in Prostate Cancer</a></td>
          <td>
            F. M. Gabhann
          </td>
          <td>2014-06-01</td>
          <td></td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Cancer is one of the most complex phenomena in biology and medicine. Extensive attempts have been made to work around this complexity. In this study, we try to take a selective approach; not modeling each particular facet in detail but rather only the pertinent and essential parts of the tumor system are simulated and followed by optimization, revealing specific traits. This leads us to a pellucid personalized model which is noteworthy as it closely approximates existing experimental results. Method For years, research has focused on modeling tumor growth but not many studies have put forward a framework for the personalization of models. In the present study, a hybrid modeling approach which consists of cellular automata for discrete cell state representation and diffusion equations to calculate distribution of relevant substances in the tumor micro-environment is favored. Moreover, naive Bayesian decision making with weighted stochastic equations and a Bayesian network to model the temporal order of mutations is presented. The model is personalized according to the evidence using Markov Chain Monte Carlo. Ultimately, this way of thinking about tumor modeling leads us to a vascular multi-scale model of tumor growth. Results To validate the tumor model, a data set belonging to the A549 cell line is used. The data represents the growth of a tumor for 30 days. We optimize the coefficients of the stochastic decision making equations using first half of the timeline. Then we predict next 15 days of growth without any other supervision. Results are promising with their low error margin and simulated growth data is in line with laboratory results. Conclusion There are many subsystems which have an effect in the growth of a tumor. A detailed model which includes all of them is currently virtually impossible to implement. We have therefore focused on a system that only includes fundamental components in this study, and have evaluated its predictions. We propose novel probability functions to obtain a personalized model and estimate the individual importance (weights) of each with parameter optimization. Our approach of using simulated annealing for parameter estimation and the subsequent validation of the prediction with in-vitro tumor growth data are, to our knowledge, unique in the literature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eb0744c0daf20e8cf77cde7d299fa213ce48d95" target='_blank'>Personalized Tumor Growth Prediction with Multiscale Tumor Modeling</a></td>
          <td>
            Serbulent Unsal, A. Acar, M. Itik, Ayşe Kabataş, O. Gedikli, Feyyaz Ozdemir, K. Turhan
          </td>
          <td>2019-01-02</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>10</td>
        </tr>

        <tr id="47 Background: Prostate cancer represents a significant economic burden for health systems; the identification of risk factors that allows the identification of subgroups of patients that require closer follow-up or more intense therapeutic regimens has a direct impact on the prognosis and expenses for the patient, there is currently great interest in patients with high-volume mHSPC, this group is defined according to the number of lesions and their location, this could represent a challenge in decision making due to the variety of presentations regarding tumor burden, currently with the development of AI algorithms it is possible to estimate the PSMA TTB by identifying patients who are candidates for specific therapies without exposing them to comorbidities associated to aggressive regimens. Methods: Baseline PSMA PET scans of patients with histopathological corroborated diagnosis of prostate cancer and considered as high volume according to 68Ga PSMA PET/CT imaging in the period from October 2017 to June 2020 were retrospectively analyzed using an automated algorithm to estimate PSMA TTB, the median follow-up was of 44 +/- 9.7 months. These patients were divided into two groups according to the optimal cutoff values of PSMA TTB and estimate progression-free survival after initial therapy with Kaplan-Meier curves and correlation index between tumor burden and prostatic specific antigen. Results: 53 patients with a mean age of 71 years (+/- 5.1) were included, 17 with Gleason 7 (4+3/3+4), 18; G8 (4+4), 15; G9 (5+4/4+5) and 3; G10 (5+5), ISUP group 3 (n=17), group 4 (n=18) group 5 (n=18), the mean PSA value at diagnosis was 62.1 ng/dL (range 25.2- 201.5). PSMA TTB mean was 100.1cm3 (range 21cm3- 518.8 cm3). Log-rank test revealed that PSMA TTB lower than 45 cm3 (n=21) presented higher time of progression-free survival in comparison with patients with PSMA TTB higher than 45 cm3 (n=32) (31.1 mo vs 47.8 mo, p <0.003). TTB and PSA demonstrated a significant correlation (Pearson r= 0.526, p< 0.0001). There was no difference between groups of low and high PSMA TTB and ISUP groups or Gleason score. Sites of visceral disease were lung (n=11), (pleura=9) liver (n=9), adrenal, (n=5) brain, (n=2), and sites of extra spinal and pelvic disease were ribs (n=24) femur (n=16), skull (n=8), scapula (n=7), humerus (n=7). All patients had one or more lesions at previously described sites. Conclusions: PSMA TTB has great value in predicting progression-free survival of patients with newly diagnosed high-volume metastatic hormone-sensitive prostate cancer and has a strong correlation with PSA, strengthening the role of next-generation molecular imaging with PSMA PET as an invaluable tool in initial diagnosis and potentially in follow-up by providing information on the biological behavior of the disease in a systemic way.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80d962749ae6e7864d12c015b0f10aaa349a6130" target='_blank'>Prognostic value of artificial intelligence (AI) –driven tumor estimation of PSMA PET total tumor burden (TTB) in newly diagnosed high-volume metastatic hormone-sensitive prostate cancer (mHSPC).</a></td>
          <td>
            F. García-Pérez, L. P. Torres Agredo, Irma Soldevilla-Gallardo, Filiberto Lemus Ramirez, M.A. Álvarez Avitia, N. Sobrevilla-Moreno
          </td>
          <td>2024-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbc0c80b88a72a50473bc7de4561b10d699c5274" target='_blank'>Comprehensive analysis of the associations between clinical factors and outcomes by machine learning, using post marketing surveillance data of cabazitaxel in patients with castration-resistant prostate cancer</a></td>
          <td>
            H. Kazama, Osamu Kawaguchi, T. Seto, Kazuhiro Suzuki, H. Matsuyama, N. Matsubara, Yuki Tajima, T. Fukao
          </td>
          <td>2022-04-29</td>
          <td>BMC Cancer</td>
          <td>1</td>
          <td>38</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>