<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Covid/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              IBD
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>IBD</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-27 06:08:11 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on IBD</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on IBD</a><br>
      <a href="#recommended_articles">3. Recommended articles on IBD</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on IBD</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>IBD</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="To help understand the complex and therapeutically challenging inflammatory bowel diseases (IBDs), we developed a systems biology model of the intestinal immune system that is able to describe main aspects of IBD and different treatment modalities thereof. The model, including key cell types and processes of the mucosal immune response, compiles a large amount of isolated experimental findings from literature into a larger context and allows for simulations of different inflammation scenarios based on the underlying data and assumptions. In the context of a large and diverse virtual IBD population, we characterized the patients based on their phenotype (in contrast to healthy individuals, they developed persistent inflammation after a trigger event) rather than on a priori assumptions on parameter differences to a healthy individual. This allowed to reproduce the enormous diversity of predispositions known to lead to IBD. Analyzing different treatment effects, the model provides insight into characteristics of individual drug therapy. We illustrate for anti‐TNF‐α therapy, how the model can be used (i) to decide for alternative treatments with best prospects in the case of nonresponse, and (ii) to identify promising combination therapies with other available treatment options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f96c7a83f219c01d8f6dd860d6912bbf755670b0" target='_blank'>
                Cell‐level systems biology model to study inflammatory bowel diseases and their treatment options
                </a>
              </td>
          <td>
            Sabine Stübler, C. Kloft, W. Huisinga
          </td>
          <td>2023-02-01</td>
          <td>CPT: Pharmacometrics & Systems Pharmacology</td>
          <td>0</td>
          <td>35</td>

            <td><a href='../recommendations/f96c7a83f219c01d8f6dd860d6912bbf755670b0' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Crohn’s Disease (CD) results from inappropriate response toward commensal flora. Earlier studies described CD as a Th1 mediated disease. Current models view both phenotypes as a continuum of various permutations between Th1, Th2 and Th17 pathways compounded by a range of Treg disfunctions. In the present paper, we develop a mathematical model, by a system of differential equations, which describe the dynamic relations among these T cells and their cytokines. The model identities four groups of CD patients according to up/down regulation of Th1 and Th2. The model simulations show that immunosuppression by TNF-α blockage benefits the group with Th1High/Th2Low while, by contrast, the group with Th1Low/Th2High will benefit from immune activation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90f152b697a8db5328c108192005baef333b7e05" target='_blank'>
                Inflammatory Bowel Disease: How Effective Is TNF-α Suppression?
                </a>
              </td>
          <td>
            W. Lo, Violeta Arsenescu, R. Arsenescu, A. Friedman
          </td>
          <td>2016-11-01</td>
          <td>PLoS ONE</td>
          <td>9</td>
          <td>76</td>

            <td><a href='../recommendations/90f152b697a8db5328c108192005baef333b7e05' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/786df5602e24720f16b5af263b0b71aecc4a1bed" target='_blank'>
                A Multiscale Model of Interleukin-6–Mediated Immune Regulation in Crohn's Disease and Its Application in Drug Discovery and Development
                </a>
              </td>
          <td>
            Gaurav Dwivedi, L. Fitz, M. Hegen, Steven W. Martin, John M. Harrold, A. Heatherington, C. Li
          </td>
          <td>2014-01-01</td>
          <td>CPT: Pharmacometrics & Systems Pharmacology</td>
          <td>32</td>
          <td>33</td>

            <td><a href='../recommendations/786df5602e24720f16b5af263b0b71aecc4a1bed' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Several PI3K inhibitors are in clinical development for the treatment of various forms of cancers, including pan‐PI3K inhibitors targeting all four PI3K isoforms (α, β, γ, and δ), and isoform‐selective inhibitors. Diarrhea and immune‐mediated colitis are among the adverse events observed with PI3K inhibition which limits the maximal tolerated dose. A quantitative systems pharmacology model was developed to investigate PI3K‐inhibitor‐induced colitis. The effects of individual PI3K isoforms on relevant cellular pathways were incorporated into a mechanistic representation of mucosal inflammation. A virtual clinical population captures the observed clinical variability in the onset timing and rates of diarrhea and colitis for seven clinically tested PI3K inhibitors. Model‐based analysis suggests that colitis development is governed by both the inhibition of PI3Kδ, which drives T cell differentiation and proliferation, and PI3Kα, which regulates epithelial barrier integrity. Specifically, when PI3Kα is inhibited below a given threshold, epithelial barrier dysfunction precipitates an exaggerated T effector response due to PI3Kδ‐inhibition, leading to risk of diarrhea and colitis. This synergy explains why the lowest diarrhea and colitis rates are seen with the weakest PI3Kδ inhibition (alpelisib), and higher rates are seen with strong PI3Kδ inhibition if PI3Kα is even mildly inhibited (e.g., idelalisib), whereas strong PI3Kδ inhibition in the absence of PI3Kα inhibition does not result in high colitis rates (umbralisib). Thus, the model‐based analysis suggests that PI3Kα and δ inhibition play unique but synergistic roles in driving colitis. Finally, we explore if and how dose‐regimen might influence colitis rates for molecules that inhibit both PI3Kα and PI3Kδ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b73dd553ae42926deeaa8377af81aa8ce9864c" target='_blank'>
                Quantitative systems pharmacology model‐based investigation of adverse gastrointestinal events associated with prolonged treatment with PI3‐kinase inhibitors
                </a>
              </td>
          <td>
            K. Gadkar, Christina M. Friedrich, V. Hurez, Maria-Luisa Ruiz, L. Dickmann, Mohit Kumar Jolly, Leah K Schutt, J. Jin, J. Ware, S. Ramanujan
          </td>
          <td>2021-12-01</td>
          <td>CPT: Pharmacometrics & Systems Pharmacology</td>
          <td>5</td>
          <td>30</td>

            <td><a href='../recommendations/42b73dd553ae42926deeaa8377af81aa8ce9864c' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="A mechanistic, multistate, mathematical model of inflammatory bowel disease (IBD) was developed by including key biological mechanisms in blood and gut, including cell differentiation, cytokine production, and clinical biomarkers. The model structure is consistent between healthy volunteers and IBD disease phenotype, with 24 parameters changed between diseases. Modular nature of the model allows for easy incorporation of new mechanisms or modification of existing interactions. Model simulations for steady‐state levels of proteins and cells in the blood and gut using a population approach are consistent with published data. By simulating the response of two clinical biomarkers, C‐reactive protein and fecal calprotectin, to parameter perturbations, the model explores hypotheses for possible treatment mechanisms. With additional experimental validation and addition of drug treatments, the model provides a platform to test hypothesis on treatment effects in IBD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f58a190602a9b2e5c598c47151457c29f2bccd" target='_blank'>
                A Dynamic Quantitative Systems Pharmacology Model of Inflammatory Bowel Disease: Part 1 – Model Framework
                </a>
              </td>
          <td>
            K. Rogers, Steven W. Martin, I. Bhattacharya, R. Singh, S. Nayak
          </td>
          <td>2020-08-21</td>
          <td>Clinical and Translational Science</td>
          <td>18</td>
          <td>22</td>

            <td><a href='../recommendations/b5f58a190602a9b2e5c598c47151457c29f2bccd' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Inflammatory bowel disease (IBD) is a chronic and relapsing disease with multiple underlying influences and notable heterogeneity among its clinical and response‐to‐treatment phenotypes. There is no cure for IBD, and none of the currently available therapies have demonstrated clinical efficacies beyond 40%–60%. Data collected about its omics, pathogenesis, and treatment strategies have grown exponentially with time making IBD a prime candidate for artificial intelligence (AI) mediated discovery support. AI can be leveraged to further understand or identify IBD features to improve clinical outcomes. Various treatment candidates are currently under evaluation in clinical trials, offering further approaches and opportunities for increasing the efficacies of treatments. However, currently, therapeutic plans are largely determined using clinical features due to the lack of specific biomarkers, and it has become necessary to step into precision medicine to predict therapeutic responses to guarantee optimal treatment efficacy. This is accompanied by the application of AI and the development of multiscale hybrid models combining mechanistic approaches and machine learning. These models ultimately lead to the creation of digital twins of given patients delivering on the promise of precision dosing and tailored treatment. Interleukin‐6 (IL‐6) is a prominent cytokine in cell‐to‐cell communication in the inflammatory responses’ regulation. Dysregulated IL‐6‐induced signaling leads to severe immunological or proliferative pathologies, such as IBD and colon cancer. This mini‐review explores multiscale models with the aim of predicting the response to therapy in IBD. Modeling IL‐6 biology and generating digital twins enhance the credibility of their prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89edceb6395ad6a9a46a561fb37a23bbccd0f06b" target='_blank'>
                Computational models in inflammatory bowel disease
                </a>
              </td>
          <td>
            P. Pinton
          </td>
          <td>2022-02-05</td>
          <td>Clinical and Translational Science</td>
          <td>5</td>
          <td>40</td>

            <td><a href='../recommendations/89edceb6395ad6a9a46a561fb37a23bbccd0f06b' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for IBD'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>IBD</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="BACKGROUND
The genetic etiology of late-onset Alzheimer's disease (LOAD) is complex, with over 75 identified loci contributing to disease risk. Recent efforts of the MODEL-AD consortia have yielded several dozen mouse strains harboring variation designed to model LOAD risk alleles. Given the complex genetic architecture of LOAD, developing animal models that combine multiple risk alleles is likely essential to improving the fidelity of these models to human disease. However, it is not clear which combinations of risk alleles are optimal for modeling disease features in mice. In this study we utilize transcriptome data from existing strains to prioritize which variants should be combined in future model development.


METHOD
We estimated the effects of genetic variants and environmental perturbations on gene expression using multiple linear regression. The computed gene-level effects were used as features in separate regression models fit to changes in human transcriptomes from AMP-AD cohorts and previously described disease subtypes. To mitigate multicollinearity and allow for non-linear relationships, we employed both regularized linear models (LASSO, SVM) and non-linear models (Random Forest, Neural Network). Variable importance was extracted from the fit models for each regression analysis. We evaluated the importance of each feature (e.g. genetic variant) across models in an ensemble approach.


RESULT
The top mouse features prioritized by the different regression models were largely similar. Moreover, similar features were also prioritized across the different human cohorts. The top mouse features included a humanized Aβ and variants in SPI1, ABCA7, Epha1, and Sorl1. Intriguingly there were distinct sets of strains prioritized for different disease sub-types. Looking across cohorts there were generally a typical (i.e. inflammatory) sub-type and a non-inflammatory sub-type. The inflammatory subtypes prioritized similar features to the overall cohorts, while the priorities for non-inflmmatory subtypes included Cd2ap, Shc2, and Mtmr4.


CONCLUSION
We identified consistent predictions regarding which mouse factors are most influential in modeling disease across model classes and human cohorts. Ensembling the models was a useful approach for feature selection. These priorities informed the selection of mouse strains to breed in the development of polygenic LOAD mouse models by the IU-JAX-PITT MODEL-AD center.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8986c1c94587029965f5bcb6049896cebe69670" target='_blank'>
              Basic Science and Pathogenesis.
              </a>
            </td>
          <td>
            G. Cary, Ravi S. Pandey, A. Uyar, Catrina A. Spruce, Annat Haber, Gregory W. Carter
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Background Genome‐wide association studies (GWAS) have identified close to one hundred loci associated with Alzheimer’s disease (AD) risk. However, for most of these loci we do not understand the underlying mechanism leading to disease. Crispr genome editing in human induced pluripotent stem cells (hiPSCs) provides a model system to study the effects of these genetic variants in a disease relevant cell type. Method We used integrative genomics to identify the cell types and genes that are most likely to explain the disease risk identified in GWAS and the direction of effect associated with elevated or reduced risk. We used crispr genome editing to modify expression of these candidate genes and tested the impact of modulating gene expression on myeloid cell function and AD associated phenotypes in vitro and in vivo. Result AD risk alleles are enriched in myeloid cell enhancers but also in certain gene families. We have focused our functional studies on two such gene families: MS4A4A/MS4A6A and LACTB/B2. MS4A4A/MS4A6A are implicated in TREM2 signaling, and are major regulators of sTREM2 levels in cerebrospinal fluid (CSF) whereas LACTB/B2 are lactamases expressed in mitochondria. LACTB is the major modulator of succinyl carnitine levels, a metabolite in CSF associated with AD. Conclusion iPSC provide powerful models for understanding the molecular mechanisms underlying genetic risk for AD">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1575374a49618e38fd6db8a54bcdf541c0140aad" target='_blank'>
              Use of human iPSC lines to model Alzheimer’s genetic risk and understand disease mechanism
              </a>
            </td>
          <td>
            Alison M. Goate, Alexandra Münch, Carmen Romero Molina, Anna Podleśny-Drabiniok, Grace Peppler, Francesca Garretti, Tulsi Patel, Hyo Lee, Edoardo Marcora
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & Dementia</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="BACKGROUND
Alzheimer's disease (AD) is the most common global cause of dementia, with no real cure available. Despite extensive genetic findings from large-scale genetic associations and similar studies, our understanding of AD is largely hindered by its long, asymptomatic progression with limited access to human brain tissue. Animal models like the marmoset allow for longitudinal analysis of disease by enabling the assaying of disease-specific phenotypes that mimic human pathophysiology. Here, we have analyzed genetic associations with disease biomarkers in a cohort of wild type and PSEN1 C410Y mutant marmosets.


METHOD
Whole genome sequencing was carried out for a cohort of largely wild-type and a few PSEN1 mutant carriers. A wide variety of disease associated biomarkers were analyzed to assess for the progression of neurodegeneration. The correlation structure underlying collected biomarkers and related phenotypes was calculated, separating for sex. Composite phenotypes were computed based on correlation strength, and genetic associations studied via QTLs for composite and individual phenotypes.


RESULT
Females showed greater correlations between disease specific biomarkers like total tau (tTau), neurofilament light protein (NfL), the astrocytic marker GFAP, and β-amyloid peptides with 40 and 42 amino acids, as well as the Aβ42:Aβ40 ratio. Correlations between markers were found to be weaker but still significant in males. Variants predicted to be pathogenic were found for many genetic loci implicated by human genome-wide association studies. Standing genetic variation in the marmosets was found akin to what is seen in humans, but overall population structures varied significantly between the two species.


CONCLUSION
Correlations between AD-relevant biomarkers in marmosets exceeded those observed in humans, which were not explained by age. This suggests a strong potential genetic contribution. Leveraging possible heritable associations with disease markers discovered in the marmosets to explain the progression of neurodegeneration could directly provide insights into human AD biology from a genetic point of view.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebe3cda9d171e22e97a6fd8a2ff6f0ec45af4148" target='_blank'>
              Basic Science and Pathogenesis.
              </a>
            </td>
          <td>
            Sonal Kumar, Catrina A. Spruce, Annat Haber, Laura Schultz, Hasi Huhe, Takeshi Murai, Sang‐Ho Choi, Stephanie Hachem, Yongshan Mou, Seung‐Kwon Ha, Jung Eun Park, Lauren Bailey, Lauren Mongeau, Andrew DeSana, Brianne Stein, L. Schaeffer, G. E. Homanics, Afonso C Silva, S. J. Rizzo, Gregory W. Carter
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND
MODEL-AD (Model Organism Development and Evaluation for Late-onset AD) is developing, characterizing, and distributing novel mouse models expressing humanized, clinically relevant genetic risk factors. Models expressing human-relevant risk genetic risk factors are expected to better phenocopy LOAD than widely used transgenic models.


METHOD
Here, two genetic risk factors APOE4 and Trem2*R47H, were incorporated into C57BL/6J (B6) mice along with humanized amyloid-beta to produce the LOAD2 model. LOAD2 and control mice were aged up to 24 months with some being provided in the absence or presence of normal chow or a high fat/high sugar diet (LOAD2 HFD) from two months of age. A phenotyping pipeline was employed to evaluate disease outcomes observed in human patients, including in vivo imaging, brain and blood biomarker and cytokine analyses, multi-omics (transcriptomics and proteomics), neuropathology and behavior.


RESULT
By 18 months, unlike control mice (e.g., LOAD2 mice fed a control diet, CD), LOAD2 HFD mice presented subtle but significant loss of neurons in the cortex, elevated levels of insoluble AΒ42 in the brain, and increased plasma neurofilament light chain (NfL). Transcriptomics and proteomics showed changes in gene/proteins relating to a variety of disease-relevant processes including lipid metabolism and synaptic function. In vivo imaging revealed an age-dependent reduction in brain region volume (MRI) and neurovascular uncoupling (PET/CT). LOAD2 HFD mice also showed a learning deficit based on a Touchscreen cognitive assay.


CONCLUSION
Despite the absence of hallmark amyloid and Tau pathologies, collectively these data support the use of LOAD2 HFD mice reveal this model as important for preclinical studies that target other features of LOAD independent of amyloid and tau.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b85088f21e24b39fd5beac702af2aa3803f31897" target='_blank'>
              Basic Science and Pathogenesis.
              </a>
            </td>
          <td>
            Adrian L. Oblak
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Alzheimer's disease (AD) is a multi-factorial and complex disease, with the risk of developing disease still largely unknown despite numerous genetic and epidemiological studies over recent years. Several genetic and modifiable lifestyle risk factors are known to contribute to disease etiology, and epigenetic mechanisms are suggested to also contribute my mediating their interaction. It is now ten years since we published the first cross-tissue epigenome-wide association study (EWAS) of DNA methylation in AD post-mortem brain samples, with subsequent studies nominating robust and reproducible alterations in genes such as ANK1, HOXA3 and RHBDF2. We have leveraged these studies to perform large-scale epigenomic meta-analyses, reporting 220 differentially methylated loci that are altered in AD cortex, and which we are currently comparing to signatures in other neurodegenerative diseases. More recently we have undertaken DNA methylomic studies in blood in a quest to identify disease-associated alterations in an easily accessible tissue, which could be potentially explored from a biomarker perspective. Finally, our most recent work has explored the contribution of non-coding RNAs to regulating gene expression in AD cortex, where we have meta-analyzed microRNA expression in ∼1,000 post-mortem cortical brain samples, establishing cell-type specific disease-associated signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc3173a776774bd6cd1cfe893f0a03b3767130af" target='_blank'>
              Basic Science and Pathogenesis.
              </a>
            </td>
          <td>
            K. Lunnon
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="BACKGROUND
Alzheimer's disease (AD) missing heritability remains extensive despite numerous genetic risk loci identified by genome-wide association or sequencing studies. This has been attributed, at least partially, to mechanisms not currently investigated by traditional single-marker/gene approaches. Polygenic Risk Scores (PRS) aggregate sparse genetic information across the genome to identify individual genetic risk profiles for disease prediction and patient risk stratification. Recent advancements have pivoted on innovative approaches utilizing OMICS data to construct such risk scores.


METHOD
We employed a random forest algorithm to identify a list of gene candidates from bulk RNA sequencing data in prefrontal cortex from 565 AD brain samples (non-Hispanic Whites, n = 399; Hispanics, n = 113; African American, n = 12) across six U.S. brain banks. Subsequently, we calculated their effect size on Braak staging using regression models to construct a polytranscriptomic risk score (PTRS). We employed two distinct models: "Ethnicity-Agnostic" Model (randomly assigning samples to training and testing samples) and "Ethnicity-Aware" Model (assigning NHW samples to training and Hispanics to testing sample). Analysis of variance and the receiver operating characteristics area under the curve (ROC AUC) was used to evaluate PTRS's classification performances. We validated findings using the Religious Orders Study/Memory and Aging Project study (ROS/MAP, n = 1,095).


RESULT
We found a significant difference in PTRS between samples with low vs. high Braak stages (≤4 vs. ≥5, p = 1*E-04; Figure 1 upper panel). AUC was found to be 79-81%, consistently in both Ethnicity-Agnostic and Ethnicity-Aware models (Figure 1 lower panel). Finally, the PTRS in ROS/MAP yielded a similar classification performance (p = 2*E-04, AUC = 77%).


CONCLUSION
contrary to prior studies, we developed a PTRS with optimal transferability across ethnicities. This underscores the importance of developing novel tools to stratify and harmonize large brain repositories for AD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/427552bf1cf235cabfde1e7980aa326defe372dc" target='_blank'>
              Basic Science and Pathogenesis.
              </a>
            </td>
          <td>
            Basilio Cieza, Zikun Yang, D. Reyes-Dumeyer, Annie J Lee, Brittany N. Dugger, Lee-Way Jin, Melissa E. Murray, Dennis W. Dickson, M. Pericak-Vance, Jeffery M. Vance, Tatiana M Foroud, Richard Mayeux, G. Tosto
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>157</td>
        </tr>

        <tr id="BACKGROUND
Alzheimer's disease (AD) exhibits considerable phenotypic heterogeneity, suggesting the potential existence of subtypes. AD is under substantial genetic influence, thus identifying systematic variation in genetic risk may provide insights into disease origins. We previously identified a genetic heterogeneity across two levels. The first level, which was disease-relevant but not disease-specific, comprised three clusters (termed "constellations") driven by different correlation patterns in a region of extended LD surrounding the MAPT locus. The second level reflected disease-specific genetic subtypes of AD. These structures were previously found to replicate in an independent dataset. Here, we investigate whether genetic subtypes demonstrate different patterns of biomarker and cognitive trajectories.


METHOD
In a discovery dataset from the UK Biobank (AD cases = 2,739, controls = 5,478), we applied principal component analysis (PCA) to AD-associated variants, allowing us to identify three constellations. Subsequently, a biclustering algorithm identified two distinct disease-specific genetic signatures among subsets of cases not found in controls. We classified individuals with mild cognitive impairment (MCI; n = 399) from the Alzheimer's Disease Neuroimaging Initiative (ADNI) into constellations and determined whether they were a member of either bicluster (i.e., genetic subtype) or not. Longitudinal change in cognitive performance (Preclinical Alzheimer's Cognitive Composite; PACC), amyloid (florbetapir PET) and p-tau (CSF) was compared between the two bicluster groups and with non-bicluster cases.


RESULT
The disease-relevant constellations and disease-specific bicluster structure was apparent in the ADNI MCI sample. Individuals with genetic signatures resembling bicluster 2 exhibited greater decline on the PACC compared to the non-bicluster (p = 0.022) and bicluster 1 groups (p = 0.004). Bicluster 1 demonstrated a somewhat greater increase in florbetapir over time compared to the non-bicluster group, but this difference was not significant (p = 0.129). Bicluster 2 demonstrated a significantly greater increase of CSF p-tau over time compared to the non-bicluster group (p = 0.041) CONCLUSION: These results provide additional evidence for the existence of AD genetic subtypes in an independent dataset. The genetic subtypes exhibit differential pathological accumulation and cognitive decline over time, suggesting that this genetic variation has meaningful consequences for downstream processes and disease manifestation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc7e8cfe6ee0d95cb62d7b4ac752cf7aad9ddf43" target='_blank'>
              Basic Science and Pathogenesis.
              </a>
            </td>
          <td>
            J. Elman, Nicholas J Schork, A. Rangan
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Over the last three decades, human genetics has gone from dissecting high-penetrance Mendelian diseases to discovering the vast and complex genetic etiology of common human diseases. In tackling this complexity, scientists have discovered the importance of numerous genetic processes – most notably functional regulatory elements – in the development and progression of these diseases. Simultaneously, scientists have increasingly used multiplex assays of variant effect to systematically phenotype the cellular consequences of millions of genetic variants. In this article, we argue that the context of genetic variants – at all scales, from other genetic variants and gene regulation to cell biology to organismal environment – are critical components of how we can employ genomics to interpret these variants, and ultimately treat these diseases. We describe approaches to extend existing experimental assays and computational approaches to examine and quantify the importance of this context, including through causal analytic approaches. Having a unified understanding of the molecular, physiological, and environmental processes governing the interpretation of genetic variants is sorely needed for the field, and this perspective argues for feasible approaches by which the combined interpretation of cellular, animal, and epidemiological data can yield that knowledge.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc2f3e163ee6e921d0ab8446acca00c0495f358b" target='_blank'>
              Understanding genetic variants in context
              </a>
            </td>
          <td>
            Nasa Sinnott-Armstrong, Stanley Fields, Fritz Roth, Lea M. Starita, Cole Trapnell, Judit Villen, Douglas M. Fowler, C. Queitsch
          </td>
          <td>2024-12-03</td>
          <td>eLife</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract Genetic risk factors for late-onset Alzheimer’s disease (LOAD) have been extensively studied in the general population of European background. Indeed, significant genetic variants with low to moderate contributions to LOAD have been used to calculate Polygenic Risk Scores (PRS) to identify individuals with a higher genetic risk of LOAD. The Long Life Family Study (LLFS) is characterized by having exceptionally long-lived individuals at lower risk of LOAD. With this study, we will uncover whether the described genetic risk factors for LOAD impact AD risk in LLFS, limited to US sites. In LLFS, PRS based on two large GWASs in the general population did not predict clinical AD status. The LLFS population did not have lower allele frequencies of risk alleles, but rather lacked putative effects in these risk alleles, with only 6.7% of SNPs from GWAS 1 and 7.7% from GWAS 2 were associated with AD. We then examine the relations between PRS and endophenotype, namely AD biomarkers (Aβ40, Aβ42, Aβ42/Aβ40, NfL, and GFAP). Levels of AD biomarkers were not positively associated with PRS, which can be explained by the low frequencies of SNPs associated with biomarker concentrations (3.3%-6.7% for GWAS 1; 2.6%-6.5% for GWAS 2). Here, we show that genetic risk factors for AD in the general population do not necessarily explain AD risk in a cohort of long-lived individuals; therefore, genetic risk factors based on large-scale GWASs ascertained from the general population need to be examined with caution when applied to a cohort ascertained for exceptional longevity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3d28c5a1867a604a8b59fadcd2b60c0897a17d2" target='_blank'>
              LIMITATIONS OF POLYGENIC RISK SCORES FOR DEMENTIA IN FAMILIAL EXCEPTIONAL AGING: IMPACT OF SAMPLING
              </a>
            </td>
          <td>
            L. Xicota, Rong Cheng, Sandra Barral Rodriguez, Lawrence Honig, Joseph Lee
          </td>
          <td>2024-12-01</td>
          <td>Innovation in Aging</td>
          <td>0</td>
          <td>3</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>