<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              9d1e185348dca2c036fe189abba3ab6cd1fb5a7a
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-21 10:42:27 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Signaling switches and bistability arising from multisite phosphorylation in protein kinase cascades</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="Markevich et al. developed a computational model for a mechanism, not previously described, whereby signaling pathways can exhibit bistable behavior and switch between stable "on" and "off" states. Mitogen-activated protein kinase (MAPK) signaling pathways, which play critical roles in cell proliferation, differentiation, and apoptosis, generally function as a three-tiered cascade in which an activated (phosphorylated) kinase at one level phosphorylates and thereby activates the kinase at the subsequent level. To be fully activated, these MAPKs, MAPK kinases (MAPKKs), and MAPKK kinases must be phosphorylated at two sites. The MAPK cascades are regulated by multiple feedback loops, and the existence of either positive or double-negative feedback regulation is generally thought to be necessary for a cascade to function as a bistable system. Markevich et al., however, present a model in which bistability of the MAPK signaling pathway arises from a distributive kinetic mechanism for the dual phosphorylation and dephosphorylation of the kinase and is independent of feedback regulation. (A distributive mechanism implies that the kinase or phosphatase releases the monophosphorylated intermediate, which must then interact with enzyme again before it is converted into the final doubly phosphorylated or completely dephosphorylated state.) Their model was consistent with the reported kinetic properties of the extracellular signal-regulated kinase (ERK) kinase (MEK) and phosphatase (MKP3). The authors suggest that multisite phosphorylation, which takes place on proteins involved in many signaling systems, may represent a widespread mechanism for conferring switch-like behavior on various signaling systems. N. I. Markevich, J. B. Hoek, B. N. Kholodenko, Signaling switches and bistability arising from multisite phosphorylation in protein kinase cascades. J. Cell Biol. 164, 353-359 (2004). [Abstract] [Full Text]">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7053bd2d18f08169a4698ad2a73b821fce3419" target='_blank'>Flipping a Signaling Switch</a></td>
          <td>

          </td>
          <td>2004-02-10</td>
          <td>Science's STKE</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="MOTIVATION
Bistability is one of the salient dynamical features in various all-or-none kinds of decision-making processes. The presence of bistability in a cell signalling network plays a key role in input-output (I/O) relation. Our study is aiming to capture and emphasise the role of motif structure influencing the I/O relation between two nodes in the context of bistability. Here, a model-based analysis is made to investigate the critical conditions responsible for the emergence of different bistable protein-protein interaction (PPI) motifs and their possible applications to find the potential drug targets.


RESULTS
The global sensitivity analysis is used to identify sensitive parameters and their role in maintaining the bistability. Additionally, the bistable switching through hysteresis is explored to develop an understanding of the underlying mechanisms involved in the cell signalling processes, when significant motifs exhibiting bistability have emerged. Further, we elaborate the application of the results by the implication of the emerged PPI motifs to identify potential drug-targets in three cancer networks, which is validated with existing databases. The influence of stochastic perturbations that could hinder desired functionality of any signalling networks is also described here.


SUPPLEMENTARY INFORMATION
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f71cec3f7de7db128541bff44d12d03cf02f12c" target='_blank'>Bistability in cell signalling and its significance in identifying potential drug-targets</a></td>
          <td>
            Suvankar Halder, Sumana Ghosh, J. Chattopadhyay, S. Chatterjee
          </td>
          <td>2021-05-22</td>
          <td>Bioinformatics</td>
          <td>8</td>
          <td>44</td>
        </tr>

        <tr id="Mitogen‐activated protein kinase (MAPK) cascades process a myriad of stimuli received by cell‐surface receptors and generate precise spatiotemporal guidance for multiple target proteins, dictating receptor‐specific cellular outcomes. Computational modeling reveals that the intrinsic topology of MAPK cascades enables them to amplify signal sensitivity and amplitude, reduce noise, and display intricate dynamic properties, which include toggle switches, excitation pulses, and oscillations. Specificity of signaling responses can be brought about by signal‐induced feedback and feedforward wiring imposed on the MAPK cascade backbone. Intracellular gradients of protein activities arise from the spatial separation of opposing reactions in kinase‐phosphatase cycles. The membrane confinement of the initiating kinase in MAPK cascades and cytosolic localization of phosphatases can result in precipitous gradients of phosphorylated signal‐transducers if they spread solely by diffusion. Endocytotic trafficking of active kinases driven by molecular motors and traveling waves of protein phosphorylation can propagate phosphorylation signals from the plasma membrane to the nucleus, especially in large cells, such as Xenopus eggs. Copyright © 2009 John Wiley & Sons, Inc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/549f0ea398377e29460e84284cf31387d6568803" target='_blank'>Four‐dimensional dynamics of MAPK information‐processing systems</a></td>
          <td>
            B. Kholodenko, M. Birtwistle
          </td>
          <td>2009-07-01</td>
          <td>Wiley Interdisciplinary Reviews: Systems Biology and Medicine</td>
          <td>87</td>
          <td>57</td>
        </tr>

        <tr id="Cellular decision-making (CDM) is a dynamic phenomenon often controlled by regulatory networks defining interactions between genes and transcription factor proteins. Traditional studies have focussed on molecular switches such as positive feedback circuits that exhibit at most bistability. However, higher-order dynamics such as tristability is also prominent in many biological processes. It is thus imperative to identify a minimal circuit that can alone explain mono, bi, and tristable dynamics. In this work, we consider a two-component positive feedback network with an autoloop and explore these regimes of stability for different degrees of multimerization and the choice of Boolean logic functions. We report that this network can exhibit numerous dynamical scenarios such as bi-and tristability, hysteresis, and biphasic kinetics, explaining the possibilities of abrupt cell state transitions and the smooth state swap without a step-like switch. Specifically, while with monomeric regulation and competitive OR logic, the circuit exhibits mono-and bistability and biphasic dynamics, with non-competitive AND and OR logics only monostability can be achieved. To obtain bistability in the latter cases, we show that the autoloop must have (at least) dimeric regulation. In pursuit of higher-order stability, we show that tristability occurs with higher degrees of multimerization and with non-competitive OR logic only. Our results, backed by rigorous analytical calculations and numerical examples, thus explain the association between multistability, multimerization, and logic in this minimal circuit. Since this circuit underlies various biological processes, including epithelial-mesenchymal transition which often drives carcinoma metastasis, these results can thus offer crucial inputs to control cell state transition by manipulating multimerization and the logic of regulation in cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c15cf389bbc4b2bcfbcbaddd5419adc5eab4dd7" target='_blank'>Logic-dependent emergence of multistability, hysteresis, and biphasic dynamics in a minimal positive feedback network with an autoloop</a></td>
          <td>
            Akriti Srivastava, Mubasher Rashid
          </td>
          <td>2024-04-08</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Many distinct signaling pathways allow the cell to receive, process, and respond to information. Often, components of different pathways interact, resulting in signaling networks. Biochemical signaling networks were constructed with experimentally obtained constants and analyzed by computational methods to understand their role in complex biological processes. These networks exhibit emergent properties such as integration of signals across multiple time scales, generation of distinct outputs depending on input strength and duration, and self-sustaining feedback loops. Feedback can result in bistable behavior with discrete steady-state activities, well-defined input thresholds for transition between states and prolonged signal output, and signal modulation in response to transient stimuli. These properties of signaling networks raise the possibility that information for "learned behavior" of biological systems may be stored within intracellular biochemical reactions that comprise signaling pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9de3b7150cc254356bd579ef99e0714c1f3b39a" target='_blank'>Emergent properties of networks of biological signaling pathways.</a></td>
          <td>
            U. Bhalla, R. Iyengar
          </td>
          <td>1999-01-15</td>
          <td>Science</td>
          <td>1645</td>
          <td>64</td>
        </tr>

        <tr id="The mitogen-activated protein kinase (MAPK) pathway is considered to be a central block in many biological signaling networks. Despite the common core cascade structure, the activation of MAPK in different biological systems can exhibit different types of dynamic behaviors. Computer modeling may help to reveal the mechanisms underlying such variations. However, so far most computational models of the MAPK cascade have been system-specific, or to reflect a particular type among the wide spectrum of possible dynamics. To obtain a general and integrated view of the relationship between the dynamics of MAPK activation and the structures and parameters of the MAPK cascade, we constructed a generic model by comparing previous models covering different specific biological systems. We assumed that reliable qualitative results could be predicted through a qualitative model with pseudo parameters. We used randomly sampled parameters instead of a specific set of “best-fit” parameters to avoid biases towards any particular systems. A range of dynamics behaviors for MAPK activation, including ultrasensitivity, bistability, transient activation and oscillation, were successfully predicted by the generic model. The results indicated that the steady state dynamics (ultrasensitivity and bistability) was jointly determined by the three-tiered structure of the MAPK cascade and the competitive substrate binding in the dual-phosphorylation processes of the central components, while the temporal dynamics (transient activation and oscillation) was mainly affected by the upstream signaling pathway and feedbacks. Moreover, MAPK kinase (MAPKK) played more important roles than the other two components in determining the dynamics of MAPK activation. We hypothesize that this is an important and advantageous property for the regulation and for the functional diversity of MAPK pathways in real cells. Finally, to assist developing generic models for signaling motifs through model comparisons, we proposed a reaction-based database to make the model data more flexible and interoperable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/558a5c17bf4b19875b0696b35de9a583a6c40956" target='_blank'>Random Parameter Sampling of a Generic Three-Tier MAPK Cascade Model Reveals Major Factors Affecting Its Versatile Dynamics</a></td>
          <td>
            Zhong Mai, Haiyan Liu
          </td>
          <td>2013-01-24</td>
          <td>PLoS ONE</td>
          <td>9</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84aa827eb6f344b68c5bff1af5def6574e991a38" target='_blank'>Sampling-based Bayesian approaches reveal the importance of quasi-bistable behavior in cellular decision processes on the example of the MAPK signaling pathway in PC-12 cell lines</a></td>
          <td>
            Antje Jensch, C. Thomaseth, N. Radde
          </td>
          <td>2017-01-25</td>
          <td>BMC Systems Biology</td>
          <td>7</td>
          <td>17</td>
        </tr>

        <tr id="For more than a decade, the MAPK (mitogen-activated protein kinase) cascade has been studied using mathematical modelling and quantitative experimentation [1]. The MAPK cascade relays the presence of extracellular stimuli such as growth hormones to the nucleus and controls the expression of hundreds of genes. MAPKs control major cell fate decisions such as proliferation, differentiation and apoptosis, mainly by inducing alterations in gene expression. In this chapter, we discuss how systems biology analysis provides insights into the functioning of this cascade. We show how this pathway assists the cell in responding properly to extracellular cues by filtering out sub-threshold stimuli, while efficiently transmitting physiologically relevant inputs. Several different receptors signal through the MAPK pathway even though they elicit opposite biological responses, thus raising the question of how specificity is achieved in MAPK signalling. Experimental studies revealed that specific biological responses are encoded by quantitative aspects of the MAPK signal such as amplitude or duration. We discuss mechanisms that enable the pathway to generate quantitatively different signals, and also explain how different signals are interpreted by the downstream gene expression machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6240690e740bb11ac1d42f43f91ffcfad059dbaf" target='_blank'>Systems analysis of MAPK signal transduction.</a></td>
          <td>
            N. Blüthgen, S. Legewie
          </td>
          <td>2008-09-30</td>
          <td>Essays in biochemistry</td>
          <td>68</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da87720f4767a44da3f92e497492607a5e8a082c" target='_blank'>When kinases meet mathematics: the systems biology of MAPK signalling</a></td>
          <td>
            W. Kolch, Muffy Calder, David R. Gilbert
          </td>
          <td>2005-03-21</td>
          <td>FEBS Letters</td>
          <td>190</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ba8b78a596fc0cb2361e538c1c13be40423941c" target='_blank'>Computational analysis reveals the coupling between bistability and the sign of a feedback loop in a TGF-β1 activation model</a></td>
          <td>
            Huipeng Li, L. Venkatraman, B. C. Narmada, Jacob K. White, Hanry Yu, L. Tucker-Kellogg
          </td>
          <td>2017-12-01</td>
          <td>BMC Systems Biology</td>
          <td>10</td>
          <td>53</td>
        </tr>

        <tr id="Different environmental stimuli often use the same set of signaling proteins to achieve very different physiological outcomes. The invasive growth and mating pathways in yeast Saccharomyces cerevisiae each employ a mitogen‐activated protein (MAP) kinase cascade that includes several common components (e.g. Ste11, Ste7), after which the pathways diverge leading to activation of different MAP kinases (Fus3 and Kss1, respectively). In order to determine how pathway specificity is achieved, we conducted a quantitative analysis of MAP kinase activity, and devised a series of six computational models that describe their behavior in vivo. As anticipated by one of the models, MAP kinase phosphorylation of shared pathway components results in diminished activation of one MAP kinase but not the other. These findings reveal how one MAP kinase can limit the activity of a competing MAP kinase and thereby ensure signal fidelity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a0034a8b7b900ff225c7b67b37624a166b457e" target='_blank'>Modeling signal specificity by feedback inhibition</a></td>
          <td>
            Nan Hao, Necmettin Yildirim, S. Parnell, B. Errede, T. Elston, H. Dohlman
          </td>
          <td>2007-04-01</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Biochemical networks, including those containing signaling pathways, display a wide range of regulatory properties. These include the ability to propagate information across different time scales and to function as switches and oscillators. The mechanisms underlying these complex behaviors involve many interacting components and cannot be understood by experiments alone. The development of computational models and the integration of these models with experiments provide valuable insight into these complex systems-level behaviors. Here we review current approaches to the development of computational models of biochemical networks and describe the insights gained from models that integrate experimental data, using three examples that deal with ultrasensitivity, flexible bistability and oscillatory behavior. These types of complex behavior from relatively simple networks highlight the necessity of using theoretical approaches in understanding higher order biological functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7892da387cb63b94afbe9f7c3bc0d0b71be6c4c2" target='_blank'>Modeling Signaling Networks</a></td>
          <td>
            Susana R. Neves, R. Iyengar
          </td>
          <td>2002-12-01</td>
          <td>Science</td>
          <td>131</td>
          <td>64</td>
        </tr>

        <tr id="Strong experimental evidence indicates that protein kinase and phosphatase (KP) cycles are critical to both the induction and maintenance of activity-dependent modifications in neurons. However, their contribution to information storage remains controversial, despite impressive modeling efforts. For instance, plasticity models based on KP cycles do not account for the maintenance of plastic modifications. Moreover, bistable KP cycle models that display memory fail to capture essential features of information storage: rapid onset, bidirectional control, graded amplitude, and finite lifetimes. Here, we show in a biophysical model that upstream activation of KP cycles, a ubiquitous mechanism, is sufficient to provide information storage with realistic induction and maintenance properties: plastic modifications are rapid, bidirectional, and graded, with finite lifetimes that are compatible with animal and human memory. The maintenance of plastic modifications relies on negligible reaction rates in basal conditions and thus depends on enzyme nonlinearity and activation properties of the activity-dependent KP cycle. Moreover, we show that information coding and memory maintenance are robust to stochastic fluctuations inherent to the molecular nature of activity-dependent KP cycle operation. This model provides a new principle for information storage where plasticity and memory emerge from a single dynamic process whose rate is controlled by neuronal activity. This principle strongly departs from the long-standing view that memory reflects stable steady states in biological systems, and offers a new perspective on memory in animals and humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181252a5c5489629938318b81d3ca3a4969e99e7" target='_blank'>A New Principle for Information Storage in an Enzymatic Pathway Model</a></td>
          <td>
            B. Delord, Hugues Berry, E. Guigon, S. Genet
          </td>
          <td>2007-05-18</td>
          <td>PLoS Computational Biology</td>
          <td>24</td>
          <td>29</td>
        </tr>

        <tr id="Significance Many long-term cellular decisions in development, synaptic plasticity, and immunity require cells to recognize input dynamics such as pulse duration or frequency. In dynamically controlled cells, incoming stimuli are often processed and filtered by a rapid-acting signaling layer, and then passed to a downstream slow-acting layer that locks in a longer-term cellular response. Directly testing how such dual-timescale networks control dynamical regulation has been challenging because most tools in synthetic biology allow rewiring of slow gene expression circuits, but not of rapid signaling circuits. In this work, we developed modular peptide tags for engineering synthetic phosphorylation circuits. We used these phospho-regulons to build synthetic dual-timescale networks in which the dynamic responsiveness of a cell fate decision can be selectively tuned. Many cells can sense and respond to time-varying stimuli, selectively triggering changes in cell fate only in response to inputs of a particular duration or frequency. A common motif in dynamically controlled cells is a dual-timescale regulatory network: although long-term fate decisions are ultimately controlled by a slow-timescale switch (e.g., gene expression), input signals are first processed by a fast-timescale signaling layer, which is hypothesized to filter what dynamic information is efficiently relayed downstream. Directly testing the design principles of how dual-timescale circuits control dynamic sensing, however, has been challenging, because most synthetic biology methods have focused solely on rewiring transcriptional circuits, which operate at a single slow timescale. Here, we report the development of a modular approach for flexibly engineering phosphorylation circuits using designed phospho-regulon motifs. By then linking rapid phospho-feedback with slower downstream transcription-based bistable switches, we can construct synthetic dual-timescale circuits in yeast in which the triggering dynamics and the end-state properties of the ON state can be selectively tuned. These phospho-regulon tools thus open up the possibility to engineer cells with customized dynamical control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/756717a1529fcc83d1a58913348081d8fcfb5cb3" target='_blank'>Engineering dynamical control of cell fate switching using synthetic phospho-regulons</a></td>
          <td>
            Russell M. Gordley, R. E. Williams, C. Bashor, Jared E. Toettcher, Shude Yan, W. Lim
          </td>
          <td>2016-11-07</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>50</td>
          <td>90</td>
        </tr>

        <tr id="Ultrasensitive response motifs, capable of converting graded stimuli into binary responses, are well-conserved in signal transduction networks. Although it has been shown that a cascade arrangement of multiple ultrasensitive modules can enhance the system’s ultrasensitivity, how a given combination of layers affects a cascade’s ultrasensitivity remains an open question for the general case. Here, we introduce a methodology that allows us to determine the presence of sequestration effects and to quantify the relative contribution of each module to the overall cascade’s ultrasensitivity. The proposed analysis framework provides a natural link between global and local ultrasensitivity descriptors and it is particularly well-suited to characterize and understand mathematical models used to study real biological systems. As a case study, we have considered three mathematical models introduced by O’Shaughnessy et al. to study a tunable synthetic MAPK cascade, and we show how our methodology can help modelers better understand alternative models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7403ccb5aec77299b40b0d3202d94fc5b21a3656" target='_blank'>Ultrasensitivity in signaling cascades revisited: Linking local and global ultrasensitivity estimations</a></td>
          <td>
            E. Altszyler, A. Ventura, A. Colman-Lerner, A. Chernomoretz
          </td>
          <td>2016-08-29</td>
          <td>PLoS ONE</td>
          <td>20</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49bf952fa513231e8a5ebcc3b53e43b7bd6a7381" target='_blank'>Antagonism and bistability in protein interaction networks.</a></td>
          <td>
            Mohsen Sabouri-Ghomi, A. Ciliberto, Sandip Kar, B. Novák, J. Tyson
          </td>
          <td>2008-01-07</td>
          <td>Journal of theoretical biology</td>
          <td>84</td>
          <td>67</td>
        </tr>

        <tr id="Cellular signaling pathways are responsible for decision making that sustains life. Most signaling pathways include post-translational modification cycles, that process multiple inputs and are tightly interconnected. Here we consider a model for phosphorylation/dephosphorylation cycles, and we show that under some assumptions they can operate as molecular neurons or perceptrons, that generate sigmoidal-like activation functions by processing sums of inputs with positive and negative weights. We carry out a steady-state and structural stability analysis for single molecular perceptrons as well as for feedforward interconnections, concluding that interconnected phosphorylation/dephosphorylation cycles may work as multi-layer biomolecular neural networks (BNNs) with the capacity to perform a variety of computations. As an application, we design signaling networks that behave as linear and non-linear classifiers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5112a01219843c2f6335c66b9a2bda740eb41fe2" target='_blank'>Signaling-based neural networks for cellular computation</a></td>
          <td>
            Christian Cuba Samaniego, Andrew Moorman, Giulia Giordano, Elisa Franco
          </td>
          <td>2020-11-10</td>
          <td>bioRxiv</td>
          <td>11</td>
          <td>27</td>
        </tr>

        <tr id="Biochemical networks related to similar functional pathways are often correlated due to cross-talk among the homologous proteins in the different networks. Using a stochastic framework, we address the functional significance of the cross-talk between two pathways. Our theoretical analysis on generic MAPK pathways reveals cross-talk is responsible for developing coordinated fluctuations between the pathways. The extent of correlation evaluated in terms of the information theoretic measure provides directionality to net information propagation. Stochastic time series and scattered plot suggest that the cross-talk generates synchronization within a cell as well as in a cellular population. Depending on the number of input and output, we identify signal integration and signal bifurcation motif that arise due to inter-pathway connectivity in the composite network. Analysis using partial information decomposition quantifies the net synergy in the information propagation through these branched pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c0ebb8b35dda6d869dfe84cf69e0ed74fdc8b95" target='_blank'>Information Theoretical Study of Cross-Talk Mediated Signal Transduction in MAPK Pathways</a></td>
          <td>
            A. Maity, P. Chaudhury, S. Banik
          </td>
          <td>2015-10-16</td>
          <td>Entropy</td>
          <td>7</td>
          <td>20</td>
        </tr>

        <tr id="Operating regimes characterizing the input-output behaviour of an activated phosphorylation-dephosphorylation reaction cycle (PdPC) such as a single MAPK cascade are dictated by the saturated/unsaturated state of the two underlying enzymatic reactions. Four combinations of the states of these two enzymatic reactions led to identification of distinct operating regimes, viz., Hyperbolic (H), Signal transducing (ST), Threshold-hyperbolic (TH) and Ultrasensitive (U). A single PdPC without an explicit feedback have been classically viewed as a module offering signal flow from upstream to downstream, that is, one-way communication. Recently it has been shown that load due to sequestration of the phosphorylated or unphosphorylated form of the substrate by corresponding downstream targets permits retroactive signalling that offers two-way communication. We systematically characterize the operating regimes of a single PdPC subject to retroactivity in either of the substrate forms. We identify five possible regime transitions that could be achieved by increasing the retroactivity strength on either of the two substrate forms. Remarkably, a retroactivity strength of 0.30 in the unphosphorylated form of the substrate is sufficient to induce a transition from ST to H regime indicating that the input-output behaviour of a PdPC is highly sensitive to the presence of a downstream load. Using sensitivity and rate-balance analysis, we show that modulation of the saturation levels of the two enzymatic reactions by increasing retroactivity is the fundamental mechanism governing operating regime transition. Highlights Characterization of operating regimes in the presence of retroactivity. Retroactivity can induce a transition between different operating regimes. Saturation levels of the two enzymatic reactions govern the regime transition. Sensitivity of the protein levels to retroactivity is dictated by saturation levels.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d027423ced733f8825b5661c384cf2168e2289" target='_blank'>Retroactivity induced operating regime transition in a phosphorylation-dephosphorylation cycle</a></td>
          <td>
            Akshay N. Parundekar, G. Viswanathan
          </td>
          <td>2020-10-02</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Cell growth critically depends on signalling pathways whose regulation is the focus of intense research. Without utilizing a priori knowledge of the relative importance of pathway components, we have applied in silico computational methods to the EGF-induced MAPK cascade. Specifically, we systematically perturbed the entire parameter space, including initial conditions, using a Monte Carlo approach, and investigate which protein components or kinetic reaction steps contribute to the differentiation of ERK responses. The model, based on previous work by Brightman and Fell (2000), is composed of 28 reactions, 27 protein molecules, and 48 parameters from both mass action and Michaelis-Menten kinetics. Our multi-parametric systems analysis confirms that Raf inactivation is one of the key steps regulating ERK responses to be either transient or sustained. Furthermore, the results of amplitude-differential ERK phosphorylations within the transient case are mainly attributed to the balance between activation and inactivation of Ras while duration-differential ERK responses for the sustained case are, in addition to Ras, markedly affected by dephospho-/phosphorylation of both MEK and ERK. Our sub-module perturbations showed that MEK and ERK's contribution to this differential ERK activation originates from fluctuations in intermediate pathway module components such as Ras and Raf, implicating a cooperative regulatory mode among the key components. The initial protein concentrations of corresponding reactions such as Ras, GAP, and Raf also influence the distinct signalling outputs of ERK activation. We then compare these results with those obtained from a single-parametric perturbation approach using an overall state sensitivity (OSS) analysis. The OSS findings indicate a more pronounced role of ERK's inhibitory feedback effect on catalysing the dissociation of the SOS complex. Both approaches reveal the presence of multiple specific reactions involved in the distinct dynamics of ERK responses and the cell fate decisions they trigger. This work adds a mechanistic insight of the contribution of key pathway components, thus may support the identification of biomarkers for pharmaceutical drug discovery processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0951d8155ada6c35ec896fb6208894bfd088af2" target='_blank'>Investigating Differential Dynamics of the MAPK Signaling Cascade Using a Multi-Parametric Global Sensitivity Analysis</a></td>
          <td>
            Jeongah Yoon, T. Deisboeck
          </td>
          <td>2009-02-23</td>
          <td>PLoS ONE</td>
          <td>40</td>
          <td>40</td>
        </tr>

        <tr id="Quantitative and qualitative nature of chemical noise propagation in a network of chemical reactions depend crucially on the topology of reaction networks. Multisite reversible phosphorylation-dephosphorylation of target proteins is one such recurrently found topology in various cellular networks regulating key functions in living cells. Here we analytically calculated the stochasticity in multistep reversible chemical reactions by determining variance of phosphorylated species at the steady state using linear noise approximation. We investigated the dependence of variance on the rate parameters in the reaction chain and the number of phosphorylation sites on the species. Assuming a quasi steady state approximation on the multistep reactions, originating from the disparity in time scales in the network, we propose a simulation scheme for coupled chemical reactions to improve the computational efficiency of stochastic simulation of the network. We performed case studies on signal transduction cascade and positive feedback loop with bistability to show the accuracy and efficiency of the method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6cbcdfcd929885a1b2acb954921a758aeb40efb" target='_blank'>Stochasticity in multi-phosphorylation and quasi steady state approximation in stochastic simulation</a></td>
          <td>
            Debashis Barik, Soutrick Das
          </td>
          <td>2018-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19ea531f407b034c82262a3c743a3f5575d55e9b" target='_blank'>Quantitative analysis of signaling networks.</a></td>
          <td>
            Herbert M. Sauro, B. Kholodenko
          </td>
          <td>2004-09-01</td>
          <td>Progress in biophysics and molecular biology</td>
          <td>213</td>
          <td>57</td>
        </tr>

        <tr id="MOTIVATION
Multi-steady state behaviour, and in particular multi-stability, provides biological systems with the capacity to take reliable decisions (such as cell fate determination). A problem arising frequently in systems biology is to elucidate whether a signal transduction mechanism or a gene regulatory network has the capacity for multi-steady state behaviour, and consequently for a switch-like response to stimuli. Bifurcation diagrams are a powerful instrument in nonlinear analysis to study the qualitative and quantitative behaviour of equilibria including bifurcation into different equilibrium branches and bistability. However, in the context of signaling pathways, the inherent large parametric uncertainty hampers the (direct) use of standard bifurcation tools.


RESULTS
We present BioSwitch, a toolbox to detect multi-steady state behaviour in signaling pathways and gene regulatory networks. The tool combines results from chemical reaction network theory with global optimization to efficiently detect whether a signaling pathway has the capacity to undergo a saddle node bifurcation, and in case of multi-stationarity, provides the exact coordinates of the bifurcation where to start a numerical continuation analysis with standard bifurcation tools, leading to two different branches of equilibria. Bistability detection in the G1/S transition pathway of S. cerevisiae is included as an illustrative example.


AVAILABILITY
BioSwitch runs under the popular MATLAB computational environment, and is available at https://sites.google.com/view/bioswitch.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e98eacddb75d4da2eb2d6559760b83f90b140ab" target='_blank'>BioSwitch: a tool for the detection of bistability and multi-steady state behaviour in signalling and gene regulatory networks</a></td>
          <td>
            Pencho Yordanov, J. Stelling, Irene Otero-Muras
          </td>
          <td>2019-10-14</td>
          <td>Bioinformatics</td>
          <td>12</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be0b6a461bc59196ef28a707135325fb1becbded" target='_blank'>A method for inverse bifurcation of biochemical switches: inferring parameters from dose response curves</a></td>
          <td>
            Irene Otero-Muras, Pencho Yordanov, J. Stelling
          </td>
          <td>2014-11-20</td>
          <td>BMC Systems Biology</td>
          <td>10</td>
          <td>40</td>
        </tr>

        <tr id="Signaling networks that convert graded stimuli into binary, all-or-none cellular responses are critical in processes ranging from cell-cycle control to lineage commitment. To exhaustively enumerate topologies that exhibit this switch-like behavior, we simulated all possible two- and three-component networks on random parameter sets, and assessed the resulting response profiles for both steepness (ultrasensitivity) and extent of memory (bistability). Simulations were used to study purely enzymatic networks, purely transcriptional networks, and hybrid enzymatic/transcriptional networks, and the topologies in each class were rank ordered by parametric robustness (i.e., the percentage of applied parameter sets exhibiting ultrasensitivity or bistability). Results reveal that the distribution of network robustness is highly skewed, with the most robust topologies clustering into a small number of motifs. Hybrid networks are the most robust in generating ultrasensitivity (up to 28%) and bistability (up to 18%); strikingly, a purely transcriptional framework is the most fragile in generating either ultrasensitive (up to 3%) or bistable (up to 1%) responses. The disparity in robustness among the network classes is due in part to zero-order ultrasensitivity, an enzyme-specific phenomenon, which repeatedly emerges as a particularly robust mechanism for generating nonlinearity and can act as a building block for switch-like responses. We also highlight experimentally studied examples of topologies enabling switching behavior, in both native and synthetic systems, that rank highly in our simulations. This unbiased approach for identifying topologies capable of a given response may be useful in discovering new natural motifs and in designing robust synthetic gene networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e923aee2814734f7789ecb3bd03dbd1e509cee2" target='_blank'>Robust Network Topologies for Generating Switch-Like Cellular Responses</a></td>
          <td>
            Najaf A. Shah, Casim A. Sarkar
          </td>
          <td>2011-06-01</td>
          <td>PLoS Computational Biology</td>
          <td>101</td>
          <td>20</td>
        </tr>

        <tr id="Cell responses are actuated by tightly controlled signal transduction pathways. Although the concept of an integrated signaling network replete with interpathway cross-talk and feedback regulation is broadly appreciated, kinetic data of the type needed to characterize such interactions in conjunction with mathematical models are lacking. In mammalian cells, the Ras/ERK pathway controls cell proliferation and other responses stimulated by growth factors, and several cross-talk and feedback mechanisms affecting its activation have been identified. In this work, we take a systematic approach to parse the magnitudes of multiple regulatory mechanisms that attenuate ERK activation through canonical (Ras-dependent) and non-canonical (PI3K-dependent) pathways. In addition to regulation of receptor and ligand levels, we consider three layers of ERK-dependent feedback: desensitization of Ras activation, negative regulation of MEK kinase (e.g. Raf) activities, and up-regulation of dual-specificity ERK phosphatases. Our results establish the second of these as the dominant mode of ERK self-regulation in mouse fibroblasts. We further demonstrate that kinetic models of signaling networks, trained on a sufficient diversity of quantitative data, can be reasonably comprehensive, accurate, and predictive in the dynamical sense.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405be6d692d4836d3b3efde9333fdf1789ff05c7" target='_blank'>Systematic Quantification of Negative Feedback Mechanisms in the Extracellular Signal-regulated Kinase (ERK) Signaling Network*</a></td>
          <td>
            M. Cirit, Chun-Chao Wang, J. Haugh
          </td>
          <td>2010-09-16</td>
          <td>The Journal of Biological Chemistry</td>
          <td>91</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b06d234407efd97d798904ba97ebb9a73b5fb31e" target='_blank'>Methodologies for the modeling and simulation of biochemical networks, illustrated for signal transduction pathways: A primer</a></td>
          <td>
            Nesma ElKalaawy, A. Wassal
          </td>
          <td>2015-03-01</td>
          <td>Bio Systems</td>
          <td>20</td>
          <td>8</td>
        </tr>

        <tr id="Mathematical models of biological switches have been proposed as a means to study the mechanism of decision making in biological systems. These conceptual models are abstract representations of the key components involved in the crucial cell fate decision underlying the biological system. In this paper, the methods of phase plane analysis and bifurcation analysis are explored and demonstrated using an example from the literature, namely the synthetic genetic circuit proposed by Gardner et al. (2000) which involved two negative loops (from two mutually inhibiting genes). Figure 1 shows a schematic diagram of the synthetic genetic circuit constructed by Gardner et al. (2000). Particularly, a saddle-node bifurcation is used as a signal response curve to capture the bistability of the system. The notion of bistability is obscure to most novice researchers or biologists because it is difficult to understand the existence of two stable steady states and how to flip from one stable steady state to another and vice versa. Thus, the main purpose of this paper is to unlock the computational techniques (bifurcation analysis implemented in a software tool called XPPAUT) in mathematical modelling of bistability through a simple example from Gardner et al. (2000). In addition, time course simulations are provided to illustrate: 1) the notion of bistability where the existence of two stable steady states and we demonstrated that for two different initial conditions one of the genes is ‘ON’ and the other gene is ‘OFF’; 2) hysteresis behaviour where the saddle-node bifurcation points as two critical points in which to turn ‘ON’ one gene happens at a larger parameter value than to turn ‘OFF’ this gene (at a lower parameter value). The hysteresis behaviour is important for irreversible decision made by cell to commit to turn ‘ON’. In conclusion, the understanding of the computational techniques in modelling biological switch is important for elucidating genetic switch that has potential for gene therapy and can provide explanation for experimental findings of bistable systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21518a4153710d2f5502382f4ecd8f6dcf91d0a0" target='_blank'>Computational techniques in mathematical modelling of biological switches</a></td>
          <td>
            K. H. Chong, S. Samarasinghe, D. Kulasiri, Jie Zheng
          </td>
          <td>2015-12-01</td>
          <td></td>
          <td>3</td>
          <td>24</td>
        </tr>

        <tr id="MOTIVATION
Multi-steady state behaviour, and in particular multi-stability, provides biological systems with the capacity to take reliable decisions (such as cell fate determination). A problem arising frequently in systems biology is to elucidate whether a signal transduction mechanism or a gene regulatory network has the capacity for multi-steady state behaviour, and consequently for a switch-like response to stimuli. Bifurcation diagrams are a powerful instrument in nonlinear analysis to study the qualitative and quantitative behaviour of equilibria including bifurcation into different equilibrium branches and bistability. However, in the context of signaling pathways, the inherent large parametric uncertainty hampers the (direct) use of standard bifurcation tools.


RESULTS
We present BioSwitch, a toolbox to detect multi-steady state behaviour in signaling pathways and gene regulatory networks. The tool combines results from chemical reaction network theory with global optimization to efficiently detect whether a signaling pathway has the capacity to undergo a saddle node bifurcation, and in case of multi-stationarity, provides the exact coordinates of the bifurcation where to start a numerical continuation analysis with standard bifurcation tools, leading to two different branches of equilibria. Bistability detection in the G1/S transition pathway of S. cerevisiae is included as an illustrative example.


AVAILABILITY
BioSwitch runs under the popular MATLAB computational environment, and is available at https://sites.google.com/view/bioswitch.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e98eacddb75d4da2eb2d6559760b83f90b140ab" target='_blank'>BioSwitch: a tool for the detection of bistability and multi-steady state behaviour in signalling and gene regulatory networks</a></td>
          <td>
            Pencho Yordanov, J. Stelling, Irene Otero-Muras
          </td>
          <td>2019-10-14</td>
          <td>Bioinformatics</td>
          <td>12</td>
          <td>19</td>
        </tr>

        <tr id="Cellular proccess that are crucial for health or disease are orchestrated by the expression of multiple genes in a network of differential complex signaling pathways, consisting of many physical and functional interactions.Unlike the stable genome structure, the highly dynamic interactome represents a major challenge for quantitative mapping of protein–protein interactions (PPIs). In this article, I will discuss the latest systems science advances by using breakthrough technologies to gain an understanding of the whole picture of the spatiotemporal signaling network as a response to dynamic individual genome–lifestyle interactions and translating it to the clinic.Biological signals control cells’ functions and their organization into tissues, organs and a whole organism’s homeostasis. A variety of extracellular and intracellu-lar signaling molecules include hormones, growth factors, cytokines, chemokines, neurotransmitters, extracellular matrix components, receptors, protein kinases, protein phosphatases and DNA-binding proteins in the nucleus. A signal is trans-mitted or amplified from the cell mem-brane to the nucleus through a dynamic complex interactions network">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2702c101690829e49078cf33e2ee81dc4f49e616" target='_blank'>Differential signaling transduction networks for clinical robustness</a></td>
          <td>
            D. Roukos
          </td>
          <td>2012-04-01</td>
          <td>Expert Review of Proteomics</td>
          <td>12</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d34891e9dacbcc2473f5b3bf678e8f1b688f4ef" target='_blank'>Theoretical Modeling of Bi-Connected Elementary Biochemical Reaction Cycles</a></td>
          <td>
            Tatsunori Nishimura, M. Tateno
          </td>
          <td>2011-02-01</td>
          <td>MAG</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Signaling pathways come together to form networks that connect receptors to many different cellular machines. Such networks not only receive and transmit signals but also process information. The complexity of these networks requires the use of computational models to understand how information is processed and how input-output relationships are determined. Two major computational approaches used to study signaling networks are graph theory and dynamical modeling. Both approaches are useful; network analysis (application of graph theory) helps us understand how the signaling network is organized and what its information-processing capabilities are, whereas dynamical modeling helps us determine how the system changes in time and space upon receiving stimuli. Computational models have helped us identify a number of emergent properties that signaling networks possess. Such properties include ultrasensitivity, bistability, robustness, and noise-filtering capabilities. These properties endow cell-signaling networks with the ability to ignore small or transient signals and/or amplify signals to drive cellular machines that spawn numerous physiological functions associated with different cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/382a84e1cb7587f5957c26d66bbc05aaa65d01e9" target='_blank'>Signaling networks: information flow, computation, and decision making.</a></td>
          <td>
            E. Azeloglu, R. Iyengar
          </td>
          <td>2015-04-01</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>94</td>
          <td>64</td>
        </tr>

        <tr id="All cells receive and respond to signals from their environment, whether they live freely or are part of a tissue. Feedback loops are one among the interactions in signal-transduction cascades, in which the information from the downstream components is sent back to the upstream components of the system. Crosstalks between signal-transduction cascades are able to combine the information's received and processed to trigger the appropriate response. To understand how signaling networks make decisions, the dynamic interactions of proteins in the signal-transduction cascades are important rather than just creating static maps of pathways. Based on reaction kinetics, we have developed mathematical models to predict and analyze the impacts of the feedback loops in JNK cascades and the crosstalks between JNK and P38 kinase cascades. Feedback loops in JNK cascades and crosstalks between JNK and P38 kinase cascades plays the complementary roles in making the impacts on the activation of JNK and P38 kinases. JNK and P38 kinase cascades with feedbacks and crosstalks is a robust system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2880a4c0be242db0837a9696c152f250827dd584" target='_blank'>Dynamic Modeling and Simulation of JNK and P38 Kinase Cascades With Feedbacks and Crosstalks</a></td>
          <td>
            Pandurangan Sundaramurthy, S. Gakkhar
          </td>
          <td>2010-08-19</td>
          <td>IEEE Transactions on NanoBioscience</td>
          <td>7</td>
          <td>24</td>
        </tr>

        <tr id="MOTIVATION
Signaling pathways capable of switching between two states are ubiquitous within living organisms. They provide the cells with the means to produce reversible or irreversible decisions. Switch-like behavior of biological systems is realized through biochemical reaction networks capable of having two or more distinct steady states which are dependent on initial conditions. Investigation of whether a certain signaling pathway can confer bistability involves a substantial amount of hypothesis testing. The cost of direct experimental testing can be prohibitive. Therefore, constraining the hypothesis space is highly beneficial. One such methodology is based on Chemical Reaction Network Theory which uses computational techniques to rule out pathways that are not capable of bistability regardless of kinetic constant values and molecule concentrations. Although useful, these methods are complicated from both pure and computational mathematics perspectives. Thus, their adoption is very limited amongst biologists.


RESULTS
We brought Chemical Reaction Network Theory approaches closer to experimental biologists by automating all the necessary steps in CRNT4SMBL. The input is based on SBML format which is the community standard for biological pathway communication. The tool parses SBML and derives C-graph representations of the biological pathway with mass action kinetics. Next steps involve an efficient search for potential saddle-node bifurcation points using an optimization technique. This type of bifurcation is important as it has the potential of acting as a switching point between two steady states. Finally, if any bifurcation points are present, continuation analysis with respect to a user defined parameter extends the steady state branches and generates a bifurcation diagram. Presence of an S-shaped bifurcation diagram indicates that the pathway acts as a bistable switch for the given optimization parameters.


AVAILABILITY
CRNT4SBML is available via the Python Package Index. The documentation can be found at https://crnt4sbml.readthedocs.io. CRNT4SBML is licensed under the Apache Software License 2.0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d70a8bbf91084839d08ab9e0cac4b787dd3e8740" target='_blank'>CRNT4SBML: a Python package for the detection of bistability in biochemical reaction networks</a></td>
          <td>
            Brandon C Reyes, Irene Otero-Muras, Michael T Shuen, A. Tartakovsky, V. Petyuk
          </td>
          <td>2020-04-14</td>
          <td>Bioinformatics</td>
          <td>5</td>
          <td>44</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>