<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../../custom.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              93dd39c86854a4c9008387177ccfd2a474105b80
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../IBD/" class="md-tabs__link">
        
  
    
  
  IBD

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Covid/" class="md-tabs__link">
        
  
    
  
  Covid

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Cancer/" class="md-tabs__link">
        
  
    
  
  Cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK1/" class="md-tabs__link">
        
  
    
  
  MAPK1

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../MAPK/" class="md-tabs__link">
        
  
    
  
  MAPK

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Virtual%20cell/" class="md-tabs__link">
        
  
    
  
  Virtual cell

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../RA/" class="md-tabs__link">
        
  
    
  
  RA

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Prostate%20cancer/" class="md-tabs__link">
        
  
    
  
  Prostate cancer

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../Virtual%20patients/" class="md-tabs__link">
        
  
    
  
  Virtual patients

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurveyModels" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurveyModels
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../IBD/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    IBD
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Covid/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Covid
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK1
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../MAPK/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    MAPK
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Virtual%20cell/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual cell
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../RA/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    RA
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Prostate%20cancer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Prostate cancer
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Virtual%20patients/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Virtual patients
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<!DOCTYPE html>
<h1 id="_1"></h1>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-29 08:54:50 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <p>
  <h3 data-intro='Recommendations for the article'>
    Recommendations for the article <i>Approaches to generating virtual patient cohorts with applications in oncology</i>
  </h3>
  <table id="table1" class="display wrap" style="width:100%">
  <thead>
    <tr>
        <th data-intro='Click to view the abstract (if available)'>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/ Conference</th>
        <th>Citation count</th>
        <th data-intro='Highest h-index among the authors'>Highest h-index</th>
    </tr>
  </thead>
  <tbody>

        <tr id="Mathematical modeling has made significant contributions to drug design, development, and optimization. Virtual clinical trials that integrate mathematical models to explore patient heterogeneity and its impact on a variety of therapeutic questions have recently risen in popularity. Here, we outline best practices for creating virtual patients from mathematical models to ultimately implement and execute a virtual clinical trial. In this practical guide, we discuss and provide examples of model design, parameter estimation, parameter sensitivity, model identifiability, and virtual patient cohort creation. Our goal is to help researchers adopt these approaches to further the use of virtual population-based analysis and virtual clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f59d303a0d7b1ae764d5c0f77fb1afbc5f25da0" target='_blank'>A practical guide for the generation of model-based virtual clinical trials</a></td>
          <td>
            M. Craig, J. Gevertz, I. Kareva, K. Wilkie
          </td>
          <td>2023-06-16</td>
          <td>Frontiers in Systems Biology</td>
          <td>13</td>
          <td>18</td>
        </tr>

        <tr id="Quantitative systems pharmacology models mechanistically describe a biological system and the effect of drug treatment on system behavior. Because these models rarely are identifiable from the available data, the uncertainty in physiological parameters may be sampled to create alternative parameterizations of the model, sometimes termed ‘Virtual Patients.’ In order to reproduce the statistics of a clinical population, Virtual Patients are often weighted to form a Virtual Population that reflects the baseline characteristics of the clinical cohort. Here we introduce a novel technique to efficiently generate Virtual Patients and, from this ensemble, demonstrate how to select a Virtual Population that matches the observed data without the need for weighting. This approach improves confidence in model predictions by mitigating the risk that spurious Virtual Patients become over-represented in Virtual Populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24682c3ba6c0856f7bd3cef71055d02b8b81ec64" target='_blank'>Efficient Generation and Selection of Virtual Populations in Quantitative Systems Pharmacology Models</a></td>
          <td>
            RJ Allen, TR Rieger, CJ Musante
          </td>
          <td>2015-10-06</td>
          <td>CPT: Pharmacometrics & Systems Pharmacology</td>
          <td>144</td>
          <td>3</td>
        </tr>

        <tr id="Virtual clinical trials (VCTs) are growing in popularity as a tool for quantitatively predicting heterogeneous treatment responses across a population. In the context of a VCT, a plausible patient is an instance of a mathematical model with parameter (or attribute) values chosen to reflect features of the disease and response to treatment for that particular patient. A number of techniques have been introduced to determine the set of model parametrizations to include in a virtual patient cohort. These methodologies generally start with a prior distribution for each model parameter and utilize some criteria to determine whether a parameter set sampled from the priors should be included or excluded from the plausible population. No standard technique exists, however, for generating these prior distributions and choosing the inclusion/exclusion criteria. In this work, we rigorously quantify the impact that VCT design choices have on VCT predictions. Rather than use real data and a complex mathematical model, a spatial model of radiotherapy is used to generate simulated patient data and the mathematical model used to describe the patient data is a two-parameter ordinary differential equations model. This controlled setup allows us to isolate the impact of both the prior distribution and the inclusion/exclusion criteria on both the heterogeneity of plausible populations and on predicted treatment response. We find that the prior distribution, rather than the inclusion/exclusion criteria, has a larger impact on the heterogeneity of the plausible population. Yet, the percent of treatment responders in the plausible population was more sensitive to the inclusion/exclusion criteria utilized. This foundational understanding of the role of virtual clinical trial design should help inform the development of future VCTs that use more complex models and real data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/231e7bc7a40e4003493fc06a15b04cc9b5fdabe2" target='_blank'>Assessing the Role of Patient Generation Techniques in Virtual Clinical Trial Outcomes</a></td>
          <td>
            J. Gevertz, Joanna R Wares
          </td>
          <td>2024-06-12</td>
          <td>Bulletin of mathematical biology</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5476f60a1b993c2a8839dea732da58040d16c13" target='_blank'>From virtual patients to digital twins in immuno-oncology: lessons learned from mechanistic quantitative systems pharmacology modeling</a></td>
          <td>
            Hanwen Wang, Theinmozhi Arulraj, Alberto Ippolito, Aleksander S. Popel
          </td>
          <td>2024-07-16</td>
          <td>NPJ Digital Medicine</td>
          <td>3</td>
          <td>12</td>
        </tr>

        <tr id="Virtual patients and digital patients/twins are two similar concepts gaining increasing attention in health care with goals to accelerate drug development and improve patients' survival, but with their own limitations. Although methods have been proposed to generate virtual patient populations using mechanistic models, there are limited number of applications in immuno-oncology research. Furthermore, due to the stricter requirements of digital twins, they are often generated in a study-specific manner with models customized to particular clinical settings (e.g., treatment, cancer, and data types). Here, we discuss the challenges for virtual patient generation in immuno-oncology with our most recent experiences, initiatives to develop digital twins, and how research on these two concepts can inform each other.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba79f86552cad2bb9dae7c8558a9879f9dd6159f" target='_blank'>From virtual patients to digital twins in immuno-oncology: lessons learned from mechanistic quantitative systems pharmacology modeling.</a></td>
          <td>
            Hanwen Wang, Theinmozhi Arulraj, Alberto Ippolito, Aleksander S. Popel
          </td>
          <td>2024-03-05</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>12</td>
        </tr>

        <tr id="Virtual patients and digital patients/twins are two similar concepts gaining increasing attention in health care with goals to accelerate drug development and improve patients' survival, but with their own limitations. Although methods have been proposed to generate virtual patient populations using mechanistic models, there are limited number of applications in immuno-oncology research. Furthermore, due to the stricter requirements of digital twins, they are often generated in a study-specific manner with models customized to particular clinical settings (e.g., treatment, cancer, and data types). Here, we discuss the challenges for virtual patient generation in immuno-oncology with our most recent experiences, initiatives to develop digital twins, and how research on these two concepts can inform each other.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba79f86552cad2bb9dae7c8558a9879f9dd6159f" target='_blank'>From virtual patients to digital twins in immuno-oncology: lessons learned from mechanistic quantitative systems pharmacology modeling.</a></td>
          <td>
            Hanwen Wang, Theinmozhi Arulraj, Alberto Ippolito, Aleksander S. Popel
          </td>
          <td>2024-03-05</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06463c8f59535518d884913bc086c23e90749299" target='_blank'>Modeling clinical phenotype variability: consideration of genomic variations, computational methods, and quantitative proteomics.</a></td>
          <td>
            J. Bai, Li-Rong Yu
          </td>
          <td>2022-10-01</td>
          <td>Journal of pharmaceutical sciences</td>
          <td>4</td>
          <td>15</td>
        </tr>

        <tr id="The generation of “virtual twins” of patients (Avatars) through integration of multiscale data gained from both the clinical profile of the patient and - omics tools, could create an appropriate environment for stratification of patients in fitting cohorts of “virtual populations”. Physiologically based pharmacokinetic & pharmacodynamic (PB/PK/PD) models as in silico clinical trial tools can estimate the PK/PD profiles in specific populations. In this work we discuss examples of how patient Avatars could be exploited in the context of in silico clinical trials and help in identifying novel biomarkers for personalized diagnosis. The PB/PK/PD models, neuroimaging and - omics data, may be fused together to further advance current decision making processes in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc03ffd4804bb091b47b3a4d07f1e8cde500d71b" target='_blank'>Exploitation of patient avatars towards stratified medicine through the development of in silico clinical trials approaches</a></td>
          <td>
            M. Spanakis, E. Papadaki, D. Kafetzopoulos, A. Karantanas, T. Maris, V. Sakkalis, K. Marias
          </td>
          <td>2013-11-01</td>
          <td>13th IEEE International Conference on BioInformatics and BioEngineering</td>
          <td>3</td>
          <td>43</td>
        </tr>

        <tr id="Virtual patients (VPs) are widely used within quantitative systems pharmacology (QSP) modeling to explore the impact of variability and uncertainty on clinical responses. In one method of generating VPs, parameters are sampled randomly from a distribution, and possible VPs are accepted or rejected based on constraints on model output behavior. This approach works but can be inefficient (i.e., the vast majority of model runs typically do not result in valid VPs). Machine learning surrogate models offer an opportunity to improve the efficiency of VP creation significantly. In this approach, surrogate models are trained using the full QSP model and subsequently used to rapidly pre‐screen for parameter combinations that result in feasible VPs. The overwhelming majority of parameter combinations pre‐vetted using the surrogate models result in valid VPs when tested in the original QSP model. This tutorial presents this novel workflow and demonstrates how a surrogate model software application can be used to select and optimize the surrogate models in a case study. We then discuss the relative efficiency of the methods and scalability of the proposed method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8262122aa5238a60a9072f5da9929a3b8df54aae" target='_blank'>Using machine learning surrogate modeling for faster QSP VP cohort generation</a></td>
          <td>
            Renée C. Myers, F. Augustin, Jérémy Huard, Christina M. Friedrich
          </td>
          <td>2023-06-16</td>
          <td>CPT: Pharmacometrics & Systems Pharmacology</td>
          <td>4</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f815189fdcfa1e70a0c0cf5d1d37532fbabfcd" target='_blank'>Emulation of Quantitative Systems Pharmacology models to accelerate virtual population inference in immuno-oncology.</a></td>
          <td>
            Tomasz Pawłowski, Grzegorz Bokota, Georgia Lazarou, Andrzej M Kierzek, Jacek Sroka
          </td>
          <td>2024-01-01</td>
          <td>Methods</td>
          <td>3</td>
          <td>3</td>
        </tr>

        <tr id="Virtual patient simulation is increasingly performed to support model‐based optimization of clinical trial designs or individualized dosing strategies. Quantitative pharmacological models typically incorporate individual‐level patient characteristics, or covariates, which enable the generation of virtual patient cohorts. The individual‐level patient characteristics, or covariates, used as input for such simulations should accurately reflect the values seen in real patient populations. Current methods often make unrealistic assumptions about the correlation between patient's covariates or require direct access to actual data sets with individual‐level patient data, which may often be limited by data sharing limitations. We propose and evaluate the use of copulas to address current shortcomings in simulation of patient‐associated covariates for virtual patient simulations for model‐based dose and trial optimization in clinical pharmacology. Copulas are multivariate distribution functions that can capture joint distributions, including the correlation, of covariate sets. We compare the performance of copulas to alternative simulation strategies, and we demonstrate their utility in several case studies. Our work demonstrates that copulas can reproduce realistic patient characteristics, both in terms of individual covariates and the dependence structure between different covariates, outperforming alternative methods, in particular when aiming to reproduce high‐dimensional covariate sets. In conclusion, copulas represent a versatile and generalizable approach for virtual patient simulation which preserve relationships between covariates, and offer an open science strategy to facilitate re‐use of patient data sets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e63eabaec5863d763b189e2ab364520b04d93eb" target='_blank'>Virtual Patient Simulation Using Copula Modeling</a></td>
          <td>
            L. Zwep, Tingjie Guo, Thomas Nagler, C. Knibbe, J. Meulman, J. Hasselt
          </td>
          <td>2023-11-10</td>
          <td>Clinical Pharmacology & Therapeutics</td>
          <td>5</td>
          <td>33</td>
        </tr>

        <tr id="Quantitative systems pharmacology (QSP) models aim to mechanistically describe disease progression and predict the effects of existing therapies or novel compounds on disease biomarkers or outcomes. For most drug development applications, it is important to predict not only the mean response to the intervention but also the inter-patient variability. In addition, given the complexity of QSP models and the sparsity of relevant human data, the parameters of QSP models are often not well-determined. One approach to overcome these limitations is to develop Virtual Populations (VPops), which allow for the exploration of parametric uncertainty and reproduce the variability in response to perturbation. Here we evaluate approaches to improve the efficiency of generating VPops. We aimed to generate these populations without sacrificing diversity of the Virtual Patients’ (VPs) pathophysiologies and phenotypes. To do this, we used our previously published approach (Allen, Rieger et al. 2016) together with two established algorithms (genetic algorithm and Metropolis-Hastings) as well as developing a novel approach, which we call “nested simulated annealing.” Each method improved our previously-published algorithm in at least one aspect. For example, all of the methods tested required significantly fewer plausible patients (PP), the precursors to VPs, to create a reasonable VPop, but we found there may be tradeoffs in terms of parametric and phenotypic diversity of the VPs. However, the improved methodologies introduced here may be appropriate in many applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12c7a13096165f576663b9d874fcb94307c8d358" target='_blank'>Improving the Generation and Selection of Virtual Populations in Quantitative Systems Pharmacology Models</a></td>
          <td>
            T. Rieger, R. Allen, Lukas Bystricky, Yuzhou Chen, Glen Wright Colopy, Yifan Cui, Angelica Gonzalez, Yifei Liu, Rebekah White, R. A. Everett, Harvey Thomas Banks, C. Musante
          </td>
          <td>2017-09-29</td>
          <td>bioRxiv</td>
          <td>68</td>
          <td>19</td>
        </tr>

        <tr id="In silico trial methods promise to improve the path to market for both medicines and medical devices, targeting the development of products, reducing reliance on animal trials, and providing adjunct evidence to bolster regulatory submissions. In silico trials are only as good as the simulated data which underpins them, consequently, often the most difficult challenge when creating robust in silico models is the generation of simulated measurements or even virtual patients that are representative of real measurements and patients. This article digests the current state of the art for generating synthetic patient data outside the context of in silico trials and outlines potential synergies to unlock the potential of in silico trials using virtual populations, by exploiting synthetic patient data to model effects on a more diverse and representative population. Synthetic data could be defined as artificial data that mimic the properties and relationships in real data. Recent advances in synthetic data generation methodologies have allowed for the generation of high-fidelity synthetic data that are both statistically and clinically, indistinguishable from real patient data. Other experimental work has demonstrated that synthetic data generation methods can be used for selective sample boosting of underrepresented groups. This article will provide a brief outline of synthetic data generation approaches and discuss how evaluation frameworks developed to assess synthetic data fidelity and utility could be adapted to evaluate the similarity of virtual patients used for in silico trials, to real patients. The article will then discuss outstanding challenges and areas for further research that would advance both synthetic data generation methods and in silico trial methods. Finally, the article will also provide a perspective on what evidence will be required to facilitate wider acceptance of in silico trials for regulatory evaluation of medicines and medical devices, including implications for post marketing safety surveillance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea76bd32d6c00bac3f7905b7ebe3bd94207a947" target='_blank'>The potential synergies between synthetic data and in silico trials in relation to generating representative virtual population cohorts</a></td>
          <td>
            P. Myles, Johan Ordish, A. Tucker
          </td>
          <td>2023-01-03</td>
          <td>Progress in Biomedical Engineering</td>
          <td>8</td>
          <td>29</td>
        </tr>

        <tr id="In silico trials are emerging as a valuable tool for improving both study design and outcomes. A key component of this process is the definition of a virtual cohort, i.e., a set of virtual patients with plausible physiological characteristics (covariates). Building on the NHANES study (2017-2020), we developed a statistical model to infer immunological parameters and a technique to generate a population of plausible immunological virtual patients. A thorough statistical analysis showed that the most appropriate model to represent our data is a conditional multivariate model. Compared to others, it is able to reproduce asymmetric distributions more accurately and is therefore more suitable in cases where there is no prior knowledge of the relationships between covariates. Our analysis also demonstrates the inter-variability and inter-dependence of the different covariates of interest. For example, age has a negative impact on the number of lymphocytes and, surprisingly, ethnicity has a minor influence on the other immunological covariates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d08776e7ce2d146338234d50ef05a97173ff4811" target='_blank'>A data-driven model for the generation of Virtual Cohorts</a></td>
          <td>
            Enrico Mastrostefano, P. Stolfi, F. Castiglione
          </td>
          <td>2021-12-09</td>
          <td>2021 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The decreasing cost-efficiency of drug development threatens to result in a severe shortage of innovative drugs, which may seriously compromise patient healthcare. This risk underlines the urgency to change the paradigm in clinical research. Here, we examine a novel concept of conducting virtual clinical trials for efficiently screening drug candidates, and for evaluating their prospects of being brought to the market successfully. The virtual clinical trials are carried out by using virtual patients (denoted Optimata Virtual Patients — OVPs). The OVP, a set of mathematical algorithms that describe the main pathological and physiological dynamic processes affected by the administered drug, has been shown to accurately predict docetaxel efficacy and safety in individual breast cancer patients. We report a test case in which virtual clinical trials have been conducted by using OVP populations for rescuing a discontinued oncology compound, ISIS-5132 (ISIS Pharmaceuticals Inc.). Our in silico study suggested that ISIS-5132 may be efficacious in combination with another drug, sunitinib malate (Sutent®, Pfizer Inc.), for the treatment of prostate cancer. The recommended combined treatment is predicted to result in a higher five-year Progression-Free Survival than monotherapy with either drug alone.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8172018701f941e71f32609e47c274a621dfd687" target='_blank'>Use of Virtual Patient Populations for Rescuing Discontinued Drug Candidates and for Reducing the Number of Patients in Clinical Trials</a></td>
          <td>
            M. Kleiman, Yael Sagi, N. Bloch, Z. Agur
          </td>
          <td>2009-09-01</td>
          <td>Alternatives to Laboratory Animals</td>
          <td>12</td>
          <td>35</td>
        </tr>

        <tr id="
 We present a new approach to predict overall survival (OS) in untested oncology trial designs, which does not require the use of clinical covariates. This approach is based on generating a virtual population from a quantitative systems pharmacology (QSP) model of cancer immunology, and linking this virtual population to real patients from previous clinical trials.
 QSP has emerged as a dominant paradigm in recent years for investigating disease mechanisms and allowing prediction of drug effects in silico. However, survival cannot be described mechanistically and intrinsically predicted from a QSP model. We develop a weakly supervised learning approach to impute labels of OS and censorship in the virtual population. This approach requires prior matching of the virtual patients to real patients on the basis of longitudinal tumor growth curves. The idea that there exists a predictive relationship between tumor dynamics and OS was motivated by previous work on tumor growth inhibition (TGI-OS) [Claret, L. et al., J. Clin. Onc., 2013]. In contrast to the TGI-OS framework, we rely solely on simulated QSP dynamical variables for predicting survival. This allows us to derive OS predictions for trial designs different than the ones used for model development.
 Data from 5 clinical trials for atezolizumab in NSCLC (BIRCH, FIR, OAK, POPLAR, and IMpower110, total N = 1641) were used to link survival labels to virtual patients. 90% of the data was used for training, and 10% was held out for model validation. The imputed OS labels were used to train a log-normal accelerated failure time model on the training data, and predictions of survival in the test data were in good agreement with a Kaplan-Meier estimate of the clinical survival. The model predicted a median OS of 471 days (95% CI 453-490) compared to the observed 475 days, (95% CI 440-517). The rate of change of tumor size under treatment in the QSP model was the most predictive feature of survival, with a hazard ratio of 1.79 between 95th percentile and median of the dynamical range (95% CI 1.69-1.81). Model variables related to tumor antigens, cytotoxic cell death, and T cell dynamics were also relevant to the prediction of survival.
 This work provides the first example of generating and validating predictions of OS without using covariates from actual clinical trials. We intend to expand this methodology across different trial designs and different subpopulations. For example, our approach could be used to estimate hazard ratios between different treatments or combination therapies, as well as for patients grouped by PDL1 expression or by line of therapy. While this work only considered overall survival as an endpoint, our approach can potentially be extended to endpoints like progression-free-survival.
 Citation Format: Matthew West, Kenta Yoshida, Jiajie Yu, Vincent Lemaire. A treatment-agnostic approach to predict patient survival from virtual clinical trials [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2022; 2022 Apr 8-13. Philadelphia (PA): AACR; Cancer Res 2022;82(12_Suppl):Abstract nr 1923.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51292f282afd7a311bc6b9102646067beaa0d67b" target='_blank'>Abstract 1923: A treatment-agnostic approach to predict patient survival from virtual clinical trials</a></td>
          <td>
            Matthew West, Kenta Yoshida, Jiajie Yu, V. Lemaire
          </td>
          <td>2022-06-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Quantitative systems pharmacology (QSP) places an emphasis on dynamic systems modeling, incorporating considerations from systems biology modeling and pharmacodynamics. The goal of QSP is often to quantitatively predict the effects of clinical therapeutics, their combinations, and their doses on clinical biomarkers and endpoints. In order to achieve this goal, strategies for incorporating clinical data into model calibration are critical. Virtual population (VPop) approaches facilitate model calibration while faced with challenges encountered in QSP model application, including modeling a breadth of clinical therapies, biomarkers, endpoints, utilizing data of varying structure and source, capturing observed clinical variability, and simulating with models that may require more substantial computational time and resources than often found in pharmacometrics applications. VPops are frequently developed in a process that may involve parameterization of isolated pathway models, integration into a larger QSP model, incorporation of clinical data, calibration, and quantitative validation that the model with the accompanying, calibrated VPop is suitable to address the intended question or help with the intended decision. Here, we introduce previous strategies for developing VPops in the context of a variety of therapeutic and safety areas: metabolic disorders, drug-induced liver injury, autoimmune diseases, and cancer. We introduce methodological considerations, prior work for sensitivity analysis and VPop algorithm design, and potential areas for future advancement. Finally, we give a more detailed application example of a VPop calibration algorithm that illustrates recent progress and many of the methodological considerations. In conclusion, although methodologies have varied, VPop strategies have been successfully applied to give valid clinical insights and predictions with the assistance of carefully defined and designed calibration and validation strategies. While a uniform VPop approach for all potential QSP applications may be challenging given the heterogeneity in use considerations, we anticipate continued innovation will help to drive VPop application for more challenging cases of greater scale while developing new rigorous methodologies and metrics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68337aa80c1390201d9ea07d9ae681ebb37bfaa5" target='_blank'>Virtual Populations for Quantitative Systems Pharmacology Models.</a></td>
          <td>
            Yougan Cheng, R. Straube, Abed Alnaif, Lu Huang, T. Leil, B. Schmidt
          </td>
          <td>2021-08-24</td>
          <td>Methods in molecular biology</td>
          <td>22</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e7d956fccc7015105d96d6e72ec20565cd792ad" target='_blank'>Accelerating robust plausible virtual patient cohort generation by substituting ODE simulations with parameter space mapping</a></td>
          <td>
            Thibaud Derippe, Sylvain Fouliard, X. Declèves, D. Mager
          </td>
          <td>2022-10-31</td>
          <td>Journal of Pharmacokinetics and Pharmacodynamics</td>
          <td>4</td>
          <td>42</td>
        </tr>

        <tr id="Artificial Intelligence has the potential to reshape the landscape of clinical trials through innovative applications, with a notable advancement being the emergence of synthetic patient generation. This process involves simulating cohorts of virtual patients that can either replace or supplement real individuals within trial settings. By leveraging synthetic patients, it becomes possible to eliminate the need for obtaining patient consent and creating control groups that mimic patients in active treatment arms. This method not only streamlines trial processes, reducing time and costs but also fortifies the protection of sensitive participant data. Furthermore, integrating synthetic patients amplifies trial efficiency by expanding the sample size. These straightforward and cost‐effective methods also enable the development of personalized subject‐specific models, enabling predictions of patient responses to interventions. Synthetic data holds great promise for generating real‐world evidence in clinical trials while upholding rigorous confidentiality standards throughout the process. Therefore, this study aims to demonstrate the applicability and performance of these methods in the context of onco‐hematological research, breaking through the theoretical and practical barriers associated with the implementation of artificial intelligence in medical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bdcab605855b7b039a99d4d58c45d8db9943a32" target='_blank'>Unlocking the potential of synthetic patients for accelerating clinical trials: Results of the first GIMEMA experience on acute myeloid leukemia patients</a></td>
          <td>
            A. Piciocchi, Marta Cipriani, Monica Messina, G. Marconi, Valentina Arena, S. Soddu, E. Crea, Maria Valeria Feraco, Marco Ferrante, E. La Sala, P. Fazi, F. Buccisano, M. Voso, Giovanni Martinelli, Adriano Venditti, M. Vignetti
          </td>
          <td>2024-03-15</td>
          <td>EJHaem</td>
          <td>1</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/139ced151db3b76b6009a58caa2164b290cd7efa" target='_blank'>Distribution-based sub-population selection (DSPS): A method for in-silico reproduction of clinical trials outcomes.</a></td>
          <td>
            Mohammadreza Ganji, Anas El Fathi, Chiara Fabris, Dayu Lv, Boris P Kovatchev, Marc D. Breton
          </td>
          <td>2024-08-30</td>
          <td>Computers in biology and medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Rare diseases, affecting millions globally, present significant drug development challenges. This is due to the limited patient populations and the unique pathophysiology of these diseases, which can make traditional clinical trial designs unfeasible. Quantitative Systems Pharmacology (QSP) models offer a promising approach to expedite drug development, particularly in rare diseases. QSP models provide a mechanistic representation of the disease and drug response in virtual patients that can complement routinely applied empirical modeling and simulation approaches. QSP models can generate digital twins of actual patients and mechanistically simulate the disease progression of rare diseases, accounting for phenotypic heterogeneity. QSP models can also support drug development in various drug modalities, such as gene therapy. Impactful QSP models case studies are presented here to illustrate their value in supporting various aspects of drug development in rare indications. As these QSP model applications continue to mature, there is a growing possibility that they could be more widely integrated into routine drug development steps. This integration could provide a robust framework for addressing some of the inherent challenges in rare disease drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/501aa60f69949c250c5052e0be685b26e600d873" target='_blank'>Quantitative Systems Pharmacology Models: Potential Tools for Advancing Drug Development for Rare Diseases.</a></td>
          <td>
            Susana Neves-Zaph, C. Kaddi
          </td>
          <td>2024-09-28</td>
          <td>Clinical pharmacology and therapeutics</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="The pharmaceutical industry has increasingly adopted model-informed drug discovery and development (MID3) to enhance productivity in drug discovery and development. Quantitative systems pharmacology (QSP), which integrates drug action mechanisms and disease complexities to predict clinical endpoints and biomarkers is central to MID3. QSP modeling has proven successful in metabolic and cardiovascular diseases and has expanded into oncology, immunotherapy, and infectious diseases. Despite its benefits, QSP model validation through clinical trial simulations using virtual patients (VPs) is challenging because of parameter variability and high computational costs. To address these challenges, this study proposes a hybrid method that combines Bayesian optimization with machine learning for efficient parameter screening. Our approach achieved an acceptance rate of 27.5% in QSP simulations, which is in sharp contrast with the 2.5% rate of conventional random search methods, indicating more than 10-fold improvement in efficiency. By facilitating faster and more diverse VPs generation, this method promises to advance clinical trial simulations and accelerate drug development in pharmaceutical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b752385c900dbfc4f30d08d8dea12e0ef790e497" target='_blank'>Accelerating virtual patient generation with a Bayesian optimization and machine learning surrogate model.</a></td>
          <td>
            Hiroaki Iwata, Ryuta Saito
          </td>
          <td>2024-12-04</td>
          <td>CPT: pharmacometrics & systems pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite a growing body of knowledge on the mechanisms underlying the onset and progression of cancer, treatment success rates in oncology are at best modest. Current approaches use statistical methods that fail to embrace the inherent and expansive complexity of the tumor/patient/drug interaction. Computational modeling, in particular mechanistic modeling, has the power to resolve this complexity. Using fundamental knowledge on the interactions occurring between the components of a complex biological system, large-scale in silico models with predictive capabilities can be generated. Here, we describe how mechanistic virtual patient models, based on systematic molecular characterization of patients and their diseases, have the potential to shift the theranostic paradigm for oncology, both in the fields of personalized medicine and targeted drug development. In particular, we highlight the mechanistic modeling platform ModCell™ for individualized prediction of patient responses to treatment, emphasizing modeling techniques and avenues of application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed989fd87c7141bf188cf54581345f04a17c5b1" target='_blank'>Predictive Modeling of Drug Treatment in the Area of Personalized Medicine</a></td>
          <td>
            L. Ogilvie, C. Wierling, T. Kessler, H. Lehrach, B. Lange
          </td>
          <td>2015-12-06</td>
          <td>Cancer Informatics</td>
          <td>21</td>
          <td>124</td>
        </tr>

        <tr id="Genomics has multiplied the number of targets for new therapeutic interventions, but this has not yet lead to a marked increase of pharma pipeline outputs. The complexity of protein function in higher order biological systems is often underestimated. Translation from in vitro and in vivo results to the human setting frequently fails due to unforeseen toxicity and efficacy issues. Biosimulation addresses these issues by capturing the complex dynamics of interacting molecules and cells in mechanistic, predictive models. A central concept is that of the virtual patient, an encapsulation of a specific pathophysiological behaviour in a biosimulation model. The authors describe how virtual patients are being used in target identification, target validation and clinical development, and discuss challenges for the acceptance of biosimulation methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0426a99b064475bbd5dc9d7ddf07d88d49e79dc5" target='_blank'>Target validation in silico: does the virtual patient cure the pharma pipeline?</a></td>
          <td>
            W. Alkema, T. Rullmann, A. van Elsas
          </td>
          <td>2006-09-18</td>
          <td>Expert Opinion on Therapeutic Targets</td>
          <td>10</td>
          <td>29</td>
        </tr>

        <tr id="Every patient is different - his/her genomes, environment, disease history and exposure to drugs. Tumours, in particular, are often heterogeneous in their genetic make-up and their response to drugs, both within and between samples. Classic clinical trials basically ignore this complexity or, as in stratified medicine, attempt to reduce it to an analysis of a small number of still enormously heterogeneous patient groups. Medicine, however, is not the only area in which we are faced with such complex ‘n = 1' (every individual case is different) situations. The weather we experience today, characterised by tens of terabytes of measurement data, has never occurred before and will never occur again. Similar to the situation in medicine, we cannot predict the development of today's weather by looking for identical weather conditions in the past, and we cannot, in real life, test drugs for every individual patient in clinical trials with large numbers of biologically identical patient replicas. We can, however, do it with the help of models duplicating the ‘n = 1' situation on the computer, an approach which will also have to be used in both patient treatment and prevention as well as drug development in the future if we do not want to continue to make dangerous and expensive mistakes in real life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46560a509d53dbfd4f1744a8621d0a1d25afa380" target='_blank'>Virtual Clinical Trials, an Essential Step in Increasing the Effectiveness of the Drug Development Process</a></td>
          <td>
            H. Lehrach
          </td>
          <td>2015-11-05</td>
          <td>Public Health Genomics</td>
          <td>19</td>
          <td>124</td>
        </tr>

        <tr id="In the context of in silico clinical trials, mechanistic computer models for pathophysiology and pharmacology (here Quantitative Systems Pharmacology models, QSP) can greatly support the decision making for drug candidates and elucidate the (potential) response of patients to existing and novel treatments. These models are built on disease mechanisms and then parametrized using (clinical study) data. Clinical variability among patients is represented by alternative model parameterizations, called virtual patients. Despite the complexity of disease modeling itself, using individual patient data to build these virtual patients is particularly challenging given the high-dimensional, potentially sparse and noisy clinical trial data. In this work, we investigate the applicability of simulation-based inference (SBI), an advanced probabilistic machine learning approach, for virtual patient generation from individual patient data and we develop and evaluate the concept of nearest patient fits (SBI NPF), which further enhances the fitting performance. At the example of rheumatoid arthritis where prediction of treatment response is notoriously difficult, our experiments demonstrate that the SBI approaches can capture large inter-patient variability in clinical data and can compete with standard fitting methods in the field. Moreover, since SBI learns a probability distribution over the virtual patient parametrization, it naturally provides the probability for alternative parametrizations. The learned distributions allow us to generate highly probable alternative virtual patient populations for rheumatoid arthritis, which could potentially enhance the assessment of drug candidates if used for in silico trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041a1690a5887dbab750003688e4c5220ead4dcd" target='_blank'>Building virtual patients using simulation-based inference</a></td>
          <td>
            Nathalie Paul, Venetia Karamitsou, Clemens Giegerich, Afshin Sadeghi, Moritz Lücke, Britta Wagenhuber, Alexander Kister, Markus Rehberg
          </td>
          <td>2024-09-12</td>
          <td>Frontiers in Systems Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2acb8715ae3fb0381f9dcb1cf1c649068ba50d2a" target='_blank'>Generation and application of avatars in pharmacometric modelling</a></td>
          <td>
            E. Chasseloup, A. Hooker, M. Karlsson
          </td>
          <td>2023-07-24</td>
          <td>Journal of Pharmacokinetics and Pharmacodynamics</td>
          <td>5</td>
          <td>29</td>
        </tr>

        <tr id="Abstract The Simcyp Simulator is a software platform widely used in the pharmaceutical industry to conduct stochastic physiologically‐based pharmacokinetic (PBPK) modeling. This approach has the advantage of combining routinely generated in vitro data on drugs and drug products with knowledge of biology and physiology parameters to predict a priori potential pharmacokinetic changes in absorption, distribution, metabolism, and excretion for populations of interest. Combining such information with pharmacodynamic knowledge of drugs enables planning for potential dosage adjustment when clinical studies are feasible. Although the conduct of dedicated clinical studies in some patient groups (e.g., with hepatic or renal diseases) is part of the regulatory path for drug development, clinical studies for all permutations of covariates potentially affecting pharmacokinetics are impossible to perform. The role of PBPK in filling the latter gap is becoming more appreciated. This tutorial describes the different input parameters required for the creation of a virtual population giving robust predictions of likely changes in pharmacokinetics. It also highlights the considerations needed to qualify the models for such contexts of use. Two case studies showing the step‐by‐step development and application of population files for obese or morbidly obese patients and individuals with Crohn's disease are provided as the backbone of our tutorial to give some hands‐on and real‐world examples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0deaff3699d43a1c8309f73d24f6a64943812bbd" target='_blank'>A guide to developing population files for physiologically‐based pharmacokinetic modeling in the Simcyp Simulator</a></td>
          <td>
            Liam Curry, Sarah Alrubia, Frederic Y Bois, Ruth Clayton, Eman El-Khateeb, Trevor N Johnson, Muhammad Faisal, S. Neuhoff, Kris Wragg, A. Rostami-Hodjegan
          </td>
          <td>2024-07-18</td>
          <td>CPT: Pharmacometrics & Systems Pharmacology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba7e80c44801604632936459d90802dc703718e" target='_blank'>Simulated Clinical Trials: Principle, Good Practices, and Focus on Virtual Patients Generation</a></td>
          <td>
            N. Savy, S. Savy, S. Andrieu, S. Marque
          </td>
          <td>2015-09-21</td>
          <td></td>
          <td>2</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4751af0866d1914929be7f8bf3c02c58b1be6e29" target='_blank'>The “Virtual Patient” system: modeling cancer using deep sequencing technologies for personalized cancer treatment</a></td>
          <td>
            Alexander Kühn, H. Lehrach
          </td>
          <td>2012-01-17</td>
          <td>Journal für Verbraucherschutz und Lebensmittelsicherheit</td>
          <td>7</td>
          <td>124</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29fe7d063e5583538b410e0525aeb528a3575fe4" target='_blank'>The “Virtual Patient” system: modeling cancer using deep sequencing technologies for personalized cancer treatment</a></td>
          <td>
            Alexander Kühn, H. Lehrach
          </td>
          <td>2012-01-17</td>
          <td>Journal für Verbraucherschutz und Lebensmittelsicherheit</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Introduction: Effective tight glycemic control (TGC) can improve outcomes, but is difficult to achieve. In silico virtual patients and trials offer significant advantages in cost, time and safety for designing effective TGC protocols. However, no such method has been fully validated. This study tests 2 matched cohorts from the Glucontrol trial treated with different protocols. The goal is to validate the ability of in-silico virtual patient models and methods to accurately predict patientspecific and clinical trial glycemic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c81677e866e5027a3ab411f0a7680c25bdadb7b" target='_blank'>Validation of a Virtual Patient and Virtual Trials Method for Accurate Prediction of TGC Protocol Performance</a></td>
          <td>
            F. Suhaimi, A. Compte, S. Penning, C. Pretty, J. Preiser, G. Shaw, T. Desaive, J. Chase
          </td>
          <td>2011-03-01</td>
          <td>Critical Care</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Drug development in oncology commonly exploits the tools of molecular biology to gain therapeutic benefit through reprograming of cellular responses. In immuno‐oncology (IO) the aim is to direct the patient’s own immune system to fight cancer. After remarkable successes of antibodies targeting PD1/PD‐L1 and CTLA4 receptors in targeted patient populations, the focus of further development has shifted toward combination therapies. However, the current drug‐development approach of exploiting a vast number of possible combination targets and dosing regimens has proven to be challenging and is arguably inefficient. In particular, the unprecedented number of clinical trials testing different combinations may no longer be sustainable by the population of available patients. Further development in IO requires a step change in selection and validation of candidate therapies to decrease development attrition rate and limit the number of clinical trials. Quantitative systems pharmacology (QSP) proposes to tackle this challenge through mechanistic modeling and simulation. Compounds’ pharmacokinetics, target binding, and mechanisms of action as well as existing knowledge on the underlying tumor and immune system biology are described by quantitative, dynamic models aiming to predict clinical results for novel combinations. Here, we review the current QSP approaches, the legacy of mathematical models available to quantitative clinical pharmacologists describing interaction between tumor and immune system, and the recent development of IO QSP platform models. We argue that QSP and virtual patients can be integrated as a new tool in existing IO drug development approaches to increase the efficiency and effectiveness of the search for novel combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe4c7719ed81361b93340d169c24788387a66ae" target='_blank'>Quantitative Systems Pharmacology Approaches for Immuno‐Oncology: Adding Virtual Patients to the Development Paradigm</a></td>
          <td>
            Vijayalakshmi Chelliah, Georgia Lazarou, Sumit Bhatnagar, J. Gibbs, M. Nijsen, Avijit Ray, B. Stoll, R. Thompson, A. Gulati, S. Soukharev, A. Yamada, Jared Weddell, Hiroyuki Sayama, Masayo Oishi, S. Wittemer-Rump, C. Patel, Christoph Niederalt, R. Burghaus, Christian Scheerans, J. Lippert, S. Kabilan, I. Kareva, N. Belousova, A. Rolfe, A. Zutshi, M. Chenel, Filippo Venezia, Sylvain Fouliard, H. Oberwittler, A. Scholer-Dahirel, H. Lelièvre, D. Bottino, Sabrina C. Collins, Hoa Q. Nguyen, Haiqing Wang, Tomoki Yoneyama, Andy Z. X. Zhu, Piet H. Graaf, A. Kierzek
          </td>
          <td>2020-07-19</td>
          <td>Clinical Pharmacology and Therapeutics</td>
          <td>51</td>
          <td>33</td>
        </tr>

        <tr id="We thank Duffull and Gulati1 for highlighting pitfalls in generating virtual patients for quantitative systems pharmacology (QSP) models. The authors demonstrate how some parameter selections lead to nonphysiological model outcomes and conclude that virtual patients are hence unsuitable in these instances. Our prior work2,3 on virtual populations avoids Duffull and Gulati’s main critique. In our workflow, the application of prior knowledge into the virtual patient generation algorithm specifically “throwsout” nonphysiological solutions. To demonstrate how to avoid some issues raised by Duffull and Gulati, we modified their first example to illustrate our workflow (Figure 1a). We generated simulated data representative of the prior information that Duffull and Gulati referred to as an “acceptable profile.” We introduced variability typical of similar clinical data.4 We then generated “plausible patients” by a MetropolisHastings algorithm.3 Because the plausible population closely matches the “data,” no further selection step was necessary for this exercise (Figure 1b,c). Following this process, we generated 2500 parameter sets with a range of values that give rise to model simulations that are plausible.5 Virtual populations are not a panacea for dealing with the uncertainty in QSP models. For example, this approach">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c333c73752c183687e9c1fa91425a97e976968e6" target='_blank'>Modeling is data driven: Use it for successful virtual patient generation</a></td>
          <td>
            T. Rieger, R. Allen, C. Musante
          </td>
          <td>2021-05-01</td>
          <td>CPT: Pharmacometrics & Systems Pharmacology</td>
          <td>10</td>
          <td>18</td>
        </tr>

        <tr id="Platform trials have gained a lot of attention recently as a possible remedy for time-consuming classical two-arm randomized controlled trials, especially in early phase drug development. This short article illustrates how to use the CohortPlat R package to simulate a cohort platform trial, where each cohort consists of a combination treatment and the respective monotherapies and standard-of-care. The endpoint is always assumed to be binary. The package oﬀers extensive ﬂexibility with respect to both platform trial trajectories, as well as treatment eﬀect scenarios and decision rules. As a special feature, the package provides a designated function for running multiple such simulations eﬃciently in parallel and saving the results in a concise manner. Many illustrations of code usage are provided.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03f5b7ea45801f8b90bc51a59569b6bff86f7b6a" target='_blank'>CohortPlat: Simulation of cohort platform trials investigating combination therapies</a></td>
          <td>
            E. Meyer, P. Mesenbrink, C. Dunger-Baldauf, E. Glimm, Franz Konig
          </td>
          <td>2022-02-04</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0dcb18bbc6d5ceb810835b9d0da8e2d97e5426" target='_blank'>A critical review on approaches to generate and validate virtual population for physiologically based pharmacokinetic models: Methodologies, case studies and way forward.</a></td>
          <td>
            Mahendra Chougule, Sivacharan Kollipara, Smritilekha Mondal, Tausif Ahmed
          </td>
          <td>2024-10-08</td>
          <td>European journal of clinical pharmacology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/febb53ab2da57fe9d114bb8d7dd94cbd71e7a324" target='_blank'>Simulation and prediction of in vivo drug metabolism in human populations from in vitro data</a></td>
          <td>
            A. Rostami-Hodjegan, G. Tucker
          </td>
          <td>2007-02-01</td>
          <td>Nature Reviews Drug Discovery</td>
          <td>519</td>
          <td>67</td>
        </tr>

        <tr id="The simulation of therapeutic models and clinical trial simulation have recently attracted attention as emerging techniques for developing new active molecules and the exploration of possible clinical trial results. Such approaches have benefited from fundamental progress in the development of 'in silico' models, as well as progress in nonlinear mixed-effect pharmacokinetic-pharmacodynamic models. Mixing the two approaches allows simulation of 'virtual' patients, who receive virtual treatments or placebo. These have various uses, such as proof of concept, decision analysis or experimental design optimisation. Also, the effect of departures from protocol on clinical trial results can easily be evaluated by the use of simulation. This technique is now implemented by the pharmaceutical industry for optimising phase II and III experimental designs when a good biomarker or a clinical outcome model is available, but the use of an in silico therapeutic model as a proof of concept is only just beginning. In order to see such methodologies used more widely in drug development, multidisciplinary efforts need to be initiated, new modelling and simulation tools developed, and sound modelling and simulation practice documents need to be adopted. A reduction in the number of failed clinical development projects, the number of negative phase II and III clinical trials, or in just their cost and duration, are among the expected benefits of modelling and simulation in clinical drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39da6e28fb4669bf26cf076f62d6e482f1724408" target='_blank'>Clinical trial simulation in drug development.</a></td>
          <td>
            P. Girard, M. Cucherat, D. Guez
          </td>
          <td>2004-05-01</td>
          <td>Therapie</td>
          <td>11</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93a1c495502edb0ff0d0e5191347712a3851108a" target='_blank'>In silico cancer immunotherapy trials uncover the consequences of therapy-specific response patterns for clinical trial design and outcome</a></td>
          <td>
            J. H. Creemers, Ankur Ankan, K. Roes, Gijs Schröder, N. Mehra, C. Figdor, I. D. de Vries, J. Textor
          </td>
          <td>2023-04-24</td>
          <td>Nature Communications</td>
          <td>12</td>
          <td>110</td>
        </tr>

        <tr id="Innovations in information and communication technology infuse all branches of science, including life sciences. Nevertheless, healthcare is historically slow in adopting technological innovation, compared with other industrial sectors. In recent years, new approaches in modelling and simulation have started to provide important insights in biomedicine, opening the way for their potential use in the reduction, refinement and partial substitution of both animal and human experimentation. In light of this evidence, the European Parliament and the United States Congress made similar recommendations to their respective regulators to allow wider use of modelling and simulation within the regulatory process. In the context of in silico medicine, the term 'in silico clinical trials' refers to the development of patient-specific models to form virtual cohorts for testing the safety and/or efficacy of new drugs and of new medical devices. Moreover, it could be envisaged that a virtual set of patients could complement a clinical trial (reducing the number of enrolled patients and improving statistical significance), and/or advise clinical decisions. This article will review the current state of in silico clinical trials and outline directions for a full-scale adoption of patient-specific modelling and simulation in the regulatory evaluation of biomedical products. In particular, we will focus on the development of vaccine therapies, which represents, in our opinion, an ideal target for this innovative approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b6497e4241f0cfe1dd6e439761557d3bd305224" target='_blank'>In silico clinical trials: concepts and early adoptions</a></td>
          <td>
            F. Pappalardo, G. Russo, Flora Musuamba Tshinanu, M. Viceconti
          </td>
          <td>2019-09-27</td>
          <td>Briefings in bioinformatics</td>
          <td>175</td>
          <td>61</td>
        </tr>

        <tr id="Abstract Simulation studies are promising in medical research in particular to improve drug development. For instance, one can aim to develop In Silico Clinical Trial in order to challenge trial’s design parameters in terms of feasibility and probability of success of the trial. Approaches based on agent-based models draw on a particularly useful framework to simulate patients evolution. In this paper, an approach based on agent-based modeling is described and discussed in the context of medical research. An R-vine copula model is used to represent the multivariate distribution of the data. A baseline data cohort can then be simulated and execution models can be developed to simulate the evolution of patients. R-vine copula models are very flexible tools which allow researchers to consider different marginal distributions than the ones observed in the data. It is then possible to perform data augmentation to explore a new population by simulating baseline data which are slightly different than those of the original population. A simulation study illustrates the efficiency of copula modeling to generate data according to specific marginal distributions but also highlights difficulties inherent to data augmentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a56bf43595a89bffcb774161827c318b7bbff1dc" target='_blank'>Agent-based modeling in medical research, virtual baseline generator and change in patients’ profile issue</a></td>
          <td>
            Philippe Saint-Pierre, N. Savy
          </td>
          <td>2023-07-11</td>
          <td>The International Journal of Biostatistics</td>
          <td>2</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/504946670c9db85d2d1af84e26d75275095f3099" target='_blank'>Next-generation, personalised, model-based critical care medicine: a state-of-the art review of in silico virtual patient models, methods, and cohorts, and how to validation them</a></td>
          <td>
            J. Chase, J. Preiser, Jennifer L. Dickson, A. Pironet, Y. Chiew, C. Pretty, G. Shaw, B. Benyó, K. Moeller, Soroush Safaei, M. Tawhai, P. Hunter, T. Desaive, Geoff nz
          </td>
          <td>2018-02-20</td>
          <td>BioMedical Engineering OnLine</td>
          <td>148</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78aab3c52b53afb2339a33752fa5beb2463fd5c6" target='_blank'>In Silico Modeling and Simulation to Guide Bioequivalence Testing for Oral Drugs in a Virtual Population</a></td>
          <td>
            Fan Zhang, Ranran Jia, Huitao Gao, Xiaofei Wu, Bo Liu, Hongyun Wang
          </td>
          <td>2021-06-30</td>
          <td>Clinical Pharmacokinetics</td>
          <td>13</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ed575610e38950d5d513fc58685891268c03e98" target='_blank'>Virtual clinical trial based on outcome modeling with iteratively redistributed extrapolation data</a></td>
          <td>
            Kohei Oguma, T. Magome, M. Someya, T. Hasegawa, K. Sakata
          </td>
          <td>2023-03-22</td>
          <td>Radiological Physics and Technology</td>
          <td>2</td>
          <td>29</td>
        </tr>

        <tr id="Every human is unique. We differ in our genomes, environment, behavior, disease history, and past and current medical treatment—a complex catalog of differences that often leads to variations in the way each of us responds to a particular therapy. We argue here that true personalization of drug therapies will rely on “virtual patient” models based on a detailed characterization of the individual patient by molecular, imaging, and sensor techniques. The models will be based, wherever possible, on the molecular mechanisms of disease processes and drug action but can also expand to hybrid models including statistics/machine learning/artificial intelligence-based elements trained on available data to address therapeutic areas or therapies for which insufficient information on mechanisms is available. Depending on the disease, its mechanisms, and the therapy, virtual patient models can be implemented at a fairly high level of abstraction, with molecular models representing cells, cell types, or organs relevant to the clinical question, interacting not only with each other but also the environment. In the future, “virtual patient/in-silico self” models may not only become a central element of our health care system, reducing otherwise unavoidable mistakes and unnecessary costs, but also act as “guardian angels” accompanying us through life to protect us against dangers and to help us to deal intelligently with our own health and wellness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a98a0af62b1dd8f00e3237aa56f67957e8749443" target='_blank'>Omics approaches to individual variation: modeling networks and the virtual patient</a></td>
          <td>
            H. Lehrach
          </td>
          <td>2016-09-01</td>
          <td>Dialogues in Clinical Neuroscience</td>
          <td>3</td>
          <td>124</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1513b961c4758fc7763b52ebec6fd2f4625c2b01" target='_blank'>Simulating complex patient populations with hierarchical learning effects to support methods development for post-market surveillance</a></td>
          <td>
            Sharon E. Davis, Henry Ssemaganda, Jejo D. Koola, Jialin Mao, D. Westerman, T. Speroff, Usha Govindarajulu, C. Ramsay, A. Sedrakyan, L. Ohno-Machado, F. Resnic, M. Matheny
          </td>
          <td>2023-04-11</td>
          <td>BMC Medical Research Methodology</td>
          <td>1</td>
          <td>63</td>
        </tr>

  </tbody>
  <tfoot>
    <tr>
        <th>Abstract</th>
        <th>Title</th>
        <th>Authors</th>
        <th>Publication Date</th>
        <th>Journal/Conference</th>
        <th>Citation count</th>
        <th>Highest h-index</th>
    </tr>
  </tfoot>
  </table>
  </p>

</body>

<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: false,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '5%', targets: 0 },
        { width: '25%', targets: 1 },
        { width: '20%', targets: 2 },
        { width: '10%', targets: 3 },
        { width: '20%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);

  var table = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.top", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>